{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "b0a1bc159b41917a",
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "source": [
    "# Database pipeline for quantification of E.coli from metagenomic data\n",
    "\n",
    "## What's in this notebook?\n",
    "\n",
    "This notebook implements the pipeline that we used to generate a JSON configuration file (`ecoli.json`) for chronostrain in our paper. Our database encodes marker-seed aligning sequences from _E.coli_, as well as all sequences of >75% similarity known to us at the family level.\n",
    "\n",
    "In principle, the JSON file is the only thing needed to run ChronoStrain; but this notebook has some other by-products:\n",
    "\n",
    "1. A repository of chromosomal assemblies for the Enterobacteriaceae family, with a TSV-formatted index (`index.tsv`)\n",
    "2. A repository of marker seed sequences and associated gene names.\n",
    "3. (Optional) A binary-stored instance of the JSON database, readable by `chronostrain`. \n",
    "\n",
    "If the sequence records listed in the JSON file do not exist (the directory is specificable by a configuration file `chronostrain.ini`), then ChronoStrain will attempt to download it by searching the ID on NCBI. This notebook creates symbolic links to the sequences found in the chromosomal assembly index (byproduct #1) to skip this process.\n",
    "After these sequences are downloaded, the database is loaded by extracting subsequences from the FASTA records; this is a very I/O-bound task and thus the pre-loaded binary (byproduct #3) helps skip the initialization.\n",
    "\n",
    "## Prerequisites\n",
    "\n",
    "### Software requirements\n",
    "\n",
    "We recommend using a `conda` environment for this notebook, with `ipywidgets` installed and updated.\n",
    "This notebook requires that the following software is installed.\n",
    "- chronostrain (python>=3.10, the basic recipe `conda_basic.yml` or the full recipe `conda_full.yml`)\n",
    "- primersearch (http://emboss.open-bio.org/, https://anaconda.org/bioconda/emboss)\n",
    "- dashing2 (2023 Baker and Langmead: https://github.com/dnbaker/dashing2)\n",
    "\n",
    "### Hardware requirements\n",
    " \n",
    "None of the operations of this notebook requires a GPU. \n",
    "As of Feb 2023, we estimate that the contents of this notebook requires ~60 GB of hard disk space. At the time that we ran this pipeline, the catalog of chromosomal assemblies totalled 38.8 GB; other files (such as the BLAST database, marker seeds and chronostrain-specific byproducts) totalled 17.2 GB with a peak of ~60 GB when accounting for temporary files."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a05bf446-8de2-44e9-91b6-3c19304d05fc",
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "source": [
    "## File paths and environment variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "5aa56187-b0f7-4db9-81c9-c52b8c3894aa",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-10-06T14:33:08.014259960Z",
     "start_time": "2023-10-06T14:33:07.966493286Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/youn/work/chronostrain/examples/database\n",
      "env: TARGET_TAXA=Enterobacteriaceae\n",
      "env: NCBI_REFSEQ_DIR=/mnt/e/ecoli_db/ref_genomes\n",
      "env: REFSEQ_INDEX=/mnt/e/ecoli_db/ref_genomes/index.tsv\n",
      "env: PATH=/usr/bin:/home/youn/mambaforge/envs/chronostrain2/bin:/home/youn/work/bin\n"
     ]
    }
   ],
   "source": [
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "\n",
    "\"\"\" ============================================ EDIT THESE SETTINGS BASED ON USER'S CHOICE. ============================================ \"\"\"\n",
    "\"\"\" RefSeq catalog settings\"\"\"\n",
    "TARGET_DIR = Path(\"/mnt/e/ecoli_db\")  # the base directory for everything else.\n",
    "TARGET_TAXA = \"Enterobacteriaceae\"  # the taxonomic identifier to download. Can be species, genus or even family.\n",
    "NCBI_REFSEQ_DIR = TARGET_DIR / \"ref_genomes\"\n",
    "REFSEQ_INDEX = NCBI_REFSEQ_DIR / \"index.tsv\"\n",
    "\n",
    "\"\"\" RefSeq BLAST database \"\"\"\n",
    "BLAST_DB_DIR = TARGET_DIR / \"blast_db\"  # The location of the Blast DB that you wish to create using RefSeq indices (they will be downloaded by this notebook).\n",
    "BLAST_DB_NAME = \"Enterobacteriaceae_refseq\"  # Blast DB to create.\n",
    "\n",
    "\"\"\" Marker seeds \"\"\"\n",
    "MARKER_SEED_DIR = TARGET_DIR / \"marker_seeds\"\n",
    "MARKER_SEED_INDEX = MARKER_SEED_DIR / \"marker_seed_index.tsv\"\n",
    "\n",
    "\"\"\" chronostrain-specific settings \"\"\"\n",
    "NUM_CORES = 8  # number of cores to use (e.g. for blastn)\n",
    "MIN_PCT_IDTY = 75  # accept BLAST hits as markers above this threshold.\n",
    "CHRONOSTRAIN_DB_DIR = TARGET_DIR / \"chronostrain_files\"  # The directory to use for chronostrain's database files.\n",
    "CHRONOSTRAIN_TARGET_JSON = CHRONOSTRAIN_DB_DIR / \"ecoli.json\"  # the desired final product.\n",
    "CHRONOSTRAIN_TARGET_CLUSTERS = CHRONOSTRAIN_DB_DIR / \"ecoli.clusters.txt\"  # the clustering file.\n",
    "CHRONOSTRAIN_TARGET_CLUSTERS_100pct = CHRONOSTRAIN_DB_DIR / \"ecoli.clusters_100pct.txt\"  # the clustering file.\n",
    "DASHING2_DIR = Path(\"/home/youn/work/bin\")  # Directory that contains the dashing2 executable.\n",
    "\n",
    "\"\"\" MetaPhlAn settings \"\"\"\n",
    "# METAPHLAN_DB_PATH = Path(\"/mnt/e/metaphlan_databases/mpa_vJan21_CHOCOPhlAnSGB_202103/mpa_vJan21_CHOCOPhlAnSGB_202103.pkl\") # MetaPhlan 3 or newer\n",
    "# METAPHLAN_TAXONOMIC_KEY = 's__Escherichia_coli'\n",
    "\n",
    "\"\"\" ============================================ DO NOT EDIT BELOW ============================================ \"\"\"\n",
    "\"\"\" environment variable extraction \"\"\"\n",
    "try:\n",
    "    VARS_SET\n",
    "except NameError:\n",
    "    VARS_SET = True\n",
    "    _cwd = %pwd\n",
    "    _parent_cwd = Path(_cwd).parent\n",
    "    _start_path = %env PATH\n",
    "\n",
    "# Work in parent directory, where all the helper scripts and settings.sh are.\n",
    "%cd \"$_parent_cwd\"\n",
    "%env TARGET_TAXA=$TARGET_TAXA\n",
    "%env NCBI_REFSEQ_DIR=$NCBI_REFSEQ_DIR\n",
    "%env REFSEQ_INDEX=$REFSEQ_INDEX\n",
    "# Need basic executables, such as \"which\" and \"basename\" (required by primersearch)\n",
    "%env PATH=/usr/bin:$_start_path:$DASHING2_DIR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "027d4506-92d5-43fd-8e93-03d81b39dd2d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "EMBOSS:6.6.0.0\n",
      "#Calling Dashing2 version v2.1.19 with command '/home/youn/work/chronostrain/examples/database/dashing2 --version'\n",
      "dashing2 has several subcommands: sketch, cmp, wsketch, and contain.\n",
      "Usage can be seen in those subcommands. (e.g., `dashing2 sketch -h`)\n",
      "\n",
      "\tsketch: converts FastX into k-mer sets/sketches, and sketches BigWig and BED files; also contains functionality from cmp, for one-step sketch and comparisons\n",
      "This is probably the most common subcommand to use.\n",
      "\n",
      "\tcmp: compares previously sketched/decomposed k-mer sets and emits results. alias: dist\n",
      "\n",
      "\tcontain: Takes a k-mer database (built with dashing2 sketch --save-kmers), then computes coverage for all k-mer references using input streams.\n",
      "\twsketch: Takes a tuple of [1-3] input binary files [(u32 or u64), (float or double), (u32 or u64)] and performs weighted minhash sketching.\n",
      "Three files are treated as Compressed Sparse Row (CSR)-format, where the third file contains indptr values, specifying the lengths of consecutive runs of pairs in the first two files corresponding to each row.\n",
      "wsketch is for sketching binary files which have already been summed, whereas sketch is for parsing and sketching (from Fast{qa}, BED, BigWig)\n",
      "\n",
      "\n",
      "Miscellania:\n",
      "printmin: Emit minimizer sequence sets in human-readable form.\n"
     ]
    }
   ],
   "source": [
    "!primersearch --version\n",
    "!dashing2 --version"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "af3195fd-9079-4993-a593-6371dc115b98",
   "metadata": {},
   "source": [
    "## Recipe starts here."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "2506deea-b0e4-4a5b-83b9-1e327f3b56b0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare directories.\n",
    "TARGET_DIR.mkdir(exist_ok=True, parents=True)\n",
    "BLAST_DB_DIR.mkdir(exist_ok=True, parents=True)\n",
    "NCBI_REFSEQ_DIR.mkdir(exist_ok=True, parents=True)\n",
    "MARKER_SEED_DIR.mkdir(exist_ok=True, parents=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c06a06d2-134e-4b4b-8388-dbef0ac304e0",
   "metadata": {},
   "source": [
    "### Step 1: Download chromosomal catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "498db6b9-37ef-473e-8f01-90b1d2720619",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "^C\n",
      "\n",
      "Aborted!\n"
     ]
    }
   ],
   "source": [
    "!bash download_ncbi2.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "334c42d7-1d69-474b-9bf5-af172b23e2de",
   "metadata": {},
   "source": [
    "### Step 2: Build Blast database."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "64765c32-0d24-4628-ac17-8800f20e2591",
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    },
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[*] Creating Blast database.\n",
      "Target fasta file: __tmp_refseqs.fasta\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Buttiauxella/agrestis/DSM_9389/NZ_AP023184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Buttiauxella/sp./3AFRM03/NZ_CP033076.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cedecea/lapagei/NCTC11466/NZ_LR134201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cedecea/neteri/FDAARGOS_392/NZ_CP023525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cedecea/neteri/M006/NZ_CP009458.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cedecea/neteri/ND14a/NZ_CP009459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cedecea/neteri/SSMD04/NZ_CP009451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/133355-SW-C4-Cam/NZ_CP041362.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/86/NZ_LT556084.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/92/NZ_LT556085.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/C3H/NZ_CP080958.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/CA71/NZ_CP064180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/FDAARGOS_122/NZ_CP014015.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/FDAARGOS_1489/NZ_CP083651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/FDAARGOS_165/NZ_CP014070.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/JCM_1661/NZ_AP024585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/amalonaticus/Y19/NZ_CP011132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/arsenatis/LY-1/NZ_CP037864.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/DY2019/NZ_CP080539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/FDAARGOS_1421/NZ_CP077405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/FDAARGOS_1422/NZ_CP077300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/FDAARGOS_253/NZ_CP020448.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/FDAARGOS_290/NZ_CP022049.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/braakii/MiY-A/NZ_CP045771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/cronae/S175-1/NZ_CP060441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/farmeri/AUSMDU00008141/NZ_CP022695.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/farmeri/CCRI-24236/NZ_CP081314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/farmeri/FDAARGOS_1423/NZ_CP077291.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/111/NZ_CP046502.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/154/NZ_CP038653.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/18-1/NZ_CP022273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/56415CZ/NZ_CP085726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/565/NZ_CP038656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/62/NZ_CP048382.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/680/NZ_CP038658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/705SK3/NZ_CP022151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/AR_0022/NZ_CP024881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/AR_0116/NZ_CP032184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/ATCC_8090/NZ_CP049015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/B38/NZ_CP016762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/C50/NZ_CP042478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CAV1321/NZ_CP011612.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CAV1741/NZ_CP011657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CAV1857/NZ_CP037734.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CF48846/NZ_CP070544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CF49969/NZ_CP070549.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CF50935/NZ_CP070559.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CFNIH1/NZ_CP007557.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CFNIH2/NZ_CP025757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CFNIH3/NZ_CP026235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CFNIH4/NZ_CP026231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CFNIH9/NZ_CP026238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/Cfr-13/NZ_CP047773.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CF_324/NZ_CP078020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CIFR51929/NZ_CP059427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CitB/NZ_CP048416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/Citrobacter_freundii_str._E2614/NZ_LS992175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/Citrobacter_freundii_str._U2785/NZ_LS992183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/CRCB-101/NZ_CP024819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/DY2010/NZ_CP086287.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/E11/NZ_CP042524.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/E33/NZ_CP042517.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/E51/NZ_CP042534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/FDAARGOS_549/NZ_CP033744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/FDAARGOS_550/NZ_CP033742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/FDAARGOS_61/NZ_CP026045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/FDAARGOS_73/NZ_CP026056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/HM38/NZ_CP024672.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/IDR1800045912-01-00/NZ_CP054294.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/IDR1900015725-01-02/NZ_CP054278.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/L75/NZ_CP047307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/LDL3-3/NZ_CP047247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/M92/NZ_CP058318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MGH279/NZ_CP060650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MGH281/NZ_CP060662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MGH281C/NZ_CP060659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MGH283/NZ_CP060654.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MGYG-HGUT-02495/NZ_LR699006.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/MSB1_1H-sc-2280393/NZ_LR890181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/N16-03880/NZ_CP047269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/N18-04078/NZ_CP047273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/N18-04085/NZ_CP047275.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/N18-04128/NZ_CP047279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/PSV/NZ_LR881934.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/R17/NZ_CP035276.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/R47/NZ_CP040698.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB12-C05/NZ_CP057868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB12-C12/NZ_CP057867.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB12-C18/NZ_CP057866.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB12-C19/NZ_CP057864.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB12-C20/NZ_CP057863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB14-C12/NZ_CP057824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB24-C03/NZ_CP057509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB24-C05/NZ_CP057506.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB29-C07/NZ_CP057353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB30-C03/NZ_CP057323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB30-C04/NZ_CP057319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB31-C16/NZ_CP057258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB35-C18/NZ_CP057149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB35-C22/NZ_CP057146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB36-C06/NZ_CP057136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHB41-C16/NZ_CP056967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00006/NZ_CP056910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00011/NZ_CP056907.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00084/NZ_CP055588.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00110/NZ_CP055582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00119/NZ_CP056852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00120/NZ_CP056850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00126/NZ_CP056839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00135/NZ_CP056827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00162/NZ_CP056809.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00200/NZ_CP055551.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00214/NZ_CP056653.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00267/NZ_CP056644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00302/NZ_CP056635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00310/NZ_CP056622.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00334/NZ_CP056597.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00342/NZ_CP056595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00350/NZ_CP056592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00355/NZ_CP056586.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00370/NZ_CP056573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00398/NZ_CP055538.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00408/NZ_CP056559.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00435/NZ_CP056527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00444/NZ_CP056515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00449/NZ_CP056505.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00477/NZ_CP056473.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00486/NZ_CP056466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00502/NZ_CP056451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00589/NZ_CP056395.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00658/NZ_CP056365.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00697/NZ_CP056336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00714/NZ_CP056333.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00830/NZ_CP056314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00857/NZ_CP056293.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00858/NZ_CP056289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00862/NZ_CP055421.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00902/NZ_CP056256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00915/NZ_CP056245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00923/NZ_CP056244.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00935/NZ_CP056238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00942/NZ_CP056235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00965/NZ_CP056232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00968/NZ_CP056229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-00985/NZ_CP056208.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/RHBSTW-01016/NZ_CP056181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/SCLZS47/NZ_CP092493.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/SL151/NZ_CP016952.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/UMH13/NZ_CP024683.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/UMH14/NZ_CP024680.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/UMH15/NZ_CP024679.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/UMH16/NZ_CP024677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/UMH19/NZ_CP024673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/Upstream_1/NZ_CP038856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/zone4/NZ_CP045726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/freundii/ZY198/NZ_CP055247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/AR_0024/NZ_CP026709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/AR_0025/NZ_CP026697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/ATCC_BAA-895/NC_009792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/FDAARGOS_1029/NZ_CP066089.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/FDAARGOS_287/NZ_CP022073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/FDAARGOS_393/NZ_CP023527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/FDAARGOS_530/NZ_CP033780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/FDAARGOS_646/NZ_CP044097.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/MPUCK001/NZ_AP023452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/koseri/NJ/NZ_CP050078.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/pasteurii/FDAARGOS_1424/NZ_CP077262.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/pasteurii/UMH17/NZ_CP024676.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/56322CZ/NZ_CP091809.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/BY9F36/NZ_CP054129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/Effluent_1/NZ_CP039327.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/FDAARGOS_617/NZ_CP044098.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/FDAARGOS_738/NZ_CP046348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/IOMTU157/NZ_AP022378.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/K218/NZ_CP089316.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/NCTC11104/NZ_LR134214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/P10159/NZ_CP012554.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/PNUCL1/NZ_CP048388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/Sb-2/NZ_CP092442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STN0717-27/NZ_AP022486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STN0717-36/NZ_AP022494.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STN0717-72/NZ_AP022513.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STW0522-26/NZ_AP022389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STW0522-27/NZ_AP022394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/STW0522-30/NZ_AP022399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/SWHIN_111/NZ_CP055089.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/portucalensis/WCHCP085039/NZ_CP043009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/rodentium/DSM_16636/NZ_CP082833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/rodentium/ICC168/NC_013716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sedlakii/3347689II/NZ_CP071070.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./172116965/NZ_CP046673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./ABFQG/NZ_CP036435.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./AR_0157/NZ_CP029727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./B72/NZ_CP069801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./BDA59-3/NZ_CP063427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./BIDMC108/NZ_CP073007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./CF971/NZ_CP041051.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./CFNIH10/NZ_CP026216.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./FDAARGOS_155/NZ_CP014030.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./H12-3-2/NZ_CP045837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./LUTT5/NZ_CP047606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./MGH103/NZ_CP073043.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./MGH105/NZ_CP073048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./R56/NZ_CP069159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB20-C15/NZ_CP057632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB20-C16/NZ_CP057630.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB21-C01/NZ_CP057620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB21-C05/NZ_CP057611.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB25-C09/NZ_CP057483.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB35-C17/NZ_CP057150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB35-C21/NZ_CP057147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHB36-C18/NZ_CP057125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00017/NZ_CP056899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00021/NZ_CP056896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00053/NZ_CP056888.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00107/NZ_CP056861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00127/NZ_CP056834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00137/NZ_CP056822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00229/NZ_CP056647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00424/NZ_CP056546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00446/NZ_CP056508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00524/NZ_CP056421.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00535/NZ_CP056409.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00570/NZ_CP056399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00599/NZ_CP056717.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00628/NZ_CP056381.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00667/NZ_CP056361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00678/NZ_CP056350.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00696/NZ_CP056341.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00821/NZ_CP056321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00859/NZ_CP056284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00881/NZ_CP056269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00887/NZ_CP056267.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00903/NZ_CP056251.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00944/NZ_CP055418.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00976/NZ_CP056219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-00986/NZ_CP056200.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-01013/NZ_CP056185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./RHBSTW-01044/NZ_CP056180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./S40-1-2/NZ_CP045840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./TBCP-5362/NZ_CP040234.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./TSA-1/NZ_CP053573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/sp./Y3/NZ_CP050009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/telavivensis/6105/NZ_CP045205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/tructae/SNU_WT2/NZ_CP038469.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/BF-6/NZ_CP019986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/FDAARGOS_364/NZ_CP023504.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/FDAARGOS_616/NZ_CP044101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/LHC5-1/NZ_CP084290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/MGYG-HGUT-02535/NZ_LR699014.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/werkmanii/UMH18/NZ_CP024675.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/youngae/L6/NZ_CP021963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Citrobacter/youngae/NCTC13709/NZ_LR134485.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/condimenti/LMG_26250/NZ_CP012264.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/dublinensis/LMG_23823/NZ_CP012266.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/malonaticus/CMCC45402/NC_023032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/malonaticus/LMG_23826/NZ_CP013940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/muytjensii/ATCC_51329/NZ_CP012268.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/5563_17/NZ_CP059045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/ATCC_29544/NZ_CP011047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/ATCC_BAA-894/NC_009778.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/C105731/NZ_CP049154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/C757/NZ_CP049150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/C767/NZ_CP049147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/C79/NZ_CP049144.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/CS-931/NZ_CP027107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/ES15/NC_017933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/GZcsf-1/NZ_CP028974.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/MOD1-H322/NZ_CP078110.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/NCTC_8155/NZ_CP012253.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sakazakii/Sp291/NC_020260.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/sp./JZ38/NZ_CP017662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Cronobacter/universalis/NCTC_9529/NZ_CP012257.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/17Nkhm-UP2/NZ_AP019630.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/1808-013/NZ_AP019632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/5549/NZ_CP093154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/A2563/NZ_AP022628.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/AEB30/NZ_CP046618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/ATCC_35953/NZ_CP011863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/CAV1043/NZ_CP011591.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/CAV1043/NZ_CP034336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/En30/NZ_AP024498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/Ent261/NZ_AP024281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/FDAARGOS_1432/NZ_CP077411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/FDAARGOS_892/NZ_CP065693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/JBIWA002/NZ_CP074177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/JBIWA007/NZ_CP074154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/L1/NZ_CP007546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/L1_delta-T1RM/NZ_CP074584.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/RHBSTW-00516/NZ_CP056716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/RHBSTW-00542/NZ_CP056711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/RHBSTW-01009/NZ_CP056126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/SD4L/NZ_CP063441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/asburiae/WW-19C/NZ_CP080107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/1367/NZ_CP039452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/220/NZ_CP039453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/EB-247/NZ_LT992502.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/FDAARGOS_1427/NZ_CP077206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/FDAARGOS_1496/NZ_CP083643.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/FDAARGOS_1497/NZ_CP083641.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/FDAARGOS_1520/NZ_CP083620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/STN0717-56/NZ_AP022508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/bugandensis/XL95/NZ_CP083403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cancerogenus/CR-Eb1/NZ_CP025225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cancerogenus/FDAARGOS_1428/NZ_CP077290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cancerogenus/HAEC1/NZ_CP082280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cancerogenus/MiY-F/NZ_CP045769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cancerogenus/UPC1/NZ_LR881936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/chengduensis/WCHECh050004/NZ_CP043318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/109/NZ_CP020525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/14240244101/NZ_CP045487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/174/NZ_CP020528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/3849/NZ_CP052870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/88701/NZ_CP045485.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/99101/NZ_CP045486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/A1137/NZ_CP021851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0002/NZ_CP018814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0050/NZ_CP021896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0053/NZ_CP021776.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0060/NZ_CP026719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0065/NZ_CP020053.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0072/NZ_CP026850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0073/NZ_CP032291.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0093/NZ_CP027604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0136/NZ_CP021902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0154/NZ_CP029716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AR_0163/NZ_CP021749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ATCC_13047/NC_014121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/AVS0889/NZ_CP092042.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/C45/NZ_LT991957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/CBG15936/NZ_CP046116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/CZ-1/NZ_CP035738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/CZ862/NZ_CP073310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/D41-sc-1712200/NZ_CP056779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/DSM_30054/NZ_CP056776.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/E3442/NZ_CP033466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL112/NZ_CP077661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL352/NZ_CP083709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL404/NZ_CP091496.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL405/NZ_CP091492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL411/NZ_CP091486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL414/NZ_CP091481.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL42/NZ_CP082147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL72/NZ_CP077659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECL78/NZ_CP077660.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECl_1.VN/NZ_CP085080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECNIH2/NZ_CP008823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECNIH4/NZ_CP009850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECNIH5/NZ_CP009854.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/ECNIH7/NZ_CP017990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/EN3600/NZ_CP035633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/FDA-CDC-AR_0132/NZ_CP027618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/FDA-CDC-AR_0164/NZ_CP028950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/FDAARGOS_1431/NZ_CP077211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/FDAARGOS_77/NZ_CP026975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/GGT036/NZ_CP009756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/GX1Z-1L/NZ_CP071861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/M12X01451/NZ_CP017475.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/MY490/NZ_CP053568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/N13-01531/NZ_CP037919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/PIMB10EC27/NZ_CP020089.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/POL9/NZ_CP086408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/SDM/NC_018079.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/STN0717-73/NZ_AP022519.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/WP5-S18-CRE-02/NZ_AP022126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/WP5-S18-ESBL-01/NZ_AP022133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/WP8-W18-CRE-04/NZ_AP022228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/cloacae/WP8-W19-CRE-02/NZ_AP022274.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/08-175/NZ_CP049290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/14102KSN/NZ_CP045312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/14280161202/NZ_CP045450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/14320055602/NZ_CP045451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/1575/NZ_CP068287.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/189/NZ_CP047965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/2013_1a/NZ_CP031565.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/20710/NZ_CP030076.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/234/NZ_CP021162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/34399/NZ_CP010384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/34977/NZ_CP010376.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/34978/NZ_CP012165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/34983/NZ_CP010377.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/34998/NZ_CP012167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/3804/NZ_CP064656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/388/NZ_CP021167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/48411CZ/NZ_CP085780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/48947CZ/NZ_CP085774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/49589CZ/NZ_CP085767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/54569CZ/NZ_CP085757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/57816CZ/NZ_CP085753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/59732CZ/NZ_CP085748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/60214CZ/NZ_CP085743.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/A1/NZ_CP031574.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AKB48/NZ_CP044335.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AMS-38/NZ_CP051132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AR432/NZ_CP029126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AR_0365/NZ_CP027142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AR_038/NZ_CP030347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/AUH-ENM30/NZ_CP045611.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/BD-50-Eh/NZ_CP063224.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/BW/NZ_CP023569.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/BW/NZ_CP027111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C15/NZ_CP042488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C15117/NZ_CP032841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C309/NZ_LT991954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C4/NZ_CP042540.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C44/NZ_CP042566.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/C45/NZ_CP042551.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CAV1176/NZ_CP011662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CAV1311/NZ_CP011572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CAV1411/NZ_CP011581.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CAV1668/NZ_CP011584.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CAV1669/NZ_CP011650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CM18-216/NZ_CP050311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/CM18-242-2/NZ_CP050506.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/DSM_14563/NZ_CP017186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/DSM_16687/NZ_CP017180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/DSM_16691/NZ_CP017179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/E5/NZ_CP042571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/E70/NZ_CP046271.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/EB_P6_L3_02.19/NZ_CP043853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/EB_P9_L5_03.19/NZ_CP043766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/EC-TO80/NZ_LS999206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/EC48293/NZ_CP070534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Ec61/NZ_CP053103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ECC59/NZ_CP080470.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ECNIH3/NZ_CP008897.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ECR091/NZ_CP008905.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/EGYMCRVIM/NZ_CP053190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eh1/NZ_CP034754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-1/NZ_CP047765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-2/NZ_CP047757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-4/NZ_CP047740.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-5/NZ_CP047736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-6/NZ_CP047729.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-7/NZ_CP066108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-8/NZ_CP066101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-E1/NZ_CP066094.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-E2/NZ_CP047715.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-E3/NZ_CP049021.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Eho-E4/NZ_CP048696.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ENCL48212/NZ_CP059414.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ENCL48880/NZ_CP059419.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ENCL49790/NZ_CP059422.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/F2/NZ_CP047570.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_1078/NZ_CP068215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_1433/NZ_CP077308.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_1434/NZ_CP077264.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_1435/NZ_CP077392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_1533/NZ_CP083613.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FDAARGOS_642/NZ_CP044107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FRM/NZ_CP019889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/FY-1/NZ_CP090792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/G17/NZ_CP079939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/GX4-8L/NZ_CP071876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/IR5283/NZ_CP064113.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/KCJ3K13/NZ_CP054410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/KCJ3K19/NZ_CP054411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/L51/NZ_CP033102.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/LMG27195/NZ_CP017183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/M1/NZ_CP090909.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ME-1/NZ_CP041733.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/MS7884A/NZ_CP022532.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/MSB1_8J/NZ_CP065939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/N1/NZ_CP031571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/NCTC11571/NZ_LR134164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/NJGLYY90-CR/NZ_CP061744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/OIPH-N069/NZ_AP019817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/OSUCZKPC4-140/NZ_CP066553.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/OSUKPC4_L/NZ_CP024908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/OSUVMCKPC4-2/NZ_CP029246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Pb204/NZ_CP030007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/PG20180049/NZ_CP043226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/PG20180056/NZ_CP043225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00012/NZ_CP058191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00016/NZ_CP058187.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00040/NZ_CP058175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00056/NZ_CP058172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00059/NZ_CP058167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00070/NZ_CP058164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00083/NZ_CP058160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00086/NZ_CP058157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00105/NZ_CP058153.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00114/NZ_CP058149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00133/NZ_CP058146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00140/NZ_CP058128.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00198/NZ_CP056756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00216/NZ_CP056751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00217/NZ_CP056747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00220/NZ_CP056742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00316/NZ_CP056727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00333/NZ_CP056719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00492/NZ_CP056455.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00564/NZ_CP056707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00626/NZ_CP056691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00642/NZ_CP056676.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00668/NZ_CP056663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00844/NZ_CP056155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RHBSTW-00916/NZ_CP056132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C010068/NZ_CP071025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C012369/NZ_CP071023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C017238/NZ_CP071021.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C017834/NZ_CP071019.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C019072/NZ_CP071017.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C019181/NZ_CP071015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C033144/NZ_CP071013.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C034177/NZ_CP071011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/RIVM_C034192/NZ_CP071009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/S11_16/NZ_CP035385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/SCEH020042/NZ_CP028538.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/SH19PTE2/NZ_CP073771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/STN0717-64/NZ_AP022510.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/TUECL19001/NZ_CP090146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/TUECL19003/NZ_CP090192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/WCHEH020038/NZ_CP031726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/WCHEH090011/NZ_CP036310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/WCHEX045001/NZ_CP043382.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Y-2/NZ_CP090796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Y2152/NZ_CP049192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Y233/NZ_CP049046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/Y323/NZ_CP049188.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/hormaechei/ZHH/NZ_CP059711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/070/NZ_CP050073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/6/NZ_CP088119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/C16/NZ_CP042578.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/DSM_13645/NZ_CP017181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/EB_P8_L5_01.19/NZ_CP043511.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/ENHKU01/NC_018405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/IB2020/NZ_CP059481.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/STN0717-53/NZ_AP022498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/STW0522-51/NZ_AP022431.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/STW0522-60/NZ_AP022446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/STW0522-62/NZ_AP022452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/kobei/WCHEK045523/NZ_CP032897.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/196/NZ_CP086100.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/197/NZ_CP086099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/994590/NZ_CP086101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/AA4/NZ_CP018785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/D42-sc-1712201/NZ_CP056119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/EcWSU1/NC_016514.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/EN-119/NZ_CP017279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/FDAARGOS_1475/NZ_CP082860.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/FDAARGOS_1498/NZ_CP083640.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/I140/NZ_CP041062.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/I42/NZ_CP040606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/JP6/NZ_CP040256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/JP9/NZ_CP040527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/P101/NZ_CP006580.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/ludwigii/UW5/NZ_CP011798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/mori/08-091/NZ_CP071063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/mori/ACYC.E9L/NZ_CP091779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/mori/BC01/NZ_CP084692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/mori/CX01/NZ_CP055276.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/mori/HSW1412/NZ_CP061801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/oligotrophicus/CCA6/NZ_AP019007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/35734/NZ_CP012162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/704SK10/NZ_CP022148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/BP10374/NZ_CP038471.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/DSM_16690/NZ_CP017184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/Ecl_20_981/NZ_CP048650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/ECY546/NZ_CP032916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/Ed-982/NZ_CP060733.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/ED5/NZ_CP058253.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/enterobacter_cloacae/NZ_CP060737.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/F1057/NZ_CP086201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/FDAARGOS_1430/NZ_CP077407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/FDAARGOS_523/NZ_CP033800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/KQ-01/NZ_CP058637.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/OIPH-N260/NZ_AP023447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/POL1/NZ_CP086405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/R11/NZ_CP019839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/RHBSTW-00002/NZ_CP058196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/RHBSTW-00309/NZ_CP056737.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/RHBSTW-00695/NZ_CP056168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/RHBSTW-00872/NZ_CP056148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/RHBSTW-00925/NZ_CP056129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/STW0522-66/NZ_AP022465.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/roggenkampii/WCHER090065/NZ_CP045064.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sichuanensis/SGAir0282/NZ_CP027986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/soli/LF7a/NC_015968.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./18A13/NZ_AP019634.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./AN-K1/NZ_CP087390.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./BWH_37/NZ_CP072970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./Crenshaw/NZ_CP020817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./CRENT-193/NZ_CP024812.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./DNB-S2/NZ_CP050520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./DR25/NZ_CP091319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./DSM_30060/NZ_CP056118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./E20/NZ_CP012999.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./E76/NZ_CP042499.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./HK169/NZ_CP017087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./JBIWA003/NZ_CP074170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./JBIWA005/NZ_CP074159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./JBIWA008/NZ_CP074149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./LU1/NZ_CP043438.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./MGH_14/NZ_CP072961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./MGH_3/NZ_CP072953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./MGH_7/NZ_CP072945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./N18-03635/NZ_CP034769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./ODB01/NZ_CP015227.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./RHBSTW-00422/NZ_CP056552.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./RHBSTW-00593/NZ_CP056393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./RHBSTW-00975/NZ_CP055883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./RHBSTW-00994/NZ_CP056199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./SA187/NZ_CP019113.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacter/sp./SGAir0187/NZ_CP025034.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Enterobacteriaceae/endosymbiont/PpusSym/NZ_CP046226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/05-3106/NZ_CP030778.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/06-3542/NZ_CP034162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/07-3866/NZ_CP030781.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/1551-2/NZ_CP025317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/2010C-3449/NZ_CP034212.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/2012EL-1823B/NZ_CP030783.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/2013C-4143/NZ_CP030787.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/2014C-4015/NZ_CP034166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/2014C-4356/NZ_CP024282.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/CB9786/NZ_AP014856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/ChinaSP140150/NZ_CP025676.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/EC06-170/NZ_AP014857.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/KF1/NZ_CP007025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/NCTC_9362/NZ_CP034213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/NIAH_Bird_3/NZ_AP014855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/RM9973/NZ_CP043271.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_165/NZ_CP070296.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_166/NZ_CP070292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_167/NZ_CP070290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_168/NZ_CP070287.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_169/NZ_CP070284.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/albertii/Sample_170/NZ_CP070281.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/00-3076/NZ_CP027584.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/005008/NZ_CP058661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/025542-1/NZ_CP035832.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/027_SK3/NZ_CP067311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/035053C1pKPC2_Vir_090515B1/NZ_CP084026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/04-00955/NZ_CP035498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/042/NC_017626.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/042/NZ_CP042934.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/05-3629/NZ_CP027373.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/06-00048/NZ_CP015229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/06-3462/NZ_CP034794.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/07-4299/NZ_CP027462.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/08-00022/NZ_CP013662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/08-3914/NZ_CP034808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/08-3918/NZ_CP034797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/09-00049/NZ_CP015228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/09-02E/NZ_AP022650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H15/NZ_CP081677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H18/NZ_CP081767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H23/NZ_CP081684.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H24/NZ_CP081774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H25/NZ_CP081690.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/0H26/NZ_CP081782.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/10/NZ_CP048330.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1000C-3/NZ_CP091427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/103/NZ_CP020495.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/103/NZ_LT615378.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/103605/NZ_CP025920.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/104/NZ_CP029164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/105/NZ_CP028700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/106/NZ_CP028698.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/106/NZ_LT615377.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/10671/NZ_CP018250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/107/NZ_CP028695.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/108/NZ_CP028693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/109/NZ_CP028690.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/10R/NZ_CP070232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/110/NZ_CP028687.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1105/NZ_CP023388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/110512/NZ_AP019761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/111/NZ_CP028685.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/11128/NC_013364.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/112/NZ_CP028683.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/11202/NZ_CP071521.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/113/NZ_CP028680.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1130/NZ_CP017434.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/11368/NC_013361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/114/NZ_CP028677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/114/NZ_CP087570.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/115/NZ_CP028674.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/116/NZ_CP028671.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1162C/NZ_CP051659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1162invT2/NZ_CP051663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1162T7/NZ_CP051661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/117/NZ_CP028668.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/118/NZ_CP028665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/118UI/NZ_CP032515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/119/NZ_CP028662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1190/NZ_CP023386.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/120/NZ_CP028659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/12009/NC_013353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/120899/NZ_CP025916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/121/NZ_CP028656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/122/NZ_CP028654.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1223/NZ_CP023383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/123/NZ_CP028652.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/124/NZ_CP028650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/124/NZ_CP048344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/127/NZ_CP023377.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/127/NZ_HG994850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/127/NZ_HG994851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1283/NZ_CP023371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/130/NZ_CP028647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1303/NZ_CP009166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/131/NZ_CP028644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/131/NZ_CP093011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/132/NZ_CP028641.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/133/NZ_CP028638.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/134/NZ_CP028635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/135/NZ_CP028632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/135/NZ_CP080260.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/136/NZ_CP028629.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/137/NZ_CP028626.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/138/NZ_CP028623.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/139/NZ_CP028620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13C1065T/NZ_CP019259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13C1079T/NZ_CP019267.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13E0725/NZ_CP020092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13E0767/NZ_CP020107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13E0780/NZ_CP020106.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13KWH46/NZ_CP019250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13P460A/NZ_CP019271.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13P477T/NZ_CP019273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13P484A/NZ_CP019280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/13TMH22/NZ_CP019256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/140/NZ_CP028617.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1409/NZ_CP063369.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/141/NZ_CP028614.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/142/NZ_CP028609.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/142/NZ_CP048337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1428/NZ_CP023366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/143/NZ_CP028607.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/144/NZ_CP023364.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/144/NZ_CP028603.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/146N4/NZ_CP076237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/147/NZ_CP028600.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/148/NZ_CP028598.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/149/NZ_CP028596.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC001/NZ_CP024127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC007/NZ_CP024131.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC017/NZ_CP024134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC020/NZ_CP024138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC029/NZ_CP024141.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/14EC033/NZ_CP024147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/15.TR.026_OXA/NZ_CP032145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/150/NZ_CP028592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1500/NZ_CP040269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1506/NZ_CP059288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1517k/NZ_CP054388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/155/NZ_CP018237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1585m1/NZ_CP086391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/16-9255/NZ_CP022407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/163/NZ_CP048371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/164/NZ_CP048367.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/165/NZ_CP020509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/16F5M1D1/NZ_AP019856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/17B6-2/NZ_CP038414.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/17MR471/NZ_CP051158.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/180-PT54/NZ_CP015832.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/18044/NZ_AP022326.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/181.1/NZ_CP059907.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/184_2aE/NZ_CP072858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/19-5/NZ_CP047010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/190/NZ_CP020520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/190693/NZ_CP048934.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1916D18/NZ_CP046000.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1916D6/NZ_CP046003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1919D3/NZ_CP046006.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1919D62/NZ_CP046009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1943/NZ_CP023359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1A/NZ_CP026361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1EC187/NZ_CP061108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1EC213/NZ_CP061101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/1FA/NZ_CP026360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2-101/NZ_CP053785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2-6-2/NZ_CP038419.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/20/NZ_CP064847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009C-3133/NZ_CP013025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009C-3378/NZ_CP034792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009C-3554/NZ_CP034803.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009C-4687/NZ_CP034799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009EL-2050/NC_018650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2009EL-2071/NC_018661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2010C-3142/NZ_CP034801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2010C-3347/NZ_CP034806.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2011C-3493/NC_018658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2011C-3911/NZ_CP015240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2011C-4251/NZ_CP027388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2011C-4315/NZ_CP024479.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2011EL-1370-2/NZ_CP022912.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012C-4221/NZ_CP027437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012C-4227/NZ_CP013029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012C-4431/NZ_CP024289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012C-4502/NZ_CP027440.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012C-4606/NZ_CP027352.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2012EL-2448/NZ_CP027586.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3033/NZ_CP027323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3181/NZ_CP027312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3250/NZ_CP027380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3252/NZ_CP027442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3264/NZ_CP027544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3277/NZ_CP027331.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3342/NZ_CP027766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3492/NZ_CP027445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-3513/NZ_CP027555.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4081/NZ_CP027573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4187/NZ_CP027546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4225/NZ_CP027577.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4282/NZ_CP027579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4361/NZ_CP027347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4465/NZ_CP015241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4538/NZ_CP027582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4830/NZ_CP027325.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4974/NZ_CP027587.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2013C-4991/NZ_CP027355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3011/NZ_CP027591.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3050/NZ_CP027472.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3051/NZ_CP027338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3057/NZ_CP027387.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3061/NZ_CP027548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3075/NZ_CP027447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3084/NZ_CP027319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3097/NZ_CP027449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3307/NZ_CP027368.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3338/NZ_CP027452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3599/NZ_CP027435.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-3655/NZ_CP027351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-4135/NZ_CP027310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-4423/NZ_CP027454.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-4587/NZ_CP027342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014C-4639/NZ_CP027361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014EL-1343-2/NZ_CP024228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014EL-1345-2/NZ_CP024223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2014EL-1346-6/NZ_CP024232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3101/NZ_CP027221.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3107/NZ_CP027317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3108/NZ_CP027307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3121/NZ_CP027340.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3163/NZ_CP027219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-3905/NZ_CP027371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-4136CT1/NZ_CP027550.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-4498/NZ_CP027552.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2015C-4944/NZ_CP027390.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/201609/NZ_CP048107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2016C-3878/NZ_CP029747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.01.04CC/NZ_AP021891.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.04.03CC/NZ_AP025205.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.06.04CC/NZ_AP025209.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.09.02CC/NZ_AP025214.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.11.01CC/NZ_AP025220.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017.15.01CC/NZ_AP021890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017C-4109/NZ_CP030767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017C-4173W12/NZ_CP030768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2017_APHA/NZ_CP051615.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-01-1CC/NZ_AP021892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-02-2CC/NZ_AP021896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-06-4CC/NZ_AP021893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-10-1CC/NZ_AP021894.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-11-3CC/NZ_AP021895.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-166/NZ_CP075632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2018-18.25PC/NZ_AP023286.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2019_APHA/NZ_CP051609.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/203740/NZ_CP025913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/204446/NZ_CP025910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/20Ec-P-124/NZ_AP017610.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/20R2R/NZ_CP062160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/210205630/NZ_CP015912.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/210221272/NZ_CP016404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/214-4/NZ_CP025840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2149/NZ_CP017436.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2159/NZ_CP017438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/219/NZ_CP020514.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/21B8/NZ_CP040309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/222/NZ_CP020516.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2273-PO3/NZ_CP061749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2452/NZ_CP031833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2547/NZ_CP072980.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2571/NZ_CP038425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/258E/NZ_CP053855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/25DN/NZ_CP063983.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/25POL/NZ_CP081010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/26561/NZ_CP027118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/266917_2/NZ_CP026723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/268.2/NZ_CP059947.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/272/NZ_CP018239.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/28Eco12/NZ_CP038505.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/28RC1/NZ_CP015020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2A/NZ_CP026358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2D/NZ_CP059003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2EC1/NZ_CP047571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2FA/NZ_CP026356.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2F_0/NZ_CP026357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2_0/NZ_CP026359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2_0/NZ_CP023853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/2_HS-C/NZ_CP038180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3/NZ_CP084096.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3-5-1/NZ_CP038416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/300709/NZ_CP025903.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/30COLEC/NZ_CP070930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/317/NZ_CP023357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/319/NZ_CP018241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/32-4/NZ_CP048310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/33-6/NZ_CP050031.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3347558/NZ_CP071073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3384/NZ_CP017440.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3385/NZ_CP029420.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/350/NZ_CP018243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/357900/NZ_CP038859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/377323_2f/NZ_CP039404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/38/NZ_CP048376.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/382634_2f/NZ_CP039403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3A/NZ_CP026355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3FA/NZ_CP026354.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/3R/NZ_CP049348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4/NZ_CP045209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/40EC/NZ_CP070920.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4276/NZ_CP017442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/472/NZ_CP018245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/493_89/NZ_CP038412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4A/NZ_CP026353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4FA/NZ_CP026352.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4_0/NZ_CP023849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4_1-1/NZ_CP023844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4_2-1/NZ_CP023834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/4_4/NZ_CP023826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/5-1/NZ_CP089445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/503025/NZ_CP025892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/503440/NZ_CP025881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/503829/NZ_CP025869.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/504211/NZ_CP025865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/504237/NZ_CP025862.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/504239/NZ_CP025859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/504838/NZ_CP025856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/50579417/NZ_CP033884.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/510016/NZ_CP025851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/51008369SK1/NZ_CP029973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/52148/NZ_CP050382.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/53/NZ_CP048359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/536/NC_008253.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/537-20/NZ_CP091534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/54/NZ_CP041284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/550/NZ_CP058223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/55989/NC_011748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/58-3/NZ_CP050036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/5A/NZ_CP026351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/5CRE51/NZ_CP021175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/5FA/NZ_CP026350.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/600468/NZ_CP025849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/602354/NZ_CP025847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/61/NZ_CP048326.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/611/NZ_CP038428.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/63-2/NZ_CP050040.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/631/NZ_CP040263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/64/NZ_CP048322.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/64.1/NZ_CP059917.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6409/NZ_CP010371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/644-PT8/NZ_CP015831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6535NalR/NZ_CP089034.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6535WT/NZ_CP089032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/65COLEC/NZ_CP070914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6666666.257727.embl/NZ_LT883142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/67-1/NZ_CP035826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/675SK2/NZ_CP027701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6A/NZ_CP026349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/6FA/NZ_CP026348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7.1_Anguil/NZ_CP076232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/702_18/NZ_CP074701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/720632/NZ_CP025842.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/730V1/NZ_CP061764.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7386Nal/NZ_CP089031.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7386WT/NZ_CP089036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/74-2/NZ_CP050044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7409/NZ_CP038423.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/746/NZ_CP023353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7636/NZ_CP038421.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7784/NZ_CP018247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/789/NZ_CP010315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7A/NZ_CP026347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7FA/NZ_CP026346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/7_2/NZ_CP023820.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/8-3-DC15/NZ_CP050862.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/8-3-Ti3/NZ_CP050865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/81/NZ_CP088131.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/81009/NZ_CP021179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/8368/NZ_CP017444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/83972/NZ_CP058220.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/86-24/NZ_CP038408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/86-3153/NZ_CP027597.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/873.10/NZ_CP061754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/88-3001/NZ_CP027363.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/88-3493/NZ_CP027457.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/88COLEC/NZ_CP070906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/89-3156/NZ_CP027366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/8A/NZ_CP026345.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/8FA/NZ_CP026344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9/NZ_CP048304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9.1_Anguil/NZ_CP076235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/90-1/NZ_CP050047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/90-3040/NZ_CP027459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/90-9280/NZ_CP024240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/90-9281/NZ_CP024243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9000/NZ_CP018252.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9234/NZ_CP017446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/94-3024/NZ_CP009106.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/94EC/NZ_CP047576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/95-3192/NZ_CP027362.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/95-3322/NZ_CP027461.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/95JB1/NZ_CP021335.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/95NR1/NZ_CP021339.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/97-3250/NZ_CP027599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/97.3/NZ_CP061758.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/98-3133/NZ_CP051001.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/99-3165/NZ_CP029981.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/99COLEC/NZ_CP070902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9A/NZ_CP026343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/9FA/NZ_CP026342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A1/NZ_CP029328.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A10/NZ_CP028764.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A11/NZ_CP028763.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A12/NZ_CP028762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A13/NZ_CP028761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A14/NZ_CP028760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A15/NZ_CP028759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A16/NZ_CP028758.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A17/NZ_CP028757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A19/NZ_CP028756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A1_136/NZ_CP040390.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A1_181/NZ_CP040067.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A2/NZ_CP028772.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A20/NZ_CP028755.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A21/NZ_CP028754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A22/NZ_CP028753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A23/NZ_CP028752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A24/NZ_CP028751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A25/NZ_CP028750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A26/NZ_CP028749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A27/NZ_CP028748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A28/NZ_CP028747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A29/NZ_CP028746.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A3/NZ_CP028771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A30/NZ_CP028745.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A31/NZ_CP028744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A32/NZ_CP028743.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A33/NZ_CP028742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A34/NZ_CP028741.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A35/NZ_CP028740.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A36/NZ_CP028739.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A37/NZ_CP029327.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A38/NZ_CP028738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A39/NZ_CP028737.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A4/NZ_CP028770.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A41/NZ_CP028735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A42/NZ_CP028734.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A5/NZ_CP028769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A6/NZ_CP028768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A7/NZ_CP028767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A8/NZ_CP028766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/A9/NZ_CP028765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABU_83972/NC_017631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABWA45/NZ_CP022154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABW_A19/NZ_CP067307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABW_A32/NZ_CP067303.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABW_A33/NZ_CP067299.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ABW_S38/NZ_CP067265.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ACN001/NZ_CP007442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ACN002/NZ_CP007491.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH01/NZ_CP055251.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH02/NZ_CP081701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH04/NZ_CP081706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH21/NZ_CP081711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH25/NZ_CP055256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH62/NZ_CP055259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AH65/NZ_CP058302.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AL.H8/NZ_CP082349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Al_Ain/NZ_CP043539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AM187/NZ_CP037857.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AMA1167/NZ_CP024801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML001_ev01/NZ_CP062241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML001_ev02/NZ_CP062240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML001_ev03/NZ_CP062239.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML001_par/NZ_CP062242.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML002_ev01/NZ_CP062249.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML002_ev02/NZ_CP062248.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML002_ev03/NZ_CP062247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML002_par/NZ_CP062250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML003_ev01/NZ_CP062245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML003_ev02/NZ_CP062244.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML003_ev03/NZ_CP062243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AML003_par/NZ_CP062246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ampC_0069/NZ_CP046396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AMSCJX02/NZ_CP031105.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AMSCJX03/NZ_CP058355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AMSCJX04/NZ_CP058308.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AMSHJX01/NZ_CP030939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/APEC_IMT5155/NZ_CP005930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/APEC_O1/NC_008563.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/APEC_O18/NZ_CP006830.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/APEC_O2-211/NZ_CP006834.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/APEC_O78/NC_020163.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AQ15/NZ_CP043487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR-0427/NZ_CP044148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR-0428/NZ_CP044145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR-0429/NZ_CP044143.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR-0430/NZ_CP044140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR202.2/NZ_CP043946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR216.2b/NZ_CP043942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR24.2b/NZ_CP035944.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR434/NZ_CP029122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR435/NZ_CP029115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR436/NZ_CP029111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR437/NZ_CP029108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ARL09_232/NZ_CP049967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0006/NZ_CP024851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0011/NZ_CP024855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0013/NZ_CP032204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0014/NZ_CP024859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0015/NZ_CP024862.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0017/NZ_CP024886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0019/NZ_CP024889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0055/NZ_CP021935.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0058/NZ_CP021689.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0061/NZ_CP020058.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0067/NZ_CP032261.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0069/NZ_CP020055.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0077/NZ_CP026755.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0081/NZ_CP027534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0085/NZ_CP029741.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0086/NZ_CP032201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0089/NZ_CP032265.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0104/NZ_CP020116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0114/NZ_CP021732.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0118/NZ_CP020048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0119/NZ_CP021535.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0128/NZ_CP021722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0137/NZ_CP021879.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0149/NZ_CP021532.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0150/NZ_CP021736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0151/NZ_CP021691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0162/NZ_CP021683.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0369/NZ_CP027140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0372/NZ_CP027134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_0374/NZ_CP027126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_451/NZ_CP030337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_452/NZ_CP030331.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_Bank_#0346/NZ_CP066366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AR_Bank_#0349/NZ_CP041996.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AS19-RrmA-/NZ_CP027430.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_11775/NZ_CP033092.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_25922/NZ_CP009072.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_25922/NZ_CP037449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_25922/NZ_CP032085.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_35150/NZ_CP038405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_43886/NZ_CP024256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_43888/NZ_CP041623.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_43889/NZ_CP015853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_43890/NZ_CP045863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_43896/NZ_CP024278.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_8739/NC_010468.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_8739/NZ_CP022959.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_8739/NZ_CP043852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_98082/NZ_CP034658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_BAA-178/NZ_CP023541.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_BAA-196/NZ_CP042867.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ATCC_BAA-196/NZ_CP042865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AUSMDU00002545/NZ_CP045975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AUSMDU00014361/NZ_CP045827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AVS0096/NZ_CP076344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AW1.7/NZ_CP072539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/AW1.7/NZ_CP073624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B2C/NZ_CP035874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B36/NZ_LR130545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B379/NZ_CP090456.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B4103-1/NZ_CP024245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B7A/NZ_CP035812.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/B95._delta_A/NZ_CP085842.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BA22372/NZ_CP040397.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Balcarce_14.2/NZ_CP076243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Balcarce_24.2/NZ_CP076245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BB24-1/NZ_CP038402.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BCE049/NZ_CP042250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BE104/NZ_CP040643.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BE2-5/NZ_CP032986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BEC1-S17-ESBL-07/NZ_AP022295.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BEC1-S17-ESBL-09/NZ_AP022298.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BEN2908/NZ_LR740776.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BEN5202/NZ_LR740758.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BH100L_substr._MG2017/NZ_CP025716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BH100N_substr._MG2017/NZ_CP025703.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BH100_substr._MG2014/NZ_CP024650.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BH100_substr._MG2017/NZ_CP025251.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BH212/NZ_CP029239.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21/NZ_CP053601.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21/NZ_CP060121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21(DE3)/NC_012892.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21(DE3)/NC_012971.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21(DE3)/NZ_CP053602.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21(DE3)/NZ_CP081489.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21(RP)/NZ_CP084683.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21-AI/NZ_CP047231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21-Gold(DE3)pLysS_AG/NC_012947.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21-RIPL/NZ_CP083273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BL21_(TaKaRa)/NZ_CP010816.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BLK16/NZ_CP014642.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BLK9/NZ_CP014641.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BLR(DE3)/NZ_CP020368.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BMH-17-0027/NZ_CP076704.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BR1220/NZ_CP093068.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BS15R-A/NZ_CP063349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BS74R-D/NZ_CP063332.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BS90R-A/NZ_CP063341.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BUCT2930/NZ_CP059723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BW25113_substr._CHXR1G20/NZ_CP069132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BW25113_substr._CHXR2G20/NZ_CP069133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BZKR2G40/NZ_CP064682.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/BZKR3G40/NZ_CP064681.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C/NZ_CP020543.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C/NZ_CP029371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C1/NZ_CP010116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C1-057/NZ_CP035366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C10/NZ_CP010132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C11/NZ_CP010133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C2/NZ_CP010117.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C21/NZ_CP052877.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C227-11/NZ_CP011331.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C2566/NZ_CP014268.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C3/NZ_CP010119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C3029/NZ_CP014269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C311/NZ_CP062204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C4/NZ_CP010121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C41(DE3)/NZ_CP010585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C43(DE3)/NZ_CP011938.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C439/NZ_CP046025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C5/NZ_CP010122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C600/NZ_CP031214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C7/NZ_CP010240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C7/NZ_CP059279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C8/NZ_CP010125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C9/NZ_CP010129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/C9/NZ_CP059281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CAR/NZ_CP026026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CAU16175/NZ_CP047378.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CB9615/NC_013941.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CCUG_73778/NZ_CP041337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CD306/NZ_CP013831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CDC#03-98/NZ_CP022689.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CE10/NC_017646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CE1628/NZ_CP053852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CE1803A/NZ_CP053851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CE2050/NZ_CP073621.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CETR1G40/NZ_CP064680.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CETR2G40/NZ_CP064679.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CETR3G40/NZ_CP064678.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFIAFB20120266/NZ_CP076706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFIAFB20140388/NZ_CP076709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFS3246/NZ_CP026929.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFS3273/NZ_CP026932.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFS3292/NZ_CP026935.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFS3313/NZ_CP026939.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN004176/NZ_CP014583.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN018748/NZ_CP028192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN027343/NZ_CP037943.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN027346/NZ_CP037945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN027350/NZ_CP037941.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN029787/NZ_CP011416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN051542/NZ_CP020835.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061761/NZ_CP042903.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061762/NZ_CP042901.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061763/NZ_CP042899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061768/NZ_CP042973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061769/NZ_CP042969.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061770/NZ_CP023142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061771/NZ_CP042896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN061772/NZ_CP042893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN064035/NZ_CP031134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN064036/NZ_CP028166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFSAN067215/NZ_CP028320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFT073/NC_004431.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFT073/NZ_CP051263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CFT073/NZ_CP058222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ChlosBP-23-1/NZ_CP048603.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CI5/NZ_CP011018.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CIT/NZ_CP026028.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CJ236/NZ_CP029238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/clone_D_i14/NC_017652.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/clone_D_i2/NC_017651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CLSC36/NZ_CP041300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CM_IVRI_KOL-1/NZ_CP033158.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CNR65D6/NZ_OU701452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CNR85I8/NZ_OU701449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Combat11I9/NZ_CP021726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Combat13F7/NZ_CP019245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Combat2C1/NZ_CP019243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP131_Sichuan/NZ_CP053720.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP53/NZ_CP033096.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP55_Sichuan/NZ_CP053731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP61_Sichuan/NZ_CP053730.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP66-6_Sichuan/NZ_CP053723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CP8-3_Sichuan/NZ_CP053736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/cq9/NZ_CP031546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CRE1/NZ_CP034399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CRE10/NZ_CP034404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CRE1493/NZ_CP019071.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CRE1540/NZ_CP019051.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CREC-532/NZ_CP024830.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CREC-544/NZ_CP024826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CREC-591/NZ_CP024821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CREC-629/NZ_CP024815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CS18F/NZ_CP074576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CT01/NZ_CP081791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CT02/NZ_CP081698.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CT29/NZ_CP032073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CT30/NZ_CP032078.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CU37RT-2/NZ_CP082774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CU39ES-1/NZ_CP082777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CU42ET-1/NZ_CP082773.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CU532-9/NZ_CP082776.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CU568-3/NZ_CP082775.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CV261/NZ_CP040316.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CV601/NZ_CP067994.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CV601gfp/NZ_CP043213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CV839-06/NZ_CP025753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CV839-15/NZ_CP024978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX04/NZ_CP081713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX05/NZ_CP081719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX07/NZ_CP081724.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX08/NZ_CP073718.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX12/NZ_CP073719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX15/NZ_CP081742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CX21/NZ_CP081762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CY598/NZ_CP068283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CY706/NZ_CP068281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/CY708/NZ_CP068279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D1/NZ_CP010134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D10/NZ_CP010157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D2/NZ_CP010137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D3/NZ_CP010140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D4/NZ_CP010143.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D5/NZ_CP010145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D6/NZ_CP010148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D7/NZ_CP010150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D72/NZ_CP035312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D8/NZ_CP010151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/D9/NZ_CP010152.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DA33133/NZ_CP029574.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DA33135/NZ_CP029576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DA33137/NZ_CP029579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DA61218/NZ_CP061206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Dam_Dcm/NZ_CP053603.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DCE7/NZ_CP090611.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DE17/NZ_CP045206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DEC4E/NZ_CP038398.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DEC5A/NZ_CP038394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DEC5B/NZ_CP038389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DEC5D/NZ_CP038386.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DEC5E/NZ_CP038383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH1/NC_017625.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH1/NC_017638.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH1Ec095/NZ_CP012125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH1Ec104/NZ_CP012126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH1Ec169/NZ_CP012127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH5a/NZ_CP076470.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH5alpha/NZ_CP025520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH5alpha/NZ_CP026085.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DH5alpha/NZ_CP045741.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DL21/NZ_CP079747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DOC14/NZ_CP091065.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Dog168/NZ_CP080116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DS26-1/NZ_CP035879.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DSM_103246/NZ_CP019944.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/DTU-1/NZ_CP026612.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E-1246/NZ_CP025573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E._coli_RL465/NZ_LT594504.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E105/NZ_CP071375.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E110019/NZ_CP035751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E118/NZ_CP049196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E138/NZ_AP022351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E148/NZ_AP024518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E166/NZ_CP032066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E189/NZ_AP024521.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2/NZ_CP048915.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2/NZ_CP086662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2348_69/NC_011601.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2348_69/NZ_CP059840.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E24377A/NC_009801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2528-C1/NZ_CP035870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E275_a/NZ_CP035865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2855/NZ_AP018796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2863/NZ_AP018802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E2865/NZ_AP018808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E302/NZ_AP022362.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E308/NZ_CP030281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E32511/NZ_CP038380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E41/NZ_CP069707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E41-1/NZ_CP028483.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E597/NZ_CP049197.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E62/NZ_CP022393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E686/NZ_CP049198.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E706/NZ_CP029687.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/E7476/NZ_CP035817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EA13/NZ_CP069712.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EA2/NZ_CP069710.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EA7/NZ_CP069709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EBJ001/NZ_CP086336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EBJ003/NZ_CP086334.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec-050/NZ_CP043227.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-107/NZ_CP043217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-129/NZ_CP038453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-16-35/NZ_CP093226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-1639/NZ_LR025101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-7215/NZ_LR025096.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec-FL1-1X/NZ_CP066836.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-JS426/NZ_CP072322.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-TO143/NZ_LS999560.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC-TO75/NZ_LS998785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC1/NZ_CP058574.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC10/NZ_CP060946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec1079/NZ_CP081306.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC11/NZ_CP027255.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC11/NZ_CP073360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC11/NZ_CP060938.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC12/NZ_CP070229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC12/NZ_CP060933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC13/NZ_CP060928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC14/NZ_CP060924.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC15/NZ_CP060919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC1515/NZ_CP021844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC16/NZ_CP074120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC1722-1/NZ_CP081192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC17GD31/NZ_CP031293.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC18/NZ_CP060914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC19/NZ_CP060908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC195/NZ_CP085188.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC2/NZ_CP041955.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC2/NZ_CP058571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC20/NZ_CP084900.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC20/NZ_CP060902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC21/NZ_CP060899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC22/NZ_CP060892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC2341/NZ_CP072976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC24/NZ_CP060885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC25/NZ_CP035123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC26/NZ_CP060880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC27/NZ_CP060873.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC28/NZ_CP049101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC29/NZ_CP060867.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC3/NZ_CP060979.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC31/NZ_CP056036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC31/NZ_CP060987.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC32/NZ_CP085619.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec387/NZ_AP024582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec394-330266/NZ_CP056114.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC4/NZ_CP060974.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec40743/NZ_CP041919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC4115/NC_011353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec42/NZ_CP059119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC42405/NZ_CP043414.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC45/NZ_CP059125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC5/NZ_CP060969.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC50/NZ_CP056073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ec55/NZ_CP059130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC5502/NZ_CP092647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC590/NZ_CP016182.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC7/NZ_CP060963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC71/NZ_CP085622.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC7518/NZ_CP085183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC8/NZ_CP060953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC9/NZ_CP060950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC93/NZ_CP061329.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC931/NZ_CP049118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC958/NZ_HG941718.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC96/NZ_CP060748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC974/NZ_CP021840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECC-1470/NZ_CP010344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCHD184/NZ_CP033250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCNB12-2/NZ_CP033635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCNB20-2/NZ_CP034787.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCRA-119/NZ_CP029242.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCTRSRTH03/NZ_CP041110.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECCWS199/NZ_CP032237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECJXMCRH7/NZ_CP067342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcMAD1/NZ_LR595691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ecMN1F/NZ_CP044410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECNB21-M121/NZ_CP083585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Eco-15/NZ_CP047710.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECO2947/NZ_CP046259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Eco889/NZ_CP015159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECOL-18-VL-LA-PA-Ryan-0026/NZ_CP041392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ecoli019/NZ_LR130532.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_224/NZ_CP018948.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_244/NZ_CP019020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_276/NZ_CP018953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_316/NZ_CP018957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_448/NZ_CP015076.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_517/NZ_CP018965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_542/NZ_CP018970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_545/NZ_CP018976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_656/NZ_CP018979.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_732/NZ_CP015138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_743/NZ_CP015069.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_745/NZ_CP015074.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_867/NZ_CP018983.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_881/NZ_CP019029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ146/NZ_CP018991.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ147/NZ_CP018995.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ153/NZ_CP019000.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ155/NZ_CP019005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ159/NZ_CP019008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ161/NZ_CP019012.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Ecol_AZ162/NZ_CP019015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECONIH1/NZ_CP009859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECONIH2/NZ_CP014667.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECONIH4/NZ_CP026399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECONIH5/NZ_CP026202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECONIH6/NZ_CP026199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP17-1298/NZ_CP040570.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP17-46/NZ_CP040572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP19-198/NZ_CP066759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP19-2498/NZ_CP066748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP19-598/NZ_CP066753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECP19-798/NZ_CP066756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF14/NZ_CP054230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF15/NZ_CP054227.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF16/NZ_CP054224.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF18/NZ_CP054219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF20/NZ_CP071441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF40/NZ_CP054214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF5/NZ_CP054236.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPF7/NZ_CP054232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPNK004/NZ_CP071439.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EcPNK006/NZ_CP071436.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECSC054/NZ_AP024112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ECY44/NZ_CP069706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EC_J_9/NZ_CP075062.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EDL933/NZ_CP008957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EE72d/NZ_CP065136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EF5-18-41/NZ_CP063492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EF7-18-51/NZ_CP063487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EF7-18-58/NZ_CP068591.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH01-18-02/NZ_CP063518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH01-18-04-A/NZ_CP063515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH08-18-36/NZ_CP063511.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH09-18-41/NZ_CP063503.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH10-18-47/NZ_CP063499.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EH41/NZ_CP045213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EJN001/NZ_CP086338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EK2009/NZ_CP040663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa10/NZ_CP083478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa2/NZ_CP083530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa4/NZ_CP083512.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa5/NZ_CP083503.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa6/NZ_CP083497.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa8/NZ_CP083492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/elppa9/NZ_CP083481.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM03-18-08/NZ_CP063483.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM06-18-14/NZ_CP063479.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM10-18-28/NZ_CP063476.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM18-18-42/NZ_CP063473.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM20-18-50/NZ_CP063467.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EM22-18-53/NZ_CP063459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EPEC1/NZ_LT903847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EPEC_E2348_69_variety_2/NZ_LT827011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ER1709/NZ_CP030240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ER1821R/NZ_CP016018.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ER2683/NZ_CP093221.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL03-1416/NZ_CP032811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL04-3476/NZ_CP032808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL05-0623/NZ_CP032805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL05-1306/NZ_CP032803.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL06-2442/NZ_CP032801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL06-2497/NZ_CP032797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERL06-2503/NZ_CP032795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ERP001/NZ_CP063214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESBL_15/NZ_CP041678.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESBL_DR28/NZ_CP067250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESC53083/NZ_CP070593.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._3426/NZ_LS992185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO124/NZ_LS992180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO148/NZ_LS992190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO217/NZ_LS992192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO6/NZ_LS992166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO60/NZ_LS992168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Escherichia_coli_str._TO73/NZ_LS992171.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESY001/NZ_CP086342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESY002/NZ_CP086556.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ESY003/NZ_CP086340.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Es_ST410_NW1_NDM_09_2017/NZ_CP031231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Es_ST80_L1_NDM_10_2017/NZ_CP031215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ET12567/NZ_CP045457.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ETEC-2264/NZ_CP023349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ETEC-2265/NZ_CP023346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ETEC6/NZ_CP041627.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ETEC_H10407/NC_017633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/EV36/NZ_CP079993.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ExPEC_XM/NZ_CP025328.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F070/NZ_AP023237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F11/NZ_CP076123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F1273/NZ_CP038376.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F1_E4/NZ_CP040307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F2_81/NZ_CP043542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F3113/NZ_CP038374.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F3398/NZ_CP040311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F5505-C1/NZ_CP024257.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F5656C1/NZ_CP024260.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F6294/NZ_CP038372.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F6321/NZ_CP038369.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F6326-C1/NZ_CP024263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F6667/NZ_CP038366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F7349/NZ_CP038363.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F7386/NZ_CP038360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F7508/NZ_CP038357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8092B/NZ_CP038355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8111-1SC3/NZ_CP024269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8492/NZ_CP038282.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8704-2/NZ_CP050998.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8797/NZ_CP038353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8798/NZ_CP038351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F8952/NZ_CP038349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/f9610206-5e81-11e8-bf7f-3c4a9275d6c8/NZ_LR536431.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/f974b26a-5e81-11e8-bf7f-3c4a9275d6c8/NZ_LR536430.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/F9792/NZ_CP024273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FAM21845/NZ_CP017220.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FC853_EC/NZ_CP040919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1058/NZ_CP065970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1059/NZ_CP066032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1077/NZ_CP068155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1248/NZ_CP069542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1249/NZ_CP069459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1250/NZ_CP069571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1251/NZ_CP069445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1252/NZ_CP069528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1253/NZ_CP069549.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1254/NZ_CP069453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1255/NZ_CP069447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1256/NZ_CP069451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1257/NZ_CP069480.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1258/NZ_CP069500.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1259/NZ_CP069559.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1260/NZ_CP069438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1261/NZ_CP069441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1262/NZ_CP069517.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1264/NZ_CP069488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1265/NZ_CP069583.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1266/NZ_CP069589.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1267/NZ_CP069522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1268/NZ_CP069501.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1276/NZ_CP070157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1277/NZ_CP070103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1279/NZ_CP069957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1280/NZ_CP069973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1281/NZ_CP070134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1282/NZ_CP069865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1283/NZ_CP070162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1284/NZ_CP070170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1285/NZ_CP070152.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1286/NZ_CP069893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1287/NZ_CP070071.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1288/NZ_CP070007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1289/NZ_CP069882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1290/NZ_CP070048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1291/NZ_CP069980.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1292/NZ_CP070124.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1293/NZ_CP070041.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1295/NZ_CP069890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1296/NZ_CP070066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1297/NZ_CP070148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1298/NZ_CP070115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1299/NZ_CP069978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1300/NZ_CP069996.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1301/NZ_CP070111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1302/NZ_CP069935.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1303/NZ_CP070045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1304/NZ_CP069990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1337/NZ_CP070026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1370/NZ_CP070098.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1371/NZ_CP070077.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1372/NZ_CP070063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_1373/NZ_CP069945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_144/NZ_CP014111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_293/NZ_CP022050.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_348/NZ_CP022086.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_349/NZ_CP027394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_401/NZ_CP023531.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_403/NZ_CP023535.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_433/NZ_CP023899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_448/NZ_CP023960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_497/NZ_CP033850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_536/NZ_CP033762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_772/NZ_CP041002.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_941/NZ_CP065624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_943/NZ_CP065615.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_944/NZ_CP065613.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_945/NZ_CP065611.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FDAARGOS_946/NZ_CP065607.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/fEC.1/NZ_CP085638.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/fECg99.1/NZ_CP085637.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FEX669/NZ_CP065152.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FHI_NMBU_03/NZ_CP019455.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FHI_NMBU_10/NZ_CP046539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FHI_NMBU_11/NZ_CP045712.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FMU073332/NZ_CP017844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_028/NZ_CP012693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_029/NZ_CP013185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_031/NZ_CP013190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_041/NZ_CP016628.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_042/NZ_CP025318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_043/NZ_CP016828.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_044/NZ_CP016755.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_064/NZ_CP022664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_069/NZ_CP023061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FORC_081/NZ_CP029057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FRIK2069/NZ_CP015846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FRIK2455/NZ_CP015843.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FRIK2533/NZ_CP015842.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FRIK804/NZ_CP034384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FRIK944/NZ_CP016625.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FSIS11705876/NZ_CP035545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FT130/NZ_CP040090.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FUJ80154/NZ_AP024687.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0001/NZ_CP031922.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0002/NZ_CP031919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0003/NZ_CP031916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0004/NZ_CP031913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0005/NZ_CP031912.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0006/NZ_CP031910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0007/NZ_CP031908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0008/NZ_CP031906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0009/NZ_CP031902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/FWSEC0010/NZ_CP031898.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G1_2/NZ_CP060083.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G3X16-2/NZ_CP038138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G3_10/NZ_CP060075.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G4_9/NZ_CP060073.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G5/NZ_CP060065.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G5295/NZ_CP038346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/g62/NZ_CP084896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G6_7/NZ_CP060061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/g74/NZ_CP084895.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G749/NZ_CP014488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/G8/NZ_CP060057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/g83/NZ_CP084894.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/g94/NZ_CP084893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GB089/NZ_CP013663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GB2005/NZ_CP079735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GB2005-delet-a-f/NZ_CP079736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/gb37/NZ_CP084897.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GD33/NZ_CP076646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GDAR2-2/NZ_CP074407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GE3/NZ_CP012376.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GF3-3/NZ_CP053259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GF4-3/NZ_CP053258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Gim1-1/NZ_CP038344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GSH8M-2/NZ_AP019675.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GW-AmxH19/NZ_CP048647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/gw25/NZ_CP084898.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GZ04-0086/NZ_CP042336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/GZEC065/NZ_CP048025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H1/NZ_CP010160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H10/NZ_CP010176.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H105/NZ_CP021454.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H14/NZ_CP010177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H15/NZ_CP010178.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H1827_12/NZ_CP013031.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H2/NZ_CP010163.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H2/NZ_CP028733.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H2495/NZ_CP038342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H3/NZ_CP010167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H3/NZ_CP028732.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H5/NZ_CP010169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H6/NZ_CP010170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H6437/NZ_CP038339.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H7/NZ_CP010171.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H8/NZ_CP010172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/H9Ecoli/NZ_CP029180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HB-Coli0/NZ_CP020933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HB37/NZ_CP053080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HB6/NZ_CP040305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HKUOPY1/NZ_CP019961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HM670/NZ_CP082772.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HS/NC_009800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HS/NZ_CP092639.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HS13-1/NZ_CP026491.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HS30-1/NZ_CP029492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HST04/NZ_CP013952.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HUE1/NZ_AP023427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/HUSEC2011/NZ_HF572917.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IAI1/NC_011741.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IAI39/NC_011750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IBS28/NZ_CP049979.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IDR1900005638-01-02/NZ_CP054282.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IGC_EcoliInv_1.1/NZ_CP054665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IGC_RcoliRes_1.1/NZ_CP054662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IHE3034/NC_017628.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IMT16316/NZ_CP023815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/INSC1002/NZ_CP045977.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/IR53017/NZ_CP064103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ISHS7/NZ_CP092252.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/J-8/NZ_CP047002.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/J31/NZ_CP053787.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/J53/NZ_CP028702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/J53-p1-pV-hybrid-1/NZ_CP068706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JA0072/NZ_CP070227.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JA0257/NZ_CP072786.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JA0265/NZ_CP072785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JA0266/NZ_CP072784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JE86-ST02/NZ_AP022811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JE86-ST05/NZ_AP022815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JEONG-1266/NZ_CP014314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JJ1886/NC_022648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JJ1887/NZ_CP014316.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JJ2434/NZ_CP013835.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JL05/NZ_CP049936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JM83/NZ_CP077969.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JME64/NZ_CP042846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JME65/NZ_CP042845.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JME66/NZ_CP042844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JME67/NZ_CP042843.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JML285/NZ_AP024114.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JNQH497/NZ_CP091925.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JNU/NZ_CP083263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JP538/NZ_AP025180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JS316/NZ_CP058618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/JW5437-1_substr._MG1655/NZ_CP014348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12/NZ_CP047127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_C3026/NZ_CP014272.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_DHB4/NZ_CP014270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_ER2796/NZ_CP009644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_ER3413/NZ_CP009789.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_J53/NZ_CP070962.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010439.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010440.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010443.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_MG1655/NZ_CP010445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_NEB_5-alpha/NZ_CP017100.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._AB1157/NZ_CP076404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._AG100/NZ_LN832404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._BW25113/NZ_CP009273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._BW2952/NC_012759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._DH10B/NC_010473.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._GM4792/NZ_CP011342.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._GM4792/NZ_CP011343.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._HMS174/NZ_LM993812.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MC4100/NZ_HG738867.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NC_000913.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP009685.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP012868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP014225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP025268.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP032667.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_CP032679.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655/NZ_LR881938.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655_TMP32XR1/NZ_CP012869.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._MG1655_TMP32XR2/NZ_CP012870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._RV308/NZ_LM995446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-12_substr._W3110/NC_007779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K-15KW01/NZ_CP016358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K1508/NZ_CP072054.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K56-43-un/NZ_CP053079.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K71-77/NZ_CP040886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KBN10P04869/NZ_CP026473.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KC-Dl-1/NZ_CP063046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KCJ3K291/NZ_CP054407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KFu014/NZ_CP091712.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KFu019/NZ_CP091700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KFu022/NZ_CP091677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KFu023/NZ_CP091662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KFu024/NZ_CP091659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KGu016/NZ_CP091704.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KKa019/NZ_CP091664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KKo003/NZ_CP091682.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KKo009/NZ_CP091673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KL53/NZ_CP030919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KLY/NZ_CP008801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KMi006/NZ_CP091691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KMi025/NZ_CP091652.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KMS2001/NZ_CP079992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KMS2005/NZ_CP079991.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KO11FL/NC_017660.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KOr019/NZ_CP091665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KOr021/NZ_CP091703.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KR2009/NZ_CP040664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KS-NP019/NZ_AP024123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KSC1031/NZ_CP019560.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KSC207/NZ_CP019558.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KSC64/NZ_CP018840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KSC9/NZ_CP018323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KTa008/NZ_CP091651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/KV7/NZ_LT795502.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/K_EC180/NZ_CP062203.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L1_E925/NZ_LR883050.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L2_E1649/NZ_LR882973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L3452210II/NZ_CP076527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L37/NZ_CP034589.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L3Cip3/NZ_CP062211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L3_CS7_E2980/NZ_LR882978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L3_E36_ETEC/NZ_LR882997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L4_E1441_ETEC/NZ_LR883012.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L5_E1779_ETEC/NZ_LR883006.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L65/NZ_CP034738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L725/NZ_CP036202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L73/NZ_CP033378.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/L7_E1373_ETEC/NZ_LR882990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LAU-OXA/NZ_CP045277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LC-1302-2020/NZ_CP091756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD22-1/NZ_CP047876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD26-1/NZ_CP047665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD27-1/NZ_CP047594.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD39-1/NZ_CP047658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD67-1/NZ_CP061185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD91-1/NZ_CP042585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LD93-1/NZ_CP047662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LF82/NC_011993.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LF82/NZ_CP082771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LHM10-1/NZ_CP037903.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LIM/NZ_CP026027.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LS4/NZ_CP024717.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LSN03-016011_A/NZ_CP035884.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LSU61/NZ_CP038336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LWY24/NZ_CP054556.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LWY6/NZ_CP072204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LY180/NC_022364.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/LZYWN_4/NZ_CP046416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M-17/NZ_CP068394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M00057/NZ_CP061339.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M1/NZ_CP010180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M10/NZ_CP010200.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M11/NZ_CP010206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M11957/NZ_CP061337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M1300706001A/NZ_CP089272.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M1300706002/NZ_CP089270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M15/NZ_CP010213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M159-1/NZ_CP058719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M160133/NZ_CP022164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M16807/NZ_CP031256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M172-1/NZ_CP058714.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M18/NZ_CP010219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M19/NZ_CP010221.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M1_5/NZ_CP053296.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M217/NZ_AP019189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M24/NZ_CP080120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M3/NZ_CP010183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M45/NZ_CP080119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M505/NZ_AP023219.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M6/NZ_CP010186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M70/NZ_CP080118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M7424/NZ_CP063153.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M7638/NZ_CP040313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M8/NZ_CP010191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M8/NZ_CP019953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M9/NZ_CP010196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/M9682-C1/NZ_CP024275.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MA11/NZ_CP040314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MB41-1/NZ_CP039834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MB9-1/NZ_CP040107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MBT-5/NZ_CP058682.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MCJCHV-1/NZ_CP030111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MDR_56/NZ_CP019903.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ME8067/NZ_CP028703.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MFDpir/NZ_CP060708.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MFDS1006657/NZ_CP073589.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MG1655-377GD/NZ_CP092037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MGY/NZ_CP019629.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN10/NZ_CP069677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN11/NZ_CP069666.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN12/NZ_CP069657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN14/NZ_CP069646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN6/NZ_CP069692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MIN9/NZ_CP069682.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_1A-sc-2280431/NZ_LR890326.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_1D-sc-2280456/NZ_LR890198.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_2E-sc-2280463/NZ_LR890651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_7C-sc-2280452/NZ_LR890270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_8D-sc-2280460/NZ_LR890606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MINF_9A-sc-2280436/NZ_LR890334.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ML35/NZ_CP025747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MNCRE44/NZ_CP010876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRSN346355/NZ_CP018121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRSN346595/NZ_CP018109.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRSN346638/NZ_CP018115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRSN346647/NZ_CP018206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRSN352231/NZ_CP018103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRY15-117/NZ_AP017617.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MRY15-131/NZ_AP017620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS1170/NZ_CP056077.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS14384/NZ_LR130562.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS14385/NZ_LR130555.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS14386/NZ_LR130552.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS14387/NZ_LR130564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS6192/NZ_CP054940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS6193/NZ_CP047405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS6198/NZ_CP015834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS7163/NZ_CP026853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MS8345/NZ_CP025401.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_1A-sc-2280383/NZ_LR890288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_1D-sc-2280324/NZ_LR898868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_3B-sc-2280406/NZ_LR890299.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_4E-sc-2280348/NZ_LR890693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_4I-sc-2280412/NZ_LR898874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_6C-sc-2280315/NZ_LR890714.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_7A-sc-2280394/NZ_LR890466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_7C-sc-2280317/NZ_LR890405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_8B-sc-2280300/NZ_LR890536.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_8G-sc-2280388/NZ_LR890603.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_9D-sc-2280338/NZ_LR890349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB1_9I-sc-2280417/NZ_LR890508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSB2_1A-sc-2280429/NZ_LR890410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSHS_133/NZ_CP041304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSHS_472/NZ_CP041302.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MSI001/NZ_CP076645.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/MT102/NZ_CP034953.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/N18-04097/NZ_CP047277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/N3/NZ_CP031609.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/N8B7-2/NZ_CP038333.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NADC_5570_86-24_6564/NZ_CP017251.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NADC_5570_86-24_6565/NZ_CP017249.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NBRC_3301/NZ_CP048439.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NC101/NZ_CP072787.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NC19/NZ_CP076654.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCCP12480/NZ_CP068680.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCCP15648/NZ_CP009050.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCCP_15955/NZ_CP041749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCM3722/NZ_CP011495.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC10444/NZ_LR134092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC10537/NZ_LR134248.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC10973/NZ_LR134216.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC11023/NZ_LS483297.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC11121/NZ_LR134220.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC11129/NZ_LR134222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC11151/NZ_LR134031.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC11476/NZ_LR134209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC122/NZ_LT906474.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC86/NZ_CP019778.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC86/NZ_LR217818.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC8623/NZ_LR134234.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC86EC/NZ_LT601384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9008/NZ_LR134236.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9022/NZ_LR134237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9038/NZ_LR134082.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9040/NZ_LR134247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9041/NZ_LR134296.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9064/NZ_LR134315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9066/NZ_LR134000.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9080/NZ_LR134295.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9082/NZ_LR134080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9084/NZ_LR134075.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9087/NZ_LR134231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9088/NZ_LR134226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9100/NZ_LR134239.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9102/NZ_LR134227.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9104/NZ_LR134152.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9107/NZ_LR134240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9112/NZ_LR134079.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9113/NZ_LR134311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9699/NZ_LR134228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9966/NZ_LS483296.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NCTC9967/NZ_LS483303.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM1/NZ_CP083888.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM2/NZ_CP083886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM3/NZ_CP083880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM4/NZ_CP083877.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM5/NZ_CP083872.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM6/NZ_CP083869.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM_11.16372/NZ_CP082129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NDM_12_14482/NZ_CP082127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NE1127/NZ_CP038321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NE122/NZ_CP038319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NE92/NZ_CP038316.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NEB10-beta/NZ_CP053604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NEB5-alpha_F'Iq/NZ_CP053607.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NEBExpress/NZ_CP053599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NEBExpress_Iq/NZ_CP053592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NEB_Turbo/NZ_CP053605.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NE_1092-2/NZ_CP038328.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NE_1169-1/NZ_CP038324.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NGF1/NZ_CP016007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Nico21(DE3)/NZ_CP053600.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NIPH17_0020/NZ_AP024560.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NIPH17_0036/NZ_AP024561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NissleGFP_pZE21/NZ_CP035486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Nissle_1917/NZ_CP007799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Nissle_1917/NZ_CP022686.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Nissle_1917/NZ_CP058217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NMBU-W10C18/NZ_CP044403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NMBU-W12E19/NZ_CP042892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NMBU-W13E19/NZ_CP043406.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NMBU_W05E18/NZ_CP042878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NMBU__W06E18/NZ_CP047609.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/no107/NZ_CP075374.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NQ3/NZ_CP024720.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NRG_857C/NC_017634.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NS-NP030/NZ_AP024126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1F10/NZ_CP075462.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1F25/NZ_CP075472.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1F31/NZ_CP045190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1F34/NZ_CP075487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1N25/NZ_CP075484.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1N28/NZ_CP075474.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1N31/NZ_CP075480.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1N34/NZ_CP075465.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1W22/NZ_CP075469.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NT1W25/NZ_CP075477.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NU14/NZ_CP019777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NZRM3614/NZ_CP032793.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NZRM4165/NZ_CP032791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/NZRM4169/NZ_CP032789.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/O177:H21/NZ_CP016546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/O18/NZ_CP007275.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/O25b:H4/NZ_CP015085.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/OK1/NZ_CP038313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Origami2-DE3/NZ_CP084529.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/OSUCMP42NDM/NZ_CP087578.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/OV24/NZ_CP080117.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/OW1E2/NZ_CP067245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/p10A/NZ_CP049081.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/p11A/NZ_CP049077.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P13-6/NZ_CP080223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P15-25/NZ_CP080233.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P15-385/NZ_CP075627.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P19_598/NZ_CP067241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P21_SE1_04.20/NZ_CP085074.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P24_WM1_05.20/NZ_CP085056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P276M/NZ_CP044293.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P27_L1_05.20/NZ_CP085069.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P30_L1_05.20/NZ_CP085060.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/P31_YH1_02.21/NZ_CP085063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/p4A/NZ_CP049085.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PA20/NZ_CP017669.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PA45B/NZ_CP021288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PapM-32-1/NZ_CP049204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PapM-36-2/NZ_CP050646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PapRG-04-4/NZ_CP049201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PapRG-06-3/NZ_CP048604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PapRG-06-5/NZ_CP048605.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PAR/NZ_CP012379.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PC-NDM34/NZ_CP082120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PC6/NZ_CP080253.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PCN033/NZ_CP006632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PCN061/NZ_CP006636.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PE15/NZ_CP041628.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PF9285/NZ_CP038791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180050/NZ_CP043211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180052/NZ_CP043209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180053/NZ_CP043207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180054/NZ_CP043205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180057/NZ_CP043181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180058/NZ_CP043203.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180059/NZ_CP043201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180060/NZ_CP043199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180061/NZ_CP043197.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180062/NZ_CP043195.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180063/NZ_CP043193.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180170/NZ_CP043187.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180171/NZ_CP043185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180172/NZ_CP043183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180173/NZ_CP043191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PG20180175/NZ_CP043189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PigCaeca_1/NZ_CP039298.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PigCaeca_2/NZ_CP038857.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PJ-T13/NZ_CP087110.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK5074/NZ_CP072802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK5086/NZ_CP080370.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK6/NZ_CP042588.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8217/NZ_CP080121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8241/NZ_CP080139.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8261/NZ_CP080155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8275/NZ_CP080163.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8276/NZ_CP080146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8277/NZ_CP080133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8566/NZ_CP080172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PK8568/NZ_CP080126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PSUO2/NZ_CP011915.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PT109/NZ_CP041031.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PT62/NZ_CP090448.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PU-1/NZ_CP042246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/PV15-279/NZ_AP018488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/QDFD216/NZ_CP053211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/QEC11-421/NZ_CP048821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR881940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR882057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR882052.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR882050.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR882493.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Quinolone_resistant/NZ_LR883965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/R1/NZ_CP031706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Rafaela_II/NZ_CP076230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RCAD0514/NZ_CP034106.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/REL606/NC_012967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-EA06-22/NZ_CP062910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-EB12-05/NZ_CP062908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-ER01-35/NZ_CP062901.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-ER02-39/NZ_CP062895.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-ER07-18/NZ_CP062889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEA06-10/NZ_CP062883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEA12-26/NZ_CP062876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEA14-20/NZ_CP062871.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB01-20/NZ_CP062868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB08-01/NZ_CP062865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB14-36/NZ_CP062863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB17-18/NZ_CP062855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB20-37-A/NZ_CP062853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PEB21-38-A/NZ_CP062851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PER02-33/NZ_CP062838.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PER04-33/NZ_CP062832.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PER12-33-A/NZ_CP062826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Lact-PER13-34/NZ_CP062824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Sevr-PER06-05-b-A/NZ_CP062815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Res13-Sevr-PER07-33/NZ_CP062811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C011532/NZ_CP068795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C012087/NZ_CP068796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C017887/NZ_CP068799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C017997/NZ_CP068800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018220/NZ_CP068801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018249/NZ_CP068802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018404/NZ_CP068803.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018563/NZ_CP068804.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018567/NZ_CP068805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018576/NZ_CP068806.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018583/NZ_CP068807.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018699/NZ_CP068808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C018707/NZ_CP068830.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C019217/NZ_CP087377.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028497/NZ_CP068809.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028536/NZ_CP068810.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028568/NZ_CP068811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028613/NZ_CP068814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028620/NZ_CP068815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028724/NZ_CP068816.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028786/NZ_CP068817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C028803/NZ_CP068818.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029020/NZ_CP068812.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029033/NZ_CP068813.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029042/NZ_CP068819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029324/NZ_CP068822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029494/NZ_CP068823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029951/NZ_CP068820.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C029952/NZ_CP068821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030197/NZ_CP068824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030256/NZ_CP068825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030300/NZ_CP068826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030371/NZ_CP068827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030443/NZ_CP068828.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C030453/NZ_CP068829.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C036569/NZ_CP086627.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C037134/NZ_CP086626.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C037166/NZ_CP086625.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C037791/NZ_CP086624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C037794/NZ_CP086623.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C037884/NZ_CP086622.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C039205/NZ_CP086620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C039227/NZ_CP086618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RIVM_C039629/NZ_CP086616.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10042/NZ_CP028122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10386/NZ_CP028126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10410/NZ_CP042350.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10425-C1/NZ_CP031343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10466/NZ_CP028381.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10740/NZ_CP043821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM10809-3/NZ_CP023165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM11626-C1/NZ_CP035768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM11911/NZ_CP044313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12275-C1/NZ_CP031341.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12367-C1/NZ_CP031345.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12522-C8/NZ_CP035767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12579/NC_017656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12581/NZ_CP007136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM12761/NZ_CP007133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM13322/NZ_CP050498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM13514/NZ_CP006027.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM13516/NZ_CP006262.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM13745/NZ_CP044312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM13752/NZ_CP044311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM14721/NZ_CP027105.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM14723/NZ_CP027103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM19259/NZ_CP046527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM7190/NZ_CP015244.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8082/NZ_CP043825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8352/NZ_CP028110.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8385/NZ_CP028112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8426/NZ_CP028116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8843-C1/NZ_CP035772.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8988-C1/NZ_CP035770.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM8995-C1/NZ_CP031355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9088/NZ_CP042298.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9131/NZ_CP028429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9154-C1/NZ_CP031353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9245/NZ_CP044314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9322/NZ_CP028117.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9387/NZ_CP009104.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9467-C1/NZ_CP031349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9872/NZ_CP028379.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9872-C1/NZ_CP024659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9873-C1/NZ_CP031347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RM9975/NZ_CP028432.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Rosetta2-DE3/NZ_CP083274.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RP437/NZ_CP083410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RR1/NZ_CP011113.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RS571/NZ_CP034389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/RUT3575/NZ_CP048010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S-P-C-029.01/NZ_CP092702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S1/NZ_CP010226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S10/NZ_CP010229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S103EC/NZ_CP076693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S10EC/NZ_CP076697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S163-2/NZ_CP058709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S17-1/NZ_CP040667.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S171-1/NZ_CP058725.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S19-141/NZ_CP075640.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S19-651/NZ_CP075852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S21/NZ_CP010230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S21EC/NZ_CP076689.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S22EC/NZ_CP076687.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S3/NZ_CP010228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S30/NZ_CP010231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S40/NZ_CP010235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S42/NZ_CP010236.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S43/NZ_CP010237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S50/NZ_CP010238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S51/NZ_CP015995.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S56/NZ_CP010242.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S65EC/NZ_CP036245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/S88/NC_011742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SA186/NZ_CP022730.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Sakai_substr._RIMD_0509952/NC_002695.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Sanji/NZ_CP011061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Santai/NZ_CP007592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SaT040/NZ_CP014495.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SB0258h1/NZ_CP071954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SBF22/NZ_CP067235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SBF45/NZ_CP067232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC406/NZ_LR778140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC407/NZ_LR778141.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC418/NZ_LR778147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC422/NZ_LR778143.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC423/NZ_LR778145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC434/NZ_LR778142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC455/NZ_LR778146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC467/NZ_LR782232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC475/NZ_LR778148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC476/NZ_LR778151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC477/NZ_LR778152.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC480/NZ_LR778153.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC487/NZ_LR778149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC492/NZ_LR778150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SC516/NZ_CP025048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCAID_URN1-2021_(19_278)/NZ_CP082824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCAID_WND1-2021_(1_128)/NZ_CP082831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCAID_WND2-2021_(3_145)/NZ_CP082827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCEC020001/NZ_CP032426.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCEC020007/NZ_CP025627.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCEC020022/NZ_CP032892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCEC020023/NZ_CP025950.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCEC020026/NZ_CP034958.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCPM-O-B-9427/NZ_CP086259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-101/NZ_CP048920.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-101/NZ_CP051849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-102/NZ_CP051753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-104/NZ_CP053284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-105/NZ_CP051738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-106/NZ_CP053234.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-108/NZ_CP051735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-109/NZ_CP051733.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-111/NZ_CP051727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-112/NZ_CP051725.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-113/NZ_CP051765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-115/NZ_CP054368.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-116/NZ_CP051719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-118/NZ_CP051716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-122/NZ_CP051714.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-123/NZ_CP051711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-124/NZ_CP051706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-125/NZ_CP051700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-152/NZ_CP051698.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-171/NZ_CP054363.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-172/NZ_CP054353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-176/NZ_CP054345.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-204/NZ_CP053251.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-301/NZ_CP051751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-306/NZ_CP053231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-308/NZ_CP053281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-313/NZ_CP051694.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-316/NZ_CP054371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-387/NZ_CP051688.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-482/NZ_CP053247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-484/NZ_CP051744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCU-485/NZ_CP053245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SCZE5/NZ_CP051222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SD134209/NZ_CP029692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SE11/NC_011415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SE15/NC_013654.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SEC470/NZ_CP007594.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SF-088/NZ_CP012635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SF-166/NZ_CP012633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SF-173/NZ_CP012631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SF-468/NZ_CP012625.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SFE8/NZ_CP051219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SH19PTE6/NZ_CP073768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Show_KS_470-1/NZ_CP038309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SI-NP020/NZ_AP024130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SJ7/NZ_CP044315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SJC33/NZ_CP080255.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SK1144/NZ_AP018784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SL112/NZ_CP043486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SLK172/NZ_CP017631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SMBL/NZ_CP061914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SMN013SH2/NZ_CP023673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SMN152SH1/NZ_CP024618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SMN197SH3/NZ_CP024056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SMS-3-5/NC_010498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SQ110/NZ_CP011322.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SQ2203/NZ_CP011324.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SQ37/NZ_CP011320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SQ88/NZ_CP011321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SRCC_1675/NZ_CP015023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SS17/NZ_CP008805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SS52/NZ_CP010304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SS_NE_1040-1/NZ_CP038305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SS_TX_313-1/NZ_CP038302.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SS_TX_754-1/NZ_CP038300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST130/NZ_CP043478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST131/NZ_CP063774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST18/NZ_CP060709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST2350/NZ_CP031321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST2747/NZ_CP007392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST2747/NZ_CP007393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST2747/NZ_CP007394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST540/NZ_CP007265.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST540/NZ_CP007390.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST540/NZ_CP007391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST648/NZ_CP008697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST865/NZ_CP077649.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST866/NZ_CP077646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ST95-32/NZ_CP043950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STB20-1/NZ_CP050173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Stbl4/NZ_CP076043.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Stbl4/NZ_CP081009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC005/NZ_CP041437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC17-183/NZ_CP075652.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC17-190/NZ_CP075649.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC2017-197/NZ_CP075663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC2018-553/NZ_CP075665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC2018-607/NZ_CP075697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC276/NZ_CP091022.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC299/NZ_CP022279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC306/NZ_CP091016.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC307/NZ_CP091018.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC308/NZ_CP091020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC309/NZ_CP041435.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC313/NZ_CP041433.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC316/NZ_CP041431.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC346/NZ_CP091044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC367/NZ_CP041429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC379/NZ_CP091024.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC383/NZ_CP091026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC384/NZ_CP091029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC388/NZ_CP041425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC409/NZ_CP041422.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC711/NZ_CP041416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STEC719/NZ_CP041411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STIN_95/NZ_CP054965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STLEFF_29/NZ_CP058879.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STLEFF_47/NZ_CP058874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STLIN_13/NZ_CP058748.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STN0717-11/NZ_AP022478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STN0717-20/NZ_AP022482.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STO_Bone1B/NZ_CP054169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STO_Bone2/NZ_CP054161.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/strain_Z247/NZ_CP021207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/STW0522-31/NZ_AP022409.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SUISSEKPC3NDM5/NZ_CP083701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Survcare045/NZ_CP076318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Survcare253/NZ_CP076305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Survcare321/NZ_CP076301.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWHEFF_58/NZ_CP055048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWHEFF_68/NZ_CP055022.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWHEFF_69/NZ_CP055019.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWYH.B349aT/NZ_CP091773.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWYHX138.261ft/NZ_CP091839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SWYHX138.33gt/NZ_CP091836.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Sx181-0-1/NZ_CP078137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SY3626C1/NZ_CP059043.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SY3626C5/NZ_CP058948.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SY3626_hybrid/NZ_CP059283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/SYNB1934/NZ_CP080246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/T159A/NZ_CP072986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/T16R/NZ_CP046716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/T28R/NZ_CP049353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/T7Express_Crystal/NZ_CP053594.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/T7Express_LysY/NZ_CP053597.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TA8571/NZ_AP024205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TB182A/NZ_CP038295.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TB21-1/NZ_CP038292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TCM3e1/NZ_CP066806.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/tcmA_3/NZ_CP059835.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TCM_B_2/NZ_CP075061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TD225-C4/NZ_CP035860.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TH9F11/NZ_CP054191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/THO-003/NZ_AP022525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/THO-006/NZ_AP022533.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/THO-008/NZ_AP022536.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/THO-010/NZ_AP022540.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/THO-015/NZ_AP022549.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TKEC21-17/NZ_CP092449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TKEC21-59/NZ_CP092452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/tolC-/NZ_CP018801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TOP10/NZ_CP080620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TR01/NZ_CP033605.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Trh52/NZ_CP047600.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TT12A/NZ_CP038496.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TT12B/NZ_CP038494.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TUM18530/NZ_AP023190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TUM18780/NZ_AP023197.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TUM18781/NZ_AP023205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TW10590/NZ_CP035850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TW10598/NZ_CP035846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TW10722/NZ_CP035841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TW11681/NZ_CP035855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TW14359/NC_013008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TX_265-1/NZ_CP038290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/TX_376-2/NZ_CP038287.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U1/NZ_CP041359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U12A/NZ_CP035476.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U13A/NZ_CP035477.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U14A/NZ_CP035516.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U15A/NZ_CP035720.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/U90/NZ_CP068035.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UK_Dog_Liverpool/NZ_CP031653.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/uk_P46212/NZ_CP013658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UM146/NC_017632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UMB200201_11/NZ_CP049343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UMNK88/NC_017641.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC-455U_190220/NZ_CP049839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC-774U_190308/NZ_CP049852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC129/NZ_CP071522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC132/NZ_CP040456.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC_26-1/NZ_CP016497.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UPEC_U013/NZ_CP058596.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UR21_11-869/NZ_OU342919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/USDA5905/NZ_CP039837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/USML2/NZ_CP011124.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UTI89/NC_007946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UTI89/NZ_CP062228.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UTI89/NZ_CP062985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/UTI89/NZ_CP064825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Vac_07.1/NZ_CP076241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/VCSW5/NZ_CP082355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/VCSW7/NZ_CP082357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/VCSW8/NZ_CP085488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/VR50/NZ_CP011134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W/NC_017635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W/NC_017664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W13-16/NZ_CP080235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W2-5/NZ_CP032989.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W25K/NZ_CP091038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W3110-P/NZ_CP084899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W5-6/NZ_CP032992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/W60/NZ_CP058342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WAT/NZ_CP012380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WB61/NZ_CP020025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC000837/NZ_CP032879.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC005215/NZ_CP092973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC005237/NZ_CP026580.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC005784/NZ_CP028578.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC020031/NZ_CP033401.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC020032/NZ_CP034966.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC025157/NZ_CP043279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC025943/NZ_CP027205.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC025970/NZ_CP036177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC025985/NZ_CP043284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC035053S1G0/NZ_CP034595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC035123/NZ_CP058651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC035125/NZ_CP058656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC035148/NZ_CP029369.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC035148/NZ_CP043274.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC040047/NZ_CP043295.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC050613/NZ_CP019213.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC115106/NZ_CP043334.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC4533/NZ_CP028589.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WCHEC96200/NZ_CP022229.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WEM25/NZ_CP061530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WG5/NZ_CP024090.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WI1_isolate/NZ_LT838196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WI2_isolate/NZ_LT838200.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Wll001/NZ_CP064167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP2-S18-ESBL-07/NZ_AP021944.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP2-S18-ESBL-08/NZ_AP021946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP2-W18-CRE-03/NZ_AP021914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP2-W18-ESBL-08/NZ_AP021933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP2-W18-ESBL-10/NZ_AP021935.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-S18-ESBL-07/NZ_AP022030.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-S18-ESBL-08/NZ_AP022032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-S18-ESBL-09/NZ_AP022036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-W18-CRE-03/NZ_AP021963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-W18-ESBL-07/NZ_AP021998.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP3-W18-ESBL-08/NZ_AP022003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-S17-ESBL-03/NZ_AP022044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-S17-ESBL-08/NZ_AP022049.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-S18-ESBL-07/NZ_AP022087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-S18-ESBL-08/NZ_AP022095.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-S18-ESBL-09/NZ_AP022098.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-W18-ESBL-08/NZ_AP022064.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP4-W18-ESBL-09/NZ_AP022069.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-S17-ESBL-10/NZ_AP022105.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-S17-ESBL-11/NZ_AP022107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-S18-ESBL-07/NZ_AP022159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-S18-ESBL-08/NZ_AP022161.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-S18-ESBL-09/NZ_AP022171.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP5-W18-ESBL-11/NZ_AP022120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP7-S17-ESBL-01/NZ_AP022173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP7-S17-ESBL-04/NZ_AP022177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP7-S18-ESBL-09/NZ_AP022207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP8-S17-ESBL-05/NZ_AP022215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP8-S17-ESBL-08/NZ_AP022221.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP8-S17-ESBL-12/NZ_AP022222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP8-S18-CRE-02/NZ_AP022245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP8-S18-ESBL-07/NZ_AP022261.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WP9-S17-ESBL-11/NZ_AP022287.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WS0115A/NZ_CP035882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WS1933D/NZ_CP035836.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WS3294A/NZ_CP035819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WS4202/NZ_CP012802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WS4437A-1/NZ_CP035822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/WW2/NZ_CP081881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XD35/NZ_CP089142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XDL/NZ_CP043033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XH1815/NZ_CP069386.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XH988/NZ_CP019355.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XH990/NZ_CP019359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XH992/NZ_CP019392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XH993/NZ_CP019361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XJW9B263/NZ_CP067399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XJW9B277/NZ_CP068041.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XL1-Blue/NZ_CP081007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/XL10-gold-ultracompetent/NZ_CP081008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Xuzhou21/NC_017906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Y5/NZ_CP013483.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YB14-1/NZ_CP038284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YD786/NZ_CP013112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YDC107/NZ_CP025707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YJ1/NZ_AP023224.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YJ3/NZ_AP023226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YJ4/NZ_AP023230.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YJ6/NZ_AP023235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YPE10/NZ_CP041448.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YPE12/NZ_CP041442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YPE3/NZ_CP041452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YSP8-1/NZ_CP037910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/YY76-1/NZ_CP040927.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1002/NZ_CP021202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1486/NZ_CP062749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1504/NZ_CP062746.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1615/NZ_CP062744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1626/NZ_CP062742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1723/NZ_CP062739.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1766/NZ_CP062736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1767/NZ_CP062733.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1768/NZ_CP062731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1769/NZ_CP062780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1811/NZ_CP062729.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1812/NZ_CP062727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1813/NZ_CP062725.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1814/NZ_CP062723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1815/NZ_CP062721.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1816/NZ_CP062719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1825/NZ_CP062717.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1826/NZ_CP062715.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1830/NZ_CP062713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1831/NZ_CP062711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1832/NZ_CP062708.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1833/NZ_CP062705.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1834/NZ_CP062702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1835/NZ_CP062700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z1836/NZ_CP062778.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z30/NZ_CP066844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z503/NZ_CP090305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z563/NZ_CP062774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z570/NZ_CP062771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z852/NZ_CP062769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z866/NZ_CP062766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z869/NZ_CP062782.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z885/NZ_CP062763.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z887/NZ_CP062761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z892/NZ_CP062758.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z903/NZ_CP062755.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/Z910/NZ_CP062752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZF31/NZ_CP047455.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZF34/NZ_CP047461.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZH063/NZ_CP014522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZH193/NZ_CP014497.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZJ3920/NZ_CP020545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/coli/ZLWT/NZ_CP066144.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/5zf15-2-1/NZ_CP079891.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/6S41-1/NZ_CP079884.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/ATCC_35469T/NC_011740.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/ATCC_35470/NZ_CP042946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/ATCC_35471/NZ_CP042945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/ATCC_35473/NZ_CP042942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/EF20JDJ4045/NZ_CP086604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/EF31/NZ_CP070951.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/EF44/NZ_CP070954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/EF91/NZ_CP070873.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/EFCF056/NZ_CP040805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/FDAARGOS_1032/NZ_CP066085.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/FDAARGOS_1438/NZ_CP077242.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/FDAARGOS_1499/NZ_CP083638.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/HNCF11W/NZ_CP053045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB02-C22/NZ_CP058057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB10-C04/NZ_CP057918.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB10-C23/NZ_CP057896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB17-C11/NZ_CP057692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB18-C04/NZ_CP057681.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB19-C05/NZ_CP057657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB23-C01/NZ_CP057565.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB23-C02/NZ_CP057561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB24-C06/NZ_CP057505.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB24-C08/NZ_CP057504.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB25-C10/NZ_CP057481.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB26-C03/NZ_CP057463.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB26-C09/NZ_CP057458.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB28-C13/NZ_CP055675.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB28-C21/NZ_CP057357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB30-C07/NZ_CP057311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB31-C13/NZ_CP057266.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB31-C17/NZ_CP057256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB32-C05/NZ_CP057243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB32-C09/NZ_CP057238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB32-C16/NZ_CP057229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB32-C22/NZ_CP057220.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB33-C04/NZ_CP057215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB33-C07/NZ_CP057210.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB38-C01/NZ_CP057104.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB38-C04/NZ_CP057101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB38-C07/NZ_CP057093.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB41-C15/NZ_CP056968.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB41-C20/NZ_CP056961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB41-C23/NZ_CP055599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB42-C01/NZ_CP056951.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB42-C13/NZ_CP056936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/fergusonii/RHB42-C24/NZ_CP056916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/HT073016/NZ_CP025979.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/NCTC11133/NZ_LR134340.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/NCTC8196/NZ_LR134270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/RHB24-C12/NZ_CP058209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/RHB42-C09/NZ_CP058207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/RHBSTW-00605/NZ_CP056697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/RHBSTW-00777/NZ_CP056165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/RHBSTW-00814/NZ_CP056159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/marmotae/W49-2/NZ_CP093239.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/sp./E4742/NZ_CP040443.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/sp./SCLE84/NZ_CP051430.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/sp./TC-EC600-tetX4/NZ_CP079924.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Escherichia/sp./TM-G17TGC/NZ_CP080247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Gibbsiella/quercinecans/FRB97/NZ_CP014136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/035/NZ_CP050067.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/18-2341/NZ_CP049600.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/AR_0007/NZ_CP024883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/AR_0009/NZ_CP024885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/AR_0018/NZ_CP024880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/AR_0062/NZ_CP026756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/AUH-KAM-9/NZ_CP048598.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/C9/NZ_CP042530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/CAV1320/NZ_CP011574.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/EA1509E/NC_020181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/EA46506/NZ_CP070520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_139/NZ_CP014748.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_1441/NZ_CP077265.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_1442/NZ_CP077293.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_152/NZ_CP014029.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_327/NZ_CP031756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_363/NZ_CP023963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_513/NZ_CP033817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/FDAARGOS_641/NZ_CP044083.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/G7/NZ_CP011539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/HNHF1/NZ_CP047669.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/Ka37751/NZ_CP041925.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/KAE3SP/NZ_CP082898.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/KA_P10_L5_03.19/NZ_CP044214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/KCTC_2190/NC_015663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/LU2/NZ_CP035466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/MINF_10B-sc-2280448/NZ_LR890184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/NCTC9644/NZ_LR134254.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/NCTC9735/NZ_LR134475.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/NTT31XS/NZ_CP077429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/RHBSTW-00898/NZ_CP056260.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/WP5-W18-CRE-01/NZ_AP022108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/Y1/NZ_CP045870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/Y3/NZ_CP045869.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/aerogenes/Y6/NZ_CP045868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/africana/200023/NZ_CP084874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/africana/FF1003/NZ_CP059391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/2481359/NZ_CP067380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/5512.56/NZ_CP067391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/LC-1303-2020/NZ_CP091752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/M5a1/NZ_CP063301.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/NCTC9146_substr._serovar_casular_type_26_substr._serovar_casular_type_26/NZ_LR594038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/NY3/NZ_CP079754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00039/NZ_CP058180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00054/NZ_CP055412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00181/NZ_CP058112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00186/NZ_CP056760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00555/NZ_CP055315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00577/NZ_CP056701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00634/NZ_CP056682.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00853/NZ_CP056150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/RHBSTW-00866/NZ_CP055309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/grimontii/SS141/NZ_CP044527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/huaxiensis/WCHKl090001/NZ_CP036175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/12084/NZ_CP072119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/53828CZ/NZ_CP085764.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/7525/NZ_CP065474.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/8-1/NZ_CP089448.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/AKKL-001/NZ_CP060111.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/AR375/NZ_CP029141.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/B106/NZ_CP067093.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/BD-50-Km/NZ_CP061930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/BD177/NZ_CP048108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/C52/NZ_CP042545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/CCRI-24235/NZ_CP081351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/CZ598/NZ_CP073305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/E718/NC_018106.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/F107/NZ_CP024643.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/FDAARGOS_647/NZ_CP044109.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/HKOPL1/NZ_CP004887.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/JNQH491/NZ_CP075881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/K342/NZ_CP093010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/K516/NZ_CP022348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/K518/NZ_CP023185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/K92/NZ_CP089315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/KCTC_1686/NC_016612.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/KM41/NZ_CP090078.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/Kmfe267/NZ_CP071393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/KNU07/NZ_CP041515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/Kox101/NZ_CP089407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/Kox58/NZ_CP089395.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/KO_408/NZ_CP091470.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/LDS17/NZ_CP065338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/M1/NZ_CP008841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/M82255/NZ_CP035214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/RHB20-C02/NZ_CP058212.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/RHBSTW-00167/NZ_CP058118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/RHBSTW-00409/NZ_CP055325.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/RHBSTW-00909/NZ_CP056136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/Sb-24/NZ_CP073236.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/THO-011/NZ_AP022547.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/X2-1/NZ_CP051427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/YL6-1/NZ_CP084415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/michiganensis/YL9/NZ_CP084542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/AR380/NZ_CP029128.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/AR_0028/NZ_CP026715.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/AR_0147/NZ_CP020358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/CAV1015/NZ_CP017928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/CAV1099/NZ_CP011597.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/CAV1335/NZ_CP011618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/CAV1752/NZ_CP018362.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_1331/NZ_CP070144.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_1332/NZ_CP069943.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_1333/NZ_CP070020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_1334/NZ_CP069911.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_335/NZ_CP027426.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/FDAARGOS_500/NZ_CP033844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/IR5392/NZ_CP064108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KONIH1/NZ_CP008788.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KONIH2/NZ_CP026285.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KONIH4/NZ_CP026269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KONIH5/NZ_CP026275.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KONSF-1/NZ_CP090245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/KoPF10/NZ_CP072914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/Kox100/NZ_CP089411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/Kox26/NZ_CP089399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/Kox71/NZ_CP089392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/MSB1_10D-sc-2280340/NZ_LR890312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/NCTC11356/NZ_LR133932.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/NCTC13727/NZ_LR134333.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/P620/NZ_CP046115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/RHB30-C02/NZ_CP057330.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/oxytoca/RHBSTW-00373/NZ_CP056567.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pasteurii/Kox205/NZ_CP089403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/0113481141/NZ_CP083075.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/015625/NZ_CP033396.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/0213293972/NZ_CP083051.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/03-9138-2/NZ_CP084873.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/04A025/NZ_CP084871.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/06KM907/NZ_CP049719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/08EU827/NZ_CP025576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1-16/NZ_CP042321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/101-17/NZ_CP042311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/103-17/NZ_CP042310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1050/NZ_CP023416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1084/NC_018522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/109-17/NZ_CP042309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11/NZ_CP016923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11219/NZ_CP030174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11305/NZ_CP030315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11311/NZ_CP030313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11420/NZ_CP026023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11454/NZ_CP030311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11492/NZ_CP026021.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1158/NZ_CP006722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1194/NZ_CP089427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/11KM256/NZ_CP049716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/12/NZ_CP082765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/121/NZ_CP031849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/12208/NZ_CP030172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1230_0791_KP/NZ_CP083070.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/13190/NZ_CP026017.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/156070/NZ_CP047648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/158590/NZ_CP047649.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/160111/NZ_CP029689.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1613015451/NZ_CP083066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1613163991/NZ_CP083063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1613258639/NZ_CP083055.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1632/NZ_CP084497.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/16BU137/NZ_CP051161.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/16HN-263/NZ_CP045263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/16_GR_13/NZ_CP027036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1756/NZ_CP019219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18-2374/NZ_CP041927.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1864/NZ_CP084492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18CPO060/NZ_CP034778.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18KM1950/NZ_CP039936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18KM2445b/NZ_CP039951.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18KM2517/NZ_CP039939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/18KM2813/NZ_CP039942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19051/NZ_CP022023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19110124/NZ_CP064174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19KM1053/NZ_CP049950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19KM1109/NZ_CP049953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19KM267/NZ_CP049947.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19KM28/NZ_CP039945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19KM57/NZ_CP039948.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/19PDR22/NZ_CP051159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/1_GR_13/NZ_CP027042.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/200/NZ_CP055293.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2016-1400/NZ_CP065034.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2016_49/NZ_CP068015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2019036D/NZ_CP047336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/203/NZ_CP021165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/205880/NZ_CP030302.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/20_GR_12/NZ_CP027048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/23/NZ_CP016926.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/23/NZ_CP030320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/283747/NZ_CP030300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2e/NZ_CP028478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2N3/NZ_CP025541.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/2_GR_12/NZ_CP027053.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/30660_NJST258_1/NZ_CP006923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/30684_NJST258_2/NZ_CP006918.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/309074/NZ_CP030296.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/3214/NZ_CP028850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/33/NZ_CP034321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/3347684I/NZ_CP071080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/3347689I/NZ_CP071086.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/33Kpn12/NZ_CP062792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/33Kpn22/NZ_CP069046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/34618/NZ_CP010392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/36/NZ_CP082759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/37/NZ_CP082753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/39427/NZ_CP054268.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/400195-1-18/NZ_CP083047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/400195-2-18/NZ_CP083043.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/41968/NZ_CP068618.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/42362/NZ_CP068614.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/45706/NZ_CP068610.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/46/NZ_CP088120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/46-16/NZ_CP042318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/4743/NZ_CP033625.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/47733/NZ_CP050360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/4906.28/NZ_CP083036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/49088/NZ_CP088997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/49210/NZ_CP089024.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/4_1-2/NZ_CP023839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/5-16/NZ_CP042320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/500_1420/NZ_CP011980.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/50595/NZ_CP050371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/50700/NZ_CP088991.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/51015/NZ_CP050376.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/51135CZ/NZ_CP085740.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/5208.51/NZ_CP083032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/53322/NZ_CP088988.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/53334/NZ_CP088985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/53357/NZ_CP088982.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/53374/NZ_CP089014.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/555/NZ_CP043932.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/56575/NZ_CP089012.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/56661/NZ_CP089001.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/57/NZ_CP088126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/59062CZ/NZ_CP085729.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/59493/NZ_CP068602.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/616/NZ_CP026495.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6604.68/NZ_CP083026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6611.48/NZ_CP083021.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6709.15-I/NZ_CP083016.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6710.71/NZ_CP083011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6711.43/NZ_CP083007.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6712.08/NZ_CP083000.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/6911.28/NZ_CP082995.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/7008.20/NZ_CP082988.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/7011.62/NZ_CP082985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/721005/NZ_CP022997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/721005/NZ_CP030292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/74-17/NZ_CP042317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/825795-1/NZ_CP017985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/8695/NZ_CP085889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/89-17/NZ_CP042315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/90-17/NZ_CP042314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/90CM2/NZ_CP071819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/911021/NZ_CP022882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AATZP/NZ_CP014755.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ABFPV/NZ_CP036442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ABFQB/NZ_CP036438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AHSWKP25/NZ_CP091048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AJ218/NZ_LR130541.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AP8555/NZ_CP035383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR142_2b/NZ_CP039954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR376/NZ_CP029137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR438/NZ_CP029099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ARLG-3135/NZ_CP033946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0046/NZ_CP032222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0047/NZ_CP021539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0049/NZ_CP018816.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0066/NZ_CP026751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0068/NZ_CP020067.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0075/NZ_CP032185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0076/NZ_CP032167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0079/NZ_CP028994.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0087/NZ_CP029738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0096/NZ_CP027612.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0097/NZ_CP032200.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0098/NZ_CP020108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0107/NZ_CP021955.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0109/NZ_CP032207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0112/NZ_CP021549.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0113/NZ_CP021751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0115/NZ_CP020071.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0117/NZ_CP020061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0120/NZ_CP021833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0125/NZ_CP021859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0126/NZ_CP021740.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0129/NZ_CP021718.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0135/NZ_CP032178.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0138/NZ_CP021757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0139/NZ_CP021960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0140/NZ_CP029722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0141/NZ_CP028953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0142/NZ_CP028990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0143/NZ_CP021708.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0145/NZ_CP021939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0146/NZ_CP021685.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0148/NZ_CP021950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0152/NZ_CP021944.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0153/NZ_CP028932.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0158/NZ_CP021696.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0160/NZ_CP032175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0361/NZ_CP027160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_0363/NZ_CP027146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AR_362/NZ_CP030341.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ATCC_35657/NZ_CP015134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ATCC_43816/NZ_CP064352.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ATCC_43816_KPPR1/NZ_CP009208.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ATCC_700721__MGH_78578/NC_009648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ATCC_BAA-2146/NZ_CP006659.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AUSMDU00003562/NZ_CP025005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AUSMDU00008079/NZ_CP022691.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/AUSMDU00008119/NZ_CP025008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/b1-2L/NZ_CP072456.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B12(AN)/NZ_CP026155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B12097/NZ_CP070445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B13243/NZ_CP072403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B1347/NZ_CP070415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B17602/NZ_CP084168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B18064/NZ_CP070449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B28484/NZ_CP070459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B28906/NZ_CP070464.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B30155/NZ_CP074580.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B31/NZ_CP035929.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B32278/NZ_CP072413.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B33769/NZ_CP072422.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B3824/NZ_CP070425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B3923/NZ_CP070430.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B4650/NZ_CP072408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B5617/NZ_CP070435.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/B960/NZ_CP070410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA10835/NZ_CP053765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA12818/NZ_CP054170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA1559/NZ_CP036187.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA2105/NZ_CP060421.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA2275/NZ_CP053364.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA25665/NZ_CP059468.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA28434/NZ_CP036327.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA34918/NZ_CP036190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BA39649/NZ_CP058940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/bat15/NZ_CP092765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/bat16/NZ_CP092759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/bat17/NZ_CP092753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp021/NZ_CP050834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp067/NZ_CP050829.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp091/NZ_CP050822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp101/NZ_CP050840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp186/NZ_CP050843.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp206/NZ_CP050845.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Bckp212/NZ_CP050826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD-13/NZ_CP082978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD-14/NZ_CP082975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD-41/NZ_CP082967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD_DM_697/NZ_CP046939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD_DM_782/NZ_CP046381.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BD_DM_914/NZ_CP046949.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Beach_Ranger/NZ_CP065453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BIC-1/NZ_CP022573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BJ1/NZ_CP084870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BJCFK909/NZ_CP034123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BK13043/NZ_CP020837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BK13048/NZ_CP045015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/blood_sample_2/NZ_CP015822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM327-1/NZ_CP063906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM334-2/NZ_CP063904.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM336-2-1/NZ_CP063913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM337-1/NZ_CP063919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM338-1/NZ_CP063922.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BM343/NZ_CP063939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BP327/NZ_CP036335.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BP3636/NZ_CP053771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BR/NZ_CP015990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BR21/NZ_CP018885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BR7/NZ_CP018883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BS317-1/NZ_CP063936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BS326-3/NZ_CP063934.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BS329-2/NZ_CP063943.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BS361-1/NZ_CP063885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BS433-2/NZ_CP063890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BTS/NZ_CP066543.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/BWHC1/NZ_CP020498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C0674/NZ_CP061832.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C11/NZ_CP084103.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C1398_mutant_C1/NZ_CP080584.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C2/NZ_CP042520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C2244/NZ_CP071464.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C2414/NZ_CP039819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C2660/NZ_CP039808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C39/NZ_CP061700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C51/NZ_CP042481.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/C789/NZ_CP034415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/carbapenem-resistant_blaNDM-1/NZ_CP009114.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1016/NZ_CP017934.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1042/NZ_CP018671.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1193/NZ_CP013322.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1217/NZ_CP018676.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1344/NZ_CP011624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1392/NZ_CP011578.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1417/NZ_CP018352.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1453/NZ_CP018356.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CAV1596/NZ_CP011647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CCRI-22199/NZ_CP035540.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CCUG_70742/NZ_CP023249.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CDC_0106/NZ_CP022611.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CDI231/NZ_CP077779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CDI694/NZ_CP077773.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CFSAN054111/NZ_CP028176.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CG43/NC_022566.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CN1/NZ_CP015382.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CNR48/NZ_LS399318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/COL17/NZ_CP072905.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CP19/NZ_CP073351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CR-HvKP1/NZ_CP040533.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/cr-hvkp3/NZ_CP024707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CR-HvKP4/NZ_CP040539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CR-HvKP5/NZ_CP040545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CR14/NZ_CP015392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir116/NZ_CP063011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir120/NZ_CP063008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir154/NZ_CP063003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir197/NZ_CP062997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir26/NZ_CP062986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir45/NZ_CP063023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CriePir75/NZ_CP063019.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRK0298/NZ_CP034039.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRKP-1215/NZ_CP024838.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRKP-2297/NZ_CP024834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRKP52R/NZ_CP066249.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRKP66R/NZ_CP063833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CRKP78R/NZ_CP066254.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CT-04/NZ_CP081891.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/CY814036/NZ_CP093151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DA12090/NZ_CP030072.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DA33140/NZ_CP029582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DA33141/NZ_CP029587.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DA33144/NZ_CP029590.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DA48896/NZ_CP024429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DD521/NZ_CP075316.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DHQP1002001/NZ_CP016811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DHQP1605752_NV/NZ_CP022127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Dirty_Dan/NZ_CP065436.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DJ/NZ_OV049808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DMC1097/NZ_CP011976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/DY1928/NZ_CP090429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/EAN23/NZ_CP039957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/EAN40/NZ_CP039960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/EC-12536/NZ_LR025099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ED2/NZ_CP016813.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ED23/NZ_CP016814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/EH13/NZ_CP089097.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ent1077/NZ_CP060281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ENT1988/NZ_CP061350.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ER17974.3/NZ_CP059295.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ESBL_DR09/NZ_CP067255.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F1/NZ_CP026130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F11/NZ_CP092901.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F127/NZ_CP026140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F13/NZ_CP026162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F132/NZ_CP026145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F138/NZ_CP026149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F4/NZ_CP090397.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F44/NZ_CP025461.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F5/NZ_CP026132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F726925/NZ_CP081820.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F77/NZ_CP026136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F81/NZ_CP026164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F89-1/NZ_CP026159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F93-1/NZ_CP026160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F93-2/NZ_CP026157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/F94/NZ_CP090398.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1076/NZ_CP068139.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1305/NZ_CP069847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1306/NZ_CP070088.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1307/NZ_CP069962.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1308/NZ_CP069968.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1309/NZ_CP070015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1310/NZ_CP070158.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1311/NZ_CP069861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1312/NZ_CP069940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1313/NZ_CP069907.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1314/NZ_CP070105.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1315/NZ_CP070172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1316/NZ_CP070092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1317/NZ_CP069961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1318/NZ_CP069939.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1319/NZ_CP070046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1320/NZ_CP070169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1321/NZ_CP069994.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1322/NZ_CP070037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1323/NZ_CP069873.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1324/NZ_CP069853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1325/NZ_CP069946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1326/NZ_CP069821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1327/NZ_CP069828.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1328/NZ_CP070087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1329/NZ_CP069992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_1330/NZ_CP070177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_156/NZ_CP014123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_439/NZ_CP023913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_444/NZ_CP023941.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_446/NZ_CP023946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_531/NZ_CP033777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_566/NZ_CP033756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_629/NZ_CP044047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_630/NZ_CP044039.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_631/NZ_CP044033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FDAARGOS_775/NZ_CP040993.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/fekpn2511/NZ_CP068972.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FK_6768/NZ_CP065554.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FO15/NZ_CP073002.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/FRPDR/NZ_CP063759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/GD4/NZ_CP025951.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/GN-2/NZ_CP019160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Gordon/NZ_CP065460.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/GR390/NZ_CP075890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/GSU10-3/NZ_AP018671.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/GZ-1/NZ_CP031934.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/H11/NZ_CP018056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HB25-1/NZ_CP039524.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HC139/NZ_CP061842.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HK31/NZ_CP073906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HK787/NZ_CP006738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HKP0018/NZ_CP061062.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HKP0067/NZ_CP061053.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HKU57/NZ_CP063215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HS11286/NC_016845.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/HZW25/NZ_CP025211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/I72/NZ_CP034281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IA-021/NZ_CP081890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IA-044/NZ_CP081889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IA565/NZ_CP030070.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IDR1800050501-01-02/NZ_CP054285.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF008-sc-2279879/NZ_LR890202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF018-sc-2279888/NZ_LR890493.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF019/NZ_CP083796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF022-sc-2279895/NZ_LR890209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF032-sc-2279918/NZ_LR890456.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF042/NZ_CP024542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF044-sc-2279940/NZ_LR890395.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF049-sc-2279950/NZ_LR890360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF059/NZ_CP024545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF065-sc-2279980/NZ_LR890665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF072-sc-2279995/NZ_LR890185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF078/NZ_CP032831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF079-sc-2280008/NZ_LR890464.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF102-sc-2279897/NZ_LR890726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF116-sc-2279924/NZ_CP031792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF118/NZ_CP083775.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF119-sc-2279930/NZ_LR890691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF122-sc-2279936/NZ_LR890600.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF125-sc-2279943/NZ_CP031795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF133-sc-2279960/NZ_LR890189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF142-sc-2279977/NZ_LR890547.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF144-sc-2279979/NZ_LR890569.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF145-sc-2279981/NZ_LR890583.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF148-sc-2279987/NZ_LR890381.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF156-sc-2280002/NZ_LR890286.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF157/NZ_CP024528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF158/NZ_CP024521.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF161-sc-2280013/NZ_LR890719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF163/NZ_CP024549.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF164/NZ_CP024556.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF168-sc-2280023/NZ_LR890471.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF171-sc-2280028/NZ_LR890446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF174-sc-2280034/NZ_LR890245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF182-sc-2280049/NZ_LR890554.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF188-sc-2280054/NZ_LR890307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF201-sc-2280069/NZ_LR890686.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF206-sc-2280074/NZ_CP031808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF215-sc-2280087/NZ_LR890317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF223-sc-2280103/NZ_LR890595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF231-sc-2280121/NZ_LR890356.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF235-sc-2280127/NZ_CP031817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF237/NZ_CP032833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF242-sc-2280140/NZ_LR890624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF247-sc-2280148/NZ_LR890562.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF249/NZ_CP024489.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF250-sc-2280154/NZ_LR890374.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF266-sc-2280182/NZ_LR890648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF274/NZ_CP024570.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF277/NZ_LR890204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF278/NZ_CP024563.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF279-sc-2280202/NZ_LR890679.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF281-sc-2280206/NZ_LR890515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF298/NZ_LR890426.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF299-sc-2280084/NZ_LR890632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF302-sc-2280090/NZ_LR890556.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF305-sc-2280094/NZ_LR890702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF310-sc-2280104/NZ_LR890254.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF313-sc-2280106/NZ_LR890415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF319-sc-2280116/NZ_LR890542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF321-sc-2280120/NZ_LR890669.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF323-sc-2280125/NZ_LR890735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF327-sc-2280132/NZ_LR890571.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF329-sc-2280137/NZ_LR890240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF331-sc-2280141/NZ_LR890424.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF333/NZ_LR890384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF334-sc-2280143/NZ_LR890235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF341-sc-2280155/NZ_LR890490.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF344-sc-2280162/NZ_LR890640.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF346-sc-2280166/NZ_LR890643.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF348-sc-2280170/NZ_LR890323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF350-sc-2280174/NZ_LR890422.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF354-sc-2280183/NZ_LR890369.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF355-sc-2280185/NZ_LR890277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF357-sc-2280189/NZ_LR890217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF358-sc-2280191/NZ_LR890474.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF359-sc-2280194/NZ_LR890616.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/INF361-sc-2280198/NZ_LR890527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR1251/NZ_CP068572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR5065/NZ_CP061948.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR5086/NZ_CP061975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR5654/NZ_CP061952.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR5726/NZ_CP061957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IR5755/NZ_CP061970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IRQBAS102c/NZ_CP071150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IRQBAS103c/NZ_CP072825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IRQBAS105c/NZ_CP072826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/IRQBAS106c-genome/NZ_CP073782.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/J1/NZ_CP013711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JNK002/NZ_CP061961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JNKPN26/NZ_CP090203.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JNQH116/NZ_CP070898.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JNQH373/NZ_CP091978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JP20/NZ_CP079762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JS161205112/NZ_CP071324.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JS187/NZ_CP025466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JS706/NZ_CP070970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JUNP254/NZ_AP021880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-1/NZ_CP064252.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-10/NZ_CP064258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-11/NZ_CP064208.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-12/NZ_CP064205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-2/NZ_CP064246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-3/NZ_CP064241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-4/NZ_CP064235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-5/NZ_CP064232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-6/NZ_CP064229.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-7/NZ_CP064223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-8/NZ_CP064217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/JX-CR-hvKP-9/NZ_CP064211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K-1L/NZ_CP072460.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K186/NZ_CP076518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K191659/NZ_CP080346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K191663/NZ_CP080353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K191773/NZ_CP080362.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K2044/NZ_CP026011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K2606/NZ_CP047633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/K66-45/NZ_CP020901.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KAM260/NZ_AP023264.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KC-Pl-HB1/NZ_CP030923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KCJ3K292/NZ_CP054403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KCJ3K293/NZ_CP054406.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KCJ3K307/NZ_CP054400.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KCTC_2242/NC_017540.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Klebsiella_pneumoniae/NZ_OU453210.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Klebsiella_pneumoniae_459/NZ_CP018306.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Klebsiella_pneumoniae_kpn154/NZ_LR745042.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KLP268/NZ_CP043047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-1/NZ_CP012883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-135LU/NZ_CP070890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-1Pi/NZ_CP071027.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-426/NZ_CP080309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-8788/NZ_CP037928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-C76/NZ_CP080297.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-CT77/NZ_CP080303.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-PA/NZ_CP084984.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP-Zhen/NZ_CP053862.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp1/NZ_CP086285.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp1/NZ_OU342706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1064WHY/NZ_CP084705.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp11-81/NZ_CP082154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP120/NZ_CP060744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp13/NZ_CP003999.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1517/NZ_CP072463.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp1604/NZ_CP085478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1677/NZ_CP041022.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP17-15/NZ_CP034076.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP17-16/NZ_CP034077.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1766/NZ_CP025146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1768/NZ_CP025140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-1/NZ_CP082001.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2050/NZ_CP082019.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2073/NZ_CP082024.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2079/NZ_CP048933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2110-2/NZ_CP084986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2113/NZ_CP082029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2138/NZ_CP082032.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-2172/NZ_CP082037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-238/NZ_CP082014.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-29/NZ_CP034249.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-3-8/NZ_CP048430.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-41/NZ_CP082010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18-8/NZ_CP082005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP18069/NZ_CP059889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP1878/NZ_CP072996.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP19-2029/NZ_CP047160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP19-2196/NZ_CP082041.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP19-3023/NZ_CP063747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP19-3088/NZ_CP063147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP19-3138/NZ_CP090622.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP2/NZ_CP041946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20191015/NZ_CP051160.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194a/NZ_CP054780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194a2/NZ_CP054774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194b/NZ_CP054768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194b2/NZ_CP054762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194c/NZ_CP054756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194c3/NZ_CP054750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194c4/NZ_CP054744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194c5/NZ_CP054738.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194d/NZ_CP054732.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194e/NZ_CP054726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP20194f/NZ_CP054720.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp202/NZ_CP041082.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP2509/NZ_CP065949.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP2723/NZ_CP072938.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP29105/NZ_CP091061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP294/NZ_CP083445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP29499/NZ_CP089598.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP30086/NZ_CP091058.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP3295/NZ_CP079725.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP33/NZ_AP018747.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp36/NZ_CP047192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP36M/NZ_CP017777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP46/NZ_CP090126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp46564/NZ_CP059317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp46564_15578211/NZ_CP089021.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp46596/NZ_CP059312.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP47693/NZ_CP070528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP4823/NZ_CP082791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP486/NZ_CP058358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP5/NZ_CP012426.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP51069/NZ_CP070564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP51248/NZ_CP070568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/kp5152/NZ_CP090462.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp52.145/NZ_FO834906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP52810/NZ_CP070579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP52813/NZ_CP070586.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP53/NZ_CP087669.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP55/NZ_CP055294.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP58/NZ_CP041373.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp6/NZ_CP082290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP617/NZ_CP012753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP63/NZ_CP077763.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP64/NZ_AP018750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP65/NZ_CP044258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP67/NZ_AP018753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP69/NZ_CP025456.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP697/NZ_CP066151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/kp7450/NZ_CP090468.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/kp757/NZ_CP015120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP77/NZ_CP058361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP7728/NZ_CP092648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp8701/NZ_CP049604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP9201/NZ_LR025092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP980/NZ_LR025088.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPA4411/NZ_CP070598.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPA9853/NZ_CP070602.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpBO_TO1/NZ_CP090629.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpBO_TO3/NZ_CP090626.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpBO_TO5/NZ_CP090623.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC-2/NZ_CP078122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpC11/NZ_CP065831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC160117/NZ_CP040033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC160121/NZ_CP040028.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC160125/NZ_CP040038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC160132/NZ_CP040023.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPC2/NZ_LR130548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpC5/NZ_CP065833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPCTRSRTH01/NZ_CP041092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPCTRSRTH07/NZ_CP041099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPCZA02/NZ_CP058226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPHS1249/NZ_CP027189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn-13/NZ_CP047707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn-14/NZ_CP047700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/kpn-hnqyy/NZ_CP074116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpN01/NZ_CP012987.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpN06/NZ_CP012992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN110/NZ_CP065172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN114/NZ_CP065167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN142/NZ_CP053875.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN1482/NZ_CP020841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN20/NZ_CP092336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn2166/NZ_LR745045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn223/NZ_CP015025.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN234/NZ_CP072653.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN236/NZ_CP072492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN246/NZ_CP072494.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN247/NZ_CP072496.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN342/NZ_CP089383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN361/NZ_CP053016.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN41053/NZ_CP052036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN528/NZ_CP020853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn555/NZ_CP015130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN55602/NZ_CP042977.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPN857/NZ_CP090432.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH1/NZ_CP008827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH10/NZ_CP007727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH24/NZ_CP008797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH27/NZ_CP007731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH29/NZ_CP009863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH30/NZ_CP009872.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH31/NZ_CP009876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH32/NZ_CP009775.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH33/NZ_CP009771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH36/NZ_CP014647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH39/NZ_CP014762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH45/NZ_CP036450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH48/NZ_CP026392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH49/NZ_CP026178.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNIH50/NZ_CP026177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNSF-2/NZ_CP090242.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPNSP-1/NZ_CP065401.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kpn_1693/NZ_CP047595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpPF25/NZ_CP065825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPR0928/NZ_CP008831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpvST147B_SE1_1_NDM/NZ_CP040724.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpvST15_NDM/NZ_CP040593.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpWEA1/NZ_AP024568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpWEA2/NZ_AP024570.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpWEA3/NZ_AP024572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpWEA4-1/NZ_AP024576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KpWEA4-2/NZ_AP024579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KPWX136/NZ_CP069170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_160/NZ_CP078033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_3/NZ_CP086724.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_121641/NZ_CP018735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_149473/NZ_CP018686.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_149832/NZ_CP018695.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_152021/NZ_CP018713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_154414/NZ_CP018337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_33208/NZ_CP018447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_62629/NZ_CP018364.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_71070/NZ_CP018450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_821588/NZ_CP018692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_822579/NZ_CP018140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_822917/NZ_CP018438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_827024/NZ_CP018701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Kp_Goe_827026/NZ_CP018707.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_Goe_828304/NZ_CP018719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_NORM_BLD_2010_71621/NZ_CP073791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_NORM_BLD_2012_85054/NZ_CP073627.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_NORM_BLD_2014_104014/NZ_CP034045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_NORM_BLD_2014_96855/NZ_CP073783.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KP_NORM_BLD_2015_115359/NZ_CP073788.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_1B/NZ_LR890291.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_1H/NZ_LR890722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_1I-sc-2280289/NZ_CP031789.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_5B/NZ_LR890353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_5D/NZ_CP024191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_5H-sc-2280283/NZ_LR890500.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_6G-sc-2280276/NZ_LR890457.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_6J/NZ_LR890230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_7E/NZ_CP024496.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_7F-sc-2280268/NZ_CP031814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_7J/NZ_CP024548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_8C-sc-2280241/NZ_LR890751.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_9A/NZ_AP024173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_9D/NZ_CP024535.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_9E/NZ_CP083768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB1_9J-sc-2280309/NZ_LR890434.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_10B-sc-2280350/NZ_LR890484.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_1C-sc-2280352/NZ_LR890620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_2B-sc-2280339/NZ_LR890343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_2C-sc-2280354/NZ_LR890391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_6C/NZ_CP083784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_9A-sc-2280332/NZ_LR890564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KSB2_9B/NZ_LR890710.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KUH-KPNHVF1/NZ_CP047675.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/KUH-KPNHVL1/NZ_CP047677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/L39_2/NZ_CP033954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/L482/NZ_CP033960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/L5-2/NZ_CP025684.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LC-1825_18/NZ_CP058325.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LEMB18/NZ_CP045782.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LH102-A/NZ_CP035194.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LH375/NZ_CP035196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LH94/NZ_CP035202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LSH-KPN148/NZ_CP040122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LSH-KPN25/NZ_CP040391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LYS105A/NZ_CP068033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LZK001/NZ_CP061965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LZKP00001/NZ_CP089986.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/LZKP00003/NZ_CP089989.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M1023-4Ar/NZ_CP063851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M1026-3Ar/NZ_CP063858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M164-1/NZ_CP063992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M169-3/NZ_CP063878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M186-2/NZ_CP063930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M212-2/NZ_CP063908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M297-1/NZ_CP051490.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M63-1/NZ_CP063863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M911-1/NZ_CP064129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/M941/NZ_AP023453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Magic_conch_shell/NZ_CP065457.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MAR14-456/NZ_CP063277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MGH83/NZ_CP073054.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MGH_39/NZ_CP072949.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MH15-269M/NZ_AP023337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MIN-109/NZ_CP086128.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MN-03/NZ_CP081897.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MNCRE53/NZ_CP018437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MNCRE69/NZ_CP018427.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MNCRE78/NZ_CP018428.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Mrs._Puff/NZ_CP065438.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MRSN752019/NZ_CP084397.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MRSN752165/NZ_CP074087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MS14393/NZ_CP054303.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MS6671/NZ_LN824133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MSB1_8A-sc-2280397/NZ_CP031800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MV-u1-SK2-O/NZ_CP085866.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/MV-v4-SK2-O/NZ_CP085863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/N1059-5At/NZ_CP063856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/N16-03892/NZ_CP047271.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/N4b/NZ_CP031577.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NB04/NZ_CP091992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NB05/NZ_CP091846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NB5/NZ_CP092656.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NB5306/NZ_CP034760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NC-01/NZ_CP081894.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NCTC11359/NZ_LR133964.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NCTC11698/NZ_LR588408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NCTC418/NZ_LR134213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NCTC9157/NZ_LR588412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NCTC9793/NZ_LR134280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NFYY0065/NZ_CP035531.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NH25/NZ_CP024874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NH34/NZ_CP034408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NH54/NZ_CP024916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NICU_2_P7/NZ_CP060049.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NKU_Kleb8A7/NZ_CP041644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NMBU-W07E18/NZ_CP042882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Nord5-1_R48/NZ_CP091594.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Nord77_R886/NZ_CP091601.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Nord9_R85/NZ_CP091582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NR5632/NZ_CP025143.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NTUH-K2044/NC_012731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE047/NZ_CP025037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE055/NZ_CP061404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE085/NZ_CP061399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE193/NZ_CP061396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE195/NZ_CP061389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE213/NZ_CP061385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NU-CRE236/NZ_CP061380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NUHL24835/NZ_CP014004.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NUHL30457/NZ_CP026586.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NY-06/NZ_CP081892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/NY9/NZ_CP015385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Ocean_Ranger/NZ_CP065447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/OCU_hvKP1/NZ_AP025246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Olaf/NZ_CP065464.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ONE259/NZ_CP090931.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/P094-1/NZ_CP043562.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/P1428/NZ_CP017994.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/P35/NZ_CP053035.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/PA-01/NZ_CP081896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Patrick_Star/NZ_CP065441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/PIMB15ND2KP27/NZ_CP041639.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/PittNDM01/NZ_CP006798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/PMK1/NZ_CP008929.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/QD23/NZ_CP042858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/QMP_B2-170/NZ_CP031798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/QS17-0029/NZ_CP024038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/QS17-0161/NZ_CP024458.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/R210-2/NZ_CP034082.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/R32/NZ_CP083443.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/R50/NZ_CP040363.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RGF20-1/NZ_CP075452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RGF99-1/NZ_CP075552.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RGT40-1/NZ_CP075548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB16-C24/NZ_CP057722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB26-C08/NZ_CP057459.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB30-C05/NZ_CP057313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB38-C06/NZ_CP057099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB39-C09/NZ_CP057069.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB39-C10/NZ_CP057066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHB39-C24/NZ_CP057034.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00062/NZ_CP056883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00066/NZ_CP056878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00113/NZ_CP056854.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00128/NZ_CP056830.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00433/NZ_CP056530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00472/NZ_CP055509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00510/NZ_CP056432.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00515/NZ_CP056425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00574/NZ_CP055451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00610/NZ_CP056385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00636/NZ_CP056372.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00807/NZ_CP056329.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00832/NZ_CP056307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00843/NZ_CP056301.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00852/NZ_CP056297.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00867/NZ_CP056275.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00973/NZ_CP056222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00982/NZ_CP056216.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RHBSTW-00995/NZ_CP056191.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RIVM_C014073/NZ_CP068797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RIVM_C015043/NZ_CP068798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJA166/NZ_CP019047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJBSI76/NZ_CP068689.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJBSI76-pV/NZ_CP068684.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJF293/NZ_CP014008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJF999/NZ_CP014010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/RJY9645/NZ_CP041353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S130-1/NZ_CP063865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S141/NZ_CP063871.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S161-2/NZ_CP058544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S165-1/NZ_CP058548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S166-1/NZ_CP063945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S183-1/NZ_CP063927.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S210-3/NZ_CP063925.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/S90-2/NZ_CP063881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SA-KpST14/NZ_CP071279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SA1/NZ_CP084868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SA12/NZ_CP084866.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB1067/NZ_CP084860.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB1139/NZ_CP084859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB1170/NZ_CP084856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB2390/NZ_CP084852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB5881/NZ_LR792628.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB611/NZ_CP084843.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB612/NZ_CP084830.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB615/NZ_CP084829.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SB617/NZ_CP084825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SC-7/NZ_CP030269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCAID_PHRX1-2021_(13_97)/NZ_CP082805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCAID_PHRX2-2021_(20_245)/NZ_CP082796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCAID_TST1-2021_(15_226)/NZ_CP082799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCH6109/NZ_CP050858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020003/NZ_CP031721.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020046/NZ_CP028783.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020049/NZ_CP028787.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020079/NZ_CP029384.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020135/NZ_CP037963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP020143/NZ_CP028548.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCKP040074/NZ_CP029388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCM96/NZ_CP028716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SCPM-O-B-8912_(11PKP_19a)/NZ_CP086664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Sea_Superman/NZ_CP065444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SGH10/NZ_CP025080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SH12/NZ_CP040833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SKGH01/NZ_CP015500.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SMKP03/NZ_AP023148.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SMU18037509/NZ_CP045661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST101:960186733/NZ_CP023487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST23/NZ_CP037742.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST307PT01/NZ_CP022924.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST307PT02/NZ_CP022922.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST307PT03/NZ_CP022919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ST307PT04/NZ_CP022916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STEFF_14/NZ_CP055186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STEFF_19/NZ_CP055174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STEFF_2/NZ_CP055210.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STEFF_9/NZ_CP055162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_77/NZ_CP055005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_79/NZ_CP055003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_84/NZ_CP054997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_90/NZ_CP054984.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_93/NZ_CP054972.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STIN_94/NZ_CP054967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLE2/NZ_CP058857.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLEFF_42/NZ_CP058866.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLEFF_44/NZ_CP058869.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLIN_10/NZ_CP058743.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLIN_19/NZ_CP058752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/STLIN_3/NZ_CP058758.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SW1780/NZ_CP073302.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHE1/NZ_CP055066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHE3/NZ_CP055061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHE4/NZ_CP055057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHEFF_62/NZ_CP055033.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHIN_100/NZ_CP055119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHIN_106/NZ_CP055106.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHIN_108/NZ_CP055096.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWHIN_114/NZ_CP055087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SWU01/NZ_CP018454.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/SZP4-9-2/NZ_CP075256.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/T69/NZ_CP084823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/T698-1/NZ_CP079781.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/T877/NZ_CP084241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TA6363/NZ_AP019665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TAKPN-1/NZ_CP091650.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TGH10/NZ_CP012744.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TGH13/NZ_CP012745.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TGH8/NZ_CP012743.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TH1/NZ_CP016159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TH12852/NZ_CP087122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TH12908/NZ_CP087123.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TH164/NZ_CP035210.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/THO-004/NZ_AP022527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TK421/NZ_CP045694.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12126/NZ_AP024750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12127/NZ_AP024753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12128/NZ_AP024756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12129/NZ_AP024760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12130/NZ_AP024763.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12131/NZ_AP024766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12132/NZ_AP024769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12133/NZ_AP024772.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12134/NZ_AP024775.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12135/NZ_AP024778.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12136/NZ_AP024781.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12137/NZ_AP024784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12138/NZ_AP024788.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12139/NZ_AP024792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TUM12140/NZ_AP024795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/TVGHCRE225/NZ_CP023722.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/UCLAOXA232KP/NZ_CP012568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/UCLAOXA232KP/NZ_CP012561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/UCLAOXA232KP_Pt0/NZ_CP012560.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/UHKPC07/NZ_CP011985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/V3/NZ_CP076863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/VBA2172/NZ_CP036320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/W14/NZ_CP015753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCGKP294/NZ_CP046612.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP015093/NZ_CP036300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020030/NZ_CP028793.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020034/NZ_CP025963.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020037/NZ_CP036371.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020098/NZ_CP036305.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020115/NZ_CP043353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP020120/NZ_CP043357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP030925/NZ_CP073285.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP040035/NZ_CP028797.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP090329/NZ_CP066517.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP090357/NZ_CP066523.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP090361/NZ_CP066528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP090374/NZ_CP066534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP095649/NZ_CP026585.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP095845/NZ_CP031885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP115068/NZ_CP036365.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP115069/NZ_CP033405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP13F2/NZ_CP028391.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP2/NZ_CP028542.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP2080/NZ_CP036361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP36/NZ_CP028583.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP7E2/NZ_CP028806.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WCHKP8F4/NZ_CP027068.3.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WGR351/NZ_CP076008.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WP2-S17-ESBL-02/NZ_AP021909.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WP2-W18-ESBL-06/NZ_AP021929.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WP4-S18-CRE-04/NZ_AP022078.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WP5-S18-ESBL-02/NZ_AP022139.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC01_S465PC/NZ_CP079115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC02_S465MC/NZ_CP079807.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC03_AI1235PC/NZ_CP079120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC04_AI1235MC/NZ_CP079131.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC05_CMC387PC/NZ_CP079138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC06_CMC387MC/NZ_CP079145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC07_CMC432PC/NZ_CP079150.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC08_CMC432MC/NZ_CP079155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC09_CMC309PC/NZ_CP079164.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC10_CMC309MC/NZ_CP079599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC11_S471PC/NZ_CP079172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC12_S471M/NZ_CP084044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC13_AI2040PC/NZ_CP079179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC14_AI2040MC/NZ_CP079186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC15_AI1547PC/NZ_CP079606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC16_AI1547MC/NZ_CP079617.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC17_CMC307PC/NZ_CP079624.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC18_CMC307MC/NZ_CP079629.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC19_AI1572C/NZ_CP079634.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC21_AI1646PC/NZ_CP079636.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC22_AI1646MC/NZ_CP079642.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC24_S468MC/NZ_CP079645.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC27_S471MPC/NZ_CP079655.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC28_CMC387MPCH/NZ_CP079668.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC29_CMC309MPCH/NZ_CP079673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC30_AI1547MPCH/NZ_CP079683.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WRC31_S468MPCH/NZ_CP079692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WSCRKP/NZ_CP091068.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WSD411/NZ_CP045674.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/WSHvKP/NZ_CP054063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/wxc2020/NZ_CP071385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/Xen39/NZ_CP040861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XH1507/NZ_CP092793.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XH1508/NZ_CP092786.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XH209/NZ_CP009461.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XH210/NZ_CP052761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XJ-K1/NZ_CP032163.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/XJ-K2/NZ_CP032240.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YH43/NZ_AP014950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YML0508/NZ_CP045193.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YTF44-1/NZ_CP075286.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YZH22127/NZ_CP090389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YZH2429/NZ_CP090393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/YZH2487/NZ_CP087706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZG2017CW4-3-1-2N/NZ_CP065351.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZG2017CW4-4-1-2/NZ_CP065343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZG2017CW4-4-2/NZ_CP065347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZG2017CW_1-1-1/NZ_CP065341.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZJ27003/NZ_CP067060.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZRKP01/NZ_CP050354.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZRKP02/NZ_CP050348.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZRKP03/NZ_CP050342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZRKP04/NZ_CP050336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZYST1/NZ_CP031613.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/pneumoniae/ZZW20/NZ_CP058960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/01A030T/NZ_CP084876.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/09A323/NZ_CP084783.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/12A476/NZ_CP084779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/18A069/NZ_CP084818.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/203_19001/NZ_CP062138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/3Z-5L/NZ_CP072515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/68-17/NZ_CP042319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/A708/NZ_CP030171.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/A708/NZ_CP026368.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/ARLG-8054/NZ_CP067966.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/ATCC_700603/NZ_CP014696.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/ATCC_700603/NZ_CP029597.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/B25110/NZ_CP070454.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/B2663/NZ_CP070420.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/B6885/NZ_CP072468.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/B9067/NZ_CP070440.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BM404-3-1/NZ_CP064044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BM419-3/NZ_CP063887.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BS324-2/NZ_CP063900.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BS418/NZ_CP063942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BS419-1/NZ_CP063896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/BS419-3/NZ_CP063902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/C3001/NZ_CP039791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/CAV1947/NZ_CP029443.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/CAV2013/NZ_CP029437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/CAV2018/NZ_CP029432.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/CIP110288/NZ_CP084777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/D120-1/NZ_CP034678.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/FDAARGOS_1443/NZ_CP077279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/FDAARGOS_1503/NZ_CP083633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/FDAARGOS_1504/NZ_CP083631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/FDAARGOS_93/NZ_CP014071.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/G4584/NZ_CP034129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/G747/NZ_CP034136.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/HKUOPA4/NZ_CP014154.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/HKUOPJ4/NZ_CP014155.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/HKUOPL4/NZ_CP014156.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/HKUOPLA/NZ_CP012252.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/HKUOPLC/NZ_CP012300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/IF3SW-P1/NZ_CP092121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/JNQH473/NZ_CP075885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KP18-31/NZ_CP045641.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KP2011790/NZ_CP066173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KPC142/NZ_CP023478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KPN710429/NZ_CP073660.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KqPF26/NZ_CP065838.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KqPF42/NZ_CP065846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/KqPF9/NZ_CP065841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/L22/NZ_CP031257.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/M17277/NZ_CP043928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/MMCC7/NZ_CP082785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/MN-07/NZ_CP081895.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/N18-04101/NZ_CP047281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/N18-04116/NZ_CP047283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/NCTC11357/NZ_LR134196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/NCTC9170/NZ_LR588411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/OSUCMP263AKPC/NZ_CP087580.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/Pinhead_Larry/NZ_CP065467.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/RHBSTW-00138/NZ_CP058131.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/S15-2/NZ_CP064046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/S174-1/NZ_CP063874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SB1124/NZ_CP084806.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SB610/NZ_CP084770.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SB98/NZ_CP084803.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SNI47/NZ_AP019687.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/STEFF_21/NZ_CP055213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/STLEFF_28/NZ_CP058859.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/STW0522-39/NZ_AP022412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SWHEFF_72/NZ_CP055010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/SWMUF35/NZ_CP068444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/TH114/NZ_CP035207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/U41/NZ_CP084789.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WI-06/NZ_CP081893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP3-S18-CRE-02/NZ_AP022005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP3-S18-ESBL-03/NZ_AP022014.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP3-W18-CRE-02/NZ_AP021950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP5-S18-ESBL-05/NZ_AP022142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP5-W18-ESBL-06/NZ_AP022112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WP7-S17-ESBL-08/NZ_AP022183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/WW-14A/NZ_CP080099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasipneumoniae/YDKL-002/NZ_CP068237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/quasivariicola/KPN1705/NZ_CP022823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./A52/NZ_CP074437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./BDA134-6/NZ_CP064784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./CTHL.F3a/NZ_CP082360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./FDAARGOS_511/NZ_CP033824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./KPN54798/NZ_CP089234.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./LTGPAF-6F/NZ_CP017450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./LY/NZ_CP022444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./M5al/NZ_CP020657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./MPUS7/NZ_CP047604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./P1927/NZ_CP073377.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./P1954/NZ_CP073373.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./P1CD1/NZ_CP033631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./PCX/NZ_CP091527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./PL-2018/NZ_CP077673.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./RHBSTW-00464/NZ_CP056483.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP3-S18-ESBL-05/NZ_AP022020.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP3-W18-ESBL-02/NZ_AP021972.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP4-W18-ESBL-05/NZ_AP022057.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP7-S18-CRE-02/NZ_AP022189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP7-S18-CRE-03/NZ_AP022195.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/sp./WP7-S18-ESBL-04/NZ_AP022201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/01A065/NZ_CP084766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/03-311-0071/NZ_LR130544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/04153260899A/NZ_LR130543.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/13450/NZ_CP030173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/13450/NZ_CP026013.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/15WZ-82/NZ_CP032354.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/342/NC_011283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/8917/NZ_CP063403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/AHKv-S01/NZ_CP047360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/AJ055/NZ_LR130539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/AJ292/NZ_LR130538.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/At-22/NC_013850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BM366-1/NZ_CP063921.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BM374-1/NZ_CP063917.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS325-2/NZ_CP063910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS325-3-1/NZ_CP063889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS326-1/NZ_CP063898.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS327-2-1/NZ_CP063916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS359-2-1/NZ_CP063912.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS369-2/NZ_CP063933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/BS375-3/NZ_CP063932.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/CDC4241-71/NZ_CP084767.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/DSM_15968/NZ_CP010523.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/DX120E/NZ_CP009274.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/E25/NZ_CP067261.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/E57-7/NZ_CP016344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/EMBL-1/NZ_CP079802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/F2R9/NZ_CP072130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/F2R9T/NZ_CP084765.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/FDAARGOS_627/NZ_CP044050.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/FH-1/NZ_CP054254.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/GJ1/NZ_CP017284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/GJ2/NZ_CP017849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/GJ3/NZ_CP017289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/GN02/NZ_CP031061.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/INF352/NZ_CP083790.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/JCM_12419/NZ_AP024592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/JNQH579/NZ_CP078146.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/KP2757/NZ_CP060807.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/KPN029/NZ_CP065162.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/KPN1481/NZ_CP020847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/KPN692/NZ_CP089385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/LEMB11/NZ_CP045783.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/LMG_23571/NZ_CP013985.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/M142-3/NZ_CP063867.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/M186-1/NZ_CP063911.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/M186-1-2/NZ_CP063915.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/ML.9ba2/NZ_CP092632.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/NCTC9668/NZ_LR134235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/RHBSTW-00883/NZ_CP056145.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/S129-1/NZ_CP063954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/S131-2/NZ_CP063953.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/S187-1/NZ_CP063926.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/WCHKP19/NZ_CP028555.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/WCHKV030666/NZ_CP027064.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/X39/NZ_CP018307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Klebsiella/variicola/ZH07/NZ_CP087978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kluyvera/ascorbata/TP1631/NZ_AP022665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kluyvera/intermedia/HR2/NZ_CP045843.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kluyvera/intermedia/N2-1/NZ_CP045845.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kluyvera/intermedia/NCTC12125/NZ_LR134138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kluyvera/sp./CRP/NZ_CP082841.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/arachidis/KACC_18508/NZ_CP045300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/cowanii/888-76/NZ_CP019445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/cowanii/FBS_223/NZ_CP035129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/cowanii/Pa82/NZ_CP069319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/oryzae/IAC-BECa90/NZ_CP065358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/oryzae/Ola_51/NZ_CP014007.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/pseudosacchari/BDA62-3/NZ_CP063425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/radicincitans/DSM_107547/NZ_CP040392.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/radicincitans/DSM_16656/NZ_CP018016.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/radicincitans/GXGL-4A/NZ_CP015113.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/radicincitans/IAC-BECa95/NZ_CP065411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/sacchari/BO-1/NZ_CP016337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/sacchari/DSM_107661/NZ_CP040677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/sacchari/SP1/NZ_CP007215.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/sp./CCTCC_M2018092/NZ_CP034225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Kosakonia/sp./SMBL-WEM22/NZ_CP051488.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/16005813/NZ_CP036199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/707804/NZ_CP049980.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/E1/NZ_CP042505.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/E61/NZ_CP042493.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/FDAARGOS_1505/NZ_CP083630.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/G426/NZ_CP043398.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/J656/NZ_CP042930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/L21/NZ_CP043397.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/P12375/NZ_CP046251.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/R25/NZ_CP035382.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/SH19PE29/NZ_CP087280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/USDA-ARS-USMARC-60222/NZ_CP013990.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/adecarboxylata/Z96-1/NZ_CP040889.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./1106151/NZ_CP046557.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./119287/NZ_CP046445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./29361/NZ_CP049786.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./4-9-1-25/NZ_CP071383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./4-9-1-25/NZ_CP076838.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./G3L/NZ_CP088265.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./J807/NZ_CP046446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./LSNIH1/NZ_CP026167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./LSNIH3/NZ_CP026387.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./w17/NZ_CP031101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Leclercia/sp./W6/NZ_CP031104.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/amnigena/A167/NZ_CP042361.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/amnigena/FDAARGOS_1444/NZ_CP077236.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/amnigena/FDAARGOS_1446/NZ_CP077331.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/amnigena/FDAARGOS_395/NZ_CP023529.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/amnigena/ZB04/NZ_CP015774.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Lelliottia/sp./WB101/NZ_CP028520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Limnobaculum/parvum/HYN0051/NZ_CP029185.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Metakosakonia/sp./MRY16-398/NZ_AP018756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Phytobacter/diazotrophicus/TA9734/NZ_AP025334.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Phytobacter/diazotrophicus/UAEU22/NZ_CP051548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Phytobacter/sp./SCO41/NZ_CP027225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Phytobacter/ursingii/CAV1151/NZ_CP011602.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Pluralibacter/gergoviae/FB2/NZ_CP009450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Pluralibacter/gergoviae/FDAARGOS_186/NZ_CP020388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Pluralibacter/gergoviae/MGYG-HGUT-02520/NZ_LR699009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Pseudocitrobacter/sp./G163CM/NZ_CP087880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/electrica/DSM_102253/NZ_CP041247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/172117885/NZ_CP046672.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/193104922/NZ_CP046671.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/23/NZ_CP048349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/A14/NZ_CP008886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/B6/NC_021066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/DSM_16926/NZ_CP059047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/LL1/NZ_CP061719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/MG/NZ_CP017802.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/NCTC13096/NZ_LR134195.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/NUITM-VR1/NZ_AP025009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/NY1/NZ_CP079752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/OHMEATKPC2_NDM5/NZ_CP054270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/Ro24724/NZ_CP021329.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/S12/NZ_CP010557.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/STEFF_15/NZ_CP055216.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/TET32/NZ_CP067413.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/WLK218/NZ_CP038281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/WM1/NZ_CP049752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/WP3-W18-ESBL-06/NZ_AP021983.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/WP8-W19-CRE-01/NZ_AP022268.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/XF201/NZ_CP076605.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/ornithinolytica/Yangling_I2/NZ_CP013338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/FDAARGOS_64/NZ_CP026047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/HH15/NZ_CP069815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/JBIWA001/NZ_CP074182.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/Rp_CZ180511/NZ_CP040183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/S25/NZ_CP044121.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/planticola/SCLZS62/NZ_CP082168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/sp./X13/NZ_CP030874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/sp./XY-1/NZ_CP067066.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/terrigena/JH01/NZ_CP050508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/terrigena/NCTC9189/NZ_LR595855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Raoultella/terrigena/Res13-Abat-PEB01-P1-04-A/NZ_CP062916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/04-0440/NZ_CP035676.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/08-0158/NZ_CP053336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/85-0051/NZ_CP053416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/92-0238/NZ_CP053417.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/CFSAN001045/NZ_CP074592.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/N268-08/NC_021870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/NCTC_12419/NC_015761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/RKS3044/NZ_CP006692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/SA19983605/NZ_CP022120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/Se40/NZ_CP067369.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/SGSC_3100/NZ_CP045766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/bongori/SGSC_3100/NZ_CP074233.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/0007-33/NZ_CP013685.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/0250/NZ_CP019197.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/0307-213/NZ_CP012599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/0315/NZ_CP019198.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/07Q015/NZ_CP077760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/08-00436/NZ_CP020492.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/08-1736/NC_021820.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/09-036813-1A/NZ_CP016525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/10ST07093/NZ_CP029839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/10TTU468x/NZ_CP032814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11-004736-1-7/NZ_CP016514.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11-01853/NZ_CP011289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11-01854/NZ_CP011292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11-01855/NZ_CP011288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11-5006/NZ_CP038601.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/119944/NZ_CP047881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11TTU1590/NZ_CP032817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11TTU1615b/NZ_CP032815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/11TTUC-046/NZ_CP032816.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/12-01738/NZ_CP027677.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/12-0523/NZ_CP039277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/12-2388/NZ_CP038595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/12-4374/NZ_CP012924.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/12888/NZ_CP045447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1315K/NZ_CP022139.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1326_28/NZ_LN649235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/138736/NZ_CP007581.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14-SA00836-0/NZ_CP054422.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028/NZ_CP034479.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028S/NC_016856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028S_substr._GXS254/NZ_CP043402.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028S_substr._GXS259/NZ_CP043401.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028S_substr._GXS275/NZ_CP043399.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/14028S_substr._JY996/NZ_CP043400.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1422-74/NZ_CP019408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/15/NZ_CP092258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/15-SA01028/NZ_CP026660.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1505/NZ_CP060834.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1554-sc-2165329/NZ_LT905064.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/159838/NZ_CP043662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/16-174478/NZ_CP063140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/16-SA00356/NZ_CP034074.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/161365/NZ_CP043664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/162835/NZ_CP043667.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/1655/NZ_CP014996.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/16A242/NZ_CP020922.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/18-SA00377-0/NZ_CP071388.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/180135033/NZ_CP088901.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/18569/NZ_CP011394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/20-369/NZ_CP075174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2008079-SE/NZ_CP090529.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2010-007898/NZ_LT905139.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2010112-SE/NZ_CP090541.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2011K-1440/NZ_CP053585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2012K-0845/NZ_CP053579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2014K-1020/NZ_CP053584.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2014LSAL01779/NZ_CP026609.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2016062-SE/NZ_CP090539.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2016089-SE/NZ_CP090535.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2017005-SE/NZ_CP090533.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2017028-SE/NZ_CP090545.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2017K-0021/NZ_CP030794.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2020LSAL11867/NZ_CP073715.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/21-SA00318-0/NZ_CP091878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/222/NZ_CP031189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/22792/NZ_CP017621.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/24249/NC_022991.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2439-64/NZ_CP053582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2511STDY5462388/NZ_LN890518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2511STDY5462413/NZ_LN890520.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2511STDY5712384/NZ_LN890522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2511STDY5712385/NZ_LN890524.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/287_91/NC_011274.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/2B_cip/NZ_CP074096.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/3/NZ_CP040701.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_201186/NZ_CP029958.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_202186/NZ_CP029956.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_203186/NZ_CP029942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_204186/NZ_CP029954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_205186/NZ_CP029950.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_206186/NZ_CP029940.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_207186/NZ_CP029938.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_208103/NZ_CP029932.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_208186/NZ_CP029949.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_209186/NZ_CP029952.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_210186/NZ_CP029946.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_211186/NZ_CP029945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_212186/NZ_CP029944.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_213186/NZ_CP029936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_214186/NZ_CP029933.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_215186/NZ_CP029930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_216103/NZ_CP029928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_217103/NZ_CP029927.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_217186/NZ_CP029646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_218186/NZ_CP029925.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_219186/NZ_CP029911.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_220186/NZ_CP029886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_221186/NZ_CP029923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_222186/NZ_CP029909.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_223186/NZ_CP029880.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_224186/NZ_CP029878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_231186/NZ_CP029904.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_232103/NZ_CP029918.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_239103/NZ_CP029908.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_252186/NZ_CP029896.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_255186/NZ_CP029885.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_256186/NZ_CP029917.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_268103/NZ_CP029902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_268186/NZ_CP029883.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_269186/NZ_CP029922.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_282186/NZ_CP029920.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/311189_291186/NZ_CP029894.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/33676/NZ_CP012681.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_202113/NZ_CP029915.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_217103/NZ_CP029914.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_227128/NZ_CP029875.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_232188/NZ_CP029900.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_241106/NZ_CP029899.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_248190/NZ_CP029882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_249107/NZ_CP029913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_252143/NZ_CP029892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_253155/NZ_CP029890.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_269157/NZ_CP029881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343076_294172/NZ_CP029888.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_211171/NZ_CP029873.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_212138/NZ_CP029919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_212159/NZ_CP029870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_213147/NZ_CP029897.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_214135/NZ_CP029872.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_214162/NZ_CP029862.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_215174/NZ_CP029868.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_228140/NZ_CP029866.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_228157/NZ_CP029864.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_255118/NZ_CP029907.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_260153/NZ_CP029861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_267164/NZ_CP029906.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_278127/NZ_CP029863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_281186/NZ_CP029853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_285138/NZ_CP029858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_286126/NZ_CP029856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343077_292148/NZ_CP029855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_201101/NZ_CP029852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_203125/NZ_CP029850.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_211126/NZ_CP029848.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_223175/NZ_CP029964.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_228140/NZ_CP029962.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_251131/NZ_CP029960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_256191/NZ_CP029959.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/343078_273110/NZ_CP029846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/361154004/NZ_CP029036.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/389598/NZ_CP054827.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/392869-2/NZ_CP015024.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/403Ty-sc-1979084/NZ_LT905062.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/41578/NC_021810.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/41578/NZ_CP045762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/420/NZ_CP019201.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/42N/NZ_CP037893.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/4316STDY6559669/NZ_LR590081.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/4316STDY6559672/NZ_LR590082.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/453/NZ_CP060855.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/460004_2-1/NZ_CP011259.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/467/NZ_CP060852.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/4714STDY6831631/NZ_LT882486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/49-2444/NZ_CP018633.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5/NZ_CP031359.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5/NZ_CP040700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/507440-20/NZ_CP007530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/50973/NZ_CP009049.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/531954/NZ_CP029035.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/56-3991/NZ_CP018635.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5ASAL05/NZ_CP090137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5ASAL07/NZ_CP090133.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5ASAL09/NZ_CP090141.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5d8c9f00c6184d00f9da85a6:sample:O1960-05/NZ_LR861808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/5d8c9f00c6184d00f9da85a6:sample:O1960-05/NZ_LR862421.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/661/NZ_CP067078.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/679/NZ_CP067082.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/69-3861/NZ_CP018637.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/7/NZ_CP040699.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/70-1605/NZ_CP018640.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/7101.67/NZ_CP039503.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/7102.58/NZ_CP039509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/724/NZ_CP060849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/733/NZ_CP060846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/74-1357/NZ_CP018642.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/77-1427/NZ_CP007598.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/77-2980/NZ_CP018644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/770/NZ_CP060844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/775W_(ATCC_43845)/NZ_CP016837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/79-2359/NZ_CP018645.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/798/NC_017046.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/805/NZ_CP060840.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/81-1435/NZ_CP018647.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/81-1607/NZ_CP018648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/81-1705/NZ_CP018651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/81-1706/NZ_CP018655.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/81741/NZ_CP019442.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/85-0120/NZ_CP054715.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/852/NZ_CP060837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/87/NZ_CP008925.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/9184/NZ_CP019035.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/92-0392/NZ_CP018657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/93-0639/NZ_CP018659.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/9355/NZ_CP045444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/94-0093/NZ_CP053581.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/95-0621/NZ_CP018661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/A1636/NZ_CP063708.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/A3ES40/NZ_CP016561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/A3EZ223/NZ_CP016563.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AB42049/NZ_CP064919.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AD19/NZ_CP043027.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AKU_12601/NC_011147.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AMR588-04-00318/NZ_CP016565.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AMR588-04-00320/NZ_CP016569.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AMR588-04-00435/NZ_CP016573.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AMR588-04-00437/NZ_CP016576.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0401/NZ_CP044188.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0402/NZ_CP044186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0403/NZ_CP044184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0404/NZ_CP044181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0405/NZ_CP044203.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0406/NZ_CP044200.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR-0407/NZ_CP044177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Articuno/NZ_CP085812.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR_0031/NZ_CP026700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AR_0127/NZ_CP032194.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_10708/NZ_CP012344.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_10717/NZ_CP019192.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_10720/NZ_CP019181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_10722/NZ_CP025218.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_10729/NZ_CP019186.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_11511/NZ_CP019185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_11997/NZ_CP019178.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_13311/NZ_CP009102.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_14028/NZ_CP034230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_14028/NZ_CP043907.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_15791/NZ_CP012349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_35640/NZ_CP007532.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_39184/NZ_CP019179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_49284/NZ_CP019184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_51958/NZ_CP019183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_51960/NZ_CP019177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_6017/NZ_CP007534.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_7378/NZ_CP012346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_8387/NZ_CP034232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_8391/NZ_CP011396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_8400/NZ_CP034233.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_9120/NZ_CP012347.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_9150/NC_006511.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_BAA-1592/NZ_CP007531.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_BAA-1673/NZ_CP019189.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_BAA-1738/NZ_CP019196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ATCC_BAA-708/NZ_CP025554.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00004549/NZ_OU015323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00005056/NZ_CP045831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00007171/NZ_OU015342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00008979/NZ_CP045952.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010527/NZ_CP045956.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010528/NZ_CP045955.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010529/NZ_CP045949.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010530/NZ_CP045947.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010531/NZ_CP045954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010532/NZ_CP045958.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00010533/NZ_CP045945.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00027944/NZ_OU015340.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/AUSMDU00027951/NZ_OU015328.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Azumarill/NZ_CP085805.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/A_cip/NZ_CP074094.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/B182/NC_017623.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/B3589/NZ_CP034968.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BAA-1586/NZ_CP023468.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BAA-1672/NZ_CP023470.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BBS1358lrg/NZ_CP061049.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BBS1358sml/NZ_CP061050.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BBS1359/NZ_CP061047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BBS1406/NZ_CP061045.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BBS1407/NZ_CP061044.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BKQZM9/NZ_CP060169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/BL10/NZ_CP024619.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Blastoise/NZ_CP085819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/C500/NZ_CP007639.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/C629/NZ_CP015724.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CD-SL01/NZ_CP028900.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC1983-67/NC_022221.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_06-0532/NZ_CP007211.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_07-0954/NZ_CP017974.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_08-1942/NZ_CP017975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2009K-0792/NZ_CP020752.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2009K-1331/NZ_CP025248.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2009K-1640/NZ_CP014975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2009K-2059/NZ_CP014983.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2010K-0257/NZ_CP020912.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2010K-1587/NZ_CP014965.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2010K-2159/NZ_CP007559.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2010K_0968/NZ_CP007528.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2011K-0870/NZ_CP007523.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2011K-1674/NZ_CP017976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2011K-1702/NZ_CP014967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2012K-0663/NZ_CP025243.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2012K-0938/NZ_CP025246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2012K-1544/NZ_CP017977.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_2013K-0218/NZ_CP017978.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_86-0391/NZ_CP007540.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CDC_H2662/NZ_CP014979.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CE-R2-11-0435/NZ_CP016517.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFIAFB20170119/NZ_CP039279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFIAFB20170120/NZ_CP038604.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFIAFB20170121/NZ_CP038608.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA1007/NZ_CP033387.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA1096/NZ_CP033348.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA12/NZ_CP033257.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA122/NZ_CP033226.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA231/NZ_CP033350.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA244/NZ_CP033255.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA300/NZ_CP033384.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA629/NZ_CP033352.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSA664/NZ_CP033356.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000189/NC_021844.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000189/NZ_CP045753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000189/NZ_CP034177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000191/NZ_CP032622.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000211/NZ_CP039500.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000212/NZ_CP039499.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000228/NZ_CP039498.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000661/NZ_CP039496.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000661/NZ_CP045759.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000669/NZ_CP039493.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000669/NZ_CP045757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000700/NZ_CP039491.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000752/NZ_CP039489.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000752/NZ_CP045754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000753/NZ_CP039487.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000827/NZ_CP074606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000951/NZ_CP039486.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000952/NZ_CP039484.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000954/NZ_CP039482.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000958/NZ_CP039480.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000960/NZ_CP039478.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000961/NZ_CP039477.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000963/NZ_CP039476.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000968/NZ_CP039473.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN000970/NZ_CP039471.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001080/NZ_CP007533.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001112/NZ_CP039469.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001115/NZ_CP039467.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001118/NZ_CP039465.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001140/NZ_CP039463.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001387/NZ_CP014994.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001588/NZ_CP012833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001660/NZ_CP016010.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001921/NC_021814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN001992/NC_020307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN002050/NC_021818.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN002064/NZ_CP005995.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN002069/NC_021812.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN003307/NZ_CP019202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN003887/NZ_CP016014.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN003890/NZ_CP016012.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN003959/NZ_CP041184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN003961/NZ_CP041183.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN004025/NZ_CP039270.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN004173/NZ_CP019204.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN004174/NZ_CP019206.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN004175/NZ_CP019172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN008081/NZ_CP074663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN008104/NZ_CP074661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN012622/NZ_CP075037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN017963/NZ_CP017720.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN018746/NZ_CP028199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN018747/NZ_CP028196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN024229/NZ_CP019174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN033541/NZ_CP020823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN033543/NZ_CP020825.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN033950/NZ_CP075141.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044865/NZ_CP075140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044875/NZ_CP075138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044885/NZ_CP075137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044888/NZ_CP075135.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044909/NZ_CP075134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044911/NZ_CP075132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044921/NZ_CP075129.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044923/NZ_CP075128.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044925/NZ_CP075127.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN044945/NZ_CP075125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN045763/NZ_CP029038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN045764/NZ_CP029039.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN047866/NZ_CP029040.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051295/NZ_CP029041.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051296/NZ_CP029336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051827/NZ_CP075122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051873/NZ_CP022003.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051882/NZ_CP075120.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN051890/NZ_CP075118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058540/NZ_CP077710.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058545/NZ_CP077706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058565/NZ_CP077704.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058588/NZ_CP077702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058591/NZ_CP077699.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058598/NZ_CP077696.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058605/NZ_CP077693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058606/NZ_CP077711.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN058620/NZ_CP077691.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN059881/NZ_CP075116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN059883/NZ_CP075115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN059898/NZ_CP075113.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN059903/NZ_CP075112.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN060804/NZ_CP075110.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN060807/NZ_CP075109.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN060808/NZ_CP075108.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN060809/NZ_CP075107.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN064033/NZ_CP028172.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN064034/NZ_CP028169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN064276/NZ_CP075106.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN067216/NZ_CP028318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN067217/NZ_CP028314.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN067218/NZ_CP028311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN070643__4083H_attTn7::_Ptrc-gfpUV-cat__bright_fluorescence__1_lactose_operon/NZ_CP024168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN070644__24N_attTn7::_Ptrc-gfpUV-cat__bright_fluorescence/NZ_CP024165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN076210/NZ_CP033345.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN076211/NZ_CP033344.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN076213/NZ_CP033343.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN076214/NZ_CP033340.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN076215/NZ_CP033338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN087304/NZ_CP037892.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CFSAN096147/NZ_CP044257.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Charizard/NZ_CP085821.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Charmeleon/NZ_CP085822.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Clefable/NZ_CP085817.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Clefairy/NZ_CP085818.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CMCC50041/NZ_CP013097.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CMCC_50503/NZ_CP009559.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CP255/NZ_CP063705.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CT18/NC_003198.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CT_02021853/NC_011205.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM19633/NC_011094.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM44454/NZ_CP016412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_20723/NZ_CP051425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_20744/NZ_CP051416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_20749/NZ_CP051413.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_20760/NZ_CP051410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_20762/NZ_CP051404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_21538/NZ_CP010282.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_21550/NZ_CP010283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_21974/NZ_CP051402.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22425/NZ_CP010279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22429/NZ_CP032396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22436/NZ_CP051398.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22437/NZ_CP051396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22448/NZ_CP051394.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22453/NZ_CP032393.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22462/NZ_CP010280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22504/NZ_CP051391.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22513/NZ_CP010281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22536/NZ_CP051389.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_22543/NZ_CP053055.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24350/NZ_CP051386.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24355/NZ_CP051384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24359/NZ_CP051381.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24362/NZ_CP051380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24374/NZ_CP051372.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24392/NZ_CP051370.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_24399/NZ_CP051368.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_28296/NZ_CP051366.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_28321-a/NZ_CP053051.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_28321-b/NZ_CP051363.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_28322/NZ_CP051358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_28336/NZ_CP051356.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_30148/NZ_CP051353.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_30155/NZ_CP053048.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_30168/NZ_CP051350.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_30176/NZ_CP051349.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_30177/NZ_CP051346.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_33822/NZ_CP051342.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_33836/NZ_CP051338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_34514/NZ_CP051329.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_34526/NZ_CP051320.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_34530/NZ_CP051317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_34981/NZ_CP032390.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_35009/NZ_CP051315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_35153/NZ_CP051313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_35161/NZ_CP051310.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_35189/NZ_CP051307.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_N1543/NZ_CP010284.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_N18486/NZ_CP009561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_N45955/NZ_CP032387.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/CVM_N53043/NZ_CP032384.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/C_cip/NZ_CP074098.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/D23580/NC_016854.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/D7795/NZ_CP063702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/D90/NZ_CP022450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/DA34821/NZ_CP029567.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/DA34827/NZ_CP029593.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/DA34833/NZ_CP029595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/DA34837/NZ_CP029568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/DT2/NC_022544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Durban/NZ_CP007507.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/E98_3139-sc-1927833/NZ_LT905143.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20090641/NZ_CP007249.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20090698/NZ_CP007248.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20100101/NZ_CP007246.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20100325/NZ_CP007360.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110221/NZ_CP007247.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110354/NZ_CP007175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110355/NZ_CP007250.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110356/NZ_CP007262.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110357/NZ_CP007261.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110358/NZ_CP007260.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110359/NZ_CP007259.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110360/NZ_CP007258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20110361/NZ_CP007263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20111095/NZ_CP007254.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20111174/NZ_CP007253.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20111175/NZ_CP007252.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20120002/NZ_CP007329.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20120005/NZ_CP007267.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20120008/NZ_CP007245.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20120200/NZ_CP007434.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20121175/NZ_CP007269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20121177/NZ_CP007333.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20121178/NZ_CP007271.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20121179/NZ_CP007272.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EC20121180/NZ_CP007273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Entei/NZ_CP085808.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/EQAS2016S1/NZ_CP033360.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ER3625/NZ_CP067091.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ERL082356/NZ_LT905140.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FC745/NZ_CP063684.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FCC0123/NZ_CP040379.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDA925033-2-17/NZ_CP037891.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDA957831-1-1/NZ_CP017719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_1066/NZ_CP066047.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_1067/NZ_CP066009.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_1271/NZ_CP069518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_312/NZ_CP022062.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_313/NZ_CP022069.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_317/NZ_CP027410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_319/NZ_CP027412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_320/NZ_CP027414.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_321/NZ_CP022070.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_368/NZ_CP023508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_373/NZ_CP023512.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_54/NZ_CP026976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_687/NZ_CP046283.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_688/NZ_CP046280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_70/NZ_CP026052.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_707/NZ_CP046279.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_708/NZ_CP046278.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_710/NZ_CP046277.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_712/NZ_CP046291.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_765/NZ_CP041011.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_768/NZ_CP041005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_878/NZ_CP065718.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_928/NZ_CP065639.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FDAARGOS_94/NZ_CP014051.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FJC33/NZ_CP041699.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC098/NZ_CP030029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC50/NZ_CP019383.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC58/NZ_CP020565.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC88/NZ_CP029029.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC93/NZ_CP032304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_007/NZ_CP009768.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_015/NZ_CP011365.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_019/NZ_CP012396.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_020/NZ_CP012144.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_038/NZ_CP015574.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_051/NZ_CP017232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_052/NZ_CP016754.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_056/NZ_CP017177.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_074/NZ_CP023436.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_075/NZ_CP023475.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_078/NZ_CP026713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_079/NZ_CP025736.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FORC_080/NZ_CP025094.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Froakie/NZ_CP085796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Frogadier/NZ_CP085795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FSIS1502169/NZ_CP016406.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FSIS1502916/NZ_CP016408.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/FSW0104/NZ_CP037894.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GJ0703-2/NZ_CP043509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Glaceon/NZ_CP085803.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Greninja/NZ_CP085794.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GSJ_2016-Sal-017/NZ_CP069297.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GSJ_2016-Sal.-018/NZ_CP069166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GSJ_2017-Sal-008/NZ_CP050130.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GSJ_2017-Sal.-009/NZ_CP050833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GT-01/NZ_CP013222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GT-38/NZ_CP013226.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/GX1006/NZ_CP060585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/HA1-SP5/NZ_CP060134.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/HA3-IN1/NZ_CP060132.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/HFCDC-SM-846/NZ_CP028729.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/HZS154/NZ_CP023345.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/IBG7b4/NZ_CP082338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ISP2825/NZ_CP080960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Ivysaur/NZ_CP085824.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/JT_01/NZ_CP028131.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Kharkiv/NZ_CP062220.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/KUFSE-SAL0043/NZ_CP047424.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/K_SA184/NZ_CP061159.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-3835/NZ_AP023288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-3837/NZ_AP023289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-3844/NZ_AP023290.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4126/NZ_AP023291.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4233/NZ_AP023292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4261/NZ_AP023294.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4334/NZ_AP023299.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4445/NZ_AP023300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4526/NZ_AP023303.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4551/NZ_AP023304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4567/NZ_AP023306.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4578/NZ_AP023309.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4596/NZ_AP023311.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4605/NZ_AP023313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4614/NZ_AP023315.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4681/NZ_AP023317.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/L-4741/NZ_AP023319.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Latias/NZ_CP085798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Latios/NZ_CP085797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LB5000/NZ_CP067397.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Leafeon/NZ_CP085804.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Levine_1/NZ_CP015923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Levine_15/NZ_CP015924.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LG24/NZ_CP045056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LG25/NZ_CP045059.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LG26/NZ_CP045063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_AZ23/NZ_CP079713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_D1/NZ_CP078142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_E3/NZ_CP079839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_SA1/NZ_CP079836.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_SA2/NZ_CP079837.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LHICA_SA3/NZ_CP079838.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LSP_389_97/NZ_CP018219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LT2/NZ_CP060507.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LT7/NZ_CP064263.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/lupe_GEN0059-sc-1979081/NZ_LT905063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/LXYSH/NZ_CP030936.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/M-3471/NZ_CP045458.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/M-3851/NZ_CP045461.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/M-5360/NZ_CP045509.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MAC15/NZ_CP030749.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Marill/NZ_CP085806.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MDUST255/NZ_OU943337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MDUST305/NZ_OU943336.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MDUST348/NZ_OU943338.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MFDS1004024/NZ_CP025745.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MFDS1004839/NZ_CP026569.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Mightyena/NZ_CP085792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Minun/NZ_CP085813.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Moltres/NZ_CP085810.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/MZ0080/NZ_CP022142.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/N13-01290/NZ_CP012930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/N1529-D3/NZ_CP019410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/N17/NZ_CP026379.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/N55391/NZ_CP016410.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NC983/NZ_CP015157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCCP16345/NZ_CP068696.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCCP_16206/NZ_CP041973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCCP_16207/NZ_CP041976.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC10080/NZ_LS483465.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC10384/NZ_LN868943.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC5706/NZ_LT571437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC9787/NZ_LS483457.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC9872/NZ_LS483455.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/NCTC_74/NZ_CP064709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/no75/NZ_CP075372.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-00D989_87-1/NZ_CP011942.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-FSR1_WB_Hawk_ST-33/NZ_CP051267.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE1-1019-1/NZ_CP009083.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE10-10052/NZ_CP009092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE11-10058/NZ_CP009093.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE2-98984-6/NZ_CP009084.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE3-98983-4/NZ_CP009085.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE4-0317-8/NZ_CP009086.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE5-1104-2/NZ_CP009087.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE6-00219-16/NZ_CP009088.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE7-100819/NZ_CP009089.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE8-1021710/NZ_CP009090.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF-SE9-10012/NZ_CP009091.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OLF_FSR1_WB_Finch_ST-13/NZ_CP051269.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/OSF005645/NZ_CP040380.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/P-stx-12/NC_016832.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/P125109/NC_011294.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/P125109/NZ_CP063700.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/P164045/NZ_CP092861.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/P281972/NZ_CP092075.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PIR00532/NZ_CP025559.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PIR00537/NZ_CP025217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PIR00538/NZ_CP025555.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PJM1/NZ_CP045038.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Plusle/NZ_CP085814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PM016_13/NZ_CP012091.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS000211/NZ_CP039716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS007087/NZ_CP044957.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS007098/NZ_CP044967.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS009777/NZ_CP036174.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS009880/NZ_CP040648.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS009887/NZ_CP040318.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS009991/NZ_CP037881.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014846/NZ_CP039558.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014847/NZ_CP039561.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014848/NZ_CP039564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014849/NZ_CP039565.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014850/NZ_CP039567.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014851/NZ_CP038847.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014852/NZ_CP039569.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014853/NZ_CP039713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014854/NZ_CP037874.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014855/NZ_CP039572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014857/NZ_CP039579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014858/NZ_CP039582.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014859/NZ_CP039585.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014860/NZ_CP039719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014861/NZ_CP039588.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014862/NZ_CP039591.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014863/NZ_CP039593.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014863/NZ_CP037879.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014864/NZ_CP039854.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014865/NZ_CP039595.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014866/NZ_CP038849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014867/NZ_CP039599.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014868/NZ_CP039603.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014873/NZ_CP039607.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014875/NZ_CP039610.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014875/NZ_CP037882.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014876/NZ_CP039856.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014879/NZ_CP040321.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014880/NZ_CP039860.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS014881/NZ_CP044961.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PNCS015054/NZ_CP037877.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Poochyena/NZ_CP085793.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PT1/NZ_CP043433.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/PU131/NZ_CP026327.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/QH/NZ_CP043773.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/QJ-2D-Sal/NZ_CP022963.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R13.1544/NZ_CP065566.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R15.2267/NZ_CP065564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R16.0431/NZ_CP062004.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R16.0556/NZ_CP061929.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R16.0676/NZ_CP029800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R16.1424/NZ_CP065572.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R17.1451/NZ_CP063294.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R17.2117/NZ_CP063330.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R17.4111/NZ_CP063289.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R17.4301/NZ_CP062795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R17.5974/NZ_CP060730.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R18.0877/NZ_CP037960.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R18.1074/NZ_CP062225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R18.1078/NZ_CP065567.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R18.1656/NZ_CP062223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R19.2839/NZ_CP046429.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R21.2429/NZ_CP090929.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/R21.2430/NZ_CP090930.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Raikou/NZ_CP085809.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RKS2983/NZ_CP006693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RKS4594/NC_012125.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM10961/NZ_CP013702.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM11060/NZ_CP022658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM11065/NZ_CP022663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM13672/NZ_CP047323.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM1984/NZ_CP012513.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM1986/NZ_CP012514.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM2968/NZ_CP028151.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM4283/NZ_CP028157.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RM6836/NC_022525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RSE04/NZ_CP034719.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RSE09/NZ_CP034717.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RSE15/NZ_CP034713.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RSE42/NZ_CP034697.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/RSK2980/NC_010067.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1087/NZ_CP019409.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1267/NZ_CP022019.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1501/NZ_CP019412.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1643/NZ_CP022117.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1687/NZ_CP019416.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1824/NZ_CP019406.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1843/NZ_CP019415.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S-1850/NZ_CP019418.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S.Tm_LT2p22_assembled/NZ_LR881463.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S01-0511/NZ_CP019116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S146/NZ_CP077662.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S15/NZ_CP089209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S16/NZ_CP081858.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S19/NZ_CP092040.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S25/NZ_CP085696.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S29/NZ_CP085699.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S304/NZ_CP061126.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S34/NZ_CP086118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S438/NZ_CP061124.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S441/NZ_CP061122.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S44712/NZ_CP035917.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S499/NZ_CP082954.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S520/NZ_CP061119.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S530/NZ_CP061118.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S585/NZ_CP061115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S61394/NZ_CP035915.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S639/NZ_CP089207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/S90/NZ_CP077670.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA01AB09084001/NZ_CP016530.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA02DT09004001/NZ_CP016521.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA02DT10168701/NZ_CP012921.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA18578/NZ_CP080513.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19950795/NZ_CP030175.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19960848/NZ_CP007280.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19970510/NZ_CP007282.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19970769/NZ_CP007283.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19980677/NZ_CP007285.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19981204/NZ_CP029991.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19981522/NZ_CP007286.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19981857/NZ_CP007288.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19992307/NZ_CP030207.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19992322/NZ_CP007291.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA19994216/NZ_CP007292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20011914/NZ_CP029992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20014981/NZ_CP022503.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20021456/NZ_CP030219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20026234/NZ_CP022489.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20026289/NZ_CP022490.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20030505/NZ_CP022500.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20030536/NZ_CP019413.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20030575/NZ_CP030181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20031245/NZ_CP030235.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20031783/NZ_CP022491.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20034636/NZ_CP022116.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20035215/NZ_CP022494.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20041063/NZ_CP019407.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20041605/NZ_CP030225.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20041608/NZ_CP022138.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20043041/NZ_CP030231.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20044251/NZ_CP022135.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20044414/NZ_CP030209.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20051401/NZ_CP030196.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20051528/NZ_CP030211.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20052327/NZ_CP030202.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20053897/NZ_CP029995.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20055162/NZ_CP030238.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20060086/NZ_CP022034.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20060561/NZ_CP019403.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20061017/NZ_CP038591.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20063285/NZ_CP030288.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20064858/NZ_CP030002.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20070548/NZ_CP040651.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20075157/NZ_CP030217.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20080453/NZ_CP030194.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20082034/NZ_CP007425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20082869/NZ_CP040668.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20083039/NZ_CP030223.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20083530/NZ_CP030203.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20084699/NZ_CP022497.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20085604/NZ_CP022015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20092095/NZ_CP019417.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20094352/NZ_CP007312.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20094620/NZ_CP030185.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20100201/NZ_CP030180.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20101045/NZ_CP030233.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20104250/NZ_CP030190.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20113174/NZ_CP029999.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20113257/NZ_CP019404.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20114778WT/NZ_CP043222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20120345/NZ_CP030024.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20121591/NZ_CP029989.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20130280/NZ_CP038593.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20141895/NZ_CP030005.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SA20143792/NZ_CP041026.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-1135/NZ_CP045464.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-2097/NZ_CP045465.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-3948/NZ_CP045513.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-3973/NZ_CP045466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-4295/NZ_CP045515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-4737/NZ_CP045516.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-5091/NZ_CP045518.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Sal-5364/NZ_CP039169.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP17-7299/NZ_CP040566.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP17-7399/NZ_CP040562.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP17-7699/NZ_CP040564.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP17-8290/NZ_CP040568.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP18-0432/NZ_CP040644.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP18-6199/NZ_CP040900.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP18-8729/NZ_CP041208.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SAP18-H9654/NZ_CP040646.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SB699/NZ_CP091300.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SC-B67/NC_006905.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SCSM4.1/NZ_CP047115.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SCSW714/NZ_CP051213.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE104/NZ_CP050712.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE109/NZ_CP050709.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE124/NZ_CP050706.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Se13/NZ_CP067337.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE211/NZ_CP084532.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Se29/NZ_CP070222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Se32/NZ_CP067339.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE74/NZ_CP050723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE81/NZ_CP050721.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE86/NZ_CP019681.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SE95/NZ_CP050716.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SEE1/NZ_CP011790.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SEE2/NZ_CP011791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SEJ/NZ_CP008928.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SEO/NZ_CP033090.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SG17-135/NZ_CP048775.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg1722-1/NZ_CP081187.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg1722-2/NZ_CP081189.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGB23/NZ_CP023166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGSC_2193/NZ_CP017728.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGSC_2206/NZ_CP017727.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGSC_2215/NZ_CP019176.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGSC_2518/NZ_CP017723.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SGSC_3086/NZ_CP045761.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg_wt5/NZ_CP036165.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg_wt7/NZ_CP036168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg_wt8/NZ_CP036167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/sg_ww281/NZ_CP036166.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH-2813-Parental_2/NZ_CP066851.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH11G0791/NZ_CP041171.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH12-003/NZ_CP016507.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH12-007/NZ_CP016504.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH13-004/NZ_CP016586.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SH13-006/NZ_CP016579.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI102/NZ_CP050771.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI108/NZ_CP050769.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI111/NZ_CP050764.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI115/NZ_CP050762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI170/NZ_CP050760.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI173/NZ_CP050757.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI174/NZ_CP050756.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI43/NZ_CP050785.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI67/NZ_CP050783.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI85/NZ_CP050779.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SI96/NZ_CP050777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10057/NZ_CP047550.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10112/NZ_CP047555.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10169/NZ_CP047548.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10231/NZ_CP047553.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10236/NZ_CP047546.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10250/NZ_CP047544.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10330/NZ_CP047542.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10359/NZ_CP047540.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10405/NZ_CP047537.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10452/NZ_CP047535.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10484/NZ_CP047531.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10640/NZ_CP047529.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10648/NZ_CP047527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10978/NZ_CP015524.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF10984/NZ_CP015526.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF11077/NZ_CP047525.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF11216/NZ_CP047522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF12367v2/NZ_CP041176.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF12519v2/NZ_CP041173.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF13520v2/NZ_CP041181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SJTUF87912v2/NZ_CP041179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL-312/NZ_CP043214.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL1344/NC_016810.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL180013/NZ_AP019692.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL254/NC_011080.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL26/NZ_CP032490.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL476/NC_011083.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL483/NC_011149.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SL7207/NZ_CP053865.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Slowbro/NZ_CP085815.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Slowpoke/NZ_CP085816.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_7627/NZ_CP060517.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_7697/NZ_CP060508.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_7966/NZ_CP060512.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_8094/NZ_CP060515.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_8245/NZ_CP080091.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SLR1_8250/NZ_CP060522.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SO2/NZ_CP014356.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SO21/NZ_CP032494.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SO3/NZ_CP014536.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SO4698-09/NZ_LN999997.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SO_8752_Stm/NZ_CP074092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SP/NZ_CP077668.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SPB7/NC_010102.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SR33/NZ_CP092911.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SRC27/NZ_CP058807.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SS2017/NZ_CP053870.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST101/NZ_CP050731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST106/NZ_CP050728.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST1120/NZ_CP021909.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST113/NZ_CP050726.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST114/NZ_CP022467.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST1539/NZ_CP035301.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST203/NZ_CP019411.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST278/NZ_CP019414.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST350/NZ_CP019405.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST45/NZ_CP050753.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST46/NZ_CP050750.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST4_74/NC_016857.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST90/NZ_CP050734.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ST931R/NZ_CP016385.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/STMU2UK/NZ_LT855376.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/subsp._enterica_serovar_Typhimurium_str._LT2/NC_003197.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Suicune/NZ_CP085807.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Swellow/NZ_CP085799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SZL_30/NZ_CP085981.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SZL_31/NZ_CP085983.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/SZL_38/NZ_CP085987.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Taillow/NZ_CP085800.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/TJWQ005/NZ_CP040458.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/TW-Stm6/NZ_CP019649.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/TXSC_TXSC08-19/NZ_CP007505.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Ty2/NC_004631.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Ty21a/NC_021176.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Ty21a/NZ_CP023975.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/UFPRLABMOR1/NZ_CP020101.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/UK-1/NC_016863.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1175/NZ_CP007483.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1676/NZ_CP014620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1677/NZ_CP014663.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1727/NZ_CP014621.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1728/NZ_CP014664.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1735/NZ_CP007584.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1736/NZ_CP014657.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1765/NZ_CP014659.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1766/NZ_CP014665.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1781/NZ_CP014666.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1783/NZ_CP014661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1808/NZ_CP014969.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1810/NZ_CP014982.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1880/NZ_CP014981.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1896/NZ_CP014977.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1898/NZ_CP014971.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1899/NZ_CP007235.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1900/NZ_CP017970.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1901/NZ_CP017971.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1903/NZ_CP007222.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1904/NZ_CP017972.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1912/NZ_CP017973.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1913/NZ_CP025278.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1923/NZ_CP025273.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1924/NZ_CP025230.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1925/NZ_CP025232.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1926/NZ_CP025234.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1927/NZ_CP007216.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1928/NZ_CP025237.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-1929/NZ_CP025241.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-60983/NZ_CP025280.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-60984/NZ_CP025276.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76332/NZ_CP025445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76333/NZ_CP025446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76334/NZ_CP025454.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76335/NZ_CP025447.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76336/NZ_CP025448.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76337/NZ_CP025449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76338/NZ_CP025450.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76339/NZ_CP025451.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76340/NZ_CP025452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA-ARS-USMARC-76341/NZ_CP025453.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USDA15WA-1/NZ_CP040686.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USMARC-69807/NZ_CP032379.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USMARC-69835/NZ_CP032444.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USMARC-69838/NZ_CP032449.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USMARC-69840/NZ_CP032446.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/USMARC-S3124.1/NC_021902.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Venusaur/NZ_CP085823.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNB151-sc-2315230/NZ_LT795114.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNP20009/NZ_CP007804.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC001/NZ_CP039445.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC002/NZ_CP039443.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC003/NZ_CP039441.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC013/NZ_CP039439.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC023/NZ_CP039437.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/VNSEC031/NZ_CP039436.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Wailmer/NZ_CP085802.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Wailord/NZ_CP085801.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Wartortle/NZ_CP085820.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/WA_14882/NZ_CP016357.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/WGS1146/NZ_CP040575.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/WW012/NZ_CP022168.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/XXB1403/NZ_CP059886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/YU07-18/NZ_CP035547.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/YU15/NZ_CP014358.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/YU39/NZ_CP011428.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/YZ20MCS16/NZ_CP077680.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/YZ21MCS4/NZ_CP089313.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/Zapdos/NZ_CP085811.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZC-S1_3rd/NZ_CP082916.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZJUJY/NZ_CP090304.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00785/NZ_CP089799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00789/NZ_CP089798.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00790/NZ_CP089797.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00813/NZ_CP089795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00814/NZ_CP089794.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00816/NZ_CP089792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00820/NZ_CP089791.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00830/NZ_CP089790.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00831/NZ_CP089789.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/enterica/ZTA19_00847/NZ_CP089788.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./A29-2/NZ_CP083731.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./A39/NZ_CP084194.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./A7/NZ_CP084001.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./JXY0409-18/NZ_CP084216.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SAL-007/NZ_CP071686.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SAL-020/NZ_CP071690.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SAL-045/NZ_CP071693.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SCFS4/NZ_CP051218.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14076/NZ_CP064674.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14146/NZ_CP064672.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14152/NZ_CP064671.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14154/NZ_CP064666.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14170/NZ_CP064663.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14178/NZ_CP064661.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF14523/NZ_CP074428.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SJTUF15034/NZ_CP074425.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SSDFZ54/NZ_CP034819.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Salmonella/sp./SSDFZ69/NZ_CP034831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Scandinavium/goeteborgense/CCUG_66741/NZ_CP054058.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/59-248/NZ_CP026766.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/600657/NZ_CP049281.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/602068/NZ_CP049285.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/83-578/NZ_CP026814.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/ATCC_9210/NZ_CP011511.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/ATCC_BAA-1247/NZ_CP026795.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/BS512__CDC_3083-94/NC_010658.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/FDAARGOS_1139/NZ_CP068090.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/KCCM_41690/NZ_CP041513.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/NCTC_9850/NZ_CP026762.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/boydii/Sb227/NC_007613.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/07-3308/NZ_CP026878.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/08-3380/NZ_CP026782.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/53-3937/NZ_CP026780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/69-3818/NZ_CP026777.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/80-547/NZ_CP026784.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/ATCC_13313/NZ_CP026774.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/BU53M1/NZ_CP024466.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/HNCMB_20080/NZ_CP061527.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/NCTC_9718/NZ_CP026786.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/Sd197/NC_007606.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/SWHE2/NZ_CP055063.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/SWHEFF_49/NZ_CP055055.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/dysenteriae/SWHEFF_51/NZ_CP055052.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/0228/NZ_CP012735.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/2002017/NC_017328.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/2003036/NZ_CP004056.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/2457T/NC_004741.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/301/NC_004337.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/61-4982/NZ_CP026792.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/83/NZ_LR878365.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/8401/NC_008258.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/981/NZ_CP012137.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/AUSMDU00008332/NZ_LR213455.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/AUSMDU00008355/NZ_LR213452.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/C32/NZ_CP041620.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/FDAARGOS_691/NZ_CP054913.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/M90T/NZ_CP037923.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/NCTC_9728/NZ_CP026799.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/PE577/NZ_CP042980.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_10/NZ_CP055194.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_12/NZ_CP055219.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_16/NZ_CP055184.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_17/NZ_CP055181.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_18/NZ_CP055179.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_22/NZ_CP055170.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_24/NZ_CP055167.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STEFF_3/NZ_CP055199.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STIN_80/NZ_CP055000.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STIN_87/NZ_CP054992.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STIN_92/NZ_CP054977.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLE1/NZ_CP058826.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLE3/NZ_CP058828.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLE4/NZ_CP058831.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_27/NZ_CP058833.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_33/NZ_CP058886.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_34/NZ_CP058839.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_35/NZ_CP058846.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_36/NZ_CP058849.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLEFF_38/NZ_CP058853.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLIN_17/NZ_CP058780.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLIN_4/NZ_CP058786.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLIN_6/NZ_CP058790.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/STLIN_8/NZ_CP058796.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHEFF_59/NZ_CP055041.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHEFF_60/NZ_CP055037.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHEFF_64/NZ_CP055030.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHEFF_67/NZ_CP055024.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHEFF_71/NZ_CP055015.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_101/NZ_CP055113.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_104/NZ_CP055109.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_107/NZ_CP055099.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_110/NZ_CP055092.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_116/NZ_CP055075.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/SWHIN_97/NZ_CP055070.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/flexneri/WW1/NZ_CP081813.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/sonnei/FDAARGOS_90/NZ_CP014099.2.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/sonnei/FORC_011/NZ_CP010829.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shigella/sonnei/SE6-1/NZ_CP055292.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shimwellia/blattae/DSM_4481/NC_017910.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shimwellia/blattae/NCTC10965/NZ_LR133996.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Shimwellia/blattae/NCTC12127/NZ_LR134215.1.chrom.fna...\n",
      "Concatenating /mnt/e/ecoli_db/ref_genomes/human_readable/refseq/bacteria/Yokenella/regensburgei/W13/NZ_CP050811.1.chrom.fna...\n",
      "\n",
      "\n",
      "Building a new DB, current time: 10/19/2023 13:56:24\n",
      "New DB name:   /mnt/e/ecoli_db/blast_db/Enterobacteriaceae_refseq\n",
      "New DB title:  Enterobacteriaceae_refseq\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 3000000000B\n",
      "Adding sequences from FASTA; added 5406 sequences in 172.652 seconds.\n",
      "\n",
      "\n",
      "/home/youn/work/chronostrain/examples/database\n"
     ]
    }
   ],
   "source": [
    "!env BLAST_DB_DIR=$BLAST_DB_DIR \\\n",
    "    BLAST_DB_NAME=$BLAST_DB_NAME \\\n",
    "    bash create_blast_db.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f4bc0bf6-5e58-4312-9cd5-aef1e8f27a7a",
   "metadata": {
    "collapsed": false,
    "jupyter": {
     "outputs_hidden": false
    }
   },
   "source": [
    "### Step 3: Build the marker seed catalog.\n",
    "\n",
    "**GOAL**: a FASTA file of marker seeds (one multi-fasta file per marker gene), and a single TSV file that catalogs them.\n",
    "\n",
    "However, to get there, we need to take a few steps...\n",
    "\n",
    "#### Step 3.1: Download MLST schema."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "0dae5906-67df-4358-a75b-f90f3612e4b1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading marker seeds from MLST schema.\n",
      "Targeting 1 taxa using MLST scheme.\n",
      "Fetching URL resource https://pubmlst.org/static/data/dbases.xml\n",
      "Got a response of size 152.35 KB.\n",
      "Schema type id: 1\n",
      "Handling locus adk\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/adk/alleles_fasta\n",
      "Got a response of size 659.89 KB.\n",
      "Handling locus fumC\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/fumC/alleles_fasta\n",
      "Got a response of size 941.86 KB.\n",
      "Handling locus gyrB\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/gyrB/alleles_fasta\n",
      "Got a response of size 552.74 KB.\n",
      "Handling locus icd\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/icd/alleles_fasta\n",
      "Got a response of size 842.08 KB.\n",
      "Handling locus mdh\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/mdh/alleles_fasta\n",
      "Got a response of size 571.89 KB.\n",
      "Handling locus purA\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/purA/alleles_fasta\n",
      "Got a response of size 537.86 KB.\n",
      "Handling locus recA\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/recA/alleles_fasta\n",
      "Got a response of size 488.47 KB.\n",
      "Schema type id: 2\n",
      "Handling locus dinB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/dinB/alleles_fasta\n",
      "Got a response of size 93.08 KB.\n",
      "Handling locus icdA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/icdA/alleles_fasta\n",
      "Got a response of size 207.75 KB.\n",
      "Handling locus pabB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/pabB/alleles_fasta\n",
      "Got a response of size 111.69 KB.\n",
      "Handling locus polB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/polB/alleles_fasta\n",
      "Got a response of size 113.34 KB.\n",
      "Handling locus putP\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/putP/alleles_fasta\n",
      "Got a response of size 117.1 KB.\n",
      "Handling locus trpA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/trpA/alleles_fasta\n",
      "Got a response of size 150.16 KB.\n",
      "Handling locus trpB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/trpB/alleles_fasta\n",
      "Got a response of size 136.96 KB.\n",
      "Handling locus uidA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/uidA/alleles_fasta\n",
      "Got a response of size 164.58 KB.\n"
     ]
    }
   ],
   "source": [
    "!python python_helpers/mlst_download.py -t \"Escherichia coli\" -w \"$TARGET_DIR\"/mlst_schema -o \"$MARKER_SEED_DIR\"/mlst_seeds.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4ac5db29-e00b-4796-ba71-7359551294b6",
   "metadata": {},
   "source": [
    "#### Step 3.2: Non-standard genes: run GFF-annotated gene search with primers across entire catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "2b6958c6-4052-444f-9fa4-779998686823",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Performing primer-based search for trpA in Escherichia coli. (FWD=GCTACGAATCTCTGTTTGCC, REV=CGCTTTCATCGGTTGTACA, len approx. 783)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:39<00:00,  7.38genome/s]\n",
      "Wrote 1787 dataframe records to trpA.feather\n",
      "Performing primer-based search for trpBA in Escherichia coli. (FWD=CGGCGATAAAGACATCTTCAC, REV=GCAACGCGGCCTGGCGGAAG, len approx. 489)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:43<00:00,  7.29genome/s]\n",
      "Wrote 2046 dataframe records to trpBA.feather\n",
      "Performing primer-based search for chuA in Escherichia coli. (FWD=ATGGTACCGGACGAACCAAC, REV=TGCCGCCAGTACCAAAGACA, len approx. 288)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:10<00:00,  8.24genome/s]\n",
      "Wrote 859 dataframe records to chuA.feather\n",
      "Performing primer-based search for yjaA in Escherichia coli. (FWD=CAAACGTGAAGTGTCAGGAG, REV=AATGCGTTCCTCAACCTGTG, len approx. 211)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:06<00:00,  8.38genome/s]\n",
      "Wrote 787 dataframe records to yjaA.feather\n",
      "Performing primer-based search for TspE4.C2 in Escherichia coli. (FWD=CACTATTCGTAAGGTCATCC, REV=AGTTTATCGCTGCGGGTCGC, len approx. 152)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:07<00:00,  8.33genome/s]\n",
      "Wrote 717 dataframe records to TspE4.C2.feather\n",
      "Performing primer-based search for arpA in Escherichia coli. (FWD=AACGCTATTCGCCAGCTTGC, REV=TCTCCCCATACCGTACGCTA, len approx. 400)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:18<00:00,  7.97genome/s]\n",
      "Wrote 1341 dataframe records to arpA.feather\n",
      "Performing primer-based search for ArpAgpE in Escherichia coli. (FWD=GATTCCATCTTGTCAAAATATGCC, REV=GAAAAGAAAAAGAATTCCCAAGAG, len approx. 301)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:59<00:00,  8.63genome/s]\n",
      "Wrote 261 dataframe records to ArpAgpE.feather\n",
      "Performing primer-based search for trpAgpC in Escherichia coli. (FWD=AGTTTTATGCCCAGTGCGAG, REV=TCTGCGCCGGTCACGCCC, len approx. 219)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:11<00:00,  8.20genome/s]\n",
      "Wrote 804 dataframe records to trpAgpC.feather\n",
      "Performing primer-based search for aesI in Escherichia coli. (FWD=CCTCTACTCACCCAAAAGTC, REV=ATCACGTAACCACAACGCAC, len approx. 315)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:45<00:00,  9.16genome/s]\n",
      "Wrote 1 dataframe records to aesI.feather\n",
      "Performing primer-based search for aesII in Escherichia coli. (FWD=CGCCTGTTGTCACTTCCACG, REV=GTTTATCACGCAGCCACAAG, len approx. 125)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:49<00:00,  9.00genome/s]\n",
      "Wrote 0 dataframe records to aesII.feather\n",
      "Performing primer-based search for chuIII in Escherichia coli. (FWD=GTGTTGAGATTGTCCGTGGG, REV=CAAAAGCACTGGCGCCCAG, len approx. 183)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:48<00:00,  9.02genome/s]\n",
      "Wrote 0 dataframe records to chuIII.feather\n",
      "Performing primer-based search for chuIV in Escherichia coli. (FWD=CTGGCGAAAGGAACCTGGA, REV=GTTATCTCATCTTGCAGCCAA, len approx. 461)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:50<00:00,  8.93genome/s]\n",
      "Wrote 0 dataframe records to chuIV.feather\n",
      "Performing primer-based search for chuV in Escherichia coli. (FWD=ACTGTATGGCAGTGGCGCAT, REV=GCAAAACTATCGGCAAACAGC, len approx. 600)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [03:48<00:00,  9.04genome/s]\n",
      "Wrote 0 dataframe records to chuV.feather\n",
      "Performing primer-based search for ybgD in Escherichia coli. (FWD=GTTGACTAAGCGCAGGTCGA, REV=TATGCGGCTGATGAAGGATC, len approx. 177)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 2063/2063 [04:08<00:00,  8.29genome/s]\n",
      "Wrote 769 dataframe records to ybgD.feather\n"
     ]
    }
   ],
   "source": [
    "\"\"\"\n",
    "Search #1:\n",
    "-------------\n",
    "Search for O-antigen gene cluster.\n",
    "JumpSTART 5'--3' primer = CATGGTAGCTGTAAAGCCAGGGGCGGTAGCGTG\n",
    "GND 5'--3' primer = CATGCTGCCATACCGACGACGCCGATCTGTTGCTTKGACA\n",
    "\n",
    "\n",
    "Phylotyping gene primers from: https://github.com/A-BN/ClermonTyping\n",
    "\"\"\"\n",
    "\n",
    "def primer_search_cluster(gene_name: str, forward_primer: str, rev_primer: str):\n",
    "    !python python_helpers/identify_gene_cluster_primersearch.py \\\n",
    "        -i \"$REFSEQ_INDEX\" \\\n",
    "        -t \"$TARGET_DIR\"/_tmp \\\n",
    "        -o \"$MARKER_SEED_DIR\"/\"$gene_name\".feather \\\n",
    "        -g \"Escherichia\" -s \"coli\" \\\n",
    "        -p1 \"$forward_primer\" -p2 \"$rev_primer\" \\\n",
    "        --mismatch-pct 30 \\\n",
    "        -n \"$gene_name\" \\\n",
    "        --use-gff\n",
    "def primer_search_gene(gene_name: str, forward_primer: str, rev_primer: str, expected_len: int):\n",
    "    !python python_helpers/identify_gene_cluster_primersearch.py \\\n",
    "        -i \"$REFSEQ_INDEX\" \\\n",
    "        -t \"$TARGET_DIR\"/_tmp \\\n",
    "        -o \"$MARKER_SEED_DIR\"/\"$gene_name\".feather \\\n",
    "        -g \"Escherichia\" -s \"coli\" \\\n",
    "        -p1 \"$forward_primer\" -p2 \"$rev_primer\" \\\n",
    "        --mismatch-pct 0 \\\n",
    "        -n \"$gene_name\" \\\n",
    "        -l \"$expected_len\" \\\n",
    "        --dont-use-gff\n",
    "\n",
    "# Note: Here, we use GFF annotation to extract the specific genes from the cluster.\n",
    "# primer_search_cluster(\"O_antigen_cluster\", \"CATGGTAGCTGTAAAGCCAGGGGCGGTAGCGTG\", \"CATGCTGCCATACCGACGACGCCGATCTGTTGCTTKGACA\", use_gff_annotations=True)\n",
    "\n",
    "primer_search_gene(\"trpA\", \"GCTACGAATCTCTGTTTGCC\", \"CGCTTTCATCGGTTGTACA\", 783)\n",
    "primer_search_gene(\"trpBA\", \"CGGCGATAAAGACATCTTCAC\", \"GCAACGCGGCCTGGCGGAAG\", 489)\n",
    "primer_search_gene(\"chuA\", \"ATGGTACCGGACGAACCAAC\", \"TGCCGCCAGTACCAAAGACA\", 288)\n",
    "primer_search_gene(\"yjaA\", \"CAAACGTGAAGTGTCAGGAG\", \"AATGCGTTCCTCAACCTGTG\", 211)\n",
    "primer_search_gene(\"TspE4.C2\", \"CACTATTCGTAAGGTCATCC\", \"AGTTTATCGCTGCGGGTCGC\", 152)\n",
    "primer_search_gene(\"arpA\", \"AACGCTATTCGCCAGCTTGC\", \"TCTCCCCATACCGTACGCTA\", 400)\n",
    "primer_search_gene(\"ArpAgpE\", \"GATTCCATCTTGTCAAAATATGCC\", \"GAAAAGAAAAAGAATTCCCAAGAG\", 301)\n",
    "primer_search_gene(\"trpAgpC\", \"AGTTTTATGCCCAGTGCGAG\", \"TCTGCGCCGGTCACGCCC\", 219)\n",
    "primer_search_gene(\"aesI\", \"CCTCTACTCACCCAAAAGTC\", \"ATCACGTAACCACAACGCAC\", 315)\n",
    "primer_search_gene(\"aesII\", \"CGCCTGTTGTCACTTCCACG\", \"GTTTATCACGCAGCCACAAG\", 125)\n",
    "primer_search_gene(\"chuIII\", \"GTGTTGAGATTGTCCGTGGG\", \"CAAAAGCACTGGCGCCCAG\", 183)\n",
    "primer_search_gene(\"chuIV\", \"CTGGCGAAAGGAACCTGGA\", \"GTTATCTCATCTTGCAGCCAA\", 461)\n",
    "primer_search_gene(\"chuV\", \"ACTGTATGGCAGTGGCGCAT\", \"GCAAAACTATCGGCAAACAGC\", 600)\n",
    "primer_search_gene(\"ybgD\", \"GTTGACTAAGCGCAGGTCGA\", \"TATGCGGCTGATGAAGGATC\", 177)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "293c0fa5-a10a-4127-a7a6-b4716dc124ea",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "100%|███████████████████████████████████████| 2063/2063 [08:17<00:00,  4.15it/s]\n"
     ]
    }
   ],
   "source": [
    "\"\"\"\n",
    "Search #2:\n",
    "------------\n",
    "fim genes: fim*\n",
    "H antigen: fliC, flk* fll* flm*\n",
    "Shigatoxin: stx*\n",
    "\"\"\"\n",
    "!python python_helpers/extract_genes_by_name.py \\\n",
    "    -i \"$REFSEQ_INDEX\" \\\n",
    "    -o \"$MARKER_SEED_DIR\"/misc_genes.feather \\\n",
    "    -g \"Escherichia\" -s \"coli\" \\\n",
    "    -p \"fim\" -p \"fliC\" -p \"flk\" -p \"fll\" -p \"flm\" -p \"stx\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5f820b76-edcf-4068-909c-97c920146f19",
   "metadata": {},
   "source": [
    "#### Step 3.3: Convert previous step to marker seed (multi-)FASTA files.\n",
    "\n",
    "ChronoStrain uses FASTA files to read in marker seeds. The previous step merely creates a catalog of raw gene catalog.\n",
    "\n",
    "We need to generate a FASTA file, but some cleanup has to be done, since sometimes genes are mis-annotated. (example: \"stx1A\" sometimes shows up as \"stxA1\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "id": "2493c953-c191-42cf-9e28-30bb96be5f97",
   "metadata": {},
   "outputs": [],
   "source": [
    "gene_dfs = []"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "id": "5fe3de48-383e-4759-bf98-627edd3dee45",
   "metadata": {},
   "outputs": [],
   "source": [
    "for gene_name in ['arpA', 'chuA', 'yjaA', 'tspE4.C4', 'ArpAgpE.f', 'trpAgpC.1']:\n",
    "    feather_file = MARKER_SEED_DIR / f'{gene_name}.feather'\n",
    "    df = pd.read_feather(feather_file)\n",
    "    if df.shape[0] > 1000:\n",
    "        df = df.sample(n=1000)\n",
    "    if df.shape[0] > 0:\n",
    "        gene_dfs.append(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "857e35b5-0ead-44e5-b6da-72ff9aefe5d3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Raw counts:\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Gene\n",
       "fim41a       1\n",
       "fimA      4701\n",
       "fimB      1625\n",
       "fimC      3420\n",
       "fimD      1719\n",
       "fimE      1765\n",
       "fimF      1858\n",
       "fimG      1872\n",
       "fimH      3469\n",
       "fimI      1771\n",
       "fimZ      1642\n",
       "fliC       220\n",
       "flk       2049\n",
       "stx1A        1\n",
       "stx1B        1\n",
       "stx2A        1\n",
       "stx2B        1\n",
       "stxA1      166\n",
       "stxA2       49\n",
       "stxB1        2\n",
       "stxB2       18\n",
       "Name: Counts Per Gene, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "After fixing & subsampling: \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Gene\n",
       "fim41a       1\n",
       "fimA      1000\n",
       "fimB      1000\n",
       "fimC      1000\n",
       "fimD      1000\n",
       "fimE      1000\n",
       "fimF      1000\n",
       "fimG      1000\n",
       "fimH      1000\n",
       "fimI      1000\n",
       "fimZ      1000\n",
       "fliC       220\n",
       "flk       1000\n",
       "stxA1      167\n",
       "stxA2       50\n",
       "stxB1        3\n",
       "stxB2       19\n",
       "Name: Counts Per Gene, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "np.random.seed(1234)  # for reproducibility\n",
    "\n",
    "# ========== Load the dataframe.\n",
    "misc_df = pd.read_feather(MARKER_SEED_DIR / \"misc_genes.feather\")\n",
    "misc_df['GeneFull'] = misc_df['Gene']\n",
    "index_df = pd.read_csv(\"/mnt/e/ecoli_db/ref_genomes/index.tsv\", sep='\\t')\n",
    "misc_df = misc_df.merge(index_df[['Accession', 'Species']], on='Accession')\n",
    "misc_df = misc_df.loc[misc_df['Species'] == 'coli']  # this is probably no longer needed as of latest version of this notebook.\n",
    "\n",
    "print(\"Raw counts:\")\n",
    "display(misc_df.groupby(\"Gene\")['Accession'].count().rename(\"Counts Per Gene\"))\n",
    "\n",
    "# Subsample if more than 1000 hits.\n",
    "misc_df = misc_df.groupby(\"Gene\").apply(\n",
    "    lambda x: x.sample(n=min(1000, x.shape[0]))\n",
    ")\n",
    "misc_df = misc_df.set_index([pd.Index(list(range(misc_df.shape[0])))])\n",
    "\n",
    "\n",
    "# STX labels are inconsistent. Fix them.\n",
    "misc_df.loc[misc_df['Gene'] == 'stx1A', 'Gene'] = 'stxA1'\n",
    "misc_df.loc[misc_df['Gene'] == 'stx2A', 'Gene'] = 'stxA2'\n",
    "misc_df.loc[misc_df['Gene'] == 'stx1B', 'Gene'] = 'stxB1'\n",
    "misc_df.loc[misc_df['Gene'] == 'stx2B', 'Gene'] = 'stxB2'\n",
    "\n",
    "# print statistics\n",
    "print(\"After fixing & subsampling: \")\n",
    "display(misc_df.groupby(\"Gene\")['Accession'].count().rename(\"Counts Per Gene\"))\n",
    "\n",
    "gene_dfs.append(misc_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "a9685435-d769-425f-95c3-18e5f296e5b9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Raw counts: \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Gene\n",
       "A0I22_RS25315       1\n",
       "A0I22_RS25325       1\n",
       "A0I22_RS25340       1\n",
       "A0I22_RS25350       1\n",
       "A0I22_RS25355       1\n",
       "                 ... \n",
       "wzy              1324\n",
       "yfgO                1\n",
       "ypfH                1\n",
       "ypfJ                1\n",
       "ypfN                1\n",
       "Name: Accession, Length: 11232, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "After fixing & subsampling: \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Gene\n",
       "acs        1\n",
       "arnB       1\n",
       "asnB      11\n",
       "bamC       1\n",
       "bcp        1\n",
       "        ... \n",
       "wzy     1000\n",
       "yfgO       1\n",
       "ypfH       1\n",
       "ypfJ       1\n",
       "ypfN       1\n",
       "Name: Accession, Length: 95, dtype: int64"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "np.random.seed(1235)  # for reproducibility\n",
    "\n",
    "# ========== Load the dataframe.\n",
    "serotype_O_df = pd.read_feather(MARKER_SEED_DIR / \"O_antigen_cluster.feather\")\n",
    "serotype_O_df['GeneFull'] = serotype_O_df['Gene']\n",
    "\n",
    "index_df = pd.read_csv(\"/mnt/e/ecoli_db/ref_genomes/index.tsv\", sep='\\t')\n",
    "serotype_O_df = serotype_O_df.merge(index_df[['Accession', 'Species']], on='Accession')\n",
    "serotype_O_df = serotype_O_df.loc[serotype_O_df['Species'] == 'coli']\n",
    "\n",
    "print(\"Raw counts: \")\n",
    "display(serotype_O_df.groupby(\"Gene\")['Accession'].count())\n",
    "\n",
    "\n",
    "# Subsample if more than 1000 hits.\n",
    "serotype_O_df = serotype_O_df.groupby(\"Gene\").apply(\n",
    "    lambda x: x.sample(n=min(1000, x.shape[0]))\n",
    ")\n",
    "serotype_O_df = serotype_O_df.set_index([pd.Index(list(range(serotype_O_df.shape[0])))])\n",
    "\n",
    "\n",
    "# Remove all putative genes (e.g. WM47_RS18200). Annotated genes typically follow a 3-letter format (followed by a specifier).\n",
    "ecoli_serotype_O_genes = set()\n",
    "for x in serotype_O_df.groupby(\"Gene\")['Accession'].count().index:\n",
    "    if \"_\" in x:\n",
    "        continue\n",
    "    ecoli_serotype_O_genes.add(x)\n",
    "serotype_O_df = serotype_O_df.loc[serotype_O_df['Gene'].isin(ecoli_serotype_O_genes)]\n",
    "\n",
    "\n",
    "# print statistics\n",
    "print(\"After fixing & subsampling: \")\n",
    "display(serotype_O_df.groupby(\"Gene\")['Accession'].count())\n",
    "\n",
    "gene_dfs.append(serotype_O_df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 59,
   "id": "0e36f53e-1e51-4efa-b3d2-ea44f6249198",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Concatenating.\n",
      "# CUSTOM GENES TOTAL = 118\n",
      "Generating FASTA files.\n",
      "Handling ArpAgpE.f (1000 records) --> /mnt/e/ecoli_db/marker_seeds/ArpAgpE.f.fasta\n",
      "Handling acs (1 records) --> /mnt/e/ecoli_db/marker_seeds/acs.fasta\n",
      "Handling arnB (1 records) --> /mnt/e/ecoli_db/marker_seeds/arnB.fasta\n",
      "Handling arpA (1000 records) --> /mnt/e/ecoli_db/marker_seeds/arpA.fasta\n",
      "Handling asnB (11 records) --> /mnt/e/ecoli_db/marker_seeds/asnB.fasta\n",
      "Handling bamC (1 records) --> /mnt/e/ecoli_db/marker_seeds/bamC.fasta\n",
      "Handling bcp (1 records) --> /mnt/e/ecoli_db/marker_seeds/bcp.fasta\n",
      "Handling bepA (1 records) --> /mnt/e/ecoli_db/marker_seeds/bepA.fasta\n",
      "Handling chuA (859 records) --> /mnt/e/ecoli_db/marker_seeds/chuA.fasta\n",
      "Handling cpsB (172 records) --> /mnt/e/ecoli_db/marker_seeds/cpsB.fasta\n",
      "Handling cpsG (570 records) --> /mnt/e/ecoli_db/marker_seeds/cpsG.fasta\n",
      "Handling dapA (1 records) --> /mnt/e/ecoli_db/marker_seeds/dapA.fasta\n",
      "Handling dapE (1 records) --> /mnt/e/ecoli_db/marker_seeds/dapE.fasta\n",
      "Handling fcf1 (1 records) --> /mnt/e/ecoli_db/marker_seeds/fcf1.fasta\n",
      "Handling fcl (111 records) --> /mnt/e/ecoli_db/marker_seeds/fcl.fasta\n",
      "Handling fdhF (1 records) --> /mnt/e/ecoli_db/marker_seeds/fdhF.fasta\n",
      "Handling fim41a (1 records) --> /mnt/e/ecoli_db/marker_seeds/fim41a.fasta\n",
      "Handling fimA (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimA.fasta\n",
      "Handling fimB (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimB.fasta\n",
      "Handling fimC (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimC.fasta\n",
      "Handling fimD (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimD.fasta\n",
      "Handling fimE (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimE.fasta\n",
      "Handling fimF (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimF.fasta\n",
      "Handling fimG (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimG.fasta\n",
      "Handling fimH (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimH.fasta\n",
      "Handling fimI (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimI.fasta\n",
      "Handling fimZ (1000 records) --> /mnt/e/ecoli_db/marker_seeds/fimZ.fasta\n",
      "Handling fliC (220 records) --> /mnt/e/ecoli_db/marker_seeds/fliC.fasta\n",
      "Handling flk (1000 records) --> /mnt/e/ecoli_db/marker_seeds/flk.fasta\n",
      "Handling fnlA (107 records) --> /mnt/e/ecoli_db/marker_seeds/fnlA.fasta\n",
      "Handling galE (413 records) --> /mnt/e/ecoli_db/marker_seeds/galE.fasta\n",
      "Handling galF (32 records) --> /mnt/e/ecoli_db/marker_seeds/galF.fasta\n",
      "Handling gcvR (1 records) --> /mnt/e/ecoli_db/marker_seeds/gcvR.fasta\n",
      "Handling glf (232 records) --> /mnt/e/ecoli_db/marker_seeds/glf.fasta\n",
      "Handling gltP (1 records) --> /mnt/e/ecoli_db/marker_seeds/gltP.fasta\n",
      "Handling gmd (473 records) --> /mnt/e/ecoli_db/marker_seeds/gmd.fasta\n",
      "Handling gmm (52 records) --> /mnt/e/ecoli_db/marker_seeds/gmm.fasta\n",
      "Handling gnd (2 records) --> /mnt/e/ecoli_db/marker_seeds/gnd.fasta\n",
      "Handling gndA (1000 records) --> /mnt/e/ecoli_db/marker_seeds/gndA.fasta\n",
      "Handling hisH (28 records) --> /mnt/e/ecoli_db/marker_seeds/hisH.fasta\n",
      "Handling insH7 (1 records) --> /mnt/e/ecoli_db/marker_seeds/insH7.fasta\n",
      "Handling ltrA (1 records) --> /mnt/e/ecoli_db/marker_seeds/ltrA.fasta\n",
      "Handling mdtP (1 records) --> /mnt/e/ecoli_db/marker_seeds/mdtP.fasta\n",
      "Handling neuB (66 records) --> /mnt/e/ecoli_db/marker_seeds/neuB.fasta\n",
      "Handling neuC (65 records) --> /mnt/e/ecoli_db/marker_seeds/neuC.fasta\n",
      "Handling neuD (1 records) --> /mnt/e/ecoli_db/marker_seeds/neuD.fasta\n",
      "Handling nrfA (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfA.fasta\n",
      "Handling nrfB (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfB.fasta\n",
      "Handling nrfC (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfC.fasta\n",
      "Handling nrfD (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfD.fasta\n",
      "Handling nrfE (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfE.fasta\n",
      "Handling nrfF (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfF.fasta\n",
      "Handling nrfG (1 records) --> /mnt/e/ecoli_db/marker_seeds/nrfG.fasta\n",
      "Handling perB (227 records) --> /mnt/e/ecoli_db/marker_seeds/perB.fasta\n",
      "Handling pseB (8 records) --> /mnt/e/ecoli_db/marker_seeds/pseB.fasta\n",
      "Handling pseC (8 records) --> /mnt/e/ecoli_db/marker_seeds/pseC.fasta\n",
      "Handling pseF (8 records) --> /mnt/e/ecoli_db/marker_seeds/pseF.fasta\n",
      "Handling pseG (7 records) --> /mnt/e/ecoli_db/marker_seeds/pseG.fasta\n",
      "Handling pseH (1 records) --> /mnt/e/ecoli_db/marker_seeds/pseH.fasta\n",
      "Handling pseI (8 records) --> /mnt/e/ecoli_db/marker_seeds/pseI.fasta\n",
      "Handling purC (1 records) --> /mnt/e/ecoli_db/marker_seeds/purC.fasta\n",
      "Handling rfbA (883 records) --> /mnt/e/ecoli_db/marker_seeds/rfbA.fasta\n",
      "Handling rfbB (887 records) --> /mnt/e/ecoli_db/marker_seeds/rfbB.fasta\n",
      "Handling rfbC (803 records) --> /mnt/e/ecoli_db/marker_seeds/rfbC.fasta\n",
      "Handling rfbD (713 records) --> /mnt/e/ecoli_db/marker_seeds/rfbD.fasta\n",
      "Handling rfbE (225 records) --> /mnt/e/ecoli_db/marker_seeds/rfbE.fasta\n",
      "Handling rfbG (12 records) --> /mnt/e/ecoli_db/marker_seeds/rfbG.fasta\n",
      "Handling rfbK (19 records) --> /mnt/e/ecoli_db/marker_seeds/rfbK.fasta\n",
      "Handling rfbX (1 records) --> /mnt/e/ecoli_db/marker_seeds/rfbX.fasta\n",
      "Handling rffA (8 records) --> /mnt/e/ecoli_db/marker_seeds/rffA.fasta\n",
      "Handling stxA1 (167 records) --> /mnt/e/ecoli_db/marker_seeds/stxA1.fasta\n",
      "Handling stxA2 (50 records) --> /mnt/e/ecoli_db/marker_seeds/stxA2.fasta\n",
      "Handling stxB1 (3 records) --> /mnt/e/ecoli_db/marker_seeds/stxB1.fasta\n",
      "Handling stxB2 (19 records) --> /mnt/e/ecoli_db/marker_seeds/stxB2.fasta\n",
      "Handling sugE (1 records) --> /mnt/e/ecoli_db/marker_seeds/sugE.fasta\n",
      "Handling tmcA (1 records) --> /mnt/e/ecoli_db/marker_seeds/tmcA.fasta\n",
      "Handling trpAgpC.1 (1000 records) --> /mnt/e/ecoli_db/marker_seeds/trpAgpC.1.fasta\n",
      "Handling tspE4.C4 (1000 records) --> /mnt/e/ecoli_db/marker_seeds/tspE4.C4.fasta\n",
      "Handling tviB (12 records) --> /mnt/e/ecoli_db/marker_seeds/tviB.fasta\n",
      "Handling tviC (9 records) --> /mnt/e/ecoli_db/marker_seeds/tviC.fasta\n",
      "Handling vioA (49 records) --> /mnt/e/ecoli_db/marker_seeds/vioA.fasta\n",
      "Handling vioB (35 records) --> /mnt/e/ecoli_db/marker_seeds/vioB.fasta\n",
      "Handling wbaP (27 records) --> /mnt/e/ecoli_db/marker_seeds/wbaP.fasta\n",
      "Handling wbbD (34 records) --> /mnt/e/ecoli_db/marker_seeds/wbbD.fasta\n",
      "Handling wbbH (1 records) --> /mnt/e/ecoli_db/marker_seeds/wbbH.fasta\n",
      "Handling wbbI (191 records) --> /mnt/e/ecoli_db/marker_seeds/wbbI.fasta\n",
      "Handling wbbJ (193 records) --> /mnt/e/ecoli_db/marker_seeds/wbbJ.fasta\n",
      "Handling wbbK (193 records) --> /mnt/e/ecoli_db/marker_seeds/wbbK.fasta\n",
      "Handling wbbL (23 records) --> /mnt/e/ecoli_db/marker_seeds/wbbL.fasta\n",
      "Handling wbdI (19 records) --> /mnt/e/ecoli_db/marker_seeds/wbdI.fasta\n",
      "Handling wcaA (55 records) --> /mnt/e/ecoli_db/marker_seeds/wcaA.fasta\n",
      "Handling wcaB (56 records) --> /mnt/e/ecoli_db/marker_seeds/wcaB.fasta\n",
      "Handling wcaC (54 records) --> /mnt/e/ecoli_db/marker_seeds/wcaC.fasta\n",
      "Handling wcaD (51 records) --> /mnt/e/ecoli_db/marker_seeds/wcaD.fasta\n",
      "Handling wcaE (51 records) --> /mnt/e/ecoli_db/marker_seeds/wcaE.fasta\n",
      "Handling wcaF (45 records) --> /mnt/e/ecoli_db/marker_seeds/wcaF.fasta\n",
      "Handling wcaG (1 records) --> /mnt/e/ecoli_db/marker_seeds/wcaG.fasta\n",
      "Handling wcaI (39 records) --> /mnt/e/ecoli_db/marker_seeds/wcaI.fasta\n",
      "Handling wcaJ (87 records) --> /mnt/e/ecoli_db/marker_seeds/wcaJ.fasta\n",
      "Handling wcaK (36 records) --> /mnt/e/ecoli_db/marker_seeds/wcaK.fasta\n",
      "Handling wcaL (38 records) --> /mnt/e/ecoli_db/marker_seeds/wcaL.fasta\n",
      "Handling wcaM (36 records) --> /mnt/e/ecoli_db/marker_seeds/wcaM.fasta\n",
      "Handling wecB (194 records) --> /mnt/e/ecoli_db/marker_seeds/wecB.fasta\n",
      "Handling wecC (1 records) --> /mnt/e/ecoli_db/marker_seeds/wecC.fasta\n",
      "Handling wfeD (2 records) --> /mnt/e/ecoli_db/marker_seeds/wfeD.fasta\n",
      "Handling wza (57 records) --> /mnt/e/ecoli_db/marker_seeds/wza.fasta\n",
      "Handling wzb (58 records) --> /mnt/e/ecoli_db/marker_seeds/wzb.fasta\n",
      "Handling wzc (55 records) --> /mnt/e/ecoli_db/marker_seeds/wzc.fasta\n",
      "Handling wzm (113 records) --> /mnt/e/ecoli_db/marker_seeds/wzm.fasta\n",
      "Handling wzt (36 records) --> /mnt/e/ecoli_db/marker_seeds/wzt.fasta\n",
      "Handling wzx (1000 records) --> /mnt/e/ecoli_db/marker_seeds/wzx.fasta\n",
      "Handling wzxC (33 records) --> /mnt/e/ecoli_db/marker_seeds/wzxC.fasta\n",
      "Handling wzy (1000 records) --> /mnt/e/ecoli_db/marker_seeds/wzy.fasta\n",
      "Handling yfgO (1 records) --> /mnt/e/ecoli_db/marker_seeds/yfgO.fasta\n",
      "Handling yjaA (787 records) --> /mnt/e/ecoli_db/marker_seeds/yjaA.fasta\n",
      "Handling ypfH (1 records) --> /mnt/e/ecoli_db/marker_seeds/ypfH.fasta\n",
      "Handling ypfJ (1 records) --> /mnt/e/ecoli_db/marker_seeds/ypfJ.fasta\n",
      "Handling ypfN (1 records) --> /mnt/e/ecoli_db/marker_seeds/ypfN.fasta\n",
      "Writing TSV file: /mnt/e/ecoli_db/marker_seeds/manual_seeds.tsv\n",
      "Cleaning up.\n"
     ]
    }
   ],
   "source": [
    "print(\"Concatenating.\")\n",
    "concat_gene_df = pd.concat(gene_dfs, ignore_index=True)\n",
    "print(\"# CUSTOM GENES TOTAL = {}\".format( len(pd.unique(concat_gene_df['Gene'])) ))\n",
    "\n",
    "\n",
    "# ======= Write FASTA files.\n",
    "print(\"Generating FASTA files.\")\n",
    "for gene, section in concat_gene_df.groupby(\"Gene\"):\n",
    "    if gene.endswith(\"*\"):\n",
    "        prefix = gene[:-1]\n",
    "        gene_name_for_file = f\"{prefix}_any\"\n",
    "    else:\n",
    "        gene_name_for_file = gene\n",
    "\n",
    "    target_fasta = MARKER_SEED_DIR / f'{gene_name_for_file}.fasta'\n",
    "    \n",
    "    print(\"Handling {} ({} records) --> {}\".format(gene, section.shape[0], target_fasta))\n",
    "    with open(target_fasta, 'wt') as f:\n",
    "        for row_idx, row in section.iterrows():\n",
    "            record = SeqRecord(\n",
    "                Seq(row['GeneSeq']),\n",
    "                id=\"{}_{}\".format(gene, row_idx),\n",
    "                description=\"Src={}:{}\".format(row['Accession'], row['GeneFull'])\n",
    "            )\n",
    "            SeqIO.write([record], f, 'fasta')\n",
    "\n",
    "            \n",
    "# ======= Human-readable metadata (to be written to TSV)\n",
    "def assign_misc_metadata(gene_name):\n",
    "    if gene_name.startswith(\"fim\"):\n",
    "        return \"Fimbrial_gene\".format(gene_name)\n",
    "    elif gene_name.startswith(\"stx\"):\n",
    "        return \"ShigaToxin\".format(gene_name)\n",
    "    elif gene_name == \"fliC\" or gene_name.startswith(\"flk\") or gene_name.startswith(\"fll\") or gene_name.startswith(\"flm\"):\n",
    "        return \"H-antigen\"\n",
    "    elif gene_name in set(pd.unique(serotype_O_df['Gene'])):\n",
    "        return \"O-antigen\"\n",
    "    else:\n",
    "        return \"Misc gene\"\n",
    "\n",
    "\n",
    "# ======= write index TSV file.\n",
    "tsv_path = MARKER_SEED_DIR / \"manual_seeds.tsv\"\n",
    "print(f\"Writing TSV file: {tsv_path}\")\n",
    "\n",
    "with open(tsv_path, 'wt') as metadata_f:\n",
    "    for gene, section in concat_gene_df.groupby(\"Gene\"):\n",
    "        if gene.endswith(\"*\"):\n",
    "            prefix = gene[:-1]\n",
    "            gene_nameAC_for_file = f\"{prefix}_any\"\n",
    "        else:\n",
    "            gene_name_for_file = gene\n",
    "\n",
    "        target_fasta = MARKER_SEED_DIR / f'{gene_name_for_file}.fasta'\n",
    "        print(\n",
    "            \"{}\\t{}\\t{}\".format(\n",
    "                gene_name_for_file, target_fasta, assign_misc_metadata(gene)\n",
    "            ), \n",
    "            file=metadata_f\n",
    "        )\n",
    "\n",
    "print(\"Cleaning up.\")\n",
    "del concat_gene_df\n",
    "del misc_df\n",
    "del serotype_O_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8324e3ed-8720-48ec-8147-2a9717bbb6a6",
   "metadata": {},
   "source": [
    "#### Step 3.4 add MetaPhlAn markers."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "id": "4028dcb1-a44e-4e2d-9fbe-4fa23efc97b3",
   "metadata": {},
   "outputs": [],
   "source": [
    "# !python python_helpers/extract_metaphlan_markers.py \\\n",
    "#     -t \"$METAPHLAN_TAXONOMIC_KEY\" \\\n",
    "#     -i \"$METAPHLAN_DB_PATH\" \\\n",
    "#     -o \"$MARKER_SEED_DIR\"/metaphlan_seeds.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7ddcd1cf-bf5c-4e8b-bda4-5b9104ba2567",
   "metadata": {},
   "source": [
    "#### Step 3.5: process and combine marker TSV files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "72b596dd-d260-480b-a453-eed574d8f6b8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Created Marker seed index: /mnt/e/ecoli_db/marker_seeds/marker_seed_index.tsv\n"
     ]
    }
   ],
   "source": [
    "!cat \"$MARKER_SEED_DIR\"/mlst_seeds.tsv > $MARKER_SEED_INDEX\n",
    "!cat \"$MARKER_SEED_DIR\"/manual_seeds.tsv >> $MARKER_SEED_INDEX\n",
    "# !cat \"$MARKER_SEED_DIR\"/metaphlan_seeds.tsv >> $MARKER_SEED_INDEX\n",
    "\n",
    "print(\"Created Marker seed index: {}\".format(MARKER_SEED_INDEX))\n",
    "assert MARKER_SEED_INDEX.exists()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "836176a5-e62d-4e74-bf2c-06b4a6ccd69c",
   "metadata": {},
   "source": [
    "### Step 4: Run Chronostrain's make-db command.\n",
    "\n",
    "By the end of the previous step, we have:\n",
    "\n",
    "1) FASTA files for each gene, listing out seed sequence(s).\n",
    "2) A TSV file (marker_seed_index.tsv) containing a list of gene names and the paths to each of these FASTA files.\n",
    "\n",
    "Using these as inputs, we now construct the database files:\n",
    "1) A JSON file of the strain records and their markers.\n",
    "2) A TXT file of strain records clustered by similarity."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "5423d9a7-dc73-4112-bda9-63538c7b09da",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-04-17 14:12:49,799 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2024-04-17 14:12:51,379 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2024-04-17 14:12:51,523 [INFO - chronostrain.cli.make_db_json] - Using marker index: /mnt/e/ecoli_db/marker_seeds/marker_seed_index.tsv\n",
      "2024-04-17 14:12:51,549 [INFO - chronostrain.cli.make_db_json] - Adding strain index catalog: /mnt/e/ecoli_db/ref_genomes/index.tsv [5406 entries; 96 species]\n",
      "2024-04-17 14:12:51,549 [INFO - chronostrain.cli.make_db_json] - Building raw DB using BLAST.\n",
      "2024-04-17 14:12:51,693 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on adk.\n",
      "2024-04-17 14:12:51,693 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fumC.\n",
      "2024-04-17 14:12:51,693 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gyrB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on icd.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on mdh.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on purA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on recA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dinB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on icdA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pabB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on polB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on putP.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on trpA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on trpB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on uidA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ArpAgpE.f.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on acs.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on arnB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on arpA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on asnB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on bamC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on bcp.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on bepA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on chuA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsG.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dapA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dapE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fcf1.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fcl.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fdhF.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fim41a.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimD.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimF.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimG.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimH.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimI.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fimZ.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fliC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on flk.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fnlA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on galE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on galF.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gcvR.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on glf.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gltP.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gmd.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gmm.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gnd.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gndA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on hisH.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on insH7.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ltrA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on mdtP.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on neuB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on neuC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on neuD.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfD.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfF.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nrfG.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on perB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseF.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseG.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseH.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pseI.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on purC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbD.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbG.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbK.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rfbX.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rffA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on stxA1.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on stxA2.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on stxB1.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on stxB2.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on sugE.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on tmcA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on trpAgpC.1.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on tspE4.C4.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on tviB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on tviC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on vioA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on vioB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbaP.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbD.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbH.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbI.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbJ.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbK.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbbL.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wbdI.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaA.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaB.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaC.\n",
      "2024-04-17 14:12:51,694 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaD.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaE.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaF.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaG.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaI.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaJ.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaK.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaL.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wcaM.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wecB.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wecC.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wfeD.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wza.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzb.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzc.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzm.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzt.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzx.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzxC.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on wzy.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yfgO.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yjaA.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ypfH.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ypfJ.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ypfN.\n",
      "2024-04-17 14:12:51,695 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `adk` (1 of 133)\n",
      "2024-04-17 14:12:53,450 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fumC` (2 of 133)\n",
      "2024-04-17 14:12:53,540 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gyrB` (3 of 133)\n",
      "2024-04-17 14:12:53,691 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `icd` (4 of 133)\n",
      "2024-04-17 14:12:53,896 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `mdh` (5 of 133)\n",
      "2024-04-17 14:12:54,074 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `purA` (6 of 133)\n",
      "2024-04-17 14:12:54,231 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `recA` (7 of 133)\n",
      "2024-04-17 14:12:54,430 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dinB` (8 of 133)\n",
      "2024-04-17 14:12:54,530 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `icdA` (9 of 133)\n",
      "2024-04-17 14:12:54,764 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pabB` (10 of 133)\n",
      "2024-04-17 14:12:54,870 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `polB` (11 of 133)\n",
      "2024-04-17 14:12:54,993 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `putP` (12 of 133)\n",
      "2024-04-17 14:12:55,127 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `trpA` (13 of 133)\n",
      "2024-04-17 14:12:55,231 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `trpB` (14 of 133)\n",
      "2024-04-17 14:12:55,349 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `uidA` (15 of 133)\n",
      "2024-04-17 14:12:55,401 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ArpAgpE.f` (16 of 133)\n",
      "2024-04-17 14:20:02,245 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `acs` (17 of 133)\n",
      "2024-04-17 14:20:02,369 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `arnB` (18 of 133)\n",
      "2024-04-17 14:20:02,379 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `arpA` (19 of 133)\n",
      "2024-04-17 14:35:34,826 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `asnB` (20 of 133)\n",
      "2024-04-17 14:35:34,831 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `bamC` (21 of 133)\n",
      "2024-04-17 14:35:34,895 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `bcp` (22 of 133)\n",
      "2024-04-17 14:35:35,002 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `bepA` (23 of 133)\n",
      "2024-04-17 14:35:35,059 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `chuA` (24 of 133)\n",
      "2024-04-17 14:41:12,685 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsB` (25 of 133)\n",
      "2024-04-17 14:41:20,816 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsG` (26 of 133)\n",
      "2024-04-17 14:44:21,650 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dapA` (27 of 133)\n",
      "2024-04-17 14:44:21,937 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dapE` (28 of 133)\n",
      "2024-04-17 14:44:22,163 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fcf1` (29 of 133)\n",
      "2024-04-17 14:44:22,165 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fcl` (30 of 133)\n",
      "2024-04-17 14:44:49,009 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fdhF` (31 of 133)\n",
      "2024-04-17 14:44:49,612 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fim41a` (32 of 133)\n",
      "2024-04-17 14:44:49,619 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimA` (33 of 133)\n",
      "2024-04-17 14:51:07,922 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimB` (34 of 133)\n",
      "2024-04-17 15:07:52,233 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimC` (35 of 133)\n",
      "2024-04-17 15:14:38,162 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimD` (36 of 133)\n",
      "2024-04-17 15:33:55,647 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimE` (37 of 133)\n",
      "2024-04-17 15:44:19,621 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimF` (38 of 133)\n",
      "2024-04-17 15:58:46,693 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimG` (39 of 133)\n",
      "2024-04-17 16:14:35,518 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimH` (40 of 133)\n",
      "2024-04-17 16:19:36,677 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimI` (41 of 133)\n",
      "2024-04-17 16:30:51,859 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fimZ` (42 of 133)\n",
      "2024-04-17 16:42:18,473 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fliC` (43 of 133)\n",
      "2024-04-17 16:43:12,277 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `flk` (44 of 133)\n",
      "2024-04-17 16:50:47,868 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fnlA` (45 of 133)\n",
      "2024-04-17 16:50:47,947 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `galE` (46 of 133)\n",
      "2024-04-17 16:50:48,499 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `galF` (47 of 133)\n",
      "2024-04-17 16:50:49,491 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gcvR` (48 of 133)\n",
      "2024-04-17 16:50:49,659 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `glf` (49 of 133)\n",
      "2024-04-17 16:50:51,431 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gltP` (50 of 133)\n",
      "2024-04-17 16:50:51,656 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gmd` (51 of 133)\n",
      "2024-04-17 16:51:07,361 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gmm` (52 of 133)\n",
      "2024-04-17 16:51:08,913 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gnd` (53 of 133)\n",
      "2024-04-17 16:51:09,158 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gndA` (54 of 133)\n",
      "2024-04-17 17:25:45,323 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `hisH` (55 of 133)\n",
      "2024-04-17 17:25:45,338 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `insH7` (56 of 133)\n",
      "2024-04-17 17:25:45,516 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ltrA` (57 of 133)\n",
      "2024-04-17 17:25:45,523 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `mdtP` (58 of 133)\n",
      "2024-04-17 17:25:45,567 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `neuB` (59 of 133)\n",
      "2024-04-17 17:25:45,583 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `neuC` (60 of 133)\n",
      "2024-04-17 17:25:45,597 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `neuD` (61 of 133)\n",
      "2024-04-17 17:25:45,603 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfA` (62 of 133)\n",
      "2024-04-17 17:25:45,680 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfB` (63 of 133)\n",
      "2024-04-17 17:25:45,769 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfC` (64 of 133)\n",
      "2024-04-17 17:25:45,844 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfD` (65 of 133)\n",
      "2024-04-17 17:25:45,928 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfE` (66 of 133)\n",
      "2024-04-17 17:25:46,011 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfF` (67 of 133)\n",
      "2024-04-17 17:25:46,079 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nrfG` (68 of 133)\n",
      "2024-04-17 17:25:46,134 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `perB` (69 of 133)\n",
      "2024-04-17 17:25:48,824 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseB` (70 of 133)\n",
      "2024-04-17 17:25:48,832 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseC` (71 of 133)\n",
      "2024-04-17 17:25:48,833 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseF` (72 of 133)\n",
      "2024-04-17 17:25:48,834 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseG` (73 of 133)\n",
      "2024-04-17 17:25:48,834 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseH` (74 of 133)\n",
      "2024-04-17 17:25:48,835 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pseI` (75 of 133)\n",
      "2024-04-17 17:25:48,835 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `purC` (76 of 133)\n",
      "2024-04-17 17:25:48,959 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbA` (77 of 133)\n",
      "2024-04-17 17:26:45,216 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbB` (78 of 133)\n",
      "2024-04-17 17:30:13,421 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbC` (79 of 133)\n",
      "2024-04-17 17:30:19,171 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbD` (80 of 133)\n",
      "2024-04-17 17:33:16,689 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbE` (81 of 133)\n",
      "2024-04-17 17:33:24,426 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbG` (82 of 133)\n",
      "2024-04-17 17:33:24,441 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbK` (83 of 133)\n",
      "2024-04-17 17:33:24,451 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rfbX` (84 of 133)\n",
      "2024-04-17 17:33:24,669 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rffA` (85 of 133)\n",
      "2024-04-17 17:33:24,676 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `stxA1` (86 of 133)\n",
      "2024-04-17 17:33:28,603 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `stxA2` (87 of 133)\n",
      "2024-04-17 17:33:29,376 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `stxB1` (88 of 133)\n",
      "2024-04-17 17:33:29,401 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `stxB2` (89 of 133)\n",
      "2024-04-17 17:33:29,601 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `sugE` (90 of 133)\n",
      "2024-04-17 17:33:29,620 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `tmcA` (91 of 133)\n",
      "2024-04-17 17:33:29,948 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `trpAgpC.1` (92 of 133)\n",
      "2024-04-17 17:33:29,963 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `tspE4.C4` (93 of 133)\n",
      "2024-04-17 17:33:30,247 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `tviB` (94 of 133)\n",
      "2024-04-17 17:33:30,250 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `tviC` (95 of 133)\n",
      "2024-04-17 17:33:30,253 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `vioA` (96 of 133)\n",
      "2024-04-17 17:33:30,284 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `vioB` (97 of 133)\n",
      "2024-04-17 17:33:30,313 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbaP` (98 of 133)\n",
      "2024-04-17 17:33:30,697 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbD` (99 of 133)\n",
      "2024-04-17 17:33:30,730 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbH` (100 of 133)\n",
      "2024-04-17 17:33:30,743 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbI` (101 of 133)\n",
      "2024-04-17 17:33:35,772 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbJ` (102 of 133)\n",
      "2024-04-17 17:33:41,589 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbK` (103 of 133)\n",
      "2024-04-17 17:33:46,654 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbbL` (104 of 133)\n",
      "2024-04-17 17:33:47,105 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wbdI` (105 of 133)\n",
      "2024-04-17 17:33:47,118 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaA` (106 of 133)\n",
      "2024-04-17 17:33:55,766 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaB` (107 of 133)\n",
      "2024-04-17 17:34:02,248 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaC` (108 of 133)\n",
      "2024-04-17 17:34:09,406 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaD` (109 of 133)\n",
      "2024-04-17 17:34:18,775 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaE` (110 of 133)\n",
      "2024-04-17 17:34:23,496 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaF` (111 of 133)\n",
      "2024-04-17 17:34:30,332 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaG` (112 of 133)\n",
      "2024-04-17 17:34:30,353 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaI` (113 of 133)\n",
      "2024-04-17 17:34:35,624 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaJ` (114 of 133)\n",
      "2024-04-17 17:34:40,659 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaK` (115 of 133)\n",
      "2024-04-17 17:34:43,151 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaL` (116 of 133)\n",
      "2024-04-17 17:34:46,577 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wcaM` (117 of 133)\n",
      "2024-04-17 17:34:48,947 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wecB` (118 of 133)\n",
      "2024-04-17 17:34:49,542 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wecC` (119 of 133)\n",
      "2024-04-17 17:34:49,543 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wfeD` (120 of 133)\n",
      "2024-04-17 17:34:49,543 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wza` (121 of 133)\n",
      "2024-04-17 17:35:01,087 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzb` (122 of 133)\n",
      "2024-04-17 17:35:08,485 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzc` (123 of 133)\n",
      "2024-04-17 17:35:17,498 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzm` (124 of 133)\n",
      "2024-04-17 17:35:18,585 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzt` (125 of 133)\n",
      "2024-04-17 17:35:18,715 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzx` (126 of 133)\n",
      "2024-04-17 17:35:25,891 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzxC` (127 of 133)\n",
      "2024-04-17 17:35:30,124 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `wzy` (128 of 133)\n",
      "2024-04-17 17:35:38,772 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yfgO` (129 of 133)\n",
      "2024-04-17 17:35:39,204 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yjaA` (130 of 133)\n",
      "2024-04-17 17:41:58,967 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ypfH` (131 of 133)\n",
      "2024-04-17 17:41:59,404 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ypfJ` (132 of 133)\n",
      "2024-04-17 17:41:59,792 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ypfN` (133 of 133)\n",
      "2024-04-17 17:42:03,833 [INFO - chronostrain.cli.make_db_json] - Wrote raw blast DB entries to /mnt/e/ecoli_db/chronostrain_files/ecoli-1raw.json.\n",
      "2024-04-17 17:42:03,833 [INFO - chronostrain.cli.make_db_json] - Resolving overlaps.\n",
      "2024-04-17 17:42:03,833 [DEBUG - chronostrain.cli.make_db_json] - Src: /mnt/e/ecoli_db/chronostrain_files/ecoli-1raw.json, Dest: /mnt/e/ecoli_db/chronostrain_files/ecoli-2overlapmerged.json\n",
      "2024-04-17 17:42:04,882 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026762.1, seqID NZ_CP026762.1 [record index 0].\n",
      "2024-04-17 17:42:05,111 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041513.1, seqID NZ_CP041513.1 [record index 0].\n",
      "2024-04-17 17:42:05,341 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068090.1, seqID NZ_CP068090.1 [record index 0].\n",
      "2024-04-17 17:42:05,645 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP026795.1, seqID NZ_CP026795.1 [record index 0].\n",
      "2024-04-17 17:42:05,872 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP011511.1, seqID NZ_CP011511.1 [record index 0].\n",
      "2024-04-17 17:42:06,273 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026814.1, seqID NZ_CP026814.1 [record index 0].\n",
      "2024-04-17 17:42:06,659 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049285.1, seqID NZ_CP049285.1 [record index 0].\n",
      "2024-04-17 17:42:07,100 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP049281.1, seqID NZ_CP049281.1 [record index 0].\n",
      "2024-04-17 17:42:07,451 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026766.1, seqID NZ_CP026766.1 [record index 0].\n",
      "2024-04-17 17:42:07,985 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP024056.1, seqID NZ_CP024056.1 [record index 0].\n",
      "2024-04-17 17:42:08,520 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP024618.1, seqID NZ_CP024618.1 [record index 0].\n",
      "2024-04-17 17:42:09,022 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP023673.1, seqID NZ_CP023673.1 [record index 0].\n",
      "2024-04-17 17:42:09,341 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_LR536431.1, seqID NZ_LR536431.1 [record index 0].\n",
      "2024-04-17 17:42:09,825 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027597.1, seqID NZ_CP027597.1 [record index 0].\n",
      "2024-04-17 17:42:10,325 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP027591.1, seqID NZ_CP027591.1 [record index 0].\n",
      "2024-04-17 17:42:10,785 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027587.1, seqID NZ_CP027587.1 [record index 0].\n",
      "2024-04-17 17:42:11,128 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055292.1, seqID NZ_CP055292.1 [record index 0].\n",
      "2024-04-17 17:42:11,502 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081813.1, seqID NZ_CP081813.1 [record index 0].\n",
      "2024-04-17 17:42:11,847 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP055099.1, seqID NZ_CP055099.1 [record index 0].\n",
      "2024-04-17 17:42:12,194 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP055109.1, seqID NZ_CP055109.1 [record index 0].\n",
      "2024-04-17 17:42:12,597 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055015.1, seqID NZ_CP055015.1 [record index 0].\n",
      "2024-04-17 17:42:12,972 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055024.1, seqID NZ_CP055024.1 [record index 0].\n",
      "2024-04-17 17:42:13,394 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055030.1, seqID NZ_CP055030.1 [record index 0].\n",
      "2024-04-17 17:42:13,824 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058790.1, seqID NZ_CP058790.1 [record index 0].\n",
      "2024-04-17 17:42:14,247 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058828.1, seqID NZ_CP058828.1 [record index 0].\n",
      "2024-04-17 17:42:14,639 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055167.1, seqID NZ_CP055167.1 [record index 0].\n",
      "2024-04-17 17:42:15,017 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055170.1, seqID NZ_CP055170.1 [record index 0].\n",
      "2024-04-17 17:42:15,417 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055179.1, seqID NZ_CP055179.1 [record index 0].\n",
      "2024-04-17 17:42:15,796 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055181.1, seqID NZ_CP055181.1 [record index 0].\n",
      "2024-04-17 17:42:16,169 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055184.1, seqID NZ_CP055184.1 [record index 0].\n",
      "2024-04-17 17:42:16,570 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055219.1, seqID NZ_CP055219.1 [record index 0].\n",
      "2024-04-17 17:42:17,005 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055194.1, seqID NZ_CP055194.1 [record index 0].\n",
      "2024-04-17 17:42:17,387 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042980.1, seqID NZ_CP042980.1 [record index 0].\n",
      "2024-04-17 17:42:17,821 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026799.1, seqID NZ_CP026799.1 [record index 0].\n",
      "2024-04-17 17:42:18,238 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP037923.1, seqID NZ_CP037923.1 [record index 0].\n",
      "2024-04-17 17:42:18,624 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054913.1, seqID NZ_CP054913.1 [record index 0].\n",
      "2024-04-17 17:42:18,994 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR213452.1, seqID NZ_LR213452.1 [record index 0].\n",
      "2024-04-17 17:42:19,396 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR213455.1, seqID NZ_LR213455.1 [record index 0].\n",
      "2024-04-17 17:42:19,844 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012137.1, seqID NZ_CP012137.1 [record index 0].\n",
      "2024-04-17 17:42:20,243 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_008258.1, seqID NC_008258.1 [record index 0].\n",
      "2024-04-17 17:42:20,631 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR878365.1, seqID NZ_LR878365.1 [record index 0].\n",
      "2024-04-17 17:42:21,022 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026792.1, seqID NZ_CP026792.1 [record index 0].\n",
      "2024-04-17 17:42:21,394 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_004337.2, seqID NC_004337.2 [record index 0].\n",
      "2024-04-17 17:42:21,882 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_004741.1, seqID NC_004741.1 [record index 0].\n",
      "2024-04-17 17:42:22,263 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP004056.1, seqID NZ_CP004056.1 [record index 0].\n",
      "2024-04-17 17:42:22,676 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017328.1, seqID NC_017328.1 [record index 0].\n",
      "2024-04-17 17:42:23,137 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012735.1, seqID NZ_CP012735.1 [record index 0].\n",
      "2024-04-17 17:42:23,558 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_010658.1, seqID NC_010658.1 [record index 0].\n",
      "2024-04-17 17:42:23,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051430.1, seqID NZ_CP051430.1 [record index 0].\n",
      "2024-04-17 17:42:24,200 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040927.1, seqID NZ_CP040927.1 [record index 0].\n",
      "2024-04-17 17:42:24,436 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP023235.1, seqID NZ_AP023235.1 [record index 0].\n",
      "2024-04-17 17:42:24,643 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023224.1, seqID NZ_AP023224.1 [record index 0].\n",
      "2024-04-17 17:42:24,901 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP013112.1, seqID NZ_CP013112.1 [record index 0].\n",
      "2024-04-17 17:42:25,175 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP019361.1, seqID NZ_CP019361.1 [record index 0].\n",
      "2024-04-17 17:42:25,487 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019359.1, seqID NZ_CP019359.1 [record index 0].\n",
      "2024-04-17 17:42:25,716 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019355.1, seqID NZ_CP019355.1 [record index 0].\n",
      "2024-04-17 17:42:25,941 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035819.1, seqID NZ_CP035819.1 [record index 0].\n",
      "2024-04-17 17:42:26,073 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP022120.1, seqID NZ_AP022120.1 [record index 0].\n",
      "2024-04-17 17:42:26,307 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022107.1, seqID NZ_AP022107.1 [record index 0].\n",
      "2024-04-17 17:42:26,706 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022095.1, seqID NZ_AP022095.1 [record index 0].\n",
      "2024-04-17 17:42:27,032 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022036.1, seqID NZ_AP022036.1 [record index 0].\n",
      "2024-04-17 17:42:27,230 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP021914.1, seqID NZ_AP021914.1 [record index 0].\n",
      "2024-04-17 17:42:27,496 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021944.1, seqID NZ_AP021944.1 [record index 0].\n",
      "2024-04-17 17:42:27,752 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043334.1, seqID NZ_CP043334.1 [record index 0].\n",
      "2024-04-17 17:42:28,150 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP019213.3, seqID NZ_CP019213.3 [record index 0].\n",
      "2024-04-17 17:42:28,363 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043274.1, seqID NZ_CP043274.1 [record index 0].\n",
      "2024-04-17 17:42:28,553 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP029369.1, seqID NZ_CP029369.1 [record index 0].\n",
      "2024-04-17 17:42:28,750 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058656.1, seqID NZ_CP058656.1 [record index 0].\n",
      "2024-04-17 17:42:28,950 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058651.1, seqID NZ_CP058651.1 [record index 0].\n",
      "2024-04-17 17:42:29,378 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043284.1, seqID NZ_CP043284.1 [record index 0].\n",
      "2024-04-17 17:42:29,736 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027205.2, seqID NZ_CP027205.2 [record index 0].\n",
      "2024-04-17 17:42:29,928 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034966.1, seqID NZ_CP034966.1 [record index 0].\n",
      "2024-04-17 17:42:30,264 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP033401.1, seqID NZ_CP033401.1 [record index 0].\n",
      "2024-04-17 17:42:30,642 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012380.1, seqID NZ_CP012380.1 [record index 0].\n",
      "2024-04-17 17:42:31,021 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017664.1, seqID NC_017664.1 [record index 0].\n",
      "2024-04-17 17:42:31,405 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017635.1, seqID NC_017635.1 [record index 0].\n",
      "2024-04-17 17:42:31,764 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP085488.1, seqID NZ_CP085488.1 [record index 0].\n",
      "2024-04-17 17:42:32,153 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_OU342919.1, seqID NZ_OU342919.1 [record index 0].\n",
      "2024-04-17 17:42:32,352 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP031653.1, seqID NZ_CP031653.1 [record index 0].\n",
      "2024-04-17 17:42:32,758 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068035.1, seqID NZ_CP068035.1 [record index 0].\n",
      "2024-04-17 17:42:33,211 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP035841.1, seqID NZ_CP035841.1 [record index 0].\n",
      "2024-04-17 17:42:33,594 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035846.1, seqID NZ_CP035846.1 [record index 0].\n",
      "2024-04-17 17:42:33,895 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023205.1, seqID NZ_AP023205.1 [record index 0].\n",
      "2024-04-17 17:42:34,252 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023197.1, seqID NZ_AP023197.1 [record index 0].\n",
      "2024-04-17 17:42:34,608 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023190.1, seqID NZ_AP023190.1 [record index 0].\n",
      "2024-04-17 17:42:35,139 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP033605.1, seqID NZ_CP033605.1 [record index 0].\n",
      "2024-04-17 17:42:35,490 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022540.1, seqID NZ_AP022540.1 [record index 0].\n",
      "2024-04-17 17:42:35,876 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035860.1, seqID NZ_CP035860.1 [record index 0].\n",
      "2024-04-17 17:42:36,231 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP075061.1, seqID NZ_CP075061.1 [record index 0].\n",
      "2024-04-17 17:42:36,596 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053597.1, seqID NZ_CP053597.1 [record index 0].\n",
      "2024-04-17 17:42:36,948 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053594.1, seqID NZ_CP053594.1 [record index 0].\n",
      "2024-04-17 17:42:37,325 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP046716.1, seqID NZ_CP046716.1 [record index 0].\n",
      "2024-04-17 17:42:37,606 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP059283.1, seqID NZ_CP059283.1 [record index 0].\n",
      "2024-04-17 17:42:37,954 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP078137.1, seqID NZ_CP078137.1 [record index 0].\n",
      "2024-04-17 17:42:38,342 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091773.1, seqID NZ_CP091773.1 [record index 0].\n",
      "2024-04-17 17:42:38,745 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076305.1, seqID NZ_CP076305.1 [record index 0].\n",
      "2024-04-17 17:42:39,082 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022409.1, seqID NZ_AP022409.1 [record index 0].\n",
      "2024-04-17 17:42:39,282 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP054169.1, seqID NZ_CP054169.1 [record index 0].\n",
      "2024-04-17 17:42:39,476 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP022482.1, seqID NZ_AP022482.1 [record index 0].\n",
      "2024-04-17 17:42:39,844 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041411.1, seqID NZ_CP041411.1 [record index 0].\n",
      "2024-04-17 17:42:40,160 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP041416.1, seqID NZ_CP041416.1 [record index 0].\n",
      "2024-04-17 17:42:40,537 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041425.1, seqID NZ_CP041425.1 [record index 0].\n",
      "2024-04-17 17:42:40,942 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091026.1, seqID NZ_CP091026.1 [record index 0].\n",
      "2024-04-17 17:42:41,266 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP041429.1, seqID NZ_CP041429.1 [record index 0].\n",
      "2024-04-17 17:42:41,478 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091044.1, seqID NZ_CP091044.1 [record index 0].\n",
      "2024-04-17 17:42:41,815 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP041431.1, seqID NZ_CP041431.1 [record index 0].\n",
      "2024-04-17 17:42:42,143 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP041433.1, seqID NZ_CP041433.1 [record index 0].\n",
      "2024-04-17 17:42:42,547 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041435.1, seqID NZ_CP041435.1 [record index 0].\n",
      "2024-04-17 17:42:42,917 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091020.1, seqID NZ_CP091020.1 [record index 0].\n",
      "2024-04-17 17:42:43,287 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091018.1, seqID NZ_CP091018.1 [record index 0].\n",
      "2024-04-17 17:42:43,654 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091016.1, seqID NZ_CP091016.1 [record index 0].\n",
      "2024-04-17 17:42:43,869 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091022.1, seqID NZ_CP091022.1 [record index 0].\n",
      "2024-04-17 17:42:44,285 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075697.1, seqID NZ_CP075697.1 [record index 0].\n",
      "2024-04-17 17:42:44,723 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075665.1, seqID NZ_CP075665.1 [record index 0].\n",
      "2024-04-17 17:42:45,114 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075663.1, seqID NZ_CP075663.1 [record index 0].\n",
      "2024-04-17 17:42:45,529 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075652.1, seqID NZ_CP075652.1 [record index 0].\n",
      "2024-04-17 17:42:45,894 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP077646.1, seqID NZ_CP077646.1 [record index 0].\n",
      "2024-04-17 17:42:46,253 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP077649.1, seqID NZ_CP077649.1 [record index 0].\n",
      "2024-04-17 17:42:46,444 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP007391.1, seqID NZ_CP007391.1 [record index 0].\n",
      "2024-04-17 17:42:46,639 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP007390.1, seqID NZ_CP007390.1 [record index 0].\n",
      "2024-04-17 17:42:46,835 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP007265.1, seqID NZ_CP007265.1 [record index 0].\n",
      "2024-04-17 17:42:47,163 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031321.1, seqID NZ_CP031321.1 [record index 0].\n",
      "2024-04-17 17:42:47,431 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043478.1, seqID NZ_CP043478.1 [record index 0].\n",
      "2024-04-17 17:42:47,652 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061914.1, seqID NZ_CP061914.1 [record index 0].\n",
      "2024-04-17 17:42:47,890 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043486.1, seqID NZ_CP043486.1 [record index 0].\n",
      "2024-04-17 17:42:48,233 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044315.1, seqID NZ_CP044315.1 [record index 0].\n",
      "2024-04-17 17:42:48,613 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP024130.1, seqID NZ_AP024130.1 [record index 0].\n",
      "2024-04-17 17:42:48,968 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051219.1, seqID NZ_CP051219.1 [record index 0].\n",
      "2024-04-17 17:42:49,334 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP007594.1, seqID NZ_CP007594.1 [record index 0].\n",
      "2024-04-17 17:42:49,726 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_011415.1, seqID NC_011415.1 [record index 0].\n",
      "2024-04-17 17:42:50,084 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP029692.1, seqID NZ_CP029692.1 [record index 0].\n",
      "2024-04-17 17:42:50,505 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP051222.1, seqID NZ_CP051222.1 [record index 0].\n",
      "2024-04-17 17:42:50,850 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054371.1, seqID NZ_CP054371.1 [record index 0].\n",
      "2024-04-17 17:42:51,051 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP053281.1, seqID NZ_CP053281.1 [record index 0].\n",
      "2024-04-17 17:42:51,381 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051698.1, seqID NZ_CP051698.1 [record index 0].\n",
      "2024-04-17 17:42:51,767 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP086259.1, seqID NZ_CP086259.1 [record index 0].\n",
      "2024-04-17 17:42:52,104 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP034958.1, seqID NZ_CP034958.1 [record index 0].\n",
      "2024-04-17 17:42:52,497 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032892.1, seqID NZ_CP032892.1 [record index 0].\n",
      "2024-04-17 17:42:52,848 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP032426.1, seqID NZ_CP032426.1 [record index 0].\n",
      "2024-04-17 17:42:53,184 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP025048.1, seqID NZ_CP025048.1 [record index 0].\n",
      "2024-04-17 17:42:53,569 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_LR778149.1, seqID NZ_LR778149.1 [record index 0].\n",
      "2024-04-17 17:42:53,888 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR778151.1, seqID NZ_LR778151.1 [record index 0].\n",
      "2024-04-17 17:42:54,199 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR778148.1, seqID NZ_LR778148.1 [record index 0].\n",
      "2024-04-17 17:42:54,518 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR782232.1, seqID NZ_LR782232.1 [record index 0].\n",
      "2024-04-17 17:42:54,869 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778145.1, seqID NZ_LR778145.1 [record index 0].\n",
      "2024-04-17 17:42:55,240 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778143.1, seqID NZ_LR778143.1 [record index 0].\n",
      "2024-04-17 17:42:55,617 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778141.1, seqID NZ_LR778141.1 [record index 0].\n",
      "2024-04-17 17:42:55,982 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778140.1, seqID NZ_LR778140.1 [record index 0].\n",
      "2024-04-17 17:42:56,340 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP067232.1, seqID NZ_CP067232.1 [record index 0].\n",
      "2024-04-17 17:42:56,791 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP007592.1, seqID NZ_CP007592.1 [record index 0].\n",
      "2024-04-17 17:42:57,162 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010242.1, seqID NZ_CP010242.1 [record index 0].\n",
      "2024-04-17 17:42:57,559 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015995.1, seqID NZ_CP015995.1 [record index 0].\n",
      "2024-04-17 17:42:57,960 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010235.1, seqID NZ_CP010235.1 [record index 0].\n",
      "2024-04-17 17:42:58,298 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010231.1, seqID NZ_CP010231.1 [record index 0].\n",
      "2024-04-17 17:42:58,653 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010228.1, seqID NZ_CP010228.1 [record index 0].\n",
      "2024-04-17 17:42:59,013 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010230.1, seqID NZ_CP010230.1 [record index 0].\n",
      "2024-04-17 17:42:59,421 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075852.1, seqID NZ_CP075852.1 [record index 0].\n",
      "2024-04-17 17:42:59,822 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075640.1, seqID NZ_CP075640.1 [record index 0].\n",
      "2024-04-17 17:43:00,188 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058725.1, seqID NZ_CP058725.1 [record index 0].\n",
      "2024-04-17 17:43:00,641 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP058709.1, seqID NZ_CP058709.1 [record index 0].\n",
      "2024-04-17 17:43:01,015 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010229.1, seqID NZ_CP010229.1 [record index 0].\n",
      "2024-04-17 17:43:01,357 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048010.1, seqID NZ_CP048010.1 [record index 0].\n",
      "2024-04-17 17:43:01,708 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083274.1, seqID NZ_CP083274.1 [record index 0].\n",
      "2024-04-17 17:43:02,049 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP028432.1, seqID NZ_CP028432.1 [record index 0].\n",
      "2024-04-17 17:43:02,387 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP009104.1, seqID NZ_CP009104.1 [record index 0].\n",
      "2024-04-17 17:43:02,718 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP028117.1, seqID NZ_CP028117.1 [record index 0].\n",
      "2024-04-17 17:43:03,081 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044314.1, seqID NZ_CP044314.1 [record index 0].\n",
      "2024-04-17 17:43:03,423 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP028429.1, seqID NZ_CP028429.1 [record index 0].\n",
      "2024-04-17 17:43:03,801 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042298.1, seqID NZ_CP042298.1 [record index 0].\n",
      "2024-04-17 17:43:04,267 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043825.1, seqID NZ_CP043825.1 [record index 0].\n",
      "2024-04-17 17:43:04,696 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015244.1, seqID NZ_CP015244.1 [record index 0].\n",
      "2024-04-17 17:43:05,027 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027105.1, seqID NZ_CP027105.1 [record index 0].\n",
      "2024-04-17 17:43:05,405 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044311.1, seqID NZ_CP044311.1 [record index 0].\n",
      "2024-04-17 17:43:05,791 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044312.1, seqID NZ_CP044312.1 [record index 0].\n",
      "2024-04-17 17:43:06,243 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050498.1, seqID NZ_CP050498.1 [record index 0].\n",
      "2024-04-17 17:43:06,608 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044313.1, seqID NZ_CP044313.1 [record index 0].\n",
      "2024-04-17 17:43:06,971 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023165.1, seqID NZ_CP023165.1 [record index 0].\n",
      "2024-04-17 17:43:07,341 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028381.1, seqID NZ_CP028381.1 [record index 0].\n",
      "2024-04-17 17:43:07,706 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028122.1, seqID NZ_CP028122.1 [record index 0].\n",
      "2024-04-17 17:43:07,895 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP068803.1, seqID NZ_CP068803.1 [record index 0].\n",
      "2024-04-17 17:43:08,283 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062855.1, seqID NZ_CP062855.1 [record index 0].\n",
      "2024-04-17 17:43:08,653 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062863.1, seqID NZ_CP062863.1 [record index 0].\n",
      "2024-04-17 17:43:09,012 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP062871.1, seqID NZ_CP062871.1 [record index 0].\n",
      "2024-04-17 17:43:09,385 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062908.1, seqID NZ_CP062908.1 [record index 0].\n",
      "2024-04-17 17:43:09,739 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP062910.1, seqID NZ_CP062910.1 [record index 0].\n",
      "2024-04-17 17:43:09,988 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_012967.1, seqID NC_012967.1 [record index 0].\n",
      "2024-04-17 17:43:10,198 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP034106.1, seqID NZ_CP034106.1 [record index 0].\n",
      "2024-04-17 17:43:10,439 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031706.1, seqID NZ_CP031706.1 [record index 0].\n",
      "2024-04-17 17:43:10,664 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR883965.1, seqID NZ_LR883965.1 [record index 0].\n",
      "2024-04-17 17:43:10,886 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR882493.1, seqID NZ_LR882493.1 [record index 0].\n",
      "2024-04-17 17:43:11,104 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048821.1, seqID NZ_CP048821.1 [record index 0].\n",
      "2024-04-17 17:43:11,369 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP053211.1, seqID NZ_CP053211.1 [record index 0].\n",
      "2024-04-17 17:43:11,603 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041031.1, seqID NZ_CP041031.1 [record index 0].\n",
      "2024-04-17 17:43:11,865 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP080126.1, seqID NZ_CP080126.1 [record index 0].\n",
      "2024-04-17 17:43:12,087 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080172.1, seqID NZ_CP080172.1 [record index 0].\n",
      "2024-04-17 17:43:12,323 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080146.1, seqID NZ_CP080146.1 [record index 0].\n",
      "2024-04-17 17:43:12,558 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080370.1, seqID NZ_CP080370.1 [record index 0].\n",
      "2024-04-17 17:43:12,858 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072802.1, seqID NZ_CP072802.1 [record index 0].\n",
      "2024-04-17 17:43:13,209 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP039298.1, seqID NZ_CP039298.1 [record index 0].\n",
      "2024-04-17 17:43:13,563 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP082120.1, seqID NZ_CP082120.1 [record index 0].\n",
      "2024-04-17 17:43:13,889 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048604.1, seqID NZ_CP048604.1 [record index 0].\n",
      "2024-04-17 17:43:14,257 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050646.1, seqID NZ_CP050646.1 [record index 0].\n",
      "2024-04-17 17:43:14,604 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP085069.1, seqID NZ_CP085069.1 [record index 0].\n",
      "2024-04-17 17:43:14,920 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP044293.1, seqID NZ_CP044293.1 [record index 0].\n",
      "2024-04-17 17:43:15,274 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP085056.1, seqID NZ_CP085056.1 [record index 0].\n",
      "2024-04-17 17:43:15,656 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075627.1, seqID NZ_CP075627.1 [record index 0].\n",
      "2024-04-17 17:43:15,995 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP045190.1, seqID NZ_CP045190.1 [record index 0].\n",
      "2024-04-17 17:43:16,367 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024126.1, seqID NZ_AP024126.1 [record index 0].\n",
      "2024-04-17 17:43:16,722 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP075374.1, seqID NZ_CP075374.1 [record index 0].\n",
      "2024-04-17 17:43:17,047 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024561.1, seqID NZ_AP024561.1 [record index 0].\n",
      "2024-04-17 17:43:17,371 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024560.1, seqID NZ_AP024560.1 [record index 0].\n",
      "2024-04-17 17:43:17,720 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053600.1, seqID NZ_CP053600.1 [record index 0].\n",
      "2024-04-17 17:43:18,067 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053592.1, seqID NZ_CP053592.1 [record index 0].\n",
      "2024-04-17 17:43:18,414 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053599.1, seqID NZ_CP053599.1 [record index 0].\n",
      "2024-04-17 17:43:18,818 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083888.1, seqID NZ_CP083888.1 [record index 0].\n",
      "2024-04-17 17:43:19,168 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS483303.1, seqID NZ_LS483303.1 [record index 0].\n",
      "2024-04-17 17:43:19,500 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134311.1, seqID NZ_LR134311.1 [record index 0].\n",
      "2024-04-17 17:43:19,873 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134079.1, seqID NZ_LR134079.1 [record index 0].\n",
      "2024-04-17 17:43:20,193 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134226.1, seqID NZ_LR134226.1 [record index 0].\n",
      "2024-04-17 17:43:20,536 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134234.1, seqID NZ_LR134234.1 [record index 0].\n",
      "2024-04-17 17:43:20,868 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134209.1, seqID NZ_LR134209.1 [record index 0].\n",
      "2024-04-17 17:43:21,252 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134222.1, seqID NZ_LR134222.1 [record index 0].\n",
      "2024-04-17 17:43:21,604 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134248.1, seqID NZ_LR134248.1 [record index 0].\n",
      "2024-04-17 17:43:22,005 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134092.1, seqID NZ_LR134092.1 [record index 0].\n",
      "2024-04-17 17:43:22,379 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP009050.1, seqID NZ_CP009050.1 [record index 0].\n",
      "2024-04-17 17:43:22,700 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068680.1, seqID NZ_CP068680.1 [record index 0].\n",
      "2024-04-17 17:43:23,035 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP076654.1, seqID NZ_CP076654.1 [record index 0].\n",
      "2024-04-17 17:43:23,243 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR898868.1, seqID NZ_LR898868.1 [record index 0].\n",
      "2024-04-17 17:43:23,446 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR130552.1, seqID NZ_LR130552.1 [record index 0].\n",
      "2024-04-17 17:43:23,819 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890651.1, seqID NZ_LR890651.1 [record index 0].\n",
      "2024-04-17 17:43:24,182 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069682.1, seqID NZ_CP069682.1 [record index 0].\n",
      "2024-04-17 17:43:24,382 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP069692.1, seqID NZ_CP069692.1 [record index 0].\n",
      "2024-04-17 17:43:24,759 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069646.1, seqID NZ_CP069646.1 [record index 0].\n",
      "2024-04-17 17:43:25,140 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP073589.1, seqID NZ_CP073589.1 [record index 0].\n",
      "2024-04-17 17:43:25,394 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024275.1, seqID NZ_CP024275.1 [record index 0].\n",
      "2024-04-17 17:43:25,617 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019953.1, seqID NZ_CP019953.1 [record index 0].\n",
      "2024-04-17 17:43:25,839 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010191.1, seqID NZ_CP010191.1 [record index 0].\n",
      "2024-04-17 17:43:26,092 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP010183.1, seqID NZ_CP010183.1 [record index 0].\n",
      "2024-04-17 17:43:26,339 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010219.1, seqID NZ_CP010219.1 [record index 0].\n",
      "2024-04-17 17:43:26,575 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058719.1, seqID NZ_CP058719.1 [record index 0].\n",
      "2024-04-17 17:43:26,846 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP010180.1, seqID NZ_CP010180.1 [record index 0].\n",
      "2024-04-17 17:43:27,071 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_022364.1, seqID NC_022364.1 [record index 0].\n",
      "2024-04-17 17:43:27,291 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP072204.1, seqID NZ_CP072204.1 [record index 0].\n",
      "2024-04-17 17:43:27,540 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035884.1, seqID NZ_CP035884.1 [record index 0].\n",
      "2024-04-17 17:43:27,769 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047662.1, seqID NZ_CP047662.1 [record index 0].\n",
      "2024-04-17 17:43:28,001 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047665.1, seqID NZ_CP047665.1 [record index 0].\n",
      "2024-04-17 17:43:28,231 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047876.1, seqID NZ_CP047876.1 [record index 0].\n",
      "2024-04-17 17:43:28,452 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR882990.1, seqID NZ_LR882990.1 [record index 0].\n",
      "2024-04-17 17:43:28,666 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP033378.1, seqID NZ_CP033378.1 [record index 0].\n",
      "2024-04-17 17:43:28,899 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP036202.1, seqID NZ_CP036202.1 [record index 0].\n",
      "2024-04-17 17:43:29,123 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP034738.1, seqID NZ_CP034738.1 [record index 0].\n",
      "2024-04-17 17:43:29,371 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR883006.1, seqID NZ_LR883006.1 [record index 0].\n",
      "2024-04-17 17:43:29,710 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR883012.1, seqID NZ_LR883012.1 [record index 0].\n",
      "2024-04-17 17:43:30,087 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR882997.1, seqID NZ_LR882997.1 [record index 0].\n",
      "2024-04-17 17:43:30,438 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP034589.1, seqID NZ_CP034589.1 [record index 0].\n",
      "2024-04-17 17:43:30,635 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP076527.1, seqID NZ_CP076527.1 [record index 0].\n",
      "2024-04-17 17:43:31,010 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR882973.1, seqID NZ_LR882973.1 [record index 0].\n",
      "2024-04-17 17:43:31,400 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR883050.1, seqID NZ_LR883050.1 [record index 0].\n",
      "2024-04-17 17:43:31,750 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LT795502.1, seqID NZ_LT795502.1 [record index 0].\n",
      "2024-04-17 17:43:32,044 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP018840.1, seqID NZ_CP018840.1 [record index 0].\n",
      "2024-04-17 17:43:32,379 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019558.1, seqID NZ_CP019558.1 [record index 0].\n",
      "2024-04-17 17:43:32,745 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019560.1, seqID NZ_CP019560.1 [record index 0].\n",
      "2024-04-17 17:43:33,107 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024123.1, seqID NZ_AP024123.1 [record index 0].\n",
      "2024-04-17 17:43:33,496 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017660.1, seqID NC_017660.1 [record index 0].\n",
      "2024-04-17 17:43:33,843 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054407.1, seqID NZ_CP054407.1 [record index 0].\n",
      "2024-04-17 17:43:34,182 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026473.1, seqID NZ_CP026473.1 [record index 0].\n",
      "2024-04-17 17:43:34,536 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP040886.1, seqID NZ_CP040886.1 [record index 0].\n",
      "2024-04-17 17:43:34,685 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058618.1, seqID NZ_CP058618.1 [record index 0].\n",
      "2024-04-17 17:43:34,907 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024114.1, seqID NZ_AP024114.1 [record index 0].\n",
      "2024-04-17 17:43:35,123 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP049936.1, seqID NZ_CP049936.1 [record index 0].\n",
      "2024-04-17 17:43:35,384 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022815.1, seqID NZ_AP022815.1 [record index 0].\n",
      "2024-04-17 17:43:35,657 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022811.1, seqID NZ_AP022811.1 [record index 0].\n",
      "2024-04-17 17:43:35,878 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053787.1, seqID NZ_CP053787.1 [record index 0].\n",
      "2024-04-17 17:43:36,124 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047002.1, seqID NZ_CP047002.1 [record index 0].\n",
      "2024-04-17 17:43:36,390 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP092252.1, seqID NZ_CP092252.1 [record index 0].\n",
      "2024-04-17 17:43:36,619 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP045977.1, seqID NZ_CP045977.1 [record index 0].\n",
      "2024-04-17 17:43:36,895 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054662.1, seqID NZ_CP054662.1 [record index 0].\n",
      "2024-04-17 17:43:37,240 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP049979.1, seqID NZ_CP049979.1 [record index 0].\n",
      "2024-04-17 17:43:37,619 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_HF572917.1, seqID NZ_HF572917.1 [record index 0].\n",
      "2024-04-17 17:43:37,975 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP023427.1, seqID NZ_AP023427.1 [record index 0].\n",
      "2024-04-17 17:43:38,243 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029492.1, seqID NZ_CP029492.1 [record index 0].\n",
      "2024-04-17 17:43:38,616 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019961.1, seqID NZ_CP019961.1 [record index 0].\n",
      "2024-04-17 17:43:38,996 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020933.1, seqID NZ_CP020933.1 [record index 0].\n",
      "2024-04-17 17:43:39,338 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP029180.1, seqID NZ_CP029180.1 [record index 0].\n",
      "2024-04-17 17:43:39,759 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013031.1, seqID NZ_CP013031.1 [record index 0].\n",
      "2024-04-17 17:43:40,089 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010178.1, seqID NZ_CP010178.1 [record index 0].\n",
      "2024-04-17 17:43:40,445 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010177.1, seqID NZ_CP010177.1 [record index 0].\n",
      "2024-04-17 17:43:40,810 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010176.1, seqID NZ_CP010176.1 [record index 0].\n",
      "2024-04-17 17:43:41,184 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048025.1, seqID NZ_CP048025.1 [record index 0].\n",
      "2024-04-17 17:43:41,557 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012376.1, seqID NZ_CP012376.1 [record index 0].\n",
      "2024-04-17 17:43:41,944 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP074407.1, seqID NZ_CP074407.1 [record index 0].\n",
      "2024-04-17 17:43:42,326 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076646.1, seqID NZ_CP076646.1 [record index 0].\n",
      "2024-04-17 17:43:42,750 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP013663.1, seqID NZ_CP013663.1 [record index 0].\n",
      "2024-04-17 17:43:43,113 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP038138.1, seqID NZ_CP038138.1 [record index 0].\n",
      "2024-04-17 17:43:43,470 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031898.1, seqID NZ_CP031898.1 [record index 0].\n",
      "2024-04-17 17:43:43,842 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031902.1, seqID NZ_CP031902.1 [record index 0].\n",
      "2024-04-17 17:43:44,233 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031906.1, seqID NZ_CP031906.1 [record index 0].\n",
      "2024-04-17 17:43:44,526 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031908.1, seqID NZ_CP031908.1 [record index 0].\n",
      "2024-04-17 17:43:44,851 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP031912.1, seqID NZ_CP031912.1 [record index 0].\n",
      "2024-04-17 17:43:45,245 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031916.1, seqID NZ_CP031916.1 [record index 0].\n",
      "2024-04-17 17:43:45,604 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023061.1, seqID NZ_CP023061.1 [record index 0].\n",
      "2024-04-17 17:43:46,014 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP016828.1, seqID NZ_CP016828.1 [record index 0].\n",
      "2024-04-17 17:43:46,298 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP016628.1, seqID NZ_CP016628.1 [record index 0].\n",
      "2024-04-17 17:43:46,570 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013190.1, seqID NZ_CP013190.1 [record index 0].\n",
      "2024-04-17 17:43:46,821 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP013185.1, seqID NZ_CP013185.1 [record index 0].\n",
      "2024-04-17 17:43:47,205 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP017844.1, seqID NZ_CP017844.1 [record index 0].\n",
      "2024-04-17 17:43:47,625 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP045712.1, seqID NZ_CP045712.1 [record index 0].\n",
      "2024-04-17 17:43:48,030 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP046539.1, seqID NZ_CP046539.1 [record index 0].\n",
      "2024-04-17 17:43:48,283 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP085638.1, seqID NZ_CP085638.1 [record index 0].\n",
      "2024-04-17 17:43:48,510 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP065613.1, seqID NZ_CP065613.1 [record index 0].\n",
      "2024-04-17 17:43:48,765 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP065615.1, seqID NZ_CP065615.1 [record index 0].\n",
      "2024-04-17 17:43:48,950 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP065624.1, seqID NZ_CP065624.1 [record index 0].\n",
      "2024-04-17 17:43:49,175 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041002.1, seqID NZ_CP041002.1 [record index 0].\n",
      "2024-04-17 17:43:49,300 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP023899.1, seqID NZ_CP023899.1 [record index 0].\n",
      "2024-04-17 17:43:49,529 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023535.1, seqID NZ_CP023535.1 [record index 0].\n",
      "2024-04-17 17:43:49,765 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023531.1, seqID NZ_CP023531.1 [record index 0].\n",
      "2024-04-17 17:43:49,996 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027394.1, seqID NZ_CP027394.1 [record index 0].\n",
      "2024-04-17 17:43:50,236 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP022086.2, seqID NZ_CP022086.2 [record index 0].\n",
      "2024-04-17 17:43:50,475 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069571.1, seqID NZ_CP069571.1 [record index 0].\n",
      "2024-04-17 17:43:50,868 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068155.1, seqID NZ_CP068155.1 [record index 0].\n",
      "2024-04-17 17:43:51,104 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP066032.1, seqID NZ_CP066032.1 [record index 0].\n",
      "2024-04-17 17:43:51,375 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP065970.1, seqID NZ_CP065970.1 [record index 0].\n",
      "2024-04-17 17:43:51,586 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024273.1, seqID NZ_CP024273.1 [record index 0].\n",
      "2024-04-17 17:43:51,793 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR536430.1, seqID NZ_LR536430.1 [record index 0].\n",
      "2024-04-17 17:43:52,065 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050998.1, seqID NZ_CP050998.1 [record index 0].\n",
      "2024-04-17 17:43:52,286 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024269.1, seqID NZ_CP024269.1 [record index 0].\n",
      "2024-04-17 17:43:52,621 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024263.1, seqID NZ_CP024263.1 [record index 0].\n",
      "2024-04-17 17:43:52,986 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024260.1, seqID NZ_CP024260.1 [record index 0].\n",
      "2024-04-17 17:43:53,360 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024257.1, seqID NZ_CP024257.1 [record index 0].\n",
      "2024-04-17 17:43:53,683 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023237.1, seqID NZ_AP023237.1 [record index 0].\n",
      "2024-04-17 17:43:54,047 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_017633.1, seqID NC_017633.1 [record index 0].\n",
      "2024-04-17 17:43:54,431 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041627.1, seqID NZ_CP041627.1 [record index 0].\n",
      "2024-04-17 17:43:54,838 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023346.1, seqID NZ_CP023346.1 [record index 0].\n",
      "2024-04-17 17:43:54,993 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP031231.1, seqID NZ_CP031231.1 [record index 0].\n",
      "2024-04-17 17:43:55,206 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041678.1, seqID NZ_CP041678.1 [record index 0].\n",
      "2024-04-17 17:43:55,443 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063459.1, seqID NZ_CP063459.1 [record index 0].\n",
      "2024-04-17 17:43:55,694 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063479.1, seqID NZ_CP063479.1 [record index 0].\n",
      "2024-04-17 17:43:55,818 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083497.1, seqID NZ_CP083497.1 [record index 0].\n",
      "2024-04-17 17:43:55,949 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083478.1, seqID NZ_CP083478.1 [record index 0].\n",
      "2024-04-17 17:43:56,178 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP045213.1, seqID NZ_CP045213.1 [record index 0].\n",
      "2024-04-17 17:43:56,302 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP068591.1, seqID NZ_CP068591.1 [record index 0].\n",
      "2024-04-17 17:43:56,515 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063487.1, seqID NZ_CP063487.1 [record index 0].\n",
      "2024-04-17 17:43:56,759 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075062.1, seqID NZ_CP075062.1 [record index 0].\n",
      "2024-04-17 17:43:57,252 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069706.1, seqID NZ_CP069706.1 [record index 0].\n",
      "2024-04-17 17:43:57,502 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071441.1, seqID NZ_CP071441.1 [record index 0].\n",
      "2024-04-17 17:43:57,999 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026202.1, seqID NZ_CP026202.1 [record index 0].\n",
      "2024-04-17 17:43:58,559 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026399.1, seqID NZ_CP026399.1 [record index 0].\n",
      "2024-04-17 17:43:58,810 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP018995.1, seqID NZ_CP018995.1 [record index 0].\n",
      "2024-04-17 17:43:59,245 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP018965.1, seqID NZ_CP018965.1 [record index 0].\n",
      "2024-04-17 17:43:59,688 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR130532.1, seqID NZ_LR130532.1 [record index 0].\n",
      "2024-04-17 17:44:00,126 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR595691.1, seqID NZ_LR595691.1 [record index 0].\n",
      "2024-04-17 17:44:00,666 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP067342.1, seqID NZ_CP067342.1 [record index 0].\n",
      "2024-04-17 17:44:01,150 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032237.1, seqID NZ_CP032237.1 [record index 0].\n",
      "2024-04-17 17:44:01,644 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029242.1, seqID NZ_CP029242.1 [record index 0].\n",
      "2024-04-17 17:44:02,072 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010344.1, seqID NZ_CP010344.1 [record index 0].\n",
      "2024-04-17 17:44:02,496 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060748.1, seqID NZ_CP060748.1 [record index 0].\n",
      "2024-04-17 17:44:02,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP049118.1, seqID NZ_CP049118.1 [record index 0].\n",
      "2024-04-17 17:44:03,372 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061329.1, seqID NZ_CP061329.1 [record index 0].\n",
      "2024-04-17 17:44:03,918 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060950.1, seqID NZ_CP060950.1 [record index 0].\n",
      "2024-04-17 17:44:04,357 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP085622.1, seqID NZ_CP085622.1 [record index 0].\n",
      "2024-04-17 17:44:04,897 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP092647.1, seqID NZ_CP092647.1 [record index 0].\n",
      "2024-04-17 17:44:05,348 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP059125.1, seqID NZ_CP059125.1 [record index 0].\n",
      "2024-04-17 17:44:05,824 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP024582.1, seqID NZ_AP024582.1 [record index 0].\n",
      "2024-04-17 17:44:06,289 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060873.1, seqID NZ_CP060873.1 [record index 0].\n",
      "2024-04-17 17:44:06,719 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035123.1, seqID NZ_CP035123.1 [record index 0].\n",
      "2024-04-17 17:44:07,201 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060885.1, seqID NZ_CP060885.1 [record index 0].\n",
      "2024-04-17 17:44:07,642 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP072976.1, seqID NZ_CP072976.1 [record index 0].\n",
      "2024-04-17 17:44:08,117 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060902.1, seqID NZ_CP060902.1 [record index 0].\n",
      "2024-04-17 17:44:08,550 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP085188.1, seqID NZ_CP085188.1 [record index 0].\n",
      "2024-04-17 17:44:09,013 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060908.1, seqID NZ_CP060908.1 [record index 0].\n",
      "2024-04-17 17:44:09,270 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP081192.1, seqID NZ_CP081192.1 [record index 0].\n",
      "2024-04-17 17:44:09,696 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP081306.1, seqID NZ_CP081306.1 [record index 0].\n",
      "2024-04-17 17:44:10,214 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060946.1, seqID NZ_CP060946.1 [record index 0].\n",
      "2024-04-17 17:44:10,618 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LS998785.1, seqID NZ_LS998785.1 [record index 0].\n",
      "2024-04-17 17:44:11,054 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP072322.1, seqID NZ_CP072322.1 [record index 0].\n",
      "2024-04-17 17:44:11,495 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR025096.1, seqID NZ_LR025096.1 [record index 0].\n",
      "2024-04-17 17:44:11,940 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035817.1, seqID NZ_CP035817.1 [record index 0].\n",
      "2024-04-17 17:44:12,455 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP029687.1, seqID NZ_CP029687.1 [record index 0].\n",
      "2024-04-17 17:44:12,937 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022393.1, seqID NZ_CP022393.1 [record index 0].\n",
      "2024-04-17 17:44:13,391 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP030281.1, seqID NZ_CP030281.1 [record index 0].\n",
      "2024-04-17 17:44:13,842 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_AP018802.1, seqID NZ_AP018802.1 [record index 0].\n",
      "2024-04-17 17:44:14,310 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_AP018796.1, seqID NZ_AP018796.1 [record index 0].\n",
      "2024-04-17 17:44:14,794 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035870.1, seqID NZ_CP035870.1 [record index 0].\n",
      "2024-04-17 17:44:15,294 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_009801.1, seqID NC_009801.1 [record index 0].\n",
      "2024-04-17 17:44:15,702 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP086662.1, seqID NZ_CP086662.1 [record index 0].\n",
      "2024-04-17 17:44:16,128 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048915.1, seqID NZ_CP048915.1 [record index 0].\n",
      "2024-04-17 17:44:16,574 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024521.1, seqID NZ_AP024521.1 [record index 0].\n",
      "2024-04-17 17:44:17,016 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP024518.1, seqID NZ_AP024518.1 [record index 0].\n",
      "2024-04-17 17:44:17,417 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP035751.1, seqID NZ_CP035751.1 [record index 0].\n",
      "2024-04-17 17:44:17,856 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LT594504.1, seqID NZ_LT594504.1 [record index 0].\n",
      "2024-04-17 17:44:18,359 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035879.1, seqID NZ_CP035879.1 [record index 0].\n",
      "2024-04-17 17:44:18,802 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP079747.1, seqID NZ_CP079747.1 [record index 0].\n",
      "2024-04-17 17:44:19,280 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026085.1, seqID NZ_CP026085.1 [record index 0].\n",
      "2024-04-17 17:44:19,705 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010150.1, seqID NZ_CP010150.1 [record index 0].\n",
      "2024-04-17 17:44:20,171 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010148.1, seqID NZ_CP010148.1 [record index 0].\n",
      "2024-04-17 17:44:20,407 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP010145.1, seqID NZ_CP010145.1 [record index 0].\n",
      "2024-04-17 17:44:20,905 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010140.1, seqID NZ_CP010140.1 [record index 0].\n",
      "2024-04-17 17:44:21,366 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010137.1, seqID NZ_CP010137.1 [record index 0].\n",
      "2024-04-17 17:44:21,864 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081762.1, seqID NZ_CP081762.1 [record index 0].\n",
      "2024-04-17 17:44:22,356 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081742.1, seqID NZ_CP081742.1 [record index 0].\n",
      "2024-04-17 17:44:22,605 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP073719.1, seqID NZ_CP073719.1 [record index 0].\n",
      "2024-04-17 17:44:22,857 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP073718.1, seqID NZ_CP073718.1 [record index 0].\n",
      "2024-04-17 17:44:23,146 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081724.1, seqID NZ_CP081724.1 [record index 0].\n",
      "2024-04-17 17:44:23,381 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081719.1, seqID NZ_CP081719.1 [record index 0].\n",
      "2024-04-17 17:44:23,634 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081713.1, seqID NZ_CP081713.1 [record index 0].\n",
      "2024-04-17 17:44:23,769 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024978.1, seqID NZ_CP024978.1 [record index 0].\n",
      "2024-04-17 17:44:24,026 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP025753.1, seqID NZ_CP025753.1 [record index 0].\n",
      "2024-04-17 17:44:24,197 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP032073.1, seqID NZ_CP032073.1 [record index 0].\n",
      "2024-04-17 17:44:24,333 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP081791.1, seqID NZ_CP081791.1 [record index 0].\n",
      "2024-04-17 17:44:24,633 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019051.1, seqID NZ_CP019051.1 [record index 0].\n",
      "2024-04-17 17:44:24,757 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034404.1, seqID NZ_CP034404.1 [record index 0].\n",
      "2024-04-17 17:44:24,882 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034399.1, seqID NZ_CP034399.1 [record index 0].\n",
      "2024-04-17 17:44:25,136 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031546.1, seqID NZ_CP031546.1 [record index 0].\n",
      "2024-04-17 17:44:25,360 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053730.1, seqID NZ_CP053730.1 [record index 0].\n",
      "2024-04-17 17:44:25,596 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053731.1, seqID NZ_CP053731.1 [record index 0].\n",
      "2024-04-17 17:44:25,746 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP033096.1, seqID NZ_CP033096.1 [record index 0].\n",
      "2024-04-17 17:44:25,969 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053720.1, seqID NZ_CP053720.1 [record index 0].\n",
      "2024-04-17 17:44:26,202 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP021726.1, seqID NZ_CP021726.1 [record index 0].\n",
      "2024-04-17 17:44:26,435 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP011018.1, seqID NZ_CP011018.1 [record index 0].\n",
      "2024-04-17 17:44:26,696 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031134.1, seqID NZ_CP031134.1 [record index 0].\n",
      "2024-04-17 17:44:26,956 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP042969.1, seqID NZ_CP042969.1 [record index 0].\n",
      "2024-04-17 17:44:27,172 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP011416.1, seqID NZ_CP011416.1 [record index 0].\n",
      "2024-04-17 17:44:27,457 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP037941.1, seqID NZ_CP037941.1 [record index 0].\n",
      "2024-04-17 17:44:27,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP014583.1, seqID NZ_CP014583.1 [record index 0].\n",
      "2024-04-17 17:44:28,186 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026939.2, seqID NZ_CP026939.2 [record index 0].\n",
      "2024-04-17 17:44:28,745 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047378.1, seqID NZ_CP047378.1 [record index 0].\n",
      "2024-04-17 17:44:29,172 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010125.1, seqID NZ_CP010125.1 [record index 0].\n",
      "2024-04-17 17:44:29,639 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010240.1, seqID NZ_CP010240.1 [record index 0].\n",
      "2024-04-17 17:44:30,176 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010122.1, seqID NZ_CP010122.1 [record index 0].\n",
      "2024-04-17 17:44:30,613 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP011938.1, seqID NZ_CP011938.1 [record index 0].\n",
      "2024-04-17 17:44:31,054 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010585.1, seqID NZ_CP010585.1 [record index 0].\n",
      "2024-04-17 17:44:31,564 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062204.1, seqID NZ_CP062204.1 [record index 0].\n",
      "2024-04-17 17:44:32,063 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014269.1, seqID NZ_CP014269.1 [record index 0].\n",
      "2024-04-17 17:44:32,519 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010119.1, seqID NZ_CP010119.1 [record index 0].\n",
      "2024-04-17 17:44:32,996 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014268.2, seqID NZ_CP014268.2 [record index 0].\n",
      "2024-04-17 17:44:33,466 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP011331.1, seqID NZ_CP011331.1 [record index 0].\n",
      "2024-04-17 17:44:33,936 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP052877.1, seqID NZ_CP052877.1 [record index 0].\n",
      "2024-04-17 17:44:34,400 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010117.1, seqID NZ_CP010117.1 [record index 0].\n",
      "2024-04-17 17:44:34,865 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010133.1, seqID NZ_CP010133.1 [record index 0].\n",
      "2024-04-17 17:44:35,288 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010132.1, seqID NZ_CP010132.1 [record index 0].\n",
      "2024-04-17 17:44:35,735 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029371.1, seqID NZ_CP029371.1 [record index 0].\n",
      "2024-04-17 17:44:36,153 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063341.1, seqID NZ_CP063341.1 [record index 0].\n",
      "2024-04-17 17:44:36,597 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020368.1, seqID NZ_CP020368.1 [record index 0].\n",
      "2024-04-17 17:44:36,822 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP014641.1, seqID NZ_CP014641.1 [record index 0].\n",
      "2024-04-17 17:44:37,026 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP014642.1, seqID NZ_CP014642.1 [record index 0].\n",
      "2024-04-17 17:44:37,418 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010816.1, seqID NZ_CP010816.1 [record index 0].\n",
      "2024-04-17 17:44:37,862 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083273.1, seqID NZ_CP083273.1 [record index 0].\n",
      "2024-04-17 17:44:38,304 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_012947.1, seqID NC_012947.1 [record index 0].\n",
      "2024-04-17 17:44:38,651 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047231.1, seqID NZ_CP047231.1 [record index 0].\n",
      "2024-04-17 17:44:38,863 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP084683.1, seqID NZ_CP084683.1 [record index 0].\n",
      "2024-04-17 17:44:39,082 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081489.1, seqID NZ_CP081489.1 [record index 0].\n",
      "2024-04-17 17:44:39,290 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053602.1, seqID NZ_CP053602.1 [record index 0].\n",
      "2024-04-17 17:44:39,549 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_012971.2, seqID NC_012971.2 [record index 0].\n",
      "2024-04-17 17:44:39,769 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_012892.2, seqID NC_012892.2 [record index 0].\n",
      "2024-04-17 17:44:39,979 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060121.1, seqID NZ_CP060121.1 [record index 0].\n",
      "2024-04-17 17:44:40,198 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053601.1, seqID NZ_CP053601.1 [record index 0].\n",
      "2024-04-17 17:44:40,402 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022295.1, seqID NZ_AP022295.1 [record index 0].\n",
      "2024-04-17 17:44:40,616 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP040397.1, seqID NZ_CP040397.1 [record index 0].\n",
      "2024-04-17 17:44:40,840 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP085842.1, seqID NZ_CP085842.1 [record index 0].\n",
      "2024-04-17 17:44:41,044 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024245.1, seqID NZ_CP024245.1 [record index 0].\n",
      "2024-04-17 17:44:41,272 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035874.1, seqID NZ_CP035874.1 [record index 0].\n",
      "2024-04-17 17:44:41,544 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP073624.1, seqID NZ_CP073624.1 [record index 0].\n",
      "2024-04-17 17:44:41,829 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072539.1, seqID NZ_CP072539.1 [record index 0].\n",
      "2024-04-17 17:44:42,120 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP045827.1, seqID NZ_CP045827.1 [record index 0].\n",
      "2024-04-17 17:44:42,333 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023541.1, seqID NZ_CP023541.1 [record index 0].\n",
      "2024-04-17 17:44:42,544 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043852.1, seqID NZ_CP043852.1 [record index 0].\n",
      "2024-04-17 17:44:42,776 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024278.1, seqID NZ_CP024278.1 [record index 0].\n",
      "2024-04-17 17:44:43,025 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024256.1, seqID NZ_CP024256.1 [record index 0].\n",
      "2024-04-17 17:44:43,237 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027430.1, seqID NZ_CP027430.1 [record index 0].\n",
      "2024-04-17 17:44:43,712 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP030331.1, seqID NZ_CP030331.1 [record index 0].\n",
      "2024-04-17 17:44:43,906 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027126.1, seqID NZ_CP027126.1 [record index 0].\n",
      "2024-04-17 17:44:44,435 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029741.1, seqID NZ_CP029741.1 [record index 0].\n",
      "2024-04-17 17:44:44,839 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP020055.1, seqID NZ_CP020055.1 [record index 0].\n",
      "2024-04-17 17:44:45,267 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024889.1, seqID NZ_CP024889.1 [record index 0].\n",
      "2024-04-17 17:44:45,503 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP029108.1, seqID NZ_CP029108.1 [record index 0].\n",
      "2024-04-17 17:44:45,694 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029111.1, seqID NZ_CP029111.1 [record index 0].\n",
      "2024-04-17 17:44:46,050 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP029122.1, seqID NZ_CP029122.1 [record index 0].\n",
      "2024-04-17 17:44:46,282 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP035944.1, seqID NZ_CP035944.1 [record index 0].\n",
      "2024-04-17 17:44:46,740 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043487.1, seqID NZ_CP043487.1 [record index 0].\n",
      "2024-04-17 17:44:47,158 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_020163.1, seqID NC_020163.1 [record index 0].\n",
      "2024-04-17 17:44:47,656 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP030939.1, seqID NZ_CP030939.1 [record index 0].\n",
      "2024-04-17 17:44:48,094 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058308.1, seqID NZ_CP058308.1 [record index 0].\n",
      "2024-04-17 17:44:48,513 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024801.1, seqID NZ_CP024801.1 [record index 0].\n",
      "2024-04-17 17:44:48,975 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082349.1, seqID NZ_CP082349.1 [record index 0].\n",
      "2024-04-17 17:44:49,223 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055256.1, seqID NZ_CP055256.1 [record index 0].\n",
      "2024-04-17 17:44:49,430 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP081706.1, seqID NZ_CP081706.1 [record index 0].\n",
      "2024-04-17 17:44:49,681 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP081701.1, seqID NZ_CP081701.1 [record index 0].\n",
      "2024-04-17 17:44:50,128 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055251.1, seqID NZ_CP055251.1 [record index 0].\n",
      "2024-04-17 17:44:50,481 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP007491.1, seqID NZ_CP007491.1 [record index 0].\n",
      "2024-04-17 17:44:50,686 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP007442.1, seqID NZ_CP007442.1 [record index 0].\n",
      "2024-04-17 17:44:51,104 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP022154.1, seqID NZ_CP022154.1 [record index 0].\n",
      "2024-04-17 17:44:51,541 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028766.1, seqID NZ_CP028766.1 [record index 0].\n",
      "2024-04-17 17:44:51,952 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028767.1, seqID NZ_CP028767.1 [record index 0].\n",
      "2024-04-17 17:44:52,398 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028769.1, seqID NZ_CP028769.1 [record index 0].\n",
      "2024-04-17 17:44:52,869 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028735.1, seqID NZ_CP028735.1 [record index 0].\n",
      "2024-04-17 17:44:53,309 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028737.1, seqID NZ_CP028737.1 [record index 0].\n",
      "2024-04-17 17:44:53,728 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028738.1, seqID NZ_CP028738.1 [record index 0].\n",
      "2024-04-17 17:44:54,176 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028740.1, seqID NZ_CP028740.1 [record index 0].\n",
      "2024-04-17 17:44:54,594 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028744.1, seqID NZ_CP028744.1 [record index 0].\n",
      "2024-04-17 17:44:55,019 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028747.1, seqID NZ_CP028747.1 [record index 0].\n",
      "2024-04-17 17:44:55,436 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028772.1, seqID NZ_CP028772.1 [record index 0].\n",
      "2024-04-17 17:44:55,656 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP040067.1, seqID NZ_CP040067.1 [record index 0].\n",
      "2024-04-17 17:44:56,104 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028757.1, seqID NZ_CP028757.1 [record index 0].\n",
      "2024-04-17 17:44:56,520 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028759.1, seqID NZ_CP028759.1 [record index 0].\n",
      "2024-04-17 17:44:56,980 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028762.1, seqID NZ_CP028762.1 [record index 0].\n",
      "2024-04-17 17:44:57,460 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029328.1, seqID NZ_CP029328.1 [record index 0].\n",
      "2024-04-17 17:44:57,878 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029981.1, seqID NZ_CP029981.1 [record index 0].\n",
      "2024-04-17 17:44:58,291 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP021339.1, seqID NZ_CP021339.1 [record index 0].\n",
      "2024-04-17 17:44:58,692 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP021335.1, seqID NZ_CP021335.1 [record index 0].\n",
      "2024-04-17 17:44:59,126 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047576.1, seqID NZ_CP047576.1 [record index 0].\n",
      "2024-04-17 17:44:59,514 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP009106.2, seqID NZ_CP009106.2 [record index 0].\n",
      "2024-04-17 17:44:59,814 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024243.1, seqID NZ_CP024243.1 [record index 0].\n",
      "2024-04-17 17:45:00,036 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027366.1, seqID NZ_CP027366.1 [record index 0].\n",
      "2024-04-17 17:45:00,259 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070906.1, seqID NZ_CP070906.1 [record index 0].\n",
      "2024-04-17 17:45:00,759 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027363.1, seqID NZ_CP027363.1 [record index 0].\n",
      "2024-04-17 17:45:01,137 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP050865.1, seqID NZ_CP050865.1 [record index 0].\n",
      "2024-04-17 17:45:01,500 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP050862.1, seqID NZ_CP050862.1 [record index 0].\n",
      "2024-04-17 17:45:01,894 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010315.1, seqID NZ_CP010315.1 [record index 0].\n",
      "2024-04-17 17:45:02,281 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP050044.1, seqID NZ_CP050044.1 [record index 0].\n",
      "2024-04-17 17:45:02,789 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP061764.1, seqID NZ_CP061764.1 [record index 0].\n",
      "2024-04-17 17:45:03,188 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LT883142.1, seqID NZ_LT883142.1 [record index 0].\n",
      "2024-04-17 17:45:03,397 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048322.1, seqID NZ_CP048322.1 [record index 0].\n",
      "2024-04-17 17:45:03,610 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048326.1, seqID NZ_CP048326.1 [record index 0].\n",
      "2024-04-17 17:45:04,046 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025847.1, seqID NZ_CP025847.1 [record index 0].\n",
      "2024-04-17 17:45:04,468 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050036.1, seqID NZ_CP050036.1 [record index 0].\n",
      "2024-04-17 17:45:04,886 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_011748.1, seqID NC_011748.1 [record index 0].\n",
      "2024-04-17 17:45:05,326 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091534.1, seqID NZ_CP091534.1 [record index 0].\n",
      "2024-04-17 17:45:05,683 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP048359.1, seqID NZ_CP048359.1 [record index 0].\n",
      "2024-04-17 17:45:06,135 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025856.1, seqID NZ_CP025856.1 [record index 0].\n",
      "2024-04-17 17:45:06,573 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025862.1, seqID NZ_CP025862.1 [record index 0].\n",
      "2024-04-17 17:45:07,008 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070920.1, seqID NZ_CP070920.1 [record index 0].\n",
      "2024-04-17 17:45:07,408 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP045209.1, seqID NZ_CP045209.1 [record index 0].\n",
      "2024-04-17 17:45:07,819 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049348.1, seqID NZ_CP049348.1 [record index 0].\n",
      "2024-04-17 17:45:08,185 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048376.1, seqID NZ_CP048376.1 [record index 0].\n",
      "2024-04-17 17:45:08,646 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP038859.1, seqID NZ_CP038859.1 [record index 0].\n",
      "2024-04-17 17:45:09,053 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050031.1, seqID NZ_CP050031.1 [record index 0].\n",
      "2024-04-17 17:45:09,448 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP084096.1, seqID NZ_CP084096.1 [record index 0].\n",
      "2024-04-17 17:45:09,815 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP047571.1, seqID NZ_CP047571.1 [record index 0].\n",
      "2024-04-17 17:45:10,032 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP059947.1, seqID NZ_CP059947.1 [record index 0].\n",
      "2024-04-17 17:45:10,238 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP053855.1, seqID NZ_CP053855.1 [record index 0].\n",
      "2024-04-17 17:45:10,642 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061749.1, seqID NZ_CP061749.1 [record index 0].\n",
      "2024-04-17 17:45:11,119 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025910.1, seqID NZ_CP025910.1 [record index 0].\n",
      "2024-04-17 17:45:11,602 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025913.1, seqID NZ_CP025913.1 [record index 0].\n",
      "2024-04-17 17:45:12,066 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075632.1, seqID NZ_CP075632.1 [record index 0].\n",
      "2024-04-17 17:45:12,478 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021892.1, seqID NZ_AP021892.1 [record index 0].\n",
      "2024-04-17 17:45:12,856 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP021890.1, seqID NZ_AP021890.1 [record index 0].\n",
      "2024-04-17 17:45:13,277 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP025220.2, seqID NZ_AP025220.2 [record index 0].\n",
      "2024-04-17 17:45:13,687 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP025205.2, seqID NZ_AP025205.2 [record index 0].\n",
      "2024-04-17 17:45:14,052 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP021891.1, seqID NZ_AP021891.1 [record index 0].\n",
      "2024-04-17 17:45:14,582 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027552.1, seqID NZ_CP027552.1 [record index 0].\n",
      "2024-04-17 17:45:14,962 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027371.1, seqID NZ_CP027371.1 [record index 0].\n",
      "2024-04-17 17:45:15,445 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027219.1, seqID NZ_CP027219.1 [record index 0].\n",
      "2024-04-17 17:45:15,926 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027340.1, seqID NZ_CP027340.1 [record index 0].\n",
      "2024-04-17 17:45:16,308 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027307.1, seqID NZ_CP027307.1 [record index 0].\n",
      "2024-04-17 17:45:16,693 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027221.1, seqID NZ_CP027221.1 [record index 0].\n",
      "2024-04-17 17:45:17,125 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024232.1, seqID NZ_CP024232.1 [record index 0].\n",
      "2024-04-17 17:45:17,514 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024223.1, seqID NZ_CP024223.1 [record index 0].\n",
      "2024-04-17 17:45:17,939 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024228.1, seqID NZ_CP024228.1 [record index 0].\n",
      "2024-04-17 17:45:18,318 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027342.1, seqID NZ_CP027342.1 [record index 0].\n",
      "2024-04-17 17:45:18,700 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027310.1, seqID NZ_CP027310.1 [record index 0].\n",
      "2024-04-17 17:45:19,135 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027368.1, seqID NZ_CP027368.1 [record index 0].\n",
      "2024-04-17 17:45:19,533 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027449.1, seqID NZ_CP027449.1 [record index 0].\n",
      "2024-04-17 17:45:20,026 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027325.1, seqID NZ_CP027325.1 [record index 0].\n",
      "2024-04-17 17:45:20,399 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027347.1, seqID NZ_CP027347.1 [record index 0].\n",
      "2024-04-17 17:45:20,788 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027573.1, seqID NZ_CP027573.1 [record index 0].\n",
      "2024-04-17 17:45:21,312 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027766.1, seqID NZ_CP027766.1 [record index 0].\n",
      "2024-04-17 17:45:21,838 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027331.1, seqID NZ_CP027331.1 [record index 0].\n",
      "2024-04-17 17:45:22,233 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027380.1, seqID NZ_CP027380.1 [record index 0].\n",
      "2024-04-17 17:45:22,702 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027323.1, seqID NZ_CP027323.1 [record index 0].\n",
      "2024-04-17 17:45:23,152 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027586.1, seqID NZ_CP027586.1 [record index 0].\n",
      "2024-04-17 17:45:23,594 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024289.1, seqID NZ_CP024289.1 [record index 0].\n",
      "2024-04-17 17:45:24,169 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP013029.1, seqID NZ_CP013029.1 [record index 0].\n",
      "2024-04-17 17:45:24,586 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022912.1, seqID NZ_CP022912.1 [record index 0].\n",
      "2024-04-17 17:45:25,003 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024479.1, seqID NZ_CP024479.1 [record index 0].\n",
      "2024-04-17 17:45:25,431 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027388.1, seqID NZ_CP027388.1 [record index 0].\n",
      "2024-04-17 17:45:25,904 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015240.1, seqID NZ_CP015240.1 [record index 0].\n",
      "2024-04-17 17:45:26,320 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_018658.1, seqID NC_018658.1 [record index 0].\n",
      "2024-04-17 17:45:26,743 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_018661.1, seqID NC_018661.1 [record index 0].\n",
      "2024-04-17 17:45:27,167 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_018650.1, seqID NC_018650.1 [record index 0].\n",
      "2024-04-17 17:45:27,384 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP061101.1, seqID NZ_CP061101.1 [record index 0].\n",
      "2024-04-17 17:45:27,764 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061108.1, seqID NZ_CP061108.1 [record index 0].\n",
      "2024-04-17 17:45:28,135 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023359.1, seqID NZ_CP023359.1 [record index 0].\n",
      "2024-04-17 17:45:28,535 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP046006.1, seqID NZ_CP046006.1 [record index 0].\n",
      "2024-04-17 17:45:28,880 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP020520.1, seqID NZ_CP020520.1 [record index 0].\n",
      "2024-04-17 17:45:29,258 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP047010.1, seqID NZ_CP047010.1 [record index 0].\n",
      "2024-04-17 17:45:29,676 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP019856.1, seqID NZ_AP019856.1 [record index 0].\n",
      "2024-04-17 17:45:30,105 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086391.1, seqID NZ_CP086391.1 [record index 0].\n",
      "2024-04-17 17:45:30,471 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP059288.1, seqID NZ_CP059288.1 [record index 0].\n",
      "2024-04-17 17:45:30,874 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040269.1, seqID NZ_CP040269.1 [record index 0].\n",
      "2024-04-17 17:45:31,275 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024141.1, seqID NZ_CP024141.1 [record index 0].\n",
      "2024-04-17 17:45:31,720 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024127.1, seqID NZ_CP024127.1 [record index 0].\n",
      "2024-04-17 17:45:31,929 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP019271.1, seqID NZ_CP019271.1 [record index 0].\n",
      "2024-04-17 17:45:32,370 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019250.1, seqID NZ_CP019250.1 [record index 0].\n",
      "2024-04-17 17:45:32,763 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP020107.1, seqID NZ_CP020107.1 [record index 0].\n",
      "2024-04-17 17:45:33,004 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019267.1, seqID NZ_CP019267.1 [record index 0].\n",
      "2024-04-17 17:45:33,262 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019259.1, seqID NZ_CP019259.1 [record index 0].\n",
      "2024-04-17 17:45:33,464 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048344.1, seqID NZ_CP048344.1 [record index 0].\n",
      "2024-04-17 17:45:33,738 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025916.1, seqID NZ_CP025916.1 [record index 0].\n",
      "2024-04-17 17:45:34,174 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_013353.1, seqID NC_013353.1 [record index 0].\n",
      "2024-04-17 17:45:34,515 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP087570.1, seqID NZ_CP087570.1 [record index 0].\n",
      "2024-04-17 17:45:34,877 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NC_013364.1, seqID NC_013364.1 [record index 0].\n",
      "2024-04-17 17:45:35,232 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_AP019761.1, seqID NZ_AP019761.1 [record index 0].\n",
      "2024-04-17 17:45:35,663 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025920.1, seqID NZ_CP025920.1 [record index 0].\n",
      "2024-04-17 17:45:36,083 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081782.1, seqID NZ_CP081782.1 [record index 0].\n",
      "2024-04-17 17:45:36,494 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081690.1, seqID NZ_CP081690.1 [record index 0].\n",
      "2024-04-17 17:45:36,909 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081774.1, seqID NZ_CP081774.1 [record index 0].\n",
      "2024-04-17 17:45:37,228 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081684.1, seqID NZ_CP081684.1 [record index 0].\n",
      "2024-04-17 17:45:37,600 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081767.1, seqID NZ_CP081767.1 [record index 0].\n",
      "2024-04-17 17:45:37,982 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081677.1, seqID NZ_CP081677.1 [record index 0].\n",
      "2024-04-17 17:45:38,345 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP015228.1, seqID NZ_CP015228.1 [record index 0].\n",
      "2024-04-17 17:45:38,554 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027462.1, seqID NZ_CP027462.1 [record index 0].\n",
      "2024-04-17 17:45:38,801 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027373.1, seqID NZ_CP027373.1 [record index 0].\n",
      "2024-04-17 17:45:38,911 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP042934.2, seqID NZ_CP042934.2 [record index 0].\n",
      "2024-04-17 17:45:39,116 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035832.1, seqID NZ_CP035832.1 [record index 0].\n",
      "2024-04-17 17:45:39,327 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027584.1, seqID NZ_CP027584.1 [record index 0].\n",
      "2024-04-17 17:45:39,664 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP010829.1, seqID NZ_CP010829.1 [record index 0].\n",
      "2024-04-17 17:45:40,003 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP014099.2, seqID NZ_CP014099.2 [record index 0].\n",
      "2024-04-17 17:45:40,381 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058846.1, seqID NZ_CP058846.1 [record index 0].\n",
      "2024-04-17 17:45:40,750 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_007613.1, seqID NC_007613.1 [record index 0].\n",
      "2024-04-17 17:45:40,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP082357.1, seqID NZ_CP082357.1 [record index 0].\n",
      "2024-04-17 17:45:41,173 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP007394.1, seqID NZ_CP007394.1 [record index 0].\n",
      "2024-04-17 17:45:41,522 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP007393.1, seqID NZ_CP007393.1 [record index 0].\n",
      "2024-04-17 17:45:41,878 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP007392.1, seqID NZ_CP007392.1 [record index 0].\n",
      "2024-04-17 17:45:42,230 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051751.1, seqID NZ_CP051751.1 [record index 0].\n",
      "2024-04-17 17:45:42,582 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010236.1, seqID NZ_CP010236.1 [record index 0].\n",
      "2024-04-17 17:45:42,942 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031347.1, seqID NZ_CP031347.1 [record index 0].\n",
      "2024-04-17 17:45:43,226 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024659.1, seqID NZ_CP024659.1 [record index 0].\n",
      "2024-04-17 17:45:43,580 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028379.1, seqID NZ_CP028379.1 [record index 0].\n",
      "2024-04-17 17:45:43,931 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031349.1, seqID NZ_CP031349.1 [record index 0].\n",
      "2024-04-17 17:45:44,242 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP031353.1, seqID NZ_CP031353.1 [record index 0].\n",
      "2024-04-17 17:45:44,603 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031355.1, seqID NZ_CP031355.1 [record index 0].\n",
      "2024-04-17 17:45:44,969 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035770.1, seqID NZ_CP035770.1 [record index 0].\n",
      "2024-04-17 17:45:45,325 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035772.1, seqID NZ_CP035772.1 [record index 0].\n",
      "2024-04-17 17:45:45,803 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP028116.1, seqID NZ_CP028116.1 [record index 0].\n",
      "2024-04-17 17:45:46,102 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028112.1, seqID NZ_CP028112.1 [record index 0].\n",
      "2024-04-17 17:45:46,466 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP006262.1, seqID NZ_CP006262.1 [record index 0].\n",
      "2024-04-17 17:45:46,844 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP006027.1, seqID NZ_CP006027.1 [record index 0].\n",
      "2024-04-17 17:45:47,214 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP007133.1, seqID NZ_CP007133.1 [record index 0].\n",
      "2024-04-17 17:45:47,578 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP007136.1, seqID NZ_CP007136.1 [record index 0].\n",
      "2024-04-17 17:45:47,935 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035767.1, seqID NZ_CP035767.1 [record index 0].\n",
      "2024-04-17 17:45:48,307 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031345.1, seqID NZ_CP031345.1 [record index 0].\n",
      "2024-04-17 17:45:48,566 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031341.1, seqID NZ_CP031341.1 [record index 0].\n",
      "2024-04-17 17:45:48,933 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035768.1, seqID NZ_CP035768.1 [record index 0].\n",
      "2024-04-17 17:45:49,196 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031343.1, seqID NZ_CP031343.1 [record index 0].\n",
      "2024-04-17 17:45:49,490 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP028126.1, seqID NZ_CP028126.1 [record index 0].\n",
      "2024-04-17 17:45:49,705 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062824.1, seqID NZ_CP062824.1 [record index 0].\n",
      "2024-04-17 17:45:49,997 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP058682.2, seqID NZ_CP058682.2 [record index 0].\n",
      "2024-04-17 17:45:50,360 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR882978.1, seqID NZ_LR882978.1 [record index 0].\n",
      "2024-04-17 17:45:50,653 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031919.1, seqID NZ_CP031919.1 [record index 0].\n",
      "2024-04-17 17:45:50,992 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP031922.1, seqID NZ_CP031922.1 [record index 0].\n",
      "2024-04-17 17:45:51,500 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP012693.1, seqID NZ_CP012693.1 [record index 0].\n",
      "2024-04-17 17:45:51,808 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP085637.1, seqID NZ_CP085637.1 [record index 0].\n",
      "2024-04-17 17:45:52,075 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP065611.1, seqID NZ_CP065611.1 [record index 0].\n",
      "2024-04-17 17:45:52,399 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023349.1, seqID NZ_CP023349.1 [record index 0].\n",
      "2024-04-17 17:45:52,622 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041919.1, seqID NZ_CP041919.1 [record index 0].\n",
      "2024-04-17 17:45:52,916 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_AP018808.1, seqID NZ_AP018808.1 [record index 0].\n",
      "2024-04-17 17:45:53,239 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP037945.1, seqID NZ_CP037945.1 [record index 0].\n",
      "2024-04-17 17:45:53,570 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP037943.1, seqID NZ_CP037943.1 [record index 0].\n",
      "2024-04-17 17:45:53,854 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP035812.1, seqID NZ_CP035812.1 [record index 0].\n",
      "2024-04-17 17:45:54,076 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028765.1, seqID NZ_CP028765.1 [record index 0].\n",
      "2024-04-17 17:45:54,371 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027599.1, seqID NZ_CP027599.1 [record index 0].\n",
      "2024-04-17 17:45:54,581 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027362.1, seqID NZ_CP027362.1 [record index 0].\n",
      "2024-04-17 17:45:54,801 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024240.1, seqID NZ_CP024240.1 [record index 0].\n",
      "2024-04-17 17:45:55,042 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027459.1, seqID NZ_CP027459.1 [record index 0].\n",
      "2024-04-17 17:45:55,283 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP025865.1, seqID NZ_CP025865.1 [record index 0].\n",
      "2024-04-17 17:45:55,524 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP025869.1, seqID NZ_CP025869.1 [record index 0].\n",
      "2024-04-17 17:45:55,762 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP025892.1, seqID NZ_CP025892.1 [record index 0].\n",
      "2024-04-17 17:45:55,995 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP015912.1, seqID NZ_CP015912.1 [record index 0].\n",
      "2024-04-17 17:45:56,184 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP021895.1, seqID NZ_AP021895.1 [record index 0].\n",
      "2024-04-17 17:45:56,474 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027390.1, seqID NZ_CP027390.1 [record index 0].\n",
      "2024-04-17 17:45:56,738 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027361.1, seqID NZ_CP027361.1 [record index 0].\n",
      "2024-04-17 17:45:56,806 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027319.1, seqID NZ_CP027319.1 [record index 0].\n",
      "2024-04-17 17:45:57,024 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027447.1, seqID NZ_CP027447.1 [record index 0].\n",
      "2024-04-17 17:45:57,237 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027548.1, seqID NZ_CP027548.1 [record index 0].\n",
      "2024-04-17 17:45:57,510 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027387.1, seqID NZ_CP027387.1 [record index 0].\n",
      "2024-04-17 17:45:57,761 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027338.1, seqID NZ_CP027338.1 [record index 0].\n",
      "2024-04-17 17:45:57,987 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027472.1, seqID NZ_CP027472.1 [record index 0].\n",
      "2024-04-17 17:45:58,214 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027582.1, seqID NZ_CP027582.1 [record index 0].\n",
      "2024-04-17 17:45:58,560 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027577.1, seqID NZ_CP027577.1 [record index 0].\n",
      "2024-04-17 17:45:58,833 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027546.1, seqID NZ_CP027546.1 [record index 0].\n",
      "2024-04-17 17:45:59,147 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027555.1, seqID NZ_CP027555.1 [record index 0].\n",
      "2024-04-17 17:45:59,390 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027445.1, seqID NZ_CP027445.1 [record index 0].\n",
      "2024-04-17 17:45:59,630 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027544.1, seqID NZ_CP027544.1 [record index 0].\n",
      "2024-04-17 17:45:59,925 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027442.1, seqID NZ_CP027442.1 [record index 0].\n",
      "2024-04-17 17:46:00,224 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP027352.1, seqID NZ_CP027352.1 [record index 0].\n",
      "2024-04-17 17:46:00,460 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP053785.1, seqID NZ_CP053785.1 [record index 0].\n",
      "2024-04-17 17:46:00,617 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP024147.1, seqID NZ_CP024147.1 [record index 0].\n",
      "2024-04-17 17:46:00,902 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP020106.1, seqID NZ_CP020106.1 [record index 0].\n",
      "2024-04-17 17:46:01,135 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP020092.1, seqID NZ_CP020092.1 [record index 0].\n",
      "2024-04-17 17:46:01,427 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_013361.1, seqID NC_013361.1 [record index 0].\n",
      "2024-04-17 17:46:01,669 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054353.1, seqID NZ_CP054353.1 [record index 0].\n",
      "2024-04-17 17:46:01,853 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR778153.1, seqID NZ_LR778153.1 [record index 0].\n",
      "2024-04-17 17:46:02,085 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR898874.1, seqID NZ_LR898874.1 [record index 0].\n",
      "2024-04-17 17:46:02,280 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010200.1, seqID NZ_CP010200.1 [record index 0].\n",
      "2024-04-17 17:46:02,519 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019944.1, seqID NZ_CP019944.1 [record index 0].\n",
      "2024-04-17 17:46:02,754 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028192.1, seqID NZ_CP028192.1 [record index 0].\n",
      "2024-04-17 17:46:02,986 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP067307.1, seqID NZ_CP067307.1 [record index 0].\n",
      "2024-04-17 17:46:03,212 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055113.1, seqID NZ_CP055113.1 [record index 0].\n",
      "2024-04-17 17:46:03,473 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055199.1, seqID NZ_CP055199.1 [record index 0].\n",
      "2024-04-17 17:46:03,690 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP066844.1, seqID NZ_CP066844.1 [record index 0].\n",
      "2024-04-17 17:46:03,902 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019392.1, seqID NZ_CP019392.1 [record index 0].\n",
      "2024-04-17 17:46:04,110 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043033.1, seqID NZ_CP043033.1 [record index 0].\n",
      "2024-04-17 17:46:04,346 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022161.1, seqID NZ_AP022161.1 [record index 0].\n",
      "2024-04-17 17:46:04,578 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022098.1, seqID NZ_AP022098.1 [record index 0].\n",
      "2024-04-17 17:46:04,815 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020025.1, seqID NZ_CP020025.1 [record index 0].\n",
      "2024-04-17 17:46:05,019 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054191.1, seqID NZ_CP054191.1 [record index 0].\n",
      "2024-04-17 17:46:05,272 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055022.1, seqID NZ_CP055022.1 [record index 0].\n",
      "2024-04-17 17:46:05,493 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP008697.1, seqID NZ_CP008697.1 [record index 0].\n",
      "2024-04-17 17:46:05,694 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_010498.1, seqID NC_010498.1 [record index 0].\n",
      "2024-04-17 17:46:05,896 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP067235.1, seqID NZ_CP067235.1 [record index 0].\n",
      "2024-04-17 17:46:06,123 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP034389.1, seqID NZ_CP034389.1 [record index 0].\n",
      "2024-04-17 17:46:06,365 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062865.1, seqID NZ_CP062865.1 [record index 0].\n",
      "2024-04-17 17:46:06,602 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062868.1, seqID NZ_CP062868.1 [record index 0].\n",
      "2024-04-17 17:46:06,816 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062889.1, seqID NZ_CP062889.1 [record index 0].\n",
      "2024-04-17 17:46:07,153 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP016546.1, seqID NZ_CP016546.1 [record index 0].\n",
      "2024-04-17 17:46:07,348 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041302.1, seqID NZ_CP041302.1 [record index 0].\n",
      "2024-04-17 17:46:07,610 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890536.1, seqID NZ_LR890536.1 [record index 0].\n",
      "2024-04-17 17:46:07,867 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015834.1, seqID NZ_CP015834.1 [record index 0].\n",
      "2024-04-17 17:46:08,070 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP047405.1, seqID NZ_CP047405.1 [record index 0].\n",
      "2024-04-17 17:46:08,273 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054940.1, seqID NZ_CP054940.1 [record index 0].\n",
      "2024-04-17 17:46:08,499 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR130555.1, seqID NZ_LR130555.1 [record index 0].\n",
      "2024-04-17 17:46:08,725 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890334.1, seqID NZ_LR890334.1 [record index 0].\n",
      "2024-04-17 17:46:08,934 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR890326.1, seqID NZ_LR890326.1 [record index 0].\n",
      "2024-04-17 17:46:09,203 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058714.1, seqID NZ_CP058714.1 [record index 0].\n",
      "2024-04-17 17:46:09,426 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP022164.1, seqID NZ_CP022164.1 [record index 0].\n",
      "2024-04-17 17:46:09,675 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP061185.1, seqID NZ_CP061185.1 [record index 0].\n",
      "2024-04-17 17:46:09,899 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP062211.1, seqID NZ_CP062211.1 [record index 0].\n",
      "2024-04-17 17:46:10,097 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP018323.1, seqID NZ_CP018323.1 [record index 0].\n",
      "2024-04-17 17:46:10,369 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091712.1, seqID NZ_CP091712.1 [record index 0].\n",
      "2024-04-17 17:46:10,608 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP064103.1, seqID NZ_CP064103.1 [record index 0].\n",
      "2024-04-17 17:46:10,804 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023815.1, seqID NZ_CP023815.1 [record index 0].\n",
      "2024-04-17 17:46:11,003 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010167.1, seqID NZ_CP010167.1 [record index 0].\n",
      "2024-04-17 17:46:11,231 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP024687.1, seqID NZ_AP024687.1 [record index 0].\n",
      "2024-04-17 17:46:11,470 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP033850.1, seqID NZ_CP033850.1 [record index 0].\n",
      "2024-04-17 17:46:11,693 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070152.1, seqID NZ_CP070152.1 [record index 0].\n",
      "2024-04-17 17:46:11,928 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040919.1, seqID NZ_CP040919.1 [record index 0].\n",
      "2024-04-17 17:46:12,135 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063476.1, seqID NZ_CP063476.1 [record index 0].\n",
      "2024-04-17 17:46:12,330 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063518.1, seqID NZ_CP063518.1 [record index 0].\n",
      "2024-04-17 17:46:12,568 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP009859.1, seqID NZ_CP009859.1 [record index 0].\n",
      "2024-04-17 17:46:12,809 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019029.1, seqID NZ_CP019029.1 [record index 0].\n",
      "2024-04-17 17:46:13,041 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018948.1, seqID NZ_CP018948.1 [record index 0].\n",
      "2024-04-17 17:46:13,267 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP085183.1, seqID NZ_CP085183.1 [record index 0].\n",
      "2024-04-17 17:46:13,504 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060899.1, seqID NZ_CP060899.1 [record index 0].\n",
      "2024-04-17 17:46:13,758 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060914.1, seqID NZ_CP060914.1 [record index 0].\n",
      "2024-04-17 17:46:14,011 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060919.1, seqID NZ_CP060919.1 [record index 0].\n",
      "2024-04-17 17:46:14,144 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060924.1, seqID NZ_CP060924.1 [record index 0].\n",
      "2024-04-17 17:46:14,406 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060933.1, seqID NZ_CP060933.1 [record index 0].\n",
      "2024-04-17 17:46:14,618 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024821.1, seqID NZ_CP024821.1 [record index 0].\n",
      "2024-04-17 17:46:14,840 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053723.1, seqID NZ_CP053723.1 [record index 0].\n",
      "2024-04-17 17:46:15,060 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041300.1, seqID NZ_CP041300.1 [record index 0].\n",
      "2024-04-17 17:46:15,279 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042896.1, seqID NZ_CP042896.1 [record index 0].\n",
      "2024-04-17 17:46:15,504 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021722.1, seqID NZ_CP021722.1 [record index 0].\n",
      "2024-04-17 17:46:15,745 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021535.1, seqID NZ_CP021535.1 [record index 0].\n",
      "2024-04-17 17:46:16,003 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020048.1, seqID NZ_CP020048.1 [record index 0].\n",
      "2024-04-17 17:46:16,202 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026755.1, seqID NZ_CP026755.1 [record index 0].\n",
      "2024-04-17 17:46:16,361 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP049967.1, seqID NZ_CP049967.1 [record index 0].\n",
      "2024-04-17 17:46:16,558 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP046396.1, seqID NZ_CP046396.1 [record index 0].\n",
      "2024-04-17 17:46:16,771 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP074701.1, seqID NZ_CP074701.1 [record index 0].\n",
      "2024-04-17 17:46:16,982 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP059003.1, seqID NZ_CP059003.1 [record index 0].\n",
      "2024-04-17 17:46:17,209 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048107.1, seqID NZ_CP048107.1 [record index 0].\n",
      "2024-04-17 17:46:17,418 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP020509.1, seqID NZ_CP020509.1 [record index 0].\n",
      "2024-04-17 17:46:17,622 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023366.1, seqID NZ_CP023366.1 [record index 0].\n",
      "2024-04-17 17:46:17,832 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048337.1, seqID NZ_CP048337.1 [record index 0].\n",
      "2024-04-17 17:46:18,085 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015229.1, seqID NZ_CP015229.1 [record index 0].\n",
      "2024-04-17 17:46:18,290 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022536.1, seqID NZ_AP022536.1 [record index 0].\n",
      "2024-04-17 17:46:18,493 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022478.1, seqID NZ_AP022478.1 [record index 0].\n",
      "2024-04-17 17:46:18,715 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022279.1, seqID NZ_CP022279.1 [record index 0].\n",
      "2024-04-17 17:46:18,924 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051765.1, seqID NZ_CP051765.1 [record index 0].\n",
      "2024-04-17 17:46:19,184 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028110.1, seqID NZ_CP028110.1 [record index 0].\n",
      "2024-04-17 17:46:19,434 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062901.1, seqID NZ_CP062901.1 [record index 0].\n",
      "2024-04-17 17:46:19,625 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134315.1, seqID NZ_LR134315.1 [record index 0].\n",
      "2024-04-17 17:46:19,826 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR890270.1, seqID NZ_LR890270.1 [record index 0].\n",
      "2024-04-17 17:46:20,031 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053259.1, seqID NZ_CP053259.1 [record index 0].\n",
      "2024-04-17 17:46:20,275 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031910.1, seqID NZ_CP031910.1 [record index 0].\n",
      "2024-04-17 17:46:20,474 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP071439.1, seqID NZ_CP071439.1 [record index 0].\n",
      "2024-04-17 17:46:20,681 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP018957.1, seqID NZ_CP018957.1 [record index 0].\n",
      "2024-04-17 17:46:20,913 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP033635.1, seqID NZ_CP033635.1 [record index 0].\n",
      "2024-04-17 17:46:21,121 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR025101.1, seqID NZ_LR025101.1 [record index 0].\n",
      "2024-04-17 17:46:21,309 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP082773.1, seqID NZ_CP082773.1 [record index 0].\n",
      "2024-04-17 17:46:21,508 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP076706.1, seqID NZ_CP076706.1 [record index 0].\n",
      "2024-04-17 17:46:21,759 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022298.1, seqID NZ_AP022298.1 [record index 0].\n",
      "2024-04-17 17:46:21,999 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028739.1, seqID NZ_CP028739.1 [record index 0].\n",
      "2024-04-17 17:46:22,267 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028743.1, seqID NZ_CP028743.1 [record index 0].\n",
      "2024-04-17 17:46:22,543 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028753.1, seqID NZ_CP028753.1 [record index 0].\n",
      "2024-04-17 17:46:22,804 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028755.1, seqID NZ_CP028755.1 [record index 0].\n",
      "2024-04-17 17:46:23,049 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027317.1, seqID NZ_CP027317.1 [record index 0].\n",
      "2024-04-17 17:46:23,289 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027454.1, seqID NZ_CP027454.1 [record index 0].\n",
      "2024-04-17 17:46:23,530 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027351.1, seqID NZ_CP027351.1 [record index 0].\n",
      "2024-04-17 17:46:23,780 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027435.1, seqID NZ_CP027435.1 [record index 0].\n",
      "2024-04-17 17:46:24,035 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022407.1, seqID NZ_CP022407.1 [record index 0].\n",
      "2024-04-17 17:46:24,275 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080260.1, seqID NZ_CP080260.1 [record index 0].\n",
      "2024-04-17 17:46:24,494 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_017626.1, seqID NC_017626.1 [record index 0].\n",
      "2024-04-17 17:46:24,718 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP066144.1, seqID NZ_CP066144.1 [record index 0].\n",
      "2024-04-17 17:46:24,959 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021202.1, seqID NZ_CP021202.1 [record index 0].\n",
      "2024-04-17 17:46:25,197 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025707.1, seqID NZ_CP025707.1 [record index 0].\n",
      "2024-04-17 17:46:25,398 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022261.1, seqID NZ_AP022261.1 [record index 0].\n",
      "2024-04-17 17:46:25,599 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022222.1, seqID NZ_AP022222.1 [record index 0].\n",
      "2024-04-17 17:46:25,811 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022207.1, seqID NZ_AP022207.1 [record index 0].\n",
      "2024-04-17 17:46:26,042 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022173.1, seqID NZ_AP022173.1 [record index 0].\n",
      "2024-04-17 17:46:26,284 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022069.1, seqID NZ_AP022069.1 [record index 0].\n",
      "2024-04-17 17:46:26,514 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022229.3, seqID NZ_CP022229.3 [record index 0].\n",
      "2024-04-17 17:46:26,743 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028589.2, seqID NZ_CP028589.2 [record index 0].\n",
      "2024-04-17 17:46:26,996 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043295.1, seqID NZ_CP043295.1 [record index 0].\n",
      "2024-04-17 17:46:27,225 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032879.1, seqID NZ_CP032879.1 [record index 0].\n",
      "2024-04-17 17:46:27,419 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP082355.1, seqID NZ_CP082355.1 [record index 0].\n",
      "2024-04-17 17:46:27,660 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058596.1, seqID NZ_CP058596.1 [record index 0].\n",
      "2024-04-17 17:46:27,934 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP016497.1, seqID NZ_CP016497.1 [record index 0].\n",
      "2024-04-17 17:46:28,162 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040456.1, seqID NZ_CP040456.1 [record index 0].\n",
      "2024-04-17 17:46:28,356 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP047600.1, seqID NZ_CP047600.1 [record index 0].\n",
      "2024-04-17 17:46:28,581 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022533.1, seqID NZ_AP022533.1 [record index 0].\n",
      "2024-04-17 17:46:28,801 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080246.1, seqID NZ_CP080246.1 [record index 0].\n",
      "2024-04-17 17:46:29,030 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076301.1, seqID NZ_CP076301.1 [record index 0].\n",
      "2024-04-17 17:46:29,258 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054161.1, seqID NZ_CP054161.1 [record index 0].\n",
      "2024-04-17 17:46:29,522 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058874.1, seqID NZ_CP058874.1 [record index 0].\n",
      "2024-04-17 17:46:29,800 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054965.1, seqID NZ_CP054965.1 [record index 0].\n",
      "2024-04-17 17:46:30,007 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053245.1, seqID NZ_CP053245.1 [record index 0].\n",
      "2024-04-17 17:46:30,250 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051744.1, seqID NZ_CP051744.1 [record index 0].\n",
      "2024-04-17 17:46:30,486 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053247.1, seqID NZ_CP053247.1 [record index 0].\n",
      "2024-04-17 17:46:30,689 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051694.1, seqID NZ_CP051694.1 [record index 0].\n",
      "2024-04-17 17:46:30,924 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054363.1, seqID NZ_CP054363.1 [record index 0].\n",
      "2024-04-17 17:46:31,219 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP051700.1, seqID NZ_CP051700.1 [record index 0].\n",
      "2024-04-17 17:46:31,435 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051706.1, seqID NZ_CP051706.1 [record index 0].\n",
      "2024-04-17 17:46:31,661 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054368.1, seqID NZ_CP054368.1 [record index 0].\n",
      "2024-04-17 17:46:31,884 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051725.1, seqID NZ_CP051725.1 [record index 0].\n",
      "2024-04-17 17:46:32,090 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051727.1, seqID NZ_CP051727.1 [record index 0].\n",
      "2024-04-17 17:46:32,299 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051738.1, seqID NZ_CP051738.1 [record index 0].\n",
      "2024-04-17 17:46:32,512 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP082827.1, seqID NZ_CP082827.1 [record index 0].\n",
      "2024-04-17 17:46:32,710 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778152.1, seqID NZ_LR778152.1 [record index 0].\n",
      "2024-04-17 17:46:32,913 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778146.1, seqID NZ_LR778146.1 [record index 0].\n",
      "2024-04-17 17:46:33,116 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778142.1, seqID NZ_LR778142.1 [record index 0].\n",
      "2024-04-17 17:46:33,305 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027103.1, seqID NZ_CP027103.1 [record index 0].\n",
      "2024-04-17 17:46:33,571 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068828.1, seqID NZ_CP068828.1 [record index 0].\n",
      "2024-04-17 17:46:33,778 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068825.1, seqID NZ_CP068825.1 [record index 0].\n",
      "2024-04-17 17:46:34,004 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068822.1, seqID NZ_CP068822.1 [record index 0].\n",
      "2024-04-17 17:46:34,233 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068817.1, seqID NZ_CP068817.1 [record index 0].\n",
      "2024-04-17 17:46:34,458 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068816.1, seqID NZ_CP068816.1 [record index 0].\n",
      "2024-04-17 17:46:34,685 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068815.1, seqID NZ_CP068815.1 [record index 0].\n",
      "2024-04-17 17:46:34,889 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP068809.1, seqID NZ_CP068809.1 [record index 0].\n",
      "2024-04-17 17:46:35,087 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068806.1, seqID NZ_CP068806.1 [record index 0].\n",
      "2024-04-17 17:46:35,346 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068799.1, seqID NZ_CP068799.1 [record index 0].\n",
      "2024-04-17 17:46:35,585 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP011915.1, seqID NZ_CP011915.1 [record index 0].\n",
      "2024-04-17 17:46:35,802 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080163.1, seqID NZ_CP080163.1 [record index 0].\n",
      "2024-04-17 17:46:36,063 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080155.1, seqID NZ_CP080155.1 [record index 0].\n",
      "2024-04-17 17:46:36,293 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043181.1, seqID NZ_CP043181.1 [record index 0].\n",
      "2024-04-17 17:46:36,507 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012379.1, seqID NZ_CP012379.1 [record index 0].\n",
      "2024-04-17 17:46:36,725 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP049201.1, seqID NZ_CP049201.1 [record index 0].\n",
      "2024-04-17 17:46:36,959 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP085063.1, seqID NZ_CP085063.1 [record index 0].\n",
      "2024-04-17 17:46:37,174 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017634.1, seqID NC_017634.1 [record index 0].\n",
      "2024-04-17 17:46:37,428 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047609.1, seqID NZ_CP047609.1 [record index 0].\n",
      "2024-04-17 17:46:37,663 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043406.1, seqID NZ_CP043406.1 [record index 0].\n",
      "2024-04-17 17:46:37,895 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP044403.1, seqID NZ_CP044403.1 [record index 0].\n",
      "2024-04-17 17:46:38,117 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058217.1, seqID NZ_CP058217.1 [record index 0].\n",
      "2024-04-17 17:46:38,349 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022686.1, seqID NZ_CP022686.1 [record index 0].\n",
      "2024-04-17 17:46:38,595 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP007799.1, seqID NZ_CP007799.1 [record index 0].\n",
      "2024-04-17 17:46:38,814 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035486.1, seqID NZ_CP035486.1 [record index 0].\n",
      "2024-04-17 17:46:39,035 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP016007.1, seqID NZ_CP016007.1 [record index 0].\n",
      "2024-04-17 17:46:39,240 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134237.1, seqID NZ_LR134237.1 [record index 0].\n",
      "2024-04-17 17:46:39,466 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072787.1, seqID NZ_CP072787.1 [record index 0].\n",
      "2024-04-17 17:46:39,668 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041304.1, seqID NZ_CP041304.1 [record index 0].\n",
      "2024-04-17 17:46:39,866 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR890405.1, seqID NZ_LR890405.1 [record index 0].\n",
      "2024-04-17 17:46:40,065 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR130564.1, seqID NZ_LR130564.1 [record index 0].\n",
      "2024-04-17 17:46:40,284 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP056077.1, seqID NZ_CP056077.1 [record index 0].\n",
      "2024-04-17 17:46:40,505 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890198.1, seqID NZ_LR890198.1 [record index 0].\n",
      "2024-04-17 17:46:40,739 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019903.1, seqID NZ_CP019903.1 [record index 0].\n",
      "2024-04-17 17:46:40,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063153.1, seqID NZ_CP063153.1 [record index 0].\n",
      "2024-04-17 17:46:41,157 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031256.1, seqID NZ_CP031256.1 [record index 0].\n",
      "2024-04-17 17:46:41,354 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061337.1, seqID NZ_CP061337.1 [record index 0].\n",
      "2024-04-17 17:46:41,566 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061339.1, seqID NZ_CP061339.1 [record index 0].\n",
      "2024-04-17 17:46:41,825 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068394.1, seqID NZ_CP068394.1 [record index 0].\n",
      "2024-04-17 17:46:42,040 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082771.1, seqID NZ_CP082771.1 [record index 0].\n",
      "2024-04-17 17:46:42,247 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_011993.1, seqID NC_011993.1 [record index 0].\n",
      "2024-04-17 17:46:42,473 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091673.1, seqID NZ_CP091673.1 [record index 0].\n",
      "2024-04-17 17:46:42,713 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063046.1, seqID NZ_CP063046.1 [record index 0].\n",
      "2024-04-17 17:46:42,939 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP016358.1, seqID NZ_CP016358.1 [record index 0].\n",
      "2024-04-17 17:46:43,201 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072784.1, seqID NZ_CP072784.1 [record index 0].\n",
      "2024-04-17 17:46:43,420 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072785.1, seqID NZ_CP072785.1 [record index 0].\n",
      "2024-04-17 17:46:43,636 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072786.1, seqID NZ_CP072786.1 [record index 0].\n",
      "2024-04-17 17:46:43,878 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070227.1, seqID NZ_CP070227.1 [record index 0].\n",
      "2024-04-17 17:46:44,121 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082772.1, seqID NZ_CP082772.1 [record index 0].\n",
      "2024-04-17 17:46:44,364 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053080.1, seqID NZ_CP053080.1 [record index 0].\n",
      "2024-04-17 17:46:44,594 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048647.1, seqID NZ_CP048647.1 [record index 0].\n",
      "2024-04-17 17:46:44,785 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053258.1, seqID NZ_CP053258.1 [record index 0].\n",
      "2024-04-17 17:46:45,034 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060075.1, seqID NZ_CP060075.1 [record index 0].\n",
      "2024-04-17 17:46:45,192 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP019455.1, seqID NZ_CP019455.1 [record index 0].\n",
      "2024-04-17 17:46:45,413 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP065152.1, seqID NZ_CP065152.1 [record index 0].\n",
      "2024-04-17 17:46:45,613 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP033762.1, seqID NZ_CP033762.1 [record index 0].\n",
      "2024-04-17 17:46:45,849 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023960.1, seqID NZ_CP023960.1 [record index 0].\n",
      "2024-04-17 17:46:46,076 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014111.1, seqID NZ_CP014111.1 [record index 0].\n",
      "2024-04-17 17:46:46,287 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069945.1, seqID NZ_CP069945.1 [record index 0].\n",
      "2024-04-17 17:46:46,477 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070063.1, seqID NZ_CP070063.1 [record index 0].\n",
      "2024-04-17 17:46:46,696 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070077.1, seqID NZ_CP070077.1 [record index 0].\n",
      "2024-04-17 17:46:46,930 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070098.1, seqID NZ_CP070098.1 [record index 0].\n",
      "2024-04-17 17:46:47,158 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070041.1, seqID NZ_CP070041.1 [record index 0].\n",
      "2024-04-17 17:46:47,364 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069980.1, seqID NZ_CP069980.1 [record index 0].\n",
      "2024-04-17 17:46:47,634 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP070048.1, seqID NZ_CP070048.1 [record index 0].\n",
      "2024-04-17 17:46:47,855 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069882.1, seqID NZ_CP069882.1 [record index 0].\n",
      "2024-04-17 17:46:48,089 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070007.1, seqID NZ_CP070007.1 [record index 0].\n",
      "2024-04-17 17:46:48,330 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070071.1, seqID NZ_CP070071.1 [record index 0].\n",
      "2024-04-17 17:46:48,578 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069893.1, seqID NZ_CP069893.1 [record index 0].\n",
      "2024-04-17 17:46:48,823 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070170.1, seqID NZ_CP070170.1 [record index 0].\n",
      "2024-04-17 17:46:49,127 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP070162.1, seqID NZ_CP070162.1 [record index 0].\n",
      "2024-04-17 17:46:49,350 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069865.1, seqID NZ_CP069865.1 [record index 0].\n",
      "2024-04-17 17:46:49,587 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070134.1, seqID NZ_CP070134.1 [record index 0].\n",
      "2024-04-17 17:46:49,823 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069973.1, seqID NZ_CP069973.1 [record index 0].\n",
      "2024-04-17 17:46:50,054 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069957.1, seqID NZ_CP069957.1 [record index 0].\n",
      "2024-04-17 17:46:50,290 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069522.1, seqID NZ_CP069522.1 [record index 0].\n",
      "2024-04-17 17:46:50,546 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069517.1, seqID NZ_CP069517.1 [record index 0].\n",
      "2024-04-17 17:46:50,776 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069441.1, seqID NZ_CP069441.1 [record index 0].\n",
      "2024-04-17 17:46:50,981 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069438.1, seqID NZ_CP069438.1 [record index 0].\n",
      "2024-04-17 17:46:51,189 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069559.1, seqID NZ_CP069559.1 [record index 0].\n",
      "2024-04-17 17:46:51,414 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069500.1, seqID NZ_CP069500.1 [record index 0].\n",
      "2024-04-17 17:46:51,643 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069480.1, seqID NZ_CP069480.1 [record index 0].\n",
      "2024-04-17 17:46:51,869 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069451.1, seqID NZ_CP069451.1 [record index 0].\n",
      "2024-04-17 17:46:52,090 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069447.1, seqID NZ_CP069447.1 [record index 0].\n",
      "2024-04-17 17:46:52,281 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069453.1, seqID NZ_CP069453.1 [record index 0].\n",
      "2024-04-17 17:46:52,464 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069549.1, seqID NZ_CP069549.1 [record index 0].\n",
      "2024-04-17 17:46:52,692 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069445.1, seqID NZ_CP069445.1 [record index 0].\n",
      "2024-04-17 17:46:52,910 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043542.1, seqID NZ_CP043542.1 [record index 0].\n",
      "2024-04-17 17:46:53,130 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076123.1, seqID NZ_CP076123.1 [record index 0].\n",
      "2024-04-17 17:46:53,392 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP031215.1, seqID NZ_CP031215.1 [record index 0].\n",
      "2024-04-17 17:46:53,585 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063214.1, seqID NZ_CP063214.1 [record index 0].\n",
      "2024-04-17 17:46:53,824 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083481.1, seqID NZ_CP083481.1 [record index 0].\n",
      "2024-04-17 17:46:54,059 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083503.1, seqID NZ_CP083503.1 [record index 0].\n",
      "2024-04-17 17:46:54,306 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083530.1, seqID NZ_CP083530.1 [record index 0].\n",
      "2024-04-17 17:46:54,533 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063492.1, seqID NZ_CP063492.1 [record index 0].\n",
      "2024-04-17 17:46:54,763 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP065136.1, seqID NZ_CP065136.1 [record index 0].\n",
      "2024-04-17 17:46:54,998 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054236.1, seqID NZ_CP054236.1 [record index 0].\n",
      "2024-04-17 17:46:55,190 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054227.1, seqID NZ_CP054227.1 [record index 0].\n",
      "2024-04-17 17:46:55,426 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054230.1, seqID NZ_CP054230.1 [record index 0].\n",
      "2024-04-17 17:46:55,668 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014667.1, seqID NZ_CP014667.1 [record index 0].\n",
      "2024-04-17 17:46:55,876 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP056036.1, seqID NZ_CP056036.1 [record index 0].\n",
      "2024-04-17 17:46:56,104 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049101.1, seqID NZ_CP049101.1 [record index 0].\n",
      "2024-04-17 17:46:56,375 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058574.1, seqID NZ_CP058574.1 [record index 0].\n",
      "2024-04-17 17:46:56,606 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP066836.1, seqID NZ_CP066836.1 [record index 0].\n",
      "2024-04-17 17:46:56,838 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086334.1, seqID NZ_CP086334.1 [record index 0].\n",
      "2024-04-17 17:46:57,052 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049198.1, seqID NZ_CP049198.1 [record index 0].\n",
      "2024-04-17 17:46:57,295 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049197.1, seqID NZ_CP049197.1 [record index 0].\n",
      "2024-04-17 17:46:57,562 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032066.1, seqID NZ_CP032066.1 [record index 0].\n",
      "2024-04-17 17:46:57,793 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049196.1, seqID NZ_CP049196.1 [record index 0].\n",
      "2024-04-17 17:46:58,020 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP071375.1, seqID NZ_CP071375.1 [record index 0].\n",
      "2024-04-17 17:46:58,239 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP090611.1, seqID NZ_CP090611.1 [record index 0].\n",
      "2024-04-17 17:46:58,492 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP061206.1, seqID NZ_CP061206.1 [record index 0].\n",
      "2024-04-17 17:46:58,728 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029579.1, seqID NZ_CP029579.1 [record index 0].\n",
      "2024-04-17 17:46:58,967 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010151.1, seqID NZ_CP010151.1 [record index 0].\n",
      "2024-04-17 17:46:59,180 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010157.1, seqID NZ_CP010157.1 [record index 0].\n",
      "2024-04-17 17:46:59,395 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082776.1, seqID NZ_CP082776.1 [record index 0].\n",
      "2024-04-17 17:46:59,638 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP082777.1, seqID NZ_CP082777.1 [record index 0].\n",
      "2024-04-17 17:46:59,924 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP082774.1, seqID NZ_CP082774.1 [record index 0].\n",
      "2024-04-17 17:47:00,108 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP032078.1, seqID NZ_CP032078.1 [record index 0].\n",
      "2024-04-17 17:47:00,306 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP074576.1, seqID NZ_CP074576.1 [record index 0].\n",
      "2024-04-17 17:47:00,583 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019243.1, seqID NZ_CP019243.1 [record index 0].\n",
      "2024-04-17 17:47:00,802 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017651.1, seqID NC_017651.1 [record index 0].\n",
      "2024-04-17 17:47:01,040 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017652.1, seqID NC_017652.1 [record index 0].\n",
      "2024-04-17 17:47:01,250 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048603.1, seqID NZ_CP048603.1 [record index 0].\n",
      "2024-04-17 17:47:01,482 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058222.1, seqID NZ_CP058222.1 [record index 0].\n",
      "2024-04-17 17:47:01,703 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051263.1, seqID NZ_CP051263.1 [record index 0].\n",
      "2024-04-17 17:47:01,930 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_004431.1, seqID NC_004431.1 [record index 0].\n",
      "2024-04-17 17:47:02,176 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP028320.1, seqID NZ_CP028320.1 [record index 0].\n",
      "2024-04-17 17:47:02,396 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023142.1, seqID NZ_CP023142.1 [record index 0].\n",
      "2024-04-17 17:47:02,589 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010121.1, seqID NZ_CP010121.1 [record index 0].\n",
      "2024-04-17 17:47:02,817 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025251.1, seqID NZ_CP025251.1 [record index 0].\n",
      "2024-04-17 17:47:03,039 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024650.2, seqID NZ_CP024650.2 [record index 0].\n",
      "2024-04-17 17:47:03,200 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP025703.1, seqID NZ_CP025703.1 [record index 0].\n",
      "2024-04-17 17:47:03,418 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025716.1, seqID NZ_CP025716.1 [record index 0].\n",
      "2024-04-17 17:47:03,650 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032085.1, seqID NZ_CP032085.1 [record index 0].\n",
      "2024-04-17 17:47:03,878 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP037449.1, seqID NZ_CP037449.1 [record index 0].\n",
      "2024-04-17 17:47:04,102 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP009072.1, seqID NZ_CP009072.1 [record index 0].\n",
      "2024-04-17 17:47:04,337 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027134.1, seqID NZ_CP027134.1 [record index 0].\n",
      "2024-04-17 17:47:04,572 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032261.1, seqID NZ_CP032261.1 [record index 0].\n",
      "2024-04-17 17:47:04,863 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP024886.1, seqID NZ_CP024886.1 [record index 0].\n",
      "2024-04-17 17:47:05,136 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024862.1, seqID NZ_CP024862.1 [record index 0].\n",
      "2024-04-17 17:47:05,360 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024851.1, seqID NZ_CP024851.1 [record index 0].\n",
      "2024-04-17 17:47:05,624 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP006834.2, seqID NZ_CP006834.2 [record index 0].\n",
      "2024-04-17 17:47:05,865 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_017631.1, seqID NC_017631.1 [record index 0].\n",
      "2024-04-17 17:47:06,067 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP028768.1, seqID NZ_CP028768.1 [record index 0].\n",
      "2024-04-17 17:47:06,275 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028734.1, seqID NZ_CP028734.1 [record index 0].\n",
      "2024-04-17 17:47:06,514 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028770.1, seqID NZ_CP028770.1 [record index 0].\n",
      "2024-04-17 17:47:06,728 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP029327.1, seqID NZ_CP029327.1 [record index 0].\n",
      "2024-04-17 17:47:06,980 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028742.1, seqID NZ_CP028742.1 [record index 0].\n",
      "2024-04-17 17:47:07,214 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028771.1, seqID NZ_CP028771.1 [record index 0].\n",
      "2024-04-17 17:47:07,448 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028749.1, seqID NZ_CP028749.1 [record index 0].\n",
      "2024-04-17 17:47:07,677 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028750.1, seqID NZ_CP028750.1 [record index 0].\n",
      "2024-04-17 17:47:07,926 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028751.1, seqID NZ_CP028751.1 [record index 0].\n",
      "2024-04-17 17:47:08,163 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028754.1, seqID NZ_CP028754.1 [record index 0].\n",
      "2024-04-17 17:47:08,366 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028761.1, seqID NZ_CP028761.1 [record index 0].\n",
      "2024-04-17 17:47:08,587 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028764.1, seqID NZ_CP028764.1 [record index 0].\n",
      "2024-04-17 17:47:08,786 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027461.1, seqID NZ_CP027461.1 [record index 0].\n",
      "2024-04-17 17:47:08,968 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP048304.1, seqID NZ_CP048304.1 [record index 0].\n",
      "2024-04-17 17:47:09,212 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP058220.1, seqID NZ_CP058220.1 [record index 0].\n",
      "2024-04-17 17:47:09,446 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023820.1, seqID NZ_CP023820.1 [record index 0].\n",
      "2024-04-17 17:47:09,719 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP059917.1, seqID NZ_CP059917.1 [record index 0].\n",
      "2024-04-17 17:47:09,924 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058223.1, seqID NZ_CP058223.1 [record index 0].\n",
      "2024-04-17 17:47:10,134 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_008253.1, seqID NC_008253.1 [record index 0].\n",
      "2024-04-17 17:47:10,375 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP033884.1, seqID NZ_CP033884.1 [record index 0].\n",
      "2024-04-17 17:47:10,590 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP071073.1, seqID NZ_CP071073.1 [record index 0].\n",
      "2024-04-17 17:47:10,810 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023357.1, seqID NZ_CP023357.1 [record index 0].\n",
      "2024-04-17 17:47:11,035 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081010.1, seqID NZ_CP081010.1 [record index 0].\n",
      "2024-04-17 17:47:11,260 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063983.1, seqID NZ_CP063983.1 [record index 0].\n",
      "2024-04-17 17:47:11,493 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020516.1, seqID NZ_CP020516.1 [record index 0].\n",
      "2024-04-17 17:47:11,697 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP017610.1, seqID NZ_AP017610.1 [record index 0].\n",
      "2024-04-17 17:47:11,906 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP030768.1, seqID NZ_CP030768.1 [record index 0].\n",
      "2024-04-17 17:47:12,097 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP025214.2, seqID NZ_AP025214.2 [record index 0].\n",
      "2024-04-17 17:47:12,288 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027452.1, seqID NZ_CP027452.1 [record index 0].\n",
      "2024-04-17 17:47:12,486 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027440.1, seqID NZ_CP027440.1 [record index 0].\n",
      "2024-04-17 17:47:12,756 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013025.1, seqID NZ_CP013025.1 [record index 0].\n",
      "2024-04-17 17:47:12,979 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP059907.1, seqID NZ_CP059907.1 [record index 0].\n",
      "2024-04-17 17:47:13,182 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024138.1, seqID NZ_CP024138.1 [record index 0].\n",
      "2024-04-17 17:47:13,440 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019273.1, seqID NZ_CP019273.1 [record index 0].\n",
      "2024-04-17 17:47:13,627 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_HG994851.1, seqID NZ_HG994851.1 [record index 0].\n",
      "2024-04-17 17:47:13,854 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_HG994850.1, seqID NZ_HG994850.1 [record index 0].\n",
      "2024-04-17 17:47:14,065 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP032515.1, seqID NZ_CP032515.1 [record index 0].\n",
      "2024-04-17 17:47:14,267 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023388.1, seqID NZ_CP023388.1 [record index 0].\n",
      "2024-04-17 17:47:14,485 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029164.1, seqID NZ_CP029164.1 [record index 0].\n",
      "2024-04-17 17:47:14,723 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020495.1, seqID NZ_CP020495.1 [record index 0].\n",
      "2024-04-17 17:47:14,960 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035498.1, seqID NZ_CP035498.1 [record index 0].\n",
      "2024-04-17 17:47:15,177 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041620.1, seqID NZ_CP041620.1 [record index 0].\n",
      "2024-04-17 17:47:15,402 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020545.1, seqID NZ_CP020545.1 [record index 0].\n",
      "2024-04-17 17:47:15,633 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014497.1, seqID NZ_CP014497.1 [record index 0].\n",
      "2024-04-17 17:47:15,860 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014522.1, seqID NZ_CP014522.1 [record index 0].\n",
      "2024-04-17 17:47:16,160 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062752.1, seqID NZ_CP062752.1 [record index 0].\n",
      "2024-04-17 17:47:16,459 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062755.1, seqID NZ_CP062755.1 [record index 0].\n",
      "2024-04-17 17:47:16,757 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062758.1, seqID NZ_CP062758.1 [record index 0].\n",
      "2024-04-17 17:47:17,062 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062761.1, seqID NZ_CP062761.1 [record index 0].\n",
      "2024-04-17 17:47:17,389 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062763.1, seqID NZ_CP062763.1 [record index 0].\n",
      "2024-04-17 17:47:17,697 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062782.1, seqID NZ_CP062782.1 [record index 0].\n",
      "2024-04-17 17:47:17,998 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062766.1, seqID NZ_CP062766.1 [record index 0].\n",
      "2024-04-17 17:47:18,287 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062769.1, seqID NZ_CP062769.1 [record index 0].\n",
      "2024-04-17 17:47:18,584 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062771.1, seqID NZ_CP062771.1 [record index 0].\n",
      "2024-04-17 17:47:18,877 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062774.1, seqID NZ_CP062774.1 [record index 0].\n",
      "2024-04-17 17:47:19,179 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062778.1, seqID NZ_CP062778.1 [record index 0].\n",
      "2024-04-17 17:47:19,474 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062700.1, seqID NZ_CP062700.1 [record index 0].\n",
      "2024-04-17 17:47:19,769 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062702.1, seqID NZ_CP062702.1 [record index 0].\n",
      "2024-04-17 17:47:20,069 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062705.1, seqID NZ_CP062705.1 [record index 0].\n",
      "2024-04-17 17:47:20,369 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062708.1, seqID NZ_CP062708.1 [record index 0].\n",
      "2024-04-17 17:47:20,668 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062711.1, seqID NZ_CP062711.1 [record index 0].\n",
      "2024-04-17 17:47:20,960 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062713.1, seqID NZ_CP062713.1 [record index 0].\n",
      "2024-04-17 17:47:21,252 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062715.1, seqID NZ_CP062715.1 [record index 0].\n",
      "2024-04-17 17:47:21,551 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062717.1, seqID NZ_CP062717.1 [record index 0].\n",
      "2024-04-17 17:47:21,861 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062719.1, seqID NZ_CP062719.1 [record index 0].\n",
      "2024-04-17 17:47:22,164 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062721.1, seqID NZ_CP062721.1 [record index 0].\n",
      "2024-04-17 17:47:22,462 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062723.1, seqID NZ_CP062723.1 [record index 0].\n",
      "2024-04-17 17:47:22,764 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062725.1, seqID NZ_CP062725.1 [record index 0].\n",
      "2024-04-17 17:47:23,058 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062727.1, seqID NZ_CP062727.1 [record index 0].\n",
      "2024-04-17 17:47:23,355 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062729.1, seqID NZ_CP062729.1 [record index 0].\n",
      "2024-04-17 17:47:23,653 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062780.1, seqID NZ_CP062780.1 [record index 0].\n",
      "2024-04-17 17:47:23,951 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062731.1, seqID NZ_CP062731.1 [record index 0].\n",
      "2024-04-17 17:47:24,243 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062733.1, seqID NZ_CP062733.1 [record index 0].\n",
      "2024-04-17 17:47:24,541 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062736.1, seqID NZ_CP062736.1 [record index 0].\n",
      "2024-04-17 17:47:24,881 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062739.1, seqID NZ_CP062739.1 [record index 0].\n",
      "2024-04-17 17:47:25,180 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062742.1, seqID NZ_CP062742.1 [record index 0].\n",
      "2024-04-17 17:47:25,486 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062744.1, seqID NZ_CP062744.1 [record index 0].\n",
      "2024-04-17 17:47:25,779 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062746.1, seqID NZ_CP062746.1 [record index 0].\n",
      "2024-04-17 17:47:26,076 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062749.1, seqID NZ_CP062749.1 [record index 0].\n",
      "2024-04-17 17:47:26,369 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038284.1, seqID NZ_CP038284.1 [record index 0].\n",
      "2024-04-17 17:47:26,656 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NC_017906.1, seqID NC_017906.1 [record index 0].\n",
      "2024-04-17 17:47:26,968 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP012802.1, seqID NZ_CP012802.1 [record index 0].\n",
      "2024-04-17 17:47:27,258 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_AP022171.1, seqID NZ_AP022171.1 [record index 0].\n",
      "2024-04-17 17:47:27,483 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022064.1, seqID NZ_AP022064.1 [record index 0].\n",
      "2024-04-17 17:47:27,706 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022087.1, seqID NZ_AP022087.1 [record index 0].\n",
      "2024-04-17 17:47:27,934 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022044.1, seqID NZ_AP022044.1 [record index 0].\n",
      "2024-04-17 17:47:28,158 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021998.1, seqID NZ_AP021998.1 [record index 0].\n",
      "2024-04-17 17:47:28,386 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021963.1, seqID NZ_AP021963.1 [record index 0].\n",
      "2024-04-17 17:47:28,606 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022032.1, seqID NZ_AP022032.1 [record index 0].\n",
      "2024-04-17 17:47:28,903 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP064167.1, seqID NZ_CP064167.1 [record index 0].\n",
      "2024-04-17 17:47:29,149 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076241.1, seqID NZ_CP076241.1 [record index 0].\n",
      "2024-04-17 17:47:29,382 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP064825.1, seqID NZ_CP064825.1 [record index 0].\n",
      "2024-04-17 17:47:29,615 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP062985.1, seqID NZ_CP062985.1 [record index 0].\n",
      "2024-04-17 17:47:29,862 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP062228.1, seqID NZ_CP062228.1 [record index 0].\n",
      "2024-04-17 17:47:30,107 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_007946.1, seqID NC_007946.1 [record index 0].\n",
      "2024-04-17 17:47:30,335 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP071522.1, seqID NZ_CP071522.1 [record index 0].\n",
      "2024-04-17 17:47:30,560 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049852.1, seqID NZ_CP049852.1 [record index 0].\n",
      "2024-04-17 17:47:30,786 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049839.1, seqID NZ_CP049839.1 [record index 0].\n",
      "2024-04-17 17:47:31,025 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_017632.1, seqID NC_017632.1 [record index 0].\n",
      "2024-04-17 17:47:31,287 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013658.1, seqID NZ_CP013658.1 [record index 0].\n",
      "2024-04-17 17:47:31,513 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035720.1, seqID NZ_CP035720.1 [record index 0].\n",
      "2024-04-17 17:47:31,741 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035516.1, seqID NZ_CP035516.1 [record index 0].\n",
      "2024-04-17 17:47:31,978 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035477.1, seqID NZ_CP035477.1 [record index 0].\n",
      "2024-04-17 17:47:32,220 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035476.1, seqID NZ_CP035476.1 [record index 0].\n",
      "2024-04-17 17:47:32,538 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038287.1, seqID NZ_CP038287.1 [record index 0].\n",
      "2024-04-17 17:47:32,845 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038290.1, seqID NZ_CP038290.1 [record index 0].\n",
      "2024-04-17 17:47:33,142 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NC_013008.1, seqID NC_013008.1 [record index 0].\n",
      "2024-04-17 17:47:33,426 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038494.1, seqID NZ_CP038494.1 [record index 0].\n",
      "2024-04-17 17:47:33,719 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038496.1, seqID NZ_CP038496.1 [record index 0].\n",
      "2024-04-17 17:47:33,947 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022549.1, seqID NZ_AP022549.1 [record index 0].\n",
      "2024-04-17 17:47:34,180 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022525.1, seqID NZ_AP022525.1 [record index 0].\n",
      "2024-04-17 17:47:34,476 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038292.1, seqID NZ_CP038292.1 [record index 0].\n",
      "2024-04-17 17:47:34,702 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021207.1, seqID NZ_CP021207.1 [record index 0].\n",
      "2024-04-17 17:47:34,937 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043950.1, seqID NZ_CP043950.1 [record index 0].\n",
      "2024-04-17 17:47:35,158 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063774.1, seqID NZ_CP063774.1 [record index 0].\n",
      "2024-04-17 17:47:35,457 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038300.1, seqID NZ_CP038300.1 [record index 0].\n",
      "2024-04-17 17:47:35,760 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038302.1, seqID NZ_CP038302.1 [record index 0].\n",
      "2024-04-17 17:47:36,058 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038305.1, seqID NZ_CP038305.1 [record index 0].\n",
      "2024-04-17 17:47:36,350 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP010304.1, seqID NZ_CP010304.1 [record index 0].\n",
      "2024-04-17 17:47:36,646 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP008805.1, seqID NZ_CP008805.1 [record index 0].\n",
      "2024-04-17 17:47:36,946 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015023.1, seqID NZ_CP015023.1 [record index 0].\n",
      "2024-04-17 17:47:37,170 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP018784.1, seqID NZ_AP018784.1 [record index 0].\n",
      "2024-04-17 17:47:37,474 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038309.1, seqID NZ_CP038309.1 [record index 0].\n",
      "2024-04-17 17:47:37,710 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012625.1, seqID NZ_CP012625.1 [record index 0].\n",
      "2024-04-17 17:47:37,956 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP012631.1, seqID NZ_CP012631.1 [record index 0].\n",
      "2024-04-17 17:47:38,176 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012633.1, seqID NZ_CP012633.1 [record index 0].\n",
      "2024-04-17 17:47:38,403 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP012635.1, seqID NZ_CP012635.1 [record index 0].\n",
      "2024-04-17 17:47:38,671 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_013654.1, seqID NC_013654.1 [record index 0].\n",
      "2024-04-17 17:47:38,927 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP051688.1, seqID NZ_CP051688.1 [record index 0].\n",
      "2024-04-17 17:47:39,170 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053231.1, seqID NZ_CP053231.1 [record index 0].\n",
      "2024-04-17 17:47:39,428 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053251.2, seqID NZ_CP053251.2 [record index 0].\n",
      "2024-04-17 17:47:39,615 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP054345.1, seqID NZ_CP054345.1 [record index 0].\n",
      "2024-04-17 17:47:39,851 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051711.1, seqID NZ_CP051711.1 [record index 0].\n",
      "2024-04-17 17:47:40,068 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051714.1, seqID NZ_CP051714.1 [record index 0].\n",
      "2024-04-17 17:47:40,293 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051719.1, seqID NZ_CP051719.1 [record index 0].\n",
      "2024-04-17 17:47:40,519 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051735.1, seqID NZ_CP051735.1 [record index 0].\n",
      "2024-04-17 17:47:40,760 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051849.1, seqID NZ_CP051849.1 [record index 0].\n",
      "2024-04-17 17:47:40,999 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048920.1, seqID NZ_CP048920.1 [record index 0].\n",
      "2024-04-17 17:47:41,253 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082831.1, seqID NZ_CP082831.1 [record index 0].\n",
      "2024-04-17 17:47:41,436 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR778150.1, seqID NZ_LR778150.1 [record index 0].\n",
      "2024-04-17 17:47:41,669 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR778147.1, seqID NZ_LR778147.1 [record index 0].\n",
      "2024-04-17 17:47:41,893 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014495.1, seqID NZ_CP014495.1 [record index 0].\n",
      "2024-04-17 17:47:42,155 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NC_002695.2, seqID NC_002695.2 [record index 0].\n",
      "2024-04-17 17:47:42,378 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022730.1, seqID NZ_CP022730.1 [record index 0].\n",
      "2024-04-17 17:47:42,584 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_011742.1, seqID NC_011742.1 [record index 0].\n",
      "2024-04-17 17:47:42,823 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP036245.1, seqID NZ_CP036245.1 [record index 0].\n",
      "2024-04-17 17:47:43,051 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076687.1, seqID NZ_CP076687.1 [record index 0].\n",
      "2024-04-17 17:47:43,305 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076689.1, seqID NZ_CP076689.1 [record index 0].\n",
      "2024-04-17 17:47:43,532 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076697.1, seqID NZ_CP076697.1 [record index 0].\n",
      "2024-04-17 17:47:43,766 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076693.1, seqID NZ_CP076693.1 [record index 0].\n",
      "2024-04-17 17:47:44,063 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP046527.1, seqID NZ_CP046527.1 [record index 0].\n",
      "2024-04-17 17:47:44,270 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068802.1, seqID NZ_CP068802.1 [record index 0].\n",
      "2024-04-17 17:47:44,579 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP076230.1, seqID NZ_CP076230.1 [record index 0].\n",
      "2024-04-17 17:47:44,894 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_AP018488.1, seqID NZ_AP018488.1 [record index 0].\n",
      "2024-04-17 17:47:45,126 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042246.1, seqID NZ_CP042246.1 [record index 0].\n",
      "2024-04-17 17:47:45,336 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049204.1, seqID NZ_CP049204.1 [record index 0].\n",
      "2024-04-17 17:47:45,566 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021288.1, seqID NZ_CP021288.1 [record index 0].\n",
      "2024-04-17 17:47:45,864 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017669.1, seqID NZ_CP017669.1 [record index 0].\n",
      "2024-04-17 17:47:46,096 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049085.2, seqID NZ_CP049085.2 [record index 0].\n",
      "2024-04-17 17:47:46,335 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049077.2, seqID NZ_CP049077.2 [record index 0].\n",
      "2024-04-17 17:47:46,554 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080117.1, seqID NZ_CP080117.1 [record index 0].\n",
      "2024-04-17 17:47:46,848 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038313.1, seqID NZ_CP038313.1 [record index 0].\n",
      "2024-04-17 17:47:47,121 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015085.1, seqID NZ_CP015085.1 [record index 0].\n",
      "2024-04-17 17:47:47,361 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP007275.1, seqID NZ_CP007275.1 [record index 0].\n",
      "2024-04-17 17:47:47,655 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032789.1, seqID NZ_CP032789.1 [record index 0].\n",
      "2024-04-17 17:47:47,949 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032791.1, seqID NZ_CP032791.1 [record index 0].\n",
      "2024-04-17 17:47:48,241 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032793.1, seqID NZ_CP032793.1 [record index 0].\n",
      "2024-04-17 17:47:48,488 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP019777.1, seqID NZ_CP019777.1 [record index 0].\n",
      "2024-04-17 17:47:48,725 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024720.1, seqID NZ_CP024720.1 [record index 0].\n",
      "2024-04-17 17:47:49,031 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038324.1, seqID NZ_CP038324.1 [record index 0].\n",
      "2024-04-17 17:47:49,363 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038328.1, seqID NZ_CP038328.1 [record index 0].\n",
      "2024-04-17 17:47:49,651 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038316.1, seqID NZ_CP038316.1 [record index 0].\n",
      "2024-04-17 17:47:49,944 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038319.1, seqID NZ_CP038319.1 [record index 0].\n",
      "2024-04-17 17:47:50,249 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038321.1, seqID NZ_CP038321.1 [record index 0].\n",
      "2024-04-17 17:47:50,471 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134031.1, seqID NZ_LR134031.1 [record index 0].\n",
      "2024-04-17 17:47:50,792 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017249.1, seqID NZ_CP017249.1 [record index 0].\n",
      "2024-04-17 17:47:51,112 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017251.1, seqID NZ_CP017251.1 [record index 0].\n",
      "2024-04-17 17:47:51,409 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038333.1, seqID NZ_CP038333.1 [record index 0].\n",
      "2024-04-17 17:47:51,596 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031609.1, seqID NZ_CP031609.1 [record index 0].\n",
      "2024-04-17 17:47:51,765 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_LR890410.1, seqID NZ_LR890410.1 [record index 0].\n",
      "2024-04-17 17:47:51,989 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890693.1, seqID NZ_LR890693.1 [record index 0].\n",
      "2024-04-17 17:47:52,228 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025401.1, seqID NZ_CP025401.1 [record index 0].\n",
      "2024-04-17 17:47:52,437 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026853.1, seqID NZ_CP026853.1 [record index 0].\n",
      "2024-04-17 17:47:52,668 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010876.1, seqID NZ_CP010876.1 [record index 0].\n",
      "2024-04-17 17:47:52,889 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP030111.1, seqID NZ_CP030111.1 [record index 0].\n",
      "2024-04-17 17:47:53,187 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040107.1, seqID NZ_CP040107.1 [record index 0].\n",
      "2024-04-17 17:47:53,486 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP039834.1, seqID NZ_CP039834.1 [record index 0].\n",
      "2024-04-17 17:47:53,789 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040314.1, seqID NZ_CP040314.1 [record index 0].\n",
      "2024-04-17 17:47:54,084 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040313.1, seqID NZ_CP040313.1 [record index 0].\n",
      "2024-04-17 17:47:54,307 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080118.1, seqID NZ_CP080118.1 [record index 0].\n",
      "2024-04-17 17:47:54,536 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080119.1, seqID NZ_CP080119.1 [record index 0].\n",
      "2024-04-17 17:47:54,766 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080120.1, seqID NZ_CP080120.1 [record index 0].\n",
      "2024-04-17 17:47:55,014 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053296.1, seqID NZ_CP053296.1 [record index 0].\n",
      "2024-04-17 17:47:55,336 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089270.1, seqID NZ_CP089270.1 [record index 0].\n",
      "2024-04-17 17:47:55,640 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089272.1, seqID NZ_CP089272.1 [record index 0].\n",
      "2024-04-17 17:47:55,941 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038336.1, seqID NZ_CP038336.1 [record index 0].\n",
      "2024-04-17 17:47:56,189 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024717.1, seqID NZ_CP024717.1 [record index 0].\n",
      "2024-04-17 17:47:56,391 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP091682.1, seqID NZ_CP091682.1 [record index 0].\n",
      "2024-04-17 17:47:56,612 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP053079.1, seqID NZ_CP053079.1 [record index 0].\n",
      "2024-04-17 17:47:56,873 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013835.1, seqID NZ_CP013835.1 [record index 0].\n",
      "2024-04-17 17:47:57,109 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP014316.1, seqID NZ_CP014316.1 [record index 0].\n",
      "2024-04-17 17:47:57,341 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_022648.1, seqID NC_022648.1 [record index 0].\n",
      "2024-04-17 17:47:57,638 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP014314.1, seqID NZ_CP014314.1 [record index 0].\n",
      "2024-04-17 17:47:57,883 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_017628.1, seqID NC_017628.1 [record index 0].\n",
      "2024-04-17 17:47:58,119 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026491.1, seqID NZ_CP026491.1 [record index 0].\n",
      "2024-04-17 17:47:58,416 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040305.1, seqID NZ_CP040305.1 [record index 0].\n",
      "2024-04-17 17:47:58,711 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038339.1, seqID NZ_CP038339.1 [record index 0].\n",
      "2024-04-17 17:47:59,009 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038342.1, seqID NZ_CP038342.1 [record index 0].\n",
      "2024-04-17 17:47:59,257 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021454.1, seqID NZ_CP021454.1 [record index 0].\n",
      "2024-04-17 17:47:59,543 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038344.1, seqID NZ_CP038344.1 [record index 0].\n",
      "2024-04-17 17:47:59,784 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP014488.1, seqID NZ_CP014488.1 [record index 0].\n",
      "2024-04-17 17:48:00,084 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038346.1, seqID NZ_CP038346.1 [record index 0].\n",
      "2024-04-17 17:48:00,371 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP031913.1, seqID NZ_CP031913.1 [record index 0].\n",
      "2024-04-17 17:48:00,673 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP035545.1, seqID NZ_CP035545.1 [record index 0].\n",
      "2024-04-17 17:48:00,989 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP016625.1, seqID NZ_CP016625.1 [record index 0].\n",
      "2024-04-17 17:48:01,284 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034384.1, seqID NZ_CP034384.1 [record index 0].\n",
      "2024-04-17 17:48:01,621 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015842.1, seqID NZ_CP015842.1 [record index 0].\n",
      "2024-04-17 17:48:01,922 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015843.2, seqID NZ_CP015843.2 [record index 0].\n",
      "2024-04-17 17:48:02,255 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015846.1, seqID NZ_CP015846.1 [record index 0].\n",
      "2024-04-17 17:48:02,583 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP016755.1, seqID NZ_CP016755.1 [record index 0].\n",
      "2024-04-17 17:48:02,879 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP022050.2, seqID NZ_CP022050.2 [record index 0].\n",
      "2024-04-17 17:48:03,109 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069990.1, seqID NZ_CP069990.1 [record index 0].\n",
      "2024-04-17 17:48:03,364 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP070045.1, seqID NZ_CP070045.1 [record index 0].\n",
      "2024-04-17 17:48:03,589 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069935.1, seqID NZ_CP069935.1 [record index 0].\n",
      "2024-04-17 17:48:03,815 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070111.1, seqID NZ_CP070111.1 [record index 0].\n",
      "2024-04-17 17:48:04,031 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069996.1, seqID NZ_CP069996.1 [record index 0].\n",
      "2024-04-17 17:48:04,279 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP069978.1, seqID NZ_CP069978.1 [record index 0].\n",
      "2024-04-17 17:48:04,509 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070115.1, seqID NZ_CP070115.1 [record index 0].\n",
      "2024-04-17 17:48:04,760 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP070148.1, seqID NZ_CP070148.1 [record index 0].\n",
      "2024-04-17 17:48:04,975 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070066.1, seqID NZ_CP070066.1 [record index 0].\n",
      "2024-04-17 17:48:05,226 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP069890.1, seqID NZ_CP069890.1 [record index 0].\n",
      "2024-04-17 17:48:05,445 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070124.1, seqID NZ_CP070124.1 [record index 0].\n",
      "2024-04-17 17:48:05,670 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069589.1, seqID NZ_CP069589.1 [record index 0].\n",
      "2024-04-17 17:48:05,905 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069583.1, seqID NZ_CP069583.1 [record index 0].\n",
      "2024-04-17 17:48:06,126 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069488.1, seqID NZ_CP069488.1 [record index 0].\n",
      "2024-04-17 17:48:06,382 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP069528.1, seqID NZ_CP069528.1 [record index 0].\n",
      "2024-04-17 17:48:06,673 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038349.1, seqID NZ_CP038349.1 [record index 0].\n",
      "2024-04-17 17:48:06,977 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038351.1, seqID NZ_CP038351.1 [record index 0].\n",
      "2024-04-17 17:48:07,268 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038353.1, seqID NZ_CP038353.1 [record index 0].\n",
      "2024-04-17 17:48:07,566 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038282.1, seqID NZ_CP038282.1 [record index 0].\n",
      "2024-04-17 17:48:07,858 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038355.1, seqID NZ_CP038355.1 [record index 0].\n",
      "2024-04-17 17:48:08,154 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038357.1, seqID NZ_CP038357.1 [record index 0].\n",
      "2024-04-17 17:48:08,445 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038360.1, seqID NZ_CP038360.1 [record index 0].\n",
      "2024-04-17 17:48:08,736 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038363.1, seqID NZ_CP038363.1 [record index 0].\n",
      "2024-04-17 17:48:09,038 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038366.1, seqID NZ_CP038366.1 [record index 0].\n",
      "2024-04-17 17:48:09,353 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038369.1, seqID NZ_CP038369.1 [record index 0].\n",
      "2024-04-17 17:48:09,650 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038372.1, seqID NZ_CP038372.1 [record index 0].\n",
      "2024-04-17 17:48:09,933 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040311.1, seqID NZ_CP040311.1 [record index 0].\n",
      "2024-04-17 17:48:10,255 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038374.1, seqID NZ_CP038374.1 [record index 0].\n",
      "2024-04-17 17:48:10,552 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040307.1, seqID NZ_CP040307.1 [record index 0].\n",
      "2024-04-17 17:48:10,860 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038376.1, seqID NZ_CP038376.1 [record index 0].\n",
      "2024-04-17 17:48:11,134 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025328.1, seqID NZ_CP025328.1 [record index 0].\n",
      "2024-04-17 17:48:11,363 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS992168.1, seqID NZ_LS992168.1 [record index 0].\n",
      "2024-04-17 17:48:11,604 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS992192.1, seqID NZ_LS992192.1 [record index 0].\n",
      "2024-04-17 17:48:11,855 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS992190.1, seqID NZ_LS992190.1 [record index 0].\n",
      "2024-04-17 17:48:12,089 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS992180.1, seqID NZ_LS992180.1 [record index 0].\n",
      "2024-04-17 17:48:12,312 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070593.1, seqID NZ_CP070593.1 [record index 0].\n",
      "2024-04-17 17:48:12,620 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032795.1, seqID NZ_CP032795.1 [record index 0].\n",
      "2024-04-17 17:48:12,931 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032797.1, seqID NZ_CP032797.1 [record index 0].\n",
      "2024-04-17 17:48:13,240 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032801.1, seqID NZ_CP032801.1 [record index 0].\n",
      "2024-04-17 17:48:13,538 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032803.1, seqID NZ_CP032803.1 [record index 0].\n",
      "2024-04-17 17:48:13,827 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032805.1, seqID NZ_CP032805.1 [record index 0].\n",
      "2024-04-17 17:48:14,129 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032808.1, seqID NZ_CP032808.1 [record index 0].\n",
      "2024-04-17 17:48:14,421 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP032811.1, seqID NZ_CP032811.1 [record index 0].\n",
      "2024-04-17 17:48:14,642 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LT827011.1, seqID NZ_LT827011.1 [record index 0].\n",
      "2024-04-17 17:48:14,862 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LT903847.1, seqID NZ_LT903847.1 [record index 0].\n",
      "2024-04-17 17:48:15,105 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP083512.1, seqID NZ_CP083512.1 [record index 0].\n",
      "2024-04-17 17:48:15,399 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP063515.1, seqID NZ_CP063515.1 [record index 0].\n",
      "2024-04-17 17:48:15,696 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP008957.1, seqID NZ_CP008957.1 [record index 0].\n",
      "2024-04-17 17:48:15,924 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054232.1, seqID NZ_CP054232.1 [record index 0].\n",
      "2024-04-17 17:48:16,147 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054214.1, seqID NZ_CP054214.1 [record index 0].\n",
      "2024-04-17 17:48:16,377 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054219.1, seqID NZ_CP054219.1 [record index 0].\n",
      "2024-04-17 17:48:16,680 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP066756.1, seqID NZ_CP066756.1 [record index 0].\n",
      "2024-04-17 17:48:16,977 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP066753.1, seqID NZ_CP066753.1 [record index 0].\n",
      "2024-04-17 17:48:17,283 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP066748.1, seqID NZ_CP066748.1 [record index 0].\n",
      "2024-04-17 17:48:17,583 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP066759.1, seqID NZ_CP066759.1 [record index 0].\n",
      "2024-04-17 17:48:17,885 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040572.1, seqID NZ_CP040572.1 [record index 0].\n",
      "2024-04-17 17:48:18,177 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040570.1, seqID NZ_CP040570.1 [record index 0].\n",
      "2024-04-17 17:48:18,411 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019015.1, seqID NZ_CP019015.1 [record index 0].\n",
      "2024-04-17 17:48:18,645 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019012.1, seqID NZ_CP019012.1 [record index 0].\n",
      "2024-04-17 17:48:18,866 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019008.1, seqID NZ_CP019008.1 [record index 0].\n",
      "2024-04-17 17:48:19,093 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP019000.1, seqID NZ_CP019000.1 [record index 0].\n",
      "2024-04-17 17:48:19,346 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018991.1, seqID NZ_CP018991.1 [record index 0].\n",
      "2024-04-17 17:48:19,567 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018983.1, seqID NZ_CP018983.1 [record index 0].\n",
      "2024-04-17 17:48:19,788 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP015074.2, seqID NZ_CP015074.2 [record index 0].\n",
      "2024-04-17 17:48:20,100 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP015069.1, seqID NZ_CP015069.1 [record index 0].\n",
      "2024-04-17 17:48:20,333 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP015138.1, seqID NZ_CP015138.1 [record index 0].\n",
      "2024-04-17 17:48:20,563 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018979.1, seqID NZ_CP018979.1 [record index 0].\n",
      "2024-04-17 17:48:20,798 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018970.1, seqID NZ_CP018970.1 [record index 0].\n",
      "2024-04-17 17:48:21,092 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP015076.1, seqID NZ_CP015076.1 [record index 0].\n",
      "2024-04-17 17:48:21,313 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP018953.1, seqID NZ_CP018953.1 [record index 0].\n",
      "2024-04-17 17:48:21,568 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP019020.1, seqID NZ_CP019020.1 [record index 0].\n",
      "2024-04-17 17:48:21,892 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP015159.1, seqID NZ_CP015159.1 [record index 0].\n",
      "2024-04-17 17:48:22,166 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP047710.1, seqID NZ_CP047710.1 [record index 0].\n",
      "2024-04-17 17:48:22,394 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021840.1, seqID NZ_CP021840.1 [record index 0].\n",
      "2024-04-17 17:48:22,617 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_HG941718.1, seqID NZ_HG941718.1 [record index 0].\n",
      "2024-04-17 17:48:22,872 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060953.1, seqID NZ_CP060953.1 [record index 0].\n",
      "2024-04-17 17:48:23,105 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP059130.1, seqID NZ_CP059130.1 [record index 0].\n",
      "2024-04-17 17:48:23,436 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NC_011353.1, seqID NC_011353.1 [record index 0].\n",
      "2024-04-17 17:48:23,667 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP056114.1, seqID NZ_CP056114.1 [record index 0].\n",
      "2024-04-17 17:48:23,896 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021844.1, seqID NZ_CP021844.1 [record index 0].\n",
      "2024-04-17 17:48:24,124 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070229.1, seqID NZ_CP070229.1 [record index 0].\n",
      "2024-04-17 17:48:24,355 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LS999560.1, seqID NZ_LS999560.1 [record index 0].\n",
      "2024-04-17 17:48:24,598 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028483.1, seqID NZ_CP028483.1 [record index 0].\n",
      "2024-04-17 17:48:24,896 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038380.1, seqID NZ_CP038380.1 [record index 0].\n",
      "2024-04-17 17:48:25,138 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022362.1, seqID NZ_AP022362.1 [record index 0].\n",
      "2024-04-17 17:48:25,316 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP059840.2, seqID NZ_CP059840.2 [record index 0].\n",
      "2024-04-17 17:48:25,543 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_011601.1, seqID NC_011601.1 [record index 0].\n",
      "2024-04-17 17:48:25,813 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_AP022351.1, seqID NZ_AP022351.1 [record index 0].\n",
      "2024-04-17 17:48:26,033 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080116.1, seqID NZ_CP080116.1 [record index 0].\n",
      "2024-04-17 17:48:26,337 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038398.1, seqID NZ_CP038398.1 [record index 0].\n",
      "2024-04-17 17:48:26,626 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP029576.1, seqID NZ_CP029576.1 [record index 0].\n",
      "2024-04-17 17:48:26,917 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040316.1, seqID NZ_CP040316.1 [record index 0].\n",
      "2024-04-17 17:48:27,138 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082775.1, seqID NZ_CP082775.1 [record index 0].\n",
      "2024-04-17 17:48:27,361 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076709.1, seqID NZ_CP076709.1 [record index 0].\n",
      "2024-04-17 17:48:27,646 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP022689.1, seqID NZ_CP022689.1 [record index 0].\n",
      "2024-04-17 17:48:27,939 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013831.1, seqID NZ_CP013831.1 [record index 0].\n",
      "2024-04-17 17:48:28,172 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041337.1, seqID NZ_CP041337.1 [record index 0].\n",
      "2024-04-17 17:48:28,468 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP059281.1, seqID NZ_CP059281.1 [record index 0].\n",
      "2024-04-17 17:48:28,751 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP059279.1, seqID NZ_CP059279.1 [record index 0].\n",
      "2024-04-17 17:48:29,046 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP035366.1, seqID NZ_CP035366.1 [record index 0].\n",
      "2024-04-17 17:48:29,281 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR740758.1, seqID NZ_LR740758.1 [record index 0].\n",
      "2024-04-17 17:48:29,510 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR740776.1, seqID NZ_LR740776.1 [record index 0].\n",
      "2024-04-17 17:48:29,740 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042250.1, seqID NZ_CP042250.1 [record index 0].\n",
      "2024-04-17 17:48:30,045 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038402.1, seqID NZ_CP038402.1 [record index 0].\n",
      "2024-04-17 17:48:30,358 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP076245.1, seqID NZ_CP076245.1 [record index 0].\n",
      "2024-04-17 17:48:30,662 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP076243.1, seqID NZ_CP076243.1 [record index 0].\n",
      "2024-04-17 17:48:30,882 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR130545.1, seqID NZ_LR130545.1 [record index 0].\n",
      "2024-04-17 17:48:31,111 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076344.1, seqID NZ_CP076344.1 [record index 0].\n",
      "2024-04-17 17:48:31,423 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP045975.1, seqID NZ_CP045975.1 [record index 0].\n",
      "2024-04-17 17:48:31,702 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP045863.1, seqID NZ_CP045863.1 [record index 0].\n",
      "2024-04-17 17:48:32,031 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015853.1, seqID NZ_CP015853.1 [record index 0].\n",
      "2024-04-17 17:48:32,316 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP041623.1, seqID NZ_CP041623.1 [record index 0].\n",
      "2024-04-17 17:48:32,605 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038405.1, seqID NZ_CP038405.1 [record index 0].\n",
      "2024-04-17 17:48:32,824 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP033092.2, seqID NZ_CP033092.2 [record index 0].\n",
      "2024-04-17 17:48:33,044 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP030337.1, seqID NZ_CP030337.1 [record index 0].\n",
      "2024-04-17 17:48:33,269 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020116.1, seqID NZ_CP020116.1 [record index 0].\n",
      "2024-04-17 17:48:33,492 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032265.1, seqID NZ_CP032265.1 [record index 0].\n",
      "2024-04-17 17:48:33,721 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032201.1, seqID NZ_CP032201.1 [record index 0].\n",
      "2024-04-17 17:48:33,865 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP027534.1, seqID NZ_CP027534.1 [record index 0].\n",
      "2024-04-17 17:48:34,089 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021689.1, seqID NZ_CP021689.1 [record index 0].\n",
      "2024-04-17 17:48:34,329 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021935.1, seqID NZ_CP021935.1 [record index 0].\n",
      "2024-04-17 17:48:34,622 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP044140.1, seqID NZ_CP044140.1 [record index 0].\n",
      "2024-04-17 17:48:34,906 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP044143.1, seqID NZ_CP044143.1 [record index 0].\n",
      "2024-04-17 17:48:35,195 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP044145.1, seqID NZ_CP044145.1 [record index 0].\n",
      "2024-04-17 17:48:35,496 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP044148.1, seqID NZ_CP044148.1 [record index 0].\n",
      "2024-04-17 17:48:35,734 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP006830.1, seqID NZ_CP006830.1 [record index 0].\n",
      "2024-04-17 17:48:35,964 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_008563.1, seqID NC_008563.1 [record index 0].\n",
      "2024-04-17 17:48:36,250 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP043539.1, seqID NZ_CP043539.1 [record index 0].\n",
      "2024-04-17 17:48:36,469 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028741.1, seqID NZ_CP028741.1 [record index 0].\n",
      "2024-04-17 17:48:36,703 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028758.1, seqID NZ_CP028758.1 [record index 0].\n",
      "2024-04-17 17:48:37,040 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017446.1, seqID NZ_CP017446.1 [record index 0].\n",
      "2024-04-17 17:48:37,341 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018252.2, seqID NZ_CP018252.2 [record index 0].\n",
      "2024-04-17 17:48:37,632 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP076235.1, seqID NZ_CP076235.1 [record index 0].\n",
      "2024-04-17 17:48:37,844 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027457.1, seqID NZ_CP027457.1 [record index 0].\n",
      "2024-04-17 17:48:38,134 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038408.1, seqID NZ_CP038408.1 [record index 0].\n",
      "2024-04-17 17:48:38,450 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017444.1, seqID NZ_CP017444.1 [record index 0].\n",
      "2024-04-17 17:48:38,677 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP021179.1, seqID NZ_CP021179.1 [record index 0].\n",
      "2024-04-17 17:48:38,960 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018247.1, seqID NZ_CP018247.1 [record index 0].\n",
      "2024-04-17 17:48:39,257 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038421.1, seqID NZ_CP038421.1 [record index 0].\n",
      "2024-04-17 17:48:39,559 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038423.1, seqID NZ_CP038423.1 [record index 0].\n",
      "2024-04-17 17:48:39,857 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089036.1, seqID NZ_CP089036.1 [record index 0].\n",
      "2024-04-17 17:48:40,142 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089031.1, seqID NZ_CP089031.1 [record index 0].\n",
      "2024-04-17 17:48:40,373 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP076232.1, seqID NZ_CP076232.1 [record index 0].\n",
      "2024-04-17 17:48:40,676 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089032.1, seqID NZ_CP089032.1 [record index 0].\n",
      "2024-04-17 17:48:40,984 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP089034.1, seqID NZ_CP089034.1 [record index 0].\n",
      "2024-04-17 17:48:41,344 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP015831.1, seqID NZ_CP015831.1 [record index 0].\n",
      "2024-04-17 17:48:41,635 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038428.1, seqID NZ_CP038428.1 [record index 0].\n",
      "2024-04-17 17:48:41,862 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP089445.1, seqID NZ_CP089445.1 [record index 0].\n",
      "2024-04-17 17:48:42,104 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023826.1, seqID NZ_CP023826.1 [record index 0].\n",
      "2024-04-17 17:48:42,340 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023844.1, seqID NZ_CP023844.1 [record index 0].\n",
      "2024-04-17 17:48:42,593 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023849.1, seqID NZ_CP023849.1 [record index 0].\n",
      "2024-04-17 17:48:42,884 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038412.1, seqID NZ_CP038412.1 [record index 0].\n",
      "2024-04-17 17:48:43,184 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018245.1, seqID NZ_CP018245.1 [record index 0].\n",
      "2024-04-17 17:48:43,508 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017442.1, seqID NZ_CP017442.1 [record index 0].\n",
      "2024-04-17 17:48:43,720 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP039403.1, seqID NZ_CP039403.1 [record index 0].\n",
      "2024-04-17 17:48:43,944 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP039404.1, seqID NZ_CP039404.1 [record index 0].\n",
      "2024-04-17 17:48:44,230 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018243.1, seqID NZ_CP018243.1 [record index 0].\n",
      "2024-04-17 17:48:44,453 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029420.1, seqID NZ_CP029420.1 [record index 0].\n",
      "2024-04-17 17:48:44,772 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017440.1, seqID NZ_CP017440.1 [record index 0].\n",
      "2024-04-17 17:48:45,063 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018241.1, seqID NZ_CP018241.1 [record index 0].\n",
      "2024-04-17 17:48:45,358 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038416.1, seqID NZ_CP038416.1 [record index 0].\n",
      "2024-04-17 17:48:45,586 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023853.1, seqID NZ_CP023853.1 [record index 0].\n",
      "2024-04-17 17:48:45,921 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015020.1, seqID NZ_CP015020.1 [record index 0].\n",
      "2024-04-17 17:48:46,215 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018239.1, seqID NZ_CP018239.1 [record index 0].\n",
      "2024-04-17 17:48:46,518 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038425.1, seqID NZ_CP038425.1 [record index 0].\n",
      "2024-04-17 17:48:46,748 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP072980.1, seqID NZ_CP072980.1 [record index 0].\n",
      "2024-04-17 17:48:47,036 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP040309.1, seqID NZ_CP040309.1 [record index 0].\n",
      "2024-04-17 17:48:47,362 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017438.1, seqID NZ_CP017438.1 [record index 0].\n",
      "2024-04-17 17:48:47,679 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017436.1, seqID NZ_CP017436.1 [record index 0].\n",
      "2024-04-17 17:48:47,970 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062160.1, seqID NZ_CP062160.1 [record index 0].\n",
      "2024-04-17 17:48:48,192 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051609.1, seqID NZ_CP051609.1 [record index 0].\n",
      "2024-04-17 17:48:48,420 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051615.1, seqID NZ_CP051615.1 [record index 0].\n",
      "2024-04-17 17:48:48,699 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP030767.1, seqID NZ_CP030767.1 [record index 0].\n",
      "2024-04-17 17:48:48,988 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034806.1, seqID NZ_CP034806.1 [record index 0].\n",
      "2024-04-17 17:48:49,274 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034801.1, seqID NZ_CP034801.1 [record index 0].\n",
      "2024-04-17 17:48:49,559 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034799.1, seqID NZ_CP034799.1 [record index 0].\n",
      "2024-04-17 17:48:49,843 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034803.1, seqID NZ_CP034803.1 [record index 0].\n",
      "2024-04-17 17:48:50,133 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034792.1, seqID NZ_CP034792.1 [record index 0].\n",
      "2024-04-17 17:48:50,436 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038419.1, seqID NZ_CP038419.1 [record index 0].\n",
      "2024-04-17 17:48:50,667 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022326.1, seqID NZ_AP022326.1 [record index 0].\n",
      "2024-04-17 17:48:50,956 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP015832.1, seqID NZ_CP015832.1 [record index 0].\n",
      "2024-04-17 17:48:51,254 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP038414.1, seqID NZ_CP038414.1 [record index 0].\n",
      "2024-04-17 17:48:51,546 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018237.1, seqID NZ_CP018237.1 [record index 0].\n",
      "2024-04-17 17:48:51,841 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028592.1, seqID NZ_CP028592.1 [record index 0].\n",
      "2024-04-17 17:48:52,142 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028596.1, seqID NZ_CP028596.1 [record index 0].\n",
      "2024-04-17 17:48:52,433 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028598.1, seqID NZ_CP028598.1 [record index 0].\n",
      "2024-04-17 17:48:52,728 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028600.1, seqID NZ_CP028600.1 [record index 0].\n",
      "2024-04-17 17:48:53,020 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028603.1, seqID NZ_CP028603.1 [record index 0].\n",
      "2024-04-17 17:48:53,326 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028607.1, seqID NZ_CP028607.1 [record index 0].\n",
      "2024-04-17 17:48:53,624 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028609.1, seqID NZ_CP028609.1 [record index 0].\n",
      "2024-04-17 17:48:53,910 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028614.1, seqID NZ_CP028614.1 [record index 0].\n",
      "2024-04-17 17:48:54,197 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028617.1, seqID NZ_CP028617.1 [record index 0].\n",
      "2024-04-17 17:48:54,489 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028620.1, seqID NZ_CP028620.1 [record index 0].\n",
      "2024-04-17 17:48:54,778 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028623.1, seqID NZ_CP028623.1 [record index 0].\n",
      "2024-04-17 17:48:55,070 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028626.1, seqID NZ_CP028626.1 [record index 0].\n",
      "2024-04-17 17:48:55,360 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028629.1, seqID NZ_CP028629.1 [record index 0].\n",
      "2024-04-17 17:48:55,643 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028632.1, seqID NZ_CP028632.1 [record index 0].\n",
      "2024-04-17 17:48:55,927 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028635.1, seqID NZ_CP028635.1 [record index 0].\n",
      "2024-04-17 17:48:56,210 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028638.1, seqID NZ_CP028638.1 [record index 0].\n",
      "2024-04-17 17:48:56,494 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028641.1, seqID NZ_CP028641.1 [record index 0].\n",
      "2024-04-17 17:48:56,813 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP093011.1, seqID NZ_CP093011.1 [record index 0].\n",
      "2024-04-17 17:48:57,099 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028644.1, seqID NZ_CP028644.1 [record index 0].\n",
      "2024-04-17 17:48:57,388 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028647.1, seqID NZ_CP028647.1 [record index 0].\n",
      "2024-04-17 17:48:57,673 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028650.1, seqID NZ_CP028650.1 [record index 0].\n",
      "2024-04-17 17:48:57,958 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028652.1, seqID NZ_CP028652.1 [record index 0].\n",
      "2024-04-17 17:48:58,244 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028654.1, seqID NZ_CP028654.1 [record index 0].\n",
      "2024-04-17 17:48:58,540 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028656.1, seqID NZ_CP028656.1 [record index 0].\n",
      "2024-04-17 17:48:58,832 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028659.1, seqID NZ_CP028659.1 [record index 0].\n",
      "2024-04-17 17:48:59,024 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP023386.1, seqID NZ_CP023386.1 [record index 0].\n",
      "2024-04-17 17:48:59,309 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028662.1, seqID NZ_CP028662.1 [record index 0].\n",
      "2024-04-17 17:48:59,599 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028665.1, seqID NZ_CP028665.1 [record index 0].\n",
      "2024-04-17 17:48:59,886 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028668.1, seqID NZ_CP028668.1 [record index 0].\n",
      "2024-04-17 17:49:00,109 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051661.1, seqID NZ_CP051661.1 [record index 0].\n",
      "2024-04-17 17:49:00,330 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051663.1, seqID NZ_CP051663.1 [record index 0].\n",
      "2024-04-17 17:49:00,551 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051659.1, seqID NZ_CP051659.1 [record index 0].\n",
      "2024-04-17 17:49:00,841 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028671.1, seqID NZ_CP028671.1 [record index 0].\n",
      "2024-04-17 17:49:01,125 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028674.1, seqID NZ_CP028674.1 [record index 0].\n",
      "2024-04-17 17:49:01,416 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028677.1, seqID NZ_CP028677.1 [record index 0].\n",
      "2024-04-17 17:49:01,727 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP017434.1, seqID NZ_CP017434.1 [record index 0].\n",
      "2024-04-17 17:49:02,008 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028680.1, seqID NZ_CP028680.1 [record index 0].\n",
      "2024-04-17 17:49:02,298 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028683.1, seqID NZ_CP028683.1 [record index 0].\n",
      "2024-04-17 17:49:02,594 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028685.1, seqID NZ_CP028685.1 [record index 0].\n",
      "2024-04-17 17:49:02,883 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028687.1, seqID NZ_CP028687.1 [record index 0].\n",
      "2024-04-17 17:49:03,170 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028690.1, seqID NZ_CP028690.1 [record index 0].\n",
      "2024-04-17 17:49:03,459 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028693.1, seqID NZ_CP028693.1 [record index 0].\n",
      "2024-04-17 17:49:03,746 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028695.1, seqID NZ_CP028695.1 [record index 0].\n",
      "2024-04-17 17:49:04,030 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP018250.1, seqID NZ_CP018250.1 [record index 0].\n",
      "2024-04-17 17:49:04,314 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028698.1, seqID NZ_CP028698.1 [record index 0].\n",
      "2024-04-17 17:49:04,603 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP028700.1, seqID NZ_CP028700.1 [record index 0].\n",
      "2024-04-17 17:49:04,833 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022650.1, seqID NZ_AP022650.1 [record index 0].\n",
      "2024-04-17 17:49:05,112 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034797.1, seqID NZ_CP034797.1 [record index 0].\n",
      "2024-04-17 17:49:05,398 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034808.1, seqID NZ_CP034808.1 [record index 0].\n",
      "2024-04-17 17:49:05,676 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP034794.1, seqID NZ_CP034794.1 [record index 0].\n",
      "2024-04-17 17:49:05,791 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP055092.1, seqID NZ_CP055092.1 [record index 0].\n",
      "2024-04-17 17:49:05,995 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055041.1, seqID NZ_CP055041.1 [record index 0].\n",
      "2024-04-17 17:49:06,223 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058796.1, seqID NZ_CP058796.1 [record index 0].\n",
      "2024-04-17 17:49:06,342 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058786.1, seqID NZ_CP058786.1 [record index 0].\n",
      "2024-04-17 17:49:06,454 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058849.1, seqID NZ_CP058849.1 [record index 0].\n",
      "2024-04-17 17:49:06,577 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058839.1, seqID NZ_CP058839.1 [record index 0].\n",
      "2024-04-17 17:49:06,698 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058886.1, seqID NZ_CP058886.1 [record index 0].\n",
      "2024-04-17 17:49:06,820 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058833.1, seqID NZ_CP058833.1 [record index 0].\n",
      "2024-04-17 17:49:06,962 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP058831.1, seqID NZ_CP058831.1 [record index 0].\n",
      "2024-04-17 17:49:07,090 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP054977.1, seqID NZ_CP054977.1 [record index 0].\n",
      "2024-04-17 17:49:07,215 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP054992.1, seqID NZ_CP054992.1 [record index 0].\n",
      "2024-04-17 17:49:07,458 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055052.1, seqID NZ_CP055052.1 [record index 0].\n",
      "2024-04-17 17:49:07,706 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP055055.1, seqID NZ_CP055055.1 [record index 0].\n",
      "2024-04-17 17:49:07,922 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055063.1, seqID NZ_CP055063.1 [record index 0].\n",
      "2024-04-17 17:49:08,181 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP080247.1, seqID NZ_CP080247.1 [record index 0].\n",
      "2024-04-17 17:49:08,443 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP079924.1, seqID NZ_CP079924.1 [record index 0].\n",
      "2024-04-17 17:49:08,667 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP047461.1, seqID NZ_CP047461.1 [record index 0].\n",
      "2024-04-17 17:49:08,891 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP047455.1, seqID NZ_CP047455.1 [record index 0].\n",
      "2024-04-17 17:49:09,107 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP090305.1, seqID NZ_CP090305.1 [record index 0].\n",
      "2024-04-17 17:49:09,316 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP037910.1, seqID NZ_CP037910.1 [record index 0].\n",
      "2024-04-17 17:49:09,541 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP041452.1, seqID NZ_CP041452.1 [record index 0].\n",
      "2024-04-17 17:49:09,776 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP041442.1, seqID NZ_CP041442.1 [record index 0].\n",
      "2024-04-17 17:49:10,014 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP041448.1, seqID NZ_CP041448.1 [record index 0].\n",
      "2024-04-17 17:49:10,122 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP023230.2, seqID NZ_AP023230.2 [record index 0].\n",
      "2024-04-17 17:49:10,230 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP023226.1, seqID NZ_AP023226.1 [record index 0].\n",
      "2024-04-17 17:49:10,352 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP013483.1, seqID NZ_CP013483.1 [record index 0].\n",
      "2024-04-17 17:49:10,619 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP081008.1, seqID NZ_CP081008.1 [record index 0].\n",
      "2024-04-17 17:49:10,883 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP081007.1, seqID NZ_CP081007.1 [record index 0].\n",
      "2024-04-17 17:49:11,000 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP068041.1, seqID NZ_CP068041.1 [record index 0].\n",
      "2024-04-17 17:49:11,109 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP067399.1, seqID NZ_CP067399.1 [record index 0].\n",
      "2024-04-17 17:49:11,381 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP069386.1, seqID NZ_CP069386.1 [record index 0].\n",
      "2024-04-17 17:49:11,500 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP089142.1, seqID NZ_CP089142.1 [record index 0].\n",
      "2024-04-17 17:49:11,707 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035822.1, seqID NZ_CP035822.1 [record index 0].\n",
      "2024-04-17 17:49:11,937 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022287.1, seqID NZ_AP022287.1 [record index 0].\n",
      "2024-04-17 17:49:12,152 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022245.1, seqID NZ_AP022245.1 [record index 0].\n",
      "2024-04-17 17:49:12,367 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022221.1, seqID NZ_AP022221.1 [record index 0].\n",
      "2024-04-17 17:49:12,578 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022177.1, seqID NZ_AP022177.1 [record index 0].\n",
      "2024-04-17 17:49:12,798 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022159.1, seqID NZ_AP022159.1 [record index 0].\n",
      "2024-04-17 17:49:12,911 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP022105.1, seqID NZ_AP022105.1 [record index 0].\n",
      "2024-04-17 17:49:13,129 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022049.1, seqID NZ_AP022049.1 [record index 0].\n",
      "2024-04-17 17:49:13,357 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP022003.1, seqID NZ_AP022003.1 [record index 0].\n",
      "2024-04-17 17:49:13,461 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP022030.1, seqID NZ_AP022030.1 [record index 0].\n",
      "2024-04-17 17:49:13,669 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021935.1, seqID NZ_AP021935.1 [record index 0].\n",
      "2024-04-17 17:49:13,790 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP021946.1, seqID NZ_AP021946.1 [record index 0].\n",
      "2024-04-17 17:49:13,901 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LT838200.1, seqID NZ_LT838200.1 [record index 0].\n",
      "2024-04-17 17:49:14,013 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LT838196.1, seqID NZ_LT838196.1 [record index 0].\n",
      "2024-04-17 17:49:14,201 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP061530.1, seqID NZ_CP061530.1 [record index 0].\n",
      "2024-04-17 17:49:14,489 [INFO - chronostrain.cli.make_db_json] - Created 14 merged markers for strain NZ_CP034595.1, seqID NZ_CP034595.1 [record index 0].\n",
      "2024-04-17 17:49:14,603 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP036177.1, seqID NZ_CP036177.1 [record index 0].\n",
      "2024-04-17 17:49:14,731 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043279.1, seqID NZ_CP043279.1 [record index 0].\n",
      "2024-04-17 17:49:14,853 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP028578.2, seqID NZ_CP028578.2 [record index 0].\n",
      "2024-04-17 17:49:14,967 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026580.3, seqID NZ_CP026580.3 [record index 0].\n",
      "2024-04-17 17:49:15,093 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP092973.1, seqID NZ_CP092973.1 [record index 0].\n",
      "2024-04-17 17:49:15,206 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058342.1, seqID NZ_CP058342.1 [record index 0].\n",
      "2024-04-17 17:49:15,314 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP032992.1, seqID NZ_CP032992.1 [record index 0].\n",
      "2024-04-17 17:49:15,574 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084899.1, seqID NZ_CP084899.1 [record index 0].\n",
      "2024-04-17 17:49:15,799 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091038.1, seqID NZ_CP091038.1 [record index 0].\n",
      "2024-04-17 17:49:16,024 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080235.1, seqID NZ_CP080235.1 [record index 0].\n",
      "2024-04-17 17:49:16,226 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP011134.1, seqID NZ_CP011134.1 [record index 0].\n",
      "2024-04-17 17:49:16,476 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011124.1, seqID NZ_CP011124.1 [record index 0].\n",
      "2024-04-17 17:49:16,668 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP039837.1, seqID NZ_CP039837.1 [record index 0].\n",
      "2024-04-17 17:49:16,943 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NC_017641.1, seqID NC_017641.1 [record index 0].\n",
      "2024-04-17 17:49:17,176 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP041359.1, seqID NZ_CP041359.1 [record index 0].\n",
      "2024-04-17 17:49:17,429 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP080620.1, seqID NZ_CP080620.1 [record index 0].\n",
      "2024-04-17 17:49:17,696 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP018801.1, seqID NZ_CP018801.1 [record index 0].\n",
      "2024-04-17 17:49:17,878 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP038295.1, seqID NZ_CP038295.1 [record index 0].\n",
      "2024-04-17 17:49:17,986 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP024205.1, seqID NZ_AP024205.1 [record index 0].\n",
      "2024-04-17 17:49:18,193 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP049353.1, seqID NZ_CP049353.1 [record index 0].\n",
      "2024-04-17 17:49:18,448 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP072986.1, seqID NZ_CP072986.1 [record index 0].\n",
      "2024-04-17 17:49:18,717 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP058948.1, seqID NZ_CP058948.1 [record index 0].\n",
      "2024-04-17 17:49:19,098 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP059043.1, seqID NZ_CP059043.1 [record index 0].\n",
      "2024-04-17 17:49:19,297 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091836.1, seqID NZ_CP091836.1 [record index 0].\n",
      "2024-04-17 17:49:19,493 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091839.1, seqID NZ_CP091839.1 [record index 0].\n",
      "2024-04-17 17:49:19,704 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055048.1, seqID NZ_CP055048.1 [record index 0].\n",
      "2024-04-17 17:49:19,819 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP076318.1, seqID NZ_CP076318.1 [record index 0].\n",
      "2024-04-17 17:49:19,941 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083701.1, seqID NZ_CP083701.1 [record index 0].\n",
      "2024-04-17 17:49:20,110 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP041422.1, seqID NZ_CP041422.1 [record index 0].\n",
      "2024-04-17 17:49:20,299 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP081009.1, seqID NZ_CP081009.1 [record index 0].\n",
      "2024-04-17 17:49:20,477 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP076043.1, seqID NZ_CP076043.1 [record index 0].\n",
      "2024-04-17 17:49:20,699 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP050173.1, seqID NZ_CP050173.1 [record index 0].\n",
      "2024-04-17 17:49:20,969 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP060709.1, seqID NZ_CP060709.1 [record index 0].\n",
      "2024-04-17 17:49:21,226 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011321.1, seqID NZ_CP011321.1 [record index 0].\n",
      "2024-04-17 17:49:21,483 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011320.1, seqID NZ_CP011320.1 [record index 0].\n",
      "2024-04-17 17:49:21,739 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011324.1, seqID NZ_CP011324.1 [record index 0].\n",
      "2024-04-17 17:49:21,992 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011322.1, seqID NZ_CP011322.1 [record index 0].\n",
      "2024-04-17 17:49:22,209 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP017631.1, seqID NZ_CP017631.1 [record index 0].\n",
      "2024-04-17 17:49:22,423 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080255.1, seqID NZ_CP080255.1 [record index 0].\n",
      "2024-04-17 17:49:22,645 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP073768.1, seqID NZ_CP073768.1 [record index 0].\n",
      "2024-04-17 17:49:22,748 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP051716.1, seqID NZ_CP051716.1 [record index 0].\n",
      "2024-04-17 17:49:22,962 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP051733.1, seqID NZ_CP051733.1 [record index 0].\n",
      "2024-04-17 17:49:23,163 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053284.1, seqID NZ_CP053284.1 [record index 0].\n",
      "2024-04-17 17:49:23,276 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP025950.3, seqID NZ_CP025950.3 [record index 0].\n",
      "2024-04-17 17:49:23,390 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP025627.1, seqID NZ_CP025627.1 [record index 0].\n",
      "2024-04-17 17:49:23,617 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP082824.1, seqID NZ_CP082824.1 [record index 0].\n",
      "2024-04-17 17:49:23,845 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP071954.1, seqID NZ_CP071954.1 [record index 0].\n",
      "2024-04-17 17:49:23,962 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP011061.1, seqID NZ_CP011061.1 [record index 0].\n",
      "2024-04-17 17:49:24,157 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010237.1, seqID NZ_CP010237.1 [record index 0].\n",
      "2024-04-17 17:49:24,426 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP040667.1, seqID NZ_CP040667.1 [record index 0].\n",
      "2024-04-17 17:49:24,638 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010226.1, seqID NZ_CP010226.1 [record index 0].\n",
      "2024-04-17 17:49:24,927 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP092702.1, seqID NZ_CP092702.1 [record index 0].\n",
      "2024-04-17 17:49:25,180 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011113.2, seqID NZ_CP011113.2 [record index 0].\n",
      "2024-04-17 17:49:25,442 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP083410.1, seqID NZ_CP083410.1 [record index 0].\n",
      "2024-04-17 17:49:25,630 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NC_017656.1, seqID NC_017656.1 [record index 0].\n",
      "2024-04-17 17:49:25,842 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042350.1, seqID NZ_CP042350.1 [record index 0].\n",
      "2024-04-17 17:49:26,065 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086616.1, seqID NZ_CP086616.1 [record index 0].\n",
      "2024-04-17 17:49:26,296 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086618.1, seqID NZ_CP086618.1 [record index 0].\n",
      "2024-04-17 17:49:26,531 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086620.1, seqID NZ_CP086620.1 [record index 0].\n",
      "2024-04-17 17:49:26,758 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086622.1, seqID NZ_CP086622.1 [record index 0].\n",
      "2024-04-17 17:49:26,982 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086623.1, seqID NZ_CP086623.1 [record index 0].\n",
      "2024-04-17 17:49:27,209 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086624.1, seqID NZ_CP086624.1 [record index 0].\n",
      "2024-04-17 17:49:27,436 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086625.1, seqID NZ_CP086625.1 [record index 0].\n",
      "2024-04-17 17:49:27,661 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086626.1, seqID NZ_CP086626.1 [record index 0].\n",
      "2024-04-17 17:49:27,884 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP086627.1, seqID NZ_CP086627.1 [record index 0].\n",
      "2024-04-17 17:49:28,108 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068829.1, seqID NZ_CP068829.1 [record index 0].\n",
      "2024-04-17 17:49:28,315 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068827.1, seqID NZ_CP068827.1 [record index 0].\n",
      "2024-04-17 17:49:28,541 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068826.1, seqID NZ_CP068826.1 [record index 0].\n",
      "2024-04-17 17:49:28,766 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068824.1, seqID NZ_CP068824.1 [record index 0].\n",
      "2024-04-17 17:49:28,993 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068821.1, seqID NZ_CP068821.1 [record index 0].\n",
      "2024-04-17 17:49:29,194 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068820.1, seqID NZ_CP068820.1 [record index 0].\n",
      "2024-04-17 17:49:29,420 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068819.1, seqID NZ_CP068819.1 [record index 0].\n",
      "2024-04-17 17:49:29,646 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068813.1, seqID NZ_CP068813.1 [record index 0].\n",
      "2024-04-17 17:49:29,849 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068812.1, seqID NZ_CP068812.1 [record index 0].\n",
      "2024-04-17 17:49:30,076 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068818.1, seqID NZ_CP068818.1 [record index 0].\n",
      "2024-04-17 17:49:30,285 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068814.1, seqID NZ_CP068814.1 [record index 0].\n",
      "2024-04-17 17:49:30,512 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068811.1, seqID NZ_CP068811.1 [record index 0].\n",
      "2024-04-17 17:49:30,739 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068810.1, seqID NZ_CP068810.1 [record index 0].\n",
      "2024-04-17 17:49:30,968 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP087377.1, seqID NZ_CP087377.1 [record index 0].\n",
      "2024-04-17 17:49:31,196 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068830.1, seqID NZ_CP068830.1 [record index 0].\n",
      "2024-04-17 17:49:31,402 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068808.1, seqID NZ_CP068808.1 [record index 0].\n",
      "2024-04-17 17:49:31,612 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068807.1, seqID NZ_CP068807.1 [record index 0].\n",
      "2024-04-17 17:49:31,818 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP068805.1, seqID NZ_CP068805.1 [record index 0].\n",
      "2024-04-17 17:49:32,044 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068804.1, seqID NZ_CP068804.1 [record index 0].\n",
      "2024-04-17 17:49:32,276 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068801.1, seqID NZ_CP068801.1 [record index 0].\n",
      "2024-04-17 17:49:32,501 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068800.1, seqID NZ_CP068800.1 [record index 0].\n",
      "2024-04-17 17:49:32,719 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068796.1, seqID NZ_CP068796.1 [record index 0].\n",
      "2024-04-17 17:49:32,833 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP068795.1, seqID NZ_CP068795.1 [record index 0].\n",
      "2024-04-17 17:49:33,034 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP062811.1, seqID NZ_CP062811.1 [record index 0].\n",
      "2024-04-17 17:49:33,249 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062815.1, seqID NZ_CP062815.1 [record index 0].\n",
      "2024-04-17 17:49:33,470 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062826.1, seqID NZ_CP062826.1 [record index 0].\n",
      "2024-04-17 17:49:33,685 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062832.1, seqID NZ_CP062832.1 [record index 0].\n",
      "2024-04-17 17:49:33,899 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062838.1, seqID NZ_CP062838.1 [record index 0].\n",
      "2024-04-17 17:49:34,117 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062883.1, seqID NZ_CP062883.1 [record index 0].\n",
      "2024-04-17 17:49:34,334 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062895.1, seqID NZ_CP062895.1 [record index 0].\n",
      "2024-04-17 17:49:34,443 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR882052.1, seqID NZ_LR882052.1 [record index 0].\n",
      "2024-04-17 17:49:34,700 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080139.1, seqID NZ_CP080139.1 [record index 0].\n",
      "2024-04-17 17:49:34,810 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP042588.1, seqID NZ_CP042588.1 [record index 0].\n",
      "2024-04-17 17:49:35,087 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP087110.1, seqID NZ_CP087110.1 [record index 0].\n",
      "2024-04-17 17:49:35,278 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP038857.1, seqID NZ_CP038857.1 [record index 0].\n",
      "2024-04-17 17:49:35,543 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043189.1, seqID NZ_CP043189.1 [record index 0].\n",
      "2024-04-17 17:49:35,806 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043191.1, seqID NZ_CP043191.1 [record index 0].\n",
      "2024-04-17 17:49:36,066 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043183.1, seqID NZ_CP043183.1 [record index 0].\n",
      "2024-04-17 17:49:36,323 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043185.1, seqID NZ_CP043185.1 [record index 0].\n",
      "2024-04-17 17:49:36,586 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043187.1, seqID NZ_CP043187.1 [record index 0].\n",
      "2024-04-17 17:49:36,850 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043193.1, seqID NZ_CP043193.1 [record index 0].\n",
      "2024-04-17 17:49:37,113 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043195.1, seqID NZ_CP043195.1 [record index 0].\n",
      "2024-04-17 17:49:37,372 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043197.1, seqID NZ_CP043197.1 [record index 0].\n",
      "2024-04-17 17:49:37,634 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043199.1, seqID NZ_CP043199.1 [record index 0].\n",
      "2024-04-17 17:49:37,894 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043201.1, seqID NZ_CP043201.1 [record index 0].\n",
      "2024-04-17 17:49:38,154 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043203.1, seqID NZ_CP043203.1 [record index 0].\n",
      "2024-04-17 17:49:38,421 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043205.1, seqID NZ_CP043205.1 [record index 0].\n",
      "2024-04-17 17:49:38,686 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043207.1, seqID NZ_CP043207.1 [record index 0].\n",
      "2024-04-17 17:49:38,948 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043209.1, seqID NZ_CP043209.1 [record index 0].\n",
      "2024-04-17 17:49:39,209 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043211.1, seqID NZ_CP043211.1 [record index 0].\n",
      "2024-04-17 17:49:39,463 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP038791.1, seqID NZ_CP038791.1 [record index 0].\n",
      "2024-04-17 17:49:39,575 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP041628.1, seqID NZ_CP041628.1 [record index 0].\n",
      "2024-04-17 17:49:39,764 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080253.1, seqID NZ_CP080253.1 [record index 0].\n",
      "2024-04-17 17:49:39,968 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048605.1, seqID NZ_CP048605.1 [record index 0].\n",
      "2024-04-17 17:49:40,080 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP085060.1, seqID NZ_CP085060.1 [record index 0].\n",
      "2024-04-17 17:49:40,193 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP085074.1, seqID NZ_CP085074.1 [record index 0].\n",
      "2024-04-17 17:49:40,415 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080233.1, seqID NZ_CP080233.1 [record index 0].\n",
      "2024-04-17 17:49:40,650 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080223.1, seqID NZ_CP080223.1 [record index 0].\n",
      "2024-04-17 17:49:40,763 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP049081.1, seqID NZ_CP049081.1 [record index 0].\n",
      "2024-04-17 17:49:40,984 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP087578.1, seqID NZ_CP087578.1 [record index 0].\n",
      "2024-04-17 17:49:41,257 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084529.1, seqID NZ_CP084529.1 [record index 0].\n",
      "2024-04-17 17:49:41,483 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP075469.1, seqID NZ_CP075469.1 [record index 0].\n",
      "2024-04-17 17:49:41,706 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP075465.1, seqID NZ_CP075465.1 [record index 0].\n",
      "2024-04-17 17:49:41,923 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP075480.1, seqID NZ_CP075480.1 [record index 0].\n",
      "2024-04-17 17:49:42,073 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP075474.1, seqID NZ_CP075474.1 [record index 0].\n",
      "2024-04-17 17:49:42,216 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP075487.1, seqID NZ_CP075487.1 [record index 0].\n",
      "2024-04-17 17:49:42,444 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP075462.1, seqID NZ_CP075462.1 [record index 0].\n",
      "2024-04-17 17:49:42,677 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042878.1, seqID NZ_CP042878.1 [record index 0].\n",
      "2024-04-17 17:49:42,914 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042892.1, seqID NZ_CP042892.1 [record index 0].\n",
      "2024-04-17 17:49:43,172 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP053605.1, seqID NZ_CP053605.1 [record index 0].\n",
      "2024-04-17 17:49:43,437 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP053607.1, seqID NZ_CP053607.1 [record index 0].\n",
      "2024-04-17 17:49:43,700 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP053604.1, seqID NZ_CP053604.1 [record index 0].\n",
      "2024-04-17 17:49:43,811 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP082127.1, seqID NZ_CP082127.1 [record index 0].\n",
      "2024-04-17 17:49:43,924 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP082129.1, seqID NZ_CP082129.1 [record index 0].\n",
      "2024-04-17 17:49:44,035 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083869.1, seqID NZ_CP083869.1 [record index 0].\n",
      "2024-04-17 17:49:44,146 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083872.1, seqID NZ_CP083872.1 [record index 0].\n",
      "2024-04-17 17:49:44,255 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083877.1, seqID NZ_CP083877.1 [record index 0].\n",
      "2024-04-17 17:49:44,366 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083880.1, seqID NZ_CP083880.1 [record index 0].\n",
      "2024-04-17 17:49:44,480 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083886.1, seqID NZ_CP083886.1 [record index 0].\n",
      "2024-04-17 17:49:44,665 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LS483296.1, seqID NZ_LS483296.1 [record index 0].\n",
      "2024-04-17 17:49:44,864 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134228.1, seqID NZ_LR134228.1 [record index 0].\n",
      "2024-04-17 17:49:45,090 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_LR134240.1, seqID NZ_LR134240.1 [record index 0].\n",
      "2024-04-17 17:49:45,276 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134152.1, seqID NZ_LR134152.1 [record index 0].\n",
      "2024-04-17 17:49:45,459 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134227.1, seqID NZ_LR134227.1 [record index 0].\n",
      "2024-04-17 17:49:45,622 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_LR134239.1, seqID NZ_LR134239.1 [record index 0].\n",
      "2024-04-17 17:49:45,802 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134231.1, seqID NZ_LR134231.1 [record index 0].\n",
      "2024-04-17 17:49:45,985 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134075.1, seqID NZ_LR134075.1 [record index 0].\n",
      "2024-04-17 17:49:46,186 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134295.1, seqID NZ_LR134295.1 [record index 0].\n",
      "2024-04-17 17:49:46,374 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134296.1, seqID NZ_LR134296.1 [record index 0].\n",
      "2024-04-17 17:49:46,573 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134247.1, seqID NZ_LR134247.1 [record index 0].\n",
      "2024-04-17 17:49:46,755 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134082.1, seqID NZ_LR134082.1 [record index 0].\n",
      "2024-04-17 17:49:46,957 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LT601384.1, seqID NZ_LT601384.1 [record index 0].\n",
      "2024-04-17 17:49:47,155 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR217818.1, seqID NZ_LR217818.1 [record index 0].\n",
      "2024-04-17 17:49:47,352 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019778.1, seqID NZ_CP019778.1 [record index 0].\n",
      "2024-04-17 17:49:47,499 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_LR134220.1, seqID NZ_LR134220.1 [record index 0].\n",
      "2024-04-17 17:49:47,682 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LS483297.1, seqID NZ_LS483297.1 [record index 0].\n",
      "2024-04-17 17:49:47,847 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_LR134216.1, seqID NZ_LR134216.1 [record index 0].\n",
      "2024-04-17 17:49:48,051 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP011495.1, seqID NZ_CP011495.1 [record index 0].\n",
      "2024-04-17 17:49:48,140 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041749.1, seqID NZ_CP041749.1 [record index 0].\n",
      "2024-04-17 17:49:48,326 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP048439.1, seqID NZ_CP048439.1 [record index 0].\n",
      "2024-04-17 17:49:48,539 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047277.1, seqID NZ_CP047277.1 [record index 0].\n",
      "2024-04-17 17:49:48,793 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP034953.2, seqID NZ_CP034953.2 [record index 0].\n",
      "2024-04-17 17:49:48,992 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP076645.1, seqID NZ_CP076645.1 [record index 0].\n",
      "2024-04-17 17:49:49,199 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890508.1, seqID NZ_LR890508.1 [record index 0].\n",
      "2024-04-17 17:49:49,417 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890349.1, seqID NZ_LR890349.1 [record index 0].\n",
      "2024-04-17 17:49:49,525 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR890603.1, seqID NZ_LR890603.1 [record index 0].\n",
      "2024-04-17 17:49:49,742 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890466.1, seqID NZ_LR890466.1 [record index 0].\n",
      "2024-04-17 17:49:49,938 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR890714.1, seqID NZ_LR890714.1 [record index 0].\n",
      "2024-04-17 17:49:50,165 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR890299.1, seqID NZ_LR890299.1 [record index 0].\n",
      "2024-04-17 17:49:50,277 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR890288.1, seqID NZ_LR890288.1 [record index 0].\n",
      "2024-04-17 17:49:50,488 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR130562.1, seqID NZ_LR130562.1 [record index 0].\n",
      "2024-04-17 17:49:50,601 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP018103.1, seqID NZ_CP018103.1 [record index 0].\n",
      "2024-04-17 17:49:50,795 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP018206.1, seqID NZ_CP018206.1 [record index 0].\n",
      "2024-04-17 17:49:50,907 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP018115.1, seqID NZ_CP018115.1 [record index 0].\n",
      "2024-04-17 17:49:51,023 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP018109.1, seqID NZ_CP018109.1 [record index 0].\n",
      "2024-04-17 17:49:51,137 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP018121.1, seqID NZ_CP018121.1 [record index 0].\n",
      "2024-04-17 17:49:51,354 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025747.1, seqID NZ_CP025747.1 [record index 0].\n",
      "2024-04-17 17:49:51,465 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR890606.1, seqID NZ_LR890606.1 [record index 0].\n",
      "2024-04-17 17:49:51,578 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP069657.1, seqID NZ_CP069657.1 [record index 0].\n",
      "2024-04-17 17:49:51,690 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP069666.1, seqID NZ_CP069666.1 [record index 0].\n",
      "2024-04-17 17:49:51,950 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP019629.1, seqID NZ_CP019629.1 [record index 0].\n",
      "2024-04-17 17:49:52,105 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP092037.1, seqID NZ_CP092037.1 [record index 0].\n",
      "2024-04-17 17:49:52,362 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP060708.1, seqID NZ_CP060708.1 [record index 0].\n",
      "2024-04-17 17:49:52,622 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP028703.1, seqID NZ_CP028703.1 [record index 0].\n",
      "2024-04-17 17:49:52,733 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP023219.2, seqID NZ_AP023219.2 [record index 0].\n",
      "2024-04-17 17:49:52,844 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP019189.1, seqID NZ_AP019189.1 [record index 0].\n",
      "2024-04-17 17:49:52,954 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP046416.1, seqID NZ_CP046416.1 [record index 0].\n",
      "2024-04-17 17:49:53,200 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026027.1, seqID NZ_CP026027.1 [record index 0].\n",
      "2024-04-17 17:49:53,403 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047594.1, seqID NZ_CP047594.1 [record index 0].\n",
      "2024-04-17 17:49:53,683 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP091756.1, seqID NZ_CP091756.1 [record index 0].\n",
      "2024-04-17 17:49:53,791 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP045277.1, seqID NZ_CP045277.1 [record index 0].\n",
      "2024-04-17 17:49:54,009 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP062203.1, seqID NZ_CP062203.1 [record index 0].\n",
      "2024-04-17 17:49:54,257 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091651.1, seqID NZ_CP091651.1 [record index 0].\n",
      "2024-04-17 17:49:54,455 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040664.1, seqID NZ_CP040664.1 [record index 0].\n",
      "2024-04-17 17:49:54,691 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091703.1, seqID NZ_CP091703.1 [record index 0].\n",
      "2024-04-17 17:49:54,853 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP079991.1, seqID NZ_CP079991.1 [record index 0].\n",
      "2024-04-17 17:49:55,018 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP079992.1, seqID NZ_CP079992.1 [record index 0].\n",
      "2024-04-17 17:49:55,144 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091652.1, seqID NZ_CP091652.1 [record index 0].\n",
      "2024-04-17 17:49:55,398 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP008801.1, seqID NZ_CP008801.1 [record index 0].\n",
      "2024-04-17 17:49:55,637 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP030919.1, seqID NZ_CP030919.1 [record index 0].\n",
      "2024-04-17 17:49:55,881 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091664.1, seqID NZ_CP091664.1 [record index 0].\n",
      "2024-04-17 17:49:56,133 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091704.1, seqID NZ_CP091704.1 [record index 0].\n",
      "2024-04-17 17:49:56,251 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091662.1, seqID NZ_CP091662.1 [record index 0].\n",
      "2024-04-17 17:49:56,487 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091677.1, seqID NZ_CP091677.1 [record index 0].\n",
      "2024-04-17 17:49:56,730 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091700.1, seqID NZ_CP091700.1 [record index 0].\n",
      "2024-04-17 17:49:56,982 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP072054.1, seqID NZ_CP072054.1 [record index 0].\n",
      "2024-04-17 17:49:57,237 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_007779.1, seqID NC_007779.1 [record index 0].\n",
      "2024-04-17 17:49:57,491 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LM995446.1, seqID NZ_LM995446.1 [record index 0].\n",
      "2024-04-17 17:49:57,782 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012870.1, seqID NZ_CP012870.1 [record index 0].\n",
      "2024-04-17 17:49:58,067 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012869.1, seqID NZ_CP012869.1 [record index 0].\n",
      "2024-04-17 17:49:58,310 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LR881938.1, seqID NZ_LR881938.1 [record index 0].\n",
      "2024-04-17 17:49:58,564 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP032679.1, seqID NZ_CP032679.1 [record index 0].\n",
      "2024-04-17 17:49:58,847 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP032667.1, seqID NZ_CP032667.1 [record index 0].\n",
      "2024-04-17 17:49:59,104 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP025268.1, seqID NZ_CP025268.1 [record index 0].\n",
      "2024-04-17 17:49:59,394 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP014225.1, seqID NZ_CP014225.1 [record index 0].\n",
      "2024-04-17 17:49:59,678 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012868.1, seqID NZ_CP012868.1 [record index 0].\n",
      "2024-04-17 17:49:59,930 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP009685.1, seqID NZ_CP009685.1 [record index 0].\n",
      "2024-04-17 17:50:00,215 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_000913.3, seqID NC_000913.3 [record index 0].\n",
      "2024-04-17 17:50:00,465 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_HG738867.1, seqID NZ_HG738867.1 [record index 0].\n",
      "2024-04-17 17:50:00,722 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LM993812.1, seqID NZ_LM993812.1 [record index 0].\n",
      "2024-04-17 17:50:01,009 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011343.2, seqID NZ_CP011343.2 [record index 0].\n",
      "2024-04-17 17:50:01,293 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP011342.2, seqID NZ_CP011342.2 [record index 0].\n",
      "2024-04-17 17:50:01,561 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_010473.1, seqID NC_010473.1 [record index 0].\n",
      "2024-04-17 17:50:01,815 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_012759.1, seqID NC_012759.1 [record index 0].\n",
      "2024-04-17 17:50:02,075 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP009273.1, seqID NZ_CP009273.1 [record index 0].\n",
      "2024-04-17 17:50:02,332 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LN832404.1, seqID NZ_LN832404.1 [record index 0].\n",
      "2024-04-17 17:50:02,588 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP076404.1, seqID NZ_CP076404.1 [record index 0].\n",
      "2024-04-17 17:50:02,837 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP017100.1, seqID NZ_CP017100.1 [record index 0].\n",
      "2024-04-17 17:50:03,092 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010445.1, seqID NZ_CP010445.1 [record index 0].\n",
      "2024-04-17 17:50:03,341 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010444.1, seqID NZ_CP010444.1 [record index 0].\n",
      "2024-04-17 17:50:03,595 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010443.1, seqID NZ_CP010443.1 [record index 0].\n",
      "2024-04-17 17:50:03,852 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010442.1, seqID NZ_CP010442.1 [record index 0].\n",
      "2024-04-17 17:50:04,102 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010441.1, seqID NZ_CP010441.1 [record index 0].\n",
      "2024-04-17 17:50:04,358 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010440.1, seqID NZ_CP010440.1 [record index 0].\n",
      "2024-04-17 17:50:04,610 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010439.1, seqID NZ_CP010439.1 [record index 0].\n",
      "2024-04-17 17:50:04,866 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP010438.1, seqID NZ_CP010438.1 [record index 0].\n",
      "2024-04-17 17:50:05,117 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP070962.1, seqID NZ_CP070962.1 [record index 0].\n",
      "2024-04-17 17:50:05,364 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP009789.1, seqID NZ_CP009789.1 [record index 0].\n",
      "2024-04-17 17:50:05,616 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP009644.1, seqID NZ_CP009644.1 [record index 0].\n",
      "2024-04-17 17:50:05,899 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP014270.1, seqID NZ_CP014270.1 [record index 0].\n",
      "2024-04-17 17:50:06,199 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP014272.1, seqID NZ_CP014272.1 [record index 0].\n",
      "2024-04-17 17:50:06,456 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP047127.1, seqID NZ_CP047127.1 [record index 0].\n",
      "2024-04-17 17:50:06,740 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP014348.1, seqID NZ_CP014348.1 [record index 0].\n",
      "2024-04-17 17:50:06,964 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP025180.1, seqID NZ_AP025180.1 [record index 0].\n",
      "2024-04-17 17:50:07,086 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091925.1, seqID NZ_CP091925.1 [record index 0].\n",
      "2024-04-17 17:50:07,329 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042843.1, seqID NZ_CP042843.1 [record index 0].\n",
      "2024-04-17 17:50:07,576 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042844.1, seqID NZ_CP042844.1 [record index 0].\n",
      "2024-04-17 17:50:07,824 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042845.1, seqID NZ_CP042845.1 [record index 0].\n",
      "2024-04-17 17:50:08,072 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042846.1, seqID NZ_CP042846.1 [record index 0].\n",
      "2024-04-17 17:50:08,310 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP077969.1, seqID NZ_CP077969.1 [record index 0].\n",
      "2024-04-17 17:50:08,578 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP068706.1, seqID NZ_CP068706.1 [record index 0].\n",
      "2024-04-17 17:50:08,840 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP028702.1, seqID NZ_CP028702.1 [record index 0].\n",
      "2024-04-17 17:50:09,122 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP054665.1, seqID NZ_CP054665.1 [record index 0].\n",
      "2024-04-17 17:50:09,407 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP013952.1, seqID NZ_CP013952.1 [record index 0].\n",
      "2024-04-17 17:50:09,518 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP010172.1, seqID NZ_CP010172.1 [record index 0].\n",
      "2024-04-17 17:50:09,739 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP010171.1, seqID NZ_CP010171.1 [record index 0].\n",
      "2024-04-17 17:50:09,948 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028732.1, seqID NZ_CP028732.1 [record index 0].\n",
      "2024-04-17 17:50:10,134 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010160.1, seqID NZ_CP010160.1 [record index 0].\n",
      "2024-04-17 17:50:10,245 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP042336.1, seqID NZ_CP042336.1 [record index 0].\n",
      "2024-04-17 17:50:10,512 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084898.1, seqID NZ_CP084898.1 [record index 0].\n",
      "2024-04-17 17:50:10,714 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP019675.1, seqID NZ_AP019675.1 [record index 0].\n",
      "2024-04-17 17:50:10,969 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084897.1, seqID NZ_CP084897.1 [record index 0].\n",
      "2024-04-17 17:50:11,221 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP079736.1, seqID NZ_CP079736.1 [record index 0].\n",
      "2024-04-17 17:50:11,474 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP079735.1, seqID NZ_CP079735.1 [record index 0].\n",
      "2024-04-17 17:50:11,730 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084893.1, seqID NZ_CP084893.1 [record index 0].\n",
      "2024-04-17 17:50:11,982 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084894.1, seqID NZ_CP084894.1 [record index 0].\n",
      "2024-04-17 17:50:12,193 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060057.1, seqID NZ_CP060057.1 [record index 0].\n",
      "2024-04-17 17:50:12,447 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084895.1, seqID NZ_CP084895.1 [record index 0].\n",
      "2024-04-17 17:50:12,655 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060061.1, seqID NZ_CP060061.1 [record index 0].\n",
      "2024-04-17 17:50:12,910 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP084896.1, seqID NZ_CP084896.1 [record index 0].\n",
      "2024-04-17 17:50:13,122 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP060065.1, seqID NZ_CP060065.1 [record index 0].\n",
      "2024-04-17 17:50:13,301 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060073.1, seqID NZ_CP060073.1 [record index 0].\n",
      "2024-04-17 17:50:13,513 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP060083.1, seqID NZ_CP060083.1 [record index 0].\n",
      "2024-04-17 17:50:13,717 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040090.1, seqID NZ_CP040090.1 [record index 0].\n",
      "2024-04-17 17:50:13,933 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029057.1, seqID NZ_CP029057.1 [record index 0].\n",
      "2024-04-17 17:50:14,135 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP022664.1, seqID NZ_CP022664.1 [record index 0].\n",
      "2024-04-17 17:50:14,323 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP065607.1, seqID NZ_CP065607.1 [record index 0].\n",
      "2024-04-17 17:50:14,511 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070026.1, seqID NZ_CP070026.1 [record index 0].\n",
      "2024-04-17 17:50:14,731 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP070157.1, seqID NZ_CP070157.1 [record index 0].\n",
      "2024-04-17 17:50:14,912 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP069459.1, seqID NZ_CP069459.1 [record index 0].\n",
      "2024-04-17 17:50:15,179 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP017220.1, seqID NZ_CP017220.1 [record index 0].\n",
      "2024-04-17 17:50:15,341 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP079993.1, seqID NZ_CP079993.1 [record index 0].\n",
      "2024-04-17 17:50:15,604 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP045457.1, seqID NZ_CP045457.1 [record index 0].\n",
      "2024-04-17 17:50:15,720 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086340.1, seqID NZ_CP086340.1 [record index 0].\n",
      "2024-04-17 17:50:15,830 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086556.1, seqID NZ_CP086556.1 [record index 0].\n",
      "2024-04-17 17:50:15,942 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086342.1, seqID NZ_CP086342.1 [record index 0].\n",
      "2024-04-17 17:50:16,051 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LS992171.1, seqID NZ_LS992171.1 [record index 0].\n",
      "2024-04-17 17:50:16,324 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LS992166.1, seqID NZ_LS992166.1 [record index 0].\n",
      "2024-04-17 17:50:16,440 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LS992185.1, seqID NZ_LS992185.1 [record index 0].\n",
      "2024-04-17 17:50:16,550 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP067250.1, seqID NZ_CP067250.1 [record index 0].\n",
      "2024-04-17 17:50:16,840 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP093221.1, seqID NZ_CP093221.1 [record index 0].\n",
      "2024-04-17 17:50:17,123 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP016018.1, seqID NZ_CP016018.1 [record index 0].\n",
      "2024-04-17 17:50:17,376 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP030240.1, seqID NZ_CP030240.1 [record index 0].\n",
      "2024-04-17 17:50:17,573 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063467.1, seqID NZ_CP063467.1 [record index 0].\n",
      "2024-04-17 17:50:17,761 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063473.1, seqID NZ_CP063473.1 [record index 0].\n",
      "2024-04-17 17:50:17,867 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP063483.1, seqID NZ_CP063483.1 [record index 0].\n",
      "2024-04-17 17:50:17,981 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083492.1, seqID NZ_CP083492.1 [record index 0].\n",
      "2024-04-17 17:50:18,180 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040663.1, seqID NZ_CP040663.1 [record index 0].\n",
      "2024-04-17 17:50:18,286 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086338.1, seqID NZ_CP086338.1 [record index 0].\n",
      "2024-04-17 17:50:18,506 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP063499.1, seqID NZ_CP063499.1 [record index 0].\n",
      "2024-04-17 17:50:18,720 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063503.1, seqID NZ_CP063503.1 [record index 0].\n",
      "2024-04-17 17:50:18,932 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063511.1, seqID NZ_CP063511.1 [record index 0].\n",
      "2024-04-17 17:50:19,156 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP024112.1, seqID NZ_AP024112.1 [record index 0].\n",
      "2024-04-17 17:50:19,363 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054224.1, seqID NZ_CP054224.1 [record index 0].\n",
      "2024-04-17 17:50:19,472 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026199.1, seqID NZ_CP026199.1 [record index 0].\n",
      "2024-04-17 17:50:19,580 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP019005.1, seqID NZ_CP019005.1 [record index 0].\n",
      "2024-04-17 17:50:19,774 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP018976.1, seqID NZ_CP018976.1 [record index 0].\n",
      "2024-04-17 17:50:19,885 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP041392.1, seqID NZ_CP041392.1 [record index 0].\n",
      "2024-04-17 17:50:20,077 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP083585.1, seqID NZ_CP083585.1 [record index 0].\n",
      "2024-04-17 17:50:20,337 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP044410.1, seqID NZ_CP044410.1 [record index 0].\n",
      "2024-04-17 17:50:20,446 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP041110.1, seqID NZ_CP041110.1 [record index 0].\n",
      "2024-04-17 17:50:20,662 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP034787.1, seqID NZ_CP034787.1 [record index 0].\n",
      "2024-04-17 17:50:20,857 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP033250.1, seqID NZ_CP033250.1 [record index 0].\n",
      "2024-04-17 17:50:20,987 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060963.1, seqID NZ_CP060963.1 [record index 0].\n",
      "2024-04-17 17:50:21,092 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP016182.2, seqID NZ_CP016182.2 [record index 0].\n",
      "2024-04-17 17:50:21,318 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP056073.1, seqID NZ_CP056073.1 [record index 0].\n",
      "2024-04-17 17:50:21,441 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060969.1, seqID NZ_CP060969.1 [record index 0].\n",
      "2024-04-17 17:50:21,637 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP043414.1, seqID NZ_CP043414.1 [record index 0].\n",
      "2024-04-17 17:50:21,755 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP059119.1, seqID NZ_CP059119.1 [record index 0].\n",
      "2024-04-17 17:50:21,878 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060974.1, seqID NZ_CP060974.1 [record index 0].\n",
      "2024-04-17 17:50:22,154 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP085619.1, seqID NZ_CP085619.1 [record index 0].\n",
      "2024-04-17 17:50:22,283 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060987.1, seqID NZ_CP060987.1 [record index 0].\n",
      "2024-04-17 17:50:22,409 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060979.1, seqID NZ_CP060979.1 [record index 0].\n",
      "2024-04-17 17:50:22,536 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060867.1, seqID NZ_CP060867.1 [record index 0].\n",
      "2024-04-17 17:50:22,657 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060880.1, seqID NZ_CP060880.1 [record index 0].\n",
      "2024-04-17 17:50:22,779 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060892.1, seqID NZ_CP060892.1 [record index 0].\n",
      "2024-04-17 17:50:22,892 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP084900.1, seqID NZ_CP084900.1 [record index 0].\n",
      "2024-04-17 17:50:23,116 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058571.1, seqID NZ_CP058571.1 [record index 0].\n",
      "2024-04-17 17:50:23,231 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP041955.1, seqID NZ_CP041955.1 [record index 0].\n",
      "2024-04-17 17:50:23,341 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP074120.1, seqID NZ_CP074120.1 [record index 0].\n",
      "2024-04-17 17:50:23,463 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060928.1, seqID NZ_CP060928.1 [record index 0].\n",
      "2024-04-17 17:50:23,592 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060938.1, seqID NZ_CP060938.1 [record index 0].\n",
      "2024-04-17 17:50:23,711 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP093226.1, seqID NZ_CP093226.1 [record index 0].\n",
      "2024-04-17 17:50:23,826 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP038453.1, seqID NZ_CP038453.1 [record index 0].\n",
      "2024-04-17 17:50:24,052 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043217.1, seqID NZ_CP043217.1 [record index 0].\n",
      "2024-04-17 17:50:24,162 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043227.1, seqID NZ_CP043227.1 [record index 0].\n",
      "2024-04-17 17:50:24,269 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086336.1, seqID NZ_CP086336.1 [record index 0].\n",
      "2024-04-17 17:50:24,420 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP069712.1, seqID NZ_CP069712.1 [record index 0].\n",
      "2024-04-17 17:50:24,637 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035865.1, seqID NZ_CP035865.1 [record index 0].\n",
      "2024-04-17 17:50:24,838 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP025573.1, seqID NZ_CP025573.1 [record index 0].\n",
      "2024-04-17 17:50:25,085 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026612.1, seqID NZ_CP026612.1 [record index 0].\n",
      "2024-04-17 17:50:25,345 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP091065.1, seqID NZ_CP091065.1 [record index 0].\n",
      "2024-04-17 17:50:25,619 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP045741.1, seqID NZ_CP045741.1 [record index 0].\n",
      "2024-04-17 17:50:25,883 [INFO - chronostrain.cli.make_db_json] - Created 14 merged markers for strain NZ_CP025520.1, seqID NZ_CP025520.1 [record index 0].\n",
      "2024-04-17 17:50:26,170 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP076470.1, seqID NZ_CP076470.1 [record index 0].\n",
      "2024-04-17 17:50:26,426 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012127.1, seqID NZ_CP012127.1 [record index 0].\n",
      "2024-04-17 17:50:26,678 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012126.1, seqID NZ_CP012126.1 [record index 0].\n",
      "2024-04-17 17:50:26,932 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP012125.1, seqID NZ_CP012125.1 [record index 0].\n",
      "2024-04-17 17:50:27,184 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_017638.1, seqID NC_017638.1 [record index 0].\n",
      "2024-04-17 17:50:27,438 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NC_017625.1, seqID NC_017625.1 [record index 0].\n",
      "2024-04-17 17:50:27,640 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP038383.1, seqID NZ_CP038383.1 [record index 0].\n",
      "2024-04-17 17:50:27,821 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP038386.1, seqID NZ_CP038386.1 [record index 0].\n",
      "2024-04-17 17:50:28,008 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP038389.1, seqID NZ_CP038389.1 [record index 0].\n",
      "2024-04-17 17:50:28,192 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP038394.1, seqID NZ_CP038394.1 [record index 0].\n",
      "2024-04-17 17:50:28,412 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP045206.1, seqID NZ_CP045206.1 [record index 0].\n",
      "2024-04-17 17:50:28,672 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP053603.1, seqID NZ_CP053603.1 [record index 0].\n",
      "2024-04-17 17:50:28,779 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP029574.1, seqID NZ_CP029574.1 [record index 0].\n",
      "2024-04-17 17:50:28,979 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010152.1, seqID NZ_CP010152.1 [record index 0].\n",
      "2024-04-17 17:50:29,182 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035312.1, seqID NZ_CP035312.1 [record index 0].\n",
      "2024-04-17 17:50:29,394 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010143.1, seqID NZ_CP010143.1 [record index 0].\n",
      "2024-04-17 17:50:29,601 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068279.1, seqID NZ_CP068279.1 [record index 0].\n",
      "2024-04-17 17:50:29,807 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068281.1, seqID NZ_CP068281.1 [record index 0].\n",
      "2024-04-17 17:50:30,014 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068283.1, seqID NZ_CP068283.1 [record index 0].\n",
      "2024-04-17 17:50:30,272 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP043213.1, seqID NZ_CP043213.1 [record index 0].\n",
      "2024-04-17 17:50:30,529 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP067994.1, seqID NZ_CP067994.1 [record index 0].\n",
      "2024-04-17 17:50:30,648 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024815.1, seqID NZ_CP024815.1 [record index 0].\n",
      "2024-04-17 17:50:30,761 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024830.1, seqID NZ_CP024830.1 [record index 0].\n",
      "2024-04-17 17:50:30,886 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP019071.1, seqID NZ_CP019071.1 [record index 0].\n",
      "2024-04-17 17:50:31,073 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053736.1, seqID NZ_CP053736.1 [record index 0].\n",
      "2024-04-17 17:50:31,272 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019245.1, seqID NZ_CP019245.1 [record index 0].\n",
      "2024-04-17 17:50:31,482 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_OU701449.1, seqID NZ_OU701449.1 [record index 0].\n",
      "2024-04-17 17:50:31,692 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_OU701452.1, seqID NZ_OU701452.1 [record index 0].\n",
      "2024-04-17 17:50:31,806 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP033158.1, seqID NZ_CP033158.1 [record index 0].\n",
      "2024-04-17 17:50:32,065 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP029238.1, seqID NZ_CP029238.1 [record index 0].\n",
      "2024-04-17 17:50:32,309 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026028.1, seqID NZ_CP026028.1 [record index 0].\n",
      "2024-04-17 17:50:32,520 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028166.1, seqID NZ_CP028166.1 [record index 0].\n",
      "2024-04-17 17:50:32,705 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042893.1, seqID NZ_CP042893.1 [record index 0].\n",
      "2024-04-17 17:50:32,895 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042973.1, seqID NZ_CP042973.1 [record index 0].\n",
      "2024-04-17 17:50:33,086 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042899.1, seqID NZ_CP042899.1 [record index 0].\n",
      "2024-04-17 17:50:33,294 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP042901.1, seqID NZ_CP042901.1 [record index 0].\n",
      "2024-04-17 17:50:33,528 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP042903.1, seqID NZ_CP042903.1 [record index 0].\n",
      "2024-04-17 17:50:33,764 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP020835.1, seqID NZ_CP020835.1 [record index 0].\n",
      "2024-04-17 17:50:33,977 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP026935.2, seqID NZ_CP026935.2 [record index 0].\n",
      "2024-04-17 17:50:34,094 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026932.2, seqID NZ_CP026932.2 [record index 0].\n",
      "2024-04-17 17:50:34,202 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026929.2, seqID NZ_CP026929.2 [record index 0].\n",
      "2024-04-17 17:50:34,462 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP064678.1, seqID NZ_CP064678.1 [record index 0].\n",
      "2024-04-17 17:50:34,720 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP064679.1, seqID NZ_CP064679.1 [record index 0].\n",
      "2024-04-17 17:50:34,978 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP064680.1, seqID NZ_CP064680.1 [record index 0].\n",
      "2024-04-17 17:50:35,196 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP073621.1, seqID NZ_CP073621.1 [record index 0].\n",
      "2024-04-17 17:50:35,383 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NC_013941.1, seqID NC_013941.1 [record index 0].\n",
      "2024-04-17 17:50:35,629 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026026.1, seqID NZ_CP026026.1 [record index 0].\n",
      "2024-04-17 17:50:35,826 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010129.1, seqID NZ_CP010129.1 [record index 0].\n",
      "2024-04-17 17:50:36,078 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP031214.1, seqID NZ_CP031214.1 [record index 0].\n",
      "2024-04-17 17:50:36,337 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP046025.1, seqID NZ_CP046025.1 [record index 0].\n",
      "2024-04-17 17:50:36,517 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010116.1, seqID NZ_CP010116.1 [record index 0].\n",
      "2024-04-17 17:50:36,774 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP064681.1, seqID NZ_CP064681.1 [record index 0].\n",
      "2024-04-17 17:50:37,030 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP064682.1, seqID NZ_CP064682.1 [record index 0].\n",
      "2024-04-17 17:50:37,284 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP069133.1, seqID NZ_CP069133.1 [record index 0].\n",
      "2024-04-17 17:50:37,538 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP069132.1, seqID NZ_CP069132.1 [record index 0].\n",
      "2024-04-17 17:50:37,747 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP059723.1, seqID NZ_CP059723.1 [record index 0].\n",
      "2024-04-17 17:50:37,944 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP063332.1, seqID NZ_CP063332.1 [record index 0].\n",
      "2024-04-17 17:50:38,141 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP093068.1, seqID NZ_CP093068.1 [record index 0].\n",
      "2024-04-17 17:50:38,398 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP029239.1, seqID NZ_CP029239.1 [record index 0].\n",
      "2024-04-17 17:50:38,598 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032986.1, seqID NZ_CP032986.1 [record index 0].\n",
      "2024-04-17 17:50:38,859 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP040643.1, seqID NZ_CP040643.1 [record index 0].\n",
      "2024-04-17 17:50:39,050 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP090456.1, seqID NZ_CP090456.1 [record index 0].\n",
      "2024-04-17 17:50:39,316 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042865.1, seqID NZ_CP042865.1 [record index 0].\n",
      "2024-04-17 17:50:39,584 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP042867.1, seqID NZ_CP042867.1 [record index 0].\n",
      "2024-04-17 17:50:39,851 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP034658.1, seqID NZ_CP034658.1 [record index 0].\n",
      "2024-04-17 17:50:39,960 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP066366.1, seqID NZ_CP066366.1 [record index 0].\n",
      "2024-04-17 17:50:40,069 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021683.1, seqID NZ_CP021683.1 [record index 0].\n",
      "2024-04-17 17:50:40,178 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021691.1, seqID NZ_CP021691.1 [record index 0].\n",
      "2024-04-17 17:50:40,292 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021736.1, seqID NZ_CP021736.1 [record index 0].\n",
      "2024-04-17 17:50:40,409 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021532.1, seqID NZ_CP021532.1 [record index 0].\n",
      "2024-04-17 17:50:40,522 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021879.1, seqID NZ_CP021879.1 [record index 0].\n",
      "2024-04-17 17:50:40,632 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021732.1, seqID NZ_CP021732.1 [record index 0].\n",
      "2024-04-17 17:50:40,743 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024859.1, seqID NZ_CP024859.1 [record index 0].\n",
      "2024-04-17 17:50:40,853 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024855.1, seqID NZ_CP024855.1 [record index 0].\n",
      "2024-04-17 17:50:40,965 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP029115.1, seqID NZ_CP029115.1 [record index 0].\n",
      "2024-04-17 17:50:41,075 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043942.1, seqID NZ_CP043942.1 [record index 0].\n",
      "2024-04-17 17:50:41,187 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043946.1, seqID NZ_CP043946.1 [record index 0].\n",
      "2024-04-17 17:50:41,405 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP005930.1, seqID NZ_CP005930.1 [record index 0].\n",
      "2024-04-17 17:50:41,594 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058355.1, seqID NZ_CP058355.1 [record index 0].\n",
      "2024-04-17 17:50:41,848 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062246.1, seqID NZ_CP062246.1 [record index 0].\n",
      "2024-04-17 17:50:42,094 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062243.1, seqID NZ_CP062243.1 [record index 0].\n",
      "2024-04-17 17:50:42,345 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062244.1, seqID NZ_CP062244.1 [record index 0].\n",
      "2024-04-17 17:50:42,592 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062245.1, seqID NZ_CP062245.1 [record index 0].\n",
      "2024-04-17 17:50:42,842 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062250.1, seqID NZ_CP062250.1 [record index 0].\n",
      "2024-04-17 17:50:43,094 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062247.1, seqID NZ_CP062247.1 [record index 0].\n",
      "2024-04-17 17:50:43,341 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062248.1, seqID NZ_CP062248.1 [record index 0].\n",
      "2024-04-17 17:50:43,592 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062249.1, seqID NZ_CP062249.1 [record index 0].\n",
      "2024-04-17 17:50:43,847 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062242.1, seqID NZ_CP062242.1 [record index 0].\n",
      "2024-04-17 17:50:44,097 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062239.1, seqID NZ_CP062239.1 [record index 0].\n",
      "2024-04-17 17:50:44,347 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062240.1, seqID NZ_CP062240.1 [record index 0].\n",
      "2024-04-17 17:50:44,591 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP062241.1, seqID NZ_CP062241.1 [record index 0].\n",
      "2024-04-17 17:50:44,845 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP037857.2, seqID NZ_CP037857.2 [record index 0].\n",
      "2024-04-17 17:50:45,029 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058302.1, seqID NZ_CP058302.1 [record index 0].\n",
      "2024-04-17 17:50:45,216 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055259.1, seqID NZ_CP055259.1 [record index 0].\n",
      "2024-04-17 17:50:45,424 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP067265.1, seqID NZ_CP067265.1 [record index 0].\n",
      "2024-04-17 17:50:45,531 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP067299.1, seqID NZ_CP067299.1 [record index 0].\n",
      "2024-04-17 17:50:45,639 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP067303.1, seqID NZ_CP067303.1 [record index 0].\n",
      "2024-04-17 17:50:45,848 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028745.1, seqID NZ_CP028745.1 [record index 0].\n",
      "2024-04-17 17:50:46,080 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040390.1, seqID NZ_CP040390.1 [record index 0].\n",
      "2024-04-17 17:50:46,290 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP028760.1, seqID NZ_CP028760.1 [record index 0].\n",
      "2024-04-17 17:50:46,542 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026342.1, seqID NZ_CP026342.1 [record index 0].\n",
      "2024-04-17 17:50:46,795 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026343.1, seqID NZ_CP026343.1 [record index 0].\n",
      "2024-04-17 17:50:46,985 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070902.1, seqID NZ_CP070902.1 [record index 0].\n",
      "2024-04-17 17:50:47,225 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP051001.1, seqID NZ_CP051001.1 [record index 0].\n",
      "2024-04-17 17:50:47,437 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP061758.1, seqID NZ_CP061758.1 [record index 0].\n",
      "2024-04-17 17:50:47,685 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026344.1, seqID NZ_CP026344.1 [record index 0].\n",
      "2024-04-17 17:50:47,930 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026345.1, seqID NZ_CP026345.1 [record index 0].\n",
      "2024-04-17 17:50:48,165 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP061754.1, seqID NZ_CP061754.1 [record index 0].\n",
      "2024-04-17 17:50:48,399 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP088131.1, seqID NZ_CP088131.1 [record index 0].\n",
      "2024-04-17 17:50:48,649 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026346.1, seqID NZ_CP026346.1 [record index 0].\n",
      "2024-04-17 17:50:48,897 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026347.1, seqID NZ_CP026347.1 [record index 0].\n",
      "2024-04-17 17:50:49,117 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023353.1, seqID NZ_CP023353.1 [record index 0].\n",
      "2024-04-17 17:50:49,227 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP025842.1, seqID NZ_CP025842.1 [record index 0].\n",
      "2024-04-17 17:50:49,475 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026348.1, seqID NZ_CP026348.1 [record index 0].\n",
      "2024-04-17 17:50:49,726 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026349.1, seqID NZ_CP026349.1 [record index 0].\n",
      "2024-04-17 17:50:49,836 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP027701.1, seqID NZ_CP027701.1 [record index 0].\n",
      "2024-04-17 17:50:50,027 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070914.1, seqID NZ_CP070914.1 [record index 0].\n",
      "2024-04-17 17:50:50,136 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP010371.1, seqID NZ_CP010371.1 [record index 0].\n",
      "2024-04-17 17:50:50,363 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP040263.1, seqID NZ_CP040263.1 [record index 0].\n",
      "2024-04-17 17:50:50,559 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP050040.1, seqID NZ_CP050040.1 [record index 0].\n",
      "2024-04-17 17:50:50,805 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026350.1, seqID NZ_CP026350.1 [record index 0].\n",
      "2024-04-17 17:50:50,921 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP021175.1, seqID NZ_CP021175.1 [record index 0].\n",
      "2024-04-17 17:50:51,175 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026351.1, seqID NZ_CP026351.1 [record index 0].\n",
      "2024-04-17 17:50:51,392 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP041284.1, seqID NZ_CP041284.1 [record index 0].\n",
      "2024-04-17 17:50:51,499 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP050382.1, seqID NZ_CP050382.1 [record index 0].\n",
      "2024-04-17 17:50:51,608 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP029973.1, seqID NZ_CP029973.1 [record index 0].\n",
      "2024-04-17 17:50:51,823 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025851.1, seqID NZ_CP025851.1 [record index 0].\n",
      "2024-04-17 17:50:51,930 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP023834.1, seqID NZ_CP023834.1 [record index 0].\n",
      "2024-04-17 17:50:52,179 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026352.1, seqID NZ_CP026352.1 [record index 0].\n",
      "2024-04-17 17:50:52,430 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026353.1, seqID NZ_CP026353.1 [record index 0].\n",
      "2024-04-17 17:50:52,678 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026354.1, seqID NZ_CP026354.1 [record index 0].\n",
      "2024-04-17 17:50:52,924 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026355.1, seqID NZ_CP026355.1 [record index 0].\n",
      "2024-04-17 17:50:53,035 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048310.1, seqID NZ_CP048310.1 [record index 0].\n",
      "2024-04-17 17:50:53,263 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025903.1, seqID NZ_CP025903.1 [record index 0].\n",
      "2024-04-17 17:50:53,508 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026359.1, seqID NZ_CP026359.1 [record index 0].\n",
      "2024-04-17 17:50:53,759 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026357.1, seqID NZ_CP026357.1 [record index 0].\n",
      "2024-04-17 17:50:54,006 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026356.1, seqID NZ_CP026356.1 [record index 0].\n",
      "2024-04-17 17:50:54,254 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026358.1, seqID NZ_CP026358.1 [record index 0].\n",
      "2024-04-17 17:50:54,477 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP038505.1, seqID NZ_CP038505.1 [record index 0].\n",
      "2024-04-17 17:50:54,713 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP026723.1, seqID NZ_CP026723.1 [record index 0].\n",
      "2024-04-17 17:50:54,817 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP027118.1, seqID NZ_CP027118.1 [record index 0].\n",
      "2024-04-17 17:50:54,994 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031833.1, seqID NZ_CP031833.1 [record index 0].\n",
      "2024-04-17 17:50:55,104 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP020514.1, seqID NZ_CP020514.1 [record index 0].\n",
      "2024-04-17 17:50:55,286 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP023286.1, seqID NZ_AP023286.1 [record index 0].\n",
      "2024-04-17 17:50:55,488 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021894.1, seqID NZ_AP021894.1 [record index 0].\n",
      "2024-04-17 17:50:55,692 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021893.1, seqID NZ_AP021893.1 [record index 0].\n",
      "2024-04-17 17:50:55,890 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP021896.1, seqID NZ_AP021896.1 [record index 0].\n",
      "2024-04-17 17:50:55,999 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP025209.2, seqID NZ_AP025209.2 [record index 0].\n",
      "2024-04-17 17:50:56,207 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP015241.1, seqID NZ_CP015241.1 [record index 0].\n",
      "2024-04-17 17:50:56,386 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP027437.1, seqID NZ_CP027437.1 [record index 0].\n",
      "2024-04-17 17:50:56,641 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026360.1, seqID NZ_CP026360.1 [record index 0].\n",
      "2024-04-17 17:50:56,887 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP026361.1, seqID NZ_CP026361.1 [record index 0].\n",
      "2024-04-17 17:50:57,070 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP046009.1, seqID NZ_CP046009.1 [record index 0].\n",
      "2024-04-17 17:50:57,292 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP046003.1, seqID NZ_CP046003.1 [record index 0].\n",
      "2024-04-17 17:50:57,507 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP048934.1, seqID NZ_CP048934.1 [record index 0].\n",
      "2024-04-17 17:50:57,615 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP072858.1, seqID NZ_CP072858.1 [record index 0].\n",
      "2024-04-17 17:50:57,798 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051158.1, seqID NZ_CP051158.1 [record index 0].\n",
      "2024-04-17 17:50:57,908 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048367.1, seqID NZ_CP048367.1 [record index 0].\n",
      "2024-04-17 17:50:58,017 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048371.1, seqID NZ_CP048371.1 [record index 0].\n",
      "2024-04-17 17:50:58,237 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032145.1, seqID NZ_CP032145.1 [record index 0].\n",
      "2024-04-17 17:50:58,536 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP076237.1, seqID NZ_CP076237.1 [record index 0].\n",
      "2024-04-17 17:50:58,758 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP023364.1, seqID NZ_CP023364.1 [record index 0].\n",
      "2024-04-17 17:50:59,036 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP063369.1, seqID NZ_CP063369.1 [record index 0].\n",
      "2024-04-17 17:50:59,252 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP009166.1, seqID NZ_CP009166.1 [record index 0].\n",
      "2024-04-17 17:50:59,360 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP023371.1, seqID NZ_CP023371.1 [record index 0].\n",
      "2024-04-17 17:50:59,471 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP023383.1, seqID NZ_CP023383.1 [record index 0].\n",
      "2024-04-17 17:50:59,730 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_CP071521.1, seqID NZ_CP071521.1 [record index 0].\n",
      "2024-04-17 17:50:59,836 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP070232.1, seqID NZ_CP070232.1 [record index 0].\n",
      "2024-04-17 17:51:00,078 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LT615377.1, seqID NZ_LT615377.1 [record index 0].\n",
      "2024-04-17 17:51:00,325 [INFO - chronostrain.cli.make_db_json] - Created 13 merged markers for strain NZ_LT615378.1, seqID NZ_LT615378.1 [record index 0].\n",
      "2024-04-17 17:51:00,442 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091427.1, seqID NZ_CP091427.1 [record index 0].\n",
      "2024-04-17 17:51:00,555 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP048330.1, seqID NZ_CP048330.1 [record index 0].\n",
      "2024-04-17 17:51:00,840 [INFO - chronostrain.cli.make_db_json] - Created 14 merged markers for strain NZ_CP084026.1, seqID NZ_CP084026.1 [record index 0].\n",
      "2024-04-17 17:51:01,045 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP067311.1, seqID NZ_CP067311.1 [record index 0].\n",
      "2024-04-17 17:51:01,153 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058661.1, seqID NZ_CP058661.1 [record index 0].\n",
      "2024-04-17 17:51:01,343 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP022215.1, seqID NZ_AP022215.1 [record index 0].\n",
      "2024-04-17 17:51:01,550 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032989.1, seqID NZ_CP032989.1 [record index 0].\n",
      "2024-04-17 17:51:01,733 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP049343.1, seqID NZ_CP049343.1 [record index 0].\n",
      "2024-04-17 17:51:01,950 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035850.1, seqID NZ_CP035850.1 [record index 0].\n",
      "2024-04-17 17:51:02,139 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP059835.1, seqID NZ_CP059835.1 [record index 0].\n",
      "2024-04-17 17:51:02,360 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091029.1, seqID NZ_CP091029.1 [record index 0].\n",
      "2024-04-17 17:51:02,573 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP091024.1, seqID NZ_CP091024.1 [record index 0].\n",
      "2024-04-17 17:51:02,796 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075649.1, seqID NZ_CP075649.1 [record index 0].\n",
      "2024-04-17 17:51:02,991 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041437.1, seqID NZ_CP041437.1 [record index 0].\n",
      "2024-04-17 17:51:03,195 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053234.1, seqID NZ_CP053234.1 [record index 0].\n",
      "2024-04-17 17:51:03,444 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP043821.1, seqID NZ_CP043821.1 [record index 0].\n",
      "2024-04-17 17:51:03,634 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP080133.1, seqID NZ_CP080133.1 [record index 0].\n",
      "2024-04-17 17:51:03,843 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP080121.1, seqID NZ_CP080121.1 [record index 0].\n",
      "2024-04-17 17:51:04,040 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075477.1, seqID NZ_CP075477.1 [record index 0].\n",
      "2024-04-17 17:51:04,234 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP075484.1, seqID NZ_CP075484.1 [record index 0].\n",
      "2024-04-17 17:51:04,429 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134080.1, seqID NZ_LR134080.1 [record index 0].\n",
      "2024-04-17 17:51:04,634 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069677.1, seqID NZ_CP069677.1 [record index 0].\n",
      "2024-04-17 17:51:04,855 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010196.1, seqID NZ_CP010196.1 [record index 0].\n",
      "2024-04-17 17:51:05,082 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010186.1, seqID NZ_CP010186.1 [record index 0].\n",
      "2024-04-17 17:51:05,304 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010221.1, seqID NZ_CP010221.1 [record index 0].\n",
      "2024-04-17 17:51:05,528 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010213.1, seqID NZ_CP010213.1 [record index 0].\n",
      "2024-04-17 17:51:05,755 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010206.1, seqID NZ_CP010206.1 [record index 0].\n",
      "2024-04-17 17:51:05,944 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP042585.1, seqID NZ_CP042585.1 [record index 0].\n",
      "2024-04-17 17:51:06,067 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091691.1, seqID NZ_CP091691.1 [record index 0].\n",
      "2024-04-17 17:51:06,192 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP091659.1, seqID NZ_CP091659.1 [record index 0].\n",
      "2024-04-17 17:51:06,421 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP083263.1, seqID NZ_CP083263.1 [record index 0].\n",
      "2024-04-17 17:51:06,621 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP054282.1, seqID NZ_CP054282.1 [record index 0].\n",
      "2024-04-17 17:51:06,800 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NC_011741.1, seqID NC_011741.1 [record index 0].\n",
      "2024-04-17 17:51:07,005 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010170.1, seqID NZ_CP010170.1 [record index 0].\n",
      "2024-04-17 17:51:07,210 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010169.1, seqID NZ_CP010169.1 [record index 0].\n",
      "2024-04-17 17:51:07,439 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028733.1, seqID NZ_CP028733.1 [record index 0].\n",
      "2024-04-17 17:51:07,636 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP025318.1, seqID NZ_CP025318.1 [record index 0].\n",
      "2024-04-17 17:51:07,868 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP069542.1, seqID NZ_CP069542.1 [record index 0].\n",
      "2024-04-17 17:51:08,083 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP046259.1, seqID NZ_CP046259.1 [record index 0].\n",
      "2024-04-17 17:51:08,191 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP073360.1, seqID NZ_CP073360.1 [record index 0].\n",
      "2024-04-17 17:51:08,407 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027255.1, seqID NZ_CP027255.1 [record index 0].\n",
      "2024-04-17 17:51:08,590 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069709.1, seqID NZ_CP069709.1 [record index 0].\n",
      "2024-04-17 17:51:08,809 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP010134.1, seqID NZ_CP010134.1 [record index 0].\n",
      "2024-04-17 17:51:09,022 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP063349.1, seqID NZ_CP063349.1 [record index 0].\n",
      "2024-04-17 17:51:09,227 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP076704.1, seqID NZ_CP076704.1 [record index 0].\n",
      "2024-04-17 17:51:09,417 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP081711.1, seqID NZ_CP081711.1 [record index 0].\n",
      "2024-04-17 17:51:09,650 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028756.1, seqID NZ_CP028756.1 [record index 0].\n",
      "2024-04-17 17:51:09,873 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028763.1, seqID NZ_CP028763.1 [record index 0].\n",
      "2024-04-17 17:51:10,059 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP050047.1, seqID NZ_CP050047.1 [record index 0].\n",
      "2024-04-17 17:51:10,241 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP016404.1, seqID NZ_CP016404.1 [record index 0].\n",
      "2024-04-17 17:51:10,433 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027550.1, seqID NZ_CP027550.1 [record index 0].\n",
      "2024-04-17 17:51:10,669 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP027355.1, seqID NZ_CP027355.1 [record index 0].\n",
      "2024-04-17 17:51:10,863 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027312.1, seqID NZ_CP027312.1 [record index 0].\n",
      "2024-04-17 17:51:11,016 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP046000.1, seqID NZ_CP046000.1 [record index 0].\n",
      "2024-04-17 17:51:11,248 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024134.1, seqID NZ_CP024134.1 [record index 0].\n",
      "2024-04-17 17:51:11,477 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024131.1, seqID NZ_CP024131.1 [record index 0].\n",
      "2024-04-17 17:51:11,602 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NC_007606.1, seqID NC_007606.1 [record index 0].\n",
      "2024-04-17 17:51:11,730 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026786.1, seqID NZ_CP026786.1 [record index 0].\n",
      "2024-04-17 17:51:11,854 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP061527.1, seqID NZ_CP061527.1 [record index 0].\n",
      "2024-04-17 17:51:11,984 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024466.1, seqID NZ_CP024466.1 [record index 0].\n",
      "2024-04-17 17:51:12,112 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026774.1, seqID NZ_CP026774.1 [record index 0].\n",
      "2024-04-17 17:51:12,238 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026784.1, seqID NZ_CP026784.1 [record index 0].\n",
      "2024-04-17 17:51:12,366 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026777.1, seqID NZ_CP026777.1 [record index 0].\n",
      "2024-04-17 17:51:12,489 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026780.1, seqID NZ_CP026780.1 [record index 0].\n",
      "2024-04-17 17:51:12,620 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026782.1, seqID NZ_CP026782.1 [record index 0].\n",
      "2024-04-17 17:51:12,746 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP026878.1, seqID NZ_CP026878.1 [record index 0].\n",
      "2024-04-17 17:51:12,943 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP024090.1, seqID NZ_CP024090.1 [record index 0].\n",
      "2024-04-17 17:51:13,130 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP051753.1, seqID NZ_CP051753.1 [record index 0].\n",
      "2024-04-17 17:51:13,324 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP010238.1, seqID NZ_CP010238.1 [record index 0].\n",
      "2024-04-17 17:51:13,523 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR882057.1, seqID NZ_LR882057.1 [record index 0].\n",
      "2024-04-17 17:51:13,719 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP006632.1, seqID NZ_CP006632.1 [record index 0].\n",
      "2024-04-17 17:51:13,919 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LT906474.1, seqID NZ_LT906474.1 [record index 0].\n",
      "2024-04-17 17:51:14,135 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020543.1, seqID NZ_CP020543.1 [record index 0].\n",
      "2024-04-17 17:51:14,331 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031105.1, seqID NZ_CP031105.1 [record index 0].\n",
      "2024-04-17 17:51:14,560 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028748.1, seqID NZ_CP028748.1 [record index 0].\n",
      "2024-04-17 17:51:14,742 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP025849.1, seqID NZ_CP025849.1 [record index 0].\n",
      "2024-04-17 17:51:14,931 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027579.1, seqID NZ_CP027579.1 [record index 0].\n",
      "2024-04-17 17:51:15,123 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP019256.1, seqID NZ_CP019256.1 [record index 0].\n",
      "2024-04-17 17:51:15,327 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP037903.1, seqID NZ_CP037903.1 [record index 0].\n",
      "2024-04-17 17:51:15,572 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP093239.1, seqID NZ_CP093239.1 [record index 0].\n",
      "2024-04-17 17:51:15,800 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP056159.1, seqID NZ_CP056159.1 [record index 0].\n",
      "2024-04-17 17:51:16,008 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP056697.1, seqID NZ_CP056697.1 [record index 0].\n",
      "2024-04-17 17:51:16,186 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058207.1, seqID NZ_CP058207.1 [record index 0].\n",
      "2024-04-17 17:51:16,393 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058209.1, seqID NZ_CP058209.1 [record index 0].\n",
      "2024-04-17 17:51:16,598 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134270.1, seqID NZ_LR134270.1 [record index 0].\n",
      "2024-04-17 17:51:16,788 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134340.1, seqID NZ_LR134340.1 [record index 0].\n",
      "2024-04-17 17:51:16,974 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP025979.1, seqID NZ_CP025979.1 [record index 0].\n",
      "2024-04-17 17:51:17,183 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP090448.1, seqID NZ_CP090448.1 [record index 0].\n",
      "2024-04-17 17:51:17,356 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP057266.1, seqID NZ_CP057266.1 [record index 0].\n",
      "2024-04-17 17:51:17,543 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP057657.1, seqID NZ_CP057657.1 [record index 0].\n",
      "2024-04-17 17:51:17,733 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP081881.1, seqID NZ_CP081881.1 [record index 0].\n",
      "2024-04-17 17:51:17,925 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP021933.1, seqID NZ_AP021933.1 [record index 0].\n",
      "2024-04-17 17:51:18,123 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP066806.1, seqID NZ_CP066806.1 [record index 0].\n",
      "2024-04-17 17:51:18,333 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP055019.1, seqID NZ_CP055019.1 [record index 0].\n",
      "2024-04-17 17:51:18,548 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058748.1, seqID NZ_CP058748.1 [record index 0].\n",
      "2024-04-17 17:51:18,762 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP058879.1, seqID NZ_CP058879.1 [record index 0].\n",
      "2024-04-17 17:51:18,968 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP068823.1, seqID NZ_CP068823.1 [record index 0].\n",
      "2024-04-17 17:51:19,163 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP067241.1, seqID NZ_CP067241.1 [record index 0].\n",
      "2024-04-17 17:51:19,387 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_AP017620.1, seqID NZ_AP017620.1 [record index 0].\n",
      "2024-04-17 17:51:19,587 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_AP017617.1, seqID NZ_AP017617.1 [record index 0].\n",
      "2024-04-17 17:51:19,778 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054556.1, seqID NZ_CP054556.1 [record index 0].\n",
      "2024-04-17 17:51:19,997 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP091665.1, seqID NZ_CP091665.1 [record index 0].\n",
      "2024-04-17 17:51:20,181 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP069501.1, seqID NZ_CP069501.1 [record index 0].\n",
      "2024-04-17 17:51:20,379 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP031293.1, seqID NZ_CP031293.1 [record index 0].\n",
      "2024-04-17 17:51:20,569 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP024826.1, seqID NZ_CP024826.1 [record index 0].\n",
      "2024-04-17 17:51:20,752 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053851.1, seqID NZ_CP053851.1 [record index 0].\n",
      "2024-04-17 17:51:20,944 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP053852.1, seqID NZ_CP053852.1 [record index 0].\n",
      "2024-04-17 17:51:21,142 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP041996.1, seqID NZ_CP041996.1 [record index 0].\n",
      "2024-04-17 17:51:21,361 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP029747.1, seqID NZ_CP029747.1 [record index 0].\n",
      "2024-04-17 17:51:21,564 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP064847.1, seqID NZ_CP064847.1 [record index 0].\n",
      "2024-04-17 17:51:21,785 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP054388.1, seqID NZ_CP054388.1 [record index 0].\n",
      "2024-04-17 17:51:21,994 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP056165.1, seqID NZ_CP056165.1 [record index 0].\n",
      "2024-04-17 17:51:22,215 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_011750.1, seqID NC_011750.1 [record index 0].\n",
      "2024-04-17 17:51:22,446 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_017646.1, seqID NC_017646.1 [record index 0].\n",
      "2024-04-17 17:51:22,683 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP032204.1, seqID NZ_CP032204.1 [record index 0].\n",
      "2024-04-17 17:51:22,870 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP027140.1, seqID NZ_CP027140.1 [record index 0].\n",
      "2024-04-17 17:51:23,064 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP040443.1, seqID NZ_CP040443.1 [record index 0].\n",
      "2024-04-17 17:51:23,186 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP055070.1, seqID NZ_CP055070.1 [record index 0].\n",
      "2024-04-17 17:51:23,308 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP055075.1, seqID NZ_CP055075.1 [record index 0].\n",
      "2024-04-17 17:51:23,429 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP055037.1, seqID NZ_CP055037.1 [record index 0].\n",
      "2024-04-17 17:51:23,548 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058853.1, seqID NZ_CP058853.1 [record index 0].\n",
      "2024-04-17 17:51:23,665 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP058826.1, seqID NZ_CP058826.1 [record index 0].\n",
      "2024-04-17 17:51:23,808 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP055000.1, seqID NZ_CP055000.1 [record index 0].\n",
      "2024-04-17 17:51:24,018 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035882.1, seqID NZ_CP035882.1 [record index 0].\n",
      "2024-04-17 17:51:24,154 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP092452.1, seqID NZ_CP092452.1 [record index 0].\n",
      "2024-04-17 17:51:24,275 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP092449.1, seqID NZ_CP092449.1 [record index 0].\n",
      "2024-04-17 17:51:24,381 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP062851.1, seqID NZ_CP062851.1 [record index 0].\n",
      "2024-04-17 17:51:24,487 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP062853.1, seqID NZ_CP062853.1 [record index 0].\n",
      "2024-04-17 17:51:24,592 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR881940.1, seqID NZ_LR881940.1 [record index 0].\n",
      "2024-04-17 17:51:24,699 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP006636.1, seqID NZ_CP006636.1 [record index 0].\n",
      "2024-04-17 17:51:24,804 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP075472.1, seqID NZ_CP075472.1 [record index 0].\n",
      "2024-04-17 17:51:25,003 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_LR134000.1, seqID NZ_LR134000.1 [record index 0].\n",
      "2024-04-17 17:51:25,211 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP047658.1, seqID NZ_CP047658.1 [record index 0].\n",
      "2024-04-17 17:51:25,325 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP092639.1, seqID NZ_CP092639.1 [record index 0].\n",
      "2024-04-17 17:51:25,427 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NC_009800.1, seqID NC_009800.1 [record index 0].\n",
      "2024-04-17 17:51:25,532 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP010163.1, seqID NZ_CP010163.1 [record index 0].\n",
      "2024-04-17 17:51:25,638 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071436.1, seqID NZ_CP071436.1 [record index 0].\n",
      "2024-04-17 17:51:25,841 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP022959.1, seqID NZ_CP022959.1 [record index 0].\n",
      "2024-04-17 17:51:26,046 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NC_010468.1, seqID NC_010468.1 [record index 0].\n",
      "2024-04-17 17:51:26,251 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP020058.1, seqID NZ_CP020058.1 [record index 0].\n",
      "2024-04-17 17:51:26,433 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070930.1, seqID NZ_CP070930.1 [record index 0].\n",
      "2024-04-17 17:51:26,536 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP038180.1, seqID NZ_CP038180.1 [record index 0].\n",
      "2024-04-17 17:51:26,653 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP019280.1, seqID NZ_CP019280.1 [record index 0].\n",
      "2024-04-17 17:51:26,766 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP023377.1, seqID NZ_CP023377.1 [record index 0].\n",
      "2024-04-17 17:51:26,994 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP058780.1, seqID NZ_CP058780.1 [record index 0].\n",
      "2024-04-17 17:51:27,212 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP035836.1, seqID NZ_CP035836.1 [record index 0].\n",
      "2024-04-17 17:51:27,406 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035855.1, seqID NZ_CP035855.1 [record index 0].\n",
      "2024-04-17 17:51:27,665 [INFO - chronostrain.cli.make_db_json] - Created 12 merged markers for strain NZ_CP062876.1, seqID NZ_CP062876.1 [record index 0].\n",
      "2024-04-17 17:51:27,850 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR134236.1, seqID NZ_LR134236.1 [record index 0].\n",
      "2024-04-17 17:51:28,030 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP070103.1, seqID NZ_CP070103.1 [record index 0].\n",
      "2024-04-17 17:51:28,230 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP069710.1, seqID NZ_CP069710.1 [record index 0].\n",
      "2024-04-17 17:51:28,401 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP069707.1, seqID NZ_CP069707.1 [record index 0].\n",
      "2024-04-17 17:51:28,606 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP081698.1, seqID NZ_CP081698.1 [record index 0].\n",
      "2024-04-17 17:51:28,831 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028746.1, seqID NZ_CP028746.1 [record index 0].\n",
      "2024-04-17 17:51:29,050 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP028752.1, seqID NZ_CP028752.1 [record index 0].\n",
      "2024-04-17 17:51:29,267 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP035826.1, seqID NZ_CP035826.1 [record index 0].\n",
      "2024-04-17 17:51:29,484 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025859.1, seqID NZ_CP025859.1 [record index 0].\n",
      "2024-04-17 17:51:29,700 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP025840.1, seqID NZ_CP025840.1 [record index 0].\n",
      "2024-04-17 17:51:29,956 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP013662.1, seqID NZ_CP013662.1 [record index 0].\n",
      "2024-04-17 17:51:30,117 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP053045.1, seqID NZ_CP053045.1 [record index 0].\n",
      "2024-04-17 17:51:30,304 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP086604.1, seqID NZ_CP086604.1 [record index 0].\n",
      "2024-04-17 17:51:30,499 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_LR882050.1, seqID NZ_LR882050.1 [record index 0].\n",
      "2024-04-17 17:51:30,715 [INFO - chronostrain.cli.make_db_json] - Created 10 merged markers for strain NZ_CP067245.1, seqID NZ_CP067245.1 [record index 0].\n",
      "2024-04-17 17:51:30,865 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057692.1, seqID NZ_CP057692.1 [record index 0].\n",
      "2024-04-17 17:51:31,003 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP077242.1, seqID NZ_CP077242.1 [record index 0].\n",
      "2024-04-17 17:51:31,150 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP042945.1, seqID NZ_CP042945.1 [record index 0].\n",
      "2024-04-17 17:51:31,299 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057210.1, seqID NZ_CP057210.1 [record index 0].\n",
      "2024-04-17 17:51:31,447 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057215.1, seqID NZ_CP057215.1 [record index 0].\n",
      "2024-04-17 17:51:31,598 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057243.1, seqID NZ_CP057243.1 [record index 0].\n",
      "2024-04-17 17:51:31,729 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP040805.1, seqID NZ_CP040805.1 [record index 0].\n",
      "2024-04-17 17:51:31,858 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_AP014856.1, seqID NZ_AP014856.1 [record index 0].\n",
      "2024-04-17 17:51:31,975 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034166.1, seqID NZ_CP034166.1 [record index 0].\n",
      "2024-04-17 17:51:32,117 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP030787.2, seqID NZ_CP030787.2 [record index 0].\n",
      "2024-04-17 17:51:32,240 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP030783.2, seqID NZ_CP030783.2 [record index 0].\n",
      "2024-04-17 17:51:32,356 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034162.1, seqID NZ_CP034162.1 [record index 0].\n",
      "2024-04-17 17:51:32,459 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP070290.1, seqID NZ_CP070290.1 [record index 0].\n",
      "2024-04-17 17:51:32,579 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_AP014855.1, seqID NZ_AP014855.1 [record index 0].\n",
      "2024-04-17 17:51:32,687 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP014857.1, seqID NZ_AP014857.1 [record index 0].\n",
      "2024-04-17 17:51:32,815 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP025317.1, seqID NZ_CP025317.1 [record index 0].\n",
      "2024-04-17 17:51:32,945 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070281.2, seqID NZ_CP070281.2 [record index 0].\n",
      "2024-04-17 17:51:33,074 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070284.2, seqID NZ_CP070284.2 [record index 0].\n",
      "2024-04-17 17:51:33,205 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070287.2, seqID NZ_CP070287.2 [record index 0].\n",
      "2024-04-17 17:51:33,312 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP043271.1, seqID NZ_CP043271.1 [record index 0].\n",
      "2024-04-17 17:51:33,433 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP034213.1, seqID NZ_CP034213.1 [record index 0].\n",
      "2024-04-17 17:51:33,545 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP025676.1, seqID NZ_CP025676.1 [record index 0].\n",
      "2024-04-17 17:51:33,657 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP024282.1, seqID NZ_CP024282.1 [record index 0].\n",
      "2024-04-17 17:51:33,800 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP030781.1, seqID NZ_CP030781.1 [record index 0].\n",
      "2024-04-17 17:51:33,936 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP030778.2, seqID NZ_CP030778.2 [record index 0].\n",
      "2024-04-17 17:51:34,065 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP056961.1, seqID NZ_CP056961.1 [record index 0].\n",
      "2024-04-17 17:51:34,205 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP056968.1, seqID NZ_CP056968.1 [record index 0].\n",
      "2024-04-17 17:51:34,333 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP057093.1, seqID NZ_CP057093.1 [record index 0].\n",
      "2024-04-17 17:51:34,482 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057357.1, seqID NZ_CP057357.1 [record index 0].\n",
      "2024-04-17 17:51:34,618 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070292.1, seqID NZ_CP070292.1 [record index 0].\n",
      "2024-04-17 17:51:34,753 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070296.1, seqID NZ_CP070296.1 [record index 0].\n",
      "2024-04-17 17:51:34,885 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP007025.1, seqID NZ_CP007025.1 [record index 0].\n",
      "2024-04-17 17:51:35,028 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP056916.1, seqID NZ_CP056916.1 [record index 0].\n",
      "2024-04-17 17:51:35,169 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP056936.1, seqID NZ_CP056936.1 [record index 0].\n",
      "2024-04-17 17:51:35,311 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP056951.1, seqID NZ_CP056951.1 [record index 0].\n",
      "2024-04-17 17:51:35,450 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057101.1, seqID NZ_CP057101.1 [record index 0].\n",
      "2024-04-17 17:51:35,593 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057220.1, seqID NZ_CP057220.1 [record index 0].\n",
      "2024-04-17 17:51:35,735 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057229.1, seqID NZ_CP057229.1 [record index 0].\n",
      "2024-04-17 17:51:35,878 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057238.1, seqID NZ_CP057238.1 [record index 0].\n",
      "2024-04-17 17:51:36,014 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057256.1, seqID NZ_CP057256.1 [record index 0].\n",
      "2024-04-17 17:51:36,156 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057311.1, seqID NZ_CP057311.1 [record index 0].\n",
      "2024-04-17 17:51:36,304 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP055675.1, seqID NZ_CP055675.1 [record index 0].\n",
      "2024-04-17 17:51:36,442 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057458.1, seqID NZ_CP057458.1 [record index 0].\n",
      "2024-04-17 17:51:36,582 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057481.1, seqID NZ_CP057481.1 [record index 0].\n",
      "2024-04-17 17:51:36,719 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057504.1, seqID NZ_CP057504.1 [record index 0].\n",
      "2024-04-17 17:51:36,855 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057505.1, seqID NZ_CP057505.1 [record index 0].\n",
      "2024-04-17 17:51:36,995 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057561.1, seqID NZ_CP057561.1 [record index 0].\n",
      "2024-04-17 17:51:37,137 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057565.1, seqID NZ_CP057565.1 [record index 0].\n",
      "2024-04-17 17:51:37,278 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057896.1, seqID NZ_CP057896.1 [record index 0].\n",
      "2024-04-17 17:51:37,420 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP057918.1, seqID NZ_CP057918.1 [record index 0].\n",
      "2024-04-17 17:51:37,567 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP058057.1, seqID NZ_CP058057.1 [record index 0].\n",
      "2024-04-17 17:51:37,709 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP083638.1, seqID NZ_CP083638.1 [record index 0].\n",
      "2024-04-17 17:51:37,850 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP066085.1, seqID NZ_CP066085.1 [record index 0].\n",
      "2024-04-17 17:51:37,975 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070873.1, seqID NZ_CP070873.1 [record index 0].\n",
      "2024-04-17 17:51:38,152 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP070954.1, seqID NZ_CP070954.1 [record index 0].\n",
      "2024-04-17 17:51:38,285 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP070951.1, seqID NZ_CP070951.1 [record index 0].\n",
      "2024-04-17 17:51:38,403 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP042942.1, seqID NZ_CP042942.1 [record index 0].\n",
      "2024-04-17 17:51:38,520 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP042946.1, seqID NZ_CP042946.1 [record index 0].\n",
      "2024-04-17 17:51:38,661 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NC_011740.1, seqID NC_011740.1 [record index 0].\n",
      "2024-04-17 17:51:38,794 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP079884.1, seqID NZ_CP079884.1 [record index 0].\n",
      "2024-04-17 17:51:38,931 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP079891.1, seqID NZ_CP079891.1 [record index 0].\n",
      "2024-04-17 17:51:39,081 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP034212.1, seqID NZ_CP034212.1 [record index 0].\n",
      "2024-04-17 17:51:39,245 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP055599.1, seqID NZ_CP055599.1 [record index 0].\n",
      "2024-04-17 17:51:39,430 [INFO - chronostrain.cli.make_db_json] - Created 9 merged markers for strain NZ_CP057104.1, seqID NZ_CP057104.1 [record index 0].\n",
      "2024-04-17 17:51:39,592 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP057463.1, seqID NZ_CP057463.1 [record index 0].\n",
      "2024-04-17 17:51:39,751 [INFO - chronostrain.cli.make_db_json] - Created 8 merged markers for strain NZ_CP057681.1, seqID NZ_CP057681.1 [record index 0].\n",
      "2024-04-17 17:51:39,832 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043225.1, seqID NZ_CP043225.1 [record index 0].\n",
      "2024-04-17 17:51:39,915 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043226.1, seqID NZ_CP043226.1 [record index 0].\n",
      "2024-04-17 17:51:40,000 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079939.1, seqID NZ_CP079939.1 [record index 0].\n",
      "2024-04-17 17:51:40,087 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032841.1, seqID NZ_CP032841.1 [record index 0].\n",
      "2024-04-17 17:51:40,178 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063224.1, seqID NZ_CP063224.1 [record index 0].\n",
      "2024-04-17 17:51:40,275 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056663.1, seqID NZ_CP056663.1 [record index 0].\n",
      "2024-04-17 17:51:40,372 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056676.1, seqID NZ_CP056676.1 [record index 0].\n",
      "2024-04-17 17:51:40,462 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056691.1, seqID NZ_CP056691.1 [record index 0].\n",
      "2024-04-17 17:51:40,549 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056455.1, seqID NZ_CP056455.1 [record index 0].\n",
      "2024-04-17 17:51:40,643 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056719.1, seqID NZ_CP056719.1 [record index 0].\n",
      "2024-04-17 17:51:40,732 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056727.1, seqID NZ_CP056727.1 [record index 0].\n",
      "2024-04-17 17:51:40,826 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058128.1, seqID NZ_CP058128.1 [record index 0].\n",
      "2024-04-17 17:51:40,918 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058146.1, seqID NZ_CP058146.1 [record index 0].\n",
      "2024-04-17 17:51:41,006 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058149.1, seqID NZ_CP058149.1 [record index 0].\n",
      "2024-04-17 17:51:41,076 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058157.1, seqID NZ_CP058157.1 [record index 0].\n",
      "2024-04-17 17:51:41,143 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP059419.1, seqID NZ_CP059419.1 [record index 0].\n",
      "2024-04-17 17:51:41,229 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP049046.1, seqID NZ_CP049046.1 [record index 0].\n",
      "2024-04-17 17:51:41,321 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090792.1, seqID NZ_CP090792.1 [record index 0].\n",
      "2024-04-17 17:51:41,405 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044107.1, seqID NZ_CP044107.1 [record index 0].\n",
      "2024-04-17 17:51:41,499 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP068215.1, seqID NZ_CP068215.1 [record index 0].\n",
      "2024-04-17 17:51:41,604 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP047570.1, seqID NZ_CP047570.1 [record index 0].\n",
      "2024-04-17 17:51:41,702 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP048696.1, seqID NZ_CP048696.1 [record index 0].\n",
      "2024-04-17 17:51:41,800 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP049021.1, seqID NZ_CP049021.1 [record index 0].\n",
      "2024-04-17 17:51:41,909 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066094.1, seqID NZ_CP066094.1 [record index 0].\n",
      "2024-04-17 17:51:41,997 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047736.1, seqID NZ_CP047736.1 [record index 0].\n",
      "2024-04-17 17:51:42,091 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047740.1, seqID NZ_CP047740.1 [record index 0].\n",
      "2024-04-17 17:51:42,184 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047757.1, seqID NZ_CP047757.1 [record index 0].\n",
      "2024-04-17 17:51:42,276 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LS999206.1, seqID NZ_LS999206.1 [record index 0].\n",
      "2024-04-17 17:51:42,372 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043853.1, seqID NZ_CP043853.1 [record index 0].\n",
      "2024-04-17 17:51:42,463 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011650.1, seqID NZ_CP011650.1 [record index 0].\n",
      "2024-04-17 17:51:42,553 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011584.1, seqID NZ_CP011584.1 [record index 0].\n",
      "2024-04-17 17:51:42,644 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011581.1, seqID NZ_CP011581.1 [record index 0].\n",
      "2024-04-17 17:51:42,735 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011572.1, seqID NZ_CP011572.1 [record index 0].\n",
      "2024-04-17 17:51:42,837 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011662.1, seqID NZ_CP011662.1 [record index 0].\n",
      "2024-04-17 17:51:42,932 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030347.1, seqID NZ_CP030347.1 [record index 0].\n",
      "2024-04-17 17:51:43,028 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085774.1, seqID NZ_CP085774.1 [record index 0].\n",
      "2024-04-17 17:51:43,115 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085780.1, seqID NZ_CP085780.1 [record index 0].\n",
      "2024-04-17 17:51:43,209 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP010376.2, seqID NZ_CP010376.2 [record index 0].\n",
      "2024-04-17 17:51:43,300 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031565.1, seqID NZ_CP031565.1 [record index 0].\n",
      "2024-04-17 17:51:43,396 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047965.1, seqID NZ_CP047965.1 [record index 0].\n",
      "2024-04-17 17:51:43,465 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP049290.1, seqID NZ_CP049290.1 [record index 0].\n",
      "2024-04-17 17:51:43,557 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP008823.1, seqID NZ_CP008823.1 [record index 0].\n",
      "2024-04-17 17:51:43,645 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077659.1, seqID NZ_CP077659.1 [record index 0].\n",
      "2024-04-17 17:51:43,739 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021749.1, seqID NZ_CP021749.1 [record index 0].\n",
      "2024-04-17 17:51:43,833 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029716.1, seqID NZ_CP029716.1 [record index 0].\n",
      "2024-04-17 17:51:43,926 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020053.1, seqID NZ_CP020053.1 [record index 0].\n",
      "2024-04-17 17:51:44,029 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026719.1, seqID NZ_CP026719.1 [record index 0].\n",
      "2024-04-17 17:51:44,118 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP072945.1, seqID NZ_CP072945.1 [record index 0].\n",
      "2024-04-17 17:51:44,209 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP072953.1, seqID NZ_CP072953.1 [record index 0].\n",
      "2024-04-17 17:51:44,302 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP072961.1, seqID NZ_CP072961.1 [record index 0].\n",
      "2024-04-17 17:51:44,370 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056118.1, seqID NZ_CP056118.1 [record index 0].\n",
      "2024-04-17 17:51:44,460 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024812.1, seqID NZ_CP024812.1 [record index 0].\n",
      "2024-04-17 17:51:44,549 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP072970.1, seqID NZ_CP072970.1 [record index 0].\n",
      "2024-04-17 17:51:44,622 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP059711.1, seqID NZ_CP059711.1 [record index 0].\n",
      "2024-04-17 17:51:44,709 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP049188.1, seqID NZ_CP049188.1 [record index 0].\n",
      "2024-04-17 17:51:44,802 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP049192.1, seqID NZ_CP049192.1 [record index 0].\n",
      "2024-04-17 17:51:44,892 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090796.1, seqID NZ_CP090796.1 [record index 0].\n",
      "2024-04-17 17:51:44,993 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043382.1, seqID NZ_CP043382.1 [record index 0].\n",
      "2024-04-17 17:51:45,081 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP036310.1, seqID NZ_CP036310.1 [record index 0].\n",
      "2024-04-17 17:51:45,173 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031726.1, seqID NZ_CP031726.1 [record index 0].\n",
      "2024-04-17 17:51:45,268 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP073771.1, seqID NZ_CP073771.1 [record index 0].\n",
      "2024-04-17 17:51:45,343 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP035385.1, seqID NZ_CP035385.1 [record index 0].\n",
      "2024-04-17 17:51:45,418 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058172.1, seqID NZ_CP058172.1 [record index 0].\n",
      "2024-04-17 17:51:45,510 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058191.1, seqID NZ_CP058191.1 [record index 0].\n",
      "2024-04-17 17:51:45,603 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029246.1, seqID NZ_CP029246.1 [record index 0].\n",
      "2024-04-17 17:51:45,697 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024908.1, seqID NZ_CP024908.1 [record index 0].\n",
      "2024-04-17 17:51:45,801 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066553.1, seqID NZ_CP066553.1 [record index 0].\n",
      "2024-04-17 17:51:45,889 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP061744.1, seqID NZ_CP061744.1 [record index 0].\n",
      "2024-04-17 17:51:45,953 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR134164.1, seqID NZ_LR134164.1 [record index 0].\n",
      "2024-04-17 17:51:46,041 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP065939.1, seqID NZ_CP065939.1 [record index 0].\n",
      "2024-04-17 17:51:46,131 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090909.1, seqID NZ_CP090909.1 [record index 0].\n",
      "2024-04-17 17:51:46,205 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033102.1, seqID NZ_CP033102.1 [record index 0].\n",
      "2024-04-17 17:51:46,292 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054411.1, seqID NZ_CP054411.1 [record index 0].\n",
      "2024-04-17 17:51:46,380 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054410.1, seqID NZ_CP054410.1 [record index 0].\n",
      "2024-04-17 17:51:46,464 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064113.1, seqID NZ_CP064113.1 [record index 0].\n",
      "2024-04-17 17:51:46,553 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP071876.1, seqID NZ_CP071876.1 [record index 0].\n",
      "2024-04-17 17:51:46,642 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077392.1, seqID NZ_CP077392.1 [record index 0].\n",
      "2024-04-17 17:51:46,727 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047765.1, seqID NZ_CP047765.1 [record index 0].\n",
      "2024-04-17 17:51:46,800 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053190.1, seqID NZ_CP053190.1 [record index 0].\n",
      "2024-04-17 17:51:46,875 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP008905.1, seqID NZ_CP008905.1 [record index 0].\n",
      "2024-04-17 17:51:46,951 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP008897.1, seqID NZ_CP008897.1 [record index 0].\n",
      "2024-04-17 17:51:47,024 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP080470.1, seqID NZ_CP080470.1 [record index 0].\n",
      "2024-04-17 17:51:47,109 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053103.1, seqID NZ_CP053103.1 [record index 0].\n",
      "2024-04-17 17:51:47,206 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046271.1, seqID NZ_CP046271.1 [record index 0].\n",
      "2024-04-17 17:51:47,277 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017186.1, seqID NZ_CP017186.1 [record index 0].\n",
      "2024-04-17 17:51:47,362 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP042488.1, seqID NZ_CP042488.1 [record index 0].\n",
      "2024-04-17 17:51:47,450 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045611.1, seqID NZ_CP045611.1 [record index 0].\n",
      "2024-04-17 17:51:47,539 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085743.1, seqID NZ_CP085743.1 [record index 0].\n",
      "2024-04-17 17:51:47,625 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085757.1, seqID NZ_CP085757.1 [record index 0].\n",
      "2024-04-17 17:51:47,709 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021167.1, seqID NZ_CP021167.1 [record index 0].\n",
      "2024-04-17 17:51:47,798 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064656.1, seqID NZ_CP064656.1 [record index 0].\n",
      "2024-04-17 17:51:47,901 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012165.1, seqID NZ_CP012165.1 [record index 0].\n",
      "2024-04-17 17:51:47,999 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP010384.1, seqID NZ_CP010384.1 [record index 0].\n",
      "2024-04-17 17:51:48,085 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021162.1, seqID NZ_CP021162.1 [record index 0].\n",
      "2024-04-17 17:51:48,184 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030076.1, seqID NZ_CP030076.1 [record index 0].\n",
      "2024-04-17 17:51:48,259 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP068287.1, seqID NZ_CP068287.1 [record index 0].\n",
      "2024-04-17 17:51:48,332 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045312.1, seqID NZ_CP045312.1 [record index 0].\n",
      "2024-04-17 17:51:48,399 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP009854.1, seqID NZ_CP009854.1 [record index 0].\n",
      "2024-04-17 17:51:48,477 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077660.1, seqID NZ_CP077660.1 [record index 0].\n",
      "2024-04-17 17:51:48,561 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP091486.1, seqID NZ_CP091486.1 [record index 0].\n",
      "2024-04-17 17:51:48,640 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP091496.1, seqID NZ_CP091496.1 [record index 0].\n",
      "2024-04-17 17:51:48,724 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077661.1, seqID NZ_CP077661.1 [record index 0].\n",
      "2024-04-17 17:51:48,817 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033466.1, seqID NZ_CP033466.1 [record index 0].\n",
      "2024-04-17 17:51:48,908 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021902.1, seqID NZ_CP021902.1 [record index 0].\n",
      "2024-04-17 17:51:48,994 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027604.1, seqID NZ_CP027604.1 [record index 0].\n",
      "2024-04-17 17:51:49,086 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021776.1, seqID NZ_CP021776.1 [record index 0].\n",
      "2024-04-17 17:51:49,186 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018814.1, seqID NZ_CP018814.1 [record index 0].\n",
      "2024-04-17 17:51:49,274 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP052870.1, seqID NZ_CP052870.1 [record index 0].\n",
      "2024-04-17 17:51:49,367 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020528.1, seqID NZ_CP020528.1 [record index 0].\n",
      "2024-04-17 17:51:49,454 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP087880.1, seqID NZ_CP087880.1 [record index 0].\n",
      "2024-04-17 17:51:49,537 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043438.2, seqID NZ_CP043438.2 [record index 0].\n",
      "2024-04-17 17:51:49,626 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP087390.1, seqID NZ_CP087390.1 [record index 0].\n",
      "2024-04-17 17:51:49,717 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031571.1, seqID NZ_CP031571.1 [record index 0].\n",
      "2024-04-17 17:51:49,807 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041733.1, seqID NZ_CP041733.1 [record index 0].\n",
      "2024-04-17 17:51:49,899 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019889.1, seqID NZ_CP019889.1 [record index 0].\n",
      "2024-04-17 17:51:49,995 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP059422.1, seqID NZ_CP059422.1 [record index 0].\n",
      "2024-04-17 17:51:50,086 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP059414.1, seqID NZ_CP059414.1 [record index 0].\n",
      "2024-04-17 17:51:50,172 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP042566.1, seqID NZ_CP042566.1 [record index 0].\n",
      "2024-04-17 17:51:50,260 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP042540.1, seqID NZ_CP042540.1 [record index 0].\n",
      "2024-04-17 17:51:50,358 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029126.1, seqID NZ_CP029126.1 [record index 0].\n",
      "2024-04-17 17:51:50,449 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031574.1, seqID NZ_CP031574.1 [record index 0].\n",
      "2024-04-17 17:51:50,543 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085748.1, seqID NZ_CP085748.1 [record index 0].\n",
      "2024-04-17 17:51:50,640 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085753.1, seqID NZ_CP085753.1 [record index 0].\n",
      "2024-04-17 17:51:50,743 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012167.1, seqID NZ_CP012167.1 [record index 0].\n",
      "2024-04-17 17:51:50,852 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP045450.1, seqID NZ_CP045450.1 [record index 0].\n",
      "2024-04-17 17:51:50,919 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP091319.1, seqID NZ_CP091319.1 [record index 0].\n",
      "2024-04-17 17:51:50,984 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090192.1, seqID NZ_CP090192.1 [record index 0].\n",
      "2024-04-17 17:51:51,052 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090146.1, seqID NZ_CP090146.1 [record index 0].\n",
      "2024-04-17 17:51:51,142 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022510.1, seqID NZ_AP022510.1 [record index 0].\n",
      "2024-04-17 17:51:51,217 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028538.2, seqID NZ_CP028538.2 [record index 0].\n",
      "2024-04-17 17:51:51,293 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071009.1, seqID NZ_CP071009.1 [record index 0].\n",
      "2024-04-17 17:51:51,370 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071011.1, seqID NZ_CP071011.1 [record index 0].\n",
      "2024-04-17 17:51:51,445 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071013.1, seqID NZ_CP071013.1 [record index 0].\n",
      "2024-04-17 17:51:51,519 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071015.1, seqID NZ_CP071015.1 [record index 0].\n",
      "2024-04-17 17:51:51,597 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071017.1, seqID NZ_CP071017.1 [record index 0].\n",
      "2024-04-17 17:51:51,672 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071019.1, seqID NZ_CP071019.1 [record index 0].\n",
      "2024-04-17 17:51:51,749 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071021.1, seqID NZ_CP071021.1 [record index 0].\n",
      "2024-04-17 17:51:51,824 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071023.1, seqID NZ_CP071023.1 [record index 0].\n",
      "2024-04-17 17:51:51,898 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071025.1, seqID NZ_CP071025.1 [record index 0].\n",
      "2024-04-17 17:51:51,967 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058164.1, seqID NZ_CP058164.1 [record index 0].\n",
      "2024-04-17 17:51:52,031 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058167.1, seqID NZ_CP058167.1 [record index 0].\n",
      "2024-04-17 17:51:52,100 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058175.1, seqID NZ_CP058175.1 [record index 0].\n",
      "2024-04-17 17:51:52,167 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058187.1, seqID NZ_CP058187.1 [record index 0].\n",
      "2024-04-17 17:51:52,259 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030007.1, seqID NZ_CP030007.1 [record index 0].\n",
      "2024-04-17 17:51:52,325 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP019817.1, seqID NZ_AP019817.1 [record index 0].\n",
      "2024-04-17 17:51:52,416 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017183.1, seqID NZ_CP017183.1 [record index 0].\n",
      "2024-04-17 17:51:52,485 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034754.1, seqID NZ_CP034754.1 [record index 0].\n",
      "2024-04-17 17:51:52,585 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP070534.1, seqID NZ_CP070534.1 [record index 0].\n",
      "2024-04-17 17:51:52,657 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP042571.1, seqID NZ_CP042571.1 [record index 0].\n",
      "2024-04-17 17:51:52,752 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP042551.1, seqID NZ_CP042551.1 [record index 0].\n",
      "2024-04-17 17:51:52,824 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027142.1, seqID NZ_CP027142.1 [record index 0].\n",
      "2024-04-17 17:51:52,920 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051132.1, seqID NZ_CP051132.1 [record index 0].\n",
      "2024-04-17 17:51:53,010 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045451.1, seqID NZ_CP045451.1 [record index 0].\n",
      "2024-04-17 17:51:53,078 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP021896.1, seqID NZ_CP021896.1 [record index 0].\n",
      "2024-04-17 17:51:53,151 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP020525.1, seqID NZ_CP020525.1 [record index 0].\n",
      "2024-04-17 17:51:53,219 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053416.1, seqID NZ_CP053416.1 [record index 0].\n",
      "2024-04-17 17:51:53,297 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP035676.1, seqID NZ_CP035676.1 [record index 0].\n",
      "2024-04-17 17:51:53,390 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022532.1, seqID NZ_CP022532.1 [record index 0].\n",
      "2024-04-17 17:51:53,482 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077264.1, seqID NZ_CP077264.1 [record index 0].\n",
      "2024-04-17 17:51:53,575 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043766.1, seqID NZ_CP043766.1 [record index 0].\n",
      "2024-04-17 17:51:53,673 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017179.1, seqID NZ_CP017179.1 [record index 0].\n",
      "2024-04-17 17:51:53,758 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044335.1, seqID NZ_CP044335.1 [record index 0].\n",
      "2024-04-17 17:51:53,859 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP074592.1, seqID NZ_CP074592.1 [record index 0].\n",
      "2024-04-17 17:51:53,933 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP091779.1, seqID NZ_CP091779.1 [record index 0].\n",
      "2024-04-17 17:51:54,001 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP083403.1, seqID NZ_CP083403.1 [record index 0].\n",
      "2024-04-17 17:51:54,065 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP083620.1, seqID NZ_CP083620.1 [record index 0].\n",
      "2024-04-17 17:51:54,129 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP083641.1, seqID NZ_CP083641.1 [record index 0].\n",
      "2024-04-17 17:51:54,192 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP083643.1, seqID NZ_CP083643.1 [record index 0].\n",
      "2024-04-17 17:51:54,254 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077206.1, seqID NZ_CP077206.1 [record index 0].\n",
      "2024-04-17 17:51:54,314 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LT992502.1, seqID NZ_LT992502.1 [record index 0].\n",
      "2024-04-17 17:51:54,417 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP056395.1, seqID NZ_CP056395.1 [record index 0].\n",
      "2024-04-17 17:51:54,504 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056473.1, seqID NZ_CP056473.1 [record index 0].\n",
      "2024-04-17 17:51:54,595 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056967.1, seqID NZ_CP056967.1 [record index 0].\n",
      "2024-04-17 17:51:54,680 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057353.1, seqID NZ_CP057353.1 [record index 0].\n",
      "2024-04-17 17:51:54,769 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP006692.1, seqID NZ_CP006692.1 [record index 0].\n",
      "2024-04-17 17:51:54,849 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053417.1, seqID NZ_CP053417.1 [record index 0].\n",
      "2024-04-17 17:51:54,920 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039453.1, seqID NZ_CP039453.1 [record index 0].\n",
      "2024-04-17 17:51:54,987 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039452.1, seqID NZ_CP039452.1 [record index 0].\n",
      "2024-04-17 17:51:55,033 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073043.1, seqID NZ_CP073043.1 [record index 0].\n",
      "2024-04-17 17:51:55,076 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024683.1, seqID NZ_CP024683.1 [record index 0].\n",
      "2024-04-17 17:51:55,124 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056505.1, seqID NZ_CP056505.1 [record index 0].\n",
      "2024-04-17 17:51:55,172 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056527.1, seqID NZ_CP056527.1 [record index 0].\n",
      "2024-04-17 17:51:55,216 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056839.1, seqID NZ_CP056839.1 [record index 0].\n",
      "2024-04-17 17:51:55,263 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056852.1, seqID NZ_CP056852.1 [record index 0].\n",
      "2024-04-17 17:51:55,330 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP057867.1, seqID NZ_CP057867.1 [record index 0].\n",
      "2024-04-17 17:51:55,382 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040698.1, seqID NZ_CP040698.1 [record index 0].\n",
      "2024-04-17 17:51:55,433 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058318.1, seqID NZ_CP058318.1 [record index 0].\n",
      "2024-04-17 17:51:55,509 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP024672.1, seqID NZ_CP024672.1 [record index 0].\n",
      "2024-04-17 17:51:55,569 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026056.1, seqID NZ_CP026056.1 [record index 0].\n",
      "2024-04-17 17:51:55,614 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024881.1, seqID NZ_CP024881.1 [record index 0].\n",
      "2024-04-17 17:51:55,663 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046502.1, seqID NZ_CP046502.1 [record index 0].\n",
      "2024-04-17 17:51:55,757 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050506.1, seqID NZ_CP050506.1 [record index 0].\n",
      "2024-04-17 17:51:55,855 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050311.1, seqID NZ_CP050311.1 [record index 0].\n",
      "2024-04-17 17:51:55,949 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT991954.1, seqID NZ_LT991954.1 [record index 0].\n",
      "2024-04-17 17:51:55,992 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049147.1, seqID NZ_CP049147.1 [record index 0].\n",
      "2024-04-17 17:51:56,034 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049150.1, seqID NZ_CP049150.1 [record index 0].\n",
      "2024-04-17 17:51:56,082 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_009778.1, seqID NC_009778.1 [record index 0].\n",
      "2024-04-17 17:51:56,121 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059045.1, seqID NZ_CP059045.1 [record index 0].\n",
      "2024-04-17 17:51:56,208 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056229.1, seqID NZ_CP056229.1 [record index 0].\n",
      "2024-04-17 17:51:56,305 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056333.1, seqID NZ_CP056333.1 [record index 0].\n",
      "2024-04-17 17:51:56,400 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056515.1, seqID NZ_CP056515.1 [record index 0].\n",
      "2024-04-17 17:51:56,501 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056559.1, seqID NZ_CP056559.1 [record index 0].\n",
      "2024-04-17 17:51:56,598 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP055538.1, seqID NZ_CP055538.1 [record index 0].\n",
      "2024-04-17 17:51:56,684 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056597.1, seqID NZ_CP056597.1 [record index 0].\n",
      "2024-04-17 17:51:56,778 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056850.1, seqID NZ_CP056850.1 [record index 0].\n",
      "2024-04-17 17:51:56,876 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056910.1, seqID NZ_CP056910.1 [record index 0].\n",
      "2024-04-17 17:51:56,981 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057506.1, seqID NZ_CP057506.1 [record index 0].\n",
      "2024-04-17 17:51:57,086 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057509.1, seqID NZ_CP057509.1 [record index 0].\n",
      "2024-04-17 17:51:57,168 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034769.1, seqID NZ_CP034769.1 [record index 0].\n",
      "2024-04-17 17:51:57,256 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020817.1, seqID NZ_CP020817.1 [record index 0].\n",
      "2024-04-17 17:51:57,320 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058253.1, seqID NZ_CP058253.1 [record index 0].\n",
      "2024-04-17 17:51:57,403 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP055276.1, seqID NZ_CP055276.1 [record index 0].\n",
      "2024-04-17 17:51:57,499 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022274.1, seqID NZ_AP022274.1 [record index 0].\n",
      "2024-04-17 17:51:57,580 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037919.1, seqID NZ_CP037919.1 [record index 0].\n",
      "2024-04-17 17:51:57,704 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP053568.1, seqID NZ_CP053568.1 [record index 0].\n",
      "2024-04-17 17:51:57,827 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP077211.1, seqID NZ_CP077211.1 [record index 0].\n",
      "2024-04-17 17:51:57,951 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP056776.1, seqID NZ_CP056776.1 [record index 0].\n",
      "2024-04-17 17:51:58,073 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP073310.1, seqID NZ_CP073310.1 [record index 0].\n",
      "2024-04-17 17:51:58,202 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP092042.1, seqID NZ_CP092042.1 [record index 0].\n",
      "2024-04-17 17:51:58,337 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NC_014121.1, seqID NC_014121.1 [record index 0].\n",
      "2024-04-17 17:51:58,405 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022508.1, seqID NZ_AP022508.1 [record index 0].\n",
      "2024-04-17 17:51:58,490 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP024498.1, seqID NZ_AP024498.1 [record index 0].\n",
      "2024-04-17 17:51:58,584 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP019630.1, seqID NZ_AP019630.1 [record index 0].\n",
      "2024-04-17 17:51:58,634 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047606.1, seqID NZ_CP047606.1 [record index 0].\n",
      "2024-04-17 17:51:58,683 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041051.1, seqID NZ_CP041051.1 [record index 0].\n",
      "2024-04-17 17:51:58,729 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036435.1, seqID NZ_CP036435.1 [record index 0].\n",
      "2024-04-17 17:51:58,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LR134214.1, seqID NZ_LR134214.1 [record index 0].\n",
      "2024-04-17 17:51:58,864 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055247.1, seqID NZ_CP055247.1 [record index 0].\n",
      "2024-04-17 17:51:58,937 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045726.1, seqID NZ_CP045726.1 [record index 0].\n",
      "2024-04-17 17:51:59,047 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016952.1, seqID NZ_CP016952.1 [record index 0].\n",
      "2024-04-17 17:51:59,099 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056208.1, seqID NZ_CP056208.1 [record index 0].\n",
      "2024-04-17 17:51:59,152 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056235.1, seqID NZ_CP056235.1 [record index 0].\n",
      "2024-04-17 17:51:59,203 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056238.1, seqID NZ_CP056238.1 [record index 0].\n",
      "2024-04-17 17:51:59,253 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056245.1, seqID NZ_CP056245.1 [record index 0].\n",
      "2024-04-17 17:51:59,303 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056256.1, seqID NZ_CP056256.1 [record index 0].\n",
      "2024-04-17 17:51:59,351 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055421.1, seqID NZ_CP055421.1 [record index 0].\n",
      "2024-04-17 17:51:59,400 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056336.1, seqID NZ_CP056336.1 [record index 0].\n",
      "2024-04-17 17:51:59,448 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056365.1, seqID NZ_CP056365.1 [record index 0].\n",
      "2024-04-17 17:51:59,495 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056451.1, seqID NZ_CP056451.1 [record index 0].\n",
      "2024-04-17 17:51:59,543 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056466.1, seqID NZ_CP056466.1 [record index 0].\n",
      "2024-04-17 17:51:59,597 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056635.1, seqID NZ_CP056635.1 [record index 0].\n",
      "2024-04-17 17:51:59,647 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056653.1, seqID NZ_CP056653.1 [record index 0].\n",
      "2024-04-17 17:51:59,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056809.1, seqID NZ_CP056809.1 [record index 0].\n",
      "2024-04-17 17:51:59,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056827.1, seqID NZ_CP056827.1 [record index 0].\n",
      "2024-04-17 17:51:59,795 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055582.1, seqID NZ_CP055582.1 [record index 0].\n",
      "2024-04-17 17:51:59,847 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055588.1, seqID NZ_CP055588.1 [record index 0].\n",
      "2024-04-17 17:51:59,896 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057864.1, seqID NZ_CP057864.1 [record index 0].\n",
      "2024-04-17 17:51:59,986 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057866.1, seqID NZ_CP057866.1 [record index 0].\n",
      "2024-04-17 17:52:00,045 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035276.1, seqID NZ_CP035276.1 [record index 0].\n",
      "2024-04-17 17:52:00,105 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR881934.1, seqID NZ_LR881934.1 [record index 0].\n",
      "2024-04-17 17:52:00,159 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047279.1, seqID NZ_CP047279.1 [record index 0].\n",
      "2024-04-17 17:52:00,214 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047275.1, seqID NZ_CP047275.1 [record index 0].\n",
      "2024-04-17 17:52:00,270 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047273.1, seqID NZ_CP047273.1 [record index 0].\n",
      "2024-04-17 17:52:00,328 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047269.1, seqID NZ_CP047269.1 [record index 0].\n",
      "2024-04-17 17:52:00,373 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890181.1, seqID NZ_LR890181.1 [record index 0].\n",
      "2024-04-17 17:52:00,423 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047307.1, seqID NZ_CP047307.1 [record index 0].\n",
      "2024-04-17 17:52:00,471 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054278.1, seqID NZ_CP054278.1 [record index 0].\n",
      "2024-04-17 17:52:00,520 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054294.1, seqID NZ_CP054294.1 [record index 0].\n",
      "2024-04-17 17:52:00,571 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042534.1, seqID NZ_CP042534.1 [record index 0].\n",
      "2024-04-17 17:52:00,621 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042517.1, seqID NZ_CP042517.1 [record index 0].\n",
      "2024-04-17 17:52:00,672 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042524.1, seqID NZ_CP042524.1 [record index 0].\n",
      "2024-04-17 17:52:00,722 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086287.1, seqID NZ_CP086287.1 [record index 0].\n",
      "2024-04-17 17:52:00,776 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LS992175.1, seqID NZ_LS992175.1 [record index 0].\n",
      "2024-04-17 17:52:00,823 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP048416.1, seqID NZ_CP048416.1 [record index 0].\n",
      "2024-04-17 17:52:00,873 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059427.1, seqID NZ_CP059427.1 [record index 0].\n",
      "2024-04-17 17:52:00,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026231.1, seqID NZ_CP026231.1 [record index 0].\n",
      "2024-04-17 17:52:00,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070559.1, seqID NZ_CP070559.1 [record index 0].\n",
      "2024-04-17 17:52:01,035 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070549.1, seqID NZ_CP070549.1 [record index 0].\n",
      "2024-04-17 17:52:01,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070544.1, seqID NZ_CP070544.1 [record index 0].\n",
      "2024-04-17 17:52:01,146 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP037734.1, seqID NZ_CP037734.1 [record index 0].\n",
      "2024-04-17 17:52:01,197 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011657.1, seqID NZ_CP011657.1 [record index 0].\n",
      "2024-04-17 17:52:01,246 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011612.1, seqID NZ_CP011612.1 [record index 0].\n",
      "2024-04-17 17:52:01,296 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042478.1, seqID NZ_CP042478.1 [record index 0].\n",
      "2024-04-17 17:52:01,379 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016762.1, seqID NZ_CP016762.1 [record index 0].\n",
      "2024-04-17 17:52:01,436 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049015.1, seqID NZ_CP049015.1 [record index 0].\n",
      "2024-04-17 17:52:01,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032184.1, seqID NZ_CP032184.1 [record index 0].\n",
      "2024-04-17 17:52:01,541 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085726.1, seqID NZ_CP085726.1 [record index 0].\n",
      "2024-04-17 17:52:01,592 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022273.1, seqID NZ_CP022273.1 [record index 0].\n",
      "2024-04-17 17:52:01,679 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056132.1, seqID NZ_CP056132.1 [record index 0].\n",
      "2024-04-17 17:52:01,769 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056707.1, seqID NZ_CP056707.1 [record index 0].\n",
      "2024-04-17 17:52:01,861 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056742.1, seqID NZ_CP056742.1 [record index 0].\n",
      "2024-04-17 17:52:01,949 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056747.1, seqID NZ_CP056747.1 [record index 0].\n",
      "2024-04-17 17:52:02,039 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056751.1, seqID NZ_CP056751.1 [record index 0].\n",
      "2024-04-17 17:52:02,128 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056756.1, seqID NZ_CP056756.1 [record index 0].\n",
      "2024-04-17 17:52:02,217 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058153.1, seqID NZ_CP058153.1 [record index 0].\n",
      "2024-04-17 17:52:02,306 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058160.1, seqID NZ_CP058160.1 [record index 0].\n",
      "2024-04-17 17:52:02,399 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP083613.1, seqID NZ_CP083613.1 [record index 0].\n",
      "2024-04-17 17:52:02,485 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077308.1, seqID NZ_CP077308.1 [record index 0].\n",
      "2024-04-17 17:52:02,574 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047715.1, seqID NZ_CP047715.1 [record index 0].\n",
      "2024-04-17 17:52:02,673 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066101.1, seqID NZ_CP066101.1 [record index 0].\n",
      "2024-04-17 17:52:02,775 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066108.1, seqID NZ_CP066108.1 [record index 0].\n",
      "2024-04-17 17:52:02,867 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047729.1, seqID NZ_CP047729.1 [record index 0].\n",
      "2024-04-17 17:52:02,963 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017180.1, seqID NZ_CP017180.1 [record index 0].\n",
      "2024-04-17 17:52:03,072 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027111.1, seqID NZ_CP027111.1 [record index 0].\n",
      "2024-04-17 17:52:03,165 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023569.1, seqID NZ_CP023569.1 [record index 0].\n",
      "2024-04-17 17:52:03,261 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085767.1, seqID NZ_CP085767.1 [record index 0].\n",
      "2024-04-17 17:52:03,355 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP010377.1, seqID NZ_CP010377.1 [record index 0].\n",
      "2024-04-17 17:52:03,443 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026850.1, seqID NZ_CP026850.1 [record index 0].\n",
      "2024-04-17 17:52:03,509 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013940.1, seqID NZ_CP013940.1 [record index 0].\n",
      "2024-04-17 17:52:03,575 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_023032.1, seqID NC_023032.1 [record index 0].\n",
      "2024-04-17 17:52:03,660 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056244.1, seqID NZ_CP056244.1 [record index 0].\n",
      "2024-04-17 17:52:03,757 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056644.1, seqID NZ_CP056644.1 [record index 0].\n",
      "2024-04-17 17:52:03,843 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057258.1, seqID NZ_CP057258.1 [record index 0].\n",
      "2024-04-17 17:52:03,900 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082360.1, seqID NZ_CP082360.1 [record index 0].\n",
      "2024-04-17 17:52:03,951 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064784.1, seqID NZ_CP064784.1 [record index 0].\n",
      "2024-04-17 17:52:04,005 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089403.1, seqID NZ_CP089403.1 [record index 0].\n",
      "2024-04-17 17:52:04,057 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073236.1, seqID NZ_CP073236.1 [record index 0].\n",
      "2024-04-17 17:52:04,143 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074159.1, seqID NZ_CP074159.1 [record index 0].\n",
      "2024-04-17 17:52:04,230 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050520.1, seqID NZ_CP050520.1 [record index 0].\n",
      "2024-04-17 17:52:04,316 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027986.1, seqID NZ_CP027986.1 [record index 0].\n",
      "2024-04-17 17:52:04,390 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022465.1, seqID NZ_AP022465.1 [record index 0].\n",
      "2024-04-17 17:52:04,459 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019839.1, seqID NZ_CP019839.1 [record index 0].\n",
      "2024-04-17 17:52:04,525 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP086405.1, seqID NZ_CP086405.1 [record index 0].\n",
      "2024-04-17 17:52:04,598 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022148.1, seqID NZ_CP022148.1 [record index 0].\n",
      "2024-04-17 17:52:04,678 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP019007.1, seqID NZ_AP019007.1 [record index 0].\n",
      "2024-04-17 17:52:04,766 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP084692.1, seqID NZ_CP084692.1 [record index 0].\n",
      "2024-04-17 17:52:04,875 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071063.1, seqID NZ_CP071063.1 [record index 0].\n",
      "2024-04-17 17:52:04,943 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022228.1, seqID NZ_AP022228.1 [record index 0].\n",
      "2024-04-17 17:52:05,063 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NC_018079.1, seqID NC_018079.1 [record index 0].\n",
      "2024-04-17 17:52:05,178 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP086408.1, seqID NZ_CP086408.1 [record index 0].\n",
      "2024-04-17 17:52:05,288 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP009756.1, seqID NZ_CP009756.1 [record index 0].\n",
      "2024-04-17 17:52:05,398 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP035633.1, seqID NZ_CP035633.1 [record index 0].\n",
      "2024-04-17 17:52:05,503 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP085080.1, seqID NZ_CP085080.1 [record index 0].\n",
      "2024-04-17 17:52:05,606 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP091492.1, seqID NZ_CP091492.1 [record index 0].\n",
      "2024-04-17 17:52:05,697 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP083709.1, seqID NZ_CP083709.1 [record index 0].\n",
      "2024-04-17 17:52:05,811 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP046116.1, seqID NZ_CP046116.1 [record index 0].\n",
      "2024-04-17 17:52:05,882 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032291.1, seqID NZ_CP032291.1 [record index 0].\n",
      "2024-04-17 17:52:05,975 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP080107.1, seqID NZ_CP080107.1 [record index 0].\n",
      "2024-04-17 17:52:06,066 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056716.1, seqID NZ_CP056716.1 [record index 0].\n",
      "2024-04-17 17:52:06,162 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022399.1, seqID NZ_AP022399.1 [record index 0].\n",
      "2024-04-17 17:52:06,260 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022394.1, seqID NZ_AP022394.1 [record index 0].\n",
      "2024-04-17 17:52:06,356 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022389.1, seqID NZ_AP022389.1 [record index 0].\n",
      "2024-04-17 17:52:06,454 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022513.1, seqID NZ_AP022513.1 [record index 0].\n",
      "2024-04-17 17:52:06,549 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022494.1, seqID NZ_AP022494.1 [record index 0].\n",
      "2024-04-17 17:52:06,644 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022486.1, seqID NZ_AP022486.1 [record index 0].\n",
      "2024-04-17 17:52:06,768 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP050078.1, seqID NZ_CP050078.1 [record index 0].\n",
      "2024-04-17 17:52:06,887 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP044097.1, seqID NZ_CP044097.1 [record index 0].\n",
      "2024-04-17 17:52:07,004 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP033780.1, seqID NZ_CP033780.1 [record index 0].\n",
      "2024-04-17 17:52:07,085 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP024677.1, seqID NZ_CP024677.1 [record index 0].\n",
      "2024-04-17 17:52:07,133 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056314.1, seqID NZ_CP056314.1 [record index 0].\n",
      "2024-04-17 17:52:07,223 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057824.1, seqID NZ_CP057824.1 [record index 0].\n",
      "2024-04-17 17:52:07,318 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP078020.1, seqID NZ_CP078020.1 [record index 0].\n",
      "2024-04-17 17:52:07,436 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP047773.1, seqID NZ_CP047773.1 [record index 0].\n",
      "2024-04-17 17:52:07,523 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045485.1, seqID NZ_CP045485.1 [record index 0].\n",
      "2024-04-17 17:52:07,591 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_017933.1, seqID NC_017933.1 [record index 0].\n",
      "2024-04-17 17:52:07,649 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP049144.1, seqID NZ_CP049144.1 [record index 0].\n",
      "2024-04-17 17:52:07,731 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LR699014.1, seqID NZ_LR699014.1 [record index 0].\n",
      "2024-04-17 17:52:07,820 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023504.1, seqID NZ_CP023504.1 [record index 0].\n",
      "2024-04-17 17:52:07,914 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038856.1, seqID NZ_CP038856.1 [record index 0].\n",
      "2024-04-17 17:52:08,014 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056586.1, seqID NZ_CP056586.1 [record index 0].\n",
      "2024-04-17 17:52:08,068 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_011283.1, seqID NC_011283.1 [record index 0].\n",
      "2024-04-17 17:52:08,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033824.1, seqID NZ_CP033824.1 [record index 0].\n",
      "2024-04-17 17:52:08,177 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP067093.1, seqID NZ_CP067093.1 [record index 0].\n",
      "2024-04-17 17:52:08,258 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019113.1, seqID NZ_CP019113.1 [record index 0].\n",
      "2024-04-17 17:52:08,360 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP076838.1, seqID NZ_CP076838.1 [record index 0].\n",
      "2024-04-17 17:52:08,464 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071383.1, seqID NZ_CP071383.1 [record index 0].\n",
      "2024-04-17 17:52:08,527 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056682.1, seqID NZ_CP056682.1 [record index 0].\n",
      "2024-04-17 17:52:08,592 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045064.2, seqID NZ_CP045064.2 [record index 0].\n",
      "2024-04-17 17:52:08,658 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056129.1, seqID NZ_CP056129.1 [record index 0].\n",
      "2024-04-17 17:52:08,725 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056168.1, seqID NZ_CP056168.1 [record index 0].\n",
      "2024-04-17 17:52:08,795 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023447.1, seqID NZ_AP023447.1 [record index 0].\n",
      "2024-04-17 17:52:08,865 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033800.1, seqID NZ_CP033800.1 [record index 0].\n",
      "2024-04-17 17:52:08,933 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060737.1, seqID NZ_CP060737.1 [record index 0].\n",
      "2024-04-17 17:52:08,998 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032916.1, seqID NZ_CP032916.1 [record index 0].\n",
      "2024-04-17 17:52:09,068 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP048650.1, seqID NZ_CP048650.1 [record index 0].\n",
      "2024-04-17 17:52:09,138 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP038471.1, seqID NZ_CP038471.1 [record index 0].\n",
      "2024-04-17 17:52:09,236 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012162.1, seqID NZ_CP012162.1 [record index 0].\n",
      "2024-04-17 17:52:09,348 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP017475.1, seqID NZ_CP017475.1 [record index 0].\n",
      "2024-04-17 17:52:09,423 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP021851.1, seqID NZ_CP021851.1 [record index 0].\n",
      "2024-04-17 17:52:09,546 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP045487.1, seqID NZ_CP045487.1 [record index 0].\n",
      "2024-04-17 17:52:09,653 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP056126.1, seqID NZ_CP056126.1 [record index 0].\n",
      "2024-04-17 17:52:09,754 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP074584.1, seqID NZ_CP074584.1 [record index 0].\n",
      "2024-04-17 17:52:09,854 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP007546.1, seqID NZ_CP007546.1 [record index 0].\n",
      "2024-04-17 17:52:09,940 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074154.1, seqID NZ_CP074154.1 [record index 0].\n",
      "2024-04-17 17:52:10,054 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP065693.1, seqID NZ_CP065693.1 [record index 0].\n",
      "2024-04-17 17:52:10,162 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP011863.1, seqID NZ_CP011863.1 [record index 0].\n",
      "2024-04-17 17:52:10,262 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP021963.1, seqID NZ_CP021963.1 [record index 0].\n",
      "2024-04-17 17:52:10,379 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP040234.1, seqID NZ_CP040234.1 [record index 0].\n",
      "2024-04-17 17:52:10,472 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP073048.1, seqID NZ_CP073048.1 [record index 0].\n",
      "2024-04-17 17:52:10,565 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP073007.1, seqID NZ_CP073007.1 [record index 0].\n",
      "2024-04-17 17:52:10,671 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069801.1, seqID NZ_CP069801.1 [record index 0].\n",
      "2024-04-17 17:52:10,739 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043009.1, seqID NZ_CP043009.1 [record index 0].\n",
      "2024-04-17 17:52:10,841 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP055089.1, seqID NZ_CP055089.1 [record index 0].\n",
      "2024-04-17 17:52:10,939 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP092442.1, seqID NZ_CP092442.1 [record index 0].\n",
      "2024-04-17 17:52:11,022 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP048388.1, seqID NZ_CP048388.1 [record index 0].\n",
      "2024-04-17 17:52:11,114 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089316.1, seqID NZ_CP089316.1 [record index 0].\n",
      "2024-04-17 17:52:11,208 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046348.1, seqID NZ_CP046348.1 [record index 0].\n",
      "2024-04-17 17:52:11,303 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044098.1, seqID NZ_CP044098.1 [record index 0].\n",
      "2024-04-17 17:52:11,401 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039327.1, seqID NZ_CP039327.1 [record index 0].\n",
      "2024-04-17 17:52:11,492 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054129.1, seqID NZ_CP054129.1 [record index 0].\n",
      "2024-04-17 17:52:11,600 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP091809.1, seqID NZ_CP091809.1 [record index 0].\n",
      "2024-04-17 17:52:11,732 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_AP023452.1, seqID NZ_AP023452.1 [record index 0].\n",
      "2024-04-17 17:52:11,857 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP026697.1, seqID NZ_CP026697.1 [record index 0].\n",
      "2024-04-17 17:52:11,950 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056181.1, seqID NZ_CP056181.1 [record index 0].\n",
      "2024-04-17 17:52:12,021 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056289.1, seqID NZ_CP056289.1 [record index 0].\n",
      "2024-04-17 17:52:12,087 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056293.1, seqID NZ_CP056293.1 [record index 0].\n",
      "2024-04-17 17:52:12,135 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056573.1, seqID NZ_CP056573.1 [record index 0].\n",
      "2024-04-17 17:52:12,184 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056622.1, seqID NZ_CP056622.1 [record index 0].\n",
      "2024-04-17 17:52:12,276 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057319.1, seqID NZ_CP057319.1 [record index 0].\n",
      "2024-04-17 17:52:12,378 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057863.1, seqID NZ_CP057863.1 [record index 0].\n",
      "2024-04-17 17:52:12,467 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057868.1, seqID NZ_CP057868.1 [record index 0].\n",
      "2024-04-17 17:52:12,563 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060654.1, seqID NZ_CP060654.1 [record index 0].\n",
      "2024-04-17 17:52:12,661 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060659.1, seqID NZ_CP060659.1 [record index 0].\n",
      "2024-04-17 17:52:12,759 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060662.1, seqID NZ_CP060662.1 [record index 0].\n",
      "2024-04-17 17:52:12,854 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060650.1, seqID NZ_CP060650.1 [record index 0].\n",
      "2024-04-17 17:52:12,906 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022151.1, seqID NZ_CP022151.1 [record index 0].\n",
      "2024-04-17 17:52:12,990 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056779.1, seqID NZ_CP056779.1 [record index 0].\n",
      "2024-04-17 17:52:13,078 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046618.1, seqID NZ_CP046618.1 [record index 0].\n",
      "2024-04-17 17:52:13,147 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP078110.1, seqID NZ_CP078110.1 [record index 0].\n",
      "2024-04-17 17:52:13,245 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034697.1, seqID NZ_CP034697.1 [record index 0].\n",
      "2024-04-17 17:52:13,344 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034717.1, seqID NZ_CP034717.1 [record index 0].\n",
      "2024-04-17 17:52:13,398 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054254.1, seqID NZ_CP054254.1 [record index 0].\n",
      "2024-04-17 17:52:13,451 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047604.1, seqID NZ_CP047604.1 [record index 0].\n",
      "2024-04-17 17:52:13,504 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020657.1, seqID NZ_CP020657.1 [record index 0].\n",
      "2024-04-17 17:52:13,559 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017994.1, seqID NZ_CP017994.1 [record index 0].\n",
      "2024-04-17 17:52:13,642 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP036175.1, seqID NZ_CP036175.1 [record index 0].\n",
      "2024-04-17 17:52:13,699 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044527.1, seqID NZ_CP044527.1 [record index 0].\n",
      "2024-04-17 17:52:13,753 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056150.1, seqID NZ_CP056150.1 [record index 0].\n",
      "2024-04-17 17:52:13,805 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056701.1, seqID NZ_CP056701.1 [record index 0].\n",
      "2024-04-17 17:52:13,861 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055315.1, seqID NZ_CP055315.1 [record index 0].\n",
      "2024-04-17 17:52:13,913 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056760.1, seqID NZ_CP056760.1 [record index 0].\n",
      "2024-04-17 17:52:13,969 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058112.1, seqID NZ_CP058112.1 [record index 0].\n",
      "2024-04-17 17:52:14,025 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055412.1, seqID NZ_CP055412.1 [record index 0].\n",
      "2024-04-17 17:52:14,082 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058180.1, seqID NZ_CP058180.1 [record index 0].\n",
      "2024-04-17 17:52:14,141 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR594038.1, seqID NZ_LR594038.1 [record index 0].\n",
      "2024-04-17 17:52:14,193 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063301.1, seqID NZ_CP063301.1 [record index 0].\n",
      "2024-04-17 17:52:14,248 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP067391.1, seqID NZ_CP067391.1 [record index 0].\n",
      "2024-04-17 17:52:14,301 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP067380.1, seqID NZ_CP067380.1 [record index 0].\n",
      "2024-04-17 17:52:14,430 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP031104.1, seqID NZ_CP031104.1 [record index 0].\n",
      "2024-04-17 17:52:14,485 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022823.1, seqID NZ_CP022823.1 [record index 0].\n",
      "2024-04-17 17:52:14,566 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015227.1, seqID NZ_CP015227.1 [record index 0].\n",
      "2024-04-17 17:52:14,657 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017087.1, seqID NZ_CP017087.1 [record index 0].\n",
      "2024-04-17 17:52:14,746 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP019634.1, seqID NZ_AP019634.1 [record index 0].\n",
      "2024-04-17 17:52:14,821 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056148.1, seqID NZ_CP056148.1 [record index 0].\n",
      "2024-04-17 17:52:14,890 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056737.1, seqID NZ_CP056737.1 [record index 0].\n",
      "2024-04-17 17:52:14,979 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP058196.1, seqID NZ_CP058196.1 [record index 0].\n",
      "2024-04-17 17:52:15,045 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP086201.1, seqID NZ_CP086201.1 [record index 0].\n",
      "2024-04-17 17:52:15,111 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060733.1, seqID NZ_CP060733.1 [record index 0].\n",
      "2024-04-17 17:52:15,203 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022452.1, seqID NZ_AP022452.1 [record index 0].\n",
      "2024-04-17 17:52:15,291 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_018405.1, seqID NC_018405.1 [record index 0].\n",
      "2024-04-17 17:52:15,385 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP042578.1, seqID NZ_CP042578.1 [record index 0].\n",
      "2024-04-17 17:52:15,486 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022126.1, seqID NZ_AP022126.1 [record index 0].\n",
      "2024-04-17 17:52:15,578 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022519.1, seqID NZ_AP022519.1 [record index 0].\n",
      "2024-04-17 17:52:15,697 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP020089.1, seqID NZ_CP020089.1 [record index 0].\n",
      "2024-04-17 17:52:15,808 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071861.1, seqID NZ_CP071861.1 [record index 0].\n",
      "2024-04-17 17:52:15,899 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026975.1, seqID NZ_CP026975.1 [record index 0].\n",
      "2024-04-17 17:52:15,988 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009850.1, seqID NZ_CP009850.1 [record index 0].\n",
      "2024-04-17 17:52:16,088 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP091481.1, seqID NZ_CP091481.1 [record index 0].\n",
      "2024-04-17 17:52:16,175 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP035738.1, seqID NZ_CP035738.1 [record index 0].\n",
      "2024-04-17 17:52:16,272 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043318.1, seqID NZ_CP043318.1 [record index 0].\n",
      "2024-04-17 17:52:16,356 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063441.1, seqID NZ_CP063441.1 [record index 0].\n",
      "2024-04-17 17:52:16,448 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074177.1, seqID NZ_CP074177.1 [record index 0].\n",
      "2024-04-17 17:52:16,534 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077411.1, seqID NZ_CP077411.1 [record index 0].\n",
      "2024-04-17 17:52:16,621 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP024281.1, seqID NZ_AP024281.1 [record index 0].\n",
      "2024-04-17 17:52:16,714 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034336.1, seqID NZ_CP034336.1 [record index 0].\n",
      "2024-04-17 17:52:16,804 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011591.1, seqID NZ_CP011591.1 [record index 0].\n",
      "2024-04-17 17:52:16,890 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022628.1, seqID NZ_AP022628.1 [record index 0].\n",
      "2024-04-17 17:52:16,989 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP093154.1, seqID NZ_CP093154.1 [record index 0].\n",
      "2024-04-17 17:52:17,063 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR134485.1, seqID NZ_LR134485.1 [record index 0].\n",
      "2024-04-17 17:52:17,141 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044101.1, seqID NZ_CP044101.1 [record index 0].\n",
      "2024-04-17 17:52:17,241 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045840.1, seqID NZ_CP045840.1 [record index 0].\n",
      "2024-04-17 17:52:17,317 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056185.1, seqID NZ_CP056185.1 [record index 0].\n",
      "2024-04-17 17:52:17,412 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056200.1, seqID NZ_CP056200.1 [record index 0].\n",
      "2024-04-17 17:52:17,487 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056219.1, seqID NZ_CP056219.1 [record index 0].\n",
      "2024-04-17 17:52:17,557 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP055418.1, seqID NZ_CP055418.1 [record index 0].\n",
      "2024-04-17 17:52:17,634 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056251.1, seqID NZ_CP056251.1 [record index 0].\n",
      "2024-04-17 17:52:17,713 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056267.1, seqID NZ_CP056267.1 [record index 0].\n",
      "2024-04-17 17:52:17,795 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056269.1, seqID NZ_CP056269.1 [record index 0].\n",
      "2024-04-17 17:52:17,878 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056284.1, seqID NZ_CP056284.1 [record index 0].\n",
      "2024-04-17 17:52:17,961 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056321.1, seqID NZ_CP056321.1 [record index 0].\n",
      "2024-04-17 17:52:18,017 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056341.1, seqID NZ_CP056341.1 [record index 0].\n",
      "2024-04-17 17:52:18,090 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056350.1, seqID NZ_CP056350.1 [record index 0].\n",
      "2024-04-17 17:52:18,171 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056361.1, seqID NZ_CP056361.1 [record index 0].\n",
      "2024-04-17 17:52:18,246 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056381.1, seqID NZ_CP056381.1 [record index 0].\n",
      "2024-04-17 17:52:18,328 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056421.1, seqID NZ_CP056421.1 [record index 0].\n",
      "2024-04-17 17:52:18,408 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056508.1, seqID NZ_CP056508.1 [record index 0].\n",
      "2024-04-17 17:52:18,491 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056546.1, seqID NZ_CP056546.1 [record index 0].\n",
      "2024-04-17 17:52:18,577 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056647.1, seqID NZ_CP056647.1 [record index 0].\n",
      "2024-04-17 17:52:18,655 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056822.1, seqID NZ_CP056822.1 [record index 0].\n",
      "2024-04-17 17:52:18,739 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056861.1, seqID NZ_CP056861.1 [record index 0].\n",
      "2024-04-17 17:52:18,817 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056888.1, seqID NZ_CP056888.1 [record index 0].\n",
      "2024-04-17 17:52:18,893 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056896.1, seqID NZ_CP056896.1 [record index 0].\n",
      "2024-04-17 17:52:18,971 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056899.1, seqID NZ_CP056899.1 [record index 0].\n",
      "2024-04-17 17:52:19,039 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP057125.1, seqID NZ_CP057125.1 [record index 0].\n",
      "2024-04-17 17:52:19,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057147.1, seqID NZ_CP057147.1 [record index 0].\n",
      "2024-04-17 17:52:19,177 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045837.1, seqID NZ_CP045837.1 [record index 0].\n",
      "2024-04-17 17:52:19,250 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014030.2, seqID NZ_CP014030.2 [record index 0].\n",
      "2024-04-17 17:52:19,325 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP024676.1, seqID NZ_CP024676.1 [record index 0].\n",
      "2024-04-17 17:52:19,417 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077262.1, seqID NZ_CP077262.1 [record index 0].\n",
      "2024-04-17 17:52:19,545 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP023527.1, seqID NZ_CP023527.1 [record index 0].\n",
      "2024-04-17 17:52:19,674 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP022073.1, seqID NZ_CP022073.1 [record index 0].\n",
      "2024-04-17 17:52:19,800 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP026709.1, seqID NZ_CP026709.1 [record index 0].\n",
      "2024-04-17 17:52:19,848 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024673.1, seqID NZ_CP024673.1 [record index 0].\n",
      "2024-04-17 17:52:19,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056592.1, seqID NZ_CP056592.1 [record index 0].\n",
      "2024-04-17 17:52:19,939 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056595.1, seqID NZ_CP056595.1 [record index 0].\n",
      "2024-04-17 17:52:20,034 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057323.1, seqID NZ_CP057323.1 [record index 0].\n",
      "2024-04-17 17:52:20,133 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026238.1, seqID NZ_CP026238.1 [record index 0].\n",
      "2024-04-17 17:52:20,216 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP026235.1, seqID NZ_CP026235.1 [record index 0].\n",
      "2024-04-17 17:52:20,272 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP038658.1, seqID NZ_CP038658.1 [record index 0].\n",
      "2024-04-17 17:52:20,323 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP048382.1, seqID NZ_CP048382.1 [record index 0].\n",
      "2024-04-17 17:52:20,372 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP038656.1, seqID NZ_CP038656.1 [record index 0].\n",
      "2024-04-17 17:52:20,421 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP038653.1, seqID NZ_CP038653.1 [record index 0].\n",
      "2024-04-17 17:52:20,498 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045771.1, seqID NZ_CP045771.1 [record index 0].\n",
      "2024-04-17 17:52:20,548 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022049.2, seqID NZ_CP022049.2 [record index 0].\n",
      "2024-04-17 17:52:20,612 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077300.1, seqID NZ_CP077300.1 [record index 0].\n",
      "2024-04-17 17:52:20,847 [INFO - chronostrain.cli.make_db_json] - Created 11 merged markers for strain NZ_CP025881.1, seqID NZ_CP025881.1 [record index 0].\n",
      "2024-04-17 17:52:20,932 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT991957.1, seqID NZ_LT991957.1 [record index 0].\n",
      "2024-04-17 17:52:21,042 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057136.1, seqID NZ_CP057136.1 [record index 0].\n",
      "2024-04-17 17:52:21,069 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP009459.1, seqID NZ_CP009459.1 [record index 0].\n",
      "2024-04-17 17:52:21,093 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023525.1, seqID NZ_CP023525.1 [record index 0].\n",
      "2024-04-17 17:52:21,155 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077662.1, seqID NZ_CP077662.1 [record index 0].\n",
      "2024-04-17 17:52:21,241 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075112.1, seqID NZ_CP075112.1 [record index 0].\n",
      "2024-04-17 17:52:21,311 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033350.2, seqID NZ_CP033350.2 [record index 0].\n",
      "2024-04-17 17:52:21,375 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028900.1, seqID NZ_CP028900.1 [record index 0].\n",
      "2024-04-17 17:52:21,424 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP087978.1, seqID NZ_CP087978.1 [record index 0].\n",
      "2024-04-17 17:52:21,475 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027064.2, seqID NZ_CP027064.2 [record index 0].\n",
      "2024-04-17 17:52:21,523 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063926.1, seqID NZ_CP063926.1 [record index 0].\n",
      "2024-04-17 17:52:21,573 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063953.1, seqID NZ_CP063953.1 [record index 0].\n",
      "2024-04-17 17:52:21,623 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063954.1, seqID NZ_CP063954.1 [record index 0].\n",
      "2024-04-17 17:52:21,674 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063915.1, seqID NZ_CP063915.1 [record index 0].\n",
      "2024-04-17 17:52:21,726 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063911.1, seqID NZ_CP063911.1 [record index 0].\n",
      "2024-04-17 17:52:21,775 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063867.1, seqID NZ_CP063867.1 [record index 0].\n",
      "2024-04-17 17:52:21,828 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP013985.1, seqID NZ_CP013985.1 [record index 0].\n",
      "2024-04-17 17:52:21,880 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020847.1, seqID NZ_CP020847.1 [record index 0].\n",
      "2024-04-17 17:52:21,938 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065162.1, seqID NZ_CP065162.1 [record index 0].\n",
      "2024-04-17 17:52:21,989 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024592.1, seqID NZ_AP024592.1 [record index 0].\n",
      "2024-04-17 17:52:22,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083790.1, seqID NZ_CP083790.1 [record index 0].\n",
      "2024-04-17 17:52:22,097 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017289.1, seqID NZ_CP017289.1 [record index 0].\n",
      "2024-04-17 17:52:22,154 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017849.1, seqID NZ_CP017849.1 [record index 0].\n",
      "2024-04-17 17:52:22,208 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017284.1, seqID NZ_CP017284.1 [record index 0].\n",
      "2024-04-17 17:52:22,258 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044050.1, seqID NZ_CP044050.1 [record index 0].\n",
      "2024-04-17 17:52:22,308 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079802.1, seqID NZ_CP079802.1 [record index 0].\n",
      "2024-04-17 17:52:22,357 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016344.1, seqID NZ_CP016344.1 [record index 0].\n",
      "2024-04-17 17:52:22,382 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP067261.1, seqID NZ_CP067261.1 [record index 0].\n",
      "2024-04-17 17:52:22,408 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084767.1, seqID NZ_CP084767.1 [record index 0].\n",
      "2024-04-17 17:52:22,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063932.1, seqID NZ_CP063932.1 [record index 0].\n",
      "2024-04-17 17:52:22,504 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063933.1, seqID NZ_CP063933.1 [record index 0].\n",
      "2024-04-17 17:52:22,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063912.1, seqID NZ_CP063912.1 [record index 0].\n",
      "2024-04-17 17:52:22,599 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063916.1, seqID NZ_CP063916.1 [record index 0].\n",
      "2024-04-17 17:52:22,625 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063898.1, seqID NZ_CP063898.1 [record index 0].\n",
      "2024-04-17 17:52:22,673 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063889.1, seqID NZ_CP063889.1 [record index 0].\n",
      "2024-04-17 17:52:22,726 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063910.1, seqID NZ_CP063910.1 [record index 0].\n",
      "2024-04-17 17:52:22,774 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063921.1, seqID NZ_CP063921.1 [record index 0].\n",
      "2024-04-17 17:52:22,822 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_013850.1, seqID NC_013850.1 [record index 0].\n",
      "2024-04-17 17:52:22,868 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130538.1, seqID NZ_LR130538.1 [record index 0].\n",
      "2024-04-17 17:52:22,915 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130539.1, seqID NZ_LR130539.1 [record index 0].\n",
      "2024-04-17 17:52:22,966 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047360.1, seqID NZ_CP047360.1 [record index 0].\n",
      "2024-04-17 17:52:23,019 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032354.1, seqID NZ_CP032354.1 [record index 0].\n",
      "2024-04-17 17:52:23,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026013.1, seqID NZ_CP026013.1 [record index 0].\n",
      "2024-04-17 17:52:23,129 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030173.1, seqID NZ_CP030173.1 [record index 0].\n",
      "2024-04-17 17:52:23,173 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130544.1, seqID NZ_LR130544.1 [record index 0].\n",
      "2024-04-17 17:52:23,223 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084766.1, seqID NZ_CP084766.1 [record index 0].\n",
      "2024-04-17 17:52:23,274 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033631.1, seqID NZ_CP033631.1 [record index 0].\n",
      "2024-04-17 17:52:23,326 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073377.1, seqID NZ_CP073377.1 [record index 0].\n",
      "2024-04-17 17:52:23,377 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089234.1, seqID NZ_CP089234.1 [record index 0].\n",
      "2024-04-17 17:52:23,429 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058960.1, seqID NZ_CP058960.1 [record index 0].\n",
      "2024-04-17 17:52:23,482 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050336.1, seqID NZ_CP050336.1 [record index 0].\n",
      "2024-04-17 17:52:23,535 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050342.1, seqID NZ_CP050342.1 [record index 0].\n",
      "2024-04-17 17:52:23,591 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050348.1, seqID NZ_CP050348.1 [record index 0].\n",
      "2024-04-17 17:52:23,643 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050354.1, seqID NZ_CP050354.1 [record index 0].\n",
      "2024-04-17 17:52:23,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP067060.1, seqID NZ_CP067060.1 [record index 0].\n",
      "2024-04-17 17:52:23,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065347.1, seqID NZ_CP065347.1 [record index 0].\n",
      "2024-04-17 17:52:23,795 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065343.1, seqID NZ_CP065343.1 [record index 0].\n",
      "2024-04-17 17:52:23,846 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075286.1, seqID NZ_CP075286.1 [record index 0].\n",
      "2024-04-17 17:52:23,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045193.1, seqID NZ_CP045193.1 [record index 0].\n",
      "2024-04-17 17:52:23,949 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032240.1, seqID NZ_CP032240.1 [record index 0].\n",
      "2024-04-17 17:52:24,001 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032163.1, seqID NZ_CP032163.1 [record index 0].\n",
      "2024-04-17 17:52:24,058 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092786.1, seqID NZ_CP092786.1 [record index 0].\n",
      "2024-04-17 17:52:24,116 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092793.1, seqID NZ_CP092793.1 [record index 0].\n",
      "2024-04-17 17:52:24,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071385.1, seqID NZ_CP071385.1 [record index 0].\n",
      "2024-04-17 17:52:24,214 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054063.1, seqID NZ_CP054063.1 [record index 0].\n",
      "2024-04-17 17:52:24,268 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091068.1, seqID NZ_CP091068.1 [record index 0].\n",
      "2024-04-17 17:52:24,321 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079692.1, seqID NZ_CP079692.1 [record index 0].\n",
      "2024-04-17 17:52:24,371 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079683.1, seqID NZ_CP079683.1 [record index 0].\n",
      "2024-04-17 17:52:24,424 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079655.1, seqID NZ_CP079655.1 [record index 0].\n",
      "2024-04-17 17:52:24,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079645.1, seqID NZ_CP079645.1 [record index 0].\n",
      "2024-04-17 17:52:24,524 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079642.1, seqID NZ_CP079642.1 [record index 0].\n",
      "2024-04-17 17:52:24,575 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079636.1, seqID NZ_CP079636.1 [record index 0].\n",
      "2024-04-17 17:52:24,620 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079634.1, seqID NZ_CP079634.1 [record index 0].\n",
      "2024-04-17 17:52:24,671 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079617.1, seqID NZ_CP079617.1 [record index 0].\n",
      "2024-04-17 17:52:24,719 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079606.1, seqID NZ_CP079606.1 [record index 0].\n",
      "2024-04-17 17:52:24,772 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079186.1, seqID NZ_CP079186.1 [record index 0].\n",
      "2024-04-17 17:52:24,824 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079179.1, seqID NZ_CP079179.1 [record index 0].\n",
      "2024-04-17 17:52:24,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084044.1, seqID NZ_CP084044.1 [record index 0].\n",
      "2024-04-17 17:52:24,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079172.1, seqID NZ_CP079172.1 [record index 0].\n",
      "2024-04-17 17:52:24,976 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079807.1, seqID NZ_CP079807.1 [record index 0].\n",
      "2024-04-17 17:52:25,027 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079115.1, seqID NZ_CP079115.1 [record index 0].\n",
      "2024-04-17 17:52:25,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP022078.1, seqID NZ_AP022078.1 [record index 0].\n",
      "2024-04-17 17:52:25,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP021909.1, seqID NZ_AP021909.1 [record index 0].\n",
      "2024-04-17 17:52:25,177 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027068.3, seqID NZ_CP027068.3 [record index 0].\n",
      "2024-04-17 17:52:25,228 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028806.2, seqID NZ_CP028806.2 [record index 0].\n",
      "2024-04-17 17:52:25,279 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028583.2, seqID NZ_CP028583.2 [record index 0].\n",
      "2024-04-17 17:52:25,331 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036361.1, seqID NZ_CP036361.1 [record index 0].\n",
      "2024-04-17 17:52:25,386 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028542.3, seqID NZ_CP028542.3 [record index 0].\n",
      "2024-04-17 17:52:25,438 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028391.2, seqID NZ_CP028391.2 [record index 0].\n",
      "2024-04-17 17:52:25,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033405.1, seqID NZ_CP033405.1 [record index 0].\n",
      "2024-04-17 17:52:25,545 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036365.1, seqID NZ_CP036365.1 [record index 0].\n",
      "2024-04-17 17:52:25,604 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026585.3, seqID NZ_CP026585.3 [record index 0].\n",
      "2024-04-17 17:52:25,657 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066534.1, seqID NZ_CP066534.1 [record index 0].\n",
      "2024-04-17 17:52:25,709 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066528.1, seqID NZ_CP066528.1 [record index 0].\n",
      "2024-04-17 17:52:25,762 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066523.1, seqID NZ_CP066523.1 [record index 0].\n",
      "2024-04-17 17:52:25,815 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066517.1, seqID NZ_CP066517.1 [record index 0].\n",
      "2024-04-17 17:52:25,866 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028797.2, seqID NZ_CP028797.2 [record index 0].\n",
      "2024-04-17 17:52:25,919 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073285.1, seqID NZ_CP073285.1 [record index 0].\n",
      "2024-04-17 17:52:25,973 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043357.1, seqID NZ_CP043357.1 [record index 0].\n",
      "2024-04-17 17:52:26,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043353.1, seqID NZ_CP043353.1 [record index 0].\n",
      "2024-04-17 17:52:26,083 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036305.1, seqID NZ_CP036305.1 [record index 0].\n",
      "2024-04-17 17:52:26,137 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036371.1, seqID NZ_CP036371.1 [record index 0].\n",
      "2024-04-17 17:52:26,192 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028793.2, seqID NZ_CP028793.2 [record index 0].\n",
      "2024-04-17 17:52:26,245 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036300.1, seqID NZ_CP036300.1 [record index 0].\n",
      "2024-04-17 17:52:26,297 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036320.1, seqID NZ_CP036320.1 [record index 0].\n",
      "2024-04-17 17:52:26,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011985.1, seqID NZ_CP011985.1 [record index 0].\n",
      "2024-04-17 17:52:26,400 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023722.1, seqID NZ_CP023722.1 [record index 0].\n",
      "2024-04-17 17:52:26,450 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024795.1, seqID NZ_AP024795.1 [record index 0].\n",
      "2024-04-17 17:52:26,502 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024792.1, seqID NZ_AP024792.1 [record index 0].\n",
      "2024-04-17 17:52:26,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024788.1, seqID NZ_AP024788.1 [record index 0].\n",
      "2024-04-17 17:52:26,602 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024784.1, seqID NZ_AP024784.1 [record index 0].\n",
      "2024-04-17 17:52:26,654 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024781.1, seqID NZ_AP024781.1 [record index 0].\n",
      "2024-04-17 17:52:26,705 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024778.1, seqID NZ_AP024778.1 [record index 0].\n",
      "2024-04-17 17:52:26,754 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024775.1, seqID NZ_AP024775.1 [record index 0].\n",
      "2024-04-17 17:52:26,804 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024772.1, seqID NZ_AP024772.1 [record index 0].\n",
      "2024-04-17 17:52:26,853 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024769.1, seqID NZ_AP024769.1 [record index 0].\n",
      "2024-04-17 17:52:26,904 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024766.1, seqID NZ_AP024766.1 [record index 0].\n",
      "2024-04-17 17:52:26,956 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024763.1, seqID NZ_AP024763.1 [record index 0].\n",
      "2024-04-17 17:52:27,008 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024760.1, seqID NZ_AP024760.1 [record index 0].\n",
      "2024-04-17 17:52:27,062 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024756.1, seqID NZ_AP024756.1 [record index 0].\n",
      "2024-04-17 17:52:27,113 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024753.1, seqID NZ_AP024753.1 [record index 0].\n",
      "2024-04-17 17:52:27,162 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024750.1, seqID NZ_AP024750.1 [record index 0].\n",
      "2024-04-17 17:52:27,213 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP022527.1, seqID NZ_AP022527.1 [record index 0].\n",
      "2024-04-17 17:52:27,266 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP087123.1, seqID NZ_CP087123.1 [record index 0].\n",
      "2024-04-17 17:52:27,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091650.1, seqID NZ_CP091650.1 [record index 0].\n",
      "2024-04-17 17:52:27,367 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP019665.1, seqID NZ_AP019665.1 [record index 0].\n",
      "2024-04-17 17:52:27,422 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084241.1, seqID NZ_CP084241.1 [record index 0].\n",
      "2024-04-17 17:52:27,477 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079781.1, seqID NZ_CP079781.1 [record index 0].\n",
      "2024-04-17 17:52:27,545 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018454.1, seqID NZ_CP018454.1 [record index 0].\n",
      "2024-04-17 17:52:27,601 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055087.1, seqID NZ_CP055087.1 [record index 0].\n",
      "2024-04-17 17:52:27,655 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055096.1, seqID NZ_CP055096.1 [record index 0].\n",
      "2024-04-17 17:52:27,710 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055061.1, seqID NZ_CP055061.1 [record index 0].\n",
      "2024-04-17 17:52:27,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073302.1, seqID NZ_CP073302.1 [record index 0].\n",
      "2024-04-17 17:52:27,820 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058758.1, seqID NZ_CP058758.1 [record index 0].\n",
      "2024-04-17 17:52:27,875 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058752.1, seqID NZ_CP058752.1 [record index 0].\n",
      "2024-04-17 17:52:27,933 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058743.1, seqID NZ_CP058743.1 [record index 0].\n",
      "2024-04-17 17:52:27,989 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058857.1, seqID NZ_CP058857.1 [record index 0].\n",
      "2024-04-17 17:52:28,045 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055003.1, seqID NZ_CP055003.1 [record index 0].\n",
      "2024-04-17 17:52:28,102 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055005.1, seqID NZ_CP055005.1 [record index 0].\n",
      "2024-04-17 17:52:28,153 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP037742.1, seqID NZ_CP037742.1 [record index 0].\n",
      "2024-04-17 17:52:28,206 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023487.1, seqID NZ_CP023487.1 [record index 0].\n",
      "2024-04-17 17:52:28,261 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040833.1, seqID NZ_CP040833.1 [record index 0].\n",
      "2024-04-17 17:52:28,317 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025080.1, seqID NZ_CP025080.1 [record index 0].\n",
      "2024-04-17 17:52:28,369 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065444.1, seqID NZ_CP065444.1 [record index 0].\n",
      "2024-04-17 17:52:28,421 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086664.1, seqID NZ_CP086664.1 [record index 0].\n",
      "2024-04-17 17:52:28,472 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029388.1, seqID NZ_CP029388.1 [record index 0].\n",
      "2024-04-17 17:52:28,527 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028548.2, seqID NZ_CP028548.2 [record index 0].\n",
      "2024-04-17 17:52:28,584 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029384.2, seqID NZ_CP029384.2 [record index 0].\n",
      "2024-04-17 17:52:28,635 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028783.2, seqID NZ_CP028783.2 [record index 0].\n",
      "2024-04-17 17:52:28,690 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031721.1, seqID NZ_CP031721.1 [record index 0].\n",
      "2024-04-17 17:52:28,743 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050858.1, seqID NZ_CP050858.1 [record index 0].\n",
      "2024-04-17 17:52:28,792 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082799.1, seqID NZ_CP082799.1 [record index 0].\n",
      "2024-04-17 17:52:28,843 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082805.1, seqID NZ_CP082805.1 [record index 0].\n",
      "2024-04-17 17:52:28,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030269.1, seqID NZ_CP030269.1 [record index 0].\n",
      "2024-04-17 17:52:28,940 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084829.1, seqID NZ_CP084829.1 [record index 0].\n",
      "2024-04-17 17:52:28,990 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084830.1, seqID NZ_CP084830.1 [record index 0].\n",
      "2024-04-17 17:52:29,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084843.1, seqID NZ_CP084843.1 [record index 0].\n",
      "2024-04-17 17:52:29,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084866.1, seqID NZ_CP084866.1 [record index 0].\n",
      "2024-04-17 17:52:29,139 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084868.1, seqID NZ_CP084868.1 [record index 0].\n",
      "2024-04-17 17:52:29,189 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063881.1, seqID NZ_CP063881.1 [record index 0].\n",
      "2024-04-17 17:52:29,236 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063925.1, seqID NZ_CP063925.1 [record index 0].\n",
      "2024-04-17 17:52:29,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063945.1, seqID NZ_CP063945.1 [record index 0].\n",
      "2024-04-17 17:52:29,342 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058548.1, seqID NZ_CP058548.1 [record index 0].\n",
      "2024-04-17 17:52:29,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014010.1, seqID NZ_CP014010.1 [record index 0].\n",
      "2024-04-17 17:52:29,447 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014008.1, seqID NZ_CP014008.1 [record index 0].\n",
      "2024-04-17 17:52:29,500 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068684.1, seqID NZ_CP068684.1 [record index 0].\n",
      "2024-04-17 17:52:29,555 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068689.1, seqID NZ_CP068689.1 [record index 0].\n",
      "2024-04-17 17:52:29,611 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP019047.1, seqID NZ_CP019047.1 [record index 0].\n",
      "2024-04-17 17:52:29,663 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056216.1, seqID NZ_CP056216.1 [record index 0].\n",
      "2024-04-17 17:52:29,717 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056297.1, seqID NZ_CP056297.1 [record index 0].\n",
      "2024-04-17 17:52:29,768 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056301.1, seqID NZ_CP056301.1 [record index 0].\n",
      "2024-04-17 17:52:29,827 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056307.1, seqID NZ_CP056307.1 [record index 0].\n",
      "2024-04-17 17:52:29,877 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056372.1, seqID NZ_CP056372.1 [record index 0].\n",
      "2024-04-17 17:52:29,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056385.1, seqID NZ_CP056385.1 [record index 0].\n",
      "2024-04-17 17:52:29,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056432.1, seqID NZ_CP056432.1 [record index 0].\n",
      "2024-04-17 17:52:30,032 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055509.1, seqID NZ_CP055509.1 [record index 0].\n",
      "2024-04-17 17:52:30,082 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056878.1, seqID NZ_CP056878.1 [record index 0].\n",
      "2024-04-17 17:52:30,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056883.1, seqID NZ_CP056883.1 [record index 0].\n",
      "2024-04-17 17:52:30,180 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057066.1, seqID NZ_CP057066.1 [record index 0].\n",
      "2024-04-17 17:52:30,228 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057069.1, seqID NZ_CP057069.1 [record index 0].\n",
      "2024-04-17 17:52:30,282 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057313.1, seqID NZ_CP057313.1 [record index 0].\n",
      "2024-04-17 17:52:30,332 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034082.1, seqID NZ_CP034082.1 [record index 0].\n",
      "2024-04-17 17:52:30,379 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031798.1, seqID NZ_CP031798.1 [record index 0].\n",
      "2024-04-17 17:52:30,429 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065441.1, seqID NZ_CP065441.1 [record index 0].\n",
      "2024-04-17 17:52:30,476 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081896.1, seqID NZ_CP081896.1 [record index 0].\n",
      "2024-04-17 17:52:30,527 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090931.1, seqID NZ_CP090931.1 [record index 0].\n",
      "2024-04-17 17:52:30,580 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065464.1, seqID NZ_CP065464.1 [record index 0].\n",
      "2024-04-17 17:52:30,631 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP025246.1, seqID NZ_AP025246.1 [record index 0].\n",
      "2024-04-17 17:52:30,683 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065447.1, seqID NZ_CP065447.1 [record index 0].\n",
      "2024-04-17 17:52:30,741 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015385.1, seqID NZ_CP015385.1 [record index 0].\n",
      "2024-04-17 17:52:30,792 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026586.1, seqID NZ_CP026586.1 [record index 0].\n",
      "2024-04-17 17:52:30,842 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061380.1, seqID NZ_CP061380.1 [record index 0].\n",
      "2024-04-17 17:52:30,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061385.1, seqID NZ_CP061385.1 [record index 0].\n",
      "2024-04-17 17:52:30,945 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061396.1, seqID NZ_CP061396.1 [record index 0].\n",
      "2024-04-17 17:52:30,997 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061399.1, seqID NZ_CP061399.1 [record index 0].\n",
      "2024-04-17 17:52:31,051 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025037.1, seqID NZ_CP025037.1 [record index 0].\n",
      "2024-04-17 17:52:31,099 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_012731.1, seqID NC_012731.1 [record index 0].\n",
      "2024-04-17 17:52:31,156 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091582.1, seqID NZ_CP091582.1 [record index 0].\n",
      "2024-04-17 17:52:31,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041644.1, seqID NZ_CP041644.1 [record index 0].\n",
      "2024-04-17 17:52:31,253 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035531.1, seqID NZ_CP035531.1 [record index 0].\n",
      "2024-04-17 17:52:31,307 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081894.1, seqID NZ_CP081894.1 [record index 0].\n",
      "2024-04-17 17:52:31,357 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034760.1, seqID NZ_CP034760.1 [record index 0].\n",
      "2024-04-17 17:52:31,414 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092656.1, seqID NZ_CP092656.1 [record index 0].\n",
      "2024-04-17 17:52:31,467 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091846.1, seqID NZ_CP091846.1 [record index 0].\n",
      "2024-04-17 17:52:31,520 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091992.1, seqID NZ_CP091992.1 [record index 0].\n",
      "2024-04-17 17:52:31,573 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031577.1, seqID NZ_CP031577.1 [record index 0].\n",
      "2024-04-17 17:52:31,623 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047271.1, seqID NZ_CP047271.1 [record index 0].\n",
      "2024-04-17 17:52:31,676 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031800.1, seqID NZ_CP031800.1 [record index 0].\n",
      "2024-04-17 17:52:31,729 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065438.1, seqID NZ_CP065438.1 [record index 0].\n",
      "2024-04-17 17:52:31,782 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018428.1, seqID NZ_CP018428.1 [record index 0].\n",
      "2024-04-17 17:52:31,834 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018427.1, seqID NZ_CP018427.1 [record index 0].\n",
      "2024-04-17 17:52:31,887 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018437.1, seqID NZ_CP018437.1 [record index 0].\n",
      "2024-04-17 17:52:31,933 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086128.1, seqID NZ_CP086128.1 [record index 0].\n",
      "2024-04-17 17:52:31,994 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063277.1, seqID NZ_CP063277.1 [record index 0].\n",
      "2024-04-17 17:52:32,043 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065457.1, seqID NZ_CP065457.1 [record index 0].\n",
      "2024-04-17 17:52:32,092 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP023453.1, seqID NZ_AP023453.1 [record index 0].\n",
      "2024-04-17 17:52:32,139 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064129.1, seqID NZ_CP064129.1 [record index 0].\n",
      "2024-04-17 17:52:32,188 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063908.1, seqID NZ_CP063908.1 [record index 0].\n",
      "2024-04-17 17:52:32,234 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063930.1, seqID NZ_CP063930.1 [record index 0].\n",
      "2024-04-17 17:52:32,290 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063878.1, seqID NZ_CP063878.1 [record index 0].\n",
      "2024-04-17 17:52:32,337 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040391.1, seqID NZ_CP040391.1 [record index 0].\n",
      "2024-04-17 17:52:32,387 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040122.1, seqID NZ_CP040122.1 [record index 0].\n",
      "2024-04-17 17:52:32,439 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035202.1, seqID NZ_CP035202.1 [record index 0].\n",
      "2024-04-17 17:52:32,487 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045782.1, seqID NZ_CP045782.1 [record index 0].\n",
      "2024-04-17 17:52:32,539 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058325.1, seqID NZ_CP058325.1 [record index 0].\n",
      "2024-04-17 17:52:32,592 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033960.1, seqID NZ_CP033960.1 [record index 0].\n",
      "2024-04-17 17:52:32,644 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033954.1, seqID NZ_CP033954.1 [record index 0].\n",
      "2024-04-17 17:52:32,695 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047677.1, seqID NZ_CP047677.1 [record index 0].\n",
      "2024-04-17 17:52:32,742 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047675.1, seqID NZ_CP047675.1 [record index 0].\n",
      "2024-04-17 17:52:32,790 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890564.1, seqID NZ_LR890564.1 [record index 0].\n",
      "2024-04-17 17:52:32,837 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083784.1, seqID NZ_CP083784.1 [record index 0].\n",
      "2024-04-17 17:52:32,882 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890391.1, seqID NZ_LR890391.1 [record index 0].\n",
      "2024-04-17 17:52:32,929 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890343.1, seqID NZ_LR890343.1 [record index 0].\n",
      "2024-04-17 17:52:32,977 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890620.1, seqID NZ_LR890620.1 [record index 0].\n",
      "2024-04-17 17:52:33,024 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890484.1, seqID NZ_LR890484.1 [record index 0].\n",
      "2024-04-17 17:52:33,073 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890434.1, seqID NZ_LR890434.1 [record index 0].\n",
      "2024-04-17 17:52:33,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024535.1, seqID NZ_CP024535.1 [record index 0].\n",
      "2024-04-17 17:52:33,171 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024496.1, seqID NZ_CP024496.1 [record index 0].\n",
      "2024-04-17 17:52:33,216 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890230.1, seqID NZ_LR890230.1 [record index 0].\n",
      "2024-04-17 17:52:33,269 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890500.1, seqID NZ_LR890500.1 [record index 0].\n",
      "2024-04-17 17:52:33,319 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024191.1, seqID NZ_CP024191.1 [record index 0].\n",
      "2024-04-17 17:52:33,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890722.1, seqID NZ_LR890722.1 [record index 0].\n",
      "2024-04-17 17:52:33,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073783.1, seqID NZ_CP073783.1 [record index 0].\n",
      "2024-04-17 17:52:33,461 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073791.1, seqID NZ_CP073791.1 [record index 0].\n",
      "2024-04-17 17:52:33,522 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018438.1, seqID NZ_CP018438.1 [record index 0].\n",
      "2024-04-17 17:52:33,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018692.1, seqID NZ_CP018692.1 [record index 0].\n",
      "2024-04-17 17:52:33,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018450.1, seqID NZ_CP018450.1 [record index 0].\n",
      "2024-04-17 17:52:33,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018364.1, seqID NZ_CP018364.1 [record index 0].\n",
      "2024-04-17 17:52:33,752 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018447.1, seqID NZ_CP018447.1 [record index 0].\n",
      "2024-04-17 17:52:33,809 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018735.1, seqID NZ_CP018735.1 [record index 0].\n",
      "2024-04-17 17:52:33,865 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069170.1, seqID NZ_CP069170.1 [record index 0].\n",
      "2024-04-17 17:52:33,914 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008831.1, seqID NZ_CP008831.1 [record index 0].\n",
      "2024-04-17 17:52:33,962 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065825.1, seqID NZ_CP065825.1 [record index 0].\n",
      "2024-04-17 17:52:34,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047595.1, seqID NZ_CP047595.1 [record index 0].\n",
      "2024-04-17 17:52:34,065 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090242.1, seqID NZ_CP090242.1 [record index 0].\n",
      "2024-04-17 17:52:34,115 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036450.1, seqID NZ_CP036450.1 [record index 0].\n",
      "2024-04-17 17:52:34,166 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014647.1, seqID NZ_CP014647.1 [record index 0].\n",
      "2024-04-17 17:52:34,218 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009771.1, seqID NZ_CP009771.1 [record index 0].\n",
      "2024-04-17 17:52:34,272 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009775.1, seqID NZ_CP009775.1 [record index 0].\n",
      "2024-04-17 17:52:34,323 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009872.1, seqID NZ_CP009872.1 [record index 0].\n",
      "2024-04-17 17:52:34,376 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008797.1, seqID NZ_CP008797.1 [record index 0].\n",
      "2024-04-17 17:52:34,429 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP007727.1, seqID NZ_CP007727.1 [record index 0].\n",
      "2024-04-17 17:52:34,482 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008827.1, seqID NZ_CP008827.1 [record index 0].\n",
      "2024-04-17 17:52:34,533 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090432.1, seqID NZ_CP090432.1 [record index 0].\n",
      "2024-04-17 17:52:34,584 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042977.1, seqID NZ_CP042977.1 [record index 0].\n",
      "2024-04-17 17:52:34,639 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015130.1, seqID NZ_CP015130.1 [record index 0].\n",
      "2024-04-17 17:52:34,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053016.1, seqID NZ_CP053016.1 [record index 0].\n",
      "2024-04-17 17:52:34,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072496.1, seqID NZ_CP072496.1 [record index 0].\n",
      "2024-04-17 17:52:34,796 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072492.1, seqID NZ_CP072492.1 [record index 0].\n",
      "2024-04-17 17:52:34,845 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072653.1, seqID NZ_CP072653.1 [record index 0].\n",
      "2024-04-17 17:52:34,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR745045.1, seqID NZ_LR745045.1 [record index 0].\n",
      "2024-04-17 17:52:34,949 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020841.1, seqID NZ_CP020841.1 [record index 0].\n",
      "2024-04-17 17:52:35,002 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053875.1, seqID NZ_CP053875.1 [record index 0].\n",
      "2024-04-17 17:52:35,054 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065172.1, seqID NZ_CP065172.1 [record index 0].\n",
      "2024-04-17 17:52:35,106 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP074116.1, seqID NZ_CP074116.1 [record index 0].\n",
      "2024-04-17 17:52:35,155 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027189.1, seqID NZ_CP027189.1 [record index 0].\n",
      "2024-04-17 17:52:35,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058226.1, seqID NZ_CP058226.1 [record index 0].\n",
      "2024-04-17 17:52:35,255 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130548.1, seqID NZ_LR130548.1 [record index 0].\n",
      "2024-04-17 17:52:35,308 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040023.1, seqID NZ_CP040023.1 [record index 0].\n",
      "2024-04-17 17:52:35,359 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040038.1, seqID NZ_CP040038.1 [record index 0].\n",
      "2024-04-17 17:52:35,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040028.1, seqID NZ_CP040028.1 [record index 0].\n",
      "2024-04-17 17:52:35,465 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040033.1, seqID NZ_CP040033.1 [record index 0].\n",
      "2024-04-17 17:52:35,518 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP078122.1, seqID NZ_CP078122.1 [record index 0].\n",
      "2024-04-17 17:52:35,569 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090623.1, seqID NZ_CP090623.1 [record index 0].\n",
      "2024-04-17 17:52:35,623 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090626.1, seqID NZ_CP090626.1 [record index 0].\n",
      "2024-04-17 17:52:35,674 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090629.1, seqID NZ_CP090629.1 [record index 0].\n",
      "2024-04-17 17:52:35,722 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070602.1, seqID NZ_CP070602.1 [record index 0].\n",
      "2024-04-17 17:52:35,774 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070598.1, seqID NZ_CP070598.1 [record index 0].\n",
      "2024-04-17 17:52:35,825 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049604.1, seqID NZ_CP049604.1 [record index 0].\n",
      "2024-04-17 17:52:35,879 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058361.1, seqID NZ_CP058361.1 [record index 0].\n",
      "2024-04-17 17:52:35,934 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015120.1, seqID NZ_CP015120.1 [record index 0].\n",
      "2024-04-17 17:52:35,988 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066151.1, seqID NZ_CP066151.1 [record index 0].\n",
      "2024-04-17 17:52:36,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025456.1, seqID NZ_CP025456.1 [record index 0].\n",
      "2024-04-17 17:52:36,091 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044258.1, seqID NZ_CP044258.1 [record index 0].\n",
      "2024-04-17 17:52:36,139 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077763.1, seqID NZ_CP077763.1 [record index 0].\n",
      "2024-04-17 17:52:36,193 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041373.1, seqID NZ_CP041373.1 [record index 0].\n",
      "2024-04-17 17:52:36,247 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055294.1, seqID NZ_CP055294.1 [record index 0].\n",
      "2024-04-17 17:52:36,298 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP087669.1, seqID NZ_CP087669.1 [record index 0].\n",
      "2024-04-17 17:52:36,349 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070579.1, seqID NZ_CP070579.1 [record index 0].\n",
      "2024-04-17 17:52:36,402 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090462.1, seqID NZ_CP090462.1 [record index 0].\n",
      "2024-04-17 17:52:36,454 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070568.1, seqID NZ_CP070568.1 [record index 0].\n",
      "2024-04-17 17:52:36,508 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070564.1, seqID NZ_CP070564.1 [record index 0].\n",
      "2024-04-17 17:52:36,560 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058358.1, seqID NZ_CP058358.1 [record index 0].\n",
      "2024-04-17 17:52:36,608 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082791.1, seqID NZ_CP082791.1 [record index 0].\n",
      "2024-04-17 17:52:36,657 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070528.1, seqID NZ_CP070528.1 [record index 0].\n",
      "2024-04-17 17:52:36,709 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089021.1, seqID NZ_CP089021.1 [record index 0].\n",
      "2024-04-17 17:52:36,762 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047192.1, seqID NZ_CP047192.1 [record index 0].\n",
      "2024-04-17 17:52:36,814 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091058.1, seqID NZ_CP091058.1 [record index 0].\n",
      "2024-04-17 17:52:36,864 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089598.1, seqID NZ_CP089598.1 [record index 0].\n",
      "2024-04-17 17:52:36,915 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083445.1, seqID NZ_CP083445.1 [record index 0].\n",
      "2024-04-17 17:52:36,965 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091061.1, seqID NZ_CP091061.1 [record index 0].\n",
      "2024-04-17 17:52:37,020 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065949.1, seqID NZ_CP065949.1 [record index 0].\n",
      "2024-04-17 17:52:37,073 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041082.1, seqID NZ_CP041082.1 [record index 0].\n",
      "2024-04-17 17:52:37,126 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054720.1, seqID NZ_CP054720.1 [record index 0].\n",
      "2024-04-17 17:52:37,180 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054726.1, seqID NZ_CP054726.1 [record index 0].\n",
      "2024-04-17 17:52:37,233 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054732.1, seqID NZ_CP054732.1 [record index 0].\n",
      "2024-04-17 17:52:37,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054738.1, seqID NZ_CP054738.1 [record index 0].\n",
      "2024-04-17 17:52:37,340 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054744.1, seqID NZ_CP054744.1 [record index 0].\n",
      "2024-04-17 17:52:37,393 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054750.1, seqID NZ_CP054750.1 [record index 0].\n",
      "2024-04-17 17:52:37,445 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054756.1, seqID NZ_CP054756.1 [record index 0].\n",
      "2024-04-17 17:52:37,498 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054762.1, seqID NZ_CP054762.1 [record index 0].\n",
      "2024-04-17 17:52:37,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054768.1, seqID NZ_CP054768.1 [record index 0].\n",
      "2024-04-17 17:52:37,606 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054774.1, seqID NZ_CP054774.1 [record index 0].\n",
      "2024-04-17 17:52:37,659 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054780.1, seqID NZ_CP054780.1 [record index 0].\n",
      "2024-04-17 17:52:37,707 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051160.1, seqID NZ_CP051160.1 [record index 0].\n",
      "2024-04-17 17:52:37,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063147.1, seqID NZ_CP063147.1 [record index 0].\n",
      "2024-04-17 17:52:37,812 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063747.1, seqID NZ_CP063747.1 [record index 0].\n",
      "2024-04-17 17:52:37,869 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082041.1, seqID NZ_CP082041.1 [record index 0].\n",
      "2024-04-17 17:52:37,922 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047160.1, seqID NZ_CP047160.1 [record index 0].\n",
      "2024-04-17 17:52:37,976 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072996.1, seqID NZ_CP072996.1 [record index 0].\n",
      "2024-04-17 17:52:38,028 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082010.1, seqID NZ_CP082010.1 [record index 0].\n",
      "2024-04-17 17:52:38,081 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP048430.1, seqID NZ_CP048430.1 [record index 0].\n",
      "2024-04-17 17:52:38,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034249.1, seqID NZ_CP034249.1 [record index 0].\n",
      "2024-04-17 17:52:38,185 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082014.1, seqID NZ_CP082014.1 [record index 0].\n",
      "2024-04-17 17:52:38,237 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082037.1, seqID NZ_CP082037.1 [record index 0].\n",
      "2024-04-17 17:52:38,288 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082032.1, seqID NZ_CP082032.1 [record index 0].\n",
      "2024-04-17 17:52:38,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084986.1, seqID NZ_CP084986.1 [record index 0].\n",
      "2024-04-17 17:52:38,389 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP048933.1, seqID NZ_CP048933.1 [record index 0].\n",
      "2024-04-17 17:52:38,442 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082024.1, seqID NZ_CP082024.1 [record index 0].\n",
      "2024-04-17 17:52:38,496 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082019.1, seqID NZ_CP082019.1 [record index 0].\n",
      "2024-04-17 17:52:38,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082001.1, seqID NZ_CP082001.1 [record index 0].\n",
      "2024-04-17 17:52:38,604 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041022.1, seqID NZ_CP041022.1 [record index 0].\n",
      "2024-04-17 17:52:38,650 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085478.1, seqID NZ_CP085478.1 [record index 0].\n",
      "2024-04-17 17:52:38,703 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072463.1, seqID NZ_CP072463.1 [record index 0].\n",
      "2024-04-17 17:52:38,754 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060744.1, seqID NZ_CP060744.1 [record index 0].\n",
      "2024-04-17 17:52:38,809 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082154.1, seqID NZ_CP082154.1 [record index 0].\n",
      "2024-04-17 17:52:38,862 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084705.1, seqID NZ_CP084705.1 [record index 0].\n",
      "2024-04-17 17:52:38,915 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OU342706.1, seqID NZ_OU342706.1 [record index 0].\n",
      "2024-04-17 17:52:38,963 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086285.1, seqID NZ_CP086285.1 [record index 0].\n",
      "2024-04-17 17:52:39,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053862.1, seqID NZ_CP053862.1 [record index 0].\n",
      "2024-04-17 17:52:39,068 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080303.1, seqID NZ_CP080303.1 [record index 0].\n",
      "2024-04-17 17:52:39,120 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080297.1, seqID NZ_CP080297.1 [record index 0].\n",
      "2024-04-17 17:52:39,172 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP037928.1, seqID NZ_CP037928.1 [record index 0].\n",
      "2024-04-17 17:52:39,225 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080309.1, seqID NZ_CP080309.1 [record index 0].\n",
      "2024-04-17 17:52:39,277 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043047.1, seqID NZ_CP043047.1 [record index 0].\n",
      "2024-04-17 17:52:39,323 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR745042.1, seqID NZ_LR745042.1 [record index 0].\n",
      "2024-04-17 17:52:39,373 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018306.1, seqID NZ_CP018306.1 [record index 0].\n",
      "2024-04-17 17:52:39,426 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020901.1, seqID NZ_CP020901.1 [record index 0].\n",
      "2024-04-17 17:52:39,476 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026011.1, seqID NZ_CP026011.1 [record index 0].\n",
      "2024-04-17 17:52:39,525 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076518.1, seqID NZ_CP076518.1 [record index 0].\n",
      "2024-04-17 17:52:39,576 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072460.1, seqID NZ_CP072460.1 [record index 0].\n",
      "2024-04-17 17:52:39,626 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064211.1, seqID NZ_CP064211.1 [record index 0].\n",
      "2024-04-17 17:52:39,677 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064217.1, seqID NZ_CP064217.1 [record index 0].\n",
      "2024-04-17 17:52:39,727 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064223.1, seqID NZ_CP064223.1 [record index 0].\n",
      "2024-04-17 17:52:39,775 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064229.1, seqID NZ_CP064229.1 [record index 0].\n",
      "2024-04-17 17:52:39,823 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064232.1, seqID NZ_CP064232.1 [record index 0].\n",
      "2024-04-17 17:52:39,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064235.1, seqID NZ_CP064235.1 [record index 0].\n",
      "2024-04-17 17:52:39,923 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064241.1, seqID NZ_CP064241.1 [record index 0].\n",
      "2024-04-17 17:52:39,975 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064246.1, seqID NZ_CP064246.1 [record index 0].\n",
      "2024-04-17 17:52:40,023 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064205.1, seqID NZ_CP064205.1 [record index 0].\n",
      "2024-04-17 17:52:40,072 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064208.1, seqID NZ_CP064208.1 [record index 0].\n",
      "2024-04-17 17:52:40,124 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064258.1, seqID NZ_CP064258.1 [record index 0].\n",
      "2024-04-17 17:52:40,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064252.1, seqID NZ_CP064252.1 [record index 0].\n",
      "2024-04-17 17:52:40,225 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070970.1, seqID NZ_CP070970.1 [record index 0].\n",
      "2024-04-17 17:52:40,276 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025466.1, seqID NZ_CP025466.1 [record index 0].\n",
      "2024-04-17 17:52:40,331 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091978.1, seqID NZ_CP091978.1 [record index 0].\n",
      "2024-04-17 17:52:40,381 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070898.1, seqID NZ_CP070898.1 [record index 0].\n",
      "2024-04-17 17:52:40,436 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090203.1, seqID NZ_CP090203.1 [record index 0].\n",
      "2024-04-17 17:52:40,488 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061961.1, seqID NZ_CP061961.1 [record index 0].\n",
      "2024-04-17 17:52:40,543 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP013711.1, seqID NZ_CP013711.1 [record index 0].\n",
      "2024-04-17 17:52:40,594 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061957.1, seqID NZ_CP061957.1 [record index 0].\n",
      "2024-04-17 17:52:40,642 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061952.1, seqID NZ_CP061952.1 [record index 0].\n",
      "2024-04-17 17:52:40,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068572.1, seqID NZ_CP068572.1 [record index 0].\n",
      "2024-04-17 17:52:40,742 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890616.1, seqID NZ_LR890616.1 [record index 0].\n",
      "2024-04-17 17:52:40,788 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890217.1, seqID NZ_LR890217.1 [record index 0].\n",
      "2024-04-17 17:52:40,835 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890643.1, seqID NZ_LR890643.1 [record index 0].\n",
      "2024-04-17 17:52:40,882 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890235.1, seqID NZ_LR890235.1 [record index 0].\n",
      "2024-04-17 17:52:40,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890384.1, seqID NZ_LR890384.1 [record index 0].\n",
      "2024-04-17 17:52:40,973 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890424.1, seqID NZ_LR890424.1 [record index 0].\n",
      "2024-04-17 17:52:41,021 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890240.1, seqID NZ_LR890240.1 [record index 0].\n",
      "2024-04-17 17:52:41,077 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890571.1, seqID NZ_LR890571.1 [record index 0].\n",
      "2024-04-17 17:52:41,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890735.1, seqID NZ_LR890735.1 [record index 0].\n",
      "2024-04-17 17:52:41,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890669.1, seqID NZ_LR890669.1 [record index 0].\n",
      "2024-04-17 17:52:41,225 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890254.1, seqID NZ_LR890254.1 [record index 0].\n",
      "2024-04-17 17:52:41,277 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890702.1, seqID NZ_LR890702.1 [record index 0].\n",
      "2024-04-17 17:52:41,325 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890556.1, seqID NZ_LR890556.1 [record index 0].\n",
      "2024-04-17 17:52:41,377 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890632.1, seqID NZ_LR890632.1 [record index 0].\n",
      "2024-04-17 17:52:41,428 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890426.1, seqID NZ_LR890426.1 [record index 0].\n",
      "2024-04-17 17:52:41,481 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890515.1, seqID NZ_LR890515.1 [record index 0].\n",
      "2024-04-17 17:52:41,534 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024563.1, seqID NZ_CP024563.1 [record index 0].\n",
      "2024-04-17 17:52:41,582 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890204.1, seqID NZ_LR890204.1 [record index 0].\n",
      "2024-04-17 17:52:41,633 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024570.1, seqID NZ_CP024570.1 [record index 0].\n",
      "2024-04-17 17:52:41,690 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890648.1, seqID NZ_LR890648.1 [record index 0].\n",
      "2024-04-17 17:52:41,740 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890562.1, seqID NZ_LR890562.1 [record index 0].\n",
      "2024-04-17 17:52:41,789 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890624.1, seqID NZ_LR890624.1 [record index 0].\n",
      "2024-04-17 17:52:41,838 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031817.1, seqID NZ_CP031817.1 [record index 0].\n",
      "2024-04-17 17:52:41,883 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890595.1, seqID NZ_LR890595.1 [record index 0].\n",
      "2024-04-17 17:52:41,930 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890317.1, seqID NZ_LR890317.1 [record index 0].\n",
      "2024-04-17 17:52:41,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031808.1, seqID NZ_CP031808.1 [record index 0].\n",
      "2024-04-17 17:52:42,027 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890554.1, seqID NZ_LR890554.1 [record index 0].\n",
      "2024-04-17 17:52:42,074 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890446.1, seqID NZ_LR890446.1 [record index 0].\n",
      "2024-04-17 17:52:42,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024556.1, seqID NZ_CP024556.1 [record index 0].\n",
      "2024-04-17 17:52:42,175 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024549.1, seqID NZ_CP024549.1 [record index 0].\n",
      "2024-04-17 17:52:42,225 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024521.1, seqID NZ_CP024521.1 [record index 0].\n",
      "2024-04-17 17:52:42,275 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024528.1, seqID NZ_CP024528.1 [record index 0].\n",
      "2024-04-17 17:52:42,323 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890381.1, seqID NZ_LR890381.1 [record index 0].\n",
      "2024-04-17 17:52:42,370 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890569.1, seqID NZ_LR890569.1 [record index 0].\n",
      "2024-04-17 17:52:42,418 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890547.1, seqID NZ_LR890547.1 [record index 0].\n",
      "2024-04-17 17:52:42,465 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890189.1, seqID NZ_LR890189.1 [record index 0].\n",
      "2024-04-17 17:52:42,493 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083775.1, seqID NZ_CP083775.1 [record index 0].\n",
      "2024-04-17 17:52:42,541 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031792.1, seqID NZ_CP031792.1 [record index 0].\n",
      "2024-04-17 17:52:42,589 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890726.1, seqID NZ_LR890726.1 [record index 0].\n",
      "2024-04-17 17:52:42,637 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890464.1, seqID NZ_LR890464.1 [record index 0].\n",
      "2024-04-17 17:52:42,688 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032831.1, seqID NZ_CP032831.1 [record index 0].\n",
      "2024-04-17 17:52:42,738 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890185.1, seqID NZ_LR890185.1 [record index 0].\n",
      "2024-04-17 17:52:42,787 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024545.1, seqID NZ_CP024545.1 [record index 0].\n",
      "2024-04-17 17:52:42,839 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890360.1, seqID NZ_LR890360.1 [record index 0].\n",
      "2024-04-17 17:52:42,888 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890395.1, seqID NZ_LR890395.1 [record index 0].\n",
      "2024-04-17 17:52:42,938 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024542.1, seqID NZ_CP024542.1 [record index 0].\n",
      "2024-04-17 17:52:42,984 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890456.1, seqID NZ_LR890456.1 [record index 0].\n",
      "2024-04-17 17:52:43,031 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890209.1, seqID NZ_LR890209.1 [record index 0].\n",
      "2024-04-17 17:52:43,082 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083796.1, seqID NZ_CP083796.1 [record index 0].\n",
      "2024-04-17 17:52:43,128 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890493.1, seqID NZ_LR890493.1 [record index 0].\n",
      "2024-04-17 17:52:43,179 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890202.1, seqID NZ_LR890202.1 [record index 0].\n",
      "2024-04-17 17:52:43,231 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054285.1, seqID NZ_CP054285.1 [record index 0].\n",
      "2024-04-17 17:52:43,281 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030070.1, seqID NZ_CP030070.1 [record index 0].\n",
      "2024-04-17 17:52:43,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081890.1, seqID NZ_CP081890.1 [record index 0].\n",
      "2024-04-17 17:52:43,381 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_016845.1, seqID NC_016845.1 [record index 0].\n",
      "2024-04-17 17:52:43,430 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006738.1, seqID NZ_CP006738.1 [record index 0].\n",
      "2024-04-17 17:52:43,478 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061842.1, seqID NZ_CP061842.1 [record index 0].\n",
      "2024-04-17 17:52:43,530 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039524.1, seqID NZ_CP039524.1 [record index 0].\n",
      "2024-04-17 17:52:43,582 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031934.1, seqID NZ_CP031934.1 [record index 0].\n",
      "2024-04-17 17:52:43,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018671.1, seqID NZ_AP018671.1 [record index 0].\n",
      "2024-04-17 17:52:43,689 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075890.1, seqID NZ_CP075890.1 [record index 0].\n",
      "2024-04-17 17:52:43,742 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065460.1, seqID NZ_CP065460.1 [record index 0].\n",
      "2024-04-17 17:52:43,792 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025951.1, seqID NZ_CP025951.1 [record index 0].\n",
      "2024-04-17 17:52:43,844 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063759.1, seqID NZ_CP063759.1 [record index 0].\n",
      "2024-04-17 17:52:43,898 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073002.1, seqID NZ_CP073002.1 [record index 0].\n",
      "2024-04-17 17:52:43,949 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065554.1, seqID NZ_CP065554.1 [record index 0].\n",
      "2024-04-17 17:52:44,006 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044047.1, seqID NZ_CP044047.1 [record index 0].\n",
      "2024-04-17 17:52:44,058 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023946.1, seqID NZ_CP023946.1 [record index 0].\n",
      "2024-04-17 17:52:44,112 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023941.1, seqID NZ_CP023941.1 [record index 0].\n",
      "2024-04-17 17:52:44,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070087.1, seqID NZ_CP070087.1 [record index 0].\n",
      "2024-04-17 17:52:44,217 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069828.1, seqID NZ_CP069828.1 [record index 0].\n",
      "2024-04-17 17:52:44,266 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070037.1, seqID NZ_CP070037.1 [record index 0].\n",
      "2024-04-17 17:52:44,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069939.1, seqID NZ_CP069939.1 [record index 0].\n",
      "2024-04-17 17:52:44,363 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069961.1, seqID NZ_CP069961.1 [record index 0].\n",
      "2024-04-17 17:52:44,416 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069907.1, seqID NZ_CP069907.1 [record index 0].\n",
      "2024-04-17 17:52:44,463 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069940.1, seqID NZ_CP069940.1 [record index 0].\n",
      "2024-04-17 17:52:44,513 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069861.1, seqID NZ_CP069861.1 [record index 0].\n",
      "2024-04-17 17:52:44,562 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068139.1, seqID NZ_CP068139.1 [record index 0].\n",
      "2024-04-17 17:52:44,612 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090398.1, seqID NZ_CP090398.1 [record index 0].\n",
      "2024-04-17 17:52:44,662 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026159.1, seqID NZ_CP026159.1 [record index 0].\n",
      "2024-04-17 17:52:44,711 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026164.1, seqID NZ_CP026164.1 [record index 0].\n",
      "2024-04-17 17:52:44,765 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026136.1, seqID NZ_CP026136.1 [record index 0].\n",
      "2024-04-17 17:52:44,819 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081820.1, seqID NZ_CP081820.1 [record index 0].\n",
      "2024-04-17 17:52:44,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026132.1, seqID NZ_CP026132.1 [record index 0].\n",
      "2024-04-17 17:52:44,929 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025461.1, seqID NZ_CP025461.1 [record index 0].\n",
      "2024-04-17 17:52:44,981 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026149.1, seqID NZ_CP026149.1 [record index 0].\n",
      "2024-04-17 17:52:45,033 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026145.1, seqID NZ_CP026145.1 [record index 0].\n",
      "2024-04-17 17:52:45,084 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026140.1, seqID NZ_CP026140.1 [record index 0].\n",
      "2024-04-17 17:52:45,138 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026130.1, seqID NZ_CP026130.1 [record index 0].\n",
      "2024-04-17 17:52:45,190 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061350.1, seqID NZ_CP061350.1 [record index 0].\n",
      "2024-04-17 17:52:45,245 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016814.1, seqID NZ_CP016814.1 [record index 0].\n",
      "2024-04-17 17:52:45,305 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016813.1, seqID NZ_CP016813.1 [record index 0].\n",
      "2024-04-17 17:52:45,357 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR025099.1, seqID NZ_LR025099.1 [record index 0].\n",
      "2024-04-17 17:52:45,407 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039960.1, seqID NZ_CP039960.1 [record index 0].\n",
      "2024-04-17 17:52:45,458 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039957.1, seqID NZ_CP039957.1 [record index 0].\n",
      "2024-04-17 17:52:45,510 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011976.1, seqID NZ_CP011976.1 [record index 0].\n",
      "2024-04-17 17:52:45,559 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065436.1, seqID NZ_CP065436.1 [record index 0].\n",
      "2024-04-17 17:52:45,613 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066254.1, seqID NZ_CP066254.1 [record index 0].\n",
      "2024-04-17 17:52:45,666 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063833.1, seqID NZ_CP063833.1 [record index 0].\n",
      "2024-04-17 17:52:45,719 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066249.1, seqID NZ_CP066249.1 [record index 0].\n",
      "2024-04-17 17:52:45,773 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034039.1, seqID NZ_CP034039.1 [record index 0].\n",
      "2024-04-17 17:52:45,828 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063019.1, seqID NZ_CP063019.1 [record index 0].\n",
      "2024-04-17 17:52:45,855 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP062997.1, seqID NZ_CP062997.1 [record index 0].\n",
      "2024-04-17 17:52:45,904 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063003.1, seqID NZ_CP063003.1 [record index 0].\n",
      "2024-04-17 17:52:45,955 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063008.1, seqID NZ_CP063008.1 [record index 0].\n",
      "2024-04-17 17:52:46,006 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063011.1, seqID NZ_CP063011.1 [record index 0].\n",
      "2024-04-17 17:52:46,064 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015392.1, seqID NZ_CP015392.1 [record index 0].\n",
      "2024-04-17 17:52:46,091 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040545.1, seqID NZ_CP040545.1 [record index 0].\n",
      "2024-04-17 17:52:46,144 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040539.1, seqID NZ_CP040539.1 [record index 0].\n",
      "2024-04-17 17:52:46,197 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040533.1, seqID NZ_CP040533.1 [record index 0].\n",
      "2024-04-17 17:52:46,247 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073351.1, seqID NZ_CP073351.1 [record index 0].\n",
      "2024-04-17 17:52:46,295 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_022566.1, seqID NC_022566.1 [record index 0].\n",
      "2024-04-17 17:52:46,348 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077773.1, seqID NZ_CP077773.1 [record index 0].\n",
      "2024-04-17 17:52:46,397 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035540.1, seqID NZ_CP035540.1 [record index 0].\n",
      "2024-04-17 17:52:46,454 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011647.1, seqID NZ_CP011647.1 [record index 0].\n",
      "2024-04-17 17:52:46,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018356.1, seqID NZ_CP018356.1 [record index 0].\n",
      "2024-04-17 17:52:46,557 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018352.1, seqID NZ_CP018352.1 [record index 0].\n",
      "2024-04-17 17:52:46,612 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011578.1, seqID NZ_CP011578.1 [record index 0].\n",
      "2024-04-17 17:52:46,663 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011624.1, seqID NZ_CP011624.1 [record index 0].\n",
      "2024-04-17 17:52:46,713 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018676.1, seqID NZ_CP018676.1 [record index 0].\n",
      "2024-04-17 17:52:46,770 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP013322.1, seqID NZ_CP013322.1 [record index 0].\n",
      "2024-04-17 17:52:46,819 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009114.1, seqID NZ_CP009114.1 [record index 0].\n",
      "2024-04-17 17:52:46,870 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034415.1, seqID NZ_CP034415.1 [record index 0].\n",
      "2024-04-17 17:52:46,921 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042481.1, seqID NZ_CP042481.1 [record index 0].\n",
      "2024-04-17 17:52:46,975 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039808.1, seqID NZ_CP039808.1 [record index 0].\n",
      "2024-04-17 17:52:47,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039819.1, seqID NZ_CP039819.1 [record index 0].\n",
      "2024-04-17 17:52:47,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071464.1, seqID NZ_CP071464.1 [record index 0].\n",
      "2024-04-17 17:52:47,131 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080584.1, seqID NZ_CP080584.1 [record index 0].\n",
      "2024-04-17 17:52:47,184 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020498.1, seqID NZ_CP020498.1 [record index 0].\n",
      "2024-04-17 17:52:47,237 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP066543.1, seqID NZ_CP066543.1 [record index 0].\n",
      "2024-04-17 17:52:47,287 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063890.1, seqID NZ_CP063890.1 [record index 0].\n",
      "2024-04-17 17:52:47,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063885.1, seqID NZ_CP063885.1 [record index 0].\n",
      "2024-04-17 17:52:47,385 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063943.1, seqID NZ_CP063943.1 [record index 0].\n",
      "2024-04-17 17:52:47,432 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063934.1, seqID NZ_CP063934.1 [record index 0].\n",
      "2024-04-17 17:52:47,477 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063936.1, seqID NZ_CP063936.1 [record index 0].\n",
      "2024-04-17 17:52:47,529 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018883.1, seqID NZ_CP018883.1 [record index 0].\n",
      "2024-04-17 17:52:47,580 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018885.1, seqID NZ_CP018885.1 [record index 0].\n",
      "2024-04-17 17:52:47,630 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063939.1, seqID NZ_CP063939.1 [record index 0].\n",
      "2024-04-17 17:52:47,677 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063919.1, seqID NZ_CP063919.1 [record index 0].\n",
      "2024-04-17 17:52:47,726 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063913.1, seqID NZ_CP063913.1 [record index 0].\n",
      "2024-04-17 17:52:47,776 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063904.1, seqID NZ_CP063904.1 [record index 0].\n",
      "2024-04-17 17:52:47,823 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063906.1, seqID NZ_CP063906.1 [record index 0].\n",
      "2024-04-17 17:52:47,881 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015822.1, seqID NZ_CP015822.1 [record index 0].\n",
      "2024-04-17 17:52:47,936 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020837.1, seqID NZ_CP020837.1 [record index 0].\n",
      "2024-04-17 17:52:47,988 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034123.1, seqID NZ_CP034123.1 [record index 0].\n",
      "2024-04-17 17:52:48,042 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022573.1, seqID NZ_CP022573.1 [record index 0].\n",
      "2024-04-17 17:52:48,092 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065453.1, seqID NZ_CP065453.1 [record index 0].\n",
      "2024-04-17 17:52:48,144 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082967.1, seqID NZ_CP082967.1 [record index 0].\n",
      "2024-04-17 17:52:48,193 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082975.1, seqID NZ_CP082975.1 [record index 0].\n",
      "2024-04-17 17:52:48,246 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082978.1, seqID NZ_CP082978.1 [record index 0].\n",
      "2024-04-17 17:52:48,296 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050845.1, seqID NZ_CP050845.1 [record index 0].\n",
      "2024-04-17 17:52:48,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050840.1, seqID NZ_CP050840.1 [record index 0].\n",
      "2024-04-17 17:52:48,412 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050822.1, seqID NZ_CP050822.1 [record index 0].\n",
      "2024-04-17 17:52:48,463 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050829.1, seqID NZ_CP050829.1 [record index 0].\n",
      "2024-04-17 17:52:48,521 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092753.1, seqID NZ_CP092753.1 [record index 0].\n",
      "2024-04-17 17:52:48,577 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092759.1, seqID NZ_CP092759.1 [record index 0].\n",
      "2024-04-17 17:52:48,633 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092765.1, seqID NZ_CP092765.1 [record index 0].\n",
      "2024-04-17 17:52:48,687 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036190.1, seqID NZ_CP036190.1 [record index 0].\n",
      "2024-04-17 17:52:48,737 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036327.1, seqID NZ_CP036327.1 [record index 0].\n",
      "2024-04-17 17:52:48,791 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053364.1, seqID NZ_CP053364.1 [record index 0].\n",
      "2024-04-17 17:52:48,839 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054170.1, seqID NZ_CP054170.1 [record index 0].\n",
      "2024-04-17 17:52:48,891 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070410.1, seqID NZ_CP070410.1 [record index 0].\n",
      "2024-04-17 17:52:48,941 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072408.1, seqID NZ_CP072408.1 [record index 0].\n",
      "2024-04-17 17:52:48,991 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070430.1, seqID NZ_CP070430.1 [record index 0].\n",
      "2024-04-17 17:52:49,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070425.1, seqID NZ_CP070425.1 [record index 0].\n",
      "2024-04-17 17:52:49,098 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072413.1, seqID NZ_CP072413.1 [record index 0].\n",
      "2024-04-17 17:52:49,150 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070459.1, seqID NZ_CP070459.1 [record index 0].\n",
      "2024-04-17 17:52:49,213 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070449.1, seqID NZ_CP070449.1 [record index 0].\n",
      "2024-04-17 17:52:49,264 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070415.1, seqID NZ_CP070415.1 [record index 0].\n",
      "2024-04-17 17:52:49,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072403.1, seqID NZ_CP072403.1 [record index 0].\n",
      "2024-04-17 17:52:49,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070445.1, seqID NZ_CP070445.1 [record index 0].\n",
      "2024-04-17 17:52:49,415 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026155.1, seqID NZ_CP026155.1 [record index 0].\n",
      "2024-04-17 17:52:49,469 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072456.1, seqID NZ_CP072456.1 [record index 0].\n",
      "2024-04-17 17:52:49,520 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025008.1, seqID NZ_CP025008.1 [record index 0].\n",
      "2024-04-17 17:52:49,575 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022691.2, seqID NZ_CP022691.2 [record index 0].\n",
      "2024-04-17 17:52:49,626 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025005.1, seqID NZ_CP025005.1 [record index 0].\n",
      "2024-04-17 17:52:49,684 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006659.2, seqID NZ_CP006659.2 [record index 0].\n",
      "2024-04-17 17:52:49,739 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015134.1, seqID NZ_CP015134.1 [record index 0].\n",
      "2024-04-17 17:52:49,792 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030341.1, seqID NZ_CP030341.1 [record index 0].\n",
      "2024-04-17 17:52:49,845 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027146.1, seqID NZ_CP027146.1 [record index 0].\n",
      "2024-04-17 17:52:49,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027160.1, seqID NZ_CP027160.1 [record index 0].\n",
      "2024-04-17 17:52:49,944 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032175.1, seqID NZ_CP032175.1 [record index 0].\n",
      "2024-04-17 17:52:49,994 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021696.1, seqID NZ_CP021696.1 [record index 0].\n",
      "2024-04-17 17:52:50,045 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021950.1, seqID NZ_CP021950.1 [record index 0].\n",
      "2024-04-17 17:52:50,096 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021685.1, seqID NZ_CP021685.1 [record index 0].\n",
      "2024-04-17 17:52:50,152 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021960.1, seqID NZ_CP021960.1 [record index 0].\n",
      "2024-04-17 17:52:50,203 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032178.1, seqID NZ_CP032178.1 [record index 0].\n",
      "2024-04-17 17:52:50,254 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021718.1, seqID NZ_CP021718.1 [record index 0].\n",
      "2024-04-17 17:52:50,305 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021859.1, seqID NZ_CP021859.1 [record index 0].\n",
      "2024-04-17 17:52:50,358 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021833.1, seqID NZ_CP021833.1 [record index 0].\n",
      "2024-04-17 17:52:50,410 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020071.1, seqID NZ_CP020071.1 [record index 0].\n",
      "2024-04-17 17:52:50,461 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021751.1, seqID NZ_CP021751.1 [record index 0].\n",
      "2024-04-17 17:52:50,513 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021549.1, seqID NZ_CP021549.1 [record index 0].\n",
      "2024-04-17 17:52:50,565 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032207.1, seqID NZ_CP032207.1 [record index 0].\n",
      "2024-04-17 17:52:50,641 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP021955.1, seqID NZ_CP021955.1 [record index 0].\n",
      "2024-04-17 17:52:50,697 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020108.1, seqID NZ_CP020108.1 [record index 0].\n",
      "2024-04-17 17:52:50,746 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032200.1, seqID NZ_CP032200.1 [record index 0].\n",
      "2024-04-17 17:52:50,797 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027612.1, seqID NZ_CP027612.1 [record index 0].\n",
      "2024-04-17 17:52:50,855 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028994.1, seqID NZ_CP028994.1 [record index 0].\n",
      "2024-04-17 17:52:50,908 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032167.1, seqID NZ_CP032167.1 [record index 0].\n",
      "2024-04-17 17:52:50,960 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018816.1, seqID NZ_CP018816.1 [record index 0].\n",
      "2024-04-17 17:52:51,011 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021539.1, seqID NZ_CP021539.1 [record index 0].\n",
      "2024-04-17 17:52:51,060 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033946.1, seqID NZ_CP033946.1 [record index 0].\n",
      "2024-04-17 17:52:51,111 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029099.1, seqID NZ_CP029099.1 [record index 0].\n",
      "2024-04-17 17:52:51,161 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035383.1, seqID NZ_CP035383.1 [record index 0].\n",
      "2024-04-17 17:52:51,209 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130541.1, seqID NZ_LR130541.1 [record index 0].\n",
      "2024-04-17 17:52:51,267 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091048.1, seqID NZ_CP091048.1 [record index 0].\n",
      "2024-04-17 17:52:51,317 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022882.1, seqID NZ_CP022882.1 [record index 0].\n",
      "2024-04-17 17:52:51,368 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071819.1, seqID NZ_CP071819.1 [record index 0].\n",
      "2024-04-17 17:52:51,422 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085889.1, seqID NZ_CP085889.1 [record index 0].\n",
      "2024-04-17 17:52:51,475 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030292.1, seqID NZ_CP030292.1 [record index 0].\n",
      "2024-04-17 17:52:51,527 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022997.1, seqID NZ_CP022997.1 [record index 0].\n",
      "2024-04-17 17:52:51,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082985.1, seqID NZ_CP082985.1 [record index 0].\n",
      "2024-04-17 17:52:51,628 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082995.1, seqID NZ_CP082995.1 [record index 0].\n",
      "2024-04-17 17:52:51,680 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083000.1, seqID NZ_CP083000.1 [record index 0].\n",
      "2024-04-17 17:52:51,733 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083007.1, seqID NZ_CP083007.1 [record index 0].\n",
      "2024-04-17 17:52:51,785 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083011.1, seqID NZ_CP083011.1 [record index 0].\n",
      "2024-04-17 17:52:51,835 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083016.1, seqID NZ_CP083016.1 [record index 0].\n",
      "2024-04-17 17:52:51,887 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083021.1, seqID NZ_CP083021.1 [record index 0].\n",
      "2024-04-17 17:52:51,939 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083026.1, seqID NZ_CP083026.1 [record index 0].\n",
      "2024-04-17 17:52:51,992 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068602.1, seqID NZ_CP068602.1 [record index 0].\n",
      "2024-04-17 17:52:52,047 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085729.1, seqID NZ_CP085729.1 [record index 0].\n",
      "2024-04-17 17:52:52,104 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088126.1, seqID NZ_CP088126.1 [record index 0].\n",
      "2024-04-17 17:52:52,153 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089012.1, seqID NZ_CP089012.1 [record index 0].\n",
      "2024-04-17 17:52:52,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089014.1, seqID NZ_CP089014.1 [record index 0].\n",
      "2024-04-17 17:52:52,262 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088982.1, seqID NZ_CP088982.1 [record index 0].\n",
      "2024-04-17 17:52:52,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085740.1, seqID NZ_CP085740.1 [record index 0].\n",
      "2024-04-17 17:52:52,366 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088991.1, seqID NZ_CP088991.1 [record index 0].\n",
      "2024-04-17 17:52:52,419 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050371.1, seqID NZ_CP050371.1 [record index 0].\n",
      "2024-04-17 17:52:52,468 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011980.1, seqID NZ_CP011980.1 [record index 0].\n",
      "2024-04-17 17:52:52,514 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042320.1, seqID NZ_CP042320.1 [record index 0].\n",
      "2024-04-17 17:52:52,566 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089024.1, seqID NZ_CP089024.1 [record index 0].\n",
      "2024-04-17 17:52:52,618 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088997.1, seqID NZ_CP088997.1 [record index 0].\n",
      "2024-04-17 17:52:52,670 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083036.1, seqID NZ_CP083036.1 [record index 0].\n",
      "2024-04-17 17:52:52,721 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033625.1, seqID NZ_CP033625.1 [record index 0].\n",
      "2024-04-17 17:52:52,769 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042318.1, seqID NZ_CP042318.1 [record index 0].\n",
      "2024-04-17 17:52:52,826 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088120.1, seqID NZ_CP088120.1 [record index 0].\n",
      "2024-04-17 17:52:52,876 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068610.1, seqID NZ_CP068610.1 [record index 0].\n",
      "2024-04-17 17:52:52,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068614.1, seqID NZ_CP068614.1 [record index 0].\n",
      "2024-04-17 17:52:52,978 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068618.1, seqID NZ_CP068618.1 [record index 0].\n",
      "2024-04-17 17:52:53,030 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054268.1, seqID NZ_CP054268.1 [record index 0].\n",
      "2024-04-17 17:52:53,083 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082753.1, seqID NZ_CP082753.1 [record index 0].\n",
      "2024-04-17 17:52:53,135 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082759.1, seqID NZ_CP082759.1 [record index 0].\n",
      "2024-04-17 17:52:53,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP010392.1, seqID NZ_CP010392.1 [record index 0].\n",
      "2024-04-17 17:52:53,239 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069046.1, seqID NZ_CP069046.1 [record index 0].\n",
      "2024-04-17 17:52:53,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071086.1, seqID NZ_CP071086.1 [record index 0].\n",
      "2024-04-17 17:52:53,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071080.1, seqID NZ_CP071080.1 [record index 0].\n",
      "2024-04-17 17:52:53,397 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028850.1, seqID NZ_CP028850.1 [record index 0].\n",
      "2024-04-17 17:52:53,447 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006918.1, seqID NZ_CP006918.1 [record index 0].\n",
      "2024-04-17 17:52:53,497 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006923.1, seqID NZ_CP006923.1 [record index 0].\n",
      "2024-04-17 17:52:53,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027053.1, seqID NZ_CP027053.1 [record index 0].\n",
      "2024-04-17 17:52:53,599 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025541.3, seqID NZ_CP025541.3 [record index 0].\n",
      "2024-04-17 17:52:53,652 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030300.1, seqID NZ_CP030300.1 [record index 0].\n",
      "2024-04-17 17:52:53,702 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030320.1, seqID NZ_CP030320.1 [record index 0].\n",
      "2024-04-17 17:52:53,754 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027048.1, seqID NZ_CP027048.1 [record index 0].\n",
      "2024-04-17 17:52:53,805 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047336.1, seqID NZ_CP047336.1 [record index 0].\n",
      "2024-04-17 17:52:53,859 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068015.1, seqID NZ_CP068015.1 [record index 0].\n",
      "2024-04-17 17:52:53,909 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065034.1, seqID NZ_CP065034.1 [record index 0].\n",
      "2024-04-17 17:52:53,959 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055293.1, seqID NZ_CP055293.1 [record index 0].\n",
      "2024-04-17 17:52:54,014 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051159.1, seqID NZ_CP051159.1 [record index 0].\n",
      "2024-04-17 17:52:54,067 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039948.1, seqID NZ_CP039948.1 [record index 0].\n",
      "2024-04-17 17:52:54,117 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039945.1, seqID NZ_CP039945.1 [record index 0].\n",
      "2024-04-17 17:52:54,170 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049947.1, seqID NZ_CP049947.1 [record index 0].\n",
      "2024-04-17 17:52:54,223 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049953.1, seqID NZ_CP049953.1 [record index 0].\n",
      "2024-04-17 17:52:54,274 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049950.1, seqID NZ_CP049950.1 [record index 0].\n",
      "2024-04-17 17:52:54,324 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064174.1, seqID NZ_CP064174.1 [record index 0].\n",
      "2024-04-17 17:52:54,374 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039942.1, seqID NZ_CP039942.1 [record index 0].\n",
      "2024-04-17 17:52:54,426 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039939.1, seqID NZ_CP039939.1 [record index 0].\n",
      "2024-04-17 17:52:54,476 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039936.1, seqID NZ_CP039936.1 [record index 0].\n",
      "2024-04-17 17:52:54,526 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034778.1, seqID NZ_CP034778.1 [record index 0].\n",
      "2024-04-17 17:52:54,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084492.1, seqID NZ_CP084492.1 [record index 0].\n",
      "2024-04-17 17:52:54,630 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027036.1, seqID NZ_CP027036.1 [record index 0].\n",
      "2024-04-17 17:52:54,684 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045263.1, seqID NZ_CP045263.1 [record index 0].\n",
      "2024-04-17 17:52:54,733 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051161.1, seqID NZ_CP051161.1 [record index 0].\n",
      "2024-04-17 17:52:54,784 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084497.1, seqID NZ_CP084497.1 [record index 0].\n",
      "2024-04-17 17:52:54,835 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083055.1, seqID NZ_CP083055.1 [record index 0].\n",
      "2024-04-17 17:52:54,887 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029689.1, seqID NZ_CP029689.1 [record index 0].\n",
      "2024-04-17 17:52:54,935 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047649.1, seqID NZ_CP047649.1 [record index 0].\n",
      "2024-04-17 17:52:54,984 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047648.1, seqID NZ_CP047648.1 [record index 0].\n",
      "2024-04-17 17:52:55,033 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030172.1, seqID NZ_CP030172.1 [record index 0].\n",
      "2024-04-17 17:52:55,083 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031849.1, seqID NZ_CP031849.1 [record index 0].\n",
      "2024-04-17 17:52:55,136 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082765.1, seqID NZ_CP082765.1 [record index 0].\n",
      "2024-04-17 17:52:55,188 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049716.1, seqID NZ_CP049716.1 [record index 0].\n",
      "2024-04-17 17:52:55,238 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089427.1, seqID NZ_CP089427.1 [record index 0].\n",
      "2024-04-17 17:52:55,287 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026021.1, seqID NZ_CP026021.1 [record index 0].\n",
      "2024-04-17 17:52:55,338 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030311.1, seqID NZ_CP030311.1 [record index 0].\n",
      "2024-04-17 17:52:55,388 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026023.1, seqID NZ_CP026023.1 [record index 0].\n",
      "2024-04-17 17:52:55,436 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030313.1, seqID NZ_CP030313.1 [record index 0].\n",
      "2024-04-17 17:52:55,491 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030174.1, seqID NZ_CP030174.1 [record index 0].\n",
      "2024-04-17 17:52:55,542 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042309.1, seqID NZ_CP042309.1 [record index 0].\n",
      "2024-04-17 17:52:55,595 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_018522.1, seqID NC_018522.1 [record index 0].\n",
      "2024-04-17 17:52:55,652 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023416.1, seqID NZ_CP023416.1 [record index 0].\n",
      "2024-04-17 17:52:55,700 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042310.1, seqID NZ_CP042310.1 [record index 0].\n",
      "2024-04-17 17:52:55,748 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042311.1, seqID NZ_CP042311.1 [record index 0].\n",
      "2024-04-17 17:52:55,796 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042321.1, seqID NZ_CP042321.1 [record index 0].\n",
      "2024-04-17 17:52:55,847 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025576.1, seqID NZ_CP025576.1 [record index 0].\n",
      "2024-04-17 17:52:55,901 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049719.1, seqID NZ_CP049719.1 [record index 0].\n",
      "2024-04-17 17:52:55,953 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033396.2, seqID NZ_CP033396.2 [record index 0].\n",
      "2024-04-17 17:52:56,011 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055309.1, seqID NZ_CP055309.1 [record index 0].\n",
      "2024-04-17 17:52:56,064 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079754.1, seqID NZ_CP079754.1 [record index 0].\n",
      "2024-04-17 17:52:56,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091752.1, seqID NZ_CP091752.1 [record index 0].\n",
      "2024-04-17 17:52:56,172 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059391.1, seqID NZ_CP059391.1 [record index 0].\n",
      "2024-04-17 17:52:56,220 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084874.1, seqID NZ_CP084874.1 [record index 0].\n",
      "2024-04-17 17:52:56,276 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134254.1, seqID NZ_LR134254.1 [record index 0].\n",
      "2024-04-17 17:52:56,366 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP061801.1, seqID NZ_CP061801.1 [record index 0].\n",
      "2024-04-17 17:52:56,457 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022133.1, seqID NZ_AP022133.1 [record index 0].\n",
      "2024-04-17 17:52:56,558 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_021870.1, seqID NC_021870.1 [record index 0].\n",
      "2024-04-17 17:52:56,638 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053336.1, seqID NZ_CP053336.1 [record index 0].\n",
      "2024-04-17 17:52:56,772 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP026167.1, seqID NZ_CP026167.1 [record index 0].\n",
      "2024-04-17 17:52:56,894 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP046446.1, seqID NZ_CP046446.1 [record index 0].\n",
      "2024-04-17 17:52:56,999 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP049786.1, seqID NZ_CP049786.1 [record index 0].\n",
      "2024-04-17 17:52:57,124 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP046557.1, seqID NZ_CP046557.1 [record index 0].\n",
      "2024-04-17 17:52:57,152 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP074437.1, seqID NZ_CP074437.1 [record index 0].\n",
      "2024-04-17 17:52:57,177 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP041925.1, seqID NZ_CP041925.1 [record index 0].\n",
      "2024-04-17 17:52:57,201 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP049600.1, seqID NZ_CP049600.1 [record index 0].\n",
      "2024-04-17 17:52:57,287 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_015968.1, seqID NC_015968.1 [record index 0].\n",
      "2024-04-17 17:52:57,352 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058637.1, seqID NZ_CP058637.1 [record index 0].\n",
      "2024-04-17 17:52:57,445 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022446.1, seqID NZ_AP022446.1 [record index 0].\n",
      "2024-04-17 17:52:57,542 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022431.1, seqID NZ_AP022431.1 [record index 0].\n",
      "2024-04-17 17:52:57,630 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022498.1, seqID NZ_AP022498.1 [record index 0].\n",
      "2024-04-17 17:52:57,746 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP059481.1, seqID NZ_CP059481.1 [record index 0].\n",
      "2024-04-17 17:52:57,865 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043511.1, seqID NZ_CP043511.1 [record index 0].\n",
      "2024-04-17 17:52:58,067 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017181.1, seqID NZ_CP017181.1 [record index 0].\n",
      "2024-04-17 17:52:58,191 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP088119.1, seqID NZ_CP088119.1 [record index 0].\n",
      "2024-04-17 17:52:58,292 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028950.1, seqID NZ_CP028950.1 [record index 0].\n",
      "2024-04-17 17:52:58,466 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017990.1, seqID NZ_CP017990.1 [record index 0].\n",
      "2024-04-17 17:52:58,700 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP082147.1, seqID NZ_CP082147.1 [record index 0].\n",
      "2024-04-17 17:52:58,836 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056711.1, seqID NZ_CP056711.1 [record index 0].\n",
      "2024-04-17 17:52:58,917 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP019632.1, seqID NZ_AP019632.1 [record index 0].\n",
      "2024-04-17 17:52:59,000 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053573.1, seqID NZ_CP053573.1 [record index 0].\n",
      "2024-04-17 17:52:59,077 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056180.1, seqID NZ_CP056180.1 [record index 0].\n",
      "2024-04-17 17:52:59,154 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056717.1, seqID NZ_CP056717.1 [record index 0].\n",
      "2024-04-17 17:52:59,271 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029727.1, seqID NZ_CP029727.1 [record index 0].\n",
      "2024-04-17 17:52:59,375 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012554.1, seqID NZ_CP012554.1 [record index 0].\n",
      "2024-04-17 17:52:59,497 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP022378.1, seqID NZ_AP022378.1 [record index 0].\n",
      "2024-04-17 17:52:59,692 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP066089.1, seqID NZ_CP066089.1 [record index 0].\n",
      "2024-04-17 17:52:59,856 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NC_009792.1, seqID NC_009792.1 [record index 0].\n",
      "2024-04-17 17:52:59,949 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024679.1, seqID NZ_CP024679.1 [record index 0].\n",
      "2024-04-17 17:53:00,031 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024680.1, seqID NZ_CP024680.1 [record index 0].\n",
      "2024-04-17 17:53:00,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092493.1, seqID NZ_CP092493.1 [record index 0].\n",
      "2024-04-17 17:53:00,210 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057146.1, seqID NZ_CP057146.1 [record index 0].\n",
      "2024-04-17 17:53:00,308 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP057149.1, seqID NZ_CP057149.1 [record index 0].\n",
      "2024-04-17 17:53:00,358 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033742.1, seqID NZ_CP033742.1 [record index 0].\n",
      "2024-04-17 17:53:00,405 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033744.1, seqID NZ_CP033744.1 [record index 0].\n",
      "2024-04-17 17:53:00,454 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LS992183.1, seqID NZ_LS992183.1 [record index 0].\n",
      "2024-04-17 17:53:00,562 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020448.2, seqID NZ_CP020448.2 [record index 0].\n",
      "2024-04-17 17:53:00,698 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077405.1, seqID NZ_CP077405.1 [record index 0].\n",
      "2024-04-17 17:53:00,823 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP080539.1, seqID NZ_CP080539.1 [record index 0].\n",
      "2024-04-17 17:53:00,917 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065167.1, seqID NZ_CP065167.1 [record index 0].\n",
      "2024-04-17 17:53:01,008 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890369.1, seqID NZ_LR890369.1 [record index 0].\n",
      "2024-04-17 17:53:01,069 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890415.1, seqID NZ_LR890415.1 [record index 0].\n",
      "2024-04-17 17:53:01,120 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890679.1, seqID NZ_LR890679.1 [record index 0].\n",
      "2024-04-17 17:53:01,173 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890374.1, seqID NZ_LR890374.1 [record index 0].\n",
      "2024-04-17 17:53:01,224 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024489.1, seqID NZ_CP024489.1 [record index 0].\n",
      "2024-04-17 17:53:01,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890600.1, seqID NZ_LR890600.1 [record index 0].\n",
      "2024-04-17 17:53:01,385 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045486.1, seqID NZ_CP045486.1 [record index 0].\n",
      "2024-04-17 17:53:01,448 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012257.1, seqID NZ_CP012257.1 [record index 0].\n",
      "2024-04-17 17:53:01,569 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP038469.1, seqID NZ_CP038469.1 [record index 0].\n",
      "2024-04-17 17:53:01,597 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134201.1, seqID NZ_LR134201.1 [record index 0].\n",
      "2024-04-17 17:53:01,673 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032444.1, seqID NZ_CP032444.1 [record index 0].\n",
      "2024-04-17 17:53:01,749 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP036166.1, seqID NZ_CP036166.1 [record index 0].\n",
      "2024-04-17 17:53:01,922 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP070222.1, seqID NZ_CP070222.1 [record index 0].\n",
      "2024-04-17 17:53:02,081 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029995.1, seqID NZ_CP029995.1 [record index 0].\n",
      "2024-04-17 17:53:02,234 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030235.1, seqID NZ_CP030235.1 [record index 0].\n",
      "2024-04-17 17:53:02,325 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019413.1, seqID NZ_CP019413.1 [record index 0].\n",
      "2024-04-17 17:53:02,417 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079837.1, seqID NZ_CP079837.1 [record index 0].\n",
      "2024-04-17 17:53:02,516 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051315.1, seqID NZ_CP051315.1 [record index 0].\n",
      "2024-04-17 17:53:02,611 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051396.1, seqID NZ_CP051396.1 [record index 0].\n",
      "2024-04-17 17:53:02,731 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051398.1, seqID NZ_CP051398.1 [record index 0].\n",
      "2024-04-17 17:53:02,824 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051425.1, seqID NZ_CP051425.1 [record index 0].\n",
      "2024-04-17 17:53:02,915 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075109.1, seqID NZ_CP075109.1 [record index 0].\n",
      "2024-04-17 17:53:02,984 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044200.1, seqID NZ_CP044200.1 [record index 0].\n",
      "2024-04-17 17:53:03,053 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027677.1, seqID NZ_CP027677.1 [record index 0].\n",
      "2024-04-17 17:53:03,138 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP082168.1, seqID NZ_CP082168.1 [record index 0].\n",
      "2024-04-17 17:53:03,266 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044121.1, seqID NZ_CP044121.1 [record index 0].\n",
      "2024-04-17 17:53:03,376 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP026047.1, seqID NZ_CP026047.1 [record index 0].\n",
      "2024-04-17 17:53:03,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018307.1, seqID NZ_CP018307.1 [record index 0].\n",
      "2024-04-17 17:53:03,508 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028555.2, seqID NZ_CP028555.2 [record index 0].\n",
      "2024-04-17 17:53:03,561 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092632.1, seqID NZ_CP092632.1 [record index 0].\n",
      "2024-04-17 17:53:03,615 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089385.1, seqID NZ_CP089385.1 [record index 0].\n",
      "2024-04-17 17:53:03,676 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060807.1, seqID NZ_CP060807.1 [record index 0].\n",
      "2024-04-17 17:53:03,733 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP078146.1, seqID NZ_CP078146.1 [record index 0].\n",
      "2024-04-17 17:53:03,788 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031061.1, seqID NZ_CP031061.1 [record index 0].\n",
      "2024-04-17 17:53:03,841 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084765.1, seqID NZ_CP084765.1 [record index 0].\n",
      "2024-04-17 17:53:03,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072130.1, seqID NZ_CP072130.1 [record index 0].\n",
      "2024-04-17 17:53:03,952 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009274.2, seqID NZ_CP009274.2 [record index 0].\n",
      "2024-04-17 17:53:04,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP010523.2, seqID NZ_CP010523.2 [record index 0].\n",
      "2024-04-17 17:53:04,067 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063917.1, seqID NZ_CP063917.1 [record index 0].\n",
      "2024-04-17 17:53:04,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR130543.1, seqID NZ_LR130543.1 [record index 0].\n",
      "2024-04-17 17:53:04,146 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091527.1, seqID NZ_CP091527.1 [record index 0].\n",
      "2024-04-17 17:53:04,198 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073373.1, seqID NZ_CP073373.1 [record index 0].\n",
      "2024-04-17 17:53:04,224 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP022444.1, seqID NZ_CP022444.1 [record index 0].\n",
      "2024-04-17 17:53:04,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022183.1, seqID NZ_AP022183.1 [record index 0].\n",
      "2024-04-17 17:53:04,278 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022112.1, seqID NZ_AP022112.1 [record index 0].\n",
      "2024-04-17 17:53:04,307 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP021950.1, seqID NZ_AP021950.1 [record index 0].\n",
      "2024-04-17 17:53:04,333 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022014.1, seqID NZ_AP022014.1 [record index 0].\n",
      "2024-04-17 17:53:04,362 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022005.1, seqID NZ_AP022005.1 [record index 0].\n",
      "2024-04-17 17:53:04,415 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP055010.1, seqID NZ_CP055010.1 [record index 0].\n",
      "2024-04-17 17:53:04,464 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP058859.1, seqID NZ_CP058859.1 [record index 0].\n",
      "2024-04-17 17:53:04,509 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084770.1, seqID NZ_CP084770.1 [record index 0].\n",
      "2024-04-17 17:53:04,553 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063874.1, seqID NZ_CP063874.1 [record index 0].\n",
      "2024-04-17 17:53:04,595 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064046.1, seqID NZ_CP064046.1 [record index 0].\n",
      "2024-04-17 17:53:04,637 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP087580.1, seqID NZ_CP087580.1 [record index 0].\n",
      "2024-04-17 17:53:04,681 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134196.1, seqID NZ_LR134196.1 [record index 0].\n",
      "2024-04-17 17:53:04,723 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP047283.1, seqID NZ_CP047283.1 [record index 0].\n",
      "2024-04-17 17:53:04,758 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP031257.1, seqID NZ_CP031257.1 [record index 0].\n",
      "2024-04-17 17:53:04,784 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065841.1, seqID NZ_CP065841.1 [record index 0].\n",
      "2024-04-17 17:53:04,810 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065846.1, seqID NZ_CP065846.1 [record index 0].\n",
      "2024-04-17 17:53:04,837 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065838.1, seqID NZ_CP065838.1 [record index 0].\n",
      "2024-04-17 17:53:04,863 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP073660.1, seqID NZ_CP073660.1 [record index 0].\n",
      "2024-04-17 17:53:04,888 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023478.1, seqID NZ_CP023478.1 [record index 0].\n",
      "2024-04-17 17:53:04,915 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066173.1, seqID NZ_CP066173.1 [record index 0].\n",
      "2024-04-17 17:53:04,943 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP092121.1, seqID NZ_CP092121.1 [record index 0].\n",
      "2024-04-17 17:53:04,966 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012300.1, seqID NZ_CP012300.1 [record index 0].\n",
      "2024-04-17 17:53:04,990 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012252.1, seqID NZ_CP012252.1 [record index 0].\n",
      "2024-04-17 17:53:05,017 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014156.1, seqID NZ_CP014156.1 [record index 0].\n",
      "2024-04-17 17:53:05,043 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014155.1, seqID NZ_CP014155.1 [record index 0].\n",
      "2024-04-17 17:53:05,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014154.1, seqID NZ_CP014154.1 [record index 0].\n",
      "2024-04-17 17:53:05,099 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP034136.1, seqID NZ_CP034136.1 [record index 0].\n",
      "2024-04-17 17:53:05,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP034129.1, seqID NZ_CP034129.1 [record index 0].\n",
      "2024-04-17 17:53:05,153 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014071.1, seqID NZ_CP014071.1 [record index 0].\n",
      "2024-04-17 17:53:05,178 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083631.1, seqID NZ_CP083631.1 [record index 0].\n",
      "2024-04-17 17:53:05,204 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP077279.1, seqID NZ_CP077279.1 [record index 0].\n",
      "2024-04-17 17:53:05,229 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP034678.1, seqID NZ_CP034678.1 [record index 0].\n",
      "2024-04-17 17:53:05,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084777.1, seqID NZ_CP084777.1 [record index 0].\n",
      "2024-04-17 17:53:05,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063902.1, seqID NZ_CP063902.1 [record index 0].\n",
      "2024-04-17 17:53:05,305 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063896.1, seqID NZ_CP063896.1 [record index 0].\n",
      "2024-04-17 17:53:05,330 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063900.1, seqID NZ_CP063900.1 [record index 0].\n",
      "2024-04-17 17:53:05,360 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063887.1, seqID NZ_CP063887.1 [record index 0].\n",
      "2024-04-17 17:53:05,384 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064044.1, seqID NZ_CP064044.1 [record index 0].\n",
      "2024-04-17 17:53:05,410 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP029597.1, seqID NZ_CP029597.1 [record index 0].\n",
      "2024-04-17 17:53:05,439 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014696.2, seqID NZ_CP014696.2 [record index 0].\n",
      "2024-04-17 17:53:05,461 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP042319.1, seqID NZ_CP042319.1 [record index 0].\n",
      "2024-04-17 17:53:05,488 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072515.1, seqID NZ_CP072515.1 [record index 0].\n",
      "2024-04-17 17:53:05,512 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084779.1, seqID NZ_CP084779.1 [record index 0].\n",
      "2024-04-17 17:53:05,537 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084783.1, seqID NZ_CP084783.1 [record index 0].\n",
      "2024-04-17 17:53:05,588 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031613.1, seqID NZ_CP031613.1 [record index 0].\n",
      "2024-04-17 17:53:05,643 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065341.1, seqID NZ_CP065341.1 [record index 0].\n",
      "2024-04-17 17:53:05,729 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065351.1, seqID NZ_CP065351.1 [record index 0].\n",
      "2024-04-17 17:53:05,820 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP014950.1, seqID NZ_AP014950.1 [record index 0].\n",
      "2024-04-17 17:53:05,879 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP052761.1, seqID NZ_CP052761.1 [record index 0].\n",
      "2024-04-17 17:53:05,930 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009461.1, seqID NZ_CP009461.1 [record index 0].\n",
      "2024-04-17 17:53:05,979 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045674.1, seqID NZ_CP045674.1 [record index 0].\n",
      "2024-04-17 17:53:06,028 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP022139.1, seqID NZ_AP022139.1 [record index 0].\n",
      "2024-04-17 17:53:06,080 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076008.1, seqID NZ_CP076008.1 [record index 0].\n",
      "2024-04-17 17:53:06,128 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031885.1, seqID NZ_CP031885.1 [record index 0].\n",
      "2024-04-17 17:53:06,179 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025963.2, seqID NZ_CP025963.2 [record index 0].\n",
      "2024-04-17 17:53:06,232 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046612.1, seqID NZ_CP046612.1 [record index 0].\n",
      "2024-04-17 17:53:06,280 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015753.1, seqID NZ_CP015753.1 [record index 0].\n",
      "2024-04-17 17:53:06,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035210.1, seqID NZ_CP035210.1 [record index 0].\n",
      "2024-04-17 17:53:06,386 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP087122.1, seqID NZ_CP087122.1 [record index 0].\n",
      "2024-04-17 17:53:06,438 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016159.1, seqID NZ_CP016159.1 [record index 0].\n",
      "2024-04-17 17:53:06,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012743.1, seqID NZ_CP012743.1 [record index 0].\n",
      "2024-04-17 17:53:06,547 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012745.1, seqID NZ_CP012745.1 [record index 0].\n",
      "2024-04-17 17:53:06,602 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012744.1, seqID NZ_CP012744.1 [record index 0].\n",
      "2024-04-17 17:53:06,628 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP075256.1, seqID NZ_CP075256.1 [record index 0].\n",
      "2024-04-17 17:53:06,683 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055106.1, seqID NZ_CP055106.1 [record index 0].\n",
      "2024-04-17 17:53:06,736 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055119.1, seqID NZ_CP055119.1 [record index 0].\n",
      "2024-04-17 17:53:06,795 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055033.1, seqID NZ_CP055033.1 [record index 0].\n",
      "2024-04-17 17:53:06,850 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055057.1, seqID NZ_CP055057.1 [record index 0].\n",
      "2024-04-17 17:53:06,905 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055066.1, seqID NZ_CP055066.1 [record index 0].\n",
      "2024-04-17 17:53:06,962 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058869.1, seqID NZ_CP058869.1 [record index 0].\n",
      "2024-04-17 17:53:07,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058866.1, seqID NZ_CP058866.1 [record index 0].\n",
      "2024-04-17 17:53:07,075 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054967.1, seqID NZ_CP054967.1 [record index 0].\n",
      "2024-04-17 17:53:07,130 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054972.1, seqID NZ_CP054972.1 [record index 0].\n",
      "2024-04-17 17:53:07,189 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054984.1, seqID NZ_CP054984.1 [record index 0].\n",
      "2024-04-17 17:53:07,245 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054997.1, seqID NZ_CP054997.1 [record index 0].\n",
      "2024-04-17 17:53:07,299 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055162.1, seqID NZ_CP055162.1 [record index 0].\n",
      "2024-04-17 17:53:07,352 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055210.1, seqID NZ_CP055210.1 [record index 0].\n",
      "2024-04-17 17:53:07,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055174.1, seqID NZ_CP055174.1 [record index 0].\n",
      "2024-04-17 17:53:07,464 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055186.1, seqID NZ_CP055186.1 [record index 0].\n",
      "2024-04-17 17:53:07,514 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022916.1, seqID NZ_CP022916.1 [record index 0].\n",
      "2024-04-17 17:53:07,566 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022919.1, seqID NZ_CP022919.1 [record index 0].\n",
      "2024-04-17 17:53:07,616 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022922.1, seqID NZ_CP022922.1 [record index 0].\n",
      "2024-04-17 17:53:07,669 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022924.1, seqID NZ_CP022924.1 [record index 0].\n",
      "2024-04-17 17:53:07,766 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP023148.1, seqID NZ_AP023148.1 [record index 0].\n",
      "2024-04-17 17:53:07,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015500.1, seqID NZ_CP015500.1 [record index 0].\n",
      "2024-04-17 17:53:07,948 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028716.1, seqID NZ_CP028716.1 [record index 0].\n",
      "2024-04-17 17:53:08,038 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP037963.1, seqID NZ_CP037963.1 [record index 0].\n",
      "2024-04-17 17:53:08,121 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028787.2, seqID NZ_CP028787.2 [record index 0].\n",
      "2024-04-17 17:53:08,176 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082796.1, seqID NZ_CP082796.1 [record index 0].\n",
      "2024-04-17 17:53:08,223 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR792628.1, seqID NZ_LR792628.1 [record index 0].\n",
      "2024-04-17 17:53:08,274 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084856.1, seqID NZ_CP084856.1 [record index 0].\n",
      "2024-04-17 17:53:08,322 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084860.1, seqID NZ_CP084860.1 [record index 0].\n",
      "2024-04-17 17:53:08,372 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063927.1, seqID NZ_CP063927.1 [record index 0].\n",
      "2024-04-17 17:53:08,426 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058544.1, seqID NZ_CP058544.1 [record index 0].\n",
      "2024-04-17 17:53:08,477 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063871.1, seqID NZ_CP063871.1 [record index 0].\n",
      "2024-04-17 17:53:08,527 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041353.1, seqID NZ_CP041353.1 [record index 0].\n",
      "2024-04-17 17:53:08,577 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068797.1, seqID NZ_CP068797.1 [record index 0].\n",
      "2024-04-17 17:53:08,632 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056222.1, seqID NZ_CP056222.1 [record index 0].\n",
      "2024-04-17 17:53:08,681 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056275.1, seqID NZ_CP056275.1 [record index 0].\n",
      "2024-04-17 17:53:08,731 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055451.1, seqID NZ_CP055451.1 [record index 0].\n",
      "2024-04-17 17:53:08,761 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP056530.1, seqID NZ_CP056530.1 [record index 0].\n",
      "2024-04-17 17:53:08,811 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056830.1, seqID NZ_CP056830.1 [record index 0].\n",
      "2024-04-17 17:53:08,861 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057034.1, seqID NZ_CP057034.1 [record index 0].\n",
      "2024-04-17 17:53:08,926 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057099.1, seqID NZ_CP057099.1 [record index 0].\n",
      "2024-04-17 17:53:09,010 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057459.1, seqID NZ_CP057459.1 [record index 0].\n",
      "2024-04-17 17:53:09,095 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057722.1, seqID NZ_CP057722.1 [record index 0].\n",
      "2024-04-17 17:53:09,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075552.1, seqID NZ_CP075552.1 [record index 0].\n",
      "2024-04-17 17:53:09,236 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040363.1, seqID NZ_CP040363.1 [record index 0].\n",
      "2024-04-17 17:53:09,288 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083443.1, seqID NZ_CP083443.1 [record index 0].\n",
      "2024-04-17 17:53:09,339 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024458.1, seqID NZ_CP024458.1 [record index 0].\n",
      "2024-04-17 17:53:09,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042858.1, seqID NZ_CP042858.1 [record index 0].\n",
      "2024-04-17 17:53:09,443 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008929.1, seqID NZ_CP008929.1 [record index 0].\n",
      "2024-04-17 17:53:09,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053035.1, seqID NZ_CP053035.1 [record index 0].\n",
      "2024-04-17 17:53:09,540 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043562.1, seqID NZ_CP043562.1 [record index 0].\n",
      "2024-04-17 17:53:09,587 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081892.1, seqID NZ_CP081892.1 [record index 0].\n",
      "2024-04-17 17:53:09,640 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061404.1, seqID NZ_CP061404.1 [record index 0].\n",
      "2024-04-17 17:53:09,696 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025143.1, seqID NZ_CP025143.1 [record index 0].\n",
      "2024-04-17 17:53:09,751 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091594.1, seqID NZ_CP091594.1 [record index 0].\n",
      "2024-04-17 17:53:09,801 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042882.1, seqID NZ_CP042882.1 [record index 0].\n",
      "2024-04-17 17:53:09,855 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024916.1, seqID NZ_CP024916.1 [record index 0].\n",
      "2024-04-17 17:53:09,909 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024874.1, seqID NZ_CP024874.1 [record index 0].\n",
      "2024-04-17 17:53:09,957 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134280.1, seqID NZ_LR134280.1 [record index 0].\n",
      "2024-04-17 17:53:10,009 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR588412.1, seqID NZ_LR588412.1 [record index 0].\n",
      "2024-04-17 17:53:10,059 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR588408.1, seqID NZ_LR588408.1 [record index 0].\n",
      "2024-04-17 17:53:10,108 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR133964.1, seqID NZ_LR133964.1 [record index 0].\n",
      "2024-04-17 17:53:10,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063856.1, seqID NZ_CP063856.1 [record index 0].\n",
      "2024-04-17 17:53:10,253 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085863.1, seqID NZ_CP085863.1 [record index 0].\n",
      "2024-04-17 17:53:10,314 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LN824133.1, seqID NZ_LN824133.1 [record index 0].\n",
      "2024-04-17 17:53:10,370 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054303.1, seqID NZ_CP054303.1 [record index 0].\n",
      "2024-04-17 17:53:10,425 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP074087.1, seqID NZ_CP074087.1 [record index 0].\n",
      "2024-04-17 17:53:10,512 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084397.1, seqID NZ_CP084397.1 [record index 0].\n",
      "2024-04-17 17:53:10,589 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081897.1, seqID NZ_CP081897.1 [record index 0].\n",
      "2024-04-17 17:53:10,676 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP023337.1, seqID NZ_AP023337.1 [record index 0].\n",
      "2024-04-17 17:53:10,730 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072949.1, seqID NZ_CP072949.1 [record index 0].\n",
      "2024-04-17 17:53:10,777 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063863.1, seqID NZ_CP063863.1 [record index 0].\n",
      "2024-04-17 17:53:10,827 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051490.1, seqID NZ_CP051490.1 [record index 0].\n",
      "2024-04-17 17:53:10,878 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063992.1, seqID NZ_CP063992.1 [record index 0].\n",
      "2024-04-17 17:53:10,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063858.1, seqID NZ_CP063858.1 [record index 0].\n",
      "2024-04-17 17:53:10,973 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063851.1, seqID NZ_CP063851.1 [record index 0].\n",
      "2024-04-17 17:53:11,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089989.1, seqID NZ_CP089989.1 [record index 0].\n",
      "2024-04-17 17:53:11,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089986.1, seqID NZ_CP089986.1 [record index 0].\n",
      "2024-04-17 17:53:11,137 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068033.1, seqID NZ_CP068033.1 [record index 0].\n",
      "2024-04-17 17:53:11,183 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035194.1, seqID NZ_CP035194.1 [record index 0].\n",
      "2024-04-17 17:53:11,209 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083768.1, seqID NZ_CP083768.1 [record index 0].\n",
      "2024-04-17 17:53:11,261 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024173.1, seqID NZ_AP024173.1 [record index 0].\n",
      "2024-04-17 17:53:11,307 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890751.1, seqID NZ_LR890751.1 [record index 0].\n",
      "2024-04-17 17:53:11,357 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024548.1, seqID NZ_CP024548.1 [record index 0].\n",
      "2024-04-17 17:53:11,405 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031814.1, seqID NZ_CP031814.1 [record index 0].\n",
      "2024-04-17 17:53:11,451 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890353.1, seqID NZ_LR890353.1 [record index 0].\n",
      "2024-04-17 17:53:11,500 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031789.1, seqID NZ_CP031789.1 [record index 0].\n",
      "2024-04-17 17:53:11,546 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890291.1, seqID NZ_LR890291.1 [record index 0].\n",
      "2024-04-17 17:53:11,597 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073788.1, seqID NZ_CP073788.1 [record index 0].\n",
      "2024-04-17 17:53:11,645 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034045.1, seqID NZ_CP034045.1 [record index 0].\n",
      "2024-04-17 17:53:11,692 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073627.1, seqID NZ_CP073627.1 [record index 0].\n",
      "2024-04-17 17:53:11,778 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018719.1, seqID NZ_CP018719.1 [record index 0].\n",
      "2024-04-17 17:53:11,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018707.1, seqID NZ_CP018707.1 [record index 0].\n",
      "2024-04-17 17:53:11,915 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018701.1, seqID NZ_CP018701.1 [record index 0].\n",
      "2024-04-17 17:53:11,971 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018140.1, seqID NZ_CP018140.1 [record index 0].\n",
      "2024-04-17 17:53:12,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018713.1, seqID NZ_CP018713.1 [record index 0].\n",
      "2024-04-17 17:53:12,091 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018695.1, seqID NZ_CP018695.1 [record index 0].\n",
      "2024-04-17 17:53:12,155 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018686.1, seqID NZ_CP018686.1 [record index 0].\n",
      "2024-04-17 17:53:12,208 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086724.1, seqID NZ_CP086724.1 [record index 0].\n",
      "2024-04-17 17:53:12,260 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024579.1, seqID NZ_AP024579.1 [record index 0].\n",
      "2024-04-17 17:53:12,312 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024576.1, seqID NZ_AP024576.1 [record index 0].\n",
      "2024-04-17 17:53:12,366 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024572.1, seqID NZ_AP024572.1 [record index 0].\n",
      "2024-04-17 17:53:12,422 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024570.1, seqID NZ_AP024570.1 [record index 0].\n",
      "2024-04-17 17:53:12,481 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP024568.1, seqID NZ_AP024568.1 [record index 0].\n",
      "2024-04-17 17:53:12,533 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040593.1, seqID NZ_CP040593.1 [record index 0].\n",
      "2024-04-17 17:53:12,616 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040724.1, seqID NZ_CP040724.1 [record index 0].\n",
      "2024-04-17 17:53:12,677 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026177.1, seqID NZ_CP026177.1 [record index 0].\n",
      "2024-04-17 17:53:12,730 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026178.1, seqID NZ_CP026178.1 [record index 0].\n",
      "2024-04-17 17:53:12,785 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026392.1, seqID NZ_CP026392.1 [record index 0].\n",
      "2024-04-17 17:53:12,832 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009876.1, seqID NZ_CP009876.1 [record index 0].\n",
      "2024-04-17 17:53:12,880 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089383.1, seqID NZ_CP089383.1 [record index 0].\n",
      "2024-04-17 17:53:12,931 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015025.1, seqID NZ_CP015025.1 [record index 0].\n",
      "2024-04-17 17:53:12,985 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092336.1, seqID NZ_CP092336.1 [record index 0].\n",
      "2024-04-17 17:53:13,047 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012992.1, seqID NZ_CP012992.1 [record index 0].\n",
      "2024-04-17 17:53:13,098 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012987.1, seqID NZ_CP012987.1 [record index 0].\n",
      "2024-04-17 17:53:13,151 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047700.1, seqID NZ_CP047700.1 [record index 0].\n",
      "2024-04-17 17:53:13,201 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047707.1, seqID NZ_CP047707.1 [record index 0].\n",
      "2024-04-17 17:53:13,255 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041099.1, seqID NZ_CP041099.1 [record index 0].\n",
      "2024-04-17 17:53:13,328 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041092.1, seqID NZ_CP041092.1 [record index 0].\n",
      "2024-04-17 17:53:13,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065833.1, seqID NZ_CP065833.1 [record index 0].\n",
      "2024-04-17 17:53:13,483 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065831.1, seqID NZ_CP065831.1 [record index 0].\n",
      "2024-04-17 17:53:13,535 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092648.1, seqID NZ_CP092648.1 [record index 0].\n",
      "2024-04-17 17:53:13,587 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090468.1, seqID NZ_CP090468.1 [record index 0].\n",
      "2024-04-17 17:53:13,635 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082290.1, seqID NZ_CP082290.1 [record index 0].\n",
      "2024-04-17 17:53:13,686 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070586.1, seqID NZ_CP070586.1 [record index 0].\n",
      "2024-04-17 17:53:13,736 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_FO834906.1, seqID NZ_FO834906.1 [record index 0].\n",
      "2024-04-17 17:53:13,791 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012426.1, seqID NZ_CP012426.1 [record index 0].\n",
      "2024-04-17 17:53:13,843 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059312.1, seqID NZ_CP059312.1 [record index 0].\n",
      "2024-04-17 17:53:13,911 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059317.1, seqID NZ_CP059317.1 [record index 0].\n",
      "2024-04-17 17:53:13,998 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090126.1, seqID NZ_CP090126.1 [record index 0].\n",
      "2024-04-17 17:53:14,062 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017777.1, seqID NZ_CP017777.1 [record index 0].\n",
      "2024-04-17 17:53:14,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079725.1, seqID NZ_CP079725.1 [record index 0].\n",
      "2024-04-17 17:53:14,164 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090622.1, seqID NZ_CP090622.1 [record index 0].\n",
      "2024-04-17 17:53:14,215 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059889.1, seqID NZ_CP059889.1 [record index 0].\n",
      "2024-04-17 17:53:14,267 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082005.1, seqID NZ_CP082005.1 [record index 0].\n",
      "2024-04-17 17:53:14,318 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082029.1, seqID NZ_CP082029.1 [record index 0].\n",
      "2024-04-17 17:53:14,372 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025140.1, seqID NZ_CP025140.1 [record index 0].\n",
      "2024-04-17 17:53:14,425 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025146.1, seqID NZ_CP025146.1 [record index 0].\n",
      "2024-04-17 17:53:14,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034077.1, seqID NZ_CP034077.1 [record index 0].\n",
      "2024-04-17 17:53:14,525 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034076.1, seqID NZ_CP034076.1 [record index 0].\n",
      "2024-04-17 17:53:14,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084984.1, seqID NZ_CP084984.1 [record index 0].\n",
      "2024-04-17 17:53:14,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071027.1, seqID NZ_CP071027.1 [record index 0].\n",
      "2024-04-17 17:53:14,698 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070890.1, seqID NZ_CP070890.1 [record index 0].\n",
      "2024-04-17 17:53:14,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OU453210.1, seqID NZ_OU453210.1 [record index 0].\n",
      "2024-04-17 17:53:14,774 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP023264.1, seqID NZ_AP023264.1 [record index 0].\n",
      "2024-04-17 17:53:14,827 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047633.1, seqID NZ_CP047633.1 [record index 0].\n",
      "2024-04-17 17:53:14,879 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080346.1, seqID NZ_CP080346.1 [record index 0].\n",
      "2024-04-17 17:53:14,933 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP021880.1, seqID NZ_AP021880.1 [record index 0].\n",
      "2024-04-17 17:53:14,987 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061970.1, seqID NZ_CP061970.1 [record index 0].\n",
      "2024-04-17 17:53:15,040 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061975.1, seqID NZ_CP061975.1 [record index 0].\n",
      "2024-04-17 17:53:15,091 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061948.1, seqID NZ_CP061948.1 [record index 0].\n",
      "2024-04-17 17:53:15,141 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890474.1, seqID NZ_LR890474.1 [record index 0].\n",
      "2024-04-17 17:53:15,193 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890422.1, seqID NZ_LR890422.1 [record index 0].\n",
      "2024-04-17 17:53:15,246 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890542.1, seqID NZ_LR890542.1 [record index 0].\n",
      "2024-04-17 17:53:15,295 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890356.1, seqID NZ_LR890356.1 [record index 0].\n",
      "2024-04-17 17:53:15,360 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890686.1, seqID NZ_LR890686.1 [record index 0].\n",
      "2024-04-17 17:53:15,440 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890307.1, seqID NZ_LR890307.1 [record index 0].\n",
      "2024-04-17 17:53:15,516 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890245.1, seqID NZ_LR890245.1 [record index 0].\n",
      "2024-04-17 17:53:15,562 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890471.1, seqID NZ_LR890471.1 [record index 0].\n",
      "2024-04-17 17:53:15,617 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890719.1, seqID NZ_LR890719.1 [record index 0].\n",
      "2024-04-17 17:53:15,666 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890286.1, seqID NZ_LR890286.1 [record index 0].\n",
      "2024-04-17 17:53:15,715 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890583.1, seqID NZ_LR890583.1 [record index 0].\n",
      "2024-04-17 17:53:15,768 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031795.1, seqID NZ_CP031795.1 [record index 0].\n",
      "2024-04-17 17:53:15,815 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890691.1, seqID NZ_LR890691.1 [record index 0].\n",
      "2024-04-17 17:53:15,869 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890665.1, seqID NZ_LR890665.1 [record index 0].\n",
      "2024-04-17 17:53:15,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034281.1, seqID NZ_CP034281.1 [record index 0].\n",
      "2024-04-17 17:53:15,979 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063215.1, seqID NZ_CP063215.1 [record index 0].\n",
      "2024-04-17 17:53:16,027 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061053.1, seqID NZ_CP061053.1 [record index 0].\n",
      "2024-04-17 17:53:16,081 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061062.1, seqID NZ_CP061062.1 [record index 0].\n",
      "2024-04-17 17:53:16,134 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073906.1, seqID NZ_CP073906.1 [record index 0].\n",
      "2024-04-17 17:53:16,187 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018056.1, seqID NZ_CP018056.1 [record index 0].\n",
      "2024-04-17 17:53:16,241 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP019160.1, seqID NZ_CP019160.1 [record index 0].\n",
      "2024-04-17 17:53:16,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068972.1, seqID NZ_CP068972.1 [record index 0].\n",
      "2024-04-17 17:53:16,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040993.1, seqID NZ_CP040993.1 [record index 0].\n",
      "2024-04-17 17:53:16,394 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044033.1, seqID NZ_CP044033.1 [record index 0].\n",
      "2024-04-17 17:53:16,444 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044039.1, seqID NZ_CP044039.1 [record index 0].\n",
      "2024-04-17 17:53:16,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033756.1, seqID NZ_CP033756.1 [record index 0].\n",
      "2024-04-17 17:53:16,544 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023913.1, seqID NZ_CP023913.1 [record index 0].\n",
      "2024-04-17 17:53:16,595 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014123.1, seqID NZ_CP014123.1 [record index 0].\n",
      "2024-04-17 17:53:16,644 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070177.1, seqID NZ_CP070177.1 [record index 0].\n",
      "2024-04-17 17:53:16,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069992.1, seqID NZ_CP069992.1 [record index 0].\n",
      "2024-04-17 17:53:16,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069821.1, seqID NZ_CP069821.1 [record index 0].\n",
      "2024-04-17 17:53:16,793 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069946.1, seqID NZ_CP069946.1 [record index 0].\n",
      "2024-04-17 17:53:16,842 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069873.1, seqID NZ_CP069873.1 [record index 0].\n",
      "2024-04-17 17:53:16,889 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069994.1, seqID NZ_CP069994.1 [record index 0].\n",
      "2024-04-17 17:53:16,956 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070169.1, seqID NZ_CP070169.1 [record index 0].\n",
      "2024-04-17 17:53:17,038 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070092.1, seqID NZ_CP070092.1 [record index 0].\n",
      "2024-04-17 17:53:17,129 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070105.1, seqID NZ_CP070105.1 [record index 0].\n",
      "2024-04-17 17:53:17,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070158.1, seqID NZ_CP070158.1 [record index 0].\n",
      "2024-04-17 17:53:17,252 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070015.1, seqID NZ_CP070015.1 [record index 0].\n",
      "2024-04-17 17:53:17,303 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070088.1, seqID NZ_CP070088.1 [record index 0].\n",
      "2024-04-17 17:53:17,348 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026157.1, seqID NZ_CP026157.1 [record index 0].\n",
      "2024-04-17 17:53:17,395 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026160.1, seqID NZ_CP026160.1 [record index 0].\n",
      "2024-04-17 17:53:17,445 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090397.1, seqID NZ_CP090397.1 [record index 0].\n",
      "2024-04-17 17:53:17,493 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026162.1, seqID NZ_CP026162.1 [record index 0].\n",
      "2024-04-17 17:53:17,547 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092901.1, seqID NZ_CP092901.1 [record index 0].\n",
      "2024-04-17 17:53:17,596 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP067255.1, seqID NZ_CP067255.1 [record index 0].\n",
      "2024-04-17 17:53:17,644 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OV049808.1, seqID NZ_OV049808.1 [record index 0].\n",
      "2024-04-17 17:53:17,697 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022127.1, seqID NZ_CP022127.1 [record index 0].\n",
      "2024-04-17 17:53:17,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075316.1, seqID NZ_CP075316.1 [record index 0].\n",
      "2024-04-17 17:53:17,795 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024429.1, seqID NZ_CP024429.1 [record index 0].\n",
      "2024-04-17 17:53:17,846 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029582.1, seqID NZ_CP029582.1 [record index 0].\n",
      "2024-04-17 17:53:17,896 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024834.1, seqID NZ_CP024834.1 [record index 0].\n",
      "2024-04-17 17:53:17,948 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024838.1, seqID NZ_CP024838.1 [record index 0].\n",
      "2024-04-17 17:53:17,996 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063023.1, seqID NZ_CP063023.1 [record index 0].\n",
      "2024-04-17 17:53:18,046 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072905.1, seqID NZ_CP072905.1 [record index 0].\n",
      "2024-04-17 17:53:18,103 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015382.1, seqID NZ_CP015382.1 [record index 0].\n",
      "2024-04-17 17:53:18,152 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028176.1, seqID NZ_CP028176.1 [record index 0].\n",
      "2024-04-17 17:53:18,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023249.1, seqID NZ_CP023249.1 [record index 0].\n",
      "2024-04-17 17:53:18,254 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018671.1, seqID NZ_CP018671.1 [record index 0].\n",
      "2024-04-17 17:53:18,315 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017934.1, seqID NZ_CP017934.1 [record index 0].\n",
      "2024-04-17 17:53:18,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042520.1, seqID NZ_CP042520.1 [record index 0].\n",
      "2024-04-17 17:53:18,418 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015990.1, seqID NZ_CP015990.1 [record index 0].\n",
      "2024-04-17 17:53:18,482 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036335.1, seqID NZ_CP036335.1 [record index 0].\n",
      "2024-04-17 17:53:18,537 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063922.1, seqID NZ_CP063922.1 [record index 0].\n",
      "2024-04-17 17:53:18,583 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084870.1, seqID NZ_CP084870.1 [record index 0].\n",
      "2024-04-17 17:53:18,634 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046949.1, seqID NZ_CP046949.1 [record index 0].\n",
      "2024-04-17 17:53:18,687 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046381.1, seqID NZ_CP046381.1 [record index 0].\n",
      "2024-04-17 17:53:18,740 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046939.1, seqID NZ_CP046939.1 [record index 0].\n",
      "2024-04-17 17:53:18,791 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050843.1, seqID NZ_CP050843.1 [record index 0].\n",
      "2024-04-17 17:53:18,842 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060421.1, seqID NZ_CP060421.1 [record index 0].\n",
      "2024-04-17 17:53:18,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036187.1, seqID NZ_CP036187.1 [record index 0].\n",
      "2024-04-17 17:53:18,943 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072422.1, seqID NZ_CP072422.1 [record index 0].\n",
      "2024-04-17 17:53:19,025 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035929.1, seqID NZ_CP035929.1 [record index 0].\n",
      "2024-04-17 17:53:19,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP074580.1, seqID NZ_CP074580.1 [record index 0].\n",
      "2024-04-17 17:53:19,201 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070464.1, seqID NZ_CP070464.1 [record index 0].\n",
      "2024-04-17 17:53:19,287 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084168.1, seqID NZ_CP084168.1 [record index 0].\n",
      "2024-04-17 17:53:19,385 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_009648.1, seqID NC_009648.1 [record index 0].\n",
      "2024-04-17 17:53:19,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009208.1, seqID NZ_CP009208.1 [record index 0].\n",
      "2024-04-17 17:53:19,503 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064352.1, seqID NZ_CP064352.1 [record index 0].\n",
      "2024-04-17 17:53:19,567 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021944.1, seqID NZ_CP021944.1 [record index 0].\n",
      "2024-04-17 17:53:19,624 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021939.1, seqID NZ_CP021939.1 [record index 0].\n",
      "2024-04-17 17:53:19,679 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021757.1, seqID NZ_CP021757.1 [record index 0].\n",
      "2024-04-17 17:53:19,729 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021740.1, seqID NZ_CP021740.1 [record index 0].\n",
      "2024-04-17 17:53:19,800 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020061.1, seqID NZ_CP020061.1 [record index 0].\n",
      "2024-04-17 17:53:19,861 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032222.1, seqID NZ_CP032222.1 [record index 0].\n",
      "2024-04-17 17:53:19,946 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029137.1, seqID NZ_CP029137.1 [record index 0].\n",
      "2024-04-17 17:53:20,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039954.1, seqID NZ_CP039954.1 [record index 0].\n",
      "2024-04-17 17:53:20,064 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036438.1, seqID NZ_CP036438.1 [record index 0].\n",
      "2024-04-17 17:53:20,112 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036442.1, seqID NZ_CP036442.1 [record index 0].\n",
      "2024-04-17 17:53:20,163 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014755.1, seqID NZ_CP014755.1 [record index 0].\n",
      "2024-04-17 17:53:20,236 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042314.1, seqID NZ_CP042314.1 [record index 0].\n",
      "2024-04-17 17:53:20,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042315.1, seqID NZ_CP042315.1 [record index 0].\n",
      "2024-04-17 17:53:20,414 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017985.1, seqID NZ_CP017985.1 [record index 0].\n",
      "2024-04-17 17:53:20,470 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042317.1, seqID NZ_CP042317.1 [record index 0].\n",
      "2024-04-17 17:53:20,539 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082988.1, seqID NZ_CP082988.1 [record index 0].\n",
      "2024-04-17 17:53:20,586 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026495.1, seqID NZ_CP026495.1 [record index 0].\n",
      "2024-04-17 17:53:20,637 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089001.1, seqID NZ_CP089001.1 [record index 0].\n",
      "2024-04-17 17:53:20,688 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043932.1, seqID NZ_CP043932.1 [record index 0].\n",
      "2024-04-17 17:53:20,739 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050376.1, seqID NZ_CP050376.1 [record index 0].\n",
      "2024-04-17 17:53:20,789 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP023839.1, seqID NZ_CP023839.1 [record index 0].\n",
      "2024-04-17 17:53:20,854 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050360.1, seqID NZ_CP050360.1 [record index 0].\n",
      "2024-04-17 17:53:20,905 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034321.1, seqID NZ_CP034321.1 [record index 0].\n",
      "2024-04-17 17:53:20,989 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030296.1, seqID NZ_CP030296.1 [record index 0].\n",
      "2024-04-17 17:53:21,054 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028478.1, seqID NZ_CP028478.1 [record index 0].\n",
      "2024-04-17 17:53:21,120 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030302.1, seqID NZ_CP030302.1 [record index 0].\n",
      "2024-04-17 17:53:21,178 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021165.1, seqID NZ_CP021165.1 [record index 0].\n",
      "2024-04-17 17:53:21,226 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027042.1, seqID NZ_CP027042.1 [record index 0].\n",
      "2024-04-17 17:53:21,276 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022023.1, seqID NZ_CP022023.1 [record index 0].\n",
      "2024-04-17 17:53:21,361 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039951.1, seqID NZ_CP039951.1 [record index 0].\n",
      "2024-04-17 17:53:21,415 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083063.1, seqID NZ_CP083063.1 [record index 0].\n",
      "2024-04-17 17:53:21,465 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083066.1, seqID NZ_CP083066.1 [record index 0].\n",
      "2024-04-17 17:53:21,515 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026017.1, seqID NZ_CP026017.1 [record index 0].\n",
      "2024-04-17 17:53:21,603 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083070.1, seqID NZ_CP083070.1 [record index 0].\n",
      "2024-04-17 17:53:21,695 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030315.1, seqID NZ_CP030315.1 [record index 0].\n",
      "2024-04-17 17:53:21,784 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084871.1, seqID NZ_CP084871.1 [record index 0].\n",
      "2024-04-17 17:53:21,876 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083075.1, seqID NZ_CP083075.1 [record index 0].\n",
      "2024-04-17 17:53:22,049 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP074233.1, seqID NZ_CP074233.1 [record index 0].\n",
      "2024-04-17 17:53:22,199 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP045766.1, seqID NZ_CP045766.1 [record index 0].\n",
      "2024-04-17 17:53:22,297 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP067369.1, seqID NZ_CP067369.1 [record index 0].\n",
      "2024-04-17 17:53:22,397 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP022120.1, seqID NZ_CP022120.1 [record index 0].\n",
      "2024-04-17 17:53:22,497 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NC_015761.1, seqID NC_015761.1 [record index 0].\n",
      "2024-04-17 17:53:22,589 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP082841.1, seqID NZ_CP082841.1 [record index 0].\n",
      "2024-04-17 17:53:22,645 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134138.1, seqID NZ_LR134138.1 [record index 0].\n",
      "2024-04-17 17:53:22,691 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP047669.1, seqID NZ_CP047669.1 [record index 0].\n",
      "2024-04-17 17:53:22,719 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP048598.1, seqID NZ_CP048598.1 [record index 0].\n",
      "2024-04-17 17:53:22,792 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025034.2, seqID NZ_CP025034.2 [record index 0].\n",
      "2024-04-17 17:53:22,943 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074149.1, seqID NZ_CP074149.1 [record index 0].\n",
      "2024-04-17 17:53:23,048 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP074170.1, seqID NZ_CP074170.1 [record index 0].\n",
      "2024-04-17 17:53:23,143 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012999.1, seqID NZ_CP012999.1 [record index 0].\n",
      "2024-04-17 17:53:23,282 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011798.1, seqID NZ_CP011798.1 [record index 0].\n",
      "2024-04-17 17:53:23,425 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP006580.1, seqID NZ_CP006580.1 [record index 0].\n",
      "2024-04-17 17:53:23,514 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040527.1, seqID NZ_CP040527.1 [record index 0].\n",
      "2024-04-17 17:53:23,604 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040256.1, seqID NZ_CP040256.1 [record index 0].\n",
      "2024-04-17 17:53:23,729 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040606.1, seqID NZ_CP040606.1 [record index 0].\n",
      "2024-04-17 17:53:23,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041062.1, seqID NZ_CP041062.1 [record index 0].\n",
      "2024-04-17 17:53:23,901 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP083640.1, seqID NZ_CP083640.1 [record index 0].\n",
      "2024-04-17 17:53:23,986 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP082860.1, seqID NZ_CP082860.1 [record index 0].\n",
      "2024-04-17 17:53:24,092 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017279.1, seqID NZ_CP017279.1 [record index 0].\n",
      "2024-04-17 17:53:24,180 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_016514.1, seqID NC_016514.1 [record index 0].\n",
      "2024-04-17 17:53:24,286 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018785.1, seqID NZ_CP018785.1 [record index 0].\n",
      "2024-04-17 17:53:24,372 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP086101.1, seqID NZ_CP086101.1 [record index 0].\n",
      "2024-04-17 17:53:24,460 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP086099.1, seqID NZ_CP086099.1 [record index 0].\n",
      "2024-04-17 17:53:24,552 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP086100.1, seqID NZ_CP086100.1 [record index 0].\n",
      "2024-04-17 17:53:24,653 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032897.1, seqID NZ_CP032897.1 [record index 0].\n",
      "2024-04-17 17:53:24,752 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050073.1, seqID NZ_CP050073.1 [record index 0].\n",
      "2024-04-17 17:53:24,837 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP082280.1, seqID NZ_CP082280.1 [record index 0].\n",
      "2024-04-17 17:53:24,936 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024675.1, seqID NZ_CP024675.1 [record index 0].\n",
      "2024-04-17 17:53:25,014 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056399.1, seqID NZ_CP056399.1 [record index 0].\n",
      "2024-04-17 17:53:25,085 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056409.1, seqID NZ_CP056409.1 [record index 0].\n",
      "2024-04-17 17:53:25,156 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056834.1, seqID NZ_CP056834.1 [record index 0].\n",
      "2024-04-17 17:53:25,257 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046673.1, seqID NZ_CP046673.1 [record index 0].\n",
      "2024-04-17 17:53:25,349 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047247.1, seqID NZ_CP047247.1 [record index 0].\n",
      "2024-04-17 17:53:25,446 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024819.1, seqID NZ_CP024819.1 [record index 0].\n",
      "2024-04-17 17:53:25,504 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035196.1, seqID NZ_CP035196.1 [record index 0].\n",
      "2024-04-17 17:53:25,556 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP009863.1, seqID NZ_CP009863.1 [record index 0].\n",
      "2024-04-17 17:53:25,620 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034831.1, seqID NZ_CP034831.1 [record index 0].\n",
      "2024-04-17 17:53:25,714 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007216.2, seqID NZ_CP007216.2 [record index 0].\n",
      "2024-04-17 17:53:25,843 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022467.1, seqID NZ_CP022467.1 [record index 0].\n",
      "2024-04-17 17:53:26,015 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041171.1, seqID NZ_CP041171.1 [record index 0].\n",
      "2024-04-17 17:53:26,162 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039169.1, seqID NZ_CP039169.1 [record index 0].\n",
      "2024-04-17 17:53:26,256 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019404.1, seqID NZ_CP019404.1 [record index 0].\n",
      "2024-04-17 17:53:26,349 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022490.1, seqID NZ_CP022490.1 [record index 0].\n",
      "2024-04-17 17:53:26,453 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022503.1, seqID NZ_CP022503.1 [record index 0].\n",
      "2024-04-17 17:53:26,574 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089207.1, seqID NZ_CP089207.1 [record index 0].\n",
      "2024-04-17 17:53:26,658 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_022525.1, seqID NC_022525.1 [record index 0].\n",
      "2024-04-17 17:53:26,791 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012514.1, seqID NZ_CP012514.1 [record index 0].\n",
      "2024-04-17 17:53:26,911 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012513.1, seqID NZ_CP012513.1 [record index 0].\n",
      "2024-04-17 17:53:27,001 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP062223.1, seqID NZ_CP062223.1 [record index 0].\n",
      "2024-04-17 17:53:27,145 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP062225.1, seqID NZ_CP062225.1 [record index 0].\n",
      "2024-04-17 17:53:27,229 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037960.1, seqID NZ_CP037960.1 [record index 0].\n",
      "2024-04-17 17:53:27,349 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP092075.1, seqID NZ_CP092075.1 [record index 0].\n",
      "2024-04-17 17:53:27,438 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025745.1, seqID NZ_CP025745.1 [record index 0].\n",
      "2024-04-17 17:53:27,529 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079838.1, seqID NZ_CP079838.1 [record index 0].\n",
      "2024-04-17 17:53:27,656 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079836.1, seqID NZ_CP079836.1 [record index 0].\n",
      "2024-04-17 17:53:27,816 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079839.1, seqID NZ_CP079839.1 [record index 0].\n",
      "2024-04-17 17:53:27,985 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015924.1, seqID NZ_CP015924.1 [record index 0].\n",
      "2024-04-17 17:53:28,148 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015923.1, seqID NZ_CP015923.1 [record index 0].\n",
      "2024-04-17 17:53:28,210 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP082338.1, seqID NZ_CP082338.1 [record index 0].\n",
      "2024-04-17 17:53:28,302 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028729.1, seqID NZ_CP028729.1 [record index 0].\n",
      "2024-04-17 17:53:28,394 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050833.1, seqID NZ_CP050833.1 [record index 0].\n",
      "2024-04-17 17:53:28,484 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032304.1, seqID NZ_CP032304.1 [record index 0].\n",
      "2024-04-17 17:53:28,556 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051389.1, seqID NZ_CP051389.1 [record index 0].\n",
      "2024-04-17 17:53:28,676 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051416.1, seqID NZ_CP051416.1 [record index 0].\n",
      "2024-04-17 17:53:28,802 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016014.1, seqID NZ_CP016014.1 [record index 0].\n",
      "2024-04-17 17:53:28,893 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016010.1, seqID NZ_CP016010.1 [record index 0].\n",
      "2024-04-17 17:53:28,992 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074606.1, seqID NZ_CP074606.1 [record index 0].\n",
      "2024-04-17 17:53:29,123 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025246.1, seqID NZ_CP025246.1 [record index 0].\n",
      "2024-04-17 17:53:29,224 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025248.1, seqID NZ_CP025248.1 [record index 0].\n",
      "2024-04-17 17:53:29,342 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019196.1, seqID NZ_CP019196.1 [record index 0].\n",
      "2024-04-17 17:53:29,431 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011396.1, seqID NZ_CP011396.1 [record index 0].\n",
      "2024-04-17 17:53:29,550 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043027.1, seqID NZ_CP043027.1 [record index 0].\n",
      "2024-04-17 17:53:29,646 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060837.1, seqID NZ_CP060837.1 [record index 0].\n",
      "2024-04-17 17:53:29,761 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060840.1, seqID NZ_CP060840.1 [record index 0].\n",
      "2024-04-17 17:53:29,914 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060844.1, seqID NZ_CP060844.1 [record index 0].\n",
      "2024-04-17 17:53:30,011 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060846.1, seqID NZ_CP060846.1 [record index 0].\n",
      "2024-04-17 17:53:30,169 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP060849.1, seqID NZ_CP060849.1 [record index 0].\n",
      "2024-04-17 17:53:30,264 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040699.1, seqID NZ_CP040699.1 [record index 0].\n",
      "2024-04-17 17:53:30,353 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090141.1, seqID NZ_CP090141.1 [record index 0].\n",
      "2024-04-17 17:53:30,442 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090133.1, seqID NZ_CP090133.1 [record index 0].\n",
      "2024-04-17 17:53:30,532 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP090137.1, seqID NZ_CP090137.1 [record index 0].\n",
      "2024-04-17 17:53:30,647 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060852.1, seqID NZ_CP060852.1 [record index 0].\n",
      "2024-04-17 17:53:30,764 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060855.1, seqID NZ_CP060855.1 [record index 0].\n",
      "2024-04-17 17:53:30,928 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060834.1, seqID NZ_CP060834.1 [record index 0].\n",
      "2024-04-17 17:53:31,060 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP074182.1, seqID NZ_CP074182.1 [record index 0].\n",
      "2024-04-17 17:53:31,143 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP069815.1, seqID NZ_CP069815.1 [record index 0].\n",
      "2024-04-17 17:53:31,197 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056145.1, seqID NZ_CP056145.1 [record index 0].\n",
      "2024-04-17 17:53:31,248 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134235.1, seqID NZ_LR134235.1 [record index 0].\n",
      "2024-04-17 17:53:31,296 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068237.1, seqID NZ_CP068237.1 [record index 0].\n",
      "2024-04-17 17:53:31,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084789.1, seqID NZ_CP084789.1 [record index 0].\n",
      "2024-04-17 17:53:31,372 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035207.1, seqID NZ_CP035207.1 [record index 0].\n",
      "2024-04-17 17:53:31,423 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055213.1, seqID NZ_CP055213.1 [record index 0].\n",
      "2024-04-17 17:53:31,449 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084806.1, seqID NZ_CP084806.1 [record index 0].\n",
      "2024-04-17 17:53:31,476 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP058131.1, seqID NZ_CP058131.1 [record index 0].\n",
      "2024-04-17 17:53:31,501 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065467.1, seqID NZ_CP065467.1 [record index 0].\n",
      "2024-04-17 17:53:31,549 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR588411.1, seqID NZ_LR588411.1 [record index 0].\n",
      "2024-04-17 17:53:31,573 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP082785.1, seqID NZ_CP082785.1 [record index 0].\n",
      "2024-04-17 17:53:31,622 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP043928.1, seqID NZ_CP043928.1 [record index 0].\n",
      "2024-04-17 17:53:31,647 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045641.1, seqID NZ_CP045641.1 [record index 0].\n",
      "2024-04-17 17:53:31,718 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP062138.1, seqID NZ_CP062138.1 [record index 0].\n",
      "2024-04-17 17:53:31,807 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084818.1, seqID NZ_CP084818.1 [record index 0].\n",
      "2024-04-17 17:53:31,864 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP087706.1, seqID NZ_CP087706.1 [record index 0].\n",
      "2024-04-17 17:53:31,916 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090393.1, seqID NZ_CP090393.1 [record index 0].\n",
      "2024-04-17 17:53:31,982 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090389.1, seqID NZ_CP090389.1 [record index 0].\n",
      "2024-04-17 17:53:32,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079673.1, seqID NZ_CP079673.1 [record index 0].\n",
      "2024-04-17 17:53:32,081 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079668.1, seqID NZ_CP079668.1 [record index 0].\n",
      "2024-04-17 17:53:32,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079629.1, seqID NZ_CP079629.1 [record index 0].\n",
      "2024-04-17 17:53:32,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079624.1, seqID NZ_CP079624.1 [record index 0].\n",
      "2024-04-17 17:53:32,235 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079599.1, seqID NZ_CP079599.1 [record index 0].\n",
      "2024-04-17 17:53:32,285 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079164.1, seqID NZ_CP079164.1 [record index 0].\n",
      "2024-04-17 17:53:32,341 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079155.1, seqID NZ_CP079155.1 [record index 0].\n",
      "2024-04-17 17:53:32,391 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079150.1, seqID NZ_CP079150.1 [record index 0].\n",
      "2024-04-17 17:53:32,480 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079145.1, seqID NZ_CP079145.1 [record index 0].\n",
      "2024-04-17 17:53:32,545 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079138.1, seqID NZ_CP079138.1 [record index 0].\n",
      "2024-04-17 17:53:32,600 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079131.1, seqID NZ_CP079131.1 [record index 0].\n",
      "2024-04-17 17:53:32,655 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079120.1, seqID NZ_CP079120.1 [record index 0].\n",
      "2024-04-17 17:53:32,706 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP021929.1, seqID NZ_AP021929.1 [record index 0].\n",
      "2024-04-17 17:53:32,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045694.1, seqID NZ_CP045694.1 [record index 0].\n",
      "2024-04-17 17:53:32,809 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084823.1, seqID NZ_CP084823.1 [record index 0].\n",
      "2024-04-17 17:53:32,877 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045661.1, seqID NZ_CP045661.1 [record index 0].\n",
      "2024-04-17 17:53:32,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084825.1, seqID NZ_CP084825.1 [record index 0].\n",
      "2024-04-17 17:53:32,997 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084852.1, seqID NZ_CP084852.1 [record index 0].\n",
      "2024-04-17 17:53:33,083 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084859.1, seqID NZ_CP084859.1 [record index 0].\n",
      "2024-04-17 17:53:33,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071279.1, seqID NZ_CP071279.1 [record index 0].\n",
      "2024-04-17 17:53:33,257 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063865.1, seqID NZ_CP063865.1 [record index 0].\n",
      "2024-04-17 17:53:33,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068798.1, seqID NZ_CP068798.1 [record index 0].\n",
      "2024-04-17 17:53:33,401 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056191.1, seqID NZ_CP056191.1 [record index 0].\n",
      "2024-04-17 17:53:33,450 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056329.1, seqID NZ_CP056329.1 [record index 0].\n",
      "2024-04-17 17:53:33,504 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056425.1, seqID NZ_CP056425.1 [record index 0].\n",
      "2024-04-17 17:53:33,552 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056854.1, seqID NZ_CP056854.1 [record index 0].\n",
      "2024-04-17 17:53:33,608 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075548.1, seqID NZ_CP075548.1 [record index 0].\n",
      "2024-04-17 17:53:33,657 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075452.1, seqID NZ_CP075452.1 [record index 0].\n",
      "2024-04-17 17:53:33,706 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006798.1, seqID NZ_CP006798.1 [record index 0].\n",
      "2024-04-17 17:53:33,763 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014004.1, seqID NZ_CP014004.1 [record index 0].\n",
      "2024-04-17 17:53:33,813 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061389.1, seqID NZ_CP061389.1 [record index 0].\n",
      "2024-04-17 17:53:33,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091601.1, seqID NZ_CP091601.1 [record index 0].\n",
      "2024-04-17 17:53:33,926 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060049.1, seqID NZ_CP060049.1 [record index 0].\n",
      "2024-04-17 17:53:33,977 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP034408.1, seqID NZ_CP034408.1 [record index 0].\n",
      "2024-04-17 17:53:34,034 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134213.1, seqID NZ_LR134213.1 [record index 0].\n",
      "2024-04-17 17:53:34,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085866.1, seqID NZ_CP085866.1 [record index 0].\n",
      "2024-04-17 17:53:34,142 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073054.1, seqID NZ_CP073054.1 [record index 0].\n",
      "2024-04-17 17:53:34,192 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061965.1, seqID NZ_CP061965.1 [record index 0].\n",
      "2024-04-17 17:53:34,243 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025684.1, seqID NZ_CP025684.1 [record index 0].\n",
      "2024-04-17 17:53:34,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890710.1, seqID NZ_LR890710.1 [record index 0].\n",
      "2024-04-17 17:53:34,345 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890457.1, seqID NZ_LR890457.1 [record index 0].\n",
      "2024-04-17 17:53:34,405 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018337.1, seqID NZ_CP018337.1 [record index 0].\n",
      "2024-04-17 17:53:34,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP078033.1, seqID NZ_CP078033.1 [record index 0].\n",
      "2024-04-17 17:53:34,530 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065401.1, seqID NZ_CP065401.1 [record index 0].\n",
      "2024-04-17 17:53:34,585 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014762.1, seqID NZ_CP014762.1 [record index 0].\n",
      "2024-04-17 17:53:34,640 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP007731.1, seqID NZ_CP007731.1 [record index 0].\n",
      "2024-04-17 17:53:34,695 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020853.1, seqID NZ_CP020853.1 [record index 0].\n",
      "2024-04-17 17:53:34,749 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP052036.1, seqID NZ_CP052036.1 [record index 0].\n",
      "2024-04-17 17:53:34,821 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072494.1, seqID NZ_CP072494.1 [record index 0].\n",
      "2024-04-17 17:53:34,890 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018753.1, seqID NZ_AP018753.1 [record index 0].\n",
      "2024-04-17 17:53:34,941 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018750.1, seqID NZ_AP018750.1 [record index 0].\n",
      "2024-04-17 17:53:34,997 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012753.1, seqID NZ_CP012753.1 [record index 0].\n",
      "2024-04-17 17:53:35,053 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018747.1, seqID NZ_AP018747.1 [record index 0].\n",
      "2024-04-17 17:53:35,106 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072938.1, seqID NZ_CP072938.1 [record index 0].\n",
      "2024-04-17 17:53:35,164 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP003999.1, seqID NZ_CP003999.1 [record index 0].\n",
      "2024-04-17 17:53:35,253 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012883.1, seqID NZ_CP012883.1 [record index 0].\n",
      "2024-04-17 17:53:35,335 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_017540.1, seqID NC_017540.1 [record index 0].\n",
      "2024-04-17 17:53:35,399 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054400.1, seqID NZ_CP054400.1 [record index 0].\n",
      "2024-04-17 17:53:35,451 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054406.1, seqID NZ_CP054406.1 [record index 0].\n",
      "2024-04-17 17:53:35,501 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP054403.1, seqID NZ_CP054403.1 [record index 0].\n",
      "2024-04-17 17:53:35,557 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030923.1, seqID NZ_CP030923.1 [record index 0].\n",
      "2024-04-17 17:53:35,608 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071324.1, seqID NZ_CP071324.1 [record index 0].\n",
      "2024-04-17 17:53:35,656 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP079762.1, seqID NZ_CP079762.1 [record index 0].\n",
      "2024-04-17 17:53:35,718 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073782.1, seqID NZ_CP073782.1 [record index 0].\n",
      "2024-04-17 17:53:35,765 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072826.1, seqID NZ_CP072826.1 [record index 0].\n",
      "2024-04-17 17:53:35,834 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072825.1, seqID NZ_CP072825.1 [record index 0].\n",
      "2024-04-17 17:53:35,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071150.1, seqID NZ_CP071150.1 [record index 0].\n",
      "2024-04-17 17:53:35,939 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890527.1, seqID NZ_LR890527.1 [record index 0].\n",
      "2024-04-17 17:53:35,988 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890277.1, seqID NZ_LR890277.1 [record index 0].\n",
      "2024-04-17 17:53:36,053 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890323.1, seqID NZ_LR890323.1 [record index 0].\n",
      "2024-04-17 17:53:36,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890640.1, seqID NZ_LR890640.1 [record index 0].\n",
      "2024-04-17 17:53:36,221 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890490.1, seqID NZ_LR890490.1 [record index 0].\n",
      "2024-04-17 17:53:36,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032833.1, seqID NZ_CP032833.1 [record index 0].\n",
      "2024-04-17 17:53:36,397 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081889.1, seqID NZ_CP081889.1 [record index 0].\n",
      "2024-04-17 17:53:36,458 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP025211.1, seqID NZ_CP025211.1 [record index 0].\n",
      "2024-04-17 17:53:36,514 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033777.1, seqID NZ_CP033777.1 [record index 0].\n",
      "2024-04-17 17:53:36,581 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069853.1, seqID NZ_CP069853.1 [record index 0].\n",
      "2024-04-17 17:53:36,641 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070046.1, seqID NZ_CP070046.1 [record index 0].\n",
      "2024-04-17 17:53:36,696 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070172.1, seqID NZ_CP070172.1 [record index 0].\n",
      "2024-04-17 17:53:36,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069968.1, seqID NZ_CP069968.1 [record index 0].\n",
      "2024-04-17 17:53:36,798 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069962.1, seqID NZ_CP069962.1 [record index 0].\n",
      "2024-04-17 17:53:36,852 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069847.1, seqID NZ_CP069847.1 [record index 0].\n",
      "2024-04-17 17:53:36,906 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059295.1, seqID NZ_CP059295.1 [record index 0].\n",
      "2024-04-17 17:53:36,961 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060281.1, seqID NZ_CP060281.1 [record index 0].\n",
      "2024-04-17 17:53:37,012 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089097.1, seqID NZ_CP089097.1 [record index 0].\n",
      "2024-04-17 17:53:37,067 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090429.1, seqID NZ_CP090429.1 [record index 0].\n",
      "2024-04-17 17:53:37,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016811.1, seqID NZ_CP016811.1 [record index 0].\n",
      "2024-04-17 17:53:37,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030072.1, seqID NZ_CP030072.1 [record index 0].\n",
      "2024-04-17 17:53:37,232 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP093151.1, seqID NZ_CP093151.1 [record index 0].\n",
      "2024-04-17 17:53:37,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081891.1, seqID NZ_CP081891.1 [record index 0].\n",
      "2024-04-17 17:53:37,339 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP062986.1, seqID NZ_CP062986.1 [record index 0].\n",
      "2024-04-17 17:53:37,391 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024707.1, seqID NZ_CP024707.1 [record index 0].\n",
      "2024-04-17 17:53:37,443 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077779.1, seqID NZ_CP077779.1 [record index 0].\n",
      "2024-04-17 17:53:37,497 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022611.1, seqID NZ_CP022611.1 [record index 0].\n",
      "2024-04-17 17:53:37,549 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061700.1, seqID NZ_CP061700.1 [record index 0].\n",
      "2024-04-17 17:53:37,596 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084103.1, seqID NZ_CP084103.1 [record index 0].\n",
      "2024-04-17 17:53:37,648 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061832.1, seqID NZ_CP061832.1 [record index 0].\n",
      "2024-04-17 17:53:37,697 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053771.1, seqID NZ_CP053771.1 [record index 0].\n",
      "2024-04-17 17:53:37,749 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050826.1, seqID NZ_CP050826.1 [record index 0].\n",
      "2024-04-17 17:53:37,803 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050834.1, seqID NZ_CP050834.1 [record index 0].\n",
      "2024-04-17 17:53:37,854 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP059468.1, seqID NZ_CP059468.1 [record index 0].\n",
      "2024-04-17 17:53:37,905 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053765.1, seqID NZ_CP053765.1 [record index 0].\n",
      "2024-04-17 17:53:37,956 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028932.1, seqID NZ_CP028932.1 [record index 0].\n",
      "2024-04-17 17:53:38,007 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP021708.1, seqID NZ_CP021708.1 [record index 0].\n",
      "2024-04-17 17:53:38,079 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028990.1, seqID NZ_CP028990.1 [record index 0].\n",
      "2024-04-17 17:53:38,131 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028953.1, seqID NZ_CP028953.1 [record index 0].\n",
      "2024-04-17 17:53:38,181 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029722.1, seqID NZ_CP029722.1 [record index 0].\n",
      "2024-04-17 17:53:38,232 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020067.1, seqID NZ_CP020067.1 [record index 0].\n",
      "2024-04-17 17:53:38,282 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026751.1, seqID NZ_CP026751.1 [record index 0].\n",
      "2024-04-17 17:53:38,331 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088985.1, seqID NZ_CP088985.1 [record index 0].\n",
      "2024-04-17 17:53:38,380 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088988.1, seqID NZ_CP088988.1 [record index 0].\n",
      "2024-04-17 17:53:38,434 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083043.1, seqID NZ_CP083043.1 [record index 0].\n",
      "2024-04-17 17:53:38,486 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083047.1, seqID NZ_CP083047.1 [record index 0].\n",
      "2024-04-17 17:53:38,537 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP062792.1, seqID NZ_CP062792.1 [record index 0].\n",
      "2024-04-17 17:53:38,601 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016926.1, seqID NZ_CP016926.1 [record index 0].\n",
      "2024-04-17 17:53:38,654 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP019219.1, seqID NZ_CP019219.1 [record index 0].\n",
      "2024-04-17 17:53:38,700 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP006722.1, seqID NZ_CP006722.1 [record index 0].\n",
      "2024-04-17 17:53:38,755 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016923.1, seqID NZ_CP016923.1 [record index 0].\n",
      "2024-04-17 17:53:38,802 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084873.1, seqID NZ_CP084873.1 [record index 0].\n",
      "2024-04-17 17:53:38,851 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083051.1, seqID NZ_CP083051.1 [record index 0].\n",
      "2024-04-17 17:53:38,978 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP031101.1, seqID NZ_CP031101.1 [record index 0].\n",
      "2024-04-17 17:53:39,025 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045845.1, seqID NZ_CP045845.1 [record index 0].\n",
      "2024-04-17 17:53:39,070 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045843.1, seqID NZ_CP045843.1 [record index 0].\n",
      "2024-04-17 17:53:39,156 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP055883.1, seqID NZ_CP055883.1 [record index 0].\n",
      "2024-04-17 17:53:39,278 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056393.1, seqID NZ_CP056393.1 [record index 0].\n",
      "2024-04-17 17:53:39,371 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056552.1, seqID NZ_CP056552.1 [record index 0].\n",
      "2024-04-17 17:53:39,447 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077407.1, seqID NZ_CP077407.1 [record index 0].\n",
      "2024-04-17 17:53:39,554 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017184.1, seqID NZ_CP017184.1 [record index 0].\n",
      "2024-04-17 17:53:39,654 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027618.1, seqID NZ_CP027618.1 [record index 0].\n",
      "2024-04-17 17:53:39,723 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045769.1, seqID NZ_CP045769.1 [record index 0].\n",
      "2024-04-17 17:53:39,786 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077290.1, seqID NZ_CP077290.1 [record index 0].\n",
      "2024-04-17 17:53:39,848 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025225.1, seqID NZ_CP025225.1 [record index 0].\n",
      "2024-04-17 17:53:39,935 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP084290.1, seqID NZ_CP084290.1 [record index 0].\n",
      "2024-04-17 17:53:40,011 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056232.1, seqID NZ_CP056232.1 [record index 0].\n",
      "2024-04-17 17:53:40,111 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP055551.1, seqID NZ_CP055551.1 [record index 0].\n",
      "2024-04-17 17:53:40,208 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056907.1, seqID NZ_CP056907.1 [record index 0].\n",
      "2024-04-17 17:53:40,283 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007557.1, seqID NZ_CP007557.1 [record index 0].\n",
      "2024-04-17 17:53:40,364 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060441.1, seqID NZ_CP060441.1 [record index 0].\n",
      "2024-04-17 17:53:40,391 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP009451.1, seqID NZ_CP009451.1 [record index 0].\n",
      "2024-04-17 17:53:40,482 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034819.1, seqID NZ_CP034819.1 [record index 0].\n",
      "2024-04-17 17:53:40,570 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP084216.1, seqID NZ_CP084216.1 [record index 0].\n",
      "2024-04-17 17:53:40,658 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP084001.1, seqID NZ_CP084001.1 [record index 0].\n",
      "2024-04-17 17:53:40,726 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP084194.1, seqID NZ_CP084194.1 [record index 0].\n",
      "2024-04-17 17:53:40,813 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP083731.1, seqID NZ_CP083731.1 [record index 0].\n",
      "2024-04-17 17:53:40,900 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089788.1, seqID NZ_CP089788.1 [record index 0].\n",
      "2024-04-17 17:53:40,988 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089789.1, seqID NZ_CP089789.1 [record index 0].\n",
      "2024-04-17 17:53:41,075 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089790.1, seqID NZ_CP089790.1 [record index 0].\n",
      "2024-04-17 17:53:41,161 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089791.1, seqID NZ_CP089791.1 [record index 0].\n",
      "2024-04-17 17:53:41,247 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089792.1, seqID NZ_CP089792.1 [record index 0].\n",
      "2024-04-17 17:53:41,332 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089794.1, seqID NZ_CP089794.1 [record index 0].\n",
      "2024-04-17 17:53:41,424 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089795.1, seqID NZ_CP089795.1 [record index 0].\n",
      "2024-04-17 17:53:41,510 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089797.1, seqID NZ_CP089797.1 [record index 0].\n",
      "2024-04-17 17:53:41,596 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089798.1, seqID NZ_CP089798.1 [record index 0].\n",
      "2024-04-17 17:53:41,684 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089799.1, seqID NZ_CP089799.1 [record index 0].\n",
      "2024-04-17 17:53:41,758 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090304.1, seqID NZ_CP090304.1 [record index 0].\n",
      "2024-04-17 17:53:41,846 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085811.1, seqID NZ_CP085811.1 [record index 0].\n",
      "2024-04-17 17:53:41,925 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP011428.1, seqID NZ_CP011428.1 [record index 0].\n",
      "2024-04-17 17:53:42,004 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014358.1, seqID NZ_CP014358.1 [record index 0].\n",
      "2024-04-17 17:53:42,081 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP035547.1, seqID NZ_CP035547.1 [record index 0].\n",
      "2024-04-17 17:53:42,155 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022168.1, seqID NZ_CP022168.1 [record index 0].\n",
      "2024-04-17 17:53:42,240 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040575.1, seqID NZ_CP040575.1 [record index 0].\n",
      "2024-04-17 17:53:42,314 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016357.1, seqID NZ_CP016357.1 [record index 0].\n",
      "2024-04-17 17:53:42,383 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085820.1, seqID NZ_CP085820.1 [record index 0].\n",
      "2024-04-17 17:53:42,451 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039436.1, seqID NZ_CP039436.1 [record index 0].\n",
      "2024-04-17 17:53:42,515 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039437.1, seqID NZ_CP039437.1 [record index 0].\n",
      "2024-04-17 17:53:42,603 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039439.1, seqID NZ_CP039439.1 [record index 0].\n",
      "2024-04-17 17:53:42,674 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007804.1, seqID NZ_CP007804.1 [record index 0].\n",
      "2024-04-17 17:53:42,739 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LT795114.1, seqID NZ_LT795114.1 [record index 0].\n",
      "2024-04-17 17:53:42,804 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_021902.1, seqID NC_021902.1 [record index 0].\n",
      "2024-04-17 17:53:42,874 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040686.1, seqID NZ_CP040686.1 [record index 0].\n",
      "2024-04-17 17:53:42,953 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025241.1, seqID NZ_CP025241.1 [record index 0].\n",
      "2024-04-17 17:53:43,034 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025237.1, seqID NZ_CP025237.1 [record index 0].\n",
      "2024-04-17 17:53:43,113 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025234.1, seqID NZ_CP025234.1 [record index 0].\n",
      "2024-04-17 17:53:43,192 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025232.1, seqID NZ_CP025232.1 [record index 0].\n",
      "2024-04-17 17:53:43,271 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025230.1, seqID NZ_CP025230.1 [record index 0].\n",
      "2024-04-17 17:53:43,351 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025273.1, seqID NZ_CP025273.1 [record index 0].\n",
      "2024-04-17 17:53:43,447 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017973.1, seqID NZ_CP017973.1 [record index 0].\n",
      "2024-04-17 17:53:43,544 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017972.1, seqID NZ_CP017972.1 [record index 0].\n",
      "2024-04-17 17:53:43,625 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007222.1, seqID NZ_CP007222.1 [record index 0].\n",
      "2024-04-17 17:53:43,719 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017971.1, seqID NZ_CP017971.1 [record index 0].\n",
      "2024-04-17 17:53:43,815 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017970.1, seqID NZ_CP017970.1 [record index 0].\n",
      "2024-04-17 17:53:43,884 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007235.2, seqID NZ_CP007235.2 [record index 0].\n",
      "2024-04-17 17:53:43,957 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014971.2, seqID NZ_CP014971.2 [record index 0].\n",
      "2024-04-17 17:53:44,038 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014977.1, seqID NZ_CP014977.1 [record index 0].\n",
      "2024-04-17 17:53:44,106 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014981.1, seqID NZ_CP014981.1 [record index 0].\n",
      "2024-04-17 17:53:44,173 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014982.2, seqID NZ_CP014982.2 [record index 0].\n",
      "2024-04-17 17:53:44,248 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014969.1, seqID NZ_CP014969.1 [record index 0].\n",
      "2024-04-17 17:53:44,323 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_016863.1, seqID NC_016863.1 [record index 0].\n",
      "2024-04-17 17:53:44,410 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020101.1, seqID NZ_CP020101.1 [record index 0].\n",
      "2024-04-17 17:53:44,512 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023975.1, seqID NZ_CP023975.1 [record index 0].\n",
      "2024-04-17 17:53:44,600 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_021176.1, seqID NC_021176.1 [record index 0].\n",
      "2024-04-17 17:53:44,687 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_004631.1, seqID NC_004631.1 [record index 0].\n",
      "2024-04-17 17:53:44,778 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007505.1, seqID NZ_CP007505.1 [record index 0].\n",
      "2024-04-17 17:53:44,860 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019649.1, seqID NZ_CP019649.1 [record index 0].\n",
      "2024-04-17 17:53:44,940 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040458.1, seqID NZ_CP040458.1 [record index 0].\n",
      "2024-04-17 17:53:45,015 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085987.1, seqID NZ_CP085987.1 [record index 0].\n",
      "2024-04-17 17:53:45,108 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085807.1, seqID NZ_CP085807.1 [record index 0].\n",
      "2024-04-17 17:53:45,205 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_003197.2, seqID NC_003197.2 [record index 0].\n",
      "2024-04-17 17:53:45,274 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LT855376.1, seqID NZ_LT855376.1 [record index 0].\n",
      "2024-04-17 17:53:45,347 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016385.1, seqID NZ_CP016385.1 [record index 0].\n",
      "2024-04-17 17:53:45,415 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050734.1, seqID NZ_CP050734.1 [record index 0].\n",
      "2024-04-17 17:53:45,488 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_016857.1, seqID NC_016857.1 [record index 0].\n",
      "2024-04-17 17:53:45,562 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050750.1, seqID NZ_CP050750.1 [record index 0].\n",
      "2024-04-17 17:53:45,636 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050753.1, seqID NZ_CP050753.1 [record index 0].\n",
      "2024-04-17 17:53:45,725 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019405.1, seqID NZ_CP019405.1 [record index 0].\n",
      "2024-04-17 17:53:45,809 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019414.1, seqID NZ_CP019414.1 [record index 0].\n",
      "2024-04-17 17:53:45,888 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP035301.1, seqID NZ_CP035301.1 [record index 0].\n",
      "2024-04-17 17:53:45,971 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050726.1, seqID NZ_CP050726.1 [record index 0].\n",
      "2024-04-17 17:53:46,040 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP021909.1, seqID NZ_CP021909.1 [record index 0].\n",
      "2024-04-17 17:53:46,109 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050728.1, seqID NZ_CP050728.1 [record index 0].\n",
      "2024-04-17 17:53:46,188 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050731.1, seqID NZ_CP050731.1 [record index 0].\n",
      "2024-04-17 17:53:46,260 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053870.1, seqID NZ_CP053870.1 [record index 0].\n",
      "2024-04-17 17:53:46,326 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP058807.1, seqID NZ_CP058807.1 [record index 0].\n",
      "2024-04-17 17:53:46,421 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP092911.1, seqID NZ_CP092911.1 [record index 0].\n",
      "2024-04-17 17:53:46,511 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_010102.1, seqID NC_010102.1 [record index 0].\n",
      "2024-04-17 17:53:46,582 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077668.1, seqID NZ_CP077668.1 [record index 0].\n",
      "2024-04-17 17:53:46,652 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP074092.1, seqID NZ_CP074092.1 [record index 0].\n",
      "2024-04-17 17:53:46,723 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN999997.1, seqID NZ_LN999997.1 [record index 0].\n",
      "2024-04-17 17:53:46,797 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014536.1, seqID NZ_CP014536.1 [record index 0].\n",
      "2024-04-17 17:53:46,869 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032494.1, seqID NZ_CP032494.1 [record index 0].\n",
      "2024-04-17 17:53:46,943 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014356.1, seqID NZ_CP014356.1 [record index 0].\n",
      "2024-04-17 17:53:47,018 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060515.1, seqID NZ_CP060515.1 [record index 0].\n",
      "2024-04-17 17:53:47,093 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053865.1, seqID NZ_CP053865.1 [record index 0].\n",
      "2024-04-17 17:53:47,195 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011083.1, seqID NC_011083.1 [record index 0].\n",
      "2024-04-17 17:53:47,269 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032490.1, seqID NZ_CP032490.1 [record index 0].\n",
      "2024-04-17 17:53:47,342 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_011080.1, seqID NC_011080.1 [record index 0].\n",
      "2024-04-17 17:53:47,415 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_016810.1, seqID NC_016810.1 [record index 0].\n",
      "2024-04-17 17:53:47,506 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043214.1, seqID NZ_CP043214.1 [record index 0].\n",
      "2024-04-17 17:53:47,588 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047522.1, seqID NZ_CP047522.1 [record index 0].\n",
      "2024-04-17 17:53:47,671 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047525.1, seqID NZ_CP047525.1 [record index 0].\n",
      "2024-04-17 17:53:47,755 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047527.1, seqID NZ_CP047527.1 [record index 0].\n",
      "2024-04-17 17:53:47,838 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047529.1, seqID NZ_CP047529.1 [record index 0].\n",
      "2024-04-17 17:53:47,931 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047531.1, seqID NZ_CP047531.1 [record index 0].\n",
      "2024-04-17 17:53:48,021 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047535.1, seqID NZ_CP047535.1 [record index 0].\n",
      "2024-04-17 17:53:48,104 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047537.1, seqID NZ_CP047537.1 [record index 0].\n",
      "2024-04-17 17:53:48,183 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047540.1, seqID NZ_CP047540.1 [record index 0].\n",
      "2024-04-17 17:53:48,266 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047542.1, seqID NZ_CP047542.1 [record index 0].\n",
      "2024-04-17 17:53:48,355 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047544.1, seqID NZ_CP047544.1 [record index 0].\n",
      "2024-04-17 17:53:48,438 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047546.1, seqID NZ_CP047546.1 [record index 0].\n",
      "2024-04-17 17:53:48,519 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047553.1, seqID NZ_CP047553.1 [record index 0].\n",
      "2024-04-17 17:53:48,601 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047548.1, seqID NZ_CP047548.1 [record index 0].\n",
      "2024-04-17 17:53:48,683 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047555.1, seqID NZ_CP047555.1 [record index 0].\n",
      "2024-04-17 17:53:48,765 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047550.1, seqID NZ_CP047550.1 [record index 0].\n",
      "2024-04-17 17:53:48,866 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016579.1, seqID NZ_CP016579.1 [record index 0].\n",
      "2024-04-17 17:53:48,964 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016586.1, seqID NZ_CP016586.1 [record index 0].\n",
      "2024-04-17 17:53:49,063 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016504.1, seqID NZ_CP016504.1 [record index 0].\n",
      "2024-04-17 17:53:49,162 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016507.1, seqID NZ_CP016507.1 [record index 0].\n",
      "2024-04-17 17:53:49,251 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066851.1, seqID NZ_CP066851.1 [record index 0].\n",
      "2024-04-17 17:53:49,341 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP036167.1, seqID NZ_CP036167.1 [record index 0].\n",
      "2024-04-17 17:53:49,443 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP036168.1, seqID NZ_CP036168.1 [record index 0].\n",
      "2024-04-17 17:53:49,546 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017723.1, seqID NZ_CP017723.1 [record index 0].\n",
      "2024-04-17 17:53:49,643 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019176.1, seqID NZ_CP019176.1 [record index 0].\n",
      "2024-04-17 17:53:49,741 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017727.1, seqID NZ_CP017727.1 [record index 0].\n",
      "2024-04-17 17:53:49,819 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017728.1, seqID NZ_CP017728.1 [record index 0].\n",
      "2024-04-17 17:53:49,909 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023166.1, seqID NZ_CP023166.1 [record index 0].\n",
      "2024-04-17 17:53:49,981 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP081189.2, seqID NZ_CP081189.2 [record index 0].\n",
      "2024-04-17 17:53:50,048 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP081187.1, seqID NZ_CP081187.1 [record index 0].\n",
      "2024-04-17 17:53:50,118 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067339.1, seqID NZ_CP067339.1 [record index 0].\n",
      "2024-04-17 17:53:50,212 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051213.1, seqID NZ_CP051213.1 [record index 0].\n",
      "2024-04-17 17:53:50,285 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047115.1, seqID NZ_CP047115.1 [record index 0].\n",
      "2024-04-17 17:53:50,390 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_006905.1, seqID NC_006905.1 [record index 0].\n",
      "2024-04-17 17:53:50,458 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041208.1, seqID NZ_CP041208.1 [record index 0].\n",
      "2024-04-17 17:53:50,524 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040900.1, seqID NZ_CP040900.1 [record index 0].\n",
      "2024-04-17 17:53:50,593 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040568.1, seqID NZ_CP040568.1 [record index 0].\n",
      "2024-04-17 17:53:50,664 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040564.1, seqID NZ_CP040564.1 [record index 0].\n",
      "2024-04-17 17:53:50,728 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040562.1, seqID NZ_CP040562.1 [record index 0].\n",
      "2024-04-17 17:53:50,795 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040566.1, seqID NZ_CP040566.1 [record index 0].\n",
      "2024-04-17 17:53:50,862 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041026.1, seqID NZ_CP041026.1 [record index 0].\n",
      "2024-04-17 17:53:50,947 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038593.1, seqID NZ_CP038593.1 [record index 0].\n",
      "2024-04-17 17:53:51,039 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043222.1, seqID NZ_CP043222.1 [record index 0].\n",
      "2024-04-17 17:53:51,136 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030190.1, seqID NZ_CP030190.1 [record index 0].\n",
      "2024-04-17 17:53:51,228 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030185.1, seqID NZ_CP030185.1 [record index 0].\n",
      "2024-04-17 17:53:51,303 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022497.1, seqID NZ_CP022497.1 [record index 0].\n",
      "2024-04-17 17:53:51,375 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040668.1, seqID NZ_CP040668.1 [record index 0].\n",
      "2024-04-17 17:53:51,442 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040651.1, seqID NZ_CP040651.1 [record index 0].\n",
      "2024-04-17 17:53:51,530 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038591.1, seqID NZ_CP038591.1 [record index 0].\n",
      "2024-04-17 17:53:51,606 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030202.1, seqID NZ_CP030202.1 [record index 0].\n",
      "2024-04-17 17:53:51,683 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030225.1, seqID NZ_CP030225.1 [record index 0].\n",
      "2024-04-17 17:53:51,775 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022494.1, seqID NZ_CP022494.1 [record index 0].\n",
      "2024-04-17 17:53:51,873 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022491.1, seqID NZ_CP022491.1 [record index 0].\n",
      "2024-04-17 17:53:51,964 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022500.1, seqID NZ_CP022500.1 [record index 0].\n",
      "2024-04-17 17:53:52,055 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029992.1, seqID NZ_CP029992.1 [record index 0].\n",
      "2024-04-17 17:53:52,142 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029991.1, seqID NZ_CP029991.1 [record index 0].\n",
      "2024-04-17 17:53:52,234 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030175.1, seqID NZ_CP030175.1 [record index 0].\n",
      "2024-04-17 17:53:52,325 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012921.1, seqID NZ_CP012921.1 [record index 0].\n",
      "2024-04-17 17:53:52,414 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016521.1, seqID NZ_CP016521.1 [record index 0].\n",
      "2024-04-17 17:53:52,504 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016530.1, seqID NZ_CP016530.1 [record index 0].\n",
      "2024-04-17 17:53:52,579 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP035915.1, seqID NZ_CP035915.1 [record index 0].\n",
      "2024-04-17 17:53:52,654 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061115.1, seqID NZ_CP061115.1 [record index 0].\n",
      "2024-04-17 17:53:52,736 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061119.1, seqID NZ_CP061119.1 [record index 0].\n",
      "2024-04-17 17:53:52,815 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP035917.1, seqID NZ_CP035917.1 [record index 0].\n",
      "2024-04-17 17:53:52,889 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061122.1, seqID NZ_CP061122.1 [record index 0].\n",
      "2024-04-17 17:53:52,958 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061124.1, seqID NZ_CP061124.1 [record index 0].\n",
      "2024-04-17 17:53:53,029 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP086118.1, seqID NZ_CP086118.1 [record index 0].\n",
      "2024-04-17 17:53:53,098 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061126.1, seqID NZ_CP061126.1 [record index 0].\n",
      "2024-04-17 17:53:53,173 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085699.1, seqID NZ_CP085699.1 [record index 0].\n",
      "2024-04-17 17:53:53,268 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085696.1, seqID NZ_CP085696.1 [record index 0].\n",
      "2024-04-17 17:53:53,360 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP081858.1, seqID NZ_CP081858.1 [record index 0].\n",
      "2024-04-17 17:53:53,436 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP089209.1, seqID NZ_CP089209.1 [record index 0].\n",
      "2024-04-17 17:53:53,514 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR881463.1, seqID NZ_LR881463.1 [record index 0].\n",
      "2024-04-17 17:53:53,581 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019418.1, seqID NZ_CP019418.1 [record index 0].\n",
      "2024-04-17 17:53:53,666 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019416.1, seqID NZ_CP019416.1 [record index 0].\n",
      "2024-04-17 17:53:53,766 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034713.1, seqID NZ_CP034713.1 [record index 0].\n",
      "2024-04-17 17:53:53,843 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034719.1, seqID NZ_CP034719.1 [record index 0].\n",
      "2024-04-17 17:53:53,921 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047323.1, seqID NZ_CP047323.1 [record index 0].\n",
      "2024-04-17 17:53:53,995 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022658.1, seqID NZ_CP022658.1 [record index 0].\n",
      "2024-04-17 17:53:54,074 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013702.1, seqID NZ_CP013702.1 [record index 0].\n",
      "2024-04-17 17:53:54,168 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_012125.1, seqID NC_012125.1 [record index 0].\n",
      "2024-04-17 17:53:54,261 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085809.1, seqID NZ_CP085809.1 [record index 0].\n",
      "2024-04-17 17:53:54,355 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046429.1, seqID NZ_CP046429.1 [record index 0].\n",
      "2024-04-17 17:53:54,425 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP065567.1, seqID NZ_CP065567.1 [record index 0].\n",
      "2024-04-17 17:53:54,490 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP063294.1, seqID NZ_CP063294.1 [record index 0].\n",
      "2024-04-17 17:53:54,576 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043773.1, seqID NZ_CP043773.1 [record index 0].\n",
      "2024-04-17 17:53:54,668 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026327.1, seqID NZ_CP026327.1 [record index 0].\n",
      "2024-04-17 17:53:54,739 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037877.1, seqID NZ_CP037877.1 [record index 0].\n",
      "2024-04-17 17:53:54,807 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044961.1, seqID NZ_CP044961.1 [record index 0].\n",
      "2024-04-17 17:53:54,874 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039860.1, seqID NZ_CP039860.1 [record index 0].\n",
      "2024-04-17 17:53:54,949 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040321.1, seqID NZ_CP040321.1 [record index 0].\n",
      "2024-04-17 17:53:55,022 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039856.1, seqID NZ_CP039856.1 [record index 0].\n",
      "2024-04-17 17:53:55,094 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037882.1, seqID NZ_CP037882.1 [record index 0].\n",
      "2024-04-17 17:53:55,170 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039610.1, seqID NZ_CP039610.1 [record index 0].\n",
      "2024-04-17 17:53:55,251 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039607.1, seqID NZ_CP039607.1 [record index 0].\n",
      "2024-04-17 17:53:55,321 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039603.1, seqID NZ_CP039603.1 [record index 0].\n",
      "2024-04-17 17:53:55,391 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039599.1, seqID NZ_CP039599.1 [record index 0].\n",
      "2024-04-17 17:53:55,468 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP038849.1, seqID NZ_CP038849.1 [record index 0].\n",
      "2024-04-17 17:53:55,537 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039595.1, seqID NZ_CP039595.1 [record index 0].\n",
      "2024-04-17 17:53:55,603 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039854.1, seqID NZ_CP039854.1 [record index 0].\n",
      "2024-04-17 17:53:55,675 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037879.1, seqID NZ_CP037879.1 [record index 0].\n",
      "2024-04-17 17:53:55,760 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039593.1, seqID NZ_CP039593.1 [record index 0].\n",
      "2024-04-17 17:53:55,833 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039591.1, seqID NZ_CP039591.1 [record index 0].\n",
      "2024-04-17 17:53:55,912 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039588.1, seqID NZ_CP039588.1 [record index 0].\n",
      "2024-04-17 17:53:55,986 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039719.1, seqID NZ_CP039719.1 [record index 0].\n",
      "2024-04-17 17:53:56,056 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039585.1, seqID NZ_CP039585.1 [record index 0].\n",
      "2024-04-17 17:53:56,123 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039582.1, seqID NZ_CP039582.1 [record index 0].\n",
      "2024-04-17 17:53:56,190 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039579.1, seqID NZ_CP039579.1 [record index 0].\n",
      "2024-04-17 17:53:56,264 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039572.1, seqID NZ_CP039572.1 [record index 0].\n",
      "2024-04-17 17:53:56,335 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037874.1, seqID NZ_CP037874.1 [record index 0].\n",
      "2024-04-17 17:53:56,402 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039713.1, seqID NZ_CP039713.1 [record index 0].\n",
      "2024-04-17 17:53:56,470 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039569.1, seqID NZ_CP039569.1 [record index 0].\n",
      "2024-04-17 17:53:56,546 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP038847.1, seqID NZ_CP038847.1 [record index 0].\n",
      "2024-04-17 17:53:56,619 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039567.1, seqID NZ_CP039567.1 [record index 0].\n",
      "2024-04-17 17:53:56,687 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039565.1, seqID NZ_CP039565.1 [record index 0].\n",
      "2024-04-17 17:53:56,756 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039564.1, seqID NZ_CP039564.1 [record index 0].\n",
      "2024-04-17 17:53:56,835 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039561.1, seqID NZ_CP039561.1 [record index 0].\n",
      "2024-04-17 17:53:56,905 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039558.1, seqID NZ_CP039558.1 [record index 0].\n",
      "2024-04-17 17:53:56,974 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037881.1, seqID NZ_CP037881.1 [record index 0].\n",
      "2024-04-17 17:53:57,058 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040318.1, seqID NZ_CP040318.1 [record index 0].\n",
      "2024-04-17 17:53:57,137 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040648.1, seqID NZ_CP040648.1 [record index 0].\n",
      "2024-04-17 17:53:57,210 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP036174.1, seqID NZ_CP036174.1 [record index 0].\n",
      "2024-04-17 17:53:57,276 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044967.1, seqID NZ_CP044967.1 [record index 0].\n",
      "2024-04-17 17:53:57,343 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044957.1, seqID NZ_CP044957.1 [record index 0].\n",
      "2024-04-17 17:53:57,410 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039716.1, seqID NZ_CP039716.1 [record index 0].\n",
      "2024-04-17 17:53:57,508 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012091.1, seqID NZ_CP012091.1 [record index 0].\n",
      "2024-04-17 17:53:57,580 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025555.1, seqID NZ_CP025555.1 [record index 0].\n",
      "2024-04-17 17:53:57,666 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025217.1, seqID NZ_CP025217.1 [record index 0].\n",
      "2024-04-17 17:53:57,776 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP092861.1, seqID NZ_CP092861.1 [record index 0].\n",
      "2024-04-17 17:53:57,876 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_016832.1, seqID NC_016832.1 [record index 0].\n",
      "2024-04-17 17:53:57,962 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040380.1, seqID NZ_CP040380.1 [record index 0].\n",
      "2024-04-17 17:53:58,038 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051269.1, seqID NZ_CP051269.1 [record index 0].\n",
      "2024-04-17 17:53:58,118 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051267.1, seqID NZ_CP051267.1 [record index 0].\n",
      "2024-04-17 17:53:58,194 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075372.1, seqID NZ_CP075372.1 [record index 0].\n",
      "2024-04-17 17:53:58,264 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LT571437.1, seqID NZ_LT571437.1 [record index 0].\n",
      "2024-04-17 17:53:58,338 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LN868943.1, seqID NZ_LN868943.1 [record index 0].\n",
      "2024-04-17 17:53:58,419 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041976.1, seqID NZ_CP041976.1 [record index 0].\n",
      "2024-04-17 17:53:58,500 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP068696.1, seqID NZ_CP068696.1 [record index 0].\n",
      "2024-04-17 17:53:58,578 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP015157.1, seqID NZ_CP015157.1 [record index 0].\n",
      "2024-04-17 17:53:58,672 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012930.1, seqID NZ_CP012930.1 [record index 0].\n",
      "2024-04-17 17:53:58,765 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085810.1, seqID NZ_CP085810.1 [record index 0].\n",
      "2024-04-17 17:53:58,864 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_OU943338.1, seqID NZ_OU943338.1 [record index 0].\n",
      "2024-04-17 17:53:58,963 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_OU943336.1, seqID NZ_OU943336.1 [record index 0].\n",
      "2024-04-17 17:53:59,064 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_OU943337.1, seqID NZ_OU943337.1 [record index 0].\n",
      "2024-04-17 17:53:59,165 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030936.1, seqID NZ_CP030936.1 [record index 0].\n",
      "2024-04-17 17:53:59,254 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905063.1, seqID NZ_LT905063.1 [record index 0].\n",
      "2024-04-17 17:53:59,326 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP064263.1, seqID NZ_CP064263.1 [record index 0].\n",
      "2024-04-17 17:53:59,389 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060507.1, seqID NZ_CP060507.1 [record index 0].\n",
      "2024-04-17 17:53:59,465 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP018219.1, seqID NZ_CP018219.1 [record index 0].\n",
      "2024-04-17 17:53:59,538 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045063.1, seqID NZ_CP045063.1 [record index 0].\n",
      "2024-04-17 17:53:59,612 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045059.1, seqID NZ_CP045059.1 [record index 0].\n",
      "2024-04-17 17:53:59,686 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045056.1, seqID NZ_CP045056.1 [record index 0].\n",
      "2024-04-17 17:53:59,763 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067397.1, seqID NZ_CP067397.1 [record index 0].\n",
      "2024-04-17 17:53:59,835 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023319.1, seqID NZ_AP023319.1 [record index 0].\n",
      "2024-04-17 17:53:59,904 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023317.1, seqID NZ_AP023317.1 [record index 0].\n",
      "2024-04-17 17:53:59,979 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023315.1, seqID NZ_AP023315.1 [record index 0].\n",
      "2024-04-17 17:54:00,053 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023313.1, seqID NZ_AP023313.1 [record index 0].\n",
      "2024-04-17 17:54:00,121 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023311.1, seqID NZ_AP023311.1 [record index 0].\n",
      "2024-04-17 17:54:00,194 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023309.1, seqID NZ_AP023309.1 [record index 0].\n",
      "2024-04-17 17:54:00,263 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023306.1, seqID NZ_AP023306.1 [record index 0].\n",
      "2024-04-17 17:54:00,338 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023304.1, seqID NZ_AP023304.1 [record index 0].\n",
      "2024-04-17 17:54:00,406 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023303.1, seqID NZ_AP023303.1 [record index 0].\n",
      "2024-04-17 17:54:00,474 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023300.1, seqID NZ_AP023300.1 [record index 0].\n",
      "2024-04-17 17:54:00,547 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023299.1, seqID NZ_AP023299.1 [record index 0].\n",
      "2024-04-17 17:54:00,614 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023294.1, seqID NZ_AP023294.1 [record index 0].\n",
      "2024-04-17 17:54:00,680 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023292.1, seqID NZ_AP023292.1 [record index 0].\n",
      "2024-04-17 17:54:00,747 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023291.1, seqID NZ_AP023291.1 [record index 0].\n",
      "2024-04-17 17:54:00,817 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023290.1, seqID NZ_AP023290.1 [record index 0].\n",
      "2024-04-17 17:54:00,889 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023289.1, seqID NZ_AP023289.1 [record index 0].\n",
      "2024-04-17 17:54:00,954 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP023288.1, seqID NZ_AP023288.1 [record index 0].\n",
      "2024-04-17 17:54:01,042 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP061159.1, seqID NZ_CP061159.1 [record index 0].\n",
      "2024-04-17 17:54:01,132 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP047424.1, seqID NZ_CP047424.1 [record index 0].\n",
      "2024-04-17 17:54:01,226 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP080960.1, seqID NZ_CP080960.1 [record index 0].\n",
      "2024-04-17 17:54:01,317 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060132.1, seqID NZ_CP060132.1 [record index 0].\n",
      "2024-04-17 17:54:01,415 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060134.1, seqID NZ_CP060134.1 [record index 0].\n",
      "2024-04-17 17:54:01,485 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050130.1, seqID NZ_CP050130.1 [record index 0].\n",
      "2024-04-17 17:54:01,580 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043509.1, seqID NZ_CP043509.1 [record index 0].\n",
      "2024-04-17 17:54:01,651 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025094.1, seqID NZ_CP025094.1 [record index 0].\n",
      "2024-04-17 17:54:01,716 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025736.1, seqID NZ_CP025736.1 [record index 0].\n",
      "2024-04-17 17:54:01,789 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP015574.1, seqID NZ_CP015574.1 [record index 0].\n",
      "2024-04-17 17:54:01,874 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029029.1, seqID NZ_CP029029.1 [record index 0].\n",
      "2024-04-17 17:54:01,972 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020565.1, seqID NZ_CP020565.1 [record index 0].\n",
      "2024-04-17 17:54:02,062 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030029.1, seqID NZ_CP030029.1 [record index 0].\n",
      "2024-04-17 17:54:02,139 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014051.2, seqID NZ_CP014051.2 [record index 0].\n",
      "2024-04-17 17:54:02,208 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP065639.1, seqID NZ_CP065639.1 [record index 0].\n",
      "2024-04-17 17:54:02,274 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP065718.1, seqID NZ_CP065718.1 [record index 0].\n",
      "2024-04-17 17:54:02,340 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041005.1, seqID NZ_CP041005.1 [record index 0].\n",
      "2024-04-17 17:54:02,421 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046277.1, seqID NZ_CP046277.1 [record index 0].\n",
      "2024-04-17 17:54:02,507 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046279.1, seqID NZ_CP046279.1 [record index 0].\n",
      "2024-04-17 17:54:02,580 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP046283.1, seqID NZ_CP046283.1 [record index 0].\n",
      "2024-04-17 17:54:02,664 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026976.1, seqID NZ_CP026976.1 [record index 0].\n",
      "2024-04-17 17:54:02,734 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP023512.1, seqID NZ_CP023512.1 [record index 0].\n",
      "2024-04-17 17:54:02,815 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023508.1, seqID NZ_CP023508.1 [record index 0].\n",
      "2024-04-17 17:54:02,887 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022070.1, seqID NZ_CP022070.1 [record index 0].\n",
      "2024-04-17 17:54:02,958 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027414.1, seqID NZ_CP027414.1 [record index 0].\n",
      "2024-04-17 17:54:03,052 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP027412.1, seqID NZ_CP027412.1 [record index 0].\n",
      "2024-04-17 17:54:03,126 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027410.1, seqID NZ_CP027410.1 [record index 0].\n",
      "2024-04-17 17:54:03,194 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022062.1, seqID NZ_CP022062.1 [record index 0].\n",
      "2024-04-17 17:54:03,285 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069518.1, seqID NZ_CP069518.1 [record index 0].\n",
      "2024-04-17 17:54:03,357 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP066009.1, seqID NZ_CP066009.1 [record index 0].\n",
      "2024-04-17 17:54:03,449 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040379.1, seqID NZ_CP040379.1 [record index 0].\n",
      "2024-04-17 17:54:03,525 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP063684.2, seqID NZ_CP063684.2 [record index 0].\n",
      "2024-04-17 17:54:03,613 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905140.1, seqID NZ_LT905140.1 [record index 0].\n",
      "2024-04-17 17:54:03,689 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067091.1, seqID NZ_CP067091.1 [record index 0].\n",
      "2024-04-17 17:54:03,782 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085808.1, seqID NZ_CP085808.1 [record index 0].\n",
      "2024-04-17 17:54:03,868 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905143.1, seqID NZ_LT905143.1 [record index 0].\n",
      "2024-04-17 17:54:03,941 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_022544.1, seqID NC_022544.1 [record index 0].\n",
      "2024-04-17 17:54:04,016 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029568.1, seqID NZ_CP029568.1 [record index 0].\n",
      "2024-04-17 17:54:04,087 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029595.1, seqID NZ_CP029595.1 [record index 0].\n",
      "2024-04-17 17:54:04,152 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029593.1, seqID NZ_CP029593.1 [record index 0].\n",
      "2024-04-17 17:54:04,230 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029567.1, seqID NZ_CP029567.1 [record index 0].\n",
      "2024-04-17 17:54:04,301 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_016854.1, seqID NC_016854.1 [record index 0].\n",
      "2024-04-17 17:54:04,397 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074098.1, seqID NZ_CP074098.1 [record index 0].\n",
      "2024-04-17 17:54:04,466 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP009561.1, seqID NZ_CP009561.1 [record index 0].\n",
      "2024-04-17 17:54:04,539 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010284.1, seqID NZ_CP010284.1 [record index 0].\n",
      "2024-04-17 17:54:04,634 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051307.1, seqID NZ_CP051307.1 [record index 0].\n",
      "2024-04-17 17:54:04,725 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051310.1, seqID NZ_CP051310.1 [record index 0].\n",
      "2024-04-17 17:54:04,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051313.1, seqID NZ_CP051313.1 [record index 0].\n",
      "2024-04-17 17:54:04,884 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051317.1, seqID NZ_CP051317.1 [record index 0].\n",
      "2024-04-17 17:54:04,972 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051329.1, seqID NZ_CP051329.1 [record index 0].\n",
      "2024-04-17 17:54:05,046 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051338.1, seqID NZ_CP051338.1 [record index 0].\n",
      "2024-04-17 17:54:05,137 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051346.1, seqID NZ_CP051346.1 [record index 0].\n",
      "2024-04-17 17:54:05,223 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051350.1, seqID NZ_CP051350.1 [record index 0].\n",
      "2024-04-17 17:54:05,318 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053048.1, seqID NZ_CP053048.1 [record index 0].\n",
      "2024-04-17 17:54:05,391 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051353.1, seqID NZ_CP051353.1 [record index 0].\n",
      "2024-04-17 17:54:05,485 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051358.1, seqID NZ_CP051358.1 [record index 0].\n",
      "2024-04-17 17:54:05,565 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051363.1, seqID NZ_CP051363.1 [record index 0].\n",
      "2024-04-17 17:54:05,642 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053051.1, seqID NZ_CP053051.1 [record index 0].\n",
      "2024-04-17 17:54:05,731 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051366.1, seqID NZ_CP051366.1 [record index 0].\n",
      "2024-04-17 17:54:05,800 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051372.1, seqID NZ_CP051372.1 [record index 0].\n",
      "2024-04-17 17:54:05,872 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051380.1, seqID NZ_CP051380.1 [record index 0].\n",
      "2024-04-17 17:54:05,963 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051381.1, seqID NZ_CP051381.1 [record index 0].\n",
      "2024-04-17 17:54:06,056 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051384.1, seqID NZ_CP051384.1 [record index 0].\n",
      "2024-04-17 17:54:06,128 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051386.1, seqID NZ_CP051386.1 [record index 0].\n",
      "2024-04-17 17:54:06,214 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053055.1, seqID NZ_CP053055.1 [record index 0].\n",
      "2024-04-17 17:54:06,288 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010281.1, seqID NZ_CP010281.1 [record index 0].\n",
      "2024-04-17 17:54:06,360 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010280.1, seqID NZ_CP010280.1 [record index 0].\n",
      "2024-04-17 17:54:06,426 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010279.1, seqID NZ_CP010279.1 [record index 0].\n",
      "2024-04-17 17:54:06,501 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010283.1, seqID NZ_CP010283.1 [record index 0].\n",
      "2024-04-17 17:54:06,578 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP010282.1, seqID NZ_CP010282.1 [record index 0].\n",
      "2024-04-17 17:54:06,650 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051404.1, seqID NZ_CP051404.1 [record index 0].\n",
      "2024-04-17 17:54:06,736 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051410.1, seqID NZ_CP051410.1 [record index 0].\n",
      "2024-04-17 17:54:06,834 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051413.1, seqID NZ_CP051413.1 [record index 0].\n",
      "2024-04-17 17:54:06,930 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011094.1, seqID NC_011094.1 [record index 0].\n",
      "2024-04-17 17:54:07,027 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_003198.1, seqID NC_003198.1 [record index 0].\n",
      "2024-04-17 17:54:07,113 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009559.1, seqID NZ_CP009559.1 [record index 0].\n",
      "2024-04-17 17:54:07,208 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033344.1, seqID NZ_CP033344.1 [record index 0].\n",
      "2024-04-17 17:54:07,308 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033345.1, seqID NZ_CP033345.1 [record index 0].\n",
      "2024-04-17 17:54:07,395 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024168.1, seqID NZ_CP024168.1 [record index 0].\n",
      "2024-04-17 17:54:07,484 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028311.1, seqID NZ_CP028311.1 [record index 0].\n",
      "2024-04-17 17:54:07,553 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028314.1, seqID NZ_CP028314.1 [record index 0].\n",
      "2024-04-17 17:54:07,624 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028318.1, seqID NZ_CP028318.1 [record index 0].\n",
      "2024-04-17 17:54:07,712 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028169.1, seqID NZ_CP028169.1 [record index 0].\n",
      "2024-04-17 17:54:07,797 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075107.1, seqID NZ_CP075107.1 [record index 0].\n",
      "2024-04-17 17:54:07,862 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075108.1, seqID NZ_CP075108.1 [record index 0].\n",
      "2024-04-17 17:54:07,947 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075110.1, seqID NZ_CP075110.1 [record index 0].\n",
      "2024-04-17 17:54:08,020 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077691.1, seqID NZ_CP077691.1 [record index 0].\n",
      "2024-04-17 17:54:08,090 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077693.1, seqID NZ_CP077693.1 [record index 0].\n",
      "2024-04-17 17:54:08,184 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077696.1, seqID NZ_CP077696.1 [record index 0].\n",
      "2024-04-17 17:54:08,271 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077702.1, seqID NZ_CP077702.1 [record index 0].\n",
      "2024-04-17 17:54:08,362 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077704.1, seqID NZ_CP077704.1 [record index 0].\n",
      "2024-04-17 17:54:08,441 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029041.1, seqID NZ_CP029041.1 [record index 0].\n",
      "2024-04-17 17:54:08,542 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029040.1, seqID NZ_CP029040.1 [record index 0].\n",
      "2024-04-17 17:54:08,641 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029038.1, seqID NZ_CP029038.1 [record index 0].\n",
      "2024-04-17 17:54:08,709 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075127.1, seqID NZ_CP075127.1 [record index 0].\n",
      "2024-04-17 17:54:08,774 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075134.1, seqID NZ_CP075134.1 [record index 0].\n",
      "2024-04-17 17:54:08,860 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075137.1, seqID NZ_CP075137.1 [record index 0].\n",
      "2024-04-17 17:54:08,953 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075141.1, seqID NZ_CP075141.1 [record index 0].\n",
      "2024-04-17 17:54:09,056 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019174.1, seqID NZ_CP019174.1 [record index 0].\n",
      "2024-04-17 17:54:09,143 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028196.1, seqID NZ_CP028196.1 [record index 0].\n",
      "2024-04-17 17:54:09,214 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028199.1, seqID NZ_CP028199.1 [record index 0].\n",
      "2024-04-17 17:54:09,295 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075037.1, seqID NZ_CP075037.1 [record index 0].\n",
      "2024-04-17 17:54:09,365 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP074663.1, seqID NZ_CP074663.1 [record index 0].\n",
      "2024-04-17 17:54:09,459 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019172.1, seqID NZ_CP019172.1 [record index 0].\n",
      "2024-04-17 17:54:09,562 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019206.1, seqID NZ_CP019206.1 [record index 0].\n",
      "2024-04-17 17:54:09,663 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019204.1, seqID NZ_CP019204.1 [record index 0].\n",
      "2024-04-17 17:54:09,757 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039270.1, seqID NZ_CP039270.1 [record index 0].\n",
      "2024-04-17 17:54:09,835 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016012.1, seqID NZ_CP016012.1 [record index 0].\n",
      "2024-04-17 17:54:09,926 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_021812.2, seqID NC_021812.2 [record index 0].\n",
      "2024-04-17 17:54:10,025 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP005995.1, seqID NZ_CP005995.1 [record index 0].\n",
      "2024-04-17 17:54:10,120 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_021818.1, seqID NC_021818.1 [record index 0].\n",
      "2024-04-17 17:54:10,207 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_020307.1, seqID NC_020307.1 [record index 0].\n",
      "2024-04-17 17:54:10,275 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_021814.1, seqID NC_021814.1 [record index 0].\n",
      "2024-04-17 17:54:10,367 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012833.1, seqID NZ_CP012833.1 [record index 0].\n",
      "2024-04-17 17:54:10,467 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP014994.1, seqID NZ_CP014994.1 [record index 0].\n",
      "2024-04-17 17:54:10,540 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039463.1, seqID NZ_CP039463.1 [record index 0].\n",
      "2024-04-17 17:54:10,611 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039465.1, seqID NZ_CP039465.1 [record index 0].\n",
      "2024-04-17 17:54:10,688 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039467.1, seqID NZ_CP039467.1 [record index 0].\n",
      "2024-04-17 17:54:10,764 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039469.1, seqID NZ_CP039469.1 [record index 0].\n",
      "2024-04-17 17:54:10,851 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007533.1, seqID NZ_CP007533.1 [record index 0].\n",
      "2024-04-17 17:54:10,920 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039471.1, seqID NZ_CP039471.1 [record index 0].\n",
      "2024-04-17 17:54:10,982 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039473.1, seqID NZ_CP039473.1 [record index 0].\n",
      "2024-04-17 17:54:11,048 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039476.1, seqID NZ_CP039476.1 [record index 0].\n",
      "2024-04-17 17:54:11,118 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039477.1, seqID NZ_CP039477.1 [record index 0].\n",
      "2024-04-17 17:54:11,181 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039478.1, seqID NZ_CP039478.1 [record index 0].\n",
      "2024-04-17 17:54:11,244 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039480.1, seqID NZ_CP039480.1 [record index 0].\n",
      "2024-04-17 17:54:11,309 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039482.1, seqID NZ_CP039482.1 [record index 0].\n",
      "2024-04-17 17:54:11,372 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039484.1, seqID NZ_CP039484.1 [record index 0].\n",
      "2024-04-17 17:54:11,434 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039486.1, seqID NZ_CP039486.1 [record index 0].\n",
      "2024-04-17 17:54:11,499 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039487.1, seqID NZ_CP039487.1 [record index 0].\n",
      "2024-04-17 17:54:11,564 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045754.1, seqID NZ_CP045754.1 [record index 0].\n",
      "2024-04-17 17:54:11,628 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039489.1, seqID NZ_CP039489.1 [record index 0].\n",
      "2024-04-17 17:54:11,693 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039491.1, seqID NZ_CP039491.1 [record index 0].\n",
      "2024-04-17 17:54:11,760 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045757.1, seqID NZ_CP045757.1 [record index 0].\n",
      "2024-04-17 17:54:11,825 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039493.1, seqID NZ_CP039493.1 [record index 0].\n",
      "2024-04-17 17:54:11,892 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045759.1, seqID NZ_CP045759.1 [record index 0].\n",
      "2024-04-17 17:54:11,955 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039496.1, seqID NZ_CP039496.1 [record index 0].\n",
      "2024-04-17 17:54:12,032 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039498.1, seqID NZ_CP039498.1 [record index 0].\n",
      "2024-04-17 17:54:12,109 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039499.1, seqID NZ_CP039499.1 [record index 0].\n",
      "2024-04-17 17:54:12,186 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039500.1, seqID NZ_CP039500.1 [record index 0].\n",
      "2024-04-17 17:54:12,256 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032622.2, seqID NZ_CP032622.2 [record index 0].\n",
      "2024-04-17 17:54:12,332 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034177.1, seqID NZ_CP034177.1 [record index 0].\n",
      "2024-04-17 17:54:12,409 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045753.1, seqID NZ_CP045753.1 [record index 0].\n",
      "2024-04-17 17:54:12,478 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_021844.1, seqID NC_021844.1 [record index 0].\n",
      "2024-04-17 17:54:12,546 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033352.2, seqID NZ_CP033352.2 [record index 0].\n",
      "2024-04-17 17:54:12,618 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033255.2, seqID NZ_CP033255.2 [record index 0].\n",
      "2024-04-17 17:54:12,695 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033226.2, seqID NZ_CP033226.2 [record index 0].\n",
      "2024-04-17 17:54:12,769 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033257.2, seqID NZ_CP033257.2 [record index 0].\n",
      "2024-04-17 17:54:12,862 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033348.2, seqID NZ_CP033348.2 [record index 0].\n",
      "2024-04-17 17:54:12,959 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016517.1, seqID NZ_CP016517.1 [record index 0].\n",
      "2024-04-17 17:54:13,040 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014979.2, seqID NZ_CP014979.2 [record index 0].\n",
      "2024-04-17 17:54:13,135 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007540.2, seqID NZ_CP007540.2 [record index 0].\n",
      "2024-04-17 17:54:13,235 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017978.1, seqID NZ_CP017978.1 [record index 0].\n",
      "2024-04-17 17:54:13,306 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025243.1, seqID NZ_CP025243.1 [record index 0].\n",
      "2024-04-17 17:54:13,384 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014967.1, seqID NZ_CP014967.1 [record index 0].\n",
      "2024-04-17 17:54:13,481 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017976.1, seqID NZ_CP017976.1 [record index 0].\n",
      "2024-04-17 17:54:13,549 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007523.1, seqID NZ_CP007523.1 [record index 0].\n",
      "2024-04-17 17:54:13,613 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007559.2, seqID NZ_CP007559.2 [record index 0].\n",
      "2024-04-17 17:54:13,683 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014965.1, seqID NZ_CP014965.1 [record index 0].\n",
      "2024-04-17 17:54:13,751 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014983.1, seqID NZ_CP014983.1 [record index 0].\n",
      "2024-04-17 17:54:13,826 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014975.1, seqID NZ_CP014975.1 [record index 0].\n",
      "2024-04-17 17:54:13,921 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020752.1, seqID NZ_CP020752.1 [record index 0].\n",
      "2024-04-17 17:54:14,017 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017975.1, seqID NZ_CP017975.1 [record index 0].\n",
      "2024-04-17 17:54:14,114 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017974.1, seqID NZ_CP017974.1 [record index 0].\n",
      "2024-04-17 17:54:14,205 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007639.1, seqID NZ_CP007639.1 [record index 0].\n",
      "2024-04-17 17:54:14,274 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085819.1, seqID NZ_CP085819.1 [record index 0].\n",
      "2024-04-17 17:54:14,370 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024619.1, seqID NZ_CP024619.1 [record index 0].\n",
      "2024-04-17 17:54:14,439 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060169.1, seqID NZ_CP060169.1 [record index 0].\n",
      "2024-04-17 17:54:14,503 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061044.1, seqID NZ_CP061044.1 [record index 0].\n",
      "2024-04-17 17:54:14,569 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061045.1, seqID NZ_CP061045.1 [record index 0].\n",
      "2024-04-17 17:54:14,636 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061047.1, seqID NZ_CP061047.1 [record index 0].\n",
      "2024-04-17 17:54:14,701 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061050.1, seqID NZ_CP061050.1 [record index 0].\n",
      "2024-04-17 17:54:14,766 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP061049.1, seqID NZ_CP061049.1 [record index 0].\n",
      "2024-04-17 17:54:14,857 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023470.1, seqID NZ_CP023470.1 [record index 0].\n",
      "2024-04-17 17:54:14,953 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023468.1, seqID NZ_CP023468.1 [record index 0].\n",
      "2024-04-17 17:54:15,031 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034968.1, seqID NZ_CP034968.1 [record index 0].\n",
      "2024-04-17 17:54:15,133 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_017623.1, seqID NC_017623.1 [record index 0].\n",
      "2024-04-17 17:54:15,243 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074094.1, seqID NZ_CP074094.1 [record index 0].\n",
      "2024-04-17 17:54:15,320 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_OU015328.1, seqID NZ_OU015328.1 [record index 0].\n",
      "2024-04-17 17:54:15,399 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_OU015340.1, seqID NZ_OU015340.1 [record index 0].\n",
      "2024-04-17 17:54:15,466 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045945.1, seqID NZ_CP045945.1 [record index 0].\n",
      "2024-04-17 17:54:15,553 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045954.1, seqID NZ_CP045954.1 [record index 0].\n",
      "2024-04-17 17:54:15,624 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045947.1, seqID NZ_CP045947.1 [record index 0].\n",
      "2024-04-17 17:54:15,691 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045949.1, seqID NZ_CP045949.1 [record index 0].\n",
      "2024-04-17 17:54:15,761 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045952.1, seqID NZ_CP045952.1 [record index 0].\n",
      "2024-04-17 17:54:15,839 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_OU015342.1, seqID NZ_OU015342.1 [record index 0].\n",
      "2024-04-17 17:54:15,932 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045831.1, seqID NZ_CP045831.1 [record index 0].\n",
      "2024-04-17 17:54:16,015 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_OU015323.1, seqID NZ_OU015323.1 [record index 0].\n",
      "2024-04-17 17:54:16,100 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_006511.1, seqID NC_006511.1 [record index 0].\n",
      "2024-04-17 17:54:16,189 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034232.1, seqID NZ_CP034232.1 [record index 0].\n",
      "2024-04-17 17:54:16,293 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019183.1, seqID NZ_CP019183.1 [record index 0].\n",
      "2024-04-17 17:54:16,373 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043907.1, seqID NZ_CP043907.1 [record index 0].\n",
      "2024-04-17 17:54:16,445 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034230.1, seqID NZ_CP034230.1 [record index 0].\n",
      "2024-04-17 17:54:16,524 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP009102.1, seqID NZ_CP009102.1 [record index 0].\n",
      "2024-04-17 17:54:16,622 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019178.1, seqID NZ_CP019178.1 [record index 0].\n",
      "2024-04-17 17:54:16,720 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019185.1, seqID NZ_CP019185.1 [record index 0].\n",
      "2024-04-17 17:54:16,808 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025218.1, seqID NZ_CP025218.1 [record index 0].\n",
      "2024-04-17 17:54:16,901 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012344.2, seqID NZ_CP012344.2 [record index 0].\n",
      "2024-04-17 17:54:17,004 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032194.1, seqID NZ_CP032194.1 [record index 0].\n",
      "2024-04-17 17:54:17,074 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP026700.1, seqID NZ_CP026700.1 [record index 0].\n",
      "2024-04-17 17:54:17,157 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085812.1, seqID NZ_CP085812.1 [record index 0].\n",
      "2024-04-17 17:54:17,254 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044177.1, seqID NZ_CP044177.1 [record index 0].\n",
      "2024-04-17 17:54:17,342 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044203.1, seqID NZ_CP044203.1 [record index 0].\n",
      "2024-04-17 17:54:17,440 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044181.1, seqID NZ_CP044181.1 [record index 0].\n",
      "2024-04-17 17:54:17,528 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044184.1, seqID NZ_CP044184.1 [record index 0].\n",
      "2024-04-17 17:54:17,617 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP044186.1, seqID NZ_CP044186.1 [record index 0].\n",
      "2024-04-17 17:54:17,687 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044188.1, seqID NZ_CP044188.1 [record index 0].\n",
      "2024-04-17 17:54:17,782 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016576.1, seqID NZ_CP016576.1 [record index 0].\n",
      "2024-04-17 17:54:17,880 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016573.1, seqID NZ_CP016573.1 [record index 0].\n",
      "2024-04-17 17:54:17,979 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016569.1, seqID NZ_CP016569.1 [record index 0].\n",
      "2024-04-17 17:54:18,076 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016565.1, seqID NZ_CP016565.1 [record index 0].\n",
      "2024-04-17 17:54:18,162 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011147.1, seqID NC_011147.1 [record index 0].\n",
      "2024-04-17 17:54:18,239 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP064919.1, seqID NZ_CP064919.1 [record index 0].\n",
      "2024-04-17 17:54:18,335 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016563.1, seqID NZ_CP016563.1 [record index 0].\n",
      "2024-04-17 17:54:18,433 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016561.1, seqID NZ_CP016561.1 [record index 0].\n",
      "2024-04-17 17:54:18,520 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053581.1, seqID NZ_CP053581.1 [record index 0].\n",
      "2024-04-17 17:54:18,607 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045444.1, seqID NZ_CP045444.1 [record index 0].\n",
      "2024-04-17 17:54:18,684 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP018657.1, seqID NZ_CP018657.1 [record index 0].\n",
      "2024-04-17 17:54:18,768 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP008925.1, seqID NZ_CP008925.1 [record index 0].\n",
      "2024-04-17 17:54:18,842 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019442.1, seqID NZ_CP019442.1 [record index 0].\n",
      "2024-04-17 17:54:18,914 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_017046.1, seqID NC_017046.1 [record index 0].\n",
      "2024-04-17 17:54:18,985 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039509.1, seqID NZ_CP039509.1 [record index 0].\n",
      "2024-04-17 17:54:19,058 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039503.1, seqID NZ_CP039503.1 [record index 0].\n",
      "2024-04-17 17:54:19,148 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP067082.1, seqID NZ_CP067082.1 [record index 0].\n",
      "2024-04-17 17:54:19,219 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR862421.1, seqID NZ_LR862421.1 [record index 0].\n",
      "2024-04-17 17:54:19,296 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR861808.1, seqID NZ_LR861808.1 [record index 0].\n",
      "2024-04-17 17:54:19,390 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009049.1, seqID NZ_CP009049.1 [record index 0].\n",
      "2024-04-17 17:54:19,491 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040700.1, seqID NZ_CP040700.1 [record index 0].\n",
      "2024-04-17 17:54:19,593 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031359.1, seqID NZ_CP031359.1 [record index 0].\n",
      "2024-04-17 17:54:19,682 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT882486.1, seqID NZ_LT882486.1 [record index 0].\n",
      "2024-04-17 17:54:19,797 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR590082.1, seqID NZ_LR590082.1 [record index 0].\n",
      "2024-04-17 17:54:19,907 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LR590081.1, seqID NZ_LR590081.1 [record index 0].\n",
      "2024-04-17 17:54:20,006 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037893.1, seqID NZ_CP037893.1 [record index 0].\n",
      "2024-04-17 17:54:20,097 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045762.1, seqID NZ_CP045762.1 [record index 0].\n",
      "2024-04-17 17:54:20,187 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_021810.1, seqID NC_021810.1 [record index 0].\n",
      "2024-04-17 17:54:20,275 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905062.1, seqID NZ_LT905062.1 [record index 0].\n",
      "2024-04-17 17:54:20,372 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029036.1, seqID NZ_CP029036.1 [record index 0].\n",
      "2024-04-17 17:54:20,474 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029846.1, seqID NZ_CP029846.1 [record index 0].\n",
      "2024-04-17 17:54:20,574 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029959.1, seqID NZ_CP029959.1 [record index 0].\n",
      "2024-04-17 17:54:20,677 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029960.1, seqID NZ_CP029960.1 [record index 0].\n",
      "2024-04-17 17:54:20,777 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029962.1, seqID NZ_CP029962.1 [record index 0].\n",
      "2024-04-17 17:54:20,879 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029964.1, seqID NZ_CP029964.1 [record index 0].\n",
      "2024-04-17 17:54:20,980 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029848.1, seqID NZ_CP029848.1 [record index 0].\n",
      "2024-04-17 17:54:21,080 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029850.1, seqID NZ_CP029850.1 [record index 0].\n",
      "2024-04-17 17:54:21,179 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029852.1, seqID NZ_CP029852.1 [record index 0].\n",
      "2024-04-17 17:54:21,280 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029855.1, seqID NZ_CP029855.1 [record index 0].\n",
      "2024-04-17 17:54:21,381 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029856.1, seqID NZ_CP029856.1 [record index 0].\n",
      "2024-04-17 17:54:21,482 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029858.1, seqID NZ_CP029858.1 [record index 0].\n",
      "2024-04-17 17:54:21,582 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029853.1, seqID NZ_CP029853.1 [record index 0].\n",
      "2024-04-17 17:54:21,681 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029863.1, seqID NZ_CP029863.1 [record index 0].\n",
      "2024-04-17 17:54:21,786 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029906.1, seqID NZ_CP029906.1 [record index 0].\n",
      "2024-04-17 17:54:21,887 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029861.1, seqID NZ_CP029861.1 [record index 0].\n",
      "2024-04-17 17:54:21,986 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029907.1, seqID NZ_CP029907.1 [record index 0].\n",
      "2024-04-17 17:54:22,087 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029864.1, seqID NZ_CP029864.1 [record index 0].\n",
      "2024-04-17 17:54:22,188 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029866.1, seqID NZ_CP029866.1 [record index 0].\n",
      "2024-04-17 17:54:22,287 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029868.1, seqID NZ_CP029868.1 [record index 0].\n",
      "2024-04-17 17:54:22,390 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029862.1, seqID NZ_CP029862.1 [record index 0].\n",
      "2024-04-17 17:54:22,491 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029872.1, seqID NZ_CP029872.1 [record index 0].\n",
      "2024-04-17 17:54:22,595 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029897.1, seqID NZ_CP029897.1 [record index 0].\n",
      "2024-04-17 17:54:22,696 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029870.1, seqID NZ_CP029870.1 [record index 0].\n",
      "2024-04-17 17:54:22,794 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029919.1, seqID NZ_CP029919.1 [record index 0].\n",
      "2024-04-17 17:54:22,895 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029873.1, seqID NZ_CP029873.1 [record index 0].\n",
      "2024-04-17 17:54:22,995 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029888.1, seqID NZ_CP029888.1 [record index 0].\n",
      "2024-04-17 17:54:23,094 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029881.1, seqID NZ_CP029881.1 [record index 0].\n",
      "2024-04-17 17:54:23,193 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029890.1, seqID NZ_CP029890.1 [record index 0].\n",
      "2024-04-17 17:54:23,297 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029892.1, seqID NZ_CP029892.1 [record index 0].\n",
      "2024-04-17 17:54:23,400 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029913.1, seqID NZ_CP029913.1 [record index 0].\n",
      "2024-04-17 17:54:23,501 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029882.1, seqID NZ_CP029882.1 [record index 0].\n",
      "2024-04-17 17:54:23,602 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029899.1, seqID NZ_CP029899.1 [record index 0].\n",
      "2024-04-17 17:54:23,702 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029900.1, seqID NZ_CP029900.1 [record index 0].\n",
      "2024-04-17 17:54:23,801 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029875.1, seqID NZ_CP029875.1 [record index 0].\n",
      "2024-04-17 17:54:23,901 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029914.1, seqID NZ_CP029914.1 [record index 0].\n",
      "2024-04-17 17:54:24,002 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029915.1, seqID NZ_CP029915.1 [record index 0].\n",
      "2024-04-17 17:54:24,080 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012681.1, seqID NZ_CP012681.1 [record index 0].\n",
      "2024-04-17 17:54:24,180 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029894.1, seqID NZ_CP029894.1 [record index 0].\n",
      "2024-04-17 17:54:24,281 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029920.1, seqID NZ_CP029920.1 [record index 0].\n",
      "2024-04-17 17:54:24,379 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029922.1, seqID NZ_CP029922.1 [record index 0].\n",
      "2024-04-17 17:54:24,480 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029883.1, seqID NZ_CP029883.1 [record index 0].\n",
      "2024-04-17 17:54:24,587 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029902.1, seqID NZ_CP029902.1 [record index 0].\n",
      "2024-04-17 17:54:24,684 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029917.1, seqID NZ_CP029917.1 [record index 0].\n",
      "2024-04-17 17:54:24,785 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029885.1, seqID NZ_CP029885.1 [record index 0].\n",
      "2024-04-17 17:54:24,887 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029896.1, seqID NZ_CP029896.1 [record index 0].\n",
      "2024-04-17 17:54:24,992 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029904.1, seqID NZ_CP029904.1 [record index 0].\n",
      "2024-04-17 17:54:25,089 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029878.1, seqID NZ_CP029878.1 [record index 0].\n",
      "2024-04-17 17:54:25,189 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029880.1, seqID NZ_CP029880.1 [record index 0].\n",
      "2024-04-17 17:54:25,287 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029909.1, seqID NZ_CP029909.1 [record index 0].\n",
      "2024-04-17 17:54:25,387 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029923.1, seqID NZ_CP029923.1 [record index 0].\n",
      "2024-04-17 17:54:25,488 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029886.1, seqID NZ_CP029886.1 [record index 0].\n",
      "2024-04-17 17:54:25,594 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029911.1, seqID NZ_CP029911.1 [record index 0].\n",
      "2024-04-17 17:54:25,695 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029925.1, seqID NZ_CP029925.1 [record index 0].\n",
      "2024-04-17 17:54:25,813 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029646.1, seqID NZ_CP029646.1 [record index 0].\n",
      "2024-04-17 17:54:25,911 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029928.1, seqID NZ_CP029928.1 [record index 0].\n",
      "2024-04-17 17:54:26,013 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029930.1, seqID NZ_CP029930.1 [record index 0].\n",
      "2024-04-17 17:54:26,111 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029933.1, seqID NZ_CP029933.1 [record index 0].\n",
      "2024-04-17 17:54:26,212 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029936.1, seqID NZ_CP029936.1 [record index 0].\n",
      "2024-04-17 17:54:26,313 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029944.1, seqID NZ_CP029944.1 [record index 0].\n",
      "2024-04-17 17:54:26,415 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029945.1, seqID NZ_CP029945.1 [record index 0].\n",
      "2024-04-17 17:54:26,519 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029946.1, seqID NZ_CP029946.1 [record index 0].\n",
      "2024-04-17 17:54:26,615 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029952.1, seqID NZ_CP029952.1 [record index 0].\n",
      "2024-04-17 17:54:26,714 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029949.1, seqID NZ_CP029949.1 [record index 0].\n",
      "2024-04-17 17:54:26,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029932.1, seqID NZ_CP029932.1 [record index 0].\n",
      "2024-04-17 17:54:26,913 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029938.1, seqID NZ_CP029938.1 [record index 0].\n",
      "2024-04-17 17:54:27,016 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029940.1, seqID NZ_CP029940.1 [record index 0].\n",
      "2024-04-17 17:54:27,120 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029950.1, seqID NZ_CP029950.1 [record index 0].\n",
      "2024-04-17 17:54:27,221 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029954.1, seqID NZ_CP029954.1 [record index 0].\n",
      "2024-04-17 17:54:27,323 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029942.1, seqID NZ_CP029942.1 [record index 0].\n",
      "2024-04-17 17:54:27,425 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029956.1, seqID NZ_CP029956.1 [record index 0].\n",
      "2024-04-17 17:54:27,525 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029958.1, seqID NZ_CP029958.1 [record index 0].\n",
      "2024-04-17 17:54:27,605 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040701.1, seqID NZ_CP040701.1 [record index 0].\n",
      "2024-04-17 17:54:27,705 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074096.1, seqID NZ_CP074096.1 [record index 0].\n",
      "2024-04-17 17:54:27,779 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN890524.1, seqID NZ_LN890524.1 [record index 0].\n",
      "2024-04-17 17:54:27,845 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN890522.1, seqID NZ_LN890522.1 [record index 0].\n",
      "2024-04-17 17:54:27,914 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN890520.1, seqID NZ_LN890520.1 [record index 0].\n",
      "2024-04-17 17:54:27,989 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN890518.1, seqID NZ_LN890518.1 [record index 0].\n",
      "2024-04-17 17:54:28,047 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP053582.1, seqID NZ_CP053582.1 [record index 0].\n",
      "2024-04-17 17:54:28,119 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017621.1, seqID NZ_CP017621.1 [record index 0].\n",
      "2024-04-17 17:54:28,217 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP091878.1, seqID NZ_CP091878.1 [record index 0].\n",
      "2024-04-17 17:54:28,291 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090545.1, seqID NZ_CP090545.1 [record index 0].\n",
      "2024-04-17 17:54:28,367 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090533.1, seqID NZ_CP090533.1 [record index 0].\n",
      "2024-04-17 17:54:28,441 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090535.1, seqID NZ_CP090535.1 [record index 0].\n",
      "2024-04-17 17:54:28,516 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090539.1, seqID NZ_CP090539.1 [record index 0].\n",
      "2024-04-17 17:54:28,607 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053585.1, seqID NZ_CP053585.1 [record index 0].\n",
      "2024-04-17 17:54:28,679 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090541.1, seqID NZ_CP090541.1 [record index 0].\n",
      "2024-04-17 17:54:28,762 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905139.1, seqID NZ_LT905139.1 [record index 0].\n",
      "2024-04-17 17:54:28,835 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090529.1, seqID NZ_CP090529.1 [record index 0].\n",
      "2024-04-17 17:54:28,909 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP088901.1, seqID NZ_CP088901.1 [record index 0].\n",
      "2024-04-17 17:54:28,981 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP020922.1, seqID NZ_CP020922.1 [record index 0].\n",
      "2024-04-17 17:54:29,059 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014996.1, seqID NZ_CP014996.1 [record index 0].\n",
      "2024-04-17 17:54:29,153 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043667.1, seqID NZ_CP043667.1 [record index 0].\n",
      "2024-04-17 17:54:29,245 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043664.1, seqID NZ_CP043664.1 [record index 0].\n",
      "2024-04-17 17:54:29,344 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063140.1, seqID NZ_CP063140.1 [record index 0].\n",
      "2024-04-17 17:54:29,420 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043662.1, seqID NZ_CP043662.1 [record index 0].\n",
      "2024-04-17 17:54:29,510 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LT905064.1, seqID NZ_LT905064.1 [record index 0].\n",
      "2024-04-17 17:54:29,599 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019408.1, seqID NZ_CP019408.1 [record index 0].\n",
      "2024-04-17 17:54:29,666 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043400.1, seqID NZ_CP043400.1 [record index 0].\n",
      "2024-04-17 17:54:29,725 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043399.1, seqID NZ_CP043399.1 [record index 0].\n",
      "2024-04-17 17:54:29,789 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043401.1, seqID NZ_CP043401.1 [record index 0].\n",
      "2024-04-17 17:54:29,851 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043402.1, seqID NZ_CP043402.1 [record index 0].\n",
      "2024-04-17 17:54:29,925 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_016856.1, seqID NC_016856.1 [record index 0].\n",
      "2024-04-17 17:54:29,995 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034479.1, seqID NZ_CP034479.1 [record index 0].\n",
      "2024-04-17 17:54:30,063 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007581.1, seqID NZ_CP007581.1 [record index 0].\n",
      "2024-04-17 17:54:30,145 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045447.1, seqID NZ_CP045447.1 [record index 0].\n",
      "2024-04-17 17:54:30,238 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012924.1, seqID NZ_CP012924.1 [record index 0].\n",
      "2024-04-17 17:54:30,326 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039277.1, seqID NZ_CP039277.1 [record index 0].\n",
      "2024-04-17 17:54:30,412 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032816.1, seqID NZ_CP032816.1 [record index 0].\n",
      "2024-04-17 17:54:30,502 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038601.1, seqID NZ_CP038601.1 [record index 0].\n",
      "2024-04-17 17:54:30,608 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016514.1, seqID NZ_CP016514.1 [record index 0].\n",
      "2024-04-17 17:54:30,679 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029839.1, seqID NZ_CP029839.1 [record index 0].\n",
      "2024-04-17 17:54:30,775 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016525.1, seqID NZ_CP016525.1 [record index 0].\n",
      "2024-04-17 17:54:30,846 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_021820.1, seqID NC_021820.1 [record index 0].\n",
      "2024-04-17 17:54:30,907 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012599.1, seqID NZ_CP012599.1 [record index 0].\n",
      "2024-04-17 17:54:30,998 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019197.1, seqID NZ_CP019197.1 [record index 0].\n",
      "2024-04-17 17:54:31,077 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013685.1, seqID NZ_CP013685.1 [record index 0].\n",
      "2024-04-17 17:54:31,154 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067066.1, seqID NZ_CP067066.1 [record index 0].\n",
      "2024-04-17 17:54:31,228 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030874.1, seqID NZ_CP030874.1 [record index 0].\n",
      "2024-04-17 17:54:31,308 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040183.1, seqID NZ_CP040183.1 [record index 0].\n",
      "2024-04-17 17:54:31,393 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013338.1, seqID NZ_CP013338.1 [record index 0].\n",
      "2024-04-17 17:54:31,470 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP076605.1, seqID NZ_CP076605.1 [record index 0].\n",
      "2024-04-17 17:54:31,548 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022268.1, seqID NZ_AP022268.1 [record index 0].\n",
      "2024-04-17 17:54:31,628 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP038281.1, seqID NZ_CP038281.1 [record index 0].\n",
      "2024-04-17 17:54:31,716 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP055216.1, seqID NZ_CP055216.1 [record index 0].\n",
      "2024-04-17 17:54:31,795 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP021329.1, seqID NZ_CP021329.1 [record index 0].\n",
      "2024-04-17 17:54:31,870 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP079752.1, seqID NZ_CP079752.1 [record index 0].\n",
      "2024-04-17 17:54:31,956 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP025009.1, seqID NZ_AP025009.1 [record index 0].\n",
      "2024-04-17 17:54:32,038 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017802.2, seqID NZ_CP017802.2 [record index 0].\n",
      "2024-04-17 17:54:32,110 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP059047.1, seqID NZ_CP059047.1 [record index 0].\n",
      "2024-04-17 17:54:32,177 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_021066.1, seqID NC_021066.1 [record index 0].\n",
      "2024-04-17 17:54:32,251 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP008886.1, seqID NZ_CP008886.1 [record index 0].\n",
      "2024-04-17 17:54:32,331 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP048349.1, seqID NZ_CP048349.1 [record index 0].\n",
      "2024-04-17 17:54:32,406 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP046671.1, seqID NZ_CP046671.1 [record index 0].\n",
      "2024-04-17 17:54:32,460 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045783.1, seqID NZ_CP045783.1 [record index 0].\n",
      "2024-04-17 17:54:32,508 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063403.1, seqID NZ_CP063403.1 [record index 0].\n",
      "2024-04-17 17:54:32,591 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP056483.1, seqID NZ_CP056483.1 [record index 0].\n",
      "2024-04-17 17:54:32,645 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081893.1, seqID NZ_CP081893.1 [record index 0].\n",
      "2024-04-17 17:54:32,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068444.1, seqID NZ_CP068444.1 [record index 0].\n",
      "2024-04-17 17:54:32,718 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022412.1, seqID NZ_AP022412.1 [record index 0].\n",
      "2024-04-17 17:54:32,743 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063942.1, seqID NZ_CP063942.1 [record index 0].\n",
      "2024-04-17 17:54:32,770 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040861.1, seqID NZ_CP040861.1 [record index 0].\n",
      "2024-04-17 17:54:32,821 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076863.1, seqID NZ_CP076863.1 [record index 0].\n",
      "2024-04-17 17:54:32,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012560.1, seqID NZ_CP012560.1 [record index 0].\n",
      "2024-04-17 17:54:32,929 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012561.1, seqID NZ_CP012561.1 [record index 0].\n",
      "2024-04-17 17:54:32,987 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP012568.1, seqID NZ_CP012568.1 [record index 0].\n",
      "2024-04-17 17:54:33,038 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP024038.1, seqID NZ_CP024038.1 [record index 0].\n",
      "2024-04-17 17:54:33,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041639.1, seqID NZ_CP041639.1 [record index 0].\n",
      "2024-04-17 17:54:33,137 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR025088.1, seqID NZ_LR025088.1 [record index 0].\n",
      "2024-04-17 17:54:33,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR025092.1, seqID NZ_LR025092.1 [record index 0].\n",
      "2024-04-17 17:54:33,236 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041946.1, seqID NZ_CP041946.1 [record index 0].\n",
      "2024-04-17 17:54:33,290 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080362.1, seqID NZ_CP080362.1 [record index 0].\n",
      "2024-04-17 17:54:33,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080353.1, seqID NZ_CP080353.1 [record index 0].\n",
      "2024-04-17 17:54:33,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029590.1, seqID NZ_CP029590.1 [record index 0].\n",
      "2024-04-17 17:54:33,440 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029587.1, seqID NZ_CP029587.1 [record index 0].\n",
      "2024-04-17 17:54:33,488 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LS399318.1, seqID NZ_LS399318.1 [record index 0].\n",
      "2024-04-17 17:54:33,541 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045015.1, seqID NZ_CP045015.1 [record index 0].\n",
      "2024-04-17 17:54:33,592 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058940.1, seqID NZ_CP058940.1 [record index 0].\n",
      "2024-04-17 17:54:33,642 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070435.1, seqID NZ_CP070435.1 [record index 0].\n",
      "2024-04-17 17:54:33,691 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029738.1, seqID NZ_CP029738.1 [record index 0].\n",
      "2024-04-17 17:54:33,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP032185.1, seqID NZ_CP032185.1 [record index 0].\n",
      "2024-04-17 17:54:33,792 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083032.1, seqID NZ_CP083032.1 [record index 0].\n",
      "2024-04-17 17:54:33,842 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041927.1, seqID NZ_CP041927.1 [record index 0].\n",
      "2024-04-17 17:54:33,946 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP035382.1, seqID NZ_CP035382.1 [record index 0].\n",
      "2024-04-17 17:54:34,022 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP021972.1, seqID NZ_AP021972.1 [record index 0].\n",
      "2024-04-17 17:54:34,096 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022020.1, seqID NZ_AP022020.1 [record index 0].\n",
      "2024-04-17 17:54:34,132 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022108.1, seqID NZ_AP022108.1 [record index 0].\n",
      "2024-04-17 17:54:34,157 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP056260.1, seqID NZ_CP056260.1 [record index 0].\n",
      "2024-04-17 17:54:34,180 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR890184.1, seqID NZ_LR890184.1 [record index 0].\n",
      "2024-04-17 17:54:34,225 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_015663.1, seqID NC_015663.1 [record index 0].\n",
      "2024-04-17 17:54:34,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP082898.1, seqID NZ_CP082898.1 [record index 0].\n",
      "2024-04-17 17:54:34,275 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP011539.1, seqID NZ_CP011539.1 [record index 0].\n",
      "2024-04-17 17:54:34,321 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044083.1, seqID NZ_CP044083.1 [record index 0].\n",
      "2024-04-17 17:54:34,344 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP033817.1, seqID NZ_CP033817.1 [record index 0].\n",
      "2024-04-17 17:54:34,369 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP031756.1, seqID NZ_CP031756.1 [record index 0].\n",
      "2024-04-17 17:54:34,401 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014029.2, seqID NZ_CP014029.2 [record index 0].\n",
      "2024-04-17 17:54:34,446 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077293.1, seqID NZ_CP077293.1 [record index 0].\n",
      "2024-04-17 17:54:34,492 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077265.1, seqID NZ_CP077265.1 [record index 0].\n",
      "2024-04-17 17:54:34,516 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014748.2, seqID NZ_CP014748.2 [record index 0].\n",
      "2024-04-17 17:54:34,542 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP070520.1, seqID NZ_CP070520.1 [record index 0].\n",
      "2024-04-17 17:54:34,569 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_020181.1, seqID NC_020181.1 [record index 0].\n",
      "2024-04-17 17:54:34,597 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP026756.1, seqID NZ_CP026756.1 [record index 0].\n",
      "2024-04-17 17:54:34,624 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP024885.1, seqID NZ_CP024885.1 [record index 0].\n",
      "2024-04-17 17:54:34,713 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056119.1, seqID NZ_CP056119.1 [record index 0].\n",
      "2024-04-17 17:54:34,790 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR881936.1, seqID NZ_LR881936.1 [record index 0].\n",
      "2024-04-17 17:54:34,879 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019986.1, seqID NZ_CP019986.1 [record index 0].\n",
      "2024-04-17 17:54:34,969 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LR699006.1, seqID NZ_LR699006.1 [record index 0].\n",
      "2024-04-17 17:54:35,064 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026045.1, seqID NZ_CP026045.1 [record index 0].\n",
      "2024-04-17 17:54:35,167 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP028520.1, seqID NZ_CP028520.1 [record index 0].\n",
      "2024-04-17 17:54:35,259 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064661.1, seqID NZ_CP064661.1 [record index 0].\n",
      "2024-04-17 17:54:35,351 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064663.1, seqID NZ_CP064663.1 [record index 0].\n",
      "2024-04-17 17:54:35,440 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064666.1, seqID NZ_CP064666.1 [record index 0].\n",
      "2024-04-17 17:54:35,536 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064671.1, seqID NZ_CP064671.1 [record index 0].\n",
      "2024-04-17 17:54:35,626 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064672.1, seqID NZ_CP064672.1 [record index 0].\n",
      "2024-04-17 17:54:35,716 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064674.1, seqID NZ_CP064674.1 [record index 0].\n",
      "2024-04-17 17:54:35,808 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051218.1, seqID NZ_CP051218.1 [record index 0].\n",
      "2024-04-17 17:54:35,896 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP089313.1, seqID NZ_CP089313.1 [record index 0].\n",
      "2024-04-17 17:54:35,984 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077680.1, seqID NZ_CP077680.1 [record index 0].\n",
      "2024-04-17 17:54:36,056 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039441.1, seqID NZ_CP039441.1 [record index 0].\n",
      "2024-04-17 17:54:36,123 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039443.1, seqID NZ_CP039443.1 [record index 0].\n",
      "2024-04-17 17:54:36,189 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP039445.1, seqID NZ_CP039445.1 [record index 0].\n",
      "2024-04-17 17:54:36,275 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032446.1, seqID NZ_CP032446.1 [record index 0].\n",
      "2024-04-17 17:54:36,368 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032449.1, seqID NZ_CP032449.1 [record index 0].\n",
      "2024-04-17 17:54:36,458 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032379.1, seqID NZ_CP032379.1 [record index 0].\n",
      "2024-04-17 17:54:36,525 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025453.1, seqID NZ_CP025453.1 [record index 0].\n",
      "2024-04-17 17:54:36,588 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025452.1, seqID NZ_CP025452.1 [record index 0].\n",
      "2024-04-17 17:54:36,652 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025451.1, seqID NZ_CP025451.1 [record index 0].\n",
      "2024-04-17 17:54:36,715 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025450.1, seqID NZ_CP025450.1 [record index 0].\n",
      "2024-04-17 17:54:36,780 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025449.1, seqID NZ_CP025449.1 [record index 0].\n",
      "2024-04-17 17:54:36,843 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025448.1, seqID NZ_CP025448.1 [record index 0].\n",
      "2024-04-17 17:54:36,906 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025447.1, seqID NZ_CP025447.1 [record index 0].\n",
      "2024-04-17 17:54:36,970 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025454.1, seqID NZ_CP025454.1 [record index 0].\n",
      "2024-04-17 17:54:37,035 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025446.1, seqID NZ_CP025446.1 [record index 0].\n",
      "2024-04-17 17:54:37,105 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025445.1, seqID NZ_CP025445.1 [record index 0].\n",
      "2024-04-17 17:54:37,195 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025276.1, seqID NZ_CP025276.1 [record index 0].\n",
      "2024-04-17 17:54:37,293 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025280.1, seqID NZ_CP025280.1 [record index 0].\n",
      "2024-04-17 17:54:37,374 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP025278.1, seqID NZ_CP025278.1 [record index 0].\n",
      "2024-04-17 17:54:37,452 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014661.1, seqID NZ_CP014661.1 [record index 0].\n",
      "2024-04-17 17:54:37,529 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014666.2, seqID NZ_CP014666.2 [record index 0].\n",
      "2024-04-17 17:54:37,608 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014665.1, seqID NZ_CP014665.1 [record index 0].\n",
      "2024-04-17 17:54:37,688 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014659.2, seqID NZ_CP014659.2 [record index 0].\n",
      "2024-04-17 17:54:37,769 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014657.1, seqID NZ_CP014657.1 [record index 0].\n",
      "2024-04-17 17:54:37,848 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007584.2, seqID NZ_CP007584.2 [record index 0].\n",
      "2024-04-17 17:54:37,926 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014664.1, seqID NZ_CP014664.1 [record index 0].\n",
      "2024-04-17 17:54:38,000 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014621.2, seqID NZ_CP014621.2 [record index 0].\n",
      "2024-04-17 17:54:38,078 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014663.2, seqID NZ_CP014663.2 [record index 0].\n",
      "2024-04-17 17:54:38,153 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP014620.1, seqID NZ_CP014620.1 [record index 0].\n",
      "2024-04-17 17:54:38,223 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007483.2, seqID NZ_CP007483.2 [record index 0].\n",
      "2024-04-17 17:54:38,287 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085983.1, seqID NZ_CP085983.1 [record index 0].\n",
      "2024-04-17 17:54:38,352 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP085981.1, seqID NZ_CP085981.1 [record index 0].\n",
      "2024-04-17 17:54:38,435 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019411.1, seqID NZ_CP019411.1 [record index 0].\n",
      "2024-04-17 17:54:38,504 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP060512.1, seqID NZ_CP060512.1 [record index 0].\n",
      "2024-04-17 17:54:38,575 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_011149.1, seqID NC_011149.1 [record index 0].\n",
      "2024-04-17 17:54:38,669 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_AP019692.1, seqID NZ_AP019692.1 [record index 0].\n",
      "2024-04-17 17:54:38,770 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041179.1, seqID NZ_CP041179.1 [record index 0].\n",
      "2024-04-17 17:54:38,872 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041181.1, seqID NZ_CP041181.1 [record index 0].\n",
      "2024-04-17 17:54:38,969 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050777.1, seqID NZ_CP050777.1 [record index 0].\n",
      "2024-04-17 17:54:39,059 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050779.1, seqID NZ_CP050779.1 [record index 0].\n",
      "2024-04-17 17:54:39,152 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050783.1, seqID NZ_CP050783.1 [record index 0].\n",
      "2024-04-17 17:54:39,244 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050785.1, seqID NZ_CP050785.1 [record index 0].\n",
      "2024-04-17 17:54:39,333 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050756.1, seqID NZ_CP050756.1 [record index 0].\n",
      "2024-04-17 17:54:39,422 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050757.1, seqID NZ_CP050757.1 [record index 0].\n",
      "2024-04-17 17:54:39,511 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050760.1, seqID NZ_CP050760.1 [record index 0].\n",
      "2024-04-17 17:54:39,602 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050762.1, seqID NZ_CP050762.1 [record index 0].\n",
      "2024-04-17 17:54:39,693 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050764.1, seqID NZ_CP050764.1 [record index 0].\n",
      "2024-04-17 17:54:39,786 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050769.1, seqID NZ_CP050769.1 [record index 0].\n",
      "2024-04-17 17:54:39,878 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050771.1, seqID NZ_CP050771.1 [record index 0].\n",
      "2024-04-17 17:54:39,968 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP036165.1, seqID NZ_CP036165.1 [record index 0].\n",
      "2024-04-17 17:54:40,038 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP048775.1, seqID NZ_CP048775.1 [record index 0].\n",
      "2024-04-17 17:54:40,098 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045518.1, seqID NZ_CP045518.1 [record index 0].\n",
      "2024-04-17 17:54:40,163 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045516.1, seqID NZ_CP045516.1 [record index 0].\n",
      "2024-04-17 17:54:40,226 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045515.1, seqID NZ_CP045515.1 [record index 0].\n",
      "2024-04-17 17:54:40,293 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045466.1, seqID NZ_CP045466.1 [record index 0].\n",
      "2024-04-17 17:54:40,358 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045513.1, seqID NZ_CP045513.1 [record index 0].\n",
      "2024-04-17 17:54:40,425 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045465.1, seqID NZ_CP045465.1 [record index 0].\n",
      "2024-04-17 17:54:40,490 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045464.1, seqID NZ_CP045464.1 [record index 0].\n",
      "2024-04-17 17:54:40,575 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030005.1, seqID NZ_CP030005.1 [record index 0].\n",
      "2024-04-17 17:54:40,666 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029999.1, seqID NZ_CP029999.1 [record index 0].\n",
      "2024-04-17 17:54:40,774 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030180.1, seqID NZ_CP030180.1 [record index 0].\n",
      "2024-04-17 17:54:40,844 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019417.1, seqID NZ_CP019417.1 [record index 0].\n",
      "2024-04-17 17:54:40,898 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022015.1, seqID NZ_CP022015.1 [record index 0].\n",
      "2024-04-17 17:54:40,991 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030217.1, seqID NZ_CP030217.1 [record index 0].\n",
      "2024-04-17 17:54:41,077 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030002.1, seqID NZ_CP030002.1 [record index 0].\n",
      "2024-04-17 17:54:41,170 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030288.1, seqID NZ_CP030288.1 [record index 0].\n",
      "2024-04-17 17:54:41,259 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019403.1, seqID NZ_CP019403.1 [record index 0].\n",
      "2024-04-17 17:54:41,336 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030211.1, seqID NZ_CP030211.1 [record index 0].\n",
      "2024-04-17 17:54:41,438 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030209.1, seqID NZ_CP030209.1 [record index 0].\n",
      "2024-04-17 17:54:41,538 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030231.1, seqID NZ_CP030231.1 [record index 0].\n",
      "2024-04-17 17:54:41,633 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022138.1, seqID NZ_CP022138.1 [record index 0].\n",
      "2024-04-17 17:54:41,703 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019407.1, seqID NZ_CP019407.1 [record index 0].\n",
      "2024-04-17 17:54:41,767 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022489.1, seqID NZ_CP022489.1 [record index 0].\n",
      "2024-04-17 17:54:41,867 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030207.1, seqID NZ_CP030207.1 [record index 0].\n",
      "2024-04-17 17:54:41,956 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077670.1, seqID NZ_CP077670.1 [record index 0].\n",
      "2024-04-17 17:54:42,055 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP061118.1, seqID NZ_CP061118.1 [record index 0].\n",
      "2024-04-17 17:54:42,132 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP092040.1, seqID NZ_CP092040.1 [record index 0].\n",
      "2024-04-17 17:54:42,224 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019116.1, seqID NZ_CP019116.1 [record index 0].\n",
      "2024-04-17 17:54:42,311 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019412.1, seqID NZ_CP019412.1 [record index 0].\n",
      "2024-04-17 17:54:42,398 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022019.1, seqID NZ_CP022019.1 [record index 0].\n",
      "2024-04-17 17:54:42,485 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019409.1, seqID NZ_CP019409.1 [record index 0].\n",
      "2024-04-17 17:54:42,582 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022663.1, seqID NZ_CP022663.1 [record index 0].\n",
      "2024-04-17 17:54:42,655 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090930.1, seqID NZ_CP090930.1 [record index 0].\n",
      "2024-04-17 17:54:42,722 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP090929.1, seqID NZ_CP090929.1 [record index 0].\n",
      "2024-04-17 17:54:42,821 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060730.2, seqID NZ_CP060730.2 [record index 0].\n",
      "2024-04-17 17:54:42,916 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP062795.1, seqID NZ_CP062795.1 [record index 0].\n",
      "2024-04-17 17:54:43,020 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063330.1, seqID NZ_CP063330.1 [record index 0].\n",
      "2024-04-17 17:54:43,123 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP065572.1, seqID NZ_CP065572.1 [record index 0].\n",
      "2024-04-17 17:54:43,217 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP029800.1, seqID NZ_CP029800.1 [record index 0].\n",
      "2024-04-17 17:54:43,322 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP061929.2, seqID NZ_CP061929.2 [record index 0].\n",
      "2024-04-17 17:54:43,427 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP062004.2, seqID NZ_CP062004.2 [record index 0].\n",
      "2024-04-17 17:54:43,518 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP065564.1, seqID NZ_CP065564.1 [record index 0].\n",
      "2024-04-17 17:54:43,609 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP065566.1, seqID NZ_CP065566.1 [record index 0].\n",
      "2024-04-17 17:54:43,703 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045038.1, seqID NZ_CP045038.1 [record index 0].\n",
      "2024-04-17 17:54:43,802 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LS483455.1, seqID NZ_LS483455.1 [record index 0].\n",
      "2024-04-17 17:54:43,898 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LS483457.1, seqID NZ_LS483457.1 [record index 0].\n",
      "2024-04-17 17:54:43,968 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016410.1, seqID NZ_CP016410.1 [record index 0].\n",
      "2024-04-17 17:54:44,058 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026379.1, seqID NZ_CP026379.1 [record index 0].\n",
      "2024-04-17 17:54:44,144 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019410.1, seqID NZ_CP019410.1 [record index 0].\n",
      "2024-04-17 17:54:44,241 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030749.1, seqID NZ_CP030749.1 [record index 0].\n",
      "2024-04-17 17:54:44,309 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045509.1, seqID NZ_CP045509.1 [record index 0].\n",
      "2024-04-17 17:54:44,373 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045461.1, seqID NZ_CP045461.1 [record index 0].\n",
      "2024-04-17 17:54:44,437 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045458.1, seqID NZ_CP045458.1 [record index 0].\n",
      "2024-04-17 17:54:44,515 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP079713.1, seqID NZ_CP079713.1 [record index 0].\n",
      "2024-04-17 17:54:44,608 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP062220.1, seqID NZ_CP062220.1 [record index 0].\n",
      "2024-04-17 17:54:44,694 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028131.1, seqID NZ_CP028131.1 [record index 0].\n",
      "2024-04-17 17:54:44,784 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060585.1, seqID NZ_CP060585.1 [record index 0].\n",
      "2024-04-17 17:54:44,856 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013226.1, seqID NZ_CP013226.1 [record index 0].\n",
      "2024-04-17 17:54:44,922 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP013222.1, seqID NZ_CP013222.1 [record index 0].\n",
      "2024-04-17 17:54:45,016 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037894.1, seqID NZ_CP037894.1 [record index 0].\n",
      "2024-04-17 17:54:45,087 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016408.1, seqID NZ_CP016408.1 [record index 0].\n",
      "2024-04-17 17:54:45,153 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016406.1, seqID NZ_CP016406.1 [record index 0].\n",
      "2024-04-17 17:54:45,217 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP011365.1, seqID NZ_CP011365.1 [record index 0].\n",
      "2024-04-17 17:54:45,312 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041699.1, seqID NZ_CP041699.1 [record index 0].\n",
      "2024-04-17 17:54:45,377 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041011.1, seqID NZ_CP041011.1 [record index 0].\n",
      "2024-04-17 17:54:45,471 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046278.1, seqID NZ_CP046278.1 [record index 0].\n",
      "2024-04-17 17:54:45,558 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046280.1, seqID NZ_CP046280.1 [record index 0].\n",
      "2024-04-17 17:54:45,634 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP017719.1, seqID NZ_CP017719.1 [record index 0].\n",
      "2024-04-17 17:54:45,731 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037891.1, seqID NZ_CP037891.1 [record index 0].\n",
      "2024-04-17 17:54:45,826 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022450.1, seqID NZ_CP022450.1 [record index 0].\n",
      "2024-04-17 17:54:45,920 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032384.1, seqID NZ_CP032384.1 [record index 0].\n",
      "2024-04-17 17:54:46,012 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032387.1, seqID NZ_CP032387.1 [record index 0].\n",
      "2024-04-17 17:54:46,104 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032390.1, seqID NZ_CP032390.1 [record index 0].\n",
      "2024-04-17 17:54:46,195 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051320.1, seqID NZ_CP051320.1 [record index 0].\n",
      "2024-04-17 17:54:46,266 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051342.1, seqID NZ_CP051342.1 [record index 0].\n",
      "2024-04-17 17:54:46,333 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051356.1, seqID NZ_CP051356.1 [record index 0].\n",
      "2024-04-17 17:54:46,420 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051391.1, seqID NZ_CP051391.1 [record index 0].\n",
      "2024-04-17 17:54:46,515 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032393.1, seqID NZ_CP032393.1 [record index 0].\n",
      "2024-04-17 17:54:46,584 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051394.1, seqID NZ_CP051394.1 [record index 0].\n",
      "2024-04-17 17:54:46,675 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP032396.1, seqID NZ_CP032396.1 [record index 0].\n",
      "2024-04-17 17:54:46,747 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051402.1, seqID NZ_CP051402.1 [record index 0].\n",
      "2024-04-17 17:54:46,812 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP016412.1, seqID NZ_CP016412.1 [record index 0].\n",
      "2024-04-17 17:54:46,898 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011205.1, seqID NC_011205.1 [record index 0].\n",
      "2024-04-17 17:54:46,967 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP044257.1, seqID NZ_CP044257.1 [record index 0].\n",
      "2024-04-17 17:54:47,053 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP037892.1, seqID NZ_CP037892.1 [record index 0].\n",
      "2024-04-17 17:54:47,130 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033338.1, seqID NZ_CP033338.1 [record index 0].\n",
      "2024-04-17 17:54:47,207 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP033343.1, seqID NZ_CP033343.1 [record index 0].\n",
      "2024-04-17 17:54:47,276 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075106.1, seqID NZ_CP075106.1 [record index 0].\n",
      "2024-04-17 17:54:47,361 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075113.1, seqID NZ_CP075113.1 [record index 0].\n",
      "2024-04-17 17:54:47,447 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075115.1, seqID NZ_CP075115.1 [record index 0].\n",
      "2024-04-17 17:54:47,528 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077711.1, seqID NZ_CP077711.1 [record index 0].\n",
      "2024-04-17 17:54:47,623 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077699.1, seqID NZ_CP077699.1 [record index 0].\n",
      "2024-04-17 17:54:47,714 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077706.1, seqID NZ_CP077706.1 [record index 0].\n",
      "2024-04-17 17:54:47,811 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077710.1, seqID NZ_CP077710.1 [record index 0].\n",
      "2024-04-17 17:54:47,901 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029336.1, seqID NZ_CP029336.1 [record index 0].\n",
      "2024-04-17 17:54:47,997 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029039.1, seqID NZ_CP029039.1 [record index 0].\n",
      "2024-04-17 17:54:48,085 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075125.1, seqID NZ_CP075125.1 [record index 0].\n",
      "2024-04-17 17:54:48,180 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075128.1, seqID NZ_CP075128.1 [record index 0].\n",
      "2024-04-17 17:54:48,271 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075129.1, seqID NZ_CP075129.1 [record index 0].\n",
      "2024-04-17 17:54:48,361 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075135.1, seqID NZ_CP075135.1 [record index 0].\n",
      "2024-04-17 17:54:48,457 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075138.1, seqID NZ_CP075138.1 [record index 0].\n",
      "2024-04-17 17:54:48,551 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075140.1, seqID NZ_CP075140.1 [record index 0].\n",
      "2024-04-17 17:54:48,620 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041183.1, seqID NZ_CP041183.1 [record index 0].\n",
      "2024-04-17 17:54:48,685 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP041184.1, seqID NZ_CP041184.1 [record index 0].\n",
      "2024-04-17 17:54:48,755 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019202.1, seqID NZ_CP019202.1 [record index 0].\n",
      "2024-04-17 17:54:48,847 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033356.2, seqID NZ_CP033356.2 [record index 0].\n",
      "2024-04-17 17:54:48,952 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033384.2, seqID NZ_CP033384.2 [record index 0].\n",
      "2024-04-17 17:54:49,054 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033387.2, seqID NZ_CP033387.2 [record index 0].\n",
      "2024-04-17 17:54:49,157 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038608.1, seqID NZ_CP038608.1 [record index 0].\n",
      "2024-04-17 17:54:49,258 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038604.1, seqID NZ_CP038604.1 [record index 0].\n",
      "2024-04-17 17:54:49,359 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP039279.1, seqID NZ_CP039279.1 [record index 0].\n",
      "2024-04-17 17:54:49,458 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017977.1, seqID NZ_CP017977.1 [record index 0].\n",
      "2024-04-17 17:54:49,559 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020912.1, seqID NZ_CP020912.1 [record index 0].\n",
      "2024-04-17 17:54:49,626 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007211.2, seqID NZ_CP007211.2 [record index 0].\n",
      "2024-04-17 17:54:49,722 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015724.1, seqID NZ_CP015724.1 [record index 0].\n",
      "2024-04-17 17:54:49,791 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045958.1, seqID NZ_CP045958.1 [record index 0].\n",
      "2024-04-17 17:54:49,876 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045955.1, seqID NZ_CP045955.1 [record index 0].\n",
      "2024-04-17 17:54:49,962 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP045956.1, seqID NZ_CP045956.1 [record index 0].\n",
      "2024-04-17 17:54:50,071 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019189.1, seqID NZ_CP019189.1 [record index 0].\n",
      "2024-04-17 17:54:50,137 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007531.1, seqID NZ_CP007531.1 [record index 0].\n",
      "2024-04-17 17:54:50,217 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012346.1, seqID NZ_CP012346.1 [record index 0].\n",
      "2024-04-17 17:54:50,304 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007534.1, seqID NZ_CP007534.1 [record index 0].\n",
      "2024-04-17 17:54:50,405 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019177.1, seqID NZ_CP019177.1 [record index 0].\n",
      "2024-04-17 17:54:50,507 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019179.1, seqID NZ_CP019179.1 [record index 0].\n",
      "2024-04-17 17:54:50,590 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007532.1, seqID NZ_CP007532.1 [record index 0].\n",
      "2024-04-17 17:54:50,682 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012349.1, seqID NZ_CP012349.1 [record index 0].\n",
      "2024-04-17 17:54:50,780 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019186.1, seqID NZ_CP019186.1 [record index 0].\n",
      "2024-04-17 17:54:50,885 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019181.1, seqID NZ_CP019181.1 [record index 0].\n",
      "2024-04-17 17:54:50,978 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019192.1, seqID NZ_CP019192.1 [record index 0].\n",
      "2024-04-17 17:54:51,081 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016837.1, seqID NZ_CP016837.1 [record index 0].\n",
      "2024-04-17 17:54:51,172 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP067078.1, seqID NZ_CP067078.1 [record index 0].\n",
      "2024-04-17 17:54:51,271 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029035.1, seqID NZ_CP029035.1 [record index 0].\n",
      "2024-04-17 17:54:51,357 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007530.1, seqID NZ_CP007530.1 [record index 0].\n",
      "2024-04-17 17:54:51,425 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP011259.2, seqID NZ_CP011259.2 [record index 0].\n",
      "2024-04-17 17:54:51,518 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019201.1, seqID NZ_CP019201.1 [record index 0].\n",
      "2024-04-17 17:54:51,587 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP015024.2, seqID NZ_CP015024.2 [record index 0].\n",
      "2024-04-17 17:54:51,671 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054827.1, seqID NZ_CP054827.1 [record index 0].\n",
      "2024-04-17 17:54:51,772 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029908.1, seqID NZ_CP029908.1 [record index 0].\n",
      "2024-04-17 17:54:51,873 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029918.1, seqID NZ_CP029918.1 [record index 0].\n",
      "2024-04-17 17:54:51,971 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029927.1, seqID NZ_CP029927.1 [record index 0].\n",
      "2024-04-17 17:54:52,040 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_022991.1, seqID NC_022991.1 [record index 0].\n",
      "2024-04-17 17:54:52,125 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP031189.1, seqID NZ_CP031189.1 [record index 0].\n",
      "2024-04-17 17:54:52,215 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026609.1, seqID NZ_CP026609.1 [record index 0].\n",
      "2024-04-17 17:54:52,303 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP053584.1, seqID NZ_CP053584.1 [record index 0].\n",
      "2024-04-17 17:54:52,421 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP053579.1, seqID NZ_CP053579.1 [record index 0].\n",
      "2024-04-17 17:54:52,492 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP071388.1, seqID NZ_CP071388.1 [record index 0].\n",
      "2024-04-17 17:54:52,576 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034074.1, seqID NZ_CP034074.1 [record index 0].\n",
      "2024-04-17 17:54:52,650 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP026660.1, seqID NZ_CP026660.1 [record index 0].\n",
      "2024-04-17 17:54:52,752 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP092258.1, seqID NZ_CP092258.1 [record index 0].\n",
      "2024-04-17 17:54:52,840 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054422.1, seqID NZ_CP054422.1 [record index 0].\n",
      "2024-04-17 17:54:52,909 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LN649235.1, seqID NZ_LN649235.1 [record index 0].\n",
      "2024-04-17 17:54:52,976 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032815.1, seqID NZ_CP032815.1 [record index 0].\n",
      "2024-04-17 17:54:53,044 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032817.1, seqID NZ_CP032817.1 [record index 0].\n",
      "2024-04-17 17:54:53,114 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP047881.1, seqID NZ_CP047881.1 [record index 0].\n",
      "2024-04-17 17:54:53,189 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP032814.1, seqID NZ_CP032814.1 [record index 0].\n",
      "2024-04-17 17:54:53,269 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP020492.1, seqID NZ_CP020492.1 [record index 0].\n",
      "2024-04-17 17:54:53,373 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019198.1, seqID NZ_CP019198.1 [record index 0].\n",
      "2024-04-17 17:54:53,457 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP021983.1, seqID NZ_AP021983.1 [record index 0].\n",
      "2024-04-17 17:54:53,535 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP049752.1, seqID NZ_CP049752.1 [record index 0].\n",
      "2024-04-17 17:54:53,612 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR134195.1, seqID NZ_LR134195.1 [record index 0].\n",
      "2024-04-17 17:54:53,690 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP046672.1, seqID NZ_CP046672.1 [record index 0].\n",
      "2024-04-17 17:54:53,748 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080099.1, seqID NZ_CP080099.1 [record index 0].\n",
      "2024-04-17 17:54:53,798 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP022142.1, seqID NZ_AP022142.1 [record index 0].\n",
      "2024-04-17 17:54:53,850 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP019687.1, seqID NZ_AP019687.1 [record index 0].\n",
      "2024-04-17 17:54:53,903 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075885.1, seqID NZ_CP075885.1 [record index 0].\n",
      "2024-04-17 17:54:53,950 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP083633.1, seqID NZ_CP083633.1 [record index 0].\n",
      "2024-04-17 17:54:54,001 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039791.1, seqID NZ_CP039791.1 [record index 0].\n",
      "2024-04-17 17:54:54,049 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP084876.1, seqID NZ_CP084876.1 [record index 0].\n",
      "2024-04-17 17:54:54,113 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077331.1, seqID NZ_CP077331.1 [record index 0].\n",
      "2024-04-17 17:54:54,200 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP088265.1, seqID NZ_CP088265.1 [record index 0].\n",
      "2024-04-17 17:54:54,291 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046445.1, seqID NZ_CP046445.1 [record index 0].\n",
      "2024-04-17 17:54:54,387 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP014007.2, seqID NZ_CP014007.2 [record index 0].\n",
      "2024-04-17 17:54:54,509 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP065358.1, seqID NZ_CP065358.1 [record index 0].\n",
      "2024-04-17 17:54:54,585 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022201.1, seqID NZ_AP022201.1 [record index 0].\n",
      "2024-04-17 17:54:54,655 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022195.1, seqID NZ_AP022195.1 [record index 0].\n",
      "2024-04-17 17:54:54,723 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022189.1, seqID NZ_AP022189.1 [record index 0].\n",
      "2024-04-17 17:54:54,790 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022057.1, seqID NZ_AP022057.1 [record index 0].\n",
      "2024-04-17 17:54:54,814 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP077429.1, seqID NZ_CP077429.1 [record index 0].\n",
      "2024-04-17 17:54:54,860 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134475.1, seqID NZ_LR134475.1 [record index 0].\n",
      "2024-04-17 17:54:54,885 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023963.1, seqID NZ_CP023963.1 [record index 0].\n",
      "2024-04-17 17:54:54,909 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP011574.1, seqID NZ_CP011574.1 [record index 0].\n",
      "2024-04-17 17:54:54,933 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP042530.1, seqID NZ_CP042530.1 [record index 0].\n",
      "2024-04-17 17:54:54,961 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP024880.1, seqID NZ_CP024880.1 [record index 0].\n",
      "2024-04-17 17:54:54,990 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP024883.1, seqID NZ_CP024883.1 [record index 0].\n",
      "2024-04-17 17:54:55,015 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050067.1, seqID NZ_CP050067.1 [record index 0].\n",
      "2024-04-17 17:54:55,083 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP037864.1, seqID NZ_CP037864.1 [record index 0].\n",
      "2024-04-17 17:54:55,167 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074425.1, seqID NZ_CP074425.1 [record index 0].\n",
      "2024-04-17 17:54:55,257 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074428.1, seqID NZ_CP074428.1 [record index 0].\n",
      "2024-04-17 17:54:55,348 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP071693.1, seqID NZ_CP071693.1 [record index 0].\n",
      "2024-04-17 17:54:55,438 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP071690.1, seqID NZ_CP071690.1 [record index 0].\n",
      "2024-04-17 17:54:55,524 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP071686.1, seqID NZ_CP071686.1 [record index 0].\n",
      "2024-04-17 17:54:55,614 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP082916.1, seqID NZ_CP082916.1 [record index 0].\n",
      "2024-04-17 17:54:55,705 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085801.1, seqID NZ_CP085801.1 [record index 0].\n",
      "2024-04-17 17:54:55,794 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085802.1, seqID NZ_CP085802.1 [record index 0].\n",
      "2024-04-17 17:54:55,881 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085823.1, seqID NZ_CP085823.1 [record index 0].\n",
      "2024-04-17 17:54:55,968 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085800.1, seqID NZ_CP085800.1 [record index 0].\n",
      "2024-04-17 17:54:56,055 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085799.1, seqID NZ_CP085799.1 [record index 0].\n",
      "2024-04-17 17:54:56,145 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060522.1, seqID NZ_CP060522.1 [record index 0].\n",
      "2024-04-17 17:54:56,233 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP080091.1, seqID NZ_CP080091.1 [record index 0].\n",
      "2024-04-17 17:54:56,325 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060517.1, seqID NZ_CP060517.1 [record index 0].\n",
      "2024-04-17 17:54:56,413 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085816.1, seqID NZ_CP085816.1 [record index 0].\n",
      "2024-04-17 17:54:56,502 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085815.1, seqID NZ_CP085815.1 [record index 0].\n",
      "2024-04-17 17:54:56,601 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041173.1, seqID NZ_CP041173.1 [record index 0].\n",
      "2024-04-17 17:54:56,701 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041176.1, seqID NZ_CP041176.1 [record index 0].\n",
      "2024-04-17 17:54:56,797 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015526.1, seqID NZ_CP015526.1 [record index 0].\n",
      "2024-04-17 17:54:56,898 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP015524.1, seqID NZ_CP015524.1 [record index 0].\n",
      "2024-04-17 17:54:57,002 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP045761.1, seqID NZ_CP045761.1 [record index 0].\n",
      "2024-04-17 17:54:57,095 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033090.1, seqID NZ_CP033090.1 [record index 0].\n",
      "2024-04-17 17:54:57,193 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP008928.1, seqID NZ_CP008928.1 [record index 0].\n",
      "2024-04-17 17:54:57,289 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011791.1, seqID NZ_CP011791.1 [record index 0].\n",
      "2024-04-17 17:54:57,383 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011790.1, seqID NZ_CP011790.1 [record index 0].\n",
      "2024-04-17 17:54:57,479 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050716.1, seqID NZ_CP050716.1 [record index 0].\n",
      "2024-04-17 17:54:57,574 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019681.1, seqID NZ_CP019681.1 [record index 0].\n",
      "2024-04-17 17:54:57,672 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050721.1, seqID NZ_CP050721.1 [record index 0].\n",
      "2024-04-17 17:54:57,763 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050723.1, seqID NZ_CP050723.1 [record index 0].\n",
      "2024-04-17 17:54:57,848 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP084532.1, seqID NZ_CP084532.1 [record index 0].\n",
      "2024-04-17 17:54:57,934 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050706.1, seqID NZ_CP050706.1 [record index 0].\n",
      "2024-04-17 17:54:58,022 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050709.1, seqID NZ_CP050709.1 [record index 0].\n",
      "2024-04-17 17:54:58,124 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP050712.1, seqID NZ_CP050712.1 [record index 0].\n",
      "2024-04-17 17:54:58,212 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP091300.1, seqID NZ_CP091300.1 [record index 0].\n",
      "2024-04-17 17:54:58,298 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040646.1, seqID NZ_CP040646.1 [record index 0].\n",
      "2024-04-17 17:54:58,386 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040644.1, seqID NZ_CP040644.1 [record index 0].\n",
      "2024-04-17 17:54:58,484 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP029989.1, seqID NZ_CP029989.1 [record index 0].\n",
      "2024-04-17 17:54:58,553 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030024.1, seqID NZ_CP030024.1 [record index 0].\n",
      "2024-04-17 17:54:58,622 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030233.1, seqID NZ_CP030233.1 [record index 0].\n",
      "2024-04-17 17:54:58,709 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007312.2, seqID NZ_CP007312.2 [record index 0].\n",
      "2024-04-17 17:54:58,819 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030203.1, seqID NZ_CP030203.1 [record index 0].\n",
      "2024-04-17 17:54:58,919 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030223.1, seqID NZ_CP030223.1 [record index 0].\n",
      "2024-04-17 17:54:59,015 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007425.1, seqID NZ_CP007425.1 [record index 0].\n",
      "2024-04-17 17:54:59,120 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP030194.1, seqID NZ_CP030194.1 [record index 0].\n",
      "2024-04-17 17:54:59,211 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022034.1, seqID NZ_CP022034.1 [record index 0].\n",
      "2024-04-17 17:54:59,329 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP030238.1, seqID NZ_CP030238.1 [record index 0].\n",
      "2024-04-17 17:54:59,411 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030196.1, seqID NZ_CP030196.1 [record index 0].\n",
      "2024-04-17 17:54:59,481 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP022135.1, seqID NZ_CP022135.1 [record index 0].\n",
      "2024-04-17 17:54:59,570 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022116.1, seqID NZ_CP022116.1 [record index 0].\n",
      "2024-04-17 17:54:59,648 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP030181.1, seqID NZ_CP030181.1 [record index 0].\n",
      "2024-04-17 17:54:59,749 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030219.1, seqID NZ_CP030219.1 [record index 0].\n",
      "2024-04-17 17:54:59,863 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007292.1, seqID NZ_CP007292.1 [record index 0].\n",
      "2024-04-17 17:54:59,952 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007291.2, seqID NZ_CP007291.2 [record index 0].\n",
      "2024-04-17 17:55:00,049 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007288.2, seqID NZ_CP007288.2 [record index 0].\n",
      "2024-04-17 17:55:00,145 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007286.2, seqID NZ_CP007286.2 [record index 0].\n",
      "2024-04-17 17:55:00,239 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007285.2, seqID NZ_CP007285.2 [record index 0].\n",
      "2024-04-17 17:55:00,331 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007283.2, seqID NZ_CP007283.2 [record index 0].\n",
      "2024-04-17 17:55:00,426 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007282.2, seqID NZ_CP007282.2 [record index 0].\n",
      "2024-04-17 17:55:00,520 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007280.2, seqID NZ_CP007280.2 [record index 0].\n",
      "2024-04-17 17:55:00,608 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019415.1, seqID NZ_CP019415.1 [record index 0].\n",
      "2024-04-17 17:55:00,697 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019406.1, seqID NZ_CP019406.1 [record index 0].\n",
      "2024-04-17 17:55:00,788 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022117.1, seqID NZ_CP022117.1 [record index 0].\n",
      "2024-04-17 17:55:00,874 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_010067.1, seqID NC_010067.1 [record index 0].\n",
      "2024-04-17 17:55:00,993 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028157.1, seqID NZ_CP028157.1 [record index 0].\n",
      "2024-04-17 17:55:01,094 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028151.1, seqID NZ_CP028151.1 [record index 0].\n",
      "2024-04-17 17:55:01,184 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063289.1, seqID NZ_CP063289.1 [record index 0].\n",
      "2024-04-17 17:55:01,284 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022963.1, seqID NZ_CP022963.1 [record index 0].\n",
      "2024-04-17 17:55:01,369 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP043433.1, seqID NZ_CP043433.1 [record index 0].\n",
      "2024-04-17 17:55:01,457 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085793.1, seqID NZ_CP085793.1 [record index 0].\n",
      "2024-04-17 17:55:01,546 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085814.1, seqID NZ_CP085814.1 [record index 0].\n",
      "2024-04-17 17:55:01,632 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025559.1, seqID NZ_CP025559.1 [record index 0].\n",
      "2024-04-17 17:55:01,726 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063700.1, seqID NZ_CP063700.1 [record index 0].\n",
      "2024-04-17 17:55:01,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011294.1, seqID NC_011294.1 [record index 0].\n",
      "2024-04-17 17:55:01,908 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009091.1, seqID NZ_CP009091.1 [record index 0].\n",
      "2024-04-17 17:55:01,993 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009090.1, seqID NZ_CP009090.1 [record index 0].\n",
      "2024-04-17 17:55:02,087 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009089.1, seqID NZ_CP009089.1 [record index 0].\n",
      "2024-04-17 17:55:02,181 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009088.1, seqID NZ_CP009088.1 [record index 0].\n",
      "2024-04-17 17:55:02,277 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009087.1, seqID NZ_CP009087.1 [record index 0].\n",
      "2024-04-17 17:55:02,376 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009086.1, seqID NZ_CP009086.1 [record index 0].\n",
      "2024-04-17 17:55:02,473 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009085.2, seqID NZ_CP009085.2 [record index 0].\n",
      "2024-04-17 17:55:02,564 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009084.2, seqID NZ_CP009084.2 [record index 0].\n",
      "2024-04-17 17:55:02,653 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009093.1, seqID NZ_CP009093.1 [record index 0].\n",
      "2024-04-17 17:55:02,747 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009092.1, seqID NZ_CP009092.1 [record index 0].\n",
      "2024-04-17 17:55:02,840 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009083.1, seqID NZ_CP009083.1 [record index 0].\n",
      "2024-04-17 17:55:02,937 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011942.1, seqID NZ_CP011942.1 [record index 0].\n",
      "2024-04-17 17:55:03,026 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP064709.1, seqID NZ_CP064709.1 [record index 0].\n",
      "2024-04-17 17:55:03,129 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_LS483465.1, seqID NZ_LS483465.1 [record index 0].\n",
      "2024-04-17 17:55:03,232 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041973.1, seqID NZ_CP041973.1 [record index 0].\n",
      "2024-04-17 17:55:03,319 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085813.1, seqID NZ_CP085813.1 [record index 0].\n",
      "2024-04-17 17:55:03,405 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085792.1, seqID NZ_CP085792.1 [record index 0].\n",
      "2024-04-17 17:55:03,494 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026569.1, seqID NZ_CP026569.1 [record index 0].\n",
      "2024-04-17 17:55:03,582 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085806.1, seqID NZ_CP085806.1 [record index 0].\n",
      "2024-04-17 17:55:03,672 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP078142.1, seqID NZ_CP078142.1 [record index 0].\n",
      "2024-04-17 17:55:03,763 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085804.1, seqID NZ_CP085804.1 [record index 0].\n",
      "2024-04-17 17:55:03,851 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085797.1, seqID NZ_CP085797.1 [record index 0].\n",
      "2024-04-17 17:55:03,938 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085798.1, seqID NZ_CP085798.1 [record index 0].\n",
      "2024-04-17 17:55:04,024 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085824.1, seqID NZ_CP085824.1 [record index 0].\n",
      "2024-04-17 17:55:04,115 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023345.1, seqID NZ_CP023345.1 [record index 0].\n",
      "2024-04-17 17:55:04,211 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069166.1, seqID NZ_CP069166.1 [record index 0].\n",
      "2024-04-17 17:55:04,306 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069297.1, seqID NZ_CP069297.1 [record index 0].\n",
      "2024-04-17 17:55:04,393 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085794.1, seqID NZ_CP085794.1 [record index 0].\n",
      "2024-04-17 17:55:04,480 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085803.1, seqID NZ_CP085803.1 [record index 0].\n",
      "2024-04-17 17:55:04,567 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085795.1, seqID NZ_CP085795.1 [record index 0].\n",
      "2024-04-17 17:55:04,655 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085796.1, seqID NZ_CP085796.1 [record index 0].\n",
      "2024-04-17 17:55:04,749 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026713.1, seqID NZ_CP026713.1 [record index 0].\n",
      "2024-04-17 17:55:04,848 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023475.1, seqID NZ_CP023475.1 [record index 0].\n",
      "2024-04-17 17:55:04,947 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP023436.1, seqID NZ_CP023436.1 [record index 0].\n",
      "2024-04-17 17:55:05,045 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017177.1, seqID NZ_CP017177.1 [record index 0].\n",
      "2024-04-17 17:55:05,140 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP016754.1, seqID NZ_CP016754.1 [record index 0].\n",
      "2024-04-17 17:55:05,234 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017232.1, seqID NZ_CP017232.1 [record index 0].\n",
      "2024-04-17 17:55:05,304 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012144.1, seqID NZ_CP012144.1 [record index 0].\n",
      "2024-04-17 17:55:05,398 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012396.1, seqID NZ_CP012396.1 [record index 0].\n",
      "2024-04-17 17:55:05,496 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP009768.1, seqID NZ_CP009768.1 [record index 0].\n",
      "2024-04-17 17:55:05,585 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019383.1, seqID NZ_CP019383.1 [record index 0].\n",
      "2024-04-17 17:55:05,671 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026052.1, seqID NZ_CP026052.1 [record index 0].\n",
      "2024-04-17 17:55:05,763 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022069.2, seqID NZ_CP022069.2 [record index 0].\n",
      "2024-04-17 17:55:05,852 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP066047.1, seqID NZ_CP066047.1 [record index 0].\n",
      "2024-04-17 17:55:05,938 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033360.1, seqID NZ_CP033360.1 [record index 0].\n",
      "2024-04-17 17:55:06,034 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007273.1, seqID NZ_CP007273.1 [record index 0].\n",
      "2024-04-17 17:55:06,130 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007272.2, seqID NZ_CP007272.2 [record index 0].\n",
      "2024-04-17 17:55:06,228 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007271.1, seqID NZ_CP007271.1 [record index 0].\n",
      "2024-04-17 17:55:06,322 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007333.1, seqID NZ_CP007333.1 [record index 0].\n",
      "2024-04-17 17:55:06,408 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007269.1, seqID NZ_CP007269.1 [record index 0].\n",
      "2024-04-17 17:55:06,505 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007434.2, seqID NZ_CP007434.2 [record index 0].\n",
      "2024-04-17 17:55:06,592 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007245.1, seqID NZ_CP007245.1 [record index 0].\n",
      "2024-04-17 17:55:06,677 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007267.2, seqID NZ_CP007267.2 [record index 0].\n",
      "2024-04-17 17:55:06,764 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007329.2, seqID NZ_CP007329.2 [record index 0].\n",
      "2024-04-17 17:55:06,850 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007252.1, seqID NZ_CP007252.1 [record index 0].\n",
      "2024-04-17 17:55:06,935 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007253.1, seqID NZ_CP007253.1 [record index 0].\n",
      "2024-04-17 17:55:07,020 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007254.1, seqID NZ_CP007254.1 [record index 0].\n",
      "2024-04-17 17:55:07,106 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007263.1, seqID NZ_CP007263.1 [record index 0].\n",
      "2024-04-17 17:55:07,193 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007258.1, seqID NZ_CP007258.1 [record index 0].\n",
      "2024-04-17 17:55:07,282 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007259.1, seqID NZ_CP007259.1 [record index 0].\n",
      "2024-04-17 17:55:07,369 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007260.1, seqID NZ_CP007260.1 [record index 0].\n",
      "2024-04-17 17:55:07,456 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007261.1, seqID NZ_CP007261.1 [record index 0].\n",
      "2024-04-17 17:55:07,543 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007262.1, seqID NZ_CP007262.1 [record index 0].\n",
      "2024-04-17 17:55:07,638 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007250.1, seqID NZ_CP007250.1 [record index 0].\n",
      "2024-04-17 17:55:07,726 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007175.1, seqID NZ_CP007175.1 [record index 0].\n",
      "2024-04-17 17:55:07,812 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007247.1, seqID NZ_CP007247.1 [record index 0].\n",
      "2024-04-17 17:55:07,902 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007360.2, seqID NZ_CP007360.2 [record index 0].\n",
      "2024-04-17 17:55:07,988 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007246.1, seqID NZ_CP007246.1 [record index 0].\n",
      "2024-04-17 17:55:08,072 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007248.1, seqID NZ_CP007248.1 [record index 0].\n",
      "2024-04-17 17:55:08,157 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007249.2, seqID NZ_CP007249.2 [record index 0].\n",
      "2024-04-17 17:55:08,247 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007507.1, seqID NZ_CP007507.1 [record index 0].\n",
      "2024-04-17 17:55:08,315 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP063702.1, seqID NZ_CP063702.1 [record index 0].\n",
      "2024-04-17 17:55:08,399 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051349.1, seqID NZ_CP051349.1 [record index 0].\n",
      "2024-04-17 17:55:08,507 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP051368.1, seqID NZ_CP051368.1 [record index 0].\n",
      "2024-04-17 17:55:08,597 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP051370.1, seqID NZ_CP051370.1 [record index 0].\n",
      "2024-04-17 17:55:08,664 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP063705.1, seqID NZ_CP063705.1 [record index 0].\n",
      "2024-04-17 17:55:08,757 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP013097.1, seqID NZ_CP013097.1 [record index 0].\n",
      "2024-04-17 17:55:08,849 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085818.1, seqID NZ_CP085818.1 [record index 0].\n",
      "2024-04-17 17:55:08,945 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085817.1, seqID NZ_CP085817.1 [record index 0].\n",
      "2024-04-17 17:55:09,034 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085822.1, seqID NZ_CP085822.1 [record index 0].\n",
      "2024-04-17 17:55:09,124 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085821.1, seqID NZ_CP085821.1 [record index 0].\n",
      "2024-04-17 17:55:09,211 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP033340.1, seqID NZ_CP033340.1 [record index 0].\n",
      "2024-04-17 17:55:09,303 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP024165.1, seqID NZ_CP024165.1 [record index 0].\n",
      "2024-04-17 17:55:09,390 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP028172.1, seqID NZ_CP028172.1 [record index 0].\n",
      "2024-04-17 17:55:09,457 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP075116.1, seqID NZ_CP075116.1 [record index 0].\n",
      "2024-04-17 17:55:09,539 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075118.1, seqID NZ_CP075118.1 [record index 0].\n",
      "2024-04-17 17:55:09,625 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075120.1, seqID NZ_CP075120.1 [record index 0].\n",
      "2024-04-17 17:55:09,722 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022003.1, seqID NZ_CP022003.1 [record index 0].\n",
      "2024-04-17 17:55:09,810 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075122.1, seqID NZ_CP075122.1 [record index 0].\n",
      "2024-04-17 17:55:09,902 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP075132.1, seqID NZ_CP075132.1 [record index 0].\n",
      "2024-04-17 17:55:09,991 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020825.1, seqID NZ_CP020825.1 [record index 0].\n",
      "2024-04-17 17:55:10,078 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP020823.1, seqID NZ_CP020823.1 [record index 0].\n",
      "2024-04-17 17:55:10,163 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP074661.1, seqID NZ_CP074661.1 [record index 0].\n",
      "2024-04-17 17:55:10,253 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007528.1, seqID NZ_CP007528.1 [record index 0].\n",
      "2024-04-17 17:55:10,348 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_022221.1, seqID NC_022221.1 [record index 0].\n",
      "2024-04-17 17:55:10,438 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP085805.1, seqID NZ_CP085805.1 [record index 0].\n",
      "2024-04-17 17:55:10,525 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP025554.1, seqID NZ_CP025554.1 [record index 0].\n",
      "2024-04-17 17:55:10,613 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012347.1, seqID NZ_CP012347.1 [record index 0].\n",
      "2024-04-17 17:55:10,699 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP034233.1, seqID NZ_CP034233.1 [record index 0].\n",
      "2024-04-17 17:55:10,784 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP063708.1, seqID NZ_CP063708.1 [record index 0].\n",
      "2024-04-17 17:55:10,881 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018661.1, seqID NZ_CP018661.1 [record index 0].\n",
      "2024-04-17 17:55:10,977 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018659.1, seqID NZ_CP018659.1 [record index 0].\n",
      "2024-04-17 17:55:11,072 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019035.1, seqID NZ_CP019035.1 [record index 0].\n",
      "2024-04-17 17:55:11,189 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP054715.1, seqID NZ_CP054715.1 [record index 0].\n",
      "2024-04-17 17:55:11,290 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018655.1, seqID NZ_CP018655.1 [record index 0].\n",
      "2024-04-17 17:55:11,389 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018651.1, seqID NZ_CP018651.1 [record index 0].\n",
      "2024-04-17 17:55:11,485 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018648.1, seqID NZ_CP018648.1 [record index 0].\n",
      "2024-04-17 17:55:11,580 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018647.1, seqID NZ_CP018647.1 [record index 0].\n",
      "2024-04-17 17:55:11,677 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018645.1, seqID NZ_CP018645.1 [record index 0].\n",
      "2024-04-17 17:55:11,775 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018644.1, seqID NZ_CP018644.1 [record index 0].\n",
      "2024-04-17 17:55:11,870 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP007598.1, seqID NZ_CP007598.1 [record index 0].\n",
      "2024-04-17 17:55:11,966 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018642.1, seqID NZ_CP018642.1 [record index 0].\n",
      "2024-04-17 17:55:12,063 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018640.1, seqID NZ_CP018640.1 [record index 0].\n",
      "2024-04-17 17:55:12,160 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018637.1, seqID NZ_CP018637.1 [record index 0].\n",
      "2024-04-17 17:55:12,256 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018635.1, seqID NZ_CP018635.1 [record index 0].\n",
      "2024-04-17 17:55:12,351 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018633.1, seqID NZ_CP018633.1 [record index 0].\n",
      "2024-04-17 17:55:12,437 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NC_011274.1, seqID NC_011274.1 [record index 0].\n",
      "2024-04-17 17:55:12,522 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP073715.1, seqID NZ_CP073715.1 [record index 0].\n",
      "2024-04-17 17:55:12,607 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP030794.1, seqID NZ_CP030794.1 [record index 0].\n",
      "2024-04-17 17:55:12,709 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP075174.1, seqID NZ_CP075174.1 [record index 0].\n",
      "2024-04-17 17:55:12,798 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011394.1, seqID NZ_CP011394.1 [record index 0].\n",
      "2024-04-17 17:55:12,884 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP038595.1, seqID NZ_CP038595.1 [record index 0].\n",
      "2024-04-17 17:55:12,979 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011288.1, seqID NZ_CP011288.1 [record index 0].\n",
      "2024-04-17 17:55:13,075 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011292.1, seqID NZ_CP011292.1 [record index 0].\n",
      "2024-04-17 17:55:13,171 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011289.1, seqID NZ_CP011289.1 [record index 0].\n",
      "2024-04-17 17:55:13,258 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP077760.1, seqID NZ_CP077760.1 [record index 0].\n",
      "2024-04-17 17:55:13,331 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067413.1, seqID NZ_CP067413.1 [record index 0].\n",
      "2024-04-17 17:55:13,401 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP054270.1, seqID NZ_CP054270.1 [record index 0].\n",
      "2024-04-17 17:55:13,431 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084803.1, seqID NZ_CP084803.1 [record index 0].\n",
      "2024-04-17 17:55:13,483 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081895.1, seqID NZ_CP081895.1 [record index 0].\n",
      "2024-04-17 17:55:13,534 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029432.1, seqID NZ_CP029432.1 [record index 0].\n",
      "2024-04-17 17:55:13,586 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029437.1, seqID NZ_CP029437.1 [record index 0].\n",
      "2024-04-17 17:55:13,637 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029443.1, seqID NZ_CP029443.1 [record index 0].\n",
      "2024-04-17 17:55:13,685 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070440.1, seqID NZ_CP070440.1 [record index 0].\n",
      "2024-04-17 17:55:13,737 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072468.1, seqID NZ_CP072468.1 [record index 0].\n",
      "2024-04-17 17:55:13,788 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070420.1, seqID NZ_CP070420.1 [record index 0].\n",
      "2024-04-17 17:55:13,838 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070454.1, seqID NZ_CP070454.1 [record index 0].\n",
      "2024-04-17 17:55:13,866 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP067966.1, seqID NZ_CP067966.1 [record index 0].\n",
      "2024-04-17 17:55:13,919 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026368.1, seqID NZ_CP026368.1 [record index 0].\n",
      "2024-04-17 17:55:13,970 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030171.1, seqID NZ_CP030171.1 [record index 0].\n",
      "2024-04-17 17:55:13,999 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084542.1, seqID NZ_CP084542.1 [record index 0].\n",
      "2024-04-17 17:55:14,031 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084415.1, seqID NZ_CP084415.1 [record index 0].\n",
      "2024-04-17 17:55:14,060 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP089315.1, seqID NZ_CP089315.1 [record index 0].\n",
      "2024-04-17 17:55:14,088 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023185.1, seqID NZ_CP023185.1 [record index 0].\n",
      "2024-04-17 17:55:14,118 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP022348.1, seqID NZ_CP022348.1 [record index 0].\n",
      "2024-04-17 17:55:14,151 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_018106.1, seqID NC_018106.1 [record index 0].\n",
      "2024-04-17 17:55:14,203 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081351.1, seqID NZ_CP081351.1 [record index 0].\n",
      "2024-04-17 17:55:14,261 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP048108.1, seqID NZ_CP048108.1 [record index 0].\n",
      "2024-04-17 17:55:14,321 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061930.1, seqID NZ_CP061930.1 [record index 0].\n",
      "2024-04-17 17:55:14,383 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP042361.1, seqID NZ_CP042361.1 [record index 0].\n",
      "2024-04-17 17:55:14,447 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP063425.1, seqID NZ_CP063425.1 [record index 0].\n",
      "2024-04-17 17:55:14,472 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045868.1, seqID NZ_CP045868.1 [record index 0].\n",
      "2024-04-17 17:55:14,496 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045869.1, seqID NZ_CP045869.1 [record index 0].\n",
      "2024-04-17 17:55:14,520 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045870.1, seqID NZ_CP045870.1 [record index 0].\n",
      "2024-04-17 17:55:14,544 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035466.1, seqID NZ_CP035466.1 [record index 0].\n",
      "2024-04-17 17:55:14,568 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP044214.1, seqID NZ_CP044214.1 [record index 0].\n",
      "2024-04-17 17:55:14,663 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057483.1, seqID NZ_CP057483.1 [record index 0].\n",
      "2024-04-17 17:55:14,757 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP059886.1, seqID NZ_CP059886.1 [record index 0].\n",
      "2024-04-17 17:55:14,844 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP060508.1, seqID NZ_CP060508.1 [record index 0].\n",
      "2024-04-17 17:55:14,913 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP067337.1, seqID NZ_CP067337.1 [record index 0].\n",
      "2024-04-17 17:55:15,001 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP080513.1, seqID NZ_CP080513.1 [record index 0].\n",
      "2024-04-17 17:55:15,104 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP082954.1, seqID NZ_CP082954.1 [record index 0].\n",
      "2024-04-17 17:55:15,177 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP006693.1, seqID NZ_CP006693.1 [record index 0].\n",
      "2024-04-17 17:55:15,271 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022142.1, seqID NZ_CP022142.1 [record index 0].\n",
      "2024-04-17 17:55:15,358 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP046291.1, seqID NZ_CP046291.1 [record index 0].\n",
      "2024-04-17 17:55:15,459 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017720.1, seqID NZ_CP017720.1 [record index 0].\n",
      "2024-04-17 17:55:15,556 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP019184.1, seqID NZ_CP019184.1 [record index 0].\n",
      "2024-04-17 17:55:15,646 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP022139.1, seqID NZ_CP022139.1 [record index 0].\n",
      "2024-04-17 17:55:15,711 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017450.1, seqID NZ_CP017450.1 [record index 0].\n",
      "2024-04-17 17:55:15,764 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026275.1, seqID NZ_CP026275.1 [record index 0].\n",
      "2024-04-17 17:55:15,824 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP018362.1, seqID NZ_CP018362.1 [record index 0].\n",
      "2024-04-17 17:55:15,878 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP035214.1, seqID NZ_CP035214.1 [record index 0].\n",
      "2024-04-17 17:55:15,930 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008841.1, seqID NZ_CP008841.1 [record index 0].\n",
      "2024-04-17 17:55:15,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_016612.1, seqID NC_016612.1 [record index 0].\n",
      "2024-04-17 17:55:16,037 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP093010.1, seqID NZ_CP093010.1 [record index 0].\n",
      "2024-04-17 17:55:16,093 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075881.1, seqID NZ_CP075881.1 [record index 0].\n",
      "2024-04-17 17:55:16,150 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP004887.1, seqID NZ_CP004887.1 [record index 0].\n",
      "2024-04-17 17:55:16,202 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042545.1, seqID NZ_CP042545.1 [record index 0].\n",
      "2024-04-17 17:55:16,256 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072119.1, seqID NZ_CP072119.1 [record index 0].\n",
      "2024-04-17 17:55:16,314 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP023529.1, seqID NZ_CP023529.1 [record index 0].\n",
      "2024-04-17 17:55:16,369 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP077236.1, seqID NZ_CP077236.1 [record index 0].\n",
      "2024-04-17 17:55:16,509 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP040889.1, seqID NZ_CP040889.1 [record index 0].\n",
      "2024-04-17 17:55:16,606 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP065411.1, seqID NZ_CP065411.1 [record index 0].\n",
      "2024-04-17 17:55:16,696 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP026285.1, seqID NZ_CP026285.1 [record index 0].\n",
      "2024-04-17 17:55:16,757 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP020358.1, seqID NZ_CP020358.1 [record index 0].\n",
      "2024-04-17 17:55:16,808 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026715.1, seqID NZ_CP026715.1 [record index 0].\n",
      "2024-04-17 17:55:16,860 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP022547.1, seqID NZ_AP022547.1 [record index 0].\n",
      "2024-04-17 17:55:16,914 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058212.1, seqID NZ_CP058212.1 [record index 0].\n",
      "2024-04-17 17:55:16,963 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065338.1, seqID NZ_CP065338.1 [record index 0].\n",
      "2024-04-17 17:55:17,017 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089407.1, seqID NZ_CP089407.1 [record index 0].\n",
      "2024-04-17 17:55:17,069 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041515.1, seqID NZ_CP041515.1 [record index 0].\n",
      "2024-04-17 17:55:17,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071393.1, seqID NZ_CP071393.1 [record index 0].\n",
      "2024-04-17 17:55:17,178 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP044109.1, seqID NZ_CP044109.1 [record index 0].\n",
      "2024-04-17 17:55:17,234 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP073305.1, seqID NZ_CP073305.1 [record index 0].\n",
      "2024-04-17 17:55:17,288 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029141.1, seqID NZ_CP029141.1 [record index 0].\n",
      "2024-04-17 17:55:17,341 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060111.1, seqID NZ_CP060111.1 [record index 0].\n",
      "2024-04-17 17:55:17,391 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089448.1, seqID NZ_CP089448.1 [record index 0].\n",
      "2024-04-17 17:55:17,444 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085764.1, seqID NZ_CP085764.1 [record index 0].\n",
      "2024-04-17 17:55:17,540 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP013990.1, seqID NZ_CP013990.1 [record index 0].\n",
      "2024-04-17 17:55:17,609 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP087280.1, seqID NZ_CP087280.1 [record index 0].\n",
      "2024-04-17 17:55:17,722 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP018016.1, seqID NZ_CP018016.1 [record index 0].\n",
      "2024-04-17 17:55:17,828 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP040392.1, seqID NZ_CP040392.1 [record index 0].\n",
      "2024-04-17 17:55:17,887 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP055325.1, seqID NZ_CP055325.1 [record index 0].\n",
      "2024-04-17 17:55:17,939 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP077673.1, seqID NZ_CP077673.1 [record index 0].\n",
      "2024-04-17 17:55:17,998 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP026269.1, seqID NZ_CP026269.1 [record index 0].\n",
      "2024-04-17 17:55:18,056 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008788.1, seqID NZ_CP008788.1 [record index 0].\n",
      "2024-04-17 17:55:18,107 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051427.1, seqID NZ_CP051427.1 [record index 0].\n",
      "2024-04-17 17:55:18,164 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP058118.1, seqID NZ_CP058118.1 [record index 0].\n",
      "2024-04-17 17:55:18,226 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091470.1, seqID NZ_CP091470.1 [record index 0].\n",
      "2024-04-17 17:55:18,280 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089395.1, seqID NZ_CP089395.1 [record index 0].\n",
      "2024-04-17 17:55:18,368 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP026387.1, seqID NZ_CP026387.1 [record index 0].\n",
      "2024-04-17 17:55:18,439 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP046251.1, seqID NZ_CP046251.1 [record index 0].\n",
      "2024-04-17 17:55:18,516 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043397.1, seqID NZ_CP043397.1 [record index 0].\n",
      "2024-04-17 17:55:18,586 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP042930.1, seqID NZ_CP042930.1 [record index 0].\n",
      "2024-04-17 17:55:18,660 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP043398.1, seqID NZ_CP043398.1 [record index 0].\n",
      "2024-04-17 17:55:18,751 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP083630.1, seqID NZ_CP083630.1 [record index 0].\n",
      "2024-04-17 17:55:18,824 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP042505.1, seqID NZ_CP042505.1 [record index 0].\n",
      "2024-04-17 17:55:18,884 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP049980.1, seqID NZ_CP049980.1 [record index 0].\n",
      "2024-04-17 17:55:18,966 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP036199.1, seqID NZ_CP036199.1 [record index 0].\n",
      "2024-04-17 17:55:19,022 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134333.1, seqID NZ_LR134333.1 [record index 0].\n",
      "2024-04-17 17:55:19,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056136.1, seqID NZ_CP056136.1 [record index 0].\n",
      "2024-04-17 17:55:19,133 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090078.1, seqID NZ_CP090078.1 [record index 0].\n",
      "2024-04-17 17:55:19,212 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP024643.1, seqID NZ_CP024643.1 [record index 0].\n",
      "2024-04-17 17:55:19,269 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065474.1, seqID NZ_CP065474.1 [record index 0].\n",
      "2024-04-17 17:55:19,340 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP042493.1, seqID NZ_CP042493.1 [record index 0].\n",
      "2024-04-17 17:55:19,413 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_AP022665.1, seqID NZ_AP022665.1 [record index 0].\n",
      "2024-04-17 17:55:19,516 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP056199.1, seqID NZ_CP056199.1 [record index 0].\n",
      "2024-04-17 17:55:19,589 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP015774.2, seqID NZ_CP015774.2 [record index 0].\n",
      "2024-04-17 17:55:19,642 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP061719.1, seqID NZ_CP061719.1 [record index 0].\n",
      "2024-04-17 17:55:19,691 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041247.1, seqID NZ_CP041247.1 [record index 0].\n",
      "2024-04-17 17:55:19,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069911.1, seqID NZ_CP069911.1 [record index 0].\n",
      "2024-04-17 17:55:19,825 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP015113.1, seqID NZ_CP015113.1 [record index 0].\n",
      "2024-04-17 17:55:19,895 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP045300.1, seqID NZ_CP045300.1 [record index 0].\n",
      "2024-04-17 17:55:19,921 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012264.1, seqID NZ_CP012264.1 [record index 0].\n",
      "2024-04-17 17:55:19,973 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046115.1, seqID NZ_CP046115.1 [record index 0].\n",
      "2024-04-17 17:55:20,025 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069943.1, seqID NZ_CP069943.1 [record index 0].\n",
      "2024-04-17 17:55:20,078 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070144.1, seqID NZ_CP070144.1 [record index 0].\n",
      "2024-04-17 17:55:20,125 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042499.1, seqID NZ_CP042499.1 [record index 0].\n",
      "2024-04-17 17:55:20,201 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP012268.1, seqID NZ_CP012268.1 [record index 0].\n",
      "2024-04-17 17:55:20,285 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP054058.1, seqID NZ_CP054058.1 [record index 0].\n",
      "2024-04-17 17:55:20,312 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP023184.1, seqID NZ_AP023184.1 [record index 0].\n",
      "2024-04-17 17:55:20,363 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR133932.1, seqID NZ_LR133932.1 [record index 0].\n",
      "2024-04-17 17:55:20,416 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089392.1, seqID NZ_CP089392.1 [record index 0].\n",
      "2024-04-17 17:55:20,472 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089399.1, seqID NZ_CP089399.1 [record index 0].\n",
      "2024-04-17 17:55:20,550 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP017662.1, seqID NZ_CP017662.1 [record index 0].\n",
      "2024-04-17 17:55:20,609 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP057330.1, seqID NZ_CP057330.1 [record index 0].\n",
      "2024-04-17 17:55:20,658 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR890312.1, seqID NZ_LR890312.1 [record index 0].\n",
      "2024-04-17 17:55:20,709 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP072914.1, seqID NZ_CP072914.1 [record index 0].\n",
      "2024-04-17 17:55:20,765 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090245.1, seqID NZ_CP090245.1 [record index 0].\n",
      "2024-04-17 17:55:20,818 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064108.1, seqID NZ_CP064108.1 [record index 0].\n",
      "2024-04-17 17:55:20,871 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033844.1, seqID NZ_CP033844.1 [record index 0].\n",
      "2024-04-17 17:55:20,922 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP070020.1, seqID NZ_CP070020.1 [record index 0].\n",
      "2024-04-17 17:55:20,979 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011618.1, seqID NZ_CP011618.1 [record index 0].\n",
      "2024-04-17 17:55:21,036 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP011597.1, seqID NZ_CP011597.1 [record index 0].\n",
      "2024-04-17 17:55:21,096 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP017928.1, seqID NZ_CP017928.1 [record index 0].\n",
      "2024-04-17 17:55:21,147 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029128.1, seqID NZ_CP029128.1 [record index 0].\n",
      "2024-04-17 17:55:21,212 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP040677.1, seqID NZ_CP040677.1 [record index 0].\n",
      "2024-04-17 17:55:21,260 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP016337.1, seqID NZ_CP016337.1 [record index 0].\n",
      "2024-04-17 17:55:21,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP089411.1, seqID NZ_CP089411.1 [record index 0].\n",
      "2024-04-17 17:55:21,363 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027426.1, seqID NZ_CP027426.1 [record index 0].\n",
      "2024-04-17 17:55:21,437 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP062916.1, seqID NZ_CP062916.1 [record index 0].\n",
      "2024-04-17 17:55:21,517 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP050508.1, seqID NZ_CP050508.1 [record index 0].\n",
      "2024-04-17 17:55:21,567 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP010557.1, seqID NZ_CP010557.1 [record index 0].\n",
      "2024-04-17 17:55:21,636 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP007215.2, seqID NZ_CP007215.2 [record index 0].\n",
      "2024-04-17 17:55:21,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP056567.1, seqID NZ_CP056567.1 [record index 0].\n",
      "2024-04-17 17:55:21,767 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_LR595855.1, seqID NZ_LR595855.1 [record index 0].\n",
      "2024-04-17 17:55:21,799 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP033076.1, seqID NZ_CP033076.1 [record index 0].\n",
      "2024-04-17 17:55:21,942 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP077291.1, seqID NZ_CP077291.1 [record index 0].\n",
      "2024-04-17 17:55:22,098 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP022695.1, seqID NZ_CP022695.1 [record index 0].\n",
      "2024-04-17 17:55:22,154 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014136.1, seqID NZ_CP014136.1 [record index 0].\n",
      "2024-04-17 17:55:22,214 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012266.1, seqID NZ_CP012266.1 [record index 0].\n",
      "2024-04-17 17:55:22,242 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP009450.1, seqID NZ_CP009450.1 [record index 0].\n",
      "2024-04-17 17:55:22,266 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR699009.1, seqID NZ_LR699009.1 [record index 0].\n",
      "2024-04-17 17:55:22,295 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP020388.1, seqID NZ_CP020388.1 [record index 0].\n",
      "2024-04-17 17:55:22,397 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP011602.1, seqID NZ_CP011602.1 [record index 0].\n",
      "2024-04-17 17:55:22,453 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP025334.1, seqID NZ_AP025334.1 [record index 0].\n",
      "2024-04-17 17:55:22,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018756.1, seqID NZ_AP018756.1 [record index 0].\n",
      "2024-04-17 17:55:22,531 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063427.1, seqID NZ_CP063427.1 [record index 0].\n",
      "2024-04-17 17:55:22,580 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051548.1, seqID NZ_CP051548.1 [record index 0].\n",
      "2024-04-17 17:55:22,649 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP034225.1, seqID NZ_CP034225.1 [record index 0].\n",
      "2024-04-17 17:55:22,715 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP019445.1, seqID NZ_CP019445.1 [record index 0].\n",
      "2024-04-17 17:55:22,852 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP025757.1, seqID NZ_CP025757.1 [record index 0].\n",
      "2024-04-17 17:55:22,956 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069319.1, seqID NZ_CP069319.1 [record index 0].\n",
      "2024-04-17 17:55:23,050 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP035129.1, seqID NZ_CP035129.1 [record index 0].\n",
      "2024-04-17 17:55:23,101 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP027225.1, seqID NZ_CP027225.1 [record index 0].\n",
      "2024-04-17 17:55:23,165 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP051488.1, seqID NZ_CP051488.1 [record index 0].\n",
      "2024-04-17 17:55:23,263 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057611.1, seqID NZ_CP057611.1 [record index 0].\n",
      "2024-04-17 17:55:23,366 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057620.1, seqID NZ_CP057620.1 [record index 0].\n",
      "2024-04-17 17:55:23,472 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP071070.1, seqID NZ_CP071070.1 [record index 0].\n",
      "2024-04-17 17:55:23,502 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050811.1, seqID NZ_CP050811.1 [record index 0].\n",
      "2024-04-17 17:55:23,636 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP014015.2, seqID NZ_CP014015.2 [record index 0].\n",
      "2024-04-17 17:55:23,781 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NC_013716.1, seqID NC_013716.1 [record index 0].\n",
      "2024-04-17 17:55:23,928 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP082833.1, seqID NZ_CP082833.1 [record index 0].\n",
      "2024-04-17 17:55:24,077 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP011132.1, seqID NZ_CP011132.1 [record index 0].\n",
      "2024-04-17 17:55:24,244 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP045205.1, seqID NZ_CP045205.1 [record index 0].\n",
      "2024-04-17 17:55:24,335 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP069159.1, seqID NZ_CP069159.1 [record index 0].\n",
      "2024-04-17 17:55:24,400 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP028974.1, seqID NZ_CP028974.1 [record index 0].\n",
      "2024-04-17 17:55:24,459 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NC_020260.1, seqID NC_020260.1 [record index 0].\n",
      "2024-04-17 17:55:24,516 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP012253.1, seqID NZ_CP012253.1 [record index 0].\n",
      "2024-04-17 17:55:24,578 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP027107.1, seqID NZ_CP027107.1 [record index 0].\n",
      "2024-04-17 17:55:24,642 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP011047.1, seqID NZ_CP011047.1 [record index 0].\n",
      "2024-04-17 17:55:24,794 [INFO - chronostrain.cli.make_db_json] - Created 7 merged markers for strain NZ_CP081314.1, seqID NZ_CP081314.1 [record index 0].\n",
      "2024-04-17 17:55:24,940 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP026216.1, seqID NZ_CP026216.1 [record index 0].\n",
      "2024-04-17 17:55:24,999 [INFO - chronostrain.cli.make_db_json] - Created 3 merged markers for strain NZ_CP049154.1, seqID NZ_CP049154.1 [record index 0].\n",
      "2024-04-17 17:55:25,125 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP050009.1, seqID NZ_CP050009.1 [record index 0].\n",
      "2024-04-17 17:55:25,250 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP057150.1, seqID NZ_CP057150.1 [record index 0].\n",
      "2024-04-17 17:55:25,357 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057630.1, seqID NZ_CP057630.1 [record index 0].\n",
      "2024-04-17 17:55:25,466 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP057632.1, seqID NZ_CP057632.1 [record index 0].\n",
      "2024-04-17 17:55:25,573 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_AP024585.1, seqID NZ_AP024585.1 [record index 0].\n",
      "2024-04-17 17:55:25,724 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_CP014070.2, seqID NZ_CP014070.2 [record index 0].\n",
      "2024-04-17 17:55:25,833 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP083651.1, seqID NZ_CP083651.1 [record index 0].\n",
      "2024-04-17 17:55:25,942 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP064180.1, seqID NZ_CP064180.1 [record index 0].\n",
      "2024-04-17 17:55:26,049 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_CP080958.1, seqID NZ_CP080958.1 [record index 0].\n",
      "2024-04-17 17:55:26,164 [INFO - chronostrain.cli.make_db_json] - Created 5 merged markers for strain NZ_LT556085.1, seqID NZ_LT556085.1 [record index 0].\n",
      "2024-04-17 17:55:26,294 [INFO - chronostrain.cli.make_db_json] - Created 6 merged markers for strain NZ_LT556084.1, seqID NZ_LT556084.1 [record index 0].\n",
      "2024-04-17 17:55:26,378 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP041362.1, seqID NZ_CP041362.1 [record index 0].\n",
      "2024-04-17 17:55:26,461 [INFO - chronostrain.cli.make_db_json] - Created 4 merged markers for strain NZ_CP056155.1, seqID NZ_CP056155.1 [record index 0].\n",
      "2024-04-17 17:55:28,703 [INFO - chronostrain.cli.make_db_json] - Database JSON file created: /mnt/e/ecoli_db/chronostrain_files/ecoli.json\n"
     ]
    }
   ],
   "source": [
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      make-db \\\n",
    "      -m $MARKER_SEED_INDEX \\\n",
    "      -r $REFSEQ_INDEX \\\n",
    "      -b $BLAST_DB_NAME -bd $BLAST_DB_DIR \\\n",
    "      --min-pct-idty $MIN_PCT_IDTY \\\n",
    "      -o $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      --threads $NUM_CORES"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "8ee37d1d-3531-4ab0-a279-7f848d2b342b",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-04-17 18:05:58,241 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2024-04-17 18:05:58,272 [INFO - chronostrain.cli.prune_json] - Pruning database via clustering\n",
      "2024-04-17 18:05:58,272 [DEBUG - chronostrain.cli.prune_json] - Src: /mnt/e/ecoli_db/chronostrain_files/ecoli.json, Output: /mnt/e/ecoli_db/chronostrain_files/ecoli.clusters.txt\n",
      "2024-04-17 18:05:58,272 [INFO - chronostrain.cli.prune_json] - Target identity threshold = 0.998\n",
      "2024-04-17 18:05:59,688 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2024-04-17 18:05:59,969 [INFO - chronostrain.cli.prune_json] - Preprocessing step -- Loading old DB instance, using data directory: /mnt/e/ecoli_db/chronostrain_files\n",
      "2024-04-17 18:05:59,969 [DEBUG - chronostrain.database.parser.json] - Couldn't find instance (/mnt/e/ecoli_db/chronostrain_files/__ecoli_/database.posix.pkl).\n",
      "2024-04-17 18:05:59,969 [INFO - chronostrain.database.parser.json] - Loading from JSON marker database file /mnt/e/ecoli_db/chronostrain_files/ecoli.json.\n",
      "2024-04-17 18:06:01,005 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026762.1 (Shigella boydii, NCTC_9850) loaded with 57 markers.\n",
      "2024-04-17 18:06:01,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041513.1 (Shigella boydii, KCCM_41690) loaded with 64 markers.\n",
      "2024-04-17 18:06:01,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068090.1 (Shigella boydii, FDAARGOS_1139) loaded with 67 markers.\n",
      "2024-04-17 18:06:01,433 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026795.1 (Shigella boydii, ATCC_BAA-1247) loaded with 63 markers.\n",
      "2024-04-17 18:06:01,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011511.1 (Shigella boydii, ATCC_9210) loaded with 68 markers.\n",
      "2024-04-17 18:06:01,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026814.1 (Shigella boydii, 83-578) loaded with 61 markers.\n",
      "2024-04-17 18:06:01,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049285.1 (Shigella boydii, 602068) loaded with 68 markers.\n",
      "2024-04-17 18:06:01,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049281.1 (Shigella boydii, 600657) loaded with 64 markers.\n",
      "2024-04-17 18:06:02,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026766.1 (Shigella boydii, 59-248) loaded with 62 markers.\n",
      "2024-04-17 18:06:02,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024056.1 (Escherichia coli, SMN197SH3) loaded with 72 markers.\n",
      "2024-04-17 18:06:02,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024618.1 (Escherichia coli, SMN152SH1) loaded with 72 markers.\n",
      "2024-04-17 18:06:02,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023673.1 (Escherichia coli, SMN013SH2) loaded with 72 markers.\n",
      "2024-04-17 18:06:02,548 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR536431.1 (Escherichia coli, f9610206-5e81-11e8-bf7f-3c4a9275d6c8) loaded with 88 markers.\n",
      "2024-04-17 18:06:02,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027597.1 (Escherichia coli, 86-3153) loaded with 74 markers.\n",
      "2024-04-17 18:06:02,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027591.1 (Escherichia coli, 2014C-3011) loaded with 72 markers.\n",
      "2024-04-17 18:06:02,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027587.1 (Escherichia coli, 2013C-4974) loaded with 76 markers.\n",
      "2024-04-17 18:06:03,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055292.1 (Shigella sonnei, SE6-1) loaded with 71 markers.\n",
      "2024-04-17 18:06:03,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081813.1 (Shigella flexneri, WW1) loaded with 76 markers.\n",
      "2024-04-17 18:06:03,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055099.1 (Shigella flexneri, SWHIN_107) loaded with 77 markers.\n",
      "2024-04-17 18:06:03,399 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055109.1 (Shigella flexneri, SWHIN_104) loaded with 77 markers.\n",
      "2024-04-17 18:06:03,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055015.1 (Shigella flexneri, SWHEFF_71) loaded with 64 markers.\n",
      "2024-04-17 18:06:03,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055024.1 (Shigella flexneri, SWHEFF_67) loaded with 65 markers.\n",
      "2024-04-17 18:06:03,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055030.1 (Shigella flexneri, SWHEFF_64) loaded with 77 markers.\n",
      "2024-04-17 18:06:03,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058790.1 (Shigella flexneri, STLIN_6) loaded with 70 markers.\n",
      "2024-04-17 18:06:03,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058828.1 (Shigella flexneri, STLE3) loaded with 82 markers.\n",
      "2024-04-17 18:06:04,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055167.1 (Shigella flexneri, STEFF_24) loaded with 72 markers.\n",
      "2024-04-17 18:06:04,166 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055170.1 (Shigella flexneri, STEFF_22) loaded with 69 markers.\n",
      "2024-04-17 18:06:04,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055179.1 (Shigella flexneri, STEFF_18) loaded with 72 markers.\n",
      "2024-04-17 18:06:04,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055181.1 (Shigella flexneri, STEFF_17) loaded with 85 markers.\n",
      "2024-04-17 18:06:04,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055184.1 (Shigella flexneri, STEFF_16) loaded with 69 markers.\n",
      "2024-04-17 18:06:04,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055219.1 (Shigella flexneri, STEFF_12) loaded with 78 markers.\n",
      "2024-04-17 18:06:04,710 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055194.1 (Shigella flexneri, STEFF_10) loaded with 71 markers.\n",
      "2024-04-17 18:06:04,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042980.1 (Shigella flexneri, PE577) loaded with 71 markers.\n",
      "2024-04-17 18:06:05,054 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026799.1 (Shigella flexneri, NCTC_9728) loaded with 71 markers.\n",
      "2024-04-17 18:06:05,239 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037923.1 (Shigella flexneri, M90T) loaded with 72 markers.\n",
      "2024-04-17 18:06:05,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054913.1 (Shigella flexneri, FDAARGOS_691) loaded with 70 markers.\n",
      "2024-04-17 18:06:05,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR213452.1 (Shigella flexneri, AUSMDU00008355) loaded with 70 markers.\n",
      "2024-04-17 18:06:05,539 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR213455.1 (Shigella flexneri, AUSMDU00008332) loaded with 71 markers.\n",
      "2024-04-17 18:06:05,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012137.1 (Shigella flexneri, 981) loaded with 71 markers.\n",
      "2024-04-17 18:06:05,738 [DEBUG - chronostrain.database.parser.json] - Strain NC_008258.1 (Shigella flexneri, 8401) loaded with 72 markers.\n",
      "2024-04-17 18:06:05,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR878365.1 (Shigella flexneri, 83) loaded with 68 markers.\n",
      "2024-04-17 18:06:05,944 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026792.1 (Shigella flexneri, 61-4982) loaded with 70 markers.\n",
      "2024-04-17 18:06:06,050 [DEBUG - chronostrain.database.parser.json] - Strain NC_004337.2 (Shigella flexneri, 301) loaded with 71 markers.\n",
      "2024-04-17 18:06:06,160 [DEBUG - chronostrain.database.parser.json] - Strain NC_004741.1 (Shigella flexneri, 2457T) loaded with 70 markers.\n",
      "2024-04-17 18:06:06,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP004056.1 (Shigella flexneri, 2003036) loaded with 71 markers.\n",
      "2024-04-17 18:06:06,391 [DEBUG - chronostrain.database.parser.json] - Strain NC_017328.1 (Shigella flexneri, 2002017) loaded with 71 markers.\n",
      "2024-04-17 18:06:06,500 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012735.1 (Shigella flexneri, 0228) loaded with 72 markers.\n",
      "2024-04-17 18:06:06,601 [DEBUG - chronostrain.database.parser.json] - Strain NC_010658.1 (Shigella boydii, BS512__CDC_3083-94) loaded with 64 markers.\n",
      "2024-04-17 18:06:06,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051430.1 (Escherichia sp., SCLE84) loaded with 70 markers.\n",
      "2024-04-17 18:06:06,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040927.1 (Escherichia coli, YY76-1) loaded with 71 markers.\n",
      "2024-04-17 18:06:06,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023235.1 (Escherichia coli, YJ6) loaded with 74 markers.\n",
      "2024-04-17 18:06:07,079 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023224.1 (Escherichia coli, YJ1) loaded with 74 markers.\n",
      "2024-04-17 18:06:07,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013112.1 (Escherichia coli, YD786) loaded with 72 markers.\n",
      "2024-04-17 18:06:07,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019361.1 (Escherichia coli, XH993) loaded with 71 markers.\n",
      "2024-04-17 18:06:07,409 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019359.1 (Escherichia coli, XH990) loaded with 77 markers.\n",
      "2024-04-17 18:06:07,514 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019355.1 (Escherichia coli, XH988) loaded with 71 markers.\n",
      "2024-04-17 18:06:07,620 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035819.1 (Escherichia coli, WS3294A) loaded with 65 markers.\n",
      "2024-04-17 18:06:07,718 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022120.1 (Escherichia coli, WP5-W18-ESBL-11) loaded with 58 markers.\n",
      "2024-04-17 18:06:07,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022107.1 (Escherichia coli, WP5-S17-ESBL-11) loaded with 64 markers.\n",
      "2024-04-17 18:06:07,929 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022095.1 (Escherichia coli, WP4-S18-ESBL-08) loaded with 75 markers.\n",
      "2024-04-17 18:06:08,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022036.1 (Escherichia coli, WP3-S18-ESBL-09) loaded with 71 markers.\n",
      "2024-04-17 18:06:08,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021914.1 (Escherichia coli, WP2-W18-CRE-03) loaded with 56 markers.\n",
      "2024-04-17 18:06:08,263 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021944.1 (Escherichia coli, WP2-S18-ESBL-07) loaded with 75 markers.\n",
      "2024-04-17 18:06:08,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043334.1 (Escherichia coli, WCHEC115106) loaded with 74 markers.\n",
      "2024-04-17 18:06:08,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019213.3 (Escherichia coli, WCHEC050613) loaded with 67 markers.\n",
      "2024-04-17 18:06:08,546 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043274.1 (Escherichia coli, WCHEC035148) loaded with 55 markers.\n",
      "2024-04-17 18:06:08,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029369.1 (Escherichia coli, WCHEC035148) loaded with 55 markers.\n",
      "2024-04-17 18:06:08,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058656.1 (Escherichia coli, WCHEC035125) loaded with 55 markers.\n",
      "2024-04-17 18:06:08,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058651.1 (Escherichia coli, WCHEC035123) loaded with 55 markers.\n",
      "2024-04-17 18:06:08,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043284.1 (Escherichia coli, WCHEC025985) loaded with 86 markers.\n",
      "2024-04-17 18:06:09,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027205.2 (Escherichia coli, WCHEC025943) loaded with 74 markers.\n",
      "2024-04-17 18:06:09,131 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034966.1 (Escherichia coli, WCHEC020032) loaded with 56 markers.\n",
      "2024-04-17 18:06:09,239 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033401.1 (Escherichia coli, WCHEC020031) loaded with 75 markers.\n",
      "2024-04-17 18:06:09,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012380.1 (Escherichia coli, WAT) loaded with 74 markers.\n",
      "2024-04-17 18:06:09,472 [DEBUG - chronostrain.database.parser.json] - Strain NC_017664.1 (Escherichia coli, W) loaded with 74 markers.\n",
      "2024-04-17 18:06:09,573 [DEBUG - chronostrain.database.parser.json] - Strain NC_017635.1 (Escherichia coli, W) loaded with 74 markers.\n",
      "2024-04-17 18:06:09,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085488.1 (Escherichia coli, VCSW8) loaded with 73 markers.\n",
      "2024-04-17 18:06:09,763 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU342919.1 (Escherichia coli, UR21_11-869) loaded with 71 markers.\n",
      "2024-04-17 18:06:09,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031653.1 (Escherichia coli, UK_Dog_Liverpool) loaded with 55 markers.\n",
      "2024-04-17 18:06:09,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068035.1 (Escherichia coli, U90) loaded with 77 markers.\n",
      "2024-04-17 18:06:10,089 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035841.1 (Escherichia coli, TW10722) loaded with 75 markers.\n",
      "2024-04-17 18:06:10,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035846.1 (Escherichia coli, TW10598) loaded with 71 markers.\n",
      "2024-04-17 18:06:10,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023205.1 (Escherichia coli, TUM18781) loaded with 61 markers.\n",
      "2024-04-17 18:06:10,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023197.1 (Escherichia coli, TUM18780) loaded with 75 markers.\n",
      "2024-04-17 18:06:10,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023190.1 (Escherichia coli, TUM18530) loaded with 75 markers.\n",
      "2024-04-17 18:06:10,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033605.1 (Escherichia coli, TR01) loaded with 80 markers.\n",
      "2024-04-17 18:06:10,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022540.1 (Escherichia coli, THO-010) loaded with 85 markers.\n",
      "2024-04-17 18:06:10,858 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035860.1 (Escherichia coli, TD225-C4) loaded with 77 markers.\n",
      "2024-04-17 18:06:10,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075061.1 (Escherichia coli, TCM_B_2) loaded with 73 markers.\n",
      "2024-04-17 18:06:11,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053597.1 (Escherichia coli, T7Express_LysY) loaded with 76 markers.\n",
      "2024-04-17 18:06:11,135 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053594.1 (Escherichia coli, T7Express_Crystal) loaded with 76 markers.\n",
      "2024-04-17 18:06:11,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046716.1 (Escherichia coli, T16R) loaded with 71 markers.\n",
      "2024-04-17 18:06:11,342 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059283.1 (Escherichia coli, SY3626_hybrid) loaded with 79 markers.\n",
      "2024-04-17 18:06:11,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078137.1 (Escherichia coli, Sx181-0-1) loaded with 71 markers.\n",
      "2024-04-17 18:06:11,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091773.1 (Escherichia coli, SWYH.B349aT) loaded with 75 markers.\n",
      "2024-04-17 18:06:11,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076305.1 (Escherichia coli, Survcare253) loaded with 82 markers.\n",
      "2024-04-17 18:06:11,755 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022409.1 (Escherichia coli, STW0522-31) loaded with 67 markers.\n",
      "2024-04-17 18:06:11,831 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054169.1 (Escherichia coli, STO_Bone1B) loaded with 43 markers.\n",
      "2024-04-17 18:06:11,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022482.1 (Escherichia coli, STN0717-20) loaded with 49 markers.\n",
      "2024-04-17 18:06:12,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041411.1 (Escherichia coli, STEC719) loaded with 73 markers.\n",
      "2024-04-17 18:06:12,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041416.1 (Escherichia coli, STEC711) loaded with 70 markers.\n",
      "2024-04-17 18:06:12,506 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041425.1 (Escherichia coli, STEC388) loaded with 70 markers.\n",
      "2024-04-17 18:06:13,020 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091026.1 (Escherichia coli, STEC383) loaded with 96 markers.\n",
      "2024-04-17 18:06:13,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041429.1 (Escherichia coli, STEC367) loaded with 71 markers.\n",
      "2024-04-17 18:06:13,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091044.1 (Escherichia coli, STEC346) loaded with 59 markers.\n",
      "2024-04-17 18:06:13,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041431.1 (Escherichia coli, STEC316) loaded with 73 markers.\n",
      "2024-04-17 18:06:13,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041433.1 (Escherichia coli, STEC313) loaded with 71 markers.\n",
      "2024-04-17 18:06:13,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041435.1 (Escherichia coli, STEC309) loaded with 73 markers.\n",
      "2024-04-17 18:06:13,718 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091020.1 (Escherichia coli, STEC308) loaded with 74 markers.\n",
      "2024-04-17 18:06:13,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091018.1 (Escherichia coli, STEC307) loaded with 74 markers.\n",
      "2024-04-17 18:06:13,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091016.1 (Escherichia coli, STEC306) loaded with 74 markers.\n",
      "2024-04-17 18:06:14,036 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091022.1 (Escherichia coli, STEC276) loaded with 51 markers.\n",
      "2024-04-17 18:06:14,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075697.1 (Escherichia coli, STEC2018-607) loaded with 73 markers.\n",
      "2024-04-17 18:06:14,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075665.1 (Escherichia coli, STEC2018-553) loaded with 75 markers.\n",
      "2024-04-17 18:06:14,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075663.1 (Escherichia coli, STEC2017-197) loaded with 75 markers.\n",
      "2024-04-17 18:06:14,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075652.1 (Escherichia coli, STEC17-183) loaded with 74 markers.\n",
      "2024-04-17 18:06:14,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077646.1 (Escherichia coli, ST866) loaded with 76 markers.\n",
      "2024-04-17 18:06:14,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077649.1 (Escherichia coli, ST865) loaded with 76 markers.\n",
      "2024-04-17 18:06:14,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007391.1 (Escherichia coli, ST540) loaded with 48 markers.\n",
      "2024-04-17 18:06:14,954 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007390.1 (Escherichia coli, ST540) loaded with 48 markers.\n",
      "2024-04-17 18:06:15,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007265.1 (Escherichia coli, ST540) loaded with 48 markers.\n",
      "2024-04-17 18:06:15,168 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031321.1 (Escherichia coli, ST2350) loaded with 72 markers.\n",
      "2024-04-17 18:06:15,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043478.1 (Escherichia coli, ST130) loaded with 76 markers.\n",
      "2024-04-17 18:06:15,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061914.1 (Escherichia coli, SMBL) loaded with 72 markers.\n",
      "2024-04-17 18:06:15,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043486.1 (Escherichia coli, SL112) loaded with 74 markers.\n",
      "2024-04-17 18:06:15,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044315.1 (Escherichia coli, SJ7) loaded with 74 markers.\n",
      "2024-04-17 18:06:15,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024130.1 (Escherichia coli, SI-NP020) loaded with 72 markers.\n",
      "2024-04-17 18:06:16,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051219.1 (Escherichia coli, SFE8) loaded with 72 markers.\n",
      "2024-04-17 18:06:16,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007594.1 (Escherichia coli, SEC470) loaded with 74 markers.\n",
      "2024-04-17 18:06:16,289 [DEBUG - chronostrain.database.parser.json] - Strain NC_011415.1 (Escherichia coli, SE11) loaded with 75 markers.\n",
      "2024-04-17 18:06:16,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029692.1 (Escherichia coli, SD134209) loaded with 69 markers.\n",
      "2024-04-17 18:06:16,691 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051222.1 (Escherichia coli, SCZE5) loaded with 71 markers.\n",
      "2024-04-17 18:06:16,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054371.1 (Escherichia coli, SCU-316) loaded with 78 markers.\n",
      "2024-04-17 18:06:16,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053281.1 (Escherichia coli, SCU-308) loaded with 57 markers.\n",
      "2024-04-17 18:06:17,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051698.1 (Escherichia coli, SCU-152) loaded with 73 markers.\n",
      "2024-04-17 18:06:17,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086259.1 (Escherichia coli, SCPM-O-B-9427) loaded with 76 markers.\n",
      "2024-04-17 18:06:17,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034958.1 (Escherichia coli, SCEC020026) loaded with 73 markers.\n",
      "2024-04-17 18:06:17,583 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032892.1 (Escherichia coli, SCEC020022) loaded with 75 markers.\n",
      "2024-04-17 18:06:17,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032426.1 (Escherichia coli, SCEC020001) loaded with 72 markers.\n",
      "2024-04-17 18:06:17,783 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025048.1 (Escherichia coli, SC516) loaded with 72 markers.\n",
      "2024-04-17 18:06:17,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778149.1 (Escherichia coli, SC487) loaded with 72 markers.\n",
      "2024-04-17 18:06:17,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778151.1 (Escherichia coli, SC476) loaded with 68 markers.\n",
      "2024-04-17 18:06:18,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778148.1 (Escherichia coli, SC475) loaded with 71 markers.\n",
      "2024-04-17 18:06:18,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR782232.1 (Escherichia coli, SC467) loaded with 70 markers.\n",
      "2024-04-17 18:06:18,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778145.1 (Escherichia coli, SC423) loaded with 71 markers.\n",
      "2024-04-17 18:06:18,464 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778143.1 (Escherichia coli, SC422) loaded with 75 markers.\n",
      "2024-04-17 18:06:18,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778141.1 (Escherichia coli, SC407) loaded with 75 markers.\n",
      "2024-04-17 18:06:18,699 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778140.1 (Escherichia coli, SC406) loaded with 75 markers.\n",
      "2024-04-17 18:06:18,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067232.1 (Escherichia coli, SBF45) loaded with 71 markers.\n",
      "2024-04-17 18:06:18,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007592.1 (Escherichia coli, Santai) loaded with 77 markers.\n",
      "2024-04-17 18:06:19,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010242.1 (Escherichia coli, S56) loaded with 72 markers.\n",
      "2024-04-17 18:06:19,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015995.1 (Escherichia coli, S51) loaded with 77 markers.\n",
      "2024-04-17 18:06:19,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010235.1 (Escherichia coli, S40) loaded with 72 markers.\n",
      "2024-04-17 18:06:19,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010231.1 (Escherichia coli, S30) loaded with 75 markers.\n",
      "2024-04-17 18:06:19,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010228.1 (Escherichia coli, S3) loaded with 73 markers.\n",
      "2024-04-17 18:06:19,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010230.1 (Escherichia coli, S21) loaded with 73 markers.\n",
      "2024-04-17 18:06:20,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075852.1 (Escherichia coli, S19-651) loaded with 73 markers.\n",
      "2024-04-17 18:06:20,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075640.1 (Escherichia coli, S19-141) loaded with 73 markers.\n",
      "2024-04-17 18:06:20,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058725.1 (Escherichia coli, S171-1) loaded with 70 markers.\n",
      "2024-04-17 18:06:20,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058709.1 (Escherichia coli, S163-2) loaded with 69 markers.\n",
      "2024-04-17 18:06:20,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010229.1 (Escherichia coli, S10) loaded with 74 markers.\n",
      "2024-04-17 18:06:20,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048010.1 (Escherichia coli, RUT3575) loaded with 74 markers.\n",
      "2024-04-17 18:06:21,045 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083274.1 (Escherichia coli, Rosetta2-DE3) loaded with 80 markers.\n",
      "2024-04-17 18:06:21,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028432.1 (Escherichia coli, RM9975) loaded with 67 markers.\n",
      "2024-04-17 18:06:21,380 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009104.1 (Escherichia coli, RM9387) loaded with 74 markers.\n",
      "2024-04-17 18:06:21,510 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028117.1 (Escherichia coli, RM9322) loaded with 67 markers.\n",
      "2024-04-17 18:06:21,647 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044314.1 (Escherichia coli, RM9245) loaded with 75 markers.\n",
      "2024-04-17 18:06:21,775 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028429.1 (Escherichia coli, RM9131) loaded with 67 markers.\n",
      "2024-04-17 18:06:21,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042298.1 (Escherichia coli, RM9088) loaded with 77 markers.\n",
      "2024-04-17 18:06:22,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043825.1 (Escherichia coli, RM8082) loaded with 76 markers.\n",
      "2024-04-17 18:06:22,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015244.1 (Escherichia coli, RM7190) loaded with 75 markers.\n",
      "2024-04-17 18:06:22,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027105.1 (Escherichia coli, RM14721) loaded with 73 markers.\n",
      "2024-04-17 18:06:22,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044311.1 (Escherichia coli, RM13752) loaded with 74 markers.\n",
      "2024-04-17 18:06:22,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044312.1 (Escherichia coli, RM13745) loaded with 74 markers.\n",
      "2024-04-17 18:06:22,808 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050498.1 (Escherichia coli, RM13322) loaded with 74 markers.\n",
      "2024-04-17 18:06:22,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044313.1 (Escherichia coli, RM11911) loaded with 74 markers.\n",
      "2024-04-17 18:06:23,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023165.1 (Escherichia coli, RM10809-3) loaded with 77 markers.\n",
      "2024-04-17 18:06:23,242 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028381.1 (Escherichia coli, RM10466) loaded with 77 markers.\n",
      "2024-04-17 18:06:23,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028122.1 (Escherichia coli, RM10042) loaded with 76 markers.\n",
      "2024-04-17 18:06:23,473 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068803.1 (Escherichia coli, RIVM_C018404) loaded with 49 markers.\n",
      "2024-04-17 18:06:23,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062855.1 (Escherichia coli, Res13-Lact-PEB17-18) loaded with 75 markers.\n",
      "2024-04-17 18:06:23,755 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062863.1 (Escherichia coli, Res13-Lact-PEB14-36) loaded with 80 markers.\n",
      "2024-04-17 18:06:23,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062871.1 (Escherichia coli, Res13-Lact-PEA14-20) loaded with 65 markers.\n",
      "2024-04-17 18:06:24,036 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062908.1 (Escherichia coli, Res13-Lact-EB12-05) loaded with 80 markers.\n",
      "2024-04-17 18:06:24,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062910.1 (Escherichia coli, Res13-Lact-EA06-22) loaded with 71 markers.\n",
      "2024-04-17 18:06:24,265 [DEBUG - chronostrain.database.parser.json] - Strain NC_012967.1 (Escherichia coli, REL606) loaded with 76 markers.\n",
      "2024-04-17 18:06:24,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034106.1 (Escherichia coli, RCAD0514) loaded with 78 markers.\n",
      "2024-04-17 18:06:24,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031706.1 (Escherichia coli, R1) loaded with 73 markers.\n",
      "2024-04-17 18:06:24,672 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR883965.1 (Escherichia coli, Quinolone_resistant) loaded with 73 markers.\n",
      "2024-04-17 18:06:24,837 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882493.1 (Escherichia coli, Quinolone_resistant) loaded with 71 markers.\n",
      "2024-04-17 18:06:24,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048821.1 (Escherichia coli, QEC11-421) loaded with 73 markers.\n",
      "2024-04-17 18:06:25,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053211.1 (Escherichia coli, QDFD216) loaded with 71 markers.\n",
      "2024-04-17 18:06:25,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041031.1 (Escherichia coli, PT109) loaded with 72 markers.\n",
      "2024-04-17 18:06:25,419 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080126.1 (Escherichia coli, PK8568) loaded with 72 markers.\n",
      "2024-04-17 18:06:25,577 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080172.1 (Escherichia coli, PK8566) loaded with 73 markers.\n",
      "2024-04-17 18:06:25,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080146.1 (Escherichia coli, PK8276) loaded with 68 markers.\n",
      "2024-04-17 18:06:25,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080370.1 (Escherichia coli, PK5086) loaded with 74 markers.\n",
      "2024-04-17 18:06:26,003 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072802.1 (Escherichia coli, PK5074) loaded with 70 markers.\n",
      "2024-04-17 18:06:26,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039298.1 (Escherichia coli, PigCaeca_1) loaded with 77 markers.\n",
      "2024-04-17 18:06:26,189 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082120.1 (Escherichia coli, PC-NDM34) loaded with 69 markers.\n",
      "2024-04-17 18:06:26,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048604.1 (Escherichia coli, PapRG-06-3) loaded with 71 markers.\n",
      "2024-04-17 18:06:26,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050646.1 (Escherichia coli, PapM-36-2) loaded with 71 markers.\n",
      "2024-04-17 18:06:26,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085069.1 (Escherichia coli, P27_L1_05.20) loaded with 70 markers.\n",
      "2024-04-17 18:06:26,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044293.1 (Escherichia coli, P276M) loaded with 71 markers.\n",
      "2024-04-17 18:06:26,698 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085056.1 (Escherichia coli, P24_WM1_05.20) loaded with 71 markers.\n",
      "2024-04-17 18:06:26,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075627.1 (Escherichia coli, P15-385) loaded with 77 markers.\n",
      "2024-04-17 18:06:26,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045190.1 (Escherichia coli, NT1F31) loaded with 90 markers.\n",
      "2024-04-17 18:06:27,061 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024126.1 (Escherichia coli, NS-NP030) loaded with 71 markers.\n",
      "2024-04-17 18:06:27,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075374.1 (Escherichia coli, no107) loaded with 72 markers.\n",
      "2024-04-17 18:06:27,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024561.1 (Escherichia coli, NIPH17_0036) loaded with 74 markers.\n",
      "2024-04-17 18:06:27,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024560.1 (Escherichia coli, NIPH17_0020) loaded with 74 markers.\n",
      "2024-04-17 18:06:27,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053600.1 (Escherichia coli, Nico21(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:27,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053592.1 (Escherichia coli, NEBExpress_Iq) loaded with 76 markers.\n",
      "2024-04-17 18:06:27,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053599.1 (Escherichia coli, NEBExpress) loaded with 76 markers.\n",
      "2024-04-17 18:06:28,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083888.1 (Escherichia coli, NDM1) loaded with 75 markers.\n",
      "2024-04-17 18:06:28,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483303.1 (Escherichia coli, NCTC9967) loaded with 75 markers.\n",
      "2024-04-17 18:06:28,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134311.1 (Escherichia coli, NCTC9113) loaded with 70 markers.\n",
      "2024-04-17 18:06:28,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134079.1 (Escherichia coli, NCTC9112) loaded with 73 markers.\n",
      "2024-04-17 18:06:28,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134226.1 (Escherichia coli, NCTC9088) loaded with 76 markers.\n",
      "2024-04-17 18:06:28,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134234.1 (Escherichia coli, NCTC8623) loaded with 74 markers.\n",
      "2024-04-17 18:06:28,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134209.1 (Escherichia coli, NCTC11476) loaded with 72 markers.\n",
      "2024-04-17 18:06:29,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134222.1 (Escherichia coli, NCTC11129) loaded with 77 markers.\n",
      "2024-04-17 18:06:29,234 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134248.1 (Escherichia coli, NCTC10537) loaded with 76 markers.\n",
      "2024-04-17 18:06:29,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134092.1 (Escherichia coli, NCTC10444) loaded with 84 markers.\n",
      "2024-04-17 18:06:29,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009050.1 (Escherichia coli, NCCP15648) loaded with 67 markers.\n",
      "2024-04-17 18:06:29,635 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068680.1 (Escherichia coli, NCCP12480) loaded with 64 markers.\n",
      "2024-04-17 18:06:29,743 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076654.1 (Escherichia coli, NC19) loaded with 74 markers.\n",
      "2024-04-17 18:06:29,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR898868.1 (Escherichia coli, MSB1_1D-sc-2280324) loaded with 69 markers.\n",
      "2024-04-17 18:06:29,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130552.1 (Escherichia coli, MS14386) loaded with 71 markers.\n",
      "2024-04-17 18:06:30,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890651.1 (Escherichia coli, MINF_2E-sc-2280463) loaded with 72 markers.\n",
      "2024-04-17 18:06:30,203 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069682.1 (Escherichia coli, MIN9) loaded with 65 markers.\n",
      "2024-04-17 18:06:30,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069692.1 (Escherichia coli, MIN6) loaded with 60 markers.\n",
      "2024-04-17 18:06:30,391 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069646.1 (Escherichia coli, MIN14) loaded with 75 markers.\n",
      "2024-04-17 18:06:30,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073589.1 (Escherichia coli, MFDS1006657) loaded with 77 markers.\n",
      "2024-04-17 18:06:30,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024275.1 (Escherichia coli, M9682-C1) loaded with 70 markers.\n",
      "2024-04-17 18:06:30,779 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019953.1 (Escherichia coli, M8) loaded with 72 markers.\n",
      "2024-04-17 18:06:30,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010191.1 (Escherichia coli, M8) loaded with 72 markers.\n",
      "2024-04-17 18:06:31,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010183.1 (Escherichia coli, M3) loaded with 73 markers.\n",
      "2024-04-17 18:06:31,203 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010219.1 (Escherichia coli, M18) loaded with 74 markers.\n",
      "2024-04-17 18:06:31,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058719.1 (Escherichia coli, M159-1) loaded with 77 markers.\n",
      "2024-04-17 18:06:31,405 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010180.1 (Escherichia coli, M1) loaded with 73 markers.\n",
      "2024-04-17 18:06:31,504 [DEBUG - chronostrain.database.parser.json] - Strain NC_022364.1 (Escherichia coli, LY180) loaded with 74 markers.\n",
      "2024-04-17 18:06:31,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072204.1 (Escherichia coli, LWY6) loaded with 69 markers.\n",
      "2024-04-17 18:06:31,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035884.1 (Escherichia coli, LSN03-016011_A) loaded with 78 markers.\n",
      "2024-04-17 18:06:31,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047662.1 (Escherichia coli, LD93-1) loaded with 76 markers.\n",
      "2024-04-17 18:06:31,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047665.1 (Escherichia coli, LD26-1) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047876.1 (Escherichia coli, LD22-1) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882990.1 (Escherichia coli, L7_E1373_ETEC) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033378.1 (Escherichia coli, L73) loaded with 72 markers.\n",
      "2024-04-17 18:06:32,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036202.1 (Escherichia coli, L725) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034738.1 (Escherichia coli, L65) loaded with 72 markers.\n",
      "2024-04-17 18:06:32,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR883006.1 (Escherichia coli, L5_E1779_ETEC) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR883012.1 (Escherichia coli, L4_E1441_ETEC) loaded with 75 markers.\n",
      "2024-04-17 18:06:32,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882997.1 (Escherichia coli, L3_E36_ETEC) loaded with 74 markers.\n",
      "2024-04-17 18:06:32,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034589.1 (Escherichia coli, L37) loaded with 73 markers.\n",
      "2024-04-17 18:06:32,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076527.1 (Escherichia coli, L3452210II) loaded with 55 markers.\n",
      "2024-04-17 18:06:33,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882973.1 (Escherichia coli, L2_E1649) loaded with 72 markers.\n",
      "2024-04-17 18:06:33,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR883050.1 (Escherichia coli, L1_E925) loaded with 62 markers.\n",
      "2024-04-17 18:06:33,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT795502.1 (Escherichia coli, KV7) loaded with 76 markers.\n",
      "2024-04-17 18:06:33,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018840.1 (Escherichia coli, KSC64) loaded with 71 markers.\n",
      "2024-04-17 18:06:33,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019558.1 (Escherichia coli, KSC207) loaded with 71 markers.\n",
      "2024-04-17 18:06:33,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019560.1 (Escherichia coli, KSC1031) loaded with 75 markers.\n",
      "2024-04-17 18:06:33,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024123.1 (Escherichia coli, KS-NP019) loaded with 71 markers.\n",
      "2024-04-17 18:06:33,896 [DEBUG - chronostrain.database.parser.json] - Strain NC_017660.1 (Escherichia coli, KO11FL) loaded with 74 markers.\n",
      "2024-04-17 18:06:34,004 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054407.1 (Escherichia coli, KCJ3K291) loaded with 76 markers.\n",
      "2024-04-17 18:06:34,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026473.1 (Escherichia coli, KBN10P04869) loaded with 74 markers.\n",
      "2024-04-17 18:06:34,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040886.1 (Escherichia coli, K71-77) loaded with 72 markers.\n",
      "2024-04-17 18:06:34,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058618.1 (Escherichia coli, JS316) loaded with 55 markers.\n",
      "2024-04-17 18:06:34,450 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024114.1 (Escherichia coli, JML285) loaded with 71 markers.\n",
      "2024-04-17 18:06:34,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049936.1 (Escherichia coli, JL05) loaded with 72 markers.\n",
      "2024-04-17 18:06:34,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022815.1 (Escherichia coli, JE86-ST05) loaded with 68 markers.\n",
      "2024-04-17 18:06:34,779 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022811.1 (Escherichia coli, JE86-ST02) loaded with 68 markers.\n",
      "2024-04-17 18:06:34,881 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053787.1 (Escherichia coli, J31) loaded with 73 markers.\n",
      "2024-04-17 18:06:34,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047002.1 (Escherichia coli, J-8) loaded with 74 markers.\n",
      "2024-04-17 18:06:35,079 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092252.1 (Escherichia coli, ISHS7) loaded with 73 markers.\n",
      "2024-04-17 18:06:35,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045977.1 (Escherichia coli, INSC1002) loaded with 76 markers.\n",
      "2024-04-17 18:06:35,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054662.1 (Escherichia coli, IGC_RcoliRes_1.1) loaded with 71 markers.\n",
      "2024-04-17 18:06:35,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049979.1 (Escherichia coli, IBS28) loaded with 71 markers.\n",
      "2024-04-17 18:06:35,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_HF572917.1 (Escherichia coli, HUSEC2011) loaded with 71 markers.\n",
      "2024-04-17 18:06:35,634 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023427.1 (Escherichia coli, HUE1) loaded with 72 markers.\n",
      "2024-04-17 18:06:35,718 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029492.1 (Escherichia coli, HS30-1) loaded with 69 markers.\n",
      "2024-04-17 18:06:35,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019961.1 (Escherichia coli, HKUOPY1) loaded with 76 markers.\n",
      "2024-04-17 18:06:35,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020933.1 (Escherichia coli, HB-Coli0) loaded with 71 markers.\n",
      "2024-04-17 18:06:36,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029180.1 (Escherichia coli, H9Ecoli) loaded with 73 markers.\n",
      "2024-04-17 18:06:36,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013031.1 (Escherichia coli, H1827_12) loaded with 74 markers.\n",
      "2024-04-17 18:06:36,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010178.1 (Escherichia coli, H15) loaded with 72 markers.\n",
      "2024-04-17 18:06:36,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010177.1 (Escherichia coli, H14) loaded with 73 markers.\n",
      "2024-04-17 18:06:36,412 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010176.1 (Escherichia coli, H10) loaded with 73 markers.\n",
      "2024-04-17 18:06:36,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048025.1 (Escherichia coli, GZEC065) loaded with 75 markers.\n",
      "2024-04-17 18:06:36,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012376.1 (Escherichia coli, GE3) loaded with 72 markers.\n",
      "2024-04-17 18:06:36,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074407.1 (Escherichia coli, GDAR2-2) loaded with 71 markers.\n",
      "2024-04-17 18:06:36,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076646.1 (Escherichia coli, GD33) loaded with 71 markers.\n",
      "2024-04-17 18:06:36,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013663.1 (Escherichia coli, GB089) loaded with 73 markers.\n",
      "2024-04-17 18:06:37,019 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038138.1 (Escherichia coli, G3X16-2) loaded with 74 markers.\n",
      "2024-04-17 18:06:37,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031898.1 (Escherichia coli, FWSEC0010) loaded with 75 markers.\n",
      "2024-04-17 18:06:37,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031902.1 (Escherichia coli, FWSEC0009) loaded with 71 markers.\n",
      "2024-04-17 18:06:37,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031906.1 (Escherichia coli, FWSEC0008) loaded with 77 markers.\n",
      "2024-04-17 18:06:37,511 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031908.1 (Escherichia coli, FWSEC0007) loaded with 74 markers.\n",
      "2024-04-17 18:06:37,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031912.1 (Escherichia coli, FWSEC0005) loaded with 69 markers.\n",
      "2024-04-17 18:06:37,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031916.1 (Escherichia coli, FWSEC0003) loaded with 74 markers.\n",
      "2024-04-17 18:06:37,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023061.1 (Escherichia coli, FORC_069) loaded with 67 markers.\n",
      "2024-04-17 18:06:38,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016828.1 (Escherichia coli, FORC_043) loaded with 67 markers.\n",
      "2024-04-17 18:06:38,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016628.1 (Escherichia coli, FORC_041) loaded with 75 markers.\n",
      "2024-04-17 18:06:38,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013190.1 (Escherichia coli, FORC_031) loaded with 65 markers.\n",
      "2024-04-17 18:06:38,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013185.1 (Escherichia coli, FORC_029) loaded with 67 markers.\n",
      "2024-04-17 18:06:38,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017844.1 (Escherichia coli, FMU073332) loaded with 73 markers.\n",
      "2024-04-17 18:06:38,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045712.1 (Escherichia coli, FHI_NMBU_11) loaded with 76 markers.\n",
      "2024-04-17 18:06:39,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046539.1 (Escherichia coli, FHI_NMBU_10) loaded with 75 markers.\n",
      "2024-04-17 18:06:39,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085638.1 (Escherichia coli, fEC.1) loaded with 71 markers.\n",
      "2024-04-17 18:06:39,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065613.1 (Escherichia coli, FDAARGOS_944) loaded with 67 markers.\n",
      "2024-04-17 18:06:39,578 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065615.1 (Escherichia coli, FDAARGOS_943) loaded with 67 markers.\n",
      "2024-04-17 18:06:39,720 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065624.1 (Escherichia coli, FDAARGOS_941) loaded with 63 markers.\n",
      "2024-04-17 18:06:39,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041002.1 (Escherichia coli, FDAARGOS_772) loaded with 75 markers.\n",
      "2024-04-17 18:06:40,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023899.1 (Escherichia coli, FDAARGOS_433) loaded with 55 markers.\n",
      "2024-04-17 18:06:40,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023535.1 (Escherichia coli, FDAARGOS_403) loaded with 71 markers.\n",
      "2024-04-17 18:06:40,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023531.1 (Escherichia coli, FDAARGOS_401) loaded with 75 markers.\n",
      "2024-04-17 18:06:40,535 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027394.1 (Escherichia coli, FDAARGOS_349) loaded with 71 markers.\n",
      "2024-04-17 18:06:40,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022086.2 (Escherichia coli, FDAARGOS_348) loaded with 74 markers.\n",
      "2024-04-17 18:06:40,866 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069571.1 (Escherichia coli, FDAARGOS_1250) loaded with 78 markers.\n",
      "2024-04-17 18:06:41,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068155.1 (Escherichia coli, FDAARGOS_1077) loaded with 71 markers.\n",
      "2024-04-17 18:06:41,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066032.1 (Escherichia coli, FDAARGOS_1059) loaded with 67 markers.\n",
      "2024-04-17 18:06:41,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065970.1 (Escherichia coli, FDAARGOS_1058) loaded with 74 markers.\n",
      "2024-04-17 18:06:41,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024273.1 (Escherichia coli, F9792) loaded with 70 markers.\n",
      "2024-04-17 18:06:41,661 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR536430.1 (Escherichia coli, f974b26a-5e81-11e8-bf7f-3c4a9275d6c8) loaded with 85 markers.\n",
      "2024-04-17 18:06:41,835 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050998.1 (Escherichia coli, F8704-2) loaded with 76 markers.\n",
      "2024-04-17 18:06:41,995 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024269.1 (Escherichia coli, F8111-1SC3) loaded with 74 markers.\n",
      "2024-04-17 18:06:42,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024263.1 (Escherichia coli, F6326-C1) loaded with 70 markers.\n",
      "2024-04-17 18:06:42,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024260.1 (Escherichia coli, F5656C1) loaded with 71 markers.\n",
      "2024-04-17 18:06:42,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024257.1 (Escherichia coli, F5505-C1) loaded with 75 markers.\n",
      "2024-04-17 18:06:42,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023237.1 (Escherichia coli, F070) loaded with 74 markers.\n",
      "2024-04-17 18:06:42,774 [DEBUG - chronostrain.database.parser.json] - Strain NC_017633.1 (Escherichia coli, ETEC_H10407) loaded with 74 markers.\n",
      "2024-04-17 18:06:42,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041627.1 (Escherichia coli, ETEC6) loaded with 74 markers.\n",
      "2024-04-17 18:06:43,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023346.1 (Escherichia coli, ETEC-2265) loaded with 74 markers.\n",
      "2024-04-17 18:06:43,598 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031231.1 (Escherichia coli, Es_ST410_NW1_NDM_09_2017) loaded with 55 markers.\n",
      "2024-04-17 18:06:43,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041678.1 (Escherichia coli, ESBL_15) loaded with 73 markers.\n",
      "2024-04-17 18:06:43,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063459.1 (Escherichia coli, EM22-18-53) loaded with 75 markers.\n",
      "2024-04-17 18:06:44,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063479.1 (Escherichia coli, EM06-18-14) loaded with 67 markers.\n",
      "2024-04-17 18:06:44,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083497.1 (Escherichia coli, elppa6) loaded with 59 markers.\n",
      "2024-04-17 18:06:44,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083478.1 (Escherichia coli, elppa10) loaded with 55 markers.\n",
      "2024-04-17 18:06:44,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045213.1 (Escherichia coli, EH41) loaded with 75 markers.\n",
      "2024-04-17 18:06:44,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068591.1 (Escherichia coli, EF7-18-58) loaded with 57 markers.\n",
      "2024-04-17 18:06:44,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063487.1 (Escherichia coli, EF7-18-51) loaded with 67 markers.\n",
      "2024-04-17 18:06:44,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075062.1 (Escherichia coli, EC_J_9) loaded with 71 markers.\n",
      "2024-04-17 18:06:45,081 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069706.1 (Escherichia coli, ECY44) loaded with 75 markers.\n",
      "2024-04-17 18:06:45,207 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071441.1 (Escherichia coli, EcPF20) loaded with 57 markers.\n",
      "2024-04-17 18:06:45,335 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026202.1 (Escherichia coli, ECONIH5) loaded with 96 markers.\n",
      "2024-04-17 18:06:45,477 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026399.1 (Escherichia coli, ECONIH4) loaded with 76 markers.\n",
      "2024-04-17 18:06:45,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018995.1 (Escherichia coli, Ecol_AZ147) loaded with 50 markers.\n",
      "2024-04-17 18:06:45,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018965.1 (Escherichia coli, Ecol_517) loaded with 73 markers.\n",
      "2024-04-17 18:06:45,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130532.1 (Escherichia coli, ecoli019) loaded with 70 markers.\n",
      "2024-04-17 18:06:45,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR595691.1 (Escherichia coli, EcMAD1) loaded with 74 markers.\n",
      "2024-04-17 18:06:45,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067342.1 (Escherichia coli, ECJXMCRH7) loaded with 71 markers.\n",
      "2024-04-17 18:06:46,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032237.1 (Escherichia coli, ECCWS199) loaded with 68 markers.\n",
      "2024-04-17 18:06:46,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029242.1 (Escherichia coli, ECCRA-119) loaded with 74 markers.\n",
      "2024-04-17 18:06:46,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010344.1 (Escherichia coli, ECC-1470) loaded with 68 markers.\n",
      "2024-04-17 18:06:46,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060748.1 (Escherichia coli, EC96) loaded with 74 markers.\n",
      "2024-04-17 18:06:46,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049118.1 (Escherichia coli, EC931) loaded with 81 markers.\n",
      "2024-04-17 18:06:46,778 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061329.1 (Escherichia coli, EC93) loaded with 70 markers.\n",
      "2024-04-17 18:06:46,884 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060950.1 (Escherichia coli, EC9) loaded with 76 markers.\n",
      "2024-04-17 18:06:46,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085622.1 (Escherichia coli, EC71) loaded with 74 markers.\n",
      "2024-04-17 18:06:47,093 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092647.1 (Escherichia coli, EC5502) loaded with 70 markers.\n",
      "2024-04-17 18:06:47,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059125.1 (Escherichia coli, EC45) loaded with 71 markers.\n",
      "2024-04-17 18:06:47,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024582.1 (Escherichia coli, Ec387) loaded with 72 markers.\n",
      "2024-04-17 18:06:47,498 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060873.1 (Escherichia coli, EC27) loaded with 72 markers.\n",
      "2024-04-17 18:06:47,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035123.1 (Escherichia coli, EC25) loaded with 72 markers.\n",
      "2024-04-17 18:06:47,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060885.1 (Escherichia coli, EC24) loaded with 73 markers.\n",
      "2024-04-17 18:06:47,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072976.1 (Escherichia coli, EC2341) loaded with 64 markers.\n",
      "2024-04-17 18:06:48,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060902.1 (Escherichia coli, EC20) loaded with 73 markers.\n",
      "2024-04-17 18:06:48,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085188.1 (Escherichia coli, EC195) loaded with 74 markers.\n",
      "2024-04-17 18:06:48,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060908.1 (Escherichia coli, EC19) loaded with 73 markers.\n",
      "2024-04-17 18:06:48,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081192.1 (Escherichia coli, EC1722-1) loaded with 57 markers.\n",
      "2024-04-17 18:06:48,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081306.1 (Escherichia coli, Ec1079) loaded with 74 markers.\n",
      "2024-04-17 18:06:48,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060946.1 (Escherichia coli, EC10) loaded with 76 markers.\n",
      "2024-04-17 18:06:48,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS998785.1 (Escherichia coli, EC-TO75) loaded with 73 markers.\n",
      "2024-04-17 18:06:48,963 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072322.1 (Escherichia coli, EC-JS426) loaded with 72 markers.\n",
      "2024-04-17 18:06:49,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR025096.1 (Escherichia coli, EC-7215) loaded with 72 markers.\n",
      "2024-04-17 18:06:49,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035817.1 (Escherichia coli, E7476) loaded with 63 markers.\n",
      "2024-04-17 18:06:49,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029687.1 (Escherichia coli, E706) loaded with 69 markers.\n",
      "2024-04-17 18:06:49,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022393.1 (Escherichia coli, E62) loaded with 69 markers.\n",
      "2024-04-17 18:06:49,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030281.1 (Escherichia coli, E308) loaded with 75 markers.\n",
      "2024-04-17 18:06:49,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018802.1 (Escherichia coli, E2863) loaded with 67 markers.\n",
      "2024-04-17 18:06:49,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018796.1 (Escherichia coli, E2855) loaded with 67 markers.\n",
      "2024-04-17 18:06:49,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035870.1 (Escherichia coli, E2528-C1) loaded with 75 markers.\n",
      "2024-04-17 18:06:50,029 [DEBUG - chronostrain.database.parser.json] - Strain NC_009801.1 (Escherichia coli, E24377A) loaded with 67 markers.\n",
      "2024-04-17 18:06:50,150 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086662.1 (Escherichia coli, E2) loaded with 74 markers.\n",
      "2024-04-17 18:06:50,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048915.1 (Escherichia coli, E2) loaded with 73 markers.\n",
      "2024-04-17 18:06:50,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024521.1 (Escherichia coli, E189) loaded with 71 markers.\n",
      "2024-04-17 18:06:50,486 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024518.1 (Escherichia coli, E148) loaded with 72 markers.\n",
      "2024-04-17 18:06:50,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035751.1 (Escherichia coli, E110019) loaded with 66 markers.\n",
      "2024-04-17 18:06:50,707 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT594504.1 (Escherichia coli, E._coli_RL465) loaded with 72 markers.\n",
      "2024-04-17 18:06:50,831 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035879.1 (Escherichia coli, DS26-1) loaded with 74 markers.\n",
      "2024-04-17 18:06:50,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079747.1 (Escherichia coli, DL21) loaded with 76 markers.\n",
      "2024-04-17 18:06:51,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026085.1 (Escherichia coli, DH5alpha) loaded with 74 markers.\n",
      "2024-04-17 18:06:51,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010150.1 (Escherichia coli, D7) loaded with 71 markers.\n",
      "2024-04-17 18:06:51,229 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010148.1 (Escherichia coli, D6) loaded with 70 markers.\n",
      "2024-04-17 18:06:51,305 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010145.1 (Escherichia coli, D5) loaded with 55 markers.\n",
      "2024-04-17 18:06:51,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010140.1 (Escherichia coli, D3) loaded with 74 markers.\n",
      "2024-04-17 18:06:51,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010137.1 (Escherichia coli, D2) loaded with 77 markers.\n",
      "2024-04-17 18:06:51,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081762.1 (Escherichia coli, CX21) loaded with 83 markers.\n",
      "2024-04-17 18:06:51,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081742.1 (Escherichia coli, CX15) loaded with 83 markers.\n",
      "2024-04-17 18:06:51,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073719.1 (Escherichia coli, CX12) loaded with 66 markers.\n",
      "2024-04-17 18:06:51,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073718.1 (Escherichia coli, CX08) loaded with 66 markers.\n",
      "2024-04-17 18:06:52,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081724.1 (Escherichia coli, CX07) loaded with 83 markers.\n",
      "2024-04-17 18:06:52,190 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081719.1 (Escherichia coli, CX05) loaded with 83 markers.\n",
      "2024-04-17 18:06:52,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081713.1 (Escherichia coli, CX04) loaded with 83 markers.\n",
      "2024-04-17 18:06:52,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024978.1 (Escherichia coli, CV839-15) loaded with 59 markers.\n",
      "2024-04-17 18:06:52,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025753.1 (Escherichia coli, CV839-06) loaded with 74 markers.\n",
      "2024-04-17 18:06:52,616 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032073.1 (Escherichia coli, CT29) loaded with 67 markers.\n",
      "2024-04-17 18:06:52,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081791.1 (Escherichia coli, CT01) loaded with 75 markers.\n",
      "2024-04-17 18:06:52,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019051.1 (Escherichia coli, CRE1540) loaded with 77 markers.\n",
      "2024-04-17 18:06:52,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034404.1 (Escherichia coli, CRE10) loaded with 56 markers.\n",
      "2024-04-17 18:06:53,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034399.1 (Escherichia coli, CRE1) loaded with 56 markers.\n",
      "2024-04-17 18:06:53,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031546.1 (Escherichia coli, cq9) loaded with 79 markers.\n",
      "2024-04-17 18:06:53,221 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053730.1 (Escherichia coli, CP61_Sichuan) loaded with 60 markers.\n",
      "2024-04-17 18:06:53,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053731.1 (Escherichia coli, CP55_Sichuan) loaded with 72 markers.\n",
      "2024-04-17 18:06:53,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033096.1 (Escherichia coli, CP53) loaded with 77 markers.\n",
      "2024-04-17 18:06:53,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053720.1 (Escherichia coli, CP131_Sichuan) loaded with 74 markers.\n",
      "2024-04-17 18:06:53,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021726.1 (Escherichia coli, Combat11I9) loaded with 72 markers.\n",
      "2024-04-17 18:06:53,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011018.1 (Escherichia coli, CI5) loaded with 71 markers.\n",
      "2024-04-17 18:06:54,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031134.1 (Escherichia coli, CFSAN064035) loaded with 76 markers.\n",
      "2024-04-17 18:06:54,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042969.1 (Escherichia coli, CFSAN061769) loaded with 80 markers.\n",
      "2024-04-17 18:06:54,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011416.1 (Escherichia coli, CFSAN029787) loaded with 72 markers.\n",
      "2024-04-17 18:06:54,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037941.1 (Escherichia coli, CFSAN027350) loaded with 76 markers.\n",
      "2024-04-17 18:06:54,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014583.1 (Escherichia coli, CFSAN004176) loaded with 80 markers.\n",
      "2024-04-17 18:06:54,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026939.2 (Escherichia coli, CFS3313) loaded with 56 markers.\n",
      "2024-04-17 18:06:54,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047378.1 (Escherichia coli, CAU16175) loaded with 75 markers.\n",
      "2024-04-17 18:06:55,070 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010125.1 (Escherichia coli, C8) loaded with 71 markers.\n",
      "2024-04-17 18:06:55,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010240.1 (Escherichia coli, C7) loaded with 74 markers.\n",
      "2024-04-17 18:06:55,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010122.1 (Escherichia coli, C5) loaded with 74 markers.\n",
      "2024-04-17 18:06:55,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011938.1 (Escherichia coli, C43(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:55,692 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010585.1 (Escherichia coli, C41(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:55,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062204.1 (Escherichia coli, C311) loaded with 75 markers.\n",
      "2024-04-17 18:06:55,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014269.1 (Escherichia coli, C3029) loaded with 77 markers.\n",
      "2024-04-17 18:06:56,036 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010119.1 (Escherichia coli, C3) loaded with 74 markers.\n",
      "2024-04-17 18:06:56,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014268.2 (Escherichia coli, C2566) loaded with 76 markers.\n",
      "2024-04-17 18:06:56,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011331.1 (Escherichia coli, C227-11) loaded with 71 markers.\n",
      "2024-04-17 18:06:56,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP052877.1 (Escherichia coli, C21) loaded with 78 markers.\n",
      "2024-04-17 18:06:56,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010117.1 (Escherichia coli, C2) loaded with 69 markers.\n",
      "2024-04-17 18:06:56,569 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010133.1 (Escherichia coli, C11) loaded with 72 markers.\n",
      "2024-04-17 18:06:56,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010132.1 (Escherichia coli, C10) loaded with 71 markers.\n",
      "2024-04-17 18:06:56,839 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029371.1 (Escherichia coli, C) loaded with 69 markers.\n",
      "2024-04-17 18:06:56,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063341.1 (Escherichia coli, BS90R-A) loaded with 71 markers.\n",
      "2024-04-17 18:06:57,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020368.1 (Escherichia coli, BLR(DE3)) loaded with 75 markers.\n",
      "2024-04-17 18:06:57,142 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014641.1 (Escherichia coli, BLK9) loaded with 44 markers.\n",
      "2024-04-17 18:06:57,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014642.1 (Escherichia coli, BLK16) loaded with 44 markers.\n",
      "2024-04-17 18:06:57,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010816.1 (Escherichia coli, BL21_(TaKaRa)) loaded with 76 markers.\n",
      "2024-04-17 18:06:57,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083273.1 (Escherichia coli, BL21-RIPL) loaded with 78 markers.\n",
      "2024-04-17 18:06:57,629 [DEBUG - chronostrain.database.parser.json] - Strain NC_012947.1 (Escherichia coli, BL21-Gold(DE3)pLysS_AG) loaded with 78 markers.\n",
      "2024-04-17 18:06:57,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047231.1 (Escherichia coli, BL21-AI) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084683.1 (Escherichia coli, BL21(RP)) loaded with 78 markers.\n",
      "2024-04-17 18:06:58,168 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081489.1 (Escherichia coli, BL21(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,328 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053602.1 (Escherichia coli, BL21(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,486 [DEBUG - chronostrain.database.parser.json] - Strain NC_012971.2 (Escherichia coli, BL21(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,632 [DEBUG - chronostrain.database.parser.json] - Strain NC_012892.2 (Escherichia coli, BL21(DE3)) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060121.1 (Escherichia coli, BL21) loaded with 76 markers.\n",
      "2024-04-17 18:06:58,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053601.1 (Escherichia coli, BL21) loaded with 76 markers.\n",
      "2024-04-17 18:06:59,069 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022295.1 (Escherichia coli, BEC1-S17-ESBL-07) loaded with 68 markers.\n",
      "2024-04-17 18:06:59,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040397.1 (Escherichia coli, BA22372) loaded with 74 markers.\n",
      "2024-04-17 18:06:59,376 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085842.1 (Escherichia coli, B95._delta_A) loaded with 76 markers.\n",
      "2024-04-17 18:06:59,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024245.1 (Escherichia coli, B4103-1) loaded with 66 markers.\n",
      "2024-04-17 18:06:59,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035874.1 (Escherichia coli, B2C) loaded with 71 markers.\n",
      "2024-04-17 18:06:59,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073624.1 (Escherichia coli, AW1.7) loaded with 73 markers.\n",
      "2024-04-17 18:06:59,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072539.1 (Escherichia coli, AW1.7) loaded with 73 markers.\n",
      "2024-04-17 18:07:00,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045827.1 (Escherichia coli, AUSMDU00014361) loaded with 76 markers.\n",
      "2024-04-17 18:07:00,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023541.1 (Escherichia coli, ATCC_BAA-178) loaded with 74 markers.\n",
      "2024-04-17 18:07:00,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043852.1 (Escherichia coli, ATCC_8739) loaded with 73 markers.\n",
      "2024-04-17 18:07:00,481 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024278.1 (Escherichia coli, ATCC_43896) loaded with 74 markers.\n",
      "2024-04-17 18:07:00,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024256.1 (Escherichia coli, ATCC_43886) loaded with 75 markers.\n",
      "2024-04-17 18:07:00,725 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027430.1 (Escherichia coli, AS19-RrmA-) loaded with 77 markers.\n",
      "2024-04-17 18:07:00,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030331.1 (Escherichia coli, AR_452) loaded with 74 markers.\n",
      "2024-04-17 18:07:00,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027126.1 (Escherichia coli, AR_0374) loaded with 54 markers.\n",
      "2024-04-17 18:07:01,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029741.1 (Escherichia coli, AR_0085) loaded with 77 markers.\n",
      "2024-04-17 18:07:01,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020055.1 (Escherichia coli, AR_0069) loaded with 76 markers.\n",
      "2024-04-17 18:07:01,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024889.1 (Escherichia coli, AR_0019) loaded with 72 markers.\n",
      "2024-04-17 18:07:01,342 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029108.1 (Escherichia coli, AR437) loaded with 55 markers.\n",
      "2024-04-17 18:07:01,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029111.1 (Escherichia coli, AR436) loaded with 54 markers.\n",
      "2024-04-17 18:07:01,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029122.1 (Escherichia coli, AR434) loaded with 71 markers.\n",
      "2024-04-17 18:07:01,615 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035944.1 (Escherichia coli, AR24.2b) loaded with 56 markers.\n",
      "2024-04-17 18:07:01,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043487.1 (Escherichia coli, AQ15) loaded with 74 markers.\n",
      "2024-04-17 18:07:01,818 [DEBUG - chronostrain.database.parser.json] - Strain NC_020163.1 (Escherichia coli, APEC_O78) loaded with 75 markers.\n",
      "2024-04-17 18:07:01,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030939.1 (Escherichia coli, AMSHJX01) loaded with 73 markers.\n",
      "2024-04-17 18:07:02,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058308.1 (Escherichia coli, AMSCJX04) loaded with 71 markers.\n",
      "2024-04-17 18:07:02,149 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024801.1 (Escherichia coli, AMA1167) loaded with 74 markers.\n",
      "2024-04-17 18:07:02,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082349.1 (Escherichia coli, AL.H8) loaded with 74 markers.\n",
      "2024-04-17 18:07:02,376 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055256.1 (Escherichia coli, AH25) loaded with 77 markers.\n",
      "2024-04-17 18:07:02,504 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081706.1 (Escherichia coli, AH04) loaded with 75 markers.\n",
      "2024-04-17 18:07:02,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081701.1 (Escherichia coli, AH02) loaded with 75 markers.\n",
      "2024-04-17 18:07:02,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055251.1 (Escherichia coli, AH01) loaded with 72 markers.\n",
      "2024-04-17 18:07:02,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007491.1 (Escherichia coli, ACN002) loaded with 74 markers.\n",
      "2024-04-17 18:07:03,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007442.1 (Escherichia coli, ACN001) loaded with 74 markers.\n",
      "2024-04-17 18:07:03,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022154.1 (Escherichia coli, ABWA45) loaded with 87 markers.\n",
      "2024-04-17 18:07:03,502 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028766.1 (Escherichia coli, A8) loaded with 72 markers.\n",
      "2024-04-17 18:07:03,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028767.1 (Escherichia coli, A7) loaded with 72 markers.\n",
      "2024-04-17 18:07:03,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028769.1 (Escherichia coli, A5) loaded with 71 markers.\n",
      "2024-04-17 18:07:03,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028735.1 (Escherichia coli, A41) loaded with 72 markers.\n",
      "2024-04-17 18:07:04,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028737.1 (Escherichia coli, A39) loaded with 74 markers.\n",
      "2024-04-17 18:07:04,276 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028738.1 (Escherichia coli, A38) loaded with 71 markers.\n",
      "2024-04-17 18:07:04,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028740.1 (Escherichia coli, A35) loaded with 73 markers.\n",
      "2024-04-17 18:07:04,587 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028744.1 (Escherichia coli, A31) loaded with 72 markers.\n",
      "2024-04-17 18:07:04,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028747.1 (Escherichia coli, A28) loaded with 71 markers.\n",
      "2024-04-17 18:07:04,894 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028772.1 (Escherichia coli, A2) loaded with 71 markers.\n",
      "2024-04-17 18:07:05,017 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040067.1 (Escherichia coli, A1_181) loaded with 56 markers.\n",
      "2024-04-17 18:07:05,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028757.1 (Escherichia coli, A17) loaded with 74 markers.\n",
      "2024-04-17 18:07:05,322 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028759.1 (Escherichia coli, A15) loaded with 69 markers.\n",
      "2024-04-17 18:07:05,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028762.1 (Escherichia coli, A12) loaded with 77 markers.\n",
      "2024-04-17 18:07:05,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029328.1 (Escherichia coli, A1) loaded with 72 markers.\n",
      "2024-04-17 18:07:05,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029981.1 (Escherichia coli, 99-3165) loaded with 73 markers.\n",
      "2024-04-17 18:07:05,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021339.1 (Escherichia coli, 95NR1) loaded with 73 markers.\n",
      "2024-04-17 18:07:05,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021335.1 (Escherichia coli, 95JB1) loaded with 69 markers.\n",
      "2024-04-17 18:07:06,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047576.1 (Escherichia coli, 94EC) loaded with 72 markers.\n",
      "2024-04-17 18:07:06,140 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009106.2 (Escherichia coli, 94-3024) loaded with 74 markers.\n",
      "2024-04-17 18:07:06,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024243.1 (Escherichia coli, 90-9281) loaded with 67 markers.\n",
      "2024-04-17 18:07:06,380 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027366.1 (Escherichia coli, 89-3156) loaded with 73 markers.\n",
      "2024-04-17 18:07:06,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070906.1 (Escherichia coli, 88COLEC) loaded with 76 markers.\n",
      "2024-04-17 18:07:06,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027363.1 (Escherichia coli, 88-3001) loaded with 75 markers.\n",
      "2024-04-17 18:07:06,730 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050865.1 (Escherichia coli, 8-3-Ti3) loaded with 59 markers.\n",
      "2024-04-17 18:07:06,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050862.1 (Escherichia coli, 8-3-DC15) loaded with 59 markers.\n",
      "2024-04-17 18:07:06,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010315.1 (Escherichia coli, 789) loaded with 74 markers.\n",
      "2024-04-17 18:07:07,045 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050044.1 (Escherichia coli, 74-2) loaded with 84 markers.\n",
      "2024-04-17 18:07:07,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061764.1 (Escherichia coli, 730V1) loaded with 87 markers.\n",
      "2024-04-17 18:07:07,290 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT883142.1 (Escherichia coli, 6666666.257727.embl) loaded with 76 markers.\n",
      "2024-04-17 18:07:07,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048322.1 (Escherichia coli, 64) loaded with 47 markers.\n",
      "2024-04-17 18:07:07,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048326.1 (Escherichia coli, 61) loaded with 47 markers.\n",
      "2024-04-17 18:07:07,579 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025847.1 (Escherichia coli, 602354) loaded with 74 markers.\n",
      "2024-04-17 18:07:07,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050036.1 (Escherichia coli, 58-3) loaded with 73 markers.\n",
      "2024-04-17 18:07:07,792 [DEBUG - chronostrain.database.parser.json] - Strain NC_011748.1 (Escherichia coli, 55989) loaded with 65 markers.\n",
      "2024-04-17 18:07:07,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091534.1 (Escherichia coli, 537-20) loaded with 72 markers.\n",
      "2024-04-17 18:07:08,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048359.1 (Escherichia coli, 53) loaded with 77 markers.\n",
      "2024-04-17 18:07:08,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025856.1 (Escherichia coli, 504838) loaded with 77 markers.\n",
      "2024-04-17 18:07:08,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025862.1 (Escherichia coli, 504237) loaded with 74 markers.\n",
      "2024-04-17 18:07:08,374 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070920.1 (Escherichia coli, 40EC) loaded with 72 markers.\n",
      "2024-04-17 18:07:08,511 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045209.1 (Escherichia coli, 4) loaded with 73 markers.\n",
      "2024-04-17 18:07:08,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049348.1 (Escherichia coli, 3R) loaded with 73 markers.\n",
      "2024-04-17 18:07:08,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048376.1 (Escherichia coli, 38) loaded with 73 markers.\n",
      "2024-04-17 18:07:08,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038859.1 (Escherichia coli, 357900) loaded with 73 markers.\n",
      "2024-04-17 18:07:08,969 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050031.1 (Escherichia coli, 33-6) loaded with 73 markers.\n",
      "2024-04-17 18:07:09,089 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084096.1 (Escherichia coli, 3) loaded with 70 markers.\n",
      "2024-04-17 18:07:09,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047571.1 (Escherichia coli, 2EC1) loaded with 70 markers.\n",
      "2024-04-17 18:07:09,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059947.1 (Escherichia coli, 268.2) loaded with 59 markers.\n",
      "2024-04-17 18:07:09,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053855.1 (Escherichia coli, 258E) loaded with 56 markers.\n",
      "2024-04-17 18:07:09,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061749.1 (Escherichia coli, 2273-PO3) loaded with 83 markers.\n",
      "2024-04-17 18:07:09,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025910.1 (Escherichia coli, 204446) loaded with 74 markers.\n",
      "2024-04-17 18:07:09,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025913.1 (Escherichia coli, 203740) loaded with 75 markers.\n",
      "2024-04-17 18:07:09,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075632.1 (Escherichia coli, 2018-166) loaded with 75 markers.\n",
      "2024-04-17 18:07:09,995 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021892.1 (Escherichia coli, 2018-01-1CC) loaded with 80 markers.\n",
      "2024-04-17 18:07:10,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021890.1 (Escherichia coli, 2017.15.01CC) loaded with 69 markers.\n",
      "2024-04-17 18:07:10,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025220.2 (Escherichia coli, 2017.11.01CC) loaded with 74 markers.\n",
      "2024-04-17 18:07:10,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025205.2 (Escherichia coli, 2017.04.03CC) loaded with 75 markers.\n",
      "2024-04-17 18:07:10,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021891.1 (Escherichia coli, 2017.01.04CC) loaded with 71 markers.\n",
      "2024-04-17 18:07:10,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027552.1 (Escherichia coli, 2015C-4498) loaded with 84 markers.\n",
      "2024-04-17 18:07:10,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027371.1 (Escherichia coli, 2015C-3905) loaded with 77 markers.\n",
      "2024-04-17 18:07:10,798 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027219.1 (Escherichia coli, 2015C-3163) loaded with 74 markers.\n",
      "2024-04-17 18:07:10,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027340.1 (Escherichia coli, 2015C-3121) loaded with 77 markers.\n",
      "2024-04-17 18:07:11,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027307.1 (Escherichia coli, 2015C-3108) loaded with 69 markers.\n",
      "2024-04-17 18:07:11,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027221.1 (Escherichia coli, 2015C-3101) loaded with 69 markers.\n",
      "2024-04-17 18:07:11,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024232.1 (Escherichia coli, 2014EL-1346-6) loaded with 71 markers.\n",
      "2024-04-17 18:07:11,407 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024223.1 (Escherichia coli, 2014EL-1345-2) loaded with 74 markers.\n",
      "2024-04-17 18:07:11,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024228.1 (Escherichia coli, 2014EL-1343-2) loaded with 75 markers.\n",
      "2024-04-17 18:07:11,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027342.1 (Escherichia coli, 2014C-4587) loaded with 73 markers.\n",
      "2024-04-17 18:07:11,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027310.1 (Escherichia coli, 2014C-4135) loaded with 73 markers.\n",
      "2024-04-17 18:07:11,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027368.1 (Escherichia coli, 2014C-3307) loaded with 74 markers.\n",
      "2024-04-17 18:07:12,003 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027449.1 (Escherichia coli, 2014C-3097) loaded with 82 markers.\n",
      "2024-04-17 18:07:12,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027325.1 (Escherichia coli, 2013C-4830) loaded with 75 markers.\n",
      "2024-04-17 18:07:12,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027347.1 (Escherichia coli, 2013C-4361) loaded with 69 markers.\n",
      "2024-04-17 18:07:12,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027573.1 (Escherichia coli, 2013C-4081) loaded with 69 markers.\n",
      "2024-04-17 18:07:12,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027766.1 (Escherichia coli, 2013C-3342) loaded with 84 markers.\n",
      "2024-04-17 18:07:12,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027331.1 (Escherichia coli, 2013C-3277) loaded with 76 markers.\n",
      "2024-04-17 18:07:12,839 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027380.1 (Escherichia coli, 2013C-3250) loaded with 67 markers.\n",
      "2024-04-17 18:07:12,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027323.1 (Escherichia coli, 2013C-3033) loaded with 77 markers.\n",
      "2024-04-17 18:07:13,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027586.1 (Escherichia coli, 2012EL-2448) loaded with 77 markers.\n",
      "2024-04-17 18:07:13,246 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024289.1 (Escherichia coli, 2012C-4431) loaded with 78 markers.\n",
      "2024-04-17 18:07:13,419 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013029.1 (Escherichia coli, 2012C-4227) loaded with 75 markers.\n",
      "2024-04-17 18:07:13,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022912.1 (Escherichia coli, 2011EL-1370-2) loaded with 76 markers.\n",
      "2024-04-17 18:07:13,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024479.1 (Escherichia coli, 2011C-4315) loaded with 70 markers.\n",
      "2024-04-17 18:07:13,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027388.1 (Escherichia coli, 2011C-4251) loaded with 77 markers.\n",
      "2024-04-17 18:07:13,938 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015240.1 (Escherichia coli, 2011C-3911) loaded with 73 markers.\n",
      "2024-04-17 18:07:14,069 [DEBUG - chronostrain.database.parser.json] - Strain NC_018658.1 (Escherichia coli, 2011C-3493) loaded with 71 markers.\n",
      "2024-04-17 18:07:14,203 [DEBUG - chronostrain.database.parser.json] - Strain NC_018661.1 (Escherichia coli, 2009EL-2071) loaded with 76 markers.\n",
      "2024-04-17 18:07:14,338 [DEBUG - chronostrain.database.parser.json] - Strain NC_018650.1 (Escherichia coli, 2009EL-2050) loaded with 74 markers.\n",
      "2024-04-17 18:07:14,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061101.1 (Escherichia coli, 1EC213) loaded with 46 markers.\n",
      "2024-04-17 18:07:14,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061108.1 (Escherichia coli, 1EC187) loaded with 72 markers.\n",
      "2024-04-17 18:07:14,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023359.1 (Escherichia coli, 1943) loaded with 72 markers.\n",
      "2024-04-17 18:07:14,729 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046006.1 (Escherichia coli, 1919D3) loaded with 74 markers.\n",
      "2024-04-17 18:07:14,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020520.1 (Escherichia coli, 190) loaded with 64 markers.\n",
      "2024-04-17 18:07:14,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047010.1 (Escherichia coli, 19-5) loaded with 70 markers.\n",
      "2024-04-17 18:07:15,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019856.1 (Escherichia coli, 16F5M1D1) loaded with 72 markers.\n",
      "2024-04-17 18:07:15,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086391.1 (Escherichia coli, 1585m1) loaded with 82 markers.\n",
      "2024-04-17 18:07:15,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059288.1 (Escherichia coli, 1506) loaded with 83 markers.\n",
      "2024-04-17 18:07:15,361 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040269.1 (Escherichia coli, 1500) loaded with 75 markers.\n",
      "2024-04-17 18:07:15,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024141.1 (Escherichia coli, 14EC029) loaded with 74 markers.\n",
      "2024-04-17 18:07:15,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024127.1 (Escherichia coli, 14EC001) loaded with 79 markers.\n",
      "2024-04-17 18:07:15,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019271.1 (Escherichia coli, 13P460A) loaded with 59 markers.\n",
      "2024-04-17 18:07:15,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019250.1 (Escherichia coli, 13KWH46) loaded with 71 markers.\n",
      "2024-04-17 18:07:15,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020107.1 (Escherichia coli, 13E0767) loaded with 70 markers.\n",
      "2024-04-17 18:07:16,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019267.1 (Escherichia coli, 13C1079T) loaded with 72 markers.\n",
      "2024-04-17 18:07:16,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019259.1 (Escherichia coli, 13C1065T) loaded with 74 markers.\n",
      "2024-04-17 18:07:16,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048344.1 (Escherichia coli, 124) loaded with 75 markers.\n",
      "2024-04-17 18:07:16,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025916.1 (Escherichia coli, 120899) loaded with 74 markers.\n",
      "2024-04-17 18:07:16,539 [DEBUG - chronostrain.database.parser.json] - Strain NC_013353.1 (Escherichia coli, 12009) loaded with 75 markers.\n",
      "2024-04-17 18:07:16,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087570.1 (Escherichia coli, 114) loaded with 73 markers.\n",
      "2024-04-17 18:07:16,778 [DEBUG - chronostrain.database.parser.json] - Strain NC_013364.1 (Escherichia coli, 11128) loaded with 69 markers.\n",
      "2024-04-17 18:07:16,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019761.1 (Escherichia coli, 110512) loaded with 67 markers.\n",
      "2024-04-17 18:07:17,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025920.1 (Escherichia coli, 103605) loaded with 74 markers.\n",
      "2024-04-17 18:07:17,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081782.1 (Escherichia coli, 0H26) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081690.1 (Escherichia coli, 0H25) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081774.1 (Escherichia coli, 0H24) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081684.1 (Escherichia coli, 0H23) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081767.1 (Escherichia coli, 0H18) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081677.1 (Escherichia coli, 0H15) loaded with 75 markers.\n",
      "2024-04-17 18:07:17,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015228.1 (Escherichia coli, 09-00049) loaded with 73 markers.\n",
      "2024-04-17 18:07:17,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027462.1 (Escherichia coli, 07-4299) loaded with 73 markers.\n",
      "2024-04-17 18:07:18,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027373.1 (Escherichia coli, 05-3629) loaded with 74 markers.\n",
      "2024-04-17 18:07:18,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042934.2 (Escherichia coli, 042) loaded with 56 markers.\n",
      "2024-04-17 18:07:18,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035832.1 (Escherichia coli, 025542-1) loaded with 74 markers.\n",
      "2024-04-17 18:07:18,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027584.1 (Escherichia coli, 00-3076) loaded with 75 markers.\n",
      "2024-04-17 18:07:18,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010829.1 (Shigella sonnei, FORC_011) loaded with 50 markers.\n",
      "2024-04-17 18:07:18,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014099.2 (Shigella sonnei, FDAARGOS_90) loaded with 50 markers.\n",
      "2024-04-17 18:07:18,973 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058846.1 (Shigella flexneri, STLEFF_35) loaded with 72 markers.\n",
      "2024-04-17 18:07:19,057 [DEBUG - chronostrain.database.parser.json] - Strain NC_007613.1 (Shigella boydii, Sb227) loaded with 68 markers.\n",
      "2024-04-17 18:07:19,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082357.1 (Escherichia coli, VCSW7) loaded with 70 markers.\n",
      "2024-04-17 18:07:19,285 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007394.1 (Escherichia coli, ST2747) loaded with 74 markers.\n",
      "2024-04-17 18:07:19,398 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007393.1 (Escherichia coli, ST2747) loaded with 75 markers.\n",
      "2024-04-17 18:07:19,549 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007392.1 (Escherichia coli, ST2747) loaded with 75 markers.\n",
      "2024-04-17 18:07:19,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051751.1 (Escherichia coli, SCU-301) loaded with 76 markers.\n",
      "2024-04-17 18:07:19,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010236.1 (Escherichia coli, S42) loaded with 73 markers.\n",
      "2024-04-17 18:07:19,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031347.1 (Escherichia coli, RM9873-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024659.1 (Escherichia coli, RM9872-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028379.1 (Escherichia coli, RM9872) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031349.1 (Escherichia coli, RM9467-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031353.1 (Escherichia coli, RM9154-C1) loaded with 78 markers.\n",
      "2024-04-17 18:07:20,608 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031355.1 (Escherichia coli, RM8995-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,729 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035770.1 (Escherichia coli, RM8988-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:20,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035772.1 (Escherichia coli, RM8843-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:21,004 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028116.1 (Escherichia coli, RM8426) loaded with 76 markers.\n",
      "2024-04-17 18:07:21,140 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028112.1 (Escherichia coli, RM8385) loaded with 74 markers.\n",
      "2024-04-17 18:07:21,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006262.1 (Escherichia coli, RM13516) loaded with 78 markers.\n",
      "2024-04-17 18:07:21,454 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006027.1 (Escherichia coli, RM13514) loaded with 79 markers.\n",
      "2024-04-17 18:07:21,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007133.1 (Escherichia coli, RM12761) loaded with 78 markers.\n",
      "2024-04-17 18:07:21,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007136.1 (Escherichia coli, RM12581) loaded with 79 markers.\n",
      "2024-04-17 18:07:21,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035767.1 (Escherichia coli, RM12522-C8) loaded with 79 markers.\n",
      "2024-04-17 18:07:22,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031345.1 (Escherichia coli, RM12367-C1) loaded with 81 markers.\n",
      "2024-04-17 18:07:22,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031341.1 (Escherichia coli, RM12275-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:22,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035768.1 (Escherichia coli, RM11626-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:22,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031343.1 (Escherichia coli, RM10425-C1) loaded with 79 markers.\n",
      "2024-04-17 18:07:22,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028126.1 (Escherichia coli, RM10386) loaded with 78 markers.\n",
      "2024-04-17 18:07:22,743 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062824.1 (Escherichia coli, Res13-Lact-PER13-34) loaded with 72 markers.\n",
      "2024-04-17 18:07:22,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058682.2 (Escherichia coli, MBT-5) loaded with 78 markers.\n",
      "2024-04-17 18:07:22,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882978.1 (Escherichia coli, L3_CS7_E2980) loaded with 73 markers.\n",
      "2024-04-17 18:07:23,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031919.1 (Escherichia coli, FWSEC0002) loaded with 79 markers.\n",
      "2024-04-17 18:07:23,276 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031922.1 (Escherichia coli, FWSEC0001) loaded with 78 markers.\n",
      "2024-04-17 18:07:23,425 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012693.1 (Escherichia coli, FORC_028) loaded with 75 markers.\n",
      "2024-04-17 18:07:23,510 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085637.1 (Escherichia coli, fECg99.1) loaded with 68 markers.\n",
      "2024-04-17 18:07:23,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065611.1 (Escherichia coli, FDAARGOS_945) loaded with 74 markers.\n",
      "2024-04-17 18:07:23,756 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023349.1 (Escherichia coli, ETEC-2264) loaded with 72 markers.\n",
      "2024-04-17 18:07:23,849 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041919.1 (Escherichia coli, Ec40743) loaded with 71 markers.\n",
      "2024-04-17 18:07:23,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018808.1 (Escherichia coli, E2865) loaded with 76 markers.\n",
      "2024-04-17 18:07:24,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037945.1 (Escherichia coli, CFSAN027346) loaded with 76 markers.\n",
      "2024-04-17 18:07:24,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037943.1 (Escherichia coli, CFSAN027343) loaded with 76 markers.\n",
      "2024-04-17 18:07:24,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035812.1 (Escherichia coli, B7A) loaded with 72 markers.\n",
      "2024-04-17 18:07:24,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028765.1 (Escherichia coli, A9) loaded with 72 markers.\n",
      "2024-04-17 18:07:24,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027599.1 (Escherichia coli, 97-3250) loaded with 77 markers.\n",
      "2024-04-17 18:07:24,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027362.1 (Escherichia coli, 95-3192) loaded with 79 markers.\n",
      "2024-04-17 18:07:24,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024240.1 (Escherichia coli, 90-9280) loaded with 72 markers.\n",
      "2024-04-17 18:07:24,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027459.1 (Escherichia coli, 90-3040) loaded with 75 markers.\n",
      "2024-04-17 18:07:25,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025865.1 (Escherichia coli, 504211) loaded with 68 markers.\n",
      "2024-04-17 18:07:25,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025869.1 (Escherichia coli, 503829) loaded with 68 markers.\n",
      "2024-04-17 18:07:25,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025892.1 (Escherichia coli, 503025) loaded with 68 markers.\n",
      "2024-04-17 18:07:25,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015912.1 (Escherichia coli, 210205630) loaded with 74 markers.\n",
      "2024-04-17 18:07:25,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021895.1 (Escherichia coli, 2018-11-3CC) loaded with 69 markers.\n",
      "2024-04-17 18:07:25,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027390.1 (Escherichia coli, 2015C-4944) loaded with 78 markers.\n",
      "2024-04-17 18:07:25,743 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027361.1 (Escherichia coli, 2014C-4639) loaded with 73 markers.\n",
      "2024-04-17 18:07:25,823 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027319.1 (Escherichia coli, 2014C-3084) loaded with 55 markers.\n",
      "2024-04-17 18:07:25,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027447.1 (Escherichia coli, 2014C-3075) loaded with 78 markers.\n",
      "2024-04-17 18:07:26,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027548.1 (Escherichia coli, 2014C-3061) loaded with 73 markers.\n",
      "2024-04-17 18:07:26,196 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027387.1 (Escherichia coli, 2014C-3057) loaded with 76 markers.\n",
      "2024-04-17 18:07:26,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027338.1 (Escherichia coli, 2014C-3051) loaded with 75 markers.\n",
      "2024-04-17 18:07:26,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027472.1 (Escherichia coli, 2014C-3050) loaded with 75 markers.\n",
      "2024-04-17 18:07:26,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027582.1 (Escherichia coli, 2013C-4538) loaded with 75 markers.\n",
      "2024-04-17 18:07:26,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027577.1 (Escherichia coli, 2013C-4225) loaded with 74 markers.\n",
      "2024-04-17 18:07:26,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027546.1 (Escherichia coli, 2013C-4187) loaded with 75 markers.\n",
      "2024-04-17 18:07:27,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027555.1 (Escherichia coli, 2013C-3513) loaded with 74 markers.\n",
      "2024-04-17 18:07:27,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027445.1 (Escherichia coli, 2013C-3492) loaded with 75 markers.\n",
      "2024-04-17 18:07:27,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027544.1 (Escherichia coli, 2013C-3264) loaded with 76 markers.\n",
      "2024-04-17 18:07:27,450 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027442.1 (Escherichia coli, 2013C-3252) loaded with 75 markers.\n",
      "2024-04-17 18:07:27,615 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027352.1 (Escherichia coli, 2012C-4606) loaded with 76 markers.\n",
      "2024-04-17 18:07:27,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053785.1 (Escherichia coli, 2-101) loaded with 84 markers.\n",
      "2024-04-17 18:07:27,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024147.1 (Escherichia coli, 14EC033) loaded with 77 markers.\n",
      "2024-04-17 18:07:27,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020106.1 (Escherichia coli, 13E0780) loaded with 72 markers.\n",
      "2024-04-17 18:07:28,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020092.1 (Escherichia coli, 13E0725) loaded with 72 markers.\n",
      "2024-04-17 18:07:28,204 [DEBUG - chronostrain.database.parser.json] - Strain NC_013361.1 (Escherichia coli, 11368) loaded with 76 markers.\n",
      "2024-04-17 18:07:28,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054353.1 (Escherichia coli, SCU-172) loaded with 77 markers.\n",
      "2024-04-17 18:07:28,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778153.1 (Escherichia coli, SC480) loaded with 71 markers.\n",
      "2024-04-17 18:07:28,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR898874.1 (Escherichia coli, MSB1_4I-sc-2280412) loaded with 79 markers.\n",
      "2024-04-17 18:07:28,710 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010200.1 (Escherichia coli, M10) loaded with 71 markers.\n",
      "2024-04-17 18:07:28,831 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019944.1 (Escherichia coli, DSM_103246) loaded with 81 markers.\n",
      "2024-04-17 18:07:28,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028192.1 (Escherichia coli, CFSAN018748) loaded with 74 markers.\n",
      "2024-04-17 18:07:29,081 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067307.1 (Escherichia coli, ABW_A19) loaded with 90 markers.\n",
      "2024-04-17 18:07:29,177 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055113.1 (Shigella flexneri, SWHIN_101) loaded with 73 markers.\n",
      "2024-04-17 18:07:29,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055199.1 (Shigella flexneri, STEFF_3) loaded with 74 markers.\n",
      "2024-04-17 18:07:29,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066844.1 (Escherichia coli, Z30) loaded with 71 markers.\n",
      "2024-04-17 18:07:29,593 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019392.1 (Escherichia coli, XH992) loaded with 71 markers.\n",
      "2024-04-17 18:07:29,693 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043033.1 (Escherichia coli, XDL) loaded with 74 markers.\n",
      "2024-04-17 18:07:29,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022161.1 (Escherichia coli, WP5-S18-ESBL-08) loaded with 76 markers.\n",
      "2024-04-17 18:07:29,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022098.1 (Escherichia coli, WP4-S18-ESBL-09) loaded with 69 markers.\n",
      "2024-04-17 18:07:30,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020025.1 (Escherichia coli, WB61) loaded with 63 markers.\n",
      "2024-04-17 18:07:30,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054191.1 (Escherichia coli, TH9F11) loaded with 78 markers.\n",
      "2024-04-17 18:07:30,209 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055022.1 (Escherichia coli, SWHEFF_68) loaded with 70 markers.\n",
      "2024-04-17 18:07:30,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008697.1 (Escherichia coli, ST648) loaded with 67 markers.\n",
      "2024-04-17 18:07:30,421 [DEBUG - chronostrain.database.parser.json] - Strain NC_010498.1 (Escherichia coli, SMS-3-5) loaded with 72 markers.\n",
      "2024-04-17 18:07:30,536 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067235.1 (Escherichia coli, SBF22) loaded with 70 markers.\n",
      "2024-04-17 18:07:30,643 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034389.1 (Escherichia coli, RS571) loaded with 71 markers.\n",
      "2024-04-17 18:07:30,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062865.1 (Escherichia coli, Res13-Lact-PEB08-01) loaded with 75 markers.\n",
      "2024-04-17 18:07:30,831 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062868.1 (Escherichia coli, Res13-Lact-PEB01-20) loaded with 74 markers.\n",
      "2024-04-17 18:07:30,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062889.1 (Escherichia coli, Res13-Lact-ER07-18) loaded with 74 markers.\n",
      "2024-04-17 18:07:31,050 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016546.1 (Escherichia coli, O177:H21) loaded with 73 markers.\n",
      "2024-04-17 18:07:31,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041302.1 (Escherichia coli, MSHS_472) loaded with 70 markers.\n",
      "2024-04-17 18:07:31,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890536.1 (Escherichia coli, MSB1_8B-sc-2280300) loaded with 76 markers.\n",
      "2024-04-17 18:07:31,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015834.1 (Escherichia coli, MS6198) loaded with 77 markers.\n",
      "2024-04-17 18:07:31,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047405.1 (Escherichia coli, MS6193) loaded with 70 markers.\n",
      "2024-04-17 18:07:31,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054940.1 (Escherichia coli, MS6192) loaded with 70 markers.\n",
      "2024-04-17 18:07:31,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130555.1 (Escherichia coli, MS14385) loaded with 69 markers.\n",
      "2024-04-17 18:07:31,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890334.1 (Escherichia coli, MINF_9A-sc-2280436) loaded with 78 markers.\n",
      "2024-04-17 18:07:31,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890326.1 (Escherichia coli, MINF_1A-sc-2280431) loaded with 79 markers.\n",
      "2024-04-17 18:07:32,013 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058714.1 (Escherichia coli, M172-1) loaded with 69 markers.\n",
      "2024-04-17 18:07:32,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022164.1 (Escherichia coli, M160133) loaded with 73 markers.\n",
      "2024-04-17 18:07:32,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061185.1 (Escherichia coli, LD67-1) loaded with 70 markers.\n",
      "2024-04-17 18:07:32,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062211.1 (Escherichia coli, L3Cip3) loaded with 74 markers.\n",
      "2024-04-17 18:07:32,410 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018323.1 (Escherichia coli, KSC9) loaded with 70 markers.\n",
      "2024-04-17 18:07:32,511 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091712.1 (Escherichia coli, KFu014) loaded with 70 markers.\n",
      "2024-04-17 18:07:32,642 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064103.1 (Escherichia coli, IR53017) loaded with 78 markers.\n",
      "2024-04-17 18:07:32,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023815.1 (Escherichia coli, IMT16316) loaded with 74 markers.\n",
      "2024-04-17 18:07:32,858 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010167.1 (Escherichia coli, H3) loaded with 72 markers.\n",
      "2024-04-17 18:07:32,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024687.1 (Escherichia coli, FUJ80154) loaded with 75 markers.\n",
      "2024-04-17 18:07:33,078 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033850.1 (Escherichia coli, FDAARGOS_497) loaded with 78 markers.\n",
      "2024-04-17 18:07:33,185 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070152.1 (Escherichia coli, FDAARGOS_1285) loaded with 77 markers.\n",
      "2024-04-17 18:07:33,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040919.1 (Escherichia coli, FC853_EC) loaded with 78 markers.\n",
      "2024-04-17 18:07:33,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063476.1 (Escherichia coli, EM10-18-28) loaded with 71 markers.\n",
      "2024-04-17 18:07:33,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063518.1 (Escherichia coli, EH01-18-02) loaded with 76 markers.\n",
      "2024-04-17 18:07:33,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009859.1 (Escherichia coli, ECONIH1) loaded with 78 markers.\n",
      "2024-04-17 18:07:34,050 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019029.1 (Escherichia coli, Ecol_881) loaded with 78 markers.\n",
      "2024-04-17 18:07:34,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018948.1 (Escherichia coli, Ecol_224) loaded with 77 markers.\n",
      "2024-04-17 18:07:34,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085183.1 (Escherichia coli, EC7518) loaded with 72 markers.\n",
      "2024-04-17 18:07:34,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060899.1 (Escherichia coli, EC21) loaded with 75 markers.\n",
      "2024-04-17 18:07:34,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060914.1 (Escherichia coli, EC18) loaded with 83 markers.\n",
      "2024-04-17 18:07:34,884 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060919.1 (Escherichia coli, EC15) loaded with 83 markers.\n",
      "2024-04-17 18:07:35,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060924.1 (Escherichia coli, EC14) loaded with 56 markers.\n",
      "2024-04-17 18:07:35,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060933.1 (Escherichia coli, EC12) loaded with 82 markers.\n",
      "2024-04-17 18:07:35,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024821.1 (Escherichia coli, CREC-591) loaded with 75 markers.\n",
      "2024-04-17 18:07:35,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053723.1 (Escherichia coli, CP66-6_Sichuan) loaded with 72 markers.\n",
      "2024-04-17 18:07:35,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041300.1 (Escherichia coli, CLSC36) loaded with 75 markers.\n",
      "2024-04-17 18:07:35,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042896.1 (Escherichia coli, CFSAN061771) loaded with 70 markers.\n",
      "2024-04-17 18:07:35,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021722.1 (Escherichia coli, AR_0128) loaded with 74 markers.\n",
      "2024-04-17 18:07:35,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021535.1 (Escherichia coli, AR_0119) loaded with 74 markers.\n",
      "2024-04-17 18:07:36,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020048.1 (Escherichia coli, AR_0118) loaded with 74 markers.\n",
      "2024-04-17 18:07:36,135 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026755.1 (Escherichia coli, AR_0077) loaded with 74 markers.\n",
      "2024-04-17 18:07:36,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049967.1 (Escherichia coli, ARL09_232) loaded with 58 markers.\n",
      "2024-04-17 18:07:36,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046396.1 (Escherichia coli, ampC_0069) loaded with 66 markers.\n",
      "2024-04-17 18:07:36,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074701.1 (Escherichia coli, 702_18) loaded with 71 markers.\n",
      "2024-04-17 18:07:36,565 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059003.1 (Escherichia coli, 2D) loaded with 79 markers.\n",
      "2024-04-17 18:07:36,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048107.1 (Escherichia coli, 201609) loaded with 75 markers.\n",
      "2024-04-17 18:07:36,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020509.1 (Escherichia coli, 165) loaded with 75 markers.\n",
      "2024-04-17 18:07:36,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023366.1 (Escherichia coli, 1428) loaded with 68 markers.\n",
      "2024-04-17 18:07:37,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048337.1 (Escherichia coli, 142) loaded with 62 markers.\n",
      "2024-04-17 18:07:37,128 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015229.1 (Escherichia coli, 06-00048) loaded with 77 markers.\n",
      "2024-04-17 18:07:37,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022536.1 (Escherichia coli, THO-008) loaded with 74 markers.\n",
      "2024-04-17 18:07:37,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022478.1 (Escherichia coli, STN0717-11) loaded with 74 markers.\n",
      "2024-04-17 18:07:37,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022279.1 (Escherichia coli, STEC299) loaded with 70 markers.\n",
      "2024-04-17 18:07:37,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051765.1 (Escherichia coli, SCU-113) loaded with 69 markers.\n",
      "2024-04-17 18:07:37,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028110.1 (Escherichia coli, RM8352) loaded with 77 markers.\n",
      "2024-04-17 18:07:37,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062901.1 (Escherichia coli, Res13-Lact-ER01-35) loaded with 74 markers.\n",
      "2024-04-17 18:07:37,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134315.1 (Escherichia coli, NCTC9064) loaded with 73 markers.\n",
      "2024-04-17 18:07:38,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890270.1 (Escherichia coli, MINF_7C-sc-2280452) loaded with 74 markers.\n",
      "2024-04-17 18:07:38,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053259.1 (Escherichia coli, GF3-3) loaded with 76 markers.\n",
      "2024-04-17 18:07:38,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031910.1 (Escherichia coli, FWSEC0006) loaded with 77 markers.\n",
      "2024-04-17 18:07:38,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071439.1 (Escherichia coli, EcPNK004) loaded with 74 markers.\n",
      "2024-04-17 18:07:38,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018957.1 (Escherichia coli, Ecol_316) loaded with 74 markers.\n",
      "2024-04-17 18:07:38,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033635.1 (Escherichia coli, ECCNB12-2) loaded with 73 markers.\n",
      "2024-04-17 18:07:38,818 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR025101.1 (Escherichia coli, EC-1639) loaded with 74 markers.\n",
      "2024-04-17 18:07:38,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082773.1 (Escherichia coli, CU42ET-1) loaded with 69 markers.\n",
      "2024-04-17 18:07:39,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076706.1 (Escherichia coli, CFIAFB20120266) loaded with 70 markers.\n",
      "2024-04-17 18:07:39,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022298.1 (Escherichia coli, BEC1-S17-ESBL-09) loaded with 73 markers.\n",
      "2024-04-17 18:07:39,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028739.1 (Escherichia coli, A36) loaded with 68 markers.\n",
      "2024-04-17 18:07:39,415 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028743.1 (Escherichia coli, A32) loaded with 72 markers.\n",
      "2024-04-17 18:07:39,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028753.1 (Escherichia coli, A22) loaded with 72 markers.\n",
      "2024-04-17 18:07:39,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028755.1 (Escherichia coli, A20) loaded with 71 markers.\n",
      "2024-04-17 18:07:39,826 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027317.1 (Escherichia coli, 2015C-3107) loaded with 77 markers.\n",
      "2024-04-17 18:07:39,978 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027454.1 (Escherichia coli, 2014C-4423) loaded with 77 markers.\n",
      "2024-04-17 18:07:40,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027351.1 (Escherichia coli, 2014C-3655) loaded with 77 markers.\n",
      "2024-04-17 18:07:40,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027435.1 (Escherichia coli, 2014C-3599) loaded with 77 markers.\n",
      "2024-04-17 18:07:40,436 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022407.1 (Escherichia coli, 16-9255) loaded with 77 markers.\n",
      "2024-04-17 18:07:40,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080260.1 (Escherichia coli, 135) loaded with 76 markers.\n",
      "2024-04-17 18:07:40,751 [DEBUG - chronostrain.database.parser.json] - Strain NC_017626.1 (Escherichia coli, 042) loaded with 64 markers.\n",
      "2024-04-17 18:07:40,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066144.1 (Escherichia coli, ZLWT) loaded with 72 markers.\n",
      "2024-04-17 18:07:41,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021202.1 (Escherichia coli, Z1002) loaded with 80 markers.\n",
      "2024-04-17 18:07:41,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025707.1 (Escherichia coli, YDC107) loaded with 77 markers.\n",
      "2024-04-17 18:07:41,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022261.1 (Escherichia coli, WP8-S18-ESBL-07) loaded with 68 markers.\n",
      "2024-04-17 18:07:41,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022222.1 (Escherichia coli, WP8-S17-ESBL-12) loaded with 74 markers.\n",
      "2024-04-17 18:07:41,686 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022207.1 (Escherichia coli, WP7-S18-ESBL-09) loaded with 74 markers.\n",
      "2024-04-17 18:07:41,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022173.1 (Escherichia coli, WP7-S17-ESBL-01) loaded with 93 markers.\n",
      "2024-04-17 18:07:42,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022069.1 (Escherichia coli, WP4-W18-ESBL-09) loaded with 76 markers.\n",
      "2024-04-17 18:07:42,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022229.3 (Escherichia coli, WCHEC96200) loaded with 76 markers.\n",
      "2024-04-17 18:07:42,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028589.2 (Escherichia coli, WCHEC4533) loaded with 76 markers.\n",
      "2024-04-17 18:07:42,420 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043295.1 (Escherichia coli, WCHEC040047) loaded with 77 markers.\n",
      "2024-04-17 18:07:42,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032879.1 (Escherichia coli, WCHEC000837) loaded with 76 markers.\n",
      "2024-04-17 18:07:42,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082355.1 (Escherichia coli, VCSW5) loaded with 72 markers.\n",
      "2024-04-17 18:07:42,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058596.1 (Escherichia coli, UPEC_U013) loaded with 72 markers.\n",
      "2024-04-17 18:07:42,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016497.1 (Escherichia coli, UPEC_26-1) loaded with 71 markers.\n",
      "2024-04-17 18:07:43,103 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040456.1 (Escherichia coli, UPEC132) loaded with 73 markers.\n",
      "2024-04-17 18:07:43,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047600.1 (Escherichia coli, Trh52) loaded with 68 markers.\n",
      "2024-04-17 18:07:43,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022533.1 (Escherichia coli, THO-006) loaded with 72 markers.\n",
      "2024-04-17 18:07:43,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080246.1 (Escherichia coli, SYNB1934) loaded with 72 markers.\n",
      "2024-04-17 18:07:43,610 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076301.1 (Escherichia coli, Survcare321) loaded with 76 markers.\n",
      "2024-04-17 18:07:43,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054161.1 (Escherichia coli, STO_Bone2) loaded with 79 markers.\n",
      "2024-04-17 18:07:43,894 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058874.1 (Escherichia coli, STLEFF_47) loaded with 67 markers.\n",
      "2024-04-17 18:07:44,019 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054965.1 (Escherichia coli, STIN_95) loaded with 73 markers.\n",
      "2024-04-17 18:07:44,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053245.1 (Escherichia coli, SCU-485) loaded with 67 markers.\n",
      "2024-04-17 18:07:44,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051744.1 (Escherichia coli, SCU-484) loaded with 73 markers.\n",
      "2024-04-17 18:07:44,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053247.1 (Escherichia coli, SCU-482) loaded with 89 markers.\n",
      "2024-04-17 18:07:44,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051694.1 (Escherichia coli, SCU-313) loaded with 79 markers.\n",
      "2024-04-17 18:07:44,708 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054363.1 (Escherichia coli, SCU-171) loaded with 66 markers.\n",
      "2024-04-17 18:07:44,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051700.1 (Escherichia coli, SCU-125) loaded with 72 markers.\n",
      "2024-04-17 18:07:44,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051706.1 (Escherichia coli, SCU-124) loaded with 73 markers.\n",
      "2024-04-17 18:07:45,083 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054368.1 (Escherichia coli, SCU-115) loaded with 73 markers.\n",
      "2024-04-17 18:07:45,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051725.1 (Escherichia coli, SCU-112) loaded with 71 markers.\n",
      "2024-04-17 18:07:45,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051727.1 (Escherichia coli, SCU-111) loaded with 68 markers.\n",
      "2024-04-17 18:07:45,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051738.1 (Escherichia coli, SCU-105) loaded with 72 markers.\n",
      "2024-04-17 18:07:45,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082827.1 (Escherichia coli, SCAID_WND2-2021_(3_145)) loaded with 81 markers.\n",
      "2024-04-17 18:07:45,704 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778152.1 (Escherichia coli, SC477) loaded with 69 markers.\n",
      "2024-04-17 18:07:45,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778146.1 (Escherichia coli, SC455) loaded with 69 markers.\n",
      "2024-04-17 18:07:45,907 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778142.1 (Escherichia coli, SC434) loaded with 69 markers.\n",
      "2024-04-17 18:07:46,020 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027103.1 (Escherichia coli, RM14723) loaded with 73 markers.\n",
      "2024-04-17 18:07:46,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068828.1 (Escherichia coli, RIVM_C030443) loaded with 74 markers.\n",
      "2024-04-17 18:07:46,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068825.1 (Escherichia coli, RIVM_C030256) loaded with 73 markers.\n",
      "2024-04-17 18:07:46,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068822.1 (Escherichia coli, RIVM_C029324) loaded with 69 markers.\n",
      "2024-04-17 18:07:46,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068817.1 (Escherichia coli, RIVM_C028786) loaded with 69 markers.\n",
      "2024-04-17 18:07:46,704 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068816.1 (Escherichia coli, RIVM_C028724) loaded with 69 markers.\n",
      "2024-04-17 18:07:46,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068815.1 (Escherichia coli, RIVM_C028620) loaded with 69 markers.\n",
      "2024-04-17 18:07:46,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068809.1 (Escherichia coli, RIVM_C028497) loaded with 60 markers.\n",
      "2024-04-17 18:07:47,062 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068806.1 (Escherichia coli, RIVM_C018576) loaded with 73 markers.\n",
      "2024-04-17 18:07:47,189 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068799.1 (Escherichia coli, RIVM_C017887) loaded with 69 markers.\n",
      "2024-04-17 18:07:47,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011915.1 (Escherichia coli, PSUO2) loaded with 73 markers.\n",
      "2024-04-17 18:07:47,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080163.1 (Escherichia coli, PK8275) loaded with 75 markers.\n",
      "2024-04-17 18:07:47,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080155.1 (Escherichia coli, PK8261) loaded with 75 markers.\n",
      "2024-04-17 18:07:47,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043181.1 (Escherichia coli, PG20180057) loaded with 72 markers.\n",
      "2024-04-17 18:07:47,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012379.1 (Escherichia coli, PAR) loaded with 73 markers.\n",
      "2024-04-17 18:07:47,978 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049201.1 (Escherichia coli, PapRG-04-4) loaded with 73 markers.\n",
      "2024-04-17 18:07:48,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085063.1 (Escherichia coli, P31_YH1_02.21) loaded with 78 markers.\n",
      "2024-04-17 18:07:48,192 [DEBUG - chronostrain.database.parser.json] - Strain NC_017634.1 (Escherichia coli, NRG_857C) loaded with 68 markers.\n",
      "2024-04-17 18:07:48,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047609.1 (Escherichia coli, NMBU__W06E18) loaded with 77 markers.\n",
      "2024-04-17 18:07:48,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043406.1 (Escherichia coli, NMBU-W13E19) loaded with 92 markers.\n",
      "2024-04-17 18:07:48,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044403.1 (Escherichia coli, NMBU-W10C18) loaded with 76 markers.\n",
      "2024-04-17 18:07:48,691 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058217.1 (Escherichia coli, Nissle_1917) loaded with 72 markers.\n",
      "2024-04-17 18:07:48,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022686.1 (Escherichia coli, Nissle_1917) loaded with 72 markers.\n",
      "2024-04-17 18:07:48,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007799.1 (Escherichia coli, Nissle_1917) loaded with 75 markers.\n",
      "2024-04-17 18:07:49,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035486.1 (Escherichia coli, NissleGFP_pZE21) loaded with 72 markers.\n",
      "2024-04-17 18:07:49,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016007.1 (Escherichia coli, NGF1) loaded with 72 markers.\n",
      "2024-04-17 18:07:49,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134237.1 (Escherichia coli, NCTC9022) loaded with 70 markers.\n",
      "2024-04-17 18:07:49,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072787.1 (Escherichia coli, NC101) loaded with 72 markers.\n",
      "2024-04-17 18:07:49,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041304.1 (Escherichia coli, MSHS_133) loaded with 70 markers.\n",
      "2024-04-17 18:07:49,618 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890405.1 (Escherichia coli, MSB1_7C-sc-2280317) loaded with 79 markers.\n",
      "2024-04-17 18:07:49,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130564.1 (Escherichia coli, MS14387) loaded with 84 markers.\n",
      "2024-04-17 18:07:49,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056077.1 (Escherichia coli, MS1170) loaded with 71 markers.\n",
      "2024-04-17 18:07:49,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890198.1 (Escherichia coli, MINF_1D-sc-2280456) loaded with 73 markers.\n",
      "2024-04-17 18:07:50,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019903.1 (Escherichia coli, MDR_56) loaded with 71 markers.\n",
      "2024-04-17 18:07:50,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063153.1 (Escherichia coli, M7424) loaded with 74 markers.\n",
      "2024-04-17 18:07:50,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031256.1 (Escherichia coli, M16807) loaded with 73 markers.\n",
      "2024-04-17 18:07:50,380 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061337.1 (Escherichia coli, M11957) loaded with 74 markers.\n",
      "2024-04-17 18:07:50,488 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061339.1 (Escherichia coli, M00057) loaded with 74 markers.\n",
      "2024-04-17 18:07:50,587 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068394.1 (Escherichia coli, M-17) loaded with 72 markers.\n",
      "2024-04-17 18:07:50,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082771.1 (Escherichia coli, LF82) loaded with 68 markers.\n",
      "2024-04-17 18:07:50,787 [DEBUG - chronostrain.database.parser.json] - Strain NC_011993.1 (Escherichia coli, LF82) loaded with 68 markers.\n",
      "2024-04-17 18:07:50,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091673.1 (Escherichia coli, KKo009) loaded with 71 markers.\n",
      "2024-04-17 18:07:51,004 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063046.1 (Escherichia coli, KC-Dl-1) loaded with 73 markers.\n",
      "2024-04-17 18:07:51,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016358.1 (Escherichia coli, K-15KW01) loaded with 71 markers.\n",
      "2024-04-17 18:07:51,234 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072784.1 (Escherichia coli, JA0266) loaded with 72 markers.\n",
      "2024-04-17 18:07:51,340 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072785.1 (Escherichia coli, JA0265) loaded with 72 markers.\n",
      "2024-04-17 18:07:51,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072786.1 (Escherichia coli, JA0257) loaded with 72 markers.\n",
      "2024-04-17 18:07:51,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070227.1 (Escherichia coli, JA0072) loaded with 72 markers.\n",
      "2024-04-17 18:07:51,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082772.1 (Escherichia coli, HM670) loaded with 71 markers.\n",
      "2024-04-17 18:07:51,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053080.1 (Escherichia coli, HB37) loaded with 70 markers.\n",
      "2024-04-17 18:07:51,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048647.1 (Escherichia coli, GW-AmxH19) loaded with 73 markers.\n",
      "2024-04-17 18:07:52,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053258.1 (Escherichia coli, GF4-3) loaded with 61 markers.\n",
      "2024-04-17 18:07:52,135 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060075.1 (Escherichia coli, G3_10) loaded with 98 markers.\n",
      "2024-04-17 18:07:52,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019455.1 (Escherichia coli, FHI_NMBU_03) loaded with 62 markers.\n",
      "2024-04-17 18:07:52,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065152.1 (Escherichia coli, FEX669) loaded with 72 markers.\n",
      "2024-04-17 18:07:52,444 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033762.1 (Escherichia coli, FDAARGOS_536) loaded with 62 markers.\n",
      "2024-04-17 18:07:52,568 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023960.1 (Escherichia coli, FDAARGOS_448) loaded with 76 markers.\n",
      "2024-04-17 18:07:52,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014111.1 (Escherichia coli, FDAARGOS_144) loaded with 72 markers.\n",
      "2024-04-17 18:07:52,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069945.1 (Escherichia coli, FDAARGOS_1373) loaded with 69 markers.\n",
      "2024-04-17 18:07:52,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070063.1 (Escherichia coli, FDAARGOS_1372) loaded with 67 markers.\n",
      "2024-04-17 18:07:52,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070077.1 (Escherichia coli, FDAARGOS_1371) loaded with 76 markers.\n",
      "2024-04-17 18:07:53,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070098.1 (Escherichia coli, FDAARGOS_1370) loaded with 76 markers.\n",
      "2024-04-17 18:07:53,207 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070041.1 (Escherichia coli, FDAARGOS_1293) loaded with 71 markers.\n",
      "2024-04-17 18:07:53,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069980.1 (Escherichia coli, FDAARGOS_1291) loaded with 76 markers.\n",
      "2024-04-17 18:07:53,436 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070048.1 (Escherichia coli, FDAARGOS_1290) loaded with 84 markers.\n",
      "2024-04-17 18:07:53,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069882.1 (Escherichia coli, FDAARGOS_1289) loaded with 74 markers.\n",
      "2024-04-17 18:07:53,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070007.1 (Escherichia coli, FDAARGOS_1288) loaded with 76 markers.\n",
      "2024-04-17 18:07:53,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070071.1 (Escherichia coli, FDAARGOS_1287) loaded with 68 markers.\n",
      "2024-04-17 18:07:53,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069893.1 (Escherichia coli, FDAARGOS_1286) loaded with 69 markers.\n",
      "2024-04-17 18:07:53,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070170.1 (Escherichia coli, FDAARGOS_1284) loaded with 67 markers.\n",
      "2024-04-17 18:07:54,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070162.1 (Escherichia coli, FDAARGOS_1283) loaded with 71 markers.\n",
      "2024-04-17 18:07:54,207 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069865.1 (Escherichia coli, FDAARGOS_1282) loaded with 76 markers.\n",
      "2024-04-17 18:07:54,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070134.1 (Escherichia coli, FDAARGOS_1281) loaded with 76 markers.\n",
      "2024-04-17 18:07:54,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069973.1 (Escherichia coli, FDAARGOS_1280) loaded with 76 markers.\n",
      "2024-04-17 18:07:54,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069957.1 (Escherichia coli, FDAARGOS_1279) loaded with 76 markers.\n",
      "2024-04-17 18:07:54,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069522.1 (Escherichia coli, FDAARGOS_1267) loaded with 73 markers.\n",
      "2024-04-17 18:07:54,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069517.1 (Escherichia coli, FDAARGOS_1262) loaded with 72 markers.\n",
      "2024-04-17 18:07:54,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069441.1 (Escherichia coli, FDAARGOS_1261) loaded with 73 markers.\n",
      "2024-04-17 18:07:55,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069438.1 (Escherichia coli, FDAARGOS_1260) loaded with 69 markers.\n",
      "2024-04-17 18:07:55,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069559.1 (Escherichia coli, FDAARGOS_1259) loaded with 68 markers.\n",
      "2024-04-17 18:07:55,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069500.1 (Escherichia coli, FDAARGOS_1258) loaded with 73 markers.\n",
      "2024-04-17 18:07:55,380 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069480.1 (Escherichia coli, FDAARGOS_1257) loaded with 70 markers.\n",
      "2024-04-17 18:07:55,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069451.1 (Escherichia coli, FDAARGOS_1256) loaded with 70 markers.\n",
      "2024-04-17 18:07:55,635 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069447.1 (Escherichia coli, FDAARGOS_1255) loaded with 71 markers.\n",
      "2024-04-17 18:07:55,764 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069453.1 (Escherichia coli, FDAARGOS_1254) loaded with 67 markers.\n",
      "2024-04-17 18:07:55,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069549.1 (Escherichia coli, FDAARGOS_1253) loaded with 68 markers.\n",
      "2024-04-17 18:07:56,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069445.1 (Escherichia coli, FDAARGOS_1251) loaded with 73 markers.\n",
      "2024-04-17 18:07:56,119 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043542.1 (Escherichia coli, F2_81) loaded with 78 markers.\n",
      "2024-04-17 18:07:56,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076123.1 (Escherichia coli, F11) loaded with 70 markers.\n",
      "2024-04-17 18:07:56,395 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031215.1 (Escherichia coli, Es_ST80_L1_NDM_10_2017) loaded with 71 markers.\n",
      "2024-04-17 18:07:56,529 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063214.1 (Escherichia coli, ERP001) loaded with 69 markers.\n",
      "2024-04-17 18:07:56,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083481.1 (Escherichia coli, elppa9) loaded with 76 markers.\n",
      "2024-04-17 18:07:56,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083503.1 (Escherichia coli, elppa5) loaded with 77 markers.\n",
      "2024-04-17 18:07:56,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083530.1 (Escherichia coli, elppa2) loaded with 70 markers.\n",
      "2024-04-17 18:07:57,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063492.1 (Escherichia coli, EF5-18-41) loaded with 74 markers.\n",
      "2024-04-17 18:07:57,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065136.1 (Escherichia coli, EE72d) loaded with 76 markers.\n",
      "2024-04-17 18:07:57,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054236.1 (Escherichia coli, EcPF5) loaded with 71 markers.\n",
      "2024-04-17 18:07:57,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054227.1 (Escherichia coli, EcPF15) loaded with 72 markers.\n",
      "2024-04-17 18:07:57,618 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054230.1 (Escherichia coli, EcPF14) loaded with 73 markers.\n",
      "2024-04-17 18:07:57,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014667.1 (Escherichia coli, ECONIH2) loaded with 70 markers.\n",
      "2024-04-17 18:07:57,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056036.1 (Escherichia coli, EC31) loaded with 75 markers.\n",
      "2024-04-17 18:07:57,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049101.1 (Escherichia coli, EC28) loaded with 68 markers.\n",
      "2024-04-17 18:07:58,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058574.1 (Escherichia coli, EC1) loaded with 82 markers.\n",
      "2024-04-17 18:07:58,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066836.1 (Escherichia coli, Ec-FL1-1X) loaded with 76 markers.\n",
      "2024-04-17 18:07:58,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086334.1 (Escherichia coli, EBJ003) loaded with 77 markers.\n",
      "2024-04-17 18:07:58,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049198.1 (Escherichia coli, E686) loaded with 76 markers.\n",
      "2024-04-17 18:07:58,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049197.1 (Escherichia coli, E597) loaded with 76 markers.\n",
      "2024-04-17 18:07:58,739 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032066.1 (Escherichia coli, E166) loaded with 71 markers.\n",
      "2024-04-17 18:07:58,889 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049196.1 (Escherichia coli, E118) loaded with 76 markers.\n",
      "2024-04-17 18:07:59,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071375.1 (Escherichia coli, E105) loaded with 78 markers.\n",
      "2024-04-17 18:07:59,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090611.1 (Escherichia coli, DCE7) loaded with 66 markers.\n",
      "2024-04-17 18:07:59,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061206.1 (Escherichia coli, DA61218) loaded with 70 markers.\n",
      "2024-04-17 18:07:59,370 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029579.1 (Escherichia coli, DA33137) loaded with 77 markers.\n",
      "2024-04-17 18:07:59,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010151.1 (Escherichia coli, D8) loaded with 73 markers.\n",
      "2024-04-17 18:07:59,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010157.1 (Escherichia coli, D10) loaded with 67 markers.\n",
      "2024-04-17 18:07:59,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082776.1 (Escherichia coli, CU532-9) loaded with 72 markers.\n",
      "2024-04-17 18:07:59,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082777.1 (Escherichia coli, CU39ES-1) loaded with 68 markers.\n",
      "2024-04-17 18:08:00,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082774.1 (Escherichia coli, CU37RT-2) loaded with 71 markers.\n",
      "2024-04-17 18:08:00,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032078.1 (Escherichia coli, CT30) loaded with 62 markers.\n",
      "2024-04-17 18:08:00,661 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074576.1 (Escherichia coli, CS18F) loaded with 78 markers.\n",
      "2024-04-17 18:08:00,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019243.1 (Escherichia coli, Combat2C1) loaded with 70 markers.\n",
      "2024-04-17 18:08:00,942 [DEBUG - chronostrain.database.parser.json] - Strain NC_017651.1 (Escherichia coli, clone_D_i2) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,067 [DEBUG - chronostrain.database.parser.json] - Strain NC_017652.1 (Escherichia coli, clone_D_i14) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,170 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048603.1 (Escherichia coli, ChlosBP-23-1) loaded with 72 markers.\n",
      "2024-04-17 18:08:01,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058222.1 (Escherichia coli, CFT073) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051263.1 (Escherichia coli, CFT073) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,600 [DEBUG - chronostrain.database.parser.json] - Strain NC_004431.1 (Escherichia coli, CFT073) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028320.1 (Escherichia coli, CFSAN067215) loaded with 71 markers.\n",
      "2024-04-17 18:08:01,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023142.1 (Escherichia coli, CFSAN061770) loaded with 78 markers.\n",
      "2024-04-17 18:08:02,015 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010121.1 (Escherichia coli, C4) loaded with 72 markers.\n",
      "2024-04-17 18:08:02,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025251.1 (Escherichia coli, BH100_substr._MG2017) loaded with 70 markers.\n",
      "2024-04-17 18:08:02,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024650.2 (Escherichia coli, BH100_substr._MG2014) loaded with 70 markers.\n",
      "2024-04-17 18:08:02,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025703.1 (Escherichia coli, BH100N_substr._MG2017) loaded with 60 markers.\n",
      "2024-04-17 18:08:02,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025716.1 (Escherichia coli, BH100L_substr._MG2017) loaded with 68 markers.\n",
      "2024-04-17 18:08:02,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032085.1 (Escherichia coli, ATCC_25922) loaded with 72 markers.\n",
      "2024-04-17 18:08:02,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037449.1 (Escherichia coli, ATCC_25922) loaded with 72 markers.\n",
      "2024-04-17 18:08:02,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009072.1 (Escherichia coli, ATCC_25922) loaded with 72 markers.\n",
      "2024-04-17 18:08:03,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027134.1 (Escherichia coli, AR_0372) loaded with 78 markers.\n",
      "2024-04-17 18:08:03,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032261.1 (Escherichia coli, AR_0067) loaded with 76 markers.\n",
      "2024-04-17 18:08:03,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024886.1 (Escherichia coli, AR_0017) loaded with 71 markers.\n",
      "2024-04-17 18:08:03,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024862.1 (Escherichia coli, AR_0015) loaded with 76 markers.\n",
      "2024-04-17 18:08:03,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024851.1 (Escherichia coli, AR_0006) loaded with 76 markers.\n",
      "2024-04-17 18:08:03,596 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006834.2 (Escherichia coli, APEC_O2-211) loaded with 70 markers.\n",
      "2024-04-17 18:08:03,704 [DEBUG - chronostrain.database.parser.json] - Strain NC_017631.1 (Escherichia coli, ABU_83972) loaded with 68 markers.\n",
      "2024-04-17 18:08:03,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028768.1 (Escherichia coli, A6) loaded with 70 markers.\n",
      "2024-04-17 18:08:03,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028734.1 (Escherichia coli, A42) loaded with 70 markers.\n",
      "2024-04-17 18:08:04,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028770.1 (Escherichia coli, A4) loaded with 73 markers.\n",
      "2024-04-17 18:08:04,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029327.1 (Escherichia coli, A37) loaded with 68 markers.\n",
      "2024-04-17 18:08:04,310 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028742.1 (Escherichia coli, A33) loaded with 72 markers.\n",
      "2024-04-17 18:08:04,459 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028771.1 (Escherichia coli, A3) loaded with 70 markers.\n",
      "2024-04-17 18:08:04,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028749.1 (Escherichia coli, A26) loaded with 70 markers.\n",
      "2024-04-17 18:08:04,768 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028750.1 (Escherichia coli, A25) loaded with 70 markers.\n",
      "2024-04-17 18:08:04,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028751.1 (Escherichia coli, A24) loaded with 70 markers.\n",
      "2024-04-17 18:08:05,036 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028754.1 (Escherichia coli, A21) loaded with 72 markers.\n",
      "2024-04-17 18:08:05,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028761.1 (Escherichia coli, A13) loaded with 67 markers.\n",
      "2024-04-17 18:08:05,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028764.1 (Escherichia coli, A10) loaded with 72 markers.\n",
      "2024-04-17 18:08:05,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027461.1 (Escherichia coli, 95-3322) loaded with 72 markers.\n",
      "2024-04-17 18:08:05,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048304.1 (Escherichia coli, 9) loaded with 65 markers.\n",
      "2024-04-17 18:08:05,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058220.1 (Escherichia coli, 83972) loaded with 68 markers.\n",
      "2024-04-17 18:08:05,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023820.1 (Escherichia coli, 7_2) loaded with 76 markers.\n",
      "2024-04-17 18:08:06,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059917.1 (Escherichia coli, 64.1) loaded with 72 markers.\n",
      "2024-04-17 18:08:06,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058223.1 (Escherichia coli, 550) loaded with 74 markers.\n",
      "2024-04-17 18:08:06,431 [DEBUG - chronostrain.database.parser.json] - Strain NC_008253.1 (Escherichia coli, 536) loaded with 69 markers.\n",
      "2024-04-17 18:08:06,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033884.1 (Escherichia coli, 50579417) loaded with 78 markers.\n",
      "2024-04-17 18:08:06,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071073.1 (Escherichia coli, 3347558) loaded with 74 markers.\n",
      "2024-04-17 18:08:06,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023357.1 (Escherichia coli, 317) loaded with 69 markers.\n",
      "2024-04-17 18:08:07,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081010.1 (Escherichia coli, 25POL) loaded with 72 markers.\n",
      "2024-04-17 18:08:07,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063983.1 (Escherichia coli, 25DN) loaded with 72 markers.\n",
      "2024-04-17 18:08:07,433 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020516.1 (Escherichia coli, 222) loaded with 76 markers.\n",
      "2024-04-17 18:08:07,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP017610.1 (Escherichia coli, 20Ec-P-124) loaded with 69 markers.\n",
      "2024-04-17 18:08:07,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030768.1 (Escherichia coli, 2017C-4173W12) loaded with 77 markers.\n",
      "2024-04-17 18:08:07,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025214.2 (Escherichia coli, 2017.09.02CC) loaded with 68 markers.\n",
      "2024-04-17 18:08:07,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027452.1 (Escherichia coli, 2014C-3338) loaded with 73 markers.\n",
      "2024-04-17 18:08:07,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027440.1 (Escherichia coli, 2012C-4502) loaded with 71 markers.\n",
      "2024-04-17 18:08:08,126 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013025.1 (Escherichia coli, 2009C-3133) loaded with 72 markers.\n",
      "2024-04-17 18:08:08,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059907.1 (Escherichia coli, 181.1) loaded with 66 markers.\n",
      "2024-04-17 18:08:08,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024138.1 (Escherichia coli, 14EC020) loaded with 69 markers.\n",
      "2024-04-17 18:08:08,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019273.1 (Escherichia coli, 13P477T) loaded with 68 markers.\n",
      "2024-04-17 18:08:08,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_HG994851.1 (Escherichia coli, 127) loaded with 60 markers.\n",
      "2024-04-17 18:08:08,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_HG994850.1 (Escherichia coli, 127) loaded with 69 markers.\n",
      "2024-04-17 18:08:08,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032515.1 (Escherichia coli, 118UI) loaded with 80 markers.\n",
      "2024-04-17 18:08:08,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023388.1 (Escherichia coli, 1105) loaded with 67 markers.\n",
      "2024-04-17 18:08:09,033 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029164.1 (Escherichia coli, 104) loaded with 71 markers.\n",
      "2024-04-17 18:08:09,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020495.1 (Escherichia coli, 103) loaded with 76 markers.\n",
      "2024-04-17 18:08:09,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035498.1 (Escherichia coli, 04-00955) loaded with 73 markers.\n",
      "2024-04-17 18:08:09,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041620.1 (Shigella flexneri, C32) loaded with 72 markers.\n",
      "2024-04-17 18:08:09,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020545.1 (Escherichia coli, ZJ3920) loaded with 71 markers.\n",
      "2024-04-17 18:08:09,594 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014497.1 (Escherichia coli, ZH193) loaded with 72 markers.\n",
      "2024-04-17 18:08:09,766 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014522.1 (Escherichia coli, ZH063) loaded with 70 markers.\n",
      "2024-04-17 18:08:09,918 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062752.1 (Escherichia coli, Z910) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062755.1 (Escherichia coli, Z903) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062758.1 (Escherichia coli, Z892) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062761.1 (Escherichia coli, Z887) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062763.1 (Escherichia coli, Z885) loaded with 78 markers.\n",
      "2024-04-17 18:08:10,633 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062782.1 (Escherichia coli, Z869) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062766.1 (Escherichia coli, Z866) loaded with 79 markers.\n",
      "2024-04-17 18:08:10,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062769.1 (Escherichia coli, Z852) loaded with 79 markers.\n",
      "2024-04-17 18:08:11,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062771.1 (Escherichia coli, Z570) loaded with 79 markers.\n",
      "2024-04-17 18:08:11,261 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062774.1 (Escherichia coli, Z563) loaded with 79 markers.\n",
      "2024-04-17 18:08:11,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062778.1 (Escherichia coli, Z1836) loaded with 81 markers.\n",
      "2024-04-17 18:08:11,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062700.1 (Escherichia coli, Z1835) loaded with 79 markers.\n",
      "2024-04-17 18:08:11,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062702.1 (Escherichia coli, Z1834) loaded with 80 markers.\n",
      "2024-04-17 18:08:11,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062705.1 (Escherichia coli, Z1833) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062708.1 (Escherichia coli, Z1832) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062711.1 (Escherichia coli, Z1831) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062713.1 (Escherichia coli, Z1830) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062715.1 (Escherichia coli, Z1826) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,686 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062717.1 (Escherichia coli, Z1825) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062719.1 (Escherichia coli, Z1816) loaded with 79 markers.\n",
      "2024-04-17 18:08:12,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062721.1 (Escherichia coli, Z1815) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062723.1 (Escherichia coli, Z1814) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062725.1 (Escherichia coli, Z1813) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062727.1 (Escherichia coli, Z1812) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062729.1 (Escherichia coli, Z1811) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062780.1 (Escherichia coli, Z1769) loaded with 79 markers.\n",
      "2024-04-17 18:08:13,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062731.1 (Escherichia coli, Z1768) loaded with 79 markers.\n",
      "2024-04-17 18:08:14,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062733.1 (Escherichia coli, Z1767) loaded with 79 markers.\n",
      "2024-04-17 18:08:14,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062736.1 (Escherichia coli, Z1766) loaded with 79 markers.\n",
      "2024-04-17 18:08:14,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062739.1 (Escherichia coli, Z1723) loaded with 80 markers.\n",
      "2024-04-17 18:08:14,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062742.1 (Escherichia coli, Z1626) loaded with 79 markers.\n",
      "2024-04-17 18:08:14,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062744.1 (Escherichia coli, Z1615) loaded with 80 markers.\n",
      "2024-04-17 18:08:14,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062746.1 (Escherichia coli, Z1504) loaded with 79 markers.\n",
      "2024-04-17 18:08:14,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062749.1 (Escherichia coli, Z1486) loaded with 79 markers.\n",
      "2024-04-17 18:08:15,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038284.1 (Escherichia coli, YB14-1) loaded with 79 markers.\n",
      "2024-04-17 18:08:15,217 [DEBUG - chronostrain.database.parser.json] - Strain NC_017906.1 (Escherichia coli, Xuzhou21) loaded with 80 markers.\n",
      "2024-04-17 18:08:15,354 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012802.1 (Escherichia coli, WS4202) loaded with 80 markers.\n",
      "2024-04-17 18:08:15,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022171.1 (Escherichia coli, WP5-S18-ESBL-09) loaded with 71 markers.\n",
      "2024-04-17 18:08:15,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022064.1 (Escherichia coli, WP4-W18-ESBL-08) loaded with 72 markers.\n",
      "2024-04-17 18:08:15,687 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022087.1 (Escherichia coli, WP4-S18-ESBL-07) loaded with 80 markers.\n",
      "2024-04-17 18:08:15,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022044.1 (Escherichia coli, WP4-S17-ESBL-03) loaded with 73 markers.\n",
      "2024-04-17 18:08:15,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021998.1 (Escherichia coli, WP3-W18-ESBL-07) loaded with 72 markers.\n",
      "2024-04-17 18:08:16,006 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021963.1 (Escherichia coli, WP3-W18-CRE-03) loaded with 74 markers.\n",
      "2024-04-17 18:08:16,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022032.1 (Escherichia coli, WP3-S18-ESBL-08) loaded with 72 markers.\n",
      "2024-04-17 18:08:16,259 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064167.1 (Escherichia coli, Wll001) loaded with 80 markers.\n",
      "2024-04-17 18:08:16,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076241.1 (Escherichia coli, Vac_07.1) loaded with 75 markers.\n",
      "2024-04-17 18:08:16,500 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064825.1 (Escherichia coli, UTI89) loaded with 69 markers.\n",
      "2024-04-17 18:08:16,616 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062985.1 (Escherichia coli, UTI89) loaded with 69 markers.\n",
      "2024-04-17 18:08:16,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062228.1 (Escherichia coli, UTI89) loaded with 69 markers.\n",
      "2024-04-17 18:08:16,867 [DEBUG - chronostrain.database.parser.json] - Strain NC_007946.1 (Escherichia coli, UTI89) loaded with 69 markers.\n",
      "2024-04-17 18:08:16,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071522.1 (Escherichia coli, UPEC129) loaded with 71 markers.\n",
      "2024-04-17 18:08:17,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049852.1 (Escherichia coli, UPEC-774U_190308) loaded with 73 markers.\n",
      "2024-04-17 18:08:17,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049839.1 (Escherichia coli, UPEC-455U_190220) loaded with 73 markers.\n",
      "2024-04-17 18:08:17,337 [DEBUG - chronostrain.database.parser.json] - Strain NC_017632.1 (Escherichia coli, UM146) loaded with 70 markers.\n",
      "2024-04-17 18:08:17,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013658.1 (Escherichia coli, uk_P46212) loaded with 72 markers.\n",
      "2024-04-17 18:08:17,657 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035720.1 (Escherichia coli, U15A) loaded with 72 markers.\n",
      "2024-04-17 18:08:17,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035516.1 (Escherichia coli, U14A) loaded with 72 markers.\n",
      "2024-04-17 18:08:17,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035477.1 (Escherichia coli, U13A) loaded with 72 markers.\n",
      "2024-04-17 18:08:18,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035476.1 (Escherichia coli, U12A) loaded with 72 markers.\n",
      "2024-04-17 18:08:18,206 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038287.1 (Escherichia coli, TX_376-2) loaded with 78 markers.\n",
      "2024-04-17 18:08:18,349 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038290.1 (Escherichia coli, TX_265-1) loaded with 81 markers.\n",
      "2024-04-17 18:08:18,489 [DEBUG - chronostrain.database.parser.json] - Strain NC_013008.1 (Escherichia coli, TW14359) loaded with 80 markers.\n",
      "2024-04-17 18:08:18,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038494.1 (Escherichia coli, TT12B) loaded with 76 markers.\n",
      "2024-04-17 18:08:18,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038496.1 (Escherichia coli, TT12A) loaded with 80 markers.\n",
      "2024-04-17 18:08:18,866 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022549.1 (Escherichia coli, THO-015) loaded with 72 markers.\n",
      "2024-04-17 18:08:18,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022525.1 (Escherichia coli, THO-003) loaded with 72 markers.\n",
      "2024-04-17 18:08:19,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038292.1 (Escherichia coli, TB21-1) loaded with 79 markers.\n",
      "2024-04-17 18:08:19,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021207.1 (Escherichia coli, strain_Z247) loaded with 70 markers.\n",
      "2024-04-17 18:08:19,351 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043950.1 (Escherichia coli, ST95-32) loaded with 70 markers.\n",
      "2024-04-17 18:08:19,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063774.1 (Escherichia coli, ST131) loaded with 72 markers.\n",
      "2024-04-17 18:08:19,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038300.1 (Escherichia coli, SS_TX_754-1) loaded with 79 markers.\n",
      "2024-04-17 18:08:19,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038302.1 (Escherichia coli, SS_TX_313-1) loaded with 79 markers.\n",
      "2024-04-17 18:08:19,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038305.1 (Escherichia coli, SS_NE_1040-1) loaded with 79 markers.\n",
      "2024-04-17 18:08:20,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010304.1 (Escherichia coli, SS52) loaded with 80 markers.\n",
      "2024-04-17 18:08:20,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008805.1 (Escherichia coli, SS17) loaded with 80 markers.\n",
      "2024-04-17 18:08:20,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015023.1 (Escherichia coli, SRCC_1675) loaded with 80 markers.\n",
      "2024-04-17 18:08:20,470 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018784.1 (Escherichia coli, SK1144) loaded with 62 markers.\n",
      "2024-04-17 18:08:20,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038309.1 (Escherichia coli, Show_KS_470-1) loaded with 81 markers.\n",
      "2024-04-17 18:08:20,734 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012625.1 (Escherichia coli, SF-468) loaded with 71 markers.\n",
      "2024-04-17 18:08:20,833 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012631.1 (Escherichia coli, SF-173) loaded with 70 markers.\n",
      "2024-04-17 18:08:20,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012633.1 (Escherichia coli, SF-166) loaded with 72 markers.\n",
      "2024-04-17 18:08:21,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012635.1 (Escherichia coli, SF-088) loaded with 74 markers.\n",
      "2024-04-17 18:08:21,134 [DEBUG - chronostrain.database.parser.json] - Strain NC_013654.1 (Escherichia coli, SE15) loaded with 71 markers.\n",
      "2024-04-17 18:08:21,248 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051688.1 (Escherichia coli, SCU-387) loaded with 70 markers.\n",
      "2024-04-17 18:08:21,351 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053231.1 (Escherichia coli, SCU-306) loaded with 71 markers.\n",
      "2024-04-17 18:08:21,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053251.2 (Escherichia coli, SCU-204) loaded with 76 markers.\n",
      "2024-04-17 18:08:21,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054345.1 (Escherichia coli, SCU-176) loaded with 64 markers.\n",
      "2024-04-17 18:08:21,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051711.1 (Escherichia coli, SCU-123) loaded with 71 markers.\n",
      "2024-04-17 18:08:21,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051714.1 (Escherichia coli, SCU-122) loaded with 72 markers.\n",
      "2024-04-17 18:08:21,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051719.1 (Escherichia coli, SCU-116) loaded with 71 markers.\n",
      "2024-04-17 18:08:21,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051735.1 (Escherichia coli, SCU-108) loaded with 71 markers.\n",
      "2024-04-17 18:08:22,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051849.1 (Escherichia coli, SCU-101) loaded with 72 markers.\n",
      "2024-04-17 18:08:22,229 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048920.1 (Escherichia coli, SCU-101) loaded with 72 markers.\n",
      "2024-04-17 18:08:22,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082831.1 (Escherichia coli, SCAID_WND1-2021_(1_128)) loaded with 74 markers.\n",
      "2024-04-17 18:08:22,454 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778150.1 (Escherichia coli, SC492) loaded with 72 markers.\n",
      "2024-04-17 18:08:22,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR778147.1 (Escherichia coli, SC418) loaded with 72 markers.\n",
      "2024-04-17 18:08:22,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014495.1 (Escherichia coli, SaT040) loaded with 70 markers.\n",
      "2024-04-17 18:08:22,856 [DEBUG - chronostrain.database.parser.json] - Strain NC_002695.2 (Escherichia coli, Sakai_substr._RIMD_0509952) loaded with 80 markers.\n",
      "2024-04-17 18:08:22,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022730.1 (Escherichia coli, SA186) loaded with 70 markers.\n",
      "2024-04-17 18:08:23,074 [DEBUG - chronostrain.database.parser.json] - Strain NC_011742.1 (Escherichia coli, S88) loaded with 69 markers.\n",
      "2024-04-17 18:08:23,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036245.1 (Escherichia coli, S65EC) loaded with 72 markers.\n",
      "2024-04-17 18:08:23,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076687.1 (Escherichia coli, S22EC) loaded with 70 markers.\n",
      "2024-04-17 18:08:23,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076689.1 (Escherichia coli, S21EC) loaded with 70 markers.\n",
      "2024-04-17 18:08:23,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076697.1 (Escherichia coli, S10EC) loaded with 72 markers.\n",
      "2024-04-17 18:08:23,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076693.1 (Escherichia coli, S103EC) loaded with 72 markers.\n",
      "2024-04-17 18:08:23,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046527.1 (Escherichia coli, RM19259) loaded with 80 markers.\n",
      "2024-04-17 18:08:24,038 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068802.1 (Escherichia coli, RIVM_C018249) loaded with 74 markers.\n",
      "2024-04-17 18:08:24,207 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076230.1 (Escherichia coli, Rafaela_II) loaded with 80 markers.\n",
      "2024-04-17 18:08:24,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018488.1 (Escherichia coli, PV15-279) loaded with 84 markers.\n",
      "2024-04-17 18:08:24,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042246.1 (Escherichia coli, PU-1) loaded with 71 markers.\n",
      "2024-04-17 18:08:24,615 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049204.1 (Escherichia coli, PapM-32-1) loaded with 77 markers.\n",
      "2024-04-17 18:08:24,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021288.1 (Escherichia coli, PA45B) loaded with 70 markers.\n",
      "2024-04-17 18:08:24,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017669.1 (Escherichia coli, PA20) loaded with 80 markers.\n",
      "2024-04-17 18:08:25,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049085.2 (Escherichia coli, p4A) loaded with 72 markers.\n",
      "2024-04-17 18:08:25,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049077.2 (Escherichia coli, p11A) loaded with 72 markers.\n",
      "2024-04-17 18:08:25,230 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080117.1 (Escherichia coli, OV24) loaded with 72 markers.\n",
      "2024-04-17 18:08:25,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038313.1 (Escherichia coli, OK1) loaded with 80 markers.\n",
      "2024-04-17 18:08:25,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015085.1 (Escherichia coli, O25b:H4) loaded with 72 markers.\n",
      "2024-04-17 18:08:25,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007275.1 (Escherichia coli, O18) loaded with 69 markers.\n",
      "2024-04-17 18:08:25,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032789.1 (Escherichia coli, NZRM4169) loaded with 79 markers.\n",
      "2024-04-17 18:08:25,907 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032791.1 (Escherichia coli, NZRM4165) loaded with 78 markers.\n",
      "2024-04-17 18:08:26,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032793.1 (Escherichia coli, NZRM3614) loaded with 77 markers.\n",
      "2024-04-17 18:08:26,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019777.1 (Escherichia coli, NU14) loaded with 69 markers.\n",
      "2024-04-17 18:08:26,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024720.1 (Escherichia coli, NQ3) loaded with 70 markers.\n",
      "2024-04-17 18:08:26,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038324.1 (Escherichia coli, NE_1169-1) loaded with 80 markers.\n",
      "2024-04-17 18:08:26,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038328.1 (Escherichia coli, NE_1092-2) loaded with 81 markers.\n",
      "2024-04-17 18:08:26,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038316.1 (Escherichia coli, NE92) loaded with 78 markers.\n",
      "2024-04-17 18:08:26,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038319.1 (Escherichia coli, NE122) loaded with 78 markers.\n",
      "2024-04-17 18:08:27,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038321.1 (Escherichia coli, NE1127) loaded with 80 markers.\n",
      "2024-04-17 18:08:27,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134031.1 (Escherichia coli, NCTC11151) loaded with 73 markers.\n",
      "2024-04-17 18:08:27,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017249.1 (Escherichia coli, NADC_5570_86-24_6565) loaded with 78 markers.\n",
      "2024-04-17 18:08:27,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017251.1 (Escherichia coli, NADC_5570_86-24_6564) loaded with 78 markers.\n",
      "2024-04-17 18:08:27,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038333.1 (Escherichia coli, N8B7-2) loaded with 79 markers.\n",
      "2024-04-17 18:08:27,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031609.1 (Escherichia coli, N3) loaded with 70 markers.\n",
      "2024-04-17 18:08:27,727 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890410.1 (Escherichia coli, MSB2_1A-sc-2280429) loaded with 62 markers.\n",
      "2024-04-17 18:08:27,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890693.1 (Escherichia coli, MSB1_4E-sc-2280348) loaded with 72 markers.\n",
      "2024-04-17 18:08:27,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025401.1 (Escherichia coli, MS8345) loaded with 72 markers.\n",
      "2024-04-17 18:08:28,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026853.1 (Escherichia coli, MS7163) loaded with 69 markers.\n",
      "2024-04-17 18:08:28,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010876.1 (Escherichia coli, MNCRE44) loaded with 72 markers.\n",
      "2024-04-17 18:08:28,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030111.1 (Escherichia coli, MCJCHV-1) loaded with 74 markers.\n",
      "2024-04-17 18:08:28,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040107.1 (Escherichia coli, MB9-1) loaded with 81 markers.\n",
      "2024-04-17 18:08:28,687 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039834.1 (Escherichia coli, MB41-1) loaded with 78 markers.\n",
      "2024-04-17 18:08:28,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040314.1 (Escherichia coli, MA11) loaded with 77 markers.\n",
      "2024-04-17 18:08:28,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040313.1 (Escherichia coli, M7638) loaded with 77 markers.\n",
      "2024-04-17 18:08:29,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080118.1 (Escherichia coli, M70) loaded with 72 markers.\n",
      "2024-04-17 18:08:29,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080119.1 (Escherichia coli, M45) loaded with 72 markers.\n",
      "2024-04-17 18:08:29,396 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080120.1 (Escherichia coli, M24) loaded with 72 markers.\n",
      "2024-04-17 18:08:29,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053296.1 (Escherichia coli, M1_5) loaded with 74 markers.\n",
      "2024-04-17 18:08:29,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089270.1 (Escherichia coli, M1300706002) loaded with 83 markers.\n",
      "2024-04-17 18:08:29,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089272.1 (Escherichia coli, M1300706001A) loaded with 80 markers.\n",
      "2024-04-17 18:08:30,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038336.1 (Escherichia coli, LSU61) loaded with 76 markers.\n",
      "2024-04-17 18:08:30,243 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024717.1 (Escherichia coli, LS4) loaded with 70 markers.\n",
      "2024-04-17 18:08:30,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091682.1 (Escherichia coli, KKo003) loaded with 62 markers.\n",
      "2024-04-17 18:08:30,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053079.1 (Escherichia coli, K56-43-un) loaded with 73 markers.\n",
      "2024-04-17 18:08:30,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013835.1 (Escherichia coli, JJ2434) loaded with 72 markers.\n",
      "2024-04-17 18:08:30,739 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014316.1 (Escherichia coli, JJ1887) loaded with 72 markers.\n",
      "2024-04-17 18:08:30,863 [DEBUG - chronostrain.database.parser.json] - Strain NC_022648.1 (Escherichia coli, JJ1886) loaded with 72 markers.\n",
      "2024-04-17 18:08:31,003 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014314.1 (Escherichia coli, JEONG-1266) loaded with 80 markers.\n",
      "2024-04-17 18:08:31,101 [DEBUG - chronostrain.database.parser.json] - Strain NC_017628.1 (Escherichia coli, IHE3034) loaded with 69 markers.\n",
      "2024-04-17 18:08:31,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026491.1 (Escherichia coli, HS13-1) loaded with 70 markers.\n",
      "2024-04-17 18:08:31,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040305.1 (Escherichia coli, HB6) loaded with 80 markers.\n",
      "2024-04-17 18:08:31,473 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038339.1 (Escherichia coli, H6437) loaded with 78 markers.\n",
      "2024-04-17 18:08:31,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038342.1 (Escherichia coli, H2495) loaded with 80 markers.\n",
      "2024-04-17 18:08:31,709 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021454.1 (Escherichia coli, H105) loaded with 72 markers.\n",
      "2024-04-17 18:08:31,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038344.1 (Escherichia coli, Gim1-1) loaded with 78 markers.\n",
      "2024-04-17 18:08:31,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014488.1 (Escherichia coli, G749) loaded with 68 markers.\n",
      "2024-04-17 18:08:32,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038346.1 (Escherichia coli, G5295) loaded with 81 markers.\n",
      "2024-04-17 18:08:32,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031913.1 (Escherichia coli, FWSEC0004) loaded with 80 markers.\n",
      "2024-04-17 18:08:32,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035545.1 (Escherichia coli, FSIS11705876) loaded with 80 markers.\n",
      "2024-04-17 18:08:32,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016625.1 (Escherichia coli, FRIK944) loaded with 81 markers.\n",
      "2024-04-17 18:08:32,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034384.1 (Escherichia coli, FRIK804) loaded with 80 markers.\n",
      "2024-04-17 18:08:32,835 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015842.1 (Escherichia coli, FRIK2533) loaded with 81 markers.\n",
      "2024-04-17 18:08:32,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015843.2 (Escherichia coli, FRIK2455) loaded with 81 markers.\n",
      "2024-04-17 18:08:33,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015846.1 (Escherichia coli, FRIK2069) loaded with 81 markers.\n",
      "2024-04-17 18:08:33,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016755.1 (Escherichia coli, FORC_044) loaded with 80 markers.\n",
      "2024-04-17 18:08:33,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022050.2 (Escherichia coli, FDAARGOS_293) loaded with 80 markers.\n",
      "2024-04-17 18:08:33,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069990.1 (Escherichia coli, FDAARGOS_1304) loaded with 71 markers.\n",
      "2024-04-17 18:08:33,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070045.1 (Escherichia coli, FDAARGOS_1303) loaded with 69 markers.\n",
      "2024-04-17 18:08:33,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069935.1 (Escherichia coli, FDAARGOS_1302) loaded with 72 markers.\n",
      "2024-04-17 18:08:33,892 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070111.1 (Escherichia coli, FDAARGOS_1301) loaded with 72 markers.\n",
      "2024-04-17 18:08:34,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069996.1 (Escherichia coli, FDAARGOS_1300) loaded with 72 markers.\n",
      "2024-04-17 18:08:34,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069978.1 (Escherichia coli, FDAARGOS_1299) loaded with 70 markers.\n",
      "2024-04-17 18:08:34,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070115.1 (Escherichia coli, FDAARGOS_1298) loaded with 72 markers.\n",
      "2024-04-17 18:08:34,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070148.1 (Escherichia coli, FDAARGOS_1297) loaded with 69 markers.\n",
      "2024-04-17 18:08:34,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070066.1 (Escherichia coli, FDAARGOS_1296) loaded with 68 markers.\n",
      "2024-04-17 18:08:34,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069890.1 (Escherichia coli, FDAARGOS_1295) loaded with 69 markers.\n",
      "2024-04-17 18:08:34,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070124.1 (Escherichia coli, FDAARGOS_1292) loaded with 72 markers.\n",
      "2024-04-17 18:08:34,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069589.1 (Escherichia coli, FDAARGOS_1266) loaded with 72 markers.\n",
      "2024-04-17 18:08:35,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069583.1 (Escherichia coli, FDAARGOS_1265) loaded with 72 markers.\n",
      "2024-04-17 18:08:35,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069488.1 (Escherichia coli, FDAARGOS_1264) loaded with 73 markers.\n",
      "2024-04-17 18:08:35,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069528.1 (Escherichia coli, FDAARGOS_1252) loaded with 70 markers.\n",
      "2024-04-17 18:08:35,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038349.1 (Escherichia coli, F8952) loaded with 78 markers.\n",
      "2024-04-17 18:08:35,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038351.1 (Escherichia coli, F8798) loaded with 82 markers.\n",
      "2024-04-17 18:08:35,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038353.1 (Escherichia coli, F8797) loaded with 80 markers.\n",
      "2024-04-17 18:08:35,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038282.1 (Escherichia coli, F8492) loaded with 80 markers.\n",
      "2024-04-17 18:08:36,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038355.1 (Escherichia coli, F8092B) loaded with 79 markers.\n",
      "2024-04-17 18:08:36,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038357.1 (Escherichia coli, F7508) loaded with 80 markers.\n",
      "2024-04-17 18:08:36,350 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038360.1 (Escherichia coli, F7386) loaded with 78 markers.\n",
      "2024-04-17 18:08:36,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038363.1 (Escherichia coli, F7349) loaded with 80 markers.\n",
      "2024-04-17 18:08:36,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038366.1 (Escherichia coli, F6667) loaded with 80 markers.\n",
      "2024-04-17 18:08:36,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038369.1 (Escherichia coli, F6321) loaded with 80 markers.\n",
      "2024-04-17 18:08:36,984 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038372.1 (Escherichia coli, F6294) loaded with 82 markers.\n",
      "2024-04-17 18:08:37,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040311.1 (Escherichia coli, F3398) loaded with 76 markers.\n",
      "2024-04-17 18:08:37,376 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038374.1 (Escherichia coli, F3113) loaded with 78 markers.\n",
      "2024-04-17 18:08:37,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040307.1 (Escherichia coli, F1_E4) loaded with 77 markers.\n",
      "2024-04-17 18:08:37,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038376.1 (Escherichia coli, F1273) loaded with 78 markers.\n",
      "2024-04-17 18:08:37,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025328.1 (Escherichia coli, ExPEC_XM) loaded with 70 markers.\n",
      "2024-04-17 18:08:38,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992168.1 (Escherichia coli, Escherichia_coli_str._TO60) loaded with 72 markers.\n",
      "2024-04-17 18:08:38,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992192.1 (Escherichia coli, Escherichia_coli_str._TO217) loaded with 72 markers.\n",
      "2024-04-17 18:08:38,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992190.1 (Escherichia coli, Escherichia_coli_str._TO148) loaded with 72 markers.\n",
      "2024-04-17 18:08:38,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992180.1 (Escherichia coli, Escherichia_coli_str._TO124) loaded with 72 markers.\n",
      "2024-04-17 18:08:38,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070593.1 (Escherichia coli, ESC53083) loaded with 72 markers.\n",
      "2024-04-17 18:08:38,999 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032795.1 (Escherichia coli, ERL06-2503) loaded with 78 markers.\n",
      "2024-04-17 18:08:39,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032797.1 (Escherichia coli, ERL06-2497) loaded with 80 markers.\n",
      "2024-04-17 18:08:39,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032801.1 (Escherichia coli, ERL06-2442) loaded with 78 markers.\n",
      "2024-04-17 18:08:39,580 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032803.1 (Escherichia coli, ERL05-1306) loaded with 78 markers.\n",
      "2024-04-17 18:08:39,745 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032805.1 (Escherichia coli, ERL05-0623) loaded with 80 markers.\n",
      "2024-04-17 18:08:39,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032808.1 (Escherichia coli, ERL04-3476) loaded with 78 markers.\n",
      "2024-04-17 18:08:40,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032811.1 (Escherichia coli, ERL03-1416) loaded with 78 markers.\n",
      "2024-04-17 18:08:40,252 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT827011.1 (Escherichia coli, EPEC_E2348_69_variety_2) loaded with 73 markers.\n",
      "2024-04-17 18:08:40,412 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT903847.1 (Escherichia coli, EPEC1) loaded with 73 markers.\n",
      "2024-04-17 18:08:40,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083512.1 (Escherichia coli, elppa4) loaded with 75 markers.\n",
      "2024-04-17 18:08:40,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063515.1 (Escherichia coli, EH01-18-04-A) loaded with 71 markers.\n",
      "2024-04-17 18:08:40,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008957.1 (Escherichia coli, EDL933) loaded with 80 markers.\n",
      "2024-04-17 18:08:41,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054232.1 (Escherichia coli, EcPF7) loaded with 70 markers.\n",
      "2024-04-17 18:08:41,334 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054214.1 (Escherichia coli, EcPF40) loaded with 73 markers.\n",
      "2024-04-17 18:08:41,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054219.1 (Escherichia coli, EcPF18) loaded with 72 markers.\n",
      "2024-04-17 18:08:41,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066756.1 (Escherichia coli, ECP19-798) loaded with 80 markers.\n",
      "2024-04-17 18:08:41,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066753.1 (Escherichia coli, ECP19-598) loaded with 81 markers.\n",
      "2024-04-17 18:08:42,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066748.1 (Escherichia coli, ECP19-2498) loaded with 81 markers.\n",
      "2024-04-17 18:08:42,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066759.1 (Escherichia coli, ECP19-198) loaded with 80 markers.\n",
      "2024-04-17 18:08:42,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040572.1 (Escherichia coli, ECP17-46) loaded with 78 markers.\n",
      "2024-04-17 18:08:42,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040570.1 (Escherichia coli, ECP17-1298) loaded with 80 markers.\n",
      "2024-04-17 18:08:42,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019015.1 (Escherichia coli, Ecol_AZ162) loaded with 72 markers.\n",
      "2024-04-17 18:08:42,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019012.1 (Escherichia coli, Ecol_AZ161) loaded with 71 markers.\n",
      "2024-04-17 18:08:42,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019008.1 (Escherichia coli, Ecol_AZ159) loaded with 71 markers.\n",
      "2024-04-17 18:08:43,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019000.1 (Escherichia coli, Ecol_AZ153) loaded with 72 markers.\n",
      "2024-04-17 18:08:43,190 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018991.1 (Escherichia coli, Ecol_AZ146) loaded with 72 markers.\n",
      "2024-04-17 18:08:43,305 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018983.1 (Escherichia coli, Ecol_867) loaded with 72 markers.\n",
      "2024-04-17 18:08:43,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015074.2 (Escherichia coli, Ecol_745) loaded with 66 markers.\n",
      "2024-04-17 18:08:43,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015069.1 (Escherichia coli, Ecol_743) loaded with 70 markers.\n",
      "2024-04-17 18:08:43,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015138.1 (Escherichia coli, Ecol_732) loaded with 72 markers.\n",
      "2024-04-17 18:08:43,811 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018979.1 (Escherichia coli, Ecol_656) loaded with 72 markers.\n",
      "2024-04-17 18:08:43,955 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018970.1 (Escherichia coli, Ecol_542) loaded with 71 markers.\n",
      "2024-04-17 18:08:44,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015076.1 (Escherichia coli, Ecol_448) loaded with 71 markers.\n",
      "2024-04-17 18:08:44,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018953.1 (Escherichia coli, Ecol_276) loaded with 72 markers.\n",
      "2024-04-17 18:08:44,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019020.1 (Escherichia coli, Ecol_244) loaded with 73 markers.\n",
      "2024-04-17 18:08:44,486 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015159.1 (Escherichia coli, Eco889) loaded with 71 markers.\n",
      "2024-04-17 18:08:44,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047710.1 (Escherichia coli, Eco-15) loaded with 71 markers.\n",
      "2024-04-17 18:08:44,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021840.1 (Escherichia coli, EC974) loaded with 78 markers.\n",
      "2024-04-17 18:08:44,899 [DEBUG - chronostrain.database.parser.json] - Strain NZ_HG941718.1 (Escherichia coli, EC958) loaded with 73 markers.\n",
      "2024-04-17 18:08:45,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060953.1 (Escherichia coli, EC8) loaded with 73 markers.\n",
      "2024-04-17 18:08:45,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059130.1 (Escherichia coli, Ec55) loaded with 72 markers.\n",
      "2024-04-17 18:08:45,411 [DEBUG - chronostrain.database.parser.json] - Strain NC_011353.1 (Escherichia coli, EC4115) loaded with 80 markers.\n",
      "2024-04-17 18:08:45,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056114.1 (Escherichia coli, Ec394-330266) loaded with 72 markers.\n",
      "2024-04-17 18:08:45,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021844.1 (Escherichia coli, EC1515) loaded with 78 markers.\n",
      "2024-04-17 18:08:45,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070229.1 (Escherichia coli, EC12) loaded with 68 markers.\n",
      "2024-04-17 18:08:46,072 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS999560.1 (Escherichia coli, EC-TO143) loaded with 74 markers.\n",
      "2024-04-17 18:08:46,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028483.1 (Escherichia coli, E41-1) loaded with 73 markers.\n",
      "2024-04-17 18:08:46,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038380.1 (Escherichia coli, E32511) loaded with 78 markers.\n",
      "2024-04-17 18:08:46,572 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022362.1 (Escherichia coli, E302) loaded with 71 markers.\n",
      "2024-04-17 18:08:46,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059840.2 (Escherichia coli, E2348_69) loaded with 63 markers.\n",
      "2024-04-17 18:08:46,874 [DEBUG - chronostrain.database.parser.json] - Strain NC_011601.1 (Escherichia coli, E2348_69) loaded with 73 markers.\n",
      "2024-04-17 18:08:47,033 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022351.1 (Escherichia coli, E138) loaded with 73 markers.\n",
      "2024-04-17 18:08:47,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080116.1 (Escherichia coli, Dog168) loaded with 72 markers.\n",
      "2024-04-17 18:08:47,289 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038398.1 (Escherichia coli, DEC4E) loaded with 78 markers.\n",
      "2024-04-17 18:08:47,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029576.1 (Escherichia coli, DA33135) loaded with 71 markers.\n",
      "2024-04-17 18:08:47,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040316.1 (Escherichia coli, CV261) loaded with 77 markers.\n",
      "2024-04-17 18:08:47,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082775.1 (Escherichia coli, CU568-3) loaded with 63 markers.\n",
      "2024-04-17 18:08:47,803 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076709.1 (Escherichia coli, CFIAFB20140388) loaded with 76 markers.\n",
      "2024-04-17 18:08:47,956 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022689.1 (Escherichia coli, CDC#03-98) loaded with 78 markers.\n",
      "2024-04-17 18:08:48,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013831.1 (Escherichia coli, CD306) loaded with 72 markers.\n",
      "2024-04-17 18:08:48,239 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041337.1 (Escherichia coli, CCUG_73778) loaded with 72 markers.\n",
      "2024-04-17 18:08:48,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059281.1 (Escherichia coli, C9) loaded with 71 markers.\n",
      "2024-04-17 18:08:48,496 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059279.1 (Escherichia coli, C7) loaded with 71 markers.\n",
      "2024-04-17 18:08:48,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035366.1 (Escherichia coli, C1-057) loaded with 77 markers.\n",
      "2024-04-17 18:08:48,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR740758.1 (Escherichia coli, BEN5202) loaded with 72 markers.\n",
      "2024-04-17 18:08:48,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR740776.1 (Escherichia coli, BEN2908) loaded with 71 markers.\n",
      "2024-04-17 18:08:49,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042250.1 (Escherichia coli, BCE049) loaded with 71 markers.\n",
      "2024-04-17 18:08:49,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038402.1 (Escherichia coli, BB24-1) loaded with 81 markers.\n",
      "2024-04-17 18:08:49,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076245.1 (Escherichia coli, Balcarce_24.2) loaded with 80 markers.\n",
      "2024-04-17 18:08:49,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076243.1 (Escherichia coli, Balcarce_14.2) loaded with 80 markers.\n",
      "2024-04-17 18:08:49,727 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130545.1 (Escherichia coli, B36) loaded with 72 markers.\n",
      "2024-04-17 18:08:49,856 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076344.1 (Escherichia coli, AVS0096) loaded with 73 markers.\n",
      "2024-04-17 18:08:50,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045975.1 (Escherichia coli, AUSMDU00002545) loaded with 80 markers.\n",
      "2024-04-17 18:08:50,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045863.1 (Escherichia coli, ATCC_43890) loaded with 78 markers.\n",
      "2024-04-17 18:08:50,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015853.1 (Escherichia coli, ATCC_43889) loaded with 80 markers.\n",
      "2024-04-17 18:08:50,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041623.1 (Escherichia coli, ATCC_43888) loaded with 76 markers.\n",
      "2024-04-17 18:08:50,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038405.1 (Escherichia coli, ATCC_35150) loaded with 80 markers.\n",
      "2024-04-17 18:08:50,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033092.2 (Escherichia coli, ATCC_11775) loaded with 72 markers.\n",
      "2024-04-17 18:08:50,818 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030337.1 (Escherichia coli, AR_451) loaded with 72 markers.\n",
      "2024-04-17 18:08:50,938 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020116.1 (Escherichia coli, AR_0104) loaded with 72 markers.\n",
      "2024-04-17 18:08:51,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032265.1 (Escherichia coli, AR_0089) loaded with 72 markers.\n",
      "2024-04-17 18:08:51,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032201.1 (Escherichia coli, AR_0086) loaded with 72 markers.\n",
      "2024-04-17 18:08:51,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027534.1 (Escherichia coli, AR_0081) loaded with 59 markers.\n",
      "2024-04-17 18:08:51,410 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021689.1 (Escherichia coli, AR_0058) loaded with 72 markers.\n",
      "2024-04-17 18:08:51,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021935.1 (Escherichia coli, AR_0055) loaded with 72 markers.\n",
      "2024-04-17 18:08:51,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044140.1 (Escherichia coli, AR-0430) loaded with 80 markers.\n",
      "2024-04-17 18:08:51,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044143.1 (Escherichia coli, AR-0429) loaded with 80 markers.\n",
      "2024-04-17 18:08:51,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044145.1 (Escherichia coli, AR-0428) loaded with 82 markers.\n",
      "2024-04-17 18:08:52,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044148.1 (Escherichia coli, AR-0427) loaded with 80 markers.\n",
      "2024-04-17 18:08:52,278 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006830.1 (Escherichia coli, APEC_O18) loaded with 69 markers.\n",
      "2024-04-17 18:08:52,400 [DEBUG - chronostrain.database.parser.json] - Strain NC_008563.1 (Escherichia coli, APEC_O1) loaded with 71 markers.\n",
      "2024-04-17 18:08:52,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043539.1 (Escherichia coli, Al_Ain) loaded with 78 markers.\n",
      "2024-04-17 18:08:52,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028741.1 (Escherichia coli, A34) loaded with 68 markers.\n",
      "2024-04-17 18:08:52,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028758.1 (Escherichia coli, A16) loaded with 74 markers.\n",
      "2024-04-17 18:08:52,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017446.1 (Escherichia coli, 9234) loaded with 78 markers.\n",
      "2024-04-17 18:08:53,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018252.2 (Escherichia coli, 9000) loaded with 80 markers.\n",
      "2024-04-17 18:08:53,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076235.1 (Escherichia coli, 9.1_Anguil) loaded with 78 markers.\n",
      "2024-04-17 18:08:53,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027457.1 (Escherichia coli, 88-3493) loaded with 75 markers.\n",
      "2024-04-17 18:08:53,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038408.1 (Escherichia coli, 86-24) loaded with 78 markers.\n",
      "2024-04-17 18:08:53,693 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017444.1 (Escherichia coli, 8368) loaded with 80 markers.\n",
      "2024-04-17 18:08:53,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021179.1 (Escherichia coli, 81009) loaded with 72 markers.\n",
      "2024-04-17 18:08:53,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018247.1 (Escherichia coli, 7784) loaded with 79 markers.\n",
      "2024-04-17 18:08:54,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038421.1 (Escherichia coli, 7636) loaded with 80 markers.\n",
      "2024-04-17 18:08:54,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038423.1 (Escherichia coli, 7409) loaded with 79 markers.\n",
      "2024-04-17 18:08:54,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089036.1 (Escherichia coli, 7386WT) loaded with 80 markers.\n",
      "2024-04-17 18:08:54,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089031.1 (Escherichia coli, 7386Nal) loaded with 80 markers.\n",
      "2024-04-17 18:08:54,764 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076232.1 (Escherichia coli, 7.1_Anguil) loaded with 69 markers.\n",
      "2024-04-17 18:08:54,929 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089032.1 (Escherichia coli, 6535WT) loaded with 82 markers.\n",
      "2024-04-17 18:08:55,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089034.1 (Escherichia coli, 6535NalR) loaded with 82 markers.\n",
      "2024-04-17 18:08:55,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015831.1 (Escherichia coli, 644-PT8) loaded with 83 markers.\n",
      "2024-04-17 18:08:55,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038428.1 (Escherichia coli, 611) loaded with 80 markers.\n",
      "2024-04-17 18:08:55,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089445.1 (Escherichia coli, 5-1) loaded with 71 markers.\n",
      "2024-04-17 18:08:55,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023826.1 (Escherichia coli, 4_4) loaded with 72 markers.\n",
      "2024-04-17 18:08:55,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023844.1 (Escherichia coli, 4_1-1) loaded with 72 markers.\n",
      "2024-04-17 18:08:56,108 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023849.1 (Escherichia coli, 4_0) loaded with 72 markers.\n",
      "2024-04-17 18:08:56,278 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038412.1 (Escherichia coli, 493_89) loaded with 78 markers.\n",
      "2024-04-17 18:08:56,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018245.1 (Escherichia coli, 472) loaded with 80 markers.\n",
      "2024-04-17 18:08:56,610 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017442.1 (Escherichia coli, 4276) loaded with 78 markers.\n",
      "2024-04-17 18:08:56,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039403.1 (Escherichia coli, 382634_2f) loaded with 68 markers.\n",
      "2024-04-17 18:08:56,835 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039404.1 (Escherichia coli, 377323_2f) loaded with 69 markers.\n",
      "2024-04-17 18:08:56,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018243.1 (Escherichia coli, 350) loaded with 80 markers.\n",
      "2024-04-17 18:08:57,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029420.1 (Escherichia coli, 3385) loaded with 72 markers.\n",
      "2024-04-17 18:08:57,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017440.1 (Escherichia coli, 3384) loaded with 80 markers.\n",
      "2024-04-17 18:08:57,469 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018241.1 (Escherichia coli, 319) loaded with 78 markers.\n",
      "2024-04-17 18:08:57,610 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038416.1 (Escherichia coli, 3-5-1) loaded with 78 markers.\n",
      "2024-04-17 18:08:57,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023853.1 (Escherichia coli, 2_0) loaded with 72 markers.\n",
      "2024-04-17 18:08:57,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015020.1 (Escherichia coli, 28RC1) loaded with 82 markers.\n",
      "2024-04-17 18:08:58,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018239.1 (Escherichia coli, 272) loaded with 78 markers.\n",
      "2024-04-17 18:08:58,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038425.1 (Escherichia coli, 2571) loaded with 80 markers.\n",
      "2024-04-17 18:08:58,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072980.1 (Escherichia coli, 2547) loaded with 72 markers.\n",
      "2024-04-17 18:08:58,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040309.1 (Escherichia coli, 21B8) loaded with 77 markers.\n",
      "2024-04-17 18:08:59,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017438.1 (Escherichia coli, 2159) loaded with 78 markers.\n",
      "2024-04-17 18:08:59,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017436.1 (Escherichia coli, 2149) loaded with 78 markers.\n",
      "2024-04-17 18:08:59,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062160.1 (Escherichia coli, 20R2R) loaded with 78 markers.\n",
      "2024-04-17 18:08:59,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051609.1 (Escherichia coli, 2019_APHA) loaded with 72 markers.\n",
      "2024-04-17 18:08:59,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051615.1 (Escherichia coli, 2017_APHA) loaded with 72 markers.\n",
      "2024-04-17 18:08:59,895 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030767.1 (Escherichia coli, 2017C-4109) loaded with 80 markers.\n",
      "2024-04-17 18:09:00,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034806.1 (Escherichia coli, 2010C-3347) loaded with 80 markers.\n",
      "2024-04-17 18:09:00,201 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034801.1 (Escherichia coli, 2010C-3142) loaded with 80 markers.\n",
      "2024-04-17 18:09:00,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034799.1 (Escherichia coli, 2009C-4687) loaded with 80 markers.\n",
      "2024-04-17 18:09:00,515 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034803.1 (Escherichia coli, 2009C-3554) loaded with 80 markers.\n",
      "2024-04-17 18:09:00,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034792.1 (Escherichia coli, 2009C-3378) loaded with 78 markers.\n",
      "2024-04-17 18:09:00,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038419.1 (Escherichia coli, 2-6-2) loaded with 81 markers.\n",
      "2024-04-17 18:09:00,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022326.1 (Escherichia coli, 18044) loaded with 72 markers.\n",
      "2024-04-17 18:09:01,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015832.1 (Escherichia coli, 180-PT54) loaded with 82 markers.\n",
      "2024-04-17 18:09:01,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038414.1 (Escherichia coli, 17B6-2) loaded with 79 markers.\n",
      "2024-04-17 18:09:01,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018237.1 (Escherichia coli, 155) loaded with 77 markers.\n",
      "2024-04-17 18:09:01,610 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028592.1 (Escherichia coli, 150) loaded with 81 markers.\n",
      "2024-04-17 18:09:01,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028596.1 (Escherichia coli, 149) loaded with 78 markers.\n",
      "2024-04-17 18:09:01,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028598.1 (Escherichia coli, 148) loaded with 78 markers.\n",
      "2024-04-17 18:09:02,070 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028600.1 (Escherichia coli, 147) loaded with 78 markers.\n",
      "2024-04-17 18:09:02,233 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028603.1 (Escherichia coli, 144) loaded with 79 markers.\n",
      "2024-04-17 18:09:02,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028607.1 (Escherichia coli, 143) loaded with 80 markers.\n",
      "2024-04-17 18:09:02,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028609.1 (Escherichia coli, 142) loaded with 80 markers.\n",
      "2024-04-17 18:09:02,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028614.1 (Escherichia coli, 141) loaded with 79 markers.\n",
      "2024-04-17 18:09:02,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028617.1 (Escherichia coli, 140) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,076 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028620.1 (Escherichia coli, 139) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028623.1 (Escherichia coli, 138) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028626.1 (Escherichia coli, 137) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028629.1 (Escherichia coli, 136) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028632.1 (Escherichia coli, 135) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028635.1 (Escherichia coli, 134) loaded with 79 markers.\n",
      "2024-04-17 18:09:03,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028638.1 (Escherichia coli, 133) loaded with 79 markers.\n",
      "2024-04-17 18:09:04,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028641.1 (Escherichia coli, 132) loaded with 79 markers.\n",
      "2024-04-17 18:09:04,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093011.1 (Escherichia coli, 131) loaded with 72 markers.\n",
      "2024-04-17 18:09:04,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028644.1 (Escherichia coli, 131) loaded with 79 markers.\n",
      "2024-04-17 18:09:04,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028647.1 (Escherichia coli, 130) loaded with 79 markers.\n",
      "2024-04-17 18:09:04,722 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028650.1 (Escherichia coli, 124) loaded with 80 markers.\n",
      "2024-04-17 18:09:04,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028652.1 (Escherichia coli, 123) loaded with 80 markers.\n",
      "2024-04-17 18:09:05,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028654.1 (Escherichia coli, 122) loaded with 78 markers.\n",
      "2024-04-17 18:09:05,159 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028656.1 (Escherichia coli, 121) loaded with 79 markers.\n",
      "2024-04-17 18:09:05,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028659.1 (Escherichia coli, 120) loaded with 79 markers.\n",
      "2024-04-17 18:09:05,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023386.1 (Escherichia coli, 1190) loaded with 73 markers.\n",
      "2024-04-17 18:09:05,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028662.1 (Escherichia coli, 119) loaded with 79 markers.\n",
      "2024-04-17 18:09:05,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028665.1 (Escherichia coli, 118) loaded with 79 markers.\n",
      "2024-04-17 18:09:05,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028668.1 (Escherichia coli, 117) loaded with 79 markers.\n",
      "2024-04-17 18:09:05,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051661.1 (Escherichia coli, 1162T7) loaded with 72 markers.\n",
      "2024-04-17 18:09:06,060 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051663.1 (Escherichia coli, 1162invT2) loaded with 72 markers.\n",
      "2024-04-17 18:09:06,181 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051659.1 (Escherichia coli, 1162C) loaded with 72 markers.\n",
      "2024-04-17 18:09:06,340 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028671.1 (Escherichia coli, 116) loaded with 79 markers.\n",
      "2024-04-17 18:09:06,504 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028674.1 (Escherichia coli, 115) loaded with 77 markers.\n",
      "2024-04-17 18:09:06,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028677.1 (Escherichia coli, 114) loaded with 79 markers.\n",
      "2024-04-17 18:09:06,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017434.1 (Escherichia coli, 1130) loaded with 78 markers.\n",
      "2024-04-17 18:09:07,004 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028680.1 (Escherichia coli, 113) loaded with 79 markers.\n",
      "2024-04-17 18:09:07,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028683.1 (Escherichia coli, 112) loaded with 78 markers.\n",
      "2024-04-17 18:09:07,338 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028685.1 (Escherichia coli, 111) loaded with 78 markers.\n",
      "2024-04-17 18:09:07,501 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028687.1 (Escherichia coli, 110) loaded with 80 markers.\n",
      "2024-04-17 18:09:07,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028690.1 (Escherichia coli, 109) loaded with 80 markers.\n",
      "2024-04-17 18:09:07,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028693.1 (Escherichia coli, 108) loaded with 80 markers.\n",
      "2024-04-17 18:09:07,951 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028695.1 (Escherichia coli, 107) loaded with 80 markers.\n",
      "2024-04-17 18:09:08,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018250.1 (Escherichia coli, 10671) loaded with 78 markers.\n",
      "2024-04-17 18:09:08,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028698.1 (Escherichia coli, 106) loaded with 80 markers.\n",
      "2024-04-17 18:09:08,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028700.1 (Escherichia coli, 105) loaded with 80 markers.\n",
      "2024-04-17 18:09:08,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022650.1 (Escherichia coli, 09-02E) loaded with 73 markers.\n",
      "2024-04-17 18:09:08,641 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034797.1 (Escherichia coli, 08-3918) loaded with 78 markers.\n",
      "2024-04-17 18:09:08,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034808.1 (Escherichia coli, 08-3914) loaded with 80 markers.\n",
      "2024-04-17 18:09:08,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034794.1 (Escherichia coli, 06-3462) loaded with 78 markers.\n",
      "2024-04-17 18:09:09,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055092.1 (Shigella flexneri, SWHIN_110) loaded with 58 markers.\n",
      "2024-04-17 18:09:09,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055041.1 (Shigella flexneri, SWHEFF_59) loaded with 65 markers.\n",
      "2024-04-17 18:09:09,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058796.1 (Shigella flexneri, STLIN_8) loaded with 79 markers.\n",
      "2024-04-17 18:09:09,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058786.1 (Shigella flexneri, STLIN_4) loaded with 60 markers.\n",
      "2024-04-17 18:09:09,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058849.1 (Shigella flexneri, STLEFF_36) loaded with 59 markers.\n",
      "2024-04-17 18:09:09,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058839.1 (Shigella flexneri, STLEFF_34) loaded with 63 markers.\n",
      "2024-04-17 18:09:09,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058886.1 (Shigella flexneri, STLEFF_33) loaded with 59 markers.\n",
      "2024-04-17 18:09:09,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058833.1 (Shigella flexneri, STLEFF_27) loaded with 64 markers.\n",
      "2024-04-17 18:09:09,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058831.1 (Shigella flexneri, STLE4) loaded with 59 markers.\n",
      "2024-04-17 18:09:10,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054977.1 (Shigella flexneri, STIN_92) loaded with 60 markers.\n",
      "2024-04-17 18:09:10,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054992.1 (Shigella flexneri, STIN_87) loaded with 60 markers.\n",
      "2024-04-17 18:09:10,392 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055052.1 (Shigella dysenteriae, SWHEFF_51) loaded with 77 markers.\n",
      "2024-04-17 18:09:10,572 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055055.1 (Shigella dysenteriae, SWHEFF_49) loaded with 77 markers.\n",
      "2024-04-17 18:09:10,745 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055063.1 (Shigella dysenteriae, SWHE2) loaded with 73 markers.\n",
      "2024-04-17 18:09:10,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080247.1 (Escherichia sp., TM-G17TGC) loaded with 89 markers.\n",
      "2024-04-17 18:09:11,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079924.1 (Escherichia sp., TC-EC600-tetX4) loaded with 85 markers.\n",
      "2024-04-17 18:09:11,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047461.1 (Escherichia coli, ZF34) loaded with 68 markers.\n",
      "2024-04-17 18:09:11,403 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047455.1 (Escherichia coli, ZF31) loaded with 68 markers.\n",
      "2024-04-17 18:09:11,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090305.1 (Escherichia coli, Z503) loaded with 94 markers.\n",
      "2024-04-17 18:09:11,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037910.1 (Escherichia coli, YSP8-1) loaded with 69 markers.\n",
      "2024-04-17 18:09:11,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041452.1 (Escherichia coli, YPE3) loaded with 66 markers.\n",
      "2024-04-17 18:09:12,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041442.1 (Escherichia coli, YPE12) loaded with 66 markers.\n",
      "2024-04-17 18:09:12,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041448.1 (Escherichia coli, YPE10) loaded with 73 markers.\n",
      "2024-04-17 18:09:12,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023230.2 (Escherichia coli, YJ4) loaded with 59 markers.\n",
      "2024-04-17 18:09:12,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023226.1 (Escherichia coli, YJ3) loaded with 58 markers.\n",
      "2024-04-17 18:09:12,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013483.1 (Escherichia coli, Y5) loaded with 53 markers.\n",
      "2024-04-17 18:09:12,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081008.1 (Escherichia coli, XL10-gold-ultracompetent) loaded with 78 markers.\n",
      "2024-04-17 18:09:12,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081007.1 (Escherichia coli, XL1-Blue) loaded with 76 markers.\n",
      "2024-04-17 18:09:13,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068041.1 (Escherichia coli, XJW9B277) loaded with 57 markers.\n",
      "2024-04-17 18:09:13,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067399.1 (Escherichia coli, XJW9B263) loaded with 59 markers.\n",
      "2024-04-17 18:09:13,374 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069386.1 (Escherichia coli, XH1815) loaded with 85 markers.\n",
      "2024-04-17 18:09:13,495 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089142.1 (Escherichia coli, XD35) loaded with 50 markers.\n",
      "2024-04-17 18:09:13,657 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035822.1 (Escherichia coli, WS4437A-1) loaded with 71 markers.\n",
      "2024-04-17 18:09:13,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022287.1 (Escherichia coli, WP9-S17-ESBL-11) loaded with 87 markers.\n",
      "2024-04-17 18:09:14,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022245.1 (Escherichia coli, WP8-S18-CRE-02) loaded with 73 markers.\n",
      "2024-04-17 18:09:14,196 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022221.1 (Escherichia coli, WP8-S17-ESBL-08) loaded with 75 markers.\n",
      "2024-04-17 18:09:14,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022177.1 (Escherichia coli, WP7-S17-ESBL-04) loaded with 74 markers.\n",
      "2024-04-17 18:09:14,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022159.1 (Escherichia coli, WP5-S18-ESBL-07) loaded with 75 markers.\n",
      "2024-04-17 18:09:14,661 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022105.1 (Escherichia coli, WP5-S17-ESBL-10) loaded with 56 markers.\n",
      "2024-04-17 18:09:14,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022049.1 (Escherichia coli, WP4-S17-ESBL-08) loaded with 76 markers.\n",
      "2024-04-17 18:09:14,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022003.1 (Escherichia coli, WP3-W18-ESBL-08) loaded with 75 markers.\n",
      "2024-04-17 18:09:15,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022030.1 (Escherichia coli, WP3-S18-ESBL-07) loaded with 62 markers.\n",
      "2024-04-17 18:09:15,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021935.1 (Escherichia coli, WP2-W18-ESBL-10) loaded with 78 markers.\n",
      "2024-04-17 18:09:15,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021946.1 (Escherichia coli, WP2-S18-ESBL-08) loaded with 69 markers.\n",
      "2024-04-17 18:09:15,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT838200.1 (Escherichia coli, WI2_isolate) loaded with 57 markers.\n",
      "2024-04-17 18:09:15,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT838196.1 (Escherichia coli, WI1_isolate) loaded with 57 markers.\n",
      "2024-04-17 18:09:15,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061530.1 (Escherichia coli, WEM25) loaded with 87 markers.\n",
      "2024-04-17 18:09:16,090 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034595.1 (Escherichia coli, WCHEC035053S1G0) loaded with 87 markers.\n",
      "2024-04-17 18:09:16,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036177.1 (Escherichia coli, WCHEC025970) loaded with 52 markers.\n",
      "2024-04-17 18:09:16,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043279.1 (Escherichia coli, WCHEC025157) loaded with 52 markers.\n",
      "2024-04-17 18:09:16,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028578.2 (Escherichia coli, WCHEC005784) loaded with 55 markers.\n",
      "2024-04-17 18:09:16,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026580.3 (Escherichia coli, WCHEC005237) loaded with 54 markers.\n",
      "2024-04-17 18:09:16,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092973.1 (Escherichia coli, WCHEC005215) loaded with 53 markers.\n",
      "2024-04-17 18:09:16,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058342.1 (Escherichia coli, W60) loaded with 54 markers.\n",
      "2024-04-17 18:09:16,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032992.1 (Escherichia coli, W5-6) loaded with 58 markers.\n",
      "2024-04-17 18:09:16,928 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084899.1 (Escherichia coli, W3110-P) loaded with 89 markers.\n",
      "2024-04-17 18:09:17,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091038.1 (Escherichia coli, W25K) loaded with 52 markers.\n",
      "2024-04-17 18:09:17,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080235.1 (Escherichia coli, W13-16) loaded with 78 markers.\n",
      "2024-04-17 18:09:17,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011134.1 (Escherichia coli, VR50) loaded with 72 markers.\n",
      "2024-04-17 18:09:17,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011124.1 (Escherichia coli, USML2) loaded with 87 markers.\n",
      "2024-04-17 18:09:17,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039837.1 (Escherichia coli, USDA5905) loaded with 75 markers.\n",
      "2024-04-17 18:09:17,619 [DEBUG - chronostrain.database.parser.json] - Strain NC_017641.1 (Escherichia coli, UMNK88) loaded with 64 markers.\n",
      "2024-04-17 18:09:17,749 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041359.1 (Escherichia coli, U1) loaded with 76 markers.\n",
      "2024-04-17 18:09:17,852 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080620.1 (Escherichia coli, TOP10) loaded with 76 markers.\n",
      "2024-04-17 18:09:17,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018801.1 (Escherichia coli, tolC-) loaded with 86 markers.\n",
      "2024-04-17 18:09:18,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038295.1 (Escherichia coli, TB182A) loaded with 72 markers.\n",
      "2024-04-17 18:09:18,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024205.1 (Escherichia coli, TA8571) loaded with 50 markers.\n",
      "2024-04-17 18:09:18,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049353.1 (Escherichia coli, T28R) loaded with 71 markers.\n",
      "2024-04-17 18:09:18,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072986.1 (Escherichia coli, T159A) loaded with 88 markers.\n",
      "2024-04-17 18:09:18,510 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058948.1 (Escherichia coli, SY3626C5) loaded with 86 markers.\n",
      "2024-04-17 18:09:18,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059043.1 (Escherichia coli, SY3626C1) loaded with 86 markers.\n",
      "2024-04-17 18:09:18,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091836.1 (Escherichia coli, SWYHX138.33gt) loaded with 73 markers.\n",
      "2024-04-17 18:09:18,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091839.1 (Escherichia coli, SWYHX138.261ft) loaded with 73 markers.\n",
      "2024-04-17 18:09:18,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055048.1 (Escherichia coli, SWHEFF_58) loaded with 70 markers.\n",
      "2024-04-17 18:09:19,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076318.1 (Escherichia coli, Survcare045) loaded with 52 markers.\n",
      "2024-04-17 18:09:19,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083701.1 (Escherichia coli, SUISSEKPC3NDM5) loaded with 57 markers.\n",
      "2024-04-17 18:09:19,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041422.1 (Escherichia coli, STEC409) loaded with 78 markers.\n",
      "2024-04-17 18:09:19,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081009.1 (Escherichia coli, Stbl4) loaded with 62 markers.\n",
      "2024-04-17 18:09:19,490 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076043.1 (Escherichia coli, Stbl4) loaded with 73 markers.\n",
      "2024-04-17 18:09:19,595 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050173.1 (Escherichia coli, STB20-1) loaded with 66 markers.\n",
      "2024-04-17 18:09:19,729 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060709.1 (Escherichia coli, ST18) loaded with 88 markers.\n",
      "2024-04-17 18:09:19,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011321.1 (Escherichia coli, SQ88) loaded with 85 markers.\n",
      "2024-04-17 18:09:19,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011320.1 (Escherichia coli, SQ37) loaded with 85 markers.\n",
      "2024-04-17 18:09:20,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011324.1 (Escherichia coli, SQ2203) loaded with 85 markers.\n",
      "2024-04-17 18:09:20,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011322.1 (Escherichia coli, SQ110) loaded with 85 markers.\n",
      "2024-04-17 18:09:20,322 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017631.1 (Escherichia coli, SLK172) loaded with 76 markers.\n",
      "2024-04-17 18:09:20,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080255.1 (Escherichia coli, SJC33) loaded with 79 markers.\n",
      "2024-04-17 18:09:20,546 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073768.1 (Escherichia coli, SH19PTE6) loaded with 66 markers.\n",
      "2024-04-17 18:09:20,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051716.1 (Escherichia coli, SCU-118) loaded with 59 markers.\n",
      "2024-04-17 18:09:20,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051733.1 (Escherichia coli, SCU-109) loaded with 78 markers.\n",
      "2024-04-17 18:09:20,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053284.1 (Escherichia coli, SCU-104) loaded with 73 markers.\n",
      "2024-04-17 18:09:20,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025950.3 (Escherichia coli, SCEC020023) loaded with 54 markers.\n",
      "2024-04-17 18:09:21,062 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025627.1 (Escherichia coli, SCEC020007) loaded with 51 markers.\n",
      "2024-04-17 18:09:21,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082824.1 (Escherichia coli, SCAID_URN1-2021_(19_278)) loaded with 74 markers.\n",
      "2024-04-17 18:09:21,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071954.1 (Escherichia coli, SB0258h1) loaded with 75 markers.\n",
      "2024-04-17 18:09:21,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011061.1 (Escherichia coli, Sanji) loaded with 52 markers.\n",
      "2024-04-17 18:09:21,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010237.1 (Escherichia coli, S43) loaded with 71 markers.\n",
      "2024-04-17 18:09:21,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040667.1 (Escherichia coli, S17-1) loaded with 88 markers.\n",
      "2024-04-17 18:09:21,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010226.1 (Escherichia coli, S1) loaded with 76 markers.\n",
      "2024-04-17 18:09:21,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092702.1 (Escherichia coli, S-P-C-029.01) loaded with 86 markers.\n",
      "2024-04-17 18:09:21,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011113.2 (Escherichia coli, RR1) loaded with 86 markers.\n",
      "2024-04-17 18:09:22,108 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083410.1 (Escherichia coli, RP437) loaded with 86 markers.\n",
      "2024-04-17 18:09:22,235 [DEBUG - chronostrain.database.parser.json] - Strain NC_017656.1 (Escherichia coli, RM12579) loaded with 73 markers.\n",
      "2024-04-17 18:09:22,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042350.1 (Escherichia coli, RM10410) loaded with 76 markers.\n",
      "2024-04-17 18:09:22,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086616.1 (Escherichia coli, RIVM_C039629) loaded with 68 markers.\n",
      "2024-04-17 18:09:22,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086618.1 (Escherichia coli, RIVM_C039227) loaded with 68 markers.\n",
      "2024-04-17 18:09:22,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086620.1 (Escherichia coli, RIVM_C039205) loaded with 68 markers.\n",
      "2024-04-17 18:09:22,903 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086622.1 (Escherichia coli, RIVM_C037884) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086623.1 (Escherichia coli, RIVM_C037794) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086624.1 (Escherichia coli, RIVM_C037791) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086625.1 (Escherichia coli, RIVM_C037166) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086626.1 (Escherichia coli, RIVM_C037134) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086627.1 (Escherichia coli, RIVM_C036569) loaded with 68 markers.\n",
      "2024-04-17 18:09:23,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068829.1 (Escherichia coli, RIVM_C030453) loaded with 77 markers.\n",
      "2024-04-17 18:09:23,818 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068827.1 (Escherichia coli, RIVM_C030371) loaded with 70 markers.\n",
      "2024-04-17 18:09:23,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068826.1 (Escherichia coli, RIVM_C030300) loaded with 77 markers.\n",
      "2024-04-17 18:09:24,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068824.1 (Escherichia coli, RIVM_C030197) loaded with 77 markers.\n",
      "2024-04-17 18:09:24,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068821.1 (Escherichia coli, RIVM_C029952) loaded with 75 markers.\n",
      "2024-04-17 18:09:24,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068820.1 (Escherichia coli, RIVM_C029951) loaded with 70 markers.\n",
      "2024-04-17 18:09:24,483 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068819.1 (Escherichia coli, RIVM_C029042) loaded with 75 markers.\n",
      "2024-04-17 18:09:24,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068813.1 (Escherichia coli, RIVM_C029033) loaded with 68 markers.\n",
      "2024-04-17 18:09:24,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068812.1 (Escherichia coli, RIVM_C029020) loaded with 70 markers.\n",
      "2024-04-17 18:09:24,876 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068818.1 (Escherichia coli, RIVM_C028803) loaded with 75 markers.\n",
      "2024-04-17 18:09:25,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068814.1 (Escherichia coli, RIVM_C028613) loaded with 72 markers.\n",
      "2024-04-17 18:09:25,155 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068811.1 (Escherichia coli, RIVM_C028568) loaded with 75 markers.\n",
      "2024-04-17 18:09:25,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068810.1 (Escherichia coli, RIVM_C028536) loaded with 77 markers.\n",
      "2024-04-17 18:09:25,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087377.1 (Escherichia coli, RIVM_C019217) loaded with 68 markers.\n",
      "2024-04-17 18:09:25,579 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068830.1 (Escherichia coli, RIVM_C018707) loaded with 75 markers.\n",
      "2024-04-17 18:09:25,716 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068808.1 (Escherichia coli, RIVM_C018699) loaded with 75 markers.\n",
      "2024-04-17 18:09:25,839 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068807.1 (Escherichia coli, RIVM_C018583) loaded with 70 markers.\n",
      "2024-04-17 18:09:25,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068805.1 (Escherichia coli, RIVM_C018567) loaded with 75 markers.\n",
      "2024-04-17 18:09:26,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068804.1 (Escherichia coli, RIVM_C018563) loaded with 76 markers.\n",
      "2024-04-17 18:09:26,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068801.1 (Escherichia coli, RIVM_C018220) loaded with 74 markers.\n",
      "2024-04-17 18:09:26,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068800.1 (Escherichia coli, RIVM_C017997) loaded with 77 markers.\n",
      "2024-04-17 18:09:26,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068796.1 (Escherichia coli, RIVM_C012087) loaded with 76 markers.\n",
      "2024-04-17 18:09:26,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068795.1 (Escherichia coli, RIVM_C011532) loaded with 56 markers.\n",
      "2024-04-17 18:09:26,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062811.1 (Escherichia coli, Res13-Sevr-PER07-33) loaded with 76 markers.\n",
      "2024-04-17 18:09:26,828 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062815.1 (Escherichia coli, Res13-Sevr-PER06-05-b-A) loaded with 77 markers.\n",
      "2024-04-17 18:09:26,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062826.1 (Escherichia coli, Res13-Lact-PER12-33-A) loaded with 77 markers.\n",
      "2024-04-17 18:09:27,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062832.1 (Escherichia coli, Res13-Lact-PER04-33) loaded with 78 markers.\n",
      "2024-04-17 18:09:27,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062838.1 (Escherichia coli, Res13-Lact-PER02-33) loaded with 78 markers.\n",
      "2024-04-17 18:09:27,259 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062883.1 (Escherichia coli, Res13-Lact-PEA06-10) loaded with 78 markers.\n",
      "2024-04-17 18:09:27,399 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062895.1 (Escherichia coli, Res13-Lact-ER02-39) loaded with 79 markers.\n",
      "2024-04-17 18:09:27,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882052.1 (Escherichia coli, Quinolone_resistant) loaded with 59 markers.\n",
      "2024-04-17 18:09:27,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080139.1 (Escherichia coli, PK8241) loaded with 75 markers.\n",
      "2024-04-17 18:09:27,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042588.1 (Escherichia coli, PK6) loaded with 63 markers.\n",
      "2024-04-17 18:09:27,898 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087110.1 (Escherichia coli, PJ-T13) loaded with 87 markers.\n",
      "2024-04-17 18:09:28,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038857.1 (Escherichia coli, PigCaeca_2) loaded with 75 markers.\n",
      "2024-04-17 18:09:28,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043189.1 (Escherichia coli, PG20180175) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043191.1 (Escherichia coli, PG20180173) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043183.1 (Escherichia coli, PG20180172) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043185.1 (Escherichia coli, PG20180171) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043187.1 (Escherichia coli, PG20180170) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043193.1 (Escherichia coli, PG20180063) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,852 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043195.1 (Escherichia coli, PG20180062) loaded with 88 markers.\n",
      "2024-04-17 18:09:28,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043197.1 (Escherichia coli, PG20180061) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,081 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043199.1 (Escherichia coli, PG20180060) loaded with 89 markers.\n",
      "2024-04-17 18:09:29,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043201.1 (Escherichia coli, PG20180059) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043203.1 (Escherichia coli, PG20180058) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043205.1 (Escherichia coli, PG20180054) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043207.1 (Escherichia coli, PG20180053) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043209.1 (Escherichia coli, PG20180052) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043211.1 (Escherichia coli, PG20180050) loaded with 88 markers.\n",
      "2024-04-17 18:09:29,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038791.1 (Escherichia coli, PF9285) loaded with 74 markers.\n",
      "2024-04-17 18:09:29,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041628.1 (Escherichia coli, PE15) loaded with 56 markers.\n",
      "2024-04-17 18:09:30,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080253.1 (Escherichia coli, PC6) loaded with 69 markers.\n",
      "2024-04-17 18:09:30,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048605.1 (Escherichia coli, PapRG-06-5) loaded with 72 markers.\n",
      "2024-04-17 18:09:30,357 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085060.1 (Escherichia coli, P30_L1_05.20) loaded with 51 markers.\n",
      "2024-04-17 18:09:30,490 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085074.1 (Escherichia coli, P21_SE1_04.20) loaded with 61 markers.\n",
      "2024-04-17 18:09:30,625 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080233.1 (Escherichia coli, P15-25) loaded with 78 markers.\n",
      "2024-04-17 18:09:30,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080223.1 (Escherichia coli, P13-6) loaded with 76 markers.\n",
      "2024-04-17 18:09:30,833 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049081.1 (Escherichia coli, p10A) loaded with 66 markers.\n",
      "2024-04-17 18:09:30,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087578.1 (Escherichia coli, OSUCMP42NDM) loaded with 74 markers.\n",
      "2024-04-17 18:09:31,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084529.1 (Escherichia coli, Origami2-DE3) loaded with 77 markers.\n",
      "2024-04-17 18:09:31,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075469.1 (Escherichia coli, NT1W22) loaded with 66 markers.\n",
      "2024-04-17 18:09:31,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075465.1 (Escherichia coli, NT1N34) loaded with 90 markers.\n",
      "2024-04-17 18:09:31,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075480.1 (Escherichia coli, NT1N31) loaded with 91 markers.\n",
      "2024-04-17 18:09:31,489 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075474.1 (Escherichia coli, NT1N28) loaded with 75 markers.\n",
      "2024-04-17 18:09:31,593 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075487.1 (Escherichia coli, NT1F34) loaded with 75 markers.\n",
      "2024-04-17 18:09:31,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075462.1 (Escherichia coli, NT1F10) loaded with 66 markers.\n",
      "2024-04-17 18:09:31,828 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042878.1 (Escherichia coli, NMBU_W05E18) loaded with 76 markers.\n",
      "2024-04-17 18:09:31,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042892.1 (Escherichia coli, NMBU-W12E19) loaded with 96 markers.\n",
      "2024-04-17 18:09:32,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053605.1 (Escherichia coli, NEB_Turbo) loaded with 86 markers.\n",
      "2024-04-17 18:09:32,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053607.1 (Escherichia coli, NEB5-alpha_F'Iq) loaded with 89 markers.\n",
      "2024-04-17 18:09:32,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053604.1 (Escherichia coli, NEB10-beta) loaded with 78 markers.\n",
      "2024-04-17 18:09:32,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082127.1 (Escherichia coli, NDM_12_14482) loaded with 51 markers.\n",
      "2024-04-17 18:09:32,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082129.1 (Escherichia coli, NDM_11.16372) loaded with 51 markers.\n",
      "2024-04-17 18:09:32,642 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083869.1 (Escherichia coli, NDM6) loaded with 53 markers.\n",
      "2024-04-17 18:09:32,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083872.1 (Escherichia coli, NDM5) loaded with 53 markers.\n",
      "2024-04-17 18:09:32,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083877.1 (Escherichia coli, NDM4) loaded with 53 markers.\n",
      "2024-04-17 18:09:32,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083880.1 (Escherichia coli, NDM3) loaded with 53 markers.\n",
      "2024-04-17 18:09:33,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083886.1 (Escherichia coli, NDM2) loaded with 59 markers.\n",
      "2024-04-17 18:09:33,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483296.1 (Escherichia coli, NCTC9966) loaded with 71 markers.\n",
      "2024-04-17 18:09:33,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134228.1 (Escherichia coli, NCTC9699) loaded with 73 markers.\n",
      "2024-04-17 18:09:33,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134240.1 (Escherichia coli, NCTC9107) loaded with 67 markers.\n",
      "2024-04-17 18:09:33,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134152.1 (Escherichia coli, NCTC9104) loaded with 73 markers.\n",
      "2024-04-17 18:09:33,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134227.1 (Escherichia coli, NCTC9102) loaded with 69 markers.\n",
      "2024-04-17 18:09:34,015 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134239.1 (Escherichia coli, NCTC9100) loaded with 76 markers.\n",
      "2024-04-17 18:09:34,173 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134231.1 (Escherichia coli, NCTC9087) loaded with 72 markers.\n",
      "2024-04-17 18:09:34,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134075.1 (Escherichia coli, NCTC9084) loaded with 72 markers.\n",
      "2024-04-17 18:09:34,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134295.1 (Escherichia coli, NCTC9080) loaded with 74 markers.\n",
      "2024-04-17 18:09:34,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134296.1 (Escherichia coli, NCTC9041) loaded with 74 markers.\n",
      "2024-04-17 18:09:34,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134247.1 (Escherichia coli, NCTC9040) loaded with 73 markers.\n",
      "2024-04-17 18:09:34,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134082.1 (Escherichia coli, NCTC9038) loaded with 72 markers.\n",
      "2024-04-17 18:09:35,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT601384.1 (Escherichia coli, NCTC86EC) loaded with 73 markers.\n",
      "2024-04-17 18:09:35,289 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR217818.1 (Escherichia coli, NCTC86) loaded with 73 markers.\n",
      "2024-04-17 18:09:35,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019778.1 (Escherichia coli, NCTC86) loaded with 73 markers.\n",
      "2024-04-17 18:09:35,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134220.1 (Escherichia coli, NCTC11121) loaded with 64 markers.\n",
      "2024-04-17 18:09:35,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483297.1 (Escherichia coli, NCTC11023) loaded with 66 markers.\n",
      "2024-04-17 18:09:35,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134216.1 (Escherichia coli, NCTC10973) loaded with 75 markers.\n",
      "2024-04-17 18:09:35,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011495.1 (Escherichia coli, NCM3722) loaded with 73 markers.\n",
      "2024-04-17 18:09:36,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041749.1 (Escherichia coli, NCCP_15955) loaded with 54 markers.\n",
      "2024-04-17 18:09:36,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048439.1 (Escherichia coli, NBRC_3301) loaded with 80 markers.\n",
      "2024-04-17 18:09:36,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047277.1 (Escherichia coli, N18-04097) loaded with 74 markers.\n",
      "2024-04-17 18:09:36,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034953.2 (Escherichia coli, MT102) loaded with 86 markers.\n",
      "2024-04-17 18:09:36,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076645.1 (Escherichia coli, MSI001) loaded with 113 markers.\n",
      "2024-04-17 18:09:36,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890508.1 (Escherichia coli, MSB1_9I-sc-2280417) loaded with 72 markers.\n",
      "2024-04-17 18:09:37,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890349.1 (Escherichia coli, MSB1_9D-sc-2280338) loaded with 66 markers.\n",
      "2024-04-17 18:09:37,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890603.1 (Escherichia coli, MSB1_8G-sc-2280388) loaded with 62 markers.\n",
      "2024-04-17 18:09:37,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890466.1 (Escherichia coli, MSB1_7A-sc-2280394) loaded with 77 markers.\n",
      "2024-04-17 18:09:37,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890714.1 (Escherichia coli, MSB1_6C-sc-2280315) loaded with 70 markers.\n",
      "2024-04-17 18:09:37,696 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890299.1 (Escherichia coli, MSB1_3B-sc-2280406) loaded with 77 markers.\n",
      "2024-04-17 18:09:37,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890288.1 (Escherichia coli, MSB1_1A-sc-2280383) loaded with 62 markers.\n",
      "2024-04-17 18:09:38,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130562.1 (Escherichia coli, MS14384) loaded with 82 markers.\n",
      "2024-04-17 18:09:38,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018103.1 (Escherichia coli, MRSN352231) loaded with 51 markers.\n",
      "2024-04-17 18:09:38,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018206.1 (Escherichia coli, MRSN346647) loaded with 72 markers.\n",
      "2024-04-17 18:09:38,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018115.1 (Escherichia coli, MRSN346638) loaded with 51 markers.\n",
      "2024-04-17 18:09:38,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018109.1 (Escherichia coli, MRSN346595) loaded with 51 markers.\n",
      "2024-04-17 18:09:38,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018121.1 (Escherichia coli, MRSN346355) loaded with 51 markers.\n",
      "2024-04-17 18:09:38,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025747.1 (Escherichia coli, ML35) loaded with 81 markers.\n",
      "2024-04-17 18:09:38,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890606.1 (Escherichia coli, MINF_8D-sc-2280460) loaded with 48 markers.\n",
      "2024-04-17 18:09:39,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069657.1 (Escherichia coli, MIN12) loaded with 61 markers.\n",
      "2024-04-17 18:09:39,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069666.1 (Escherichia coli, MIN11) loaded with 58 markers.\n",
      "2024-04-17 18:09:39,426 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019629.1 (Escherichia coli, MGY) loaded with 86 markers.\n",
      "2024-04-17 18:09:39,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092037.1 (Escherichia coli, MG1655-377GD) loaded with 63 markers.\n",
      "2024-04-17 18:09:39,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060708.1 (Escherichia coli, MFDpir) loaded with 85 markers.\n",
      "2024-04-17 18:09:39,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028703.1 (Escherichia coli, ME8067) loaded with 89 markers.\n",
      "2024-04-17 18:09:40,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023219.2 (Escherichia coli, M505) loaded with 59 markers.\n",
      "2024-04-17 18:09:40,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019189.1 (Escherichia coli, M217) loaded with 52 markers.\n",
      "2024-04-17 18:09:40,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046416.1 (Escherichia coli, LZYWN_4) loaded with 52 markers.\n",
      "2024-04-17 18:09:40,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026027.1 (Escherichia coli, LIM) loaded with 85 markers.\n",
      "2024-04-17 18:09:40,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047594.1 (Escherichia coli, LD27-1) loaded with 76 markers.\n",
      "2024-04-17 18:09:40,577 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091756.1 (Escherichia coli, LC-1302-2020) loaded with 73 markers.\n",
      "2024-04-17 18:09:40,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045277.1 (Escherichia coli, LAU-OXA) loaded with 48 markers.\n",
      "2024-04-17 18:09:40,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062203.1 (Escherichia coli, K_EC180) loaded with 78 markers.\n",
      "2024-04-17 18:09:40,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091651.1 (Escherichia coli, KTa008) loaded with 75 markers.\n",
      "2024-04-17 18:09:41,033 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040664.1 (Escherichia coli, KR2009) loaded with 75 markers.\n",
      "2024-04-17 18:09:41,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091703.1 (Escherichia coli, KOr021) loaded with 76 markers.\n",
      "2024-04-17 18:09:41,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079991.1 (Escherichia coli, KMS2005) loaded with 68 markers.\n",
      "2024-04-17 18:09:41,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079992.1 (Escherichia coli, KMS2001) loaded with 68 markers.\n",
      "2024-04-17 18:09:41,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091652.1 (Escherichia coli, KMi025) loaded with 59 markers.\n",
      "2024-04-17 18:09:41,598 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008801.1 (Escherichia coli, KLY) loaded with 86 markers.\n",
      "2024-04-17 18:09:41,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030919.1 (Escherichia coli, KL53) loaded with 71 markers.\n",
      "2024-04-17 18:09:41,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091664.1 (Escherichia coli, KKa019) loaded with 76 markers.\n",
      "2024-04-17 18:09:41,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091704.1 (Escherichia coli, KGu016) loaded with 76 markers.\n",
      "2024-04-17 18:09:42,024 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091662.1 (Escherichia coli, KFu023) loaded with 60 markers.\n",
      "2024-04-17 18:09:42,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091677.1 (Escherichia coli, KFu022) loaded with 75 markers.\n",
      "2024-04-17 18:09:42,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091700.1 (Escherichia coli, KFu019) loaded with 74 markers.\n",
      "2024-04-17 18:09:42,334 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072054.1 (Escherichia coli, K1508) loaded with 86 markers.\n",
      "2024-04-17 18:09:42,445 [DEBUG - chronostrain.database.parser.json] - Strain NC_007779.1 (Escherichia coli, K-12_substr._W3110) loaded with 92 markers.\n",
      "2024-04-17 18:09:42,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LM995446.1 (Escherichia coli, K-12_substr._RV308) loaded with 85 markers.\n",
      "2024-04-17 18:09:42,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012870.1 (Escherichia coli, K-12_substr._MG1655_TMP32XR2) loaded with 86 markers.\n",
      "2024-04-17 18:09:42,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012869.1 (Escherichia coli, K-12_substr._MG1655_TMP32XR1) loaded with 86 markers.\n",
      "2024-04-17 18:09:42,866 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR881938.1 (Escherichia coli, K-12_substr._MG1655) loaded with 85 markers.\n",
      "2024-04-17 18:09:42,938 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide W (possible IUPAC code) in NZ_CP032679.1,IDX=0, while parsing marker insH7:BLAST<849>. Skipping strain NZ_CP032679.1.\n",
      "2024-04-17 18:09:43,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032667.1 (Escherichia coli, K-12_substr._MG1655) loaded with 85 markers.\n",
      "2024-04-17 18:09:43,156 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025268.1 (Escherichia coli, K-12_substr._MG1655) loaded with 85 markers.\n",
      "2024-04-17 18:09:43,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014225.1 (Escherichia coli, K-12_substr._MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:43,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012868.1 (Escherichia coli, K-12_substr._MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:43,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009685.1 (Escherichia coli, K-12_substr._MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:43,596 [DEBUG - chronostrain.database.parser.json] - Strain NC_000913.3 (Escherichia coli, K-12_substr._MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:43,710 [DEBUG - chronostrain.database.parser.json] - Strain NZ_HG738867.1 (Escherichia coli, K-12_substr._MC4100) loaded with 86 markers.\n",
      "2024-04-17 18:09:43,826 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LM993812.1 (Escherichia coli, K-12_substr._HMS174) loaded with 89 markers.\n",
      "2024-04-17 18:09:43,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011343.2 (Escherichia coli, K-12_substr._GM4792) loaded with 86 markers.\n",
      "2024-04-17 18:09:44,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011342.2 (Escherichia coli, K-12_substr._GM4792) loaded with 86 markers.\n",
      "2024-04-17 18:09:44,165 [DEBUG - chronostrain.database.parser.json] - Strain NC_010473.1 (Escherichia coli, K-12_substr._DH10B) loaded with 80 markers.\n",
      "2024-04-17 18:09:44,277 [DEBUG - chronostrain.database.parser.json] - Strain NC_012759.1 (Escherichia coli, K-12_substr._BW2952) loaded with 87 markers.\n",
      "2024-04-17 18:09:44,391 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009273.1 (Escherichia coli, K-12_substr._BW25113) loaded with 85 markers.\n",
      "2024-04-17 18:09:44,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN832404.1 (Escherichia coli, K-12_substr._AG100) loaded with 84 markers.\n",
      "2024-04-17 18:09:44,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076404.1 (Escherichia coli, K-12_substr._AB1157) loaded with 88 markers.\n",
      "2024-04-17 18:09:44,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017100.1 (Escherichia coli, K-12_NEB_5-alpha) loaded with 89 markers.\n",
      "2024-04-17 18:09:44,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010445.1 (Escherichia coli, K-12_MG1655) loaded with 85 markers.\n",
      "2024-04-17 18:09:44,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010444.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010443.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010442.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,272 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010441.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010440.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010439.1 (Escherichia coli, K-12_MG1655) loaded with 87 markers.\n",
      "2024-04-17 18:09:45,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010438.1 (Escherichia coli, K-12_MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:45,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070962.1 (Escherichia coli, K-12_J53) loaded with 91 markers.\n",
      "2024-04-17 18:09:45,987 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009789.1 (Escherichia coli, K-12_ER3413) loaded with 78 markers.\n",
      "2024-04-17 18:09:46,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009644.1 (Escherichia coli, K-12_ER2796) loaded with 78 markers.\n",
      "2024-04-17 18:09:46,335 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014270.1 (Escherichia coli, K-12_DHB4) loaded with 86 markers.\n",
      "2024-04-17 18:09:46,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014272.1 (Escherichia coli, K-12_C3026) loaded with 86 markers.\n",
      "2024-04-17 18:09:46,698 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047127.1 (Escherichia coli, K-12) loaded with 89 markers.\n",
      "2024-04-17 18:09:46,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014348.1 (Escherichia coli, JW5437-1_substr._MG1655) loaded with 86 markers.\n",
      "2024-04-17 18:09:47,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025180.1 (Escherichia coli, JP538) loaded with 77 markers.\n",
      "2024-04-17 18:09:47,174 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091925.1 (Escherichia coli, JNQH497) loaded with 56 markers.\n",
      "2024-04-17 18:09:47,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042843.1 (Escherichia coli, JME67) loaded with 85 markers.\n",
      "2024-04-17 18:09:47,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042844.1 (Escherichia coli, JME66) loaded with 85 markers.\n",
      "2024-04-17 18:09:47,696 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042845.1 (Escherichia coli, JME65) loaded with 85 markers.\n",
      "2024-04-17 18:09:47,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042846.1 (Escherichia coli, JME64) loaded with 85 markers.\n",
      "2024-04-17 18:09:48,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077969.1 (Escherichia coli, JM83) loaded with 85 markers.\n",
      "2024-04-17 18:09:48,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068706.1 (Escherichia coli, J53-p1-pV-hybrid-1) loaded with 86 markers.\n",
      "2024-04-17 18:09:48,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028702.1 (Escherichia coli, J53) loaded with 86 markers.\n",
      "2024-04-17 18:09:48,577 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054665.1 (Escherichia coli, IGC_EcoliInv_1.1) loaded with 84 markers.\n",
      "2024-04-17 18:09:48,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013952.1 (Escherichia coli, HST04) loaded with 85 markers.\n",
      "2024-04-17 18:09:48,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010172.1 (Escherichia coli, H8) loaded with 60 markers.\n",
      "2024-04-17 18:09:48,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010171.1 (Escherichia coli, H7) loaded with 68 markers.\n",
      "2024-04-17 18:09:49,089 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028732.1 (Escherichia coli, H3) loaded with 72 markers.\n",
      "2024-04-17 18:09:49,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010160.1 (Escherichia coli, H1) loaded with 70 markers.\n",
      "2024-04-17 18:09:49,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042336.1 (Escherichia coli, GZ04-0086) loaded with 58 markers.\n",
      "2024-04-17 18:09:49,518 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084898.1 (Escherichia coli, gw25) loaded with 89 markers.\n",
      "2024-04-17 18:09:49,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019675.1 (Escherichia coli, GSH8M-2) loaded with 73 markers.\n",
      "2024-04-17 18:09:49,905 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084897.1 (Escherichia coli, gb37) loaded with 89 markers.\n",
      "2024-04-17 18:09:50,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079736.1 (Escherichia coli, GB2005-delet-a-f) loaded with 77 markers.\n",
      "2024-04-17 18:09:50,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079735.1 (Escherichia coli, GB2005) loaded with 77 markers.\n",
      "2024-04-17 18:09:50,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084893.1 (Escherichia coli, g94) loaded with 89 markers.\n",
      "2024-04-17 18:09:50,576 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084894.1 (Escherichia coli, g83) loaded with 89 markers.\n",
      "2024-04-17 18:09:50,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060057.1 (Escherichia coli, G8) loaded with 71 markers.\n",
      "2024-04-17 18:09:50,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084895.1 (Escherichia coli, g74) loaded with 89 markers.\n",
      "2024-04-17 18:09:51,072 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060061.1 (Escherichia coli, G6_7) loaded with 71 markers.\n",
      "2024-04-17 18:09:51,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084896.1 (Escherichia coli, g62) loaded with 89 markers.\n",
      "2024-04-17 18:09:51,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060065.1 (Escherichia coli, G5) loaded with 76 markers.\n",
      "2024-04-17 18:09:51,390 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060073.1 (Escherichia coli, G4_9) loaded with 71 markers.\n",
      "2024-04-17 18:09:51,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060083.1 (Escherichia coli, G1_2) loaded with 72 markers.\n",
      "2024-04-17 18:09:51,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040090.1 (Escherichia coli, FT130) loaded with 75 markers.\n",
      "2024-04-17 18:09:51,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029057.1 (Escherichia coli, FORC_081) loaded with 75 markers.\n",
      "2024-04-17 18:09:51,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022664.1 (Escherichia coli, FORC_064) loaded with 73 markers.\n",
      "2024-04-17 18:09:52,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065607.1 (Escherichia coli, FDAARGOS_946) loaded with 73 markers.\n",
      "2024-04-17 18:09:52,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070026.1 (Escherichia coli, FDAARGOS_1337) loaded with 71 markers.\n",
      "2024-04-17 18:09:52,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070157.1 (Escherichia coli, FDAARGOS_1276) loaded with 76 markers.\n",
      "2024-04-17 18:09:52,384 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069459.1 (Escherichia coli, FDAARGOS_1249) loaded with 69 markers.\n",
      "2024-04-17 18:09:52,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017220.1 (Escherichia coli, FAM21845) loaded with 82 markers.\n",
      "2024-04-17 18:09:52,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079993.1 (Escherichia coli, EV36) loaded with 68 markers.\n",
      "2024-04-17 18:09:52,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045457.1 (Escherichia coli, ET12567) loaded with 89 markers.\n",
      "2024-04-17 18:09:52,905 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086340.1 (Escherichia coli, ESY003) loaded with 53 markers.\n",
      "2024-04-17 18:09:53,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086556.1 (Escherichia coli, ESY002) loaded with 54 markers.\n",
      "2024-04-17 18:09:53,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086342.1 (Escherichia coli, ESY001) loaded with 54 markers.\n",
      "2024-04-17 18:09:53,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992171.1 (Escherichia coli, Escherichia_coli_str._TO73) loaded with 59 markers.\n",
      "2024-04-17 18:09:53,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992166.1 (Escherichia coli, Escherichia_coli_str._TO6) loaded with 72 markers.\n",
      "2024-04-17 18:09:53,469 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992185.1 (Escherichia coli, Escherichia_coli_str._3426) loaded with 57 markers.\n",
      "2024-04-17 18:09:53,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067250.1 (Escherichia coli, ESBL_DR28) loaded with 54 markers.\n",
      "2024-04-17 18:09:53,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093221.1 (Escherichia coli, ER2683) loaded with 89 markers.\n",
      "2024-04-17 18:09:53,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016018.1 (Escherichia coli, ER1821R) loaded with 90 markers.\n",
      "2024-04-17 18:09:54,021 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030240.1 (Escherichia coli, ER1709) loaded with 86 markers.\n",
      "2024-04-17 18:09:54,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063467.1 (Escherichia coli, EM20-18-50) loaded with 70 markers.\n",
      "2024-04-17 18:09:54,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063473.1 (Escherichia coli, EM18-18-42) loaded with 65 markers.\n",
      "2024-04-17 18:09:54,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063483.1 (Escherichia coli, EM03-18-08) loaded with 59 markers.\n",
      "2024-04-17 18:09:54,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083492.1 (Escherichia coli, elppa8) loaded with 53 markers.\n",
      "2024-04-17 18:09:54,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040663.1 (Escherichia coli, EK2009) loaded with 75 markers.\n",
      "2024-04-17 18:09:54,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086338.1 (Escherichia coli, EJN001) loaded with 53 markers.\n",
      "2024-04-17 18:09:54,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063499.1 (Escherichia coli, EH10-18-47) loaded with 63 markers.\n",
      "2024-04-17 18:09:54,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063503.1 (Escherichia coli, EH09-18-41) loaded with 72 markers.\n",
      "2024-04-17 18:09:55,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063511.1 (Escherichia coli, EH08-18-36) loaded with 77 markers.\n",
      "2024-04-17 18:09:55,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024112.1 (Escherichia coli, ECSC054) loaded with 87 markers.\n",
      "2024-04-17 18:09:55,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054224.1 (Escherichia coli, EcPF16) loaded with 75 markers.\n",
      "2024-04-17 18:09:55,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026199.1 (Escherichia coli, ECONIH6) loaded with 53 markers.\n",
      "2024-04-17 18:09:55,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019005.1 (Escherichia coli, Ecol_AZ155) loaded with 58 markers.\n",
      "2024-04-17 18:09:55,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018976.1 (Escherichia coli, Ecol_545) loaded with 69 markers.\n",
      "2024-04-17 18:09:55,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041392.1 (Escherichia coli, ECOL-18-VL-LA-PA-Ryan-0026) loaded with 54 markers.\n",
      "2024-04-17 18:09:55,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083585.1 (Escherichia coli, ECNB21-M121) loaded with 74 markers.\n",
      "2024-04-17 18:09:55,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044410.1 (Escherichia coli, ecMN1F) loaded with 86 markers.\n",
      "2024-04-17 18:09:56,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041110.1 (Escherichia coli, ECCTRSRTH03) loaded with 64 markers.\n",
      "2024-04-17 18:09:56,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034787.1 (Escherichia coli, ECCNB20-2) loaded with 74 markers.\n",
      "2024-04-17 18:09:56,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033250.1 (Escherichia coli, ECCHD184) loaded with 80 markers.\n",
      "2024-04-17 18:09:56,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060963.1 (Escherichia coli, EC7) loaded with 58 markers.\n",
      "2024-04-17 18:09:56,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016182.2 (Escherichia coli, EC590) loaded with 58 markers.\n",
      "2024-04-17 18:09:56,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056073.1 (Escherichia coli, EC50) loaded with 68 markers.\n",
      "2024-04-17 18:09:56,621 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060969.1 (Escherichia coli, EC5) loaded with 56 markers.\n",
      "2024-04-17 18:09:56,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043414.1 (Escherichia coli, EC42405) loaded with 73 markers.\n",
      "2024-04-17 18:09:56,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059119.1 (Escherichia coli, Ec42) loaded with 61 markers.\n",
      "2024-04-17 18:09:57,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060974.1 (Escherichia coli, EC4) loaded with 66 markers.\n",
      "2024-04-17 18:09:57,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085619.1 (Escherichia coli, EC32) loaded with 71 markers.\n",
      "2024-04-17 18:09:57,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060987.1 (Escherichia coli, EC31) loaded with 53 markers.\n",
      "2024-04-17 18:09:57,349 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060979.1 (Escherichia coli, EC3) loaded with 58 markers.\n",
      "2024-04-17 18:09:57,433 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060867.1 (Escherichia coli, EC29) loaded with 47 markers.\n",
      "2024-04-17 18:09:57,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060880.1 (Escherichia coli, EC26) loaded with 60 markers.\n",
      "2024-04-17 18:09:57,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060892.1 (Escherichia coli, EC22) loaded with 55 markers.\n",
      "2024-04-17 18:09:57,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084900.1 (Escherichia coli, EC20) loaded with 60 markers.\n",
      "2024-04-17 18:09:57,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058571.1 (Escherichia coli, EC2) loaded with 76 markers.\n",
      "2024-04-17 18:09:57,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041955.1 (Escherichia coli, EC2) loaded with 52 markers.\n",
      "2024-04-17 18:09:58,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074120.1 (Escherichia coli, EC16) loaded with 53 markers.\n",
      "2024-04-17 18:09:58,269 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060928.1 (Escherichia coli, EC13) loaded with 61 markers.\n",
      "2024-04-17 18:09:58,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060938.1 (Escherichia coli, EC11) loaded with 53 markers.\n",
      "2024-04-17 18:09:58,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093226.1 (Escherichia coli, EC-16-35) loaded with 52 markers.\n",
      "2024-04-17 18:09:58,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038453.1 (Escherichia coli, EC-129) loaded with 49 markers.\n",
      "2024-04-17 18:09:58,698 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043217.1 (Escherichia coli, EC-107) loaded with 74 markers.\n",
      "2024-04-17 18:09:58,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043227.1 (Escherichia coli, Ec-050) loaded with 53 markers.\n",
      "2024-04-17 18:09:58,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086336.1 (Escherichia coli, EBJ001) loaded with 51 markers.\n",
      "2024-04-17 18:09:59,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069712.1 (Escherichia coli, EA13) loaded with 59 markers.\n",
      "2024-04-17 18:09:59,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035865.1 (Escherichia coli, E275_a) loaded with 74 markers.\n",
      "2024-04-17 18:09:59,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025573.1 (Escherichia coli, E-1246) loaded with 72 markers.\n",
      "2024-04-17 18:09:59,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026612.1 (Escherichia coli, DTU-1) loaded with 84 markers.\n",
      "2024-04-17 18:09:59,579 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091065.1 (Escherichia coli, DOC14) loaded with 85 markers.\n",
      "2024-04-17 18:09:59,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045741.1 (Escherichia coli, DH5alpha) loaded with 89 markers.\n",
      "2024-04-17 18:09:59,803 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025520.1 (Escherichia coli, DH5alpha) loaded with 90 markers.\n",
      "2024-04-17 18:09:59,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076470.1 (Escherichia coli, DH5a) loaded with 86 markers.\n",
      "2024-04-17 18:10:00,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012127.1 (Escherichia coli, DH1Ec169) loaded with 93 markers.\n",
      "2024-04-17 18:10:00,361 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012126.1 (Escherichia coli, DH1Ec104) loaded with 93 markers.\n",
      "2024-04-17 18:10:00,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012125.1 (Escherichia coli, DH1Ec095) loaded with 93 markers.\n",
      "2024-04-17 18:10:00,717 [DEBUG - chronostrain.database.parser.json] - Strain NC_017638.1 (Escherichia coli, DH1) loaded with 93 markers.\n",
      "2024-04-17 18:10:00,898 [DEBUG - chronostrain.database.parser.json] - Strain NC_017625.1 (Escherichia coli, DH1) loaded with 90 markers.\n",
      "2024-04-17 18:10:01,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038383.1 (Escherichia coli, DEC5E) loaded with 72 markers.\n",
      "2024-04-17 18:10:01,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038386.1 (Escherichia coli, DEC5D) loaded with 73 markers.\n",
      "2024-04-17 18:10:01,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038389.1 (Escherichia coli, DEC5B) loaded with 73 markers.\n",
      "2024-04-17 18:10:01,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038394.1 (Escherichia coli, DEC5A) loaded with 73 markers.\n",
      "2024-04-17 18:10:01,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045206.1 (Escherichia coli, DE17) loaded with 72 markers.\n",
      "2024-04-17 18:10:01,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053603.1 (Escherichia coli, Dam_Dcm) loaded with 87 markers.\n",
      "2024-04-17 18:10:02,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029574.1 (Escherichia coli, DA33133) loaded with 60 markers.\n",
      "2024-04-17 18:10:02,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010152.1 (Escherichia coli, D9) loaded with 74 markers.\n",
      "2024-04-17 18:10:02,412 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035312.1 (Escherichia coli, D72) loaded with 72 markers.\n",
      "2024-04-17 18:10:02,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010143.1 (Escherichia coli, D4) loaded with 74 markers.\n",
      "2024-04-17 18:10:02,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068279.1 (Escherichia coli, CY708) loaded with 72 markers.\n",
      "2024-04-17 18:10:02,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068281.1 (Escherichia coli, CY706) loaded with 74 markers.\n",
      "2024-04-17 18:10:03,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068283.1 (Escherichia coli, CY598) loaded with 74 markers.\n",
      "2024-04-17 18:10:03,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043213.1 (Escherichia coli, CV601gfp) loaded with 88 markers.\n",
      "2024-04-17 18:10:03,392 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067994.1 (Escherichia coli, CV601) loaded with 88 markers.\n",
      "2024-04-17 18:10:03,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024815.1 (Escherichia coli, CREC-629) loaded with 58 markers.\n",
      "2024-04-17 18:10:03,647 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024830.1 (Escherichia coli, CREC-532) loaded with 56 markers.\n",
      "2024-04-17 18:10:03,766 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019071.1 (Escherichia coli, CRE1493) loaded with 53 markers.\n",
      "2024-04-17 18:10:03,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053736.1 (Escherichia coli, CP8-3_Sichuan) loaded with 75 markers.\n",
      "2024-04-17 18:10:03,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019245.1 (Escherichia coli, Combat13F7) loaded with 71 markers.\n",
      "2024-04-17 18:10:04,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU701449.1 (Escherichia coli, CNR85I8) loaded with 77 markers.\n",
      "2024-04-17 18:10:04,289 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU701452.1 (Escherichia coli, CNR65D6) loaded with 76 markers.\n",
      "2024-04-17 18:10:04,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033158.1 (Escherichia coli, CM_IVRI_KOL-1) loaded with 53 markers.\n",
      "2024-04-17 18:10:04,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029238.1 (Escherichia coli, CJ236) loaded with 86 markers.\n",
      "2024-04-17 18:10:04,764 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026028.1 (Escherichia coli, CIT) loaded with 85 markers.\n",
      "2024-04-17 18:10:04,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028166.1 (Escherichia coli, CFSAN064036) loaded with 74 markers.\n",
      "2024-04-17 18:10:05,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042893.1 (Escherichia coli, CFSAN061772) loaded with 71 markers.\n",
      "2024-04-17 18:10:05,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042973.1 (Escherichia coli, CFSAN061768) loaded with 69 markers.\n",
      "2024-04-17 18:10:05,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042899.1 (Escherichia coli, CFSAN061763) loaded with 69 markers.\n",
      "2024-04-17 18:10:05,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042901.1 (Escherichia coli, CFSAN061762) loaded with 75 markers.\n",
      "2024-04-17 18:10:05,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042903.1 (Escherichia coli, CFSAN061761) loaded with 70 markers.\n",
      "2024-04-17 18:10:05,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020835.1 (Escherichia coli, CFSAN051542) loaded with 74 markers.\n",
      "2024-04-17 18:10:05,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026935.2 (Escherichia coli, CFS3292) loaded with 92 markers.\n",
      "2024-04-17 18:10:06,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026932.2 (Escherichia coli, CFS3273) loaded with 57 markers.\n",
      "2024-04-17 18:10:06,181 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026929.2 (Escherichia coli, CFS3246) loaded with 56 markers.\n",
      "2024-04-17 18:10:06,259 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide K (possible IUPAC code) in NZ_CP064678.1,IDX=0, while parsing marker fimE:BLAST<129842...>. Skipping strain NZ_CP064678.1.\n",
      "2024-04-17 18:10:06,344 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide K (possible IUPAC code) in NZ_CP064679.1,IDX=0, while parsing marker fimE:BLAST<129841...>. Skipping strain NZ_CP064679.1.\n",
      "2024-04-17 18:10:06,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064680.1 (Escherichia coli, CETR1G40) loaded with 85 markers.\n",
      "2024-04-17 18:10:06,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073621.1 (Escherichia coli, CE2050) loaded with 78 markers.\n",
      "2024-04-17 18:10:06,821 [DEBUG - chronostrain.database.parser.json] - Strain NC_013941.1 (Escherichia coli, CB9615) loaded with 73 markers.\n",
      "2024-04-17 18:10:07,013 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026026.1 (Escherichia coli, CAR) loaded with 84 markers.\n",
      "2024-04-17 18:10:07,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010129.1 (Escherichia coli, C9) loaded with 65 markers.\n",
      "2024-04-17 18:10:07,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031214.1 (Escherichia coli, C600) loaded with 85 markers.\n",
      "2024-04-17 18:10:07,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046025.1 (Escherichia coli, C439) loaded with 88 markers.\n",
      "2024-04-17 18:10:07,549 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010116.1 (Escherichia coli, C1) loaded with 69 markers.\n",
      "2024-04-17 18:10:07,610 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide K (possible IUPAC code) in NZ_CP064681.1,IDX=0, while parsing marker fimE:BLAST<129840...>. Skipping strain NZ_CP064681.1.\n",
      "2024-04-17 18:10:07,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064682.1 (Escherichia coli, BZKR2G40) loaded with 85 markers.\n",
      "2024-04-17 18:10:07,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069133.1 (Escherichia coli, BW25113_substr._CHXR2G20) loaded with 85 markers.\n",
      "2024-04-17 18:10:07,963 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069132.1 (Escherichia coli, BW25113_substr._CHXR1G20) loaded with 85 markers.\n",
      "2024-04-17 18:10:08,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059723.1 (Escherichia coli, BUCT2930) loaded with 68 markers.\n",
      "2024-04-17 18:10:08,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063332.1 (Escherichia coli, BS74R-D) loaded with 72 markers.\n",
      "2024-04-17 18:10:08,506 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093068.1 (Escherichia coli, BR1220) loaded with 75 markers.\n",
      "2024-04-17 18:10:08,704 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029239.1 (Escherichia coli, BH212) loaded with 88 markers.\n",
      "2024-04-17 18:10:08,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032986.1 (Escherichia coli, BE2-5) loaded with 75 markers.\n",
      "2024-04-17 18:10:08,917 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040643.1 (Escherichia coli, BE104) loaded with 85 markers.\n",
      "2024-04-17 18:10:09,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090456.1 (Escherichia coli, B379) loaded with 79 markers.\n",
      "2024-04-17 18:10:09,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042865.1 (Escherichia coli, ATCC_BAA-196) loaded with 89 markers.\n",
      "2024-04-17 18:10:10,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042867.1 (Escherichia coli, ATCC_BAA-196) loaded with 89 markers.\n",
      "2024-04-17 18:10:10,338 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034658.1 (Escherichia coli, ATCC_98082) loaded with 85 markers.\n",
      "2024-04-17 18:10:10,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066366.1 (Escherichia coli, AR_Bank_#0346) loaded with 67 markers.\n",
      "2024-04-17 18:10:10,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021683.1 (Escherichia coli, AR_0162) loaded with 51 markers.\n",
      "2024-04-17 18:10:10,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021691.1 (Escherichia coli, AR_0151) loaded with 51 markers.\n",
      "2024-04-17 18:10:10,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021736.1 (Escherichia coli, AR_0150) loaded with 52 markers.\n",
      "2024-04-17 18:10:10,828 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021532.1 (Escherichia coli, AR_0149) loaded with 52 markers.\n",
      "2024-04-17 18:10:10,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021879.1 (Escherichia coli, AR_0137) loaded with 51 markers.\n",
      "2024-04-17 18:10:10,993 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021732.1 (Escherichia coli, AR_0114) loaded with 51 markers.\n",
      "2024-04-17 18:10:11,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024859.1 (Escherichia coli, AR_0014) loaded with 52 markers.\n",
      "2024-04-17 18:10:11,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024855.1 (Escherichia coli, AR_0011) loaded with 53 markers.\n",
      "2024-04-17 18:10:11,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029115.1 (Escherichia coli, AR435) loaded with 53 markers.\n",
      "2024-04-17 18:10:11,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043942.1 (Escherichia coli, AR216.2b) loaded with 53 markers.\n",
      "2024-04-17 18:10:11,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043946.1 (Escherichia coli, AR202.2) loaded with 53 markers.\n",
      "2024-04-17 18:10:11,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP005930.1 (Escherichia coli, APEC_IMT5155) loaded with 71 markers.\n",
      "2024-04-17 18:10:11,659 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058355.1 (Escherichia coli, AMSCJX03) loaded with 77 markers.\n",
      "2024-04-17 18:10:11,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062246.1 (Escherichia coli, AML003_par) loaded with 85 markers.\n",
      "2024-04-17 18:10:11,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062243.1 (Escherichia coli, AML003_ev03) loaded with 86 markers.\n",
      "2024-04-17 18:10:11,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062244.1 (Escherichia coli, AML003_ev02) loaded with 85 markers.\n",
      "2024-04-17 18:10:12,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062245.1 (Escherichia coli, AML003_ev01) loaded with 85 markers.\n",
      "2024-04-17 18:10:12,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062250.1 (Escherichia coli, AML002_par) loaded with 85 markers.\n",
      "2024-04-17 18:10:12,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062247.1 (Escherichia coli, AML002_ev03) loaded with 86 markers.\n",
      "2024-04-17 18:10:12,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062248.1 (Escherichia coli, AML002_ev02) loaded with 85 markers.\n",
      "2024-04-17 18:10:12,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062249.1 (Escherichia coli, AML002_ev01) loaded with 86 markers.\n",
      "2024-04-17 18:10:12,639 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062242.1 (Escherichia coli, AML001_par) loaded with 85 markers.\n",
      "2024-04-17 18:10:12,833 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062239.1 (Escherichia coli, AML001_ev03) loaded with 85 markers.\n",
      "2024-04-17 18:10:13,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062240.1 (Escherichia coli, AML001_ev02) loaded with 86 markers.\n",
      "2024-04-17 18:10:13,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062241.1 (Escherichia coli, AML001_ev01) loaded with 86 markers.\n",
      "2024-04-17 18:10:13,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037857.2 (Escherichia coli, AM187) loaded with 78 markers.\n",
      "2024-04-17 18:10:13,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058302.1 (Escherichia coli, AH65) loaded with 72 markers.\n",
      "2024-04-17 18:10:13,686 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055259.1 (Escherichia coli, AH62) loaded with 72 markers.\n",
      "2024-04-17 18:10:13,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067265.1 (Escherichia coli, ABW_S38) loaded with 76 markers.\n",
      "2024-04-17 18:10:13,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067299.1 (Escherichia coli, ABW_A33) loaded with 61 markers.\n",
      "2024-04-17 18:10:14,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067303.1 (Escherichia coli, ABW_A32) loaded with 61 markers.\n",
      "2024-04-17 18:10:14,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028745.1 (Escherichia coli, A30) loaded with 74 markers.\n",
      "2024-04-17 18:10:14,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040390.1 (Escherichia coli, A1_136) loaded with 75 markers.\n",
      "2024-04-17 18:10:14,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028760.1 (Escherichia coli, A14) loaded with 71 markers.\n",
      "2024-04-17 18:10:14,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026342.1 (Escherichia coli, 9FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:14,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026343.1 (Escherichia coli, 9A) loaded with 86 markers.\n",
      "2024-04-17 18:10:15,015 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070902.1 (Escherichia coli, 99COLEC) loaded with 68 markers.\n",
      "2024-04-17 18:10:15,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051001.1 (Escherichia coli, 98-3133) loaded with 75 markers.\n",
      "2024-04-17 18:10:15,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061758.1 (Escherichia coli, 97.3) loaded with 86 markers.\n",
      "2024-04-17 18:10:15,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026344.1 (Escherichia coli, 8FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:15,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026345.1 (Escherichia coli, 8A) loaded with 86 markers.\n",
      "2024-04-17 18:10:15,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061754.1 (Escherichia coli, 873.10) loaded with 74 markers.\n",
      "2024-04-17 18:10:16,060 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088131.1 (Escherichia coli, 81) loaded with 89 markers.\n",
      "2024-04-17 18:10:16,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026346.1 (Escherichia coli, 7FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:16,298 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026347.1 (Escherichia coli, 7A) loaded with 86 markers.\n",
      "2024-04-17 18:10:16,425 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023353.1 (Escherichia coli, 746) loaded with 74 markers.\n",
      "2024-04-17 18:10:16,515 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025842.1 (Escherichia coli, 720632) loaded with 52 markers.\n",
      "2024-04-17 18:10:16,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026348.1 (Escherichia coli, 6FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:16,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026349.1 (Escherichia coli, 6A) loaded with 86 markers.\n",
      "2024-04-17 18:10:16,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027701.1 (Escherichia coli, 675SK2) loaded with 51 markers.\n",
      "2024-04-17 18:10:16,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070914.1 (Escherichia coli, 65COLEC) loaded with 70 markers.\n",
      "2024-04-17 18:10:17,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010371.1 (Escherichia coli, 6409) loaded with 59 markers.\n",
      "2024-04-17 18:10:17,237 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040263.1 (Escherichia coli, 631) loaded with 76 markers.\n",
      "2024-04-17 18:10:17,340 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050040.1 (Escherichia coli, 63-2) loaded with 72 markers.\n",
      "2024-04-17 18:10:17,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026350.1 (Escherichia coli, 5FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:17,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021175.1 (Escherichia coli, 5CRE51) loaded with 58 markers.\n",
      "2024-04-17 18:10:17,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026351.1 (Escherichia coli, 5A) loaded with 86 markers.\n",
      "2024-04-17 18:10:17,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041284.1 (Escherichia coli, 54) loaded with 66 markers.\n",
      "2024-04-17 18:10:17,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050382.1 (Escherichia coli, 52148) loaded with 51 markers.\n",
      "2024-04-17 18:10:18,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029973.1 (Escherichia coli, 51008369SK1) loaded with 53 markers.\n",
      "2024-04-17 18:10:18,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025851.1 (Escherichia coli, 510016) loaded with 64 markers.\n",
      "2024-04-17 18:10:18,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023834.1 (Escherichia coli, 4_2-1) loaded with 53 markers.\n",
      "2024-04-17 18:10:18,355 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026352.1 (Escherichia coli, 4FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:18,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026353.1 (Escherichia coli, 4A) loaded with 86 markers.\n",
      "2024-04-17 18:10:18,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026354.1 (Escherichia coli, 3FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:18,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026355.1 (Escherichia coli, 3A) loaded with 86 markers.\n",
      "2024-04-17 18:10:18,907 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048310.1 (Escherichia coli, 32-4) loaded with 51 markers.\n",
      "2024-04-17 18:10:19,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025903.1 (Escherichia coli, 300709) loaded with 76 markers.\n",
      "2024-04-17 18:10:19,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026359.1 (Escherichia coli, 2_0) loaded with 86 markers.\n",
      "2024-04-17 18:10:19,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026357.1 (Escherichia coli, 2F_0) loaded with 86 markers.\n",
      "2024-04-17 18:10:19,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026356.1 (Escherichia coli, 2FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:19,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026358.1 (Escherichia coli, 2A) loaded with 86 markers.\n",
      "2024-04-17 18:10:19,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038505.1 (Escherichia coli, 28Eco12) loaded with 76 markers.\n",
      "2024-04-17 18:10:19,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026723.1 (Escherichia coli, 266917_2) loaded with 73 markers.\n",
      "2024-04-17 18:10:20,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027118.1 (Escherichia coli, 26561) loaded with 59 markers.\n",
      "2024-04-17 18:10:20,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031833.1 (Escherichia coli, 2452) loaded with 75 markers.\n",
      "2024-04-17 18:10:20,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020514.1 (Escherichia coli, 219) loaded with 52 markers.\n",
      "2024-04-17 18:10:20,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023286.1 (Escherichia coli, 2018-18.25PC) loaded with 72 markers.\n",
      "2024-04-17 18:10:20,681 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021894.1 (Escherichia coli, 2018-10-1CC) loaded with 66 markers.\n",
      "2024-04-17 18:10:20,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021893.1 (Escherichia coli, 2018-06-4CC) loaded with 74 markers.\n",
      "2024-04-17 18:10:20,892 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021896.1 (Escherichia coli, 2018-02-2CC) loaded with 68 markers.\n",
      "2024-04-17 18:10:21,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025209.2 (Escherichia coli, 2017.06.04CC) loaded with 91 markers.\n",
      "2024-04-17 18:10:21,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015241.1 (Escherichia coli, 2013C-4465) loaded with 75 markers.\n",
      "2024-04-17 18:10:21,309 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027437.1 (Escherichia coli, 2012C-4221) loaded with 75 markers.\n",
      "2024-04-17 18:10:21,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026360.1 (Escherichia coli, 1FA) loaded with 86 markers.\n",
      "2024-04-17 18:10:21,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026361.1 (Escherichia coli, 1A) loaded with 86 markers.\n",
      "2024-04-17 18:10:21,629 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046009.1 (Escherichia coli, 1919D62) loaded with 76 markers.\n",
      "2024-04-17 18:10:21,775 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046003.1 (Escherichia coli, 1916D6) loaded with 66 markers.\n",
      "2024-04-17 18:10:21,944 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048934.1 (Escherichia coli, 190693) loaded with 68 markers.\n",
      "2024-04-17 18:10:22,090 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072858.1 (Escherichia coli, 184_2aE) loaded with 59 markers.\n",
      "2024-04-17 18:10:22,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051158.1 (Escherichia coli, 17MR471) loaded with 75 markers.\n",
      "2024-04-17 18:10:22,342 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048367.1 (Escherichia coli, 164) loaded with 51 markers.\n",
      "2024-04-17 18:10:22,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048371.1 (Escherichia coli, 163) loaded with 52 markers.\n",
      "2024-04-17 18:10:22,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032145.1 (Escherichia coli, 15.TR.026_OXA) loaded with 75 markers.\n",
      "2024-04-17 18:10:22,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076237.1 (Escherichia coli, 146N4) loaded with 80 markers.\n",
      "2024-04-17 18:10:22,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023364.1 (Escherichia coli, 144) loaded with 78 markers.\n",
      "2024-04-17 18:10:23,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063369.1 (Escherichia coli, 1409) loaded with 75 markers.\n",
      "2024-04-17 18:10:23,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009166.1 (Escherichia coli, 1303) loaded with 73 markers.\n",
      "2024-04-17 18:10:23,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023371.1 (Escherichia coli, 1283) loaded with 63 markers.\n",
      "2024-04-17 18:10:23,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023383.1 (Escherichia coli, 1223) loaded with 60 markers.\n",
      "2024-04-17 18:10:23,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071521.1 (Escherichia coli, 11202) loaded with 88 markers.\n",
      "2024-04-17 18:10:23,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070232.1 (Escherichia coli, 10R) loaded with 59 markers.\n",
      "2024-04-17 18:10:23,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT615377.1 (Escherichia coli, 106) loaded with 84 markers.\n",
      "2024-04-17 18:10:24,008 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide Y (possible IUPAC code) in NZ_LT615378.1,IDX=0, while parsing marker fimD:BLAST<1100717...>. Skipping strain NZ_LT615378.1.\n",
      "2024-04-17 18:10:24,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091427.1 (Escherichia coli, 1000C-3) loaded with 53 markers.\n",
      "2024-04-17 18:10:24,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048330.1 (Escherichia coli, 10) loaded with 61 markers.\n",
      "2024-04-17 18:10:24,338 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084026.1 (Escherichia coli, 035053C1pKPC2_Vir_090515B1) loaded with 87 markers.\n",
      "2024-04-17 18:10:24,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067311.1 (Escherichia coli, 027_SK3) loaded with 76 markers.\n",
      "2024-04-17 18:10:24,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058661.1 (Escherichia coli, 005008) loaded with 52 markers.\n",
      "2024-04-17 18:10:24,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022215.1 (Escherichia coli, WP8-S17-ESBL-05) loaded with 72 markers.\n",
      "2024-04-17 18:10:24,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032989.1 (Escherichia coli, W2-5) loaded with 78 markers.\n",
      "2024-04-17 18:10:24,859 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049343.1 (Escherichia coli, UMB200201_11) loaded with 70 markers.\n",
      "2024-04-17 18:10:24,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035850.1 (Escherichia coli, TW10590) loaded with 76 markers.\n",
      "2024-04-17 18:10:25,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059835.1 (Escherichia coli, tcmA_3) loaded with 72 markers.\n",
      "2024-04-17 18:10:25,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091029.1 (Escherichia coli, STEC384) loaded with 75 markers.\n",
      "2024-04-17 18:10:25,374 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091024.1 (Escherichia coli, STEC379) loaded with 75 markers.\n",
      "2024-04-17 18:10:25,515 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075649.1 (Escherichia coli, STEC17-190) loaded with 74 markers.\n",
      "2024-04-17 18:10:25,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041437.1 (Escherichia coli, STEC005) loaded with 74 markers.\n",
      "2024-04-17 18:10:25,801 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053234.1 (Escherichia coli, SCU-106) loaded with 71 markers.\n",
      "2024-04-17 18:10:25,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043821.1 (Escherichia coli, RM10740) loaded with 79 markers.\n",
      "2024-04-17 18:10:26,076 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080133.1 (Escherichia coli, PK8277) loaded with 86 markers.\n",
      "2024-04-17 18:10:26,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080121.1 (Escherichia coli, PK8217) loaded with 87 markers.\n",
      "2024-04-17 18:10:26,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075477.1 (Escherichia coli, NT1W25) loaded with 77 markers.\n",
      "2024-04-17 18:10:26,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075484.1 (Escherichia coli, NT1N25) loaded with 77 markers.\n",
      "2024-04-17 18:10:26,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134080.1 (Escherichia coli, NCTC9082) loaded with 71 markers.\n",
      "2024-04-17 18:10:26,748 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069677.1 (Escherichia coli, MIN10) loaded with 73 markers.\n",
      "2024-04-17 18:10:26,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010196.1 (Escherichia coli, M9) loaded with 68 markers.\n",
      "2024-04-17 18:10:27,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010186.1 (Escherichia coli, M6) loaded with 68 markers.\n",
      "2024-04-17 18:10:27,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010221.1 (Escherichia coli, M19) loaded with 68 markers.\n",
      "2024-04-17 18:10:27,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010213.1 (Escherichia coli, M15) loaded with 68 markers.\n",
      "2024-04-17 18:10:27,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010206.1 (Escherichia coli, M11) loaded with 67 markers.\n",
      "2024-04-17 18:10:27,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042585.1 (Escherichia coli, LD91-1) loaded with 71 markers.\n",
      "2024-04-17 18:10:27,597 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091691.1 (Escherichia coli, KMi006) loaded with 51 markers.\n",
      "2024-04-17 18:10:27,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091659.1 (Escherichia coli, KFu024) loaded with 54 markers.\n",
      "2024-04-17 18:10:27,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083263.1 (Escherichia coli, JNU) loaded with 74 markers.\n",
      "2024-04-17 18:10:27,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054282.1 (Escherichia coli, IDR1900005638-01-02) loaded with 69 markers.\n",
      "2024-04-17 18:10:27,994 [DEBUG - chronostrain.database.parser.json] - Strain NC_011741.1 (Escherichia coli, IAI1) loaded with 71 markers.\n",
      "2024-04-17 18:10:28,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010170.1 (Escherichia coli, H6) loaded with 81 markers.\n",
      "2024-04-17 18:10:28,221 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010169.1 (Escherichia coli, H5) loaded with 81 markers.\n",
      "2024-04-17 18:10:28,328 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028733.1 (Escherichia coli, H2) loaded with 76 markers.\n",
      "2024-04-17 18:10:28,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025318.1 (Escherichia coli, FORC_042) loaded with 71 markers.\n",
      "2024-04-17 18:10:28,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069542.1 (Escherichia coli, FDAARGOS_1248) loaded with 71 markers.\n",
      "2024-04-17 18:10:28,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046259.1 (Escherichia coli, ECO2947) loaded with 69 markers.\n",
      "2024-04-17 18:10:28,755 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073360.1 (Escherichia coli, EC11) loaded with 56 markers.\n",
      "2024-04-17 18:10:28,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027255.1 (Escherichia coli, EC11) loaded with 71 markers.\n",
      "2024-04-17 18:10:29,006 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069709.1 (Escherichia coli, EA7) loaded with 103 markers.\n",
      "2024-04-17 18:10:29,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010134.1 (Escherichia coli, D1) loaded with 74 markers.\n",
      "2024-04-17 18:10:29,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063349.1 (Escherichia coli, BS15R-A) loaded with 72 markers.\n",
      "2024-04-17 18:10:29,411 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076704.1 (Escherichia coli, BMH-17-0027) loaded with 75 markers.\n",
      "2024-04-17 18:10:29,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081711.1 (Escherichia coli, AH21) loaded with 77 markers.\n",
      "2024-04-17 18:10:29,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028756.1 (Escherichia coli, A19) loaded with 70 markers.\n",
      "2024-04-17 18:10:29,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028763.1 (Escherichia coli, A11) loaded with 70 markers.\n",
      "2024-04-17 18:10:29,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050047.1 (Escherichia coli, 90-1) loaded with 73 markers.\n",
      "2024-04-17 18:10:29,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016404.1 (Escherichia coli, 210221272) loaded with 68 markers.\n",
      "2024-04-17 18:10:30,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027550.1 (Escherichia coli, 2015C-4136CT1) loaded with 74 markers.\n",
      "2024-04-17 18:10:30,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027355.1 (Escherichia coli, 2013C-4991) loaded with 75 markers.\n",
      "2024-04-17 18:10:30,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027312.1 (Escherichia coli, 2013C-3181) loaded with 73 markers.\n",
      "2024-04-17 18:10:30,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046000.1 (Escherichia coli, 1916D18) loaded with 82 markers.\n",
      "2024-04-17 18:10:30,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024134.1 (Escherichia coli, 14EC017) loaded with 70 markers.\n",
      "2024-04-17 18:10:30,805 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024131.1 (Escherichia coli, 14EC007) loaded with 74 markers.\n",
      "2024-04-17 18:10:30,868 [DEBUG - chronostrain.database.parser.json] - Strain NC_007606.1 (Shigella dysenteriae, Sd197) loaded with 42 markers.\n",
      "2024-04-17 18:10:30,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026786.1 (Shigella dysenteriae, NCTC_9718) loaded with 42 markers.\n",
      "2024-04-17 18:10:30,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061527.1 (Shigella dysenteriae, HNCMB_20080) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,050 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024466.1 (Shigella dysenteriae, BU53M1) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026774.1 (Shigella dysenteriae, ATCC_13313) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026784.1 (Shigella dysenteriae, 80-547) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,257 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026777.1 (Shigella dysenteriae, 69-3818) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026780.1 (Shigella dysenteriae, 53-3937) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026782.1 (Shigella dysenteriae, 08-3380) loaded with 41 markers.\n",
      "2024-04-17 18:10:31,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026878.1 (Shigella dysenteriae, 07-3308) loaded with 42 markers.\n",
      "2024-04-17 18:10:31,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024090.1 (Escherichia coli, WG5) loaded with 63 markers.\n",
      "2024-04-17 18:10:31,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051753.1 (Escherichia coli, SCU-102) loaded with 75 markers.\n",
      "2024-04-17 18:10:31,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010238.1 (Escherichia coli, S50) loaded with 67 markers.\n",
      "2024-04-17 18:10:31,889 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882057.1 (Escherichia coli, Quinolone_resistant) loaded with 71 markers.\n",
      "2024-04-17 18:10:32,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006632.1 (Escherichia coli, PCN033) loaded with 75 markers.\n",
      "2024-04-17 18:10:32,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT906474.1 (Escherichia coli, NCTC122) loaded with 63 markers.\n",
      "2024-04-17 18:10:32,216 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020543.1 (Escherichia coli, C) loaded with 63 markers.\n",
      "2024-04-17 18:10:32,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031105.1 (Escherichia coli, AMSCJX02) loaded with 70 markers.\n",
      "2024-04-17 18:10:32,426 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028748.1 (Escherichia coli, A27) loaded with 64 markers.\n",
      "2024-04-17 18:10:32,527 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025849.1 (Escherichia coli, 600468) loaded with 72 markers.\n",
      "2024-04-17 18:10:32,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027579.1 (Escherichia coli, 2013C-4282) loaded with 80 markers.\n",
      "2024-04-17 18:10:32,783 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019256.1 (Escherichia coli, 13TMH22) loaded with 71 markers.\n",
      "2024-04-17 18:10:32,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037903.1 (Escherichia coli, LHM10-1) loaded with 77 markers.\n",
      "2024-04-17 18:10:33,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093239.1 (Escherichia marmotae, W49-2) loaded with 70 markers.\n",
      "2024-04-17 18:10:33,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056159.1 (Escherichia marmotae, RHBSTW-00814) loaded with 69 markers.\n",
      "2024-04-17 18:10:33,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056697.1 (Escherichia marmotae, RHBSTW-00605) loaded with 70 markers.\n",
      "2024-04-17 18:10:33,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058207.1 (Escherichia marmotae, RHB42-C09) loaded with 71 markers.\n",
      "2024-04-17 18:10:33,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058209.1 (Escherichia marmotae, RHB24-C12) loaded with 69 markers.\n",
      "2024-04-17 18:10:33,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134270.1 (Escherichia marmotae, NCTC8196) loaded with 69 markers.\n",
      "2024-04-17 18:10:33,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134340.1 (Escherichia marmotae, NCTC11133) loaded with 69 markers.\n",
      "2024-04-17 18:10:34,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025979.1 (Escherichia marmotae, HT073016) loaded with 68 markers.\n",
      "2024-04-17 18:10:34,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090448.1 (Escherichia coli, PT62) loaded with 65 markers.\n",
      "2024-04-17 18:10:34,350 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057266.1 (Escherichia fergusonii, RHB31-C13) loaded with 53 markers.\n",
      "2024-04-17 18:10:34,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057657.1 (Escherichia fergusonii, RHB19-C05) loaded with 53 markers.\n",
      "2024-04-17 18:10:34,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081881.1 (Escherichia coli, WW2) loaded with 75 markers.\n",
      "2024-04-17 18:10:34,811 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021933.1 (Escherichia coli, WP2-W18-ESBL-08) loaded with 75 markers.\n",
      "2024-04-17 18:10:34,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066806.1 (Escherichia coli, TCM3e1) loaded with 79 markers.\n",
      "2024-04-17 18:10:35,128 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055019.1 (Escherichia coli, SWHEFF_69) loaded with 75 markers.\n",
      "2024-04-17 18:10:35,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058748.1 (Escherichia coli, STLIN_13) loaded with 75 markers.\n",
      "2024-04-17 18:10:35,399 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058879.1 (Escherichia coli, STLEFF_29) loaded with 76 markers.\n",
      "2024-04-17 18:10:35,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068823.1 (Escherichia coli, RIVM_C029494) loaded with 76 markers.\n",
      "2024-04-17 18:10:35,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067241.1 (Escherichia coli, P19_598) loaded with 76 markers.\n",
      "2024-04-17 18:10:35,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP017620.1 (Escherichia coli, MRY15-131) loaded with 76 markers.\n",
      "2024-04-17 18:10:35,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP017617.1 (Escherichia coli, MRY15-117) loaded with 75 markers.\n",
      "2024-04-17 18:10:35,969 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054556.1 (Escherichia coli, LWY24) loaded with 76 markers.\n",
      "2024-04-17 18:10:36,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091665.1 (Escherichia coli, KOr019) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,195 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069501.1 (Escherichia coli, FDAARGOS_1268) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031293.1 (Escherichia coli, EC17GD31) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024826.1 (Escherichia coli, CREC-544) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053851.1 (Escherichia coli, CE1803A) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053852.1 (Escherichia coli, CE1628) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041996.1 (Escherichia coli, AR_Bank_#0349) loaded with 75 markers.\n",
      "2024-04-17 18:10:36,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029747.1 (Escherichia coli, 2016C-3878) loaded with 76 markers.\n",
      "2024-04-17 18:10:37,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064847.1 (Escherichia coli, 20) loaded with 76 markers.\n",
      "2024-04-17 18:10:37,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054388.1 (Escherichia coli, 1517k) loaded with 77 markers.\n",
      "2024-04-17 18:10:37,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056165.1 (Escherichia marmotae, RHBSTW-00777) loaded with 68 markers.\n",
      "2024-04-17 18:10:37,417 [DEBUG - chronostrain.database.parser.json] - Strain NC_011750.1 (Escherichia coli, IAI39) loaded with 79 markers.\n",
      "2024-04-17 18:10:37,567 [DEBUG - chronostrain.database.parser.json] - Strain NC_017646.1 (Escherichia coli, CE10) loaded with 79 markers.\n",
      "2024-04-17 18:10:37,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032204.1 (Escherichia coli, AR_0013) loaded with 81 markers.\n",
      "2024-04-17 18:10:37,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027140.1 (Escherichia coli, AR_0369) loaded with 68 markers.\n",
      "2024-04-17 18:10:37,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040443.1 (Escherichia sp., E4742) loaded with 59 markers.\n",
      "2024-04-17 18:10:38,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055070.1 (Shigella flexneri, SWHIN_97) loaded with 57 markers.\n",
      "2024-04-17 18:10:38,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055075.1 (Shigella flexneri, SWHIN_116) loaded with 50 markers.\n",
      "2024-04-17 18:10:38,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055037.1 (Shigella flexneri, SWHEFF_60) loaded with 58 markers.\n",
      "2024-04-17 18:10:38,281 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058853.1 (Shigella flexneri, STLEFF_38) loaded with 58 markers.\n",
      "2024-04-17 18:10:38,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058826.1 (Shigella flexneri, STLE1) loaded with 49 markers.\n",
      "2024-04-17 18:10:38,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055000.1 (Shigella flexneri, STIN_80) loaded with 54 markers.\n",
      "2024-04-17 18:10:38,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035882.1 (Escherichia coli, WS0115A) loaded with 75 markers.\n",
      "2024-04-17 18:10:38,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092452.1 (Escherichia coli, TKEC21-59) loaded with 55 markers.\n",
      "2024-04-17 18:10:38,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092449.1 (Escherichia coli, TKEC21-17) loaded with 55 markers.\n",
      "2024-04-17 18:10:38,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062851.1 (Escherichia coli, Res13-Lact-PEB21-38-A) loaded with 56 markers.\n",
      "2024-04-17 18:10:38,859 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062853.1 (Escherichia coli, Res13-Lact-PEB20-37-A) loaded with 56 markers.\n",
      "2024-04-17 18:10:38,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR881940.1 (Escherichia coli, Quinolone_resistant) loaded with 54 markers.\n",
      "2024-04-17 18:10:39,038 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006636.1 (Escherichia coli, PCN061) loaded with 58 markers.\n",
      "2024-04-17 18:10:39,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075472.1 (Escherichia coli, NT1F25) loaded with 62 markers.\n",
      "2024-04-17 18:10:39,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134000.1 (Escherichia coli, NCTC9066) loaded with 71 markers.\n",
      "2024-04-17 18:10:39,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047658.1 (Escherichia coli, LD39-1) loaded with 68 markers.\n",
      "2024-04-17 18:10:39,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092639.1 (Escherichia coli, HS) loaded with 55 markers.\n",
      "2024-04-17 18:10:39,477 [DEBUG - chronostrain.database.parser.json] - Strain NC_009800.1 (Escherichia coli, HS) loaded with 55 markers.\n",
      "2024-04-17 18:10:39,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010163.1 (Escherichia coli, H2) loaded with 52 markers.\n",
      "2024-04-17 18:10:39,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071436.1 (Escherichia coli, EcPNK006) loaded with 56 markers.\n",
      "2024-04-17 18:10:39,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022959.1 (Escherichia coli, ATCC_8739) loaded with 69 markers.\n",
      "2024-04-17 18:10:39,813 [DEBUG - chronostrain.database.parser.json] - Strain NC_010468.1 (Escherichia coli, ATCC_8739) loaded with 69 markers.\n",
      "2024-04-17 18:10:39,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020058.1 (Escherichia coli, AR_0061) loaded with 74 markers.\n",
      "2024-04-17 18:10:40,003 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070930.1 (Escherichia coli, 30COLEC) loaded with 66 markers.\n",
      "2024-04-17 18:10:40,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038180.1 (Escherichia coli, 2_HS-C) loaded with 56 markers.\n",
      "2024-04-17 18:10:40,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019280.1 (Escherichia coli, 13P484A) loaded with 55 markers.\n",
      "2024-04-17 18:10:40,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023377.1 (Escherichia coli, 127) loaded with 55 markers.\n",
      "2024-04-17 18:10:40,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058780.1 (Shigella flexneri, STLIN_17) loaded with 77 markers.\n",
      "2024-04-17 18:10:40,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035836.1 (Escherichia coli, WS1933D) loaded with 75 markers.\n",
      "2024-04-17 18:10:40,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035855.1 (Escherichia coli, TW11681) loaded with 70 markers.\n",
      "2024-04-17 18:10:40,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062876.1 (Escherichia coli, Res13-Lact-PEA12-26) loaded with 74 markers.\n",
      "2024-04-17 18:10:40,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134236.1 (Escherichia coli, NCTC9008) loaded with 72 markers.\n",
      "2024-04-17 18:10:40,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070103.1 (Escherichia coli, FDAARGOS_1277) loaded with 80 markers.\n",
      "2024-04-17 18:10:41,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069710.1 (Escherichia coli, EA2) loaded with 74 markers.\n",
      "2024-04-17 18:10:41,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069707.1 (Escherichia coli, E41) loaded with 56 markers.\n",
      "2024-04-17 18:10:41,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081698.1 (Escherichia coli, CT02) loaded with 80 markers.\n",
      "2024-04-17 18:10:41,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028746.1 (Escherichia coli, A29) loaded with 71 markers.\n",
      "2024-04-17 18:10:41,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028752.1 (Escherichia coli, A23) loaded with 65 markers.\n",
      "2024-04-17 18:10:41,637 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035826.1 (Escherichia coli, 67-1) loaded with 70 markers.\n",
      "2024-04-17 18:10:41,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025859.1 (Escherichia coli, 504239) loaded with 77 markers.\n",
      "2024-04-17 18:10:41,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025840.1 (Escherichia coli, 214-4) loaded with 65 markers.\n",
      "2024-04-17 18:10:42,094 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013662.1 (Escherichia coli, 08-00022) loaded with 68 markers.\n",
      "2024-04-17 18:10:42,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053045.1 (Escherichia fergusonii, HNCF11W) loaded with 48 markers.\n",
      "2024-04-17 18:10:42,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086604.1 (Escherichia fergusonii, EF20JDJ4045) loaded with 51 markers.\n",
      "2024-04-17 18:10:42,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR882050.1 (Escherichia coli, Quinolone_resistant) loaded with 64 markers.\n",
      "2024-04-17 18:10:42,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067245.1 (Escherichia coli, OW1E2) loaded with 78 markers.\n",
      "2024-04-17 18:10:42,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057692.1 (Escherichia fergusonii, RHB17-C11) loaded with 52 markers.\n",
      "2024-04-17 18:10:42,692 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077242.1 (Escherichia fergusonii, FDAARGOS_1438) loaded with 52 markers.\n",
      "2024-04-17 18:10:42,778 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042945.1 (Escherichia fergusonii, ATCC_35471) loaded with 52 markers.\n",
      "2024-04-17 18:10:42,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057210.1 (Escherichia fergusonii, RHB33-C07) loaded with 54 markers.\n",
      "2024-04-17 18:10:42,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057215.1 (Escherichia fergusonii, RHB33-C04) loaded with 54 markers.\n",
      "2024-04-17 18:10:43,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057243.1 (Escherichia fergusonii, RHB32-C05) loaded with 65 markers.\n",
      "2024-04-17 18:10:43,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040805.1 (Escherichia fergusonii, EFCF056) loaded with 107 markers.\n",
      "2024-04-17 18:10:43,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP014856.1 (Escherichia albertii, CB9786) loaded with 54 markers.\n",
      "2024-04-17 18:10:43,370 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034166.1 (Escherichia albertii, 2014C-4015) loaded with 51 markers.\n",
      "2024-04-17 18:10:43,458 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030787.2 (Escherichia albertii, 2013C-4143) loaded with 54 markers.\n",
      "2024-04-17 18:10:43,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030783.2 (Escherichia albertii, 2012EL-1823B) loaded with 57 markers.\n",
      "2024-04-17 18:10:43,642 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034162.1 (Escherichia albertii, 06-3542) loaded with 50 markers.\n",
      "2024-04-17 18:10:43,723 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070290.1 (Escherichia albertii, Sample_167) loaded with 51 markers.\n",
      "2024-04-17 18:10:43,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP014855.1 (Escherichia albertii, NIAH_Bird_3) loaded with 55 markers.\n",
      "2024-04-17 18:10:44,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP014857.1 (Escherichia albertii, EC06-170) loaded with 51 markers.\n",
      "2024-04-17 18:10:44,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025317.1 (Escherichia albertii, 1551-2) loaded with 54 markers.\n",
      "2024-04-17 18:10:44,272 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070281.2 (Escherichia albertii, Sample_170) loaded with 54 markers.\n",
      "2024-04-17 18:10:44,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070284.2 (Escherichia albertii, Sample_169) loaded with 54 markers.\n",
      "2024-04-17 18:10:44,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070287.2 (Escherichia albertii, Sample_168) loaded with 50 markers.\n",
      "2024-04-17 18:10:44,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043271.1 (Escherichia albertii, RM9973) loaded with 54 markers.\n",
      "2024-04-17 18:10:44,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034213.1 (Escherichia albertii, NCTC_9362) loaded with 54 markers.\n",
      "2024-04-17 18:10:44,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025676.1 (Escherichia albertii, ChinaSP140150) loaded with 51 markers.\n",
      "2024-04-17 18:10:44,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024282.1 (Escherichia albertii, 2014C-4356) loaded with 52 markers.\n",
      "2024-04-17 18:10:45,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030781.1 (Escherichia albertii, 07-3866) loaded with 54 markers.\n",
      "2024-04-17 18:10:45,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030778.2 (Escherichia albertii, 05-3106) loaded with 54 markers.\n",
      "2024-04-17 18:10:45,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056961.1 (Escherichia fergusonii, RHB41-C20) loaded with 47 markers.\n",
      "2024-04-17 18:10:45,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056968.1 (Escherichia fergusonii, RHB41-C15) loaded with 47 markers.\n",
      "2024-04-17 18:10:45,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057093.1 (Escherichia fergusonii, RHB38-C07) loaded with 47 markers.\n",
      "2024-04-17 18:10:45,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057357.1 (Escherichia fergusonii, RHB28-C21) loaded with 57 markers.\n",
      "2024-04-17 18:10:45,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070292.1 (Escherichia albertii, Sample_166) loaded with 52 markers.\n",
      "2024-04-17 18:10:45,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070296.1 (Escherichia albertii, Sample_165) loaded with 52 markers.\n",
      "2024-04-17 18:10:45,956 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007025.1 (Escherichia albertii, KF1) loaded with 52 markers.\n",
      "2024-04-17 18:10:46,078 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056916.1 (Escherichia fergusonii, RHB42-C24) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056936.1 (Escherichia fergusonii, RHB42-C13) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056951.1 (Escherichia fergusonii, RHB42-C01) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057101.1 (Escherichia fergusonii, RHB38-C04) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057220.1 (Escherichia fergusonii, RHB32-C22) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057229.1 (Escherichia fergusonii, RHB32-C16) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,779 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057238.1 (Escherichia fergusonii, RHB32-C09) loaded with 51 markers.\n",
      "2024-04-17 18:10:46,877 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057256.1 (Escherichia fergusonii, RHB31-C17) loaded with 49 markers.\n",
      "2024-04-17 18:10:46,991 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057311.1 (Escherichia fergusonii, RHB30-C07) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,115 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055675.1 (Escherichia fergusonii, RHB28-C13) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057458.1 (Escherichia fergusonii, RHB26-C09) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057481.1 (Escherichia fergusonii, RHB25-C10) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,444 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057504.1 (Escherichia fergusonii, RHB24-C08) loaded with 50 markers.\n",
      "2024-04-17 18:10:47,548 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057505.1 (Escherichia fergusonii, RHB24-C06) loaded with 50 markers.\n",
      "2024-04-17 18:10:47,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057561.1 (Escherichia fergusonii, RHB23-C02) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057565.1 (Escherichia fergusonii, RHB23-C01) loaded with 49 markers.\n",
      "2024-04-17 18:10:47,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057896.1 (Escherichia fergusonii, RHB10-C23) loaded with 51 markers.\n",
      "2024-04-17 18:10:47,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057918.1 (Escherichia fergusonii, RHB10-C04) loaded with 51 markers.\n",
      "2024-04-17 18:10:47,944 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058057.1 (Escherichia fergusonii, RHB02-C22) loaded with 49 markers.\n",
      "2024-04-17 18:10:48,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083638.1 (Escherichia fergusonii, FDAARGOS_1499) loaded with 50 markers.\n",
      "2024-04-17 18:10:48,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066085.1 (Escherichia fergusonii, FDAARGOS_1032) loaded with 50 markers.\n",
      "2024-04-17 18:10:48,150 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070873.1 (Escherichia fergusonii, EF91) loaded with 51 markers.\n",
      "2024-04-17 18:10:48,221 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070954.1 (Escherichia fergusonii, EF44) loaded with 52 markers.\n",
      "2024-04-17 18:10:48,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070951.1 (Escherichia fergusonii, EF31) loaded with 49 markers.\n",
      "2024-04-17 18:10:48,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042942.1 (Escherichia fergusonii, ATCC_35473) loaded with 49 markers.\n",
      "2024-04-17 18:10:48,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042946.1 (Escherichia fergusonii, ATCC_35470) loaded with 47 markers.\n",
      "2024-04-17 18:10:48,611 [DEBUG - chronostrain.database.parser.json] - Strain NC_011740.1 (Escherichia fergusonii, ATCC_35469T) loaded with 50 markers.\n",
      "2024-04-17 18:10:48,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079884.1 (Escherichia fergusonii, 6S41-1) loaded with 51 markers.\n",
      "2024-04-17 18:10:48,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079891.1 (Escherichia fergusonii, 5zf15-2-1) loaded with 49 markers.\n",
      "2024-04-17 18:10:48,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034212.1 (Escherichia albertii, 2010C-3449) loaded with 52 markers.\n",
      "2024-04-17 18:10:49,091 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055599.1 (Escherichia fergusonii, RHB41-C23) loaded with 52 markers.\n",
      "2024-04-17 18:10:49,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057104.1 (Escherichia fergusonii, RHB38-C01) loaded with 56 markers.\n",
      "2024-04-17 18:10:49,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057463.1 (Escherichia fergusonii, RHB26-C03) loaded with 52 markers.\n",
      "2024-04-17 18:10:49,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057681.1 (Escherichia fergusonii, RHB18-C04) loaded with 51 markers.\n",
      "2024-04-17 18:10:49,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043225.1 (Enterobacter hormaechei, PG20180056) loaded with 34 markers.\n",
      "2024-04-17 18:10:49,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043226.1 (Enterobacter hormaechei, PG20180049) loaded with 34 markers.\n",
      "2024-04-17 18:10:49,620 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079939.1 (Enterobacter hormaechei, G17) loaded with 34 markers.\n",
      "2024-04-17 18:10:49,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032841.1 (Enterobacter hormaechei, C15117) loaded with 32 markers.\n",
      "2024-04-17 18:10:49,781 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063224.1 (Enterobacter hormaechei, BD-50-Eh) loaded with 39 markers.\n",
      "2024-04-17 18:10:49,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056663.1 (Enterobacter hormaechei, RHBSTW-00668) loaded with 35 markers.\n",
      "2024-04-17 18:10:49,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056676.1 (Enterobacter hormaechei, RHBSTW-00642) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056691.1 (Enterobacter hormaechei, RHBSTW-00626) loaded with 36 markers.\n",
      "2024-04-17 18:10:50,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056455.1 (Enterobacter hormaechei, RHBSTW-00492) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056719.1 (Enterobacter hormaechei, RHBSTW-00333) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056727.1 (Enterobacter hormaechei, RHBSTW-00316) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058128.1 (Enterobacter hormaechei, RHBSTW-00140) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,672 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058146.1 (Enterobacter hormaechei, RHBSTW-00133) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,784 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058149.1 (Enterobacter hormaechei, RHBSTW-00114) loaded with 35 markers.\n",
      "2024-04-17 18:10:50,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058157.1 (Enterobacter hormaechei, RHBSTW-00086) loaded with 47 markers.\n",
      "2024-04-17 18:10:50,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059419.1 (Enterobacter hormaechei, ENCL48880) loaded with 37 markers.\n",
      "2024-04-17 18:10:51,073 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049046.1 (Enterobacter hormaechei, Y233) loaded with 36 markers.\n",
      "2024-04-17 18:10:51,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090792.1 (Enterobacter hormaechei, FY-1) loaded with 47 markers.\n",
      "2024-04-17 18:10:51,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044107.1 (Enterobacter hormaechei, FDAARGOS_642) loaded with 32 markers.\n",
      "2024-04-17 18:10:51,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068215.1 (Enterobacter hormaechei, FDAARGOS_1078) loaded with 46 markers.\n",
      "2024-04-17 18:10:51,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047570.1 (Enterobacter hormaechei, F2) loaded with 33 markers.\n",
      "2024-04-17 18:10:51,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048696.1 (Enterobacter hormaechei, Eho-E4) loaded with 37 markers.\n",
      "2024-04-17 18:10:51,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049021.1 (Enterobacter hormaechei, Eho-E3) loaded with 37 markers.\n",
      "2024-04-17 18:10:51,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066094.1 (Enterobacter hormaechei, Eho-E1) loaded with 37 markers.\n",
      "2024-04-17 18:10:51,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047736.1 (Enterobacter hormaechei, Eho-5) loaded with 35 markers.\n",
      "2024-04-17 18:10:51,758 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047740.1 (Enterobacter hormaechei, Eho-4) loaded with 37 markers.\n",
      "2024-04-17 18:10:51,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047757.1 (Enterobacter hormaechei, Eho-2) loaded with 38 markers.\n",
      "2024-04-17 18:10:51,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS999206.1 (Enterobacter hormaechei, EC-TO80) loaded with 34 markers.\n",
      "2024-04-17 18:10:52,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043853.1 (Enterobacter hormaechei, EB_P6_L3_02.19) loaded with 34 markers.\n",
      "2024-04-17 18:10:52,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011650.1 (Enterobacter hormaechei, CAV1669) loaded with 35 markers.\n",
      "2024-04-17 18:10:52,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011584.1 (Enterobacter hormaechei, CAV1668) loaded with 35 markers.\n",
      "2024-04-17 18:10:52,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011581.1 (Enterobacter hormaechei, CAV1411) loaded with 36 markers.\n",
      "2024-04-17 18:10:52,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011572.1 (Enterobacter hormaechei, CAV1311) loaded with 35 markers.\n",
      "2024-04-17 18:10:52,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011662.1 (Enterobacter hormaechei, CAV1176) loaded with 35 markers.\n",
      "2024-04-17 18:10:52,444 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030347.1 (Enterobacter hormaechei, AR_038) loaded with 40 markers.\n",
      "2024-04-17 18:10:52,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085774.1 (Enterobacter hormaechei, 48947CZ) loaded with 33 markers.\n",
      "2024-04-17 18:10:52,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085780.1 (Enterobacter hormaechei, 48411CZ) loaded with 33 markers.\n",
      "2024-04-17 18:10:52,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010376.2 (Enterobacter hormaechei, 34977) loaded with 35 markers.\n",
      "2024-04-17 18:10:52,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031565.1 (Enterobacter hormaechei, 2013_1a) loaded with 38 markers.\n",
      "2024-04-17 18:10:52,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047965.1 (Enterobacter hormaechei, 189) loaded with 36 markers.\n",
      "2024-04-17 18:10:52,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049290.1 (Enterobacter hormaechei, 08-175) loaded with 32 markers.\n",
      "2024-04-17 18:10:53,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008823.1 (Enterobacter cloacae, ECNIH2) loaded with 35 markers.\n",
      "2024-04-17 18:10:53,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077659.1 (Enterobacter cloacae, ECL72) loaded with 36 markers.\n",
      "2024-04-17 18:10:53,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021749.1 (Enterobacter cloacae, AR_0163) loaded with 37 markers.\n",
      "2024-04-17 18:10:53,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029716.1 (Enterobacter cloacae, AR_0154) loaded with 38 markers.\n",
      "2024-04-17 18:10:53,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020053.1 (Enterobacter cloacae, AR_0065) loaded with 34 markers.\n",
      "2024-04-17 18:10:53,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026719.1 (Enterobacter cloacae, AR_0060) loaded with 35 markers.\n",
      "2024-04-17 18:10:53,571 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072945.1 (Enterobacter sp., MGH_7) loaded with 33 markers.\n",
      "2024-04-17 18:10:53,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072953.1 (Enterobacter sp., MGH_3) loaded with 32 markers.\n",
      "2024-04-17 18:10:53,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072961.1 (Enterobacter sp., MGH_14) loaded with 32 markers.\n",
      "2024-04-17 18:10:53,835 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056118.1 (Enterobacter sp., DSM_30060) loaded with 36 markers.\n",
      "2024-04-17 18:10:53,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024812.1 (Enterobacter sp., CRENT-193) loaded with 33 markers.\n",
      "2024-04-17 18:10:53,999 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072970.1 (Enterobacter sp., BWH_37) loaded with 33 markers.\n",
      "2024-04-17 18:10:54,070 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059711.1 (Enterobacter hormaechei, ZHH) loaded with 36 markers.\n",
      "2024-04-17 18:10:54,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049188.1 (Enterobacter hormaechei, Y323) loaded with 35 markers.\n",
      "2024-04-17 18:10:54,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049192.1 (Enterobacter hormaechei, Y2152) loaded with 35 markers.\n",
      "2024-04-17 18:10:54,384 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090796.1 (Enterobacter hormaechei, Y-2) loaded with 32 markers.\n",
      "2024-04-17 18:10:54,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043382.1 (Enterobacter hormaechei, WCHEX045001) loaded with 37 markers.\n",
      "2024-04-17 18:10:54,569 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036310.1 (Enterobacter hormaechei, WCHEH090011) loaded with 36 markers.\n",
      "2024-04-17 18:10:54,657 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031726.1 (Enterobacter hormaechei, WCHEH020038) loaded with 33 markers.\n",
      "2024-04-17 18:10:54,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073771.1 (Enterobacter hormaechei, SH19PTE2) loaded with 33 markers.\n",
      "2024-04-17 18:10:54,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035385.1 (Enterobacter hormaechei, S11_16) loaded with 34 markers.\n",
      "2024-04-17 18:10:54,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058172.1 (Enterobacter hormaechei, RHBSTW-00056) loaded with 39 markers.\n",
      "2024-04-17 18:10:55,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058191.1 (Enterobacter hormaechei, RHBSTW-00012) loaded with 46 markers.\n",
      "2024-04-17 18:10:55,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029246.1 (Enterobacter hormaechei, OSUVMCKPC4-2) loaded with 35 markers.\n",
      "2024-04-17 18:10:55,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024908.1 (Enterobacter hormaechei, OSUKPC4_L) loaded with 33 markers.\n",
      "2024-04-17 18:10:55,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066553.1 (Enterobacter hormaechei, OSUCZKPC4-140) loaded with 33 markers.\n",
      "2024-04-17 18:10:55,466 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061744.1 (Enterobacter hormaechei, NJGLYY90-CR) loaded with 31 markers.\n",
      "2024-04-17 18:10:55,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134164.1 (Enterobacter hormaechei, NCTC11571) loaded with 36 markers.\n",
      "2024-04-17 18:10:55,658 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065939.1 (Enterobacter hormaechei, MSB1_8J) loaded with 32 markers.\n",
      "2024-04-17 18:10:55,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090909.1 (Enterobacter hormaechei, M1) loaded with 33 markers.\n",
      "2024-04-17 18:10:55,894 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033102.1 (Enterobacter hormaechei, L51) loaded with 36 markers.\n",
      "2024-04-17 18:10:55,991 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054411.1 (Enterobacter hormaechei, KCJ3K19) loaded with 34 markers.\n",
      "2024-04-17 18:10:56,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054410.1 (Enterobacter hormaechei, KCJ3K13) loaded with 33 markers.\n",
      "2024-04-17 18:10:56,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064113.1 (Enterobacter hormaechei, IR5283) loaded with 33 markers.\n",
      "2024-04-17 18:10:56,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071876.1 (Enterobacter hormaechei, GX4-8L) loaded with 35 markers.\n",
      "2024-04-17 18:10:56,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077392.1 (Enterobacter hormaechei, FDAARGOS_1435) loaded with 43 markers.\n",
      "2024-04-17 18:10:56,354 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047765.1 (Enterobacter hormaechei, Eho-1) loaded with 34 markers.\n",
      "2024-04-17 18:10:56,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053190.1 (Enterobacter hormaechei, EGYMCRVIM) loaded with 33 markers.\n",
      "2024-04-17 18:10:56,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008905.1 (Enterobacter hormaechei, ECR091) loaded with 39 markers.\n",
      "2024-04-17 18:10:56,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008897.1 (Enterobacter hormaechei, ECNIH3) loaded with 39 markers.\n",
      "2024-04-17 18:10:56,621 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080470.1 (Enterobacter hormaechei, ECC59) loaded with 35 markers.\n",
      "2024-04-17 18:10:56,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053103.1 (Enterobacter hormaechei, Ec61) loaded with 33 markers.\n",
      "2024-04-17 18:10:56,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046271.1 (Enterobacter hormaechei, E70) loaded with 34 markers.\n",
      "2024-04-17 18:10:56,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017186.1 (Enterobacter hormaechei, DSM_14563) loaded with 31 markers.\n",
      "2024-04-17 18:10:56,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042488.1 (Enterobacter hormaechei, C15) loaded with 32 markers.\n",
      "2024-04-17 18:10:56,962 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045611.1 (Enterobacter hormaechei, AUH-ENM30) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,027 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085743.1 (Enterobacter hormaechei, 60214CZ) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085757.1 (Enterobacter hormaechei, 54569CZ) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021167.1 (Enterobacter hormaechei, 388) loaded with 35 markers.\n",
      "2024-04-17 18:10:57,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064656.1 (Enterobacter hormaechei, 3804) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012165.1 (Enterobacter hormaechei, 34978) loaded with 33 markers.\n",
      "2024-04-17 18:10:57,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010384.1 (Enterobacter hormaechei, 34399) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021162.1 (Enterobacter hormaechei, 234) loaded with 35 markers.\n",
      "2024-04-17 18:10:57,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030076.1 (Enterobacter hormaechei, 20710) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068287.1 (Enterobacter hormaechei, 1575) loaded with 33 markers.\n",
      "2024-04-17 18:10:57,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045312.1 (Enterobacter hormaechei, 14102KSN) loaded with 34 markers.\n",
      "2024-04-17 18:10:57,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009854.1 (Enterobacter cloacae, ECNIH5) loaded with 39 markers.\n",
      "2024-04-17 18:10:57,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077660.1 (Enterobacter cloacae, ECL78) loaded with 32 markers.\n",
      "2024-04-17 18:10:57,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091486.1 (Enterobacter cloacae, ECL411) loaded with 34 markers.\n",
      "2024-04-17 18:10:57,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091496.1 (Enterobacter cloacae, ECL404) loaded with 34 markers.\n",
      "2024-04-17 18:10:58,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077661.1 (Enterobacter cloacae, ECL112) loaded with 36 markers.\n",
      "2024-04-17 18:10:58,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033466.1 (Enterobacter cloacae, E3442) loaded with 31 markers.\n",
      "2024-04-17 18:10:58,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021902.1 (Enterobacter cloacae, AR_0136) loaded with 33 markers.\n",
      "2024-04-17 18:10:58,278 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027604.1 (Enterobacter cloacae, AR_0093) loaded with 32 markers.\n",
      "2024-04-17 18:10:58,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021776.1 (Enterobacter cloacae, AR_0053) loaded with 33 markers.\n",
      "2024-04-17 18:10:58,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018814.1 (Enterobacter cloacae, AR_0002) loaded with 33 markers.\n",
      "2024-04-17 18:10:58,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP052870.1 (Enterobacter cloacae, 3849) loaded with 36 markers.\n",
      "2024-04-17 18:10:58,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020528.1 (Enterobacter cloacae, 174) loaded with 37 markers.\n",
      "2024-04-17 18:10:58,730 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087880.1 (Pseudocitrobacter sp., G163CM) loaded with 37 markers.\n",
      "2024-04-17 18:10:58,826 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043438.2 (Enterobacter sp., LU1) loaded with 35 markers.\n",
      "2024-04-17 18:10:58,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087390.1 (Enterobacter sp., AN-K1) loaded with 36 markers.\n",
      "2024-04-17 18:10:58,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031571.1 (Enterobacter hormaechei, N1) loaded with 39 markers.\n",
      "2024-04-17 18:10:59,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041733.1 (Enterobacter hormaechei, ME-1) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019889.1 (Enterobacter hormaechei, FRM) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059422.1 (Enterobacter hormaechei, ENCL49790) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059414.1 (Enterobacter hormaechei, ENCL48212) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,401 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042566.1 (Enterobacter hormaechei, C44) loaded with 36 markers.\n",
      "2024-04-17 18:10:59,480 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042540.1 (Enterobacter hormaechei, C4) loaded with 36 markers.\n",
      "2024-04-17 18:10:59,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029126.1 (Enterobacter hormaechei, AR432) loaded with 37 markers.\n",
      "2024-04-17 18:10:59,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031574.1 (Enterobacter hormaechei, A1) loaded with 36 markers.\n",
      "2024-04-17 18:10:59,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085748.1 (Enterobacter hormaechei, 59732CZ) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085753.1 (Enterobacter hormaechei, 57816CZ) loaded with 32 markers.\n",
      "2024-04-17 18:10:59,962 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012167.1 (Enterobacter hormaechei, 34998) loaded with 36 markers.\n",
      "2024-04-17 18:11:00,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045450.1 (Enterobacter hormaechei, 14280161202) loaded with 36 markers.\n",
      "2024-04-17 18:11:00,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091319.1 (Enterobacter sp., DR25) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090192.1 (Enterobacter hormaechei, TUECL19003) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,409 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090146.1 (Enterobacter hormaechei, TUECL19001) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022510.1 (Enterobacter hormaechei, STN0717-64) loaded with 35 markers.\n",
      "2024-04-17 18:11:00,621 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028538.2 (Enterobacter hormaechei, SCEH020042) loaded with 34 markers.\n",
      "2024-04-17 18:11:00,723 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071009.1 (Enterobacter hormaechei, RIVM_C034192) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071011.1 (Enterobacter hormaechei, RIVM_C034177) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071013.1 (Enterobacter hormaechei, RIVM_C033144) loaded with 33 markers.\n",
      "2024-04-17 18:11:00,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071015.1 (Enterobacter hormaechei, RIVM_C019181) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,074 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071017.1 (Enterobacter hormaechei, RIVM_C019072) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071019.1 (Enterobacter hormaechei, RIVM_C017834) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071021.1 (Enterobacter hormaechei, RIVM_C017238) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071023.1 (Enterobacter hormaechei, RIVM_C012369) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071025.1 (Enterobacter hormaechei, RIVM_C010068) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058164.1 (Enterobacter hormaechei, RHBSTW-00070) loaded with 33 markers.\n",
      "2024-04-17 18:11:01,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058167.1 (Enterobacter hormaechei, RHBSTW-00059) loaded with 38 markers.\n",
      "2024-04-17 18:11:01,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058175.1 (Enterobacter hormaechei, RHBSTW-00040) loaded with 37 markers.\n",
      "2024-04-17 18:11:01,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058187.1 (Enterobacter hormaechei, RHBSTW-00016) loaded with 35 markers.\n",
      "2024-04-17 18:11:01,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030007.1 (Enterobacter hormaechei, Pb204) loaded with 35 markers.\n",
      "2024-04-17 18:11:02,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019817.1 (Enterobacter hormaechei, OIPH-N069) loaded with 33 markers.\n",
      "2024-04-17 18:11:02,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017183.1 (Enterobacter hormaechei, LMG27195) loaded with 34 markers.\n",
      "2024-04-17 18:11:02,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034754.1 (Enterobacter hormaechei, Eh1) loaded with 33 markers.\n",
      "2024-04-17 18:11:02,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070534.1 (Enterobacter hormaechei, EC48293) loaded with 35 markers.\n",
      "2024-04-17 18:11:02,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042571.1 (Enterobacter hormaechei, E5) loaded with 37 markers.\n",
      "2024-04-17 18:11:02,466 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042551.1 (Enterobacter hormaechei, C45) loaded with 36 markers.\n",
      "2024-04-17 18:11:02,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027142.1 (Enterobacter hormaechei, AR_0365) loaded with 34 markers.\n",
      "2024-04-17 18:11:02,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051132.1 (Enterobacter hormaechei, AMS-38) loaded with 35 markers.\n",
      "2024-04-17 18:11:02,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045451.1 (Enterobacter hormaechei, 14320055602) loaded with 31 markers.\n",
      "2024-04-17 18:11:02,755 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021896.1 (Enterobacter cloacae, AR_0050) loaded with 33 markers.\n",
      "2024-04-17 18:11:02,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020525.1 (Enterobacter cloacae, 109) loaded with 36 markers.\n",
      "2024-04-17 18:11:02,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053416.1 (Salmonella bongori, 85-0051) loaded with 32 markers.\n",
      "2024-04-17 18:11:02,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035676.1 (Salmonella bongori, 04-0440) loaded with 30 markers.\n",
      "2024-04-17 18:11:03,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022532.1 (Enterobacter hormaechei, MS7884A) loaded with 35 markers.\n",
      "2024-04-17 18:11:03,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077264.1 (Enterobacter hormaechei, FDAARGOS_1434) loaded with 37 markers.\n",
      "2024-04-17 18:11:03,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043766.1 (Enterobacter hormaechei, EB_P9_L5_03.19) loaded with 39 markers.\n",
      "2024-04-17 18:11:03,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017179.1 (Enterobacter hormaechei, DSM_16691) loaded with 37 markers.\n",
      "2024-04-17 18:11:03,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044335.1 (Enterobacter hormaechei, AKB48) loaded with 36 markers.\n",
      "2024-04-17 18:11:03,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074592.1 (Salmonella bongori, CFSAN001045) loaded with 30 markers.\n",
      "2024-04-17 18:11:03,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091779.1 (Enterobacter mori, ACYC.E9L) loaded with 34 markers.\n",
      "2024-04-17 18:11:03,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083403.1 (Enterobacter bugandensis, XL95) loaded with 36 markers.\n",
      "2024-04-17 18:11:03,749 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083620.1 (Enterobacter bugandensis, FDAARGOS_1520) loaded with 36 markers.\n",
      "2024-04-17 18:11:03,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083641.1 (Enterobacter bugandensis, FDAARGOS_1497) loaded with 36 markers.\n",
      "2024-04-17 18:11:03,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083643.1 (Enterobacter bugandensis, FDAARGOS_1496) loaded with 36 markers.\n",
      "2024-04-17 18:11:04,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077206.1 (Enterobacter bugandensis, FDAARGOS_1427) loaded with 36 markers.\n",
      "2024-04-17 18:11:04,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT992502.1 (Enterobacter bugandensis, EB-247) loaded with 36 markers.\n",
      "2024-04-17 18:11:04,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056395.1 (Citrobacter freundii, RHBSTW-00589) loaded with 44 markers.\n",
      "2024-04-17 18:11:04,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056473.1 (Citrobacter freundii, RHBSTW-00477) loaded with 41 markers.\n",
      "2024-04-17 18:11:04,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056967.1 (Citrobacter freundii, RHB41-C16) loaded with 39 markers.\n",
      "2024-04-17 18:11:04,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057353.1 (Citrobacter freundii, RHB29-C07) loaded with 40 markers.\n",
      "2024-04-17 18:11:04,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006692.1 (Salmonella bongori, RKS3044) loaded with 34 markers.\n",
      "2024-04-17 18:11:04,658 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053417.1 (Salmonella bongori, 92-0238) loaded with 34 markers.\n",
      "2024-04-17 18:11:04,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039453.1 (Enterobacter bugandensis, 220) loaded with 32 markers.\n",
      "2024-04-17 18:11:04,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039452.1 (Enterobacter bugandensis, 1367) loaded with 35 markers.\n",
      "2024-04-17 18:11:04,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073043.1 (Citrobacter sp., MGH103) loaded with 55 markers.\n",
      "2024-04-17 18:11:04,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024683.1 (Citrobacter freundii, UMH13) loaded with 48 markers.\n",
      "2024-04-17 18:11:05,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056505.1 (Citrobacter freundii, RHBSTW-00449) loaded with 46 markers.\n",
      "2024-04-17 18:11:05,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056527.1 (Citrobacter freundii, RHBSTW-00435) loaded with 58 markers.\n",
      "2024-04-17 18:11:05,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056839.1 (Citrobacter freundii, RHBSTW-00126) loaded with 48 markers.\n",
      "2024-04-17 18:11:05,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056852.1 (Citrobacter freundii, RHBSTW-00119) loaded with 44 markers.\n",
      "2024-04-17 18:11:05,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057867.1 (Citrobacter freundii, RHB12-C12) loaded with 42 markers.\n",
      "2024-04-17 18:11:05,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040698.1 (Citrobacter freundii, R47) loaded with 43 markers.\n",
      "2024-04-17 18:11:05,738 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058318.1 (Citrobacter freundii, M92) loaded with 47 markers.\n",
      "2024-04-17 18:11:05,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024672.1 (Citrobacter freundii, HM38) loaded with 41 markers.\n",
      "2024-04-17 18:11:05,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026056.1 (Citrobacter freundii, FDAARGOS_73) loaded with 46 markers.\n",
      "2024-04-17 18:11:05,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024881.1 (Citrobacter freundii, AR_0022) loaded with 44 markers.\n",
      "2024-04-17 18:11:06,076 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046502.1 (Citrobacter freundii, 111) loaded with 49 markers.\n",
      "2024-04-17 18:11:06,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050506.1 (Enterobacter hormaechei, CM18-242-2) loaded with 34 markers.\n",
      "2024-04-17 18:11:06,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050311.1 (Enterobacter hormaechei, CM18-216) loaded with 34 markers.\n",
      "2024-04-17 18:11:06,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT991954.1 (Enterobacter hormaechei, C309) loaded with 35 markers.\n",
      "2024-04-17 18:11:06,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049147.1 (Cronobacter sakazakii, C767) loaded with 30 markers.\n",
      "2024-04-17 18:11:06,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049150.1 (Cronobacter sakazakii, C757) loaded with 30 markers.\n",
      "2024-04-17 18:11:06,474 [DEBUG - chronostrain.database.parser.json] - Strain NC_009778.1 (Cronobacter sakazakii, ATCC_BAA-894) loaded with 29 markers.\n",
      "2024-04-17 18:11:06,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059045.1 (Cronobacter sakazakii, 5563_17) loaded with 29 markers.\n",
      "2024-04-17 18:11:06,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056229.1 (Citrobacter freundii, RHBSTW-00968) loaded with 36 markers.\n",
      "2024-04-17 18:11:06,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056333.1 (Citrobacter freundii, RHBSTW-00714) loaded with 45 markers.\n",
      "2024-04-17 18:11:06,781 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056515.1 (Citrobacter freundii, RHBSTW-00444) loaded with 40 markers.\n",
      "2024-04-17 18:11:06,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056559.1 (Citrobacter freundii, RHBSTW-00408) loaded with 36 markers.\n",
      "2024-04-17 18:11:06,963 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055538.1 (Citrobacter freundii, RHBSTW-00398) loaded with 44 markers.\n",
      "2024-04-17 18:11:07,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056597.1 (Citrobacter freundii, RHBSTW-00334) loaded with 44 markers.\n",
      "2024-04-17 18:11:07,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056850.1 (Citrobacter freundii, RHBSTW-00120) loaded with 49 markers.\n",
      "2024-04-17 18:11:07,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056910.1 (Citrobacter freundii, RHBSTW-00006) loaded with 64 markers.\n",
      "2024-04-17 18:11:07,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057506.1 (Citrobacter freundii, RHB24-C05) loaded with 40 markers.\n",
      "2024-04-17 18:11:07,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057509.1 (Citrobacter freundii, RHB24-C03) loaded with 40 markers.\n",
      "2024-04-17 18:11:07,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034769.1 (Enterobacter sp., N18-03635) loaded with 33 markers.\n",
      "2024-04-17 18:11:07,776 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020817.1 (Enterobacter sp., Crenshaw) loaded with 35 markers.\n",
      "2024-04-17 18:11:07,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058253.1 (Enterobacter roggenkampii, ED5) loaded with 37 markers.\n",
      "2024-04-17 18:11:07,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055276.1 (Enterobacter mori, CX01) loaded with 34 markers.\n",
      "2024-04-17 18:11:08,054 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022274.1 (Enterobacter cloacae, WP8-W19-CRE-02) loaded with 33 markers.\n",
      "2024-04-17 18:11:08,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037919.1 (Enterobacter cloacae, N13-01531) loaded with 37 markers.\n",
      "2024-04-17 18:11:08,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053568.1 (Enterobacter cloacae, MY490) loaded with 34 markers.\n",
      "2024-04-17 18:11:08,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077211.1 (Enterobacter cloacae, FDAARGOS_1431) loaded with 34 markers.\n",
      "2024-04-17 18:11:08,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056776.1 (Enterobacter cloacae, DSM_30054) loaded with 34 markers.\n",
      "2024-04-17 18:11:08,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073310.1 (Enterobacter cloacae, CZ862) loaded with 81 markers.\n",
      "2024-04-17 18:11:08,748 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092042.1 (Enterobacter cloacae, AVS0889) loaded with 35 markers.\n",
      "2024-04-17 18:11:08,830 [DEBUG - chronostrain.database.parser.json] - Strain NC_014121.1 (Enterobacter cloacae, ATCC_13047) loaded with 34 markers.\n",
      "2024-04-17 18:11:08,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022508.1 (Enterobacter bugandensis, STN0717-56) loaded with 32 markers.\n",
      "2024-04-17 18:11:08,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024498.1 (Enterobacter asburiae, En30) loaded with 35 markers.\n",
      "2024-04-17 18:11:09,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019630.1 (Enterobacter asburiae, 17Nkhm-UP2) loaded with 35 markers.\n",
      "2024-04-17 18:11:09,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047606.1 (Citrobacter sp., LUTT5) loaded with 44 markers.\n",
      "2024-04-17 18:11:09,224 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041051.1 (Citrobacter sp., CF971) loaded with 51 markers.\n",
      "2024-04-17 18:11:09,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036435.1 (Citrobacter sp., ABFQG) loaded with 41 markers.\n",
      "2024-04-17 18:11:09,376 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134214.1 (Citrobacter portucalensis, NCTC11104) loaded with 44 markers.\n",
      "2024-04-17 18:11:09,458 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055247.1 (Citrobacter freundii, ZY198) loaded with 44 markers.\n",
      "2024-04-17 18:11:09,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045726.1 (Citrobacter freundii, zone4) loaded with 44 markers.\n",
      "2024-04-17 18:11:09,657 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016952.1 (Citrobacter freundii, SL151) loaded with 49 markers.\n",
      "2024-04-17 18:11:09,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056208.1 (Citrobacter freundii, RHBSTW-00985) loaded with 68 markers.\n",
      "2024-04-17 18:11:09,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056235.1 (Citrobacter freundii, RHBSTW-00942) loaded with 60 markers.\n",
      "2024-04-17 18:11:09,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056238.1 (Citrobacter freundii, RHBSTW-00935) loaded with 45 markers.\n",
      "2024-04-17 18:11:10,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056245.1 (Citrobacter freundii, RHBSTW-00915) loaded with 44 markers.\n",
      "2024-04-17 18:11:10,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056256.1 (Citrobacter freundii, RHBSTW-00902) loaded with 45 markers.\n",
      "2024-04-17 18:11:10,269 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055421.1 (Citrobacter freundii, RHBSTW-00862) loaded with 76 markers.\n",
      "2024-04-17 18:11:10,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056336.1 (Citrobacter freundii, RHBSTW-00697) loaded with 49 markers.\n",
      "2024-04-17 18:11:10,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056365.1 (Citrobacter freundii, RHBSTW-00658) loaded with 43 markers.\n",
      "2024-04-17 18:11:10,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056451.1 (Citrobacter freundii, RHBSTW-00502) loaded with 45 markers.\n",
      "2024-04-17 18:11:10,784 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056466.1 (Citrobacter freundii, RHBSTW-00486) loaded with 74 markers.\n",
      "2024-04-17 18:11:10,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056635.1 (Citrobacter freundii, RHBSTW-00302) loaded with 88 markers.\n",
      "2024-04-17 18:11:11,083 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056653.1 (Citrobacter freundii, RHBSTW-00214) loaded with 41 markers.\n",
      "2024-04-17 18:11:11,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056809.1 (Citrobacter freundii, RHBSTW-00162) loaded with 45 markers.\n",
      "2024-04-17 18:11:11,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056827.1 (Citrobacter freundii, RHBSTW-00135) loaded with 46 markers.\n",
      "2024-04-17 18:11:11,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055582.1 (Citrobacter freundii, RHBSTW-00110) loaded with 75 markers.\n",
      "2024-04-17 18:11:11,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055588.1 (Citrobacter freundii, RHBSTW-00084) loaded with 75 markers.\n",
      "2024-04-17 18:11:11,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057864.1 (Citrobacter freundii, RHB12-C19) loaded with 40 markers.\n",
      "2024-04-17 18:11:11,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057866.1 (Citrobacter freundii, RHB12-C18) loaded with 45 markers.\n",
      "2024-04-17 18:11:11,851 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035276.1 (Citrobacter freundii, R17) loaded with 45 markers.\n",
      "2024-04-17 18:11:11,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR881934.1 (Citrobacter freundii, PSV) loaded with 46 markers.\n",
      "2024-04-17 18:11:12,038 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047279.1 (Citrobacter freundii, N18-04128) loaded with 43 markers.\n",
      "2024-04-17 18:11:12,126 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047275.1 (Citrobacter freundii, N18-04085) loaded with 43 markers.\n",
      "2024-04-17 18:11:12,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047273.1 (Citrobacter freundii, N18-04078) loaded with 45 markers.\n",
      "2024-04-17 18:11:12,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047269.1 (Citrobacter freundii, N16-03880) loaded with 43 markers.\n",
      "2024-04-17 18:11:12,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890181.1 (Citrobacter freundii, MSB1_1H-sc-2280393) loaded with 44 markers.\n",
      "2024-04-17 18:11:12,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047307.1 (Citrobacter freundii, L75) loaded with 46 markers.\n",
      "2024-04-17 18:11:12,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054278.1 (Citrobacter freundii, IDR1900015725-01-02) loaded with 42 markers.\n",
      "2024-04-17 18:11:12,646 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054294.1 (Citrobacter freundii, IDR1800045912-01-00) loaded with 42 markers.\n",
      "2024-04-17 18:11:12,743 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042534.1 (Citrobacter freundii, E51) loaded with 48 markers.\n",
      "2024-04-17 18:11:12,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042517.1 (Citrobacter freundii, E33) loaded with 45 markers.\n",
      "2024-04-17 18:11:12,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042524.1 (Citrobacter freundii, E11) loaded with 47 markers.\n",
      "2024-04-17 18:11:13,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086287.1 (Citrobacter freundii, DY2010) loaded with 42 markers.\n",
      "2024-04-17 18:11:13,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992175.1 (Citrobacter freundii, Citrobacter_freundii_str._E2614) loaded with 42 markers.\n",
      "2024-04-17 18:11:13,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048416.1 (Citrobacter freundii, CitB) loaded with 41 markers.\n",
      "2024-04-17 18:11:13,281 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059427.1 (Citrobacter freundii, CIFR51929) loaded with 51 markers.\n",
      "2024-04-17 18:11:13,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026231.1 (Citrobacter freundii, CFNIH4) loaded with 58 markers.\n",
      "2024-04-17 18:11:13,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070559.1 (Citrobacter freundii, CF50935) loaded with 43 markers.\n",
      "2024-04-17 18:11:13,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070549.1 (Citrobacter freundii, CF49969) loaded with 45 markers.\n",
      "2024-04-17 18:11:13,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070544.1 (Citrobacter freundii, CF48846) loaded with 43 markers.\n",
      "2024-04-17 18:11:13,720 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037734.1 (Citrobacter freundii, CAV1857) loaded with 42 markers.\n",
      "2024-04-17 18:11:13,849 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011657.1 (Citrobacter freundii, CAV1741) loaded with 42 markers.\n",
      "2024-04-17 18:11:13,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011612.1 (Citrobacter freundii, CAV1321) loaded with 42 markers.\n",
      "2024-04-17 18:11:14,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042478.1 (Citrobacter freundii, C50) loaded with 45 markers.\n",
      "2024-04-17 18:11:14,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016762.1 (Citrobacter freundii, B38) loaded with 44 markers.\n",
      "2024-04-17 18:11:14,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049015.1 (Citrobacter freundii, ATCC_8090) loaded with 46 markers.\n",
      "2024-04-17 18:11:14,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032184.1 (Citrobacter freundii, AR_0116) loaded with 48 markers.\n",
      "2024-04-17 18:11:14,459 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085726.1 (Citrobacter freundii, 56415CZ) loaded with 52 markers.\n",
      "2024-04-17 18:11:14,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022273.1 (Citrobacter freundii, 18-1) loaded with 50 markers.\n",
      "2024-04-17 18:11:14,635 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056132.1 (Enterobacter hormaechei, RHBSTW-00916) loaded with 35 markers.\n",
      "2024-04-17 18:11:14,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056707.1 (Enterobacter hormaechei, RHBSTW-00564) loaded with 37 markers.\n",
      "2024-04-17 18:11:14,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056742.1 (Enterobacter hormaechei, RHBSTW-00220) loaded with 31 markers.\n",
      "2024-04-17 18:11:14,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056747.1 (Enterobacter hormaechei, RHBSTW-00217) loaded with 32 markers.\n",
      "2024-04-17 18:11:14,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056751.1 (Enterobacter hormaechei, RHBSTW-00216) loaded with 31 markers.\n",
      "2024-04-17 18:11:15,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056756.1 (Enterobacter hormaechei, RHBSTW-00198) loaded with 31 markers.\n",
      "2024-04-17 18:11:15,175 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058153.1 (Enterobacter hormaechei, RHBSTW-00105) loaded with 55 markers.\n",
      "2024-04-17 18:11:15,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058160.1 (Enterobacter hormaechei, RHBSTW-00083) loaded with 54 markers.\n",
      "2024-04-17 18:11:15,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083613.1 (Enterobacter hormaechei, FDAARGOS_1533) loaded with 31 markers.\n",
      "2024-04-17 18:11:15,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077308.1 (Enterobacter hormaechei, FDAARGOS_1433) loaded with 35 markers.\n",
      "2024-04-17 18:11:15,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047715.1 (Enterobacter hormaechei, Eho-E2) loaded with 37 markers.\n",
      "2024-04-17 18:11:15,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066101.1 (Enterobacter hormaechei, Eho-8) loaded with 35 markers.\n",
      "2024-04-17 18:11:15,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066108.1 (Enterobacter hormaechei, Eho-7) loaded with 35 markers.\n",
      "2024-04-17 18:11:15,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047729.1 (Enterobacter hormaechei, Eho-6) loaded with 35 markers.\n",
      "2024-04-17 18:11:16,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017180.1 (Enterobacter hormaechei, DSM_16687) loaded with 31 markers.\n",
      "2024-04-17 18:11:16,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027111.1 (Enterobacter hormaechei, BW) loaded with 32 markers.\n",
      "2024-04-17 18:11:16,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023569.1 (Enterobacter hormaechei, BW) loaded with 32 markers.\n",
      "2024-04-17 18:11:16,310 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085767.1 (Enterobacter hormaechei, 49589CZ) loaded with 31 markers.\n",
      "2024-04-17 18:11:16,415 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010377.1 (Enterobacter hormaechei, 34983) loaded with 36 markers.\n",
      "2024-04-17 18:11:16,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026850.1 (Enterobacter cloacae, AR_0072) loaded with 31 markers.\n",
      "2024-04-17 18:11:16,565 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013940.1 (Cronobacter malonaticus, LMG_23826) loaded with 25 markers.\n",
      "2024-04-17 18:11:16,636 [DEBUG - chronostrain.database.parser.json] - Strain NC_023032.1 (Cronobacter malonaticus, CMCC45402) loaded with 26 markers.\n",
      "2024-04-17 18:11:16,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056244.1 (Citrobacter freundii, RHBSTW-00923) loaded with 40 markers.\n",
      "2024-04-17 18:11:16,844 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056644.1 (Citrobacter freundii, RHBSTW-00267) loaded with 63 markers.\n",
      "2024-04-17 18:11:16,945 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057258.1 (Citrobacter freundii, RHB31-C16) loaded with 40 markers.\n",
      "2024-04-17 18:11:17,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082360.1 (Klebsiella sp., CTHL.F3a) loaded with 21 markers.\n",
      "2024-04-17 18:11:17,135 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064784.1 (Klebsiella sp., BDA134-6) loaded with 19 markers.\n",
      "2024-04-17 18:11:17,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089403.1 (Klebsiella pasteurii, Kox205) loaded with 20 markers.\n",
      "2024-04-17 18:11:17,338 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073236.1 (Klebsiella michiganensis, Sb-24) loaded with 19 markers.\n",
      "2024-04-17 18:11:17,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074159.1 (Enterobacter sp., JBIWA005) loaded with 34 markers.\n",
      "2024-04-17 18:11:17,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050520.1 (Enterobacter sp., DNB-S2) loaded with 41 markers.\n",
      "2024-04-17 18:11:17,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027986.1 (Enterobacter sichuanensis, SGAir0282) loaded with 34 markers.\n",
      "2024-04-17 18:11:17,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022465.1 (Enterobacter roggenkampii, STW0522-66) loaded with 70 markers.\n",
      "2024-04-17 18:11:17,805 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019839.1 (Enterobacter roggenkampii, R11) loaded with 52 markers.\n",
      "2024-04-17 18:11:17,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086405.1 (Enterobacter roggenkampii, POL1) loaded with 52 markers.\n",
      "2024-04-17 18:11:18,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022148.1 (Enterobacter roggenkampii, 704SK10) loaded with 47 markers.\n",
      "2024-04-17 18:11:18,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019007.1 (Enterobacter oligotrophicus, CCA6) loaded with 35 markers.\n",
      "2024-04-17 18:11:18,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084692.1 (Enterobacter mori, BC01) loaded with 33 markers.\n",
      "2024-04-17 18:11:18,257 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071063.1 (Enterobacter mori, 08-091) loaded with 32 markers.\n",
      "2024-04-17 18:11:18,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022228.1 (Enterobacter cloacae, WP8-W18-CRE-04) loaded with 38 markers.\n",
      "2024-04-17 18:11:18,491 [DEBUG - chronostrain.database.parser.json] - Strain NC_018079.1 (Enterobacter cloacae, SDM) loaded with 34 markers.\n",
      "2024-04-17 18:11:18,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086408.1 (Enterobacter cloacae, POL9) loaded with 35 markers.\n",
      "2024-04-17 18:11:18,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009756.1 (Enterobacter cloacae, GGT036) loaded with 33 markers.\n",
      "2024-04-17 18:11:18,763 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035633.1 (Enterobacter cloacae, EN3600) loaded with 40 markers.\n",
      "2024-04-17 18:11:18,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085080.1 (Enterobacter cloacae, ECl_1.VN) loaded with 35 markers.\n",
      "2024-04-17 18:11:19,013 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091492.1 (Enterobacter cloacae, ECL405) loaded with 45 markers.\n",
      "2024-04-17 18:11:19,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083709.1 (Enterobacter cloacae, ECL352) loaded with 39 markers.\n",
      "2024-04-17 18:11:19,166 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046116.1 (Enterobacter cloacae, CBG15936) loaded with 34 markers.\n",
      "2024-04-17 18:11:19,318 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032291.1 (Enterobacter cloacae, AR_0073) loaded with 61 markers.\n",
      "2024-04-17 18:11:19,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080107.1 (Enterobacter asburiae, WW-19C) loaded with 43 markers.\n",
      "2024-04-17 18:11:19,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056716.1 (Enterobacter asburiae, RHBSTW-00516) loaded with 39 markers.\n",
      "2024-04-17 18:11:19,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022399.1 (Citrobacter portucalensis, STW0522-30) loaded with 45 markers.\n",
      "2024-04-17 18:11:19,731 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022394.1 (Citrobacter portucalensis, STW0522-27) loaded with 39 markers.\n",
      "2024-04-17 18:11:19,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022389.1 (Citrobacter portucalensis, STW0522-26) loaded with 40 markers.\n",
      "2024-04-17 18:11:19,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022513.1 (Citrobacter portucalensis, STN0717-72) loaded with 40 markers.\n",
      "2024-04-17 18:11:20,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022494.1 (Citrobacter portucalensis, STN0717-36) loaded with 44 markers.\n",
      "2024-04-17 18:11:20,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022486.1 (Citrobacter portucalensis, STN0717-27) loaded with 46 markers.\n",
      "2024-04-17 18:11:20,219 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050078.1 (Citrobacter koseri, NJ) loaded with 42 markers.\n",
      "2024-04-17 18:11:20,298 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044097.1 (Citrobacter koseri, FDAARGOS_646) loaded with 39 markers.\n",
      "2024-04-17 18:11:20,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033780.1 (Citrobacter koseri, FDAARGOS_530) loaded with 39 markers.\n",
      "2024-04-17 18:11:20,470 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024677.1 (Citrobacter freundii, UMH16) loaded with 45 markers.\n",
      "2024-04-17 18:11:20,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056314.1 (Citrobacter freundii, RHBSTW-00830) loaded with 43 markers.\n",
      "2024-04-17 18:11:20,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057824.1 (Citrobacter freundii, RHB14-C12) loaded with 39 markers.\n",
      "2024-04-17 18:11:20,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078020.1 (Citrobacter freundii, CF_324) loaded with 65 markers.\n",
      "2024-04-17 18:11:20,837 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047773.1 (Citrobacter freundii, Cfr-13) loaded with 46 markers.\n",
      "2024-04-17 18:11:20,907 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045485.1 (Enterobacter cloacae, 88701) loaded with 35 markers.\n",
      "2024-04-17 18:11:20,982 [DEBUG - chronostrain.database.parser.json] - Strain NC_017933.1 (Cronobacter sakazakii, ES15) loaded with 33 markers.\n",
      "2024-04-17 18:11:21,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049144.1 (Cronobacter sakazakii, C79) loaded with 30 markers.\n",
      "2024-04-17 18:11:21,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR699014.1 (Citrobacter werkmanii, MGYG-HGUT-02535) loaded with 43 markers.\n",
      "2024-04-17 18:11:21,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023504.1 (Citrobacter werkmanii, FDAARGOS_364) loaded with 43 markers.\n",
      "2024-04-17 18:11:21,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038856.1 (Citrobacter freundii, Upstream_1) loaded with 40 markers.\n",
      "2024-04-17 18:11:21,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056586.1 (Citrobacter freundii, RHBSTW-00355) loaded with 49 markers.\n",
      "2024-04-17 18:11:21,533 [DEBUG - chronostrain.database.parser.json] - Strain NC_011283.1 (Klebsiella variicola, 342) loaded with 16 markers.\n",
      "2024-04-17 18:11:21,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033824.1 (Klebsiella sp., FDAARGOS_511) loaded with 19 markers.\n",
      "2024-04-17 18:11:21,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067093.1 (Klebsiella michiganensis, B106) loaded with 20 markers.\n",
      "2024-04-17 18:11:21,823 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019113.1 (Enterobacter sp., SA187) loaded with 32 markers.\n",
      "2024-04-17 18:11:21,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076838.1 (Leclercia sp., 4-9-1-25) loaded with 27 markers.\n",
      "2024-04-17 18:11:21,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071383.1 (Leclercia sp., 4-9-1-25) loaded with 27 markers.\n",
      "2024-04-17 18:11:22,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056682.1 (Klebsiella grimontii, RHBSTW-00634) loaded with 29 markers.\n",
      "2024-04-17 18:11:22,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045064.2 (Enterobacter roggenkampii, WCHER090065) loaded with 44 markers.\n",
      "2024-04-17 18:11:22,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056129.1 (Enterobacter roggenkampii, RHBSTW-00925) loaded with 78 markers.\n",
      "2024-04-17 18:11:22,425 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056168.1 (Enterobacter roggenkampii, RHBSTW-00695) loaded with 43 markers.\n",
      "2024-04-17 18:11:22,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023447.1 (Enterobacter roggenkampii, OIPH-N260) loaded with 70 markers.\n",
      "2024-04-17 18:11:22,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033800.1 (Enterobacter roggenkampii, FDAARGOS_523) loaded with 41 markers.\n",
      "2024-04-17 18:11:22,775 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060737.1 (Enterobacter roggenkampii, enterobacter_cloacae) loaded with 69 markers.\n",
      "2024-04-17 18:11:22,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032916.1 (Enterobacter roggenkampii, ECY546) loaded with 35 markers.\n",
      "2024-04-17 18:11:22,951 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048650.1 (Enterobacter roggenkampii, Ecl_20_981) loaded with 43 markers.\n",
      "2024-04-17 18:11:23,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038471.1 (Enterobacter roggenkampii, BP10374) loaded with 33 markers.\n",
      "2024-04-17 18:11:23,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012162.1 (Enterobacter roggenkampii, 35734) loaded with 41 markers.\n",
      "2024-04-17 18:11:23,252 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017475.1 (Enterobacter cloacae, M12X01451) loaded with 34 markers.\n",
      "2024-04-17 18:11:23,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021851.1 (Enterobacter cloacae, A1137) loaded with 40 markers.\n",
      "2024-04-17 18:11:23,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045487.1 (Enterobacter cloacae, 14240244101) loaded with 33 markers.\n",
      "2024-04-17 18:11:23,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056126.1 (Enterobacter asburiae, RHBSTW-01009) loaded with 49 markers.\n",
      "2024-04-17 18:11:23,601 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074584.1 (Enterobacter asburiae, L1_delta-T1RM) loaded with 37 markers.\n",
      "2024-04-17 18:11:23,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007546.1 (Enterobacter asburiae, L1) loaded with 37 markers.\n",
      "2024-04-17 18:11:23,773 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074154.1 (Enterobacter asburiae, JBIWA007) loaded with 35 markers.\n",
      "2024-04-17 18:11:23,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065693.1 (Enterobacter asburiae, FDAARGOS_892) loaded with 62 markers.\n",
      "2024-04-17 18:11:23,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011863.1 (Enterobacter asburiae, ATCC_35953) loaded with 62 markers.\n",
      "2024-04-17 18:11:24,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021963.1 (Citrobacter youngae, L6) loaded with 47 markers.\n",
      "2024-04-17 18:11:24,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040234.1 (Citrobacter sp., TBCP-5362) loaded with 39 markers.\n",
      "2024-04-17 18:11:24,251 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073048.1 (Citrobacter sp., MGH105) loaded with 42 markers.\n",
      "2024-04-17 18:11:24,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073007.1 (Citrobacter sp., BIDMC108) loaded with 45 markers.\n",
      "2024-04-17 18:11:24,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069801.1 (Citrobacter sp., B72) loaded with 42 markers.\n",
      "2024-04-17 18:11:24,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043009.1 (Citrobacter portucalensis, WCHCP085039) loaded with 43 markers.\n",
      "2024-04-17 18:11:24,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055089.1 (Citrobacter portucalensis, SWHIN_111) loaded with 45 markers.\n",
      "2024-04-17 18:11:24,709 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092442.1 (Citrobacter portucalensis, Sb-2) loaded with 43 markers.\n",
      "2024-04-17 18:11:24,784 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048388.1 (Citrobacter portucalensis, PNUCL1) loaded with 43 markers.\n",
      "2024-04-17 18:11:24,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089316.1 (Citrobacter portucalensis, K218) loaded with 41 markers.\n",
      "2024-04-17 18:11:24,956 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046348.1 (Citrobacter portucalensis, FDAARGOS_738) loaded with 42 markers.\n",
      "2024-04-17 18:11:25,036 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044098.1 (Citrobacter portucalensis, FDAARGOS_617) loaded with 46 markers.\n",
      "2024-04-17 18:11:25,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039327.1 (Citrobacter portucalensis, Effluent_1) loaded with 56 markers.\n",
      "2024-04-17 18:11:25,238 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054129.1 (Citrobacter portucalensis, BY9F36) loaded with 47 markers.\n",
      "2024-04-17 18:11:25,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091809.1 (Citrobacter portucalensis, 56322CZ) loaded with 45 markers.\n",
      "2024-04-17 18:11:25,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023452.1 (Citrobacter koseri, MPUCK001) loaded with 45 markers.\n",
      "2024-04-17 18:11:25,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026697.1 (Citrobacter koseri, AR_0025) loaded with 46 markers.\n",
      "2024-04-17 18:11:25,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056181.1 (Citrobacter freundii, RHBSTW-01016) loaded with 48 markers.\n",
      "2024-04-17 18:11:25,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056289.1 (Citrobacter freundii, RHBSTW-00858) loaded with 51 markers.\n",
      "2024-04-17 18:11:25,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056293.1 (Citrobacter freundii, RHBSTW-00857) loaded with 51 markers.\n",
      "2024-04-17 18:11:25,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056573.1 (Citrobacter freundii, RHBSTW-00370) loaded with 45 markers.\n",
      "2024-04-17 18:11:26,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056622.1 (Citrobacter freundii, RHBSTW-00310) loaded with 44 markers.\n",
      "2024-04-17 18:11:26,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057319.1 (Citrobacter freundii, RHB30-C04) loaded with 45 markers.\n",
      "2024-04-17 18:11:26,230 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057863.1 (Citrobacter freundii, RHB12-C20) loaded with 42 markers.\n",
      "2024-04-17 18:11:26,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057868.1 (Citrobacter freundii, RHB12-C05) loaded with 47 markers.\n",
      "2024-04-17 18:11:26,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060654.1 (Citrobacter freundii, MGH283) loaded with 42 markers.\n",
      "2024-04-17 18:11:26,516 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060659.1 (Citrobacter freundii, MGH281C) loaded with 42 markers.\n",
      "2024-04-17 18:11:26,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060662.1 (Citrobacter freundii, MGH281) loaded with 42 markers.\n",
      "2024-04-17 18:11:26,693 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060650.1 (Citrobacter freundii, MGH279) loaded with 42 markers.\n",
      "2024-04-17 18:11:26,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022151.1 (Citrobacter freundii, 705SK3) loaded with 50 markers.\n",
      "2024-04-17 18:11:26,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056779.1 (Enterobacter cloacae, D41-sc-1712200) loaded with 30 markers.\n",
      "2024-04-17 18:11:26,955 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046618.1 (Enterobacter asburiae, AEB30) loaded with 34 markers.\n",
      "2024-04-17 18:11:27,021 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078110.1 (Cronobacter sakazakii, MOD1-H322) loaded with 35 markers.\n",
      "2024-04-17 18:11:27,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034697.1 (Salmonella enterica, RSE42) loaded with 42 markers.\n",
      "2024-04-17 18:11:27,185 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034717.1 (Salmonella enterica, RSE09) loaded with 42 markers.\n",
      "2024-04-17 18:11:27,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054254.1 (Klebsiella variicola, FH-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:27,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047604.1 (Klebsiella sp., MPUS7) loaded with 22 markers.\n",
      "2024-04-17 18:11:27,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020657.1 (Klebsiella sp., M5al) loaded with 22 markers.\n",
      "2024-04-17 18:11:27,527 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017994.1 (Klebsiella pneumoniae, P1428) loaded with 20 markers.\n",
      "2024-04-17 18:11:27,643 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036175.1 (Klebsiella huaxiensis, WCHKl090001) loaded with 28 markers.\n",
      "2024-04-17 18:11:27,747 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044527.1 (Klebsiella grimontii, SS141) loaded with 26 markers.\n",
      "2024-04-17 18:11:27,863 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056150.1 (Klebsiella grimontii, RHBSTW-00853) loaded with 26 markers.\n",
      "2024-04-17 18:11:27,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056701.1 (Klebsiella grimontii, RHBSTW-00577) loaded with 26 markers.\n",
      "2024-04-17 18:11:28,048 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055315.1 (Klebsiella grimontii, RHBSTW-00555) loaded with 30 markers.\n",
      "2024-04-17 18:11:28,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056760.1 (Klebsiella grimontii, RHBSTW-00186) loaded with 24 markers.\n",
      "2024-04-17 18:11:28,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058112.1 (Klebsiella grimontii, RHBSTW-00181) loaded with 24 markers.\n",
      "2024-04-17 18:11:28,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055412.1 (Klebsiella grimontii, RHBSTW-00054) loaded with 55 markers.\n",
      "2024-04-17 18:11:28,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058180.1 (Klebsiella grimontii, RHBSTW-00039) loaded with 40 markers.\n",
      "2024-04-17 18:11:28,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR594038.1 (Klebsiella grimontii, NCTC9146_substr._serovar_casular_type_26_substr._serovar_casular_type_26) loaded with 20 markers.\n",
      "2024-04-17 18:11:28,667 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063301.1 (Klebsiella grimontii, M5a1) loaded with 22 markers.\n",
      "2024-04-17 18:11:28,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067391.1 (Klebsiella grimontii, 5512.56) loaded with 22 markers.\n",
      "2024-04-17 18:11:28,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067380.1 (Klebsiella grimontii, 2481359) loaded with 23 markers.\n",
      "2024-04-17 18:11:28,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031104.1 (Leclercia sp., W6) loaded with 38 markers.\n",
      "2024-04-17 18:11:29,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022823.1 (Klebsiella quasivariicola, KPN1705) loaded with 22 markers.\n",
      "2024-04-17 18:11:29,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015227.1 (Enterobacter sp., ODB01) loaded with 34 markers.\n",
      "2024-04-17 18:11:29,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017087.1 (Enterobacter sp., HK169) loaded with 32 markers.\n",
      "2024-04-17 18:11:29,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019634.1 (Enterobacter sp., 18A13) loaded with 33 markers.\n",
      "2024-04-17 18:11:29,341 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056148.1 (Enterobacter roggenkampii, RHBSTW-00872) loaded with 49 markers.\n",
      "2024-04-17 18:11:29,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056737.1 (Enterobacter roggenkampii, RHBSTW-00309) loaded with 56 markers.\n",
      "2024-04-17 18:11:29,583 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058196.1 (Enterobacter roggenkampii, RHBSTW-00002) loaded with 104 markers.\n",
      "2024-04-17 18:11:29,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086201.1 (Enterobacter roggenkampii, F1057) loaded with 35 markers.\n",
      "2024-04-17 18:11:29,748 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060733.1 (Enterobacter roggenkampii, Ed-982) loaded with 45 markers.\n",
      "2024-04-17 18:11:29,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022452.1 (Enterobacter kobei, STW0522-62) loaded with 41 markers.\n",
      "2024-04-17 18:11:29,904 [DEBUG - chronostrain.database.parser.json] - Strain NC_018405.1 (Enterobacter kobei, ENHKU01) loaded with 33 markers.\n",
      "2024-04-17 18:11:29,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042578.1 (Enterobacter kobei, C16) loaded with 34 markers.\n",
      "2024-04-17 18:11:30,082 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022126.1 (Enterobacter cloacae, WP5-S18-CRE-02) loaded with 40 markers.\n",
      "2024-04-17 18:11:30,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022519.1 (Enterobacter cloacae, STN0717-73) loaded with 39 markers.\n",
      "2024-04-17 18:11:30,290 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020089.1 (Enterobacter cloacae, PIMB10EC27) loaded with 59 markers.\n",
      "2024-04-17 18:11:30,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071861.1 (Enterobacter cloacae, GX1Z-1L) loaded with 40 markers.\n",
      "2024-04-17 18:11:30,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026975.1 (Enterobacter cloacae, FDAARGOS_77) loaded with 40 markers.\n",
      "2024-04-17 18:11:30,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009850.1 (Enterobacter cloacae, ECNIH4) loaded with 35 markers.\n",
      "2024-04-17 18:11:30,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091481.1 (Enterobacter cloacae, ECL414) loaded with 39 markers.\n",
      "2024-04-17 18:11:30,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035738.1 (Enterobacter cloacae, CZ-1) loaded with 35 markers.\n",
      "2024-04-17 18:11:30,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043318.1 (Enterobacter chengduensis, WCHECh050004) loaded with 35 markers.\n",
      "2024-04-17 18:11:30,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063441.1 (Enterobacter asburiae, SD4L) loaded with 34 markers.\n",
      "2024-04-17 18:11:30,969 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074177.1 (Enterobacter asburiae, JBIWA002) loaded with 35 markers.\n",
      "2024-04-17 18:11:31,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077411.1 (Enterobacter asburiae, FDAARGOS_1432) loaded with 38 markers.\n",
      "2024-04-17 18:11:31,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024281.1 (Enterobacter asburiae, Ent261) loaded with 41 markers.\n",
      "2024-04-17 18:11:31,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034336.1 (Enterobacter asburiae, CAV1043) loaded with 50 markers.\n",
      "2024-04-17 18:11:31,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011591.1 (Enterobacter asburiae, CAV1043) loaded with 54 markers.\n",
      "2024-04-17 18:11:31,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022628.1 (Enterobacter asburiae, A2563) loaded with 37 markers.\n",
      "2024-04-17 18:11:31,489 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093154.1 (Enterobacter asburiae, 5549) loaded with 39 markers.\n",
      "2024-04-17 18:11:31,585 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134485.1 (Citrobacter youngae, NCTC13709) loaded with 38 markers.\n",
      "2024-04-17 18:11:31,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044101.1 (Citrobacter werkmanii, FDAARGOS_616) loaded with 41 markers.\n",
      "2024-04-17 18:11:31,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045840.1 (Citrobacter sp., S40-1-2) loaded with 43 markers.\n",
      "2024-04-17 18:11:31,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056185.1 (Citrobacter sp., RHBSTW-01013) loaded with 47 markers.\n",
      "2024-04-17 18:11:31,969 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056200.1 (Citrobacter sp., RHBSTW-00986) loaded with 47 markers.\n",
      "2024-04-17 18:11:32,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056219.1 (Citrobacter sp., RHBSTW-00976) loaded with 68 markers.\n",
      "2024-04-17 18:11:32,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055418.1 (Citrobacter sp., RHBSTW-00944) loaded with 53 markers.\n",
      "2024-04-17 18:11:32,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056251.1 (Citrobacter sp., RHBSTW-00903) loaded with 61 markers.\n",
      "2024-04-17 18:11:32,410 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056267.1 (Citrobacter sp., RHBSTW-00887) loaded with 53 markers.\n",
      "2024-04-17 18:11:32,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056269.1 (Citrobacter sp., RHBSTW-00881) loaded with 61 markers.\n",
      "2024-04-17 18:11:32,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056284.1 (Citrobacter sp., RHBSTW-00859) loaded with 54 markers.\n",
      "2024-04-17 18:11:32,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056321.1 (Citrobacter sp., RHBSTW-00821) loaded with 53 markers.\n",
      "2024-04-17 18:11:32,995 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056341.1 (Citrobacter sp., RHBSTW-00696) loaded with 50 markers.\n",
      "2024-04-17 18:11:33,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056350.1 (Citrobacter sp., RHBSTW-00678) loaded with 63 markers.\n",
      "2024-04-17 18:11:33,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056361.1 (Citrobacter sp., RHBSTW-00667) loaded with 70 markers.\n",
      "2024-04-17 18:11:33,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056381.1 (Citrobacter sp., RHBSTW-00628) loaded with 68 markers.\n",
      "2024-04-17 18:11:33,594 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056421.1 (Citrobacter sp., RHBSTW-00524) loaded with 52 markers.\n",
      "2024-04-17 18:11:33,727 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056508.1 (Citrobacter sp., RHBSTW-00446) loaded with 54 markers.\n",
      "2024-04-17 18:11:33,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056546.1 (Citrobacter sp., RHBSTW-00424) loaded with 61 markers.\n",
      "2024-04-17 18:11:33,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056647.1 (Citrobacter sp., RHBSTW-00229) loaded with 46 markers.\n",
      "2024-04-17 18:11:34,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056822.1 (Citrobacter sp., RHBSTW-00137) loaded with 67 markers.\n",
      "2024-04-17 18:11:34,269 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056861.1 (Citrobacter sp., RHBSTW-00107) loaded with 54 markers.\n",
      "2024-04-17 18:11:34,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056888.1 (Citrobacter sp., RHBSTW-00053) loaded with 64 markers.\n",
      "2024-04-17 18:11:34,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056896.1 (Citrobacter sp., RHBSTW-00021) loaded with 69 markers.\n",
      "2024-04-17 18:11:34,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056899.1 (Citrobacter sp., RHBSTW-00017) loaded with 67 markers.\n",
      "2024-04-17 18:11:34,877 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057125.1 (Citrobacter sp., RHB36-C18) loaded with 44 markers.\n",
      "2024-04-17 18:11:34,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057147.1 (Citrobacter sp., RHB35-C21) loaded with 47 markers.\n",
      "2024-04-17 18:11:35,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045837.1 (Citrobacter sp., H12-3-2) loaded with 43 markers.\n",
      "2024-04-17 18:11:35,142 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014030.2 (Citrobacter sp., FDAARGOS_155) loaded with 35 markers.\n",
      "2024-04-17 18:11:35,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024676.1 (Citrobacter pasteurii, UMH17) loaded with 38 markers.\n",
      "2024-04-17 18:11:35,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077262.1 (Citrobacter pasteurii, FDAARGOS_1424) loaded with 33 markers.\n",
      "2024-04-17 18:11:35,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023527.1 (Citrobacter koseri, FDAARGOS_393) loaded with 40 markers.\n",
      "2024-04-17 18:11:35,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022073.1 (Citrobacter koseri, FDAARGOS_287) loaded with 41 markers.\n",
      "2024-04-17 18:11:35,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026709.1 (Citrobacter koseri, AR_0024) loaded with 40 markers.\n",
      "2024-04-17 18:11:35,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024673.1 (Citrobacter freundii, UMH19) loaded with 46 markers.\n",
      "2024-04-17 18:11:35,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056592.1 (Citrobacter freundii, RHBSTW-00350) loaded with 49 markers.\n",
      "2024-04-17 18:11:36,054 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056595.1 (Citrobacter freundii, RHBSTW-00342) loaded with 51 markers.\n",
      "2024-04-17 18:11:36,177 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057323.1 (Citrobacter freundii, RHB30-C03) loaded with 48 markers.\n",
      "2024-04-17 18:11:36,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026238.1 (Citrobacter freundii, CFNIH9) loaded with 74 markers.\n",
      "2024-04-17 18:11:36,500 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026235.1 (Citrobacter freundii, CFNIH3) loaded with 64 markers.\n",
      "2024-04-17 18:11:36,625 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038658.1 (Citrobacter freundii, 680) loaded with 52 markers.\n",
      "2024-04-17 18:11:36,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048382.1 (Citrobacter freundii, 62) loaded with 57 markers.\n",
      "2024-04-17 18:11:36,899 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038656.1 (Citrobacter freundii, 565) loaded with 52 markers.\n",
      "2024-04-17 18:11:37,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038653.1 (Citrobacter freundii, 154) loaded with 52 markers.\n",
      "2024-04-17 18:11:37,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045771.1 (Citrobacter braakii, MiY-A) loaded with 43 markers.\n",
      "2024-04-17 18:11:37,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022049.2 (Citrobacter braakii, FDAARGOS_290) loaded with 43 markers.\n",
      "2024-04-17 18:11:37,361 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077300.1 (Citrobacter braakii, FDAARGOS_1422) loaded with 45 markers.\n",
      "2024-04-17 18:11:37,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025881.1 (Escherichia coli, 503440) loaded with 69 markers.\n",
      "2024-04-17 18:11:37,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT991957.1 (Enterobacter cloacae, C45) loaded with 33 markers.\n",
      "2024-04-17 18:11:37,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057136.1 (Citrobacter freundii, RHB36-C06) loaded with 45 markers.\n",
      "2024-04-17 18:11:37,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009459.1 (Cedecea neteri, ND14a) loaded with 26 markers.\n",
      "2024-04-17 18:11:37,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009458.1 (Cedecea neteri, M006) loaded with 29 markers.\n",
      "2024-04-17 18:11:37,995 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023525.1 (Cedecea neteri, FDAARGOS_392) loaded with 25 markers.\n",
      "2024-04-17 18:11:38,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077662.1 (Salmonella enterica, S146) loaded with 42 markers.\n",
      "2024-04-17 18:11:38,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075112.1 (Salmonella enterica, CFSAN059903) loaded with 41 markers.\n",
      "2024-04-17 18:11:38,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033350.2 (Salmonella enterica, CFSA231) loaded with 42 markers.\n",
      "2024-04-17 18:11:38,409 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028900.1 (Salmonella enterica, CD-SL01) loaded with 42 markers.\n",
      "2024-04-17 18:11:38,496 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087978.1 (Klebsiella variicola, ZH07) loaded with 17 markers.\n",
      "2024-04-17 18:11:38,565 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027064.2 (Klebsiella variicola, WCHKV030666) loaded with 17 markers.\n",
      "2024-04-17 18:11:38,646 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063926.1 (Klebsiella variicola, S187-1) loaded with 16 markers.\n",
      "2024-04-17 18:11:38,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063953.1 (Klebsiella variicola, S131-2) loaded with 18 markers.\n",
      "2024-04-17 18:11:38,828 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063954.1 (Klebsiella variicola, S129-1) loaded with 18 markers.\n",
      "2024-04-17 18:11:38,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063915.1 (Klebsiella variicola, M186-1-2) loaded with 16 markers.\n",
      "2024-04-17 18:11:38,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063911.1 (Klebsiella variicola, M186-1) loaded with 16 markers.\n",
      "2024-04-17 18:11:39,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063867.1 (Klebsiella variicola, M142-3) loaded with 15 markers.\n",
      "2024-04-17 18:11:39,159 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013985.1 (Klebsiella variicola, LMG_23571) loaded with 18 markers.\n",
      "2024-04-17 18:11:39,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020847.1 (Klebsiella variicola, KPN1481) loaded with 17 markers.\n",
      "2024-04-17 18:11:39,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065162.1 (Klebsiella variicola, KPN029) loaded with 18 markers.\n",
      "2024-04-17 18:11:39,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024592.1 (Klebsiella variicola, JCM_12419) loaded with 18 markers.\n",
      "2024-04-17 18:11:39,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083790.1 (Klebsiella variicola, INF352) loaded with 18 markers.\n",
      "2024-04-17 18:11:39,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017289.1 (Klebsiella variicola, GJ3) loaded with 16 markers.\n",
      "2024-04-17 18:11:39,691 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017849.1 (Klebsiella variicola, GJ2) loaded with 16 markers.\n",
      "2024-04-17 18:11:39,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017284.1 (Klebsiella variicola, GJ1) loaded with 16 markers.\n",
      "2024-04-17 18:11:39,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044050.1 (Klebsiella variicola, FDAARGOS_627) loaded with 17 markers.\n",
      "2024-04-17 18:11:39,924 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079802.1 (Klebsiella variicola, EMBL-1) loaded with 16 markers.\n",
      "2024-04-17 18:11:40,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016344.1 (Klebsiella variicola, E57-7) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,076 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067261.1 (Klebsiella variicola, E25) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,156 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084767.1 (Klebsiella variicola, CDC4241-71) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063932.1 (Klebsiella variicola, BS375-3) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063933.1 (Klebsiella variicola, BS369-2) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063912.1 (Klebsiella variicola, BS359-2-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,464 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063916.1 (Klebsiella variicola, BS327-2-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063898.1 (Klebsiella variicola, BS326-1) loaded with 18 markers.\n",
      "2024-04-17 18:11:40,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063889.1 (Klebsiella variicola, BS325-3-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063910.1 (Klebsiella variicola, BS325-2) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,774 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063921.1 (Klebsiella variicola, BM366-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,858 [DEBUG - chronostrain.database.parser.json] - Strain NC_013850.1 (Klebsiella variicola, At-22) loaded with 17 markers.\n",
      "2024-04-17 18:11:40,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130538.1 (Klebsiella variicola, AJ292) loaded with 17 markers.\n",
      "2024-04-17 18:11:41,017 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130539.1 (Klebsiella variicola, AJ055) loaded with 16 markers.\n",
      "2024-04-17 18:11:41,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047360.1 (Klebsiella variicola, AHKv-S01) loaded with 18 markers.\n",
      "2024-04-17 18:11:41,170 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032354.1 (Klebsiella variicola, 15WZ-82) loaded with 17 markers.\n",
      "2024-04-17 18:11:41,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026013.1 (Klebsiella variicola, 13450) loaded with 17 markers.\n",
      "2024-04-17 18:11:41,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030173.1 (Klebsiella variicola, 13450) loaded with 17 markers.\n",
      "2024-04-17 18:11:41,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130544.1 (Klebsiella variicola, 03-311-0071) loaded with 19 markers.\n",
      "2024-04-17 18:11:41,480 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084766.1 (Klebsiella variicola, 01A065) loaded with 17 markers.\n",
      "2024-04-17 18:11:41,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033631.1 (Klebsiella sp., P1CD1) loaded with 16 markers.\n",
      "2024-04-17 18:11:41,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073377.1 (Klebsiella sp., P1927) loaded with 35 markers.\n",
      "2024-04-17 18:11:41,722 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089234.1 (Klebsiella sp., KPN54798) loaded with 20 markers.\n",
      "2024-04-17 18:11:41,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058960.1 (Klebsiella pneumoniae, ZZW20) loaded with 19 markers.\n",
      "2024-04-17 18:11:41,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050336.1 (Klebsiella pneumoniae, ZRKP04) loaded with 37 markers.\n",
      "2024-04-17 18:11:42,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050342.1 (Klebsiella pneumoniae, ZRKP03) loaded with 37 markers.\n",
      "2024-04-17 18:11:42,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050348.1 (Klebsiella pneumoniae, ZRKP02) loaded with 37 markers.\n",
      "2024-04-17 18:11:42,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050354.1 (Klebsiella pneumoniae, ZRKP01) loaded with 37 markers.\n",
      "2024-04-17 18:11:42,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067060.1 (Klebsiella pneumoniae, ZJ27003) loaded with 20 markers.\n",
      "2024-04-17 18:11:42,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065347.1 (Klebsiella pneumoniae, ZG2017CW4-4-2) loaded with 16 markers.\n",
      "2024-04-17 18:11:42,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065343.1 (Klebsiella pneumoniae, ZG2017CW4-4-1-2) loaded with 16 markers.\n",
      "2024-04-17 18:11:42,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075286.1 (Klebsiella pneumoniae, YTF44-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:42,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045193.1 (Klebsiella pneumoniae, YML0508) loaded with 18 markers.\n",
      "2024-04-17 18:11:42,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032240.1 (Klebsiella pneumoniae, XJ-K2) loaded with 39 markers.\n",
      "2024-04-17 18:11:42,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032163.1 (Klebsiella pneumoniae, XJ-K1) loaded with 34 markers.\n",
      "2024-04-17 18:11:42,945 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092786.1 (Klebsiella pneumoniae, XH1508) loaded with 31 markers.\n",
      "2024-04-17 18:11:43,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092793.1 (Klebsiella pneumoniae, XH1507) loaded with 31 markers.\n",
      "2024-04-17 18:11:43,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071385.1 (Klebsiella pneumoniae, wxc2020) loaded with 16 markers.\n",
      "2024-04-17 18:11:43,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054063.1 (Klebsiella pneumoniae, WSHvKP) loaded with 19 markers.\n",
      "2024-04-17 18:11:43,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091068.1 (Klebsiella pneumoniae, WSCRKP) loaded with 35 markers.\n",
      "2024-04-17 18:11:43,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079692.1 (Klebsiella pneumoniae, WRC31_S468MPCH) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079683.1 (Klebsiella pneumoniae, WRC30_AI1547MPCH) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,502 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079655.1 (Klebsiella pneumoniae, WRC27_S471MPC) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079645.1 (Klebsiella pneumoniae, WRC24_S468MC) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079642.1 (Klebsiella pneumoniae, WRC22_AI1646MC) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079636.1 (Klebsiella pneumoniae, WRC21_AI1646PC) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079634.1 (Klebsiella pneumoniae, WRC19_AI1572C) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,897 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079617.1 (Klebsiella pneumoniae, WRC16_AI1547MC) loaded with 18 markers.\n",
      "2024-04-17 18:11:43,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079606.1 (Klebsiella pneumoniae, WRC15_AI1547PC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079186.1 (Klebsiella pneumoniae, WRC14_AI2040MC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079179.1 (Klebsiella pneumoniae, WRC13_AI2040PC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084044.1 (Klebsiella pneumoniae, WRC12_S471M) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079172.1 (Klebsiella pneumoniae, WRC11_S471PC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,342 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079807.1 (Klebsiella pneumoniae, WRC02_S465MC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079115.1 (Klebsiella pneumoniae, WRC01_S465PC) loaded with 18 markers.\n",
      "2024-04-17 18:11:44,496 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022078.1 (Klebsiella pneumoniae, WP4-S18-CRE-04) loaded with 19 markers.\n",
      "2024-04-17 18:11:44,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021909.1 (Klebsiella pneumoniae, WP2-S17-ESBL-02) loaded with 16 markers.\n",
      "2024-04-17 18:11:44,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027068.3 (Klebsiella pneumoniae, WCHKP8F4) loaded with 34 markers.\n",
      "2024-04-17 18:11:44,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028806.2 (Klebsiella pneumoniae, WCHKP7E2) loaded with 19 markers.\n",
      "2024-04-17 18:11:44,826 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028583.2 (Klebsiella pneumoniae, WCHKP36) loaded with 34 markers.\n",
      "2024-04-17 18:11:44,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036361.1 (Klebsiella pneumoniae, WCHKP2080) loaded with 34 markers.\n",
      "2024-04-17 18:11:45,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028542.3 (Klebsiella pneumoniae, WCHKP2) loaded with 35 markers.\n",
      "2024-04-17 18:11:45,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028391.2 (Klebsiella pneumoniae, WCHKP13F2) loaded with 16 markers.\n",
      "2024-04-17 18:11:45,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033405.1 (Klebsiella pneumoniae, WCHKP115069) loaded with 34 markers.\n",
      "2024-04-17 18:11:45,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036365.1 (Klebsiella pneumoniae, WCHKP115068) loaded with 37 markers.\n",
      "2024-04-17 18:11:45,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026585.3 (Klebsiella pneumoniae, WCHKP095649) loaded with 33 markers.\n",
      "2024-04-17 18:11:45,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066534.1 (Klebsiella pneumoniae, WCHKP090374) loaded with 38 markers.\n",
      "2024-04-17 18:11:45,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066528.1 (Klebsiella pneumoniae, WCHKP090361) loaded with 32 markers.\n",
      "2024-04-17 18:11:45,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066523.1 (Klebsiella pneumoniae, WCHKP090357) loaded with 38 markers.\n",
      "2024-04-17 18:11:45,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066517.1 (Klebsiella pneumoniae, WCHKP090329) loaded with 32 markers.\n",
      "2024-04-17 18:11:46,091 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028797.2 (Klebsiella pneumoniae, WCHKP040035) loaded with 39 markers.\n",
      "2024-04-17 18:11:46,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073285.1 (Klebsiella pneumoniae, WCHKP030925) loaded with 20 markers.\n",
      "2024-04-17 18:11:46,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043357.1 (Klebsiella pneumoniae, WCHKP020120) loaded with 37 markers.\n",
      "2024-04-17 18:11:46,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043353.1 (Klebsiella pneumoniae, WCHKP020115) loaded with 34 markers.\n",
      "2024-04-17 18:11:46,511 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036305.1 (Klebsiella pneumoniae, WCHKP020098) loaded with 32 markers.\n",
      "2024-04-17 18:11:46,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036371.1 (Klebsiella pneumoniae, WCHKP020037) loaded with 36 markers.\n",
      "2024-04-17 18:11:46,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028793.2 (Klebsiella pneumoniae, WCHKP020030) loaded with 32 markers.\n",
      "2024-04-17 18:11:46,831 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036300.1 (Klebsiella pneumoniae, WCHKP015093) loaded with 37 markers.\n",
      "2024-04-17 18:11:46,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036320.1 (Klebsiella pneumoniae, VBA2172) loaded with 18 markers.\n",
      "2024-04-17 18:11:47,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011985.1 (Klebsiella pneumoniae, UHKPC07) loaded with 25 markers.\n",
      "2024-04-17 18:11:47,173 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023722.1 (Klebsiella pneumoniae, TVGHCRE225) loaded with 35 markers.\n",
      "2024-04-17 18:11:47,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024795.1 (Klebsiella pneumoniae, TUM12140) loaded with 33 markers.\n",
      "2024-04-17 18:11:47,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024792.1 (Klebsiella pneumoniae, TUM12139) loaded with 32 markers.\n",
      "2024-04-17 18:11:47,502 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024788.1 (Klebsiella pneumoniae, TUM12138) loaded with 32 markers.\n",
      "2024-04-17 18:11:47,597 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024784.1 (Klebsiella pneumoniae, TUM12137) loaded with 32 markers.\n",
      "2024-04-17 18:11:47,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024781.1 (Klebsiella pneumoniae, TUM12136) loaded with 33 markers.\n",
      "2024-04-17 18:11:47,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024778.1 (Klebsiella pneumoniae, TUM12135) loaded with 31 markers.\n",
      "2024-04-17 18:11:47,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024775.1 (Klebsiella pneumoniae, TUM12134) loaded with 32 markers.\n",
      "2024-04-17 18:11:47,918 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024772.1 (Klebsiella pneumoniae, TUM12133) loaded with 31 markers.\n",
      "2024-04-17 18:11:48,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024769.1 (Klebsiella pneumoniae, TUM12132) loaded with 32 markers.\n",
      "2024-04-17 18:11:48,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024766.1 (Klebsiella pneumoniae, TUM12131) loaded with 33 markers.\n",
      "2024-04-17 18:11:48,236 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024763.1 (Klebsiella pneumoniae, TUM12130) loaded with 32 markers.\n",
      "2024-04-17 18:11:48,328 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024760.1 (Klebsiella pneumoniae, TUM12129) loaded with 31 markers.\n",
      "2024-04-17 18:11:48,425 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024756.1 (Klebsiella pneumoniae, TUM12128) loaded with 30 markers.\n",
      "2024-04-17 18:11:48,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024753.1 (Klebsiella pneumoniae, TUM12127) loaded with 32 markers.\n",
      "2024-04-17 18:11:48,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024750.1 (Klebsiella pneumoniae, TUM12126) loaded with 30 markers.\n",
      "2024-04-17 18:11:48,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022527.1 (Klebsiella pneumoniae, THO-004) loaded with 18 markers.\n",
      "2024-04-17 18:11:48,757 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087123.1 (Klebsiella pneumoniae, TH12908) loaded with 20 markers.\n",
      "2024-04-17 18:11:48,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091650.1 (Klebsiella pneumoniae, TAKPN-1) loaded with 19 markers.\n",
      "2024-04-17 18:11:48,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019665.1 (Klebsiella pneumoniae, TA6363) loaded with 16 markers.\n",
      "2024-04-17 18:11:48,963 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084241.1 (Klebsiella pneumoniae, T877) loaded with 17 markers.\n",
      "2024-04-17 18:11:49,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079781.1 (Klebsiella pneumoniae, T698-1) loaded with 17 markers.\n",
      "2024-04-17 18:11:49,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018454.1 (Klebsiella pneumoniae, SWU01) loaded with 33 markers.\n",
      "2024-04-17 18:11:49,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055087.1 (Klebsiella pneumoniae, SWHIN_114) loaded with 18 markers.\n",
      "2024-04-17 18:11:49,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055096.1 (Klebsiella pneumoniae, SWHIN_108) loaded with 16 markers.\n",
      "2024-04-17 18:11:49,426 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055061.1 (Klebsiella pneumoniae, SWHE3) loaded with 18 markers.\n",
      "2024-04-17 18:11:49,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073302.1 (Klebsiella pneumoniae, SW1780) loaded with 38 markers.\n",
      "2024-04-17 18:11:49,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058758.1 (Klebsiella pneumoniae, STLIN_3) loaded with 17 markers.\n",
      "2024-04-17 18:11:49,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058752.1 (Klebsiella pneumoniae, STLIN_19) loaded with 17 markers.\n",
      "2024-04-17 18:11:49,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058743.1 (Klebsiella pneumoniae, STLIN_10) loaded with 17 markers.\n",
      "2024-04-17 18:11:49,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058857.1 (Klebsiella pneumoniae, STLE2) loaded with 18 markers.\n",
      "2024-04-17 18:11:49,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055003.1 (Klebsiella pneumoniae, STIN_79) loaded with 19 markers.\n",
      "2024-04-17 18:11:50,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055005.1 (Klebsiella pneumoniae, STIN_77) loaded with 18 markers.\n",
      "2024-04-17 18:11:50,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037742.1 (Klebsiella pneumoniae, ST23) loaded with 20 markers.\n",
      "2024-04-17 18:11:50,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023487.1 (Klebsiella pneumoniae, ST101:960186733) loaded with 17 markers.\n",
      "2024-04-17 18:11:50,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040833.1 (Klebsiella pneumoniae, SH12) loaded with 33 markers.\n",
      "2024-04-17 18:11:50,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025080.1 (Klebsiella pneumoniae, SGH10) loaded with 20 markers.\n",
      "2024-04-17 18:11:50,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065444.1 (Klebsiella pneumoniae, Sea_Superman) loaded with 35 markers.\n",
      "2024-04-17 18:11:50,514 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086664.1 (Klebsiella pneumoniae, SCPM-O-B-8912_(11PKP_19a)) loaded with 27 markers.\n",
      "2024-04-17 18:11:50,583 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029388.1 (Klebsiella pneumoniae, SCKP040074) loaded with 22 markers.\n",
      "2024-04-17 18:11:50,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028548.2 (Klebsiella pneumoniae, SCKP020143) loaded with 36 markers.\n",
      "2024-04-17 18:11:50,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029384.2 (Klebsiella pneumoniae, SCKP020079) loaded with 37 markers.\n",
      "2024-04-17 18:11:50,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028783.2 (Klebsiella pneumoniae, SCKP020046) loaded with 18 markers.\n",
      "2024-04-17 18:11:50,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031721.1 (Klebsiella pneumoniae, SCKP020003) loaded with 36 markers.\n",
      "2024-04-17 18:11:51,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050858.1 (Klebsiella pneumoniae, SCH6109) loaded with 18 markers.\n",
      "2024-04-17 18:11:51,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082799.1 (Klebsiella pneumoniae, SCAID_TST1-2021_(15_226)) loaded with 18 markers.\n",
      "2024-04-17 18:11:51,175 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082805.1 (Klebsiella pneumoniae, SCAID_PHRX1-2021_(13_97)) loaded with 23 markers.\n",
      "2024-04-17 18:11:51,251 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030269.1 (Klebsiella pneumoniae, SC-7) loaded with 20 markers.\n",
      "2024-04-17 18:11:51,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084829.1 (Klebsiella pneumoniae, SB615) loaded with 20 markers.\n",
      "2024-04-17 18:11:51,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084830.1 (Klebsiella pneumoniae, SB612) loaded with 17 markers.\n",
      "2024-04-17 18:11:51,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084843.1 (Klebsiella pneumoniae, SB611) loaded with 22 markers.\n",
      "2024-04-17 18:11:51,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084866.1 (Klebsiella pneumoniae, SA12) loaded with 20 markers.\n",
      "2024-04-17 18:11:51,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084868.1 (Klebsiella pneumoniae, SA1) loaded with 16 markers.\n",
      "2024-04-17 18:11:51,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063881.1 (Klebsiella pneumoniae, S90-2) loaded with 17 markers.\n",
      "2024-04-17 18:11:51,748 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063925.1 (Klebsiella pneumoniae, S210-3) loaded with 27 markers.\n",
      "2024-04-17 18:11:51,811 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063945.1 (Klebsiella pneumoniae, S166-1) loaded with 15 markers.\n",
      "2024-04-17 18:11:51,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058548.1 (Klebsiella pneumoniae, S165-1) loaded with 19 markers.\n",
      "2024-04-17 18:11:51,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014010.1 (Klebsiella pneumoniae, RJF999) loaded with 20 markers.\n",
      "2024-04-17 18:11:52,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014008.1 (Klebsiella pneumoniae, RJF293) loaded with 17 markers.\n",
      "2024-04-17 18:11:52,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068684.1 (Klebsiella pneumoniae, RJBSI76-pV) loaded with 32 markers.\n",
      "2024-04-17 18:11:52,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068689.1 (Klebsiella pneumoniae, RJBSI76) loaded with 32 markers.\n",
      "2024-04-17 18:11:52,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019047.1 (Klebsiella pneumoniae, RJA166) loaded with 20 markers.\n",
      "2024-04-17 18:11:52,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056216.1 (Klebsiella pneumoniae, RHBSTW-00982) loaded with 18 markers.\n",
      "2024-04-17 18:11:52,436 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056297.1 (Klebsiella pneumoniae, RHBSTW-00852) loaded with 18 markers.\n",
      "2024-04-17 18:11:52,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056301.1 (Klebsiella pneumoniae, RHBSTW-00843) loaded with 17 markers.\n",
      "2024-04-17 18:11:52,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056307.1 (Klebsiella pneumoniae, RHBSTW-00832) loaded with 31 markers.\n",
      "2024-04-17 18:11:52,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056372.1 (Klebsiella pneumoniae, RHBSTW-00636) loaded with 20 markers.\n",
      "2024-04-17 18:11:52,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056385.1 (Klebsiella pneumoniae, RHBSTW-00610) loaded with 18 markers.\n",
      "2024-04-17 18:11:52,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056432.1 (Klebsiella pneumoniae, RHBSTW-00510) loaded with 17 markers.\n",
      "2024-04-17 18:11:52,917 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055509.1 (Klebsiella pneumoniae, RHBSTW-00472) loaded with 31 markers.\n",
      "2024-04-17 18:11:52,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056878.1 (Klebsiella pneumoniae, RHBSTW-00066) loaded with 18 markers.\n",
      "2024-04-17 18:11:53,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056883.1 (Klebsiella pneumoniae, RHBSTW-00062) loaded with 18 markers.\n",
      "2024-04-17 18:11:53,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057066.1 (Klebsiella pneumoniae, RHB39-C10) loaded with 18 markers.\n",
      "2024-04-17 18:11:53,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057069.1 (Klebsiella pneumoniae, RHB39-C09) loaded with 17 markers.\n",
      "2024-04-17 18:11:53,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057313.1 (Klebsiella pneumoniae, RHB30-C05) loaded with 18 markers.\n",
      "2024-04-17 18:11:53,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034082.1 (Klebsiella pneumoniae, R210-2) loaded with 20 markers.\n",
      "2024-04-17 18:11:53,405 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031798.1 (Klebsiella pneumoniae, QMP_B2-170) loaded with 17 markers.\n",
      "2024-04-17 18:11:53,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065441.1 (Klebsiella pneumoniae, Patrick_Star) loaded with 31 markers.\n",
      "2024-04-17 18:11:53,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081896.1 (Klebsiella pneumoniae, PA-01) loaded with 20 markers.\n",
      "2024-04-17 18:11:53,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090931.1 (Klebsiella pneumoniae, ONE259) loaded with 17 markers.\n",
      "2024-04-17 18:11:53,725 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065464.1 (Klebsiella pneumoniae, Olaf) loaded with 40 markers.\n",
      "2024-04-17 18:11:53,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025246.1 (Klebsiella pneumoniae, OCU_hvKP1) loaded with 20 markers.\n",
      "2024-04-17 18:11:53,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065447.1 (Klebsiella pneumoniae, Ocean_Ranger) loaded with 19 markers.\n",
      "2024-04-17 18:11:53,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015385.1 (Klebsiella pneumoniae, NY9) loaded with 34 markers.\n",
      "2024-04-17 18:11:54,048 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026586.1 (Klebsiella pneumoniae, NUHL30457) loaded with 16 markers.\n",
      "2024-04-17 18:11:54,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061380.1 (Klebsiella pneumoniae, NU-CRE236) loaded with 17 markers.\n",
      "2024-04-17 18:11:54,195 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061385.1 (Klebsiella pneumoniae, NU-CRE213) loaded with 26 markers.\n",
      "2024-04-17 18:11:54,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061396.1 (Klebsiella pneumoniae, NU-CRE193) loaded with 29 markers.\n",
      "2024-04-17 18:11:54,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061399.1 (Klebsiella pneumoniae, NU-CRE085) loaded with 30 markers.\n",
      "2024-04-17 18:11:54,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025037.1 (Klebsiella pneumoniae, NU-CRE047) loaded with 29 markers.\n",
      "2024-04-17 18:11:54,575 [DEBUG - chronostrain.database.parser.json] - Strain NC_012731.1 (Klebsiella pneumoniae, NTUH-K2044) loaded with 20 markers.\n",
      "2024-04-17 18:11:54,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091582.1 (Klebsiella pneumoniae, Nord9_R85) loaded with 16 markers.\n",
      "2024-04-17 18:11:54,707 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041644.1 (Klebsiella pneumoniae, NKU_Kleb8A7) loaded with 17 markers.\n",
      "2024-04-17 18:11:54,775 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035531.1 (Klebsiella pneumoniae, NFYY0065) loaded with 18 markers.\n",
      "2024-04-17 18:11:54,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081894.1 (Klebsiella pneumoniae, NC-01) loaded with 18 markers.\n",
      "2024-04-17 18:11:54,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034760.1 (Klebsiella pneumoniae, NB5306) loaded with 17 markers.\n",
      "2024-04-17 18:11:54,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092656.1 (Klebsiella pneumoniae, NB5) loaded with 20 markers.\n",
      "2024-04-17 18:11:55,050 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091846.1 (Klebsiella pneumoniae, NB05) loaded with 20 markers.\n",
      "2024-04-17 18:11:55,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091992.1 (Klebsiella pneumoniae, NB04) loaded with 20 markers.\n",
      "2024-04-17 18:11:55,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031577.1 (Klebsiella pneumoniae, N4b) loaded with 18 markers.\n",
      "2024-04-17 18:11:55,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047271.1 (Klebsiella pneumoniae, N16-03892) loaded with 19 markers.\n",
      "2024-04-17 18:11:55,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031800.1 (Klebsiella pneumoniae, MSB1_8A-sc-2280397) loaded with 19 markers.\n",
      "2024-04-17 18:11:55,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065438.1 (Klebsiella pneumoniae, Mrs._Puff) loaded with 32 markers.\n",
      "2024-04-17 18:11:55,480 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018428.1 (Klebsiella pneumoniae, MNCRE78) loaded with 27 markers.\n",
      "2024-04-17 18:11:55,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018427.1 (Klebsiella pneumoniae, MNCRE69) loaded with 27 markers.\n",
      "2024-04-17 18:11:55,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018437.1 (Klebsiella pneumoniae, MNCRE53) loaded with 28 markers.\n",
      "2024-04-17 18:11:55,738 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086128.1 (Klebsiella pneumoniae, MIN-109) loaded with 18 markers.\n",
      "2024-04-17 18:11:55,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063277.1 (Klebsiella pneumoniae, MAR14-456) loaded with 20 markers.\n",
      "2024-04-17 18:11:55,928 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065457.1 (Klebsiella pneumoniae, Magic_conch_shell) loaded with 18 markers.\n",
      "2024-04-17 18:11:56,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023453.1 (Klebsiella pneumoniae, M941) loaded with 17 markers.\n",
      "2024-04-17 18:11:56,087 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064129.1 (Klebsiella pneumoniae, M911-1) loaded with 19 markers.\n",
      "2024-04-17 18:11:56,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063908.1 (Klebsiella pneumoniae, M212-2) loaded with 20 markers.\n",
      "2024-04-17 18:11:56,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063930.1 (Klebsiella pneumoniae, M186-2) loaded with 18 markers.\n",
      "2024-04-17 18:11:56,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063878.1 (Klebsiella pneumoniae, M169-3) loaded with 17 markers.\n",
      "2024-04-17 18:11:56,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040391.1 (Klebsiella pneumoniae, LSH-KPN25) loaded with 18 markers.\n",
      "2024-04-17 18:11:56,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040122.1 (Klebsiella pneumoniae, LSH-KPN148) loaded with 33 markers.\n",
      "2024-04-17 18:11:56,602 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035202.1 (Klebsiella pneumoniae, LH94) loaded with 18 markers.\n",
      "2024-04-17 18:11:56,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045782.1 (Klebsiella pneumoniae, LEMB18) loaded with 17 markers.\n",
      "2024-04-17 18:11:56,749 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058325.1 (Klebsiella pneumoniae, LC-1825_18) loaded with 28 markers.\n",
      "2024-04-17 18:11:56,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033960.1 (Klebsiella pneumoniae, L482) loaded with 29 markers.\n",
      "2024-04-17 18:11:57,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033954.1 (Klebsiella pneumoniae, L39_2) loaded with 32 markers.\n",
      "2024-04-17 18:11:57,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047677.1 (Klebsiella pneumoniae, KUH-KPNHVL1) loaded with 20 markers.\n",
      "2024-04-17 18:11:57,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047675.1 (Klebsiella pneumoniae, KUH-KPNHVF1) loaded with 20 markers.\n",
      "2024-04-17 18:11:58,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890564.1 (Klebsiella pneumoniae, KSB2_9A-sc-2280332) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083784.1 (Klebsiella pneumoniae, KSB2_6C) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,124 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890391.1 (Klebsiella pneumoniae, KSB2_2C-sc-2280354) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890343.1 (Klebsiella pneumoniae, KSB2_2B-sc-2280339) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,251 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890620.1 (Klebsiella pneumoniae, KSB2_1C-sc-2280352) loaded with 22 markers.\n",
      "2024-04-17 18:11:58,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890484.1 (Klebsiella pneumoniae, KSB2_10B-sc-2280350) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890434.1 (Klebsiella pneumoniae, KSB1_9J-sc-2280309) loaded with 18 markers.\n",
      "2024-04-17 18:11:58,459 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024535.1 (Klebsiella pneumoniae, KSB1_9D) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,518 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024496.1 (Klebsiella pneumoniae, KSB1_7E) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890230.1 (Klebsiella pneumoniae, KSB1_6J) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890500.1 (Klebsiella pneumoniae, KSB1_5H-sc-2280283) loaded with 19 markers.\n",
      "2024-04-17 18:11:58,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024191.1 (Klebsiella pneumoniae, KSB1_5D) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890722.1 (Klebsiella pneumoniae, KSB1_1H) loaded with 16 markers.\n",
      "2024-04-17 18:11:58,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073783.1 (Klebsiella pneumoniae, KP_NORM_BLD_2014_96855) loaded with 15 markers.\n",
      "2024-04-17 18:11:58,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073791.1 (Klebsiella pneumoniae, KP_NORM_BLD_2010_71621) loaded with 18 markers.\n",
      "2024-04-17 18:11:58,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018438.1 (Klebsiella pneumoniae, Kp_Goe_822917) loaded with 17 markers.\n",
      "2024-04-17 18:11:59,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018692.1 (Klebsiella pneumoniae, Kp_Goe_821588) loaded with 17 markers.\n",
      "2024-04-17 18:11:59,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018450.1 (Klebsiella pneumoniae, Kp_Goe_71070) loaded with 17 markers.\n",
      "2024-04-17 18:11:59,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018364.1 (Klebsiella pneumoniae, Kp_Goe_62629) loaded with 20 markers.\n",
      "2024-04-17 18:11:59,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018447.1 (Klebsiella pneumoniae, Kp_Goe_33208) loaded with 17 markers.\n",
      "2024-04-17 18:11:59,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018735.1 (Klebsiella pneumoniae, Kp_Goe_121641) loaded with 17 markers.\n",
      "2024-04-17 18:11:59,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069170.1 (Klebsiella pneumoniae, KPWX136) loaded with 35 markers.\n",
      "2024-04-17 18:11:59,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008831.1 (Klebsiella pneumoniae, KPR0928) loaded with 28 markers.\n",
      "2024-04-17 18:11:59,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065825.1 (Klebsiella pneumoniae, KpPF25) loaded with 19 markers.\n",
      "2024-04-17 18:11:59,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047595.1 (Klebsiella pneumoniae, Kpn_1693) loaded with 20 markers.\n",
      "2024-04-17 18:11:59,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090242.1 (Klebsiella pneumoniae, KPNSF-2) loaded with 21 markers.\n",
      "2024-04-17 18:11:59,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036450.1 (Klebsiella pneumoniae, KPNIH45) loaded with 22 markers.\n",
      "2024-04-17 18:12:00,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014647.1 (Klebsiella pneumoniae, KPNIH36) loaded with 31 markers.\n",
      "2024-04-17 18:12:00,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009771.1 (Klebsiella pneumoniae, KPNIH33) loaded with 26 markers.\n",
      "2024-04-17 18:12:00,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009775.1 (Klebsiella pneumoniae, KPNIH32) loaded with 28 markers.\n",
      "2024-04-17 18:12:00,390 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009872.1 (Klebsiella pneumoniae, KPNIH30) loaded with 26 markers.\n",
      "2024-04-17 18:12:00,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008797.1 (Klebsiella pneumoniae, KPNIH24) loaded with 26 markers.\n",
      "2024-04-17 18:12:00,599 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007727.1 (Klebsiella pneumoniae, KPNIH10) loaded with 29 markers.\n",
      "2024-04-17 18:12:00,707 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008827.1 (Klebsiella pneumoniae, KPNIH1) loaded with 28 markers.\n",
      "2024-04-17 18:12:00,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090432.1 (Klebsiella pneumoniae, KPN857) loaded with 35 markers.\n",
      "2024-04-17 18:12:00,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042977.1 (Klebsiella pneumoniae, KPN55602) loaded with 16 markers.\n",
      "2024-04-17 18:12:00,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015130.1 (Klebsiella pneumoniae, Kpn555) loaded with 20 markers.\n",
      "2024-04-17 18:12:01,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053016.1 (Klebsiella pneumoniae, KPN361) loaded with 34 markers.\n",
      "2024-04-17 18:12:01,201 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072496.1 (Klebsiella pneumoniae, KPN247) loaded with 19 markers.\n",
      "2024-04-17 18:12:01,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072492.1 (Klebsiella pneumoniae, KPN236) loaded with 20 markers.\n",
      "2024-04-17 18:12:01,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072653.1 (Klebsiella pneumoniae, KPN234) loaded with 15 markers.\n",
      "2024-04-17 18:12:01,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR745045.1 (Klebsiella pneumoniae, Kpn2166) loaded with 16 markers.\n",
      "2024-04-17 18:12:01,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020841.1 (Klebsiella pneumoniae, KPN1482) loaded with 20 markers.\n",
      "2024-04-17 18:12:01,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053875.1 (Klebsiella pneumoniae, KPN142) loaded with 32 markers.\n",
      "2024-04-17 18:12:01,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065172.1 (Klebsiella pneumoniae, KPN110) loaded with 16 markers.\n",
      "2024-04-17 18:12:01,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074116.1 (Klebsiella pneumoniae, kpn-hnqyy) loaded with 40 markers.\n",
      "2024-04-17 18:12:01,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027189.1 (Klebsiella pneumoniae, KPHS1249) loaded with 20 markers.\n",
      "2024-04-17 18:12:02,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058226.1 (Klebsiella pneumoniae, KPCZA02) loaded with 36 markers.\n",
      "2024-04-17 18:12:02,107 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130548.1 (Klebsiella pneumoniae, KPC2) loaded with 24 markers.\n",
      "2024-04-17 18:12:02,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040023.1 (Klebsiella pneumoniae, KPC160132) loaded with 18 markers.\n",
      "2024-04-17 18:12:02,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040038.1 (Klebsiella pneumoniae, KPC160125) loaded with 18 markers.\n",
      "2024-04-17 18:12:02,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040028.1 (Klebsiella pneumoniae, KPC160121) loaded with 18 markers.\n",
      "2024-04-17 18:12:02,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040033.1 (Klebsiella pneumoniae, KPC160117) loaded with 18 markers.\n",
      "2024-04-17 18:12:02,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078122.1 (Klebsiella pneumoniae, KPC-2) loaded with 37 markers.\n",
      "2024-04-17 18:12:02,580 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090623.1 (Klebsiella pneumoniae, KpBO_TO5) loaded with 28 markers.\n",
      "2024-04-17 18:12:02,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090626.1 (Klebsiella pneumoniae, KpBO_TO3) loaded with 27 markers.\n",
      "2024-04-17 18:12:02,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090629.1 (Klebsiella pneumoniae, KpBO_TO1) loaded with 27 markers.\n",
      "2024-04-17 18:12:02,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070602.1 (Klebsiella pneumoniae, KPA9853) loaded with 17 markers.\n",
      "2024-04-17 18:12:02,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070598.1 (Klebsiella pneumoniae, KPA4411) loaded with 16 markers.\n",
      "2024-04-17 18:12:03,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049604.1 (Klebsiella pneumoniae, Kp8701) loaded with 18 markers.\n",
      "2024-04-17 18:12:03,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058361.1 (Klebsiella pneumoniae, KP77) loaded with 18 markers.\n",
      "2024-04-17 18:12:03,196 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015120.1 (Klebsiella pneumoniae, kp757) loaded with 16 markers.\n",
      "2024-04-17 18:12:03,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066151.1 (Klebsiella pneumoniae, KP697) loaded with 35 markers.\n",
      "2024-04-17 18:12:03,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025456.1 (Klebsiella pneumoniae, KP69) loaded with 34 markers.\n",
      "2024-04-17 18:12:03,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044258.1 (Klebsiella pneumoniae, KP65) loaded with 37 markers.\n",
      "2024-04-17 18:12:03,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077763.1 (Klebsiella pneumoniae, KP63) loaded with 38 markers.\n",
      "2024-04-17 18:12:03,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041373.1 (Klebsiella pneumoniae, KP58) loaded with 45 markers.\n",
      "2024-04-17 18:12:03,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055294.1 (Klebsiella pneumoniae, KP55) loaded with 33 markers.\n",
      "2024-04-17 18:12:03,849 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087669.1 (Klebsiella pneumoniae, KP53) loaded with 16 markers.\n",
      "2024-04-17 18:12:03,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070579.1 (Klebsiella pneumoniae, KP52810) loaded with 16 markers.\n",
      "2024-04-17 18:12:04,015 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090462.1 (Klebsiella pneumoniae, kp5152) loaded with 38 markers.\n",
      "2024-04-17 18:12:04,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070568.1 (Klebsiella pneumoniae, KP51248) loaded with 18 markers.\n",
      "2024-04-17 18:12:04,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070564.1 (Klebsiella pneumoniae, KP51069) loaded with 28 markers.\n",
      "2024-04-17 18:12:04,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058358.1 (Klebsiella pneumoniae, KP486) loaded with 18 markers.\n",
      "2024-04-17 18:12:04,328 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082791.1 (Klebsiella pneumoniae, KP4823) loaded with 16 markers.\n",
      "2024-04-17 18:12:04,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070528.1 (Klebsiella pneumoniae, KP47693) loaded with 17 markers.\n",
      "2024-04-17 18:12:04,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089021.1 (Klebsiella pneumoniae, Kp46564_15578211) loaded with 20 markers.\n",
      "2024-04-17 18:12:04,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047192.1 (Klebsiella pneumoniae, Kp36) loaded with 36 markers.\n",
      "2024-04-17 18:12:04,680 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091058.1 (Klebsiella pneumoniae, KP30086) loaded with 20 markers.\n",
      "2024-04-17 18:12:04,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089598.1 (Klebsiella pneumoniae, KP29499) loaded with 20 markers.\n",
      "2024-04-17 18:12:04,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083445.1 (Klebsiella pneumoniae, KP294) loaded with 18 markers.\n",
      "2024-04-17 18:12:04,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091061.1 (Klebsiella pneumoniae, KP29105) loaded with 20 markers.\n",
      "2024-04-17 18:12:05,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065949.1 (Klebsiella pneumoniae, KP2509) loaded with 28 markers.\n",
      "2024-04-17 18:12:05,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041082.1 (Klebsiella pneumoniae, Kp202) loaded with 21 markers.\n",
      "2024-04-17 18:12:05,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054720.1 (Klebsiella pneumoniae, KP20194f) loaded with 35 markers.\n",
      "2024-04-17 18:12:05,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054726.1 (Klebsiella pneumoniae, KP20194e) loaded with 35 markers.\n",
      "2024-04-17 18:12:05,536 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054732.1 (Klebsiella pneumoniae, KP20194d) loaded with 35 markers.\n",
      "2024-04-17 18:12:05,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054738.1 (Klebsiella pneumoniae, KP20194c5) loaded with 35 markers.\n",
      "2024-04-17 18:12:05,749 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054744.1 (Klebsiella pneumoniae, KP20194c4) loaded with 36 markers.\n",
      "2024-04-17 18:12:05,858 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054750.1 (Klebsiella pneumoniae, KP20194c3) loaded with 36 markers.\n",
      "2024-04-17 18:12:05,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054756.1 (Klebsiella pneumoniae, KP20194c) loaded with 35 markers.\n",
      "2024-04-17 18:12:06,066 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054762.1 (Klebsiella pneumoniae, KP20194b2) loaded with 35 markers.\n",
      "2024-04-17 18:12:06,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054768.1 (Klebsiella pneumoniae, KP20194b) loaded with 35 markers.\n",
      "2024-04-17 18:12:06,281 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054774.1 (Klebsiella pneumoniae, KP20194a2) loaded with 35 markers.\n",
      "2024-04-17 18:12:06,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054780.1 (Klebsiella pneumoniae, KP20194a) loaded with 35 markers.\n",
      "2024-04-17 18:12:06,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051160.1 (Klebsiella pneumoniae, KP20191015) loaded with 17 markers.\n",
      "2024-04-17 18:12:06,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063147.1 (Klebsiella pneumoniae, KP19-3088) loaded with 20 markers.\n",
      "2024-04-17 18:12:06,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063747.1 (Klebsiella pneumoniae, KP19-3023) loaded with 18 markers.\n",
      "2024-04-17 18:12:06,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082041.1 (Klebsiella pneumoniae, KP19-2196) loaded with 38 markers.\n",
      "2024-04-17 18:12:06,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047160.1 (Klebsiella pneumoniae, KP19-2029) loaded with 36 markers.\n",
      "2024-04-17 18:12:06,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072996.1 (Klebsiella pneumoniae, KP1878) loaded with 34 markers.\n",
      "2024-04-17 18:12:07,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082010.1 (Klebsiella pneumoniae, KP18-41) loaded with 38 markers.\n",
      "2024-04-17 18:12:07,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048430.1 (Klebsiella pneumoniae, KP18-3-8) loaded with 31 markers.\n",
      "2024-04-17 18:12:07,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034249.1 (Klebsiella pneumoniae, KP18-29) loaded with 16 markers.\n",
      "2024-04-17 18:12:07,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082014.1 (Klebsiella pneumoniae, KP18-238) loaded with 37 markers.\n",
      "2024-04-17 18:12:07,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082037.1 (Klebsiella pneumoniae, KP18-2172) loaded with 37 markers.\n",
      "2024-04-17 18:12:07,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082032.1 (Klebsiella pneumoniae, KP18-2138) loaded with 37 markers.\n",
      "2024-04-17 18:12:07,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084986.1 (Klebsiella pneumoniae, KP18-2110-2) loaded with 18 markers.\n",
      "2024-04-17 18:12:07,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048933.1 (Klebsiella pneumoniae, KP18-2079) loaded with 34 markers.\n",
      "2024-04-17 18:12:07,879 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082024.1 (Klebsiella pneumoniae, KP18-2073) loaded with 37 markers.\n",
      "2024-04-17 18:12:08,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082019.1 (Klebsiella pneumoniae, KP18-2050) loaded with 39 markers.\n",
      "2024-04-17 18:12:08,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082001.1 (Klebsiella pneumoniae, KP18-1) loaded with 39 markers.\n",
      "2024-04-17 18:12:08,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041022.1 (Klebsiella pneumoniae, KP1677) loaded with 16 markers.\n",
      "2024-04-17 18:12:08,243 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085478.1 (Klebsiella pneumoniae, Kp1604) loaded with 16 markers.\n",
      "2024-04-17 18:12:08,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072463.1 (Klebsiella pneumoniae, KP1517) loaded with 36 markers.\n",
      "2024-04-17 18:12:08,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060744.1 (Klebsiella pneumoniae, KP120) loaded with 27 markers.\n",
      "2024-04-17 18:12:08,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082154.1 (Klebsiella pneumoniae, Kp11-81) loaded with 18 markers.\n",
      "2024-04-17 18:12:08,646 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084705.1 (Klebsiella pneumoniae, KP1064WHY) loaded with 32 markers.\n",
      "2024-04-17 18:12:08,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU342706.1 (Klebsiella pneumoniae, Kp1) loaded with 19 markers.\n",
      "2024-04-17 18:12:08,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086285.1 (Klebsiella pneumoniae, Kp1) loaded with 19 markers.\n",
      "2024-04-17 18:12:08,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053862.1 (Klebsiella pneumoniae, KP-Zhen) loaded with 20 markers.\n",
      "2024-04-17 18:12:09,027 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080303.1 (Klebsiella pneumoniae, KP-CT77) loaded with 33 markers.\n",
      "2024-04-17 18:12:09,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080297.1 (Klebsiella pneumoniae, KP-C76) loaded with 33 markers.\n",
      "2024-04-17 18:12:09,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037928.1 (Klebsiella pneumoniae, KP-8788) loaded with 35 markers.\n",
      "2024-04-17 18:12:09,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080309.1 (Klebsiella pneumoniae, KP-426) loaded with 34 markers.\n",
      "2024-04-17 18:12:09,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043047.1 (Klebsiella pneumoniae, KLP268) loaded with 27 markers.\n",
      "2024-04-17 18:12:09,496 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR745042.1 (Klebsiella pneumoniae, Klebsiella_pneumoniae_kpn154) loaded with 16 markers.\n",
      "2024-04-17 18:12:09,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018306.1 (Klebsiella pneumoniae, Klebsiella_pneumoniae_459) loaded with 17 markers.\n",
      "2024-04-17 18:12:09,625 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020901.1 (Klebsiella pneumoniae, K66-45) loaded with 17 markers.\n",
      "2024-04-17 18:12:09,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026011.1 (Klebsiella pneumoniae, K2044) loaded with 20 markers.\n",
      "2024-04-17 18:12:09,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076518.1 (Klebsiella pneumoniae, K186) loaded with 18 markers.\n",
      "2024-04-17 18:12:09,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072460.1 (Klebsiella pneumoniae, K-1L) loaded with 19 markers.\n",
      "2024-04-17 18:12:09,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064211.1 (Klebsiella pneumoniae, JX-CR-hvKP-9) loaded with 29 markers.\n",
      "2024-04-17 18:12:09,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064217.1 (Klebsiella pneumoniae, JX-CR-hvKP-8) loaded with 33 markers.\n",
      "2024-04-17 18:12:10,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064223.1 (Klebsiella pneumoniae, JX-CR-hvKP-7) loaded with 30 markers.\n",
      "2024-04-17 18:12:10,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064229.1 (Klebsiella pneumoniae, JX-CR-hvKP-6) loaded with 20 markers.\n",
      "2024-04-17 18:12:10,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064232.1 (Klebsiella pneumoniae, JX-CR-hvKP-5) loaded with 20 markers.\n",
      "2024-04-17 18:12:10,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064235.1 (Klebsiella pneumoniae, JX-CR-hvKP-4) loaded with 34 markers.\n",
      "2024-04-17 18:12:10,405 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064241.1 (Klebsiella pneumoniae, JX-CR-hvKP-3) loaded with 33 markers.\n",
      "2024-04-17 18:12:10,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064246.1 (Klebsiella pneumoniae, JX-CR-hvKP-2) loaded with 37 markers.\n",
      "2024-04-17 18:12:10,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064205.1 (Klebsiella pneumoniae, JX-CR-hvKP-12) loaded with 20 markers.\n",
      "2024-04-17 18:12:10,641 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064208.1 (Klebsiella pneumoniae, JX-CR-hvKP-11) loaded with 20 markers.\n",
      "2024-04-17 18:12:10,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064258.1 (Klebsiella pneumoniae, JX-CR-hvKP-10) loaded with 29 markers.\n",
      "2024-04-17 18:12:10,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064252.1 (Klebsiella pneumoniae, JX-CR-hvKP-1) loaded with 33 markers.\n",
      "2024-04-17 18:12:10,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070970.1 (Klebsiella pneumoniae, JS706) loaded with 16 markers.\n",
      "2024-04-17 18:12:10,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025466.1 (Klebsiella pneumoniae, JS187) loaded with 16 markers.\n",
      "2024-04-17 18:12:10,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091978.1 (Klebsiella pneumoniae, JNQH373) loaded with 21 markers.\n",
      "2024-04-17 18:12:11,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070898.1 (Klebsiella pneumoniae, JNQH116) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,140 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090203.1 (Klebsiella pneumoniae, JNKPN26) loaded with 34 markers.\n",
      "2024-04-17 18:12:11,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061961.1 (Klebsiella pneumoniae, JNK002) loaded with 18 markers.\n",
      "2024-04-17 18:12:11,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013711.1 (Klebsiella pneumoniae, J1) loaded with 19 markers.\n",
      "2024-04-17 18:12:11,398 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061957.1 (Klebsiella pneumoniae, IR5726) loaded with 35 markers.\n",
      "2024-04-17 18:12:11,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061952.1 (Klebsiella pneumoniae, IR5654) loaded with 17 markers.\n",
      "2024-04-17 18:12:11,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068572.1 (Klebsiella pneumoniae, IR1251) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890616.1 (Klebsiella pneumoniae, INF359-sc-2280194) loaded with 19 markers.\n",
      "2024-04-17 18:12:11,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890217.1 (Klebsiella pneumoniae, INF357-sc-2280189) loaded with 18 markers.\n",
      "2024-04-17 18:12:11,722 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890643.1 (Klebsiella pneumoniae, INF346-sc-2280166) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890235.1 (Klebsiella pneumoniae, INF334-sc-2280143) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890384.1 (Klebsiella pneumoniae, INF333) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890424.1 (Klebsiella pneumoniae, INF331-sc-2280141) loaded with 16 markers.\n",
      "2024-04-17 18:12:11,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890240.1 (Klebsiella pneumoniae, INF329-sc-2280137) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890571.1 (Klebsiella pneumoniae, INF327-sc-2280132) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890735.1 (Klebsiella pneumoniae, INF323-sc-2280125) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890669.1 (Klebsiella pneumoniae, INF321-sc-2280120) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890254.1 (Klebsiella pneumoniae, INF310-sc-2280104) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890702.1 (Klebsiella pneumoniae, INF305-sc-2280094) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890556.1 (Klebsiella pneumoniae, INF302-sc-2280090) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890632.1 (Klebsiella pneumoniae, INF299-sc-2280084) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890426.1 (Klebsiella pneumoniae, INF298) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890515.1 (Klebsiella pneumoniae, INF281-sc-2280206) loaded with 19 markers.\n",
      "2024-04-17 18:12:12,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024563.1 (Klebsiella pneumoniae, INF278) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,729 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890204.1 (Klebsiella pneumoniae, INF277) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024570.1 (Klebsiella pneumoniae, INF274) loaded with 16 markers.\n",
      "2024-04-17 18:12:12,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890648.1 (Klebsiella pneumoniae, INF266-sc-2280182) loaded with 20 markers.\n",
      "2024-04-17 18:12:12,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890562.1 (Klebsiella pneumoniae, INF247-sc-2280148) loaded with 17 markers.\n",
      "2024-04-17 18:12:13,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890624.1 (Klebsiella pneumoniae, INF242-sc-2280140) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031817.1 (Klebsiella pneumoniae, INF235-sc-2280127) loaded with 21 markers.\n",
      "2024-04-17 18:12:13,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890595.1 (Klebsiella pneumoniae, INF223-sc-2280103) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,234 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890317.1 (Klebsiella pneumoniae, INF215-sc-2280087) loaded with 19 markers.\n",
      "2024-04-17 18:12:13,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031808.1 (Klebsiella pneumoniae, INF206-sc-2280074) loaded with 19 markers.\n",
      "2024-04-17 18:12:13,361 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890554.1 (Klebsiella pneumoniae, INF182-sc-2280049) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,438 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890446.1 (Klebsiella pneumoniae, INF171-sc-2280028) loaded with 19 markers.\n",
      "2024-04-17 18:12:13,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024556.1 (Klebsiella pneumoniae, INF164) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024549.1 (Klebsiella pneumoniae, INF163) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,618 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024521.1 (Klebsiella pneumoniae, INF158) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024528.1 (Klebsiella pneumoniae, INF157) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890381.1 (Klebsiella pneumoniae, INF148-sc-2279987) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,803 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890569.1 (Klebsiella pneumoniae, INF144-sc-2279979) loaded with 18 markers.\n",
      "2024-04-17 18:12:13,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890547.1 (Klebsiella pneumoniae, INF142-sc-2279977) loaded with 16 markers.\n",
      "2024-04-17 18:12:13,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890189.1 (Klebsiella pneumoniae, INF133-sc-2279960) loaded with 16 markers.\n",
      "2024-04-17 18:12:14,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083775.1 (Klebsiella pneumoniae, INF118) loaded with 40 markers.\n",
      "2024-04-17 18:12:14,103 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031792.1 (Klebsiella pneumoniae, INF116-sc-2279924) loaded with 17 markers.\n",
      "2024-04-17 18:12:14,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890726.1 (Klebsiella pneumoniae, INF102-sc-2279897) loaded with 30 markers.\n",
      "2024-04-17 18:12:14,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890464.1 (Klebsiella pneumoniae, INF079-sc-2280008) loaded with 20 markers.\n",
      "2024-04-17 18:12:14,312 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032831.1 (Klebsiella pneumoniae, INF078) loaded with 16 markers.\n",
      "2024-04-17 18:12:14,374 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890185.1 (Klebsiella pneumoniae, INF072-sc-2279995) loaded with 17 markers.\n",
      "2024-04-17 18:12:14,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024545.1 (Klebsiella pneumoniae, INF059) loaded with 18 markers.\n",
      "2024-04-17 18:12:14,539 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890360.1 (Klebsiella pneumoniae, INF049-sc-2279950) loaded with 22 markers.\n",
      "2024-04-17 18:12:14,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890395.1 (Klebsiella pneumoniae, INF044-sc-2279940) loaded with 18 markers.\n",
      "2024-04-17 18:12:14,708 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024542.1 (Klebsiella pneumoniae, INF042) loaded with 18 markers.\n",
      "2024-04-17 18:12:14,788 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890456.1 (Klebsiella pneumoniae, INF032-sc-2279918) loaded with 17 markers.\n",
      "2024-04-17 18:12:14,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890209.1 (Klebsiella pneumoniae, INF022-sc-2279895) loaded with 37 markers.\n",
      "2024-04-17 18:12:14,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083796.1 (Klebsiella pneumoniae, INF019) loaded with 18 markers.\n",
      "2024-04-17 18:12:15,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890493.1 (Klebsiella pneumoniae, INF018-sc-2279888) loaded with 16 markers.\n",
      "2024-04-17 18:12:15,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890202.1 (Klebsiella pneumoniae, INF008-sc-2279879) loaded with 16 markers.\n",
      "2024-04-17 18:12:15,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054285.1 (Klebsiella pneumoniae, IDR1800050501-01-02) loaded with 17 markers.\n",
      "2024-04-17 18:12:15,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030070.1 (Klebsiella pneumoniae, IA565) loaded with 16 markers.\n",
      "2024-04-17 18:12:15,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081890.1 (Klebsiella pneumoniae, IA-021) loaded with 18 markers.\n",
      "2024-04-17 18:12:15,444 [DEBUG - chronostrain.database.parser.json] - Strain NC_016845.1 (Klebsiella pneumoniae, HS11286) loaded with 17 markers.\n",
      "2024-04-17 18:12:15,521 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006738.1 (Klebsiella pneumoniae, HK787) loaded with 16 markers.\n",
      "2024-04-17 18:12:15,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061842.1 (Klebsiella pneumoniae, HC139) loaded with 18 markers.\n",
      "2024-04-17 18:12:15,691 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039524.1 (Klebsiella pneumoniae, HB25-1) loaded with 25 markers.\n",
      "2024-04-17 18:12:15,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031934.1 (Klebsiella pneumoniae, GZ-1) loaded with 34 markers.\n",
      "2024-04-17 18:12:15,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018671.1 (Klebsiella pneumoniae, GSU10-3) loaded with 37 markers.\n",
      "2024-04-17 18:12:15,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075890.1 (Klebsiella pneumoniae, GR390) loaded with 18 markers.\n",
      "2024-04-17 18:12:16,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065460.1 (Klebsiella pneumoniae, Gordon) loaded with 44 markers.\n",
      "2024-04-17 18:12:16,190 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025951.1 (Klebsiella pneumoniae, GD4) loaded with 29 markers.\n",
      "2024-04-17 18:12:16,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063759.1 (Klebsiella pneumoniae, FRPDR) loaded with 38 markers.\n",
      "2024-04-17 18:12:16,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073002.1 (Klebsiella pneumoniae, FO15) loaded with 30 markers.\n",
      "2024-04-17 18:12:16,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065554.1 (Klebsiella pneumoniae, FK_6768) loaded with 36 markers.\n",
      "2024-04-17 18:12:16,597 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044047.1 (Klebsiella pneumoniae, FDAARGOS_629) loaded with 18 markers.\n",
      "2024-04-17 18:12:16,661 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023946.1 (Klebsiella pneumoniae, FDAARGOS_446) loaded with 18 markers.\n",
      "2024-04-17 18:12:16,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023941.1 (Klebsiella pneumoniae, FDAARGOS_444) loaded with 33 markers.\n",
      "2024-04-17 18:12:16,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070087.1 (Klebsiella pneumoniae, FDAARGOS_1328) loaded with 25 markers.\n",
      "2024-04-17 18:12:16,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069828.1 (Klebsiella pneumoniae, FDAARGOS_1327) loaded with 20 markers.\n",
      "2024-04-17 18:12:16,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070037.1 (Klebsiella pneumoniae, FDAARGOS_1322) loaded with 16 markers.\n",
      "2024-04-17 18:12:17,054 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069939.1 (Klebsiella pneumoniae, FDAARGOS_1318) loaded with 17 markers.\n",
      "2024-04-17 18:12:17,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069961.1 (Klebsiella pneumoniae, FDAARGOS_1317) loaded with 16 markers.\n",
      "2024-04-17 18:12:17,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069907.1 (Klebsiella pneumoniae, FDAARGOS_1313) loaded with 18 markers.\n",
      "2024-04-17 18:12:17,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069940.1 (Klebsiella pneumoniae, FDAARGOS_1312) loaded with 20 markers.\n",
      "2024-04-17 18:12:17,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069861.1 (Klebsiella pneumoniae, FDAARGOS_1311) loaded with 19 markers.\n",
      "2024-04-17 18:12:17,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068139.1 (Klebsiella pneumoniae, FDAARGOS_1076) loaded with 18 markers.\n",
      "2024-04-17 18:12:17,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090398.1 (Klebsiella pneumoniae, F94) loaded with 35 markers.\n",
      "2024-04-17 18:12:17,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026159.1 (Klebsiella pneumoniae, F89-1) loaded with 16 markers.\n",
      "2024-04-17 18:12:17,628 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026164.1 (Klebsiella pneumoniae, F81) loaded with 19 markers.\n",
      "2024-04-17 18:12:17,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026136.1 (Klebsiella pneumoniae, F77) loaded with 35 markers.\n",
      "2024-04-17 18:12:17,897 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081820.1 (Klebsiella pneumoniae, F726925) loaded with 35 markers.\n",
      "2024-04-17 18:12:18,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026132.1 (Klebsiella pneumoniae, F5) loaded with 34 markers.\n",
      "2024-04-17 18:12:18,128 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025461.1 (Klebsiella pneumoniae, F44) loaded with 32 markers.\n",
      "2024-04-17 18:12:18,238 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026149.1 (Klebsiella pneumoniae, F138) loaded with 32 markers.\n",
      "2024-04-17 18:12:18,350 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026145.1 (Klebsiella pneumoniae, F132) loaded with 33 markers.\n",
      "2024-04-17 18:12:18,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026140.1 (Klebsiella pneumoniae, F127) loaded with 35 markers.\n",
      "2024-04-17 18:12:18,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026130.1 (Klebsiella pneumoniae, F1) loaded with 35 markers.\n",
      "2024-04-17 18:12:18,675 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061350.1 (Klebsiella pneumoniae, ENT1988) loaded with 16 markers.\n",
      "2024-04-17 18:12:18,766 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016814.1 (Klebsiella pneumoniae, ED23) loaded with 20 markers.\n",
      "2024-04-17 18:12:18,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016813.1 (Klebsiella pneumoniae, ED2) loaded with 20 markers.\n",
      "2024-04-17 18:12:18,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR025099.1 (Klebsiella pneumoniae, EC-12536) loaded with 18 markers.\n",
      "2024-04-17 18:12:18,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039960.1 (Klebsiella pneumoniae, EAN40) loaded with 18 markers.\n",
      "2024-04-17 18:12:19,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039957.1 (Klebsiella pneumoniae, EAN23) loaded with 18 markers.\n",
      "2024-04-17 18:12:19,124 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011976.1 (Klebsiella pneumoniae, DMC1097) loaded with 26 markers.\n",
      "2024-04-17 18:12:19,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065436.1 (Klebsiella pneumoniae, Dirty_Dan) loaded with 19 markers.\n",
      "2024-04-17 18:12:19,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066254.1 (Klebsiella pneumoniae, CRKP78R) loaded with 33 markers.\n",
      "2024-04-17 18:12:19,390 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063833.1 (Klebsiella pneumoniae, CRKP66R) loaded with 37 markers.\n",
      "2024-04-17 18:12:19,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066249.1 (Klebsiella pneumoniae, CRKP52R) loaded with 32 markers.\n",
      "2024-04-17 18:12:19,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034039.1 (Klebsiella pneumoniae, CRK0298) loaded with 23 markers.\n",
      "2024-04-17 18:12:19,646 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063019.1 (Klebsiella pneumoniae, CriePir75) loaded with 21 markers.\n",
      "2024-04-17 18:12:19,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062997.1 (Klebsiella pneumoniae, CriePir197) loaded with 22 markers.\n",
      "2024-04-17 18:12:19,787 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063003.1 (Klebsiella pneumoniae, CriePir154) loaded with 17 markers.\n",
      "2024-04-17 18:12:19,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063008.1 (Klebsiella pneumoniae, CriePir120) loaded with 16 markers.\n",
      "2024-04-17 18:12:19,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063011.1 (Klebsiella pneumoniae, CriePir116) loaded with 16 markers.\n",
      "2024-04-17 18:12:19,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015392.1 (Klebsiella pneumoniae, CR14) loaded with 28 markers.\n",
      "2024-04-17 18:12:20,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040545.1 (Klebsiella pneumoniae, CR-HvKP5) loaded with 31 markers.\n",
      "2024-04-17 18:12:20,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040539.1 (Klebsiella pneumoniae, CR-HvKP4) loaded with 31 markers.\n",
      "2024-04-17 18:12:20,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040533.1 (Klebsiella pneumoniae, CR-HvKP1) loaded with 31 markers.\n",
      "2024-04-17 18:12:20,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073351.1 (Klebsiella pneumoniae, CP19) loaded with 31 markers.\n",
      "2024-04-17 18:12:20,459 [DEBUG - chronostrain.database.parser.json] - Strain NC_022566.1 (Klebsiella pneumoniae, CG43) loaded with 16 markers.\n",
      "2024-04-17 18:12:20,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077773.1 (Klebsiella pneumoniae, CDI694) loaded with 32 markers.\n",
      "2024-04-17 18:12:20,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035540.1 (Klebsiella pneumoniae, CCRI-22199) loaded with 17 markers.\n",
      "2024-04-17 18:12:20,720 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011647.1 (Klebsiella pneumoniae, CAV1596) loaded with 31 markers.\n",
      "2024-04-17 18:12:20,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018356.1 (Klebsiella pneumoniae, CAV1453) loaded with 21 markers.\n",
      "2024-04-17 18:12:20,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018352.1 (Klebsiella pneumoniae, CAV1417) loaded with 20 markers.\n",
      "2024-04-17 18:12:20,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011578.1 (Klebsiella pneumoniae, CAV1392) loaded with 20 markers.\n",
      "2024-04-17 18:12:21,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011624.1 (Klebsiella pneumoniae, CAV1344) loaded with 27 markers.\n",
      "2024-04-17 18:12:21,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018676.1 (Klebsiella pneumoniae, CAV1217) loaded with 19 markers.\n",
      "2024-04-17 18:12:21,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013322.1 (Klebsiella pneumoniae, CAV1193) loaded with 26 markers.\n",
      "2024-04-17 18:12:21,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009114.1 (Klebsiella pneumoniae, carbapenem-resistant_blaNDM-1) loaded with 20 markers.\n",
      "2024-04-17 18:12:21,334 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034415.1 (Klebsiella pneumoniae, C789) loaded with 32 markers.\n",
      "2024-04-17 18:12:21,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042481.1 (Klebsiella pneumoniae, C51) loaded with 18 markers.\n",
      "2024-04-17 18:12:21,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039808.1 (Klebsiella pneumoniae, C2660) loaded with 36 markers.\n",
      "2024-04-17 18:12:21,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039819.1 (Klebsiella pneumoniae, C2414) loaded with 34 markers.\n",
      "2024-04-17 18:12:21,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071464.1 (Klebsiella pneumoniae, C2244) loaded with 35 markers.\n",
      "2024-04-17 18:12:21,778 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080584.1 (Klebsiella pneumoniae, C1398_mutant_C1) loaded with 32 markers.\n",
      "2024-04-17 18:12:21,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020498.1 (Klebsiella pneumoniae, BWHC1) loaded with 25 markers.\n",
      "2024-04-17 18:12:21,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066543.1 (Klebsiella pneumoniae, BTS) loaded with 40 markers.\n",
      "2024-04-17 18:12:22,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063890.1 (Klebsiella pneumoniae, BS433-2) loaded with 20 markers.\n",
      "2024-04-17 18:12:22,093 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063885.1 (Klebsiella pneumoniae, BS361-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063943.1 (Klebsiella pneumoniae, BS329-2) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063934.1 (Klebsiella pneumoniae, BS326-3) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063936.1 (Klebsiella pneumoniae, BS317-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018883.1 (Klebsiella pneumoniae, BR7) loaded with 18 markers.\n",
      "2024-04-17 18:12:22,411 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018885.1 (Klebsiella pneumoniae, BR21) loaded with 18 markers.\n",
      "2024-04-17 18:12:22,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063939.1 (Klebsiella pneumoniae, BM343) loaded with 18 markers.\n",
      "2024-04-17 18:12:22,549 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063919.1 (Klebsiella pneumoniae, BM337-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063913.1 (Klebsiella pneumoniae, BM336-2-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063904.1 (Klebsiella pneumoniae, BM334-2) loaded with 16 markers.\n",
      "2024-04-17 18:12:22,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063906.1 (Klebsiella pneumoniae, BM327-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:22,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015822.1 (Klebsiella pneumoniae, blood_sample_2) loaded with 28 markers.\n",
      "2024-04-17 18:12:22,899 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020837.1 (Klebsiella pneumoniae, BK13043) loaded with 24 markers.\n",
      "2024-04-17 18:12:22,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034123.1 (Klebsiella pneumoniae, BJCFK909) loaded with 31 markers.\n",
      "2024-04-17 18:12:23,070 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022573.1 (Klebsiella pneumoniae, BIC-1) loaded with 25 markers.\n",
      "2024-04-17 18:12:23,140 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065453.1 (Klebsiella pneumoniae, Beach_Ranger) loaded with 18 markers.\n",
      "2024-04-17 18:12:23,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082967.1 (Klebsiella pneumoniae, BD-41) loaded with 19 markers.\n",
      "2024-04-17 18:12:23,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082975.1 (Klebsiella pneumoniae, BD-14) loaded with 18 markers.\n",
      "2024-04-17 18:12:23,342 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082978.1 (Klebsiella pneumoniae, BD-13) loaded with 17 markers.\n",
      "2024-04-17 18:12:23,405 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050845.1 (Klebsiella pneumoniae, Bckp206) loaded with 19 markers.\n",
      "2024-04-17 18:12:23,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050840.1 (Klebsiella pneumoniae, Bckp101) loaded with 18 markers.\n",
      "2024-04-17 18:12:23,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050822.1 (Klebsiella pneumoniae, Bckp091) loaded with 19 markers.\n",
      "2024-04-17 18:12:23,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050829.1 (Klebsiella pneumoniae, Bckp067) loaded with 17 markers.\n",
      "2024-04-17 18:12:23,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092753.1 (Klebsiella pneumoniae, bat17) loaded with 28 markers.\n",
      "2024-04-17 18:12:23,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092759.1 (Klebsiella pneumoniae, bat16) loaded with 28 markers.\n",
      "2024-04-17 18:12:23,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092765.1 (Klebsiella pneumoniae, bat15) loaded with 28 markers.\n",
      "2024-04-17 18:12:23,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036190.1 (Klebsiella pneumoniae, BA34918) loaded with 20 markers.\n",
      "2024-04-17 18:12:23,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036327.1 (Klebsiella pneumoniae, BA28434) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053364.1 (Klebsiella pneumoniae, BA2275) loaded with 19 markers.\n",
      "2024-04-17 18:12:24,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054170.1 (Klebsiella pneumoniae, BA12818) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070410.1 (Klebsiella pneumoniae, B960) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072408.1 (Klebsiella pneumoniae, B4650) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,286 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070430.1 (Klebsiella pneumoniae, B3923) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070425.1 (Klebsiella pneumoniae, B3824) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072413.1 (Klebsiella pneumoniae, B32278) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070459.1 (Klebsiella pneumoniae, B28484) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,616 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070449.1 (Klebsiella pneumoniae, B18064) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,707 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070415.1 (Klebsiella pneumoniae, B1347) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072403.1 (Klebsiella pneumoniae, B13243) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070445.1 (Klebsiella pneumoniae, B12097) loaded with 18 markers.\n",
      "2024-04-17 18:12:24,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026155.1 (Klebsiella pneumoniae, B12(AN)) loaded with 17 markers.\n",
      "2024-04-17 18:12:25,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072456.1 (Klebsiella pneumoniae, b1-2L) loaded with 19 markers.\n",
      "2024-04-17 18:12:25,108 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025008.1 (Klebsiella pneumoniae, AUSMDU00008119) loaded with 25 markers.\n",
      "2024-04-17 18:12:25,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022691.2 (Klebsiella pneumoniae, AUSMDU00008079) loaded with 25 markers.\n",
      "2024-04-17 18:12:25,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025005.1 (Klebsiella pneumoniae, AUSMDU00003562) loaded with 27 markers.\n",
      "2024-04-17 18:12:25,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006659.2 (Klebsiella pneumoniae, ATCC_BAA-2146) loaded with 19 markers.\n",
      "2024-04-17 18:12:25,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015134.1 (Klebsiella pneumoniae, ATCC_35657) loaded with 19 markers.\n",
      "2024-04-17 18:12:25,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030341.1 (Klebsiella pneumoniae, AR_362) loaded with 24 markers.\n",
      "2024-04-17 18:12:25,568 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027146.1 (Klebsiella pneumoniae, AR_0363) loaded with 28 markers.\n",
      "2024-04-17 18:12:25,635 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027160.1 (Klebsiella pneumoniae, AR_0361) loaded with 22 markers.\n",
      "2024-04-17 18:12:25,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032175.1 (Klebsiella pneumoniae, AR_0160) loaded with 16 markers.\n",
      "2024-04-17 18:12:25,756 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021696.1 (Klebsiella pneumoniae, AR_0158) loaded with 16 markers.\n",
      "2024-04-17 18:12:25,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021950.1 (Klebsiella pneumoniae, AR_0148) loaded with 17 markers.\n",
      "2024-04-17 18:12:25,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021685.1 (Klebsiella pneumoniae, AR_0146) loaded with 17 markers.\n",
      "2024-04-17 18:12:25,993 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021960.1 (Klebsiella pneumoniae, AR_0139) loaded with 20 markers.\n",
      "2024-04-17 18:12:26,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032178.1 (Klebsiella pneumoniae, AR_0135) loaded with 17 markers.\n",
      "2024-04-17 18:12:26,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021718.1 (Klebsiella pneumoniae, AR_0129) loaded with 26 markers.\n",
      "2024-04-17 18:12:26,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021859.1 (Klebsiella pneumoniae, AR_0125) loaded with 25 markers.\n",
      "2024-04-17 18:12:26,297 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021833.1 (Klebsiella pneumoniae, AR_0120) loaded with 27 markers.\n",
      "2024-04-17 18:12:26,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020071.1 (Klebsiella pneumoniae, AR_0115) loaded with 24 markers.\n",
      "2024-04-17 18:12:26,450 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021751.1 (Klebsiella pneumoniae, AR_0113) loaded with 23 markers.\n",
      "2024-04-17 18:12:26,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021549.1 (Klebsiella pneumoniae, AR_0112) loaded with 23 markers.\n",
      "2024-04-17 18:12:26,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032207.1 (Klebsiella pneumoniae, AR_0109) loaded with 17 markers.\n",
      "2024-04-17 18:12:26,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021955.1 (Klebsiella pneumoniae, AR_0107) loaded with 21 markers.\n",
      "2024-04-17 18:12:26,729 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020108.1 (Klebsiella pneumoniae, AR_0098) loaded with 29 markers.\n",
      "2024-04-17 18:12:26,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032200.1 (Klebsiella pneumoniae, AR_0097) loaded with 17 markers.\n",
      "2024-04-17 18:12:26,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027612.1 (Klebsiella pneumoniae, AR_0096) loaded with 17 markers.\n",
      "2024-04-17 18:12:26,917 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028994.1 (Klebsiella pneumoniae, AR_0079) loaded with 21 markers.\n",
      "2024-04-17 18:12:26,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032167.1 (Klebsiella pneumoniae, AR_0076) loaded with 17 markers.\n",
      "2024-04-17 18:12:27,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018816.1 (Klebsiella pneumoniae, AR_0049) loaded with 19 markers.\n",
      "2024-04-17 18:12:27,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021539.1 (Klebsiella pneumoniae, AR_0047) loaded with 29 markers.\n",
      "2024-04-17 18:12:27,173 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033946.1 (Klebsiella pneumoniae, ARLG-3135) loaded with 18 markers.\n",
      "2024-04-17 18:12:27,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029099.1 (Klebsiella pneumoniae, AR438) loaded with 28 markers.\n",
      "2024-04-17 18:12:27,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035383.1 (Klebsiella pneumoniae, AP8555) loaded with 20 markers.\n",
      "2024-04-17 18:12:27,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130541.1 (Klebsiella pneumoniae, AJ218) loaded with 21 markers.\n",
      "2024-04-17 18:12:27,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091048.1 (Klebsiella pneumoniae, AHSWKP25) loaded with 16 markers.\n",
      "2024-04-17 18:12:27,539 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022882.1 (Klebsiella pneumoniae, 911021) loaded with 33 markers.\n",
      "2024-04-17 18:12:27,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071819.1 (Klebsiella pneumoniae, 90CM2) loaded with 18 markers.\n",
      "2024-04-17 18:12:27,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085889.1 (Klebsiella pneumoniae, 8695) loaded with 38 markers.\n",
      "2024-04-17 18:12:27,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030292.1 (Klebsiella pneumoniae, 721005) loaded with 29 markers.\n",
      "2024-04-17 18:12:27,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022997.1 (Klebsiella pneumoniae, 721005) loaded with 29 markers.\n",
      "2024-04-17 18:12:28,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082985.1 (Klebsiella pneumoniae, 7011.62) loaded with 18 markers.\n",
      "2024-04-17 18:12:28,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082995.1 (Klebsiella pneumoniae, 6911.28) loaded with 20 markers.\n",
      "2024-04-17 18:12:28,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083000.1 (Klebsiella pneumoniae, 6712.08) loaded with 17 markers.\n",
      "2024-04-17 18:12:28,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083007.1 (Klebsiella pneumoniae, 6711.43) loaded with 39 markers.\n",
      "2024-04-17 18:12:28,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083011.1 (Klebsiella pneumoniae, 6710.71) loaded with 17 markers.\n",
      "2024-04-17 18:12:28,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083016.1 (Klebsiella pneumoniae, 6709.15-I) loaded with 23 markers.\n",
      "2024-04-17 18:12:28,580 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083021.1 (Klebsiella pneumoniae, 6611.48) loaded with 23 markers.\n",
      "2024-04-17 18:12:28,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083026.1 (Klebsiella pneumoniae, 6604.68) loaded with 23 markers.\n",
      "2024-04-17 18:12:28,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068602.1 (Klebsiella pneumoniae, 59493) loaded with 30 markers.\n",
      "2024-04-17 18:12:28,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085729.1 (Klebsiella pneumoniae, 59062CZ) loaded with 19 markers.\n",
      "2024-04-17 18:12:28,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088126.1 (Klebsiella pneumoniae, 57) loaded with 17 markers.\n",
      "2024-04-17 18:12:29,020 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089012.1 (Klebsiella pneumoniae, 56575) loaded with 16 markers.\n",
      "2024-04-17 18:12:29,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089014.1 (Klebsiella pneumoniae, 53374) loaded with 21 markers.\n",
      "2024-04-17 18:12:29,216 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088982.1 (Klebsiella pneumoniae, 53357) loaded with 20 markers.\n",
      "2024-04-17 18:12:29,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085740.1 (Klebsiella pneumoniae, 51135CZ) loaded with 32 markers.\n",
      "2024-04-17 18:12:29,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088991.1 (Klebsiella pneumoniae, 50700) loaded with 32 markers.\n",
      "2024-04-17 18:12:29,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050371.1 (Klebsiella pneumoniae, 50595) loaded with 19 markers.\n",
      "2024-04-17 18:12:29,633 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011980.1 (Klebsiella pneumoniae, 500_1420) loaded with 24 markers.\n",
      "2024-04-17 18:12:29,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042320.1 (Klebsiella pneumoniae, 5-16) loaded with 31 markers.\n",
      "2024-04-17 18:12:29,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089024.1 (Klebsiella pneumoniae, 49210) loaded with 20 markers.\n",
      "2024-04-17 18:12:29,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088997.1 (Klebsiella pneumoniae, 49088) loaded with 34 markers.\n",
      "2024-04-17 18:12:30,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083036.1 (Klebsiella pneumoniae, 4906.28) loaded with 17 markers.\n",
      "2024-04-17 18:12:30,157 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033625.1 (Klebsiella pneumoniae, 4743) loaded with 18 markers.\n",
      "2024-04-17 18:12:30,281 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042318.1 (Klebsiella pneumoniae, 46-16) loaded with 31 markers.\n",
      "2024-04-17 18:12:30,368 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088120.1 (Klebsiella pneumoniae, 46) loaded with 17 markers.\n",
      "2024-04-17 18:12:30,464 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068610.1 (Klebsiella pneumoniae, 45706) loaded with 29 markers.\n",
      "2024-04-17 18:12:30,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068614.1 (Klebsiella pneumoniae, 42362) loaded with 27 markers.\n",
      "2024-04-17 18:12:30,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068618.1 (Klebsiella pneumoniae, 41968) loaded with 28 markers.\n",
      "2024-04-17 18:12:30,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054268.1 (Klebsiella pneumoniae, 39427) loaded with 29 markers.\n",
      "2024-04-17 18:12:30,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082753.1 (Klebsiella pneumoniae, 37) loaded with 32 markers.\n",
      "2024-04-17 18:12:30,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082759.1 (Klebsiella pneumoniae, 36) loaded with 32 markers.\n",
      "2024-04-17 18:12:31,045 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010392.1 (Klebsiella pneumoniae, 34618) loaded with 29 markers.\n",
      "2024-04-17 18:12:31,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069046.1 (Klebsiella pneumoniae, 33Kpn22) loaded with 26 markers.\n",
      "2024-04-17 18:12:31,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071086.1 (Klebsiella pneumoniae, 3347689I) loaded with 20 markers.\n",
      "2024-04-17 18:12:31,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071080.1 (Klebsiella pneumoniae, 3347684I) loaded with 16 markers.\n",
      "2024-04-17 18:12:31,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028850.1 (Klebsiella pneumoniae, 3214) loaded with 20 markers.\n",
      "2024-04-17 18:12:31,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006918.1 (Klebsiella pneumoniae, 30684_NJST258_2) loaded with 25 markers.\n",
      "2024-04-17 18:12:31,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006923.1 (Klebsiella pneumoniae, 30660_NJST258_1) loaded with 22 markers.\n",
      "2024-04-17 18:12:31,731 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027053.1 (Klebsiella pneumoniae, 2_GR_12) loaded with 32 markers.\n",
      "2024-04-17 18:12:31,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025541.3 (Klebsiella pneumoniae, 2N3) loaded with 18 markers.\n",
      "2024-04-17 18:12:31,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030300.1 (Klebsiella pneumoniae, 283747) loaded with 35 markers.\n",
      "2024-04-17 18:12:31,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030320.1 (Klebsiella pneumoniae, 23) loaded with 16 markers.\n",
      "2024-04-17 18:12:31,999 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027048.1 (Klebsiella pneumoniae, 20_GR_12) loaded with 25 markers.\n",
      "2024-04-17 18:12:32,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047336.1 (Klebsiella pneumoniae, 2019036D) loaded with 19 markers.\n",
      "2024-04-17 18:12:32,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068015.1 (Klebsiella pneumoniae, 2016_49) loaded with 20 markers.\n",
      "2024-04-17 18:12:32,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065034.1 (Klebsiella pneumoniae, 2016-1400) loaded with 17 markers.\n",
      "2024-04-17 18:12:32,236 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055293.1 (Klebsiella pneumoniae, 200) loaded with 17 markers.\n",
      "2024-04-17 18:12:32,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051159.1 (Klebsiella pneumoniae, 19PDR22) loaded with 33 markers.\n",
      "2024-04-17 18:12:32,370 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039948.1 (Klebsiella pneumoniae, 19KM57) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039945.1 (Klebsiella pneumoniae, 19KM28) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049947.1 (Klebsiella pneumoniae, 19KM267) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049953.1 (Klebsiella pneumoniae, 19KM1109) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049950.1 (Klebsiella pneumoniae, 19KM1053) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064174.1 (Klebsiella pneumoniae, 19110124) loaded with 16 markers.\n",
      "2024-04-17 18:12:32,731 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039942.1 (Klebsiella pneumoniae, 18KM2813) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039939.1 (Klebsiella pneumoniae, 18KM2517) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,851 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039936.1 (Klebsiella pneumoniae, 18KM1950) loaded with 18 markers.\n",
      "2024-04-17 18:12:32,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034778.1 (Klebsiella pneumoniae, 18CPO060) loaded with 16 markers.\n",
      "2024-04-17 18:12:32,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084492.1 (Klebsiella pneumoniae, 1864) loaded with 35 markers.\n",
      "2024-04-17 18:12:33,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027036.1 (Klebsiella pneumoniae, 16_GR_13) loaded with 18 markers.\n",
      "2024-04-17 18:12:33,131 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045263.1 (Klebsiella pneumoniae, 16HN-263) loaded with 35 markers.\n",
      "2024-04-17 18:12:33,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051161.1 (Klebsiella pneumoniae, 16BU137) loaded with 17 markers.\n",
      "2024-04-17 18:12:33,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084497.1 (Klebsiella pneumoniae, 1632) loaded with 35 markers.\n",
      "2024-04-17 18:12:33,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083055.1 (Klebsiella pneumoniae, 1613258639) loaded with 18 markers.\n",
      "2024-04-17 18:12:33,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029689.1 (Klebsiella pneumoniae, 160111) loaded with 37 markers.\n",
      "2024-04-17 18:12:33,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047649.1 (Klebsiella pneumoniae, 158590) loaded with 34 markers.\n",
      "2024-04-17 18:12:33,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047648.1 (Klebsiella pneumoniae, 156070) loaded with 34 markers.\n",
      "2024-04-17 18:12:33,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030172.1 (Klebsiella pneumoniae, 12208) loaded with 19 markers.\n",
      "2024-04-17 18:12:33,732 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031849.1 (Klebsiella pneumoniae, 121) loaded with 20 markers.\n",
      "2024-04-17 18:12:33,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082765.1 (Klebsiella pneumoniae, 12) loaded with 38 markers.\n",
      "2024-04-17 18:12:33,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049716.1 (Klebsiella pneumoniae, 11KM256) loaded with 18 markers.\n",
      "2024-04-17 18:12:33,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089427.1 (Klebsiella pneumoniae, 1194) loaded with 17 markers.\n",
      "2024-04-17 18:12:34,006 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026021.1 (Klebsiella pneumoniae, 11492) loaded with 20 markers.\n",
      "2024-04-17 18:12:34,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030311.1 (Klebsiella pneumoniae, 11454) loaded with 16 markers.\n",
      "2024-04-17 18:12:34,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026023.1 (Klebsiella pneumoniae, 11420) loaded with 20 markers.\n",
      "2024-04-17 18:12:34,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030313.1 (Klebsiella pneumoniae, 11311) loaded with 16 markers.\n",
      "2024-04-17 18:12:34,251 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030174.1 (Klebsiella pneumoniae, 11219) loaded with 20 markers.\n",
      "2024-04-17 18:12:34,319 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide R (possible IUPAC code) in NZ_CP042309.1,IDX=0, while parsing marker insH7:BLAST<9703>. Skipping strain NZ_CP042309.1.\n",
      "2024-04-17 18:12:34,381 [DEBUG - chronostrain.database.parser.json] - Strain NC_018522.1 (Klebsiella pneumoniae, 1084) loaded with 20 markers.\n",
      "2024-04-17 18:12:34,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023416.1 (Klebsiella pneumoniae, 1050) loaded with 20 markers.\n",
      "2024-04-17 18:12:34,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042310.1 (Klebsiella pneumoniae, 103-17) loaded with 31 markers.\n",
      "2024-04-17 18:12:34,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042311.1 (Klebsiella pneumoniae, 101-17) loaded with 31 markers.\n",
      "2024-04-17 18:12:34,774 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042321.1 (Klebsiella pneumoniae, 1-16) loaded with 30 markers.\n",
      "2024-04-17 18:12:34,849 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025576.1 (Klebsiella pneumoniae, 08EU827) loaded with 16 markers.\n",
      "2024-04-17 18:12:34,938 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049719.1 (Klebsiella pneumoniae, 06KM907) loaded with 18 markers.\n",
      "2024-04-17 18:12:35,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033396.2 (Klebsiella pneumoniae, 015625) loaded with 36 markers.\n",
      "2024-04-17 18:12:35,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055309.1 (Klebsiella grimontii, RHBSTW-00866) loaded with 32 markers.\n",
      "2024-04-17 18:12:35,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079754.1 (Klebsiella grimontii, NY3) loaded with 23 markers.\n",
      "2024-04-17 18:12:35,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091752.1 (Klebsiella grimontii, LC-1303-2020) loaded with 20 markers.\n",
      "2024-04-17 18:12:35,467 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059391.1 (Klebsiella africana, FF1003) loaded with 17 markers.\n",
      "2024-04-17 18:12:35,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084874.1 (Klebsiella africana, 200023) loaded with 17 markers.\n",
      "2024-04-17 18:12:35,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134254.1 (Klebsiella aerogenes, NCTC9644) loaded with 20 markers.\n",
      "2024-04-17 18:12:35,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061801.1 (Enterobacter mori, HSW1412) loaded with 40 markers.\n",
      "2024-04-17 18:12:35,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022133.1 (Enterobacter cloacae, WP5-S18-ESBL-01) loaded with 36 markers.\n",
      "2024-04-17 18:12:35,777 [DEBUG - chronostrain.database.parser.json] - Strain NC_021870.1 (Salmonella bongori, N268-08) loaded with 34 markers.\n",
      "2024-04-17 18:12:35,833 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053336.1 (Salmonella bongori, 08-0158) loaded with 34 markers.\n",
      "2024-04-17 18:12:35,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026167.1 (Leclercia sp., LSNIH1) loaded with 37 markers.\n",
      "2024-04-17 18:12:35,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046446.1 (Leclercia sp., J807) loaded with 32 markers.\n",
      "2024-04-17 18:12:36,027 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049786.1 (Leclercia sp., 29361) loaded with 31 markers.\n",
      "2024-04-17 18:12:36,094 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046557.1 (Leclercia sp., 1106151) loaded with 35 markers.\n",
      "2024-04-17 18:12:36,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074437.1 (Klebsiella sp., A52) loaded with 16 markers.\n",
      "2024-04-17 18:12:36,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041925.1 (Klebsiella aerogenes, Ka37751) loaded with 17 markers.\n",
      "2024-04-17 18:12:36,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049600.1 (Klebsiella aerogenes, 18-2341) loaded with 17 markers.\n",
      "2024-04-17 18:12:36,326 [DEBUG - chronostrain.database.parser.json] - Strain NC_015968.1 (Enterobacter soli, LF7a) loaded with 30 markers.\n",
      "2024-04-17 18:12:36,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058637.1 (Enterobacter roggenkampii, KQ-01) loaded with 37 markers.\n",
      "2024-04-17 18:12:36,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022446.1 (Enterobacter kobei, STW0522-60) loaded with 47 markers.\n",
      "2024-04-17 18:12:36,696 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022431.1 (Enterobacter kobei, STW0522-51) loaded with 35 markers.\n",
      "2024-04-17 18:12:36,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022498.1 (Enterobacter kobei, STN0717-53) loaded with 49 markers.\n",
      "2024-04-17 18:12:36,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059481.1 (Enterobacter kobei, IB2020) loaded with 48 markers.\n",
      "2024-04-17 18:12:36,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043511.1 (Enterobacter kobei, EB_P8_L5_01.19) loaded with 53 markers.\n",
      "2024-04-17 18:12:37,078 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017181.1 (Enterobacter kobei, DSM_13645) loaded with 47 markers.\n",
      "2024-04-17 18:12:37,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088119.1 (Enterobacter kobei, 6) loaded with 38 markers.\n",
      "2024-04-17 18:12:37,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028950.1 (Enterobacter cloacae, FDA-CDC-AR_0164) loaded with 35 markers.\n",
      "2024-04-17 18:12:37,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017990.1 (Enterobacter cloacae, ECNIH7) loaded with 35 markers.\n",
      "2024-04-17 18:12:37,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082147.1 (Enterobacter cloacae, ECL42) loaded with 47 markers.\n",
      "2024-04-17 18:12:37,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056711.1 (Enterobacter asburiae, RHBSTW-00542) loaded with 44 markers.\n",
      "2024-04-17 18:12:37,511 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019632.1 (Enterobacter asburiae, 1808-013) loaded with 41 markers.\n",
      "2024-04-17 18:12:37,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053573.1 (Citrobacter sp., TSA-1) loaded with 44 markers.\n",
      "2024-04-17 18:12:37,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056180.1 (Citrobacter sp., RHBSTW-01044) loaded with 42 markers.\n",
      "2024-04-17 18:12:37,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056717.1 (Citrobacter sp., RHBSTW-00599) loaded with 45 markers.\n",
      "2024-04-17 18:12:37,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029727.1 (Citrobacter sp., AR_0157) loaded with 46 markers.\n",
      "2024-04-17 18:12:38,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012554.1 (Citrobacter portucalensis, P10159) loaded with 45 markers.\n",
      "2024-04-17 18:12:38,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022378.1 (Citrobacter portucalensis, IOMTU157) loaded with 43 markers.\n",
      "2024-04-17 18:12:38,170 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066089.1 (Citrobacter koseri, FDAARGOS_1029) loaded with 40 markers.\n",
      "2024-04-17 18:12:38,231 [DEBUG - chronostrain.database.parser.json] - Strain NC_009792.1 (Citrobacter koseri, ATCC_BAA-895) loaded with 40 markers.\n",
      "2024-04-17 18:12:38,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024679.1 (Citrobacter freundii, UMH15) loaded with 45 markers.\n",
      "2024-04-17 18:12:38,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024680.1 (Citrobacter freundii, UMH14) loaded with 42 markers.\n",
      "2024-04-17 18:12:38,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092493.1 (Citrobacter freundii, SCLZS47) loaded with 49 markers.\n",
      "2024-04-17 18:12:38,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057146.1 (Citrobacter freundii, RHB35-C22) loaded with 45 markers.\n",
      "2024-04-17 18:12:38,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057149.1 (Citrobacter freundii, RHB35-C18) loaded with 45 markers.\n",
      "2024-04-17 18:12:38,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033742.1 (Citrobacter freundii, FDAARGOS_550) loaded with 43 markers.\n",
      "2024-04-17 18:12:38,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033744.1 (Citrobacter freundii, FDAARGOS_549) loaded with 42 markers.\n",
      "2024-04-17 18:12:38,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS992183.1 (Citrobacter freundii, Citrobacter_freundii_str._U2785) loaded with 48 markers.\n",
      "2024-04-17 18:12:39,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020448.2 (Citrobacter braakii, FDAARGOS_253) loaded with 42 markers.\n",
      "2024-04-17 18:12:39,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077405.1 (Citrobacter braakii, FDAARGOS_1421) loaded with 44 markers.\n",
      "2024-04-17 18:12:39,174 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080539.1 (Citrobacter braakii, DY2019) loaded with 43 markers.\n",
      "2024-04-17 18:12:39,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065167.1 (Klebsiella pneumoniae, KPN114) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890369.1 (Klebsiella pneumoniae, INF354-sc-2280183) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890415.1 (Klebsiella pneumoniae, INF313-sc-2280106) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890679.1 (Klebsiella pneumoniae, INF279-sc-2280202) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890374.1 (Klebsiella pneumoniae, INF250-sc-2280154) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,539 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024489.1 (Klebsiella pneumoniae, INF249) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,596 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890600.1 (Klebsiella pneumoniae, INF122-sc-2279936) loaded with 18 markers.\n",
      "2024-04-17 18:12:39,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045486.1 (Enterobacter cloacae, 99101) loaded with 31 markers.\n",
      "2024-04-17 18:12:39,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012257.1 (Cronobacter universalis, NCTC_9529) loaded with 28 markers.\n",
      "2024-04-17 18:12:39,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038469.1 (Citrobacter tructae, SNU_WT2) loaded with 47 markers.\n",
      "2024-04-17 18:12:39,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134201.1 (Cedecea lapagei, NCTC11466) loaded with 26 markers.\n",
      "2024-04-17 18:12:39,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032444.1 (Salmonella enterica, USMARC-69835) loaded with 43 markers.\n",
      "2024-04-17 18:12:39,956 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036166.1 (Salmonella enterica, sg_ww281) loaded with 43 markers.\n",
      "2024-04-17 18:12:40,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070222.1 (Salmonella enterica, Se29) loaded with 45 markers.\n",
      "2024-04-17 18:12:40,087 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029995.1 (Salmonella enterica, SA20053897) loaded with 39 markers.\n",
      "2024-04-17 18:12:40,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030235.1 (Salmonella enterica, SA20031245) loaded with 42 markers.\n",
      "2024-04-17 18:12:40,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019413.1 (Salmonella enterica, SA20030536) loaded with 43 markers.\n",
      "2024-04-17 18:12:40,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079837.1 (Salmonella enterica, LHICA_SA2) loaded with 43 markers.\n",
      "2024-04-17 18:12:40,502 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051315.1 (Salmonella enterica, CVM_35009) loaded with 42 markers.\n",
      "2024-04-17 18:12:40,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051396.1 (Salmonella enterica, CVM_22437) loaded with 42 markers.\n",
      "2024-04-17 18:12:40,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051398.1 (Salmonella enterica, CVM_22436) loaded with 42 markers.\n",
      "2024-04-17 18:12:40,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051425.1 (Salmonella enterica, CVM_20723) loaded with 42 markers.\n",
      "2024-04-17 18:12:40,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075109.1 (Salmonella enterica, CFSAN060807) loaded with 42 markers.\n",
      "2024-04-17 18:12:41,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044200.1 (Salmonella enterica, AR-0406) loaded with 41 markers.\n",
      "2024-04-17 18:12:41,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027677.1 (Salmonella enterica, 12-01738) loaded with 41 markers.\n",
      "2024-04-17 18:12:41,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082168.1 (Raoultella planticola, SCLZS62) loaded with 19 markers.\n",
      "2024-04-17 18:12:41,261 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044121.1 (Raoultella planticola, S25) loaded with 17 markers.\n",
      "2024-04-17 18:12:41,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026047.1 (Raoultella planticola, FDAARGOS_64) loaded with 18 markers.\n",
      "2024-04-17 18:12:41,403 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018307.1 (Klebsiella variicola, X39) loaded with 18 markers.\n",
      "2024-04-17 18:12:41,490 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028555.2 (Klebsiella variicola, WCHKP19) loaded with 28 markers.\n",
      "2024-04-17 18:12:41,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092632.1 (Klebsiella variicola, ML.9ba2) loaded with 16 markers.\n",
      "2024-04-17 18:12:41,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089385.1 (Klebsiella variicola, KPN692) loaded with 16 markers.\n",
      "2024-04-17 18:12:41,680 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060807.1 (Klebsiella variicola, KP2757) loaded with 18 markers.\n",
      "2024-04-17 18:12:41,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078146.1 (Klebsiella variicola, JNQH579) loaded with 25 markers.\n",
      "2024-04-17 18:12:41,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031061.1 (Klebsiella variicola, GN02) loaded with 17 markers.\n",
      "2024-04-17 18:12:41,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084765.1 (Klebsiella variicola, F2R9T) loaded with 18 markers.\n",
      "2024-04-17 18:12:41,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072130.1 (Klebsiella variicola, F2R9) loaded with 18 markers.\n",
      "2024-04-17 18:12:42,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009274.2 (Klebsiella variicola, DX120E) loaded with 16 markers.\n",
      "2024-04-17 18:12:42,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010523.2 (Klebsiella variicola, DSM_15968) loaded with 18 markers.\n",
      "2024-04-17 18:12:42,152 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063917.1 (Klebsiella variicola, BM374-1) loaded with 16 markers.\n",
      "2024-04-17 18:12:42,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130543.1 (Klebsiella variicola, 04153260899A) loaded with 17 markers.\n",
      "2024-04-17 18:12:42,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091527.1 (Klebsiella sp., PCX) loaded with 18 markers.\n",
      "2024-04-17 18:12:42,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073373.1 (Klebsiella sp., P1954) loaded with 19 markers.\n",
      "2024-04-17 18:12:42,398 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022444.1 (Klebsiella sp., LY) loaded with 19 markers.\n",
      "2024-04-17 18:12:42,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022183.1 (Klebsiella quasipneumoniae, WP7-S17-ESBL-08) loaded with 19 markers.\n",
      "2024-04-17 18:12:42,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022112.1 (Klebsiella quasipneumoniae, WP5-W18-ESBL-06) loaded with 33 markers.\n",
      "2024-04-17 18:12:42,641 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021950.1 (Klebsiella quasipneumoniae, WP3-W18-CRE-02) loaded with 27 markers.\n",
      "2024-04-17 18:12:42,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022014.1 (Klebsiella quasipneumoniae, WP3-S18-ESBL-03) loaded with 25 markers.\n",
      "2024-04-17 18:12:42,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022005.1 (Klebsiella quasipneumoniae, WP3-S18-CRE-02) loaded with 32 markers.\n",
      "2024-04-17 18:12:42,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055010.1 (Klebsiella quasipneumoniae, SWHEFF_72) loaded with 25 markers.\n",
      "2024-04-17 18:12:42,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058859.1 (Klebsiella quasipneumoniae, STLEFF_28) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084770.1 (Klebsiella quasipneumoniae, SB610) loaded with 25 markers.\n",
      "2024-04-17 18:12:43,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063874.1 (Klebsiella quasipneumoniae, S174-1) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064046.1 (Klebsiella quasipneumoniae, S15-2) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087580.1 (Klebsiella quasipneumoniae, OSUCMP263AKPC) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134196.1 (Klebsiella quasipneumoniae, NCTC11357) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047283.1 (Klebsiella quasipneumoniae, N18-04116) loaded with 17 markers.\n",
      "2024-04-17 18:12:43,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047281.1 (Klebsiella quasipneumoniae, N18-04101) loaded with 26 markers.\n",
      "2024-04-17 18:12:43,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031257.1 (Klebsiella quasipneumoniae, L22) loaded with 20 markers.\n",
      "2024-04-17 18:12:43,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065841.1 (Klebsiella quasipneumoniae, KqPF9) loaded with 20 markers.\n",
      "2024-04-17 18:12:43,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065846.1 (Klebsiella quasipneumoniae, KqPF42) loaded with 17 markers.\n",
      "2024-04-17 18:12:43,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065838.1 (Klebsiella quasipneumoniae, KqPF26) loaded with 17 markers.\n",
      "2024-04-17 18:12:43,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073660.1 (Klebsiella quasipneumoniae, KPN710429) loaded with 34 markers.\n",
      "2024-04-17 18:12:43,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023478.1 (Klebsiella quasipneumoniae, KPC142) loaded with 19 markers.\n",
      "2024-04-17 18:12:43,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066173.1 (Klebsiella quasipneumoniae, KP2011790) loaded with 20 markers.\n",
      "2024-04-17 18:12:43,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092121.1 (Klebsiella quasipneumoniae, IF3SW-P1) loaded with 18 markers.\n",
      "2024-04-17 18:12:43,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012300.1 (Klebsiella quasipneumoniae, HKUOPLC) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012252.1 (Klebsiella quasipneumoniae, HKUOPLA) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014156.1 (Klebsiella quasipneumoniae, HKUOPL4) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014155.1 (Klebsiella quasipneumoniae, HKUOPJ4) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,209 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014154.1 (Klebsiella quasipneumoniae, HKUOPA4) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034136.1 (Klebsiella quasipneumoniae, G747) loaded with 25 markers.\n",
      "2024-04-17 18:12:44,355 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034129.1 (Klebsiella quasipneumoniae, G4584) loaded with 24 markers.\n",
      "2024-04-17 18:12:44,415 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014071.1 (Klebsiella quasipneumoniae, FDAARGOS_93) loaded with 18 markers.\n",
      "2024-04-17 18:12:44,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083631.1 (Klebsiella quasipneumoniae, FDAARGOS_1504) loaded with 18 markers.\n",
      "2024-04-17 18:12:44,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077279.1 (Klebsiella quasipneumoniae, FDAARGOS_1443) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034678.1 (Klebsiella quasipneumoniae, D120-1) loaded with 20 markers.\n",
      "2024-04-17 18:12:44,647 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084777.1 (Klebsiella quasipneumoniae, CIP110288) loaded with 17 markers.\n",
      "2024-04-17 18:12:44,706 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063902.1 (Klebsiella quasipneumoniae, BS419-3) loaded with 19 markers.\n",
      "2024-04-17 18:12:44,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063896.1 (Klebsiella quasipneumoniae, BS419-1) loaded with 19 markers.\n",
      "2024-04-17 18:12:44,821 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063900.1 (Klebsiella quasipneumoniae, BS324-2) loaded with 19 markers.\n",
      "2024-04-17 18:12:44,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063887.1 (Klebsiella quasipneumoniae, BM419-3) loaded with 19 markers.\n",
      "2024-04-17 18:12:44,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064044.1 (Klebsiella quasipneumoniae, BM404-3-1) loaded with 19 markers.\n",
      "2024-04-17 18:12:44,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029597.1 (Klebsiella quasipneumoniae, ATCC_700603) loaded with 19 markers.\n",
      "2024-04-17 18:12:45,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014696.2 (Klebsiella quasipneumoniae, ATCC_700603) loaded with 18 markers.\n",
      "2024-04-17 18:12:45,102 [WARNING - chronostrain.database.parser.json] - Encountered an unknown nucleotide R (possible IUPAC code) in NZ_CP042319.1,IDX=0, while parsing marker rfbB:BLAST<778850...>. Skipping strain NZ_CP042319.1.\n",
      "2024-04-17 18:12:45,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072515.1 (Klebsiella quasipneumoniae, 3Z-5L) loaded with 19 markers.\n",
      "2024-04-17 18:12:45,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084779.1 (Klebsiella quasipneumoniae, 12A476) loaded with 18 markers.\n",
      "2024-04-17 18:12:45,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084783.1 (Klebsiella quasipneumoniae, 09A323) loaded with 17 markers.\n",
      "2024-04-17 18:12:45,335 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031613.1 (Klebsiella pneumoniae, ZYST1) loaded with 23 markers.\n",
      "2024-04-17 18:12:45,395 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065341.1 (Klebsiella pneumoniae, ZG2017CW_1-1-1) loaded with 22 markers.\n",
      "2024-04-17 18:12:45,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065351.1 (Klebsiella pneumoniae, ZG2017CW4-3-1-2N) loaded with 22 markers.\n",
      "2024-04-17 18:12:45,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP014950.1 (Klebsiella pneumoniae, YH43) loaded with 20 markers.\n",
      "2024-04-17 18:12:45,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP052761.1 (Klebsiella pneumoniae, XH210) loaded with 18 markers.\n",
      "2024-04-17 18:12:45,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009461.1 (Klebsiella pneumoniae, XH209) loaded with 18 markers.\n",
      "2024-04-17 18:12:45,686 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045674.1 (Klebsiella pneumoniae, WSD411) loaded with 17 markers.\n",
      "2024-04-17 18:12:45,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022139.1 (Klebsiella pneumoniae, WP5-S18-ESBL-02) loaded with 16 markers.\n",
      "2024-04-17 18:12:45,801 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076008.1 (Klebsiella pneumoniae, WGR351) loaded with 19 markers.\n",
      "2024-04-17 18:12:45,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031885.1 (Klebsiella pneumoniae, WCHKP095845) loaded with 22 markers.\n",
      "2024-04-17 18:12:45,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025963.2 (Klebsiella pneumoniae, WCHKP020034) loaded with 17 markers.\n",
      "2024-04-17 18:12:45,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046612.1 (Klebsiella pneumoniae, WCGKP294) loaded with 18 markers.\n",
      "2024-04-17 18:12:46,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015753.1 (Klebsiella pneumoniae, W14) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035210.1 (Klebsiella pneumoniae, TH164) loaded with 19 markers.\n",
      "2024-04-17 18:12:46,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087122.1 (Klebsiella pneumoniae, TH12852) loaded with 18 markers.\n",
      "2024-04-17 18:12:46,234 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016159.1 (Klebsiella pneumoniae, TH1) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,298 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012743.1 (Klebsiella pneumoniae, TGH8) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012745.1 (Klebsiella pneumoniae, TGH13) loaded with 19 markers.\n",
      "2024-04-17 18:12:46,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012744.1 (Klebsiella pneumoniae, TGH10) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075256.1 (Klebsiella pneumoniae, SZP4-9-2) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055106.1 (Klebsiella pneumoniae, SWHIN_106) loaded with 18 markers.\n",
      "2024-04-17 18:12:46,579 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055119.1 (Klebsiella pneumoniae, SWHIN_100) loaded with 18 markers.\n",
      "2024-04-17 18:12:46,633 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055033.1 (Klebsiella pneumoniae, SWHEFF_62) loaded with 20 markers.\n",
      "2024-04-17 18:12:46,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055057.1 (Klebsiella pneumoniae, SWHE4) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055066.1 (Klebsiella pneumoniae, SWHE1) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058869.1 (Klebsiella pneumoniae, STLEFF_44) loaded with 22 markers.\n",
      "2024-04-17 18:12:46,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058866.1 (Klebsiella pneumoniae, STLEFF_42) loaded with 17 markers.\n",
      "2024-04-17 18:12:46,918 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054967.1 (Klebsiella pneumoniae, STIN_94) loaded with 19 markers.\n",
      "2024-04-17 18:12:46,974 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054972.1 (Klebsiella pneumoniae, STIN_93) loaded with 18 markers.\n",
      "2024-04-17 18:12:47,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054984.1 (Klebsiella pneumoniae, STIN_90) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054997.1 (Klebsiella pneumoniae, STIN_84) loaded with 16 markers.\n",
      "2024-04-17 18:12:47,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055162.1 (Klebsiella pneumoniae, STEFF_9) loaded with 18 markers.\n",
      "2024-04-17 18:12:47,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055210.1 (Klebsiella pneumoniae, STEFF_2) loaded with 20 markers.\n",
      "2024-04-17 18:12:47,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055174.1 (Klebsiella pneumoniae, STEFF_19) loaded with 18 markers.\n",
      "2024-04-17 18:12:47,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055186.1 (Klebsiella pneumoniae, STEFF_14) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,384 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022916.1 (Klebsiella pneumoniae, ST307PT04) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022919.1 (Klebsiella pneumoniae, ST307PT03) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,495 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022922.1 (Klebsiella pneumoniae, ST307PT02) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022924.1 (Klebsiella pneumoniae, ST307PT01) loaded with 17 markers.\n",
      "2024-04-17 18:12:47,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023148.1 (Klebsiella pneumoniae, SMKP03) loaded with 18 markers.\n",
      "2024-04-17 18:12:47,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015500.1 (Klebsiella pneumoniae, SKGH01) loaded with 19 markers.\n",
      "2024-04-17 18:12:47,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028716.1 (Klebsiella pneumoniae, SCM96) loaded with 18 markers.\n",
      "2024-04-17 18:12:47,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037963.1 (Klebsiella pneumoniae, SCKP020135) loaded with 22 markers.\n",
      "2024-04-17 18:12:47,851 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028787.2 (Klebsiella pneumoniae, SCKP020049) loaded with 23 markers.\n",
      "2024-04-17 18:12:47,907 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082796.1 (Klebsiella pneumoniae, SCAID_PHRX2-2021_(20_245)) loaded with 16 markers.\n",
      "2024-04-17 18:12:47,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR792628.1 (Klebsiella pneumoniae, SB5881) loaded with 16 markers.\n",
      "2024-04-17 18:12:48,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084856.1 (Klebsiella pneumoniae, SB1170) loaded with 17 markers.\n",
      "2024-04-17 18:12:48,081 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084860.1 (Klebsiella pneumoniae, SB1067) loaded with 22 markers.\n",
      "2024-04-17 18:12:48,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063927.1 (Klebsiella pneumoniae, S183-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:48,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058544.1 (Klebsiella pneumoniae, S161-2) loaded with 20 markers.\n",
      "2024-04-17 18:12:48,263 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063871.1 (Klebsiella pneumoniae, S141) loaded with 19 markers.\n",
      "2024-04-17 18:12:48,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041353.1 (Klebsiella pneumoniae, RJY9645) loaded with 18 markers.\n",
      "2024-04-17 18:12:48,376 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068797.1 (Klebsiella pneumoniae, RIVM_C014073) loaded with 17 markers.\n",
      "2024-04-17 18:12:48,431 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056222.1 (Klebsiella pneumoniae, RHBSTW-00973) loaded with 17 markers.\n",
      "2024-04-17 18:12:48,488 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056275.1 (Klebsiella pneumoniae, RHBSTW-00867) loaded with 18 markers.\n",
      "2024-04-17 18:12:48,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055451.1 (Klebsiella pneumoniae, RHBSTW-00574) loaded with 35 markers.\n",
      "2024-04-17 18:12:48,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056530.1 (Klebsiella pneumoniae, RHBSTW-00433) loaded with 34 markers.\n",
      "2024-04-17 18:12:48,700 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056830.1 (Klebsiella pneumoniae, RHBSTW-00128) loaded with 19 markers.\n",
      "2024-04-17 18:12:48,755 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057034.1 (Klebsiella pneumoniae, RHB39-C24) loaded with 17 markers.\n",
      "2024-04-17 18:12:48,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057099.1 (Klebsiella pneumoniae, RHB38-C06) loaded with 18 markers.\n",
      "2024-04-17 18:12:48,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057459.1 (Klebsiella pneumoniae, RHB26-C08) loaded with 18 markers.\n",
      "2024-04-17 18:12:48,928 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057722.1 (Klebsiella pneumoniae, RHB16-C24) loaded with 18 markers.\n",
      "2024-04-17 18:12:48,987 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075552.1 (Klebsiella pneumoniae, RGF99-1) loaded with 22 markers.\n",
      "2024-04-17 18:12:49,042 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040363.1 (Klebsiella pneumoniae, R50) loaded with 17 markers.\n",
      "2024-04-17 18:12:49,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083443.1 (Klebsiella pneumoniae, R32) loaded with 23 markers.\n",
      "2024-04-17 18:12:49,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024458.1 (Klebsiella pneumoniae, QS17-0161) loaded with 17 markers.\n",
      "2024-04-17 18:12:49,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042858.1 (Klebsiella pneumoniae, QD23) loaded with 22 markers.\n",
      "2024-04-17 18:12:49,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008929.1 (Klebsiella pneumoniae, PMK1) loaded with 18 markers.\n",
      "2024-04-17 18:12:49,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053035.1 (Klebsiella pneumoniae, P35) loaded with 17 markers.\n",
      "2024-04-17 18:12:49,401 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043562.1 (Klebsiella pneumoniae, P094-1) loaded with 18 markers.\n",
      "2024-04-17 18:12:49,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081892.1 (Klebsiella pneumoniae, NY-06) loaded with 19 markers.\n",
      "2024-04-17 18:12:49,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061404.1 (Klebsiella pneumoniae, NU-CRE055) loaded with 35 markers.\n",
      "2024-04-17 18:12:49,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025143.1 (Klebsiella pneumoniae, NR5632) loaded with 19 markers.\n",
      "2024-04-17 18:12:49,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091594.1 (Klebsiella pneumoniae, Nord5-1_R48) loaded with 19 markers.\n",
      "2024-04-17 18:12:49,706 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042882.1 (Klebsiella pneumoniae, NMBU-W07E18) loaded with 19 markers.\n",
      "2024-04-17 18:12:49,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024916.1 (Klebsiella pneumoniae, NH54) loaded with 18 markers.\n",
      "2024-04-17 18:12:49,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024874.1 (Klebsiella pneumoniae, NH25) loaded with 21 markers.\n",
      "2024-04-17 18:12:49,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134280.1 (Klebsiella pneumoniae, NCTC9793) loaded with 18 markers.\n",
      "2024-04-17 18:12:49,947 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR588412.1 (Klebsiella pneumoniae, NCTC9157) loaded with 17 markers.\n",
      "2024-04-17 18:12:50,003 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR588408.1 (Klebsiella pneumoniae, NCTC11698) loaded with 16 markers.\n",
      "2024-04-17 18:12:50,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR133964.1 (Klebsiella pneumoniae, NCTC11359) loaded with 16 markers.\n",
      "2024-04-17 18:12:50,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063856.1 (Klebsiella pneumoniae, N1059-5At) loaded with 15 markers.\n",
      "2024-04-17 18:12:50,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085863.1 (Klebsiella pneumoniae, MV-v4-SK2-O) loaded with 17 markers.\n",
      "2024-04-17 18:12:50,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN824133.1 (Klebsiella pneumoniae, MS6671) loaded with 19 markers.\n",
      "2024-04-17 18:12:50,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054303.1 (Klebsiella pneumoniae, MS14393) loaded with 16 markers.\n",
      "2024-04-17 18:12:50,349 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074087.1 (Klebsiella pneumoniae, MRSN752165) loaded with 19 markers.\n",
      "2024-04-17 18:12:50,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084397.1 (Klebsiella pneumoniae, MRSN752019) loaded with 19 markers.\n",
      "2024-04-17 18:12:50,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081897.1 (Klebsiella pneumoniae, MN-03) loaded with 17 markers.\n",
      "2024-04-17 18:12:50,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023337.1 (Klebsiella pneumoniae, MH15-269M) loaded with 20 markers.\n",
      "2024-04-17 18:12:50,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072949.1 (Klebsiella pneumoniae, MGH_39) loaded with 18 markers.\n",
      "2024-04-17 18:12:50,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063863.1 (Klebsiella pneumoniae, M63-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:50,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051490.1 (Klebsiella pneumoniae, M297-1) loaded with 16 markers.\n",
      "2024-04-17 18:12:50,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063992.1 (Klebsiella pneumoniae, M164-1) loaded with 21 markers.\n",
      "2024-04-17 18:12:50,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063858.1 (Klebsiella pneumoniae, M1026-3Ar) loaded with 22 markers.\n",
      "2024-04-17 18:12:50,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063851.1 (Klebsiella pneumoniae, M1023-4Ar) loaded with 22 markers.\n",
      "2024-04-17 18:12:51,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089989.1 (Klebsiella pneumoniae, LZKP00003) loaded with 24 markers.\n",
      "2024-04-17 18:12:51,181 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089986.1 (Klebsiella pneumoniae, LZKP00001) loaded with 18 markers.\n",
      "2024-04-17 18:12:51,259 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068033.1 (Klebsiella pneumoniae, LYS105A) loaded with 18 markers.\n",
      "2024-04-17 18:12:51,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035194.1 (Klebsiella pneumoniae, LH102-A) loaded with 16 markers.\n",
      "2024-04-17 18:12:51,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083768.1 (Klebsiella pneumoniae, KSB1_9E) loaded with 26 markers.\n",
      "2024-04-17 18:12:51,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024173.1 (Klebsiella pneumoniae, KSB1_9A) loaded with 31 markers.\n",
      "2024-04-17 18:12:51,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890751.1 (Klebsiella pneumoniae, KSB1_8C-sc-2280241) loaded with 17 markers.\n",
      "2024-04-17 18:12:51,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024548.1 (Klebsiella pneumoniae, KSB1_7J) loaded with 18 markers.\n",
      "2024-04-17 18:12:51,823 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031814.1 (Klebsiella pneumoniae, KSB1_7F-sc-2280268) loaded with 20 markers.\n",
      "2024-04-17 18:12:51,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890353.1 (Klebsiella pneumoniae, KSB1_5B) loaded with 17 markers.\n",
      "2024-04-17 18:12:51,962 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031789.1 (Klebsiella pneumoniae, KSB1_1I-sc-2280289) loaded with 16 markers.\n",
      "2024-04-17 18:12:52,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890291.1 (Klebsiella pneumoniae, KSB1_1B) loaded with 17 markers.\n",
      "2024-04-17 18:12:52,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073788.1 (Klebsiella pneumoniae, KP_NORM_BLD_2015_115359) loaded with 17 markers.\n",
      "2024-04-17 18:12:52,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034045.1 (Klebsiella pneumoniae, KP_NORM_BLD_2014_104014) loaded with 17 markers.\n",
      "2024-04-17 18:12:52,260 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073627.1 (Klebsiella pneumoniae, KP_NORM_BLD_2012_85054) loaded with 16 markers.\n",
      "2024-04-17 18:12:52,351 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018719.1 (Klebsiella pneumoniae, KP_Goe_828304) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018707.1 (Klebsiella pneumoniae, Kp_Goe_827026) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018701.1 (Klebsiella pneumoniae, Kp_Goe_827024) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018140.1 (Klebsiella pneumoniae, Kp_Goe_822579) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018713.1 (Klebsiella pneumoniae, Kp_Goe_152021) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,768 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018695.1 (Klebsiella pneumoniae, Kp_Goe_149832) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,835 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018686.1 (Klebsiella pneumoniae, Kp_Goe_149473) loaded with 19 markers.\n",
      "2024-04-17 18:12:52,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086724.1 (Klebsiella pneumoniae, KP_3) loaded with 17 markers.\n",
      "2024-04-17 18:12:52,993 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024579.1 (Klebsiella pneumoniae, KpWEA4-2) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024576.1 (Klebsiella pneumoniae, KpWEA4-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024572.1 (Klebsiella pneumoniae, KpWEA3) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024570.1 (Klebsiella pneumoniae, KpWEA2) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,298 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024568.1 (Klebsiella pneumoniae, KpWEA1) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040593.1 (Klebsiella pneumoniae, KpvST15_NDM) loaded with 21 markers.\n",
      "2024-04-17 18:12:53,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040724.1 (Klebsiella pneumoniae, KpvST147B_SE1_1_NDM) loaded with 19 markers.\n",
      "2024-04-17 18:12:53,569 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026177.1 (Klebsiella pneumoniae, KPNIH50) loaded with 34 markers.\n",
      "2024-04-17 18:12:53,680 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026178.1 (Klebsiella pneumoniae, KPNIH49) loaded with 33 markers.\n",
      "2024-04-17 18:12:53,773 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026392.1 (Klebsiella pneumoniae, KPNIH48) loaded with 32 markers.\n",
      "2024-04-17 18:12:53,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009876.1 (Klebsiella pneumoniae, KPNIH31) loaded with 17 markers.\n",
      "2024-04-17 18:12:53,898 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089383.1 (Klebsiella pneumoniae, KPN342) loaded with 18 markers.\n",
      "2024-04-17 18:12:53,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015025.1 (Klebsiella pneumoniae, Kpn223) loaded with 20 markers.\n",
      "2024-04-17 18:12:54,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092336.1 (Klebsiella pneumoniae, KPN20) loaded with 17 markers.\n",
      "2024-04-17 18:12:54,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012992.1 (Klebsiella pneumoniae, KpN06) loaded with 17 markers.\n",
      "2024-04-17 18:12:54,152 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012987.1 (Klebsiella pneumoniae, KpN01) loaded with 16 markers.\n",
      "2024-04-17 18:12:54,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047700.1 (Klebsiella pneumoniae, Kpn-14) loaded with 21 markers.\n",
      "2024-04-17 18:12:54,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047707.1 (Klebsiella pneumoniae, Kpn-13) loaded with 18 markers.\n",
      "2024-04-17 18:12:54,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041099.1 (Klebsiella pneumoniae, KPCTRSRTH07) loaded with 21 markers.\n",
      "2024-04-17 18:12:54,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041092.1 (Klebsiella pneumoniae, KPCTRSRTH01) loaded with 16 markers.\n",
      "2024-04-17 18:12:54,438 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065833.1 (Klebsiella pneumoniae, KpC5) loaded with 17 markers.\n",
      "2024-04-17 18:12:54,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065831.1 (Klebsiella pneumoniae, KpC11) loaded with 16 markers.\n",
      "2024-04-17 18:12:54,578 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092648.1 (Klebsiella pneumoniae, KP7728) loaded with 19 markers.\n",
      "2024-04-17 18:12:54,658 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090468.1 (Klebsiella pneumoniae, kp7450) loaded with 17 markers.\n",
      "2024-04-17 18:12:54,716 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082290.1 (Klebsiella pneumoniae, Kp6) loaded with 21 markers.\n",
      "2024-04-17 18:12:54,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070586.1 (Klebsiella pneumoniae, KP52813) loaded with 19 markers.\n",
      "2024-04-17 18:12:54,837 [DEBUG - chronostrain.database.parser.json] - Strain NZ_FO834906.1 (Klebsiella pneumoniae, Kp52.145) loaded with 16 markers.\n",
      "2024-04-17 18:12:54,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012426.1 (Klebsiella pneumoniae, KP5) loaded with 19 markers.\n",
      "2024-04-17 18:12:54,974 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059312.1 (Klebsiella pneumoniae, Kp46596) loaded with 19 markers.\n",
      "2024-04-17 18:12:55,052 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059317.1 (Klebsiella pneumoniae, Kp46564) loaded with 19 markers.\n",
      "2024-04-17 18:12:55,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090126.1 (Klebsiella pneumoniae, KP46) loaded with 17 markers.\n",
      "2024-04-17 18:12:55,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017777.1 (Klebsiella pneumoniae, KP36M) loaded with 25 markers.\n",
      "2024-04-17 18:12:55,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079725.1 (Klebsiella pneumoniae, KP3295) loaded with 17 markers.\n",
      "2024-04-17 18:12:55,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090622.1 (Klebsiella pneumoniae, KP19-3138) loaded with 24 markers.\n",
      "2024-04-17 18:12:55,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059889.1 (Klebsiella pneumoniae, KP18069) loaded with 19 markers.\n",
      "2024-04-17 18:12:55,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082005.1 (Klebsiella pneumoniae, KP18-8) loaded with 23 markers.\n",
      "2024-04-17 18:12:55,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082029.1 (Klebsiella pneumoniae, KP18-2113) loaded with 22 markers.\n",
      "2024-04-17 18:12:55,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025140.1 (Klebsiella pneumoniae, KP1768) loaded with 20 markers.\n",
      "2024-04-17 18:12:55,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025146.1 (Klebsiella pneumoniae, KP1766) loaded with 20 markers.\n",
      "2024-04-17 18:12:55,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034077.1 (Klebsiella pneumoniae, KP17-16) loaded with 21 markers.\n",
      "2024-04-17 18:12:55,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034076.1 (Klebsiella pneumoniae, KP17-15) loaded with 21 markers.\n",
      "2024-04-17 18:12:55,897 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084984.1 (Klebsiella pneumoniae, KP-PA) loaded with 19 markers.\n",
      "2024-04-17 18:12:55,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071027.1 (Klebsiella pneumoniae, KP-1Pi) loaded with 19 markers.\n",
      "2024-04-17 18:12:56,060 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070890.1 (Klebsiella pneumoniae, KP-135LU) loaded with 19 markers.\n",
      "2024-04-17 18:12:56,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU453210.1 (Klebsiella pneumoniae, Klebsiella_pneumoniae) loaded with 17 markers.\n",
      "2024-04-17 18:12:56,242 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023264.1 (Klebsiella pneumoniae, KAM260) loaded with 50 markers.\n",
      "2024-04-17 18:12:56,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047633.1 (Klebsiella pneumoniae, K2606) loaded with 17 markers.\n",
      "2024-04-17 18:12:56,391 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080346.1 (Klebsiella pneumoniae, K191659) loaded with 20 markers.\n",
      "2024-04-17 18:12:56,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021880.1 (Klebsiella pneumoniae, JUNP254) loaded with 18 markers.\n",
      "2024-04-17 18:12:56,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061970.1 (Klebsiella pneumoniae, IR5755) loaded with 19 markers.\n",
      "2024-04-17 18:12:56,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061975.1 (Klebsiella pneumoniae, IR5086) loaded with 19 markers.\n",
      "2024-04-17 18:12:56,730 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061948.1 (Klebsiella pneumoniae, IR5065) loaded with 18 markers.\n",
      "2024-04-17 18:12:56,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890474.1 (Klebsiella pneumoniae, INF358-sc-2280191) loaded with 17 markers.\n",
      "2024-04-17 18:12:56,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890422.1 (Klebsiella pneumoniae, INF350-sc-2280174) loaded with 20 markers.\n",
      "2024-04-17 18:12:56,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890542.1 (Klebsiella pneumoniae, INF319-sc-2280116) loaded with 21 markers.\n",
      "2024-04-17 18:12:57,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890356.1 (Klebsiella pneumoniae, INF231-sc-2280121) loaded with 17 markers.\n",
      "2024-04-17 18:12:57,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890686.1 (Klebsiella pneumoniae, INF201-sc-2280069) loaded with 17 markers.\n",
      "2024-04-17 18:12:57,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890307.1 (Klebsiella pneumoniae, INF188-sc-2280054) loaded with 27 markers.\n",
      "2024-04-17 18:12:57,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890245.1 (Klebsiella pneumoniae, INF174-sc-2280034) loaded with 18 markers.\n",
      "2024-04-17 18:12:57,272 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890471.1 (Klebsiella pneumoniae, INF168-sc-2280023) loaded with 18 markers.\n",
      "2024-04-17 18:12:57,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890719.1 (Klebsiella pneumoniae, INF161-sc-2280013) loaded with 16 markers.\n",
      "2024-04-17 18:12:57,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890286.1 (Klebsiella pneumoniae, INF156-sc-2280002) loaded with 16 markers.\n",
      "2024-04-17 18:12:57,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890583.1 (Klebsiella pneumoniae, INF145-sc-2279981) loaded with 16 markers.\n",
      "2024-04-17 18:12:57,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031795.1 (Klebsiella pneumoniae, INF125-sc-2279943) loaded with 17 markers.\n",
      "2024-04-17 18:12:57,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890691.1 (Klebsiella pneumoniae, INF119-sc-2279930) loaded with 18 markers.\n",
      "2024-04-17 18:12:57,620 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890665.1 (Klebsiella pneumoniae, INF065-sc-2279980) loaded with 18 markers.\n",
      "2024-04-17 18:12:57,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034281.1 (Klebsiella pneumoniae, I72) loaded with 16 markers.\n",
      "2024-04-17 18:12:57,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063215.1 (Klebsiella pneumoniae, HKU57) loaded with 18 markers.\n",
      "2024-04-17 18:12:57,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061053.1 (Klebsiella pneumoniae, HKP0067) loaded with 19 markers.\n",
      "2024-04-17 18:12:57,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061062.1 (Klebsiella pneumoniae, HKP0018) loaded with 19 markers.\n",
      "2024-04-17 18:12:57,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073906.1 (Klebsiella pneumoniae, HK31) loaded with 19 markers.\n",
      "2024-04-17 18:12:58,027 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018056.1 (Klebsiella pneumoniae, H11) loaded with 18 markers.\n",
      "2024-04-17 18:12:58,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019160.1 (Klebsiella pneumoniae, GN-2) loaded with 18 markers.\n",
      "2024-04-17 18:12:58,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068972.1 (Klebsiella pneumoniae, fekpn2511) loaded with 16 markers.\n",
      "2024-04-17 18:12:58,239 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040993.1 (Klebsiella pneumoniae, FDAARGOS_775) loaded with 16 markers.\n",
      "2024-04-17 18:12:58,297 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044033.1 (Klebsiella pneumoniae, FDAARGOS_631) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044039.1 (Klebsiella pneumoniae, FDAARGOS_630) loaded with 19 markers.\n",
      "2024-04-17 18:12:58,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033756.1 (Klebsiella pneumoniae, FDAARGOS_566) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,483 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023913.1 (Klebsiella pneumoniae, FDAARGOS_439) loaded with 19 markers.\n",
      "2024-04-17 18:12:58,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014123.1 (Klebsiella pneumoniae, FDAARGOS_156) loaded with 18 markers.\n",
      "2024-04-17 18:12:58,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070177.1 (Klebsiella pneumoniae, FDAARGOS_1330) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069992.1 (Klebsiella pneumoniae, FDAARGOS_1329) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,727 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069821.1 (Klebsiella pneumoniae, FDAARGOS_1326) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069946.1 (Klebsiella pneumoniae, FDAARGOS_1325) loaded with 19 markers.\n",
      "2024-04-17 18:12:58,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069873.1 (Klebsiella pneumoniae, FDAARGOS_1323) loaded with 16 markers.\n",
      "2024-04-17 18:12:58,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069994.1 (Klebsiella pneumoniae, FDAARGOS_1321) loaded with 17 markers.\n",
      "2024-04-17 18:12:58,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070169.1 (Klebsiella pneumoniae, FDAARGOS_1320) loaded with 18 markers.\n",
      "2024-04-17 18:12:59,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070092.1 (Klebsiella pneumoniae, FDAARGOS_1316) loaded with 20 markers.\n",
      "2024-04-17 18:12:59,107 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070105.1 (Klebsiella pneumoniae, FDAARGOS_1314) loaded with 21 markers.\n",
      "2024-04-17 18:12:59,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070158.1 (Klebsiella pneumoniae, FDAARGOS_1310) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070015.1 (Klebsiella pneumoniae, FDAARGOS_1309) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070088.1 (Klebsiella pneumoniae, FDAARGOS_1306) loaded with 18 markers.\n",
      "2024-04-17 18:12:59,357 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026157.1 (Klebsiella pneumoniae, F93-2) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026160.1 (Klebsiella pneumoniae, F93-1) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,477 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090397.1 (Klebsiella pneumoniae, F4) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026162.1 (Klebsiella pneumoniae, F13) loaded with 16 markers.\n",
      "2024-04-17 18:12:59,689 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092901.1 (Klebsiella pneumoniae, F11) loaded with 22 markers.\n",
      "2024-04-17 18:12:59,776 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067255.1 (Klebsiella pneumoniae, ESBL_DR09) loaded with 16 markers.\n",
      "2024-04-17 18:12:59,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OV049808.1 (Klebsiella pneumoniae, DJ) loaded with 19 markers.\n",
      "2024-04-17 18:12:59,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022127.1 (Klebsiella pneumoniae, DHQP1605752_NV) loaded with 17 markers.\n",
      "2024-04-17 18:12:59,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075316.1 (Klebsiella pneumoniae, DD521) loaded with 24 markers.\n",
      "2024-04-17 18:13:00,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024429.1 (Klebsiella pneumoniae, DA48896) loaded with 18 markers.\n",
      "2024-04-17 18:13:00,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029582.1 (Klebsiella pneumoniae, DA33140) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,174 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024834.1 (Klebsiella pneumoniae, CRKP-2297) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,248 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024838.1 (Klebsiella pneumoniae, CRKP-1215) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063023.1 (Klebsiella pneumoniae, CriePir45) loaded with 17 markers.\n",
      "2024-04-17 18:13:00,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072905.1 (Klebsiella pneumoniae, COL17) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015382.1 (Klebsiella pneumoniae, CN1) loaded with 16 markers.\n",
      "2024-04-17 18:13:00,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028176.1 (Klebsiella pneumoniae, CFSAN054111) loaded with 17 markers.\n",
      "2024-04-17 18:13:00,553 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023249.1 (Klebsiella pneumoniae, CCUG_70742) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,620 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018671.1 (Klebsiella pneumoniae, CAV1042) loaded with 16 markers.\n",
      "2024-04-17 18:13:00,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017934.1 (Klebsiella pneumoniae, CAV1016) loaded with 18 markers.\n",
      "2024-04-17 18:13:00,758 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042520.1 (Klebsiella pneumoniae, C2) loaded with 19 markers.\n",
      "2024-04-17 18:13:00,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015990.1 (Klebsiella pneumoniae, BR) loaded with 17 markers.\n",
      "2024-04-17 18:13:00,894 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036335.1 (Klebsiella pneumoniae, BP327) loaded with 17 markers.\n",
      "2024-04-17 18:13:00,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063922.1 (Klebsiella pneumoniae, BM338-1) loaded with 17 markers.\n",
      "2024-04-17 18:13:01,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084870.1 (Klebsiella pneumoniae, BJ1) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,090 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046949.1 (Klebsiella pneumoniae, BD_DM_914) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,149 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046381.1 (Klebsiella pneumoniae, BD_DM_782) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046939.1 (Klebsiella pneumoniae, BD_DM_697) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050843.1 (Klebsiella pneumoniae, Bckp186) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060421.1 (Klebsiella pneumoniae, BA2105) loaded with 16 markers.\n",
      "2024-04-17 18:13:01,392 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036187.1 (Klebsiella pneumoniae, BA1559) loaded with 17 markers.\n",
      "2024-04-17 18:13:01,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072422.1 (Klebsiella pneumoniae, B33769) loaded with 18 markers.\n",
      "2024-04-17 18:13:01,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035929.1 (Klebsiella pneumoniae, B31) loaded with 21 markers.\n",
      "2024-04-17 18:13:01,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074580.1 (Klebsiella pneumoniae, B30155) loaded with 18 markers.\n",
      "2024-04-17 18:13:01,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070464.1 (Klebsiella pneumoniae, B28906) loaded with 18 markers.\n",
      "2024-04-17 18:13:01,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084168.1 (Klebsiella pneumoniae, B17602) loaded with 18 markers.\n",
      "2024-04-17 18:13:02,002 [DEBUG - chronostrain.database.parser.json] - Strain NC_009648.1 (Klebsiella pneumoniae, ATCC_700721__MGH_78578) loaded with 22 markers.\n",
      "2024-04-17 18:13:02,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009208.1 (Klebsiella pneumoniae, ATCC_43816_KPPR1) loaded with 17 markers.\n",
      "2024-04-17 18:13:02,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064352.1 (Klebsiella pneumoniae, ATCC_43816) loaded with 17 markers.\n",
      "2024-04-17 18:13:02,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021944.1 (Klebsiella pneumoniae, AR_0152) loaded with 19 markers.\n",
      "2024-04-17 18:13:02,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021939.1 (Klebsiella pneumoniae, AR_0145) loaded with 19 markers.\n",
      "2024-04-17 18:13:02,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021757.1 (Klebsiella pneumoniae, AR_0138) loaded with 19 markers.\n",
      "2024-04-17 18:13:02,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021740.1 (Klebsiella pneumoniae, AR_0126) loaded with 18 markers.\n",
      "2024-04-17 18:13:02,572 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020061.1 (Klebsiella pneumoniae, AR_0117) loaded with 18 markers.\n",
      "2024-04-17 18:13:02,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032222.1 (Klebsiella pneumoniae, AR_0046) loaded with 23 markers.\n",
      "2024-04-17 18:13:02,763 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029137.1 (Klebsiella pneumoniae, AR376) loaded with 17 markers.\n",
      "2024-04-17 18:13:02,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039954.1 (Klebsiella pneumoniae, AR142_2b) loaded with 16 markers.\n",
      "2024-04-17 18:13:02,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036438.1 (Klebsiella pneumoniae, ABFQB) loaded with 17 markers.\n",
      "2024-04-17 18:13:03,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036442.1 (Klebsiella pneumoniae, ABFPV) loaded with 18 markers.\n",
      "2024-04-17 18:13:03,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014755.1 (Klebsiella pneumoniae, AATZP) loaded with 19 markers.\n",
      "2024-04-17 18:13:03,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042314.1 (Klebsiella pneumoniae, 90-17) loaded with 15 markers.\n",
      "2024-04-17 18:13:03,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042315.1 (Klebsiella pneumoniae, 89-17) loaded with 21 markers.\n",
      "2024-04-17 18:13:03,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017985.1 (Klebsiella pneumoniae, 825795-1) loaded with 19 markers.\n",
      "2024-04-17 18:13:03,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042317.1 (Klebsiella pneumoniae, 74-17) loaded with 16 markers.\n",
      "2024-04-17 18:13:03,533 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082988.1 (Klebsiella pneumoniae, 7008.20) loaded with 19 markers.\n",
      "2024-04-17 18:13:03,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026495.1 (Klebsiella pneumoniae, 616) loaded with 16 markers.\n",
      "2024-04-17 18:13:03,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089001.1 (Klebsiella pneumoniae, 56661) loaded with 25 markers.\n",
      "2024-04-17 18:13:03,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043932.1 (Klebsiella pneumoniae, 555) loaded with 19 markers.\n",
      "2024-04-17 18:13:03,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050376.1 (Klebsiella pneumoniae, 51015) loaded with 17 markers.\n",
      "2024-04-17 18:13:04,024 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023839.1 (Klebsiella pneumoniae, 4_1-2) loaded with 19 markers.\n",
      "2024-04-17 18:13:04,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050360.1 (Klebsiella pneumoniae, 47733) loaded with 19 markers.\n",
      "2024-04-17 18:13:04,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034321.1 (Klebsiella pneumoniae, 33) loaded with 17 markers.\n",
      "2024-04-17 18:13:04,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030296.1 (Klebsiella pneumoniae, 309074) loaded with 20 markers.\n",
      "2024-04-17 18:13:04,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028478.1 (Klebsiella pneumoniae, 2e) loaded with 18 markers.\n",
      "2024-04-17 18:13:04,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030302.1 (Klebsiella pneumoniae, 205880) loaded with 16 markers.\n",
      "2024-04-17 18:13:04,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021165.1 (Klebsiella pneumoniae, 203) loaded with 20 markers.\n",
      "2024-04-17 18:13:04,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027042.1 (Klebsiella pneumoniae, 1_GR_13) loaded with 23 markers.\n",
      "2024-04-17 18:13:04,732 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022023.1 (Klebsiella pneumoniae, 19051) loaded with 17 markers.\n",
      "2024-04-17 18:13:04,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039951.1 (Klebsiella pneumoniae, 18KM2445b) loaded with 16 markers.\n",
      "2024-04-17 18:13:04,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083063.1 (Klebsiella pneumoniae, 1613163991) loaded with 19 markers.\n",
      "2024-04-17 18:13:04,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083066.1 (Klebsiella pneumoniae, 1613015451) loaded with 17 markers.\n",
      "2024-04-17 18:13:05,061 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026017.1 (Klebsiella pneumoniae, 13190) loaded with 17 markers.\n",
      "2024-04-17 18:13:05,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083070.1 (Klebsiella pneumoniae, 1230_0791_KP) loaded with 16 markers.\n",
      "2024-04-17 18:13:05,189 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030315.1 (Klebsiella pneumoniae, 11305) loaded with 19 markers.\n",
      "2024-04-17 18:13:05,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084871.1 (Klebsiella pneumoniae, 04A025) loaded with 17 markers.\n",
      "2024-04-17 18:13:05,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083075.1 (Klebsiella pneumoniae, 0113481141) loaded with 18 markers.\n",
      "2024-04-17 18:13:05,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074233.1 (Salmonella bongori, SGSC_3100) loaded with 28 markers.\n",
      "2024-04-17 18:13:05,495 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045766.1 (Salmonella bongori, SGSC_3100) loaded with 28 markers.\n",
      "2024-04-17 18:13:05,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067369.1 (Salmonella bongori, Se40) loaded with 28 markers.\n",
      "2024-04-17 18:13:05,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022120.1 (Salmonella bongori, SA19983605) loaded with 28 markers.\n",
      "2024-04-17 18:13:05,725 [DEBUG - chronostrain.database.parser.json] - Strain NC_015761.1 (Salmonella bongori, NCTC_12419) loaded with 28 markers.\n",
      "2024-04-17 18:13:05,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082841.1 (Kluyvera sp., CRP) loaded with 37 markers.\n",
      "2024-04-17 18:13:05,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134138.1 (Kluyvera intermedia, NCTC12125) loaded with 35 markers.\n",
      "2024-04-17 18:13:06,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047669.1 (Klebsiella aerogenes, HNHF1) loaded with 21 markers.\n",
      "2024-04-17 18:13:06,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048598.1 (Klebsiella aerogenes, AUH-KAM-9) loaded with 18 markers.\n",
      "2024-04-17 18:13:06,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025034.2 (Enterobacter sp., SGAir0187) loaded with 34 markers.\n",
      "2024-04-17 18:13:06,286 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074149.1 (Enterobacter sp., JBIWA008) loaded with 34 markers.\n",
      "2024-04-17 18:13:06,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074170.1 (Enterobacter sp., JBIWA003) loaded with 32 markers.\n",
      "2024-04-17 18:13:06,489 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012999.1 (Enterobacter sp., E20) loaded with 40 markers.\n",
      "2024-04-17 18:13:06,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011798.1 (Enterobacter ludwigii, UW5) loaded with 34 markers.\n",
      "2024-04-17 18:13:06,692 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006580.1 (Enterobacter ludwigii, P101) loaded with 33 markers.\n",
      "2024-04-17 18:13:06,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040527.1 (Enterobacter ludwigii, JP9) loaded with 34 markers.\n",
      "2024-04-17 18:13:06,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040256.1 (Enterobacter ludwigii, JP6) loaded with 34 markers.\n",
      "2024-04-17 18:13:06,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040606.1 (Enterobacter ludwigii, I42) loaded with 32 markers.\n",
      "2024-04-17 18:13:07,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041062.1 (Enterobacter ludwigii, I140) loaded with 32 markers.\n",
      "2024-04-17 18:13:07,206 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083640.1 (Enterobacter ludwigii, FDAARGOS_1498) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082860.1 (Enterobacter ludwigii, FDAARGOS_1475) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,413 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017279.1 (Enterobacter ludwigii, EN-119) loaded with 34 markers.\n",
      "2024-04-17 18:13:07,505 [DEBUG - chronostrain.database.parser.json] - Strain NC_016514.1 (Enterobacter ludwigii, EcWSU1) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018785.1 (Enterobacter ludwigii, AA4) loaded with 33 markers.\n",
      "2024-04-17 18:13:07,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086101.1 (Enterobacter ludwigii, 994590) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086099.1 (Enterobacter ludwigii, 197) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,879 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086100.1 (Enterobacter ludwigii, 196) loaded with 35 markers.\n",
      "2024-04-17 18:13:07,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032897.1 (Enterobacter kobei, WCHEK045523) loaded with 36 markers.\n",
      "2024-04-17 18:13:08,074 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050073.1 (Enterobacter kobei, 070) loaded with 39 markers.\n",
      "2024-04-17 18:13:08,157 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082280.1 (Enterobacter cancerogenus, HAEC1) loaded with 31 markers.\n",
      "2024-04-17 18:13:08,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024675.1 (Citrobacter werkmanii, UMH18) loaded with 44 markers.\n",
      "2024-04-17 18:13:08,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056399.1 (Citrobacter sp., RHBSTW-00570) loaded with 47 markers.\n",
      "2024-04-17 18:13:08,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056409.1 (Citrobacter sp., RHBSTW-00535) loaded with 70 markers.\n",
      "2024-04-17 18:13:08,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056834.1 (Citrobacter sp., RHBSTW-00127) loaded with 58 markers.\n",
      "2024-04-17 18:13:08,808 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046673.1 (Citrobacter sp., 172116965) loaded with 50 markers.\n",
      "2024-04-17 18:13:08,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047247.1 (Citrobacter freundii, LDL3-3) loaded with 43 markers.\n",
      "2024-04-17 18:13:09,024 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024819.1 (Citrobacter freundii, CRCB-101) loaded with 41 markers.\n",
      "2024-04-17 18:13:09,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035196.1 (Klebsiella pneumoniae, LH375) loaded with 18 markers.\n",
      "2024-04-17 18:13:09,149 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009863.1 (Klebsiella pneumoniae, KPNIH29) loaded with 17 markers.\n",
      "2024-04-17 18:13:09,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034831.1 (Salmonella sp., SSDFZ69) loaded with 41 markers.\n",
      "2024-04-17 18:13:09,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007216.2 (Salmonella enterica, USDA-ARS-USMARC-1927) loaded with 41 markers.\n",
      "2024-04-17 18:13:09,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022467.1 (Salmonella enterica, ST114) loaded with 40 markers.\n",
      "2024-04-17 18:13:09,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041171.1 (Salmonella enterica, SH11G0791) loaded with 39 markers.\n",
      "2024-04-17 18:13:09,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039169.1 (Salmonella enterica, Sal-5364) loaded with 40 markers.\n",
      "2024-04-17 18:13:09,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019404.1 (Salmonella enterica, SA20113257) loaded with 41 markers.\n",
      "2024-04-17 18:13:09,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022490.1 (Salmonella enterica, SA20026289) loaded with 38 markers.\n",
      "2024-04-17 18:13:09,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022503.1 (Salmonella enterica, SA20014981) loaded with 38 markers.\n",
      "2024-04-17 18:13:10,061 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089207.1 (Salmonella enterica, S639) loaded with 39 markers.\n",
      "2024-04-17 18:13:10,195 [DEBUG - chronostrain.database.parser.json] - Strain NC_022525.1 (Salmonella enterica, RM6836) loaded with 39 markers.\n",
      "2024-04-17 18:13:10,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012514.1 (Salmonella enterica, RM1986) loaded with 39 markers.\n",
      "2024-04-17 18:13:10,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012513.1 (Salmonella enterica, RM1984) loaded with 39 markers.\n",
      "2024-04-17 18:13:10,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062223.1 (Salmonella enterica, R18.1656) loaded with 40 markers.\n",
      "2024-04-17 18:13:10,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062225.1 (Salmonella enterica, R18.1074) loaded with 41 markers.\n",
      "2024-04-17 18:13:10,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037960.1 (Salmonella enterica, R18.0877) loaded with 40 markers.\n",
      "2024-04-17 18:13:10,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092075.1 (Salmonella enterica, P281972) loaded with 45 markers.\n",
      "2024-04-17 18:13:10,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025745.1 (Salmonella enterica, MFDS1004024) loaded with 39 markers.\n",
      "2024-04-17 18:13:11,083 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079838.1 (Salmonella enterica, LHICA_SA3) loaded with 42 markers.\n",
      "2024-04-17 18:13:11,224 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079836.1 (Salmonella enterica, LHICA_SA1) loaded with 42 markers.\n",
      "2024-04-17 18:13:11,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079839.1 (Salmonella enterica, LHICA_E3) loaded with 42 markers.\n",
      "2024-04-17 18:13:11,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015924.1 (Salmonella enterica, Levine_15) loaded with 41 markers.\n",
      "2024-04-17 18:13:11,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015923.1 (Salmonella enterica, Levine_1) loaded with 41 markers.\n",
      "2024-04-17 18:13:11,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082338.1 (Salmonella enterica, IBG7b4) loaded with 44 markers.\n",
      "2024-04-17 18:13:11,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028729.1 (Salmonella enterica, HFCDC-SM-846) loaded with 39 markers.\n",
      "2024-04-17 18:13:11,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050833.1 (Salmonella enterica, GSJ_2017-Sal.-009) loaded with 39 markers.\n",
      "2024-04-17 18:13:12,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032304.1 (Salmonella enterica, FORC93) loaded with 38 markers.\n",
      "2024-04-17 18:13:12,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051389.1 (Salmonella enterica, CVM_22536) loaded with 44 markers.\n",
      "2024-04-17 18:13:12,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051416.1 (Salmonella enterica, CVM_20744) loaded with 44 markers.\n",
      "2024-04-17 18:13:12,420 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016014.1 (Salmonella enterica, CFSAN003887) loaded with 44 markers.\n",
      "2024-04-17 18:13:12,535 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016010.1 (Salmonella enterica, CFSAN001660) loaded with 41 markers.\n",
      "2024-04-17 18:13:12,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074606.1 (Salmonella enterica, CFSAN000827) loaded with 41 markers.\n",
      "2024-04-17 18:13:12,781 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025246.1 (Salmonella enterica, CDC_2012K-0938) loaded with 41 markers.\n",
      "2024-04-17 18:13:12,894 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025248.1 (Salmonella enterica, CDC_2009K-1331) loaded with 41 markers.\n",
      "2024-04-17 18:13:13,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019196.1 (Salmonella enterica, ATCC_BAA-1738) loaded with 39 markers.\n",
      "2024-04-17 18:13:13,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011396.1 (Salmonella enterica, ATCC_8391) loaded with 39 markers.\n",
      "2024-04-17 18:13:13,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043027.1 (Salmonella enterica, AD19) loaded with 41 markers.\n",
      "2024-04-17 18:13:13,367 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060837.1 (Salmonella enterica, 852) loaded with 40 markers.\n",
      "2024-04-17 18:13:13,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060840.1 (Salmonella enterica, 805) loaded with 40 markers.\n",
      "2024-04-17 18:13:13,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060844.1 (Salmonella enterica, 770) loaded with 38 markers.\n",
      "2024-04-17 18:13:13,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060846.1 (Salmonella enterica, 733) loaded with 40 markers.\n",
      "2024-04-17 18:13:13,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060849.1 (Salmonella enterica, 724) loaded with 39 markers.\n",
      "2024-04-17 18:13:13,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040699.1 (Salmonella enterica, 7) loaded with 39 markers.\n",
      "2024-04-17 18:13:13,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090141.1 (Salmonella enterica, 5ASAL09) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090133.1 (Salmonella enterica, 5ASAL07) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,237 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090137.1 (Salmonella enterica, 5ASAL05) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060852.1 (Salmonella enterica, 467) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060855.1 (Salmonella enterica, 453) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060834.1 (Salmonella enterica, 1505) loaded with 40 markers.\n",
      "2024-04-17 18:13:14,659 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074182.1 (Raoultella planticola, JBIWA001) loaded with 20 markers.\n",
      "2024-04-17 18:13:14,756 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069815.1 (Raoultella planticola, HH15) loaded with 20 markers.\n",
      "2024-04-17 18:13:14,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056145.1 (Klebsiella variicola, RHBSTW-00883) loaded with 16 markers.\n",
      "2024-04-17 18:13:14,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134235.1 (Klebsiella variicola, NCTC9668) loaded with 18 markers.\n",
      "2024-04-17 18:13:15,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068237.1 (Klebsiella quasipneumoniae, YDKL-002) loaded with 20 markers.\n",
      "2024-04-17 18:13:15,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084789.1 (Klebsiella quasipneumoniae, U41) loaded with 35 markers.\n",
      "2024-04-17 18:13:15,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035207.1 (Klebsiella quasipneumoniae, TH114) loaded with 17 markers.\n",
      "2024-04-17 18:13:15,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055213.1 (Klebsiella quasipneumoniae, STEFF_21) loaded with 18 markers.\n",
      "2024-04-17 18:13:15,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084806.1 (Klebsiella quasipneumoniae, SB1124) loaded with 19 markers.\n",
      "2024-04-17 18:13:15,428 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058131.1 (Klebsiella quasipneumoniae, RHBSTW-00138) loaded with 20 markers.\n",
      "2024-04-17 18:13:15,498 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065467.1 (Klebsiella quasipneumoniae, Pinhead_Larry) loaded with 19 markers.\n",
      "2024-04-17 18:13:15,569 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR588411.1 (Klebsiella quasipneumoniae, NCTC9170) loaded with 26 markers.\n",
      "2024-04-17 18:13:15,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082785.1 (Klebsiella quasipneumoniae, MMCC7) loaded with 19 markers.\n",
      "2024-04-17 18:13:15,696 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043928.1 (Klebsiella quasipneumoniae, M17277) loaded with 18 markers.\n",
      "2024-04-17 18:13:15,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045641.1 (Klebsiella quasipneumoniae, KP18-31) loaded with 24 markers.\n",
      "2024-04-17 18:13:15,844 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062138.1 (Klebsiella quasipneumoniae, 203_19001) loaded with 17 markers.\n",
      "2024-04-17 18:13:15,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084818.1 (Klebsiella quasipneumoniae, 18A069) loaded with 17 markers.\n",
      "2024-04-17 18:13:15,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087706.1 (Klebsiella pneumoniae, YZH2487) loaded with 18 markers.\n",
      "2024-04-17 18:13:16,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090393.1 (Klebsiella pneumoniae, YZH2429) loaded with 18 markers.\n",
      "2024-04-17 18:13:16,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090389.1 (Klebsiella pneumoniae, YZH22127) loaded with 18 markers.\n",
      "2024-04-17 18:13:16,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079673.1 (Klebsiella pneumoniae, WRC29_CMC309MPCH) loaded with 28 markers.\n",
      "2024-04-17 18:13:16,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079668.1 (Klebsiella pneumoniae, WRC28_CMC387MPCH) loaded with 26 markers.\n",
      "2024-04-17 18:13:16,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079629.1 (Klebsiella pneumoniae, WRC18_CMC307MC) loaded with 31 markers.\n",
      "2024-04-17 18:13:16,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079624.1 (Klebsiella pneumoniae, WRC17_CMC307PC) loaded with 31 markers.\n",
      "2024-04-17 18:13:16,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079599.1 (Klebsiella pneumoniae, WRC10_CMC309MC) loaded with 29 markers.\n",
      "2024-04-17 18:13:16,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079164.1 (Klebsiella pneumoniae, WRC09_CMC309PC) loaded with 29 markers.\n",
      "2024-04-17 18:13:16,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079155.1 (Klebsiella pneumoniae, WRC08_CMC432MC) loaded with 24 markers.\n",
      "2024-04-17 18:13:16,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079150.1 (Klebsiella pneumoniae, WRC07_CMC432PC) loaded with 25 markers.\n",
      "2024-04-17 18:13:16,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079145.1 (Klebsiella pneumoniae, WRC06_CMC387MC) loaded with 25 markers.\n",
      "2024-04-17 18:13:17,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079138.1 (Klebsiella pneumoniae, WRC05_CMC387PC) loaded with 25 markers.\n",
      "2024-04-17 18:13:17,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079131.1 (Klebsiella pneumoniae, WRC04_AI1235MC) loaded with 27 markers.\n",
      "2024-04-17 18:13:17,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079120.1 (Klebsiella pneumoniae, WRC03_AI1235PC) loaded with 27 markers.\n",
      "2024-04-17 18:13:17,355 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021929.1 (Klebsiella pneumoniae, WP2-W18-ESBL-06) loaded with 18 markers.\n",
      "2024-04-17 18:13:17,420 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045694.1 (Klebsiella pneumoniae, TK421) loaded with 17 markers.\n",
      "2024-04-17 18:13:17,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084823.1 (Klebsiella pneumoniae, T69) loaded with 16 markers.\n",
      "2024-04-17 18:13:17,565 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045661.1 (Klebsiella pneumoniae, SMU18037509) loaded with 29 markers.\n",
      "2024-04-17 18:13:17,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084825.1 (Klebsiella pneumoniae, SB617) loaded with 20 markers.\n",
      "2024-04-17 18:13:17,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084852.1 (Klebsiella pneumoniae, SB2390) loaded with 17 markers.\n",
      "2024-04-17 18:13:17,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084859.1 (Klebsiella pneumoniae, SB1139) loaded with 16 markers.\n",
      "2024-04-17 18:13:17,856 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071279.1 (Klebsiella pneumoniae, SA-KpST14) loaded with 31 markers.\n",
      "2024-04-17 18:13:17,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063865.1 (Klebsiella pneumoniae, S130-1) loaded with 16 markers.\n",
      "2024-04-17 18:13:17,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068798.1 (Klebsiella pneumoniae, RIVM_C015043) loaded with 17 markers.\n",
      "2024-04-17 18:13:18,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056191.1 (Klebsiella pneumoniae, RHBSTW-00995) loaded with 20 markers.\n",
      "2024-04-17 18:13:18,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056329.1 (Klebsiella pneumoniae, RHBSTW-00807) loaded with 16 markers.\n",
      "2024-04-17 18:13:18,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056425.1 (Klebsiella pneumoniae, RHBSTW-00515) loaded with 22 markers.\n",
      "2024-04-17 18:13:18,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056854.1 (Klebsiella pneumoniae, RHBSTW-00113) loaded with 33 markers.\n",
      "2024-04-17 18:13:18,384 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075548.1 (Klebsiella pneumoniae, RGT40-1) loaded with 17 markers.\n",
      "2024-04-17 18:13:18,444 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075452.1 (Klebsiella pneumoniae, RGF20-1) loaded with 17 markers.\n",
      "2024-04-17 18:13:18,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006798.1 (Klebsiella pneumoniae, PittNDM01) loaded with 23 markers.\n",
      "2024-04-17 18:13:18,594 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014004.1 (Klebsiella pneumoniae, NUHL24835) loaded with 19 markers.\n",
      "2024-04-17 18:13:18,672 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061389.1 (Klebsiella pneumoniae, NU-CRE195) loaded with 17 markers.\n",
      "2024-04-17 18:13:18,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091601.1 (Klebsiella pneumoniae, Nord77_R886) loaded with 22 markers.\n",
      "2024-04-17 18:13:18,811 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060049.1 (Klebsiella pneumoniae, NICU_2_P7) loaded with 16 markers.\n",
      "2024-04-17 18:13:18,884 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034408.1 (Klebsiella pneumoniae, NH34) loaded with 26 markers.\n",
      "2024-04-17 18:13:18,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134213.1 (Klebsiella pneumoniae, NCTC418) loaded with 16 markers.\n",
      "2024-04-17 18:13:19,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085866.1 (Klebsiella pneumoniae, MV-u1-SK2-O) loaded with 17 markers.\n",
      "2024-04-17 18:13:19,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073054.1 (Klebsiella pneumoniae, MGH83) loaded with 17 markers.\n",
      "2024-04-17 18:13:19,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061965.1 (Klebsiella pneumoniae, LZK001) loaded with 17 markers.\n",
      "2024-04-17 18:13:19,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025684.1 (Klebsiella pneumoniae, L5-2) loaded with 17 markers.\n",
      "2024-04-17 18:13:19,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890710.1 (Klebsiella pneumoniae, KSB2_9B) loaded with 20 markers.\n",
      "2024-04-17 18:13:19,332 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890457.1 (Klebsiella pneumoniae, KSB1_6G-sc-2280276) loaded with 24 markers.\n",
      "2024-04-17 18:13:19,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018337.1 (Klebsiella pneumoniae, Kp_Goe_154414) loaded with 17 markers.\n",
      "2024-04-17 18:13:19,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078033.1 (Klebsiella pneumoniae, KP_160) loaded with 27 markers.\n",
      "2024-04-17 18:13:19,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065401.1 (Klebsiella pneumoniae, KPNSP-1) loaded with 16 markers.\n",
      "2024-04-17 18:13:19,629 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014762.1 (Klebsiella pneumoniae, KPNIH39) loaded with 24 markers.\n",
      "2024-04-17 18:13:19,695 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007731.1 (Klebsiella pneumoniae, KPNIH27) loaded with 18 markers.\n",
      "2024-04-17 18:13:19,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020853.1 (Klebsiella pneumoniae, KPN528) loaded with 23 markers.\n",
      "2024-04-17 18:13:19,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP052036.1 (Klebsiella pneumoniae, KPN41053) loaded with 18 markers.\n",
      "2024-04-17 18:13:19,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072494.1 (Klebsiella pneumoniae, KPN246) loaded with 18 markers.\n",
      "2024-04-17 18:13:19,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018753.1 (Klebsiella pneumoniae, KP67) loaded with 30 markers.\n",
      "2024-04-17 18:13:20,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018750.1 (Klebsiella pneumoniae, KP64) loaded with 30 markers.\n",
      "2024-04-17 18:13:20,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012753.1 (Klebsiella pneumoniae, KP617) loaded with 24 markers.\n",
      "2024-04-17 18:13:20,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018747.1 (Klebsiella pneumoniae, KP33) loaded with 26 markers.\n",
      "2024-04-17 18:13:20,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072938.1 (Klebsiella pneumoniae, KP2723) loaded with 18 markers.\n",
      "2024-04-17 18:13:20,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP003999.1 (Klebsiella pneumoniae, Kp13) loaded with 21 markers.\n",
      "2024-04-17 18:13:20,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012883.1 (Klebsiella pneumoniae, KP-1) loaded with 19 markers.\n",
      "2024-04-17 18:13:20,519 [DEBUG - chronostrain.database.parser.json] - Strain NC_017540.1 (Klebsiella pneumoniae, KCTC_2242) loaded with 16 markers.\n",
      "2024-04-17 18:13:20,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054400.1 (Klebsiella pneumoniae, KCJ3K307) loaded with 17 markers.\n",
      "2024-04-17 18:13:20,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054406.1 (Klebsiella pneumoniae, KCJ3K293) loaded with 17 markers.\n",
      "2024-04-17 18:13:20,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054403.1 (Klebsiella pneumoniae, KCJ3K292) loaded with 17 markers.\n",
      "2024-04-17 18:13:20,765 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030923.1 (Klebsiella pneumoniae, KC-Pl-HB1) loaded with 16 markers.\n",
      "2024-04-17 18:13:20,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071324.1 (Klebsiella pneumoniae, JS161205112) loaded with 16 markers.\n",
      "2024-04-17 18:13:20,903 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079762.1 (Klebsiella pneumoniae, JP20) loaded with 17 markers.\n",
      "2024-04-17 18:13:20,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073782.1 (Klebsiella pneumoniae, IRQBAS106c-genome) loaded with 16 markers.\n",
      "2024-04-17 18:13:21,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072826.1 (Klebsiella pneumoniae, IRQBAS105c) loaded with 16 markers.\n",
      "2024-04-17 18:13:21,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072825.1 (Klebsiella pneumoniae, IRQBAS103c) loaded with 17 markers.\n",
      "2024-04-17 18:13:21,177 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071150.1 (Klebsiella pneumoniae, IRQBAS102c) loaded with 16 markers.\n",
      "2024-04-17 18:13:21,246 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890527.1 (Klebsiella pneumoniae, INF361-sc-2280198) loaded with 19 markers.\n",
      "2024-04-17 18:13:21,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890277.1 (Klebsiella pneumoniae, INF355-sc-2280185) loaded with 19 markers.\n",
      "2024-04-17 18:13:21,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890323.1 (Klebsiella pneumoniae, INF348-sc-2280170) loaded with 26 markers.\n",
      "2024-04-17 18:13:21,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890640.1 (Klebsiella pneumoniae, INF344-sc-2280162) loaded with 19 markers.\n",
      "2024-04-17 18:13:21,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890490.1 (Klebsiella pneumoniae, INF341-sc-2280155) loaded with 19 markers.\n",
      "2024-04-17 18:13:21,580 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032833.1 (Klebsiella pneumoniae, INF237) loaded with 16 markers.\n",
      "2024-04-17 18:13:21,641 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081889.1 (Klebsiella pneumoniae, IA-044) loaded with 17 markers.\n",
      "2024-04-17 18:13:21,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025211.1 (Klebsiella pneumoniae, HZW25) loaded with 33 markers.\n",
      "2024-04-17 18:13:21,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033777.1 (Klebsiella pneumoniae, FDAARGOS_531) loaded with 16 markers.\n",
      "2024-04-17 18:13:21,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069853.1 (Klebsiella pneumoniae, FDAARGOS_1324) loaded with 17 markers.\n",
      "2024-04-17 18:13:21,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070046.1 (Klebsiella pneumoniae, FDAARGOS_1319) loaded with 21 markers.\n",
      "2024-04-17 18:13:22,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070172.1 (Klebsiella pneumoniae, FDAARGOS_1315) loaded with 33 markers.\n",
      "2024-04-17 18:13:22,081 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069968.1 (Klebsiella pneumoniae, FDAARGOS_1308) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069962.1 (Klebsiella pneumoniae, FDAARGOS_1307) loaded with 18 markers.\n",
      "2024-04-17 18:13:22,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069847.1 (Klebsiella pneumoniae, FDAARGOS_1305) loaded with 22 markers.\n",
      "2024-04-17 18:13:22,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059295.1 (Klebsiella pneumoniae, ER17974.3) loaded with 16 markers.\n",
      "2024-04-17 18:13:22,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060281.1 (Klebsiella pneumoniae, ent1077) loaded with 16 markers.\n",
      "2024-04-17 18:13:22,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089097.1 (Klebsiella pneumoniae, EH13) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090429.1 (Klebsiella pneumoniae, DY1928) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016811.1 (Klebsiella pneumoniae, DHQP1002001) loaded with 20 markers.\n",
      "2024-04-17 18:13:22,595 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030072.1 (Klebsiella pneumoniae, DA12090) loaded with 19 markers.\n",
      "2024-04-17 18:13:22,658 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093151.1 (Klebsiella pneumoniae, CY814036) loaded with 16 markers.\n",
      "2024-04-17 18:13:22,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081891.1 (Klebsiella pneumoniae, CT-04) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,781 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062986.1 (Klebsiella pneumoniae, CriePir26) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024707.1 (Klebsiella pneumoniae, cr-hvkp3) loaded with 16 markers.\n",
      "2024-04-17 18:13:22,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077779.1 (Klebsiella pneumoniae, CDI231) loaded with 17 markers.\n",
      "2024-04-17 18:13:22,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022611.1 (Klebsiella pneumoniae, CDC_0106) loaded with 23 markers.\n",
      "2024-04-17 18:13:23,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061700.1 (Klebsiella pneumoniae, C39) loaded with 16 markers.\n",
      "2024-04-17 18:13:23,107 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084103.1 (Klebsiella pneumoniae, C11) loaded with 18 markers.\n",
      "2024-04-17 18:13:23,173 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061832.1 (Klebsiella pneumoniae, C0674) loaded with 17 markers.\n",
      "2024-04-17 18:13:23,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053771.1 (Klebsiella pneumoniae, BP3636) loaded with 23 markers.\n",
      "2024-04-17 18:13:23,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050826.1 (Klebsiella pneumoniae, Bckp212) loaded with 24 markers.\n",
      "2024-04-17 18:13:23,375 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050834.1 (Klebsiella pneumoniae, Bckp021) loaded with 18 markers.\n",
      "2024-04-17 18:13:23,456 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059468.1 (Klebsiella pneumoniae, BA25665) loaded with 25 markers.\n",
      "2024-04-17 18:13:23,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053765.1 (Klebsiella pneumoniae, BA10835) loaded with 23 markers.\n",
      "2024-04-17 18:13:23,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028932.1 (Klebsiella pneumoniae, AR_0153) loaded with 23 markers.\n",
      "2024-04-17 18:13:23,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021708.1 (Klebsiella pneumoniae, AR_0143) loaded with 23 markers.\n",
      "2024-04-17 18:13:23,758 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028990.1 (Klebsiella pneumoniae, AR_0142) loaded with 24 markers.\n",
      "2024-04-17 18:13:23,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028953.1 (Klebsiella pneumoniae, AR_0141) loaded with 20 markers.\n",
      "2024-04-17 18:13:23,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029722.1 (Klebsiella pneumoniae, AR_0140) loaded with 21 markers.\n",
      "2024-04-17 18:13:23,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020067.1 (Klebsiella pneumoniae, AR_0068) loaded with 24 markers.\n",
      "2024-04-17 18:13:24,065 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026751.1 (Klebsiella pneumoniae, AR_0066) loaded with 23 markers.\n",
      "2024-04-17 18:13:24,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088985.1 (Klebsiella pneumoniae, 53334) loaded with 17 markers.\n",
      "2024-04-17 18:13:24,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088988.1 (Klebsiella pneumoniae, 53322) loaded with 16 markers.\n",
      "2024-04-17 18:13:24,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083043.1 (Klebsiella pneumoniae, 400195-2-18) loaded with 26 markers.\n",
      "2024-04-17 18:13:24,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083047.1 (Klebsiella pneumoniae, 400195-1-18) loaded with 26 markers.\n",
      "2024-04-17 18:13:24,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062792.1 (Klebsiella pneumoniae, 33Kpn12) loaded with 18 markers.\n",
      "2024-04-17 18:13:24,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016926.1 (Klebsiella pneumoniae, 23) loaded with 17 markers.\n",
      "2024-04-17 18:13:24,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019219.1 (Klebsiella pneumoniae, 1756) loaded with 17 markers.\n",
      "2024-04-17 18:13:24,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006722.1 (Klebsiella pneumoniae, 1158) loaded with 16 markers.\n",
      "2024-04-17 18:13:24,695 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016923.1 (Klebsiella pneumoniae, 11) loaded with 23 markers.\n",
      "2024-04-17 18:13:24,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084873.1 (Klebsiella pneumoniae, 03-9138-2) loaded with 16 markers.\n",
      "2024-04-17 18:13:24,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083051.1 (Klebsiella pneumoniae, 0213293972) loaded with 16 markers.\n",
      "2024-04-17 18:13:24,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031101.1 (Leclercia sp., w17) loaded with 34 markers.\n",
      "2024-04-17 18:13:24,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045845.1 (Kluyvera intermedia, N2-1) loaded with 34 markers.\n",
      "2024-04-17 18:13:25,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045843.1 (Kluyvera intermedia, HR2) loaded with 34 markers.\n",
      "2024-04-17 18:13:25,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055883.1 (Enterobacter sp., RHBSTW-00975) loaded with 34 markers.\n",
      "2024-04-17 18:13:25,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056393.1 (Enterobacter sp., RHBSTW-00593) loaded with 32 markers.\n",
      "2024-04-17 18:13:25,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056552.1 (Enterobacter sp., RHBSTW-00422) loaded with 32 markers.\n",
      "2024-04-17 18:13:25,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077407.1 (Enterobacter roggenkampii, FDAARGOS_1430) loaded with 36 markers.\n",
      "2024-04-17 18:13:25,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017184.1 (Enterobacter roggenkampii, DSM_16690) loaded with 36 markers.\n",
      "2024-04-17 18:13:25,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027618.1 (Enterobacter cloacae, FDA-CDC-AR_0132) loaded with 36 markers.\n",
      "2024-04-17 18:13:25,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045769.1 (Enterobacter cancerogenus, MiY-F) loaded with 31 markers.\n",
      "2024-04-17 18:13:25,620 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077290.1 (Enterobacter cancerogenus, FDAARGOS_1428) loaded with 30 markers.\n",
      "2024-04-17 18:13:25,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025225.1 (Enterobacter cancerogenus, CR-Eb1) loaded with 32 markers.\n",
      "2024-04-17 18:13:25,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084290.1 (Citrobacter werkmanii, LHC5-1) loaded with 41 markers.\n",
      "2024-04-17 18:13:25,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056232.1 (Citrobacter freundii, RHBSTW-00965) loaded with 44 markers.\n",
      "2024-04-17 18:13:25,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055551.1 (Citrobacter freundii, RHBSTW-00200) loaded with 45 markers.\n",
      "2024-04-17 18:13:26,074 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056907.1 (Citrobacter freundii, RHBSTW-00011) loaded with 42 markers.\n",
      "2024-04-17 18:13:26,170 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007557.1 (Citrobacter freundii, CFNIH1) loaded with 43 markers.\n",
      "2024-04-17 18:13:26,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060441.1 (Citrobacter cronae, S175-1) loaded with 43 markers.\n",
      "2024-04-17 18:13:26,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009451.1 (Cedecea neteri, SSMD04) loaded with 25 markers.\n",
      "2024-04-17 18:13:26,396 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034819.1 (Salmonella sp., SSDFZ54) loaded with 44 markers.\n",
      "2024-04-17 18:13:26,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084216.1 (Salmonella sp., JXY0409-18) loaded with 42 markers.\n",
      "2024-04-17 18:13:26,518 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084001.1 (Salmonella sp., A7) loaded with 43 markers.\n",
      "2024-04-17 18:13:26,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084194.1 (Salmonella sp., A39) loaded with 44 markers.\n",
      "2024-04-17 18:13:26,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083731.1 (Salmonella sp., A29-2) loaded with 41 markers.\n",
      "2024-04-17 18:13:26,723 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089788.1 (Salmonella enterica, ZTA19_00847) loaded with 41 markers.\n",
      "2024-04-17 18:13:26,787 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089789.1 (Salmonella enterica, ZTA19_00831) loaded with 41 markers.\n",
      "2024-04-17 18:13:26,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089790.1 (Salmonella enterica, ZTA19_00830) loaded with 41 markers.\n",
      "2024-04-17 18:13:26,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089791.1 (Salmonella enterica, ZTA19_00820) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089792.1 (Salmonella enterica, ZTA19_00816) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,076 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089794.1 (Salmonella enterica, ZTA19_00814) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089795.1 (Salmonella enterica, ZTA19_00813) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089797.1 (Salmonella enterica, ZTA19_00790) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,305 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089798.1 (Salmonella enterica, ZTA19_00789) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,394 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089799.1 (Salmonella enterica, ZTA19_00785) loaded with 41 markers.\n",
      "2024-04-17 18:13:27,492 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090304.1 (Salmonella enterica, ZJUJY) loaded with 44 markers.\n",
      "2024-04-17 18:13:27,586 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085811.1 (Salmonella enterica, Zapdos) loaded with 42 markers.\n",
      "2024-04-17 18:13:27,696 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011428.1 (Salmonella enterica, YU39) loaded with 44 markers.\n",
      "2024-04-17 18:13:27,803 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014358.1 (Salmonella enterica, YU15) loaded with 44 markers.\n",
      "2024-04-17 18:13:27,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035547.1 (Salmonella enterica, YU07-18) loaded with 44 markers.\n",
      "2024-04-17 18:13:28,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022168.1 (Salmonella enterica, WW012) loaded with 44 markers.\n",
      "2024-04-17 18:13:28,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040575.1 (Salmonella enterica, WGS1146) loaded with 42 markers.\n",
      "2024-04-17 18:13:28,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016357.1 (Salmonella enterica, WA_14882) loaded with 43 markers.\n",
      "2024-04-17 18:13:28,313 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085820.1 (Salmonella enterica, Wartortle) loaded with 44 markers.\n",
      "2024-04-17 18:13:28,415 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039436.1 (Salmonella enterica, VNSEC031) loaded with 45 markers.\n",
      "2024-04-17 18:13:28,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039437.1 (Salmonella enterica, VNSEC023) loaded with 43 markers.\n",
      "2024-04-17 18:13:28,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039439.1 (Salmonella enterica, VNSEC013) loaded with 41 markers.\n",
      "2024-04-17 18:13:28,723 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007804.1 (Salmonella enterica, VNP20009) loaded with 44 markers.\n",
      "2024-04-17 18:13:28,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT795114.1 (Salmonella enterica, VNB151-sc-2315230) loaded with 44 markers.\n",
      "2024-04-17 18:13:28,949 [DEBUG - chronostrain.database.parser.json] - Strain NC_021902.1 (Salmonella enterica, USMARC-S3124.1) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,073 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040686.1 (Salmonella enterica, USDA15WA-1) loaded with 44 markers.\n",
      "2024-04-17 18:13:29,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025241.1 (Salmonella enterica, USDA-ARS-USMARC-1929) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025237.1 (Salmonella enterica, USDA-ARS-USMARC-1928) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025234.1 (Salmonella enterica, USDA-ARS-USMARC-1926) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025232.1 (Salmonella enterica, USDA-ARS-USMARC-1925) loaded with 42 markers.\n",
      "2024-04-17 18:13:29,548 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025230.1 (Salmonella enterica, USDA-ARS-USMARC-1924) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025273.1 (Salmonella enterica, USDA-ARS-USMARC-1923) loaded with 43 markers.\n",
      "2024-04-17 18:13:29,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017973.1 (Salmonella enterica, USDA-ARS-USMARC-1912) loaded with 34 markers.\n",
      "2024-04-17 18:13:29,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017972.1 (Salmonella enterica, USDA-ARS-USMARC-1904) loaded with 39 markers.\n",
      "2024-04-17 18:13:29,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007222.1 (Salmonella enterica, USDA-ARS-USMARC-1903) loaded with 39 markers.\n",
      "2024-04-17 18:13:29,851 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017971.1 (Salmonella enterica, USDA-ARS-USMARC-1901) loaded with 39 markers.\n",
      "2024-04-17 18:13:29,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017970.1 (Salmonella enterica, USDA-ARS-USMARC-1900) loaded with 39 markers.\n",
      "2024-04-17 18:13:29,984 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007235.2 (Salmonella enterica, USDA-ARS-USMARC-1899) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014971.2 (Salmonella enterica, USDA-ARS-USMARC-1898) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,124 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014977.1 (Salmonella enterica, USDA-ARS-USMARC-1896) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014981.1 (Salmonella enterica, USDA-ARS-USMARC-1880) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014982.2 (Salmonella enterica, USDA-ARS-USMARC-1810) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014969.1 (Salmonella enterica, USDA-ARS-USMARC-1808) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,410 [DEBUG - chronostrain.database.parser.json] - Strain NC_016863.1 (Salmonella enterica, UK-1) loaded with 44 markers.\n",
      "2024-04-17 18:13:30,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020101.1 (Salmonella enterica, UFPRLABMOR1) loaded with 41 markers.\n",
      "2024-04-17 18:13:30,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023975.1 (Salmonella enterica, Ty21a) loaded with 42 markers.\n",
      "2024-04-17 18:13:30,611 [DEBUG - chronostrain.database.parser.json] - Strain NC_021176.1 (Salmonella enterica, Ty21a) loaded with 42 markers.\n",
      "2024-04-17 18:13:30,679 [DEBUG - chronostrain.database.parser.json] - Strain NC_004631.1 (Salmonella enterica, Ty2) loaded with 42 markers.\n",
      "2024-04-17 18:13:30,759 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007505.1 (Salmonella enterica, TXSC_TXSC08-19) loaded with 39 markers.\n",
      "2024-04-17 18:13:30,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019649.1 (Salmonella enterica, TW-Stm6) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,013 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040458.1 (Salmonella enterica, TJWQ005) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,115 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085987.1 (Salmonella enterica, SZL_38) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,190 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085807.1 (Salmonella enterica, Suicune) loaded with 42 markers.\n",
      "2024-04-17 18:13:31,281 [DEBUG - chronostrain.database.parser.json] - Strain NC_003197.2 (Salmonella enterica, subsp._enterica_serovar_Typhimurium_str._LT2) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT855376.1 (Salmonella enterica, STMU2UK) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016385.1 (Salmonella enterica, ST931R) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,535 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050734.1 (Salmonella enterica, ST90) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,623 [DEBUG - chronostrain.database.parser.json] - Strain NC_016857.1 (Salmonella enterica, ST4_74) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,716 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050750.1 (Salmonella enterica, ST46) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050753.1 (Salmonella enterica, ST45) loaded with 44 markers.\n",
      "2024-04-17 18:13:31,905 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019405.1 (Salmonella enterica, ST350) loaded with 40 markers.\n",
      "2024-04-17 18:13:31,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019414.1 (Salmonella enterica, ST278) loaded with 42 markers.\n",
      "2024-04-17 18:13:32,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035301.1 (Salmonella enterica, ST1539) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050726.1 (Salmonella enterica, ST113) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021909.1 (Salmonella enterica, ST1120) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,285 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050728.1 (Salmonella enterica, ST106) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050731.1 (Salmonella enterica, ST101) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053870.1 (Salmonella enterica, SS2017) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058807.1 (Salmonella enterica, SRC27) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092911.1 (Salmonella enterica, SR33) loaded with 40 markers.\n",
      "2024-04-17 18:13:32,630 [DEBUG - chronostrain.database.parser.json] - Strain NC_010102.1 (Salmonella enterica, SPB7) loaded with 43 markers.\n",
      "2024-04-17 18:13:32,707 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077668.1 (Salmonella enterica, SP) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074092.1 (Salmonella enterica, SO_8752_Stm) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,849 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN999997.1 (Salmonella enterica, SO4698-09) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014536.1 (Salmonella enterica, SO3) loaded with 44 markers.\n",
      "2024-04-17 18:13:32,991 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032494.1 (Salmonella enterica, SO21) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014356.1 (Salmonella enterica, SO2) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060515.1 (Salmonella enterica, SLR1_8094) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053865.1 (Salmonella enterica, SL7207) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,390 [DEBUG - chronostrain.database.parser.json] - Strain NC_011083.1 (Salmonella enterica, SL476) loaded with 51 markers.\n",
      "2024-04-17 18:13:33,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032490.1 (Salmonella enterica, SL26) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,535 [DEBUG - chronostrain.database.parser.json] - Strain NC_011080.1 (Salmonella enterica, SL254) loaded with 43 markers.\n",
      "2024-04-17 18:13:33,613 [DEBUG - chronostrain.database.parser.json] - Strain NC_016810.1 (Salmonella enterica, SL1344) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043214.1 (Salmonella enterica, SL-312) loaded with 41 markers.\n",
      "2024-04-17 18:13:33,774 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047522.1 (Salmonella enterica, SJTUF11216) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047525.1 (Salmonella enterica, SJTUF11077) loaded with 44 markers.\n",
      "2024-04-17 18:13:33,954 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047527.1 (Salmonella enterica, SJTUF10648) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047529.1 (Salmonella enterica, SJTUF10640) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047531.1 (Salmonella enterica, SJTUF10484) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,185 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047535.1 (Salmonella enterica, SJTUF10452) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,257 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047537.1 (Salmonella enterica, SJTUF10405) loaded with 43 markers.\n",
      "2024-04-17 18:13:34,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047540.1 (Salmonella enterica, SJTUF10359) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,425 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047542.1 (Salmonella enterica, SJTUF10330) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047544.1 (Salmonella enterica, SJTUF10250) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,593 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047546.1 (Salmonella enterica, SJTUF10236) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047553.1 (Salmonella enterica, SJTUF10231) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,748 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047548.1 (Salmonella enterica, SJTUF10169) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047555.1 (Salmonella enterica, SJTUF10112) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047550.1 (Salmonella enterica, SJTUF10057) loaded with 44 markers.\n",
      "2024-04-17 18:13:34,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016579.1 (Salmonella enterica, SH13-006) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,060 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016586.1 (Salmonella enterica, SH13-004) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,185 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016504.1 (Salmonella enterica, SH12-007) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016507.1 (Salmonella enterica, SH12-003) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066851.1 (Salmonella enterica, SH-2813-Parental_2) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036167.1 (Salmonella enterica, sg_wt8) loaded with 42 markers.\n",
      "2024-04-17 18:13:35,637 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036168.1 (Salmonella enterica, sg_wt7) loaded with 44 markers.\n",
      "2024-04-17 18:13:35,709 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017723.1 (Salmonella enterica, SGSC_2518) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019176.1 (Salmonella enterica, SGSC_2215) loaded with 41 markers.\n",
      "2024-04-17 18:13:35,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017727.1 (Salmonella enterica, SGSC_2206) loaded with 42 markers.\n",
      "2024-04-17 18:13:35,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017728.1 (Salmonella enterica, SGSC_2193) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023166.1 (Salmonella enterica, SGB23) loaded with 41 markers.\n",
      "2024-04-17 18:13:36,087 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081189.2 (Salmonella enterica, sg1722-2) loaded with 43 markers.\n",
      "2024-04-17 18:13:36,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081187.1 (Salmonella enterica, sg1722-1) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067339.1 (Salmonella enterica, Se32) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,326 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051213.1 (Salmonella enterica, SCSW714) loaded with 40 markers.\n",
      "2024-04-17 18:13:36,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047115.1 (Salmonella enterica, SCSM4.1) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,477 [DEBUG - chronostrain.database.parser.json] - Strain NC_006905.1 (Salmonella enterica, SC-B67) loaded with 41 markers.\n",
      "2024-04-17 18:13:36,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041208.1 (Salmonella enterica, SAP18-8729) loaded with 43 markers.\n",
      "2024-04-17 18:13:36,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040900.1 (Salmonella enterica, SAP18-6199) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040568.1 (Salmonella enterica, SAP17-8290) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,801 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040564.1 (Salmonella enterica, SAP17-7699) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040562.1 (Salmonella enterica, SAP17-7399) loaded with 44 markers.\n",
      "2024-04-17 18:13:36,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040566.1 (Salmonella enterica, SAP17-7299) loaded with 44 markers.\n",
      "2024-04-17 18:13:37,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041026.1 (Salmonella enterica, SA20143792) loaded with 44 markers.\n",
      "2024-04-17 18:13:37,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038593.1 (Salmonella enterica, SA20130280) loaded with 43 markers.\n",
      "2024-04-17 18:13:37,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043222.1 (Salmonella enterica, SA20114778WT) loaded with 44 markers.\n",
      "2024-04-17 18:13:37,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030190.1 (Salmonella enterica, SA20104250) loaded with 40 markers.\n",
      "2024-04-17 18:13:37,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030185.1 (Salmonella enterica, SA20094620) loaded with 40 markers.\n",
      "2024-04-17 18:13:37,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022497.1 (Salmonella enterica, SA20084699) loaded with 42 markers.\n",
      "2024-04-17 18:13:37,518 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040668.1 (Salmonella enterica, SA20082869) loaded with 44 markers.\n",
      "2024-04-17 18:13:37,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040651.1 (Salmonella enterica, SA20070548) loaded with 44 markers.\n",
      "2024-04-17 18:13:37,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038591.1 (Salmonella enterica, SA20061017) loaded with 43 markers.\n",
      "2024-04-17 18:13:37,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030202.1 (Salmonella enterica, SA20052327) loaded with 43 markers.\n",
      "2024-04-17 18:13:37,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030225.1 (Salmonella enterica, SA20041605) loaded with 41 markers.\n",
      "2024-04-17 18:13:37,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022494.1 (Salmonella enterica, SA20035215) loaded with 42 markers.\n",
      "2024-04-17 18:13:37,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022491.1 (Salmonella enterica, SA20031783) loaded with 41 markers.\n",
      "2024-04-17 18:13:38,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022500.1 (Salmonella enterica, SA20030505) loaded with 41 markers.\n",
      "2024-04-17 18:13:38,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029992.1 (Salmonella enterica, SA20011914) loaded with 42 markers.\n",
      "2024-04-17 18:13:38,177 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029991.1 (Salmonella enterica, SA19981204) loaded with 42 markers.\n",
      "2024-04-17 18:13:38,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030175.1 (Salmonella enterica, SA19950795) loaded with 42 markers.\n",
      "2024-04-17 18:13:38,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012921.1 (Salmonella enterica, SA02DT10168701) loaded with 41 markers.\n",
      "2024-04-17 18:13:38,420 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016521.1 (Salmonella enterica, SA02DT09004001) loaded with 41 markers.\n",
      "2024-04-17 18:13:38,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016530.1 (Salmonella enterica, SA01AB09084001) loaded with 41 markers.\n",
      "2024-04-17 18:13:38,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035915.1 (Salmonella enterica, S61394) loaded with 44 markers.\n",
      "2024-04-17 18:13:38,735 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061115.1 (Salmonella enterica, S585) loaded with 44 markers.\n",
      "2024-04-17 18:13:38,839 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061119.1 (Salmonella enterica, S520) loaded with 44 markers.\n",
      "2024-04-17 18:13:38,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035917.1 (Salmonella enterica, S44712) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061122.1 (Salmonella enterica, S441) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061124.1 (Salmonella enterica, S438) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086118.1 (Salmonella enterica, S34) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061126.1 (Salmonella enterica, S304) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085699.1 (Salmonella enterica, S29) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085696.1 (Salmonella enterica, S25) loaded with 41 markers.\n",
      "2024-04-17 18:13:39,635 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081858.1 (Salmonella enterica, S16) loaded with 43 markers.\n",
      "2024-04-17 18:13:39,706 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089209.1 (Salmonella enterica, S15) loaded with 45 markers.\n",
      "2024-04-17 18:13:39,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR881463.1 (Salmonella enterica, S.Tm_LT2p22_assembled) loaded with 44 markers.\n",
      "2024-04-17 18:13:39,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019418.1 (Salmonella enterica, S-1850) loaded with 41 markers.\n",
      "2024-04-17 18:13:39,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019416.1 (Salmonella enterica, S-1687) loaded with 41 markers.\n",
      "2024-04-17 18:13:40,017 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034713.1 (Salmonella enterica, RSE15) loaded with 39 markers.\n",
      "2024-04-17 18:13:40,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034719.1 (Salmonella enterica, RSE04) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047323.1 (Salmonella enterica, RM13672) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,238 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022658.1 (Salmonella enterica, RM11060) loaded with 43 markers.\n",
      "2024-04-17 18:13:40,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013702.1 (Salmonella enterica, RM10961) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,388 [DEBUG - chronostrain.database.parser.json] - Strain NC_012125.1 (Salmonella enterica, RKS4594) loaded with 41 markers.\n",
      "2024-04-17 18:13:40,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085809.1 (Salmonella enterica, Raikou) loaded with 42 markers.\n",
      "2024-04-17 18:13:40,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046429.1 (Salmonella enterica, R19.2839) loaded with 42 markers.\n",
      "2024-04-17 18:13:40,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065567.1 (Salmonella enterica, R18.1078) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063294.1 (Salmonella enterica, R17.1451) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,745 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043773.1 (Salmonella enterica, QH) loaded with 41 markers.\n",
      "2024-04-17 18:13:40,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026327.1 (Salmonella enterica, PU131) loaded with 41 markers.\n",
      "2024-04-17 18:13:40,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037877.1 (Salmonella enterica, PNCS015054) loaded with 44 markers.\n",
      "2024-04-17 18:13:40,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044961.1 (Salmonella enterica, PNCS014881) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039860.1 (Salmonella enterica, PNCS014880) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040321.1 (Salmonella enterica, PNCS014879) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039856.1 (Salmonella enterica, PNCS014876) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037882.1 (Salmonella enterica, PNCS014875) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,340 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039610.1 (Salmonella enterica, PNCS014875) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,407 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039607.1 (Salmonella enterica, PNCS014873) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039603.1 (Salmonella enterica, PNCS014868) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039599.1 (Salmonella enterica, PNCS014867) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,639 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038849.1 (Salmonella enterica, PNCS014866) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039595.1 (Salmonella enterica, PNCS014865) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,776 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039854.1 (Salmonella enterica, PNCS014864) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037879.1 (Salmonella enterica, PNCS014863) loaded with 44 markers.\n",
      "2024-04-17 18:13:41,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039593.1 (Salmonella enterica, PNCS014863) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039591.1 (Salmonella enterica, PNCS014862) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039588.1 (Salmonella enterica, PNCS014861) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,157 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039719.1 (Salmonella enterica, PNCS014860) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039585.1 (Salmonella enterica, PNCS014859) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039582.1 (Salmonella enterica, PNCS014858) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039579.1 (Salmonella enterica, PNCS014857) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039572.1 (Salmonella enterica, PNCS014855) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,515 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037874.1 (Salmonella enterica, PNCS014854) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039713.1 (Salmonella enterica, PNCS014853) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,657 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039569.1 (Salmonella enterica, PNCS014852) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,749 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038847.1 (Salmonella enterica, PNCS014851) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039567.1 (Salmonella enterica, PNCS014850) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039565.1 (Salmonella enterica, PNCS014849) loaded with 44 markers.\n",
      "2024-04-17 18:13:42,999 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039564.1 (Salmonella enterica, PNCS014848) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039561.1 (Salmonella enterica, PNCS014847) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,170 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039558.1 (Salmonella enterica, PNCS014846) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037881.1 (Salmonella enterica, PNCS009991) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040318.1 (Salmonella enterica, PNCS009887) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040648.1 (Salmonella enterica, PNCS009880) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036174.1 (Salmonella enterica, PNCS009777) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044967.1 (Salmonella enterica, PNCS007098) loaded with 42 markers.\n",
      "2024-04-17 18:13:43,673 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044957.1 (Salmonella enterica, PNCS007087) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039716.1 (Salmonella enterica, PNCS000211) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,814 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012091.1 (Salmonella enterica, PM016_13) loaded with 42 markers.\n",
      "2024-04-17 18:13:43,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025555.1 (Salmonella enterica, PIR00538) loaded with 44 markers.\n",
      "2024-04-17 18:13:43,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025217.1 (Salmonella enterica, PIR00537) loaded with 39 markers.\n",
      "2024-04-17 18:13:44,020 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092861.1 (Salmonella enterica, P164045) loaded with 40 markers.\n",
      "2024-04-17 18:13:44,089 [DEBUG - chronostrain.database.parser.json] - Strain NC_016832.1 (Salmonella enterica, P-stx-12) loaded with 42 markers.\n",
      "2024-04-17 18:13:44,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040380.1 (Salmonella enterica, OSF005645) loaded with 39 markers.\n",
      "2024-04-17 18:13:44,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051269.1 (Salmonella enterica, OLF_FSR1_WB_Finch_ST-13) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,288 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051267.1 (Salmonella enterica, OLF-FSR1_WB_Hawk_ST-33) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,359 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075372.1 (Salmonella enterica, no75) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT571437.1 (Salmonella enterica, NCTC5706) loaded with 43 markers.\n",
      "2024-04-17 18:13:44,467 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN868943.1 (Salmonella enterica, NCTC10384) loaded with 40 markers.\n",
      "2024-04-17 18:13:44,535 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041976.1 (Salmonella enterica, NCCP_16207) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,602 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068696.1 (Salmonella enterica, NCCP16345) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015157.1 (Salmonella enterica, NC983) loaded with 44 markers.\n",
      "2024-04-17 18:13:44,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012930.1 (Salmonella enterica, N13-01290) loaded with 41 markers.\n",
      "2024-04-17 18:13:44,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085810.1 (Salmonella enterica, Moltres) loaded with 42 markers.\n",
      "2024-04-17 18:13:44,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU943338.1 (Salmonella enterica, MDUST348) loaded with 42 markers.\n",
      "2024-04-17 18:13:44,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU943336.1 (Salmonella enterica, MDUST305) loaded with 42 markers.\n",
      "2024-04-17 18:13:44,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU943337.1 (Salmonella enterica, MDUST255) loaded with 42 markers.\n",
      "2024-04-17 18:13:45,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030936.1 (Salmonella enterica, LXYSH) loaded with 42 markers.\n",
      "2024-04-17 18:13:45,165 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905063.1 (Salmonella enterica, lupe_GEN0059-sc-1979081) loaded with 42 markers.\n",
      "2024-04-17 18:13:45,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064263.1 (Salmonella enterica, LT7) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060507.1 (Salmonella enterica, LT2) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018219.1 (Salmonella enterica, LSP_389_97) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,716 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045063.1 (Salmonella enterica, LG26) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,823 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045059.1 (Salmonella enterica, LG25) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045056.1 (Salmonella enterica, LG24) loaded with 44 markers.\n",
      "2024-04-17 18:13:45,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067397.1 (Salmonella enterica, LB5000) loaded with 45 markers.\n",
      "2024-04-17 18:13:46,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023319.1 (Salmonella enterica, L-4741) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023317.1 (Salmonella enterica, L-4681) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023315.1 (Salmonella enterica, L-4614) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023313.1 (Salmonella enterica, L-4605) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023311.1 (Salmonella enterica, L-4596) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023309.1 (Salmonella enterica, L-4578) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,722 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023306.1 (Salmonella enterica, L-4567) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023304.1 (Salmonella enterica, L-4551) loaded with 44 markers.\n",
      "2024-04-17 18:13:46,903 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023303.1 (Salmonella enterica, L-4526) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023300.1 (Salmonella enterica, L-4445) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023299.1 (Salmonella enterica, L-4334) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,221 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023294.1 (Salmonella enterica, L-4261) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,355 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023292.1 (Salmonella enterica, L-4233) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023291.1 (Salmonella enterica, L-4126) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023290.1 (Salmonella enterica, L-3844) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023289.1 (Salmonella enterica, L-3837) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023288.1 (Salmonella enterica, L-3835) loaded with 44 markers.\n",
      "2024-04-17 18:13:47,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061159.1 (Salmonella enterica, K_SA184) loaded with 45 markers.\n",
      "2024-04-17 18:13:47,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047424.1 (Salmonella enterica, KUFSE-SAL0043) loaded with 43 markers.\n",
      "2024-04-17 18:13:48,095 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080960.1 (Salmonella enterica, ISP2825) loaded with 42 markers.\n",
      "2024-04-17 18:13:48,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060132.1 (Salmonella enterica, HA3-IN1) loaded with 42 markers.\n",
      "2024-04-17 18:13:48,261 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060134.1 (Salmonella enterica, HA1-SP5) loaded with 42 markers.\n",
      "2024-04-17 18:13:48,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050130.1 (Salmonella enterica, GSJ_2017-Sal-008) loaded with 44 markers.\n",
      "2024-04-17 18:13:48,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043509.1 (Salmonella enterica, GJ0703-2) loaded with 40 markers.\n",
      "2024-04-17 18:13:48,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025094.1 (Salmonella enterica, FORC_080) loaded with 43 markers.\n",
      "2024-04-17 18:13:48,659 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025736.1 (Salmonella enterica, FORC_079) loaded with 44 markers.\n",
      "2024-04-17 18:13:48,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015574.1 (Salmonella enterica, FORC_038) loaded with 43 markers.\n",
      "2024-04-17 18:13:48,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029029.1 (Salmonella enterica, FORC88) loaded with 39 markers.\n",
      "2024-04-17 18:13:48,879 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020565.1 (Salmonella enterica, FORC58) loaded with 41 markers.\n",
      "2024-04-17 18:13:48,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030029.1 (Salmonella enterica, FORC098) loaded with 39 markers.\n",
      "2024-04-17 18:13:49,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014051.2 (Salmonella enterica, FDAARGOS_94) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065639.1 (Salmonella enterica, FDAARGOS_928) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,234 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065718.1 (Salmonella enterica, FDAARGOS_878) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,322 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041005.1 (Salmonella enterica, FDAARGOS_768) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046277.1 (Salmonella enterica, FDAARGOS_710) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046279.1 (Salmonella enterica, FDAARGOS_707) loaded with 42 markers.\n",
      "2024-04-17 18:13:49,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046283.1 (Salmonella enterica, FDAARGOS_687) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026976.1 (Salmonella enterica, FDAARGOS_54) loaded with 41 markers.\n",
      "2024-04-17 18:13:49,695 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023512.1 (Salmonella enterica, FDAARGOS_373) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023508.1 (Salmonella enterica, FDAARGOS_368) loaded with 43 markers.\n",
      "2024-04-17 18:13:49,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022070.1 (Salmonella enterica, FDAARGOS_321) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027414.1 (Salmonella enterica, FDAARGOS_320) loaded with 44 markers.\n",
      "2024-04-17 18:13:49,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027412.1 (Salmonella enterica, FDAARGOS_319) loaded with 41 markers.\n",
      "2024-04-17 18:13:50,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027410.1 (Salmonella enterica, FDAARGOS_317) loaded with 44 markers.\n",
      "2024-04-17 18:13:50,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022062.1 (Salmonella enterica, FDAARGOS_312) loaded with 44 markers.\n",
      "2024-04-17 18:13:50,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069518.1 (Salmonella enterica, FDAARGOS_1271) loaded with 39 markers.\n",
      "2024-04-17 18:13:50,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066009.1 (Salmonella enterica, FDAARGOS_1067) loaded with 44 markers.\n",
      "2024-04-17 18:13:50,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040379.1 (Salmonella enterica, FCC0123) loaded with 39 markers.\n",
      "2024-04-17 18:13:50,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063684.2 (Salmonella enterica, FC745) loaded with 43 markers.\n",
      "2024-04-17 18:13:50,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905140.1 (Salmonella enterica, ERL082356) loaded with 42 markers.\n",
      "2024-04-17 18:13:50,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067091.1 (Salmonella enterica, ER3625) loaded with 45 markers.\n",
      "2024-04-17 18:13:50,659 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085808.1 (Salmonella enterica, Entei) loaded with 42 markers.\n",
      "2024-04-17 18:13:50,756 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905143.1 (Salmonella enterica, E98_3139-sc-1927833) loaded with 42 markers.\n",
      "2024-04-17 18:13:50,850 [DEBUG - chronostrain.database.parser.json] - Strain NC_022544.1 (Salmonella enterica, DT2) loaded with 44 markers.\n",
      "2024-04-17 18:13:50,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029568.1 (Salmonella enterica, DA34837) loaded with 44 markers.\n",
      "2024-04-17 18:13:50,997 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029595.1 (Salmonella enterica, DA34833) loaded with 44 markers.\n",
      "2024-04-17 18:13:51,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029593.1 (Salmonella enterica, DA34827) loaded with 44 markers.\n",
      "2024-04-17 18:13:51,140 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029567.1 (Salmonella enterica, DA34821) loaded with 44 markers.\n",
      "2024-04-17 18:13:51,210 [DEBUG - chronostrain.database.parser.json] - Strain NC_016854.1 (Salmonella enterica, D23580) loaded with 44 markers.\n",
      "2024-04-17 18:13:51,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074098.1 (Salmonella enterica, C_cip) loaded with 41 markers.\n",
      "2024-04-17 18:13:51,362 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009561.1 (Salmonella enterica, CVM_N18486) loaded with 43 markers.\n",
      "2024-04-17 18:13:51,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010284.1 (Salmonella enterica, CVM_N1543) loaded with 43 markers.\n",
      "2024-04-17 18:13:51,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051307.1 (Salmonella enterica, CVM_35189) loaded with 39 markers.\n",
      "2024-04-17 18:13:51,586 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051310.1 (Salmonella enterica, CVM_35161) loaded with 41 markers.\n",
      "2024-04-17 18:13:51,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051313.1 (Salmonella enterica, CVM_35153) loaded with 42 markers.\n",
      "2024-04-17 18:13:51,709 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051317.1 (Salmonella enterica, CVM_34530) loaded with 43 markers.\n",
      "2024-04-17 18:13:51,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051329.1 (Salmonella enterica, CVM_34514) loaded with 43 markers.\n",
      "2024-04-17 18:13:51,895 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051338.1 (Salmonella enterica, CVM_33836) loaded with 44 markers.\n",
      "2024-04-17 18:13:51,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051346.1 (Salmonella enterica, CVM_30177) loaded with 41 markers.\n",
      "2024-04-17 18:13:52,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051350.1 (Salmonella enterica, CVM_30168) loaded with 42 markers.\n",
      "2024-04-17 18:13:52,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053048.1 (Salmonella enterica, CVM_30155) loaded with 42 markers.\n",
      "2024-04-17 18:13:52,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051353.1 (Salmonella enterica, CVM_30148) loaded with 44 markers.\n",
      "2024-04-17 18:13:52,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051358.1 (Salmonella enterica, CVM_28322) loaded with 41 markers.\n",
      "2024-04-17 18:13:52,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051363.1 (Salmonella enterica, CVM_28321-b) loaded with 44 markers.\n",
      "2024-04-17 18:13:52,533 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053051.1 (Salmonella enterica, CVM_28321-a) loaded with 44 markers.\n",
      "2024-04-17 18:13:52,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051366.1 (Salmonella enterica, CVM_28296) loaded with 41 markers.\n",
      "2024-04-17 18:13:52,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051372.1 (Salmonella enterica, CVM_24374) loaded with 42 markers.\n",
      "2024-04-17 18:13:52,717 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051380.1 (Salmonella enterica, CVM_24362) loaded with 44 markers.\n",
      "2024-04-17 18:13:52,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051381.1 (Salmonella enterica, CVM_24359) loaded with 41 markers.\n",
      "2024-04-17 18:13:52,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051384.1 (Salmonella enterica, CVM_24355) loaded with 46 markers.\n",
      "2024-04-17 18:13:52,945 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051386.1 (Salmonella enterica, CVM_24350) loaded with 44 markers.\n",
      "2024-04-17 18:13:53,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053055.1 (Salmonella enterica, CVM_22543) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,108 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010281.1 (Salmonella enterica, CVM_22513) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010280.1 (Salmonella enterica, CVM_22462) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,298 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010279.1 (Salmonella enterica, CVM_22425) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010283.1 (Salmonella enterica, CVM_21550) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010282.1 (Salmonella enterica, CVM_21538) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051404.1 (Salmonella enterica, CVM_20762) loaded with 44 markers.\n",
      "2024-04-17 18:13:53,593 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051410.1 (Salmonella enterica, CVM_20760) loaded with 41 markers.\n",
      "2024-04-17 18:13:53,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051413.1 (Salmonella enterica, CVM_20749) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,721 [DEBUG - chronostrain.database.parser.json] - Strain NC_011094.1 (Salmonella enterica, CVM19633) loaded with 42 markers.\n",
      "2024-04-17 18:13:53,783 [DEBUG - chronostrain.database.parser.json] - Strain NC_003198.1 (Salmonella enterica, CT18) loaded with 42 markers.\n",
      "2024-04-17 18:13:53,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009559.1 (Salmonella enterica, CMCC_50503) loaded with 43 markers.\n",
      "2024-04-17 18:13:53,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033344.1 (Salmonella enterica, CFSAN076211) loaded with 39 markers.\n",
      "2024-04-17 18:13:53,995 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033345.1 (Salmonella enterica, CFSAN076210) loaded with 39 markers.\n",
      "2024-04-17 18:13:54,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024168.1 (Salmonella enterica, CFSAN070643__4083H_attTn7::_Ptrc-gfpUV-cat__bright_fluorescence__1_lactose_operon) loaded with 39 markers.\n",
      "2024-04-17 18:13:54,136 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028311.1 (Salmonella enterica, CFSAN067218) loaded with 41 markers.\n",
      "2024-04-17 18:13:54,201 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028314.1 (Salmonella enterica, CFSAN067217) loaded with 45 markers.\n",
      "2024-04-17 18:13:54,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028318.1 (Salmonella enterica, CFSAN067216) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028169.1 (Salmonella enterica, CFSAN064034) loaded with 43 markers.\n",
      "2024-04-17 18:13:54,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075107.1 (Salmonella enterica, CFSAN060809) loaded with 42 markers.\n",
      "2024-04-17 18:13:54,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075108.1 (Salmonella enterica, CFSAN060808) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075110.1 (Salmonella enterica, CFSAN060804) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077691.1 (Salmonella enterica, CFSAN058620) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077693.1 (Salmonella enterica, CFSAN058605) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,861 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077696.1 (Salmonella enterica, CFSAN058598) loaded with 44 markers.\n",
      "2024-04-17 18:13:54,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077702.1 (Salmonella enterica, CFSAN058588) loaded with 43 markers.\n",
      "2024-04-17 18:13:55,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077704.1 (Salmonella enterica, CFSAN058565) loaded with 41 markers.\n",
      "2024-04-17 18:13:55,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029041.1 (Salmonella enterica, CFSAN051295) loaded with 43 markers.\n",
      "2024-04-17 18:13:55,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029040.1 (Salmonella enterica, CFSAN047866) loaded with 43 markers.\n",
      "2024-04-17 18:13:55,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029038.1 (Salmonella enterica, CFSAN045763) loaded with 43 markers.\n",
      "2024-04-17 18:13:55,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075127.1 (Salmonella enterica, CFSAN044925) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075134.1 (Salmonella enterica, CFSAN044909) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075137.1 (Salmonella enterica, CFSAN044885) loaded with 42 markers.\n",
      "2024-04-17 18:13:55,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075141.1 (Salmonella enterica, CFSAN033950) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019174.1 (Salmonella enterica, CFSAN024229) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028196.1 (Salmonella enterica, CFSAN018747) loaded with 40 markers.\n",
      "2024-04-17 18:13:55,756 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028199.1 (Salmonella enterica, CFSAN018746) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075037.1 (Salmonella enterica, CFSAN012622) loaded with 44 markers.\n",
      "2024-04-17 18:13:55,978 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074663.1 (Salmonella enterica, CFSAN008081) loaded with 44 markers.\n",
      "2024-04-17 18:13:56,073 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019172.1 (Salmonella enterica, CFSAN004175) loaded with 41 markers.\n",
      "2024-04-17 18:13:56,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019206.1 (Salmonella enterica, CFSAN004174) loaded with 41 markers.\n",
      "2024-04-17 18:13:56,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019204.1 (Salmonella enterica, CFSAN004173) loaded with 41 markers.\n",
      "2024-04-17 18:13:56,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039270.1 (Salmonella enterica, CFSAN004025) loaded with 46 markers.\n",
      "2024-04-17 18:13:56,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016012.1 (Salmonella enterica, CFSAN003890) loaded with 43 markers.\n",
      "2024-04-17 18:13:56,402 [DEBUG - chronostrain.database.parser.json] - Strain NC_021812.2 (Salmonella enterica, CFSAN002069) loaded with 41 markers.\n",
      "2024-04-17 18:13:56,505 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP005995.1 (Salmonella enterica, CFSAN002064) loaded with 41 markers.\n",
      "2024-04-17 18:13:56,611 [DEBUG - chronostrain.database.parser.json] - Strain NC_021818.1 (Salmonella enterica, CFSAN002050) loaded with 44 markers.\n",
      "2024-04-17 18:13:56,715 [DEBUG - chronostrain.database.parser.json] - Strain NC_020307.1 (Salmonella enterica, CFSAN001992) loaded with 43 markers.\n",
      "2024-04-17 18:13:56,816 [DEBUG - chronostrain.database.parser.json] - Strain NC_021814.1 (Salmonella enterica, CFSAN001921) loaded with 44 markers.\n",
      "2024-04-17 18:13:56,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012833.1 (Salmonella enterica, CFSAN001588) loaded with 42 markers.\n",
      "2024-04-17 18:13:56,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014994.1 (Salmonella enterica, CFSAN001387) loaded with 39 markers.\n",
      "2024-04-17 18:13:57,048 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039463.1 (Salmonella enterica, CFSAN001140) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039465.1 (Salmonella enterica, CFSAN001118) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,203 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039467.1 (Salmonella enterica, CFSAN001115) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039469.1 (Salmonella enterica, CFSAN001112) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007533.1 (Salmonella enterica, CFSAN001080) loaded with 42 markers.\n",
      "2024-04-17 18:13:57,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039471.1 (Salmonella enterica, CFSAN000970) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039473.1 (Salmonella enterica, CFSAN000968) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039476.1 (Salmonella enterica, CFSAN000963) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039477.1 (Salmonella enterica, CFSAN000961) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039478.1 (Salmonella enterica, CFSAN000960) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039480.1 (Salmonella enterica, CFSAN000958) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039482.1 (Salmonella enterica, CFSAN000954) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039484.1 (Salmonella enterica, CFSAN000952) loaded with 41 markers.\n",
      "2024-04-17 18:13:57,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039486.1 (Salmonella enterica, CFSAN000951) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039487.1 (Salmonella enterica, CFSAN000753) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045754.1 (Salmonella enterica, CFSAN000752) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039489.1 (Salmonella enterica, CFSAN000752) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,224 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039491.1 (Salmonella enterica, CFSAN000700) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045757.1 (Salmonella enterica, CFSAN000669) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039493.1 (Salmonella enterica, CFSAN000669) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,438 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045759.1 (Salmonella enterica, CFSAN000661) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,527 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039496.1 (Salmonella enterica, CFSAN000661) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039498.1 (Salmonella enterica, CFSAN000228) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,670 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039499.1 (Salmonella enterica, CFSAN000212) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,738 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039500.1 (Salmonella enterica, CFSAN000211) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032622.2 (Salmonella enterica, CFSAN000191) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034177.1 (Salmonella enterica, CFSAN000189) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045753.1 (Salmonella enterica, CFSAN000189) loaded with 41 markers.\n",
      "2024-04-17 18:13:58,999 [DEBUG - chronostrain.database.parser.json] - Strain NC_021844.1 (Salmonella enterica, CFSAN000189) loaded with 41 markers.\n",
      "2024-04-17 18:13:59,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033352.2 (Salmonella enterica, CFSA629) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033255.2 (Salmonella enterica, CFSA244) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033226.2 (Salmonella enterica, CFSA122) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,362 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033257.2 (Salmonella enterica, CFSA12) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033348.2 (Salmonella enterica, CFSA1096) loaded with 42 markers.\n",
      "2024-04-17 18:13:59,504 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016517.1 (Salmonella enterica, CE-R2-11-0435) loaded with 41 markers.\n",
      "2024-04-17 18:13:59,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014979.2 (Salmonella enterica, CDC_H2662) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007540.2 (Salmonella enterica, CDC_86-0391) loaded with 39 markers.\n",
      "2024-04-17 18:13:59,698 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017978.1 (Salmonella enterica, CDC_2013K-0218) loaded with 39 markers.\n",
      "2024-04-17 18:13:59,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025243.1 (Salmonella enterica, CDC_2012K-0663) loaded with 43 markers.\n",
      "2024-04-17 18:13:59,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014967.1 (Salmonella enterica, CDC_2011K-1702) loaded with 44 markers.\n",
      "2024-04-17 18:13:59,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017976.1 (Salmonella enterica, CDC_2011K-1674) loaded with 38 markers.\n",
      "2024-04-17 18:13:59,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007523.1 (Salmonella enterica, CDC_2011K-0870) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007559.2 (Salmonella enterica, CDC_2010K-2159) loaded with 42 markers.\n",
      "2024-04-17 18:14:00,094 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014965.1 (Salmonella enterica, CDC_2010K-1587) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,168 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014983.1 (Salmonella enterica, CDC_2009K-2059) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,233 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014975.1 (Salmonella enterica, CDC_2009K-1640) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020752.1 (Salmonella enterica, CDC_2009K-0792) loaded with 39 markers.\n",
      "2024-04-17 18:14:00,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017975.1 (Salmonella enterica, CDC_08-1942) loaded with 39 markers.\n",
      "2024-04-17 18:14:00,419 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017974.1 (Salmonella enterica, CDC_07-0954) loaded with 38 markers.\n",
      "2024-04-17 18:14:00,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007639.1 (Salmonella enterica, C500) loaded with 41 markers.\n",
      "2024-04-17 18:14:00,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085819.1 (Salmonella enterica, Blastoise) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024619.1 (Salmonella enterica, BL10) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060169.1 (Salmonella enterica, BKQZM9) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,764 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061044.1 (Salmonella enterica, BBS1407) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,839 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061045.1 (Salmonella enterica, BBS1406) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061047.1 (Salmonella enterica, BBS1359) loaded with 44 markers.\n",
      "2024-04-17 18:14:00,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061050.1 (Salmonella enterica, BBS1358sml) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061049.1 (Salmonella enterica, BBS1358lrg) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023470.1 (Salmonella enterica, BAA-1672) loaded with 43 markers.\n",
      "2024-04-17 18:14:01,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023468.1 (Salmonella enterica, BAA-1586) loaded with 41 markers.\n",
      "2024-04-17 18:14:01,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034968.1 (Salmonella enterica, B3589) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,318 [DEBUG - chronostrain.database.parser.json] - Strain NC_017623.1 (Salmonella enterica, B182) loaded with 41 markers.\n",
      "2024-04-17 18:14:01,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074094.1 (Salmonella enterica, A_cip) loaded with 41 markers.\n",
      "2024-04-17 18:14:01,459 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015328.1 (Salmonella enterica, AUSMDU00027951) loaded with 43 markers.\n",
      "2024-04-17 18:14:01,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015340.1 (Salmonella enterica, AUSMDU00027944) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045945.1 (Salmonella enterica, AUSMDU00010533) loaded with 43 markers.\n",
      "2024-04-17 18:14:01,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045954.1 (Salmonella enterica, AUSMDU00010531) loaded with 41 markers.\n",
      "2024-04-17 18:14:01,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045947.1 (Salmonella enterica, AUSMDU00010530) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045949.1 (Salmonella enterica, AUSMDU00010529) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045952.1 (Salmonella enterica, AUSMDU00008979) loaded with 44 markers.\n",
      "2024-04-17 18:14:01,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015342.1 (Salmonella enterica, AUSMDU00007171) loaded with 44 markers.\n",
      "2024-04-17 18:14:02,069 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045831.1 (Salmonella enterica, AUSMDU00005056) loaded with 39 markers.\n",
      "2024-04-17 18:14:02,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015323.1 (Salmonella enterica, AUSMDU00004549) loaded with 44 markers.\n",
      "2024-04-17 18:14:02,241 [DEBUG - chronostrain.database.parser.json] - Strain NC_006511.1 (Salmonella enterica, ATCC_9150) loaded with 43 markers.\n",
      "2024-04-17 18:14:02,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034232.1 (Salmonella enterica, ATCC_8387) loaded with 34 markers.\n",
      "2024-04-17 18:14:02,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019183.1 (Salmonella enterica, ATCC_51958) loaded with 41 markers.\n",
      "2024-04-17 18:14:02,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043907.1 (Salmonella enterica, ATCC_14028) loaded with 44 markers.\n",
      "2024-04-17 18:14:02,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034230.1 (Salmonella enterica, ATCC_14028) loaded with 44 markers.\n",
      "2024-04-17 18:14:02,595 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009102.1 (Salmonella enterica, ATCC_13311) loaded with 44 markers.\n",
      "2024-04-17 18:14:02,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019178.1 (Salmonella enterica, ATCC_11997) loaded with 42 markers.\n",
      "2024-04-17 18:14:02,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019185.1 (Salmonella enterica, ATCC_11511) loaded with 43 markers.\n",
      "2024-04-17 18:14:02,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025218.1 (Salmonella enterica, ATCC_10722) loaded with 39 markers.\n",
      "2024-04-17 18:14:02,876 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012344.2 (Salmonella enterica, ATCC_10708) loaded with 41 markers.\n",
      "2024-04-17 18:14:02,944 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032194.1 (Salmonella enterica, AR_0127) loaded with 44 markers.\n",
      "2024-04-17 18:14:03,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026700.1 (Salmonella enterica, AR_0031) loaded with 44 markers.\n",
      "2024-04-17 18:14:03,079 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085812.1 (Salmonella enterica, Articuno) loaded with 42 markers.\n",
      "2024-04-17 18:14:03,155 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044177.1 (Salmonella enterica, AR-0407) loaded with 41 markers.\n",
      "2024-04-17 18:14:03,263 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044203.1 (Salmonella enterica, AR-0405) loaded with 43 markers.\n",
      "2024-04-17 18:14:03,363 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044181.1 (Salmonella enterica, AR-0404) loaded with 41 markers.\n",
      "2024-04-17 18:14:03,466 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044184.1 (Salmonella enterica, AR-0403) loaded with 43 markers.\n",
      "2024-04-17 18:14:03,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044186.1 (Salmonella enterica, AR-0402) loaded with 44 markers.\n",
      "2024-04-17 18:14:03,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044188.1 (Salmonella enterica, AR-0401) loaded with 45 markers.\n",
      "2024-04-17 18:14:03,786 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016576.1 (Salmonella enterica, AMR588-04-00437) loaded with 41 markers.\n",
      "2024-04-17 18:14:03,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016573.1 (Salmonella enterica, AMR588-04-00435) loaded with 41 markers.\n",
      "2024-04-17 18:14:03,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016569.1 (Salmonella enterica, AMR588-04-00320) loaded with 41 markers.\n",
      "2024-04-17 18:14:04,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016565.1 (Salmonella enterica, AMR588-04-00318) loaded with 41 markers.\n",
      "2024-04-17 18:14:04,188 [DEBUG - chronostrain.database.parser.json] - Strain NC_011147.1 (Salmonella enterica, AKU_12601) loaded with 43 markers.\n",
      "2024-04-17 18:14:04,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064919.1 (Salmonella enterica, AB42049) loaded with 44 markers.\n",
      "2024-04-17 18:14:04,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016563.1 (Salmonella enterica, A3EZ223) loaded with 41 markers.\n",
      "2024-04-17 18:14:04,504 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016561.1 (Salmonella enterica, A3ES40) loaded with 41 markers.\n",
      "2024-04-17 18:14:04,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053581.1 (Salmonella enterica, 94-0093) loaded with 35 markers.\n",
      "2024-04-17 18:14:04,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045444.1 (Salmonella enterica, 9355) loaded with 42 markers.\n",
      "2024-04-17 18:14:04,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018657.1 (Salmonella enterica, 92-0392) loaded with 44 markers.\n",
      "2024-04-17 18:14:04,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008925.1 (Salmonella enterica, 87) loaded with 42 markers.\n",
      "2024-04-17 18:14:05,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019442.1 (Salmonella enterica, 81741) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,125 [DEBUG - chronostrain.database.parser.json] - Strain NC_017046.1 (Salmonella enterica, 798) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,236 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039509.1 (Salmonella enterica, 7102.58) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039503.1 (Salmonella enterica, 7101.67) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067082.1 (Salmonella enterica, 679) loaded with 42 markers.\n",
      "2024-04-17 18:14:05,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR862421.1 (Salmonella enterica, 5d8c9f00c6184d00f9da85a6:sample:O1960-05) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR861808.1 (Salmonella enterica, 5d8c9f00c6184d00f9da85a6:sample:O1960-05) loaded with 44 markers.\n",
      "2024-04-17 18:14:05,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009049.1 (Salmonella enterica, 50973) loaded with 43 markers.\n",
      "2024-04-17 18:14:05,837 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040700.1 (Salmonella enterica, 5) loaded with 41 markers.\n",
      "2024-04-17 18:14:05,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031359.1 (Salmonella enterica, 5) loaded with 41 markers.\n",
      "2024-04-17 18:14:06,024 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT882486.1 (Salmonella enterica, 4714STDY6831631) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,125 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR590082.1 (Salmonella enterica, 4316STDY6559672) loaded with 43 markers.\n",
      "2024-04-17 18:14:06,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR590081.1 (Salmonella enterica, 4316STDY6559669) loaded with 43 markers.\n",
      "2024-04-17 18:14:06,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037893.1 (Salmonella enterica, 42N) loaded with 38 markers.\n",
      "2024-04-17 18:14:06,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045762.1 (Salmonella enterica, 41578) loaded with 41 markers.\n",
      "2024-04-17 18:14:06,491 [DEBUG - chronostrain.database.parser.json] - Strain NC_021810.1 (Salmonella enterica, 41578) loaded with 41 markers.\n",
      "2024-04-17 18:14:06,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905062.1 (Salmonella enterica, 403Ty-sc-1979084) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,625 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029036.1 (Salmonella enterica, 361154004) loaded with 43 markers.\n",
      "2024-04-17 18:14:06,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029846.1 (Salmonella enterica, 343078_273110) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,766 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029959.1 (Salmonella enterica, 343078_256191) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029960.1 (Salmonella enterica, 343078_251131) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,899 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029962.1 (Salmonella enterica, 343078_228140) loaded with 42 markers.\n",
      "2024-04-17 18:14:06,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029964.1 (Salmonella enterica, 343078_223175) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029848.1 (Salmonella enterica, 343078_211126) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029850.1 (Salmonella enterica, 343078_203125) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,159 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029852.1 (Salmonella enterica, 343078_201101) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029855.1 (Salmonella enterica, 343077_292148) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,289 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029856.1 (Salmonella enterica, 343077_286126) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029858.1 (Salmonella enterica, 343077_285138) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029853.1 (Salmonella enterica, 343077_281186) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029863.1 (Salmonella enterica, 343077_278127) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029906.1 (Salmonella enterica, 343077_267164) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029861.1 (Salmonella enterica, 343077_260153) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029907.1 (Salmonella enterica, 343077_255118) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,773 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029864.1 (Salmonella enterica, 343077_228157) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029866.1 (Salmonella enterica, 343077_228140) loaded with 42 markers.\n",
      "2024-04-17 18:14:07,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029868.1 (Salmonella enterica, 343077_215174) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029862.1 (Salmonella enterica, 343077_214162) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,156 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029872.1 (Salmonella enterica, 343077_214135) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029897.1 (Salmonella enterica, 343077_213147) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029870.1 (Salmonella enterica, 343077_212159) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029919.1 (Salmonella enterica, 343077_212138) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029873.1 (Salmonella enterica, 343077_211171) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029888.1 (Salmonella enterica, 343076_294172) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,710 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029881.1 (Salmonella enterica, 343076_269157) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,779 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029890.1 (Salmonella enterica, 343076_253155) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,876 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029892.1 (Salmonella enterica, 343076_252143) loaded with 42 markers.\n",
      "2024-04-17 18:14:08,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029913.1 (Salmonella enterica, 343076_249107) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,089 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029882.1 (Salmonella enterica, 343076_248190) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029899.1 (Salmonella enterica, 343076_241106) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029900.1 (Salmonella enterica, 343076_232188) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029875.1 (Salmonella enterica, 343076_227128) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029914.1 (Salmonella enterica, 343076_217103) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029915.1 (Salmonella enterica, 343076_202113) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,637 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012681.1 (Salmonella enterica, 33676) loaded with 44 markers.\n",
      "2024-04-17 18:14:09,705 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029894.1 (Salmonella enterica, 311189_291186) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029920.1 (Salmonella enterica, 311189_282186) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029922.1 (Salmonella enterica, 311189_269186) loaded with 42 markers.\n",
      "2024-04-17 18:14:09,954 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029883.1 (Salmonella enterica, 311189_268186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029902.1 (Salmonella enterica, 311189_268103) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029917.1 (Salmonella enterica, 311189_256186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029885.1 (Salmonella enterica, 311189_255186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,310 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029896.1 (Salmonella enterica, 311189_252186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,410 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029904.1 (Salmonella enterica, 311189_231186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029878.1 (Salmonella enterica, 311189_224186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029880.1 (Salmonella enterica, 311189_223186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,618 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029909.1 (Salmonella enterica, 311189_222186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,680 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029923.1 (Salmonella enterica, 311189_221186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029886.1 (Salmonella enterica, 311189_220186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029911.1 (Salmonella enterica, 311189_219186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,884 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029925.1 (Salmonella enterica, 311189_218186) loaded with 42 markers.\n",
      "2024-04-17 18:14:10,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029646.1 (Salmonella enterica, 311189_217186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029928.1 (Salmonella enterica, 311189_216103) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,117 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029930.1 (Salmonella enterica, 311189_215186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029933.1 (Salmonella enterica, 311189_214186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029936.1 (Salmonella enterica, 311189_213186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029944.1 (Salmonella enterica, 311189_212186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029945.1 (Salmonella enterica, 311189_211186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029946.1 (Salmonella enterica, 311189_210186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029952.1 (Salmonella enterica, 311189_209186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,596 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029949.1 (Salmonella enterica, 311189_208186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029932.1 (Salmonella enterica, 311189_208103) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,722 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029938.1 (Salmonella enterica, 311189_207186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029940.1 (Salmonella enterica, 311189_206186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029950.1 (Salmonella enterica, 311189_205186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029954.1 (Salmonella enterica, 311189_204186) loaded with 42 markers.\n",
      "2024-04-17 18:14:11,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029942.1 (Salmonella enterica, 311189_203186) loaded with 42 markers.\n",
      "2024-04-17 18:14:12,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029956.1 (Salmonella enterica, 311189_202186) loaded with 42 markers.\n",
      "2024-04-17 18:14:12,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029958.1 (Salmonella enterica, 311189_201186) loaded with 42 markers.\n",
      "2024-04-17 18:14:12,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040701.1 (Salmonella enterica, 3) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,276 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074096.1 (Salmonella enterica, 2B_cip) loaded with 41 markers.\n",
      "2024-04-17 18:14:12,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN890524.1 (Salmonella enterica, 2511STDY5712385) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN890522.1 (Salmonella enterica, 2511STDY5712384) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,501 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN890520.1 (Salmonella enterica, 2511STDY5462413) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,587 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN890518.1 (Salmonella enterica, 2511STDY5462388) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,643 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053582.1 (Salmonella enterica, 2439-64) loaded with 29 markers.\n",
      "2024-04-17 18:14:12,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017621.1 (Salmonella enterica, 22792) loaded with 43 markers.\n",
      "2024-04-17 18:14:12,775 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091878.1 (Salmonella enterica, 21-SA00318-0) loaded with 39 markers.\n",
      "2024-04-17 18:14:12,858 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090545.1 (Salmonella enterica, 2017028-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:12,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090533.1 (Salmonella enterica, 2017005-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,005 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090535.1 (Salmonella enterica, 2016089-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,074 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090539.1 (Salmonella enterica, 2016062-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,148 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053585.1 (Salmonella enterica, 2011K-1440) loaded with 42 markers.\n",
      "2024-04-17 18:14:13,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090541.1 (Salmonella enterica, 2010112-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,293 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905139.1 (Salmonella enterica, 2010-007898) loaded with 42 markers.\n",
      "2024-04-17 18:14:13,377 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090529.1 (Salmonella enterica, 2008079-SE) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,473 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088901.1 (Salmonella enterica, 180135033) loaded with 43 markers.\n",
      "2024-04-17 18:14:13,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020922.1 (Salmonella enterica, 16A242) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014996.1 (Salmonella enterica, 1655) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043667.1 (Salmonella enterica, 162835) loaded with 41 markers.\n",
      "2024-04-17 18:14:13,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043664.1 (Salmonella enterica, 161365) loaded with 41 markers.\n",
      "2024-04-17 18:14:13,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063140.1 (Salmonella enterica, 16-174478) loaded with 44 markers.\n",
      "2024-04-17 18:14:13,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043662.1 (Salmonella enterica, 159838) loaded with 40 markers.\n",
      "2024-04-17 18:14:14,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT905064.1 (Salmonella enterica, 1554-sc-2165329) loaded with 42 markers.\n",
      "2024-04-17 18:14:14,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019408.1 (Salmonella enterica, 1422-74) loaded with 41 markers.\n",
      "2024-04-17 18:14:14,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043400.1 (Salmonella enterica, 14028S_substr._JY996) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,401 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043399.1 (Salmonella enterica, 14028S_substr._GXS275) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043401.1 (Salmonella enterica, 14028S_substr._GXS259) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043402.1 (Salmonella enterica, 14028S_substr._GXS254) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,642 [DEBUG - chronostrain.database.parser.json] - Strain NC_016856.1 (Salmonella enterica, 14028S) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034479.1 (Salmonella enterica, 14028) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007581.1 (Salmonella enterica, 138736) loaded with 44 markers.\n",
      "2024-04-17 18:14:14,847 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045447.1 (Salmonella enterica, 12888) loaded with 42 markers.\n",
      "2024-04-17 18:14:14,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012924.1 (Salmonella enterica, 12-4374) loaded with 41 markers.\n",
      "2024-04-17 18:14:14,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039277.1 (Salmonella enterica, 12-0523) loaded with 39 markers.\n",
      "2024-04-17 18:14:15,044 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032816.1 (Salmonella enterica, 11TTUC-046) loaded with 39 markers.\n",
      "2024-04-17 18:14:15,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038601.1 (Salmonella enterica, 11-5006) loaded with 45 markers.\n",
      "2024-04-17 18:14:15,190 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016514.1 (Salmonella enterica, 11-004736-1-7) loaded with 41 markers.\n",
      "2024-04-17 18:14:15,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029839.1 (Salmonella enterica, 10ST07093) loaded with 44 markers.\n",
      "2024-04-17 18:14:15,388 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016525.1 (Salmonella enterica, 09-036813-1A) loaded with 41 markers.\n",
      "2024-04-17 18:14:15,495 [DEBUG - chronostrain.database.parser.json] - Strain NC_021820.1 (Salmonella enterica, 08-1736) loaded with 44 markers.\n",
      "2024-04-17 18:14:15,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012599.1 (Salmonella enterica, 0307-213) loaded with 35 markers.\n",
      "2024-04-17 18:14:15,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019197.1 (Salmonella enterica, 0250) loaded with 39 markers.\n",
      "2024-04-17 18:14:15,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013685.1 (Salmonella enterica, 0007-33) loaded with 44 markers.\n",
      "2024-04-17 18:14:15,784 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067066.1 (Raoultella sp., XY-1) loaded with 21 markers.\n",
      "2024-04-17 18:14:15,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030874.1 (Raoultella sp., X13) loaded with 19 markers.\n",
      "2024-04-17 18:14:15,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040183.1 (Raoultella planticola, Rp_CZ180511) loaded with 21 markers.\n",
      "2024-04-17 18:14:15,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013338.1 (Raoultella ornithinolytica, Yangling_I2) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,040 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076605.1 (Raoultella ornithinolytica, XF201) loaded with 23 markers.\n",
      "2024-04-17 18:14:16,100 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022268.1 (Raoultella ornithinolytica, WP8-W19-CRE-01) loaded with 20 markers.\n",
      "2024-04-17 18:14:16,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038281.1 (Raoultella ornithinolytica, WLK218) loaded with 21 markers.\n",
      "2024-04-17 18:14:16,236 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055216.1 (Raoultella ornithinolytica, STEFF_15) loaded with 19 markers.\n",
      "2024-04-17 18:14:16,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021329.1 (Raoultella ornithinolytica, Ro24724) loaded with 19 markers.\n",
      "2024-04-17 18:14:16,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079752.1 (Raoultella ornithinolytica, NY1) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025009.1 (Raoultella ornithinolytica, NUITM-VR1) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,486 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017802.2 (Raoultella ornithinolytica, MG) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,547 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059047.1 (Raoultella ornithinolytica, DSM_16926) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,607 [DEBUG - chronostrain.database.parser.json] - Strain NC_021066.1 (Raoultella ornithinolytica, B6) loaded with 19 markers.\n",
      "2024-04-17 18:14:16,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008886.1 (Raoultella ornithinolytica, A14) loaded with 20 markers.\n",
      "2024-04-17 18:14:16,763 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048349.1 (Raoultella ornithinolytica, 23) loaded with 21 markers.\n",
      "2024-04-17 18:14:16,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046671.1 (Raoultella ornithinolytica, 193104922) loaded with 22 markers.\n",
      "2024-04-17 18:14:16,901 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045783.1 (Klebsiella variicola, LEMB11) loaded with 17 markers.\n",
      "2024-04-17 18:14:16,955 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063403.1 (Klebsiella variicola, 8917) loaded with 16 markers.\n",
      "2024-04-17 18:14:17,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056483.1 (Klebsiella sp., RHBSTW-00464) loaded with 24 markers.\n",
      "2024-04-17 18:14:17,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081893.1 (Klebsiella quasipneumoniae, WI-06) loaded with 17 markers.\n",
      "2024-04-17 18:14:17,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068444.1 (Klebsiella quasipneumoniae, SWMUF35) loaded with 17 markers.\n",
      "2024-04-17 18:14:17,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022412.1 (Klebsiella quasipneumoniae, STW0522-39) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,279 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063942.1 (Klebsiella quasipneumoniae, BS418) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040861.1 (Klebsiella pneumoniae, Xen39) loaded with 20 markers.\n",
      "2024-04-17 18:14:17,391 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076863.1 (Klebsiella pneumoniae, V3) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,454 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012560.1 (Klebsiella pneumoniae, UCLAOXA232KP_Pt0) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,509 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012561.1 (Klebsiella pneumoniae, UCLAOXA232KP) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012568.1 (Klebsiella pneumoniae, UCLAOXA232KP) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024038.1 (Klebsiella pneumoniae, QS17-0029) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041639.1 (Klebsiella pneumoniae, PIMB15ND2KP27) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR025088.1 (Klebsiella pneumoniae, KP980) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR025092.1 (Klebsiella pneumoniae, KP9201) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041946.1 (Klebsiella pneumoniae, KP2) loaded with 18 markers.\n",
      "2024-04-17 18:14:17,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080362.1 (Klebsiella pneumoniae, K191773) loaded with 19 markers.\n",
      "2024-04-17 18:14:17,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080353.1 (Klebsiella pneumoniae, K191663) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,019 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029590.1 (Klebsiella pneumoniae, DA33144) loaded with 19 markers.\n",
      "2024-04-17 18:14:18,070 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029587.1 (Klebsiella pneumoniae, DA33141) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS399318.1 (Klebsiella pneumoniae, CNR48) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045015.1 (Klebsiella pneumoniae, BK13048) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058940.1 (Klebsiella pneumoniae, BA39649) loaded with 20 markers.\n",
      "2024-04-17 18:14:18,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070435.1 (Klebsiella pneumoniae, B5617) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029738.1 (Klebsiella pneumoniae, AR_0087) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032185.1 (Klebsiella pneumoniae, AR_0075) loaded with 20 markers.\n",
      "2024-04-17 18:14:18,469 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083032.1 (Klebsiella pneumoniae, 5208.51) loaded with 19 markers.\n",
      "2024-04-17 18:14:18,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041927.1 (Klebsiella pneumoniae, 18-2374) loaded with 19 markers.\n",
      "2024-04-17 18:14:18,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035382.1 (Leclercia adecarboxylata, R25) loaded with 35 markers.\n",
      "2024-04-17 18:14:18,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021972.1 (Klebsiella sp., WP3-W18-ESBL-02) loaded with 45 markers.\n",
      "2024-04-17 18:14:18,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022020.1 (Klebsiella sp., WP3-S18-ESBL-05) loaded with 45 markers.\n",
      "2024-04-17 18:14:18,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022108.1 (Klebsiella aerogenes, WP5-W18-CRE-01) loaded with 26 markers.\n",
      "2024-04-17 18:14:18,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056260.1 (Klebsiella aerogenes, RHBSTW-00898) loaded with 20 markers.\n",
      "2024-04-17 18:14:18,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890184.1 (Klebsiella aerogenes, MINF_10B-sc-2280448) loaded with 18 markers.\n",
      "2024-04-17 18:14:18,980 [DEBUG - chronostrain.database.parser.json] - Strain NC_015663.1 (Klebsiella aerogenes, KCTC_2190) loaded with 19 markers.\n",
      "2024-04-17 18:14:19,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082898.1 (Klebsiella aerogenes, KAE3SP) loaded with 18 markers.\n",
      "2024-04-17 18:14:19,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011539.1 (Klebsiella aerogenes, G7) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,157 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044083.1 (Klebsiella aerogenes, FDAARGOS_641) loaded with 19 markers.\n",
      "2024-04-17 18:14:19,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033817.1 (Klebsiella aerogenes, FDAARGOS_513) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,285 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031756.1 (Klebsiella aerogenes, FDAARGOS_327) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014029.2 (Klebsiella aerogenes, FDAARGOS_152) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077293.1 (Klebsiella aerogenes, FDAARGOS_1442) loaded with 19 markers.\n",
      "2024-04-17 18:14:19,456 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077265.1 (Klebsiella aerogenes, FDAARGOS_1441) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014748.2 (Klebsiella aerogenes, FDAARGOS_139) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,563 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070520.1 (Klebsiella aerogenes, EA46506) loaded with 18 markers.\n",
      "2024-04-17 18:14:19,616 [DEBUG - chronostrain.database.parser.json] - Strain NC_020181.1 (Klebsiella aerogenes, EA1509E) loaded with 19 markers.\n",
      "2024-04-17 18:14:19,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026756.1 (Klebsiella aerogenes, AR_0062) loaded with 24 markers.\n",
      "2024-04-17 18:14:19,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024885.1 (Klebsiella aerogenes, AR_0009) loaded with 20 markers.\n",
      "2024-04-17 18:14:19,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056119.1 (Enterobacter ludwigii, D42-sc-1712201) loaded with 33 markers.\n",
      "2024-04-17 18:14:19,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR881936.1 (Enterobacter cancerogenus, UPC1) loaded with 30 markers.\n",
      "2024-04-17 18:14:19,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019986.1 (Citrobacter werkmanii, BF-6) loaded with 42 markers.\n",
      "2024-04-17 18:14:20,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR699006.1 (Citrobacter freundii, MGYG-HGUT-02495) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026045.1 (Citrobacter freundii, FDAARGOS_61) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028520.1 (Lelliottia sp., WB101) loaded with 34 markers.\n",
      "2024-04-17 18:14:20,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064661.1 (Salmonella sp., SJTUF14178) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,415 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064663.1 (Salmonella sp., SJTUF14170) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,517 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064666.1 (Salmonella sp., SJTUF14154) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,616 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064671.1 (Salmonella sp., SJTUF14152) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,718 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064672.1 (Salmonella sp., SJTUF14146) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,808 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064674.1 (Salmonella sp., SJTUF14076) loaded with 43 markers.\n",
      "2024-04-17 18:14:20,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051218.1 (Salmonella sp., SCFS4) loaded with 44 markers.\n",
      "2024-04-17 18:14:21,005 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089313.1 (Salmonella enterica, YZ21MCS4) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,103 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077680.1 (Salmonella enterica, YZ20MCS16) loaded with 41 markers.\n",
      "2024-04-17 18:14:21,195 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039441.1 (Salmonella enterica, VNSEC003) loaded with 40 markers.\n",
      "2024-04-17 18:14:21,261 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039443.1 (Salmonella enterica, VNSEC002) loaded with 40 markers.\n",
      "2024-04-17 18:14:21,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039445.1 (Salmonella enterica, VNSEC001) loaded with 40 markers.\n",
      "2024-04-17 18:14:21,405 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032446.1 (Salmonella enterica, USMARC-69840) loaded with 41 markers.\n",
      "2024-04-17 18:14:21,473 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032449.1 (Salmonella enterica, USMARC-69838) loaded with 41 markers.\n",
      "2024-04-17 18:14:21,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032379.1 (Salmonella enterica, USMARC-69807) loaded with 41 markers.\n",
      "2024-04-17 18:14:21,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025453.1 (Salmonella enterica, USDA-ARS-USMARC-76341) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025452.1 (Salmonella enterica, USDA-ARS-USMARC-76340) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025451.1 (Salmonella enterica, USDA-ARS-USMARC-76339) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025450.1 (Salmonella enterica, USDA-ARS-USMARC-76338) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,874 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025449.1 (Salmonella enterica, USDA-ARS-USMARC-76337) loaded with 43 markers.\n",
      "2024-04-17 18:14:21,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025448.1 (Salmonella enterica, USDA-ARS-USMARC-76336) loaded with 43 markers.\n",
      "2024-04-17 18:14:22,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025447.1 (Salmonella enterica, USDA-ARS-USMARC-76335) loaded with 43 markers.\n",
      "2024-04-17 18:14:22,091 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025454.1 (Salmonella enterica, USDA-ARS-USMARC-76334) loaded with 43 markers.\n",
      "2024-04-17 18:14:22,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025446.1 (Salmonella enterica, USDA-ARS-USMARC-76333) loaded with 43 markers.\n",
      "2024-04-17 18:14:22,248 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025445.1 (Salmonella enterica, USDA-ARS-USMARC-76332) loaded with 43 markers.\n",
      "2024-04-17 18:14:22,322 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025276.1 (Salmonella enterica, USDA-ARS-USMARC-60984) loaded with 40 markers.\n",
      "2024-04-17 18:14:22,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025280.1 (Salmonella enterica, USDA-ARS-USMARC-60983) loaded with 42 markers.\n",
      "2024-04-17 18:14:22,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025278.1 (Salmonella enterica, USDA-ARS-USMARC-1913) loaded with 40 markers.\n",
      "2024-04-17 18:14:22,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014661.1 (Salmonella enterica, USDA-ARS-USMARC-1783) loaded with 44 markers.\n",
      "2024-04-17 18:14:22,597 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014666.2 (Salmonella enterica, USDA-ARS-USMARC-1781) loaded with 44 markers.\n",
      "2024-04-17 18:14:22,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014665.1 (Salmonella enterica, USDA-ARS-USMARC-1766) loaded with 44 markers.\n",
      "2024-04-17 18:14:22,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014659.2 (Salmonella enterica, USDA-ARS-USMARC-1765) loaded with 44 markers.\n",
      "2024-04-17 18:14:22,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014657.1 (Salmonella enterica, USDA-ARS-USMARC-1736) loaded with 44 markers.\n",
      "2024-04-17 18:14:22,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007584.2 (Salmonella enterica, USDA-ARS-USMARC-1735) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014664.1 (Salmonella enterica, USDA-ARS-USMARC-1728) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014621.2 (Salmonella enterica, USDA-ARS-USMARC-1727) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,159 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014663.2 (Salmonella enterica, USDA-ARS-USMARC-1677) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014620.1 (Salmonella enterica, USDA-ARS-USMARC-1676) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007483.2 (Salmonella enterica, USDA-ARS-USMARC-1175) loaded with 44 markers.\n",
      "2024-04-17 18:14:23,370 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085983.1 (Salmonella enterica, SZL_31) loaded with 40 markers.\n",
      "2024-04-17 18:14:23,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085981.1 (Salmonella enterica, SZL_30) loaded with 40 markers.\n",
      "2024-04-17 18:14:23,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019411.1 (Salmonella enterica, ST203) loaded with 40 markers.\n",
      "2024-04-17 18:14:23,568 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060512.1 (Salmonella enterica, SLR1_7966) loaded with 40 markers.\n",
      "2024-04-17 18:14:23,640 [DEBUG - chronostrain.database.parser.json] - Strain NC_011149.1 (Salmonella enterica, SL483) loaded with 43 markers.\n",
      "2024-04-17 18:14:23,723 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019692.1 (Salmonella enterica, SL180013) loaded with 61 markers.\n",
      "2024-04-17 18:14:23,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041179.1 (Salmonella enterica, SJTUF87912v2) loaded with 43 markers.\n",
      "2024-04-17 18:14:23,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041181.1 (Salmonella enterica, SJTUF13520v2) loaded with 43 markers.\n",
      "2024-04-17 18:14:23,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050777.1 (Salmonella enterica, SI96) loaded with 43 markers.\n",
      "2024-04-17 18:14:23,993 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050779.1 (Salmonella enterica, SI85) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050783.1 (Salmonella enterica, SI67) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,126 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050785.1 (Salmonella enterica, SI43) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050756.1 (Salmonella enterica, SI174) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,260 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050757.1 (Salmonella enterica, SI173) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050760.1 (Salmonella enterica, SI170) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,395 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050762.1 (Salmonella enterica, SI115) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050764.1 (Salmonella enterica, SI111) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050769.1 (Salmonella enterica, SI108) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050771.1 (Salmonella enterica, SI102) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036165.1 (Salmonella enterica, sg_wt5) loaded with 42 markers.\n",
      "2024-04-17 18:14:24,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048775.1 (Salmonella enterica, SG17-135) loaded with 43 markers.\n",
      "2024-04-17 18:14:24,841 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045518.1 (Salmonella enterica, Sal-5091) loaded with 44 markers.\n",
      "2024-04-17 18:14:24,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045516.1 (Salmonella enterica, Sal-4737) loaded with 44 markers.\n",
      "2024-04-17 18:14:24,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045515.1 (Salmonella enterica, Sal-4295) loaded with 44 markers.\n",
      "2024-04-17 18:14:25,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045466.1 (Salmonella enterica, Sal-3973) loaded with 44 markers.\n",
      "2024-04-17 18:14:25,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045513.1 (Salmonella enterica, Sal-3948) loaded with 45 markers.\n",
      "2024-04-17 18:14:25,195 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045465.1 (Salmonella enterica, Sal-2097) loaded with 45 markers.\n",
      "2024-04-17 18:14:25,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045464.1 (Salmonella enterica, Sal-1135) loaded with 45 markers.\n",
      "2024-04-17 18:14:25,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030005.1 (Salmonella enterica, SA20141895) loaded with 40 markers.\n",
      "2024-04-17 18:14:25,419 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029999.1 (Salmonella enterica, SA20113174) loaded with 42 markers.\n",
      "2024-04-17 18:14:25,514 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030180.1 (Salmonella enterica, SA20100201) loaded with 42 markers.\n",
      "2024-04-17 18:14:25,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019417.1 (Salmonella enterica, SA20092095) loaded with 43 markers.\n",
      "2024-04-17 18:14:25,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022015.1 (Salmonella enterica, SA20085604) loaded with 24 markers.\n",
      "2024-04-17 18:14:25,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030217.1 (Salmonella enterica, SA20075157) loaded with 42 markers.\n",
      "2024-04-17 18:14:25,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030002.1 (Salmonella enterica, SA20064858) loaded with 42 markers.\n",
      "2024-04-17 18:14:25,855 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030288.1 (Salmonella enterica, SA20063285) loaded with 41 markers.\n",
      "2024-04-17 18:14:25,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019403.1 (Salmonella enterica, SA20060561) loaded with 41 markers.\n",
      "2024-04-17 18:14:25,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030211.1 (Salmonella enterica, SA20051528) loaded with 44 markers.\n",
      "2024-04-17 18:14:26,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030209.1 (Salmonella enterica, SA20044414) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030231.1 (Salmonella enterica, SA20043041) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022138.1 (Salmonella enterica, SA20041608) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019407.1 (Salmonella enterica, SA20041063) loaded with 43 markers.\n",
      "2024-04-17 18:14:26,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022489.1 (Salmonella enterica, SA20026234) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030207.1 (Salmonella enterica, SA19992307) loaded with 41 markers.\n",
      "2024-04-17 18:14:26,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077670.1 (Salmonella enterica, S90) loaded with 43 markers.\n",
      "2024-04-17 18:14:26,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061118.1 (Salmonella enterica, S530) loaded with 43 markers.\n",
      "2024-04-17 18:14:26,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092040.1 (Salmonella enterica, S19) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019116.1 (Salmonella enterica, S01-0511) loaded with 40 markers.\n",
      "2024-04-17 18:14:26,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019412.1 (Salmonella enterica, S-1501) loaded with 43 markers.\n",
      "2024-04-17 18:14:26,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022019.1 (Salmonella enterica, S-1267) loaded with 38 markers.\n",
      "2024-04-17 18:14:26,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019409.1 (Salmonella enterica, S-1087) loaded with 41 markers.\n",
      "2024-04-17 18:14:26,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022663.1 (Salmonella enterica, RM11065) loaded with 39 markers.\n",
      "2024-04-17 18:14:26,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090930.1 (Salmonella enterica, R21.2430) loaded with 43 markers.\n",
      "2024-04-17 18:14:27,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090929.1 (Salmonella enterica, R21.2429) loaded with 43 markers.\n",
      "2024-04-17 18:14:27,105 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060730.2 (Salmonella enterica, R17.5974) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062795.1 (Salmonella enterica, R17.4301) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,237 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063330.1 (Salmonella enterica, R17.2117) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065572.1 (Salmonella enterica, R16.1424) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029800.1 (Salmonella enterica, R16.0676) loaded with 44 markers.\n",
      "2024-04-17 18:14:27,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061929.2 (Salmonella enterica, R16.0556) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062004.2 (Salmonella enterica, R16.0431) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065564.1 (Salmonella enterica, R15.2267) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,643 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065566.1 (Salmonella enterica, R13.1544) loaded with 42 markers.\n",
      "2024-04-17 18:14:27,724 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045038.1 (Salmonella enterica, PJM1) loaded with 46 markers.\n",
      "2024-04-17 18:14:27,790 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483455.1 (Salmonella enterica, NCTC9872) loaded with 38 markers.\n",
      "2024-04-17 18:14:27,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483457.1 (Salmonella enterica, NCTC9787) loaded with 41 markers.\n",
      "2024-04-17 18:14:27,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016410.1 (Salmonella enterica, N55391) loaded with 40 markers.\n",
      "2024-04-17 18:14:28,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026379.1 (Salmonella enterica, N17) loaded with 87 markers.\n",
      "2024-04-17 18:14:28,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019410.1 (Salmonella enterica, N1529-D3) loaded with 42 markers.\n",
      "2024-04-17 18:14:28,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030749.1 (Salmonella enterica, MAC15) loaded with 41 markers.\n",
      "2024-04-17 18:14:28,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045509.1 (Salmonella enterica, M-5360) loaded with 44 markers.\n",
      "2024-04-17 18:14:28,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045461.1 (Salmonella enterica, M-3851) loaded with 44 markers.\n",
      "2024-04-17 18:14:28,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045458.1 (Salmonella enterica, M-3471) loaded with 44 markers.\n",
      "2024-04-17 18:14:28,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079713.1 (Salmonella enterica, LHICA_AZ23) loaded with 44 markers.\n",
      "2024-04-17 18:14:28,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062220.1 (Salmonella enterica, Kharkiv) loaded with 43 markers.\n",
      "2024-04-17 18:14:29,395 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028131.1 (Salmonella enterica, JT_01) loaded with 43 markers.\n",
      "2024-04-17 18:14:29,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060585.1 (Salmonella enterica, GX1006) loaded with 42 markers.\n",
      "2024-04-17 18:14:29,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013226.1 (Salmonella enterica, GT-38) loaded with 45 markers.\n",
      "2024-04-17 18:14:29,596 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013222.1 (Salmonella enterica, GT-01) loaded with 44 markers.\n",
      "2024-04-17 18:14:29,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037894.1 (Salmonella enterica, FSW0104) loaded with 42 markers.\n",
      "2024-04-17 18:14:29,730 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016408.1 (Salmonella enterica, FSIS1502916) loaded with 40 markers.\n",
      "2024-04-17 18:14:29,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016406.1 (Salmonella enterica, FSIS1502169) loaded with 40 markers.\n",
      "2024-04-17 18:14:29,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011365.1 (Salmonella enterica, FORC_015) loaded with 39 markers.\n",
      "2024-04-17 18:14:29,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041699.1 (Salmonella enterica, FJC33) loaded with 43 markers.\n",
      "2024-04-17 18:14:30,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041011.1 (Salmonella enterica, FDAARGOS_765) loaded with 42 markers.\n",
      "2024-04-17 18:14:30,079 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046278.1 (Salmonella enterica, FDAARGOS_708) loaded with 44 markers.\n",
      "2024-04-17 18:14:30,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046280.1 (Salmonella enterica, FDAARGOS_688) loaded with 42 markers.\n",
      "2024-04-17 18:14:30,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017719.1 (Salmonella enterica, FDA957831-1-1) loaded with 44 markers.\n",
      "2024-04-17 18:14:30,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037891.1 (Salmonella enterica, FDA925033-2-17) loaded with 41 markers.\n",
      "2024-04-17 18:14:30,353 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022450.1 (Salmonella enterica, D90) loaded with 43 markers.\n",
      "2024-04-17 18:14:30,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032384.1 (Salmonella enterica, CVM_N53043) loaded with 43 markers.\n",
      "2024-04-17 18:14:30,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032387.1 (Salmonella enterica, CVM_N45955) loaded with 41 markers.\n",
      "2024-04-17 18:14:30,543 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032390.1 (Salmonella enterica, CVM_34981) loaded with 41 markers.\n",
      "2024-04-17 18:14:30,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051320.1 (Salmonella enterica, CVM_34526) loaded with 45 markers.\n",
      "2024-04-17 18:14:30,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051342.1 (Salmonella enterica, CVM_33822) loaded with 44 markers.\n",
      "2024-04-17 18:14:30,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051356.1 (Salmonella enterica, CVM_28336) loaded with 42 markers.\n",
      "2024-04-17 18:14:30,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051391.1 (Salmonella enterica, CVM_22504) loaded with 45 markers.\n",
      "2024-04-17 18:14:30,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032393.1 (Salmonella enterica, CVM_22453) loaded with 41 markers.\n",
      "2024-04-17 18:14:30,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051394.1 (Salmonella enterica, CVM_22448) loaded with 44 markers.\n",
      "2024-04-17 18:14:30,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032396.1 (Salmonella enterica, CVM_22429) loaded with 41 markers.\n",
      "2024-04-17 18:14:31,061 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051402.1 (Salmonella enterica, CVM_21974) loaded with 43 markers.\n",
      "2024-04-17 18:14:31,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016412.1 (Salmonella enterica, CVM44454) loaded with 40 markers.\n",
      "2024-04-17 18:14:31,190 [DEBUG - chronostrain.database.parser.json] - Strain NC_011205.1 (Salmonella enterica, CT_02021853) loaded with 41 markers.\n",
      "2024-04-17 18:14:31,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044257.1 (Salmonella enterica, CFSAN096147) loaded with 40 markers.\n",
      "2024-04-17 18:14:31,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037892.1 (Salmonella enterica, CFSAN087304) loaded with 42 markers.\n",
      "2024-04-17 18:14:31,385 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033338.1 (Salmonella enterica, CFSAN076215) loaded with 44 markers.\n",
      "2024-04-17 18:14:31,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033343.1 (Salmonella enterica, CFSAN076213) loaded with 35 markers.\n",
      "2024-04-17 18:14:31,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075106.1 (Salmonella enterica, CFSAN064276) loaded with 44 markers.\n",
      "2024-04-17 18:14:31,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075113.1 (Salmonella enterica, CFSAN059898) loaded with 41 markers.\n",
      "2024-04-17 18:14:31,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075115.1 (Salmonella enterica, CFSAN059883) loaded with 42 markers.\n",
      "2024-04-17 18:14:31,705 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077711.1 (Salmonella enterica, CFSAN058606) loaded with 42 markers.\n",
      "2024-04-17 18:14:31,769 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077699.1 (Salmonella enterica, CFSAN058591) loaded with 44 markers.\n",
      "2024-04-17 18:14:31,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077706.1 (Salmonella enterica, CFSAN058545) loaded with 44 markers.\n",
      "2024-04-17 18:14:31,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077710.1 (Salmonella enterica, CFSAN058540) loaded with 42 markers.\n",
      "2024-04-17 18:14:32,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029336.1 (Salmonella enterica, CFSAN051296) loaded with 39 markers.\n",
      "2024-04-17 18:14:32,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029039.1 (Salmonella enterica, CFSAN045764) loaded with 39 markers.\n",
      "2024-04-17 18:14:32,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075125.1 (Salmonella enterica, CFSAN044945) loaded with 43 markers.\n",
      "2024-04-17 18:14:32,325 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075128.1 (Salmonella enterica, CFSAN044923) loaded with 42 markers.\n",
      "2024-04-17 18:14:32,398 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075129.1 (Salmonella enterica, CFSAN044921) loaded with 44 markers.\n",
      "2024-04-17 18:14:32,469 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075135.1 (Salmonella enterica, CFSAN044888) loaded with 44 markers.\n",
      "2024-04-17 18:14:32,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075138.1 (Salmonella enterica, CFSAN044875) loaded with 37 markers.\n",
      "2024-04-17 18:14:32,610 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075140.1 (Salmonella enterica, CFSAN044865) loaded with 42 markers.\n",
      "2024-04-17 18:14:32,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041183.1 (Salmonella enterica, CFSAN003961) loaded with 44 markers.\n",
      "2024-04-17 18:14:32,752 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041184.1 (Salmonella enterica, CFSAN003959) loaded with 44 markers.\n",
      "2024-04-17 18:14:32,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019202.1 (Salmonella enterica, CFSAN003307) loaded with 39 markers.\n",
      "2024-04-17 18:14:32,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033356.2 (Salmonella enterica, CFSA664) loaded with 43 markers.\n",
      "2024-04-17 18:14:32,951 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033384.2 (Salmonella enterica, CFSA300) loaded with 43 markers.\n",
      "2024-04-17 18:14:33,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033387.2 (Salmonella enterica, CFSA1007) loaded with 43 markers.\n",
      "2024-04-17 18:14:33,152 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038608.1 (Salmonella enterica, CFIAFB20170121) loaded with 86 markers.\n",
      "2024-04-17 18:14:33,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038604.1 (Salmonella enterica, CFIAFB20170120) loaded with 86 markers.\n",
      "2024-04-17 18:14:33,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039279.1 (Salmonella enterica, CFIAFB20170119) loaded with 86 markers.\n",
      "2024-04-17 18:14:33,486 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017977.1 (Salmonella enterica, CDC_2012K-1544) loaded with 39 markers.\n",
      "2024-04-17 18:14:33,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020912.1 (Salmonella enterica, CDC_2010K-0257) loaded with 39 markers.\n",
      "2024-04-17 18:14:33,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007211.2 (Salmonella enterica, CDC_06-0532) loaded with 44 markers.\n",
      "2024-04-17 18:14:33,779 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015724.1 (Salmonella enterica, C629) loaded with 43 markers.\n",
      "2024-04-17 18:14:33,881 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045958.1 (Salmonella enterica, AUSMDU00010532) loaded with 40 markers.\n",
      "2024-04-17 18:14:33,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045955.1 (Salmonella enterica, AUSMDU00010528) loaded with 41 markers.\n",
      "2024-04-17 18:14:34,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045956.1 (Salmonella enterica, AUSMDU00010527) loaded with 41 markers.\n",
      "2024-04-17 18:14:34,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019189.1 (Salmonella enterica, ATCC_BAA-1673) loaded with 42 markers.\n",
      "2024-04-17 18:14:34,152 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007531.1 (Salmonella enterica, ATCC_BAA-1592) loaded with 44 markers.\n",
      "2024-04-17 18:14:34,219 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012346.1 (Salmonella enterica, ATCC_7378) loaded with 43 markers.\n",
      "2024-04-17 18:14:34,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007534.1 (Salmonella enterica, ATCC_6017) loaded with 43 markers.\n",
      "2024-04-17 18:14:34,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019177.1 (Salmonella enterica, ATCC_51960) loaded with 42 markers.\n",
      "2024-04-17 18:14:34,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019179.1 (Salmonella enterica, ATCC_39184) loaded with 41 markers.\n",
      "2024-04-17 18:14:34,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007532.1 (Salmonella enterica, ATCC_35640) loaded with 45 markers.\n",
      "2024-04-17 18:14:34,856 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012349.1 (Salmonella enterica, ATCC_15791) loaded with 43 markers.\n",
      "2024-04-17 18:14:34,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019186.1 (Salmonella enterica, ATCC_10729) loaded with 40 markers.\n",
      "2024-04-17 18:14:34,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019181.1 (Salmonella enterica, ATCC_10720) loaded with 38 markers.\n",
      "2024-04-17 18:14:35,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019192.1 (Salmonella enterica, ATCC_10717) loaded with 46 markers.\n",
      "2024-04-17 18:14:35,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016837.1 (Salmonella enterica, 775W_(ATCC_43845)) loaded with 43 markers.\n",
      "2024-04-17 18:14:35,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067078.1 (Salmonella enterica, 661) loaded with 42 markers.\n",
      "2024-04-17 18:14:35,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029035.1 (Salmonella enterica, 531954) loaded with 39 markers.\n",
      "2024-04-17 18:14:35,392 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007530.1 (Salmonella enterica, 507440-20) loaded with 39 markers.\n",
      "2024-04-17 18:14:35,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011259.2 (Salmonella enterica, 460004_2-1) loaded with 43 markers.\n",
      "2024-04-17 18:14:35,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019201.1 (Salmonella enterica, 420) loaded with 45 markers.\n",
      "2024-04-17 18:14:35,612 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015024.2 (Salmonella enterica, 392869-2) loaded with 43 markers.\n",
      "2024-04-17 18:14:35,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054827.1 (Salmonella enterica, 389598) loaded with 42 markers.\n",
      "2024-04-17 18:14:35,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029908.1 (Salmonella enterica, 311189_239103) loaded with 42 markers.\n",
      "2024-04-17 18:14:35,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029918.1 (Salmonella enterica, 311189_232103) loaded with 42 markers.\n",
      "2024-04-17 18:14:35,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029927.1 (Salmonella enterica, 311189_217103) loaded with 42 markers.\n",
      "2024-04-17 18:14:35,933 [DEBUG - chronostrain.database.parser.json] - Strain NC_022991.1 (Salmonella enterica, 24249) loaded with 43 markers.\n",
      "2024-04-17 18:14:36,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031189.1 (Salmonella enterica, 222) loaded with 44 markers.\n",
      "2024-04-17 18:14:36,069 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026609.1 (Salmonella enterica, 2014LSAL01779) loaded with 41 markers.\n",
      "2024-04-17 18:14:36,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053584.1 (Salmonella enterica, 2014K-1020) loaded with 42 markers.\n",
      "2024-04-17 18:14:36,194 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053579.1 (Salmonella enterica, 2012K-0845) loaded with 41 markers.\n",
      "2024-04-17 18:14:36,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071388.1 (Salmonella enterica, 18-SA00377-0) loaded with 43 markers.\n",
      "2024-04-17 18:14:36,334 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034074.1 (Salmonella enterica, 16-SA00356) loaded with 42 markers.\n",
      "2024-04-17 18:14:36,403 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026660.1 (Salmonella enterica, 15-SA01028) loaded with 40 markers.\n",
      "2024-04-17 18:14:36,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092258.1 (Salmonella enterica, 15) loaded with 43 markers.\n",
      "2024-04-17 18:14:36,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054422.1 (Salmonella enterica, 14-SA00836-0) loaded with 42 markers.\n",
      "2024-04-17 18:14:36,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN649235.1 (Salmonella enterica, 1326_28) loaded with 40 markers.\n",
      "2024-04-17 18:14:36,667 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032815.1 (Salmonella enterica, 11TTU1615b) loaded with 40 markers.\n",
      "2024-04-17 18:14:36,740 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032817.1 (Salmonella enterica, 11TTU1590) loaded with 40 markers.\n",
      "2024-04-17 18:14:36,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047881.1 (Salmonella enterica, 119944) loaded with 41 markers.\n",
      "2024-04-17 18:14:36,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032814.1 (Salmonella enterica, 10TTU468x) loaded with 40 markers.\n",
      "2024-04-17 18:14:36,947 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020492.1 (Salmonella enterica, 08-00436) loaded with 44 markers.\n",
      "2024-04-17 18:14:37,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019198.1 (Salmonella enterica, 0315) loaded with 45 markers.\n",
      "2024-04-17 18:14:37,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021983.1 (Raoultella ornithinolytica, WP3-W18-ESBL-06) loaded with 25 markers.\n",
      "2024-04-17 18:14:37,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049752.1 (Raoultella ornithinolytica, WM1) loaded with 20 markers.\n",
      "2024-04-17 18:14:37,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134195.1 (Raoultella ornithinolytica, NCTC13096) loaded with 21 markers.\n",
      "2024-04-17 18:14:37,394 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046672.1 (Raoultella ornithinolytica, 172117885) loaded with 23 markers.\n",
      "2024-04-17 18:14:37,467 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080099.1 (Klebsiella quasipneumoniae, WW-14A) loaded with 27 markers.\n",
      "2024-04-17 18:14:37,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022142.1 (Klebsiella quasipneumoniae, WP5-S18-ESBL-05) loaded with 28 markers.\n",
      "2024-04-17 18:14:37,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019687.1 (Klebsiella quasipneumoniae, SNI47) loaded with 18 markers.\n",
      "2024-04-17 18:14:37,651 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075885.1 (Klebsiella quasipneumoniae, JNQH473) loaded with 16 markers.\n",
      "2024-04-17 18:14:37,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083633.1 (Klebsiella quasipneumoniae, FDAARGOS_1503) loaded with 17 markers.\n",
      "2024-04-17 18:14:37,768 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039791.1 (Klebsiella quasipneumoniae, C3001) loaded with 20 markers.\n",
      "2024-04-17 18:14:37,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084876.1 (Klebsiella quasipneumoniae, 01A030T) loaded with 17 markers.\n",
      "2024-04-17 18:14:37,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077331.1 (Lelliottia amnigena, FDAARGOS_1446) loaded with 40 markers.\n",
      "2024-04-17 18:14:37,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088265.1 (Leclercia sp., G3L) loaded with 38 markers.\n",
      "2024-04-17 18:14:38,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046445.1 (Leclercia sp., 119287) loaded with 34 markers.\n",
      "2024-04-17 18:14:38,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014007.2 (Kosakonia oryzae, Ola_51) loaded with 37 markers.\n",
      "2024-04-17 18:14:38,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065358.1 (Kosakonia oryzae, IAC-BECa90) loaded with 37 markers.\n",
      "2024-04-17 18:14:38,246 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022201.1 (Klebsiella sp., WP7-S18-ESBL-04) loaded with 56 markers.\n",
      "2024-04-17 18:14:38,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022195.1 (Klebsiella sp., WP7-S18-CRE-03) loaded with 53 markers.\n",
      "2024-04-17 18:14:38,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022189.1 (Klebsiella sp., WP7-S18-CRE-02) loaded with 53 markers.\n",
      "2024-04-17 18:14:38,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022057.1 (Klebsiella sp., WP4-W18-ESBL-05) loaded with 44 markers.\n",
      "2024-04-17 18:14:38,599 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077429.1 (Klebsiella aerogenes, NTT31XS) loaded with 18 markers.\n",
      "2024-04-17 18:14:38,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134475.1 (Klebsiella aerogenes, NCTC9735) loaded with 19 markers.\n",
      "2024-04-17 18:14:38,734 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023963.1 (Klebsiella aerogenes, FDAARGOS_363) loaded with 20 markers.\n",
      "2024-04-17 18:14:38,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011574.1 (Klebsiella aerogenes, CAV1320) loaded with 20 markers.\n",
      "2024-04-17 18:14:38,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042530.1 (Klebsiella aerogenes, C9) loaded with 18 markers.\n",
      "2024-04-17 18:14:38,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024880.1 (Klebsiella aerogenes, AR_0018) loaded with 18 markers.\n",
      "2024-04-17 18:14:38,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024883.1 (Klebsiella aerogenes, AR_0007) loaded with 20 markers.\n",
      "2024-04-17 18:14:38,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050067.1 (Klebsiella aerogenes, 035) loaded with 18 markers.\n",
      "2024-04-17 18:14:39,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037864.1 (Citrobacter arsenatis, LY-1) loaded with 46 markers.\n",
      "2024-04-17 18:14:39,131 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074425.1 (Salmonella sp., SJTUF15034) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074428.1 (Salmonella sp., SJTUF14523) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,251 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071693.1 (Salmonella sp., SAL-045) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071690.1 (Salmonella sp., SAL-020) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071686.1 (Salmonella sp., SAL-007) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,443 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082916.1 (Salmonella enterica, ZC-S1_3rd) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,505 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085801.1 (Salmonella enterica, Wailord) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085802.1 (Salmonella enterica, Wailmer) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,637 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085823.1 (Salmonella enterica, Venusaur) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085800.1 (Salmonella enterica, Taillow) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,763 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085799.1 (Salmonella enterica, Swellow) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060522.1 (Salmonella enterica, SLR1_8250) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080091.1 (Salmonella enterica, SLR1_8245) loaded with 40 markers.\n",
      "2024-04-17 18:14:39,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060517.1 (Salmonella enterica, SLR1_7627) loaded with 41 markers.\n",
      "2024-04-17 18:14:40,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085816.1 (Salmonella enterica, Slowpoke) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,087 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085815.1 (Salmonella enterica, Slowbro) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041173.1 (Salmonella enterica, SJTUF12519v2) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041176.1 (Salmonella enterica, SJTUF12367v2) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015526.1 (Salmonella enterica, SJTUF10984) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,341 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015524.1 (Salmonella enterica, SJTUF10978) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,402 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045761.1 (Salmonella enterica, SGSC_3086) loaded with 41 markers.\n",
      "2024-04-17 18:14:40,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033090.1 (Salmonella enterica, SEO) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008928.1 (Salmonella enterica, SEJ) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011791.1 (Salmonella enterica, SEE2) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011790.1 (Salmonella enterica, SEE1) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050716.1 (Salmonella enterica, SE95) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019681.1 (Salmonella enterica, SE86) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,844 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050721.1 (Salmonella enterica, SE81) loaded with 42 markers.\n",
      "2024-04-17 18:14:40,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050723.1 (Salmonella enterica, SE74) loaded with 40 markers.\n",
      "2024-04-17 18:14:40,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084532.1 (Salmonella enterica, SE211) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,027 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050706.1 (Salmonella enterica, SE124) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050709.1 (Salmonella enterica, SE109) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050712.1 (Salmonella enterica, SE104) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091300.1 (Salmonella enterica, SB699) loaded with 41 markers.\n",
      "2024-04-17 18:14:41,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040646.1 (Salmonella enterica, SAP18-H9654) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040644.1 (Salmonella enterica, SAP18-0432) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029989.1 (Salmonella enterica, SA20121591) loaded with 44 markers.\n",
      "2024-04-17 18:14:41,501 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030024.1 (Salmonella enterica, SA20120345) loaded with 41 markers.\n",
      "2024-04-17 18:14:41,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030233.1 (Salmonella enterica, SA20101045) loaded with 43 markers.\n",
      "2024-04-17 18:14:41,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007312.2 (Salmonella enterica, SA20094352) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,704 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030203.1 (Salmonella enterica, SA20083530) loaded with 45 markers.\n",
      "2024-04-17 18:14:41,778 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030223.1 (Salmonella enterica, SA20083039) loaded with 44 markers.\n",
      "2024-04-17 18:14:41,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007425.1 (Salmonella enterica, SA20082034) loaded with 40 markers.\n",
      "2024-04-17 18:14:41,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030194.1 (Salmonella enterica, SA20080453) loaded with 41 markers.\n",
      "2024-04-17 18:14:42,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022034.1 (Salmonella enterica, SA20060086) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030238.1 (Salmonella enterica, SA20055162) loaded with 41 markers.\n",
      "2024-04-17 18:14:42,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030196.1 (Salmonella enterica, SA20051401) loaded with 41 markers.\n",
      "2024-04-17 18:14:42,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022135.1 (Salmonella enterica, SA20044251) loaded with 42 markers.\n",
      "2024-04-17 18:14:42,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022116.1 (Salmonella enterica, SA20034636) loaded with 41 markers.\n",
      "2024-04-17 18:14:42,399 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030181.1 (Salmonella enterica, SA20030575) loaded with 41 markers.\n",
      "2024-04-17 18:14:42,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030219.1 (Salmonella enterica, SA20021456) loaded with 42 markers.\n",
      "2024-04-17 18:14:42,552 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007292.1 (Salmonella enterica, SA19994216) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007291.2 (Salmonella enterica, SA19992322) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007288.2 (Salmonella enterica, SA19981857) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007286.2 (Salmonella enterica, SA19981522) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,818 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007285.2 (Salmonella enterica, SA19980677) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007283.2 (Salmonella enterica, SA19970769) loaded with 40 markers.\n",
      "2024-04-17 18:14:42,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007282.2 (Salmonella enterica, SA19970510) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,029 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007280.2 (Salmonella enterica, SA19960848) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,093 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019415.1 (Salmonella enterica, S-1843) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019406.1 (Salmonella enterica, S-1824) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,230 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022117.1 (Salmonella enterica, S-1643) loaded with 41 markers.\n",
      "2024-04-17 18:14:43,296 [DEBUG - chronostrain.database.parser.json] - Strain NC_010067.1 (Salmonella enterica, RSK2980) loaded with 35 markers.\n",
      "2024-04-17 18:14:43,354 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028157.1 (Salmonella enterica, RM4283) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028151.1 (Salmonella enterica, RM2968) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,483 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063289.1 (Salmonella enterica, R17.4111) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022963.1 (Salmonella enterica, QJ-2D-Sal) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043433.1 (Salmonella enterica, PT1) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085793.1 (Salmonella enterica, Poochyena) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085814.1 (Salmonella enterica, Plusle) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025559.1 (Salmonella enterica, PIR00532) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063700.1 (Salmonella enterica, P125109) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,931 [DEBUG - chronostrain.database.parser.json] - Strain NC_011294.1 (Salmonella enterica, P125109) loaded with 40 markers.\n",
      "2024-04-17 18:14:43,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009091.1 (Salmonella enterica, OLF-SE9-10012) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009090.1 (Salmonella enterica, OLF-SE8-1021710) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009089.1 (Salmonella enterica, OLF-SE7-100819) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009088.1 (Salmonella enterica, OLF-SE6-00219-16) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009087.1 (Salmonella enterica, OLF-SE5-1104-2) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009086.1 (Salmonella enterica, OLF-SE4-0317-8) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009085.2 (Salmonella enterica, OLF-SE3-98983-4) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,489 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009084.2 (Salmonella enterica, OLF-SE2-98984-6) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,565 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009093.1 (Salmonella enterica, OLF-SE11-10058) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,627 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009092.1 (Salmonella enterica, OLF-SE10-10052) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009083.1 (Salmonella enterica, OLF-SE1-1019-1) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,761 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011942.1 (Salmonella enterica, OLF-00D989_87-1) loaded with 40 markers.\n",
      "2024-04-17 18:14:44,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064709.1 (Salmonella enterica, NCTC_74) loaded with 43 markers.\n",
      "2024-04-17 18:14:44,918 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483465.1 (Salmonella enterica, NCTC10080) loaded with 45 markers.\n",
      "2024-04-17 18:14:44,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041973.1 (Salmonella enterica, NCCP_16206) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085813.1 (Salmonella enterica, Minun) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085792.1 (Salmonella enterica, Mightyena) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026569.1 (Salmonella enterica, MFDS1004839) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085806.1 (Salmonella enterica, Marill) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,315 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078142.1 (Salmonella enterica, LHICA_D1) loaded with 43 markers.\n",
      "2024-04-17 18:14:45,380 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085804.1 (Salmonella enterica, Leafeon) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085797.1 (Salmonella enterica, Latios) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,508 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085798.1 (Salmonella enterica, Latias) loaded with 39 markers.\n",
      "2024-04-17 18:14:45,577 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085824.1 (Salmonella enterica, Ivysaur) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,644 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023345.1 (Salmonella enterica, HZS154) loaded with 42 markers.\n",
      "2024-04-17 18:14:45,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069166.1 (Salmonella enterica, GSJ_2016-Sal.-018) loaded with 41 markers.\n",
      "2024-04-17 18:14:45,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069297.1 (Salmonella enterica, GSJ_2016-Sal-017) loaded with 41 markers.\n",
      "2024-04-17 18:14:45,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085794.1 (Salmonella enterica, Greninja) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,909 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085803.1 (Salmonella enterica, Glaceon) loaded with 40 markers.\n",
      "2024-04-17 18:14:45,978 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085795.1 (Salmonella enterica, Frogadier) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085796.1 (Salmonella enterica, Froakie) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026713.1 (Salmonella enterica, FORC_078) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,168 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023475.1 (Salmonella enterica, FORC_075) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023436.1 (Salmonella enterica, FORC_074) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017177.1 (Salmonella enterica, FORC_056) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016754.1 (Salmonella enterica, FORC_052) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017232.1 (Salmonella enterica, FORC_051) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,517 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012144.1 (Salmonella enterica, FORC_020) loaded with 44 markers.\n",
      "2024-04-17 18:14:46,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012396.1 (Salmonella enterica, FORC_019) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,650 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009768.1 (Salmonella enterica, FORC_007) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019383.1 (Salmonella enterica, FORC50) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,792 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026052.1 (Salmonella enterica, FDAARGOS_70) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022069.2 (Salmonella enterica, FDAARGOS_313) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066047.1 (Salmonella enterica, FDAARGOS_1066) loaded with 40 markers.\n",
      "2024-04-17 18:14:46,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033360.1 (Salmonella enterica, EQAS2016S1) loaded with 41 markers.\n",
      "2024-04-17 18:14:47,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007273.1 (Salmonella enterica, EC20121180) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007272.2 (Salmonella enterica, EC20121179) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007271.1 (Salmonella enterica, EC20121178) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,246 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007333.1 (Salmonella enterica, EC20121177) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,325 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007269.1 (Salmonella enterica, EC20121175) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007434.2 (Salmonella enterica, EC20120200) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007245.1 (Salmonella enterica, EC20120008) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007267.2 (Salmonella enterica, EC20120005) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,581 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007329.2 (Salmonella enterica, EC20120002) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,651 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007252.1 (Salmonella enterica, EC20111175) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,709 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007253.1 (Salmonella enterica, EC20111174) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007254.1 (Salmonella enterica, EC20111095) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,828 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007263.1 (Salmonella enterica, EC20110361) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007258.1 (Salmonella enterica, EC20110360) loaded with 40 markers.\n",
      "2024-04-17 18:14:47,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007259.1 (Salmonella enterica, EC20110359) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,062 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007260.1 (Salmonella enterica, EC20110358) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007261.1 (Salmonella enterica, EC20110357) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007262.1 (Salmonella enterica, EC20110356) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,260 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007250.1 (Salmonella enterica, EC20110355) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,320 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007175.1 (Salmonella enterica, EC20110354) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,391 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007247.1 (Salmonella enterica, EC20110221) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,451 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007360.2 (Salmonella enterica, EC20100325) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007246.1 (Salmonella enterica, EC20100101) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,586 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007248.1 (Salmonella enterica, EC20090698) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,661 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007249.2 (Salmonella enterica, EC20090641) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007507.1 (Salmonella enterica, Durban) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063702.1 (Salmonella enterica, D7795) loaded with 40 markers.\n",
      "2024-04-17 18:14:48,869 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051349.1 (Salmonella enterica, CVM_30176) loaded with 41 markers.\n",
      "2024-04-17 18:14:48,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051368.1 (Salmonella enterica, CVM_24399) loaded with 42 markers.\n",
      "2024-04-17 18:14:49,013 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051370.1 (Salmonella enterica, CVM_24392) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063705.1 (Salmonella enterica, CP255) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,161 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013097.1 (Salmonella enterica, CMCC50041) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,229 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085818.1 (Salmonella enterica, Clefairy) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085817.1 (Salmonella enterica, Clefable) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,364 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085822.1 (Salmonella enterica, Charmeleon) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085821.1 (Salmonella enterica, Charizard) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,496 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033340.1 (Salmonella enterica, CFSAN076214) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024165.1 (Salmonella enterica, CFSAN070644__24N_attTn7::_Ptrc-gfpUV-cat__bright_fluorescence) loaded with 42 markers.\n",
      "2024-04-17 18:14:49,615 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028172.1 (Salmonella enterica, CFSAN064033) loaded with 41 markers.\n",
      "2024-04-17 18:14:49,681 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075116.1 (Salmonella enterica, CFSAN059881) loaded with 43 markers.\n",
      "2024-04-17 18:14:49,739 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075118.1 (Salmonella enterica, CFSAN051890) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,814 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075120.1 (Salmonella enterica, CFSAN051882) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022003.1 (Salmonella enterica, CFSAN051873) loaded with 40 markers.\n",
      "2024-04-17 18:14:49,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075122.1 (Salmonella enterica, CFSAN051827) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,005 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075132.1 (Salmonella enterica, CFSAN044911) loaded with 38 markers.\n",
      "2024-04-17 18:14:50,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020825.1 (Salmonella enterica, CFSAN033543) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020823.1 (Salmonella enterica, CFSAN033541) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074661.1 (Salmonella enterica, CFSAN008104) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007528.1 (Salmonella enterica, CDC_2010K_0968) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,321 [DEBUG - chronostrain.database.parser.json] - Strain NC_022221.1 (Salmonella enterica, CDC1983-67) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085805.1 (Salmonella enterica, Azumarill) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,448 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025554.1 (Salmonella enterica, ATCC_BAA-708) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012347.1 (Salmonella enterica, ATCC_9120) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,585 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034233.1 (Salmonella enterica, ATCC_8400) loaded with 44 markers.\n",
      "2024-04-17 18:14:50,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063708.1 (Salmonella enterica, A1636) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018661.1 (Salmonella enterica, 95-0621) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,777 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018659.1 (Salmonella enterica, 93-0639) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,837 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019035.1 (Salmonella enterica, 9184) loaded with 40 markers.\n",
      "2024-04-17 18:14:50,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054715.1 (Salmonella enterica, 85-0120) loaded with 39 markers.\n",
      "2024-04-17 18:14:50,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018655.1 (Salmonella enterica, 81-1706) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018651.1 (Salmonella enterica, 81-1705) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018648.1 (Salmonella enterica, 81-1607) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,159 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018647.1 (Salmonella enterica, 81-1435) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,226 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018645.1 (Salmonella enterica, 79-2359) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,285 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018644.1 (Salmonella enterica, 77-2980) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,344 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007598.1 (Salmonella enterica, 77-1427) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,410 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018642.1 (Salmonella enterica, 74-1357) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018640.1 (Salmonella enterica, 70-1605) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018637.1 (Salmonella enterica, 69-3861) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,598 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018635.1 (Salmonella enterica, 56-3991) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018633.1 (Salmonella enterica, 49-2444) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,720 [DEBUG - chronostrain.database.parser.json] - Strain NC_011274.1 (Salmonella enterica, 287_91) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,795 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073715.1 (Salmonella enterica, 2020LSAL11867) loaded with 41 markers.\n",
      "2024-04-17 18:14:51,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030794.1 (Salmonella enterica, 2017K-0021) loaded with 40 markers.\n",
      "2024-04-17 18:14:51,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075174.1 (Salmonella enterica, 20-369) loaded with 41 markers.\n",
      "2024-04-17 18:14:51,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011394.1 (Salmonella enterica, 18569) loaded with 40 markers.\n",
      "2024-04-17 18:14:52,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038595.1 (Salmonella enterica, 12-2388) loaded with 40 markers.\n",
      "2024-04-17 18:14:52,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011288.1 (Salmonella enterica, 11-01855) loaded with 42 markers.\n",
      "2024-04-17 18:14:52,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011292.1 (Salmonella enterica, 11-01854) loaded with 42 markers.\n",
      "2024-04-17 18:14:52,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011289.1 (Salmonella enterica, 11-01853) loaded with 42 markers.\n",
      "2024-04-17 18:14:52,312 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077760.1 (Salmonella enterica, 07Q015) loaded with 40 markers.\n",
      "2024-04-17 18:14:52,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067413.1 (Raoultella ornithinolytica, TET32) loaded with 20 markers.\n",
      "2024-04-17 18:14:52,439 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054270.1 (Raoultella ornithinolytica, OHMEATKPC2_NDM5) loaded with 25 markers.\n",
      "2024-04-17 18:14:52,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084803.1 (Klebsiella quasipneumoniae, SB98) loaded with 17 markers.\n",
      "2024-04-17 18:14:52,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081895.1 (Klebsiella quasipneumoniae, MN-07) loaded with 20 markers.\n",
      "2024-04-17 18:14:52,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029432.1 (Klebsiella quasipneumoniae, CAV2018) loaded with 18 markers.\n",
      "2024-04-17 18:14:52,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029437.1 (Klebsiella quasipneumoniae, CAV2013) loaded with 18 markers.\n",
      "2024-04-17 18:14:52,734 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029443.1 (Klebsiella quasipneumoniae, CAV1947) loaded with 18 markers.\n",
      "2024-04-17 18:14:52,798 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070440.1 (Klebsiella quasipneumoniae, B9067) loaded with 19 markers.\n",
      "2024-04-17 18:14:52,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072468.1 (Klebsiella quasipneumoniae, B6885) loaded with 20 markers.\n",
      "2024-04-17 18:14:52,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070420.1 (Klebsiella quasipneumoniae, B2663) loaded with 20 markers.\n",
      "2024-04-17 18:14:52,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070454.1 (Klebsiella quasipneumoniae, B25110) loaded with 21 markers.\n",
      "2024-04-17 18:14:53,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067966.1 (Klebsiella quasipneumoniae, ARLG-8054) loaded with 17 markers.\n",
      "2024-04-17 18:14:53,097 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026368.1 (Klebsiella quasipneumoniae, A708) loaded with 16 markers.\n",
      "2024-04-17 18:14:53,150 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030171.1 (Klebsiella quasipneumoniae, A708) loaded with 16 markers.\n",
      "2024-04-17 18:14:53,238 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084542.1 (Klebsiella michiganensis, YL9) loaded with 21 markers.\n",
      "2024-04-17 18:14:53,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084415.1 (Klebsiella michiganensis, YL6-1) loaded with 21 markers.\n",
      "2024-04-17 18:14:53,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089315.1 (Klebsiella michiganensis, K92) loaded with 21 markers.\n",
      "2024-04-17 18:14:53,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023185.1 (Klebsiella michiganensis, K518) loaded with 20 markers.\n",
      "2024-04-17 18:14:53,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022348.1 (Klebsiella michiganensis, K516) loaded with 20 markers.\n",
      "2024-04-17 18:14:53,631 [DEBUG - chronostrain.database.parser.json] - Strain NC_018106.1 (Klebsiella michiganensis, E718) loaded with 20 markers.\n",
      "2024-04-17 18:14:53,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081351.1 (Klebsiella michiganensis, CCRI-24235) loaded with 19 markers.\n",
      "2024-04-17 18:14:53,783 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP048108.1 (Klebsiella michiganensis, BD177) loaded with 22 markers.\n",
      "2024-04-17 18:14:53,859 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061930.1 (Klebsiella michiganensis, BD-50-Km) loaded with 19 markers.\n",
      "2024-04-17 18:14:53,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042361.1 (Lelliottia amnigena, A167) loaded with 32 markers.\n",
      "2024-04-17 18:14:53,977 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063425.1 (Kosakonia pseudosacchari, BDA62-3) loaded with 35 markers.\n",
      "2024-04-17 18:14:54,038 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045868.1 (Klebsiella aerogenes, Y6) loaded with 16 markers.\n",
      "2024-04-17 18:14:54,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045869.1 (Klebsiella aerogenes, Y3) loaded with 16 markers.\n",
      "2024-04-17 18:14:54,150 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045870.1 (Klebsiella aerogenes, Y1) loaded with 16 markers.\n",
      "2024-04-17 18:14:54,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035466.1 (Klebsiella aerogenes, LU2) loaded with 18 markers.\n",
      "2024-04-17 18:14:54,260 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044214.1 (Klebsiella aerogenes, KA_P10_L5_03.19) loaded with 20 markers.\n",
      "2024-04-17 18:14:54,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057483.1 (Citrobacter sp., RHB25-C09) loaded with 43 markers.\n",
      "2024-04-17 18:14:54,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059886.1 (Salmonella enterica, XXB1403) loaded with 48 markers.\n",
      "2024-04-17 18:14:54,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060508.1 (Salmonella enterica, SLR1_7697) loaded with 41 markers.\n",
      "2024-04-17 18:14:54,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067337.1 (Salmonella enterica, Se13) loaded with 48 markers.\n",
      "2024-04-17 18:14:54,594 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080513.1 (Salmonella enterica, SA18578) loaded with 42 markers.\n",
      "2024-04-17 18:14:54,672 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082954.1 (Salmonella enterica, S499) loaded with 47 markers.\n",
      "2024-04-17 18:14:54,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006693.1 (Salmonella enterica, RKS2983) loaded with 48 markers.\n",
      "2024-04-17 18:14:54,823 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022142.1 (Salmonella enterica, MZ0080) loaded with 43 markers.\n",
      "2024-04-17 18:14:54,889 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046291.1 (Salmonella enterica, FDAARGOS_712) loaded with 42 markers.\n",
      "2024-04-17 18:14:54,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017720.1 (Salmonella enterica, CFSAN017963) loaded with 41 markers.\n",
      "2024-04-17 18:14:55,017 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019184.1 (Salmonella enterica, ATCC_49284) loaded with 41 markers.\n",
      "2024-04-17 18:14:55,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022139.1 (Salmonella enterica, 1315K) loaded with 41 markers.\n",
      "2024-04-17 18:14:55,181 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017450.1 (Klebsiella sp., LTGPAF-6F) loaded with 23 markers.\n",
      "2024-04-17 18:14:55,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026275.1 (Klebsiella oxytoca, KONIH5) loaded with 20 markers.\n",
      "2024-04-17 18:14:55,332 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018362.1 (Klebsiella oxytoca, CAV1752) loaded with 25 markers.\n",
      "2024-04-17 18:14:55,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035214.1 (Klebsiella michiganensis, M82255) loaded with 25 markers.\n",
      "2024-04-17 18:14:55,490 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008841.1 (Klebsiella michiganensis, M1) loaded with 20 markers.\n",
      "2024-04-17 18:14:55,575 [DEBUG - chronostrain.database.parser.json] - Strain NC_016612.1 (Klebsiella michiganensis, KCTC_1686) loaded with 19 markers.\n",
      "2024-04-17 18:14:55,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093010.1 (Klebsiella michiganensis, K342) loaded with 20 markers.\n",
      "2024-04-17 18:14:55,734 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075881.1 (Klebsiella michiganensis, JNQH491) loaded with 20 markers.\n",
      "2024-04-17 18:14:55,803 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP004887.1 (Klebsiella michiganensis, HKOPL1) loaded with 19 markers.\n",
      "2024-04-17 18:14:55,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042545.1 (Klebsiella michiganensis, C52) loaded with 20 markers.\n",
      "2024-04-17 18:14:55,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072119.1 (Klebsiella michiganensis, 12084) loaded with 19 markers.\n",
      "2024-04-17 18:14:56,046 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023529.1 (Lelliottia amnigena, FDAARGOS_395) loaded with 31 markers.\n",
      "2024-04-17 18:14:56,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077236.1 (Lelliottia amnigena, FDAARGOS_1444) loaded with 31 markers.\n",
      "2024-04-17 18:14:56,207 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040889.1 (Leclercia adecarboxylata, Z96-1) loaded with 33 markers.\n",
      "2024-04-17 18:14:56,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065411.1 (Kosakonia radicincitans, IAC-BECa95) loaded with 41 markers.\n",
      "2024-04-17 18:14:56,412 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026285.1 (Klebsiella oxytoca, KONIH2) loaded with 35 markers.\n",
      "2024-04-17 18:14:56,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020358.1 (Klebsiella oxytoca, AR_0147) loaded with 22 markers.\n",
      "2024-04-17 18:14:56,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026715.1 (Klebsiella oxytoca, AR_0028) loaded with 19 markers.\n",
      "2024-04-17 18:14:56,659 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022547.1 (Klebsiella michiganensis, THO-011) loaded with 19 markers.\n",
      "2024-04-17 18:14:56,727 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058212.1 (Klebsiella michiganensis, RHB20-C02) loaded with 18 markers.\n",
      "2024-04-17 18:14:56,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065338.1 (Klebsiella michiganensis, LDS17) loaded with 18 markers.\n",
      "2024-04-17 18:14:56,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089407.1 (Klebsiella michiganensis, Kox101) loaded with 24 markers.\n",
      "2024-04-17 18:14:56,979 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041515.1 (Klebsiella michiganensis, KNU07) loaded with 39 markers.\n",
      "2024-04-17 18:14:57,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071393.1 (Klebsiella michiganensis, Kmfe267) loaded with 18 markers.\n",
      "2024-04-17 18:14:57,134 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044109.1 (Klebsiella michiganensis, FDAARGOS_647) loaded with 21 markers.\n",
      "2024-04-17 18:14:57,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP073305.1 (Klebsiella michiganensis, CZ598) loaded with 24 markers.\n",
      "2024-04-17 18:14:57,309 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029141.1 (Klebsiella michiganensis, AR375) loaded with 20 markers.\n",
      "2024-04-17 18:14:57,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060111.1 (Klebsiella michiganensis, AKKL-001) loaded with 19 markers.\n",
      "2024-04-17 18:14:57,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089448.1 (Klebsiella michiganensis, 8-1) loaded with 19 markers.\n",
      "2024-04-17 18:14:57,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085764.1 (Klebsiella michiganensis, 53828CZ) loaded with 20 markers.\n",
      "2024-04-17 18:14:57,624 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013990.1 (Leclercia adecarboxylata, USDA-ARS-USMARC-60222) loaded with 34 markers.\n",
      "2024-04-17 18:14:57,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087280.1 (Leclercia adecarboxylata, SH19PE29) loaded with 36 markers.\n",
      "2024-04-17 18:14:57,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018016.1 (Kosakonia radicincitans, DSM_16656) loaded with 41 markers.\n",
      "2024-04-17 18:14:57,889 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040392.1 (Kosakonia radicincitans, DSM_107547) loaded with 37 markers.\n",
      "2024-04-17 18:14:57,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055325.1 (Klebsiella michiganensis, RHBSTW-00409) loaded with 18 markers.\n",
      "2024-04-17 18:14:58,056 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077673.1 (Klebsiella sp., PL-2018) loaded with 19 markers.\n",
      "2024-04-17 18:14:58,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026269.1 (Klebsiella oxytoca, KONIH4) loaded with 36 markers.\n",
      "2024-04-17 18:14:58,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008788.1 (Klebsiella oxytoca, KONIH1) loaded with 36 markers.\n",
      "2024-04-17 18:14:58,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051427.1 (Klebsiella michiganensis, X2-1) loaded with 16 markers.\n",
      "2024-04-17 18:14:58,433 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058118.1 (Klebsiella michiganensis, RHBSTW-00167) loaded with 23 markers.\n",
      "2024-04-17 18:14:58,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091470.1 (Klebsiella michiganensis, KO_408) loaded with 20 markers.\n",
      "2024-04-17 18:14:58,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089395.1 (Klebsiella michiganensis, Kox58) loaded with 50 markers.\n",
      "2024-04-17 18:14:58,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026387.1 (Leclercia sp., LSNIH3) loaded with 33 markers.\n",
      "2024-04-17 18:14:58,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046251.1 (Leclercia adecarboxylata, P12375) loaded with 37 markers.\n",
      "2024-04-17 18:14:58,960 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043397.1 (Leclercia adecarboxylata, L21) loaded with 42 markers.\n",
      "2024-04-17 18:14:59,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042930.1 (Leclercia adecarboxylata, J656) loaded with 34 markers.\n",
      "2024-04-17 18:14:59,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043398.1 (Leclercia adecarboxylata, G426) loaded with 34 markers.\n",
      "2024-04-17 18:14:59,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083630.1 (Leclercia adecarboxylata, FDAARGOS_1505) loaded with 36 markers.\n",
      "2024-04-17 18:14:59,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042505.1 (Leclercia adecarboxylata, E1) loaded with 37 markers.\n",
      "2024-04-17 18:14:59,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049980.1 (Leclercia adecarboxylata, 707804) loaded with 36 markers.\n",
      "2024-04-17 18:14:59,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036199.1 (Leclercia adecarboxylata, 16005813) loaded with 32 markers.\n",
      "2024-04-17 18:14:59,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134333.1 (Klebsiella oxytoca, NCTC13727) loaded with 20 markers.\n",
      "2024-04-17 18:14:59,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056136.1 (Klebsiella michiganensis, RHBSTW-00909) loaded with 37 markers.\n",
      "2024-04-17 18:14:59,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090078.1 (Klebsiella michiganensis, KM41) loaded with 18 markers.\n",
      "2024-04-17 18:14:59,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024643.1 (Klebsiella michiganensis, F107) loaded with 18 markers.\n",
      "2024-04-17 18:14:59,736 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065474.1 (Klebsiella michiganensis, 7525) loaded with 17 markers.\n",
      "2024-04-17 18:14:59,797 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042493.1 (Leclercia adecarboxylata, E61) loaded with 36 markers.\n",
      "2024-04-17 18:14:59,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022665.1 (Kluyvera ascorbata, TP1631) loaded with 33 markers.\n",
      "2024-04-17 18:14:59,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056199.1 (Enterobacter sp., RHBSTW-00994) loaded with 29 markers.\n",
      "2024-04-17 18:14:59,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015774.2 (Lelliottia amnigena, ZB04) loaded with 29 markers.\n",
      "2024-04-17 18:15:00,021 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061719.1 (Raoultella ornithinolytica, LL1) loaded with 21 markers.\n",
      "2024-04-17 18:15:00,082 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041247.1 (Raoultella electrica, DSM_102253) loaded with 19 markers.\n",
      "2024-04-17 18:15:00,175 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069911.1 (Klebsiella oxytoca, FDAARGOS_1334) loaded with 20 markers.\n",
      "2024-04-17 18:15:00,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015113.1 (Kosakonia radicincitans, GXGL-4A) loaded with 42 markers.\n",
      "2024-04-17 18:15:00,383 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045300.1 (Kosakonia arachidis, KACC_18508) loaded with 33 markers.\n",
      "2024-04-17 18:15:00,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012264.1 (Cronobacter condimenti, LMG_26250) loaded with 28 markers.\n",
      "2024-04-17 18:15:00,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046115.1 (Klebsiella oxytoca, P620) loaded with 24 markers.\n",
      "2024-04-17 18:15:00,584 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069943.1 (Klebsiella oxytoca, FDAARGOS_1332) loaded with 20 markers.\n",
      "2024-04-17 18:15:00,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070144.1 (Klebsiella oxytoca, FDAARGOS_1331) loaded with 21 markers.\n",
      "2024-04-17 18:15:00,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042499.1 (Enterobacter sp., E76) loaded with 28 markers.\n",
      "2024-04-17 18:15:00,787 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012268.1 (Cronobacter muytjensii, ATCC_51329) loaded with 27 markers.\n",
      "2024-04-17 18:15:00,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP054058.1 (Scandinavium goeteborgense, CCUG_66741) loaded with 36 markers.\n",
      "2024-04-17 18:15:00,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP023184.1 (Buttiauxella agrestis, DSM_9389) loaded with 26 markers.\n",
      "2024-04-17 18:15:01,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR133932.1 (Klebsiella oxytoca, NCTC11356) loaded with 21 markers.\n",
      "2024-04-17 18:15:01,085 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089392.1 (Klebsiella oxytoca, Kox71) loaded with 21 markers.\n",
      "2024-04-17 18:15:01,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089399.1 (Klebsiella oxytoca, Kox26) loaded with 21 markers.\n",
      "2024-04-17 18:15:01,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017662.1 (Cronobacter sp., JZ38) loaded with 26 markers.\n",
      "2024-04-17 18:15:01,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057330.1 (Klebsiella oxytoca, RHB30-C02) loaded with 22 markers.\n",
      "2024-04-17 18:15:01,343 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR890312.1 (Klebsiella oxytoca, MSB1_10D-sc-2280340) loaded with 22 markers.\n",
      "2024-04-17 18:15:01,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072914.1 (Klebsiella oxytoca, KoPF10) loaded with 20 markers.\n",
      "2024-04-17 18:15:01,488 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090245.1 (Klebsiella oxytoca, KONSF-1) loaded with 19 markers.\n",
      "2024-04-17 18:15:01,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064108.1 (Klebsiella oxytoca, IR5392) loaded with 21 markers.\n",
      "2024-04-17 18:15:01,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033844.1 (Klebsiella oxytoca, FDAARGOS_500) loaded with 18 markers.\n",
      "2024-04-17 18:15:01,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070020.1 (Klebsiella oxytoca, FDAARGOS_1333) loaded with 19 markers.\n",
      "2024-04-17 18:15:01,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011618.1 (Klebsiella oxytoca, CAV1335) loaded with 17 markers.\n",
      "2024-04-17 18:15:01,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011597.1 (Klebsiella oxytoca, CAV1099) loaded with 17 markers.\n",
      "2024-04-17 18:15:01,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017928.1 (Klebsiella oxytoca, CAV1015) loaded with 17 markers.\n",
      "2024-04-17 18:15:02,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029128.1 (Klebsiella oxytoca, AR380) loaded with 21 markers.\n",
      "2024-04-17 18:15:02,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040677.1 (Kosakonia sacchari, DSM_107661) loaded with 34 markers.\n",
      "2024-04-17 18:15:02,174 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016337.1 (Kosakonia sacchari, BO-1) loaded with 34 markers.\n",
      "2024-04-17 18:15:02,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089411.1 (Klebsiella oxytoca, Kox100) loaded with 24 markers.\n",
      "2024-04-17 18:15:02,341 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027426.1 (Klebsiella oxytoca, FDAARGOS_335) loaded with 20 markers.\n",
      "2024-04-17 18:15:02,416 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062916.1 (Raoultella terrigena, Res13-Abat-PEB01-P1-04-A) loaded with 20 markers.\n",
      "2024-04-17 18:15:02,477 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050508.1 (Raoultella terrigena, JH01) loaded with 18 markers.\n",
      "2024-04-17 18:15:02,555 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010557.1 (Raoultella ornithinolytica, S12) loaded with 17 markers.\n",
      "2024-04-17 18:15:02,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007215.2 (Kosakonia sacchari, SP1) loaded with 33 markers.\n",
      "2024-04-17 18:15:02,682 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056567.1 (Klebsiella oxytoca, RHBSTW-00373) loaded with 18 markers.\n",
      "2024-04-17 18:15:02,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR595855.1 (Raoultella terrigena, NCTC9189) loaded with 16 markers.\n",
      "2024-04-17 18:15:02,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033076.1 (Buttiauxella sp., 3AFRM03) loaded with 34 markers.\n",
      "2024-04-17 18:15:02,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077291.1 (Citrobacter farmeri, FDAARGOS_1423) loaded with 42 markers.\n",
      "2024-04-17 18:15:02,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022695.1 (Citrobacter farmeri, AUSMDU00008141) loaded with 42 markers.\n",
      "2024-04-17 18:15:03,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134215.1 (Shimwellia blattae, NCTC12127) loaded with 11 markers.\n",
      "2024-04-17 18:15:03,050 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR133996.1 (Shimwellia blattae, NCTC10965) loaded with 11 markers.\n",
      "2024-04-17 18:15:03,093 [DEBUG - chronostrain.database.parser.json] - Strain NC_017910.1 (Shimwellia blattae, DSM_4481) loaded with 11 markers.\n",
      "2024-04-17 18:15:03,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014136.1 (Gibbsiella quercinecans, FRB97) loaded with 6 markers.\n",
      "2024-04-17 18:15:03,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012266.1 (Cronobacter dublinensis, LMG_23823) loaded with 27 markers.\n",
      "2024-04-17 18:15:03,259 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP009450.1 (Pluralibacter gergoviae, FB2) loaded with 15 markers.\n",
      "2024-04-17 18:15:03,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR699009.1 (Pluralibacter gergoviae, MGYG-HGUT-02520) loaded with 19 markers.\n",
      "2024-04-17 18:15:03,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020388.1 (Pluralibacter gergoviae, FDAARGOS_186) loaded with 19 markers.\n",
      "2024-04-17 18:15:03,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011602.1 (Phytobacter ursingii, CAV1151) loaded with 29 markers.\n",
      "2024-04-17 18:15:03,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025334.1 (Phytobacter diazotrophicus, TA9734) loaded with 39 markers.\n",
      "2024-04-17 18:15:03,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018756.1 (Metakosakonia sp., MRY16-398) loaded with 38 markers.\n",
      "2024-04-17 18:15:03,733 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063427.1 (Citrobacter sp., BDA59-3) loaded with 31 markers.\n",
      "2024-04-17 18:15:03,805 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051548.1 (Phytobacter diazotrophicus, UAEU22) loaded with 28 markers.\n",
      "2024-04-17 18:15:03,871 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034225.1 (Kosakonia sp., CCTCC_M2018092) loaded with 35 markers.\n",
      "2024-04-17 18:15:03,926 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019445.1 (Kosakonia cowanii, 888-76) loaded with 35 markers.\n",
      "2024-04-17 18:15:04,019 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025757.1 (Citrobacter freundii, CFNIH2) loaded with 41 markers.\n",
      "2024-04-17 18:15:04,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069319.1 (Kosakonia cowanii, Pa82) loaded with 34 markers.\n",
      "2024-04-17 18:15:04,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035129.1 (Kosakonia cowanii, FBS_223) loaded with 32 markers.\n",
      "2024-04-17 18:15:04,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027225.1 (Phytobacter sp., SCO41) loaded with 29 markers.\n",
      "2024-04-17 18:15:04,295 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051488.1 (Kosakonia sp., SMBL-WEM22) loaded with 34 markers.\n",
      "2024-04-17 18:15:04,358 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057611.1 (Citrobacter sp., RHB21-C05) loaded with 39 markers.\n",
      "2024-04-17 18:15:04,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057620.1 (Citrobacter sp., RHB21-C01) loaded with 39 markers.\n",
      "2024-04-17 18:15:04,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071070.1 (Citrobacter sedlakii, 3347689II) loaded with 37 markers.\n",
      "2024-04-17 18:15:04,577 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050811.1 (Yokenella regensburgei, W13) loaded with 29 markers.\n",
      "2024-04-17 18:15:04,649 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014015.2 (Citrobacter amalonaticus, FDAARGOS_122) loaded with 47 markers.\n",
      "2024-04-17 18:15:04,727 [DEBUG - chronostrain.database.parser.json] - Strain NC_013716.1 (Citrobacter rodentium, ICC168) loaded with 46 markers.\n",
      "2024-04-17 18:15:04,815 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082833.1 (Citrobacter rodentium, DSM_16636) loaded with 46 markers.\n",
      "2024-04-17 18:15:04,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011132.1 (Citrobacter amalonaticus, Y19) loaded with 46 markers.\n",
      "2024-04-17 18:15:05,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045205.1 (Citrobacter telavivensis, 6105) loaded with 39 markers.\n",
      "2024-04-17 18:15:05,082 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069159.1 (Citrobacter sp., R56) loaded with 44 markers.\n",
      "2024-04-17 18:15:05,132 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028974.1 (Cronobacter sakazakii, GZcsf-1) loaded with 29 markers.\n",
      "2024-04-17 18:15:05,180 [DEBUG - chronostrain.database.parser.json] - Strain NC_020260.1 (Cronobacter sakazakii, Sp291) loaded with 33 markers.\n",
      "2024-04-17 18:15:05,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012253.1 (Cronobacter sakazakii, NCTC_8155) loaded with 33 markers.\n",
      "2024-04-17 18:15:05,281 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027107.1 (Cronobacter sakazakii, CS-931) loaded with 30 markers.\n",
      "2024-04-17 18:15:05,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011047.1 (Cronobacter sakazakii, ATCC_29544) loaded with 30 markers.\n",
      "2024-04-17 18:15:05,407 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081314.1 (Citrobacter farmeri, CCRI-24236) loaded with 42 markers.\n",
      "2024-04-17 18:15:05,481 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP026216.1 (Citrobacter sp., CFNIH10) loaded with 42 markers.\n",
      "2024-04-17 18:15:05,529 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049154.1 (Cronobacter sakazakii, C105731) loaded with 31 markers.\n",
      "2024-04-17 18:15:05,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050009.1 (Citrobacter sp., Y3) loaded with 46 markers.\n",
      "2024-04-17 18:15:05,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057150.1 (Citrobacter sp., RHB35-C17) loaded with 42 markers.\n",
      "2024-04-17 18:15:05,780 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057630.1 (Citrobacter sp., RHB20-C16) loaded with 38 markers.\n",
      "2024-04-17 18:15:05,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP057632.1 (Citrobacter sp., RHB20-C15) loaded with 38 markers.\n",
      "2024-04-17 18:15:05,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024585.1 (Citrobacter amalonaticus, JCM_1661) loaded with 38 markers.\n",
      "2024-04-17 18:15:06,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014070.2 (Citrobacter amalonaticus, FDAARGOS_165) loaded with 37 markers.\n",
      "2024-04-17 18:15:06,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083651.1 (Citrobacter amalonaticus, FDAARGOS_1489) loaded with 38 markers.\n",
      "2024-04-17 18:15:06,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064180.1 (Citrobacter amalonaticus, CA71) loaded with 38 markers.\n",
      "2024-04-17 18:15:06,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080958.1 (Citrobacter amalonaticus, C3H) loaded with 38 markers.\n",
      "2024-04-17 18:15:06,325 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT556085.1 (Citrobacter amalonaticus, 92) loaded with 48 markers.\n",
      "2024-04-17 18:15:06,403 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT556084.1 (Citrobacter amalonaticus, 86) loaded with 45 markers.\n",
      "2024-04-17 18:15:06,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041362.1 (Citrobacter amalonaticus, 133355-SW-C4-Cam) loaded with 29 markers.\n",
      "2024-04-17 18:15:06,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056155.1 (Enterobacter hormaechei, RHBSTW-00844) loaded with 31 markers.\n",
      "2024-04-17 18:15:06,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029185.2 (Limnobaculum parvum, HYN0051) loaded with 1 markers.\n",
      "2024-04-17 18:15:06,654 [INFO - chronostrain.database.database] - Instantiating database `ecoli`.\n",
      "2024-04-17 18:15:13,622 [DEBUG - chronostrain.database.parser.json] - Using `DictionaryBackend` for database backend.\n",
      "2024-04-17 18:15:13,624 [INFO - chronostrain.cli.prune_json] - Computing all-to-all Jaccard distance calculations.\n",
      "2024-04-17 18:15:13,624 [INFO - chronostrain.cli.prune_json] - Using ProbMinHash method for sketching (sz=8192).\n",
      "2024-04-17 18:15:25,299 [WARNING - chronostrain.cli.prune_json] - Strain NC_004431.1 (Escherichia coli, CFT073) -- marker wzc-wzb has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:36,537 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP012127.1 (Escherichia coli, DH1Ec169) -- marker fimE has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:56,862 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007291.2 (Salmonella enterica, SA19992322) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:56,862 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007288.2 (Salmonella enterica, SA19981857) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:56,862 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007288.2 (Salmonella enterica, SA19981857) -- marker rfbB has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:56,863 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007286.2 (Salmonella enterica, SA19981522) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:56,864 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007285.2 (Salmonella enterica, SA19980677) -- marker wzc has an N. This strain will be excluded.\n",
      "2024-04-17 18:15:57,482 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007360.2 (Salmonella enterica, EC20100325) -- marker nrfE has an N. This strain will be excluded.\n",
      "2024-04-17 18:19:18,220 [INFO - chronostrain.cli.prune_json] - Computing clusters.\n",
      "2024-04-17 18:20:12,566 [INFO - chronostrain.cli.prune_json] - Before clustering: 5398 genomes\n",
      "2024-04-17 18:20:12,566 [INFO - chronostrain.cli.prune_json] - After clustering: 2323 genomes\n",
      "2024-04-17 18:20:12,566 [DEBUG - chronostrain.cli.prune_json] - Cleaning up. Removing tmpdir = /mnt/e/ecoli_db/chronostrain_files/__cluster_tmp\n"
     ]
    }
   ],
   "source": [
    "# Perform clustering\n",
    "thresh = 0.998\n",
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      cluster-db \\\n",
    "      -i $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      -o $CHRONOSTRAIN_TARGET_CLUSTERS \\\n",
    "      --ident-threshold $thresh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "827c51ec-27d4-4fac-bdd0-e3b22b3e9fe9",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-04-17 18:21:19,458 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2024-04-17 18:21:19,472 [INFO - chronostrain.cli.prune_json] - Pruning database via clustering\n",
      "2024-04-17 18:21:19,472 [DEBUG - chronostrain.cli.prune_json] - Src: /mnt/e/ecoli_db/chronostrain_files/ecoli.json, Output: /mnt/e/ecoli_db/chronostrain_files/ecoli.clusters_100pct.txt\n",
      "2024-04-17 18:21:19,472 [INFO - chronostrain.cli.prune_json] - Target identity threshold = 0.9999999999\n",
      "2024-04-17 18:21:20,460 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2024-04-17 18:21:20,593 [INFO - chronostrain.cli.prune_json] - Preprocessing step -- Loading old DB instance, using data directory: /mnt/e/ecoli_db/chronostrain_files\n",
      "2024-04-17 18:21:22,360 [INFO - chronostrain.database.database] - Instantiating database `ecoli`.\n",
      "2024-04-17 18:21:22,362 [DEBUG - chronostrain.database.parser.json] - Loaded database instance from /mnt/e/ecoli_db/chronostrain_files/__ecoli_/database.posix.pkl.\n",
      "2024-04-17 18:21:22,364 [DEBUG - chronostrain.database.parser.json] - Using `DictionaryBackend` for database backend.\n",
      "2024-04-17 18:21:22,366 [INFO - chronostrain.cli.prune_json] - Computing all-to-all Jaccard distance calculations.\n",
      "2024-04-17 18:21:22,367 [INFO - chronostrain.cli.prune_json] - Using ProbMinHash method for sketching (sz=8192).\n",
      "2024-04-17 18:21:28,060 [WARNING - chronostrain.cli.prune_json] - Strain NC_004431.1 (Escherichia coli, CFT073) -- marker wzc-wzb has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:33,368 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP012127.1 (Escherichia coli, DH1Ec169) -- marker fimE has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:42,957 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007291.2 (Salmonella enterica, SA19992322) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:42,957 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007288.2 (Salmonella enterica, SA19981857) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:42,957 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007288.2 (Salmonella enterica, SA19981857) -- marker rfbB has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:42,957 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007286.2 (Salmonella enterica, SA19981522) -- marker cpsG has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:42,957 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007285.2 (Salmonella enterica, SA19980677) -- marker wzc has an N. This strain will be excluded.\n",
      "2024-04-17 18:21:43,255 [WARNING - chronostrain.cli.prune_json] - Strain NZ_CP007360.2 (Salmonella enterica, EC20100325) -- marker nrfE has an N. This strain will be excluded.\n",
      "2024-04-17 18:23:26,567 [INFO - chronostrain.cli.prune_json] - Computing clusters.\n",
      "2024-04-17 18:26:06,847 [INFO - chronostrain.cli.prune_json] - Before clustering: 5398 genomes\n",
      "2024-04-17 18:26:06,847 [INFO - chronostrain.cli.prune_json] - After clustering: 4107 genomes\n",
      "2024-04-17 18:26:06,847 [DEBUG - chronostrain.cli.prune_json] - Cleaning up. Removing tmpdir = /mnt/e/ecoli_db/chronostrain_files/__cluster_tmp\n"
     ]
    }
   ],
   "source": [
    "# Perform clustering\n",
    "thresh = 0.9999999999\n",
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      cluster-db \\\n",
    "      -i $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      -o $CHRONOSTRAIN_TARGET_CLUSTERS_100pct \\\n",
    "      --ident-threshold $thresh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "83e81c62-6add-4420-84e9-505592d76768",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4082\n",
      "missing strains:\n",
      "done.\n"
     ]
    }
   ],
   "source": [
    "with open(\"/data/cctm/youn/chronostrain_zenodo_new/done/umb/databases/chronostrain_files/ecoli.100pct.txt\", \"rt\") as f:\n",
    "    f.readline()\n",
    "    _x = [l.strip().split(\"\\t\")[0] for l in f]\n",
    "\n",
    "print(len(_x))\n",
    "print(\"missing strains:\")\n",
    "_s_ids = {s.id for s in src_db.all_strains()}\n",
    "for k in _x:\n",
    "    if k not in _s_ids:\n",
    "        print(k)\n",
    "print(\"done.\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f23a7c20-6ecd-4f6b-a529-f4d053d64de3",
   "metadata": {},
   "source": [
    "# OPTIONAL: compute some statistics\n",
    "\n",
    "<!-- ChronoStrain's database is specified by a single JSON file. In this notebook, it is given by the variable `CHRONOSTRAIN_TARGET_JSON`. The `JSONParser` object will attempt to parse the file, and extract the markers (each specified by a start and end position on the assembly) from the chromosomal assembly.\n",
    "\n",
    "The directory in which the extracted markers are stored is in `<data_dir>` (in this notebook it is given by the variable `CHRONOSTRAIN_DB_DIR`), or more specifically `<data_dir>/__<database_name>_/markers/<strain_id>/<marker_id>.fasta`.\n",
    "Note that the JSON generated by this notebook has the sequence FASTA paths embedded into the entries (`seqs` field of each strain).\n",
    "Without that field, ChronoStrain will attempt to download it from NCBI (by assuming that the sequence accession is an NCBI nucleotide acession).\n",
    "\n",
    "The marker extraction is disk I/O-bound. The database will be stored in binary via `pickle`, so that on future usages it will load faster. (try running this cell a second time and see the difference!) -->"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "22abb209-7d9f-4c4e-860d-b4cbbbd82e26",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-04-17 18:26:30,930 [INFO - chronostrain.database.database] - Instantiating database `ecoli`.\n",
      "# of E. coli entries: 2058\n"
     ]
    }
   ],
   "source": [
    "from chronostrain.database import JSONParser\n",
    "src_db = JSONParser(\n",
    "    entries_file=CHRONOSTRAIN_TARGET_JSON,\n",
    "    data_dir=CHRONOSTRAIN_DB_DIR,\n",
    "    marker_max_len=50000,\n",
    "    force_refresh=False\n",
    ").parse()\n",
    "\n",
    "n = 0\n",
    "for s in src_db.all_strains():\n",
    "    if s.metadata.genus == 'Escherichia' and s.metadata.species == 'coli':\n",
    "        n += 1\n",
    "print(\"# of E. coli entries:\", n)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "7696dca3-086d-4810-b6b5-fdce89b3b0dc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Database's E.coli marker fraction of genome: 0.015044251863146507 [mean]\n"
     ]
    }
   ],
   "source": [
    "n = 0\n",
    "ratios = []\n",
    "for s in src_db.all_strains():\n",
    "    if s.metadata.genus == 'Escherichia' and s.metadata.species == 'coli':\n",
    "        n += 1\n",
    "        marker_len = sum(len(m) for m in s.markers)\n",
    "        genome_len = s.metadata.total_len\n",
    "        ratios.append(\n",
    "            marker_len / genome_len\n",
    "        )\n",
    "print(\"Database's E.coli marker fraction of genome: {} [mean]\".format(np.mean(ratios)))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "f4cbc303-9a4f-4af4-a17e-86ba144522f3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# clusters = 2323\n",
      "# ecoli clusters = 842\n"
     ]
    }
   ],
   "source": [
    "df_entries = []\n",
    "with open(CHRONOSTRAIN_TARGET_CLUSTERS, \"rt\") as f:\n",
    "    for line in f:\n",
    "        if line.startswith(\"#\"):\n",
    "            continue\n",
    "        tokens = line.rstrip().split(\"\\t\")\n",
    "        rep = tokens[0]\n",
    "        members = tokens[1].split(\",\")\n",
    "        for member in members:\n",
    "            # if member.startswith(\"GCA\"):  # only include isolates\n",
    "            df_entries.append({'Accession': member, 'Cluster': rep})\n",
    "cluster_df = pd.DataFrame(df_entries)\n",
    "del df_entries\n",
    "print(\"# clusters = {}\".format(len(pd.unique(cluster_df['Cluster']))))\n",
    "\n",
    "\n",
    "n_efaecalis = 0\n",
    "for cluster_id in pd.unique(cluster_df['Cluster']):\n",
    "    s = src_db.get_strain(cluster_id)\n",
    "    if s.metadata.genus == 'Escherichia' and s.metadata.species == 'coli':\n",
    "        n_efaecalis += 1\n",
    "print(\"# ecoli clusters = {}\".format(n_efaecalis))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "chronostrain",
   "language": "python",
   "name": "chronostrain"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
