{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "8e311a3a-ed85-4b86-ac34-5c194ef42c82",
   "metadata": {},
   "source": [
    "# Database pipeline for quantification of E.coli from metagenomic data\n",
    "\n",
    "This notebook is the simplified version of `ecoli.ipynb` found in this same directory, using a small subset of markers (MLST genes only).\n",
    "\n",
    "**[IMPORTANT!!!] This notebook is NOT for constructing the database used in our paper. This only downloads _REFERENCE_ genomes, and _ONLY_ from the species \"Escherichia coli\" using NCBI datasets API. Please note that running analyses on metagenomic samples should (probably) use all available genomes at the Family-level.**\n",
    "\n",
    "## Prerequisites\n",
    "\n",
    "### Software requirements\n",
    "\n",
    "We recommend using a `conda` environment for this notebook.\n",
    "This notebook requires that the following software is installed.\n",
    "- chronostrain (python>=3.10, the basic recipe `conda_basic.yml` or the full recipe `conda_full.yml`)\n",
    "- dashing2 (2023 Baker and Langmead: https://github.com/dnbaker/dashing2)\n",
    "\n",
    "### Hardware requirements\n",
    "\n",
    "None of the operations of this notebook requires a GPU. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "b242412b-3585-437d-ae05-45078ed0c9a7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/youn/work/chronostrain/examples/database\n",
      "env: TARGET_TAXA=Escherichia coli\n",
      "env: NCBI_REFSEQ_DIR=ecoli_db_simple/ref_genomes\n",
      "env: REFSEQ_INDEX=ecoli_db_simple/ref_genomes/index.tsv\n",
      "env: PATH=/usr/bin:/usr/bin:/home/youn/mambaforge/envs/chronostrain2/bin:/home/youn/work/bin\n"
     ]
    }
   ],
   "source": [
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "\n",
    "\"\"\" ============================================ EDIT THESE SETTINGS BASED ON USER'S CHOICE. ============================================ \"\"\"\n",
    "\"\"\" RefSeq catalog settings\"\"\"\n",
    "TARGET_DIR = Path(\"./ecoli_db_simple\")  # the base directory for everything else.\n",
    "TARGET_TAXA = \"Escherichia coli\"  # the taxonomic identifier to download. Can be species, genus or even family.\n",
    "NCBI_REFSEQ_DIR = TARGET_DIR / \"ref_genomes\"\n",
    "REFSEQ_INDEX = NCBI_REFSEQ_DIR / \"index.tsv\"\n",
    "\n",
    "\"\"\" RefSeq BLAST database \"\"\"\n",
    "BLAST_DB_DIR = TARGET_DIR / \"blast_db\"  # The location of the Blast DB that you wish to create using RefSeq indices (they will be downloaded by this notebook).\n",
    "BLAST_DB_NAME = \"Enterobacteriaceae_refseq\"  # Blast DB to create.\n",
    "\n",
    "\"\"\" Marker seeds \"\"\"\n",
    "MARKER_SEED_DIR = TARGET_DIR / \"marker_seeds\"\n",
    "\n",
    "\"\"\" chronost#rain-specific settings \"\"\"\n",
    "NUM_CORES = 8  # number of cores to use (e.g. for blastn)\n",
    "MIN_PCT_IDTY = 75  # accept BLAST hits as markers above this threshold.\n",
    "CHRONOSTRAIN_DB_DIR = TARGET_DIR / \"chronostrain_files\"  # The directory to use for chronostrain's database files.\n",
    "CHRONOSTRAIN_TARGET_JSON = CHRONOSTRAIN_DB_DIR / \"ecoli.json\"  # the desired final product.\n",
    "CHRONOSTRAIN_TARGET_CLUSTERS = CHRONOSTRAIN_DB_DIR / \"ecoli.clusters.txt\"  # the clustering file.\n",
    "DASHING2_DIR = Path(\"/home/youn/work/bin\")  # Directory that contains the dashing2 executable.\n",
    "\n",
    "\"\"\" MetaPhlAn settings \"\"\"\n",
    "# METAPHLAN_DB_PATH = Path(\"/mnt/e/metaphlan_databases/mpa_vJan21_CHOCOPhlAnSGB_202103/mpa_vJan21_CHOCOPhlAnSGB_202103.pkl\") # MetaPhlan 3 or newer\n",
    "# METAPHLAN_TAXONOMIC_KEY = 's__Escherichia_coli'\n",
    "\n",
    "\"\"\" ============================================ DO NOT EDIT BELOW ============================================ \"\"\"\n",
    "\"\"\" environment variable extraction \"\"\"\n",
    "try:\n",
    "    VARS_SET\n",
    "except NameError:\n",
    "    VARS_SET = True\n",
    "    _cwd = %pwd\n",
    "    _parent_cwd = Path(_cwd).parent\n",
    "    _start_path = %env PATH\n",
    "\n",
    "# Work in parent directory, where all the helper scripts and settings.sh are.\n",
    "%cd \"$_parent_cwd\"\n",
    "%env TARGET_TAXA=$TARGET_TAXA\n",
    "%env NCBI_REFSEQ_DIR=$NCBI_REFSEQ_DIR\n",
    "%env REFSEQ_INDEX=$REFSEQ_INDEX\n",
    "# Need basic executables, such as \"which\" and \"basename\" (required by primersearch)\n",
    "%env PATH=/usr/bin:$_start_path:$DASHING2_DIR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "66052ced-e4e5-4ea7-9453-d577841dcf25",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "#Calling Dashing2 version v2.1.19 with command '/home/youn/work/chronostrain/examples/database/dashing2 --version'\n",
      "dashing2 has several subcommands: sketch, cmp, wsketch, and contain.\n",
      "Usage can be seen in those subcommands. (e.g., `dashing2 sketch -h`)\n",
      "\n",
      "\tsketch: converts FastX into k-mer sets/sketches, and sketches BigWig and BED files; also contains functionality from cmp, for one-step sketch and comparisons\n",
      "This is probably the most common subcommand to use.\n",
      "\n",
      "\tcmp: compares previously sketched/decomposed k-mer sets and emits results. alias: dist\n",
      "\n",
      "\tcontain: Takes a k-mer database (built with dashing2 sketch --save-kmers), then computes coverage for all k-mer references using input streams.\n",
      "\twsketch: Takes a tuple of [1-3] input binary files [(u32 or u64), (float or double), (u32 or u64)] and performs weighted minhash sketching.\n",
      "Three files are treated as Compressed Sparse Row (CSR)-format, where the third file contains indptr values, specifying the lengths of consecutive runs of pairs in the first two files corresponding to each row.\n",
      "wsketch is for sketching binary files which have already been summed, whereas sketch is for parsing and sketching (from Fast{qa}, BED, BigWig)\n",
      "\n",
      "\n",
      "Miscellania:\n",
      "printmin: Emit minimizer sequence sets in human-readable form.\n"
     ]
    }
   ],
   "source": [
    "!dashing2 --version"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "515078da-eddc-4843-9174-4af8d9a01a33",
   "metadata": {},
   "source": [
    "# Recipe starts here."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "b8627279-a477-44c8-be91-6d079da0bd6b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare directories.\n",
    "TARGET_DIR.mkdir(exist_ok=True, parents=True)\n",
    "BLAST_DB_DIR.mkdir(exist_ok=True, parents=True)\n",
    "NCBI_REFSEQ_DIR.mkdir(exist_ok=True, parents=True)\n",
    "MARKER_SEED_DIR.mkdir(exist_ok=True, parents=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd634593-16a7-4d8a-9662-1fe7cce9364a",
   "metadata": {},
   "source": [
    "### Step 1: Download chromosomal catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "569398fd-7c96-4c42-bdce-c8833f62558e",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2025-05-17 05:29:33,257 - Got reference only = True. Fetching reference genomes only.\n",
      "2025-05-17 05:29:33,257 - Handling taxid Escherichia coli\n",
      "2025-05-17 05:29:33,257 - [EXECUTE] datasets summary genome taxon \"Escherichia coli\" --assembly-source all --assembly-version latest --assembly-level complete --exclude-atypical --mag exclude --reference> ecoli_db_simple/ref_genomes/catalog.Escherichia_coli.json\n",
      "2025-05-17 05:29:33,542 - Found 2 raw records.\n",
      "2025-05-17 05:29:33,543 - [Escherichia coli] Found 2 unique strain records.\n",
      "SLEEPING -- Escherichia coli\n",
      "2025-05-17 05:29:34,544 - Downloading assemblies to directory ecoli_db_simple/ref_genomes\n",
      "[EXECUTE] datasets download genome accession --inputfile ecoli_db_simple/ref_genomes/__ncbi_input.txt --include genome,gff3 --assembly-version latest --filename ecoli_db_simple/ref_genomes/ncbi_dataset.zip\n",
      "Collecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [------------------------------------------------]   0% 0/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    4.38kB 13.3MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    18.3kB 989kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    18.3kB 989kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    18.3kB 989kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    32.8kB 773kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    32.8kB 773kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    65.5kB 920kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    65.5kB 920kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    65.5kB 920kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    98.3kB 975kB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    131kB 1.14MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    131kB 1.14MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    164kB 1.28MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    197kB 1.38MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    262kB 1.66MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    295kB 1.76MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    328kB 1.91MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    393kB 2.11MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    524kB 2.62MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    557kB 2.66MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    655kB 3.07MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    786kB 3.46MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    885kB 3.68MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    918kB 3.64MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.02MB 3.99MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.15MB 4.27MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.28MB 4.52MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.31MB 4.45MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.44MB 4.84MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.54MB 4.93MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.67MB 5.12MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.7MB 5.06MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.82MB 5.33MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.82MB 5.33MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.82MB 5.33MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.82MB 4.85MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    1.93MB 5.01MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.06MB 5.16MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.1MB 5.11MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.2MB 5.3MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.33MB 5.44MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.46MB 5.56MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.49MB 5.5MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.56MB 5.61MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.69MB 5.73MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.82MB 5.83MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.87MB 5.8MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    2.98MB 3.25MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.11MB 3.34MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.18MB 3.37MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.24MB 3.4MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.34MB 3.49MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.48MB 3.58MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.6MB 3.28MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.7MB 3.33MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.74MB 3.35MB/s\n",
      "\u001b[1A\u001b[2K\u001b[1A\u001b[2K\u001b[1A\u001b[2KCollecting 2 genome records [================================================] 100% 2/2\n",
      "Downloading: ecoli_db_simple/ref_genomes/ncbi_dataset.zip    3.83MB valid zip structure -- files not checked\n",
      "Validating package [================================================] 100% 8/8\n",
      "Indexing GCF_000008865.2 (Escherichia coli, Sakai substr. RIMD 0509952)\n",
      "Indexing GCF_000005845.2 (Escherichia coli, K-12 substr. MG1655)\n"
     ]
    }
   ],
   "source": [
    "!python python_helpers/download_ncbi.py -t \"Escherichia coli\" -d $NCBI_REFSEQ_DIR -o $REFSEQ_INDEX --level \"complete\" --reference-only"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2a83a65d-12de-4d0a-832b-c27a44fa6ebf",
   "metadata": {},
   "source": [
    "## Step 2: Download MLST markers."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "ddd1fe7e-70fb-4565-b030-827ed44413bd",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading marker seeds from MLST schema.\n",
      "Targeting 1 taxa using MLST scheme.\n",
      "Fetching URL resource https://pubmlst.org/static/data/dbases.xml\n",
      "Got a response of size 152.35 KB.\n",
      "Schema type id: 1\n",
      "Handling locus adk\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/adk/alleles_fasta\n",
      "Got a response of size 1021.48 KB.\n",
      "Handling locus fumC\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/fumC/alleles_fasta\n",
      "Got a response of size 1.19 MB.\n",
      "Handling locus gyrB\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/gyrB/alleles_fasta\n",
      "Got a response of size 752.67 KB.\n",
      "Handling locus icd\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/icd/alleles_fasta\n",
      "Got a response of size 1.05 MB.\n",
      "Handling locus mdh\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/mdh/alleles_fasta\n",
      "Got a response of size 740.53 KB.\n",
      "Handling locus purA\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/purA/alleles_fasta\n",
      "Got a response of size 710.46 KB.\n",
      "Handling locus recA\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_ecoli_achtman_seqdef/loci/recA/alleles_fasta\n",
      "Got a response of size 634.16 KB.\n",
      "Schema type id: 2\n",
      "Handling locus dinB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/dinB/alleles_fasta\n",
      "Got a response of size 142.16 KB.\n",
      "Handling locus icdA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/icdA/alleles_fasta\n",
      "Got a response of size 271.05 KB.\n",
      "Handling locus pabB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/pabB/alleles_fasta\n",
      "Got a response of size 141.63 KB.\n",
      "Handling locus polB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/polB/alleles_fasta\n",
      "Got a response of size 153.41 KB.\n",
      "Handling locus putP\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/putP/alleles_fasta\n",
      "Got a response of size 156.75 KB.\n",
      "Handling locus trpA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/trpA/alleles_fasta\n",
      "Got a response of size 212.16 KB.\n",
      "Handling locus trpB\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/trpB/alleles_fasta\n",
      "Got a response of size 198.41 KB.\n",
      "Handling locus uidA\n",
      "Fetching URL resource https://bigsdb.pasteur.fr/api/db/pubmlst_ecoli_seqdef/loci/uidA/alleles_fasta\n",
      "Got a response of size 229.61 KB.\n"
     ]
    }
   ],
   "source": [
    "!python python_helpers/mlst_download.py -t \"Escherichia coli\" -w \"$TARGET_DIR\"/mlst_schema -o \"$MARKER_SEED_DIR\"/mlst_seeds.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7d99db50-c3bb-45cf-8589-62f2c58dc8ee",
   "metadata": {},
   "source": [
    "## Step 3: Invoke `chronostrain make-db`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "5f075187-e006-40f2-ad9a-56ff2d6dcd86",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2025-05-17 05:21:49,732 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2025-05-17 05:21:50,401 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2025-05-17 05:21:50,489 [INFO - chronostrain.cli.make_db_json] - Using marker index: ecoli_db_simple/marker_seeds/mlst_seeds.tsv\n",
      "2025-05-17 05:21:50,491 [INFO - chronostrain.cli.make_db_json] - Adding strain index catalog: ecoli_db_simple/ref_genomes/index.tsv [2 entries; 1 species]\n",
      "2025-05-17 05:21:50,491 [INFO - chronostrain.cli.make_db_json] - Building raw DB using BLAST.\n",
      "2025-05-17 05:21:50,492 [INFO - chronostrain.cli.make_db_json] - Blast DB `Enterobacteriaceae_refseq` not found in ecoli_db_simple/blast_db. Running makeblastdb.\n",
      "2025-05-17 05:21:50,492 [INFO - chronostrain.cli.make_db_json] - Concatenating input files from 2 entries.\n",
      "2025-05-17 05:21:50,510 [DEBUG - chronostrain.util.external.commandline] - EXECUTE [cwd=ecoli_db_simple/blast_db] makeblastdb -in /home/youn/work/chronostrain/examples/database/ecoli_db_simple/blast_db/Enterobacteriaceae_refseq.fasta -out Enterobacteriaceae_refseq -dbtype nucl -title Enterobacteriaceae_refseq -parse_seqids\n",
      "2025-05-17 05:21:50,609 [DEBUG - chronostrain.util.external.commandline] - STDOUT: Building a new DB, current time: 05/17/2025 05:21:50\n",
      "New DB name:   /home/youn/work/chronostrain/examples/database/ecoli_db_simple/blast_db/Enterobacteriaceae_refseq\n",
      "New DB title:  Enterobacteriaceae_refseq\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 3000000000B\n",
      "Adding sequences from FASTA; added 2 sequences in 0.0290999 seconds.\n",
      "2025-05-17 05:21:50,609 [INFO - chronostrain.cli.make_db_json] - Cleaning up.\n",
      "2025-05-17 05:21:50,611 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on adk.\n",
      "2025-05-17 05:21:50,611 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_adk.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/adk.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,672 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fumC.\n",
      "2025-05-17 05:21:50,672 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_fumC.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/fumC.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,720 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gyrB.\n",
      "2025-05-17 05:21:50,720 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_gyrB.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/gyrB.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,773 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on icd.\n",
      "2025-05-17 05:21:50,773 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_icd.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/icd.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,826 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on mdh.\n",
      "2025-05-17 05:21:50,826 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_mdh.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/mdh.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,874 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on purA.\n",
      "2025-05-17 05:21:50,874 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_purA.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/purA.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,926 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on recA.\n",
      "2025-05-17 05:21:50,926 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_1_recA.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/recA.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:50,974 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dinB.\n",
      "2025-05-17 05:21:50,974 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_dinB.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/dinB.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,023 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on icdA.\n",
      "2025-05-17 05:21:51,023 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_icdA.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/icdA.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,079 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pabB.\n",
      "2025-05-17 05:21:51,079 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_pabB.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/pabB.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,138 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on polB.\n",
      "2025-05-17 05:21:51,138 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_polB.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/polB.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,192 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on putP.\n",
      "2025-05-17 05:21:51,192 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_putP.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/putP.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,245 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on trpA.\n",
      "2025-05-17 05:21:51,245 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_trpA.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/trpA.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,295 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on trpB.\n",
      "2025-05-17 05:21:51,296 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_trpB.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/trpB.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,348 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on uidA.\n",
      "2025-05-17 05:21:51,348 [DEBUG - chronostrain.util.external.commandline] - EXECUTE blastn -db Enterobacteriaceae_refseq -query /home/youn/work/chronostrain/examples/database/ecoli_db_simple/marker_seeds/Escherichia_coli_2_uidA.fasta -outfmt 6 saccver sstart send slen qstart qend evalue pident gaps qcovhsp -out ecoli_db_simple/chronostrain_files/_BLAST_ecoli/uidA.tsv -strand both -num_threads 8 -perc_identity 75 -max_target_seqs 20\n",
      "2025-05-17 05:21:51,398 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `adk` (1 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fumC` (2 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gyrB` (3 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `icd` (4 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `mdh` (5 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `purA` (6 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `recA` (7 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dinB` (8 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `icdA` (9 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pabB` (10 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `polB` (11 of 15)\n",
      "2025-05-17 05:21:51,400 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `putP` (12 of 15)\n",
      "2025-05-17 05:21:51,401 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `trpA` (13 of 15)\n",
      "2025-05-17 05:21:51,401 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `trpB` (14 of 15)\n",
      "2025-05-17 05:21:51,401 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `uidA` (15 of 15)\n",
      "2025-05-17 05:21:51,401 [INFO - chronostrain.cli.make_db_json] - Wrote raw blast DB entries to ecoli_db_simple/chronostrain_files/ecoli-1raw.json.\n",
      "2025-05-17 05:21:51,401 [INFO - chronostrain.cli.make_db_json] - Resolving overlaps.\n",
      "2025-05-17 05:21:51,401 [DEBUG - chronostrain.cli.make_db_json] - Src: ecoli_db_simple/chronostrain_files/ecoli-1raw.json, Dest: ecoli_db_simple/chronostrain_files/ecoli-2overlapmerged.json\n",
      "2025-05-17 05:21:51,415 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_002695.2, seqID NC_002695.2 [record index 0].\n",
      "2025-05-17 05:21:51,429 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_000913.3, seqID NC_000913.3 [record index 0].\n",
      "2025-05-17 05:21:51,429 [INFO - chronostrain.cli.make_db_json] - Database JSON file created: ecoli_db_simple/chronostrain_files/ecoli.json\n"
     ]
    }
   ],
   "source": [
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      make-db \\\n",
    "      -m \"$MARKER_SEED_DIR\"/mlst_seeds.tsv \\\n",
    "      -r $REFSEQ_INDEX \\\n",
    "      -b $BLAST_DB_NAME -bd $BLAST_DB_DIR \\\n",
    "      --min-pct-idty $MIN_PCT_IDTY \\\n",
    "      -o $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      --threads $NUM_CORES"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0c4c8dd-d461-4c78-a9e3-a1cb1024b634",
   "metadata": {},
   "source": [
    "# Clustering?\n",
    "\n",
    "Note that this example downloads just a few genomes (two E. coli genomes, as of May 2025) since we only are requesting reference E. coli genomes from NCBI.\n",
    "These two genomes are fairly different strains; and thus there is no need to cluster. \n",
    "\n",
    "However, if you DID want to cluster, you would do something like this:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "f4fd1c08-3346-4861-a383-08196100630b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2025-05-17 05:25:11,444 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2025-05-17 05:25:11,456 [INFO - chronostrain.cli.prune_json] - Pruning database via clustering\n",
      "2025-05-17 05:25:11,456 [DEBUG - chronostrain.cli.prune_json] - Src: ecoli_db_simple/chronostrain_files/ecoli.json, Output: ecoli_db_simple/chronostrain_files/ecoli.clusters.txt\n",
      "2025-05-17 05:25:11,456 [INFO - chronostrain.cli.prune_json] - Target identity threshold = 0.998\n",
      "2025-05-17 05:25:12,114 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2025-05-17 05:25:12,203 [INFO - chronostrain.cli.prune_json] - Preprocessing step -- Loading old DB instance, using data directory: ecoli_db_simple/chronostrain_files\n",
      "2025-05-17 05:25:12,203 [DEBUG - chronostrain.database.parser.json] - Couldn't find instance (ecoli_db_simple/chronostrain_files/__ecoli_/database.posix.pkl).\n",
      "2025-05-17 05:25:12,203 [INFO - chronostrain.database.parser.json] - Loading from JSON marker database file ecoli_db_simple/chronostrain_files/ecoli.json.\n",
      "2025-05-17 05:25:12,224 [DEBUG - chronostrain.database.parser.json] - Strain NC_002695.2 (Escherichia coli, Sakai substr. RIMD 0509952) loaded with 14 markers.\n",
      "2025-05-17 05:25:12,238 [DEBUG - chronostrain.database.parser.json] - Strain NC_000913.3 (Escherichia coli, K-12 substr. MG1655) loaded with 14 markers.\n",
      "2025-05-17 05:25:12,238 [INFO - chronostrain.database.database] - Instantiating database `ecoli`.\n",
      "2025-05-17 05:25:12,239 [DEBUG - chronostrain.database.parser.json] - Using `DictionaryBackend` for database backend.\n",
      "2025-05-17 05:25:12,239 [INFO - chronostrain.cli.prune_json] - Computing all-to-all Jaccard distance calculations.\n",
      "2025-05-17 05:25:12,239 [INFO - chronostrain.cli.prune_json] - Using ProbMinHash method for sketching (sz=8192).\n",
      "2025-05-17 05:25:12,272 [INFO - chronostrain.cli.prune_json] - Computing clusters.\n",
      "2025-05-17 05:25:12,276 [INFO - chronostrain.cli.prune_json] - Before clustering: 2 genomes\n",
      "2025-05-17 05:25:12,276 [INFO - chronostrain.cli.prune_json] - After clustering: 2 genomes\n",
      "2025-05-17 05:25:12,276 [DEBUG - chronostrain.cli.prune_json] - Cleaning up. Removing tmpdir = /home/youn/work/chronostrain/examples/database/ecoli_db_simple/chronostrain_files/__cluster_tmp\n"
     ]
    }
   ],
   "source": [
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      cluster-db \\\n",
    "      -i $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      -o $CHRONOSTRAIN_TARGET_CLUSTERS \\\n",
    "      --ident-threshold 0.998"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "chronostrain",
   "language": "python",
   "name": "chronostrain"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
