{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "7a07fb2ef7d1774e",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "# Database pipeline for quantification of E.faecalis from metagenomic data\n",
    "\n",
    "## What's in this notebook?\n",
    "\n",
    "This notebook implements a typical pipeline that might use to generate JSON configuration file (`efaecalis.json`) for chronostrain. Our database encodes marker-seed aligning sequences from _E.faecalis_, as well as all sequences of >75% similarity known to us at the family level.\n",
    "\n",
    "In principle, the JSON file is the only thing needed to run ChronoStrain; but this notebook has some other by-products:\n",
    "\n",
    "1. A repository of chromosomal assemblies for the Enterococcaceae family, with a TSV-formatted index (`index.tsv`)\n",
    "2. A repository of marker seed sequences and associated gene names.\n",
    "3. (Optional) A binary-stored instance of the JSON database, readable by `chronostrain`. \n",
    "\n",
    "If the sequence records listed in the JSON file do not exist (the directory is specificable by a configuration file `chronostrain.ini`), then ChronoStrain will attempt to download it by searching the ID on NCBI. This notebook creates symbolic links to the sequences found in the chromosomal assembly index (byproduct #1) to skip this process.\n",
    "After these sequences are downloaded, the database is loaded by extracting subsequences from the FASTA records; this is a very I/O-bound task and thus the pre-loaded binary (byproduct #3) helps skip the initialization.\n",
    "\n",
    "## Prerequisites\n",
    "\n",
    "We recommend using a `conda` environment for this notebook, with `ipywidgets` installed and updated. None of the operations of this notebook requires a GPU.\n",
    "This notebook requires that the following software is installed.\n",
    "- chronostrain (python>=3.10, the basic recipe `conda_basic.yml` or the full recipe `conda_full.yml`)\n",
    "- primersearch (http://emboss.open-bio.org/, https://anaconda.org/bioconda/emboss)\n",
    "- dashing2 (2023 Baker and Langmead: https://github.com/dnbaker/dashing2)\n",
    "\n",
    "### Hardware requirements\n",
    " \n",
    "None of the operations of this notebook requires a GPU. \n",
    "As of Aug 2023, we estimate that the contents of this notebook requires ~20 GB of hard disk space. \n",
    "At the time that we ran this pipeline, the catalog of chromosomal assemblies totalled 14.3 GB.\n",
    "Other files (such as the BLAST database, marker seeds and chronostrain-specific byproducts) totalled 5.4 GB, with a peak of ~28 GB when accounting for temporary files."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2246f795-a67f-4524-8b24-b9962e04a908",
   "metadata": {
    "collapsed": false
   },
   "source": [
    "## File paths and environment variables"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "3a7a4e2f-7823-4e2a-b046-bbd897a65e4d",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2023-09-28T13:57:12.606748999Z",
     "start_time": "2023-09-28T13:57:12.432061566Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/youn/work/chronostrain/examples/database\n",
      "env: JAX_PLATFORM_NAME=cpu\n",
      "env: TARGET_TAXA=Enterococcaceae\n",
      "env: NCBI_REFSEQ_DIR=/mnt/e/infant_nt/database/ref_genomes\n",
      "env: REFSEQ_INDEX=/mnt/e/infant_nt/database/ref_genomes/index.tsv\n",
      "env: PATH=/usr/bin:/home/youn/mambaforge/envs/chronostrain2/bin:/home/youn/work/bin\n"
     ]
    }
   ],
   "source": [
    "from pathlib import Path\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "from typing import *\n",
    "\n",
    "from Bio import SeqIO\n",
    "from Bio.Seq import Seq\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "\n",
    "\"\"\" ============================================ EDIT THESE SETTINGS BASED ON USER'S CHOICE. ============================================ \"\"\"\n",
    "\"\"\" RefSeq catalog settings\"\"\"\n",
    "TARGET_DIR = Path(\"/mnt/e/efaecalis_db\")  # the base directory for everything else.\n",
    "TARGET_TAXA = \"Enterococcaceae\"  # the taxonomic identifier to download. Can be species, genus or even family.\n",
    "NCBI_REFSEQ_DIR = TARGET_DIR / \"ref_genomes\"\n",
    "REFSEQ_INDEX = NCBI_REFSEQ_DIR / \"index.tsv\"\n",
    "\n",
    "\"\"\" RefSeq BLAST database \"\"\"\n",
    "BLAST_DB_DIR = TARGET_DIR / \"blast_db\"  # The location of the Blast DB that you wish to create using RefSeq indices (they will be downloaded by this notebook).\n",
    "BLAST_DB_NAME = \"Efaecalis_refseq\"  # Blast DB to create.\n",
    "\n",
    "\"\"\" Marker seeds \"\"\"\n",
    "MARKER_SEED_DIR = TARGET_DIR / \"marker_seeds\"\n",
    "MARKER_SEED_INDEX = MARKER_SEED_DIR / \"marker_seed_index.tsv\"\n",
    "\n",
    "\"\"\" chronostrain-specific settings \"\"\"\n",
    "NUM_CORES = 8  # number of cores to use (e.g. for blastn)\n",
    "MIN_PCT_IDTY = 75  # accept BLAST hits as markers above this threshold.\n",
    "CHRONOSTRAIN_DB_DIR = TARGET_DIR / \"chronostrain_files\"  # The directory to use for chronostrain's database files.\n",
    "CHRONOSTRAIN_TARGET_JSON = CHRONOSTRAIN_DB_DIR / \"efaecalis.json\"  # the desired final product.\n",
    "CHRONOSTRAIN_TARGET_CLUSTERS = CHRONOSTRAIN_DB_DIR / \"efaecalis.clusters.txt\"  # the clustering file.\n",
    "DASHING2_DIR = Path(\"/home/youn/work/bin\")  # Directory that contains the dashing2 executable.\n",
    "\n",
    "\"\"\" MetaPhlAn settings \"\"\"\n",
    "METAPHLAN_DB_PATH = Path(\"/mnt/e/metaphlan_databases/mpa_vJan21_CHOCOPhlAnSGB_202103/mpa_vJan21_CHOCOPhlAnSGB_202103.pkl\") # MetaPhlan 3 or newer\n",
    "METAPHLAN_TAXONOMIC_KEY = 's__Enterococcus_faecalis'\n",
    "\n",
    "\n",
    "\"\"\" ============================================ DO NOT EDIT BELOW ============================================ \"\"\"\n",
    "\"\"\" environment variable extraction \"\"\"\n",
    "try:\n",
    "    VARS_SET\n",
    "except NameError:\n",
    "    VARS_SET = True\n",
    "    _cwd = %pwd\n",
    "    _parent_cwd = Path(_cwd).parent\n",
    "    _start_path = %env PATH\n",
    "\n",
    "# Work in parent directory, where all the helper scripts and settings.sh are.\n",
    "%cd \"$_parent_cwd\"\n",
    "# Don't use GPU when importing jaxlib through chronostrain.\n",
    "%env JAX_PLATFORM_NAME=cpu  \n",
    "%env TARGET_TAXA=$TARGET_TAXA\n",
    "%env NCBI_REFSEQ_DIR=$NCBI_REFSEQ_DIR\n",
    "%env REFSEQ_INDEX=$REFSEQ_INDEX\n",
    "# Need basic executables, such as \"which\" and \"basename\" (required by primersearch)\n",
    "%env PATH=/usr/bin:$_start_path:$DASHING2_DIR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "fe9ac0c8-1267-4d81-a72d-43384889c37f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "checking EMBOSS primersearch.\n",
      "EMBOSS:6.6.0.0\n",
      "\n",
      "checking dashing2.\n",
      "#Calling Dashing2 version v2.1.19 with command '/home/youn/work/chronostrain/examples/database/dashing2 --version'\n",
      "dashing2 has several subcommands: sketch, cmp, wsketch, and contain.\n",
      "Usage can be seen in those subcommands. (e.g., `dashing2 sketch -h`)\n",
      "\n",
      "\tsketch: converts FastX into k-mer sets/sketches, and sketches BigWig and BED files; also contains functionality from cmp, for one-step sketch and comparisons\n",
      "This is probably the most common subcommand to use.\n",
      "\n",
      "\tcmp: compares previously sketched/decomposed k-mer sets and emits results. alias: dist\n",
      "\n",
      "\tcontain: Takes a k-mer database (built with dashing2 sketch --save-kmers), then computes coverage for all k-mer references using input streams.\n",
      "\twsketch: Takes a tuple of [1-3] input binary files [(u32 or u64), (float or double), (u32 or u64)] and performs weighted minhash sketching.\n",
      "Three files are treated as Compressed Sparse Row (CSR)-format, where the third file contains indptr values, specifying the lengths of consecutive runs of pairs in the first two files corresponding to each row.\n",
      "wsketch is for sketching binary files which have already been summed, whereas sketch is for parsing and sketching (from Fast{qa}, BED, BigWig)\n",
      "\n",
      "\n",
      "Miscellania:\n",
      "printmin: Emit minimizer sequence sets in human-readable form.\n",
      "\n",
      "checking pgap.\n",
      "PGAP version 2023-10-03.build7061 is up to date.\n"
     ]
    }
   ],
   "source": [
    "# === Ensure that these commands work.\n",
    "print(\"checking EMBOSS primersearch.\")\n",
    "!primersearch --version\n",
    "\n",
    "print(\"\\nchecking dashing2.\")\n",
    "!dashing2 --version\n",
    "\n",
    "print(\"\\nchecking pgap.\")\n",
    "!pgap --version"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "adda6630-d7c5-4bd7-ac66-e5d1fc8dbed8",
   "metadata": {},
   "source": [
    "## Recipe starts here."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0f279e6f-8d09-4f7e-86f2-6d4e5eb0b478",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Prepare directories.\n",
    "TARGET_DIR.mkdir(exist_ok=True, parents=True)\n",
    "BLAST_DB_DIR.mkdir(exist_ok=True, parents=True)\n",
    "NCBI_REFSEQ_DIR.mkdir(exist_ok=True, parents=True)\n",
    "MARKER_SEED_DIR.mkdir(exist_ok=True, parents=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e85afcc9-2976-4a29-a7e6-05725c3c45d8",
   "metadata": {},
   "source": [
    "### Step 1: Download RefSeq catalog."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "29a46a3d-29bf-4848-b6cf-44b9f51ceab0",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2023-10-03 13:05:22,082 [INFO - chronostrain.download_ncbi] - Found 2197 records.\n",
      "2023-10-03 13:05:22,082 [INFO - chronostrain.download_ncbi] - Downloading assemblies to directory /mnt/e/infant_nt/database/ref_genomes\n",
      "Indexing GCF_003711125.1 (FDAARGOS_184)\n",
      "Indexing GCF_005347505.1 (352)\n",
      "Indexing GCF_013394325.1 (G-15)\n",
      "Indexing GCA_003711125.1 (FDAARGOS_184)\n",
      "Indexing GCA_005347505.1 (352)\n",
      "Indexing GCA_013394325.1 (G-15)\n",
      "Indexing GCF_003641225.1 (EC-369)\n",
      "Indexing GCF_009707345.1 (EC291)\n",
      "Indexing GCF_014844215.1 (EGM182)\n",
      "Indexing GCF_016127635.1 (FDAARGOS_998)\n",
      "Indexing GCF_016727305.1 (FDAARGOS_1122)\n",
      "Indexing GCF_016727325.1 (FDAARGOS_1121)\n",
      "Indexing GCF_016727345.1 (FDAARGOS_1120)\n",
      "Indexing GCF_022870765.1 (ECB140)\n",
      "Indexing GCF_023523715.1 (SP11)\n",
      "Indexing GCF_027944555.1 (CQFYY22-063)\n",
      "Indexing GCF_029201225.1 (ASE2)\n",
      "Indexing GCF_029215545.1 (ASE4)\n",
      "Indexing GCA_003641225.1 (EC-369)\n",
      "Indexing GCA_009707345.1 (EC291)\n",
      "Indexing GCA_014844215.1 (EGM182)\n",
      "Indexing GCA_016127635.1 (FDAARGOS_998)\n",
      "Indexing GCA_016727305.1 (FDAARGOS_1122)\n",
      "Indexing GCA_016727325.1 (FDAARGOS_1121)\n",
      "Indexing GCA_016727345.1 (FDAARGOS_1120)\n",
      "Indexing GCA_022870765.1 (ECB140)\n",
      "Indexing GCA_023523715.1 (SP11)\n",
      "Indexing GCA_027944555.1 (CQFYY22-063)\n",
      "Indexing GCA_029201225.1 (ASE2)\n",
      "Indexing GCA_029215545.1 (ASE4)\n",
      "Indexing GCF_000157355.2 (EC20)\n",
      "Indexing GCA_000157355.2 (EC20)\n",
      "Indexing GCF_900474605.1 (NCTC12421)\n",
      "Indexing GCF_947054945.1 (CIRMBP-1281)\n",
      "Indexing GCF_947054965.1 (CIRMBP-1246)\n",
      "Indexing GCF_947055055.1 (CIRMBP-1261)\n",
      "Indexing GCF_947055095.1 (CIRMBP-1212)\n",
      "Indexing GCF_947055105.1 (CIRMBP-1274)\n",
      "Indexing GCF_947055185.1 (CIRMBP-1228)\n",
      "Indexing GCF_947055195.1 (CIRMBP-1283)\n",
      "Indexing GCF_947055595.1 (CIRMBP-1302)\n",
      "Indexing GCF_947055965.1 (CIRMBP-1287)\n",
      "Indexing GCA_900474605.1 (NCTC12421)\n",
      "Indexing GCA_947054945.1 (CIRMBP-1281)\n",
      "Indexing GCA_947054965.1 (CIRMBP-1246)\n",
      "Indexing GCA_947055055.1 (CIRMBP-1261)\n",
      "Indexing GCA_947055095.1 (CIRMBP-1212)\n",
      "Indexing GCA_947055105.1 (CIRMBP-1274)\n",
      "Indexing GCA_947055185.1 (CIRMBP-1228)\n",
      "Indexing GCA_947055195.1 (CIRMBP-1283)\n",
      "Indexing GCA_947055595.1 (CIRMBP-1302)\n",
      "Indexing GCA_947055965.1 (CIRMBP-1287)\n",
      "Indexing GCF_028335365.1 (K205-4b)\n",
      "Indexing GCA_028335365.1 (K205-4b)\n",
      "Indexing GCF_002277935.1 (BDGP3)\n",
      "Indexing GCF_001267395.1 (KLDS 6.0933)\n",
      "Indexing GCF_001267865.1 (KLDS 6.0930)\n",
      "Indexing GCF_015999505.1 (KCTC 13289)\n",
      "Indexing GCF_023375285.1 (S8-3)\n",
      "Indexing GCF_028335325.1 (R59-1)\n",
      "Indexing GCF_028335645.1 (R109-1)\n",
      "Indexing GCA_001267395.1 (KLDS6.0933)\n",
      "Indexing GCA_001267865.1 (KLDS 6.0930)\n",
      "Indexing GCA_002277935.1 (BDGP3)\n",
      "Indexing GCA_015999505.1 (KCTC 13289)\n",
      "Indexing GCA_023375285.1 (S8-3)\n",
      "Indexing GCA_028335325.1 (R59-1)\n",
      "Indexing GCA_028335645.1 (R109-1)\n",
      "Indexing GCF_001598635.1 (LD33)\n",
      "Indexing GCF_001689055.2 (KB1)\n",
      "Indexing GCF_001989555.1 (CLB21560)\n",
      "Indexing GCF_002163735.1 (sorialis)\n",
      "Indexing GCF_002208945.1 (FDAARGOS_338)\n",
      "Indexing GCF_002289045.2 (H25)\n",
      "Indexing GCF_002355755.1 (W11)\n",
      "Indexing GCF_003030425.1 (FDAARGOS_324)\n",
      "Indexing GCF_003345275.1 (TY1)\n",
      "Indexing GCF_003812665.1 (FDAARGOS_528)\n",
      "Indexing GCF_003966385.1 (KUB3006)\n",
      "Indexing GCF_003966405.1 (KUB3007)\n",
      "Indexing GCF_004332015.1 (CVM N48037F)\n",
      "Indexing GCF_004332035.1 (CVM N60443F)\n",
      "Indexing GCF_005158025.1 (FC)\n",
      "Indexing GCF_005484525.1 (SGAir0397)\n",
      "Indexing GCF_005979285.1 (C25)\n",
      "Indexing GCF_005979305.1 (C54)\n",
      "Indexing GCF_006349345.1 (HA-1)\n",
      "Indexing GCF_006364815.1 (FDAARGOS_611)\n",
      "Indexing GCF_006494835.1 (VE14089)\n",
      "Indexing GCF_006494855.1 (VE18379)\n",
      "Indexing GCF_006494875.1 (VE18395)\n",
      "Indexing GCF_006766145.1 (DM01)\n",
      "Indexing GCF_009498175.3 (L8)\n",
      "Indexing GCF_010103655.1 (JY32)\n",
      "Indexing GCF_011193375.1 (ES-1)\n",
      "Indexing GCF_012594215.1 (TH4125)\n",
      "Indexing GCF_013344605.1 (133170041-3)\n",
      "Indexing GCF_013344625.1 (111540047-1)\n",
      "Indexing GCF_013344645.1 (111540027-2)\n",
      "Indexing GCF_013344665.1 (092160007-3)\n",
      "Indexing GCF_014489455.1 (SF28073)\n",
      "Indexing GCF_014844235.1 (EGM183)\n",
      "Indexing GCF_015377745.1 (TK-P4B)\n",
      "Indexing GCF_016743895.1 (T90-1)\n",
      "Indexing GCF_016812095.1 (L11)\n",
      "Indexing GCF_017280035.1 (L13)\n",
      "Indexing GCF_017280235.1 (L18)\n",
      "Indexing GCF_017357805.1 (BFF1B1)\n",
      "Indexing GCF_017639585.1 (EF-2001)\n",
      "Indexing GCF_018517125.1 (18-243)\n",
      "Indexing GCF_018986755.2 (1207/14)\n",
      "Indexing GCF_019195415.1 (SY-1)\n",
      "Indexing GCF_019222705.1 (NS2)\n",
      "Indexing GCF_019774535.1 (E006xJH2-2-TC1)\n",
      "Indexing GCF_019856595.1 (E006)\n",
      "Indexing GCF_020616775.1 (EFS17)\n",
      "Indexing GCF_020616875.1 (EFS36)\n",
      "Indexing GCF_020616935.1 (EFS108)\n",
      "Indexing GCF_020844125.1 (AR_0780)\n",
      "Indexing GCF_020881975.1 (R5)\n",
      "Indexing GCF_020882035.1 (E512)\n",
      "Indexing GCF_020882055.1 (E512-TC2)\n",
      "Indexing GCF_020882075.1 (E509)\n",
      "Indexing GCF_020882095.1 (E512-TC1)\n",
      "Indexing GCF_020882115.1 (Fac74)\n",
      "Indexing GCF_020906605.1 (E533)\n",
      "Indexing GCF_021484825.1 (PCH555)\n",
      "Indexing GCF_021610105.1 (UK045)\n",
      "Indexing GCF_021655635.1 (K-4)\n",
      "Indexing GCF_022212845.1 (152)\n",
      "Indexing GCF_022212865.1 (143-1)\n",
      "Indexing GCF_022212885.1 (142-1)\n",
      "Indexing GCF_022212905.1 (133-1)\n",
      "Indexing GCF_022212925.1 (119-2)\n",
      "Indexing GCF_022212945.1 (119-1)\n",
      "Indexing GCF_022212965.1 (101-1)\n",
      "Indexing GCF_022212985.1 (59)\n",
      "Indexing GCF_022213005.1 (43-2)\n",
      "Indexing GCF_022213025.1 (2-1)\n",
      "Indexing GCF_022557275.1 (LJX 0909)\n",
      "Indexing GCF_022648285.1 (UAMS_EL56)\n",
      "Indexing GCF_022648425.1 (UAMS_EL54)\n",
      "Indexing GCF_022648595.1 (UAMS_EL53)\n",
      "Indexing GCF_022691465.1 (VSE-WC032)\n",
      "Indexing GCF_022691485.1 (VRE-WC031)\n",
      "Indexing GCF_022699585.1 (NY13321)\n",
      "Indexing GCF_022810805.1 (RF_1_1_25_1)\n",
      "Indexing GCF_022869705.1 (PartL-Efaecalis-RM8376)\n",
      "Indexing GCF_023182775.2 (AKSZ-208)\n",
      "Indexing GCF_023299225.1 (AT49a)\n",
      "Indexing GCF_023299425.1 (AT48)\n",
      "Indexing GCF_023299465.1 (LS05-1)\n",
      "Indexing GCF_023299485.1 (LS05-2)\n",
      "Indexing GCF_023299505.1 (AT40b)\n",
      "Indexing GCF_023299525.1 (LS06-1)\n",
      "Indexing GCF_023299545.1 (AT09)\n",
      "Indexing GCF_023299585.1 (AT40a)\n",
      "Indexing GCF_023299605.1 (AT46a)\n",
      "Indexing GCF_023299625.1 (AT39)\n",
      "Indexing GCF_023299645.1 (AT50)\n",
      "Indexing GCF_023299665.1 (AT29)\n",
      "Indexing GCF_023299685.1 (AT22)\n",
      "Indexing GCF_023299705.1 (AT34)\n",
      "Indexing GCF_023299785.1 (AT43a)\n",
      "Indexing GCF_023300025.1 (AT41)\n",
      "Indexing GCF_023300165.1 (AT04)\n",
      "Indexing GCF_023375245.1 (S11-6)\n",
      "Indexing GCF_023375265.1 (S39-4)\n",
      "Indexing GCF_023375465.1 (661)\n",
      "Indexing GCF_023375485.1 (732)\n",
      "Indexing GCF_023375505.1 (249)\n",
      "Indexing GCF_023375525.1 (705)\n",
      "Indexing GCF_023375545.1 (1521)\n",
      "Indexing GCF_023523795.1 (FMAMOB6)\n",
      "Indexing GCF_023612035.1 (M817)\n",
      "Indexing GCF_023650735.1 (QZ076)\n",
      "Indexing GCF_023702145.1 (CQ025)\n",
      "Indexing GCF_023746575.1 (M9)\n",
      "Indexing GCF_024267595.1 (Chr-JH 2-2)\n",
      "Indexing GCF_024579755.1 (JF3A-223)\n",
      "Indexing GCF_024579795.1 (JF3A-4253)\n",
      "Indexing GCF_024927865.1 (SJ82)\n",
      "Indexing GCF_025957845.1 (BE70)\n",
      "Indexing GCF_025957925.1 (BE69)\n",
      "Indexing GCF_025958025.1 (BE52)\n",
      "Indexing GCF_025958065.1 (BE47)\n",
      "Indexing GCF_025958085.1 (BE45)\n",
      "Indexing GCF_025958125.1 (BE43)\n",
      "Indexing GCF_025958325.1 (BE33)\n",
      "Indexing GCF_025958445.1 (BE32)\n",
      "Indexing GCF_025958625.1 (BE25)\n",
      "Indexing GCF_025958785.1 (BE17)\n",
      "Indexing GCF_025958805.1 (BE16)\n",
      "Indexing GCF_025958825.1 (BE15)\n",
      "Indexing GCF_025958845.1 (BE13)\n",
      "Indexing GCF_025958865.1 (BE11)\n",
      "Indexing GCF_025958885.1 (BE8)\n",
      "Indexing GCF_025958905.1 (BE7)\n",
      "Indexing GCF_025958925.1 (BE5)\n",
      "Indexing GCF_026650885.1 (FA2-2)\n",
      "Indexing GCF_026802365.1 (T30)\n",
      "Indexing GCF_026802385.1 (M61)\n",
      "Indexing GCF_027942035.1 (ESC1)\n",
      "Indexing GCF_028131645.1 (SVR2330)\n",
      "Indexing GCF_028335305.1 (K198-1)\n",
      "Indexing GCF_028335345.1 (K190-1)\n",
      "Indexing GCF_028335425.1 (K70-12a)\n",
      "Indexing GCF_028335445.1 (K137-1a)\n",
      "Indexing GCF_028335505.1 (K194-1)\n",
      "Indexing GCF_028335545.1 (K72-1)\n",
      "Indexing GCF_028335665.1 (K80-2b)\n",
      "Indexing GCF_028335685.1 (K188-1)\n",
      "Indexing GCF_028335705.1 (K205-3)\n",
      "Indexing GCF_028335725.1 (K191-1)\n",
      "Indexing GCF_028532705.1 (DM86)\n",
      "Indexing GCF_028598965.2 (SVJ-EF01)\n",
      "Indexing GCF_028621735.1 (CQJXZ21-077)\n",
      "Indexing GCF_028621755.1 (CQJXZ21-076)\n",
      "Indexing GCF_028621775.1 (CQNBZ21-135)\n",
      "Indexing GCF_028743535.1 (B-537)\n",
      "Indexing GCF_029000425.1 (W5)\n",
      "Indexing GCF_029011255.1 (VSI48)\n",
      "Indexing GCF_029024925.1 (DSM 20478)\n",
      "Indexing GCF_029167565.1 (GENOMIC22-006)\n",
      "Indexing GCF_029278545.1 (3143)\n",
      "Indexing GCF_029866805.1 (NY13412)\n",
      "Indexing GCF_029910175.1 (NY13410)\n",
      "Indexing GCF_030020155.1 (EfsPNK7)\n",
      "Indexing GCF_030020175.1 (EfsPF15)\n",
      "Indexing GCF_030020195.1 (EfsPF13)\n",
      "Indexing GCF_030020215.1 (EfsPF5)\n",
      "Indexing GCF_030020235.1 (EfsPF2)\n",
      "Indexing GCF_030020255.1 (EfsC130)\n",
      "Indexing GCF_030020425.1 (EfsPF20)\n",
      "Indexing GCF_030020455.1 (EfsC123)\n",
      "Indexing GCF_030020475.1 (EfsC116)\n",
      "Indexing GCF_030020495.1 (EfsC109-2)\n",
      "Indexing GCF_030020515.1 (EfsC108)\n",
      "Indexing GCF_030020535.1 (EfsC94)\n",
      "Indexing GCF_030020555.1 (EfsC85)\n",
      "Indexing GCF_030020575.1 (EfsC61)\n",
      "Indexing GCF_030020595.1 (EfsC49)\n",
      "Indexing GCF_030020615.1 (EfsC33)\n",
      "Indexing GCF_030020645.1 (EfsC20)\n",
      "Indexing GCF_030020665.1 (EfsC17)\n",
      "Indexing GCF_030020685.1 (EfsC12)\n",
      "Indexing GCF_030020705.1 (EfsC11)\n",
      "Indexing GCF_030020725.1 (EfsC9)\n",
      "Indexing GCF_030020745.1 (EfsC8)\n",
      "Indexing GCF_030064715.1 (AFL-109F)\n",
      "Indexing GCF_030264755.1 (90_2023)\n",
      "Indexing GCF_030295445.1 (JARB-HU0796)\n",
      "Indexing GCF_030295805.1 (L6D)\n",
      "Indexing GCF_030295845.1 (N4E)\n",
      "Indexing GCF_030376825.1 (RE25)\n",
      "Indexing GCF_030512425.1 (HL1)\n",
      "Indexing GCF_030517125.1 (EfsPF36)\n",
      "Indexing GCF_030517355.1 (EfsPF27)\n",
      "Indexing GCF_030517385.1 (EfsPF23)\n",
      "Indexing GCF_030517415.1 (EfsC129)\n",
      "Indexing GCF_030517445.1 (EfsC43)\n",
      "Indexing GCF_030517475.1 (EfsC29)\n",
      "Indexing GCF_030517505.1 (EfsC23)\n",
      "Indexing GCF_030517535.1 (EfsPF24-2)\n",
      "Indexing GCF_030517565.1 (EfsC1)\n",
      "Indexing GCF_030517625.1 (EfsC13)\n",
      "Indexing GCF_031186115.1 (JF4A2234)\n",
      "Indexing GCF_031206965.1 (N775_PC7)\n",
      "Indexing GCF_031208705.1 (N775_PC2)\n",
      "Indexing GCF_900636775.1 (NCTC8745)\n",
      "Indexing GCF_901553725.1 (NCTC8732)\n",
      "Indexing GCF_902379005.2 (MGYG-HGUT-01694)\n",
      "Indexing GCF_905120545.1 (26975_1#66)\n",
      "Indexing GCF_905120555.1 (26975_1#5)\n",
      "Indexing GCF_905120565.1 (26975_1#52)\n",
      "Indexing GCF_905120575.1 (26975_2#148)\n",
      "Indexing GCF_905120585.1 (26975_1#18)\n",
      "Indexing GCF_905120595.1 (26975_1#11)\n",
      "Indexing GCF_905120605.1 (26975_1#45)\n",
      "Indexing GCF_905120615.1 (26975_1#33)\n",
      "Indexing GCF_905120625.1 (26975_1#32)\n",
      "Indexing GCF_905120635.1 (26975_2#150)\n",
      "Indexing GCF_905120645.1 (26975_1#37)\n",
      "Indexing GCF_905120655.1 (26975_1#13)\n",
      "Indexing GCF_905120665.1 (26975_1#48)\n",
      "Indexing GCF_905120675.1 (26975_2#147)\n",
      "Indexing GCF_905120685.1 (26975_2#145)\n",
      "Indexing GCF_905120695.1 (26975_1#25)\n",
      "Indexing GCF_905120705.1 (26975_1#12)\n",
      "Indexing GCF_905120715.1 (26975_1#22)\n",
      "Indexing GCF_905120725.1 (26975_1#31)\n",
      "Indexing GCF_905120735.1 (26975_1#63)\n",
      "Indexing GCF_905120745.1 (26975_1#105)\n",
      "Indexing GCF_905120755.1 (26975_1#88)\n",
      "Indexing GCF_905120765.1 (26975_1#107)\n",
      "Indexing GCF_905120775.1 (26975_1#367)\n",
      "Indexing GCF_905120785.1 (26975_1#70)\n",
      "Indexing GCF_905120795.1 (26975_1#74)\n",
      "Indexing GCF_905120805.1 (26975_1#132)\n",
      "Indexing GCF_905120815.1 (26975_1#129)\n",
      "Indexing GCF_905120825.1 (26975_1#145)\n",
      "Indexing GCF_905120835.1 (26975_1#118)\n",
      "Indexing GCF_905120845.1 (26975_1#111)\n",
      "Indexing GCF_905120855.1 (26975_1#117)\n",
      "Indexing GCF_905120865.1 (26975_2#152)\n",
      "Indexing GCF_905120875.1 (26975_1#89)\n",
      "Indexing GCF_905120885.1 (26975_1#160)\n",
      "Indexing GCF_905120895.1 (26975_2#162)\n",
      "Indexing GCF_905120905.1 (26975_2#165)\n",
      "Indexing GCF_905120915.1 (26975_1#146)\n",
      "Indexing GCF_905120925.1 (26975_1#168)\n",
      "Indexing GCF_905120935.1 (26975_2#160)\n",
      "Indexing GCF_905120945.1 (26975_2#190)\n",
      "Indexing GCF_905120955.1 (26975_2#198)\n",
      "Indexing GCF_905120965.1 (26975_2#200)\n",
      "Indexing GCF_905120975.1 (26975_2#184)\n",
      "Indexing GCF_905120985.1 (26975_2#204)\n",
      "Indexing GCF_905120995.1 (26975_2#138)\n",
      "Indexing GCF_905121005.1 (26975_2#195)\n",
      "Indexing GCF_905121015.1 (26975_2#191)\n",
      "Indexing GCF_905121025.1 (26975_2#201)\n",
      "Indexing GCF_905121035.1 (26975_2#180)\n",
      "Indexing GCF_905121045.1 (26975_1#180)\n",
      "Indexing GCF_905121055.1 (26975_1#182)\n",
      "Indexing GCF_905121065.1 (26975_1#366)\n",
      "Indexing GCF_905121075.1 (26975_2#175)\n",
      "Indexing GCF_905121085.1 (26975_2#206)\n",
      "Indexing GCF_905121095.1 (26975_2#221)\n",
      "Indexing GCF_905121105.1 (26975_2#225)\n",
      "Indexing GCF_905121115.1 (26975_1#188)\n",
      "Indexing GCF_905121125.1 (26975_1#194)\n",
      "Indexing GCF_905121135.1 (26975_1#236)\n",
      "Indexing GCF_905121145.1 (26975_1#253)\n",
      "Indexing GCF_905121155.1 (26975_1#276)\n",
      "Indexing GCF_905121165.1 (26975_1#268)\n",
      "Indexing GCF_905121175.1 (26975_1#228)\n",
      "Indexing GCF_905121185.1 (26975_1#214)\n",
      "Indexing GCF_905121195.1 (26975_1#279)\n",
      "Indexing GCF_905121205.1 (26975_1#246)\n",
      "Indexing GCF_905121215.1 (26975_1#280)\n",
      "Indexing GCF_905121225.1 (26975_1#283)\n",
      "Indexing GCF_905121235.1 (26975_1#263)\n",
      "Indexing GCF_905121245.1 (26975_1#241)\n",
      "Indexing GCF_905121255.1 (26975_1#262)\n",
      "Indexing GCF_905121265.1 (26975_2#226)\n",
      "Indexing GCF_905121275.1 (26975_1#381)\n",
      "Indexing GCF_905121285.1 (26975_1#368)\n",
      "Indexing GCF_905121295.1 (26975_1#372)\n",
      "Indexing GCF_905121305.1 (26975_1#382)\n",
      "Indexing GCF_905121315.1 (26975_2#247)\n",
      "Indexing GCF_905121325.1 (26975_1#369)\n",
      "Indexing GCF_905121335.1 (26975_2#74)\n",
      "Indexing GCF_905121345.1 (26975_2#69)\n",
      "Indexing GCF_905121355.1 (26975_2#102)\n",
      "Indexing GCF_905121365.1 (26975_2#229)\n",
      "Indexing GCF_905121375.1 (26975_2#91)\n",
      "Indexing GCF_905121385.1 (26975_2#115)\n",
      "Indexing GCF_905121395.1 (26975_2#107)\n",
      "Indexing GCF_905121405.1 (26975_2#106)\n",
      "Indexing GCF_905121415.1 (26975_2#2)\n",
      "Indexing GCF_905121425.1 (26975_2#231)\n",
      "Indexing GCF_905121435.1 (26975_2#97)\n",
      "Indexing GCF_905121445.1 (26975_2#114)\n",
      "Indexing GCF_905121455.1 (26975_2#83)\n",
      "Indexing GCF_905121465.1 (26975_2#130)\n",
      "Indexing GCF_905121475.1 (27688_1#347)\n",
      "Indexing GCF_905121485.1 (27688_1#225)\n",
      "Indexing GCF_905121495.1 (27725_1#182)\n",
      "Indexing GCF_905121505.1 (27725_1#206)\n",
      "Indexing GCF_905121515.1 (27688_1#304)\n",
      "Indexing GCF_905121525.1 (27725_1#242)\n",
      "Indexing GCF_905121535.1 (27688_1#375)\n",
      "Indexing GCF_905121545.1 (27688_1#68)\n",
      "Indexing GCF_905121555.1 (27725_1#262)\n",
      "Indexing GCF_905121565.1 (27725_1#198)\n",
      "Indexing GCF_905121575.1 (27688_1#241)\n",
      "Indexing GCF_905121585.1 (27688_1#303)\n",
      "Indexing GCF_905121595.1 (27688_1#265)\n",
      "Indexing GCF_905121605.1 (27725_1#226)\n",
      "Indexing GCF_905121615.1 (27688_1#45)\n",
      "Indexing GCF_905121625.1 (27688_1#76)\n",
      "Indexing GCF_905121635.1 (27688_1#88)\n",
      "Indexing GCF_905121645.1 (27725_1#254)\n",
      "Indexing GCF_905121655.1 (27688_1#269)\n",
      "Indexing GCF_905121665.1 (27688_1#80)\n",
      "Indexing GCF_905121675.1 (27688_1#21)\n",
      "Indexing GCF_905121685.1 (27688_1#157)\n",
      "Indexing GCF_905121695.1 (27688_1#153)\n",
      "Indexing GCF_905121705.1 (27688_1#380)\n",
      "Indexing GCF_905121715.1 (27688_1#194)\n",
      "Indexing GCF_905121725.1 (27688_1#57)\n",
      "Indexing GCF_905121735.1 (27688_1#101)\n",
      "Indexing GCF_905121745.1 (27688_1#231)\n",
      "Indexing GCF_905121755.1 (27688_1#266)\n",
      "Indexing GCF_905121765.1 (27725_1#319)\n",
      "Indexing GCF_905121775.1 (27688_1#114)\n",
      "Indexing GCF_905121785.1 (27688_1#202)\n",
      "Indexing GCF_905121795.1 (27688_1#106)\n",
      "Indexing GCF_905121805.1 (27725_1#295)\n",
      "Indexing GCF_905121815.1 (27688_1#230)\n",
      "Indexing GCF_905121825.1 (27688_1#234)\n",
      "Indexing GCF_905121835.1 (27725_1#12)\n",
      "Indexing GCF_905121845.1 (27725_1#343)\n",
      "Indexing GCF_905121855.1 (28099_2#254)\n",
      "Indexing GCF_905121865.1 (27688_1#267)\n",
      "Indexing GCF_905121875.1 (27688_1#93)\n",
      "Indexing GCF_905121885.1 (28157_4#249)\n",
      "Indexing GCF_905121895.1 (27688_1#305)\n",
      "Indexing GCF_905121905.1 (28157_4#3)\n",
      "Indexing GCF_905121915.1 (27688_1#170)\n",
      "Indexing GCF_905121925.1 (27725_1#8)\n",
      "Indexing GCF_905121935.1 (28157_4#168)\n",
      "Indexing GCF_905121945.1 (27725_1#347)\n",
      "Indexing GCF_905121955.1 (28157_4#164)\n",
      "Indexing GCF_905121965.1 (28099_2#340)\n",
      "Indexing GCF_905121975.1 (27688_1#292)\n",
      "Indexing GCF_905121985.1 (27725_1#24)\n",
      "Indexing GCF_905121995.1 (28099_2#281)\n",
      "Indexing GCF_905122005.1 (28157_4#208)\n",
      "Indexing GCF_905122015.1 (27725_1#307)\n",
      "Indexing GCF_905122025.1 (28099_2#67)\n",
      "Indexing GCF_905122035.1 (27688_1#235)\n",
      "Indexing GCF_905122045.1 (27688_1#215)\n",
      "Indexing GCF_905122055.1 (28157_4#300)\n",
      "Indexing GCF_905122065.1 (28157_4#316)\n",
      "Indexing GCF_905122075.1 (28157_4#38)\n",
      "Indexing GCF_905122085.1 (28157_4#11)\n",
      "Indexing GCF_905122095.1 (28157_4#90)\n",
      "Indexing GCF_905122105.1 (28157_4#228)\n",
      "Indexing GCF_905122115.1 (28157_4#308)\n",
      "Indexing GCF_905122125.1 (28157_4#7)\n",
      "Indexing GCF_905122135.1 (28157_4#283)\n",
      "Indexing GCF_905122145.1 (28157_4#86)\n",
      "Indexing GCF_905122155.1 (28099_2#241)\n",
      "Indexing GCF_905122165.1 (28157_4#221)\n",
      "Indexing GCF_905122175.1 (28157_4#241)\n",
      "Indexing GCF_905122185.1 (28157_4#70)\n",
      "Indexing GCF_905122195.1 (28157_4#315)\n",
      "Indexing GCF_905122205.1 (28157_4#191)\n",
      "Indexing GCF_905122215.1 (28157_4#152)\n",
      "Indexing GCF_905122225.1 (28157_4#245)\n",
      "Indexing GCF_905122235.1 (28157_4#65)\n",
      "Indexing GCF_905122245.1 (28157_4#258)\n",
      "Indexing GCF_905122255.1 (28157_4#179)\n",
      "Indexing GCF_905122265.1 (28157_4#207)\n",
      "Indexing GCF_905122275.1 (28157_4#72)\n",
      "Indexing GCF_905122285.1 (28157_4#92)\n",
      "Indexing GCF_905122295.1 (28157_4#39)\n",
      "Indexing GCF_905122305.1 (28157_4#93)\n",
      "Indexing GCF_905122315.1 (28157_4#96)\n",
      "Indexing GCF_905122325.1 (28157_4#278)\n",
      "Indexing GCF_905122335.1 (28157_4#140)\n",
      "Indexing GCF_905122345.1 (28157_4#218)\n",
      "Indexing GCF_905122355.1 (28157_4#139)\n",
      "Indexing GCF_905122365.1 (28099_2#314)\n",
      "Indexing GCF_905122375.1 (28157_4#282)\n",
      "Indexing GCF_905122385.1 (28157_4#281)\n",
      "Indexing GCF_905122395.1 (28157_4#137)\n",
      "Indexing GCF_905122405.1 (28157_4#317)\n",
      "Indexing GCF_905122415.1 (28157_4#276)\n",
      "Indexing GCF_905122425.1 (28157_4#177)\n",
      "Indexing GCF_905122435.1 (28157_4#40)\n",
      "Indexing GCF_905122445.1 (28157_4#173)\n",
      "Indexing GCF_905122455.1 (28157_4#260)\n",
      "Indexing GCF_905122465.1 (28157_4#264)\n",
      "Indexing GCF_905122475.1 (28157_4#109)\n",
      "Indexing GCF_905122485.1 (28157_4#356)\n",
      "Indexing GCF_905122495.1 (28157_4#348)\n",
      "Indexing GCF_905122505.1 (28099_2#370)\n",
      "Indexing GCF_905122515.1 (28157_4#117)\n",
      "Indexing GCF_905122525.1 (28157_4#166)\n",
      "Indexing GCF_905122535.1 (28157_4#145)\n",
      "Indexing GCF_905122545.1 (28099_2#139)\n",
      "Indexing GCF_905122555.1 (28157_4#18)\n",
      "Indexing GCF_905122565.1 (28157_4#135)\n",
      "Indexing GCF_905122575.1 (28099_2#233)\n",
      "Indexing GCF_905122585.1 (28099_2#18)\n",
      "Indexing GCF_905122595.1 (28099_2#118)\n",
      "Indexing GCF_905122605.1 (28099_2#54)\n",
      "Indexing GCF_905122615.1 (28099_2#292)\n",
      "Indexing GCF_905122625.1 (28157_4#310)\n",
      "Indexing GCF_905122635.1 (28157_4#381)\n",
      "Indexing GCF_905122645.1 (28157_4#363)\n",
      "Indexing GCF_905122655.1 (28157_4#48)\n",
      "Indexing GCF_905122665.1 (28099_2#361)\n",
      "Indexing GCF_905122675.1 (28157_4#330)\n",
      "Indexing GCF_905122685.1 (28157_4#87)\n",
      "Indexing GCF_905122695.1 (27688_1#172)\n",
      "Indexing GCF_905122705.1 (28099_2#221)\n",
      "Indexing GCF_905122715.1 (28157_4#338)\n",
      "Indexing GCF_905122725.1 (27688_1#208)\n",
      "Indexing GCF_905122735.1 (27725_1#19)\n",
      "Indexing GCF_905122745.1 (28157_4#85)\n",
      "Indexing GCF_905122755.1 (27688_1#325)\n",
      "Indexing GCF_905122765.1 (27725_1#59)\n",
      "Indexing GCF_905122775.1 (27688_1#240)\n",
      "Indexing GCF_905122785.1 (27725_1#112)\n",
      "Indexing GCF_905122795.1 (27688_1#329)\n",
      "Indexing GCF_905122805.1 (27725_1#11)\n",
      "Indexing GCF_905122815.1 (27688_1#192)\n",
      "Indexing GCF_905122825.1 (27688_1#353)\n",
      "Indexing GCF_905122835.1 (27688_1#78)\n",
      "Indexing GCF_905122845.1 (27725_1#7)\n",
      "Indexing GCF_905122855.1 (27725_1#47)\n",
      "Indexing GCF_905122865.1 (27688_1#43)\n",
      "Indexing GCF_905122875.1 (27688_1#212)\n",
      "Indexing GCF_905122885.1 (27725_1#72)\n",
      "Indexing GCF_905122895.1 (27688_1#260)\n",
      "Indexing GCF_905122905.1 (27688_1#256)\n",
      "Indexing GCF_905122915.1 (27688_1#337)\n",
      "Indexing GCF_905122925.1 (27688_1#103)\n",
      "Indexing GCF_905122935.1 (27688_1#220)\n",
      "Indexing GCF_905122945.1 (27688_1#341)\n",
      "Indexing GCF_905122955.1 (27688_1#204)\n",
      "Indexing GCF_905122965.1 (27688_1#128)\n",
      "Indexing GCF_905122975.1 (27688_1#160)\n",
      "Indexing GCF_905122985.1 (27688_1#136)\n",
      "Indexing GCF_905122995.1 (27688_1#285)\n",
      "Indexing GCF_905123005.1 (27688_1#55)\n",
      "Indexing GCF_905123015.1 (27688_1#124)\n",
      "Indexing GCF_905123025.1 (27688_1#152)\n",
      "Indexing GCF_905123035.1 (27688_1#59)\n",
      "Indexing GCF_905123045.1 (27688_1#112)\n",
      "Indexing GCF_905123055.1 (27688_1#74)\n",
      "Indexing GCF_905123065.1 (27688_1#382)\n",
      "Indexing GCF_905123075.1 (27688_1#132)\n",
      "Indexing GCF_905123085.1 (26975_1#75)\n",
      "Indexing GCF_905123095.1 (26975_1#26)\n",
      "Indexing GCF_905123115.1 (26975_1#39)\n",
      "Indexing GCF_905123135.1 (26975_1#29)\n",
      "Indexing GCF_905123155.1 (26975_2#149)\n",
      "Indexing GCF_905123165.1 (26975_2#16)\n",
      "Indexing GCF_905123175.1 (26975_2#24)\n",
      "Indexing GCF_905123185.1 (26975_2#17)\n",
      "Indexing GCF_905123195.1 (26975_1#150)\n",
      "Indexing GCF_905123205.1 (26975_2#18)\n",
      "Indexing GCF_905123215.1 (26975_2#20)\n",
      "Indexing GCF_905123225.1 (26975_2#23)\n",
      "Indexing GCF_905123235.1 (26975_1#85)\n",
      "Indexing GCF_905123245.1 (26975_2#21)\n",
      "Indexing GCF_905123255.1 (26975_1#204)\n",
      "Indexing GCF_905123265.1 (26975_2#19)\n",
      "Indexing GCF_905123275.1 (26975_2#25)\n",
      "Indexing GCF_905123285.1 (26975_2#26)\n",
      "Indexing GCF_905123295.1 (26975_2#27)\n",
      "Indexing GCF_905123305.1 (26975_2#29)\n",
      "Indexing GCF_905123315.1 (26975_2#30)\n",
      "Indexing GCF_905123325.1 (26975_2#31)\n",
      "Indexing GCF_905123335.1 (26975_2#39)\n",
      "Indexing GCF_905123345.1 (26975_2#227)\n",
      "Indexing GCF_905123355.1 (26975_2#38)\n",
      "Indexing GCF_905123365.1 (26975_2#33)\n",
      "Indexing GCF_905123375.1 (26975_2#35)\n",
      "Indexing GCF_905123385.1 (26975_2#36)\n",
      "Indexing GCF_905123395.1 (26975_2#40)\n",
      "Indexing GCF_905123405.1 (26975_2#228)\n",
      "Indexing GCF_905123415.1 (26975_2#32)\n",
      "Indexing GCF_905123425.1 (26975_2#37)\n",
      "Indexing GCF_905123435.1 (26975_2#34)\n",
      "Indexing GCF_905123445.1 (26975_2#242)\n",
      "Indexing GCF_905123455.1 (26975_2#43)\n",
      "Indexing GCF_905123465.1 (26975_2#44)\n",
      "Indexing GCF_905123475.1 (26975_2#45)\n",
      "Indexing GCF_905123485.1 (26975_2#41)\n",
      "Indexing GCF_905123495.1 (26975_2#42)\n",
      "Indexing GCF_905123525.1 (28157_4#171)\n",
      "Indexing GCF_905123535.1 (28157_4#255)\n",
      "Indexing GCF_905123545.1 (28157_4#36)\n",
      "Indexing GCF_905123555.1 (28157_4#280)\n",
      "Indexing GCF_905123565.1 (28157_4#99)\n",
      "Indexing GCF_905123575.1 (27688_1#210)\n",
      "Indexing GCF_905123585.1 (27688_1#300)\n",
      "Indexing GCF_905123595.1 (28099_2#176)\n",
      "Indexing GCF_905123605.1 (28157_4#120)\n",
      "Indexing GCF_905123615.1 (28157_4#299)\n",
      "Indexing GCF_905123625.1 (28157_4#319)\n",
      "Indexing GCF_905123635.1 (28099_2#75)\n",
      "Indexing GCF_905123645.1 (28157_4#185)\n",
      "Indexing GCF_905123655.1 (28157_4#209)\n",
      "Indexing GCF_905123665.1 (28157_4#156)\n",
      "Indexing GCF_905123675.1 (28099_2#73)\n",
      "Indexing GCF_905123685.1 (28157_4#153)\n",
      "Indexing GCF_905123695.1 (28099_2#212)\n",
      "Indexing GCF_905123705.1 (28157_4#78)\n",
      "Indexing GCF_905123715.1 (28157_4#88)\n",
      "Indexing GCF_905123725.1 (27688_1#349)\n",
      "Indexing GCF_905123735.1 (28157_4#263)\n",
      "Indexing GCF_905123745.1 (28157_4#163)\n",
      "Indexing GCF_905123755.1 (27688_1#67)\n",
      "Indexing GCF_905123765.1 (28157_4#150)\n",
      "Indexing GCF_905123775.1 (28157_4#64)\n",
      "Indexing GCF_905123785.1 (27725_1#165)\n",
      "Indexing GCF_905123795.1 (28157_4#332)\n",
      "Indexing GCF_905123805.1 (27688_1#63)\n",
      "Indexing GCF_905123815.1 (28157_4#306)\n",
      "Indexing GCF_905123825.1 (27688_1#342)\n",
      "Indexing GCF_905123835.1 (28157_4#42)\n",
      "Indexing GCF_905123845.1 (28157_4#211)\n",
      "Indexing GCF_905123855.1 (28157_4#56)\n",
      "Indexing GCF_905123865.1 (27688_1#370)\n",
      "Indexing GCF_905123875.1 (28157_4#95)\n",
      "Indexing GCF_906464835.1 (TM6294)\n",
      "Indexing GCF_906464865.1 (WE0438)\n",
      "Indexing GCF_906464875.1 (BX8117)\n",
      "Indexing GCF_906464895.1 (WE0851)\n",
      "Indexing GCF_906464915.2 (WE0254)\n",
      "Indexing GCF_906464925.1 (BX5936)\n",
      "Indexing GCA_001598635.1 (LD33)\n",
      "Indexing GCA_001689055.2 (KB1)\n",
      "Indexing GCA_001989555.1 (CLB21560)\n",
      "Indexing GCA_002163735.1 (sorialis)\n",
      "Indexing GCA_002208945.2 (FDAARGOS_338)\n",
      "Indexing GCA_002289045.2 (H25)\n",
      "Indexing GCA_002355755.1 (W11)\n",
      "Indexing GCA_003030425.1 (FDAARGOS_324)\n",
      "Indexing GCA_003345275.1 (TY1)\n",
      "Indexing GCA_003812665.1 (FDAARGOS_528)\n",
      "Indexing GCA_003966385.1 (KUB3006)\n",
      "Indexing GCA_003966405.1 (KUB3007)\n",
      "Indexing GCA_004332015.1 (CVM N48037F)\n",
      "Indexing GCA_004332035.1 (CVM N60443F)\n",
      "Indexing GCA_005158025.1 (FC)\n",
      "Indexing GCA_005484525.1 (SGAir0397)\n",
      "Indexing GCA_005979285.1 (C25)\n",
      "Indexing GCA_005979305.1 (C54)\n",
      "Indexing GCA_006349345.1 (HA-1)\n",
      "Indexing GCA_006364815.1 (FDAARGOS_611)\n",
      "Indexing GCA_006494835.1 (VE14089)\n",
      "Indexing GCA_006494855.1 (VE18379)\n",
      "Indexing GCA_006494875.1 (VE18395)\n",
      "Indexing GCA_006766145.1 (DM01)\n",
      "Indexing GCA_009498175.3 (L8)\n",
      "Indexing GCA_010103655.1 (JY32)\n",
      "Indexing GCA_011193375.1 (ES-1)\n",
      "Indexing GCA_012594215.1 (TH4125)\n",
      "Indexing GCA_013344605.1 (133170041-3)\n",
      "Indexing GCA_013344625.1 (111540047-1)\n",
      "Indexing GCA_013344645.1 (111540027-2)\n",
      "Indexing GCA_013344665.1 (092160007-3)\n",
      "Indexing GCA_014489455.1 (SF28073)\n",
      "Indexing GCA_014844235.1 (EGM183)\n",
      "Indexing GCA_015377745.1 (TK-P4B)\n",
      "Indexing GCA_016743835.1 (T90-2)\n",
      "Indexing GCA_016743895.1 (T90-1)\n",
      "Indexing GCA_016812095.1 (L11)\n",
      "Indexing GCA_017280035.1 (L13)\n",
      "Indexing GCA_017280235.1 (L18)\n",
      "Indexing GCA_017357805.1 (BFF1B1)\n",
      "Indexing GCA_017639585.1 (EF-2001)\n",
      "Indexing GCA_018517125.1 (18-243)\n",
      "Indexing GCA_018986755.2 (1207/14)\n",
      "Indexing GCA_019195415.1 (SY-1)\n",
      "Indexing GCA_019222705.1 (NS2)\n",
      "Indexing GCA_019774535.1 (E006xJH2-2-TC1)\n",
      "Indexing GCA_019856595.1 (E006)\n",
      "Indexing GCA_020616775.1 (EFS17)\n",
      "Indexing GCA_020616875.1 (EFS36)\n",
      "Indexing GCA_020616935.1 (EFS108)\n",
      "Indexing GCA_020844125.1 (AR_0780)\n",
      "Indexing GCA_020881975.1 (R5)\n",
      "Indexing GCA_020882035.1 (E512)\n",
      "Indexing GCA_020882055.1 (E512-TC2)\n",
      "Indexing GCA_020882075.1 (E509)\n",
      "Indexing GCA_020882095.1 (E512-TC1)\n",
      "Indexing GCA_020882115.1 (Fac74)\n",
      "Indexing GCA_020906605.1 (E533)\n",
      "Indexing GCA_021484825.1 (PCH555)\n",
      "Indexing GCA_021610105.1 (UK045)\n",
      "Indexing GCA_021655635.1 (K-4)\n",
      "Indexing GCA_022212845.1 (152)\n",
      "Indexing GCA_022212865.1 (143-1)\n",
      "Indexing GCA_022212885.1 (142-1)\n",
      "Indexing GCA_022212905.1 (133-1)\n",
      "Indexing GCA_022212925.1 (119-2)\n",
      "Indexing GCA_022212945.1 (119-1)\n",
      "Indexing GCA_022212965.1 (101-1)\n",
      "Indexing GCA_022212985.1 (59)\n",
      "Indexing GCA_022213005.1 (43-2)\n",
      "Indexing GCA_022213025.1 (2-1)\n",
      "Indexing GCA_022557275.1 (LJX 0909)\n",
      "Indexing GCA_022648285.1 (UAMS_EL56)\n",
      "Indexing GCA_022648425.1 (UAMS_EL54)\n",
      "Indexing GCA_022648595.1 (UAMS_EL53)\n",
      "Indexing GCA_022691465.1 (VSE-WC032)\n",
      "Indexing GCA_022691485.1 (VRE-WC031)\n",
      "Indexing GCA_022699585.1 (NY13321)\n",
      "Indexing GCA_022810805.1 (RF_1_1_25_1)\n",
      "Indexing GCA_022869705.1 (PartL-Efaecalis-RM8376)\n",
      "Indexing GCA_023182775.2 (AKSZ-208)\n",
      "Indexing GCA_023299225.1 (AT49a)\n",
      "Indexing GCA_023299425.1 (AT48)\n",
      "Indexing GCA_023299465.1 (LS05-1)\n",
      "Indexing GCA_023299485.1 (LS05-2)\n",
      "Indexing GCA_023299505.1 (AT40b)\n",
      "Indexing GCA_023299525.1 (LS06-1)\n",
      "Indexing GCA_023299545.1 (AT09)\n",
      "Indexing GCA_023299585.1 (AT40a)\n",
      "Indexing GCA_023299605.1 (AT46a)\n",
      "Indexing GCA_023299625.1 (AT39)\n",
      "Indexing GCA_023299645.1 (AT50)\n",
      "Indexing GCA_023299665.1 (AT29)\n",
      "Indexing GCA_023299685.1 (AT22)\n",
      "Indexing GCA_023299705.1 (AT34)\n",
      "Indexing GCA_023299785.1 (AT43a)\n",
      "Indexing GCA_023300025.1 (AT41)\n",
      "Indexing GCA_023300165.1 (AT04)\n",
      "Indexing GCA_023375245.1 (S11-6)\n",
      "Indexing GCA_023375265.1 (S39-4)\n",
      "Indexing GCA_023375465.1 (661)\n",
      "Indexing GCA_023375485.1 (732)\n",
      "Indexing GCA_023375505.1 (249)\n",
      "Indexing GCA_023375525.1 (705)\n",
      "Indexing GCA_023375545.1 (1521)\n",
      "Indexing GCA_023523795.1 (FMAMOB6)\n",
      "Indexing GCA_023612035.1 (M817)\n",
      "Indexing GCA_023650735.1 (QZ076)\n",
      "Indexing GCA_023702145.1 (CQ025)\n",
      "Indexing GCA_023746575.1 (M9)\n",
      "Indexing GCA_024267595.1 (Chr-JH 2-2)\n",
      "Indexing GCA_024579755.1 (JF3A-223)\n",
      "Indexing GCA_024579795.1 (JF3A-4253)\n",
      "Indexing GCA_024927865.1 (SJ82)\n",
      "Indexing GCA_025957845.1 (BE70)\n",
      "Indexing GCA_025957925.1 (BE69)\n",
      "Indexing GCA_025957945.1 (BE67)\n",
      "Indexing GCA_025957965.1 (BE65)\n",
      "Indexing GCA_025957985.1 (BE57)\n",
      "Indexing GCA_025958005.1 (BE54)\n",
      "Indexing GCA_025958025.1 (BE52)\n",
      "Indexing GCA_025958045.1 (BE51)\n",
      "Indexing GCA_025958065.1 (BE47)\n",
      "Indexing GCA_025958085.1 (BE45)\n",
      "Indexing GCA_025958125.1 (BE43)\n",
      "Indexing GCA_025958325.1 (BE33)\n",
      "Indexing GCA_025958445.1 (BE32)\n",
      "Indexing GCA_025958625.1 (BE25)\n",
      "Indexing GCA_025958785.1 (BE17)\n",
      "Indexing GCA_025958805.1 (BE16)\n",
      "Indexing GCA_025958825.1 (BE15)\n",
      "Indexing GCA_025958845.1 (BE13)\n",
      "Indexing GCA_025958865.1 (BE11)\n",
      "Indexing GCA_025958885.1 (BE8)\n",
      "Indexing GCA_025958905.1 (BE7)\n",
      "Indexing GCA_025958925.1 (BE5)\n",
      "Indexing GCA_026650885.1 (FA2-2)\n",
      "Indexing GCA_026802365.1 (T30)\n",
      "Indexing GCA_026802385.1 (M61)\n",
      "Indexing GCA_027942035.1 (ESC1)\n",
      "Indexing GCA_028131375.1 (SVR2281)\n",
      "Indexing GCA_028131645.1 (SVR2330)\n",
      "Indexing GCA_028131945.1 (SVR2331)\n",
      "Indexing GCA_028132325.1 (SVR2332)\n",
      "Indexing GCA_028335305.1 (K198-1)\n",
      "Indexing GCA_028335345.1 (K190-1)\n",
      "Indexing GCA_028335425.1 (K70-12a)\n",
      "Indexing GCA_028335445.1 (K137-1a)\n",
      "Indexing GCA_028335505.1 (K194-1)\n",
      "Indexing GCA_028335545.1 (K72-1)\n",
      "Indexing GCA_028335665.1 (K80-2b)\n",
      "Indexing GCA_028335685.1 (K188-1)\n",
      "Indexing GCA_028335705.1 (K205-3)\n",
      "Indexing GCA_028335725.1 (K191-1)\n",
      "Indexing GCA_028532705.1 (DM86)\n",
      "Indexing GCA_028598965.2 (SVJ-EF01)\n",
      "Indexing GCA_028621735.1 (CQJXZ21-077)\n",
      "Indexing GCA_028621755.1 (CQJXZ21-076)\n",
      "Indexing GCA_028621775.1 (CQNBZ21-135)\n",
      "Indexing GCA_028743535.1 (B-537)\n",
      "Indexing GCA_029000425.1 (W5)\n",
      "Indexing GCA_029011255.1 (VSI48)\n",
      "Indexing GCA_029011395.1 (VSI36)\n",
      "Indexing GCA_029011535.1 (VSI19)\n",
      "Indexing GCA_029011745.1 (VSI09)\n",
      "Indexing GCA_029024925.1 (DSM 20478)\n",
      "Indexing GCA_029167565.1 (GENOMIC22-006)\n",
      "Indexing GCA_029278545.1 (3143)\n",
      "Indexing GCA_029866805.1 (NY13412)\n",
      "Indexing GCA_029910175.1 (NY13410)\n",
      "Indexing GCA_030020155.1 (EfsPNK7)\n",
      "Indexing GCA_030020175.1 (EfsPF15)\n",
      "Indexing GCA_030020195.1 (EfsPF13)\n",
      "Indexing GCA_030020215.1 (EfsPF5)\n",
      "Indexing GCA_030020235.1 (EfsPF2)\n",
      "Indexing GCA_030020255.1 (EfsC130)\n",
      "Indexing GCA_030020425.1 (EfsPF20)\n",
      "Indexing GCA_030020455.1 (EfsC123)\n",
      "Indexing GCA_030020475.1 (EfsC116)\n",
      "Indexing GCA_030020495.1 (EfsC109-2)\n",
      "Indexing GCA_030020515.1 (EfsC108)\n",
      "Indexing GCA_030020535.1 (EfsC94)\n",
      "Indexing GCA_030020555.1 (EfsC85)\n",
      "Indexing GCA_030020575.1 (EfsC61)\n",
      "Indexing GCA_030020595.1 (EfsC49)\n",
      "Indexing GCA_030020615.1 (EfsC33)\n",
      "Indexing GCA_030020645.1 (EfsC20)\n",
      "Indexing GCA_030020665.1 (EfsC17)\n",
      "Indexing GCA_030020685.1 (EfsC12)\n",
      "Indexing GCA_030020705.1 (EfsC11)\n",
      "Indexing GCA_030020725.1 (EfsC9)\n",
      "Indexing GCA_030020745.1 (EfsC8)\n",
      "Indexing GCA_030064715.1 (AFL-109F)\n",
      "Indexing GCA_030264755.1 (90_2023)\n",
      "Indexing GCA_030295445.1 (JARB-HU0796)\n",
      "Indexing GCA_030295805.1 (L6D)\n",
      "Indexing GCA_030295845.1 (N4E)\n",
      "Indexing GCA_030376825.1 (RE25)\n",
      "Indexing GCA_030512425.1 (HL1)\n",
      "Indexing GCA_030517125.1 (EfsPF36)\n",
      "Indexing GCA_030517355.1 (EfsPF27)\n",
      "Indexing GCA_030517385.1 (EfsPF23)\n",
      "Indexing GCA_030517415.1 (EfsC129)\n",
      "Indexing GCA_030517445.1 (EfsC43)\n",
      "Indexing GCA_030517475.1 (EfsC29)\n",
      "Indexing GCA_030517505.1 (EfsC23)\n",
      "Indexing GCA_030517535.1 (EfsPF24-2)\n",
      "Indexing GCA_030517565.1 (EfsC1)\n",
      "Indexing GCA_030517625.1 (EfsC13)\n",
      "Indexing GCA_031186115.1 (JF4A2234)\n",
      "Indexing GCA_031206965.1 (N775_PC7)\n",
      "Indexing GCA_031208705.1 (N775_PC2)\n",
      "Indexing GCA_900636775.1 (NCTC8745)\n",
      "Indexing GCA_901553725.1 (NCTC8732)\n",
      "Indexing GCA_902379005.2 (MGYG-HGUT-01694)\n",
      "Indexing GCA_905120545.1 (26975_1#66)\n",
      "Indexing GCA_905120555.1 (26975_1#5)\n",
      "Indexing GCA_905120565.1 (26975_1#52)\n",
      "Indexing GCA_905120575.1 (26975_2#148)\n",
      "Indexing GCA_905120585.1 (26975_1#18)\n",
      "Indexing GCA_905120595.1 (26975_1#11)\n",
      "Indexing GCA_905120605.1 (26975_1#45)\n",
      "Indexing GCA_905120615.1 (26975_1#33)\n",
      "Indexing GCA_905120625.1 (26975_1#32)\n",
      "Indexing GCA_905120635.1 (26975_2#150)\n",
      "Indexing GCA_905120645.1 (26975_1#37)\n",
      "Indexing GCA_905120655.1 (26975_1#13)\n",
      "Indexing GCA_905120665.1 (26975_1#48)\n",
      "Indexing GCA_905120675.1 (26975_2#147)\n",
      "Indexing GCA_905120685.1 (26975_2#145)\n",
      "Indexing GCA_905120695.1 (26975_1#25)\n",
      "Indexing GCA_905120705.1 (26975_1#12)\n",
      "Indexing GCA_905120715.1 (26975_1#22)\n",
      "Indexing GCA_905120725.1 (26975_1#31)\n",
      "Indexing GCA_905120735.1 (26975_1#63)\n",
      "Indexing GCA_905120745.1 (26975_1#105)\n",
      "Indexing GCA_905120755.1 (26975_1#88)\n",
      "Indexing GCA_905120765.1 (26975_1#107)\n",
      "Indexing GCA_905120775.1 (26975_1#367)\n",
      "Indexing GCA_905120785.1 (26975_1#70)\n",
      "Indexing GCA_905120795.1 (26975_1#74)\n",
      "Indexing GCA_905120805.1 (26975_1#132)\n",
      "Indexing GCA_905120815.1 (26975_1#129)\n",
      "Indexing GCA_905120825.1 (26975_1#145)\n",
      "Indexing GCA_905120835.1 (26975_1#118)\n",
      "Indexing GCA_905120845.1 (26975_1#111)\n",
      "Indexing GCA_905120855.1 (26975_1#117)\n",
      "Indexing GCA_905120865.1 (26975_2#152)\n",
      "Indexing GCA_905120875.1 (26975_1#89)\n",
      "Indexing GCA_905120885.1 (26975_1#160)\n",
      "Indexing GCA_905120895.1 (26975_2#162)\n",
      "Indexing GCA_905120905.1 (26975_2#165)\n",
      "Indexing GCA_905120915.1 (26975_1#146)\n",
      "Indexing GCA_905120925.1 (26975_1#168)\n",
      "Indexing GCA_905120935.1 (26975_2#160)\n",
      "Indexing GCA_905120945.1 (26975_2#190)\n",
      "Indexing GCA_905120955.1 (26975_2#198)\n",
      "Indexing GCA_905120965.1 (26975_2#200)\n",
      "Indexing GCA_905120975.1 (26975_2#184)\n",
      "Indexing GCA_905120985.1 (26975_2#204)\n",
      "Indexing GCA_905120995.1 (26975_2#138)\n",
      "Indexing GCA_905121005.1 (26975_2#195)\n",
      "Indexing GCA_905121015.1 (26975_2#191)\n",
      "Indexing GCA_905121025.1 (26975_2#201)\n",
      "Indexing GCA_905121035.1 (26975_2#180)\n",
      "Indexing GCA_905121045.1 (26975_1#180)\n",
      "Indexing GCA_905121055.1 (26975_1#182)\n",
      "Indexing GCA_905121065.1 (26975_1#366)\n",
      "Indexing GCA_905121075.1 (26975_2#175)\n",
      "Indexing GCA_905121085.1 (26975_2#206)\n",
      "Indexing GCA_905121095.1 (26975_2#221)\n",
      "Indexing GCA_905121105.1 (26975_2#225)\n",
      "Indexing GCA_905121115.1 (26975_1#188)\n",
      "Indexing GCA_905121125.1 (26975_1#194)\n",
      "Indexing GCA_905121135.1 (26975_1#236)\n",
      "Indexing GCA_905121145.1 (26975_1#253)\n",
      "Indexing GCA_905121155.1 (26975_1#276)\n",
      "Indexing GCA_905121165.1 (26975_1#268)\n",
      "Indexing GCA_905121175.1 (26975_1#228)\n",
      "Indexing GCA_905121185.1 (26975_1#214)\n",
      "Indexing GCA_905121195.1 (26975_1#279)\n",
      "Indexing GCA_905121205.1 (26975_1#246)\n",
      "Indexing GCA_905121215.1 (26975_1#280)\n",
      "Indexing GCA_905121225.1 (26975_1#283)\n",
      "Indexing GCA_905121235.1 (26975_1#263)\n",
      "Indexing GCA_905121245.1 (26975_1#241)\n",
      "Indexing GCA_905121255.1 (26975_1#262)\n",
      "Indexing GCA_905121265.1 (26975_2#226)\n",
      "Indexing GCA_905121275.1 (26975_1#381)\n",
      "Indexing GCA_905121285.1 (26975_1#368)\n",
      "Indexing GCA_905121295.1 (26975_1#372)\n",
      "Indexing GCA_905121305.1 (26975_1#382)\n",
      "Indexing GCA_905121315.1 (26975_2#247)\n",
      "Indexing GCA_905121325.1 (26975_1#369)\n",
      "Indexing GCA_905121335.1 (26975_2#74)\n",
      "Indexing GCA_905121345.1 (26975_2#69)\n",
      "Indexing GCA_905121355.1 (26975_2#102)\n",
      "Indexing GCA_905121365.1 (26975_2#229)\n",
      "Indexing GCA_905121375.1 (26975_2#91)\n",
      "Indexing GCA_905121385.1 (26975_2#115)\n",
      "Indexing GCA_905121395.1 (26975_2#107)\n",
      "Indexing GCA_905121405.1 (26975_2#106)\n",
      "Indexing GCA_905121415.1 (26975_2#2)\n",
      "Indexing GCA_905121425.1 (26975_2#231)\n",
      "Indexing GCA_905121435.1 (26975_2#97)\n",
      "Indexing GCA_905121445.1 (26975_2#114)\n",
      "Indexing GCA_905121455.1 (26975_2#83)\n",
      "Indexing GCA_905121465.1 (26975_2#130)\n",
      "Indexing GCA_905121475.1 (27688_1#347)\n",
      "Indexing GCA_905121485.1 (27688_1#225)\n",
      "Indexing GCA_905121495.1 (27725_1#182)\n",
      "Indexing GCA_905121505.1 (27725_1#206)\n",
      "Indexing GCA_905121515.1 (27688_1#304)\n",
      "Indexing GCA_905121525.1 (27725_1#242)\n",
      "Indexing GCA_905121535.1 (27688_1#375)\n",
      "Indexing GCA_905121545.1 (27688_1#68)\n",
      "Indexing GCA_905121555.1 (27725_1#262)\n",
      "Indexing GCA_905121565.1 (27725_1#198)\n",
      "Indexing GCA_905121575.1 (27688_1#241)\n",
      "Indexing GCA_905121585.1 (27688_1#303)\n",
      "Indexing GCA_905121595.1 (27688_1#265)\n",
      "Indexing GCA_905121605.1 (27725_1#226)\n",
      "Indexing GCA_905121615.1 (27688_1#45)\n",
      "Indexing GCA_905121625.1 (27688_1#76)\n",
      "Indexing GCA_905121635.1 (27688_1#88)\n",
      "Indexing GCA_905121645.1 (27725_1#254)\n",
      "Indexing GCA_905121655.1 (27688_1#269)\n",
      "Indexing GCA_905121665.1 (27688_1#80)\n",
      "Indexing GCA_905121675.1 (27688_1#21)\n",
      "Indexing GCA_905121685.1 (27688_1#157)\n",
      "Indexing GCA_905121695.1 (27688_1#153)\n",
      "Indexing GCA_905121705.1 (27688_1#380)\n",
      "Indexing GCA_905121715.1 (27688_1#194)\n",
      "Indexing GCA_905121725.1 (27688_1#57)\n",
      "Indexing GCA_905121735.1 (27688_1#101)\n",
      "Indexing GCA_905121745.1 (27688_1#231)\n",
      "Indexing GCA_905121755.1 (27688_1#266)\n",
      "Indexing GCA_905121765.1 (27725_1#319)\n",
      "Indexing GCA_905121775.1 (27688_1#114)\n",
      "Indexing GCA_905121785.1 (27688_1#202)\n",
      "Indexing GCA_905121795.1 (27688_1#106)\n",
      "Indexing GCA_905121805.1 (27725_1#295)\n",
      "Indexing GCA_905121815.1 (27688_1#230)\n",
      "Indexing GCA_905121825.1 (27688_1#234)\n",
      "Indexing GCA_905121835.1 (27725_1#12)\n",
      "Indexing GCA_905121845.1 (27725_1#343)\n",
      "Indexing GCA_905121855.1 (28099_2#254)\n",
      "Indexing GCA_905121865.1 (27688_1#267)\n",
      "Indexing GCA_905121875.1 (27688_1#93)\n",
      "Indexing GCA_905121885.1 (28157_4#249)\n",
      "Indexing GCA_905121895.1 (27688_1#305)\n",
      "Indexing GCA_905121905.1 (28157_4#3)\n",
      "Indexing GCA_905121915.1 (27688_1#170)\n",
      "Indexing GCA_905121925.1 (27725_1#8)\n",
      "Indexing GCA_905121935.1 (28157_4#168)\n",
      "Indexing GCA_905121945.1 (27725_1#347)\n",
      "Indexing GCA_905121955.1 (28157_4#164)\n",
      "Indexing GCA_905121965.1 (28099_2#340)\n",
      "Indexing GCA_905121975.1 (27688_1#292)\n",
      "Indexing GCA_905121985.1 (27725_1#24)\n",
      "Indexing GCA_905121995.1 (28099_2#281)\n",
      "Indexing GCA_905122005.1 (28157_4#208)\n",
      "Indexing GCA_905122015.1 (27725_1#307)\n",
      "Indexing GCA_905122025.1 (28099_2#67)\n",
      "Indexing GCA_905122035.1 (27688_1#235)\n",
      "Indexing GCA_905122045.1 (27688_1#215)\n",
      "Indexing GCA_905122055.1 (28157_4#300)\n",
      "Indexing GCA_905122065.1 (28157_4#316)\n",
      "Indexing GCA_905122075.1 (28157_4#38)\n",
      "Indexing GCA_905122085.1 (28157_4#11)\n",
      "Indexing GCA_905122095.1 (28157_4#90)\n",
      "Indexing GCA_905122105.1 (28157_4#228)\n",
      "Indexing GCA_905122115.1 (28157_4#308)\n",
      "Indexing GCA_905122125.1 (28157_4#7)\n",
      "Indexing GCA_905122135.1 (28157_4#283)\n",
      "Indexing GCA_905122145.1 (28157_4#86)\n",
      "Indexing GCA_905122155.1 (28099_2#241)\n",
      "Indexing GCA_905122165.1 (28157_4#221)\n",
      "Indexing GCA_905122175.1 (28157_4#241)\n",
      "Indexing GCA_905122185.1 (28157_4#70)\n",
      "Indexing GCA_905122195.1 (28157_4#315)\n",
      "Indexing GCA_905122205.1 (28157_4#191)\n",
      "Indexing GCA_905122215.1 (28157_4#152)\n",
      "Indexing GCA_905122225.1 (28157_4#245)\n",
      "Indexing GCA_905122235.1 (28157_4#65)\n",
      "Indexing GCA_905122245.1 (28157_4#258)\n",
      "Indexing GCA_905122255.1 (28157_4#179)\n",
      "Indexing GCA_905122265.1 (28157_4#207)\n",
      "Indexing GCA_905122275.1 (28157_4#72)\n",
      "Indexing GCA_905122285.1 (28157_4#92)\n",
      "Indexing GCA_905122295.1 (28157_4#39)\n",
      "Indexing GCA_905122305.1 (28157_4#93)\n",
      "Indexing GCA_905122315.1 (28157_4#96)\n",
      "Indexing GCA_905122325.1 (28157_4#278)\n",
      "Indexing GCA_905122335.1 (28157_4#140)\n",
      "Indexing GCA_905122345.1 (28157_4#218)\n",
      "Indexing GCA_905122355.1 (28157_4#139)\n",
      "Indexing GCA_905122365.1 (28099_2#314)\n",
      "Indexing GCA_905122375.1 (28157_4#282)\n",
      "Indexing GCA_905122385.1 (28157_4#281)\n",
      "Indexing GCA_905122395.1 (28157_4#137)\n",
      "Indexing GCA_905122405.1 (28157_4#317)\n",
      "Indexing GCA_905122415.1 (28157_4#276)\n",
      "Indexing GCA_905122425.1 (28157_4#177)\n",
      "Indexing GCA_905122435.1 (28157_4#40)\n",
      "Indexing GCA_905122445.1 (28157_4#173)\n",
      "Indexing GCA_905122455.1 (28157_4#260)\n",
      "Indexing GCA_905122465.1 (28157_4#264)\n",
      "Indexing GCA_905122475.1 (28157_4#109)\n",
      "Indexing GCA_905122485.1 (28157_4#356)\n",
      "Indexing GCA_905122495.1 (28157_4#348)\n",
      "Indexing GCA_905122505.1 (28099_2#370)\n",
      "Indexing GCA_905122515.1 (28157_4#117)\n",
      "Indexing GCA_905122525.1 (28157_4#166)\n",
      "Indexing GCA_905122535.1 (28157_4#145)\n",
      "Indexing GCA_905122545.1 (28099_2#139)\n",
      "Indexing GCA_905122555.1 (28157_4#18)\n",
      "Indexing GCA_905122565.1 (28157_4#135)\n",
      "Indexing GCA_905122575.1 (28099_2#233)\n",
      "Indexing GCA_905122585.1 (28099_2#18)\n",
      "Indexing GCA_905122595.1 (28099_2#118)\n",
      "Indexing GCA_905122605.1 (28099_2#54)\n",
      "Indexing GCA_905122615.1 (28099_2#292)\n",
      "Indexing GCA_905122625.1 (28157_4#310)\n",
      "Indexing GCA_905122635.1 (28157_4#381)\n",
      "Indexing GCA_905122645.1 (28157_4#363)\n",
      "Indexing GCA_905122655.1 (28157_4#48)\n",
      "Indexing GCA_905122665.1 (28099_2#361)\n",
      "Indexing GCA_905122675.1 (28157_4#330)\n",
      "Indexing GCA_905122685.1 (28157_4#87)\n",
      "Indexing GCA_905122695.1 (27688_1#172)\n",
      "Indexing GCA_905122705.1 (28099_2#221)\n",
      "Indexing GCA_905122715.1 (28157_4#338)\n",
      "Indexing GCA_905122725.1 (27688_1#208)\n",
      "Indexing GCA_905122735.1 (27725_1#19)\n",
      "Indexing GCA_905122745.1 (28157_4#85)\n",
      "Indexing GCA_905122755.1 (27688_1#325)\n",
      "Indexing GCA_905122765.1 (27725_1#59)\n",
      "Indexing GCA_905122775.1 (27688_1#240)\n",
      "Indexing GCA_905122785.1 (27725_1#112)\n",
      "Indexing GCA_905122795.1 (27688_1#329)\n",
      "Indexing GCA_905122805.1 (27725_1#11)\n",
      "Indexing GCA_905122815.1 (27688_1#192)\n",
      "Indexing GCA_905122825.1 (27688_1#353)\n",
      "Indexing GCA_905122835.1 (27688_1#78)\n",
      "Indexing GCA_905122845.1 (27725_1#7)\n",
      "Indexing GCA_905122855.1 (27725_1#47)\n",
      "Indexing GCA_905122865.1 (27688_1#43)\n",
      "Indexing GCA_905122875.1 (27688_1#212)\n",
      "Indexing GCA_905122885.1 (27725_1#72)\n",
      "Indexing GCA_905122895.1 (27688_1#260)\n",
      "Indexing GCA_905122905.1 (27688_1#256)\n",
      "Indexing GCA_905122915.1 (27688_1#337)\n",
      "Indexing GCA_905122925.1 (27688_1#103)\n",
      "Indexing GCA_905122935.1 (27688_1#220)\n",
      "Indexing GCA_905122945.1 (27688_1#341)\n",
      "Indexing GCA_905122955.1 (27688_1#204)\n",
      "Indexing GCA_905122965.1 (27688_1#128)\n",
      "Indexing GCA_905122975.1 (27688_1#160)\n",
      "Indexing GCA_905122985.1 (27688_1#136)\n",
      "Indexing GCA_905122995.1 (27688_1#285)\n",
      "Indexing GCA_905123005.1 (27688_1#55)\n",
      "Indexing GCA_905123015.1 (27688_1#124)\n",
      "Indexing GCA_905123025.1 (27688_1#152)\n",
      "Indexing GCA_905123035.1 (27688_1#59)\n",
      "Indexing GCA_905123045.1 (27688_1#112)\n",
      "Indexing GCA_905123055.1 (27688_1#74)\n",
      "Indexing GCA_905123065.1 (27688_1#382)\n",
      "Indexing GCA_905123075.1 (27688_1#132)\n",
      "Indexing GCA_905123085.1 (26975_1#75)\n",
      "Indexing GCA_905123095.1 (26975_1#26)\n",
      "Indexing GCA_905123115.1 (26975_1#39)\n",
      "Indexing GCA_905123135.1 (26975_1#29)\n",
      "Indexing GCA_905123155.1 (26975_2#149)\n",
      "Indexing GCA_905123165.1 (26975_2#16)\n",
      "Indexing GCA_905123175.1 (26975_2#24)\n",
      "Indexing GCA_905123185.1 (26975_2#17)\n",
      "Indexing GCA_905123195.1 (26975_1#150)\n",
      "Indexing GCA_905123205.1 (26975_2#18)\n",
      "Indexing GCA_905123215.1 (26975_2#20)\n",
      "Indexing GCA_905123225.1 (26975_2#23)\n",
      "Indexing GCA_905123235.1 (26975_1#85)\n",
      "Indexing GCA_905123245.1 (26975_2#21)\n",
      "Indexing GCA_905123255.1 (26975_1#204)\n",
      "Indexing GCA_905123265.1 (26975_2#19)\n",
      "Indexing GCA_905123275.1 (26975_2#25)\n",
      "Indexing GCA_905123285.1 (26975_2#26)\n",
      "Indexing GCA_905123295.1 (26975_2#27)\n",
      "Indexing GCA_905123305.1 (26975_2#29)\n",
      "Indexing GCA_905123315.1 (26975_2#30)\n",
      "Indexing GCA_905123325.1 (26975_2#31)\n",
      "Indexing GCA_905123335.1 (26975_2#39)\n",
      "Indexing GCA_905123345.1 (26975_2#227)\n",
      "Indexing GCA_905123355.1 (26975_2#38)\n",
      "Indexing GCA_905123365.1 (26975_2#33)\n",
      "Indexing GCA_905123375.1 (26975_2#35)\n",
      "Indexing GCA_905123385.1 (26975_2#36)\n",
      "Indexing GCA_905123395.1 (26975_2#40)\n",
      "Indexing GCA_905123405.1 (26975_2#228)\n",
      "Indexing GCA_905123415.1 (26975_2#32)\n",
      "Indexing GCA_905123425.1 (26975_2#37)\n",
      "Indexing GCA_905123435.1 (26975_2#34)\n",
      "Indexing GCA_905123445.1 (26975_2#242)\n",
      "Indexing GCA_905123455.1 (26975_2#43)\n",
      "Indexing GCA_905123465.1 (26975_2#44)\n",
      "Indexing GCA_905123475.1 (26975_2#45)\n",
      "Indexing GCA_905123485.1 (26975_2#41)\n",
      "Indexing GCA_905123495.1 (26975_2#42)\n",
      "Indexing GCA_905123525.1 (28157_4#171)\n",
      "Indexing GCA_905123535.1 (28157_4#255)\n",
      "Indexing GCA_905123545.1 (28157_4#36)\n",
      "Indexing GCA_905123555.1 (28157_4#280)\n",
      "Indexing GCA_905123565.1 (28157_4#99)\n",
      "Indexing GCA_905123575.1 (27688_1#210)\n",
      "Indexing GCA_905123585.1 (27688_1#300)\n",
      "Indexing GCA_905123595.1 (28099_2#176)\n",
      "Indexing GCA_905123605.1 (28157_4#120)\n",
      "Indexing GCA_905123615.1 (28157_4#299)\n",
      "Indexing GCA_905123625.1 (28157_4#319)\n",
      "Indexing GCA_905123635.1 (28099_2#75)\n",
      "Indexing GCA_905123645.1 (28157_4#185)\n",
      "Indexing GCA_905123655.1 (28157_4#209)\n",
      "Indexing GCA_905123665.1 (28157_4#156)\n",
      "Indexing GCA_905123675.1 (28099_2#73)\n",
      "Indexing GCA_905123685.1 (28157_4#153)\n",
      "Indexing GCA_905123695.1 (28099_2#212)\n",
      "Indexing GCA_905123705.1 (28157_4#78)\n",
      "Indexing GCA_905123715.1 (28157_4#88)\n",
      "Indexing GCA_905123725.1 (27688_1#349)\n",
      "Indexing GCA_905123735.1 (28157_4#263)\n",
      "Indexing GCA_905123745.1 (28157_4#163)\n",
      "Indexing GCA_905123755.1 (27688_1#67)\n",
      "Indexing GCA_905123765.1 (28157_4#150)\n",
      "Indexing GCA_905123775.1 (28157_4#64)\n",
      "Indexing GCA_905123785.1 (27725_1#165)\n",
      "Indexing GCA_905123795.1 (28157_4#332)\n",
      "Indexing GCA_905123805.1 (27688_1#63)\n",
      "Indexing GCA_905123815.1 (28157_4#306)\n",
      "Indexing GCA_905123825.1 (27688_1#342)\n",
      "Indexing GCA_905123835.1 (28157_4#42)\n",
      "Indexing GCA_905123845.1 (28157_4#211)\n",
      "Indexing GCA_905123855.1 (28157_4#56)\n",
      "Indexing GCA_905123865.1 (27688_1#370)\n",
      "Indexing GCA_905123875.1 (28157_4#95)\n",
      "Indexing GCA_906464835.1 (TM6294)\n",
      "Indexing GCA_906464865.1 (WE0438)\n",
      "Indexing GCA_906464875.1 (BX8117)\n",
      "Indexing GCA_906464895.1 (WE0851)\n",
      "Indexing GCA_906464915.2 (WE0254)\n",
      "Indexing GCA_906464925.1 (BX5936)\n",
      "Indexing GCF_000211255.2 (62)\n",
      "Indexing GCA_000211255.2 (62)\n",
      "Indexing GCF_002814115.1 (ARO1/DG)\n",
      "Indexing GCA_002814115.1 (ARO1/DG)\n",
      "Indexing GCF_000742975.1 (29212)\n",
      "Indexing GCA_000742975.1 (ATCC 29212)\n",
      "Indexing GCF_000281195.1 (D32)\n",
      "Indexing GCA_000281195.1 (D32)\n",
      "Indexing GCF_000550745.1 (DENG1)\n",
      "Indexing GCA_000550745.1 (DENG1)\n",
      "Indexing GCF_000391485.2 (B594)\n",
      "Indexing GCA_000391485.2 (B594)\n",
      "Indexing GCF_000172575.2 (OG1RF)\n",
      "Indexing GCA_000172575.2 (OG1RF)\n",
      "Indexing GCF_009662495.1 (R712)\n",
      "Indexing GCA_009662495.1 (R712)\n",
      "Indexing GCF_000317915.1 (Symbioflor 1)\n",
      "Indexing GCA_000317915.1 (Symbioflor 1)\n",
      "Indexing GCF_000007785.1 (V583)\n",
      "Indexing GCA_000007785.1 (V583)\n",
      "Indexing GCF_009734005.1 (SRR24)\n",
      "Indexing GCF_001298485.1 (64/3)\n",
      "Indexing GCF_001412695.1 (UW7606x64/3 TC1)\n",
      "Indexing GCF_001518735.1 (6E6)\n",
      "Indexing GCF_001587115.1 (UW8175)\n",
      "Indexing GCF_001594345.1 (ATCC 700221)\n",
      "Indexing GCF_001635875.1 (E39)\n",
      "Indexing GCF_001720945.1 (ISMMS_VRE_1)\n",
      "Indexing GCF_001721065.1 (ISMMS_VRE_7)\n",
      "Indexing GCF_001721905.1 (ISMMS_VRE_11)\n",
      "Indexing GCF_001750885.1 (E745)\n",
      "Indexing GCF_001886635.1 (E1)\n",
      "Indexing GCF_001895905.1 (VRE001)\n",
      "Indexing GCF_001953235.1 (ISMMS_VRE_12)\n",
      "Indexing GCF_001953255.1 (ISMMS_VRE_9)\n",
      "Indexing GCF_002007625.1 (2014-VREF-41)\n",
      "Indexing GCF_002024245.1 (2014-VREF-114)\n",
      "Indexing GCF_002025045.1 (2014-VREF-268)\n",
      "Indexing GCF_002025065.1 (2014-VREF-63)\n",
      "Indexing GCF_002334625.1 (K60-39)\n",
      "Indexing GCF_002442955.1 (A_020709_82)\n",
      "Indexing GCF_002761255.1 (E240)\n",
      "Indexing GCF_002761275.1 (E243)\n",
      "Indexing GCF_002761555.1 (16-346)\n",
      "Indexing GCF_002777275.1 (E232)\n",
      "Indexing GCF_002831505.1 (LS170308)\n",
      "Indexing GCF_002848385.1 (SC4)\n",
      "Indexing GCF_002848625.1 (Efaecium_ER03933.3A)\n",
      "Indexing GCF_002848645.1 (Efaecium_ER04120.3A)\n",
      "Indexing GCF_002848665.1 (Efaecium_ER04484.3A)\n",
      "Indexing GCF_002848685.1 (Efaecium_ER04462.3A)\n",
      "Indexing GCF_002848705.1 (Efaecium_ER04619.3A)\n",
      "Indexing GCF_002848725.1 (Efaecium_ER04526.5A)\n",
      "Indexing GCF_002848745.1 (Efaecium_ER04526.3A)\n",
      "Indexing GCF_002850515.1 (WEFA23)\n",
      "Indexing GCF_002880635.1 (AALTL)\n",
      "Indexing GCF_002973755.2 (15-307-1)\n",
      "Indexing GCF_002983785.1 (FDAARGOS_323)\n",
      "Indexing GCF_003020685.1 (AUSMDU00004167)\n",
      "Indexing GCF_003020705.1 (AUSMDU00004055)\n",
      "Indexing GCF_003020725.1 (AUSMDU00004028)\n",
      "Indexing GCF_003020745.1 (AUSMDU00004024)\n",
      "Indexing GCF_003020765.1 (AUSMDU00004142)\n",
      "Indexing GCF_003071425.1 (CFSAN059070)\n",
      "Indexing GCF_003071445.1 (CFSAN059071)\n",
      "Indexing GCF_003269465.1 (BM4105-RF)\n",
      "Indexing GCF_003574925.1 (HY07)\n",
      "Indexing GCF_003667965.1 (Enterococcus faecium JE1)\n",
      "Indexing GCF_003711605.1 (Gr17)\n",
      "Indexing GCF_003957785.1 (RBWH1)\n",
      "Indexing GCF_004015145.1 (CBA7134)\n",
      "Indexing GCF_004101385.1 (SRCM103341)\n",
      "Indexing GCF_004103475.1 (SRCM103470)\n",
      "Indexing GCF_004332055.1 (FSIS1608820)\n",
      "Indexing GCF_005166365.1 (NM213)\n",
      "Indexing GCF_005517315.1 (VB3025)\n",
      "Indexing GCF_005576735.1 (VB3240)\n",
      "Indexing GCF_005886545.1 (UAMSEF_01)\n",
      "Indexing GCF_005886655.1 (UAMSEF_08)\n",
      "Indexing GCF_005886715.1 (UAMSEF_09)\n",
      "Indexing GCF_005886735.1 (UAMSEF_20)\n",
      "Indexing GCF_005952885.1 (HOU503)\n",
      "Indexing GCF_006007925.1 (VRE1)\n",
      "Indexing GCF_006280355.1 (F17E0263)\n",
      "Indexing GCF_006337045.1 (DB-1)\n",
      "Indexing GCF_006337145.1 (HB-1)\n",
      "Indexing GCF_006351785.1 (FB-1)\n",
      "Indexing GCF_006351845.1 (N56454)\n",
      "Indexing GCF_006575625.1 (515)\n",
      "Indexing GCF_006741355.1 (QU 50)\n",
      "Indexing GCF_007917035.2 (VVEswe-R)\n",
      "Indexing GCF_007917315.3 (VVEswe-S)\n",
      "Indexing GCF_007923785.2 (YN2-1)\n",
      "Indexing GCF_007923865.2 (SC3-1)\n",
      "Indexing GCF_007923885.2 (SDGJP3)\n",
      "Indexing GCF_007923905.2 (HN11)\n",
      "Indexing GCF_007923925.2 (HB2-2)\n",
      "Indexing GCF_007923945.2 (GZ8)\n",
      "Indexing GCF_007923965.2 (SDGJQ5)\n",
      "Indexing GCF_008000855.1 (FA3)\n",
      "Indexing GCF_008330605.1 (DMEA02)\n",
      "Indexing GCF_008728455.1 (V1836)\n",
      "Indexing GCF_008728475.1 (V2937)\n",
      "Indexing GCF_009036045.1 (LAC7.2)\n",
      "Indexing GCF_009697285.1 (VRE)\n",
      "Indexing GCF_009938075.1 (ZY11)\n",
      "Indexing GCF_009938285.1 (KUHS13)\n",
      "Indexing GCF_010120755.1 (ZY2)\n",
      "Indexing GCF_011745645.1 (DT1-1)\n",
      "Indexing GCF_012045365.1 (BIOPOP-3 ALE)\n",
      "Indexing GCF_012045505.1 (BIOPOP-3 WT)\n",
      "Indexing GCF_012932975.2 (BA17124)\n",
      "Indexing GCF_012932985.2 (BP5067)\n",
      "Indexing GCF_012933055.2 (BP3378)\n",
      "Indexing GCF_012933075.2 (BP657)\n",
      "Indexing GCF_012933165.2 (A7214)\n",
      "Indexing GCF_012933195.2 (A6521)\n",
      "Indexing GCF_012933245.2 (A4694)\n",
      "Indexing GCF_012933265.2 (A3895)\n",
      "Indexing GCF_012933285.2 (A11051)\n",
      "Indexing GCF_012933295.2 (A15023)\n",
      "Indexing GCF_012933345.2 (A10290)\n",
      "Indexing GCF_013201055.1 (FS 86)\n",
      "Indexing GCF_014490015.1 (AML0157)\n",
      "Indexing GCF_014874615.1 (VB3338)\n",
      "Indexing GCF_015325925.1 (VRE3370)\n",
      "Indexing GCF_015377765.1 (TK-P5D)\n",
      "Indexing GCF_015476295.1 (VRE3363)\n",
      "Indexing GCF_015767695.1 (KCCM 12118)\n",
      "Indexing GCF_015999405.1 (VRE3389)\n",
      "Indexing GCF_015999425.1 (VRE3382)\n",
      "Indexing GCF_015999605.1 (Dallas 1)\n",
      "Indexing GCF_016126675.1 (Dallas 5)\n",
      "Indexing GCF_016406345.1 (Dallas 91)\n",
      "Indexing GCF_016406365.1 (Dallas 53_1)\n",
      "Indexing GCF_016406385.1 (Dallas 55)\n",
      "Indexing GCF_016406405.1 (Dallas 66)\n",
      "Indexing GCF_016406425.1 (Dallas 83)\n",
      "Indexing GCF_016406445.1 (Dallas 17_1)\n",
      "Indexing GCF_016406465.1 (Dallas 51_4)\n",
      "Indexing GCF_016406485.1 (Dallas 16_1)\n",
      "Indexing GCF_016406505.1 (Dallas 87_1)\n",
      "Indexing GCF_016406525.1 (Dallas 71_2)\n",
      "Indexing GCF_016406545.1 (Dallas 53_2)\n",
      "Indexing GCF_016406565.1 (Dallas 57)\n",
      "Indexing GCF_016406585.1 (Dallas 93_3)\n",
      "Indexing GCF_016406605.1 (Dallas 93_2)\n",
      "Indexing GCF_016415025.1 (Dallas 97_1)\n",
      "Indexing GCF_016415105.1 (Dallas 103)\n",
      "Indexing GCF_016415285.1 (Dallas 100_1)\n",
      "Indexing GCF_016415325.1 (Dallas 107_1)\n",
      "Indexing GCF_016415345.1 (Dallas 111)\n",
      "Indexing GCF_016415365.1 (Dallas 124_3)\n",
      "Indexing GCF_016415385.1 (Dallas 124_1)\n",
      "Indexing GCF_016415405.1 (Dallas 144_1)\n",
      "Indexing GCF_016415425.1 (Dallas 148)\n",
      "Indexing GCF_016415445.1 (Dallas 137)\n",
      "Indexing GCF_016415465.1 (Dallas 131_2)\n",
      "Indexing GCF_016415485.1 (Dallas 163_1)\n",
      "Indexing GCF_016415505.1 (Dallas 155)\n",
      "Indexing GCF_016415545.1 (Dallas 154_1)\n",
      "Indexing GCF_016415565.1 (Dallas 158)\n",
      "Indexing GCF_016743855.1 (fac90)\n",
      "Indexing GCF_016864255.1 (WGS1811-4-7)\n",
      "Indexing GCF_017584065.1 (VRE3355)\n",
      "Indexing GCF_017603725.1 (VB13828)\n",
      "Indexing GCF_017815655.1 (VB12993)\n",
      "Indexing GCF_017815675.1 (VB3895)\n",
      "Indexing GCF_017815695.1 (VB976)\n",
      "Indexing GCF_017897965.1 (VB6171)\n",
      "Indexing GCF_017898005.1 (VB6521)\n",
      "Indexing GCF_017898025.1 (VB3378)\n",
      "Indexing GCF_018219285.1 (PR05720-3)\n",
      "Indexing GCF_018219325.1 (PR01996-12)\n",
      "Indexing GCF_018279145.1 (116)\n",
      "Indexing GCF_018516845.1 (18-465)\n",
      "Indexing GCF_018516925.1 (18-276)\n",
      "Indexing GCF_018517045.1 (17-508)\n",
      "Indexing GCF_018517065.1 (17-318)\n",
      "Indexing GCF_018517085.1 (16-164)\n",
      "Indexing GCF_018517105.1 (16-021)\n",
      "Indexing GCF_018517185.1 (18-042)\n",
      "Indexing GCF_019175425.1 (F88)\n",
      "Indexing GCF_019175445.1 (F39)\n",
      "Indexing GCF_019175465.1 (F179)\n",
      "Indexing GCF_019175525.1 (AVS0243)\n",
      "Indexing GCF_019356355.1 (7150)\n",
      "Indexing GCF_019456555.1 (VBO96)\n",
      "Indexing GCF_019456575.1 (VBO39)\n",
      "Indexing GCF_019456595.1 (VBR48)\n",
      "Indexing GCF_019977495.1 (AA620)\n",
      "Indexing GCF_019977575.1 (AA622)\n",
      "Indexing GCF_020091325.1 (HJP554)\n",
      "Indexing GCF_020162155.1 (V1225)\n",
      "Indexing GCF_020162175.1 (V1164)\n",
      "Indexing GCF_020221735.2 (4995/20)\n",
      "Indexing GCF_020736585.1 (SC1762)\n",
      "Indexing GCF_020736625.1 (SC1762-D)\n",
      "Indexing GCF_021172105.3 (V13-21-E11-012-001)\n",
      "Indexing GCF_021228615.1 (NMVRE-001)\n",
      "Indexing GCF_021398505.1 (H7)\n",
      "Indexing GCF_021560195.1 (P47)\n",
      "Indexing GCF_021713095.1 (UK043)\n",
      "Indexing GCF_021899455.1 (UK040)\n",
      "Indexing GCF_022647825.1 (UAMS_EF58)\n",
      "Indexing GCF_022647945.1 (UAMS_EF57)\n",
      "Indexing GCF_022648125.1 (UAMS_EF55)\n",
      "Indexing GCF_022691445.1 (VRE-WC072)\n",
      "Indexing GCF_022699225.1 (NY2032)\n",
      "Indexing GCF_022699545.1 (NY11067)\n",
      "Indexing GCF_022699565.1 (NY11066)\n",
      "Indexing GCF_022749455.1 (NY13320)\n",
      "Indexing GCF_023182815.2 (AKSZ-47)\n",
      "Indexing GCF_023182835.2 (AKSZ-142)\n",
      "Indexing GCF_023204955.1 (UCH1)\n",
      "Indexing GCF_023204975.1 (UCH5)\n",
      "Indexing GCF_023299325.1 (AT47b)\n",
      "Indexing GCF_023299725.1 (AT45b)\n",
      "Indexing GCF_023299745.1 (AT44)\n",
      "Indexing GCF_023299805.1 (AT02)\n",
      "Indexing GCF_023375205.1 (S2-6b)\n",
      "Indexing GCF_023375225.1 (S39-7a)\n",
      "Indexing GCF_023375345.1 (1818)\n",
      "Indexing GCF_023375365.1 (211b)\n",
      "Indexing GCF_023375385.1 (264a)\n",
      "Indexing GCF_023375405.1 (1525)\n",
      "Indexing GCF_023375425.1 (642)\n",
      "Indexing GCF_023375445.1 (237)\n",
      "Indexing GCF_023658035.1 (ef332)\n",
      "Indexing GCF_024665675.1 (ENCfa-68)\n",
      "Indexing GCF_025800545.1 (ATF64)\n",
      "Indexing GCF_025800625.1 (BTF71-1)\n",
      "Indexing GCF_025800645.1 (IC25)\n",
      "Indexing GCF_025913595.1 (TF51-2)\n",
      "Indexing GCF_025916235.1 (T17-1)\n",
      "Indexing GCF_025998395.1 (NUITM-VRE1)\n",
      "Indexing GCF_026073115.1 (JHP9)\n",
      "Indexing GCF_026073415.1 (JHP80)\n",
      "Indexing GCF_026073435.1 (AA55)\n",
      "Indexing GCF_026073455.1 (AA94)\n",
      "Indexing GCF_026073625.1 (GK923)\n",
      "Indexing GCF_026073735.1 (GK941)\n",
      "Indexing GCF_026073915.1 (GK961)\n",
      "Indexing GCF_026409165.1 (DVT1681)\n",
      "Indexing GCF_026427575.1 (WN-18)\n",
      "Indexing GCF_027286245.1 (EF0656)\n",
      "Indexing GCF_027915235.1 (DMEA09)\n",
      "Indexing GCF_027944535.1 (CQFYH22-006)\n",
      "Indexing GCF_028335205.1 (K195-6b)\n",
      "Indexing GCF_028335225.1 (K191-3)\n",
      "Indexing GCF_028335245.1 (R39-1)\n",
      "Indexing GCF_028335265.1 (R186-2)\n",
      "Indexing GCF_028335465.1 (K69-1a)\n",
      "Indexing GCF_028335485.1 (K188-5)\n",
      "Indexing GCF_028335525.1 (K162-1)\n",
      "Indexing GCF_028335565.1 (K75-1)\n",
      "Indexing GCF_028335585.1 (K70-1a)\n",
      "Indexing GCF_028335605.1 (K189-3)\n",
      "Indexing GCF_028335625.1 (K80-15b)\n",
      "Indexing GCF_028891525.1 (W6)\n",
      "Indexing GCF_029023785.1 (DSM 20477)\n",
      "Indexing GCF_029167665.1 (GENOMIC22-005)\n",
      "Indexing GCF_030064735.1 (XJ78NG)\n",
      "Indexing GCF_030295785.1 (K6D)\n",
      "Indexing GCF_030316155.1 (A2)\n",
      "Indexing GCF_030406065.1 (AK_C_05)\n",
      "Indexing GCF_030581675.1 (C0019EM0007)\n",
      "Indexing GCF_030581775.1 (C0019EM0014)\n",
      "Indexing GCF_030581795.1 (C0019EM0008)\n",
      "Indexing GCF_030584205.1 (A0019EM0021)\n",
      "Indexing GCF_030584225.1 (A0019EM0020)\n",
      "Indexing GCF_030584245.1 (A0019EM0022)\n",
      "Indexing GCF_030584265.1 (A0019EM0035)\n",
      "Indexing GCF_030584285.1 (B0019EM0006)\n",
      "Indexing GCF_030584305.1 (B0019EM0004)\n",
      "Indexing GCF_030584325.1 (B0019EM0014)\n",
      "Indexing GCF_030584345.1 (B0019EM0012)\n",
      "Indexing GCF_030584365.1 (B0019EM0015)\n",
      "Indexing GCF_030584385.1 (B0019EM0017)\n",
      "Indexing GCF_030584405.1 (B0019EM0019)\n",
      "Indexing GCF_030584425.1 (B0019EM0026)\n",
      "Indexing GCF_030584445.1 (B0019EM0030)\n",
      "Indexing GCF_030584465.1 (B0019EM0022)\n",
      "Indexing GCF_030584485.1 (B0019EM0028)\n",
      "Indexing GCF_030584505.1 (B0019EM0023)\n",
      "Indexing GCF_030584525.1 (B0019EM0032)\n",
      "Indexing GCF_030584545.1 (B0019EM0016)\n",
      "Indexing GCF_030584585.1 (B0019EM0040)\n",
      "Indexing GCF_030584605.1 (C0019EM0016)\n",
      "Indexing GCF_030584625.1 (A0019EM0008)\n",
      "Indexing GCF_030584645.1 (A0019EM0010)\n",
      "Indexing GCF_030584665.1 (A0019EM0007)\n",
      "Indexing GCF_030584685.1 (B0019EM0034)\n",
      "Indexing GCF_030643745.1 (CX2-6)\n",
      "Indexing GCF_030644745.1 (B0019EM0038)\n",
      "Indexing GCF_030644765.1 (B0019EM0045)\n",
      "Indexing GCF_030644785.1 (B0019EM0041)\n",
      "Indexing GCF_030644805.1 (C0019EM0002)\n",
      "Indexing GCF_030644825.1 (C0019EM0006)\n",
      "Indexing GCF_030719015.1 (TC64_3x4995_20_TV8)\n",
      "Indexing GCF_900044005.1 (EFE11651)\n",
      "Indexing GCF_900066025.1 (EFE10021)\n",
      "Indexing GCF_900092475.1 (Ef_aus00233)\n",
      "Indexing GCF_900094185.1 (Ef_DMG1500501)\n",
      "Indexing GCF_900634805.1 (E0139)\n",
      "Indexing GCF_900635365.1 (E1334)\n",
      "Indexing GCF_900635415.1 (E6043)\n",
      "Indexing GCF_900637035.1 (NCTC7174)\n",
      "Indexing GCF_900638765.1 (E2079)\n",
      "Indexing GCF_900638775.1 (E4227)\n",
      "Indexing GCF_900638785.1 (E1774)\n",
      "Indexing GCF_900638805.1 (E4402)\n",
      "Indexing GCF_900638815.1 (E0595)\n",
      "Indexing GCF_900638825.1 (E4438)\n",
      "Indexing GCF_900639335.1 (E7067)\n",
      "Indexing GCF_900639345.1 (E6055)\n",
      "Indexing GCF_900639355.1 (E7025)\n",
      "Indexing GCF_900639365.1 (E7171)\n",
      "Indexing GCF_900639385.1 (E7040)\n",
      "Indexing GCF_900639395.1 (E6975)\n",
      "Indexing GCF_900639405.1 (E4457)\n",
      "Indexing GCF_900639415.1 (E6988)\n",
      "Indexing GCF_900639425.1 (E7098)\n",
      "Indexing GCF_900639445.1 (E7199)\n",
      "Indexing GCF_900639455.1 (E7237)\n",
      "Indexing GCF_900639465.1 (E7429)\n",
      "Indexing GCF_900639485.1 (E7240)\n",
      "Indexing GCF_900639495.1 (E7471)\n",
      "Indexing GCF_900639505.1 (E7356)\n",
      "Indexing GCF_900639515.1 (E7654)\n",
      "Indexing GCF_900639525.1 (E7663)\n",
      "Indexing GCF_900639535.1 (E8202)\n",
      "Indexing GCF_900639545.1 (E7933)\n",
      "Indexing GCF_900639555.1 (E8172)\n",
      "Indexing GCF_900639565.1 (E8195)\n",
      "Indexing GCF_900639575.1 (E8014)\n",
      "Indexing GCF_900639585.1 (E7948)\n",
      "Indexing GCF_900639595.1 (E8377)\n",
      "Indexing GCF_900639605.1 (E8290)\n",
      "Indexing GCF_900639615.1 (E8328)\n",
      "Indexing GCF_900639625.1 (E8284)\n",
      "Indexing GCF_900639635.1 (E8927)\n",
      "Indexing GCF_900639645.1 (E8440)\n",
      "Indexing GCF_900639655.1 (E8691)\n",
      "Indexing GCF_900639665.1 (E7114)\n",
      "Indexing GCF_900639705.1 (E9101)\n",
      "Indexing GCF_900639715.1 (E8414)\n",
      "Indexing GCF_900639725.1 (E8423)\n",
      "Indexing GCF_900639745.1 (E4456)\n",
      "Indexing GCF_900640265.1 (E4239)\n",
      "Indexing GCF_900683475.1 (3012STDY6244127)\n",
      "Indexing GCF_907163255.1 (AUS2001)\n",
      "Indexing GCF_907163315.1 (AUS2002)\n",
      "Indexing GCF_907165365.1 (USZ_VRE32_P32)\n",
      "Indexing GCF_907176135.1 (USZ_VRE5_P5)\n",
      "Indexing GCF_907176815.1 (USZ_VRE67_P60)\n",
      "Indexing GCF_907177285.1 (USZ_VRE53_P46)\n",
      "Indexing GCA_001298485.1 (64/3)\n",
      "Indexing GCA_001412695.1 (UW7606x64/3 TC1)\n",
      "Indexing GCA_001518735.1 (6E6)\n",
      "Indexing GCA_001587115.1 (UW8175)\n",
      "Indexing GCA_001594345.1 (ATCC 700221)\n",
      "Indexing GCA_001635875.1 (E39)\n",
      "Indexing GCA_001720945.1 (ISMMS_VRE_1)\n",
      "Indexing GCA_001721065.1 (ISMMS_VRE_7)\n",
      "Indexing GCA_001721905.1 (ISMMS_VRE_11)\n",
      "Indexing GCA_001750885.1 (E745)\n",
      "Indexing GCA_001886635.1 (E1)\n",
      "Indexing GCA_001895905.1 (VRE001)\n",
      "Indexing GCA_001953235.1 (ISMMS_VRE_12)\n",
      "Indexing GCA_001953255.1 (ISMMS_VRE_9)\n",
      "Indexing GCA_002007625.1 (2014-VREF-41)\n",
      "Indexing GCA_002024245.1 (2014-VREF-114)\n",
      "Indexing GCA_002025045.1 (2014-VREF-268)\n",
      "Indexing GCA_002025065.1 (2014-VREF-63)\n",
      "Indexing GCA_002334625.1 (K60-39)\n",
      "Indexing GCA_002442955.1 (A_020709_82)\n",
      "Indexing GCA_002761255.1 (E240)\n",
      "Indexing GCA_002761275.1 (E243)\n",
      "Indexing GCA_002761555.1 (16-346)\n",
      "Indexing GCA_002777275.1 (E232)\n",
      "Indexing GCA_002831505.1 (LS170308)\n",
      "Indexing GCA_002848385.1 (SC4)\n",
      "Indexing GCA_002848625.1 (Efaecium_ER03933.3A)\n",
      "Indexing GCA_002848645.1 (Efaecium_ER04120.3A)\n",
      "Indexing GCA_002848665.1 (Efaecium_ER04484.3A)\n",
      "Indexing GCA_002848685.1 (Efaecium_ER04462.3A)\n",
      "Indexing GCA_002848705.1 (Efaecium_ER04619.3A)\n",
      "Indexing GCA_002848725.1 (Efaecium_ER04526.5A)\n",
      "Indexing GCA_002848745.1 (Efaecium_ER04526.3A)\n",
      "Indexing GCA_002850515.1 (WEFA23)\n",
      "Indexing GCA_002880635.1 (AALTL)\n",
      "Indexing GCA_002973755.2 (15-307-1)\n",
      "Indexing GCA_002983785.1 (FDAARGOS_323)\n",
      "Indexing GCA_003020685.1 (AUSMDU00004167)\n",
      "Indexing GCA_003020705.1 (AUSMDU00004055)\n",
      "Indexing GCA_003020725.1 (AUSMDU00004028)\n",
      "Indexing GCA_003020745.1 (AUSMDU00004024)\n",
      "Indexing GCA_003020765.1 (AUSMDU00004142)\n",
      "Indexing GCA_003071425.1 (CFSAN059070)\n",
      "Indexing GCA_003071445.1 (CFSAN059071)\n",
      "Indexing GCA_003269465.1 (BM4105-RF)\n",
      "Indexing GCA_003574925.1 (HY07)\n",
      "Indexing GCA_003667965.1 (JE1)\n",
      "Indexing GCA_003711605.1 (Gr17)\n",
      "Indexing GCA_003957785.1 (RBWH1)\n",
      "Indexing GCA_004015145.1 (CBA7134)\n",
      "Indexing GCA_004101385.1 (SRCM103341)\n",
      "Indexing GCA_004103475.1 (SRCM103470)\n",
      "Indexing GCA_004332055.1 (FSIS1608820)\n",
      "Indexing GCA_005166365.1 (NM213)\n",
      "Indexing GCA_005517315.1 (VB3025)\n",
      "Indexing GCA_005576735.1 (VB3240)\n",
      "Indexing GCA_005886545.1 (UAMSEF_01)\n",
      "Indexing GCA_005886655.1 (UAMSEF_08)\n",
      "Indexing GCA_005886715.1 (UAMSEF_09)\n",
      "Indexing GCA_005886735.1 (UAMSEF_20)\n",
      "Indexing GCA_005952885.1 (HOU503)\n",
      "Indexing GCA_006007925.1 (VRE1)\n",
      "Indexing GCA_006280355.1 (F17E0263)\n",
      "Indexing GCA_006337045.1 (DB-1)\n",
      "Indexing GCA_006337145.1 (HB-1)\n",
      "Indexing GCA_006351785.1 (FB-1)\n",
      "Indexing GCA_006351845.1 (N56454)\n",
      "Indexing GCA_006575625.1 (515)\n",
      "Indexing GCA_006741355.1 (QU 50)\n",
      "Indexing GCA_007917035.3 (VVEswe-R)\n",
      "Indexing GCA_007917315.3 (VVEswe-S)\n",
      "Indexing GCA_007923785.2 (YN2-1)\n",
      "Indexing GCA_007923865.2 (SC3-1)\n",
      "Indexing GCA_007923885.2 (SDGJP3)\n",
      "Indexing GCA_007923905.2 (HN11)\n",
      "Indexing GCA_007923925.2 (HB2-2)\n",
      "Indexing GCA_007923945.2 (GZ8)\n",
      "Indexing GCA_007923965.2 (SDGJQ5)\n",
      "Indexing GCA_008000855.1 (FA3)\n",
      "Indexing GCA_008330605.1 (DMEA02)\n",
      "Indexing GCA_008728455.1 (V1836)\n",
      "Indexing GCA_008728475.1 (V2937)\n",
      "Indexing GCA_009036045.1 (LAC7.2)\n",
      "Indexing GCA_009697285.1 (VRE)\n",
      "Indexing GCA_009734005.2 (SRR24)\n",
      "Indexing GCA_009938075.1 (ZY11)\n",
      "Indexing GCA_009938285.1 (KUHS13)\n",
      "Indexing GCA_010120755.1 (ZY2)\n",
      "Indexing GCA_011745645.1 (DT1-1)\n",
      "Indexing GCA_012045365.1 (BIOPOP-3 ALE)\n",
      "Indexing GCA_012045505.1 (BIOPOP-3 WT)\n",
      "Indexing GCA_012932975.2 (BA17124)\n",
      "Indexing GCA_012932985.2 (BP5067)\n",
      "Indexing GCA_012933055.2 (BP3378)\n",
      "Indexing GCA_012933075.2 (BP657)\n",
      "Indexing GCA_012933165.2 (A7214)\n",
      "Indexing GCA_012933195.2 (A6521)\n",
      "Indexing GCA_012933245.2 (A4694)\n",
      "Indexing GCA_012933265.2 (A3895)\n",
      "Indexing GCA_012933285.2 (A11051)\n",
      "Indexing GCA_012933295.2 (A15023)\n",
      "Indexing GCA_012933345.2 (A10290)\n",
      "Indexing GCA_013201055.1 (FS 86)\n",
      "Indexing GCA_014490015.1 (AML0157)\n",
      "Indexing GCA_014874615.1 (VB3338)\n",
      "Indexing GCA_015325925.1 (VRE3370)\n",
      "Indexing GCA_015356095.1 (Cairo)\n",
      "Indexing GCA_015377765.1 (TK-P5D)\n",
      "Indexing GCA_015476295.1 (VRE3363)\n",
      "Indexing GCA_015767695.1 (KCCM 12118)\n",
      "Indexing GCA_015999405.1 (VRE3389)\n",
      "Indexing GCA_015999425.1 (VRE3382)\n",
      "Indexing GCA_015999605.1 (Dallas 1)\n",
      "Indexing GCA_016126675.1 (Dallas 5)\n",
      "Indexing GCA_016406345.1 (Dallas 91)\n",
      "Indexing GCA_016406365.1 (Dallas 53_1)\n",
      "Indexing GCA_016406385.1 (Dallas 55)\n",
      "Indexing GCA_016406405.1 (Dallas 66)\n",
      "Indexing GCA_016406425.1 (Dallas 83)\n",
      "Indexing GCA_016406445.1 (Dallas 17_1)\n",
      "Indexing GCA_016406465.1 (Dallas 51_4)\n",
      "Indexing GCA_016406485.1 (Dallas 16_1)\n",
      "Indexing GCA_016406505.1 (Dallas 87_1)\n",
      "Indexing GCA_016406525.1 (Dallas 71_2)\n",
      "Indexing GCA_016406545.1 (Dallas 53_2)\n",
      "Indexing GCA_016406565.1 (Dallas 57)\n",
      "Indexing GCA_016406585.1 (Dallas 93_3)\n",
      "Indexing GCA_016406605.1 (Dallas 93_2)\n",
      "Indexing GCA_016415025.1 (Dallas 97_1)\n",
      "Indexing GCA_016415105.1 (Dallas 103)\n",
      "Indexing GCA_016415285.1 (Dallas 100_1)\n",
      "Indexing GCA_016415325.1 (Dallas 107_1)\n",
      "Indexing GCA_016415345.1 (Dallas 111)\n",
      "Indexing GCA_016415365.1 (Dallas 124_3)\n",
      "Indexing GCA_016415385.1 (Dallas 124_1)\n",
      "Indexing GCA_016415405.1 (Dallas 144_1)\n",
      "Indexing GCA_016415425.1 (Dallas 148)\n",
      "Indexing GCA_016415445.1 (Dallas 137)\n",
      "Indexing GCA_016415465.1 (Dallas 131_2)\n",
      "Indexing GCA_016415485.1 (Dallas 163_1)\n",
      "Indexing GCA_016415505.1 (Dallas 155)\n",
      "Indexing GCA_016415545.1 (Dallas 154_1)\n",
      "Indexing GCA_016415565.1 (Dallas 158)\n",
      "Indexing GCA_016743855.1 (fac90)\n",
      "Indexing GCA_016864255.1 (WGS1811-4-7)\n",
      "Indexing GCA_017301355.1 (AUSMDU00011555)\n",
      "Indexing GCA_017584065.1 (VRE3355)\n",
      "Indexing GCA_017603725.1 (VB13828)\n",
      "Indexing GCA_017815655.1 (VB12993)\n",
      "Indexing GCA_017815675.1 (VB3895)\n",
      "Indexing GCA_017815695.1 (VB976)\n",
      "Indexing GCA_017897965.1 (VB6171)\n",
      "Indexing GCA_017898005.1 (VB6521)\n",
      "Indexing GCA_017898025.1 (VB3378)\n",
      "Indexing GCA_018219285.1 (PR05720-3)\n",
      "Indexing GCA_018219325.1 (PR01996-12)\n",
      "Indexing GCA_018279145.1 (116)\n",
      "Indexing GCA_018516845.1 (18-465)\n",
      "Indexing GCA_018516925.1 (18-276)\n",
      "Indexing GCA_018517025.1 (18-133)\n",
      "Indexing GCA_018517045.1 (17-508)\n",
      "Indexing GCA_018517065.1 (17-318)\n",
      "Indexing GCA_018517085.1 (16-164)\n",
      "Indexing GCA_018517105.1 (16-021)\n",
      "Indexing GCA_018517145.1 (18-204)\n",
      "Indexing GCA_018517165.1 (18-201)\n",
      "Indexing GCA_018517185.1 (18-042)\n",
      "Indexing GCA_019175425.1 (F88)\n",
      "Indexing GCA_019175445.1 (F39)\n",
      "Indexing GCA_019175465.1 (F179)\n",
      "Indexing GCA_019175525.1 (AVS0243)\n",
      "Indexing GCA_019356355.1 (7150)\n",
      "Indexing GCA_019456555.1 (VBO96)\n",
      "Indexing GCA_019456575.1 (VBO39)\n",
      "Indexing GCA_019456595.1 (VBR48)\n",
      "Indexing GCA_019774555.1 (E843xGE-1-TC1)\n",
      "Indexing GCA_019977495.1 (AA620)\n",
      "Indexing GCA_019977575.1 (AA622)\n",
      "Indexing GCA_020091325.1 (HJP554)\n",
      "Indexing GCA_020162155.1 (V1225)\n",
      "Indexing GCA_020162175.1 (V1164)\n",
      "Indexing GCA_020221735.2 (4995/20)\n",
      "Indexing GCA_020736585.1 (SC1762)\n",
      "Indexing GCA_020736625.1 (SC1762-D)\n",
      "Indexing GCA_021172105.3 (V13-21-E11-012-001)\n",
      "Indexing GCA_021228615.1 (NMVRE-001)\n",
      "Indexing GCA_021398505.1 (H7)\n",
      "Indexing GCA_021560195.1 (P47)\n",
      "Indexing GCA_021713095.1 (UK043)\n",
      "Indexing GCA_021899455.1 (UK040)\n",
      "Indexing GCA_022647825.1 (UAMS_EF58)\n",
      "Indexing GCA_022647945.1 (UAMS_EF57)\n",
      "Indexing GCA_022648125.1 (UAMS_EF55)\n",
      "Indexing GCA_022691445.1 (VRE-WC072)\n",
      "Indexing GCA_022699225.1 (NY2032)\n",
      "Indexing GCA_022699545.1 (NY11067)\n",
      "Indexing GCA_022699565.1 (NY11066)\n",
      "Indexing GCA_022749455.1 (NY13320)\n",
      "Indexing GCA_022811085.1 (BMT-5-1-10)\n",
      "Indexing GCA_022811125.1 (BMT-2-2-11)\n",
      "Indexing GCA_022811145.1 (BMT-4-1-1)\n",
      "Indexing GCA_022811165.1 (ICU-1-2)\n",
      "Indexing GCA_022811185.1 (BMT-1-1-10)\n",
      "Indexing GCA_022811205.1 (ICU-3-1)\n",
      "Indexing GCA_022811225.1 (BMT-1-1-11)\n",
      "Indexing GCA_022811245.1 (BMT-1-1-12)\n",
      "Indexing GCA_022811265.1 (BMT-4-1-11)\n",
      "Indexing GCA_022811285.1 (BMT-2-1-1)\n",
      "Indexing GCA_022811305.1 (BMT-1-3-1)\n",
      "Indexing GCA_022811325.1 (BMT-3-2-1)\n",
      "Indexing GCA_022811345.1 (BMT-2-3-8)\n",
      "Indexing GCA_022811365.1 (BMT-2-1-3)\n",
      "Indexing GCA_022811385.1 (ICU-1-1)\n",
      "Indexing GCA_022811405.1 (BMT-4-1-3)\n",
      "Indexing GCA_022811425.1 (ICU-4-1)\n",
      "Indexing GCA_022811445.1 (ICU-5-1)\n",
      "Indexing GCA_022811465.1 (ICU-2-1)\n",
      "Indexing GCA_023182815.2 (AKSZ-47)\n",
      "Indexing GCA_023182835.2 (AKSZ-142)\n",
      "Indexing GCA_023204955.1 (UCH1)\n",
      "Indexing GCA_023204975.1 (UCH5)\n",
      "Indexing GCA_023299325.1 (AT47b)\n",
      "Indexing GCA_023299725.1 (AT45b)\n",
      "Indexing GCA_023299745.1 (AT44)\n",
      "Indexing GCA_023299805.1 (AT02)\n",
      "Indexing GCA_023375205.1 (S2-6b)\n",
      "Indexing GCA_023375225.1 (S39-7a)\n",
      "Indexing GCA_023375345.1 (1818)\n",
      "Indexing GCA_023375365.1 (211b)\n",
      "Indexing GCA_023375385.1 (264a)\n",
      "Indexing GCA_023375405.1 (1525)\n",
      "Indexing GCA_023375425.1 (642)\n",
      "Indexing GCA_023375445.1 (237)\n",
      "Indexing GCA_023658035.1 (ef332)\n",
      "Indexing GCA_024665675.1 (ENCfa-68)\n",
      "Indexing GCA_025800545.1 (ATF64)\n",
      "Indexing GCA_025800625.1 (BTF71-1)\n",
      "Indexing GCA_025800645.1 (IC25)\n",
      "Indexing GCA_025913595.1 (TF51-2)\n",
      "Indexing GCA_025916235.1 (T17-1)\n",
      "Indexing GCA_025998395.1 (NUITM-VRE1)\n",
      "Indexing GCA_026073115.1 (JHP9)\n",
      "Indexing GCA_026073255.1 (JHP10)\n",
      "Indexing GCA_026073355.1 (JHP35)\n",
      "Indexing GCA_026073375.1 (JHP36)\n",
      "Indexing GCA_026073395.1 (JHP38)\n",
      "Indexing GCA_026073415.1 (JHP80)\n",
      "Indexing GCA_026073435.1 (AA55)\n",
      "Indexing GCA_026073455.1 (AA94)\n",
      "Indexing GCA_026073475.1 (AA96)\n",
      "Indexing GCA_026073495.1 (AA242)\n",
      "Indexing GCA_026073515.1 (AA290)\n",
      "Indexing GCA_026073535.1 (AA610)\n",
      "Indexing GCA_026073555.1 (AA818)\n",
      "Indexing GCA_026073625.1 (GK923)\n",
      "Indexing GCA_026073735.1 (GK941)\n",
      "Indexing GCA_026073915.1 (GK961)\n",
      "Indexing GCA_026409165.1 (DVT1681)\n",
      "Indexing GCA_026427575.1 (WN-18)\n",
      "Indexing GCA_027286245.1 (EF0656)\n",
      "Indexing GCA_027915235.1 (DMEA09)\n",
      "Indexing GCA_027944535.1 (CQFYH22-006)\n",
      "Indexing GCA_028335205.1 (K195-6b)\n",
      "Indexing GCA_028335225.1 (K191-3)\n",
      "Indexing GCA_028335245.1 (R39-1)\n",
      "Indexing GCA_028335265.1 (R186-2)\n",
      "Indexing GCA_028335465.1 (K69-1a)\n",
      "Indexing GCA_028335485.1 (K188-5)\n",
      "Indexing GCA_028335525.1 (K162-1)\n",
      "Indexing GCA_028335565.1 (K75-1)\n",
      "Indexing GCA_028335585.1 (K70-1a)\n",
      "Indexing GCA_028335605.1 (K189-3)\n",
      "Indexing GCA_028335625.1 (K80-15b)\n",
      "Indexing GCA_028891525.1 (W6)\n",
      "Indexing GCA_029023785.1 (DSM 20477)\n",
      "Indexing GCA_029167665.1 (GENOMIC22-005)\n",
      "Indexing GCA_030064735.1 (XJ78NG)\n",
      "Indexing GCA_030295785.1 (K6D)\n",
      "Indexing GCA_030316155.1 (A2)\n",
      "Indexing GCA_030406065.1 (AK_C_05)\n",
      "Indexing GCA_030581675.1 (C0019EM0007)\n",
      "Indexing GCA_030581775.1 (C0019EM0014)\n",
      "Indexing GCA_030581795.1 (C0019EM0008)\n",
      "Indexing GCA_030584205.1 (A0019EM0021)\n",
      "Indexing GCA_030584225.1 (A0019EM0020)\n",
      "Indexing GCA_030584245.1 (A0019EM0022)\n",
      "Indexing GCA_030584265.1 (A0019EM0035)\n",
      "Indexing GCA_030584285.1 (B0019EM0006)\n",
      "Indexing GCA_030584305.1 (B0019EM0004)\n",
      "Indexing GCA_030584325.1 (B0019EM0014)\n",
      "Indexing GCA_030584345.1 (B0019EM0012)\n",
      "Indexing GCA_030584365.1 (B0019EM0015)\n",
      "Indexing GCA_030584385.1 (B0019EM0017)\n",
      "Indexing GCA_030584405.1 (B0019EM0019)\n",
      "Indexing GCA_030584425.1 (B0019EM0026)\n",
      "Indexing GCA_030584445.1 (B0019EM0030)\n",
      "Indexing GCA_030584465.1 (B0019EM0022)\n",
      "Indexing GCA_030584485.1 (B0019EM0028)\n",
      "Indexing GCA_030584505.1 (B0019EM0023)\n",
      "Indexing GCA_030584525.1 (B0019EM0032)\n",
      "Indexing GCA_030584545.1 (B0019EM0016)\n",
      "Indexing GCA_030584585.1 (B0019EM0040)\n",
      "Indexing GCA_030584605.1 (C0019EM0016)\n",
      "Indexing GCA_030584625.1 (A0019EM0008)\n",
      "Indexing GCA_030584645.1 (A0019EM0010)\n",
      "Indexing GCA_030584665.1 (A0019EM0007)\n",
      "Indexing GCA_030584685.1 (B0019EM0034)\n",
      "Indexing GCA_030643745.1 (CX2-6)\n",
      "Indexing GCA_030644745.1 (B0019EM0038)\n",
      "Indexing GCA_030644765.1 (B0019EM0045)\n",
      "Indexing GCA_030644785.1 (B0019EM0041)\n",
      "Indexing GCA_030644805.1 (C0019EM0002)\n",
      "Indexing GCA_030644825.1 (C0019EM0006)\n",
      "Indexing GCA_030656695.1 (C0019EM0037)\n",
      "Indexing GCA_030656715.1 (C0019EM0019)\n",
      "Indexing GCA_030656735.1 (C0019EM0018)\n",
      "Indexing GCA_030656755.1 (C0019EM0029)\n",
      "Indexing GCA_030656775.1 (C0019EM0020)\n",
      "Indexing GCA_030656795.1 (C0019EM0030)\n",
      "Indexing GCA_030656815.1 (C0019EM0021)\n",
      "Indexing GCA_030656835.1 (C0019EM0028)\n",
      "Indexing GCA_030656855.1 (C0019EM0031)\n",
      "Indexing GCA_030656895.1 (D0019EM0001)\n",
      "Indexing GCA_030656915.1 (D0019EM0013)\n",
      "Indexing GCA_030656935.1 (D0019EM0009)\n",
      "Indexing GCA_030656955.1 (D0019EM0012)\n",
      "Indexing GCA_030656975.1 (C0019EM0033)\n",
      "Indexing GCA_030656995.1 (C0019EM0032)\n",
      "Indexing GCA_030657015.1 (D0019EM0008)\n",
      "Indexing GCA_030657035.1 (C0019EM0035)\n",
      "Indexing GCA_030657055.1 (F0019EM0001)\n",
      "Indexing GCA_030657075.1 (F0019EM0006)\n",
      "Indexing GCA_030657095.1 (F0019EM0008)\n",
      "Indexing GCA_030657115.1 (F0019EM0017)\n",
      "Indexing GCA_030657135.1 (F0019EM0011)\n",
      "Indexing GCA_030657155.1 (F0019EM0041)\n",
      "Indexing GCA_030657175.1 (D0019EM0023)\n",
      "Indexing GCA_030657195.1 (D0019EM0024)\n",
      "Indexing GCA_030657215.1 (D0019EM0026)\n",
      "Indexing GCA_030657235.1 (D0019EM0029)\n",
      "Indexing GCA_030657255.1 (E0019EM0001)\n",
      "Indexing GCA_030657275.1 (E0019EM0002)\n",
      "Indexing GCA_030657295.1 (E0019EM0007)\n",
      "Indexing GCA_030657315.1 (E0019EM0009)\n",
      "Indexing GCA_030664785.1 (E0019EM0016)\n",
      "Indexing GCA_030664805.1 (E0019EM0013)\n",
      "Indexing GCA_030664825.1 (E0019EM0020)\n",
      "Indexing GCA_030664845.1 (E0019EM0026)\n",
      "Indexing GCA_030664865.1 (E0019EM0072)\n",
      "Indexing GCA_030664885.1 (E0019EM0077)\n",
      "Indexing GCA_030664905.1 (F0019EM0002)\n",
      "Indexing GCA_030664925.1 (E0019EM0059)\n",
      "Indexing GCA_030664945.1 (E0019EM0047)\n",
      "Indexing GCA_030664965.1 (E0019EM0079)\n",
      "Indexing GCA_030664985.1 (E0019EM0078)\n",
      "Indexing GCA_030665005.1 (E0019EM0024)\n",
      "Indexing GCA_030665025.1 (E0019EM0028)\n",
      "Indexing GCA_030665045.1 (E0019EM0042)\n",
      "Indexing GCA_030665065.1 (E0019EM0021)\n",
      "Indexing GCA_030665085.1 (F0019EM0005)\n",
      "Indexing GCA_030665105.1 (F0019EM0004)\n",
      "Indexing GCA_030665125.1 (F0019EM0024)\n",
      "Indexing GCA_030665145.1 (F0019EM0020)\n",
      "Indexing GCA_030665165.1 (F0019EM0021)\n",
      "Indexing GCA_030665185.1 (F0019EM0026)\n",
      "Indexing GCA_030665205.1 (F0019EM0027)\n",
      "Indexing GCA_030665225.1 (F0019EM0032)\n",
      "Indexing GCA_030665245.1 (F0019EM0031)\n",
      "Indexing GCA_030665265.1 (F0019EM0033)\n",
      "Indexing GCA_030665285.1 (F0019EM0035)\n",
      "Indexing GCA_030665305.1 (F0019EM0037)\n",
      "Indexing GCA_030665325.1 (F0019EM0030)\n",
      "Indexing GCA_030665345.1 (F0019EM0039)\n",
      "Indexing GCA_030665365.1 (F0019EM0049)\n",
      "Indexing GCA_030665385.1 (F0019EM0056)\n",
      "Indexing GCA_030665405.1 (F0019EM0050)\n",
      "Indexing GCA_030685595.1 (G0019EM0003)\n",
      "Indexing GCA_030685655.1 (D0019EM0022)\n",
      "Indexing GCA_030685675.1 (G0019EM0011)\n",
      "Indexing GCA_030685695.1 (G0019EM0012)\n",
      "Indexing GCA_030685715.1 (G0019EM0023)\n",
      "Indexing GCA_030685735.1 (G0019EM0022)\n",
      "Indexing GCA_030685755.1 (H0019EM0007)\n",
      "Indexing GCA_030685775.1 (H0019EM0009)\n",
      "Indexing GCA_030685795.1 (H0019EM0003)\n",
      "Indexing GCA_030685815.1 (G0019EM0027)\n",
      "Indexing GCA_030685835.1 (H0019EM0006)\n",
      "Indexing GCA_030685855.1 (H0019EM0013)\n",
      "Indexing GCA_030685875.1 (G0019EM0001)\n",
      "Indexing GCA_030685895.1 (G0019EM0002)\n",
      "Indexing GCA_030685915.1 (G0019EM0008)\n",
      "Indexing GCA_030685935.1 (G0019EM0010)\n",
      "Indexing GCA_030685955.1 (G0019EM0019)\n",
      "Indexing GCA_030685975.1 (G0019EM0017)\n",
      "Indexing GCA_030685995.1 (G0019EM0006)\n",
      "Indexing GCA_030686015.1 (G0019EM0013)\n",
      "Indexing GCA_030686035.1 (G0019EM0015)\n",
      "Indexing GCA_030686055.1 (G0019EM0021)\n",
      "Indexing GCA_030686075.1 (H0019EM0021)\n",
      "Indexing GCA_030686095.1 (H0019EM0023)\n",
      "Indexing GCA_030719015.1 (TC64_3x4995_20_TV8)\n",
      "Indexing GCA_900044005.1 (EFE11651)\n",
      "Indexing GCA_900066025.1 (EFE10021)\n",
      "Indexing GCA_900092475.1 (Ef_aus00233)\n",
      "Indexing GCA_900094185.1 (Ef_DMG1500501)\n",
      "Indexing GCA_900634805.1 (E0139)\n",
      "Indexing GCA_900635365.1 (E1334)\n",
      "Indexing GCA_900635415.1 (E6043)\n",
      "Indexing GCA_900637035.1 (NCTC7174)\n",
      "Indexing GCA_900638765.1 (E2079)\n",
      "Indexing GCA_900638775.1 (E4227)\n",
      "Indexing GCA_900638785.1 (E1774)\n",
      "Indexing GCA_900638805.1 (E4402)\n",
      "Indexing GCA_900638815.1 (E0595)\n",
      "Indexing GCA_900638825.1 (E4438)\n",
      "Indexing GCA_900639335.1 (E7067)\n",
      "Indexing GCA_900639345.1 (E6055)\n",
      "Indexing GCA_900639355.1 (E7025)\n",
      "Indexing GCA_900639365.1 (E7171)\n",
      "Indexing GCA_900639385.1 (E7040)\n",
      "Indexing GCA_900639395.1 (E6975)\n",
      "Indexing GCA_900639405.1 (E4457)\n",
      "Indexing GCA_900639415.1 (E6988)\n",
      "Indexing GCA_900639425.1 (E7098)\n",
      "Indexing GCA_900639445.1 (E7199)\n",
      "Indexing GCA_900639455.1 (E7237)\n",
      "Indexing GCA_900639465.1 (E7429)\n",
      "Indexing GCA_900639485.1 (E7240)\n",
      "Indexing GCA_900639495.1 (E7471)\n",
      "Indexing GCA_900639505.1 (E7356)\n",
      "Indexing GCA_900639515.1 (E7654)\n",
      "Indexing GCA_900639525.1 (E7663)\n",
      "Indexing GCA_900639535.1 (E8202)\n",
      "Indexing GCA_900639545.1 (E7933)\n",
      "Indexing GCA_900639555.1 (E8172)\n",
      "Indexing GCA_900639565.1 (E8195)\n",
      "Indexing GCA_900639575.1 (E8014)\n",
      "Indexing GCA_900639585.1 (E7948)\n",
      "Indexing GCA_900639595.1 (E8377)\n",
      "Indexing GCA_900639605.1 (E8290)\n",
      "Indexing GCA_900639615.1 (E8328)\n",
      "Indexing GCA_900639625.1 (E8284)\n",
      "Indexing GCA_900639635.1 (E8927)\n",
      "Indexing GCA_900639645.1 (E8440)\n",
      "Indexing GCA_900639655.1 (E8691)\n",
      "Indexing GCA_900639665.1 (E7114)\n",
      "Indexing GCA_900639705.1 (E9101)\n",
      "Indexing GCA_900639715.1 (E8414)\n",
      "Indexing GCA_900639725.1 (E8423)\n",
      "Indexing GCA_900639745.1 (E4456)\n",
      "Indexing GCA_900640265.1 (E4239)\n",
      "Indexing GCA_900683475.1 (3012STDY6244127)\n",
      "Indexing GCA_907163255.1 (AUS2001)\n",
      "Indexing GCA_907163315.1 (AUS2002)\n",
      "Indexing GCA_907165365.1 (USZ_VRE32_P32)\n",
      "Indexing GCA_907176135.1 (USZ_VRE5_P5)\n",
      "Indexing GCA_907176815.1 (USZ_VRE67_P60)\n",
      "Indexing GCA_907177285.1 (USZ_VRE53_P46)\n",
      "Indexing GCF_000336405.1 (NRRL B-2354)\n",
      "Indexing GCA_000336405.1 (NRRL B-2354)\n",
      "Indexing GCF_000250945.1 (Aus0004)\n",
      "Indexing GCA_000250945.1 (Aus0004)\n",
      "Indexing GCF_000444405.1 (AUS0085)\n",
      "Indexing GCA_000444405.1 (AUS0085)\n",
      "Indexing GCF_000174395.2 (DO)\n",
      "Indexing GCA_000174395.2 (DO)\n",
      "Indexing GCF_000737555.1 (T110)\n",
      "Indexing GCA_000737555.1 (T110)\n",
      "Indexing GCF_001558875.2 (FDAARGOS_163)\n",
      "Indexing GCF_009730155.1 (FDAARGOS_728)\n",
      "Indexing GCF_014844195.1 (EGM181)\n",
      "Indexing GCF_021496385.1 (EG81)\n",
      "Indexing GCF_023375325.1 (674)\n",
      "Indexing GCF_025263125.1 (fecal_iso1)\n",
      "Indexing GCF_028335185.1 (K205-4a)\n",
      "Indexing GCA_001558875.2 (FDAARGOS_163)\n",
      "Indexing GCA_009730155.1 (FDAARGOS_728)\n",
      "Indexing GCA_014844195.1 (EGM181)\n",
      "Indexing GCA_021496385.1 (EG81)\n",
      "Indexing GCA_023375325.1 (674)\n",
      "Indexing GCA_025263125.1 (fecal_iso1)\n",
      "Indexing GCA_028335185.1 (K205-4a)\n",
      "Indexing GCF_003343305.1 (CR1)\n",
      "Indexing GCA_003343305.1 (CR1)\n",
      "Indexing GCF_002278015.2 (FDAARGOS_234)\n",
      "Indexing GCF_001641305.1 (R17)\n",
      "Indexing GCF_012222705.1 (CQP3-9)\n",
      "Indexing GCF_013391145.1 (13344)\n",
      "Indexing GCF_013391165.1 (13152)\n",
      "Indexing GCF_013391185.1 (13144)\n",
      "Indexing GCF_013391205.1 (708)\n",
      "Indexing GCF_014218175.1 (HDC14-2)\n",
      "Indexing GCF_016127235.1 (FDAARGOS_1057)\n",
      "Indexing GCF_016727125.1 (FDAARGOS_1124)\n",
      "Indexing GCF_016727265.1 (FDAARGOS_1125)\n",
      "Indexing GCF_019175505.1 (F104)\n",
      "Indexing GCF_023373765.1 (GDIAS-5)\n",
      "Indexing GCF_023375155.1 (S2-7)\n",
      "Indexing GCF_023375175.1 (S46-3)\n",
      "Indexing GCF_023375305.1 (697)\n",
      "Indexing GCF_025913615.1 (T16-1)\n",
      "Indexing GCF_025913635.1 (TF14-3)\n",
      "Indexing GCF_025913655.1 (TF34)\n",
      "Indexing GCF_025913675.1 (TF52)\n",
      "Indexing GCF_029814255.1 (DSM 20160)\n",
      "Indexing GCF_030295825.1 (L6E)\n",
      "Indexing GCF_900636615.1 (NCTC12368)\n",
      "Indexing GCA_001641305.1 (R17)\n",
      "Indexing GCA_002278015.2 (FDAARGOS_234)\n",
      "Indexing GCA_012222705.1 (CQP3-9)\n",
      "Indexing GCA_013391145.1 (13344)\n",
      "Indexing GCA_013391165.1 (13152)\n",
      "Indexing GCA_013391185.1 (13144)\n",
      "Indexing GCA_013391205.1 (708)\n",
      "Indexing GCA_014218175.1 (HDC14-2)\n",
      "Indexing GCA_016127235.1 (FDAARGOS_1057)\n",
      "Indexing GCA_016727125.1 (FDAARGOS_1124)\n",
      "Indexing GCA_016727265.1 (FDAARGOS_1125)\n",
      "Indexing GCA_019175505.1 (F104)\n",
      "Indexing GCA_023373765.1 (GDIAS-5)\n",
      "Indexing GCA_023375155.1 (S2-7)\n",
      "Indexing GCA_023375175.1 (S46-3)\n",
      "Indexing GCA_023375305.1 (697)\n",
      "Indexing GCA_025913615.1 (T16-1)\n",
      "Indexing GCA_025913635.1 (TF14-3)\n",
      "Indexing GCA_025913655.1 (TF34)\n",
      "Indexing GCA_025913675.1 (TF52)\n",
      "Indexing GCA_029814255.1 (DSM 20160)\n",
      "Indexing GCA_030295825.1 (L6E)\n",
      "Indexing GCA_900636615.1 (NCTC12368)\n",
      "Indexing GCF_000271405.2 (ATCC 9790)\n",
      "Indexing GCA_000271405.2 (ATCC 9790)\n",
      "Indexing GCA_905186115.1 (Enterococcus hirae canine HQ MAG)\n",
      "Indexing GCF_022846515.1 (DB-1)\n",
      "Indexing GCA_022846515.1 (DB-1)\n",
      "Indexing GCF_019343125.1 (CX 2-6_2)\n",
      "Indexing GCF_015767715.1 (KCTC 21015)\n",
      "Indexing GCF_019203145.1 (JDM1)\n",
      "Indexing GCF_019880345.1 (E843)\n",
      "Indexing GCF_019967715.1 (HJS001)\n",
      "Indexing GCF_020268645.1 (DH9003)\n",
      "Indexing GCF_023612275.1 (IDCC 2105)\n",
      "Indexing GCF_030490365.1 (SU-B46)\n",
      "Indexing GCA_015767715.1 (KCTC 21015)\n",
      "Indexing GCA_019203145.1 (JDM1)\n",
      "Indexing GCA_019343125.1 (CX 2-6_2)\n",
      "Indexing GCA_019880345.1 (E843)\n",
      "Indexing GCA_019967715.1 (HJS001)\n",
      "Indexing GCA_020268645.1 (DH9003)\n",
      "Indexing GCA_023612275.1 (IDCC 2105)\n",
      "Indexing GCA_030490365.1 (SU-B46)\n",
      "Indexing GCF_002813755.1 (DSM 4838)\n",
      "Indexing GCF_002216705.1 (EMB156)\n",
      "Indexing GCF_003957375.1 (Pe161)\n",
      "Indexing GCF_005158285.1 (Pe103)\n",
      "Indexing GCF_013394305.1 (15-1A)\n",
      "Indexing GCF_020149485.1 (P6)\n",
      "Indexing GCF_931972305.1 (P2005)\n",
      "Indexing GCA_002216705.1 (EMB156)\n",
      "Indexing GCA_002813755.1 (DSM 4838)\n",
      "Indexing GCA_003957375.1 (Pe161)\n",
      "Indexing GCA_005158285.1 (Pe103)\n",
      "Indexing GCA_013394305.1 (15-1A)\n",
      "Indexing GCA_020149485.1 (P6)\n",
      "Indexing GCA_931972305.1 (P2005)\n",
      "Indexing GCF_000504125.1 (QU25)\n",
      "Indexing GCA_000504125.1 (QU25)\n",
      "Indexing GCF_019175485.1 (F162_2)\n",
      "Indexing GCF_019797905.1 (CX012922)\n",
      "Indexing GCF_025309655.1 (HG-5)\n",
      "Indexing GCF_025402795.1 (Er676)\n",
      "Indexing GCF_025402855.1 (ATCC 49464)\n",
      "Indexing GCA_019175485.1 (F162_2)\n",
      "Indexing GCA_019797905.1 (CX012922)\n",
      "Indexing GCA_025309655.1 (HG-5)\n",
      "Indexing GCA_025402795.1 (Er676)\n",
      "Indexing GCA_025402855.1 (ATCC 49464)\n",
      "Indexing GCF_029023825.1 (DSM 20726)\n",
      "Indexing GCA_029023825.1 (DSM 20726)\n",
      "Indexing GCF_011397115.1 (VE80)\n",
      "Indexing GCA_011397115.1 (VE80)\n",
      "Indexing GCA_001465115.1 (LMG 23085)\n",
      "Indexing GCF_029983095.1 (CoE-012-22)\n",
      "Indexing GCA_029983095.1 (CoE-012-22)\n",
      "Indexing GCF_002850555.1 (CR-Ec1)\n",
      "Indexing GCA_002850555.1 (CR-Ec1)\n",
      "Indexing GCF_008000875.1 (DA9)\n",
      "Indexing GCA_008000875.1 (DA9)\n",
      "Indexing GCF_002393265.1 (FDAARGOS_375)\n",
      "Indexing GCA_002393265.1 (FDAARGOS_375)\n",
      "Indexing GCF_003812305.1 (FDAARGOS_553)\n",
      "Indexing GCA_003812305.1 (FDAARGOS_553)\n",
      "Indexing GCF_030263915.1 (FZMF)\n",
      "Indexing GCA_030263915.1 (FZMF)\n",
      "Indexing GCF_023539285.1 (LX10)\n",
      "Indexing GCA_023539285.1 (LX10)\n",
      "Indexing GCF_005843965.1 (M190262)\n",
      "Indexing GCA_005843965.1 (M190262)\n",
      "Indexing GCA_023168365.1 (PLM3)\n",
      "Indexing GCF_030053755.1 (ZQ21)\n",
      "Indexing GCA_030053755.1 (ZQ21)\n",
      "Indexing GCF_002290025.1 (a523)\n",
      "Indexing GCA_002290025.1 (a523)\n",
      "Indexing GCF_002197645.1 (MN05)\n",
      "Indexing GCA_002197645.1 (MN05)\n",
      "Indexing GCF_004001205.1 (DAT585)\n",
      "Indexing GCF_009731445.1 (DAT1033)\n",
      "Indexing GCA_004001205.1 (DAT585)\n",
      "Indexing GCA_009731445.1 (DAT1033)\n",
      "Indexing GCF_000283975.1 (DAT561)\n",
      "Indexing GCA_000283975.1 (DAT561)\n",
      "Indexing GCF_000747585.1 (S1)\n",
      "Indexing GCA_000747585.1 (S1)\n",
      "Indexing GCF_001712815.1 (MJ4)\n",
      "Indexing GCF_002243535.1 (KUD23)\n",
      "Indexing GCF_003795105.1 (LMG 26042)\n",
      "Indexing GCF_009720125.1 (YJ1)\n",
      "Indexing GCF_030388485.1 (SNTH-1)\n",
      "Indexing GCA_001712815.1 (MJ4)\n",
      "Indexing GCA_002243535.1 (KUD23)\n",
      "Indexing GCA_003795105.1 (LMG 26042)\n",
      "Indexing GCA_009720125.1 (YJ1)\n",
      "Indexing GCA_019222785.1 (CY54)\n",
      "Indexing GCA_030388485.1 (SNTH-1)\n",
      "Indexing GCF_000283615.1 (NBRC 12172)\n",
      "Indexing GCA_000283615.1 (NBRC 12172)\n",
      "Indexing GCF_003795145.1 (KCTC 3924)\n",
      "Indexing GCA_003795145.1 (KCTC 3924)\n",
      "Indexing GCF_003795125.1 (JCM 31126)\n",
      "Indexing GCA_003795125.1 (JCM 31126)\n",
      "Indexing GCF_014397115.1 (ATCC BAA-640)\n",
      "Indexing GCA_014397115.1 (ATCC BAA-640)\n",
      "Indexing GCF_011303955.1 (HDW17A)\n",
      "Indexing GCA_011303955.1 (HDW17A)\n",
      "Indexing GCF_020628455.1 (36B2)\n",
      "Indexing GCF_020628355.1 (110B2)\n",
      "Indexing GCF_020628555.1 (35B2)\n",
      "Indexing GCF_020628665.1 (25B2)\n",
      "Indexing GCF_020628735.1 (12B2)\n",
      "Indexing GCA_020628355.1 (110B2)\n",
      "Indexing GCA_020628455.1 (36B2)\n",
      "Indexing GCA_020628555.1 (35B2)\n",
      "Indexing GCA_020628665.1 (25B2)\n",
      "Indexing GCA_020628735.1 (12B2)\n",
      "Indexing GCF_011304195.1 (HDW17B)\n",
      "Indexing GCA_011304195.1 (HDW17B)\n",
      "Indexing GCF_029144185.1 (STAA11)\n",
      "Indexing GCF_029144405.1 (STAA25)\n",
      "Indexing GCA_029144185.1 (STAA11)\n",
      "Indexing GCA_029144405.1 (STAA25)\n",
      "Indexing GCF_024637875.1 (G314F)\n",
      "Indexing GCA_024637875.1 (G314F)\n",
      "Indexing GCF_023299185.1 (AT46b)\n",
      "Indexing GCF_019797785.1 (BN31)\n",
      "Indexing GCF_023299345.1 (AT19)\n",
      "Indexing GCF_023299445.1 (AT33)\n",
      "Indexing GCF_023299565.1 (AT15)\n",
      "Indexing GCF_023299765.1 (AT32)\n",
      "Indexing GCF_023300045.1 (AT03)\n",
      "Indexing GCF_028335165.1 (K136-2)\n",
      "Indexing GCF_028335285.1 (K204-1)\n",
      "Indexing GCF_029011965.1 (VF20)\n",
      "Indexing GCA_019797785.1 (BN31)\n",
      "Indexing GCA_023299185.1 (AT46b)\n",
      "Indexing GCA_023299345.1 (AT19)\n",
      "Indexing GCA_023299445.1 (AT33)\n",
      "Indexing GCA_023299565.1 (AT15)\n",
      "Indexing GCA_023299765.1 (AT32)\n",
      "Indexing GCA_023300045.1 (AT03)\n",
      "Indexing GCA_028335165.1 (K136-2)\n",
      "Indexing GCA_028335285.1 (K204-1)\n",
      "Indexing GCA_029011965.1 (VF20)\n",
      "Indexing GCF_001998885.1 (CD276T)\n",
      "Indexing GCA_001998885.1 (CD276)\n",
      "Indexing GCF_022655055.1 (CY52-2)\n",
      "Indexing GCA_022655055.1 (CY52-2)\n",
      "Indexing GCF_001870205.1 (DSM 21459)\n",
      "Indexing GCA_001870205.1 (DSM 21459)\n",
      "Indexing GCF_004792515.1 (CF-49)\n",
      "Indexing GCA_004792515.1 (personal::cf-49)\n",
      "Indexing GCF_005145005.1 (MN-17)\n",
      "Indexing GCA_005145005.1 (MN-17)\n"
     ]
    }
   ],
   "source": [
    "!bash download_ncbi2.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e24765a8-e03c-4d00-8850-a01d1deb9e60",
   "metadata": {},
   "source": [
    "### Step 2: Build Blast database."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "edb80e45-2eaf-43da-87c3-915a20538eb0",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[*] Creating Blast database.\n",
      "Target fasta file: __tmp_refseqs.fasta\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003711125.1/NZ_CP024590.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005347505.1/NZ_CP034169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013394325.1/NZ_AP019814.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003711125.1/CP024590.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005347505.1/CP034169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013394325.1/AP019814.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003641225.1/NZ_CP032739.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009707345.1/NZ_CP046123.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014844215.1/NZ_CP050490.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016127635.1/NZ_CP066024.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016727305.1/NZ_CP068126.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016727325.1/NZ_CP068128.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016727345.1/NZ_CP068132.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022870765.1/NZ_CP086066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023523715.1/NZ_CP053180.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_027944555.1/NZ_CP116026.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029201225.1/NZ_CP119296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029215545.1/NZ_CP119393.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003641225.1/CP032739.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009707345.1/CP046123.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014844215.1/CP050490.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016127635.1/CP066024.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016727305.1/CP068126.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016727325.1/CP068128.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016727345.1/CP068132.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022870765.1/CP086066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023523715.1/CP053180.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_027944555.1/CP116026.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029201225.1/CP119296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029215545.1/CP119393.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000157355.2/NC_020995.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000157355.2/CP004856.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900474605.1/NZ_LS483306.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947054945.1/NZ_OX346404.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947054965.1/NZ_OX346401.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055055.1/NZ_OX346402.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055095.1/NZ_OX346406.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055105.1/NZ_OX346407.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055185.1/NZ_OX346405.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055195.1/NZ_OX346403.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055595.1/NZ_OX346408.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_947055965.1/NZ_OX346411.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900474605.1/LS483306.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947054945.1/OX346404.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947054965.1/OX346401.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055055.1/OX346402.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055095.1/OX346406.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055105.1/OX346407.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055185.1/OX346405.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055195.1/OX346403.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055595.1/OX346408.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_947055965.1/OX346411.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335365.1/NZ_CP116583.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335365.1/CP116583.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002277935.1/NZ_CP022930.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001267395.1/NZ_CP012366.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001267865.1/NZ_CP012384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015999505.1/NZ_CP065535.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375285.1/NZ_CP088203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335325.1/NZ_CP116581.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335645.1/NZ_CP116579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001267395.1/CP012366.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001267865.1/CP012384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002277935.1/CP022930.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015999505.1/CP065535.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375285.1/CP088203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335325.1/CP116581.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335645.1/CP116579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001598635.1/NZ_CP014949.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001689055.2/NZ_CP015410.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001989555.1/NZ_CP019512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002163735.1/NZ_CP015883.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002208945.1/NZ_CP022059.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002289045.2/NZ_CP029612.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002355755.1/NZ_AP017623.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003030425.1/NZ_CP028285.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003345275.1/NZ_CP031027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003812665.1/NZ_CP033787.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003966385.1/NZ_AP018538.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003966405.1/NZ_AP018543.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004332015.1/NZ_CP028720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004332035.1/NZ_CP028724.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005158025.1/NZ_CP028835.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005484525.1/NZ_CP039434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005979285.1/NZ_CP030042.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005979305.1/NZ_CP030045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006349345.1/NZ_CP040898.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006364815.1/NZ_CP041012.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006494835.1/NZ_CP039296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006494855.1/NZ_CP039548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006494875.1/NZ_CP039549.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006766145.1/NZ_CP041344.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009498175.3/NZ_CP042216.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_010103655.1/NZ_CP045045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_011193375.1/NZ_CP049775.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012594215.1/NZ_CP051005.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013344605.1/NZ_CP046108.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013344625.1/NZ_CP046111.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013344645.1/NZ_CP046112.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013344665.1/NZ_CP046113.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014489455.1/NZ_CP060804.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014844235.1/NZ_CP050491.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015377745.1/NZ_CP045598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016743895.1/NZ_CP068249.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016812095.1/NZ_CP069185.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017280035.1/NZ_CP071172.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017280235.1/NZ_CP071176.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017357805.1/NZ_CP046022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017639585.1/NZ_CP060796.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517125.1/NZ_CP065784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018986755.2/NZ_CP075604.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019195415.1/NZ_CP078015.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019222705.1/NZ_CP078162.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019774535.1/NZ_CP081505.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019856595.1/NZ_CP082231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020616775.1/NZ_CP085289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020616875.1/NZ_CP085291.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020616935.1/NZ_CP085294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020844125.1/NZ_CP063980.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020881975.1/NZ_CP086411.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020882035.1/NZ_CP086558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020882055.1/NZ_CP086566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020882075.1/NZ_CP086560.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020882095.1/NZ_CP086564.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020882115.1/NZ_CP086592.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020906605.1/NZ_CP086726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021484825.1/NZ_CP090558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021610105.1/NZ_CP091198.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021655635.1/NZ_AP025270.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212845.1/NZ_CP091884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212865.1/NZ_CP091886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212885.1/NZ_CP091889.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212905.1/NZ_CP091892.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212925.1/NZ_CP091895.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212945.1/NZ_CP091897.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212965.1/NZ_CP091899.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022212985.1/NZ_CP091901.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022213005.1/NZ_CP091904.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022213025.1/NZ_CP091906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022557275.1/NZ_CP092907.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022648285.1/NZ_CP076488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022648425.1/NZ_CP076493.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022648595.1/NZ_CP076498.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022691465.1/NZ_CP092574.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022691485.1/NZ_CP092576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022699585.1/NZ_CP093957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022810805.1/NZ_CP046247.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022869705.1/NZ_CP064374.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023182775.2/NZ_CP110289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299225.1/NZ_CP097066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299425.1/NZ_CP097010.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299465.1/NZ_CP097006.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299485.1/NZ_CP097004.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299505.1/NZ_CP097034.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299525.1/NZ_CP097002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299545.1/NZ_CP097056.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299585.1/NZ_CP097069.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299605.1/NZ_CP097020.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299625.1/NZ_CP097038.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299645.1/NZ_CP097008.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299665.1/NZ_CP097046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299685.1/NZ_CP097048.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299705.1/NZ_CP097042.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299785.1/NZ_CP097030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023300025.1/NZ_CP097031.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023300165.1/NZ_CP097059.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375245.1/NZ_CP088198.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375265.1/NZ_CP088200.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375465.1/NZ_CP091227.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375485.1/NZ_CP091231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375505.1/NZ_CP091240.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375525.1/NZ_CP091235.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375545.1/NZ_CP091237.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023523795.1/NZ_CP053181.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023612035.1/NZ_CP080759.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023650735.1/NZ_CP098025.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023702145.1/NZ_CP098418.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023746575.1/NZ_CP098743.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024267595.1/NZ_CP100596.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024579755.1/NZ_CP102065.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024579795.1/NZ_CP102070.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024927865.1/NZ_CP103863.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025957845.1/NZ_CP110019.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025957925.1/NZ_CP110020.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958025.1/NZ_CP110036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958065.1/NZ_CP110039.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958085.1/NZ_CP110040.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958125.1/NZ_CP110041.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958325.1/NZ_CP110047.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958445.1/NZ_CP110050.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958625.1/NZ_CP110053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958785.1/NZ_CP110054.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958805.1/NZ_CP110059.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958825.1/NZ_CP110063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958845.1/NZ_CP110068.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958865.1/NZ_CP110069.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958885.1/NZ_CP110071.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958905.1/NZ_CP110072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025958925.1/NZ_CP110074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026650885.1/NZ_CP085841.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026802365.1/NZ_CP113831.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026802385.1/NZ_CP113832.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_027942035.1/NZ_CP115992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028131645.1/NZ_AP026721.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335305.1/NZ_CP116569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335345.1/NZ_CP116571.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335425.1/NZ_CP116555.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335445.1/NZ_CP116559.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335505.1/NZ_CP116557.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335545.1/NZ_CP116561.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335665.1/NZ_CP116553.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335685.1/NZ_CP116564.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335705.1/NZ_CP116575.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335725.1/NZ_CP116566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028532705.1/NZ_CP116962.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028598965.2/NZ_CP121233.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028621735.1/NZ_CP117508.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028621755.1/NZ_CP117503.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028621775.1/NZ_CP117511.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028743535.1/NZ_CP117970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029000425.1/NZ_CP118755.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029011255.1/NZ_CP118048.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029024925.1/NZ_CP118962.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029167565.1/NZ_CP119159.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029278545.1/NZ_CP119528.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029866805.1/NZ_CP104856.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029910175.1/NZ_CP106757.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020155.1/NZ_CP124887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020175.1/NZ_CP124884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020195.1/NZ_CP124886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020215.1/NZ_CP124889.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020235.1/NZ_CP124890.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020255.1/NZ_CP124891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020425.1/NZ_CP124882.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020455.1/NZ_CP124893.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020475.1/NZ_CP124894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020495.1/NZ_CP124898.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020515.1/NZ_CP124900.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020535.1/NZ_CP124901.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020555.1/NZ_CP124905.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020575.1/NZ_CP124911.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020595.1/NZ_CP124906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020615.1/NZ_CP124908.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020645.1/NZ_CP124913.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020665.1/NZ_CP124916.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020685.1/NZ_CP124921.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020705.1/NZ_CP124923.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020725.1/NZ_CP124918.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030020745.1/NZ_CP124914.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030064715.1/NZ_CP109611.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030264755.1/NZ_CP127177.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030295445.1/NZ_AP027136.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030295805.1/NZ_AP027297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030295845.1/NZ_AP027302.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030376825.1/NZ_CP128464.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030512425.1/NZ_CP124778.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517125.1/NZ_CP124925.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517355.1/NZ_CP124928.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517385.1/NZ_CP124935.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517415.1/NZ_CP124939.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517445.1/NZ_CP124947.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517475.1/NZ_CP124950.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517505.1/NZ_CP124953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517535.1/NZ_CP124931.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517565.1/NZ_CP124960.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030517625.1/NZ_CP124956.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_031186115.1/NZ_CP133450.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_031206965.1/NZ_CP102120.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_031208705.1/NZ_CP102121.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900636775.1/NZ_LR134312.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_901553725.1/NZ_LR594051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_902379005.2/NZ_LR698844.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120545.1/NZ_LR961961.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120555.1/NZ_LR961959.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120565.1/NZ_LR961923.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120575.1/NZ_LR961970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120585.1/NZ_LR961953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120595.1/NZ_LR961924.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120605.1/NZ_LR961933.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120615.1/NZ_LR961966.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120625.1/NZ_LR961980.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120635.1/NZ_LR961948.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120645.1/NZ_LR961975.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120655.1/NZ_LR961942.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120665.1/NZ_LR961938.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120675.1/NZ_LR961925.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120685.1/NZ_LR961935.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120695.1/NZ_LR961945.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120705.1/NZ_LR961951.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120715.1/NZ_LR961957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120725.1/NZ_LR961965.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120735.1/NZ_LR961978.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120745.1/NZ_LR961981.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120755.1/NZ_LR962008.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120765.1/NZ_LR961997.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120775.1/NZ_LR961999.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120785.1/NZ_LR961991.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120795.1/NZ_LR962016.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120805.1/NZ_LR962002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120815.1/NZ_LR961986.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120825.1/NZ_LR962013.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120835.1/NZ_LR961994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120845.1/NZ_LR961989.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120855.1/NZ_LR961996.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120865.1/NZ_LR962010.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120875.1/NZ_LR962004.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120885.1/NZ_LR962021.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120895.1/NZ_LR962019.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120905.1/NZ_LR962023.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120915.1/NZ_LR962025.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120925.1/NZ_LR962029.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120935.1/NZ_LR962017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120945.1/NZ_LR962051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120955.1/NZ_LR962063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120965.1/NZ_LR962056.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120975.1/NZ_LR962053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120985.1/NZ_LR962072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905120995.1/NZ_LR962060.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121005.1/NZ_LR962030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121015.1/NZ_LR962068.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121025.1/NZ_LR962049.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121035.1/NZ_LR962046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121045.1/NZ_LR962034.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121055.1/NZ_LR962036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121065.1/NZ_LR962039.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121075.1/NZ_LR962043.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121085.1/NZ_LR962075.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121095.1/NZ_LR962074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121105.1/NZ_LR962085.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121115.1/NZ_LR962078.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121125.1/NZ_LR962083.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121135.1/NZ_LR962110.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121145.1/NZ_LR962117.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121155.1/NZ_LR962098.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121165.1/NZ_LR962093.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121175.1/NZ_LR962119.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121185.1/NZ_LR962107.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121195.1/NZ_LR962086.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121205.1/NZ_LR962103.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121215.1/NZ_LR962101.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121225.1/NZ_LR962112.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121235.1/NZ_LR962091.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121245.1/NZ_LR962089.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121255.1/NZ_LR962115.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121265.1/NZ_LR962097.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121275.1/NZ_LR962129.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121285.1/NZ_LR962125.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121295.1/NZ_LR962123.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121305.1/NZ_LR962121.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121315.1/NZ_LR962137.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121325.1/NZ_LR962132.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121335.1/NZ_LR962165.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121345.1/NZ_LR962141.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121355.1/NZ_LR962146.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121365.1/NZ_LR962157.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121375.1/NZ_LR962161.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121385.1/NZ_LR962158.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121395.1/NZ_LR962163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121405.1/NZ_LR962144.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121415.1/NZ_LR962169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121425.1/NZ_LR962171.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121435.1/NZ_LR962155.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121445.1/NZ_LR962138.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121455.1/NZ_LR962152.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121465.1/NZ_LR962149.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121475.1/NZ_LR962174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121485.1/NZ_LR962178.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121495.1/NZ_LR962182.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121505.1/NZ_LR962180.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121515.1/NZ_LR962194.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121525.1/NZ_LR962230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121535.1/NZ_LR962197.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121545.1/NZ_LR962217.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121555.1/NZ_LR962227.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121565.1/NZ_LR962211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121575.1/NZ_LR962190.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121585.1/NZ_LR962188.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121595.1/NZ_LR962200.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121605.1/NZ_LR962189.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121615.1/NZ_LR962213.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121625.1/NZ_LR962210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121635.1/NZ_LR962196.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121645.1/NZ_LR962203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121655.1/NZ_LR962185.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121665.1/NZ_LR962225.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121675.1/NZ_LR962234.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121685.1/NZ_LR962239.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121695.1/NZ_LR962231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121705.1/NZ_LR962236.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121715.1/NZ_LR962258.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121725.1/NZ_LR962264.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121735.1/NZ_LR962242.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121745.1/NZ_LR962254.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121755.1/NZ_LR962248.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121765.1/NZ_LR962277.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121775.1/NZ_LR962251.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121785.1/NZ_LR962256.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121795.1/NZ_LR962280.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121805.1/NZ_LR962275.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121815.1/NZ_LR962283.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121825.1/NZ_LR962271.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121835.1/NZ_LR962267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121845.1/NZ_LR962284.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121855.1/NZ_LR962300.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121865.1/NZ_LR962302.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121875.1/NZ_LR962316.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121885.1/NZ_LR962298.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121895.1/NZ_LR962296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121905.1/NZ_LR962288.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121915.1/NZ_LR962293.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121925.1/NZ_LR962306.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121935.1/NZ_LR962343.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121945.1/NZ_LR962328.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121955.1/NZ_LR962315.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121965.1/NZ_LR962331.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121975.1/NZ_LR962323.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121985.1/NZ_LR962311.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905121995.1/NZ_LR962336.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122005.1/NZ_LR962326.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122015.1/NZ_LR962340.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122025.1/NZ_LR962334.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122035.1/NZ_LR962319.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122045.1/NZ_LR962321.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122055.1/NZ_LR962348.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122065.1/NZ_LR962346.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122075.1/NZ_LR962351.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122085.1/NZ_LR962353.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122095.1/NZ_LR962356.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122105.1/NZ_LR962359.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122115.1/NZ_LR962361.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122125.1/NZ_LR962363.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122135.1/NZ_LR962372.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122145.1/NZ_LR962367.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122155.1/NZ_LR962375.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122165.1/NZ_LR962370.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122175.1/NZ_LR962393.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122185.1/NZ_LR962378.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122195.1/NZ_LR962395.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122205.1/NZ_LR962390.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122215.1/NZ_LR962400.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122225.1/NZ_LR962396.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122235.1/NZ_LR962386.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122245.1/NZ_LR962382.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122255.1/NZ_LR962407.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122265.1/NZ_LR962403.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122275.1/NZ_LR962415.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122285.1/NZ_LR962414.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122295.1/NZ_LR962421.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122305.1/NZ_LR962410.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122315.1/NZ_LR962423.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122325.1/NZ_LR962427.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122335.1/NZ_LR962412.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122345.1/NZ_LR962418.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122355.1/NZ_LR962426.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122365.1/NZ_LR962450.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122375.1/NZ_LR962432.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122385.1/NZ_LR962444.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122395.1/NZ_LR962447.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122405.1/NZ_LR962464.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122415.1/NZ_LR962454.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122425.1/NZ_LR962437.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122435.1/NZ_LR962461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122445.1/NZ_LR962428.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122455.1/NZ_LR962440.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122465.1/NZ_LR962434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122475.1/NZ_LR962486.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122485.1/NZ_LR962476.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122495.1/NZ_LR962480.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122505.1/NZ_LR962468.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122515.1/NZ_LR962474.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122525.1/NZ_LR962466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122535.1/NZ_LR962495.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122545.1/NZ_LR962497.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122555.1/NZ_LR962489.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122565.1/NZ_LR962523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122575.1/NZ_LR962512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122585.1/NZ_LR962507.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122595.1/NZ_LR962525.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122605.1/NZ_LR962502.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122615.1/NZ_LR962521.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122625.1/NZ_LR962517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122635.1/NZ_LR962526.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122645.1/NZ_LR962509.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122655.1/NZ_LR962519.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122665.1/NZ_LR962501.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122675.1/NZ_LR962548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122685.1/NZ_LR962541.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122695.1/NZ_LR962539.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122705.1/NZ_LR962531.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122715.1/NZ_LR962545.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122725.1/NZ_LR962542.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122735.1/NZ_LR962546.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122745.1/NZ_LR962550.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122755.1/NZ_LR962551.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122765.1/NZ_LR962554.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122775.1/NZ_LR962565.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122785.1/NZ_LR962579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122795.1/NZ_LR962574.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122805.1/NZ_LR962572.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122815.1/NZ_LR962569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122825.1/NZ_LR962558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122835.1/NZ_LR962567.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122845.1/NZ_LR962561.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122855.1/NZ_LR962559.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122865.1/NZ_LR962576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122875.1/NZ_LR962580.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122885.1/NZ_LR962589.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122895.1/NZ_LR962595.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122905.1/NZ_LR962584.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122915.1/NZ_LR962591.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122925.1/NZ_LR962586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122935.1/NZ_LR962600.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122945.1/NZ_LR962598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122955.1/NZ_LR962602.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122965.1/NZ_LR962611.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122975.1/NZ_LR962618.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122985.1/NZ_LR962634.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905122995.1/NZ_LR962608.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123005.1/NZ_LR962606.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123015.1/NZ_LR962604.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123025.1/NZ_LR962628.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123035.1/NZ_LR962617.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123045.1/NZ_LR962620.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123055.1/NZ_LR962615.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123065.1/NZ_LR962632.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123075.1/NZ_LR962626.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123085.1/NZ_LR962642.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123095.1/NZ_LR962636.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123115.1/NZ_LR962649.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123135.1/NZ_LR962647.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123155.1/NZ_LR962639.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123165.1/NZ_LR962670.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123175.1/NZ_LR962666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123185.1/NZ_LR962681.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123195.1/NZ_LR962671.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123205.1/NZ_LR962663.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123215.1/NZ_LR962656.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123225.1/NZ_LR962676.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123235.1/NZ_LR962680.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123245.1/NZ_LR962650.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123255.1/NZ_LR962658.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123265.1/NZ_LR962654.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123275.1/NZ_LR962687.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123285.1/NZ_LR962695.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123295.1/NZ_LR962684.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123305.1/NZ_LR962691.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123315.1/NZ_LR962694.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123325.1/NZ_LR962698.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123335.1/NZ_LR962703.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123345.1/NZ_LR962717.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123355.1/NZ_LR962714.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123365.1/NZ_LR962700.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123375.1/NZ_LR962721.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123385.1/NZ_LR962723.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123395.1/NZ_LR962702.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123405.1/NZ_LR962711.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123415.1/NZ_LR962708.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123425.1/NZ_LR962720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123435.1/NZ_LR962716.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123445.1/NZ_LR962726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123455.1/NZ_LR962731.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123465.1/NZ_LR962729.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123475.1/NZ_LR962738.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123485.1/NZ_LR962736.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123495.1/NZ_LR962732.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123525.1/NZ_LR962782.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123535.1/NZ_LR962771.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123545.1/NZ_LR962807.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123555.1/NZ_LR962805.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123565.1/NZ_LR962796.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123575.1/NZ_LR962788.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123585.1/NZ_LR962772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123595.1/NZ_LR962802.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123605.1/NZ_LR962784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123615.1/NZ_LR962820.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123625.1/NZ_LR962811.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123635.1/NZ_LR962775.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123645.1/NZ_LR962814.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123655.1/NZ_LR962792.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123665.1/NZ_LR962777.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123675.1/NZ_LR962806.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123685.1/NZ_LR962797.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123695.1/NZ_LR962818.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123705.1/NZ_LR962821.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123715.1/NZ_LR962825.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123725.1/NZ_LR962859.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123735.1/NZ_LR962827.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123745.1/NZ_LR962834.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123755.1/NZ_LR962830.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123765.1/NZ_LR962832.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123775.1/NZ_LR962857.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123785.1/NZ_LR962840.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123795.1/NZ_LR962836.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123805.1/NZ_LR962846.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123815.1/NZ_LR962862.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123825.1/NZ_LR962850.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123835.1/NZ_LR962843.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123845.1/NZ_LR962842.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123855.1/NZ_LR962826.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123865.1/NZ_LR962855.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_905123875.1/NZ_LR962852.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464835.1/NZ_OD940431.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464865.1/NZ_OD940420.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464875.1/NZ_OD940437.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464895.1/NZ_OD940422.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464915.2/NZ_OD940440.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_906464925.1/NZ_OD940434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001598635.1/CP014949.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001689055.2/CP015410.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001989555.1/CP019512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002163735.1/CP015883.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002208945.2/CP022059.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002289045.2/CP029612.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002355755.1/AP017623.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003030425.1/CP028285.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003345275.1/CP031027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003812665.1/CP033787.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003966385.1/AP018538.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003966405.1/AP018543.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004332015.1/CP028720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004332035.1/CP028724.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005158025.1/CP028835.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005484525.1/CP039434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005979285.1/CP030042.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005979305.1/CP030045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006349345.1/CP040898.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006364815.1/CP041012.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006494835.1/CP039296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006494855.1/CP039548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006494875.1/CP039549.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006766145.1/CP041344.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009498175.3/CP042216.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_010103655.1/CP045045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_011193375.1/CP049775.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012594215.1/CP051005.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013344605.1/CP046108.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013344625.1/CP046111.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013344645.1/CP046112.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013344665.1/CP046113.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014489455.1/CP060804.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014844235.1/CP050491.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015377745.1/CP045598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016743835.1/CP068242.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016743895.1/CP068249.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016812095.1/CP069185.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017280035.1/CP071172.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017280235.1/CP071176.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017357805.1/CP046022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017639585.1/CP060796.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517125.1/CP065784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018986755.2/CP075604.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019195415.1/CP078015.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019222705.1/CP078162.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019774535.1/CP081505.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019856595.1/CP082231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020616775.1/CP085289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020616875.1/CP085291.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020616935.1/CP085294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020844125.1/CP063980.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020881975.1/CP086411.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020882035.1/CP086558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020882055.1/CP086566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020882075.1/CP086560.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020882095.1/CP086564.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020882115.1/CP086592.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020906605.1/CP086726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021484825.1/CP090558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021610105.1/CP091198.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021655635.1/AP025270.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212845.1/CP091884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212865.1/CP091886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212885.1/CP091889.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212905.1/CP091892.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212925.1/CP091895.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212945.1/CP091897.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212965.1/CP091899.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022212985.1/CP091901.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022213005.1/CP091904.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022213025.1/CP091906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022557275.1/CP092907.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022648285.1/CP076488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022648425.1/CP076493.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022648595.1/CP076498.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022691465.1/CP092574.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022691485.1/CP092576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022699585.1/CP093957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022810805.1/CP046247.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022869705.1/CP064374.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023182775.2/CP110289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299225.1/CP097066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299425.1/CP097010.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299465.1/CP097006.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299485.1/CP097004.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299505.1/CP097034.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299525.1/CP097002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299545.1/CP097056.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299585.1/CP097069.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299605.1/CP097020.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299625.1/CP097038.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299645.1/CP097008.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299665.1/CP097046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299685.1/CP097048.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299705.1/CP097042.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299785.1/CP097030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023300025.1/CP097031.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023300165.1/CP097059.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375245.1/CP088198.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375265.1/CP088200.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375465.1/CP091227.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375485.1/CP091231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375505.1/CP091240.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375525.1/CP091235.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375545.1/CP091237.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023523795.1/CP053181.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023612035.1/CP080759.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023650735.1/CP098025.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023702145.1/CP098418.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023746575.1/CP098743.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024267595.1/CP100596.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024579755.1/CP102065.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024579795.1/CP102070.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024927865.1/CP103863.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025957845.1/CP110019.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025957925.1/CP110020.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025957945.1/CP110022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025957965.1/CP110030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025957985.1/CP110032.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958005.1/CP110035.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958025.1/CP110036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958045.1/CP110038.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958065.1/CP110039.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958085.1/CP110040.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958125.1/CP110041.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958325.1/CP110047.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958445.1/CP110050.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958625.1/CP110053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958785.1/CP110054.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958805.1/CP110059.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958825.1/CP110063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958845.1/CP110068.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958865.1/CP110069.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958885.1/CP110071.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958905.1/CP110072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025958925.1/CP110074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026650885.1/CP085841.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026802365.1/CP113831.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026802385.1/CP113832.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_027942035.1/CP115992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028131375.1/AP026718.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028131645.1/AP026721.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028131945.1/AP026726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028132325.1/AP026729.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335305.1/CP116569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335345.1/CP116571.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335425.1/CP116555.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335445.1/CP116559.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335505.1/CP116557.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335545.1/CP116561.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335665.1/CP116553.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335685.1/CP116564.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335705.1/CP116575.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335725.1/CP116566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028532705.1/CP116962.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028598965.2/CP121233.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028621735.1/CP117508.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028621755.1/CP117503.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028621775.1/CP117511.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028743535.1/CP117970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029000425.1/CP118755.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029011255.1/CP118048.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029011395.1/CP118057.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029011535.1/CP118072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029011745.1/CP118085.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029024925.1/CP118962.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029167565.1/CP119159.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029278545.1/CP119528.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029866805.1/CP104856.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029910175.1/CP106757.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020155.1/CP124887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020175.1/CP124884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020195.1/CP124886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020215.1/CP124889.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020235.1/CP124890.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020255.1/CP124891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020425.1/CP124882.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020455.1/CP124893.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020475.1/CP124894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020495.1/CP124898.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020515.1/CP124900.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020535.1/CP124901.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020555.1/CP124905.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020575.1/CP124911.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020595.1/CP124906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020615.1/CP124908.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020645.1/CP124913.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020665.1/CP124916.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020685.1/CP124921.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020705.1/CP124923.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020725.1/CP124918.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030020745.1/CP124914.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030064715.1/CP109611.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030264755.1/CP127177.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030295445.1/AP027136.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030295805.1/AP027297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030295845.1/AP027302.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030376825.1/CP128464.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030512425.1/CP124778.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517125.1/CP124925.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517355.1/CP124928.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517385.1/CP124935.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517415.1/CP124939.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517445.1/CP124947.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517475.1/CP124950.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517505.1/CP124953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517535.1/CP124931.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517565.1/CP124960.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030517625.1/CP124956.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_031186115.1/CP133450.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_031206965.1/CP102120.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_031208705.1/CP102121.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900636775.1/LR134312.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_901553725.1/LR594051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_902379005.2/LR698844.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120545.1/LR961961.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120555.1/LR961959.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120565.1/LR961923.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120575.1/LR961970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120585.1/LR961953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120595.1/LR961924.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120605.1/LR961933.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120615.1/LR961966.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120625.1/LR961980.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120635.1/LR961948.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120645.1/LR961975.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120655.1/LR961942.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120665.1/LR961938.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120675.1/LR961925.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120685.1/LR961935.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120695.1/LR961945.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120705.1/LR961951.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120715.1/LR961957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120725.1/LR961965.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120735.1/LR961978.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120745.1/LR961981.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120755.1/LR962008.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120765.1/LR961997.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120775.1/LR961999.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120785.1/LR961991.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120795.1/LR962016.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120805.1/LR962002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120815.1/LR961986.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120825.1/LR962013.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120835.1/LR961994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120845.1/LR961989.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120855.1/LR961996.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120865.1/LR962010.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120875.1/LR962004.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120885.1/LR962021.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120895.1/LR962019.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120905.1/LR962023.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120915.1/LR962025.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120925.1/LR962029.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120935.1/LR962017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120945.1/LR962051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120955.1/LR962063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120965.1/LR962056.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120975.1/LR962053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120985.1/LR962072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905120995.1/LR962060.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121005.1/LR962030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121015.1/LR962068.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121025.1/LR962049.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121035.1/LR962046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121045.1/LR962034.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121055.1/LR962036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121065.1/LR962039.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121075.1/LR962043.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121085.1/LR962075.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121095.1/LR962074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121105.1/LR962085.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121115.1/LR962078.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121125.1/LR962083.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121135.1/LR962110.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121145.1/LR962117.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121155.1/LR962098.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121165.1/LR962093.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121175.1/LR962119.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121185.1/LR962107.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121195.1/LR962086.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121205.1/LR962103.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121215.1/LR962101.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121225.1/LR962112.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121235.1/LR962091.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121245.1/LR962089.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121255.1/LR962115.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121265.1/LR962097.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121275.1/LR962129.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121285.1/LR962125.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121295.1/LR962123.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121305.1/LR962121.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121315.1/LR962137.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121325.1/LR962132.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121335.1/LR962165.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121345.1/LR962141.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121355.1/LR962146.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121365.1/LR962157.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121375.1/LR962161.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121385.1/LR962158.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121395.1/LR962163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121405.1/LR962144.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121415.1/LR962169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121425.1/LR962171.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121435.1/LR962155.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121445.1/LR962138.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121455.1/LR962152.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121465.1/LR962149.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121475.1/LR962174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121485.1/LR962178.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121495.1/LR962182.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121505.1/LR962180.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121515.1/LR962194.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121525.1/LR962230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121535.1/LR962197.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121545.1/LR962217.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121555.1/LR962227.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121565.1/LR962211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121575.1/LR962190.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121585.1/LR962188.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121595.1/LR962200.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121605.1/LR962189.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121615.1/LR962213.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121625.1/LR962210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121635.1/LR962196.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121645.1/LR962203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121655.1/LR962185.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121665.1/LR962225.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121675.1/LR962234.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121685.1/LR962239.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121695.1/LR962231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121705.1/LR962236.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121715.1/LR962258.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121725.1/LR962264.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121735.1/LR962242.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121745.1/LR962254.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121755.1/LR962248.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121765.1/LR962277.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121775.1/LR962251.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121785.1/LR962256.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121795.1/LR962280.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121805.1/LR962275.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121815.1/LR962283.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121825.1/LR962271.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121835.1/LR962267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121845.1/LR962284.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121855.1/LR962300.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121865.1/LR962302.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121875.1/LR962316.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121885.1/LR962298.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121895.1/LR962296.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121905.1/LR962288.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121915.1/LR962293.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121925.1/LR962306.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121935.1/LR962343.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121945.1/LR962328.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121955.1/LR962315.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121965.1/LR962331.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121975.1/LR962323.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121985.1/LR962311.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905121995.1/LR962336.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122005.1/LR962326.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122015.1/LR962340.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122025.1/LR962334.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122035.1/LR962319.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122045.1/LR962321.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122055.1/LR962348.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122065.1/LR962346.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122075.1/LR962351.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122085.1/LR962353.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122095.1/LR962356.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122105.1/LR962359.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122115.1/LR962361.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122125.1/LR962363.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122135.1/LR962372.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122145.1/LR962367.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122155.1/LR962375.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122165.1/LR962370.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122175.1/LR962393.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122185.1/LR962378.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122195.1/LR962395.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122205.1/LR962390.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122215.1/LR962400.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122225.1/LR962396.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122235.1/LR962386.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122245.1/LR962382.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122255.1/LR962407.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122265.1/LR962403.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122275.1/LR962415.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122285.1/LR962414.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122295.1/LR962421.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122305.1/LR962410.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122315.1/LR962423.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122325.1/LR962427.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122335.1/LR962412.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122345.1/LR962418.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122355.1/LR962426.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122365.1/LR962450.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122375.1/LR962432.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122385.1/LR962444.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122395.1/LR962447.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122405.1/LR962464.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122415.1/LR962454.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122425.1/LR962437.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122435.1/LR962461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122445.1/LR962428.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122455.1/LR962440.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122465.1/LR962434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122475.1/LR962486.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122485.1/LR962476.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122495.1/LR962480.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122505.1/LR962468.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122515.1/LR962474.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122525.1/LR962466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122535.1/LR962495.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122545.1/LR962497.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122555.1/LR962489.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122565.1/LR962523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122575.1/LR962512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122585.1/LR962507.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122595.1/LR962525.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122605.1/LR962502.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122615.1/LR962521.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122625.1/LR962517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122635.1/LR962526.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122645.1/LR962509.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122655.1/LR962519.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122665.1/LR962501.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122675.1/LR962548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122685.1/LR962541.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122695.1/LR962539.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122705.1/LR962531.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122715.1/LR962545.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122725.1/LR962542.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122735.1/LR962546.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122745.1/LR962550.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122755.1/LR962551.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122765.1/LR962554.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122775.1/LR962565.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122785.1/LR962579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122795.1/LR962574.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122805.1/LR962572.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122815.1/LR962569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122825.1/LR962558.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122835.1/LR962567.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122845.1/LR962561.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122855.1/LR962559.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122865.1/LR962576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122875.1/LR962580.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122885.1/LR962589.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122895.1/LR962595.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122905.1/LR962584.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122915.1/LR962591.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122925.1/LR962586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122935.1/LR962600.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122945.1/LR962598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122955.1/LR962602.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122965.1/LR962611.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122975.1/LR962618.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122985.1/LR962634.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905122995.1/LR962608.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123005.1/LR962606.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123015.1/LR962604.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123025.1/LR962628.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123035.1/LR962617.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123045.1/LR962620.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123055.1/LR962615.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123065.1/LR962632.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123075.1/LR962626.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123085.1/LR962642.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123095.1/LR962636.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123115.1/LR962649.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123135.1/LR962647.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123155.1/LR962639.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123165.1/LR962670.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123175.1/LR962666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123185.1/LR962681.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123195.1/LR962671.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123205.1/LR962663.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123215.1/LR962656.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123225.1/LR962676.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123235.1/LR962680.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123245.1/LR962650.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123255.1/LR962658.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123265.1/LR962654.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123275.1/LR962687.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123285.1/LR962695.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123295.1/LR962684.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123305.1/LR962691.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123315.1/LR962694.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123325.1/LR962698.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123335.1/LR962703.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123345.1/LR962717.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123355.1/LR962714.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123365.1/LR962700.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123375.1/LR962721.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123385.1/LR962723.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123395.1/LR962702.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123405.1/LR962711.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123415.1/LR962708.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123425.1/LR962720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123435.1/LR962716.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123445.1/LR962726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123455.1/LR962731.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123465.1/LR962729.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123475.1/LR962738.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123485.1/LR962736.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123495.1/LR962732.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123525.1/LR962782.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123535.1/LR962771.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123545.1/LR962807.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123555.1/LR962805.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123565.1/LR962796.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123575.1/LR962788.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123585.1/LR962772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123595.1/LR962802.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123605.1/LR962784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123615.1/LR962820.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123625.1/LR962811.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123635.1/LR962775.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123645.1/LR962814.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123655.1/LR962792.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123665.1/LR962777.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123675.1/LR962806.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123685.1/LR962797.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123695.1/LR962818.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123705.1/LR962821.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123715.1/LR962825.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123725.1/LR962859.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123735.1/LR962827.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123745.1/LR962834.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123755.1/LR962830.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123765.1/LR962832.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123775.1/LR962857.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123785.1/LR962840.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123795.1/LR962836.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123805.1/LR962846.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123815.1/LR962862.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123825.1/LR962850.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123835.1/LR962843.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123845.1/LR962842.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123855.1/LR962826.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123865.1/LR962855.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905123875.1/LR962852.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464835.1/OD940431.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464865.1/OD940420.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464875.1/OD940437.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464895.1/OD940422.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464915.2/OD940440.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_906464925.1/OD940434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000211255.2/NC_017312.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000211255.2/CP002491.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002814115.1/NZ_CP022488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002814115.1/CP022488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000742975.1/NZ_CP008816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000742975.1/CP008816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000281195.1/NC_018221.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000281195.1/CP003726.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000550745.1/NZ_CP004081.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000550745.1/CP004081.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000391485.2/NZ_CP041738.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000391485.2/CP041738.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000172575.2/NC_017316.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000172575.2/CP002621.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009662495.1/NZ_CP036249.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009662495.1/CP036249.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000317915.1/NC_019770.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000317915.1/HF558530.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000007785.1/NC_004668.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000007785.1/AE016830.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009734005.1/NZ_CP038996.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001298485.1/NZ_CP012522.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001412695.1/NZ_CP013009.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001518735.1/NZ_CP013994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001587115.1/NZ_CP011828.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001594345.1/NZ_CP014449.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001635875.1/NZ_CP011281.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001720945.1/NZ_CP012430.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001721065.1/NZ_CP012460.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001721905.1/NZ_CP016163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001750885.1/NZ_CP014529.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001886635.1/NZ_CP018065.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001895905.1/NZ_CP018071.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001953235.1/NZ_CP018828.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001953255.1/NZ_CP018830.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002007625.1/NZ_CP019208.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002024245.1/NZ_CP019970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002025045.1/NZ_CP019992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002025065.1/NZ_CP019988.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002334625.1/NZ_CP023423.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002442955.1/NZ_CP018128.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002761255.1/NZ_CP017792.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002761275.1/NZ_CP017797.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002761555.1/NZ_CP021849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002777275.1/NZ_CP017787.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002831505.1/NZ_CP025077.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848385.1/NZ_CP025425.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848625.1/NZ_CP023780.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848645.1/NZ_CP023784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848665.1/NZ_CP023794.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848685.1/NZ_CP023789.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848705.1/NZ_CP023804.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848725.1/NZ_CP023799.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002848745.1/NZ_CP023808.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002850515.1/NZ_CP021885.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002880635.1/NZ_CP025754.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002973755.2/NZ_CP044316.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002983785.1/NZ_CP027402.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003020685.1/NZ_CP027497.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003020705.1/NZ_CP027506.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003020725.1/NZ_CP027512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003020745.1/NZ_CP027517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003020765.1/NZ_CP027501.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003071425.1/NZ_CP020484.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003071445.1/NZ_CP020488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003269465.1/NZ_CP030110.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003574925.1/NZ_CP032308.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003667965.1/NZ_CP033041.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003711605.1/NZ_CP033376.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003957785.1/NZ_CP033206.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004015145.1/NZ_CP025685.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004101385.1/NZ_CP035136.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004103475.1/NZ_CP035220.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004332055.1/NZ_CP028727.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005166365.1/NZ_CP034949.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005517315.1/NZ_CP040236.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005576735.1/NZ_CP040368.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005886545.1/NZ_CP035648.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005886655.1/NZ_CP035654.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005886715.1/NZ_CP035660.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005886735.1/NZ_CP035666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005952885.1/NZ_CP040706.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006007925.1/NZ_CP040740.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006280355.1/NZ_CP040849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006337045.1/NZ_CP040875.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006337145.1/NZ_CP040878.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006351785.1/NZ_CP040907.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006351845.1/NZ_CP040904.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006575625.1/NZ_CP041257.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_006741355.1/NZ_AP019394.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007917035.2/NZ_CP041261.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007917315.3/NZ_CP041270.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923785.2/NZ_CP038172.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923865.2/NZ_CP038166.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923885.2/NZ_CP038170.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923905.2/NZ_CP038177.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923925.2/NZ_CP038164.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923945.2/NZ_CP038163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_007923965.2/NZ_CP038174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_008000855.1/NZ_CP042834.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_008330605.1/NZ_CP043484.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_008728455.1/NZ_CP044264.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_008728475.1/NZ_CP044274.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009036045.1/NZ_CP045012.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009697285.1/NZ_CP046077.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009938075.1/NZ_CP038995.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009938285.1/NZ_AP022341.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_010120755.1/NZ_CP039729.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_011745645.1/NZ_CP050255.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012045365.1/NZ_CP050648.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012045505.1/NZ_CP050650.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012932975.2/NZ_CP059784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012932985.2/NZ_CP059806.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933055.2/NZ_CP059793.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933075.2/NZ_CP059816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933165.2/NZ_CP059747.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933195.2/NZ_CP061817.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933245.2/NZ_CP059739.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933265.2/NZ_CP059772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933285.2/NZ_CP059764.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933295.2/NZ_CP059823.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012933345.2/NZ_CP059755.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013201055.1/NZ_CP053704.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014490015.1/NZ_CP060861.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014874615.1/NZ_CP062265.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015325925.1/NZ_CP064049.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015377765.1/NZ_CP045602.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015476295.1/NZ_CP064343.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015767695.1/NZ_CP065210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015999405.1/NZ_CP065523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015999425.1/NZ_CP065527.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015999605.1/NZ_CP065547.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016126675.1/NZ_CP065928.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406345.1/NZ_CP066466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406365.1/NZ_CP066415.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406385.1/NZ_CP066421.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406405.1/NZ_CP066434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406425.1/NZ_CP066451.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406445.1/NZ_CP066389.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406465.1/NZ_CP066396.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406485.1/NZ_CP066381.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406505.1/NZ_CP066457.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406525.1/NZ_CP066443.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406545.1/NZ_CP066409.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406565.1/NZ_CP066427.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406585.1/NZ_CP066480.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016406605.1/NZ_CP066473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415025.1/NZ_CP066568.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415105.1/NZ_CP066583.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415285.1/NZ_CP066576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415325.1/NZ_CP066591.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415345.1/NZ_CP066598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415365.1/NZ_CP066614.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415385.1/NZ_CP066606.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415405.1/NZ_CP066637.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415425.1/NZ_CP066645.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415445.1/NZ_CP066629.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415465.1/NZ_CP066622.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415485.1/NZ_CP066673.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415505.1/NZ_CP066659.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415545.1/NZ_CP066653.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016415565.1/NZ_CP066667.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016743855.1/NZ_CP068244.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016864255.1/NZ_CP046166.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017584065.1/NZ_CP071931.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017603725.1/NZ_CP071958.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017815655.1/NZ_CP072579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017815675.1/NZ_CP072567.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017815695.1/NZ_CP072586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017897965.1/NZ_CP072731.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017898005.1/NZ_CP072720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_017898025.1/NZ_CP072707.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018219285.1/NZ_CP064406.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018219325.1/NZ_CP064424.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018279145.1/NZ_CP047326.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018516845.1/NZ_CP065752.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018516925.1/NZ_CP065754.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517045.1/NZ_CP065767.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517065.1/NZ_CP065770.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517085.1/NZ_CP065773.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517105.1/NZ_CP065777.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_018517185.1/NZ_CP066215.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175425.1/NZ_CP072878.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175445.1/NZ_CP072881.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175465.1/NZ_CP072884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175525.1/NZ_CP072894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019356355.1/NZ_CP079927.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019456555.1/NZ_CP074105.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019456575.1/NZ_CP074107.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019456595.1/NZ_CP074109.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019977495.1/NZ_AP024831.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019977575.1/NZ_AP024837.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020091325.1/NZ_CP083473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020162155.1/NZ_CP083912.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020162175.1/NZ_CP083920.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020221735.2/NZ_CP084178.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020736585.1/NZ_CP085894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020736625.1/NZ_CP085906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021172105.3/NZ_CP089092.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021228615.1/NZ_CP089319.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021398505.1/NZ_CP083179.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021560195.1/NZ_CP091100.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021713095.1/NZ_CP091301.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021899455.1/NZ_CP091577.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022647825.1/NZ_CP076471.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022647945.1/NZ_CP076477.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022648125.1/NZ_CP076483.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022691445.1/NZ_CP092569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022699225.1/NZ_CP093941.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022699545.1/NZ_CP093953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022699565.1/NZ_CP093947.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022749455.1/NZ_CP094230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023182815.2/NZ_CP110294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023182835.2/NZ_CP110360.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023204955.1/NZ_CP096211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023204975.1/NZ_CP096219.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299325.1/NZ_CP097014.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299725.1/NZ_CP097022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299745.1/NZ_CP097027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299805.1/NZ_CP097061.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375205.1/NZ_CP088191.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375225.1/NZ_CP088195.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375345.1/NZ_CP091206.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375365.1/NZ_CP091210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375385.1/NZ_CP091213.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375405.1/NZ_CP091217.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375425.1/NZ_CP091221.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375445.1/NZ_CP091224.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023658035.1/NZ_CP058891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024665675.1/NZ_CP102534.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025800545.1/NZ_JAOXXX010000002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025800625.1/NZ_JAOXXY010000002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025800645.1/NZ_JAOXXZ010000001.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025913595.1/NZ_CP109798.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025916235.1/NZ_CP109839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025998395.1/NZ_AP026772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073115.1/NZ_AP026566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073415.1/NZ_AP026601.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073435.1/NZ_AP026604.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073455.1/NZ_AP026612.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073625.1/NZ_AP026655.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073735.1/NZ_AP026660.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026073915.1/NZ_AP026666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026409165.1/NZ_CP112862.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_026427575.1/NZ_CP099979.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_027286245.1/NZ_CP114617.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_027915235.1/NZ_CP115812.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_027944535.1/NZ_CP116023.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335205.1/NZ_CP116514.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335225.1/NZ_CP116532.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335245.1/NZ_CP116517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335265.1/NZ_CP116520.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335465.1/NZ_CP116523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335485.1/NZ_CP116528.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335525.1/NZ_CP116536.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335565.1/NZ_CP116540.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335585.1/NZ_CP116543.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335605.1/NZ_CP116546.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335625.1/NZ_CP116549.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028891525.1/NZ_CP118548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029023785.1/NZ_CP118955.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029167665.1/NZ_CP119150.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030064735.1/NZ_CP109758.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030295785.1/NZ_AP027294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030316155.1/NZ_CP128188.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030406065.1/NZ_CP128995.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030581675.1/NZ_CP130829.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030581775.1/NZ_CP130825.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030581795.1/NZ_CP130827.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584205.1/NZ_CP130871.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584225.1/NZ_CP130873.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584245.1/NZ_CP130869.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584265.1/NZ_CP130867.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584285.1/NZ_CP130863.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584305.1/NZ_CP130865.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584325.1/NZ_CP130859.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584345.1/NZ_CP130861.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584365.1/NZ_CP130851.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584385.1/NZ_CP130847.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584405.1/NZ_CP130845.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584425.1/NZ_CP130839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584445.1/NZ_CP130835.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584465.1/NZ_CP130843.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584485.1/NZ_CP130837.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584505.1/NZ_CP130841.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584525.1/NZ_CP130833.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584545.1/NZ_CP130849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584585.1/NZ_CP130882.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584605.1/NZ_CP130881.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584625.1/NZ_CP130877.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584645.1/NZ_CP130875.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584665.1/NZ_CP130879.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030584685.1/NZ_CP130883.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030643745.1/NZ_CP041665.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030644745.1/NZ_CP130906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030644765.1/NZ_CP130910.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030644785.1/NZ_CP130908.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030644805.1/NZ_CP130912.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030644825.1/NZ_CP130914.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030719015.1/NZ_CP132226.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900044005.1/NZ_LN999987.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900066025.1/NZ_LN999844.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900092475.1/NZ_LT598663.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900094185.1/NZ_LT603678.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900634805.1/NZ_LR132067.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900635365.1/NZ_LR134095.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900635415.1/NZ_LR134105.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900637035.1/NZ_LR134337.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638765.1/NZ_LR135169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638775.1/NZ_LR135170.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638785.1/NZ_LR135181.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638805.1/NZ_LR135174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638815.1/NZ_LR135179.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900638825.1/NZ_LR135191.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639335.1/NZ_LR135235.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639345.1/NZ_LR135197.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639355.1/NZ_LR135226.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639365.1/NZ_LR135203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639385.1/NZ_LR135219.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639395.1/NZ_LR135278.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639405.1/NZ_LR135258.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639415.1/NZ_LR135243.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639425.1/NZ_LR135254.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639445.1/NZ_LR135287.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639455.1/NZ_LR135293.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639465.1/NZ_LR135297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639485.1/NZ_LR135307.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639495.1/NZ_LR135331.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639505.1/NZ_LR135339.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639515.1/NZ_LR135324.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639525.1/NZ_LR135317.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639535.1/NZ_LR135344.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639545.1/NZ_LR135384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639555.1/NZ_LR135372.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639565.1/NZ_LR135364.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639575.1/NZ_LR135351.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639585.1/NZ_LR135357.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639595.1/NZ_LR135401.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639605.1/NZ_LR135394.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639615.1/NZ_LR135414.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639625.1/NZ_LR135408.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639635.1/NZ_LR135428.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639645.1/NZ_LR135419.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639655.1/NZ_LR135435.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639665.1/NZ_LR135443.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639705.1/NZ_LR135474.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639715.1/NZ_LR135488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639725.1/NZ_LR135475.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900639745.1/NZ_LR135482.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900640265.1/NZ_LR135782.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900683475.1/NZ_LR215968.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907163255.1/NZ_OU015345.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907163315.1/NZ_OU015350.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907165365.1/NZ_OU015708.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907176135.1/NZ_OU015946.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907176815.1/NZ_OU016027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_907177285.1/NZ_OU016036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001298485.1/CP012522.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001412695.1/CP013009.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001518735.1/CP013994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001587115.1/CP011828.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001594345.1/CP014449.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001635875.1/CP011281.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001720945.1/CP012430.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001721065.1/CP012460.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001721905.1/CP016163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001750885.1/CP014529.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001886635.1/CP018065.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001895905.1/CP018071.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001953235.1/CP018828.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001953255.1/CP018830.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002007625.1/CP019208.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002024245.1/CP019970.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002025045.1/CP019992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002025065.1/CP019988.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002334625.1/CP023423.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002442955.1/CP018128.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002761255.1/CP017792.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002761275.1/CP017797.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002761555.1/CP021849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002777275.1/CP017787.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002831505.1/CP025077.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848385.1/CP025425.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848625.1/CP023780.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848645.1/CP023784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848665.1/CP023794.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848685.1/CP023789.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848705.1/CP023804.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848725.1/CP023799.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002848745.1/CP023808.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002850515.1/CP021885.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002880635.1/CP025754.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002973755.2/CP044316.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002983785.1/CP027402.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003020685.1/CP027497.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003020705.1/CP027506.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003020725.1/CP027512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003020745.1/CP027517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003020765.1/CP027501.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003071425.1/CP020484.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003071445.1/CP020488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003269465.1/CP030110.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003574925.1/CP032308.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003667965.1/CP033041.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003711605.1/CP033376.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003957785.1/CP033206.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004015145.1/CP025685.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004101385.1/CP035136.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004103475.1/CP035220.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004332055.1/CP028727.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005166365.1/CP034949.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005517315.1/CP040236.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005576735.1/CP040368.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005886545.1/CP035648.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005886655.1/CP035654.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005886715.1/CP035660.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005886735.1/CP035666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005952885.1/CP040706.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006007925.1/CP040740.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006280355.1/CP040849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006337045.1/CP040875.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006337145.1/CP040878.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006351785.1/CP040907.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006351845.1/CP040904.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006575625.1/CP041257.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_006741355.1/AP019394.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007917035.3/CP041261.3.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007917315.3/CP041270.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923785.2/CP038172.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923865.2/CP038166.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923885.2/CP038170.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923905.2/CP038177.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923925.2/CP038164.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923945.2/CP038163.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_007923965.2/CP038174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_008000855.1/CP042834.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_008330605.1/CP043484.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_008728455.1/CP044264.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_008728475.1/CP044274.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009036045.1/CP045012.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009697285.1/CP046077.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009734005.2/CP038996.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009938075.1/CP038995.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009938285.1/AP022341.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_010120755.1/CP039729.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_011745645.1/CP050255.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012045365.1/CP050648.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012045505.1/CP050650.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012932975.2/CP059784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012932985.2/CP059806.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933055.2/CP059793.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933075.2/CP059816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933165.2/CP059747.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933195.2/CP061817.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933245.2/CP059739.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933265.2/CP059772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933285.2/CP059764.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933295.2/CP059823.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012933345.2/CP059755.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013201055.1/CP053704.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014490015.1/CP060861.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014874615.1/CP062265.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015325925.1/CP064049.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015356095.1/CP064279.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015377765.1/CP045602.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015476295.1/CP064343.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015767695.1/CP065210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015999405.1/CP065523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015999425.1/CP065527.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015999605.1/CP065547.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016126675.1/CP065928.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406345.1/CP066466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406365.1/CP066415.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406385.1/CP066421.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406405.1/CP066434.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406425.1/CP066451.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406445.1/CP066389.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406465.1/CP066396.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406485.1/CP066381.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406505.1/CP066457.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406525.1/CP066443.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406545.1/CP066409.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406565.1/CP066427.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406585.1/CP066480.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016406605.1/CP066473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415025.1/CP066568.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415105.1/CP066583.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415285.1/CP066576.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415325.1/CP066591.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415345.1/CP066598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415365.1/CP066614.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415385.1/CP066606.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415405.1/CP066637.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415425.1/CP066645.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415445.1/CP066629.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415465.1/CP066622.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415485.1/CP066673.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415505.1/CP066659.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415545.1/CP066653.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016415565.1/CP066667.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016743855.1/CP068244.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016864255.1/CP046166.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017301355.1/CP071271.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017584065.1/CP071931.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017603725.1/CP071958.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017815655.1/CP072579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017815675.1/CP072567.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017815695.1/CP072586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017897965.1/CP072731.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017898005.1/CP072720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_017898025.1/CP072707.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018219285.1/CP064406.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018219325.1/CP064424.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018279145.1/CP047326.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018516845.1/CP065752.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018516925.1/CP065754.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517025.1/CP065764.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517045.1/CP065767.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517065.1/CP065770.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517085.1/CP065773.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517105.1/CP065777.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517145.1/CP066205.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517165.1/CP066212.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_018517185.1/CP066215.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175425.1/CP072878.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175445.1/CP072881.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175465.1/CP072884.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175525.1/CP072894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019356355.1/CP079927.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019456555.1/CP074105.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019456575.1/CP074107.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019456595.1/CP074109.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019774555.1/CP081504.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019977495.1/AP024831.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019977575.1/AP024837.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020091325.1/CP083473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020162155.1/CP083912.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020162175.1/CP083920.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020221735.2/CP084178.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020736585.1/CP085894.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020736625.1/CP085906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021172105.3/CP089092.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021228615.1/CP089319.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021398505.1/CP083179.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021560195.1/CP091100.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021713095.1/CP091301.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021899455.1/CP091577.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022647825.1/CP076471.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022647945.1/CP076477.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022648125.1/CP076483.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022691445.1/CP092569.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022699225.1/CP093941.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022699545.1/CP093953.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022699565.1/CP093947.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022749455.1/CP094230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811085.1/CP063283.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811125.1/CP063537.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811145.1/CP063542.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811165.1/CP063549.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811185.1/CP063555.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811205.1/CP063562.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811225.1/CP063568.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811245.1/CP063574.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811265.1/CP063579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811285.1/CP063586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811305.1/CP063593.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811325.1/CP063598.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811345.1/CP063604.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811365.1/CP063647.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811385.1/CP063611.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811405.1/CP063618.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811425.1/CP063626.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811445.1/CP063633.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022811465.1/CP063639.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023182815.2/CP110294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023182835.2/CP110360.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023204955.1/CP096211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023204975.1/CP096219.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299325.1/CP097014.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299725.1/CP097022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299745.1/CP097027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299805.1/CP097061.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375205.1/CP088191.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375225.1/CP088195.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375345.1/CP091206.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375365.1/CP091210.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375385.1/CP091213.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375405.1/CP091217.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375425.1/CP091221.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375445.1/CP091224.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023658035.1/CP058891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024665675.1/CP102534.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025800545.1/JAOXXX010000002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025800625.1/JAOXXY010000002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025800645.1/JAOXXZ010000001.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025913595.1/CP109798.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025916235.1/CP109839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025998395.1/AP026772.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073115.1/AP026566.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073255.1/AP026573.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073355.1/AP026579.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073375.1/AP026586.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073395.1/AP026593.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073415.1/AP026601.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073435.1/AP026604.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073455.1/AP026612.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073475.1/AP026617.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073495.1/AP026628.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073515.1/AP026632.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073535.1/AP026641.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073555.1/AP026650.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073625.1/AP026655.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073735.1/AP026660.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026073915.1/AP026666.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026409165.1/CP112862.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_026427575.1/CP099979.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_027286245.1/CP114617.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_027915235.1/CP115812.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_027944535.1/CP116023.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335205.1/CP116514.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335225.1/CP116532.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335245.1/CP116517.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335265.1/CP116520.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335465.1/CP116523.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335485.1/CP116528.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335525.1/CP116536.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335565.1/CP116540.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335585.1/CP116543.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335605.1/CP116546.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335625.1/CP116549.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028891525.1/CP118548.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029023785.1/CP118955.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029167665.1/CP119150.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030064735.1/CP109758.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030295785.1/AP027294.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030316155.1/CP128188.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030406065.1/CP128995.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030581675.1/CP130829.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030581775.1/CP130825.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030581795.1/CP130827.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584205.1/CP130871.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584225.1/CP130873.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584245.1/CP130869.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584265.1/CP130867.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584285.1/CP130863.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584305.1/CP130865.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584325.1/CP130859.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584345.1/CP130861.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584365.1/CP130851.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584385.1/CP130847.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584405.1/CP130845.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584425.1/CP130839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584445.1/CP130835.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584465.1/CP130843.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584485.1/CP130837.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584505.1/CP130841.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584525.1/CP130833.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584545.1/CP130849.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584585.1/CP130882.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584605.1/CP130881.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584625.1/CP130877.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584645.1/CP130875.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584665.1/CP130879.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030584685.1/CP130883.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030643745.1/CP041665.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030644745.1/CP130906.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030644765.1/CP130910.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030644785.1/CP130908.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030644805.1/CP130912.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030644825.1/CP130914.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656695.1/CP131024.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656715.1/CP130994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656735.1/CP130992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656755.1/CP131002.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656775.1/CP130996.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656795.1/CP131004.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656815.1/CP130998.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656835.1/CP131000.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656855.1/CP131006.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656895.1/CP131029.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656915.1/CP131037.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656935.1/CP131033.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656955.1/CP131035.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656975.1/CP131025.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030656995.1/CP131008.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657015.1/CP131031.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657035.1/CP131027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657055.1/CP131010.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657075.1/CP131012.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657095.1/CP131014.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657115.1/CP131018.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657135.1/CP131016.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657155.1/CP131020.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657175.1/CP131040.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657195.1/CP131042.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657215.1/CP131044.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657235.1/CP131046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657255.1/CP131048.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657275.1/CP131050.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657295.1/CP131052.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030657315.1/CP131054.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664785.1/CP131065.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664805.1/CP131063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664825.1/CP131067.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664845.1/CP131073.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664865.1/CP131083.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664885.1/CP131085.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664905.1/CP131091.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664925.1/CP131081.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664945.1/CP131079.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664965.1/CP131089.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030664985.1/CP131087.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665005.1/CP131071.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665025.1/CP131075.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665045.1/CP131077.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665065.1/CP131069.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665085.1/CP131095.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665105.1/CP131093.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665125.1/CP131101.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665145.1/CP131097.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665165.1/CP131099.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665185.1/CP131103.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665205.1/CP131105.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665225.1/CP131111.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665245.1/CP131109.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665265.1/CP131113.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665285.1/CP131115.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665305.1/CP131117.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665325.1/CP131107.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665345.1/CP131119.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665365.1/CP131121.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665385.1/CP131125.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030665405.1/CP131123.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685595.1/CP131489.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685655.1/CP131490.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685675.1/CP131500.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685695.1/CP131502.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685715.1/CP131516.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685735.1/CP131514.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685755.1/CP131524.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685775.1/CP131526.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685795.1/CP131520.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685815.1/CP131518.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685835.1/CP131522.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685855.1/CP131528.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685875.1/CP131492.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685895.1/CP131494.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685915.1/CP131610.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685935.1/CP131498.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685955.1/CP131510.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685975.1/CP131508.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030685995.1/CP131496.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030686015.1/CP131504.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030686035.1/CP131506.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030686055.1/CP131512.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030686075.1/CP131530.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030686095.1/CP131532.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030719015.1/CP132226.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900044005.1/LN999987.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900066025.1/LN999844.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900092475.1/LT598663.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900094185.1/LT603678.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900634805.1/LR132067.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900635365.1/LR134095.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900635415.1/LR134105.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900637035.1/LR134337.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638765.1/LR135169.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638775.1/LR135170.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638785.1/LR135181.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638805.1/LR135174.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638815.1/LR135179.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900638825.1/LR135191.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639335.1/LR135235.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639345.1/LR135197.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639355.1/LR135226.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639365.1/LR135203.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639385.1/LR135219.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639395.1/LR135278.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639405.1/LR135258.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639415.1/LR135243.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639425.1/LR135254.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639445.1/LR135287.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639455.1/LR135293.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639465.1/LR135297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639485.1/LR135307.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639495.1/LR135331.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639505.1/LR135339.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639515.1/LR135324.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639525.1/LR135317.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639535.1/LR135344.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639545.1/LR135384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639555.1/LR135372.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639565.1/LR135364.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639575.1/LR135351.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639585.1/LR135357.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639595.1/LR135401.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639605.1/LR135394.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639615.1/LR135414.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639625.1/LR135408.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639635.1/LR135428.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639645.1/LR135419.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639655.1/LR135435.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639665.1/LR135443.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639705.1/LR135474.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639715.1/LR135488.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639725.1/LR135475.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900639745.1/LR135482.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900640265.1/LR135782.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900683475.1/LR215968.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907163255.1/OU015345.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907163315.1/OU015350.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907165365.1/OU015708.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907176135.1/OU015946.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907176815.1/OU016027.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_907177285.1/OU016036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000336405.1/NC_020207.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000336405.1/CP004063.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000250945.1/NC_017022.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000250945.1/CP003351.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000444405.1/NC_021994.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000444405.1/CP006620.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000174395.2/NC_017960.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000174395.2/CP003583.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000737555.1/NZ_CP006030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000737555.1/CP006030.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001558875.2/NZ_CP014067.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009730155.1/NZ_CP046307.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014844195.1/NZ_CP050485.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_021496385.1/NZ_CP050816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375325.1/NZ_CP091204.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025263125.1/NZ_CP078505.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335185.1/NZ_CP116510.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001558875.2/CP014067.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009730155.1/CP046307.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014844195.1/CP050485.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_021496385.1/CP050816.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375325.1/CP091204.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025263125.1/CP078505.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335185.1/CP116510.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003343305.1/NZ_CP030932.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003343305.1/CP030932.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002278015.2/NZ_CP023011.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001641305.1/NZ_CP015516.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_012222705.1/NZ_CP037955.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013391145.1/NZ_CP055229.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013391165.1/NZ_CP055230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013391185.1/NZ_CP055231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013391205.1/NZ_CP055232.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014218175.1/NZ_CP042289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016127235.1/NZ_CP065992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016727125.1/NZ_CP068122.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_016727265.1/NZ_CP068118.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175505.1/NZ_CP072891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023373765.1/NZ_CP093926.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375155.1/NZ_CP088183.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375175.1/NZ_CP088187.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023375305.1/NZ_CP091201.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025913615.1/NZ_CP109802.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025913635.1/NZ_CP109803.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025913655.1/NZ_CP109807.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025913675.1/NZ_CP109808.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029814255.1/NZ_CP122304.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030295825.1/NZ_AP027299.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_900636615.1/NZ_LR134297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001641305.1/CP015516.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002278015.2/CP023011.2.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_012222705.1/CP037955.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013391145.1/CP055229.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013391165.1/CP055230.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013391185.1/CP055231.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013391205.1/CP055232.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014218175.1/CP042289.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016127235.1/CP065992.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016727125.1/CP068122.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_016727265.1/CP068118.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175505.1/CP072891.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023373765.1/CP093926.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375155.1/CP088183.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375175.1/CP088187.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023375305.1/CP091201.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025913615.1/CP109802.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025913635.1/CP109803.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025913655.1/CP109807.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025913675.1/CP109808.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029814255.1/CP122304.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030295825.1/AP027299.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_900636615.1/LR134297.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000271405.2/NC_018081.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000271405.2/CP003504.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_905186115.1/LR994474.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022846515.1/NZ_AP025635.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022846515.1/AP025635.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019343125.1/NZ_CP079880.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_015767715.1/NZ_CP065211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019203145.1/NZ_CP078094.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019880345.1/NZ_CP082267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019967715.1/NZ_CP082963.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020268645.1/NZ_CP084248.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023612275.1/NZ_CP063766.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030490365.1/NZ_CP129887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_015767715.1/CP065211.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019203145.1/CP078094.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019343125.1/CP079880.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019880345.1/CP082267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019967715.1/CP082963.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020268645.1/CP084248.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023612275.1/CP063766.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030490365.1/CP129887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002813755.1/NZ_CP018061.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002216705.1/NZ_CP022340.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003957375.1/NZ_CP025473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005158285.1/NZ_CP029066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_013394305.1/NZ_AP019810.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020149485.1/NZ_CP083695.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_931972305.1/NZ_OV996158.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002216705.1/CP022340.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002813755.1/CP018061.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003957375.1/CP025473.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005158285.1/CP029066.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_013394305.1/AP019810.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020149485.1/CP083695.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_931972305.1/OV996158.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000504125.1/NC_022878.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000504125.1/AP013036.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019175485.1/NZ_CP072888.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019797905.1/NZ_CP081846.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025309655.1/NZ_CP104392.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025402795.1/NZ_CP104764.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_025402855.1/NZ_CP104762.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019175485.1/CP072888.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019797905.1/CP081846.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025309655.1/CP104392.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025402795.1/CP104764.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_025402855.1/CP104762.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029023825.1/NZ_CP118957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029023825.1/CP118957.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_011397115.1/NZ_AP022822.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_011397115.1/AP022822.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001465115.1/CP013614.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029983095.1/NZ_CP120467.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029983095.1/CP120467.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002850555.1/NZ_CP025223.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002850555.1/CP025223.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_008000875.1/NZ_CP042839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_008000875.1/CP042839.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002393265.1/NZ_CP023515.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002393265.1/CP023515.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003812305.1/NZ_CP033740.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003812305.1/CP033740.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030263915.1/NZ_CP127160.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030263915.1/CP127160.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023539285.1/NZ_CP092784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023539285.1/CP092784.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005843965.1/NZ_CP040461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005843965.1/CP040461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023168365.1/AP025690.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030053755.1/NZ_CP125658.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030053755.1/CP125658.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002290025.1/NZ_CP023074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002290025.1/CP023074.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002197645.1/NZ_CP021874.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002197645.1/CP021874.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004001205.1/NZ_AP018524.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009731445.1/NZ_AP021885.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004001205.1/AP018524.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009731445.1/AP021885.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000283975.1/NC_016938.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000283975.1/AP012282.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000747585.1/NZ_CP006683.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000747585.1/CP006683.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001712815.1/NZ_CP012047.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_002243535.1/NZ_CP020017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003795105.1/NZ_CP027768.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_009720125.1/NZ_CP046246.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_030388485.1/NZ_CP074114.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001712815.1/CP012047.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_002243535.1/CP020017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003795105.1/CP027768.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_009720125.1/CP046246.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019222785.1/CP078164.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_030388485.1/CP074114.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_000283615.1/NC_016052.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_000283615.1/AP012046.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003795145.1/NZ_CP027786.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003795145.1/CP027786.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_003795125.1/NZ_CP027783.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_003795125.1/CP027783.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_014397115.1/NZ_CP060720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_014397115.1/CP060720.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_011303955.1/NZ_CP049886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_011303955.1/CP049886.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020628455.1/NZ_CP081461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020628355.1/NZ_CP081459.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020628555.1/NZ_CP081466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020628665.1/NZ_CP081470.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_020628735.1/NZ_CP081472.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020628355.1/CP081459.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020628455.1/CP081461.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020628555.1/CP081466.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020628665.1/CP081470.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_020628735.1/CP081472.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_011304195.1/NZ_CP049887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_011304195.1/CP049887.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029144185.1/NZ_CP110232.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029144405.1/NZ_CP110237.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029144185.1/CP110232.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029144405.1/CP110237.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_024637875.1/NZ_CP102451.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_024637875.1/CP102451.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299185.1/NZ_CP097017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_019797785.1/NZ_CP081833.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299345.1/NZ_CP097051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299445.1/NZ_CP097044.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299565.1/NZ_CP097053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023299765.1/NZ_CP097045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_023300045.1/NZ_CP097072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335165.1/NZ_CP116503.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_028335285.1/NZ_CP116507.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_029011965.1/NZ_CP090216.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_019797785.1/CP081833.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299185.1/CP097017.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299345.1/CP097051.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299445.1/CP097044.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299565.1/CP097053.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023299765.1/CP097045.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_023300045.1/CP097072.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335165.1/CP116503.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_028335285.1/CP116507.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_029011965.1/CP090216.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001998885.1/NZ_CP019609.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001998885.1/CP019609.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_022655055.1/NZ_CP093384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_022655055.1/CP093384.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_001870205.1/NZ_CP017267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_001870205.1/CP017267.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_004792515.1/NZ_CP038865.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_004792515.1/CP038865.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCF_005145005.1/NZ_CP039712.1.chrom.fna...\n",
      "Concatenating /mnt/e/infant_nt/database/ref_genomes/ncbi_dataset/data/GCA_005145005.1/CP039712.1.chrom.fna...\n",
      "\n",
      "\n",
      "Building a new DB, current time: 10/03/2023 13:48:05\n",
      "New DB name:   /mnt/e/infant_nt/database/blast_db/Efaecalis_refseq\n",
      "New DB title:  Efaecalis_refseq\n",
      "Sequence type: Nucleotide\n",
      "Keep MBits: T\n",
      "Maximum file size: 3000000000B\n",
      "Adding sequences from FASTA; added 2197 sequences in 40.7332 seconds.\n",
      "\n",
      "\n",
      "/home/youn/work/chronostrain/examples/database\n"
     ]
    }
   ],
   "source": [
    "!env BLAST_DB_DIR=$BLAST_DB_DIR \\\n",
    "    BLAST_DB_NAME=$BLAST_DB_NAME \\\n",
    "    bash create_blast_db.sh"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dbef23e0-cf03-4139-adcd-527b932c0026",
   "metadata": {},
   "source": [
    "### Step 3: Build the marker seed catalog.\n",
    "\n",
    "ChronoStrain only needs a FASTA file of marker seeds (one multi-fasta file per marker gene), and a single TSV file that catalogs them.\n",
    "However, to get there, we need to take a few steps...\n",
    "\n",
    "#### Step 3.1: Download MLST schema."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "053f68f2-641e-4a38-866e-9ba5b9fcbd19",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Downloading marker seeds from MLST schema.\n",
      "Targeting 1 taxa using MLST scheme.\n",
      "Fetching URL resource https://pubmlst.org/static/data/dbases.xml\n",
      "Got a response of size 152.35 KB.\n",
      "Schema type id: \n",
      "Handling locus gdh\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/gdh/alleles_fasta\n",
      "Got a response of size 69.5 KB.\n",
      "Handling locus gyd\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/gyd/alleles_fasta\n",
      "Got a response of size 20.51 KB.\n",
      "Handling locus pstS\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/pstS/alleles_fasta\n",
      "Got a response of size 74.72 KB.\n",
      "Handling locus gki\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/gki/alleles_fasta\n",
      "Got a response of size 54.14 KB.\n",
      "Handling locus aroE\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/aroE/alleles_fasta\n",
      "Got a response of size 60.48 KB.\n",
      "Handling locus xpt\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/xpt/alleles_fasta\n",
      "Got a response of size 54.96 KB.\n",
      "Handling locus yqiL\n",
      "Fetching URL resource https://rest.pubmlst.org/db/pubmlst_efaecalis_seqdef/loci/yqiL/alleles_fasta\n",
      "Got a response of size 54.46 KB.\n"
     ]
    }
   ],
   "source": [
    "!python python_helpers/mlst_download.py -t \"Enterococcus faecalis\" -w \"$TARGET_DIR\"/mlst_schema -o \"$MARKER_SEED_DIR\"/mlst_seeds.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e0d382c6-2e7a-4579-8a74-22fc1f2e37a3",
   "metadata": {},
   "source": [
    "#### Step 3.2: Non-standard genes: run PCR primer search across entire catalog.\n",
    "\n",
    "First, perform in-silico primer search."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "4ad3e596-618e-4d0b-9f52-0fb80e60fca8",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Performing primer-based search for cpsA in Enterococcus faecalis. (FWD=GTAGAAGAAGCAAGCCAGTACGCC, REV=CCTCTGCAGCAATCTGTTTCATGG, len approx. 478)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:59<00:00,  6.02genome/s]\n",
      "Performing primer-based search for cpsB in Enterococcus faecalis. (FWD=GTGTCATCACAGCTATCGTCGC, REV=CCGGCATTGGATAAGGAAATAGCC, len approx. 603)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [03:00<00:00,  6.00genome/s]\n",
      "Performing primer-based search for cpsC in Enterococcus faecalis. (FWD=CCTGAATATCAATGTATTTGGGCAGTC, REV=CCAACGCTTTGCTTCTTGAATGAC, len approx. 300)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:56<00:00,  6.11genome/s]\n",
      "Performing primer-based search for cpsD in Enterococcus faecalis. (FWD=GGATTCTCTTGTTCAACAAACCATTGG, REV=CGCATGGCTTCATAAAAGAACAGC, len approx. 522)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [03:01<00:00,  5.97genome/s]\n",
      "Performing primer-based search for cpsE in Enterococcus faecalis. (FWD=GAGGTTGAGCGAGATATATTATGGC, REV=CACTTCATAAACCGACTCATCACG, len approx. 450)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:40<00:00,  6.73genome/s]\n",
      "Performing primer-based search for cpsF in Enterococcus faecalis. (FWD=GCATTACAAGGTTATACAGTTGATGG, REV=GACTGTTCCATCTTATCTTTTATTCGG, len approx. 580)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:40<00:00,  6.72genome/s]\n",
      "Performing primer-based search for cpsG in Enterococcus faecalis. (FWD=GGCTCTGATCAAATGTGGAATCCC, REV=GGTGTATCTTCAGAAACATATTCTACTG, len approx. 503)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:25<00:00,  7.42genome/s]\n",
      "Performing primer-based search for cpsH in Enterococcus faecalis. (FWD=GTGTCTTTAGCAATTGGTATCGGTTG, REV=CACTAGAGTAGCTAATACTTTTTTTTCCC, len approx. 366)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:53<00:00,  6.25genome/s]\n",
      "Performing primer-based search for cpsI in Enterococcus faecalis. (FWD=GCTTGTGAAGCAGCTAAACGAGG, REV=CTCTGATAAGTAAGTTTCTTTCTCTGCC, len approx. 630)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:55<00:00,  6.17genome/s]\n",
      "Performing primer-based search for cpsJ in Enterococcus faecalis. (FWD=CCTCGACGTATATTCTGGAGAAAC, REV=GCTTAGTTTCACCAAATGCACGTAG, len approx. 553)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:34<00:00,  7.01genome/s]\n",
      "Performing primer-based search for cpsK in Enterococcus faecalis. (FWD=GCGTTGCACAACGAATTGCTAAATAC, REV=CGCTACAATATAGTAAGGTAGCTGAATC, len approx. 422)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:51<00:00,  6.31genome/s]\n",
      "Performing primer-based search for cylA in Enterococcus faecalis. (FWD=GGTTATGCATCAGATCTCTCAA, REV=TCTTCAGGTTTAAAATCTGG, len approx. 223)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:54<00:00,  6.20genome/s]\n",
      "Performing primer-based search for cylB in Enterococcus faecalis. (FWD=GGAGAATTAGTGTTTAGAGCG, REV=GCTTCATAACCATTGTTACTATAGAAAC, len approx. 522)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:33<00:00,  7.04genome/s]\n",
      "Performing primer-based search for cylM in Enterococcus faecalis. (FWD=AAGATTGTCTGTGCCATGGA, REV=TACTCACTTCCGGCAACCTT, len approx. 159)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [03:08<00:00,  5.75genome/s]\n",
      "Performing primer-based search for cbh in Enterococcus faecalis. (FWD=CTCATAGGATCCATCACCAACATCAC, REV=TGGCTGGAATTCACTTTTCAGGCTAT, len approx. 580)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:27<00:00,  7.35genome/s]\n",
      "Performing primer-based search for gelE in Enterococcus faecalis. (FWD=TTGTTGGAAGTTCATGTCTA, REV=TTCATTGACCAGAACAGATT, len approx. 1484)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [06:28<00:00,  2.78genome/s]\n",
      "Performing primer-based search for fsrB in Enterococcus faecalis. (FWD=GCATTGTTATCTATGTCGCCATACC, REV=GGCTTAGTTCCCACACCATC, len approx. 396)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:55<00:00,  6.17genome/s]\n",
      "Performing primer-based search for gls24_like in Enterococcus faecalis. (FWD=GCATTAGATGAGATTGATGGTC, REV=GCGAGGTTCAGTTTCTTC, len approx. 446)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:58<00:00,  6.07genome/s]\n",
      "Performing primer-based search for esp in Enterococcus faecalis. (FWD=CGATAAAGAAGAGAGCGGAG, REV=GCAAACTCTACATCCACGTC, len approx. 539)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [03:26<00:00,  5.22genome/s]\n",
      "Performing primer-based search for psaA in Enterococcus faecalis. (FWD=CTATTTTGCAGCAAGTGATG, REV=CGCATAGTAACTATCACCATCTTG, len approx. 540)\n",
      "Will NOT use GFF files; primer PCR hits will be interpreted as gene hits.\n",
      "100%|███████████████████████████████████| 1081/1081 [02:33<00:00,  7.03genome/s]\n",
      "Wrote 1081 dataframe records to psaA.feather\n"
     ]
    }
   ],
   "source": [
    "def perform_primer_search(gene_name: str, forward_primer: str, rev_primer: str, est_length: int):\n",
    "    !python python_helpers/identify_gene_cluster_primersearch.py \\\n",
    "        -i \"$REFSEQ_INDEX\" \\\n",
    "        -t \"$TARGET_DIR\"/_tmp \\\n",
    "        -o \"$MARKER_SEED_DIR\"/\"$gene_name\".feather \\\n",
    "        -g \"Enterococcus\" \\\n",
    "        -s \"faecalis\" \\\n",
    "        -p1 \"$forward_primer\" -p2 \"$rev_primer\" \\\n",
    "        -n \"$gene_name\" \\\n",
    "        -l \"$est_length\" \\\n",
    "        --mismatch-pct 0 \\\n",
    "        --dont-use-gff\n",
    "\n",
    "\n",
    "# ====== Known strain polymorphisms\n",
    "\"\"\" https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0000582#s5 \"\"\"\n",
    "perform_primer_search(\"cpsA\", \"GTAGAAGAAGCAAGCCAGTACGCC\", \"CCTCTGCAGCAATCTGTTTCATGG\", 478)\n",
    "perform_primer_search(\"cpsB\", \"GTGTCATCACAGCTATCGTCGC\", \"CCGGCATTGGATAAGGAAATAGCC\", 603)\n",
    "perform_primer_search(\"cpsC\", \"CCTGAATATCAATGTATTTGGGCAGTC\", \"CCAACGCTTTGCTTCTTGAATGAC\", 300)\n",
    "perform_primer_search(\"cpsD\", \"GGATTCTCTTGTTCAACAAACCATTGG\", \"CGCATGGCTTCATAAAAGAACAGC\", 522)\n",
    "perform_primer_search(\"cpsE\", \"GAGGTTGAGCGAGATATATTATGGC\", \"CACTTCATAAACCGACTCATCACG\", 450)\n",
    "perform_primer_search(\"cpsF\", \"GCATTACAAGGTTATACAGTTGATGG\", \"GACTGTTCCATCTTATCTTTTATTCGG\", 580)\n",
    "perform_primer_search(\"cpsG\", \"GGCTCTGATCAAATGTGGAATCCC\", \"GGTGTATCTTCAGAAACATATTCTACTG\", 503)\n",
    "perform_primer_search(\"cpsH\", \"GTGTCTTTAGCAATTGGTATCGGTTG\", \"CACTAGAGTAGCTAATACTTTTTTTTCCC\", 366)\n",
    "perform_primer_search(\"cpsI\", \"GCTTGTGAAGCAGCTAAACGAGG\", \"CTCTGATAAGTAAGTTTCTTTCTCTGCC\", 630)\n",
    "perform_primer_search(\"cpsJ\", \"CCTCGACGTATATTCTGGAGAAAC\", \"GCTTAGTTTCACCAAATGCACGTAG\", 553)\n",
    "perform_primer_search(\"cpsK\", \"GCGTTGCACAACGAATTGCTAAATAC\", \"CGCTACAATATAGTAAGGTAGCTGAATC\", 422)\n",
    "\n",
    "# ======= suspected virulence determinants\n",
    "perform_primer_search(\"cylA\", \"GGTTATGCATCAGATCTCTCAA\", \"TCTTCAGGTTTAAAATCTGG\", 223)\n",
    "perform_primer_search(\"cylB\", \"GGAGAATTAGTGTTTAGAGCG\", \"GCTTCATAACCATTGTTACTATAGAAAC\", 522)\n",
    "perform_primer_search(\"cylM\", \"AAGATTGTCTGTGCCATGGA\", \"TACTCACTTCCGGCAACCTT\", 159)\n",
    "perform_primer_search(\"cbh\", \"CTCATAGGATCCATCACCAACATCAC\", \"TGGCTGGAATTCACTTTTCAGGCTAT\", 580)\n",
    "perform_primer_search(\"gelE\", \"TTGTTGGAAGTTCATGTCTA\", \"TTCATTGACCAGAACAGATT\", 1484)\n",
    "perform_primer_search(\"fsrB\", \"GCATTGTTATCTATGTCGCCATACC\", \"GGCTTAGTTCCCACACCATC\", 396)\n",
    "\n",
    "# ======= pathogenicity island\n",
    "perform_primer_search(\"gls24_like\", \"GCATTAGATGAGATTGATGGTC\", \"GCGAGGTTCAGTTTCTTC\", 446)\n",
    "perform_primer_search(\"esp\", \"CGATAAAGAAGAGAGCGGAG\", \"GCAAACTCTACATCCACGTC\", 539)\n",
    "perform_primer_search(\"psaA\", \"CTATTTTGCAGCAAGTGATG\", \"CGCATAGTAACTATCACCATCTTG\", 540)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "282f45fc-6f13-41b2-b612-ed42b47edd15",
   "metadata": {},
   "source": [
    "Next, convert previous step to marker seed (multi-)FASTA files.\n",
    "\n",
    "ChronoStrain uses FASTA files to read in marker seeds. The previous step merely creates a catalog of raw gene catalog. We need to generate a FASTA file, but some cleanup has to be done!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "3ab860e5-fac0-405a-acff-4a7b2aff10f4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "cpsK.feather -> cpsK.fasta\n",
      "cbh.feather -> cbh.fasta\n",
      "cpsA.feather -> cpsA.fasta\n",
      "cpsB.feather -> cpsB.fasta\n",
      "cpsC.feather -> cpsC.fasta\n",
      "cpsD.feather -> cpsD.fasta\n",
      "cpsE.feather -> cpsE.fasta\n",
      "cpsF.feather -> cpsF.fasta\n",
      "cpsG.feather -> cpsG.fasta\n",
      "cpsH.feather -> cpsH.fasta\n",
      "cpsI.feather -> cpsI.fasta\n",
      "cpsJ.feather -> cpsJ.fasta\n",
      "cylA.feather -> cylA.fasta\n",
      "cylB.feather -> cylB.fasta\n",
      "cylM.feather -> cylM.fasta\n",
      "esp.feather -> esp.fasta\n",
      "fsrB.feather -> fsrB.fasta\n",
      "gelE.feather -> gelE.fasta\n",
      "gls24_like.feather -> gls24_like.fasta\n",
      "psaA.feather -> psaA.fasta\n"
     ]
    }
   ],
   "source": [
    "with open(MARKER_SEED_DIR / \"manual_seeds.tsv\", \"wt\") as metadata_tsv:\n",
    "    for feather_file in MARKER_SEED_DIR.glob(\"*.feather\"):\n",
    "        gene_name = feather_file.stem\n",
    "        fasta_path = MARKER_SEED_DIR / f'{gene_name}.fasta'\n",
    "        _df = pd.read_feather(feather_file)\n",
    "        with open(fasta_path, 'wt') as f:\n",
    "            for row_idx, row in _df.iterrows():\n",
    "                record = SeqRecord(\n",
    "                    Seq(row['GeneSeq']),\n",
    "                    id=\"{}_{}\".format(gene_name, row_idx),\n",
    "                    description=\"Src={}:{}\".format(row['Accession'], row['Gene'])\n",
    "                )\n",
    "                SeqIO.write([record], f, 'fasta')\n",
    "        print(f'{feather_file.name} -> {fasta_path.name}')\n",
    "        \n",
    "        print(\n",
    "            \"{}\\t{}\\t{}\".format(\n",
    "                gene_name, fasta_path, f\"POLYMORPHIC_{gene_name}\"\n",
    "            ), \n",
    "            file=metadata_tsv\n",
    "        )"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0bf56924-e7e2-4b6e-bab3-cdd7afb7c239",
   "metadata": {},
   "source": [
    "#### 3.4 Extract MetaPhlAn markers."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "991c1e35-3269-40b0-ba78-7b4eb0d384d7",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "!python python_helpers/extract_metaphlan_markers.py \\\n",
    "    -t \"$METAPHLAN_TAXONOMIC_KEY\" \\\n",
    "    -i \"$METAPHLAN_DB_PATH\" \\\n",
    "    -o \"$MARKER_SEED_DIR\"/metaphlan_seeds.tsv"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "feb4b1d1-f33a-476e-bc11-3d11e596914b",
   "metadata": {},
   "source": [
    "#### 3.5 Combine marker seed files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 77,
   "id": "5942cb82-efea-4c0c-99e5-cd601ca98852",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/youn/mambaforge/envs/chronostrain2/lib/python3.10/pty.py:89: RuntimeWarning: os.fork() was called. os.fork() is incompatible with multithreaded code, and JAX is multithreaded, so this will likely lead to a deadlock.\n",
      "  pid, fd = os.forkpty()\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Created Marker seed index: /mnt/e/infant_nt/database/marker_seeds/marker_seed_index.tsv\n"
     ]
    }
   ],
   "source": [
    "!cat \"$MARKER_SEED_DIR\"/mlst_seeds.tsv > $MARKER_SEED_INDEX\n",
    "!cat \"$MARKER_SEED_DIR\"/manual_seeds.tsv >> $MARKER_SEED_INDEX\n",
    "!cat \"$MARKER_SEED_DIR\"/metaphlan_seeds.tsv >> $MARKER_SEED_INDEX\n",
    "\n",
    "print(\"Created Marker seed index: {}\".format(MARKER_SEED_INDEX))\n",
    "assert MARKER_SEED_INDEX.exists()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8e13da06-b7af-43ad-acf6-820b48e08520",
   "metadata": {},
   "source": [
    "### Step 4: Run Chronostrain's make-db command.\n",
    "\n",
    "By the end of the previous step, we have:\n",
    "\n",
    "1) FASTA files for each gene, listing out seed sequence(s).\n",
    "2) A TSV file (marker_seed_index.tsv) containing a list of gene names and the paths to each of these FASTA files.\n",
    "\n",
    "Using these as inputs, we now construct the database files:\n",
    "1) A JSON file of the strain records and their markers.\n",
    "2) A TXT file of strain records clustered by similarity."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "63a29b9f-4556-43c0-9aae-501917058d5a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-02-25 12:13:14,646 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2024-02-25 12:13:15,304 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2024-02-25 12:13:15,398 [INFO - chronostrain.cli.make_db_json] - Building raw DB using BLAST.\n",
      "2024-02-25 12:13:15,401 [INFO - chronostrain.cli.make_db_json] - Creating strain entries from catalog /mnt/e/infant_nt/database/ref_genomes/index.tsv\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gdh.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gyd.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pstS.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gki.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on aroE.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on xpt.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yqiL.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsK.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cbh.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsB.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsC.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsD.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsE.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsF.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsG.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsH.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsI.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsJ.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylB.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylM.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on esp.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fsrB.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gelE.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gls24_like.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on psaA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on prgU.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on terS.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on xdh.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rplC.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpoC.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on greA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nfsA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on lacD.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ygjK.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on abcf.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on lpdA.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on amaP.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rplD.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gap.\n",
      "2024-02-25 12:13:15,452 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on purK.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ltrA.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on sea1.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpmF.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on mgtA.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pnp.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pgmB.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpmA.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cls.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dfr.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fabZ.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on guaB.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yqeK.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on walK.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on metG.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on uvrC.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on groL.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on zwf.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on guaD.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dnaE.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pyk.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pdhA.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pepT.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 12:13:15,453 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gdh`.\n",
      "2024-02-25 12:13:15,718 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gyd`.\n",
      "2024-02-25 12:13:15,996 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pstS`.\n",
      "2024-02-25 12:13:16,062 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gki`.\n",
      "2024-02-25 12:13:16,081 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `aroE`.\n",
      "2024-02-25 12:13:16,100 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `xpt`.\n",
      "2024-02-25 12:13:16,152 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yqiL`.\n",
      "2024-02-25 12:13:16,171 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsK`.\n",
      "2024-02-25 12:13:48,285 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cbh`.\n",
      "2024-02-25 12:15:24,323 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsA`.\n",
      "2024-02-25 12:21:52,512 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsB`.\n",
      "2024-02-25 12:29:00,090 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsC`.\n",
      "2024-02-25 12:29:44,111 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsD`.\n",
      "2024-02-25 12:30:26,585 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsE`.\n",
      "2024-02-25 12:31:07,809 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsF`.\n",
      "2024-02-25 12:31:19,786 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsG`.\n",
      "2024-02-25 12:31:48,389 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsH`.\n",
      "2024-02-25 12:32:30,975 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsI`.\n",
      "2024-02-25 12:33:04,972 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsJ`.\n",
      "2024-02-25 12:33:49,473 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylA`.\n",
      "2024-02-25 12:33:50,675 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylB`.\n",
      "2024-02-25 12:33:51,887 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylM`.\n",
      "2024-02-25 12:33:52,570 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `esp`.\n",
      "2024-02-25 12:34:10,842 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fsrB`.\n",
      "2024-02-25 12:36:16,716 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gelE`.\n",
      "2024-02-25 12:36:17,073 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gls24_like`.\n",
      "2024-02-25 12:36:17,074 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `psaA`.\n",
      "2024-02-25 12:36:33,536 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `prgU`.\n",
      "2024-02-25 12:36:44,624 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `terS`.\n",
      "2024-02-25 12:36:48,410 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `xdh`.\n",
      "2024-02-25 12:37:24,245 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rplC`.\n",
      "2024-02-25 12:38:28,716 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpoC`.\n",
      "2024-02-25 12:39:37,387 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `greA`.\n",
      "2024-02-25 12:40:11,362 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nfsA`.\n",
      "2024-02-25 12:40:45,145 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `lacD`.\n",
      "2024-02-25 12:41:54,699 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ygjK`.\n",
      "2024-02-25 12:42:30,736 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `abcf`.\n",
      "2024-02-25 12:43:06,467 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `lpdA`.\n",
      "2024-02-25 12:44:19,195 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `amaP`.\n",
      "2024-02-25 12:44:58,836 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rplD`.\n",
      "2024-02-25 12:46:07,120 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gap`.\n",
      "2024-02-25 12:47:49,876 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `purK`.\n",
      "2024-02-25 12:48:51,467 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ltrA`.\n",
      "2024-02-25 12:48:52,356 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `sea1`.\n",
      "2024-02-25 12:48:53,161 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpmF`.\n",
      "2024-02-25 12:50:03,294 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `mgtA`.\n",
      "2024-02-25 12:51:09,228 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pnp`.\n",
      "2024-02-25 12:52:10,552 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pgmB`.\n",
      "2024-02-25 12:53:21,876 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpmA`.\n",
      "2024-02-25 12:53:55,972 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cls`.\n",
      "2024-02-25 12:55:07,232 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dfr`.\n",
      "2024-02-25 12:55:43,945 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fabZ`.\n",
      "2024-02-25 12:56:52,439 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `guaB`.\n",
      "2024-02-25 12:57:54,662 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yqeK`.\n",
      "2024-02-25 12:58:36,524 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `walK`.\n",
      "2024-02-25 12:59:13,863 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `metG`.\n",
      "2024-02-25 13:00:13,777 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `uvrC`.\n",
      "2024-02-25 13:00:56,511 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `groL`.\n",
      "2024-02-25 13:01:52,615 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `zwf`.\n",
      "2024-02-25 13:02:29,880 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `guaD`.\n",
      "2024-02-25 13:03:06,967 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dnaE`.\n",
      "2024-02-25 13:03:44,591 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pyk`.\n",
      "2024-02-25 13:04:24,403 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pdhA`.\n",
      "2024-02-25 13:05:06,261 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pepT`.\n",
      "2024-02-25 13:06:17,740 [INFO - chronostrain.cli.make_db_json] - Creating strain entries from catalog /mnt/e/infant_nt/database/infant_isolates/index.tsv\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gdh.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gyd.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pstS.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gki.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on aroE.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on xpt.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yqiL.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsK.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cbh.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsB.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsC.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsD.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsE.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsF.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsG.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsH.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsI.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cpsJ.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylB.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cylM.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on esp.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fsrB.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gelE.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gls24_like.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on psaA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on prgU.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on terS.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on xdh.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rplC.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpoC.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on greA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on nfsA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on lacD.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ygjK.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on abcf.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on lpdA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on amaP.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rplD.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on gap.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on purK.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on ltrA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on sea1.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpmF.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on mgtA.\n",
      "2024-02-25 13:06:17,795 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pnp.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pgmB.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on rpmA.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on cls.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dfr.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on fabZ.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on guaB.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on yqeK.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on walK.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on metG.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on uvrC.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on groL.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on zwf.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on guaD.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on dnaE.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pyk.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pdhA.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - Running blastn on pepT.\n",
      "2024-02-25 13:06:17,796 [DEBUG - chronostrain.cli.make_db_json] - skipping actual blastn call. (using previously evaluated ones) -- TODO undo this temporary change.\n",
      "2024-02-25 13:06:17,797 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gdh`.\n",
      "2024-02-25 13:06:17,898 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gyd`.\n",
      "2024-02-25 13:06:17,905 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pstS`.\n",
      "2024-02-25 13:06:17,911 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gki`.\n",
      "2024-02-25 13:06:17,917 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `aroE`.\n",
      "2024-02-25 13:06:17,923 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `xpt`.\n",
      "2024-02-25 13:06:17,929 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yqiL`.\n",
      "2024-02-25 13:06:17,935 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsK`.\n",
      "2024-02-25 13:06:25,158 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cbh`.\n",
      "2024-02-25 13:06:41,715 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsA`.\n",
      "2024-02-25 13:08:59,007 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsB`.\n",
      "2024-02-25 13:11:23,559 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsC`.\n",
      "2024-02-25 13:11:31,335 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsD`.\n",
      "2024-02-25 13:11:39,247 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsE`.\n",
      "2024-02-25 13:11:46,897 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsF`.\n",
      "2024-02-25 13:11:48,860 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsG`.\n",
      "2024-02-25 13:11:53,592 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsH`.\n",
      "2024-02-25 13:12:01,203 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsI`.\n",
      "2024-02-25 13:12:08,082 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cpsJ`.\n",
      "2024-02-25 13:12:15,904 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylA`.\n",
      "2024-02-25 13:12:16,884 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylB`.\n",
      "2024-02-25 13:12:17,928 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cylM`.\n",
      "2024-02-25 13:12:18,124 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `esp`.\n",
      "2024-02-25 13:12:22,829 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fsrB`.\n",
      "2024-02-25 13:12:55,628 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gelE`.\n",
      "2024-02-25 13:12:55,739 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gls24_like`.\n",
      "2024-02-25 13:12:55,739 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `psaA`.\n",
      "2024-02-25 13:13:00,041 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `prgU`.\n",
      "2024-02-25 13:13:10,582 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `terS`.\n",
      "2024-02-25 13:13:11,967 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `xdh`.\n",
      "2024-02-25 13:13:21,429 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rplC`.\n",
      "2024-02-25 13:13:31,455 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpoC`.\n",
      "2024-02-25 13:13:41,123 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `greA`.\n",
      "2024-02-25 13:13:51,005 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `nfsA`.\n",
      "2024-02-25 13:14:00,436 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `lacD`.\n",
      "2024-02-25 13:14:22,797 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ygjK`.\n",
      "2024-02-25 13:14:32,295 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `abcf`.\n",
      "2024-02-25 13:14:41,655 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `lpdA`.\n",
      "2024-02-25 13:15:03,063 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `amaP`.\n",
      "2024-02-25 13:15:15,928 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rplD`.\n",
      "2024-02-25 13:15:25,482 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `gap`.\n",
      "2024-02-25 13:15:47,794 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `purK`.\n",
      "2024-02-25 13:16:06,768 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `ltrA`.\n",
      "2024-02-25 13:16:07,123 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `sea1`.\n",
      "2024-02-25 13:16:11,487 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpmF`.\n",
      "2024-02-25 13:16:35,013 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `mgtA`.\n",
      "2024-02-25 13:16:54,432 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pnp`.\n",
      "2024-02-25 13:17:04,279 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pgmB`.\n",
      "2024-02-25 13:17:26,311 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `rpmA`.\n",
      "2024-02-25 13:17:34,745 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `cls`.\n",
      "2024-02-25 13:17:56,737 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dfr`.\n",
      "2024-02-25 13:18:06,591 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `fabZ`.\n",
      "2024-02-25 13:18:28,599 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `guaB`.\n",
      "2024-02-25 13:18:38,098 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `yqeK`.\n",
      "2024-02-25 13:18:50,650 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `walK`.\n",
      "2024-02-25 13:19:00,389 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `metG`.\n",
      "2024-02-25 13:19:10,148 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `uvrC`.\n",
      "2024-02-25 13:19:23,298 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `groL`.\n",
      "2024-02-25 13:19:33,079 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `zwf`.\n",
      "2024-02-25 13:19:42,852 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `guaD`.\n",
      "2024-02-25 13:19:52,684 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `dnaE`.\n",
      "2024-02-25 13:20:03,023 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pyk`.\n",
      "2024-02-25 13:20:12,876 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pdhA`.\n",
      "2024-02-25 13:20:22,652 [DEBUG - chronostrain.cli.make_db_json] - Parsing BLAST hits for gene `pepT`.\n",
      "2024-02-25 13:20:45,907 [INFO - chronostrain.cli.make_db_json] - Wrote raw blast DB entries to /mnt/e/infant_nt/database/chronostrain_files/efaecalis-1raw.json.\n",
      "2024-02-25 13:20:45,907 [INFO - chronostrain.cli.make_db_json] - Resolving overlaps.\n",
      "2024-02-25 13:20:45,907 [DEBUG - chronostrain.cli.make_db_json] - Src: /mnt/e/infant_nt/database/chronostrain_files/efaecalis-1raw.json, Dest: /mnt/e/infant_nt/database/chronostrain_files/efaecalis-2overlapmerged.json\n",
      "2024-02-25 13:20:46,040 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP022488.1, seqID CP022488.1.\n",
      "2024-02-25 13:20:46,057 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022488.1, seqID NZ_CP022488.1.\n",
      "2024-02-25 13:20:46,072 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940434.1, seqID OD940434.1.\n",
      "2024-02-25 13:20:46,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940440.2, seqID OD940440.2.\n",
      "2024-02-25 13:20:46,103 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940422.1, seqID OD940422.1.\n",
      "2024-02-25 13:20:46,104 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962855.1.\n",
      "2024-02-25 13:20:46,104 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962855.1.\n",
      "2024-02-25 13:20:46,104 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962855.1, seqID LR962855.1.\n",
      "2024-02-25 13:20:46,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962846.1.\n",
      "2024-02-25 13:20:46,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962846.1.\n",
      "2024-02-25 13:20:46,105 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962846.1, seqID LR962846.1.\n",
      "2024-02-25 13:20:46,107 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962827.1.\n",
      "2024-02-25 13:20:46,107 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962827.1.\n",
      "2024-02-25 13:20:46,107 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962827.1, seqID LR962827.1.\n",
      "2024-02-25 13:20:46,108 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962806.1.\n",
      "2024-02-25 13:20:46,108 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962806.1.\n",
      "2024-02-25 13:20:46,108 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962806.1, seqID LR962806.1.\n",
      "2024-02-25 13:20:46,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962796.1.\n",
      "2024-02-25 13:20:46,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962796.1.\n",
      "2024-02-25 13:20:46,109 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962796.1, seqID LR962796.1.\n",
      "2024-02-25 13:20:46,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962716.1.\n",
      "2024-02-25 13:20:46,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962716.1.\n",
      "2024-02-25 13:20:46,110 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962716.1, seqID LR962716.1.\n",
      "2024-02-25 13:20:46,112 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962708.1.\n",
      "2024-02-25 13:20:46,112 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962708.1.\n",
      "2024-02-25 13:20:46,112 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962708.1, seqID LR962708.1.\n",
      "2024-02-25 13:20:46,113 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962700.1.\n",
      "2024-02-25 13:20:46,113 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962700.1.\n",
      "2024-02-25 13:20:46,113 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962700.1, seqID LR962700.1.\n",
      "2024-02-25 13:20:46,114 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962684.1.\n",
      "2024-02-25 13:20:46,114 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962684.1.\n",
      "2024-02-25 13:20:46,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962684.1, seqID LR962684.1.\n",
      "2024-02-25 13:20:46,115 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962642.1.\n",
      "2024-02-25 13:20:46,115 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962642.1.\n",
      "2024-02-25 13:20:46,115 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962642.1, seqID LR962642.1.\n",
      "2024-02-25 13:20:46,117 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962509.1.\n",
      "2024-02-25 13:20:46,117 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962509.1.\n",
      "2024-02-25 13:20:46,117 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962509.1, seqID LR962509.1.\n",
      "2024-02-25 13:20:46,118 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962521.1.\n",
      "2024-02-25 13:20:46,118 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962521.1.\n",
      "2024-02-25 13:20:46,118 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962521.1, seqID LR962521.1.\n",
      "2024-02-25 13:20:46,119 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962474.1.\n",
      "2024-02-25 13:20:46,119 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962474.1.\n",
      "2024-02-25 13:20:46,119 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962474.1, seqID LR962474.1.\n",
      "2024-02-25 13:20:46,120 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962440.1.\n",
      "2024-02-25 13:20:46,120 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962440.1.\n",
      "2024-02-25 13:20:46,120 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962440.1, seqID LR962440.1.\n",
      "2024-02-25 13:20:46,121 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962444.1.\n",
      "2024-02-25 13:20:46,121 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962444.1.\n",
      "2024-02-25 13:20:46,121 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962444.1, seqID LR962444.1.\n",
      "2024-02-25 13:20:46,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962403.1.\n",
      "2024-02-25 13:20:46,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962403.1.\n",
      "2024-02-25 13:20:46,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962403.1, seqID LR962403.1.\n",
      "2024-02-25 13:20:46,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962396.1.\n",
      "2024-02-25 13:20:46,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962396.1.\n",
      "2024-02-25 13:20:46,124 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962396.1, seqID LR962396.1.\n",
      "2024-02-25 13:20:46,125 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962363.1.\n",
      "2024-02-25 13:20:46,125 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962363.1.\n",
      "2024-02-25 13:20:46,125 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962363.1, seqID LR962363.1.\n",
      "2024-02-25 13:20:46,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962361.1.\n",
      "2024-02-25 13:20:46,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962361.1.\n",
      "2024-02-25 13:20:46,126 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962361.1, seqID LR962361.1.\n",
      "2024-02-25 13:20:46,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962359.1.\n",
      "2024-02-25 13:20:46,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962359.1.\n",
      "2024-02-25 13:20:46,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962359.1, seqID LR962359.1.\n",
      "2024-02-25 13:20:46,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962353.1.\n",
      "2024-02-25 13:20:46,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962353.1.\n",
      "2024-02-25 13:20:46,128 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962353.1, seqID LR962353.1.\n",
      "2024-02-25 13:20:46,130 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962336.1.\n",
      "2024-02-25 13:20:46,130 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962336.1.\n",
      "2024-02-25 13:20:46,130 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962336.1, seqID LR962336.1.\n",
      "2024-02-25 13:20:46,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962298.1.\n",
      "2024-02-25 13:20:46,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962298.1.\n",
      "2024-02-25 13:20:46,131 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962298.1, seqID LR962298.1.\n",
      "2024-02-25 13:20:46,132 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962185.1.\n",
      "2024-02-25 13:20:46,132 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962185.1.\n",
      "2024-02-25 13:20:46,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962185.1, seqID LR962185.1.\n",
      "2024-02-25 13:20:46,133 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962115.1.\n",
      "2024-02-25 13:20:46,133 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962115.1.\n",
      "2024-02-25 13:20:46,133 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962115.1, seqID LR962115.1.\n",
      "2024-02-25 13:20:46,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962036.1.\n",
      "2024-02-25 13:20:46,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962036.1.\n",
      "2024-02-25 13:20:46,134 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962036.1, seqID LR962036.1.\n",
      "2024-02-25 13:20:46,136 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962060.1.\n",
      "2024-02-25 13:20:46,136 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962060.1.\n",
      "2024-02-25 13:20:46,136 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962060.1, seqID LR962060.1.\n",
      "2024-02-25 13:20:46,152 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR594051.1, seqID LR594051.1.\n",
      "2024-02-25 13:20:46,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR134312.1, seqID LR134312.1.\n",
      "2024-02-25 13:20:46,185 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124918.1, seqID CP124918.1.\n",
      "2024-02-25 13:20:46,203 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124906.1, seqID CP124906.1.\n",
      "2024-02-25 13:20:46,222 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124889.1, seqID CP124889.1.\n",
      "2024-02-25 13:20:46,240 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124886.1, seqID CP124886.1.\n",
      "2024-02-25 13:20:46,258 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118072.1, seqID CP118072.1.\n",
      "2024-02-25 13:20:46,279 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP117508.1, seqID CP117508.1.\n",
      "2024-02-25 13:20:46,299 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116575.1, seqID CP116575.1.\n",
      "2024-02-25 13:20:46,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP103863.1, seqID CP103863.1.\n",
      "2024-02-25 13:20:46,334 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091231.1, seqID CP091231.1.\n",
      "2024-02-25 13:20:46,350 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP088200.1, seqID CP088200.1.\n",
      "2024-02-25 13:20:46,366 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP025270.1, seqID AP025270.1.\n",
      "2024-02-25 13:20:46,383 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP065784.1, seqID CP065784.1.\n",
      "2024-02-25 13:20:46,400 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP022059.2, seqID CP022059.2.\n",
      "2024-02-25 13:20:46,419 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940434.1, seqID NZ_OD940434.1.\n",
      "2024-02-25 13:20:46,438 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940440.2, seqID NZ_OD940440.2.\n",
      "2024-02-25 13:20:46,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940422.1, seqID NZ_OD940422.1.\n",
      "2024-02-25 13:20:46,473 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962855.1, seqID NZ_LR962855.1.\n",
      "2024-02-25 13:20:46,489 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962846.1, seqID NZ_LR962846.1.\n",
      "2024-02-25 13:20:46,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962827.1, seqID NZ_LR962827.1.\n",
      "2024-02-25 13:20:46,522 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962806.1, seqID NZ_LR962806.1.\n",
      "2024-02-25 13:20:46,538 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962796.1, seqID NZ_LR962796.1.\n",
      "2024-02-25 13:20:46,554 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962716.1, seqID NZ_LR962716.1.\n",
      "2024-02-25 13:20:46,570 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962708.1, seqID NZ_LR962708.1.\n",
      "2024-02-25 13:20:46,587 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962700.1, seqID NZ_LR962700.1.\n",
      "2024-02-25 13:20:46,603 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962684.1, seqID NZ_LR962684.1.\n",
      "2024-02-25 13:20:46,620 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962642.1, seqID NZ_LR962642.1.\n",
      "2024-02-25 13:20:46,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962509.1, seqID NZ_LR962509.1.\n",
      "2024-02-25 13:20:46,651 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962521.1, seqID NZ_LR962521.1.\n",
      "2024-02-25 13:20:46,667 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962474.1, seqID NZ_LR962474.1.\n",
      "2024-02-25 13:20:46,683 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962440.1, seqID NZ_LR962440.1.\n",
      "2024-02-25 13:20:46,699 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962444.1, seqID NZ_LR962444.1.\n",
      "2024-02-25 13:20:46,714 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962403.1, seqID NZ_LR962403.1.\n",
      "2024-02-25 13:20:46,731 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962396.1, seqID NZ_LR962396.1.\n",
      "2024-02-25 13:20:46,747 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962363.1, seqID NZ_LR962363.1.\n",
      "2024-02-25 13:20:46,763 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962361.1, seqID NZ_LR962361.1.\n",
      "2024-02-25 13:20:46,779 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962359.1, seqID NZ_LR962359.1.\n",
      "2024-02-25 13:20:46,795 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962353.1, seqID NZ_LR962353.1.\n",
      "2024-02-25 13:20:46,812 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962336.1, seqID NZ_LR962336.1.\n",
      "2024-02-25 13:20:46,828 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962298.1, seqID NZ_LR962298.1.\n",
      "2024-02-25 13:20:46,843 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962185.1, seqID NZ_LR962185.1.\n",
      "2024-02-25 13:20:46,860 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962115.1, seqID NZ_LR962115.1.\n",
      "2024-02-25 13:20:46,876 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962036.1, seqID NZ_LR962036.1.\n",
      "2024-02-25 13:20:46,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962060.1, seqID NZ_LR962060.1.\n",
      "2024-02-25 13:20:46,912 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR594051.1, seqID NZ_LR594051.1.\n",
      "2024-02-25 13:20:46,929 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR134312.1, seqID NZ_LR134312.1.\n",
      "2024-02-25 13:20:46,948 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124918.1, seqID NZ_CP124918.1.\n",
      "2024-02-25 13:20:46,966 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124906.1, seqID NZ_CP124906.1.\n",
      "2024-02-25 13:20:46,985 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124889.1, seqID NZ_CP124889.1.\n",
      "2024-02-25 13:20:47,004 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124886.1, seqID NZ_CP124886.1.\n",
      "2024-02-25 13:20:47,022 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP117508.1, seqID NZ_CP117508.1.\n",
      "2024-02-25 13:20:47,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116575.1, seqID NZ_CP116575.1.\n",
      "2024-02-25 13:20:47,060 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP103863.1, seqID NZ_CP103863.1.\n",
      "2024-02-25 13:20:47,080 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091231.1, seqID NZ_CP091231.1.\n",
      "2024-02-25 13:20:47,099 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088200.1, seqID NZ_CP088200.1.\n",
      "2024-02-25 13:20:47,117 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP025270.1, seqID NZ_AP025270.1.\n",
      "2024-02-25 13:20:47,136 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP065784.1, seqID NZ_CP065784.1.\n",
      "2024-02-25 13:20:47,155 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP022059.1, seqID NZ_CP022059.1.\n",
      "2024-02-25 13:20:47,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AE016830.1, seqID AE016830.1.\n",
      "2024-02-25 13:20:47,195 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_004668.1, seqID NC_004668.1.\n",
      "2024-02-25 13:20:47,196 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP041738.1.\n",
      "2024-02-25 13:20:47,196 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP041738.1.\n",
      "2024-02-25 13:20:47,196 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP041738.1, seqID CP041738.1.\n",
      "2024-02-25 13:20:47,215 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041738.1, seqID NZ_CP041738.1.\n",
      "2024-02-25 13:20:47,231 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP004081.1, seqID CP004081.1.\n",
      "2024-02-25 13:20:47,251 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP004081.1, seqID NZ_CP004081.1.\n",
      "2024-02-25 13:20:47,252 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962862.1.\n",
      "2024-02-25 13:20:47,252 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962862.1.\n",
      "2024-02-25 13:20:47,252 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962862.1, seqID LR962862.1.\n",
      "2024-02-25 13:20:47,253 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962859.1.\n",
      "2024-02-25 13:20:47,253 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962859.1.\n",
      "2024-02-25 13:20:47,253 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962859.1, seqID LR962859.1.\n",
      "2024-02-25 13:20:47,255 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962821.1.\n",
      "2024-02-25 13:20:47,255 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962821.1.\n",
      "2024-02-25 13:20:47,255 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962821.1, seqID LR962821.1.\n",
      "2024-02-25 13:20:47,256 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962784.1.\n",
      "2024-02-25 13:20:47,256 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962784.1.\n",
      "2024-02-25 13:20:47,256 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962784.1, seqID LR962784.1.\n",
      "2024-02-25 13:20:47,257 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962782.1.\n",
      "2024-02-25 13:20:47,257 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962782.1.\n",
      "2024-02-25 13:20:47,257 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962782.1, seqID LR962782.1.\n",
      "2024-02-25 13:20:47,258 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962731.1.\n",
      "2024-02-25 13:20:47,258 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962731.1.\n",
      "2024-02-25 13:20:47,258 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962731.1, seqID LR962731.1.\n",
      "2024-02-25 13:20:47,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962702.1.\n",
      "2024-02-25 13:20:47,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962702.1.\n",
      "2024-02-25 13:20:47,260 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962702.1, seqID LR962702.1.\n",
      "2024-02-25 13:20:47,261 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962639.1.\n",
      "2024-02-25 13:20:47,261 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962639.1.\n",
      "2024-02-25 13:20:47,261 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962639.1, seqID LR962639.1.\n",
      "2024-02-25 13:20:47,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962647.1.\n",
      "2024-02-25 13:20:47,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962647.1.\n",
      "2024-02-25 13:20:47,263 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962647.1, seqID LR962647.1.\n",
      "2024-02-25 13:20:47,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962636.1.\n",
      "2024-02-25 13:20:47,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962636.1.\n",
      "2024-02-25 13:20:47,264 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962636.1, seqID LR962636.1.\n",
      "2024-02-25 13:20:47,265 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962615.1.\n",
      "2024-02-25 13:20:47,265 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962615.1.\n",
      "2024-02-25 13:20:47,265 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962615.1, seqID LR962615.1.\n",
      "2024-02-25 13:20:47,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962628.1.\n",
      "2024-02-25 13:20:47,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962628.1.\n",
      "2024-02-25 13:20:47,266 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962628.1, seqID LR962628.1.\n",
      "2024-02-25 13:20:47,268 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962611.1.\n",
      "2024-02-25 13:20:47,268 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962611.1.\n",
      "2024-02-25 13:20:47,268 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962611.1, seqID LR962611.1.\n",
      "2024-02-25 13:20:47,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962586.1.\n",
      "2024-02-25 13:20:47,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962586.1.\n",
      "2024-02-25 13:20:47,269 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962586.1, seqID LR962586.1.\n",
      "2024-02-25 13:20:47,271 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962584.1.\n",
      "2024-02-25 13:20:47,271 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962584.1.\n",
      "2024-02-25 13:20:47,271 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962584.1, seqID LR962584.1.\n",
      "2024-02-25 13:20:47,272 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962559.1.\n",
      "2024-02-25 13:20:47,272 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962559.1.\n",
      "2024-02-25 13:20:47,272 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962559.1, seqID LR962559.1.\n",
      "2024-02-25 13:20:47,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962561.1.\n",
      "2024-02-25 13:20:47,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962561.1.\n",
      "2024-02-25 13:20:47,273 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962561.1, seqID LR962561.1.\n",
      "2024-02-25 13:20:47,275 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962558.1.\n",
      "2024-02-25 13:20:47,275 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962558.1.\n",
      "2024-02-25 13:20:47,275 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962558.1, seqID LR962558.1.\n",
      "2024-02-25 13:20:47,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962569.1.\n",
      "2024-02-25 13:20:47,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962569.1.\n",
      "2024-02-25 13:20:47,276 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962569.1, seqID LR962569.1.\n",
      "2024-02-25 13:20:47,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962574.1.\n",
      "2024-02-25 13:20:47,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962574.1.\n",
      "2024-02-25 13:20:47,277 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962574.1, seqID LR962574.1.\n",
      "2024-02-25 13:20:47,278 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962565.1.\n",
      "2024-02-25 13:20:47,278 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962565.1.\n",
      "2024-02-25 13:20:47,278 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962565.1, seqID LR962565.1.\n",
      "2024-02-25 13:20:47,280 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962551.1.\n",
      "2024-02-25 13:20:47,280 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962551.1.\n",
      "2024-02-25 13:20:47,280 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962551.1, seqID LR962551.1.\n",
      "2024-02-25 13:20:47,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962542.1.\n",
      "2024-02-25 13:20:47,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962542.1.\n",
      "2024-02-25 13:20:47,281 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962542.1, seqID LR962542.1.\n",
      "2024-02-25 13:20:47,282 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962526.1.\n",
      "2024-02-25 13:20:47,282 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962526.1.\n",
      "2024-02-25 13:20:47,282 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962526.1, seqID LR962526.1.\n",
      "2024-02-25 13:20:47,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962502.1.\n",
      "2024-02-25 13:20:47,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962502.1.\n",
      "2024-02-25 13:20:47,284 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962502.1, seqID LR962502.1.\n",
      "2024-02-25 13:20:47,285 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962507.1.\n",
      "2024-02-25 13:20:47,285 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962507.1.\n",
      "2024-02-25 13:20:47,285 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962507.1, seqID LR962507.1.\n",
      "2024-02-25 13:20:47,286 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962523.1.\n",
      "2024-02-25 13:20:47,286 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962523.1.\n",
      "2024-02-25 13:20:47,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962523.1, seqID LR962523.1.\n",
      "2024-02-25 13:20:47,288 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962495.1.\n",
      "2024-02-25 13:20:47,288 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962495.1.\n",
      "2024-02-25 13:20:47,288 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962495.1, seqID LR962495.1.\n",
      "2024-02-25 13:20:47,289 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962464.1.\n",
      "2024-02-25 13:20:47,289 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962464.1.\n",
      "2024-02-25 13:20:47,289 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962464.1, seqID LR962464.1.\n",
      "2024-02-25 13:20:47,291 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962418.1.\n",
      "2024-02-25 13:20:47,291 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962418.1.\n",
      "2024-02-25 13:20:47,291 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962418.1, seqID LR962418.1.\n",
      "2024-02-25 13:20:47,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962423.1.\n",
      "2024-02-25 13:20:47,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962423.1.\n",
      "2024-02-25 13:20:47,292 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962423.1, seqID LR962423.1.\n",
      "2024-02-25 13:20:47,293 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962410.1.\n",
      "2024-02-25 13:20:47,293 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962410.1.\n",
      "2024-02-25 13:20:47,293 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962410.1, seqID LR962410.1.\n",
      "2024-02-25 13:20:47,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962421.1.\n",
      "2024-02-25 13:20:47,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962421.1.\n",
      "2024-02-25 13:20:47,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962421.1, seqID LR962421.1.\n",
      "2024-02-25 13:20:47,296 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962407.1.\n",
      "2024-02-25 13:20:47,296 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962407.1.\n",
      "2024-02-25 13:20:47,296 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962407.1, seqID LR962407.1.\n",
      "2024-02-25 13:20:47,297 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962386.1.\n",
      "2024-02-25 13:20:47,297 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962386.1.\n",
      "2024-02-25 13:20:47,297 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962386.1, seqID LR962386.1.\n",
      "2024-02-25 13:20:47,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962378.1.\n",
      "2024-02-25 13:20:47,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962378.1.\n",
      "2024-02-25 13:20:47,298 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962378.1, seqID LR962378.1.\n",
      "2024-02-25 13:20:47,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962367.1.\n",
      "2024-02-25 13:20:47,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962367.1.\n",
      "2024-02-25 13:20:47,300 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962367.1, seqID LR962367.1.\n",
      "2024-02-25 13:20:47,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962372.1.\n",
      "2024-02-25 13:20:47,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962372.1.\n",
      "2024-02-25 13:20:47,301 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962372.1, seqID LR962372.1.\n",
      "2024-02-25 13:20:47,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962356.1.\n",
      "2024-02-25 13:20:47,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962356.1.\n",
      "2024-02-25 13:20:47,302 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962356.1, seqID LR962356.1.\n",
      "2024-02-25 13:20:47,303 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962321.1.\n",
      "2024-02-25 13:20:47,303 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962321.1.\n",
      "2024-02-25 13:20:47,303 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962321.1, seqID LR962321.1.\n",
      "2024-02-25 13:20:47,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962331.1.\n",
      "2024-02-25 13:20:47,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962331.1.\n",
      "2024-02-25 13:20:47,305 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962331.1, seqID LR962331.1.\n",
      "2024-02-25 13:20:47,306 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962306.1.\n",
      "2024-02-25 13:20:47,306 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962306.1.\n",
      "2024-02-25 13:20:47,306 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962306.1, seqID LR962306.1.\n",
      "2024-02-25 13:20:47,307 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962302.1.\n",
      "2024-02-25 13:20:47,307 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962302.1.\n",
      "2024-02-25 13:20:47,307 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962302.1, seqID LR962302.1.\n",
      "2024-02-25 13:20:47,309 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962267.1.\n",
      "2024-02-25 13:20:47,309 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962267.1.\n",
      "2024-02-25 13:20:47,309 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962267.1, seqID LR962267.1.\n",
      "2024-02-25 13:20:47,310 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962271.1.\n",
      "2024-02-25 13:20:47,310 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962271.1.\n",
      "2024-02-25 13:20:47,310 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962271.1, seqID LR962271.1.\n",
      "2024-02-25 13:20:47,311 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962277.1.\n",
      "2024-02-25 13:20:47,311 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962277.1.\n",
      "2024-02-25 13:20:47,311 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962277.1, seqID LR962277.1.\n",
      "2024-02-25 13:20:47,313 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962248.1.\n",
      "2024-02-25 13:20:47,313 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962248.1.\n",
      "2024-02-25 13:20:47,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962248.1, seqID LR962248.1.\n",
      "2024-02-25 13:20:47,314 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962203.1.\n",
      "2024-02-25 13:20:47,314 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962203.1.\n",
      "2024-02-25 13:20:47,314 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962203.1, seqID LR962203.1.\n",
      "2024-02-25 13:20:47,316 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962196.1.\n",
      "2024-02-25 13:20:47,316 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962196.1.\n",
      "2024-02-25 13:20:47,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962196.1, seqID LR962196.1.\n",
      "2024-02-25 13:20:47,317 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962227.1.\n",
      "2024-02-25 13:20:47,317 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962227.1.\n",
      "2024-02-25 13:20:47,317 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962227.1, seqID LR962227.1.\n",
      "2024-02-25 13:20:47,318 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962174.1.\n",
      "2024-02-25 13:20:47,318 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962174.1.\n",
      "2024-02-25 13:20:47,318 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962174.1, seqID LR962174.1.\n",
      "2024-02-25 13:20:47,320 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962152.1.\n",
      "2024-02-25 13:20:47,320 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962152.1.\n",
      "2024-02-25 13:20:47,320 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962152.1, seqID LR962152.1.\n",
      "2024-02-25 13:20:47,321 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962155.1.\n",
      "2024-02-25 13:20:47,321 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962155.1.\n",
      "2024-02-25 13:20:47,321 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962155.1, seqID LR962155.1.\n",
      "2024-02-25 13:20:47,322 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962169.1.\n",
      "2024-02-25 13:20:47,322 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962169.1.\n",
      "2024-02-25 13:20:47,322 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962169.1, seqID LR962169.1.\n",
      "2024-02-25 13:20:47,323 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962157.1.\n",
      "2024-02-25 13:20:47,323 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962157.1.\n",
      "2024-02-25 13:20:47,323 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962157.1, seqID LR962157.1.\n",
      "2024-02-25 13:20:47,325 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962141.1.\n",
      "2024-02-25 13:20:47,325 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962141.1.\n",
      "2024-02-25 13:20:47,325 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962141.1, seqID LR962141.1.\n",
      "2024-02-25 13:20:47,326 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962165.1.\n",
      "2024-02-25 13:20:47,326 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962165.1.\n",
      "2024-02-25 13:20:47,326 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962165.1, seqID LR962165.1.\n",
      "2024-02-25 13:20:47,327 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962121.1.\n",
      "2024-02-25 13:20:47,327 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962121.1.\n",
      "2024-02-25 13:20:47,327 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962121.1, seqID LR962121.1.\n",
      "2024-02-25 13:20:47,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962129.1.\n",
      "2024-02-25 13:20:47,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962129.1.\n",
      "2024-02-25 13:20:47,329 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962129.1, seqID LR962129.1.\n",
      "2024-02-25 13:20:47,330 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962119.1.\n",
      "2024-02-25 13:20:47,330 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962119.1.\n",
      "2024-02-25 13:20:47,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962119.1, seqID LR962119.1.\n",
      "2024-02-25 13:20:47,331 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962056.1.\n",
      "2024-02-25 13:20:47,331 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962056.1.\n",
      "2024-02-25 13:20:47,331 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962056.1, seqID LR962056.1.\n",
      "2024-02-25 13:20:47,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962025.1.\n",
      "2024-02-25 13:20:47,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962025.1.\n",
      "2024-02-25 13:20:47,333 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962025.1, seqID LR962025.1.\n",
      "2024-02-25 13:20:47,334 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962004.1.\n",
      "2024-02-25 13:20:47,334 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962004.1.\n",
      "2024-02-25 13:20:47,334 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962004.1, seqID LR962004.1.\n",
      "2024-02-25 13:20:47,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962013.1.\n",
      "2024-02-25 13:20:47,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962013.1.\n",
      "2024-02-25 13:20:47,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962013.1, seqID LR962013.1.\n",
      "2024-02-25 13:20:47,337 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961991.1.\n",
      "2024-02-25 13:20:47,337 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961991.1.\n",
      "2024-02-25 13:20:47,337 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961991.1, seqID LR961991.1.\n",
      "2024-02-25 13:20:47,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961997.1.\n",
      "2024-02-25 13:20:47,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961997.1.\n",
      "2024-02-25 13:20:47,338 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961997.1, seqID LR961997.1.\n",
      "2024-02-25 13:20:47,340 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961935.1.\n",
      "2024-02-25 13:20:47,340 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961935.1.\n",
      "2024-02-25 13:20:47,340 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961935.1, seqID LR961935.1.\n",
      "2024-02-25 13:20:47,356 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP027297.1, seqID AP027297.1.\n",
      "2024-02-25 13:20:47,374 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124921.1, seqID CP124921.1.\n",
      "2024-02-25 13:20:47,393 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124908.1, seqID CP124908.1.\n",
      "2024-02-25 13:20:47,412 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124894.1, seqID CP124894.1.\n",
      "2024-02-25 13:20:47,430 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124884.1, seqID CP124884.1.\n",
      "2024-02-25 13:20:47,448 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP106757.1, seqID CP106757.1.\n",
      "2024-02-25 13:20:47,468 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP119528.1, seqID CP119528.1.\n",
      "2024-02-25 13:20:47,488 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP119159.1, seqID CP119159.1.\n",
      "2024-02-25 13:20:47,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118085.1, seqID CP118085.1.\n",
      "2024-02-25 13:20:47,524 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118057.1, seqID CP118057.1.\n",
      "2024-02-25 13:20:47,542 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118755.1, seqID CP118755.1.\n",
      "2024-02-25 13:20:47,560 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP113831.1, seqID CP113831.1.\n",
      "2024-02-25 13:20:47,579 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110030.1, seqID CP110030.1.\n",
      "2024-02-25 13:20:47,580 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP102070.1.\n",
      "2024-02-25 13:20:47,580 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP102070.1.\n",
      "2024-02-25 13:20:47,580 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP102070.1, seqID CP102070.1.\n",
      "2024-02-25 13:20:47,596 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097002.1, seqID CP097002.1.\n",
      "2024-02-25 13:20:47,613 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP092576.1, seqID CP092576.1.\n",
      "2024-02-25 13:20:47,631 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP092574.1, seqID CP092574.1.\n",
      "2024-02-25 13:20:47,648 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP076498.1, seqID CP076498.1.\n",
      "2024-02-25 13:20:47,665 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP076493.1, seqID CP076493.1.\n",
      "2024-02-25 13:20:47,683 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP076488.1, seqID CP076488.1.\n",
      "2024-02-25 13:20:47,700 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091901.1, seqID CP091901.1.\n",
      "2024-02-25 13:20:47,717 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091899.1, seqID CP091899.1.\n",
      "2024-02-25 13:20:47,733 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091892.1, seqID CP091892.1.\n",
      "2024-02-25 13:20:47,751 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091889.1, seqID CP091889.1.\n",
      "2024-02-25 13:20:47,771 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091198.1, seqID CP091198.1.\n",
      "2024-02-25 13:20:47,788 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086726.1, seqID CP086726.1.\n",
      "2024-02-25 13:20:47,805 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086592.1, seqID CP086592.1.\n",
      "2024-02-25 13:20:47,822 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086558.1, seqID CP086558.1.\n",
      "2024-02-25 13:20:47,838 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP082231.1, seqID CP082231.1.\n",
      "2024-02-25 13:20:47,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP075604.2, seqID CP075604.2.\n",
      "2024-02-25 13:20:47,872 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046113.1, seqID CP046113.1.\n",
      "2024-02-25 13:20:47,888 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP049775.1, seqID CP049775.1.\n",
      "2024-02-25 13:20:47,904 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP041344.1, seqID CP041344.1.\n",
      "2024-02-25 13:20:47,905 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039549.1.\n",
      "2024-02-25 13:20:47,905 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039549.1.\n",
      "2024-02-25 13:20:47,905 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP039549.1, seqID CP039549.1.\n",
      "2024-02-25 13:20:47,907 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039548.1.\n",
      "2024-02-25 13:20:47,907 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039548.1.\n",
      "2024-02-25 13:20:47,907 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP039548.1, seqID CP039548.1.\n",
      "2024-02-25 13:20:47,908 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039296.1.\n",
      "2024-02-25 13:20:47,908 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP039296.1.\n",
      "2024-02-25 13:20:47,908 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP039296.1, seqID CP039296.1.\n",
      "2024-02-25 13:20:47,925 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP028285.1, seqID CP028285.1.\n",
      "2024-02-25 13:20:47,939 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP017623.1, seqID AP017623.1.\n",
      "2024-02-25 13:20:47,954 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP015883.1, seqID CP015883.1.\n",
      "2024-02-25 13:20:47,971 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962862.1, seqID NZ_LR962862.1.\n",
      "2024-02-25 13:20:47,987 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962859.1, seqID NZ_LR962859.1.\n",
      "2024-02-25 13:20:48,004 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962821.1, seqID NZ_LR962821.1.\n",
      "2024-02-25 13:20:48,021 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962784.1, seqID NZ_LR962784.1.\n",
      "2024-02-25 13:20:48,037 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962782.1, seqID NZ_LR962782.1.\n",
      "2024-02-25 13:20:48,052 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962731.1, seqID NZ_LR962731.1.\n",
      "2024-02-25 13:20:48,069 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962702.1, seqID NZ_LR962702.1.\n",
      "2024-02-25 13:20:48,087 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962639.1, seqID NZ_LR962639.1.\n",
      "2024-02-25 13:20:48,104 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962647.1, seqID NZ_LR962647.1.\n",
      "2024-02-25 13:20:48,119 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962636.1, seqID NZ_LR962636.1.\n",
      "2024-02-25 13:20:48,135 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962615.1, seqID NZ_LR962615.1.\n",
      "2024-02-25 13:20:48,150 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962628.1, seqID NZ_LR962628.1.\n",
      "2024-02-25 13:20:48,166 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962611.1, seqID NZ_LR962611.1.\n",
      "2024-02-25 13:20:48,182 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962586.1, seqID NZ_LR962586.1.\n",
      "2024-02-25 13:20:48,200 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962584.1, seqID NZ_LR962584.1.\n",
      "2024-02-25 13:20:48,217 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962559.1, seqID NZ_LR962559.1.\n",
      "2024-02-25 13:20:48,233 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962561.1, seqID NZ_LR962561.1.\n",
      "2024-02-25 13:20:48,249 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962558.1, seqID NZ_LR962558.1.\n",
      "2024-02-25 13:20:48,265 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962569.1, seqID NZ_LR962569.1.\n",
      "2024-02-25 13:20:48,282 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962574.1, seqID NZ_LR962574.1.\n",
      "2024-02-25 13:20:48,298 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962565.1, seqID NZ_LR962565.1.\n",
      "2024-02-25 13:20:48,314 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962551.1, seqID NZ_LR962551.1.\n",
      "2024-02-25 13:20:48,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962542.1, seqID NZ_LR962542.1.\n",
      "2024-02-25 13:20:48,349 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962526.1, seqID NZ_LR962526.1.\n",
      "2024-02-25 13:20:48,367 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962502.1, seqID NZ_LR962502.1.\n",
      "2024-02-25 13:20:48,385 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962507.1, seqID NZ_LR962507.1.\n",
      "2024-02-25 13:20:48,401 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962523.1, seqID NZ_LR962523.1.\n",
      "2024-02-25 13:20:48,418 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962495.1, seqID NZ_LR962495.1.\n",
      "2024-02-25 13:20:48,436 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962464.1, seqID NZ_LR962464.1.\n",
      "2024-02-25 13:20:48,454 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962418.1, seqID NZ_LR962418.1.\n",
      "2024-02-25 13:20:48,472 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962423.1, seqID NZ_LR962423.1.\n",
      "2024-02-25 13:20:48,489 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962410.1, seqID NZ_LR962410.1.\n",
      "2024-02-25 13:20:48,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962421.1, seqID NZ_LR962421.1.\n",
      "2024-02-25 13:20:48,522 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962407.1, seqID NZ_LR962407.1.\n",
      "2024-02-25 13:20:48,540 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962386.1, seqID NZ_LR962386.1.\n",
      "2024-02-25 13:20:48,558 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962378.1, seqID NZ_LR962378.1.\n",
      "2024-02-25 13:20:48,577 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962367.1, seqID NZ_LR962367.1.\n",
      "2024-02-25 13:20:48,595 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962372.1, seqID NZ_LR962372.1.\n",
      "2024-02-25 13:20:48,613 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962356.1, seqID NZ_LR962356.1.\n",
      "2024-02-25 13:20:48,629 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962321.1, seqID NZ_LR962321.1.\n",
      "2024-02-25 13:20:48,645 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962331.1, seqID NZ_LR962331.1.\n",
      "2024-02-25 13:20:48,663 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962306.1, seqID NZ_LR962306.1.\n",
      "2024-02-25 13:20:48,681 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962302.1, seqID NZ_LR962302.1.\n",
      "2024-02-25 13:20:48,699 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962267.1, seqID NZ_LR962267.1.\n",
      "2024-02-25 13:20:48,717 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962271.1, seqID NZ_LR962271.1.\n",
      "2024-02-25 13:20:48,734 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962277.1, seqID NZ_LR962277.1.\n",
      "2024-02-25 13:20:48,752 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962248.1, seqID NZ_LR962248.1.\n",
      "2024-02-25 13:20:48,771 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962203.1, seqID NZ_LR962203.1.\n",
      "2024-02-25 13:20:48,787 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962196.1, seqID NZ_LR962196.1.\n",
      "2024-02-25 13:20:48,806 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962227.1, seqID NZ_LR962227.1.\n",
      "2024-02-25 13:20:48,825 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962174.1, seqID NZ_LR962174.1.\n",
      "2024-02-25 13:20:48,843 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962152.1, seqID NZ_LR962152.1.\n",
      "2024-02-25 13:20:48,860 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962155.1, seqID NZ_LR962155.1.\n",
      "2024-02-25 13:20:48,877 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962169.1, seqID NZ_LR962169.1.\n",
      "2024-02-25 13:20:48,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962157.1, seqID NZ_LR962157.1.\n",
      "2024-02-25 13:20:48,911 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962141.1, seqID NZ_LR962141.1.\n",
      "2024-02-25 13:20:48,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962165.1, seqID NZ_LR962165.1.\n",
      "2024-02-25 13:20:48,944 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962121.1, seqID NZ_LR962121.1.\n",
      "2024-02-25 13:20:48,962 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962129.1, seqID NZ_LR962129.1.\n",
      "2024-02-25 13:20:48,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962119.1, seqID NZ_LR962119.1.\n",
      "2024-02-25 13:20:48,999 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962056.1, seqID NZ_LR962056.1.\n",
      "2024-02-25 13:20:49,018 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962025.1, seqID NZ_LR962025.1.\n",
      "2024-02-25 13:20:49,035 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962004.1, seqID NZ_LR962004.1.\n",
      "2024-02-25 13:20:49,053 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962013.1, seqID NZ_LR962013.1.\n",
      "2024-02-25 13:20:49,070 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961991.1, seqID NZ_LR961991.1.\n",
      "2024-02-25 13:20:49,087 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961997.1, seqID NZ_LR961997.1.\n",
      "2024-02-25 13:20:49,106 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961935.1, seqID NZ_LR961935.1.\n",
      "2024-02-25 13:20:49,126 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP027297.1, seqID NZ_AP027297.1.\n",
      "2024-02-25 13:20:49,145 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124921.1, seqID NZ_CP124921.1.\n",
      "2024-02-25 13:20:49,165 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124908.1, seqID NZ_CP124908.1.\n",
      "2024-02-25 13:20:49,185 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124894.1, seqID NZ_CP124894.1.\n",
      "2024-02-25 13:20:49,204 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124884.1, seqID NZ_CP124884.1.\n",
      "2024-02-25 13:20:49,224 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP106757.1, seqID NZ_CP106757.1.\n",
      "2024-02-25 13:20:49,245 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP119528.1, seqID NZ_CP119528.1.\n",
      "2024-02-25 13:20:49,267 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP119159.1, seqID NZ_CP119159.1.\n",
      "2024-02-25 13:20:49,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP118755.1, seqID NZ_CP118755.1.\n",
      "2024-02-25 13:20:49,306 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP113831.1, seqID NZ_CP113831.1.\n",
      "2024-02-25 13:20:49,325 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP102070.1, seqID NZ_CP102070.1.\n",
      "2024-02-25 13:20:49,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097002.1, seqID NZ_CP097002.1.\n",
      "2024-02-25 13:20:49,365 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092576.1, seqID NZ_CP092576.1.\n",
      "2024-02-25 13:20:49,386 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092574.1, seqID NZ_CP092574.1.\n",
      "2024-02-25 13:20:49,408 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076498.1, seqID NZ_CP076498.1.\n",
      "2024-02-25 13:20:49,430 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076493.1, seqID NZ_CP076493.1.\n",
      "2024-02-25 13:20:49,452 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP076488.1, seqID NZ_CP076488.1.\n",
      "2024-02-25 13:20:49,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091901.1, seqID NZ_CP091901.1.\n",
      "2024-02-25 13:20:49,494 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091899.1, seqID NZ_CP091899.1.\n",
      "2024-02-25 13:20:49,514 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091892.1, seqID NZ_CP091892.1.\n",
      "2024-02-25 13:20:49,535 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091889.1, seqID NZ_CP091889.1.\n",
      "2024-02-25 13:20:49,558 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091198.1, seqID NZ_CP091198.1.\n",
      "2024-02-25 13:20:49,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086726.1, seqID NZ_CP086726.1.\n",
      "2024-02-25 13:20:49,598 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086592.1, seqID NZ_CP086592.1.\n",
      "2024-02-25 13:20:49,618 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086558.1, seqID NZ_CP086558.1.\n",
      "2024-02-25 13:20:49,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP082231.1, seqID NZ_CP082231.1.\n",
      "2024-02-25 13:20:49,659 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP075604.2, seqID NZ_CP075604.2.\n",
      "2024-02-25 13:20:49,677 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046113.1, seqID NZ_CP046113.1.\n",
      "2024-02-25 13:20:49,696 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP049775.1, seqID NZ_CP049775.1.\n",
      "2024-02-25 13:20:49,715 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041344.1, seqID NZ_CP041344.1.\n",
      "2024-02-25 13:20:49,735 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039549.1, seqID NZ_CP039549.1.\n",
      "2024-02-25 13:20:49,756 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039548.1, seqID NZ_CP039548.1.\n",
      "2024-02-25 13:20:49,777 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039296.1, seqID NZ_CP039296.1.\n",
      "2024-02-25 13:20:49,796 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028285.1, seqID NZ_CP028285.1.\n",
      "2024-02-25 13:20:49,814 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP017623.1, seqID NZ_AP017623.1.\n",
      "2024-02-25 13:20:49,833 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015883.1, seqID NZ_CP015883.1.\n",
      "2024-02-25 13:20:49,834 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962068.1.\n",
      "2024-02-25 13:20:49,834 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962068.1.\n",
      "2024-02-25 13:20:49,834 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962068.1, seqID LR962068.1.\n",
      "2024-02-25 13:20:49,851 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962068.1, seqID NZ_LR962068.1.\n",
      "2024-02-25 13:20:49,867 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP025690.1, seqID AP025690.1.\n",
      "2024-02-25 13:20:49,883 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain HF558530.1, seqID HF558530.1.\n",
      "2024-02-25 13:20:49,901 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_019770.1, seqID NC_019770.1.\n",
      "2024-02-25 13:20:49,917 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP002621.1, seqID CP002621.1.\n",
      "2024-02-25 13:20:49,935 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_017316.1, seqID NC_017316.1.\n",
      "2024-02-25 13:20:49,950 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP003726.1, seqID CP003726.1.\n",
      "2024-02-25 13:20:49,970 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_018221.1, seqID NC_018221.1.\n",
      "2024-02-25 13:20:49,986 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP008816.1, seqID CP008816.1.\n",
      "2024-02-25 13:20:50,005 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP008816.1, seqID NZ_CP008816.1.\n",
      "2024-02-25 13:20:50,022 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP002491.1, seqID CP002491.1.\n",
      "2024-02-25 13:20:50,042 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NC_017312.1, seqID NC_017312.1.\n",
      "2024-02-25 13:20:50,059 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940420.1, seqID OD940420.1.\n",
      "2024-02-25 13:20:50,060 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962836.1.\n",
      "2024-02-25 13:20:50,060 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962836.1.\n",
      "2024-02-25 13:20:50,060 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962836.1, seqID LR962836.1.\n",
      "2024-02-25 13:20:50,061 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962830.1.\n",
      "2024-02-25 13:20:50,061 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962830.1.\n",
      "2024-02-25 13:20:50,061 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962830.1, seqID LR962830.1.\n",
      "2024-02-25 13:20:50,062 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962834.1.\n",
      "2024-02-25 13:20:50,062 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962834.1.\n",
      "2024-02-25 13:20:50,062 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962834.1, seqID LR962834.1.\n",
      "2024-02-25 13:20:50,063 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962818.1.\n",
      "2024-02-25 13:20:50,063 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962818.1.\n",
      "2024-02-25 13:20:50,063 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962818.1, seqID LR962818.1.\n",
      "2024-02-25 13:20:50,064 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962777.1.\n",
      "2024-02-25 13:20:50,065 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962777.1.\n",
      "2024-02-25 13:20:50,065 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962777.1, seqID LR962777.1.\n",
      "2024-02-25 13:20:50,066 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962792.1.\n",
      "2024-02-25 13:20:50,066 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962792.1.\n",
      "2024-02-25 13:20:50,066 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962792.1, seqID LR962792.1.\n",
      "2024-02-25 13:20:50,067 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962811.1.\n",
      "2024-02-25 13:20:50,067 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962811.1.\n",
      "2024-02-25 13:20:50,067 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962811.1, seqID LR962811.1.\n",
      "2024-02-25 13:20:50,068 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962802.1.\n",
      "2024-02-25 13:20:50,068 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962802.1.\n",
      "2024-02-25 13:20:50,068 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962802.1, seqID LR962802.1.\n",
      "2024-02-25 13:20:50,069 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962732.1.\n",
      "2024-02-25 13:20:50,069 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962732.1.\n",
      "2024-02-25 13:20:50,069 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962732.1, seqID LR962732.1.\n",
      "2024-02-25 13:20:50,070 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962720.1.\n",
      "2024-02-25 13:20:50,070 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962720.1.\n",
      "2024-02-25 13:20:50,070 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962720.1, seqID LR962720.1.\n",
      "2024-02-25 13:20:50,071 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962721.1.\n",
      "2024-02-25 13:20:50,071 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962721.1.\n",
      "2024-02-25 13:20:50,072 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962721.1, seqID LR962721.1.\n",
      "2024-02-25 13:20:50,073 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962717.1.\n",
      "2024-02-25 13:20:50,073 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962717.1.\n",
      "2024-02-25 13:20:50,073 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962717.1, seqID LR962717.1.\n",
      "2024-02-25 13:20:50,074 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962698.1.\n",
      "2024-02-25 13:20:50,074 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962698.1.\n",
      "2024-02-25 13:20:50,074 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962698.1, seqID LR962698.1.\n",
      "2024-02-25 13:20:50,075 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962694.1.\n",
      "2024-02-25 13:20:50,075 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962694.1.\n",
      "2024-02-25 13:20:50,075 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962694.1, seqID LR962694.1.\n",
      "2024-02-25 13:20:50,076 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962695.1.\n",
      "2024-02-25 13:20:50,076 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962695.1.\n",
      "2024-02-25 13:20:50,076 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962695.1, seqID LR962695.1.\n",
      "2024-02-25 13:20:50,077 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962654.1.\n",
      "2024-02-25 13:20:50,078 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962654.1.\n",
      "2024-02-25 13:20:50,078 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962654.1, seqID LR962654.1.\n",
      "2024-02-25 13:20:50,079 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962656.1.\n",
      "2024-02-25 13:20:50,079 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962656.1.\n",
      "2024-02-25 13:20:50,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962656.1, seqID LR962656.1.\n",
      "2024-02-25 13:20:50,080 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962663.1.\n",
      "2024-02-25 13:20:50,080 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962663.1.\n",
      "2024-02-25 13:20:50,080 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962663.1, seqID LR962663.1.\n",
      "2024-02-25 13:20:50,081 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962670.1.\n",
      "2024-02-25 13:20:50,081 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962670.1.\n",
      "2024-02-25 13:20:50,081 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962670.1, seqID LR962670.1.\n",
      "2024-02-25 13:20:50,082 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962649.1.\n",
      "2024-02-25 13:20:50,082 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962649.1.\n",
      "2024-02-25 13:20:50,082 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962649.1, seqID LR962649.1.\n",
      "2024-02-25 13:20:50,084 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962620.1.\n",
      "2024-02-25 13:20:50,084 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962620.1.\n",
      "2024-02-25 13:20:50,084 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962620.1, seqID LR962620.1.\n",
      "2024-02-25 13:20:50,085 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962608.1.\n",
      "2024-02-25 13:20:50,085 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962608.1.\n",
      "2024-02-25 13:20:50,085 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962608.1, seqID LR962608.1.\n",
      "2024-02-25 13:20:50,086 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962589.1.\n",
      "2024-02-25 13:20:50,086 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962589.1.\n",
      "2024-02-25 13:20:50,086 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962589.1, seqID LR962589.1.\n",
      "2024-02-25 13:20:50,087 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962580.1.\n",
      "2024-02-25 13:20:50,087 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962580.1.\n",
      "2024-02-25 13:20:50,087 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962580.1, seqID LR962580.1.\n",
      "2024-02-25 13:20:50,088 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962572.1.\n",
      "2024-02-25 13:20:50,088 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962572.1.\n",
      "2024-02-25 13:20:50,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962572.1, seqID LR962572.1.\n",
      "2024-02-25 13:20:50,090 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962579.1.\n",
      "2024-02-25 13:20:50,090 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962579.1.\n",
      "2024-02-25 13:20:50,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962579.1, seqID LR962579.1.\n",
      "2024-02-25 13:20:50,091 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962554.1.\n",
      "2024-02-25 13:20:50,091 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962554.1.\n",
      "2024-02-25 13:20:50,091 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962554.1, seqID LR962554.1.\n",
      "2024-02-25 13:20:50,092 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962512.1.\n",
      "2024-02-25 13:20:50,092 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962512.1.\n",
      "2024-02-25 13:20:50,092 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962512.1, seqID LR962512.1.\n",
      "2024-02-25 13:20:50,093 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962466.1.\n",
      "2024-02-25 13:20:50,093 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962466.1.\n",
      "2024-02-25 13:20:50,093 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962466.1, seqID LR962466.1.\n",
      "2024-02-25 13:20:50,094 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962486.1.\n",
      "2024-02-25 13:20:50,094 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962486.1.\n",
      "2024-02-25 13:20:50,094 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962486.1, seqID LR962486.1.\n",
      "2024-02-25 13:20:50,095 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962461.1.\n",
      "2024-02-25 13:20:50,095 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962461.1.\n",
      "2024-02-25 13:20:50,095 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962461.1, seqID LR962461.1.\n",
      "2024-02-25 13:20:50,097 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962437.1.\n",
      "2024-02-25 13:20:50,097 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962437.1.\n",
      "2024-02-25 13:20:50,097 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962437.1, seqID LR962437.1.\n",
      "2024-02-25 13:20:50,098 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962414.1.\n",
      "2024-02-25 13:20:50,098 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962414.1.\n",
      "2024-02-25 13:20:50,098 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962414.1, seqID LR962414.1.\n",
      "2024-02-25 13:20:50,099 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962415.1.\n",
      "2024-02-25 13:20:50,099 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962415.1.\n",
      "2024-02-25 13:20:50,099 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962415.1, seqID LR962415.1.\n",
      "2024-02-25 13:20:50,100 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962382.1.\n",
      "2024-02-25 13:20:50,100 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962382.1.\n",
      "2024-02-25 13:20:50,100 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962382.1, seqID LR962382.1.\n",
      "2024-02-25 13:20:50,101 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962400.1.\n",
      "2024-02-25 13:20:50,101 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962400.1.\n",
      "2024-02-25 13:20:50,102 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962400.1, seqID LR962400.1.\n",
      "2024-02-25 13:20:50,103 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962390.1.\n",
      "2024-02-25 13:20:50,103 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962390.1.\n",
      "2024-02-25 13:20:50,103 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962390.1, seqID LR962390.1.\n",
      "2024-02-25 13:20:50,104 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962370.1.\n",
      "2024-02-25 13:20:50,104 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962370.1.\n",
      "2024-02-25 13:20:50,104 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962370.1, seqID LR962370.1.\n",
      "2024-02-25 13:20:50,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962351.1.\n",
      "2024-02-25 13:20:50,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962351.1.\n",
      "2024-02-25 13:20:50,105 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962351.1, seqID LR962351.1.\n",
      "2024-02-25 13:20:50,107 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962319.1.\n",
      "2024-02-25 13:20:50,107 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962319.1.\n",
      "2024-02-25 13:20:50,107 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962319.1, seqID LR962319.1.\n",
      "2024-02-25 13:20:50,108 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962334.1.\n",
      "2024-02-25 13:20:50,108 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962334.1.\n",
      "2024-02-25 13:20:50,108 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962334.1, seqID LR962334.1.\n",
      "2024-02-25 13:20:50,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962340.1.\n",
      "2024-02-25 13:20:50,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962340.1.\n",
      "2024-02-25 13:20:50,109 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962340.1, seqID LR962340.1.\n",
      "2024-02-25 13:20:50,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962326.1.\n",
      "2024-02-25 13:20:50,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962326.1.\n",
      "2024-02-25 13:20:50,110 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962326.1, seqID LR962326.1.\n",
      "2024-02-25 13:20:50,111 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962311.1.\n",
      "2024-02-25 13:20:50,111 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962311.1.\n",
      "2024-02-25 13:20:50,111 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962311.1, seqID LR962311.1.\n",
      "2024-02-25 13:20:50,112 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962323.1.\n",
      "2024-02-25 13:20:50,112 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962323.1.\n",
      "2024-02-25 13:20:50,112 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962323.1, seqID LR962323.1.\n",
      "2024-02-25 13:20:50,114 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962343.1.\n",
      "2024-02-25 13:20:50,114 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962343.1.\n",
      "2024-02-25 13:20:50,114 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962343.1, seqID LR962343.1.\n",
      "2024-02-25 13:20:50,115 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962293.1.\n",
      "2024-02-25 13:20:50,115 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962293.1.\n",
      "2024-02-25 13:20:50,115 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962293.1, seqID LR962293.1.\n",
      "2024-02-25 13:20:50,116 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962296.1.\n",
      "2024-02-25 13:20:50,116 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962296.1.\n",
      "2024-02-25 13:20:50,116 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962296.1, seqID LR962296.1.\n",
      "2024-02-25 13:20:50,117 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962300.1.\n",
      "2024-02-25 13:20:50,117 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962300.1.\n",
      "2024-02-25 13:20:50,117 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962300.1, seqID LR962300.1.\n",
      "2024-02-25 13:20:50,118 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962283.1.\n",
      "2024-02-25 13:20:50,118 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962283.1.\n",
      "2024-02-25 13:20:50,118 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962283.1, seqID LR962283.1.\n",
      "2024-02-25 13:20:50,120 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962256.1.\n",
      "2024-02-25 13:20:50,120 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962256.1.\n",
      "2024-02-25 13:20:50,120 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962256.1, seqID LR962256.1.\n",
      "2024-02-25 13:20:50,121 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962242.1.\n",
      "2024-02-25 13:20:50,121 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962242.1.\n",
      "2024-02-25 13:20:50,121 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962242.1, seqID LR962242.1.\n",
      "2024-02-25 13:20:50,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962236.1.\n",
      "2024-02-25 13:20:50,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962236.1.\n",
      "2024-02-25 13:20:50,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962236.1, seqID LR962236.1.\n",
      "2024-02-25 13:20:50,123 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962200.1.\n",
      "2024-02-25 13:20:50,123 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962200.1.\n",
      "2024-02-25 13:20:50,123 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962200.1, seqID LR962200.1.\n",
      "2024-02-25 13:20:50,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962190.1.\n",
      "2024-02-25 13:20:50,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962190.1.\n",
      "2024-02-25 13:20:50,124 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962190.1, seqID LR962190.1.\n",
      "2024-02-25 13:20:50,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962217.1.\n",
      "2024-02-25 13:20:50,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962217.1.\n",
      "2024-02-25 13:20:50,126 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962217.1, seqID LR962217.1.\n",
      "2024-02-25 13:20:50,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962194.1.\n",
      "2024-02-25 13:20:50,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962194.1.\n",
      "2024-02-25 13:20:50,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962194.1, seqID LR962194.1.\n",
      "2024-02-25 13:20:50,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962180.1.\n",
      "2024-02-25 13:20:50,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962180.1.\n",
      "2024-02-25 13:20:50,128 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962180.1, seqID LR962180.1.\n",
      "2024-02-25 13:20:50,129 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962182.1.\n",
      "2024-02-25 13:20:50,129 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962182.1.\n",
      "2024-02-25 13:20:50,129 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962182.1, seqID LR962182.1.\n",
      "2024-02-25 13:20:50,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962123.1.\n",
      "2024-02-25 13:20:50,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962123.1.\n",
      "2024-02-25 13:20:50,131 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962123.1, seqID LR962123.1.\n",
      "2024-02-25 13:20:50,132 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962097.1.\n",
      "2024-02-25 13:20:50,132 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962097.1.\n",
      "2024-02-25 13:20:50,132 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962097.1, seqID LR962097.1.\n",
      "2024-02-25 13:20:50,133 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962089.1.\n",
      "2024-02-25 13:20:50,133 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962089.1.\n",
      "2024-02-25 13:20:50,133 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962089.1, seqID LR962089.1.\n",
      "2024-02-25 13:20:50,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962101.1.\n",
      "2024-02-25 13:20:50,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962101.1.\n",
      "2024-02-25 13:20:50,134 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962101.1, seqID LR962101.1.\n",
      "2024-02-25 13:20:50,135 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962093.1.\n",
      "2024-02-25 13:20:50,136 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962093.1.\n",
      "2024-02-25 13:20:50,136 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962093.1, seqID LR962093.1.\n",
      "2024-02-25 13:20:50,137 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962110.1.\n",
      "2024-02-25 13:20:50,137 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962110.1.\n",
      "2024-02-25 13:20:50,137 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962110.1, seqID LR962110.1.\n",
      "2024-02-25 13:20:50,138 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962083.1.\n",
      "2024-02-25 13:20:50,138 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962083.1.\n",
      "2024-02-25 13:20:50,138 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962083.1, seqID LR962083.1.\n",
      "2024-02-25 13:20:50,139 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962043.1.\n",
      "2024-02-25 13:20:50,139 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962043.1.\n",
      "2024-02-25 13:20:50,139 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962043.1, seqID LR962043.1.\n",
      "2024-02-25 13:20:50,140 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962049.1.\n",
      "2024-02-25 13:20:50,140 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962049.1.\n",
      "2024-02-25 13:20:50,140 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962049.1, seqID LR962049.1.\n",
      "2024-02-25 13:20:50,142 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962072.1.\n",
      "2024-02-25 13:20:50,142 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962072.1.\n",
      "2024-02-25 13:20:50,142 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962072.1, seqID LR962072.1.\n",
      "2024-02-25 13:20:50,143 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962063.1.\n",
      "2024-02-25 13:20:50,143 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962063.1.\n",
      "2024-02-25 13:20:50,143 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962063.1, seqID LR962063.1.\n",
      "2024-02-25 13:20:50,144 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962051.1.\n",
      "2024-02-25 13:20:50,144 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962051.1.\n",
      "2024-02-25 13:20:50,144 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962051.1, seqID LR962051.1.\n",
      "2024-02-25 13:20:50,145 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962023.1.\n",
      "2024-02-25 13:20:50,145 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962023.1.\n",
      "2024-02-25 13:20:50,145 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962023.1, seqID LR962023.1.\n",
      "2024-02-25 13:20:50,147 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962019.1.\n",
      "2024-02-25 13:20:50,147 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962019.1.\n",
      "2024-02-25 13:20:50,147 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962019.1, seqID LR962019.1.\n",
      "2024-02-25 13:20:50,148 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962021.1.\n",
      "2024-02-25 13:20:50,148 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962021.1.\n",
      "2024-02-25 13:20:50,148 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962021.1, seqID LR962021.1.\n",
      "2024-02-25 13:20:50,149 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961989.1.\n",
      "2024-02-25 13:20:50,149 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961989.1.\n",
      "2024-02-25 13:20:50,149 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961989.1, seqID LR961989.1.\n",
      "2024-02-25 13:20:50,150 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961999.1.\n",
      "2024-02-25 13:20:50,150 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961999.1.\n",
      "2024-02-25 13:20:50,150 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961999.1, seqID LR961999.1.\n",
      "2024-02-25 13:20:50,151 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961978.1.\n",
      "2024-02-25 13:20:50,151 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961978.1.\n",
      "2024-02-25 13:20:50,151 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961978.1, seqID LR961978.1.\n",
      "2024-02-25 13:20:50,153 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961965.1.\n",
      "2024-02-25 13:20:50,153 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961965.1.\n",
      "2024-02-25 13:20:50,153 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961965.1, seqID LR961965.1.\n",
      "2024-02-25 13:20:50,154 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961945.1.\n",
      "2024-02-25 13:20:50,154 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961945.1.\n",
      "2024-02-25 13:20:50,154 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961945.1, seqID LR961945.1.\n",
      "2024-02-25 13:20:50,155 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961925.1.\n",
      "2024-02-25 13:20:50,155 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961925.1.\n",
      "2024-02-25 13:20:50,155 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961925.1, seqID LR961925.1.\n",
      "2024-02-25 13:20:50,156 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961938.1.\n",
      "2024-02-25 13:20:50,156 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961938.1.\n",
      "2024-02-25 13:20:50,156 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961938.1, seqID LR961938.1.\n",
      "2024-02-25 13:20:50,157 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961975.1.\n",
      "2024-02-25 13:20:50,158 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961975.1.\n",
      "2024-02-25 13:20:50,158 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961975.1, seqID LR961975.1.\n",
      "2024-02-25 13:20:50,159 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961966.1.\n",
      "2024-02-25 13:20:50,159 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961966.1.\n",
      "2024-02-25 13:20:50,159 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961966.1, seqID LR961966.1.\n",
      "2024-02-25 13:20:50,160 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961933.1.\n",
      "2024-02-25 13:20:50,160 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961933.1.\n",
      "2024-02-25 13:20:50,160 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961933.1, seqID LR961933.1.\n",
      "2024-02-25 13:20:50,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961970.1.\n",
      "2024-02-25 13:20:50,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961970.1.\n",
      "2024-02-25 13:20:50,161 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961970.1, seqID LR961970.1.\n",
      "2024-02-25 13:20:50,162 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961961.1.\n",
      "2024-02-25 13:20:50,162 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961961.1.\n",
      "2024-02-25 13:20:50,162 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961961.1, seqID LR961961.1.\n",
      "2024-02-25 13:20:50,163 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR698844.1.\n",
      "2024-02-25 13:20:50,163 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR698844.1.\n",
      "2024-02-25 13:20:50,163 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR698844.1, seqID LR698844.1.\n",
      "2024-02-25 13:20:50,180 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP102121.1, seqID CP102121.1.\n",
      "2024-02-25 13:20:50,197 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP102120.1, seqID CP102120.1.\n",
      "2024-02-25 13:20:50,198 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP133450.1.\n",
      "2024-02-25 13:20:50,198 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP133450.1.\n",
      "2024-02-25 13:20:50,198 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP133450.1, seqID CP133450.1.\n",
      "2024-02-25 13:20:50,216 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124931.1, seqID CP124931.1.\n",
      "2024-02-25 13:20:50,233 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124953.1, seqID CP124953.1.\n",
      "2024-02-25 13:20:50,251 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124947.1, seqID CP124947.1.\n",
      "2024-02-25 13:20:50,269 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124928.1, seqID CP124928.1.\n",
      "2024-02-25 13:20:50,288 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP128464.1, seqID CP128464.1.\n",
      "2024-02-25 13:20:50,304 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP027302.1, seqID AP027302.1.\n",
      "2024-02-25 13:20:50,319 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP027136.1, seqID AP027136.1.\n",
      "2024-02-25 13:20:50,337 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124923.1, seqID CP124923.1.\n",
      "2024-02-25 13:20:50,356 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124916.1, seqID CP124916.1.\n",
      "2024-02-25 13:20:50,374 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124911.1, seqID CP124911.1.\n",
      "2024-02-25 13:20:50,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124901.1, seqID CP124901.1.\n",
      "2024-02-25 13:20:50,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124882.1, seqID CP124882.1.\n",
      "2024-02-25 13:20:50,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP118962.1.\n",
      "2024-02-25 13:20:50,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP118962.1.\n",
      "2024-02-25 13:20:50,412 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118962.1, seqID CP118962.1.\n",
      "2024-02-25 13:20:50,413 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP117970.1.\n",
      "2024-02-25 13:20:50,413 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP117970.1.\n",
      "2024-02-25 13:20:50,413 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP117970.1, seqID CP117970.1.\n",
      "2024-02-25 13:20:50,431 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116962.1, seqID CP116962.1.\n",
      "2024-02-25 13:20:50,449 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116571.1, seqID CP116571.1.\n",
      "2024-02-25 13:20:50,467 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110074.1, seqID CP110074.1.\n",
      "2024-02-25 13:20:50,485 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110069.1, seqID CP110069.1.\n",
      "2024-02-25 13:20:50,503 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110063.1, seqID CP110063.1.\n",
      "2024-02-25 13:20:50,520 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110050.1, seqID CP110050.1.\n",
      "2024-02-25 13:20:50,538 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110047.1, seqID CP110047.1.\n",
      "2024-02-25 13:20:50,556 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110041.1, seqID CP110041.1.\n",
      "2024-02-25 13:20:50,574 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110032.1, seqID CP110032.1.\n",
      "2024-02-25 13:20:50,575 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP102065.1.\n",
      "2024-02-25 13:20:50,575 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP102065.1.\n",
      "2024-02-25 13:20:50,575 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP102065.1, seqID CP102065.1.\n",
      "2024-02-25 13:20:50,592 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP100596.1, seqID CP100596.1.\n",
      "2024-02-25 13:20:50,608 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP053181.1, seqID CP053181.1.\n",
      "2024-02-25 13:20:50,625 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097048.1, seqID CP097048.1.\n",
      "2024-02-25 13:20:50,642 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097046.1, seqID CP097046.1.\n",
      "2024-02-25 13:20:50,659 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097020.1, seqID CP097020.1.\n",
      "2024-02-25 13:20:50,675 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097069.1, seqID CP097069.1.\n",
      "2024-02-25 13:20:50,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097056.1, seqID CP097056.1.\n",
      "2024-02-25 13:20:50,710 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097010.1, seqID CP097010.1.\n",
      "2024-02-25 13:20:50,727 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097066.1, seqID CP097066.1.\n",
      "2024-02-25 13:20:50,743 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP064374.1, seqID CP064374.1.\n",
      "2024-02-25 13:20:50,759 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091906.1, seqID CP091906.1.\n",
      "2024-02-25 13:20:50,760 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP090558.1.\n",
      "2024-02-25 13:20:50,760 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP090558.1.\n",
      "2024-02-25 13:20:50,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP090558.1, seqID CP090558.1.\n",
      "2024-02-25 13:20:50,775 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086564.1, seqID CP086564.1.\n",
      "2024-02-25 13:20:50,791 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086566.1, seqID CP086566.1.\n",
      "2024-02-25 13:20:50,808 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086411.1, seqID CP086411.1.\n",
      "2024-02-25 13:20:50,824 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP085294.1, seqID CP085294.1.\n",
      "2024-02-25 13:20:50,839 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP078162.1, seqID CP078162.1.\n",
      "2024-02-25 13:20:50,840 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP046022.1.\n",
      "2024-02-25 13:20:50,840 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP046022.1.\n",
      "2024-02-25 13:20:50,840 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046022.1, seqID CP046022.1.\n",
      "2024-02-25 13:20:50,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046112.1, seqID CP046112.1.\n",
      "2024-02-25 13:20:50,874 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046111.1, seqID CP046111.1.\n",
      "2024-02-25 13:20:50,891 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046108.1, seqID CP046108.1.\n",
      "2024-02-25 13:20:50,908 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP041012.1, seqID CP041012.1.\n",
      "2024-02-25 13:20:50,924 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP030042.1, seqID CP030042.1.\n",
      "2024-02-25 13:20:50,940 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP014949.1, seqID CP014949.1.\n",
      "2024-02-25 13:20:50,960 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940420.1, seqID NZ_OD940420.1.\n",
      "2024-02-25 13:20:50,976 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962836.1, seqID NZ_LR962836.1.\n",
      "2024-02-25 13:20:50,991 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962830.1, seqID NZ_LR962830.1.\n",
      "2024-02-25 13:20:51,007 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962834.1, seqID NZ_LR962834.1.\n",
      "2024-02-25 13:20:51,023 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962818.1, seqID NZ_LR962818.1.\n",
      "2024-02-25 13:20:51,040 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962777.1, seqID NZ_LR962777.1.\n",
      "2024-02-25 13:20:51,057 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962792.1, seqID NZ_LR962792.1.\n",
      "2024-02-25 13:20:51,073 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962811.1, seqID NZ_LR962811.1.\n",
      "2024-02-25 13:20:51,090 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962802.1, seqID NZ_LR962802.1.\n",
      "2024-02-25 13:20:51,107 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962732.1, seqID NZ_LR962732.1.\n",
      "2024-02-25 13:20:51,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962720.1, seqID NZ_LR962720.1.\n",
      "2024-02-25 13:20:51,139 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962721.1, seqID NZ_LR962721.1.\n",
      "2024-02-25 13:20:51,155 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962717.1, seqID NZ_LR962717.1.\n",
      "2024-02-25 13:20:51,172 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962698.1, seqID NZ_LR962698.1.\n",
      "2024-02-25 13:20:51,188 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962694.1, seqID NZ_LR962694.1.\n",
      "2024-02-25 13:20:51,205 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962695.1, seqID NZ_LR962695.1.\n",
      "2024-02-25 13:20:51,222 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962654.1, seqID NZ_LR962654.1.\n",
      "2024-02-25 13:20:51,238 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962656.1, seqID NZ_LR962656.1.\n",
      "2024-02-25 13:20:51,255 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962663.1, seqID NZ_LR962663.1.\n",
      "2024-02-25 13:20:51,271 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962670.1, seqID NZ_LR962670.1.\n",
      "2024-02-25 13:20:51,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962649.1, seqID NZ_LR962649.1.\n",
      "2024-02-25 13:20:51,302 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962620.1, seqID NZ_LR962620.1.\n",
      "2024-02-25 13:20:51,319 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962608.1, seqID NZ_LR962608.1.\n",
      "2024-02-25 13:20:51,334 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962589.1, seqID NZ_LR962589.1.\n",
      "2024-02-25 13:20:51,349 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962580.1, seqID NZ_LR962580.1.\n",
      "2024-02-25 13:20:51,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962572.1, seqID NZ_LR962572.1.\n",
      "2024-02-25 13:20:51,381 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962579.1, seqID NZ_LR962579.1.\n",
      "2024-02-25 13:20:51,396 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962554.1, seqID NZ_LR962554.1.\n",
      "2024-02-25 13:20:51,414 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962512.1, seqID NZ_LR962512.1.\n",
      "2024-02-25 13:20:51,430 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962466.1, seqID NZ_LR962466.1.\n",
      "2024-02-25 13:20:51,447 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962486.1, seqID NZ_LR962486.1.\n",
      "2024-02-25 13:20:51,462 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962461.1, seqID NZ_LR962461.1.\n",
      "2024-02-25 13:20:51,479 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962437.1, seqID NZ_LR962437.1.\n",
      "2024-02-25 13:20:51,494 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962414.1, seqID NZ_LR962414.1.\n",
      "2024-02-25 13:20:51,510 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962415.1, seqID NZ_LR962415.1.\n",
      "2024-02-25 13:20:51,526 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962382.1, seqID NZ_LR962382.1.\n",
      "2024-02-25 13:20:51,543 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962400.1, seqID NZ_LR962400.1.\n",
      "2024-02-25 13:20:51,559 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962390.1, seqID NZ_LR962390.1.\n",
      "2024-02-25 13:20:51,575 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962370.1, seqID NZ_LR962370.1.\n",
      "2024-02-25 13:20:51,592 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962351.1, seqID NZ_LR962351.1.\n",
      "2024-02-25 13:20:51,607 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962319.1, seqID NZ_LR962319.1.\n",
      "2024-02-25 13:20:51,622 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962334.1, seqID NZ_LR962334.1.\n",
      "2024-02-25 13:20:51,639 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962340.1, seqID NZ_LR962340.1.\n",
      "2024-02-25 13:20:51,656 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962326.1, seqID NZ_LR962326.1.\n",
      "2024-02-25 13:20:51,671 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962311.1, seqID NZ_LR962311.1.\n",
      "2024-02-25 13:20:51,686 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962323.1, seqID NZ_LR962323.1.\n",
      "2024-02-25 13:20:51,702 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962343.1, seqID NZ_LR962343.1.\n",
      "2024-02-25 13:20:51,719 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962293.1, seqID NZ_LR962293.1.\n",
      "2024-02-25 13:20:51,736 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962296.1, seqID NZ_LR962296.1.\n",
      "2024-02-25 13:20:51,751 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962300.1, seqID NZ_LR962300.1.\n",
      "2024-02-25 13:20:51,767 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962283.1, seqID NZ_LR962283.1.\n",
      "2024-02-25 13:20:51,782 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962256.1, seqID NZ_LR962256.1.\n",
      "2024-02-25 13:20:51,799 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962242.1, seqID NZ_LR962242.1.\n",
      "2024-02-25 13:20:51,814 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962236.1, seqID NZ_LR962236.1.\n",
      "2024-02-25 13:20:51,829 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962200.1, seqID NZ_LR962200.1.\n",
      "2024-02-25 13:20:51,845 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962190.1, seqID NZ_LR962190.1.\n",
      "2024-02-25 13:20:51,863 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962217.1, seqID NZ_LR962217.1.\n",
      "2024-02-25 13:20:51,878 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962194.1, seqID NZ_LR962194.1.\n",
      "2024-02-25 13:20:51,894 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962180.1, seqID NZ_LR962180.1.\n",
      "2024-02-25 13:20:51,910 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962182.1, seqID NZ_LR962182.1.\n",
      "2024-02-25 13:20:51,927 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962123.1, seqID NZ_LR962123.1.\n",
      "2024-02-25 13:20:51,941 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962097.1, seqID NZ_LR962097.1.\n",
      "2024-02-25 13:20:51,958 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962089.1, seqID NZ_LR962089.1.\n",
      "2024-02-25 13:20:51,974 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962101.1, seqID NZ_LR962101.1.\n",
      "2024-02-25 13:20:51,990 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962093.1, seqID NZ_LR962093.1.\n",
      "2024-02-25 13:20:52,007 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962110.1, seqID NZ_LR962110.1.\n",
      "2024-02-25 13:20:52,023 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962083.1, seqID NZ_LR962083.1.\n",
      "2024-02-25 13:20:52,039 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962043.1, seqID NZ_LR962043.1.\n",
      "2024-02-25 13:20:52,056 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962049.1, seqID NZ_LR962049.1.\n",
      "2024-02-25 13:20:52,073 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962072.1, seqID NZ_LR962072.1.\n",
      "2024-02-25 13:20:52,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962063.1, seqID NZ_LR962063.1.\n",
      "2024-02-25 13:20:52,103 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962051.1, seqID NZ_LR962051.1.\n",
      "2024-02-25 13:20:52,119 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962023.1, seqID NZ_LR962023.1.\n",
      "2024-02-25 13:20:52,136 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962019.1, seqID NZ_LR962019.1.\n",
      "2024-02-25 13:20:52,152 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962021.1, seqID NZ_LR962021.1.\n",
      "2024-02-25 13:20:52,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961989.1, seqID NZ_LR961989.1.\n",
      "2024-02-25 13:20:52,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961999.1, seqID NZ_LR961999.1.\n",
      "2024-02-25 13:20:52,202 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961978.1, seqID NZ_LR961978.1.\n",
      "2024-02-25 13:20:52,219 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961965.1, seqID NZ_LR961965.1.\n",
      "2024-02-25 13:20:52,234 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961945.1, seqID NZ_LR961945.1.\n",
      "2024-02-25 13:20:52,251 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961925.1, seqID NZ_LR961925.1.\n",
      "2024-02-25 13:20:52,267 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961938.1, seqID NZ_LR961938.1.\n",
      "2024-02-25 13:20:52,282 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961975.1, seqID NZ_LR961975.1.\n",
      "2024-02-25 13:20:52,298 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961966.1, seqID NZ_LR961966.1.\n",
      "2024-02-25 13:20:52,314 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961933.1, seqID NZ_LR961933.1.\n",
      "2024-02-25 13:20:52,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961970.1, seqID NZ_LR961970.1.\n",
      "2024-02-25 13:20:52,345 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961961.1, seqID NZ_LR961961.1.\n",
      "2024-02-25 13:20:52,363 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR698844.1, seqID NZ_LR698844.1.\n",
      "2024-02-25 13:20:52,382 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP102121.1, seqID NZ_CP102121.1.\n",
      "2024-02-25 13:20:52,402 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP102120.1, seqID NZ_CP102120.1.\n",
      "2024-02-25 13:20:52,420 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP133450.1, seqID NZ_CP133450.1.\n",
      "2024-02-25 13:20:52,440 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124931.1, seqID NZ_CP124931.1.\n",
      "2024-02-25 13:20:52,459 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124953.1, seqID NZ_CP124953.1.\n",
      "2024-02-25 13:20:52,478 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124947.1, seqID NZ_CP124947.1.\n",
      "2024-02-25 13:20:52,498 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124928.1, seqID NZ_CP124928.1.\n",
      "2024-02-25 13:20:52,517 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP128464.1, seqID NZ_CP128464.1.\n",
      "2024-02-25 13:20:52,537 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP027302.1, seqID NZ_AP027302.1.\n",
      "2024-02-25 13:20:52,555 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP027136.1, seqID NZ_AP027136.1.\n",
      "2024-02-25 13:20:52,574 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124923.1, seqID NZ_CP124923.1.\n",
      "2024-02-25 13:20:52,593 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124916.1, seqID NZ_CP124916.1.\n",
      "2024-02-25 13:20:52,612 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124911.1, seqID NZ_CP124911.1.\n",
      "2024-02-25 13:20:52,631 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124901.1, seqID NZ_CP124901.1.\n",
      "2024-02-25 13:20:52,650 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124882.1, seqID NZ_CP124882.1.\n",
      "2024-02-25 13:20:52,667 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP118962.1, seqID NZ_CP118962.1.\n",
      "2024-02-25 13:20:52,685 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP117970.1, seqID NZ_CP117970.1.\n",
      "2024-02-25 13:20:52,704 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116962.1, seqID NZ_CP116962.1.\n",
      "2024-02-25 13:20:52,723 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116571.1, seqID NZ_CP116571.1.\n",
      "2024-02-25 13:20:52,742 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110074.1, seqID NZ_CP110074.1.\n",
      "2024-02-25 13:20:52,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110069.1, seqID NZ_CP110069.1.\n",
      "2024-02-25 13:20:52,779 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110063.1, seqID NZ_CP110063.1.\n",
      "2024-02-25 13:20:52,797 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110050.1, seqID NZ_CP110050.1.\n",
      "2024-02-25 13:20:52,815 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110047.1, seqID NZ_CP110047.1.\n",
      "2024-02-25 13:20:52,834 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110041.1, seqID NZ_CP110041.1.\n",
      "2024-02-25 13:20:52,852 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP102065.1, seqID NZ_CP102065.1.\n",
      "2024-02-25 13:20:52,871 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP100596.1, seqID NZ_CP100596.1.\n",
      "2024-02-25 13:20:52,890 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP053181.1, seqID NZ_CP053181.1.\n",
      "2024-02-25 13:20:52,909 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097048.1, seqID NZ_CP097048.1.\n",
      "2024-02-25 13:20:52,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097046.1, seqID NZ_CP097046.1.\n",
      "2024-02-25 13:20:52,946 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097020.1, seqID NZ_CP097020.1.\n",
      "2024-02-25 13:20:52,965 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097069.1, seqID NZ_CP097069.1.\n",
      "2024-02-25 13:20:52,984 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097056.1, seqID NZ_CP097056.1.\n",
      "2024-02-25 13:20:53,003 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097010.1, seqID NZ_CP097010.1.\n",
      "2024-02-25 13:20:53,021 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097066.1, seqID NZ_CP097066.1.\n",
      "2024-02-25 13:20:53,041 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP064374.1, seqID NZ_CP064374.1.\n",
      "2024-02-25 13:20:53,058 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091906.1, seqID NZ_CP091906.1.\n",
      "2024-02-25 13:20:53,075 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP090558.1, seqID NZ_CP090558.1.\n",
      "2024-02-25 13:20:53,093 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086564.1, seqID NZ_CP086564.1.\n",
      "2024-02-25 13:20:53,110 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086566.1, seqID NZ_CP086566.1.\n",
      "2024-02-25 13:20:53,129 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086411.1, seqID NZ_CP086411.1.\n",
      "2024-02-25 13:20:53,147 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085294.1, seqID NZ_CP085294.1.\n",
      "2024-02-25 13:20:53,164 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP078162.1, seqID NZ_CP078162.1.\n",
      "2024-02-25 13:20:53,182 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046022.1, seqID NZ_CP046022.1.\n",
      "2024-02-25 13:20:53,201 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046112.1, seqID NZ_CP046112.1.\n",
      "2024-02-25 13:20:53,220 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046111.1, seqID NZ_CP046111.1.\n",
      "2024-02-25 13:20:53,239 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046108.1, seqID NZ_CP046108.1.\n",
      "2024-02-25 13:20:53,258 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP041012.1, seqID NZ_CP041012.1.\n",
      "2024-02-25 13:20:53,275 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030042.1, seqID NZ_CP030042.1.\n",
      "2024-02-25 13:20:53,293 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP014949.1, seqID NZ_CP014949.1.\n",
      "2024-02-25 13:20:53,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP092784.1.\n",
      "2024-02-25 13:20:53,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP092784.1.\n",
      "2024-02-25 13:20:53,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP092784.1, seqID CP092784.1.\n",
      "2024-02-25 13:20:53,311 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092784.1, seqID NZ_CP092784.1.\n",
      "2024-02-25 13:20:53,326 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940431.1, seqID OD940431.1.\n",
      "2024-02-25 13:20:53,328 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962842.1.\n",
      "2024-02-25 13:20:53,328 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962842.1.\n",
      "2024-02-25 13:20:53,328 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962842.1, seqID LR962842.1.\n",
      "2024-02-25 13:20:53,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962843.1.\n",
      "2024-02-25 13:20:53,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962843.1.\n",
      "2024-02-25 13:20:53,329 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962843.1, seqID LR962843.1.\n",
      "2024-02-25 13:20:53,330 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962850.1.\n",
      "2024-02-25 13:20:53,330 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962850.1.\n",
      "2024-02-25 13:20:53,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962850.1, seqID LR962850.1.\n",
      "2024-02-25 13:20:53,332 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962840.1.\n",
      "2024-02-25 13:20:53,332 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962840.1.\n",
      "2024-02-25 13:20:53,332 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962840.1, seqID LR962840.1.\n",
      "2024-02-25 13:20:53,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962857.1.\n",
      "2024-02-25 13:20:53,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962857.1.\n",
      "2024-02-25 13:20:53,333 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962857.1, seqID LR962857.1.\n",
      "2024-02-25 13:20:53,334 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962832.1.\n",
      "2024-02-25 13:20:53,334 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962832.1.\n",
      "2024-02-25 13:20:53,334 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962832.1, seqID LR962832.1.\n",
      "2024-02-25 13:20:53,335 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962814.1.\n",
      "2024-02-25 13:20:53,335 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962814.1.\n",
      "2024-02-25 13:20:53,335 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962814.1, seqID LR962814.1.\n",
      "2024-02-25 13:20:53,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962775.1.\n",
      "2024-02-25 13:20:53,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962775.1.\n",
      "2024-02-25 13:20:53,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962775.1, seqID LR962775.1.\n",
      "2024-02-25 13:20:53,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962788.1.\n",
      "2024-02-25 13:20:53,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962788.1.\n",
      "2024-02-25 13:20:53,338 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962788.1, seqID LR962788.1.\n",
      "2024-02-25 13:20:53,339 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962736.1.\n",
      "2024-02-25 13:20:53,339 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962736.1.\n",
      "2024-02-25 13:20:53,339 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962736.1, seqID LR962736.1.\n",
      "2024-02-25 13:20:53,340 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962729.1.\n",
      "2024-02-25 13:20:53,340 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962729.1.\n",
      "2024-02-25 13:20:53,340 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962729.1, seqID LR962729.1.\n",
      "2024-02-25 13:20:53,341 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962726.1.\n",
      "2024-02-25 13:20:53,341 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962726.1.\n",
      "2024-02-25 13:20:53,341 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962726.1, seqID LR962726.1.\n",
      "2024-02-25 13:20:53,343 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962711.1.\n",
      "2024-02-25 13:20:53,343 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962711.1.\n",
      "2024-02-25 13:20:53,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962711.1, seqID LR962711.1.\n",
      "2024-02-25 13:20:53,344 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962714.1.\n",
      "2024-02-25 13:20:53,344 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962714.1.\n",
      "2024-02-25 13:20:53,344 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962714.1, seqID LR962714.1.\n",
      "2024-02-25 13:20:53,345 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962658.1.\n",
      "2024-02-25 13:20:53,345 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962658.1.\n",
      "2024-02-25 13:20:53,345 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962658.1, seqID LR962658.1.\n",
      "2024-02-25 13:20:53,347 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962671.1.\n",
      "2024-02-25 13:20:53,347 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962671.1.\n",
      "2024-02-25 13:20:53,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962671.1, seqID LR962671.1.\n",
      "2024-02-25 13:20:53,348 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962632.1.\n",
      "2024-02-25 13:20:53,348 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962632.1.\n",
      "2024-02-25 13:20:53,348 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962632.1, seqID LR962632.1.\n",
      "2024-02-25 13:20:53,349 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962604.1.\n",
      "2024-02-25 13:20:53,349 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962604.1.\n",
      "2024-02-25 13:20:53,349 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962604.1, seqID LR962604.1.\n",
      "2024-02-25 13:20:53,351 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962606.1.\n",
      "2024-02-25 13:20:53,351 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962606.1.\n",
      "2024-02-25 13:20:53,351 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962606.1, seqID LR962606.1.\n",
      "2024-02-25 13:20:53,352 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962634.1.\n",
      "2024-02-25 13:20:53,352 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962634.1.\n",
      "2024-02-25 13:20:53,352 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962634.1, seqID LR962634.1.\n",
      "2024-02-25 13:20:53,353 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962618.1.\n",
      "2024-02-25 13:20:53,353 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962618.1.\n",
      "2024-02-25 13:20:53,353 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962618.1, seqID LR962618.1.\n",
      "2024-02-25 13:20:53,354 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962602.1.\n",
      "2024-02-25 13:20:53,354 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962602.1.\n",
      "2024-02-25 13:20:53,354 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962602.1, seqID LR962602.1.\n",
      "2024-02-25 13:20:53,356 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962598.1.\n",
      "2024-02-25 13:20:53,356 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962598.1.\n",
      "2024-02-25 13:20:53,356 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962598.1, seqID LR962598.1.\n",
      "2024-02-25 13:20:53,357 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962600.1.\n",
      "2024-02-25 13:20:53,357 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962600.1.\n",
      "2024-02-25 13:20:53,357 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962600.1, seqID LR962600.1.\n",
      "2024-02-25 13:20:53,358 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962591.1.\n",
      "2024-02-25 13:20:53,358 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962591.1.\n",
      "2024-02-25 13:20:53,358 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962591.1, seqID LR962591.1.\n",
      "2024-02-25 13:20:53,359 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962595.1.\n",
      "2024-02-25 13:20:53,359 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962595.1.\n",
      "2024-02-25 13:20:53,359 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962595.1, seqID LR962595.1.\n",
      "2024-02-25 13:20:53,361 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962576.1.\n",
      "2024-02-25 13:20:53,361 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962576.1.\n",
      "2024-02-25 13:20:53,361 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962576.1, seqID LR962576.1.\n",
      "2024-02-25 13:20:53,362 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962567.1.\n",
      "2024-02-25 13:20:53,362 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962567.1.\n",
      "2024-02-25 13:20:53,362 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962567.1, seqID LR962567.1.\n",
      "2024-02-25 13:20:53,363 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962546.1.\n",
      "2024-02-25 13:20:53,363 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962546.1.\n",
      "2024-02-25 13:20:53,363 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962546.1, seqID LR962546.1.\n",
      "2024-02-25 13:20:53,364 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962545.1.\n",
      "2024-02-25 13:20:53,364 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962545.1.\n",
      "2024-02-25 13:20:53,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962545.1, seqID LR962545.1.\n",
      "2024-02-25 13:20:53,366 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962531.1.\n",
      "2024-02-25 13:20:53,366 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962531.1.\n",
      "2024-02-25 13:20:53,366 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962531.1, seqID LR962531.1.\n",
      "2024-02-25 13:20:53,367 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962539.1.\n",
      "2024-02-25 13:20:53,367 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962539.1.\n",
      "2024-02-25 13:20:53,367 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962539.1, seqID LR962539.1.\n",
      "2024-02-25 13:20:53,368 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962497.1.\n",
      "2024-02-25 13:20:53,368 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962497.1.\n",
      "2024-02-25 13:20:53,368 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962497.1, seqID LR962497.1.\n",
      "2024-02-25 13:20:53,369 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962434.1.\n",
      "2024-02-25 13:20:53,369 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962434.1.\n",
      "2024-02-25 13:20:53,369 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962434.1, seqID LR962434.1.\n",
      "2024-02-25 13:20:53,370 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962454.1.\n",
      "2024-02-25 13:20:53,371 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962454.1.\n",
      "2024-02-25 13:20:53,371 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962454.1, seqID LR962454.1.\n",
      "2024-02-25 13:20:53,372 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962432.1.\n",
      "2024-02-25 13:20:53,372 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962432.1.\n",
      "2024-02-25 13:20:53,372 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962432.1, seqID LR962432.1.\n",
      "2024-02-25 13:20:53,373 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962450.1.\n",
      "2024-02-25 13:20:53,373 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962450.1.\n",
      "2024-02-25 13:20:53,373 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962450.1, seqID LR962450.1.\n",
      "2024-02-25 13:20:53,374 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962393.1.\n",
      "2024-02-25 13:20:53,374 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962393.1.\n",
      "2024-02-25 13:20:53,374 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962393.1, seqID LR962393.1.\n",
      "2024-02-25 13:20:53,376 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962375.1.\n",
      "2024-02-25 13:20:53,376 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962375.1.\n",
      "2024-02-25 13:20:53,376 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962375.1, seqID LR962375.1.\n",
      "2024-02-25 13:20:53,377 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962346.1.\n",
      "2024-02-25 13:20:53,377 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962346.1.\n",
      "2024-02-25 13:20:53,377 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962346.1, seqID LR962346.1.\n",
      "2024-02-25 13:20:53,378 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962348.1.\n",
      "2024-02-25 13:20:53,378 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962348.1.\n",
      "2024-02-25 13:20:53,378 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962348.1, seqID LR962348.1.\n",
      "2024-02-25 13:20:53,379 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962315.1.\n",
      "2024-02-25 13:20:53,380 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962315.1.\n",
      "2024-02-25 13:20:53,380 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962315.1, seqID LR962315.1.\n",
      "2024-02-25 13:20:53,381 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962328.1.\n",
      "2024-02-25 13:20:53,381 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962328.1.\n",
      "2024-02-25 13:20:53,381 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962328.1, seqID LR962328.1.\n",
      "2024-02-25 13:20:53,382 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962239.1.\n",
      "2024-02-25 13:20:53,382 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962239.1.\n",
      "2024-02-25 13:20:53,382 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962239.1, seqID LR962239.1.\n",
      "2024-02-25 13:20:53,383 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962234.1.\n",
      "2024-02-25 13:20:53,383 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962234.1.\n",
      "2024-02-25 13:20:53,383 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962234.1, seqID LR962234.1.\n",
      "2024-02-25 13:20:53,385 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962225.1.\n",
      "2024-02-25 13:20:53,385 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962225.1.\n",
      "2024-02-25 13:20:53,385 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962225.1, seqID LR962225.1.\n",
      "2024-02-25 13:20:53,386 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962210.1.\n",
      "2024-02-25 13:20:53,386 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962210.1.\n",
      "2024-02-25 13:20:53,386 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962210.1, seqID LR962210.1.\n",
      "2024-02-25 13:20:53,387 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962213.1.\n",
      "2024-02-25 13:20:53,387 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962213.1.\n",
      "2024-02-25 13:20:53,387 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962213.1, seqID LR962213.1.\n",
      "2024-02-25 13:20:53,389 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962189.1.\n",
      "2024-02-25 13:20:53,389 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962189.1.\n",
      "2024-02-25 13:20:53,389 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962189.1, seqID LR962189.1.\n",
      "2024-02-25 13:20:53,390 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962188.1.\n",
      "2024-02-25 13:20:53,390 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962188.1.\n",
      "2024-02-25 13:20:53,390 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962188.1, seqID LR962188.1.\n",
      "2024-02-25 13:20:53,391 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962197.1.\n",
      "2024-02-25 13:20:53,391 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962197.1.\n",
      "2024-02-25 13:20:53,391 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962197.1, seqID LR962197.1.\n",
      "2024-02-25 13:20:53,392 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962230.1.\n",
      "2024-02-25 13:20:53,392 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962230.1.\n",
      "2024-02-25 13:20:53,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962230.1, seqID LR962230.1.\n",
      "2024-02-25 13:20:53,394 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962171.1.\n",
      "2024-02-25 13:20:53,394 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962171.1.\n",
      "2024-02-25 13:20:53,394 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962171.1, seqID LR962171.1.\n",
      "2024-02-25 13:20:53,395 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962163.1.\n",
      "2024-02-25 13:20:53,395 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962163.1.\n",
      "2024-02-25 13:20:53,395 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962163.1, seqID LR962163.1.\n",
      "2024-02-25 13:20:53,396 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962161.1.\n",
      "2024-02-25 13:20:53,396 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962161.1.\n",
      "2024-02-25 13:20:53,396 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962161.1, seqID LR962161.1.\n",
      "2024-02-25 13:20:53,397 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962132.1.\n",
      "2024-02-25 13:20:53,397 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962132.1.\n",
      "2024-02-25 13:20:53,397 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962132.1, seqID LR962132.1.\n",
      "2024-02-25 13:20:53,399 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962137.1.\n",
      "2024-02-25 13:20:53,399 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962137.1.\n",
      "2024-02-25 13:20:53,399 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962137.1, seqID LR962137.1.\n",
      "2024-02-25 13:20:53,400 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962112.1.\n",
      "2024-02-25 13:20:53,400 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962112.1.\n",
      "2024-02-25 13:20:53,400 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962112.1, seqID LR962112.1.\n",
      "2024-02-25 13:20:53,401 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962103.1.\n",
      "2024-02-25 13:20:53,402 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962103.1.\n",
      "2024-02-25 13:20:53,402 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962103.1, seqID LR962103.1.\n",
      "2024-02-25 13:20:53,403 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962107.1.\n",
      "2024-02-25 13:20:53,403 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962107.1.\n",
      "2024-02-25 13:20:53,403 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962107.1, seqID LR962107.1.\n",
      "2024-02-25 13:20:53,404 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962075.1.\n",
      "2024-02-25 13:20:53,404 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962075.1.\n",
      "2024-02-25 13:20:53,404 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962075.1, seqID LR962075.1.\n",
      "2024-02-25 13:20:53,406 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962039.1.\n",
      "2024-02-25 13:20:53,406 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962039.1.\n",
      "2024-02-25 13:20:53,406 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962039.1, seqID LR962039.1.\n",
      "2024-02-25 13:20:53,407 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962034.1.\n",
      "2024-02-25 13:20:53,407 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962034.1.\n",
      "2024-02-25 13:20:53,407 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962034.1, seqID LR962034.1.\n",
      "2024-02-25 13:20:53,408 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962030.1.\n",
      "2024-02-25 13:20:53,408 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962030.1.\n",
      "2024-02-25 13:20:53,408 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962030.1, seqID LR962030.1.\n",
      "2024-02-25 13:20:53,409 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962017.1.\n",
      "2024-02-25 13:20:53,409 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962017.1.\n",
      "2024-02-25 13:20:53,409 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962017.1, seqID LR962017.1.\n",
      "2024-02-25 13:20:53,410 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962010.1.\n",
      "2024-02-25 13:20:53,411 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962010.1.\n",
      "2024-02-25 13:20:53,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962010.1, seqID LR962010.1.\n",
      "2024-02-25 13:20:53,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961996.1.\n",
      "2024-02-25 13:20:53,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961996.1.\n",
      "2024-02-25 13:20:53,412 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961996.1, seqID LR961996.1.\n",
      "2024-02-25 13:20:53,413 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961994.1.\n",
      "2024-02-25 13:20:53,413 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961994.1.\n",
      "2024-02-25 13:20:53,413 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961994.1, seqID LR961994.1.\n",
      "2024-02-25 13:20:53,414 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961986.1.\n",
      "2024-02-25 13:20:53,414 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961986.1.\n",
      "2024-02-25 13:20:53,414 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961986.1, seqID LR961986.1.\n",
      "2024-02-25 13:20:53,416 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962008.1.\n",
      "2024-02-25 13:20:53,416 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962008.1.\n",
      "2024-02-25 13:20:53,416 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962008.1, seqID LR962008.1.\n",
      "2024-02-25 13:20:53,418 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961981.1.\n",
      "2024-02-25 13:20:53,418 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961981.1.\n",
      "2024-02-25 13:20:53,418 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961981.1, seqID LR961981.1.\n",
      "2024-02-25 13:20:53,419 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961957.1.\n",
      "2024-02-25 13:20:53,419 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961957.1.\n",
      "2024-02-25 13:20:53,419 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961957.1, seqID LR961957.1.\n",
      "2024-02-25 13:20:53,421 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961953.1.\n",
      "2024-02-25 13:20:53,421 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961953.1.\n",
      "2024-02-25 13:20:53,421 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961953.1, seqID LR961953.1.\n",
      "2024-02-25 13:20:53,422 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961923.1.\n",
      "2024-02-25 13:20:53,422 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961923.1.\n",
      "2024-02-25 13:20:53,422 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961923.1, seqID LR961923.1.\n",
      "2024-02-25 13:20:53,423 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961959.1.\n",
      "2024-02-25 13:20:53,423 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961959.1.\n",
      "2024-02-25 13:20:53,423 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961959.1, seqID LR961959.1.\n",
      "2024-02-25 13:20:53,439 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124778.1, seqID CP124778.1.\n",
      "2024-02-25 13:20:53,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP127177.1, seqID CP127177.1.\n",
      "2024-02-25 13:20:53,473 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP109611.1, seqID CP109611.1.\n",
      "2024-02-25 13:20:53,489 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124914.1, seqID CP124914.1.\n",
      "2024-02-25 13:20:53,507 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124891.1, seqID CP124891.1.\n",
      "2024-02-25 13:20:53,524 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP118048.1, seqID CP118048.1.\n",
      "2024-02-25 13:20:53,541 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP117511.1, seqID CP117511.1.\n",
      "2024-02-25 13:20:53,559 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP117503.1, seqID CP117503.1.\n",
      "2024-02-25 13:20:53,577 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP121233.1, seqID CP121233.1.\n",
      "2024-02-25 13:20:53,595 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116566.1, seqID CP116566.1.\n",
      "2024-02-25 13:20:53,612 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116564.1, seqID CP116564.1.\n",
      "2024-02-25 13:20:53,630 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116561.1, seqID CP116561.1.\n",
      "2024-02-25 13:20:53,646 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116569.1, seqID CP116569.1.\n",
      "2024-02-25 13:20:53,668 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP026729.1, seqID AP026729.1.\n",
      "2024-02-25 13:20:53,691 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP026726.1, seqID AP026726.1.\n",
      "2024-02-25 13:20:53,709 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP026721.1, seqID AP026721.1.\n",
      "2024-02-25 13:20:53,727 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP115992.1, seqID CP115992.1.\n",
      "2024-02-25 13:20:53,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110038.1, seqID CP110038.1.\n",
      "2024-02-25 13:20:53,762 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110036.1, seqID CP110036.1.\n",
      "2024-02-25 13:20:53,779 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110020.1, seqID CP110020.1.\n",
      "2024-02-25 13:20:53,796 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110019.1, seqID CP110019.1.\n",
      "2024-02-25 13:20:53,814 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP098418.1, seqID CP098418.1.\n",
      "2024-02-25 13:20:53,833 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP098025.1, seqID CP098025.1.\n",
      "2024-02-25 13:20:53,847 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091237.1, seqID CP091237.1.\n",
      "2024-02-25 13:20:53,863 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091240.1, seqID CP091240.1.\n",
      "2024-02-25 13:20:53,879 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091227.1, seqID CP091227.1.\n",
      "2024-02-25 13:20:53,897 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097059.1, seqID CP097059.1.\n",
      "2024-02-25 13:20:53,913 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097030.1, seqID CP097030.1.\n",
      "2024-02-25 13:20:53,929 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097008.1, seqID CP097008.1.\n",
      "2024-02-25 13:20:53,946 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097038.1, seqID CP097038.1.\n",
      "2024-02-25 13:20:53,962 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097004.1, seqID CP097004.1.\n",
      "2024-02-25 13:20:53,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110289.1, seqID CP110289.1.\n",
      "2024-02-25 13:20:53,999 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP093957.1, seqID CP093957.1.\n",
      "2024-02-25 13:20:54,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP092907.1, seqID CP092907.1.\n",
      "2024-02-25 13:20:54,032 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091886.1, seqID CP091886.1.\n",
      "2024-02-25 13:20:54,047 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP063980.1, seqID CP063980.1.\n",
      "2024-02-25 13:20:54,063 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP085289.1, seqID CP085289.1.\n",
      "2024-02-25 13:20:54,079 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP045598.1, seqID CP045598.1.\n",
      "2024-02-25 13:20:54,093 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP050491.1, seqID CP050491.1.\n",
      "2024-02-25 13:20:54,110 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP060804.1, seqID CP060804.1.\n",
      "2024-02-25 13:20:54,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP030045.1, seqID CP030045.1.\n",
      "2024-02-25 13:20:54,142 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP039434.1, seqID CP039434.1.\n",
      "2024-02-25 13:20:54,159 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP028835.1, seqID CP028835.1.\n",
      "2024-02-25 13:20:54,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP028720.1.\n",
      "2024-02-25 13:20:54,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP028720.1.\n",
      "2024-02-25 13:20:54,161 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP028720.1, seqID CP028720.1.\n",
      "2024-02-25 13:20:54,176 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP018543.1, seqID AP018543.1.\n",
      "2024-02-25 13:20:54,191 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP018538.1, seqID AP018538.1.\n",
      "2024-02-25 13:20:54,208 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP033787.1, seqID CP033787.1.\n",
      "2024-02-25 13:20:54,226 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP031027.1, seqID CP031027.1.\n",
      "2024-02-25 13:20:54,245 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP029612.1, seqID CP029612.1.\n",
      "2024-02-25 13:20:54,263 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP019512.1, seqID CP019512.1.\n",
      "2024-02-25 13:20:54,279 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP015410.2, seqID CP015410.2.\n",
      "2024-02-25 13:20:54,298 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940431.1, seqID NZ_OD940431.1.\n",
      "2024-02-25 13:20:54,312 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962842.1, seqID NZ_LR962842.1.\n",
      "2024-02-25 13:20:54,330 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962843.1, seqID NZ_LR962843.1.\n",
      "2024-02-25 13:20:54,347 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962850.1, seqID NZ_LR962850.1.\n",
      "2024-02-25 13:20:54,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962840.1, seqID NZ_LR962840.1.\n",
      "2024-02-25 13:20:54,379 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962857.1, seqID NZ_LR962857.1.\n",
      "2024-02-25 13:20:54,395 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962832.1, seqID NZ_LR962832.1.\n",
      "2024-02-25 13:20:54,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962814.1, seqID NZ_LR962814.1.\n",
      "2024-02-25 13:20:54,426 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962775.1, seqID NZ_LR962775.1.\n",
      "2024-02-25 13:20:54,444 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962788.1, seqID NZ_LR962788.1.\n",
      "2024-02-25 13:20:54,460 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962736.1, seqID NZ_LR962736.1.\n",
      "2024-02-25 13:20:54,476 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962729.1, seqID NZ_LR962729.1.\n",
      "2024-02-25 13:20:54,491 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962726.1, seqID NZ_LR962726.1.\n",
      "2024-02-25 13:20:54,506 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962711.1, seqID NZ_LR962711.1.\n",
      "2024-02-25 13:20:54,524 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962714.1, seqID NZ_LR962714.1.\n",
      "2024-02-25 13:20:54,541 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962658.1, seqID NZ_LR962658.1.\n",
      "2024-02-25 13:20:54,560 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962671.1, seqID NZ_LR962671.1.\n",
      "2024-02-25 13:20:54,578 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962632.1, seqID NZ_LR962632.1.\n",
      "2024-02-25 13:20:54,594 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962604.1, seqID NZ_LR962604.1.\n",
      "2024-02-25 13:20:54,612 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962606.1, seqID NZ_LR962606.1.\n",
      "2024-02-25 13:20:54,628 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962634.1, seqID NZ_LR962634.1.\n",
      "2024-02-25 13:20:54,644 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962618.1, seqID NZ_LR962618.1.\n",
      "2024-02-25 13:20:54,659 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962602.1, seqID NZ_LR962602.1.\n",
      "2024-02-25 13:20:54,676 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962598.1, seqID NZ_LR962598.1.\n",
      "2024-02-25 13:20:54,693 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962600.1, seqID NZ_LR962600.1.\n",
      "2024-02-25 13:20:54,708 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962591.1, seqID NZ_LR962591.1.\n",
      "2024-02-25 13:20:54,723 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962595.1, seqID NZ_LR962595.1.\n",
      "2024-02-25 13:20:54,741 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962576.1, seqID NZ_LR962576.1.\n",
      "2024-02-25 13:20:54,757 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962567.1, seqID NZ_LR962567.1.\n",
      "2024-02-25 13:20:54,772 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962546.1, seqID NZ_LR962546.1.\n",
      "2024-02-25 13:20:54,787 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962545.1, seqID NZ_LR962545.1.\n",
      "2024-02-25 13:20:54,803 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962531.1, seqID NZ_LR962531.1.\n",
      "2024-02-25 13:20:54,819 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962539.1, seqID NZ_LR962539.1.\n",
      "2024-02-25 13:20:54,836 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962497.1, seqID NZ_LR962497.1.\n",
      "2024-02-25 13:20:54,851 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962434.1, seqID NZ_LR962434.1.\n",
      "2024-02-25 13:20:54,868 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962454.1, seqID NZ_LR962454.1.\n",
      "2024-02-25 13:20:54,886 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962432.1, seqID NZ_LR962432.1.\n",
      "2024-02-25 13:20:54,902 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962450.1, seqID NZ_LR962450.1.\n",
      "2024-02-25 13:20:54,918 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962393.1, seqID NZ_LR962393.1.\n",
      "2024-02-25 13:20:54,934 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962375.1, seqID NZ_LR962375.1.\n",
      "2024-02-25 13:20:54,950 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962346.1, seqID NZ_LR962346.1.\n",
      "2024-02-25 13:20:54,966 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962348.1, seqID NZ_LR962348.1.\n",
      "2024-02-25 13:20:54,981 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962315.1, seqID NZ_LR962315.1.\n",
      "2024-02-25 13:20:54,997 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962328.1, seqID NZ_LR962328.1.\n",
      "2024-02-25 13:20:55,015 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962239.1, seqID NZ_LR962239.1.\n",
      "2024-02-25 13:20:55,034 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962234.1, seqID NZ_LR962234.1.\n",
      "2024-02-25 13:20:55,050 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962225.1, seqID NZ_LR962225.1.\n",
      "2024-02-25 13:20:55,066 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962210.1, seqID NZ_LR962210.1.\n",
      "2024-02-25 13:20:55,084 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962213.1, seqID NZ_LR962213.1.\n",
      "2024-02-25 13:20:55,100 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962189.1, seqID NZ_LR962189.1.\n",
      "2024-02-25 13:20:55,116 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962188.1, seqID NZ_LR962188.1.\n",
      "2024-02-25 13:20:55,131 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962197.1, seqID NZ_LR962197.1.\n",
      "2024-02-25 13:20:55,147 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962230.1, seqID NZ_LR962230.1.\n",
      "2024-02-25 13:20:55,165 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962171.1, seqID NZ_LR962171.1.\n",
      "2024-02-25 13:20:55,180 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962163.1, seqID NZ_LR962163.1.\n",
      "2024-02-25 13:20:55,195 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962161.1, seqID NZ_LR962161.1.\n",
      "2024-02-25 13:20:55,212 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962132.1, seqID NZ_LR962132.1.\n",
      "2024-02-25 13:20:55,226 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962137.1, seqID NZ_LR962137.1.\n",
      "2024-02-25 13:20:55,243 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962112.1, seqID NZ_LR962112.1.\n",
      "2024-02-25 13:20:55,260 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962103.1, seqID NZ_LR962103.1.\n",
      "2024-02-25 13:20:55,277 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962107.1, seqID NZ_LR962107.1.\n",
      "2024-02-25 13:20:55,294 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962075.1, seqID NZ_LR962075.1.\n",
      "2024-02-25 13:20:55,311 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962039.1, seqID NZ_LR962039.1.\n",
      "2024-02-25 13:20:55,326 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962034.1, seqID NZ_LR962034.1.\n",
      "2024-02-25 13:20:55,343 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962030.1, seqID NZ_LR962030.1.\n",
      "2024-02-25 13:20:55,358 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962017.1, seqID NZ_LR962017.1.\n",
      "2024-02-25 13:20:55,372 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962010.1, seqID NZ_LR962010.1.\n",
      "2024-02-25 13:20:55,387 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961996.1, seqID NZ_LR961996.1.\n",
      "2024-02-25 13:20:55,402 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961994.1, seqID NZ_LR961994.1.\n",
      "2024-02-25 13:20:55,417 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961986.1, seqID NZ_LR961986.1.\n",
      "2024-02-25 13:20:55,433 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962008.1, seqID NZ_LR962008.1.\n",
      "2024-02-25 13:20:55,450 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961981.1, seqID NZ_LR961981.1.\n",
      "2024-02-25 13:20:55,466 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961957.1, seqID NZ_LR961957.1.\n",
      "2024-02-25 13:20:55,482 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961953.1, seqID NZ_LR961953.1.\n",
      "2024-02-25 13:20:55,499 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961923.1, seqID NZ_LR961923.1.\n",
      "2024-02-25 13:20:55,514 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961959.1, seqID NZ_LR961959.1.\n",
      "2024-02-25 13:20:55,531 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124778.1, seqID NZ_CP124778.1.\n",
      "2024-02-25 13:20:55,549 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP127177.1, seqID NZ_CP127177.1.\n",
      "2024-02-25 13:20:55,568 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP109611.1, seqID NZ_CP109611.1.\n",
      "2024-02-25 13:20:55,585 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124914.1, seqID NZ_CP124914.1.\n",
      "2024-02-25 13:20:55,604 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124891.1, seqID NZ_CP124891.1.\n",
      "2024-02-25 13:20:55,621 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP118048.1, seqID NZ_CP118048.1.\n",
      "2024-02-25 13:20:55,639 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP117511.1, seqID NZ_CP117511.1.\n",
      "2024-02-25 13:20:55,657 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP117503.1, seqID NZ_CP117503.1.\n",
      "2024-02-25 13:20:55,676 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP121233.1, seqID NZ_CP121233.1.\n",
      "2024-02-25 13:20:55,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116566.1, seqID NZ_CP116566.1.\n",
      "2024-02-25 13:20:55,712 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116564.1, seqID NZ_CP116564.1.\n",
      "2024-02-25 13:20:55,730 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116561.1, seqID NZ_CP116561.1.\n",
      "2024-02-25 13:20:55,747 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116569.1, seqID NZ_CP116569.1.\n",
      "2024-02-25 13:20:55,768 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP026721.1, seqID NZ_AP026721.1.\n",
      "2024-02-25 13:20:55,788 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP115992.1, seqID NZ_CP115992.1.\n",
      "2024-02-25 13:20:55,806 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110036.1, seqID NZ_CP110036.1.\n",
      "2024-02-25 13:20:55,824 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110020.1, seqID NZ_CP110020.1.\n",
      "2024-02-25 13:20:55,843 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110019.1, seqID NZ_CP110019.1.\n",
      "2024-02-25 13:20:55,862 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP098418.1, seqID NZ_CP098418.1.\n",
      "2024-02-25 13:20:55,882 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP098025.1, seqID NZ_CP098025.1.\n",
      "2024-02-25 13:20:55,899 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091237.1, seqID NZ_CP091237.1.\n",
      "2024-02-25 13:20:55,918 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091240.1, seqID NZ_CP091240.1.\n",
      "2024-02-25 13:20:55,936 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091227.1, seqID NZ_CP091227.1.\n",
      "2024-02-25 13:20:55,955 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097059.1, seqID NZ_CP097059.1.\n",
      "2024-02-25 13:20:55,974 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097030.1, seqID NZ_CP097030.1.\n",
      "2024-02-25 13:20:55,991 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097008.1, seqID NZ_CP097008.1.\n",
      "2024-02-25 13:20:56,010 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097038.1, seqID NZ_CP097038.1.\n",
      "2024-02-25 13:20:56,028 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097004.1, seqID NZ_CP097004.1.\n",
      "2024-02-25 13:20:56,046 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110289.1, seqID NZ_CP110289.1.\n",
      "2024-02-25 13:20:56,067 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP093957.1, seqID NZ_CP093957.1.\n",
      "2024-02-25 13:20:56,086 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP092907.1, seqID NZ_CP092907.1.\n",
      "2024-02-25 13:20:56,105 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091886.1, seqID NZ_CP091886.1.\n",
      "2024-02-25 13:20:56,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP063980.1, seqID NZ_CP063980.1.\n",
      "2024-02-25 13:20:56,140 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085289.1, seqID NZ_CP085289.1.\n",
      "2024-02-25 13:20:56,157 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045598.1, seqID NZ_CP045598.1.\n",
      "2024-02-25 13:20:56,174 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP050491.1, seqID NZ_CP050491.1.\n",
      "2024-02-25 13:20:56,194 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060804.1, seqID NZ_CP060804.1.\n",
      "2024-02-25 13:20:56,212 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP030045.1, seqID NZ_CP030045.1.\n",
      "2024-02-25 13:20:56,231 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP039434.1, seqID NZ_CP039434.1.\n",
      "2024-02-25 13:20:56,250 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028835.1, seqID NZ_CP028835.1.\n",
      "2024-02-25 13:20:56,268 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028720.1, seqID NZ_CP028720.1.\n",
      "2024-02-25 13:20:56,286 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018543.1, seqID NZ_AP018543.1.\n",
      "2024-02-25 13:20:56,304 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_AP018538.1, seqID NZ_AP018538.1.\n",
      "2024-02-25 13:20:56,324 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP033787.1, seqID NZ_CP033787.1.\n",
      "2024-02-25 13:20:56,344 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP031027.1, seqID NZ_CP031027.1.\n",
      "2024-02-25 13:20:56,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP029612.1, seqID NZ_CP029612.1.\n",
      "2024-02-25 13:20:56,384 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP019512.1, seqID NZ_CP019512.1.\n",
      "2024-02-25 13:20:56,403 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP015410.2, seqID NZ_CP015410.2.\n",
      "2024-02-25 13:20:56,420 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP036249.1, seqID CP036249.1.\n",
      "2024-02-25 13:20:56,439 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP036249.1, seqID NZ_CP036249.1.\n",
      "2024-02-25 13:20:56,440 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962826.1.\n",
      "2024-02-25 13:20:56,440 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962826.1.\n",
      "2024-02-25 13:20:56,440 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962826.1, seqID LR962826.1.\n",
      "2024-02-25 13:20:56,441 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962797.1.\n",
      "2024-02-25 13:20:56,441 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962797.1.\n",
      "2024-02-25 13:20:56,441 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962797.1, seqID LR962797.1.\n",
      "2024-02-25 13:20:56,443 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962807.1.\n",
      "2024-02-25 13:20:56,443 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962807.1.\n",
      "2024-02-25 13:20:56,443 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962807.1, seqID LR962807.1.\n",
      "2024-02-25 13:20:56,444 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962738.1.\n",
      "2024-02-25 13:20:56,444 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962738.1.\n",
      "2024-02-25 13:20:56,444 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962738.1, seqID LR962738.1.\n",
      "2024-02-25 13:20:56,445 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962723.1.\n",
      "2024-02-25 13:20:56,445 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962723.1.\n",
      "2024-02-25 13:20:56,445 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962723.1, seqID LR962723.1.\n",
      "2024-02-25 13:20:56,446 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962703.1.\n",
      "2024-02-25 13:20:56,446 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962703.1.\n",
      "2024-02-25 13:20:56,447 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962703.1, seqID LR962703.1.\n",
      "2024-02-25 13:20:56,448 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962687.1.\n",
      "2024-02-25 13:20:56,448 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962687.1.\n",
      "2024-02-25 13:20:56,448 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962687.1, seqID LR962687.1.\n",
      "2024-02-25 13:20:56,449 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962681.1.\n",
      "2024-02-25 13:20:56,449 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962681.1.\n",
      "2024-02-25 13:20:56,449 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962681.1, seqID LR962681.1.\n",
      "2024-02-25 13:20:56,450 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962666.1.\n",
      "2024-02-25 13:20:56,450 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962666.1.\n",
      "2024-02-25 13:20:56,450 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962666.1, seqID LR962666.1.\n",
      "2024-02-25 13:20:56,452 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962617.1.\n",
      "2024-02-25 13:20:56,452 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962617.1.\n",
      "2024-02-25 13:20:56,452 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962617.1, seqID LR962617.1.\n",
      "2024-02-25 13:20:56,453 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962550.1.\n",
      "2024-02-25 13:20:56,453 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962550.1.\n",
      "2024-02-25 13:20:56,453 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962550.1, seqID LR962550.1.\n",
      "2024-02-25 13:20:56,454 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962525.1.\n",
      "2024-02-25 13:20:56,454 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962525.1.\n",
      "2024-02-25 13:20:56,454 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962525.1, seqID LR962525.1.\n",
      "2024-02-25 13:20:56,455 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962489.1.\n",
      "2024-02-25 13:20:56,455 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962489.1.\n",
      "2024-02-25 13:20:56,455 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962489.1, seqID LR962489.1.\n",
      "2024-02-25 13:20:56,456 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962468.1.\n",
      "2024-02-25 13:20:56,456 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962468.1.\n",
      "2024-02-25 13:20:56,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962468.1, seqID LR962468.1.\n",
      "2024-02-25 13:20:56,457 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962480.1.\n",
      "2024-02-25 13:20:56,457 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962480.1.\n",
      "2024-02-25 13:20:56,457 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962480.1, seqID LR962480.1.\n",
      "2024-02-25 13:20:56,459 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962476.1.\n",
      "2024-02-25 13:20:56,459 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962476.1.\n",
      "2024-02-25 13:20:56,459 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962476.1, seqID LR962476.1.\n",
      "2024-02-25 13:20:56,460 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962428.1.\n",
      "2024-02-25 13:20:56,460 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962428.1.\n",
      "2024-02-25 13:20:56,460 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962428.1, seqID LR962428.1.\n",
      "2024-02-25 13:20:56,461 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962447.1.\n",
      "2024-02-25 13:20:56,461 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962447.1.\n",
      "2024-02-25 13:20:56,461 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962447.1, seqID LR962447.1.\n",
      "2024-02-25 13:20:56,462 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962395.1.\n",
      "2024-02-25 13:20:56,462 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962395.1.\n",
      "2024-02-25 13:20:56,462 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962395.1, seqID LR962395.1.\n",
      "2024-02-25 13:20:56,464 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962284.1.\n",
      "2024-02-25 13:20:56,464 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962284.1.\n",
      "2024-02-25 13:20:56,464 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962284.1, seqID LR962284.1.\n",
      "2024-02-25 13:20:56,465 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962280.1.\n",
      "2024-02-25 13:20:56,465 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962280.1.\n",
      "2024-02-25 13:20:56,465 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962280.1, seqID LR962280.1.\n",
      "2024-02-25 13:20:56,466 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962251.1.\n",
      "2024-02-25 13:20:56,466 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962251.1.\n",
      "2024-02-25 13:20:56,466 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962251.1, seqID LR962251.1.\n",
      "2024-02-25 13:20:56,467 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962254.1.\n",
      "2024-02-25 13:20:56,467 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962254.1.\n",
      "2024-02-25 13:20:56,467 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962254.1, seqID LR962254.1.\n",
      "2024-02-25 13:20:56,469 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962258.1.\n",
      "2024-02-25 13:20:56,469 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962258.1.\n",
      "2024-02-25 13:20:56,469 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962258.1, seqID LR962258.1.\n",
      "2024-02-25 13:20:56,470 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962231.1.\n",
      "2024-02-25 13:20:56,470 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962231.1.\n",
      "2024-02-25 13:20:56,470 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962231.1, seqID LR962231.1.\n",
      "2024-02-25 13:20:56,472 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962178.1.\n",
      "2024-02-25 13:20:56,472 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962178.1.\n",
      "2024-02-25 13:20:56,472 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962178.1, seqID LR962178.1.\n",
      "2024-02-25 13:20:56,473 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962138.1.\n",
      "2024-02-25 13:20:56,473 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962138.1.\n",
      "2024-02-25 13:20:56,473 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962138.1, seqID LR962138.1.\n",
      "2024-02-25 13:20:56,474 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962144.1.\n",
      "2024-02-25 13:20:56,474 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962144.1.\n",
      "2024-02-25 13:20:56,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962144.1, seqID LR962144.1.\n",
      "2024-02-25 13:20:56,475 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962158.1.\n",
      "2024-02-25 13:20:56,476 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962158.1.\n",
      "2024-02-25 13:20:56,476 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962158.1, seqID LR962158.1.\n",
      "2024-02-25 13:20:56,477 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962146.1.\n",
      "2024-02-25 13:20:56,477 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962146.1.\n",
      "2024-02-25 13:20:56,477 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962146.1, seqID LR962146.1.\n",
      "2024-02-25 13:20:56,478 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962125.1.\n",
      "2024-02-25 13:20:56,478 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962125.1.\n",
      "2024-02-25 13:20:56,478 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962125.1, seqID LR962125.1.\n",
      "2024-02-25 13:20:56,480 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962091.1.\n",
      "2024-02-25 13:20:56,480 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962091.1.\n",
      "2024-02-25 13:20:56,480 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962091.1, seqID LR962091.1.\n",
      "2024-02-25 13:20:56,481 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962086.1.\n",
      "2024-02-25 13:20:56,481 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962086.1.\n",
      "2024-02-25 13:20:56,481 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962086.1, seqID LR962086.1.\n",
      "2024-02-25 13:20:56,482 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962098.1.\n",
      "2024-02-25 13:20:56,482 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962098.1.\n",
      "2024-02-25 13:20:56,482 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962098.1, seqID LR962098.1.\n",
      "2024-02-25 13:20:56,483 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962078.1.\n",
      "2024-02-25 13:20:56,483 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962078.1.\n",
      "2024-02-25 13:20:56,484 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962078.1, seqID LR962078.1.\n",
      "2024-02-25 13:20:56,485 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962046.1.\n",
      "2024-02-25 13:20:56,485 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962046.1.\n",
      "2024-02-25 13:20:56,485 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962046.1, seqID LR962046.1.\n",
      "2024-02-25 13:20:56,486 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962053.1.\n",
      "2024-02-25 13:20:56,486 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962053.1.\n",
      "2024-02-25 13:20:56,486 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962053.1, seqID LR962053.1.\n",
      "2024-02-25 13:20:56,487 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962029.1.\n",
      "2024-02-25 13:20:56,487 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962029.1.\n",
      "2024-02-25 13:20:56,487 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962029.1, seqID LR962029.1.\n",
      "2024-02-25 13:20:56,489 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962002.1.\n",
      "2024-02-25 13:20:56,489 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962002.1.\n",
      "2024-02-25 13:20:56,489 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962002.1, seqID LR962002.1.\n",
      "2024-02-25 13:20:56,490 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961951.1.\n",
      "2024-02-25 13:20:56,490 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961951.1.\n",
      "2024-02-25 13:20:56,490 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961951.1, seqID LR961951.1.\n",
      "2024-02-25 13:20:56,491 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961942.1.\n",
      "2024-02-25 13:20:56,491 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961942.1.\n",
      "2024-02-25 13:20:56,491 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961942.1, seqID LR961942.1.\n",
      "2024-02-25 13:20:56,492 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961948.1.\n",
      "2024-02-25 13:20:56,492 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961948.1.\n",
      "2024-02-25 13:20:56,492 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961948.1, seqID LR961948.1.\n",
      "2024-02-25 13:20:56,510 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124956.1, seqID CP124956.1.\n",
      "2024-02-25 13:20:56,533 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain AP026718.1, seqID AP026718.1.\n",
      "2024-02-25 13:20:56,550 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP113832.1, seqID CP113832.1.\n",
      "2024-02-25 13:20:56,565 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP085841.1, seqID CP085841.1.\n",
      "2024-02-25 13:20:56,583 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110068.1, seqID CP110068.1.\n",
      "2024-02-25 13:20:56,599 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110059.1, seqID CP110059.1.\n",
      "2024-02-25 13:20:56,616 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110054.1, seqID CP110054.1.\n",
      "2024-02-25 13:20:56,632 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110040.1, seqID CP110040.1.\n",
      "2024-02-25 13:20:56,648 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110039.1, seqID CP110039.1.\n",
      "2024-02-25 13:20:56,665 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP098743.1, seqID CP098743.1.\n",
      "2024-02-25 13:20:56,681 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097031.1, seqID CP097031.1.\n",
      "2024-02-25 13:20:56,698 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091904.1, seqID CP091904.1.\n",
      "2024-02-25 13:20:56,714 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP085291.1, seqID CP085291.1.\n",
      "2024-02-25 13:20:56,731 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP081505.1, seqID CP081505.1.\n",
      "2024-02-25 13:20:56,746 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP060796.1, seqID CP060796.1.\n",
      "2024-02-25 13:20:56,763 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP068249.1, seqID CP068249.1.\n",
      "2024-02-25 13:20:56,778 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962826.1, seqID NZ_LR962826.1.\n",
      "2024-02-25 13:20:56,793 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962797.1, seqID NZ_LR962797.1.\n",
      "2024-02-25 13:20:56,810 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962807.1, seqID NZ_LR962807.1.\n",
      "2024-02-25 13:20:56,825 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962738.1, seqID NZ_LR962738.1.\n",
      "2024-02-25 13:20:56,841 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962723.1, seqID NZ_LR962723.1.\n",
      "2024-02-25 13:20:56,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962703.1, seqID NZ_LR962703.1.\n",
      "2024-02-25 13:20:56,872 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962687.1, seqID NZ_LR962687.1.\n",
      "2024-02-25 13:20:56,888 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962681.1, seqID NZ_LR962681.1.\n",
      "2024-02-25 13:20:56,904 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962666.1, seqID NZ_LR962666.1.\n",
      "2024-02-25 13:20:56,919 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962617.1, seqID NZ_LR962617.1.\n",
      "2024-02-25 13:20:56,934 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962550.1, seqID NZ_LR962550.1.\n",
      "2024-02-25 13:20:56,949 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962525.1, seqID NZ_LR962525.1.\n",
      "2024-02-25 13:20:56,964 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962489.1, seqID NZ_LR962489.1.\n",
      "2024-02-25 13:20:56,980 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962468.1, seqID NZ_LR962468.1.\n",
      "2024-02-25 13:20:56,995 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962480.1, seqID NZ_LR962480.1.\n",
      "2024-02-25 13:20:57,011 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962476.1, seqID NZ_LR962476.1.\n",
      "2024-02-25 13:20:57,027 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962428.1, seqID NZ_LR962428.1.\n",
      "2024-02-25 13:20:57,042 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962447.1, seqID NZ_LR962447.1.\n",
      "2024-02-25 13:20:57,055 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962395.1, seqID NZ_LR962395.1.\n",
      "2024-02-25 13:20:57,072 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962284.1, seqID NZ_LR962284.1.\n",
      "2024-02-25 13:20:57,088 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962280.1, seqID NZ_LR962280.1.\n",
      "2024-02-25 13:20:57,103 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962251.1, seqID NZ_LR962251.1.\n",
      "2024-02-25 13:20:57,119 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962254.1, seqID NZ_LR962254.1.\n",
      "2024-02-25 13:20:57,135 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962258.1, seqID NZ_LR962258.1.\n",
      "2024-02-25 13:20:57,151 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962231.1, seqID NZ_LR962231.1.\n",
      "2024-02-25 13:20:57,167 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962178.1, seqID NZ_LR962178.1.\n",
      "2024-02-25 13:20:57,184 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962138.1, seqID NZ_LR962138.1.\n",
      "2024-02-25 13:20:57,198 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962144.1, seqID NZ_LR962144.1.\n",
      "2024-02-25 13:20:57,215 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962158.1, seqID NZ_LR962158.1.\n",
      "2024-02-25 13:20:57,233 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962146.1, seqID NZ_LR962146.1.\n",
      "2024-02-25 13:20:57,250 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962125.1, seqID NZ_LR962125.1.\n",
      "2024-02-25 13:20:57,267 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962091.1, seqID NZ_LR962091.1.\n",
      "2024-02-25 13:20:57,285 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962086.1, seqID NZ_LR962086.1.\n",
      "2024-02-25 13:20:57,302 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962098.1, seqID NZ_LR962098.1.\n",
      "2024-02-25 13:20:57,319 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962078.1, seqID NZ_LR962078.1.\n",
      "2024-02-25 13:20:57,336 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962046.1, seqID NZ_LR962046.1.\n",
      "2024-02-25 13:20:57,355 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962053.1, seqID NZ_LR962053.1.\n",
      "2024-02-25 13:20:57,371 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962029.1, seqID NZ_LR962029.1.\n",
      "2024-02-25 13:20:57,387 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962002.1, seqID NZ_LR962002.1.\n",
      "2024-02-25 13:20:57,403 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961951.1, seqID NZ_LR961951.1.\n",
      "2024-02-25 13:20:57,420 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961942.1, seqID NZ_LR961942.1.\n",
      "2024-02-25 13:20:57,436 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961948.1, seqID NZ_LR961948.1.\n",
      "2024-02-25 13:20:57,456 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124956.1, seqID NZ_CP124956.1.\n",
      "2024-02-25 13:20:57,474 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP113832.1, seqID NZ_CP113832.1.\n",
      "2024-02-25 13:20:57,492 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085841.1, seqID NZ_CP085841.1.\n",
      "2024-02-25 13:20:57,510 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110068.1, seqID NZ_CP110068.1.\n",
      "2024-02-25 13:20:57,528 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110059.1, seqID NZ_CP110059.1.\n",
      "2024-02-25 13:20:57,547 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110054.1, seqID NZ_CP110054.1.\n",
      "2024-02-25 13:20:57,564 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110040.1, seqID NZ_CP110040.1.\n",
      "2024-02-25 13:20:57,581 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110039.1, seqID NZ_CP110039.1.\n",
      "2024-02-25 13:20:57,599 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP098743.1, seqID NZ_CP098743.1.\n",
      "2024-02-25 13:20:57,617 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097031.1, seqID NZ_CP097031.1.\n",
      "2024-02-25 13:20:57,636 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091904.1, seqID NZ_CP091904.1.\n",
      "2024-02-25 13:20:57,655 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP085291.1, seqID NZ_CP085291.1.\n",
      "2024-02-25 13:20:57,673 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP081505.1, seqID NZ_CP081505.1.\n",
      "2024-02-25 13:20:57,691 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP060796.1, seqID NZ_CP060796.1.\n",
      "2024-02-25 13:20:57,711 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP068249.1, seqID NZ_CP068249.1.\n",
      "2024-02-25 13:20:57,727 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain OD940437.1, seqID OD940437.1.\n",
      "2024-02-25 13:20:57,729 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962852.1.\n",
      "2024-02-25 13:20:57,729 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962852.1.\n",
      "2024-02-25 13:20:57,729 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962852.1, seqID LR962852.1.\n",
      "2024-02-25 13:20:57,730 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962825.1.\n",
      "2024-02-25 13:20:57,730 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962825.1.\n",
      "2024-02-25 13:20:57,730 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962825.1, seqID LR962825.1.\n",
      "2024-02-25 13:20:57,731 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962820.1.\n",
      "2024-02-25 13:20:57,731 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962820.1.\n",
      "2024-02-25 13:20:57,731 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962820.1, seqID LR962820.1.\n",
      "2024-02-25 13:20:57,733 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962772.1.\n",
      "2024-02-25 13:20:57,733 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962772.1.\n",
      "2024-02-25 13:20:57,733 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962772.1, seqID LR962772.1.\n",
      "2024-02-25 13:20:57,734 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962805.1.\n",
      "2024-02-25 13:20:57,734 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962805.1.\n",
      "2024-02-25 13:20:57,734 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962805.1, seqID LR962805.1.\n",
      "2024-02-25 13:20:57,735 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962771.1.\n",
      "2024-02-25 13:20:57,735 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962771.1.\n",
      "2024-02-25 13:20:57,735 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962771.1, seqID LR962771.1.\n",
      "2024-02-25 13:20:57,737 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962691.1.\n",
      "2024-02-25 13:20:57,737 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962691.1.\n",
      "2024-02-25 13:20:57,737 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962691.1, seqID LR962691.1.\n",
      "2024-02-25 13:20:57,738 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962650.1.\n",
      "2024-02-25 13:20:57,738 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962650.1.\n",
      "2024-02-25 13:20:57,738 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962650.1, seqID LR962650.1.\n",
      "2024-02-25 13:20:57,739 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962680.1.\n",
      "2024-02-25 13:20:57,739 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962680.1.\n",
      "2024-02-25 13:20:57,739 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962680.1, seqID LR962680.1.\n",
      "2024-02-25 13:20:57,740 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962676.1.\n",
      "2024-02-25 13:20:57,740 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962676.1.\n",
      "2024-02-25 13:20:57,740 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962676.1, seqID LR962676.1.\n",
      "2024-02-25 13:20:57,742 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962626.1.\n",
      "2024-02-25 13:20:57,742 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962626.1.\n",
      "2024-02-25 13:20:57,742 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962626.1, seqID LR962626.1.\n",
      "2024-02-25 13:20:57,743 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962541.1.\n",
      "2024-02-25 13:20:57,743 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962541.1.\n",
      "2024-02-25 13:20:57,743 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962541.1, seqID LR962541.1.\n",
      "2024-02-25 13:20:57,744 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962501.1.\n",
      "2024-02-25 13:20:57,744 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962501.1.\n",
      "2024-02-25 13:20:57,744 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962501.1, seqID LR962501.1.\n",
      "2024-02-25 13:20:57,746 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962519.1.\n",
      "2024-02-25 13:20:57,746 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962519.1.\n",
      "2024-02-25 13:20:57,746 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962519.1, seqID LR962519.1.\n",
      "2024-02-25 13:20:57,747 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962426.1.\n",
      "2024-02-25 13:20:57,747 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962426.1.\n",
      "2024-02-25 13:20:57,747 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962426.1, seqID LR962426.1.\n",
      "2024-02-25 13:20:57,748 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962412.1.\n",
      "2024-02-25 13:20:57,748 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962412.1.\n",
      "2024-02-25 13:20:57,748 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962412.1, seqID LR962412.1.\n",
      "2024-02-25 13:20:57,750 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962427.1.\n",
      "2024-02-25 13:20:57,750 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962427.1.\n",
      "2024-02-25 13:20:57,750 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962427.1, seqID LR962427.1.\n",
      "2024-02-25 13:20:57,751 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962288.1.\n",
      "2024-02-25 13:20:57,751 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962288.1.\n",
      "2024-02-25 13:20:57,751 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962288.1, seqID LR962288.1.\n",
      "2024-02-25 13:20:57,752 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962316.1.\n",
      "2024-02-25 13:20:57,752 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962316.1.\n",
      "2024-02-25 13:20:57,752 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962316.1, seqID LR962316.1.\n",
      "2024-02-25 13:20:57,754 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962275.1.\n",
      "2024-02-25 13:20:57,754 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962275.1.\n",
      "2024-02-25 13:20:57,754 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962275.1, seqID LR962275.1.\n",
      "2024-02-25 13:20:57,755 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962264.1.\n",
      "2024-02-25 13:20:57,755 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962264.1.\n",
      "2024-02-25 13:20:57,755 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962264.1, seqID LR962264.1.\n",
      "2024-02-25 13:20:57,756 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962211.1.\n",
      "2024-02-25 13:20:57,756 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962211.1.\n",
      "2024-02-25 13:20:57,756 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962211.1, seqID LR962211.1.\n",
      "2024-02-25 13:20:57,757 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962149.1.\n",
      "2024-02-25 13:20:57,757 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962149.1.\n",
      "2024-02-25 13:20:57,757 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962149.1, seqID LR962149.1.\n",
      "2024-02-25 13:20:57,759 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962117.1.\n",
      "2024-02-25 13:20:57,759 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962117.1.\n",
      "2024-02-25 13:20:57,759 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962117.1, seqID LR962117.1.\n",
      "2024-02-25 13:20:57,760 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962085.1.\n",
      "2024-02-25 13:20:57,760 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962085.1.\n",
      "2024-02-25 13:20:57,760 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962085.1, seqID LR962085.1.\n",
      "2024-02-25 13:20:57,762 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962074.1.\n",
      "2024-02-25 13:20:57,762 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962074.1.\n",
      "2024-02-25 13:20:57,762 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962074.1, seqID LR962074.1.\n",
      "2024-02-25 13:20:57,763 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962016.1.\n",
      "2024-02-25 13:20:57,763 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962016.1.\n",
      "2024-02-25 13:20:57,763 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962016.1, seqID LR962016.1.\n",
      "2024-02-25 13:20:57,765 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961980.1.\n",
      "2024-02-25 13:20:57,765 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961980.1.\n",
      "2024-02-25 13:20:57,765 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961980.1, seqID LR961980.1.\n",
      "2024-02-25 13:20:57,766 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961924.1.\n",
      "2024-02-25 13:20:57,766 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR961924.1.\n",
      "2024-02-25 13:20:57,766 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR961924.1, seqID LR961924.1.\n",
      "2024-02-25 13:20:57,784 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124960.1, seqID CP124960.1.\n",
      "2024-02-25 13:20:57,802 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124950.1, seqID CP124950.1.\n",
      "2024-02-25 13:20:57,820 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124939.1, seqID CP124939.1.\n",
      "2024-02-25 13:20:57,838 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124935.1, seqID CP124935.1.\n",
      "2024-02-25 13:20:57,857 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124925.1, seqID CP124925.1.\n",
      "2024-02-25 13:20:57,875 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124913.1, seqID CP124913.1.\n",
      "2024-02-25 13:20:57,893 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124905.1, seqID CP124905.1.\n",
      "2024-02-25 13:20:57,911 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124898.1, seqID CP124898.1.\n",
      "2024-02-25 13:20:57,928 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124893.1, seqID CP124893.1.\n",
      "2024-02-25 13:20:57,945 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124890.1, seqID CP124890.1.\n",
      "2024-02-25 13:20:57,962 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124887.1, seqID CP124887.1.\n",
      "2024-02-25 13:20:57,981 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP104856.1, seqID CP104856.1.\n",
      "2024-02-25 13:20:57,996 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116553.1, seqID CP116553.1.\n",
      "2024-02-25 13:20:58,013 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116557.1, seqID CP116557.1.\n",
      "2024-02-25 13:20:58,029 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116559.1, seqID CP116559.1.\n",
      "2024-02-25 13:20:58,045 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP116555.1, seqID CP116555.1.\n",
      "2024-02-25 13:20:58,063 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110072.1, seqID CP110072.1.\n",
      "2024-02-25 13:20:58,080 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110071.1, seqID CP110071.1.\n",
      "2024-02-25 13:20:58,097 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110053.1, seqID CP110053.1.\n",
      "2024-02-25 13:20:58,116 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110035.1, seqID CP110035.1.\n",
      "2024-02-25 13:20:58,133 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP110022.1, seqID CP110022.1.\n",
      "2024-02-25 13:20:58,150 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP080759.1, seqID CP080759.1.\n",
      "2024-02-25 13:20:58,166 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091235.1, seqID CP091235.1.\n",
      "2024-02-25 13:20:58,181 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP088198.1, seqID CP088198.1.\n",
      "2024-02-25 13:20:58,198 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097042.1, seqID CP097042.1.\n",
      "2024-02-25 13:20:58,215 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097034.1, seqID CP097034.1.\n",
      "2024-02-25 13:20:58,231 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP097006.1, seqID CP097006.1.\n",
      "2024-02-25 13:20:58,247 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP046247.1, seqID CP046247.1.\n",
      "2024-02-25 13:20:58,263 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091897.1, seqID CP091897.1.\n",
      "2024-02-25 13:20:58,279 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091895.1, seqID CP091895.1.\n",
      "2024-02-25 13:20:58,296 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP091884.1, seqID CP091884.1.\n",
      "2024-02-25 13:20:58,313 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP086560.1, seqID CP086560.1.\n",
      "2024-02-25 13:20:58,329 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP078015.1, seqID CP078015.1.\n",
      "2024-02-25 13:20:58,346 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP071176.1, seqID CP071176.1.\n",
      "2024-02-25 13:20:58,364 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP071172.1, seqID CP071172.1.\n",
      "2024-02-25 13:20:58,380 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP069185.1, seqID CP069185.1.\n",
      "2024-02-25 13:20:58,397 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP051005.1, seqID CP051005.1.\n",
      "2024-02-25 13:20:58,415 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP045045.1, seqID CP045045.1.\n",
      "2024-02-25 13:20:58,432 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP042216.2, seqID CP042216.2.\n",
      "2024-02-25 13:20:58,449 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP040898.1, seqID CP040898.1.\n",
      "2024-02-25 13:20:58,450 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP028724.1.\n",
      "2024-02-25 13:20:58,450 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP028724.1.\n",
      "2024-02-25 13:20:58,450 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP028724.1, seqID CP028724.1.\n",
      "2024-02-25 13:20:58,470 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_OD940437.1, seqID NZ_OD940437.1.\n",
      "2024-02-25 13:20:58,487 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962852.1, seqID NZ_LR962852.1.\n",
      "2024-02-25 13:20:58,504 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962825.1, seqID NZ_LR962825.1.\n",
      "2024-02-25 13:20:58,521 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962820.1, seqID NZ_LR962820.1.\n",
      "2024-02-25 13:20:58,538 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962772.1, seqID NZ_LR962772.1.\n",
      "2024-02-25 13:20:58,555 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962805.1, seqID NZ_LR962805.1.\n",
      "2024-02-25 13:20:58,572 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962771.1, seqID NZ_LR962771.1.\n",
      "2024-02-25 13:20:58,589 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962691.1, seqID NZ_LR962691.1.\n",
      "2024-02-25 13:20:58,608 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962650.1, seqID NZ_LR962650.1.\n",
      "2024-02-25 13:20:58,624 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962680.1, seqID NZ_LR962680.1.\n",
      "2024-02-25 13:20:58,643 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962676.1, seqID NZ_LR962676.1.\n",
      "2024-02-25 13:20:58,660 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962626.1, seqID NZ_LR962626.1.\n",
      "2024-02-25 13:20:58,677 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962541.1, seqID NZ_LR962541.1.\n",
      "2024-02-25 13:20:58,694 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962501.1, seqID NZ_LR962501.1.\n",
      "2024-02-25 13:20:58,711 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962519.1, seqID NZ_LR962519.1.\n",
      "2024-02-25 13:20:58,728 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962426.1, seqID NZ_LR962426.1.\n",
      "2024-02-25 13:20:58,745 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962412.1, seqID NZ_LR962412.1.\n",
      "2024-02-25 13:20:58,761 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962427.1, seqID NZ_LR962427.1.\n",
      "2024-02-25 13:20:58,779 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962288.1, seqID NZ_LR962288.1.\n",
      "2024-02-25 13:20:58,797 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962316.1, seqID NZ_LR962316.1.\n",
      "2024-02-25 13:20:58,813 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962275.1, seqID NZ_LR962275.1.\n",
      "2024-02-25 13:20:58,831 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962264.1, seqID NZ_LR962264.1.\n",
      "2024-02-25 13:20:58,847 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962211.1, seqID NZ_LR962211.1.\n",
      "2024-02-25 13:20:58,864 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962149.1, seqID NZ_LR962149.1.\n",
      "2024-02-25 13:20:58,880 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962117.1, seqID NZ_LR962117.1.\n",
      "2024-02-25 13:20:58,898 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962085.1, seqID NZ_LR962085.1.\n",
      "2024-02-25 13:20:58,914 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962074.1, seqID NZ_LR962074.1.\n",
      "2024-02-25 13:20:58,931 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962016.1, seqID NZ_LR962016.1.\n",
      "2024-02-25 13:20:58,947 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961980.1, seqID NZ_LR961980.1.\n",
      "2024-02-25 13:20:58,964 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR961924.1, seqID NZ_LR961924.1.\n",
      "2024-02-25 13:20:58,985 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124960.1, seqID NZ_CP124960.1.\n",
      "2024-02-25 13:20:59,005 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124950.1, seqID NZ_CP124950.1.\n",
      "2024-02-25 13:20:59,026 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124939.1, seqID NZ_CP124939.1.\n",
      "2024-02-25 13:20:59,048 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124935.1, seqID NZ_CP124935.1.\n",
      "2024-02-25 13:20:59,069 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124925.1, seqID NZ_CP124925.1.\n",
      "2024-02-25 13:20:59,089 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124913.1, seqID NZ_CP124913.1.\n",
      "2024-02-25 13:20:59,108 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124905.1, seqID NZ_CP124905.1.\n",
      "2024-02-25 13:20:59,127 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124898.1, seqID NZ_CP124898.1.\n",
      "2024-02-25 13:20:59,147 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124893.1, seqID NZ_CP124893.1.\n",
      "2024-02-25 13:20:59,166 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124890.1, seqID NZ_CP124890.1.\n",
      "2024-02-25 13:20:59,186 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124887.1, seqID NZ_CP124887.1.\n",
      "2024-02-25 13:20:59,206 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP104856.1, seqID NZ_CP104856.1.\n",
      "2024-02-25 13:20:59,224 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116553.1, seqID NZ_CP116553.1.\n",
      "2024-02-25 13:20:59,242 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116557.1, seqID NZ_CP116557.1.\n",
      "2024-02-25 13:20:59,260 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116559.1, seqID NZ_CP116559.1.\n",
      "2024-02-25 13:20:59,278 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP116555.1, seqID NZ_CP116555.1.\n",
      "2024-02-25 13:20:59,297 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110072.1, seqID NZ_CP110072.1.\n",
      "2024-02-25 13:20:59,316 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110071.1, seqID NZ_CP110071.1.\n",
      "2024-02-25 13:20:59,334 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP110053.1, seqID NZ_CP110053.1.\n",
      "2024-02-25 13:20:59,354 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP080759.1, seqID NZ_CP080759.1.\n",
      "2024-02-25 13:20:59,374 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091235.1, seqID NZ_CP091235.1.\n",
      "2024-02-25 13:20:59,392 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP088198.1, seqID NZ_CP088198.1.\n",
      "2024-02-25 13:20:59,411 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097042.1, seqID NZ_CP097042.1.\n",
      "2024-02-25 13:20:59,431 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097034.1, seqID NZ_CP097034.1.\n",
      "2024-02-25 13:20:59,449 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP097006.1, seqID NZ_CP097006.1.\n",
      "2024-02-25 13:20:59,469 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP046247.1, seqID NZ_CP046247.1.\n",
      "2024-02-25 13:20:59,487 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091897.1, seqID NZ_CP091897.1.\n",
      "2024-02-25 13:20:59,505 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091895.1, seqID NZ_CP091895.1.\n",
      "2024-02-25 13:20:59,525 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP091884.1, seqID NZ_CP091884.1.\n",
      "2024-02-25 13:20:59,544 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP086560.1, seqID NZ_CP086560.1.\n",
      "2024-02-25 13:20:59,564 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP078015.1, seqID NZ_CP078015.1.\n",
      "2024-02-25 13:20:59,583 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071176.1, seqID NZ_CP071176.1.\n",
      "2024-02-25 13:20:59,602 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP071172.1, seqID NZ_CP071172.1.\n",
      "2024-02-25 13:20:59,620 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP069185.1, seqID NZ_CP069185.1.\n",
      "2024-02-25 13:20:59,640 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP051005.1, seqID NZ_CP051005.1.\n",
      "2024-02-25 13:20:59,659 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP045045.1, seqID NZ_CP045045.1.\n",
      "2024-02-25 13:20:59,678 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP042216.2, seqID NZ_CP042216.2.\n",
      "2024-02-25 13:20:59,698 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP040898.1, seqID NZ_CP040898.1.\n",
      "2024-02-25 13:20:59,715 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP028724.1, seqID NZ_CP028724.1.\n",
      "2024-02-25 13:20:59,716 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962548.1.\n",
      "2024-02-25 13:20:59,716 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962548.1.\n",
      "2024-02-25 13:20:59,716 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962548.1, seqID LR962548.1.\n",
      "2024-02-25 13:20:59,717 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962517.1.\n",
      "2024-02-25 13:20:59,717 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR962517.1.\n",
      "2024-02-25 13:20:59,717 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain LR962517.1, seqID LR962517.1.\n",
      "2024-02-25 13:20:59,735 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP124900.1, seqID CP124900.1.\n",
      "2024-02-25 13:20:59,751 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962548.1, seqID NZ_LR962548.1.\n",
      "2024-02-25 13:20:59,767 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_LR962517.1, seqID NZ_LR962517.1.\n",
      "2024-02-25 13:20:59,786 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain NZ_CP124900.1, seqID NZ_CP124900.1.\n",
      "2024-02-25 13:20:59,803 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP068242.1, seqID CP068242.1.\n",
      "2024-02-25 13:20:59,823 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain CP013614.1, seqID CP013614.1.\n",
      "2024-02-25 13:20:59,828 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP006683.1, seqID CP006683.1.\n",
      "2024-02-25 13:20:59,835 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP006683.1, seqID NZ_CP006683.1.\n",
      "2024-02-25 13:20:59,840 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP021885.1, seqID AP021885.1.\n",
      "2024-02-25 13:20:59,845 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP018524.1, seqID AP018524.1.\n",
      "2024-02-25 13:20:59,852 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP021885.1, seqID NZ_AP021885.1.\n",
      "2024-02-25 13:20:59,858 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP018524.1, seqID NZ_AP018524.1.\n",
      "2024-02-25 13:20:59,871 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP127160.1, seqID CP127160.1.\n",
      "2024-02-25 13:20:59,883 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP127160.1, seqID NZ_CP127160.1.\n",
      "2024-02-25 13:20:59,893 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP014067.2, seqID CP014067.2.\n",
      "2024-02-25 13:20:59,905 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014067.2, seqID NZ_CP014067.2.\n",
      "2024-02-25 13:20:59,915 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116026.1, seqID CP116026.1.\n",
      "2024-02-25 13:20:59,925 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP053180.1, seqID CP053180.1.\n",
      "2024-02-25 13:20:59,935 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050490.1, seqID CP050490.1.\n",
      "2024-02-25 13:20:59,946 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116026.1, seqID NZ_CP116026.1.\n",
      "2024-02-25 13:20:59,957 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP053180.1, seqID NZ_CP053180.1.\n",
      "2024-02-25 13:20:59,968 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050490.1, seqID NZ_CP050490.1.\n",
      "2024-02-25 13:20:59,977 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP025635.1, seqID AP025635.1.\n",
      "2024-02-25 13:20:59,989 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP025635.1, seqID NZ_AP025635.1.\n",
      "2024-02-25 13:21:00,000 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP119393.1, seqID CP119393.1.\n",
      "2024-02-25 13:21:00,009 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068132.1, seqID CP068132.1.\n",
      "2024-02-25 13:21:00,019 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068126.1, seqID CP068126.1.\n",
      "2024-02-25 13:21:00,031 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP119393.1, seqID NZ_CP119393.1.\n",
      "2024-02-25 13:21:00,042 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068132.1, seqID NZ_CP068132.1.\n",
      "2024-02-25 13:21:00,053 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068126.1, seqID NZ_CP068126.1.\n",
      "2024-02-25 13:21:00,061 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP004856.1, seqID CP004856.1.\n",
      "2024-02-25 13:21:00,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_020995.1, seqID NC_020995.1.\n",
      "2024-02-25 13:21:00,082 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066024.1, seqID CP066024.1.\n",
      "2024-02-25 13:21:00,093 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP046123.1, seqID CP046123.1.\n",
      "2024-02-25 13:21:00,103 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP032739.1, seqID CP032739.1.\n",
      "2024-02-25 13:21:00,114 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066024.1, seqID NZ_CP066024.1.\n",
      "2024-02-25 13:21:00,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP046123.1, seqID NZ_CP046123.1.\n",
      "2024-02-25 13:21:00,137 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP032739.1, seqID NZ_CP032739.1.\n",
      "2024-02-25 13:21:00,147 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023515.1, seqID CP023515.1.\n",
      "2024-02-25 13:21:00,158 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023515.1, seqID NZ_CP023515.1.\n",
      "2024-02-25 13:21:00,169 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP025223.1, seqID CP025223.1.\n",
      "2024-02-25 13:21:00,181 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP025223.1, seqID NZ_CP025223.1.\n",
      "2024-02-25 13:21:00,192 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP119296.1, seqID CP119296.1.\n",
      "2024-02-25 13:21:00,202 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP086066.1, seqID CP086066.1.\n",
      "2024-02-25 13:21:00,212 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068128.1, seqID CP068128.1.\n",
      "2024-02-25 13:21:00,223 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP119296.1, seqID NZ_CP119296.1.\n",
      "2024-02-25 13:21:00,235 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP086066.1, seqID NZ_CP086066.1.\n",
      "2024-02-25 13:21:00,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068128.1, seqID NZ_CP068128.1.\n",
      "2024-02-25 13:21:00,254 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP088203.1, seqID CP088203.1.\n",
      "2024-02-25 13:21:00,263 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP022930.1, seqID CP022930.1.\n",
      "2024-02-25 13:21:00,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP088203.1, seqID NZ_CP088203.1.\n",
      "2024-02-25 13:21:00,282 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP022930.1, seqID NZ_CP022930.1.\n",
      "2024-02-25 13:21:00,291 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116579.1, seqID CP116579.1.\n",
      "2024-02-25 13:21:00,300 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116581.1, seqID CP116581.1.\n",
      "2024-02-25 13:21:00,309 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065535.1, seqID CP065535.1.\n",
      "2024-02-25 13:21:00,317 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP012384.1, seqID CP012384.1.\n",
      "2024-02-25 13:21:00,325 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP012366.1, seqID CP012366.1.\n",
      "2024-02-25 13:21:00,334 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116579.1, seqID NZ_CP116579.1.\n",
      "2024-02-25 13:21:00,344 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116581.1, seqID NZ_CP116581.1.\n",
      "2024-02-25 13:21:00,354 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065535.1, seqID NZ_CP065535.1.\n",
      "2024-02-25 13:21:00,363 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012384.1, seqID NZ_CP012384.1.\n",
      "2024-02-25 13:21:00,372 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012366.1, seqID NZ_CP012366.1.\n",
      "2024-02-25 13:21:00,373 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OV996158.1.\n",
      "2024-02-25 13:21:00,373 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OV996158.1, seqID OV996158.1.\n",
      "2024-02-25 13:21:00,381 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018061.1, seqID CP018061.1.\n",
      "2024-02-25 13:21:00,382 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP022340.1.\n",
      "2024-02-25 13:21:00,382 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP022340.1, seqID CP022340.1.\n",
      "2024-02-25 13:21:00,392 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OV996158.1, seqID NZ_OV996158.1.\n",
      "2024-02-25 13:21:00,401 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP022340.1, seqID NZ_CP022340.1.\n",
      "2024-02-25 13:21:00,411 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018061.1, seqID NZ_CP018061.1.\n",
      "2024-02-25 13:21:00,419 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR134297.1, seqID LR134297.1.\n",
      "2024-02-25 13:21:00,428 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134297.1, seqID NZ_LR134297.1.\n",
      "2024-02-25 13:21:00,438 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040461.1, seqID CP040461.1.\n",
      "2024-02-25 13:21:00,449 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040461.1, seqID NZ_CP040461.1.\n",
      "2024-02-25 13:21:00,458 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP033740.1, seqID CP033740.1.\n",
      "2024-02-25 13:21:00,469 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP033740.1, seqID NZ_CP033740.1.\n",
      "2024-02-25 13:21:00,477 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116510.1, seqID CP116510.1.\n",
      "2024-02-25 13:21:00,487 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP078505.1, seqID CP078505.1.\n",
      "2024-02-25 13:21:00,496 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091204.1, seqID CP091204.1.\n",
      "2024-02-25 13:21:00,506 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050816.1, seqID CP050816.1.\n",
      "2024-02-25 13:21:00,516 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050485.1, seqID CP050485.1.\n",
      "2024-02-25 13:21:00,526 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP046307.1, seqID CP046307.1.\n",
      "2024-02-25 13:21:00,535 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116510.1, seqID NZ_CP116510.1.\n",
      "2024-02-25 13:21:00,546 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP078505.1, seqID NZ_CP078505.1.\n",
      "2024-02-25 13:21:00,557 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091204.1, seqID NZ_CP091204.1.\n",
      "2024-02-25 13:21:00,568 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050816.1, seqID NZ_CP050816.1.\n",
      "2024-02-25 13:21:00,580 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050485.1, seqID NZ_CP050485.1.\n",
      "2024-02-25 13:21:00,591 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP046307.1, seqID NZ_CP046307.1.\n",
      "2024-02-25 13:21:00,601 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP013036.1, seqID AP013036.1.\n",
      "2024-02-25 13:21:00,611 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_022878.1, seqID NC_022878.1.\n",
      "2024-02-25 13:21:00,620 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP083695.1, seqID CP083695.1.\n",
      "2024-02-25 13:21:00,629 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP019810.1, seqID AP019810.1.\n",
      "2024-02-25 13:21:00,638 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP029066.1, seqID CP029066.1.\n",
      "2024-02-25 13:21:00,648 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083695.1, seqID NZ_CP083695.1.\n",
      "2024-02-25 13:21:00,658 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP019810.1, seqID NZ_AP019810.1.\n",
      "2024-02-25 13:21:00,668 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP029066.1, seqID NZ_CP029066.1.\n",
      "2024-02-25 13:21:00,668 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR994474.1.\n",
      "2024-02-25 13:21:00,668 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR994474.1, seqID LR994474.1.\n",
      "2024-02-25 13:21:00,676 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP003504.1, seqID CP003504.1.\n",
      "2024-02-25 13:21:00,685 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_018081.1, seqID NC_018081.1.\n",
      "2024-02-25 13:21:00,694 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP027299.1, seqID AP027299.1.\n",
      "2024-02-25 13:21:00,703 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP122304.1, seqID CP122304.1.\n",
      "2024-02-25 13:21:00,711 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109808.1, seqID CP109808.1.\n",
      "2024-02-25 13:21:00,720 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109807.1, seqID CP109807.1.\n",
      "2024-02-25 13:21:00,729 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109803.1, seqID CP109803.1.\n",
      "2024-02-25 13:21:00,737 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109802.1, seqID CP109802.1.\n",
      "2024-02-25 13:21:00,745 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091201.1, seqID CP091201.1.\n",
      "2024-02-25 13:21:00,753 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP088187.1, seqID CP088187.1.\n",
      "2024-02-25 13:21:00,761 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP088183.1, seqID CP088183.1.\n",
      "2024-02-25 13:21:00,770 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP093926.1, seqID CP093926.1.\n",
      "2024-02-25 13:21:00,778 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072891.1, seqID CP072891.1.\n",
      "2024-02-25 13:21:00,786 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068118.1, seqID CP068118.1.\n",
      "2024-02-25 13:21:00,794 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068122.1, seqID CP068122.1.\n",
      "2024-02-25 13:21:00,801 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065992.1, seqID CP065992.1.\n",
      "2024-02-25 13:21:00,809 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP055232.1, seqID CP055232.1.\n",
      "2024-02-25 13:21:00,817 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP055231.1, seqID CP055231.1.\n",
      "2024-02-25 13:21:00,825 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP055230.1, seqID CP055230.1.\n",
      "2024-02-25 13:21:00,833 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP055229.1, seqID CP055229.1.\n",
      "2024-02-25 13:21:00,841 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP037955.1, seqID CP037955.1.\n",
      "2024-02-25 13:21:00,848 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023011.2, seqID CP023011.2.\n",
      "2024-02-25 13:21:00,856 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP015516.1, seqID CP015516.1.\n",
      "2024-02-25 13:21:00,866 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP027299.1, seqID NZ_AP027299.1.\n",
      "2024-02-25 13:21:00,875 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP122304.1, seqID NZ_CP122304.1.\n",
      "2024-02-25 13:21:00,885 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109808.1, seqID NZ_CP109808.1.\n",
      "2024-02-25 13:21:00,894 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109807.1, seqID NZ_CP109807.1.\n",
      "2024-02-25 13:21:00,903 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109803.1, seqID NZ_CP109803.1.\n",
      "2024-02-25 13:21:00,912 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109802.1, seqID NZ_CP109802.1.\n",
      "2024-02-25 13:21:00,921 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091201.1, seqID NZ_CP091201.1.\n",
      "2024-02-25 13:21:00,931 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP088187.1, seqID NZ_CP088187.1.\n",
      "2024-02-25 13:21:00,940 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP088183.1, seqID NZ_CP088183.1.\n",
      "2024-02-25 13:21:00,950 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP093926.1, seqID NZ_CP093926.1.\n",
      "2024-02-25 13:21:00,959 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072891.1, seqID NZ_CP072891.1.\n",
      "2024-02-25 13:21:00,968 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068118.1, seqID NZ_CP068118.1.\n",
      "2024-02-25 13:21:00,978 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068122.1, seqID NZ_CP068122.1.\n",
      "2024-02-25 13:21:00,987 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065992.1, seqID NZ_CP065992.1.\n",
      "2024-02-25 13:21:00,996 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP055232.1, seqID NZ_CP055232.1.\n",
      "2024-02-25 13:21:01,005 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP055231.1, seqID NZ_CP055231.1.\n",
      "2024-02-25 13:21:01,015 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP055230.1, seqID NZ_CP055230.1.\n",
      "2024-02-25 13:21:01,024 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP055229.1, seqID NZ_CP055229.1.\n",
      "2024-02-25 13:21:01,033 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP037955.1, seqID NZ_CP037955.1.\n",
      "2024-02-25 13:21:01,042 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP015516.1, seqID NZ_CP015516.1.\n",
      "2024-02-25 13:21:01,051 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023011.2, seqID NZ_CP023011.2.\n",
      "2024-02-25 13:21:01,059 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023074.1, seqID CP023074.1.\n",
      "2024-02-25 13:21:01,068 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023074.1, seqID NZ_CP023074.1.\n",
      "2024-02-25 13:21:01,069 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP042839.1.\n",
      "2024-02-25 13:21:01,069 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP042839.1, seqID CP042839.1.\n",
      "2024-02-25 13:21:01,077 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP042839.1, seqID NZ_CP042839.1.\n",
      "2024-02-25 13:21:01,086 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP082267.1, seqID CP082267.1.\n",
      "2024-02-25 13:21:01,094 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065211.1, seqID CP065211.1.\n",
      "2024-02-25 13:21:01,103 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP082267.1, seqID NZ_CP082267.1.\n",
      "2024-02-25 13:21:01,112 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065211.1, seqID NZ_CP065211.1.\n",
      "2024-02-25 13:21:01,121 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP006620.1, seqID CP006620.1.\n",
      "2024-02-25 13:21:01,131 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_021994.1, seqID NC_021994.1.\n",
      "2024-02-25 13:21:01,139 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP003351.1, seqID CP003351.1.\n",
      "2024-02-25 13:21:01,148 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_017022.1, seqID NC_017022.1.\n",
      "2024-02-25 13:21:01,156 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP004063.1, seqID CP004063.1.\n",
      "2024-02-25 13:21:01,164 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_020207.1, seqID NC_020207.1.\n",
      "2024-02-25 13:21:01,165 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU016036.1.\n",
      "2024-02-25 13:21:01,165 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU016036.1, seqID OU016036.1.\n",
      "2024-02-25 13:21:01,166 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU016027.1.\n",
      "2024-02-25 13:21:01,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU016027.1, seqID OU016027.1.\n",
      "2024-02-25 13:21:01,166 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU015946.1.\n",
      "2024-02-25 13:21:01,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU015946.1, seqID OU015946.1.\n",
      "2024-02-25 13:21:01,167 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU015708.1.\n",
      "2024-02-25 13:21:01,167 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU015708.1, seqID OU015708.1.\n",
      "2024-02-25 13:21:01,167 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU015350.1.\n",
      "2024-02-25 13:21:01,168 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU015350.1, seqID OU015350.1.\n",
      "2024-02-25 13:21:01,168 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for OU015345.1.\n",
      "2024-02-25 13:21:01,168 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain OU015345.1, seqID OU015345.1.\n",
      "2024-02-25 13:21:01,176 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR215968.1, seqID LR215968.1.\n",
      "2024-02-25 13:21:01,176 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135482.1.\n",
      "2024-02-25 13:21:01,176 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135482.1, seqID LR135482.1.\n",
      "2024-02-25 13:21:01,177 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135475.1.\n",
      "2024-02-25 13:21:01,177 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135475.1, seqID LR135475.1.\n",
      "2024-02-25 13:21:01,178 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135488.1.\n",
      "2024-02-25 13:21:01,178 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135488.1, seqID LR135488.1.\n",
      "2024-02-25 13:21:01,178 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135474.1.\n",
      "2024-02-25 13:21:01,178 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135474.1, seqID LR135474.1.\n",
      "2024-02-25 13:21:01,179 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135443.1.\n",
      "2024-02-25 13:21:01,179 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135443.1, seqID LR135443.1.\n",
      "2024-02-25 13:21:01,179 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135435.1.\n",
      "2024-02-25 13:21:01,179 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135435.1, seqID LR135435.1.\n",
      "2024-02-25 13:21:01,180 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135419.1.\n",
      "2024-02-25 13:21:01,180 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135419.1, seqID LR135419.1.\n",
      "2024-02-25 13:21:01,180 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135428.1.\n",
      "2024-02-25 13:21:01,180 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135428.1, seqID LR135428.1.\n",
      "2024-02-25 13:21:01,181 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135408.1.\n",
      "2024-02-25 13:21:01,181 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135408.1, seqID LR135408.1.\n",
      "2024-02-25 13:21:01,182 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135414.1.\n",
      "2024-02-25 13:21:01,182 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135414.1, seqID LR135414.1.\n",
      "2024-02-25 13:21:01,182 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135394.1.\n",
      "2024-02-25 13:21:01,182 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135394.1, seqID LR135394.1.\n",
      "2024-02-25 13:21:01,183 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135401.1.\n",
      "2024-02-25 13:21:01,183 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135401.1, seqID LR135401.1.\n",
      "2024-02-25 13:21:01,183 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135357.1.\n",
      "2024-02-25 13:21:01,183 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135357.1, seqID LR135357.1.\n",
      "2024-02-25 13:21:01,184 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135351.1.\n",
      "2024-02-25 13:21:01,184 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135351.1, seqID LR135351.1.\n",
      "2024-02-25 13:21:01,184 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135364.1.\n",
      "2024-02-25 13:21:01,185 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135364.1, seqID LR135364.1.\n",
      "2024-02-25 13:21:01,185 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135372.1.\n",
      "2024-02-25 13:21:01,185 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135372.1, seqID LR135372.1.\n",
      "2024-02-25 13:21:01,186 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135384.1.\n",
      "2024-02-25 13:21:01,186 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135384.1, seqID LR135384.1.\n",
      "2024-02-25 13:21:01,186 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135344.1.\n",
      "2024-02-25 13:21:01,186 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135344.1, seqID LR135344.1.\n",
      "2024-02-25 13:21:01,187 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135317.1.\n",
      "2024-02-25 13:21:01,187 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135317.1, seqID LR135317.1.\n",
      "2024-02-25 13:21:01,187 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135324.1.\n",
      "2024-02-25 13:21:01,188 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135324.1, seqID LR135324.1.\n",
      "2024-02-25 13:21:01,188 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135339.1.\n",
      "2024-02-25 13:21:01,188 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135339.1, seqID LR135339.1.\n",
      "2024-02-25 13:21:01,189 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135307.1.\n",
      "2024-02-25 13:21:01,189 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135307.1, seqID LR135307.1.\n",
      "2024-02-25 13:21:01,189 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135297.1.\n",
      "2024-02-25 13:21:01,189 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135297.1, seqID LR135297.1.\n",
      "2024-02-25 13:21:01,190 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135293.1.\n",
      "2024-02-25 13:21:01,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135293.1, seqID LR135293.1.\n",
      "2024-02-25 13:21:01,190 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135254.1.\n",
      "2024-02-25 13:21:01,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135254.1, seqID LR135254.1.\n",
      "2024-02-25 13:21:01,191 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135243.1.\n",
      "2024-02-25 13:21:01,191 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135243.1, seqID LR135243.1.\n",
      "2024-02-25 13:21:01,192 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135258.1.\n",
      "2024-02-25 13:21:01,192 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135258.1, seqID LR135258.1.\n",
      "2024-02-25 13:21:01,192 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135278.1.\n",
      "2024-02-25 13:21:01,192 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135278.1, seqID LR135278.1.\n",
      "2024-02-25 13:21:01,193 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135219.1.\n",
      "2024-02-25 13:21:01,193 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135219.1, seqID LR135219.1.\n",
      "2024-02-25 13:21:01,193 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135235.1.\n",
      "2024-02-25 13:21:01,193 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135235.1, seqID LR135235.1.\n",
      "2024-02-25 13:21:01,194 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135191.1.\n",
      "2024-02-25 13:21:01,194 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135191.1, seqID LR135191.1.\n",
      "2024-02-25 13:21:01,194 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135179.1.\n",
      "2024-02-25 13:21:01,194 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135179.1, seqID LR135179.1.\n",
      "2024-02-25 13:21:01,195 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135174.1.\n",
      "2024-02-25 13:21:01,195 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135174.1, seqID LR135174.1.\n",
      "2024-02-25 13:21:01,196 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135169.1.\n",
      "2024-02-25 13:21:01,196 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135169.1, seqID LR135169.1.\n",
      "2024-02-25 13:21:01,203 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR134337.1, seqID LR134337.1.\n",
      "2024-02-25 13:21:01,204 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR134105.1.\n",
      "2024-02-25 13:21:01,204 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR134105.1, seqID LR134105.1.\n",
      "2024-02-25 13:21:01,204 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR134095.1.\n",
      "2024-02-25 13:21:01,204 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR134095.1, seqID LR134095.1.\n",
      "2024-02-25 13:21:01,205 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR132067.1.\n",
      "2024-02-25 13:21:01,205 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR132067.1, seqID LR132067.1.\n",
      "2024-02-25 13:21:01,206 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LT603678.1.\n",
      "2024-02-25 13:21:01,206 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LT603678.1, seqID LT603678.1.\n",
      "2024-02-25 13:21:01,206 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LT598663.1.\n",
      "2024-02-25 13:21:01,206 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LT598663.1, seqID LT598663.1.\n",
      "2024-02-25 13:21:01,213 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LN999844.1, seqID LN999844.1.\n",
      "2024-02-25 13:21:01,221 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LN999987.1, seqID LN999987.1.\n",
      "2024-02-25 13:21:01,230 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP132226.1, seqID CP132226.1.\n",
      "2024-02-25 13:21:01,230 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131532.1.\n",
      "2024-02-25 13:21:01,230 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131532.1, seqID CP131532.1.\n",
      "2024-02-25 13:21:01,231 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131530.1.\n",
      "2024-02-25 13:21:01,231 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131530.1, seqID CP131530.1.\n",
      "2024-02-25 13:21:01,232 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131512.1.\n",
      "2024-02-25 13:21:01,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131512.1, seqID CP131512.1.\n",
      "2024-02-25 13:21:01,232 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131506.1.\n",
      "2024-02-25 13:21:01,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131506.1, seqID CP131506.1.\n",
      "2024-02-25 13:21:01,233 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131504.1.\n",
      "2024-02-25 13:21:01,233 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131504.1, seqID CP131504.1.\n",
      "2024-02-25 13:21:01,233 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131496.1.\n",
      "2024-02-25 13:21:01,233 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131496.1, seqID CP131496.1.\n",
      "2024-02-25 13:21:01,234 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131508.1.\n",
      "2024-02-25 13:21:01,234 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131508.1, seqID CP131508.1.\n",
      "2024-02-25 13:21:01,234 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131510.1.\n",
      "2024-02-25 13:21:01,234 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131510.1, seqID CP131510.1.\n",
      "2024-02-25 13:21:01,235 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131498.1.\n",
      "2024-02-25 13:21:01,235 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131498.1, seqID CP131498.1.\n",
      "2024-02-25 13:21:01,245 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131610.1, seqID CP131610.1.\n",
      "2024-02-25 13:21:01,245 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131494.1.\n",
      "2024-02-25 13:21:01,245 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131494.1, seqID CP131494.1.\n",
      "2024-02-25 13:21:01,246 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131492.1.\n",
      "2024-02-25 13:21:01,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131492.1, seqID CP131492.1.\n",
      "2024-02-25 13:21:01,246 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131528.1.\n",
      "2024-02-25 13:21:01,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131528.1, seqID CP131528.1.\n",
      "2024-02-25 13:21:01,247 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131522.1.\n",
      "2024-02-25 13:21:01,247 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131522.1, seqID CP131522.1.\n",
      "2024-02-25 13:21:01,247 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131518.1.\n",
      "2024-02-25 13:21:01,248 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131518.1, seqID CP131518.1.\n",
      "2024-02-25 13:21:01,248 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131520.1.\n",
      "2024-02-25 13:21:01,248 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131520.1, seqID CP131520.1.\n",
      "2024-02-25 13:21:01,249 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131526.1.\n",
      "2024-02-25 13:21:01,249 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131526.1, seqID CP131526.1.\n",
      "2024-02-25 13:21:01,249 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131524.1.\n",
      "2024-02-25 13:21:01,249 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131524.1, seqID CP131524.1.\n",
      "2024-02-25 13:21:01,250 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131514.1.\n",
      "2024-02-25 13:21:01,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131514.1, seqID CP131514.1.\n",
      "2024-02-25 13:21:01,250 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131516.1.\n",
      "2024-02-25 13:21:01,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131516.1, seqID CP131516.1.\n",
      "2024-02-25 13:21:01,251 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131502.1.\n",
      "2024-02-25 13:21:01,251 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131502.1, seqID CP131502.1.\n",
      "2024-02-25 13:21:01,252 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131500.1.\n",
      "2024-02-25 13:21:01,252 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131500.1, seqID CP131500.1.\n",
      "2024-02-25 13:21:01,252 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131490.1.\n",
      "2024-02-25 13:21:01,252 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131490.1, seqID CP131490.1.\n",
      "2024-02-25 13:21:01,253 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131489.1.\n",
      "2024-02-25 13:21:01,253 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131489.1, seqID CP131489.1.\n",
      "2024-02-25 13:21:01,254 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131123.1.\n",
      "2024-02-25 13:21:01,254 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131123.1, seqID CP131123.1.\n",
      "2024-02-25 13:21:01,254 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131125.1.\n",
      "2024-02-25 13:21:01,254 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131125.1, seqID CP131125.1.\n",
      "2024-02-25 13:21:01,255 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131121.1.\n",
      "2024-02-25 13:21:01,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131121.1, seqID CP131121.1.\n",
      "2024-02-25 13:21:01,255 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131119.1.\n",
      "2024-02-25 13:21:01,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131119.1, seqID CP131119.1.\n",
      "2024-02-25 13:21:01,256 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131107.1.\n",
      "2024-02-25 13:21:01,256 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131107.1, seqID CP131107.1.\n",
      "2024-02-25 13:21:01,257 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131117.1.\n",
      "2024-02-25 13:21:01,257 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131117.1, seqID CP131117.1.\n",
      "2024-02-25 13:21:01,257 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131115.1.\n",
      "2024-02-25 13:21:01,257 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131115.1, seqID CP131115.1.\n",
      "2024-02-25 13:21:01,258 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131113.1.\n",
      "2024-02-25 13:21:01,258 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131113.1, seqID CP131113.1.\n",
      "2024-02-25 13:21:01,258 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131109.1.\n",
      "2024-02-25 13:21:01,258 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131109.1, seqID CP131109.1.\n",
      "2024-02-25 13:21:01,259 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131111.1.\n",
      "2024-02-25 13:21:01,259 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131111.1, seqID CP131111.1.\n",
      "2024-02-25 13:21:01,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131105.1.\n",
      "2024-02-25 13:21:01,260 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131105.1, seqID CP131105.1.\n",
      "2024-02-25 13:21:01,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131103.1.\n",
      "2024-02-25 13:21:01,260 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131103.1, seqID CP131103.1.\n",
      "2024-02-25 13:21:01,261 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131099.1.\n",
      "2024-02-25 13:21:01,261 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131099.1, seqID CP131099.1.\n",
      "2024-02-25 13:21:01,261 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131097.1.\n",
      "2024-02-25 13:21:01,261 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131097.1, seqID CP131097.1.\n",
      "2024-02-25 13:21:01,262 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131101.1.\n",
      "2024-02-25 13:21:01,262 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131101.1, seqID CP131101.1.\n",
      "2024-02-25 13:21:01,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131093.1.\n",
      "2024-02-25 13:21:01,263 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131093.1, seqID CP131093.1.\n",
      "2024-02-25 13:21:01,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131095.1.\n",
      "2024-02-25 13:21:01,263 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131095.1, seqID CP131095.1.\n",
      "2024-02-25 13:21:01,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131069.1.\n",
      "2024-02-25 13:21:01,264 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131069.1, seqID CP131069.1.\n",
      "2024-02-25 13:21:01,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131077.1.\n",
      "2024-02-25 13:21:01,264 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131077.1, seqID CP131077.1.\n",
      "2024-02-25 13:21:01,265 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131075.1.\n",
      "2024-02-25 13:21:01,265 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131075.1, seqID CP131075.1.\n",
      "2024-02-25 13:21:01,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131071.1.\n",
      "2024-02-25 13:21:01,266 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131071.1, seqID CP131071.1.\n",
      "2024-02-25 13:21:01,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131087.1.\n",
      "2024-02-25 13:21:01,266 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131087.1, seqID CP131087.1.\n",
      "2024-02-25 13:21:01,267 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131089.1.\n",
      "2024-02-25 13:21:01,267 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131089.1, seqID CP131089.1.\n",
      "2024-02-25 13:21:01,267 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131079.1.\n",
      "2024-02-25 13:21:01,267 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131079.1, seqID CP131079.1.\n",
      "2024-02-25 13:21:01,268 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131081.1.\n",
      "2024-02-25 13:21:01,268 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131081.1, seqID CP131081.1.\n",
      "2024-02-25 13:21:01,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131091.1.\n",
      "2024-02-25 13:21:01,269 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131091.1, seqID CP131091.1.\n",
      "2024-02-25 13:21:01,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131085.1.\n",
      "2024-02-25 13:21:01,269 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131085.1, seqID CP131085.1.\n",
      "2024-02-25 13:21:01,270 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131083.1.\n",
      "2024-02-25 13:21:01,270 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131083.1, seqID CP131083.1.\n",
      "2024-02-25 13:21:01,270 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131073.1.\n",
      "2024-02-25 13:21:01,270 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131073.1, seqID CP131073.1.\n",
      "2024-02-25 13:21:01,271 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131067.1.\n",
      "2024-02-25 13:21:01,271 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131067.1, seqID CP131067.1.\n",
      "2024-02-25 13:21:01,272 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131063.1.\n",
      "2024-02-25 13:21:01,272 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131063.1, seqID CP131063.1.\n",
      "2024-02-25 13:21:01,272 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131065.1.\n",
      "2024-02-25 13:21:01,272 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131065.1, seqID CP131065.1.\n",
      "2024-02-25 13:21:01,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131054.1.\n",
      "2024-02-25 13:21:01,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131054.1, seqID CP131054.1.\n",
      "2024-02-25 13:21:01,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131052.1.\n",
      "2024-02-25 13:21:01,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131052.1, seqID CP131052.1.\n",
      "2024-02-25 13:21:01,274 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131050.1.\n",
      "2024-02-25 13:21:01,274 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131050.1, seqID CP131050.1.\n",
      "2024-02-25 13:21:01,275 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131048.1.\n",
      "2024-02-25 13:21:01,275 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131048.1, seqID CP131048.1.\n",
      "2024-02-25 13:21:01,275 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131046.1.\n",
      "2024-02-25 13:21:01,275 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131046.1, seqID CP131046.1.\n",
      "2024-02-25 13:21:01,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131044.1.\n",
      "2024-02-25 13:21:01,276 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131044.1, seqID CP131044.1.\n",
      "2024-02-25 13:21:01,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131042.1.\n",
      "2024-02-25 13:21:01,276 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131042.1, seqID CP131042.1.\n",
      "2024-02-25 13:21:01,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131040.1.\n",
      "2024-02-25 13:21:01,277 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131040.1, seqID CP131040.1.\n",
      "2024-02-25 13:21:01,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131020.1.\n",
      "2024-02-25 13:21:01,277 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131020.1, seqID CP131020.1.\n",
      "2024-02-25 13:21:01,278 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131016.1.\n",
      "2024-02-25 13:21:01,278 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131016.1, seqID CP131016.1.\n",
      "2024-02-25 13:21:01,279 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131018.1.\n",
      "2024-02-25 13:21:01,279 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131018.1, seqID CP131018.1.\n",
      "2024-02-25 13:21:01,279 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131014.1.\n",
      "2024-02-25 13:21:01,279 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131014.1, seqID CP131014.1.\n",
      "2024-02-25 13:21:01,280 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131012.1.\n",
      "2024-02-25 13:21:01,280 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131012.1, seqID CP131012.1.\n",
      "2024-02-25 13:21:01,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131010.1.\n",
      "2024-02-25 13:21:01,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131010.1, seqID CP131010.1.\n",
      "2024-02-25 13:21:01,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131027.1.\n",
      "2024-02-25 13:21:01,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131027.1, seqID CP131027.1.\n",
      "2024-02-25 13:21:01,282 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131031.1.\n",
      "2024-02-25 13:21:01,282 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131031.1, seqID CP131031.1.\n",
      "2024-02-25 13:21:01,282 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131025.1.\n",
      "2024-02-25 13:21:01,282 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131025.1, seqID CP131025.1.\n",
      "2024-02-25 13:21:01,283 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131035.1.\n",
      "2024-02-25 13:21:01,283 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131035.1, seqID CP131035.1.\n",
      "2024-02-25 13:21:01,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131033.1.\n",
      "2024-02-25 13:21:01,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131033.1, seqID CP131033.1.\n",
      "2024-02-25 13:21:01,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131037.1.\n",
      "2024-02-25 13:21:01,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131037.1, seqID CP131037.1.\n",
      "2024-02-25 13:21:01,285 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131029.1.\n",
      "2024-02-25 13:21:01,285 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131029.1, seqID CP131029.1.\n",
      "2024-02-25 13:21:01,286 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131006.1.\n",
      "2024-02-25 13:21:01,286 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131006.1, seqID CP131006.1.\n",
      "2024-02-25 13:21:01,286 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131000.1.\n",
      "2024-02-25 13:21:01,286 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131000.1, seqID CP131000.1.\n",
      "2024-02-25 13:21:01,287 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130998.1.\n",
      "2024-02-25 13:21:01,287 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130998.1, seqID CP130998.1.\n",
      "2024-02-25 13:21:01,287 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131004.1.\n",
      "2024-02-25 13:21:01,287 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131004.1, seqID CP131004.1.\n",
      "2024-02-25 13:21:01,288 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130996.1.\n",
      "2024-02-25 13:21:01,288 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130996.1, seqID CP130996.1.\n",
      "2024-02-25 13:21:01,289 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131002.1.\n",
      "2024-02-25 13:21:01,289 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131002.1, seqID CP131002.1.\n",
      "2024-02-25 13:21:01,289 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130992.1.\n",
      "2024-02-25 13:21:01,289 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130992.1, seqID CP130992.1.\n",
      "2024-02-25 13:21:01,290 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130994.1.\n",
      "2024-02-25 13:21:01,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130994.1, seqID CP130994.1.\n",
      "2024-02-25 13:21:01,290 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP131024.1.\n",
      "2024-02-25 13:21:01,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP131024.1, seqID CP131024.1.\n",
      "2024-02-25 13:21:01,291 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130914.1.\n",
      "2024-02-25 13:21:01,291 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130914.1, seqID CP130914.1.\n",
      "2024-02-25 13:21:01,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130912.1.\n",
      "2024-02-25 13:21:01,292 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130912.1, seqID CP130912.1.\n",
      "2024-02-25 13:21:01,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130908.1.\n",
      "2024-02-25 13:21:01,292 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130908.1, seqID CP130908.1.\n",
      "2024-02-25 13:21:01,293 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130910.1.\n",
      "2024-02-25 13:21:01,293 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130910.1, seqID CP130910.1.\n",
      "2024-02-25 13:21:01,293 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130906.1.\n",
      "2024-02-25 13:21:01,294 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130906.1, seqID CP130906.1.\n",
      "2024-02-25 13:21:01,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130883.1.\n",
      "2024-02-25 13:21:01,294 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130883.1, seqID CP130883.1.\n",
      "2024-02-25 13:21:01,295 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130879.1.\n",
      "2024-02-25 13:21:01,295 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130879.1, seqID CP130879.1.\n",
      "2024-02-25 13:21:01,295 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130875.1.\n",
      "2024-02-25 13:21:01,295 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130875.1, seqID CP130875.1.\n",
      "2024-02-25 13:21:01,296 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130877.1.\n",
      "2024-02-25 13:21:01,296 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130877.1, seqID CP130877.1.\n",
      "2024-02-25 13:21:01,296 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130881.1.\n",
      "2024-02-25 13:21:01,296 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130881.1, seqID CP130881.1.\n",
      "2024-02-25 13:21:01,297 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130882.1.\n",
      "2024-02-25 13:21:01,297 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130882.1, seqID CP130882.1.\n",
      "2024-02-25 13:21:01,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130849.1.\n",
      "2024-02-25 13:21:01,298 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130849.1, seqID CP130849.1.\n",
      "2024-02-25 13:21:01,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130833.1.\n",
      "2024-02-25 13:21:01,298 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130833.1, seqID CP130833.1.\n",
      "2024-02-25 13:21:01,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130841.1.\n",
      "2024-02-25 13:21:01,299 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130841.1, seqID CP130841.1.\n",
      "2024-02-25 13:21:01,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130837.1.\n",
      "2024-02-25 13:21:01,299 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130837.1, seqID CP130837.1.\n",
      "2024-02-25 13:21:01,300 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130843.1.\n",
      "2024-02-25 13:21:01,300 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130843.1, seqID CP130843.1.\n",
      "2024-02-25 13:21:01,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130835.1.\n",
      "2024-02-25 13:21:01,301 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130835.1, seqID CP130835.1.\n",
      "2024-02-25 13:21:01,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130839.1.\n",
      "2024-02-25 13:21:01,301 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130839.1, seqID CP130839.1.\n",
      "2024-02-25 13:21:01,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130845.1.\n",
      "2024-02-25 13:21:01,302 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130845.1, seqID CP130845.1.\n",
      "2024-02-25 13:21:01,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130847.1.\n",
      "2024-02-25 13:21:01,302 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130847.1, seqID CP130847.1.\n",
      "2024-02-25 13:21:01,303 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130851.1.\n",
      "2024-02-25 13:21:01,303 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130851.1, seqID CP130851.1.\n",
      "2024-02-25 13:21:01,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130861.1.\n",
      "2024-02-25 13:21:01,304 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130861.1, seqID CP130861.1.\n",
      "2024-02-25 13:21:01,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130859.1.\n",
      "2024-02-25 13:21:01,304 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130859.1, seqID CP130859.1.\n",
      "2024-02-25 13:21:01,305 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130865.1.\n",
      "2024-02-25 13:21:01,305 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130865.1, seqID CP130865.1.\n",
      "2024-02-25 13:21:01,305 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130863.1.\n",
      "2024-02-25 13:21:01,305 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130863.1, seqID CP130863.1.\n",
      "2024-02-25 13:21:01,306 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130867.1.\n",
      "2024-02-25 13:21:01,306 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130867.1, seqID CP130867.1.\n",
      "2024-02-25 13:21:01,307 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130869.1.\n",
      "2024-02-25 13:21:01,307 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130869.1, seqID CP130869.1.\n",
      "2024-02-25 13:21:01,307 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130873.1.\n",
      "2024-02-25 13:21:01,307 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130873.1, seqID CP130873.1.\n",
      "2024-02-25 13:21:01,308 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130871.1.\n",
      "2024-02-25 13:21:01,308 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130871.1, seqID CP130871.1.\n",
      "2024-02-25 13:21:01,308 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130827.1.\n",
      "2024-02-25 13:21:01,308 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130827.1, seqID CP130827.1.\n",
      "2024-02-25 13:21:01,309 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130825.1.\n",
      "2024-02-25 13:21:01,309 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130825.1, seqID CP130825.1.\n",
      "2024-02-25 13:21:01,310 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP130829.1.\n",
      "2024-02-25 13:21:01,310 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP130829.1, seqID CP130829.1.\n",
      "2024-02-25 13:21:01,310 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP128188.1.\n",
      "2024-02-25 13:21:01,310 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP128188.1, seqID CP128188.1.\n",
      "2024-02-25 13:21:01,318 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP027294.1, seqID AP027294.1.\n",
      "2024-02-25 13:21:01,327 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109758.1, seqID CP109758.1.\n",
      "2024-02-25 13:21:01,336 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP119150.1, seqID CP119150.1.\n",
      "2024-02-25 13:21:01,337 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP118955.1.\n",
      "2024-02-25 13:21:01,337 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP118955.1, seqID CP118955.1.\n",
      "2024-02-25 13:21:01,346 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP118548.1, seqID CP118548.1.\n",
      "2024-02-25 13:21:01,354 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116528.1, seqID CP116528.1.\n",
      "2024-02-25 13:21:01,363 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116523.1, seqID CP116523.1.\n",
      "2024-02-25 13:21:01,372 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116520.1, seqID CP116520.1.\n",
      "2024-02-25 13:21:01,380 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116517.1, seqID CP116517.1.\n",
      "2024-02-25 13:21:01,389 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116532.1, seqID CP116532.1.\n",
      "2024-02-25 13:21:01,398 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116514.1, seqID CP116514.1.\n",
      "2024-02-25 13:21:01,406 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116023.1, seqID CP116023.1.\n",
      "2024-02-25 13:21:01,414 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP115812.1, seqID CP115812.1.\n",
      "2024-02-25 13:21:01,423 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP114617.1, seqID CP114617.1.\n",
      "2024-02-25 13:21:01,432 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP112862.1, seqID CP112862.1.\n",
      "2024-02-25 13:21:01,441 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026666.1, seqID AP026666.1.\n",
      "2024-02-25 13:21:01,450 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026660.1, seqID AP026660.1.\n",
      "2024-02-25 13:21:01,458 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026655.1, seqID AP026655.1.\n",
      "2024-02-25 13:21:01,467 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026650.1, seqID AP026650.1.\n",
      "2024-02-25 13:21:01,476 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026641.1, seqID AP026641.1.\n",
      "2024-02-25 13:21:01,485 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026632.1, seqID AP026632.1.\n",
      "2024-02-25 13:21:01,495 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026628.1, seqID AP026628.1.\n",
      "2024-02-25 13:21:01,504 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026617.1, seqID AP026617.1.\n",
      "2024-02-25 13:21:01,513 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026604.1, seqID AP026604.1.\n",
      "2024-02-25 13:21:01,521 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026601.1, seqID AP026601.1.\n",
      "2024-02-25 13:21:01,531 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026593.1, seqID AP026593.1.\n",
      "2024-02-25 13:21:01,540 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026586.1, seqID AP026586.1.\n",
      "2024-02-25 13:21:01,550 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026579.1, seqID AP026579.1.\n",
      "2024-02-25 13:21:01,559 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026573.1, seqID AP026573.1.\n",
      "2024-02-25 13:21:01,568 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026566.1, seqID AP026566.1.\n",
      "2024-02-25 13:21:01,576 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026772.1, seqID AP026772.1.\n",
      "2024-02-25 13:21:01,585 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109798.1, seqID CP109798.1.\n",
      "2024-02-25 13:21:01,593 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain JAOXXZ010000001.1, seqID JAOXXZ010000001.1.\n",
      "2024-02-25 13:21:01,602 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain JAOXXY010000002.1, seqID JAOXXY010000002.1.\n",
      "2024-02-25 13:21:01,610 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain JAOXXX010000002.1, seqID JAOXXX010000002.1.\n",
      "2024-02-25 13:21:01,618 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP102534.1, seqID CP102534.1.\n",
      "2024-02-25 13:21:01,627 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP058891.1, seqID CP058891.1.\n",
      "2024-02-25 13:21:01,635 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091224.1, seqID CP091224.1.\n",
      "2024-02-25 13:21:01,642 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091217.1, seqID CP091217.1.\n",
      "2024-02-25 13:21:01,650 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091213.1, seqID CP091213.1.\n",
      "2024-02-25 13:21:01,657 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091210.1, seqID CP091210.1.\n",
      "2024-02-25 13:21:01,665 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091206.1, seqID CP091206.1.\n",
      "2024-02-25 13:21:01,673 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP088191.1, seqID CP088191.1.\n",
      "2024-02-25 13:21:01,681 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097022.1, seqID CP097022.1.\n",
      "2024-02-25 13:21:01,690 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097014.1, seqID CP097014.1.\n",
      "2024-02-25 13:21:01,699 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP096219.1, seqID CP096219.1.\n",
      "2024-02-25 13:21:01,708 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP096211.1, seqID CP096211.1.\n",
      "2024-02-25 13:21:01,717 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP110360.1, seqID CP110360.1.\n",
      "2024-02-25 13:21:01,725 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP110294.1, seqID CP110294.1.\n",
      "2024-02-25 13:21:01,734 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063633.1, seqID CP063633.1.\n",
      "2024-02-25 13:21:01,743 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063626.1, seqID CP063626.1.\n",
      "2024-02-25 13:21:01,751 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063618.1, seqID CP063618.1.\n",
      "2024-02-25 13:21:01,760 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063611.1, seqID CP063611.1.\n",
      "2024-02-25 13:21:01,768 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063647.1, seqID CP063647.1.\n",
      "2024-02-25 13:21:01,777 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063604.1, seqID CP063604.1.\n",
      "2024-02-25 13:21:01,786 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063598.1, seqID CP063598.1.\n",
      "2024-02-25 13:21:01,794 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063593.1, seqID CP063593.1.\n",
      "2024-02-25 13:21:01,802 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063586.1, seqID CP063586.1.\n",
      "2024-02-25 13:21:01,810 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063574.1, seqID CP063574.1.\n",
      "2024-02-25 13:21:01,818 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063568.1, seqID CP063568.1.\n",
      "2024-02-25 13:21:01,827 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063562.1, seqID CP063562.1.\n",
      "2024-02-25 13:21:01,836 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063555.1, seqID CP063555.1.\n",
      "2024-02-25 13:21:01,844 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063549.1, seqID CP063549.1.\n",
      "2024-02-25 13:21:01,853 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063537.1, seqID CP063537.1.\n",
      "2024-02-25 13:21:01,862 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063283.1, seqID CP063283.1.\n",
      "2024-02-25 13:21:01,871 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP094230.1, seqID CP094230.1.\n",
      "2024-02-25 13:21:01,880 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP093947.1, seqID CP093947.1.\n",
      "2024-02-25 13:21:01,889 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP093953.1, seqID CP093953.1.\n",
      "2024-02-25 13:21:01,898 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP093941.1, seqID CP093941.1.\n",
      "2024-02-25 13:21:01,907 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP092569.1, seqID CP092569.1.\n",
      "2024-02-25 13:21:01,915 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP076483.1, seqID CP076483.1.\n",
      "2024-02-25 13:21:01,923 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP076477.1, seqID CP076477.1.\n",
      "2024-02-25 13:21:01,932 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP076471.1, seqID CP076471.1.\n",
      "2024-02-25 13:21:01,940 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091577.1, seqID CP091577.1.\n",
      "2024-02-25 13:21:01,949 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091301.1, seqID CP091301.1.\n",
      "2024-02-25 13:21:01,956 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP091100.1, seqID CP091100.1.\n",
      "2024-02-25 13:21:01,957 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP083179.1.\n",
      "2024-02-25 13:21:01,957 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP083179.1, seqID CP083179.1.\n",
      "2024-02-25 13:21:01,965 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP089319.1, seqID CP089319.1.\n",
      "2024-02-25 13:21:01,974 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP089092.2, seqID CP089092.2.\n",
      "2024-02-25 13:21:01,983 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP085906.1, seqID CP085906.1.\n",
      "2024-02-25 13:21:01,992 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP085894.1, seqID CP085894.1.\n",
      "2024-02-25 13:21:01,992 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP083473.1.\n",
      "2024-02-25 13:21:01,992 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP083473.1, seqID CP083473.1.\n",
      "2024-02-25 13:21:02,001 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP024837.1, seqID AP024837.1.\n",
      "2024-02-25 13:21:02,011 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP024831.1, seqID AP024831.1.\n",
      "2024-02-25 13:21:02,018 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP081504.1, seqID CP081504.1.\n",
      "2024-02-25 13:21:02,025 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP074109.1, seqID CP074109.1.\n",
      "2024-02-25 13:21:02,033 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP074107.1, seqID CP074107.1.\n",
      "2024-02-25 13:21:02,040 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP074105.1, seqID CP074105.1.\n",
      "2024-02-25 13:21:02,048 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP079927.1, seqID CP079927.1.\n",
      "2024-02-25 13:21:02,056 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072894.1, seqID CP072894.1.\n",
      "2024-02-25 13:21:02,064 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072884.1, seqID CP072884.1.\n",
      "2024-02-25 13:21:02,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066212.1, seqID CP066212.1.\n",
      "2024-02-25 13:21:02,080 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065777.1, seqID CP065777.1.\n",
      "2024-02-25 13:21:02,087 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065773.1, seqID CP065773.1.\n",
      "2024-02-25 13:21:02,095 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065770.1, seqID CP065770.1.\n",
      "2024-02-25 13:21:02,103 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065767.1, seqID CP065767.1.\n",
      "2024-02-25 13:21:02,111 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065764.1, seqID CP065764.1.\n",
      "2024-02-25 13:21:02,119 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065754.1, seqID CP065754.1.\n",
      "2024-02-25 13:21:02,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065752.1, seqID CP065752.1.\n",
      "2024-02-25 13:21:02,133 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP047326.1, seqID CP047326.1.\n",
      "2024-02-25 13:21:02,141 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP064406.1, seqID CP064406.1.\n",
      "2024-02-25 13:21:02,149 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072707.1, seqID CP072707.1.\n",
      "2024-02-25 13:21:02,157 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072720.1, seqID CP072720.1.\n",
      "2024-02-25 13:21:02,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072731.1, seqID CP072731.1.\n",
      "2024-02-25 13:21:02,174 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072586.1, seqID CP072586.1.\n",
      "2024-02-25 13:21:02,182 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072567.1, seqID CP072567.1.\n",
      "2024-02-25 13:21:02,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072579.1, seqID CP072579.1.\n",
      "2024-02-25 13:21:02,198 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP071958.1, seqID CP071958.1.\n",
      "2024-02-25 13:21:02,207 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP071931.1, seqID CP071931.1.\n",
      "2024-02-25 13:21:02,215 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP071271.1, seqID CP071271.1.\n",
      "2024-02-25 13:21:02,224 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP046166.1, seqID CP046166.1.\n",
      "2024-02-25 13:21:02,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP068244.1, seqID CP068244.1.\n",
      "2024-02-25 13:21:02,240 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066667.1, seqID CP066667.1.\n",
      "2024-02-25 13:21:02,248 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066653.1, seqID CP066653.1.\n",
      "2024-02-25 13:21:02,256 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066659.1, seqID CP066659.1.\n",
      "2024-02-25 13:21:02,265 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066622.1, seqID CP066622.1.\n",
      "2024-02-25 13:21:02,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066629.1, seqID CP066629.1.\n",
      "2024-02-25 13:21:02,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066645.1, seqID CP066645.1.\n",
      "2024-02-25 13:21:02,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066598.1, seqID CP066598.1.\n",
      "2024-02-25 13:21:02,298 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066591.1, seqID CP066591.1.\n",
      "2024-02-25 13:21:02,306 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066583.1, seqID CP066583.1.\n",
      "2024-02-25 13:21:02,315 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066473.1, seqID CP066473.1.\n",
      "2024-02-25 13:21:02,323 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066480.1, seqID CP066480.1.\n",
      "2024-02-25 13:21:02,332 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066427.1, seqID CP066427.1.\n",
      "2024-02-25 13:21:02,340 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066409.1, seqID CP066409.1.\n",
      "2024-02-25 13:21:02,349 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066443.1, seqID CP066443.1.\n",
      "2024-02-25 13:21:02,357 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066457.1, seqID CP066457.1.\n",
      "2024-02-25 13:21:02,365 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066381.1, seqID CP066381.1.\n",
      "2024-02-25 13:21:02,373 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066389.1, seqID CP066389.1.\n",
      "2024-02-25 13:21:02,382 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066451.1, seqID CP066451.1.\n",
      "2024-02-25 13:21:02,390 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066421.1, seqID CP066421.1.\n",
      "2024-02-25 13:21:02,399 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066415.1, seqID CP066415.1.\n",
      "2024-02-25 13:21:02,407 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066466.1, seqID CP066466.1.\n",
      "2024-02-25 13:21:02,416 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065547.1, seqID CP065547.1.\n",
      "2024-02-25 13:21:02,424 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065523.1, seqID CP065523.1.\n",
      "2024-02-25 13:21:02,432 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065210.1, seqID CP065210.1.\n",
      "2024-02-25 13:21:02,440 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP064343.1, seqID CP064343.1.\n",
      "2024-02-25 13:21:02,448 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP064049.1, seqID CP064049.1.\n",
      "2024-02-25 13:21:02,456 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP062265.1, seqID CP062265.1.\n",
      "2024-02-25 13:21:02,457 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP060861.1.\n",
      "2024-02-25 13:21:02,457 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP060861.1, seqID CP060861.1.\n",
      "2024-02-25 13:21:02,465 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP053704.1, seqID CP053704.1.\n",
      "2024-02-25 13:21:02,473 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059755.1, seqID CP059755.1.\n",
      "2024-02-25 13:21:02,481 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059823.1, seqID CP059823.1.\n",
      "2024-02-25 13:21:02,490 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059764.1, seqID CP059764.1.\n",
      "2024-02-25 13:21:02,498 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059772.1, seqID CP059772.1.\n",
      "2024-02-25 13:21:02,506 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059739.1, seqID CP059739.1.\n",
      "2024-02-25 13:21:02,515 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP061817.1, seqID CP061817.1.\n",
      "2024-02-25 13:21:02,523 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059747.1, seqID CP059747.1.\n",
      "2024-02-25 13:21:02,531 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059816.1, seqID CP059816.1.\n",
      "2024-02-25 13:21:02,540 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059793.1, seqID CP059793.1.\n",
      "2024-02-25 13:21:02,548 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059806.1, seqID CP059806.1.\n",
      "2024-02-25 13:21:02,556 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP059784.1, seqID CP059784.1.\n",
      "2024-02-25 13:21:02,564 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050650.1, seqID CP050650.1.\n",
      "2024-02-25 13:21:02,572 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050648.1, seqID CP050648.1.\n",
      "2024-02-25 13:21:02,579 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP050255.1, seqID CP050255.1.\n",
      "2024-02-25 13:21:02,588 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP039729.1, seqID CP039729.1.\n",
      "2024-02-25 13:21:02,596 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP022341.1, seqID AP022341.1.\n",
      "2024-02-25 13:21:02,605 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038995.1, seqID CP038995.1.\n",
      "2024-02-25 13:21:02,613 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038996.1, seqID CP038996.1.\n",
      "2024-02-25 13:21:02,622 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP046077.1, seqID CP046077.1.\n",
      "2024-02-25 13:21:02,630 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP045012.1, seqID CP045012.1.\n",
      "2024-02-25 13:21:02,639 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP044274.1, seqID CP044274.1.\n",
      "2024-02-25 13:21:02,648 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP044264.1, seqID CP044264.1.\n",
      "2024-02-25 13:21:02,656 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038174.1, seqID CP038174.1.\n",
      "2024-02-25 13:21:02,664 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038163.1, seqID CP038163.1.\n",
      "2024-02-25 13:21:02,672 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038164.1, seqID CP038164.1.\n",
      "2024-02-25 13:21:02,679 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038177.1, seqID CP038177.1.\n",
      "2024-02-25 13:21:02,687 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038170.1, seqID CP038170.1.\n",
      "2024-02-25 13:21:02,695 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038172.1, seqID CP038172.1.\n",
      "2024-02-25 13:21:02,704 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP041270.2, seqID CP041270.2.\n",
      "2024-02-25 13:21:02,712 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP041261.3, seqID CP041261.3.\n",
      "2024-02-25 13:21:02,720 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP019394.1, seqID AP019394.1.\n",
      "2024-02-25 13:21:02,720 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP041257.1.\n",
      "2024-02-25 13:21:02,720 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP041257.1, seqID CP041257.1.\n",
      "2024-02-25 13:21:02,728 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040904.1, seqID CP040904.1.\n",
      "2024-02-25 13:21:02,728 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP040907.1.\n",
      "2024-02-25 13:21:02,728 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040907.1, seqID CP040907.1.\n",
      "2024-02-25 13:21:02,737 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040875.1, seqID CP040875.1.\n",
      "2024-02-25 13:21:02,745 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040849.1, seqID CP040849.1.\n",
      "2024-02-25 13:21:02,754 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040740.1, seqID CP040740.1.\n",
      "2024-02-25 13:21:02,762 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040706.1, seqID CP040706.1.\n",
      "2024-02-25 13:21:02,772 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP035666.1, seqID CP035666.1.\n",
      "2024-02-25 13:21:02,781 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP035660.1, seqID CP035660.1.\n",
      "2024-02-25 13:21:02,790 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP035654.1, seqID CP035654.1.\n",
      "2024-02-25 13:21:02,799 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP035648.1, seqID CP035648.1.\n",
      "2024-02-25 13:21:02,808 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040368.1, seqID CP040368.1.\n",
      "2024-02-25 13:21:02,817 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP040236.1, seqID CP040236.1.\n",
      "2024-02-25 13:21:02,818 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP028727.1.\n",
      "2024-02-25 13:21:02,818 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP028727.1, seqID CP028727.1.\n",
      "2024-02-25 13:21:02,826 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP035220.1, seqID CP035220.1.\n",
      "2024-02-25 13:21:02,835 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP033206.1, seqID CP033206.1.\n",
      "2024-02-25 13:21:02,843 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP030110.1, seqID CP030110.1.\n",
      "2024-02-25 13:21:02,852 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP020488.1, seqID CP020488.1.\n",
      "2024-02-25 13:21:02,861 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP020484.1, seqID CP020484.1.\n",
      "2024-02-25 13:21:02,861 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP027501.1.\n",
      "2024-02-25 13:21:02,862 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027501.1, seqID CP027501.1.\n",
      "2024-02-25 13:21:02,862 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP027517.1.\n",
      "2024-02-25 13:21:02,862 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027517.1, seqID CP027517.1.\n",
      "2024-02-25 13:21:02,863 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP027512.1.\n",
      "2024-02-25 13:21:02,863 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027512.1, seqID CP027512.1.\n",
      "2024-02-25 13:21:02,863 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP027506.1.\n",
      "2024-02-25 13:21:02,863 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027506.1, seqID CP027506.1.\n",
      "2024-02-25 13:21:02,864 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP027497.1.\n",
      "2024-02-25 13:21:02,864 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027497.1, seqID CP027497.1.\n",
      "2024-02-25 13:21:02,872 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP027402.1, seqID CP027402.1.\n",
      "2024-02-25 13:21:02,881 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP044316.1, seqID CP044316.1.\n",
      "2024-02-25 13:21:02,889 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP025754.1, seqID CP025754.1.\n",
      "2024-02-25 13:21:02,897 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP021885.1, seqID CP021885.1.\n",
      "2024-02-25 13:21:02,906 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023808.1, seqID CP023808.1.\n",
      "2024-02-25 13:21:02,915 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023799.1, seqID CP023799.1.\n",
      "2024-02-25 13:21:02,923 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023804.1, seqID CP023804.1.\n",
      "2024-02-25 13:21:02,932 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023789.1, seqID CP023789.1.\n",
      "2024-02-25 13:21:02,940 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023794.1, seqID CP023794.1.\n",
      "2024-02-25 13:21:02,949 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023784.1, seqID CP023784.1.\n",
      "2024-02-25 13:21:02,957 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023780.1, seqID CP023780.1.\n",
      "2024-02-25 13:21:02,966 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP025425.1, seqID CP025425.1.\n",
      "2024-02-25 13:21:02,974 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP017787.1, seqID CP017787.1.\n",
      "2024-02-25 13:21:02,982 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP021849.1, seqID CP021849.1.\n",
      "2024-02-25 13:21:02,990 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP017797.1, seqID CP017797.1.\n",
      "2024-02-25 13:21:02,998 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP017792.1, seqID CP017792.1.\n",
      "2024-02-25 13:21:02,999 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP018128.1.\n",
      "2024-02-25 13:21:02,999 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018128.1, seqID CP018128.1.\n",
      "2024-02-25 13:21:03,007 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP023423.1, seqID CP023423.1.\n",
      "2024-02-25 13:21:03,015 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP019988.1, seqID CP019988.1.\n",
      "2024-02-25 13:21:03,024 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP019992.1, seqID CP019992.1.\n",
      "2024-02-25 13:21:03,024 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP019970.1.\n",
      "2024-02-25 13:21:03,024 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP019970.1, seqID CP019970.1.\n",
      "2024-02-25 13:21:03,033 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP019208.1, seqID CP019208.1.\n",
      "2024-02-25 13:21:03,041 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018830.1, seqID CP018830.1.\n",
      "2024-02-25 13:21:03,050 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018828.1, seqID CP018828.1.\n",
      "2024-02-25 13:21:03,058 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018071.1, seqID CP018071.1.\n",
      "2024-02-25 13:21:03,066 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP018065.1, seqID CP018065.1.\n",
      "2024-02-25 13:21:03,074 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP014529.1, seqID CP014529.1.\n",
      "2024-02-25 13:21:03,082 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP012460.1, seqID CP012460.1.\n",
      "2024-02-25 13:21:03,090 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP011281.1, seqID CP011281.1.\n",
      "2024-02-25 13:21:03,099 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP014449.1, seqID CP014449.1.\n",
      "2024-02-25 13:21:03,106 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP011828.1, seqID CP011828.1.\n",
      "2024-02-25 13:21:03,115 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP013994.1, seqID CP013994.1.\n",
      "2024-02-25 13:21:03,122 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP013009.1, seqID CP013009.1.\n",
      "2024-02-25 13:21:03,130 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP012522.1, seqID CP012522.1.\n",
      "2024-02-25 13:21:03,139 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU016036.1, seqID NZ_OU016036.1.\n",
      "2024-02-25 13:21:03,149 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU016027.1, seqID NZ_OU016027.1.\n",
      "2024-02-25 13:21:03,159 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU015946.1, seqID NZ_OU015946.1.\n",
      "2024-02-25 13:21:03,168 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU015708.1, seqID NZ_OU015708.1.\n",
      "2024-02-25 13:21:03,178 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU015350.1, seqID NZ_OU015350.1.\n",
      "2024-02-25 13:21:03,188 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_OU015345.1, seqID NZ_OU015345.1.\n",
      "2024-02-25 13:21:03,196 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR215968.1, seqID NZ_LR215968.1.\n",
      "2024-02-25 13:21:03,205 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135482.1, seqID NZ_LR135482.1.\n",
      "2024-02-25 13:21:03,214 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135475.1, seqID NZ_LR135475.1.\n",
      "2024-02-25 13:21:03,224 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135488.1, seqID NZ_LR135488.1.\n",
      "2024-02-25 13:21:03,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135474.1, seqID NZ_LR135474.1.\n",
      "2024-02-25 13:21:03,242 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135443.1, seqID NZ_LR135443.1.\n",
      "2024-02-25 13:21:03,251 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135435.1, seqID NZ_LR135435.1.\n",
      "2024-02-25 13:21:03,261 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135419.1, seqID NZ_LR135419.1.\n",
      "2024-02-25 13:21:03,270 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135428.1, seqID NZ_LR135428.1.\n",
      "2024-02-25 13:21:03,280 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135408.1, seqID NZ_LR135408.1.\n",
      "2024-02-25 13:21:03,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135414.1, seqID NZ_LR135414.1.\n",
      "2024-02-25 13:21:03,300 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135394.1, seqID NZ_LR135394.1.\n",
      "2024-02-25 13:21:03,309 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135401.1, seqID NZ_LR135401.1.\n",
      "2024-02-25 13:21:03,318 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135357.1, seqID NZ_LR135357.1.\n",
      "2024-02-25 13:21:03,328 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135351.1, seqID NZ_LR135351.1.\n",
      "2024-02-25 13:21:03,338 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135364.1, seqID NZ_LR135364.1.\n",
      "2024-02-25 13:21:03,347 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135372.1, seqID NZ_LR135372.1.\n",
      "2024-02-25 13:21:03,357 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135384.1, seqID NZ_LR135384.1.\n",
      "2024-02-25 13:21:03,366 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135344.1, seqID NZ_LR135344.1.\n",
      "2024-02-25 13:21:03,375 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135317.1, seqID NZ_LR135317.1.\n",
      "2024-02-25 13:21:03,384 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135324.1, seqID NZ_LR135324.1.\n",
      "2024-02-25 13:21:03,394 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135339.1, seqID NZ_LR135339.1.\n",
      "2024-02-25 13:21:03,404 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135307.1, seqID NZ_LR135307.1.\n",
      "2024-02-25 13:21:03,414 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135297.1, seqID NZ_LR135297.1.\n",
      "2024-02-25 13:21:03,423 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135293.1, seqID NZ_LR135293.1.\n",
      "2024-02-25 13:21:03,432 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135254.1, seqID NZ_LR135254.1.\n",
      "2024-02-25 13:21:03,441 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135243.1, seqID NZ_LR135243.1.\n",
      "2024-02-25 13:21:03,450 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135258.1, seqID NZ_LR135258.1.\n",
      "2024-02-25 13:21:03,459 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135278.1, seqID NZ_LR135278.1.\n",
      "2024-02-25 13:21:03,468 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135219.1, seqID NZ_LR135219.1.\n",
      "2024-02-25 13:21:03,477 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135235.1, seqID NZ_LR135235.1.\n",
      "2024-02-25 13:21:03,486 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135191.1, seqID NZ_LR135191.1.\n",
      "2024-02-25 13:21:03,494 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135179.1, seqID NZ_LR135179.1.\n",
      "2024-02-25 13:21:03,503 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135174.1, seqID NZ_LR135174.1.\n",
      "2024-02-25 13:21:03,511 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135169.1, seqID NZ_LR135169.1.\n",
      "2024-02-25 13:21:03,519 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134337.1, seqID NZ_LR134337.1.\n",
      "2024-02-25 13:21:03,528 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134105.1, seqID NZ_LR134105.1.\n",
      "2024-02-25 13:21:03,536 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR134095.1, seqID NZ_LR134095.1.\n",
      "2024-02-25 13:21:03,544 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR132067.1, seqID NZ_LR132067.1.\n",
      "2024-02-25 13:21:03,553 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LT603678.1, seqID NZ_LT603678.1.\n",
      "2024-02-25 13:21:03,563 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LT598663.1, seqID NZ_LT598663.1.\n",
      "2024-02-25 13:21:03,571 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LN999844.1, seqID NZ_LN999844.1.\n",
      "2024-02-25 13:21:03,580 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LN999987.1, seqID NZ_LN999987.1.\n",
      "2024-02-25 13:21:03,589 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP132226.1, seqID NZ_CP132226.1.\n",
      "2024-02-25 13:21:03,598 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130914.1, seqID NZ_CP130914.1.\n",
      "2024-02-25 13:21:03,608 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130912.1, seqID NZ_CP130912.1.\n",
      "2024-02-25 13:21:03,617 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130908.1, seqID NZ_CP130908.1.\n",
      "2024-02-25 13:21:03,626 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130910.1, seqID NZ_CP130910.1.\n",
      "2024-02-25 13:21:03,636 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130906.1, seqID NZ_CP130906.1.\n",
      "2024-02-25 13:21:03,645 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130883.1, seqID NZ_CP130883.1.\n",
      "2024-02-25 13:21:03,654 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130879.1, seqID NZ_CP130879.1.\n",
      "2024-02-25 13:21:03,664 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130875.1, seqID NZ_CP130875.1.\n",
      "2024-02-25 13:21:03,673 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130877.1, seqID NZ_CP130877.1.\n",
      "2024-02-25 13:21:03,683 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130881.1, seqID NZ_CP130881.1.\n",
      "2024-02-25 13:21:03,692 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130882.1, seqID NZ_CP130882.1.\n",
      "2024-02-25 13:21:03,702 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130849.1, seqID NZ_CP130849.1.\n",
      "2024-02-25 13:21:03,711 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130833.1, seqID NZ_CP130833.1.\n",
      "2024-02-25 13:21:03,720 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130841.1, seqID NZ_CP130841.1.\n",
      "2024-02-25 13:21:03,729 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130837.1, seqID NZ_CP130837.1.\n",
      "2024-02-25 13:21:03,738 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130843.1, seqID NZ_CP130843.1.\n",
      "2024-02-25 13:21:03,748 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130835.1, seqID NZ_CP130835.1.\n",
      "2024-02-25 13:21:03,757 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130839.1, seqID NZ_CP130839.1.\n",
      "2024-02-25 13:21:03,767 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130845.1, seqID NZ_CP130845.1.\n",
      "2024-02-25 13:21:03,776 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130847.1, seqID NZ_CP130847.1.\n",
      "2024-02-25 13:21:03,785 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130851.1, seqID NZ_CP130851.1.\n",
      "2024-02-25 13:21:03,794 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130861.1, seqID NZ_CP130861.1.\n",
      "2024-02-25 13:21:03,803 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130859.1, seqID NZ_CP130859.1.\n",
      "2024-02-25 13:21:03,813 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130865.1, seqID NZ_CP130865.1.\n",
      "2024-02-25 13:21:03,822 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130863.1, seqID NZ_CP130863.1.\n",
      "2024-02-25 13:21:03,831 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130867.1, seqID NZ_CP130867.1.\n",
      "2024-02-25 13:21:03,840 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130869.1, seqID NZ_CP130869.1.\n",
      "2024-02-25 13:21:03,849 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130873.1, seqID NZ_CP130873.1.\n",
      "2024-02-25 13:21:03,859 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130871.1, seqID NZ_CP130871.1.\n",
      "2024-02-25 13:21:03,867 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130827.1, seqID NZ_CP130827.1.\n",
      "2024-02-25 13:21:03,877 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130825.1, seqID NZ_CP130825.1.\n",
      "2024-02-25 13:21:03,886 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP130829.1, seqID NZ_CP130829.1.\n",
      "2024-02-25 13:21:03,895 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP128188.1, seqID NZ_CP128188.1.\n",
      "2024-02-25 13:21:03,904 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP027294.1, seqID NZ_AP027294.1.\n",
      "2024-02-25 13:21:03,914 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109758.1, seqID NZ_CP109758.1.\n",
      "2024-02-25 13:21:03,923 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP119150.1, seqID NZ_CP119150.1.\n",
      "2024-02-25 13:21:03,931 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP118955.1, seqID NZ_CP118955.1.\n",
      "2024-02-25 13:21:03,941 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP118548.1, seqID NZ_CP118548.1.\n",
      "2024-02-25 13:21:03,949 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116528.1, seqID NZ_CP116528.1.\n",
      "2024-02-25 13:21:03,958 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116523.1, seqID NZ_CP116523.1.\n",
      "2024-02-25 13:21:03,967 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116520.1, seqID NZ_CP116520.1.\n",
      "2024-02-25 13:21:03,975 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116517.1, seqID NZ_CP116517.1.\n",
      "2024-02-25 13:21:03,984 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116532.1, seqID NZ_CP116532.1.\n",
      "2024-02-25 13:21:03,992 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116514.1, seqID NZ_CP116514.1.\n",
      "2024-02-25 13:21:04,001 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116023.1, seqID NZ_CP116023.1.\n",
      "2024-02-25 13:21:04,009 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP115812.1, seqID NZ_CP115812.1.\n",
      "2024-02-25 13:21:04,019 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP114617.1, seqID NZ_CP114617.1.\n",
      "2024-02-25 13:21:04,029 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP112862.1, seqID NZ_CP112862.1.\n",
      "2024-02-25 13:21:04,039 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026666.1, seqID NZ_AP026666.1.\n",
      "2024-02-25 13:21:04,049 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026660.1, seqID NZ_AP026660.1.\n",
      "2024-02-25 13:21:04,059 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026655.1, seqID NZ_AP026655.1.\n",
      "2024-02-25 13:21:04,069 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026604.1, seqID NZ_AP026604.1.\n",
      "2024-02-25 13:21:04,078 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026601.1, seqID NZ_AP026601.1.\n",
      "2024-02-25 13:21:04,089 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026566.1, seqID NZ_AP026566.1.\n",
      "2024-02-25 13:21:04,098 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026772.1, seqID NZ_AP026772.1.\n",
      "2024-02-25 13:21:04,107 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109798.1, seqID NZ_CP109798.1.\n",
      "2024-02-25 13:21:04,116 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_JAOXXZ010000001.1, seqID NZ_JAOXXZ010000001.1.\n",
      "2024-02-25 13:21:04,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_JAOXXY010000002.1, seqID NZ_JAOXXY010000002.1.\n",
      "2024-02-25 13:21:04,135 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_JAOXXX010000002.1, seqID NZ_JAOXXX010000002.1.\n",
      "2024-02-25 13:21:04,144 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP102534.1, seqID NZ_CP102534.1.\n",
      "2024-02-25 13:21:04,154 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP058891.1, seqID NZ_CP058891.1.\n",
      "2024-02-25 13:21:04,163 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091224.1, seqID NZ_CP091224.1.\n",
      "2024-02-25 13:21:04,172 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091217.1, seqID NZ_CP091217.1.\n",
      "2024-02-25 13:21:04,181 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091213.1, seqID NZ_CP091213.1.\n",
      "2024-02-25 13:21:04,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091210.1, seqID NZ_CP091210.1.\n",
      "2024-02-25 13:21:04,198 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091206.1, seqID NZ_CP091206.1.\n",
      "2024-02-25 13:21:04,207 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP088191.1, seqID NZ_CP088191.1.\n",
      "2024-02-25 13:21:04,217 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097022.1, seqID NZ_CP097022.1.\n",
      "2024-02-25 13:21:04,226 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097014.1, seqID NZ_CP097014.1.\n",
      "2024-02-25 13:21:04,236 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP096219.1, seqID NZ_CP096219.1.\n",
      "2024-02-25 13:21:04,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP096211.1, seqID NZ_CP096211.1.\n",
      "2024-02-25 13:21:04,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP110360.1, seqID NZ_CP110360.1.\n",
      "2024-02-25 13:21:04,264 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP110294.1, seqID NZ_CP110294.1.\n",
      "2024-02-25 13:21:04,274 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP094230.1, seqID NZ_CP094230.1.\n",
      "2024-02-25 13:21:04,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP093947.1, seqID NZ_CP093947.1.\n",
      "2024-02-25 13:21:04,294 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP093953.1, seqID NZ_CP093953.1.\n",
      "2024-02-25 13:21:04,303 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP093941.1, seqID NZ_CP093941.1.\n",
      "2024-02-25 13:21:04,313 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP092569.1, seqID NZ_CP092569.1.\n",
      "2024-02-25 13:21:04,323 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP076483.1, seqID NZ_CP076483.1.\n",
      "2024-02-25 13:21:04,333 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP076477.1, seqID NZ_CP076477.1.\n",
      "2024-02-25 13:21:04,342 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP076471.1, seqID NZ_CP076471.1.\n",
      "2024-02-25 13:21:04,351 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091577.1, seqID NZ_CP091577.1.\n",
      "2024-02-25 13:21:04,361 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091301.1, seqID NZ_CP091301.1.\n",
      "2024-02-25 13:21:04,369 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP091100.1, seqID NZ_CP091100.1.\n",
      "2024-02-25 13:21:04,377 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083179.1, seqID NZ_CP083179.1.\n",
      "2024-02-25 13:21:04,387 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP089319.1, seqID NZ_CP089319.1.\n",
      "2024-02-25 13:21:04,396 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP089092.2, seqID NZ_CP089092.2.\n",
      "2024-02-25 13:21:04,406 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP085906.1, seqID NZ_CP085906.1.\n",
      "2024-02-25 13:21:04,416 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP085894.1, seqID NZ_CP085894.1.\n",
      "2024-02-25 13:21:04,425 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083473.1, seqID NZ_CP083473.1.\n",
      "2024-02-25 13:21:04,435 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP024837.1, seqID NZ_AP024837.1.\n",
      "2024-02-25 13:21:04,445 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP024831.1, seqID NZ_AP024831.1.\n",
      "2024-02-25 13:21:04,453 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP074109.1, seqID NZ_CP074109.1.\n",
      "2024-02-25 13:21:04,462 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP074107.1, seqID NZ_CP074107.1.\n",
      "2024-02-25 13:21:04,470 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP074105.1, seqID NZ_CP074105.1.\n",
      "2024-02-25 13:21:04,480 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP079927.1, seqID NZ_CP079927.1.\n",
      "2024-02-25 13:21:04,489 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072894.1, seqID NZ_CP072894.1.\n",
      "2024-02-25 13:21:04,498 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072884.1, seqID NZ_CP072884.1.\n",
      "2024-02-25 13:21:04,507 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065777.1, seqID NZ_CP065777.1.\n",
      "2024-02-25 13:21:04,516 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065773.1, seqID NZ_CP065773.1.\n",
      "2024-02-25 13:21:04,525 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065770.1, seqID NZ_CP065770.1.\n",
      "2024-02-25 13:21:04,535 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065767.1, seqID NZ_CP065767.1.\n",
      "2024-02-25 13:21:04,544 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065754.1, seqID NZ_CP065754.1.\n",
      "2024-02-25 13:21:04,553 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065752.1, seqID NZ_CP065752.1.\n",
      "2024-02-25 13:21:04,561 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP047326.1, seqID NZ_CP047326.1.\n",
      "2024-02-25 13:21:04,571 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064406.1, seqID NZ_CP064406.1.\n",
      "2024-02-25 13:21:04,580 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072707.1, seqID NZ_CP072707.1.\n",
      "2024-02-25 13:21:04,590 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072720.1, seqID NZ_CP072720.1.\n",
      "2024-02-25 13:21:04,600 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072731.1, seqID NZ_CP072731.1.\n",
      "2024-02-25 13:21:04,609 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072586.1, seqID NZ_CP072586.1.\n",
      "2024-02-25 13:21:04,619 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072567.1, seqID NZ_CP072567.1.\n",
      "2024-02-25 13:21:04,629 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072579.1, seqID NZ_CP072579.1.\n",
      "2024-02-25 13:21:04,638 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP071958.1, seqID NZ_CP071958.1.\n",
      "2024-02-25 13:21:04,648 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP071931.1, seqID NZ_CP071931.1.\n",
      "2024-02-25 13:21:04,658 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP046166.1, seqID NZ_CP046166.1.\n",
      "2024-02-25 13:21:04,666 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP068244.1, seqID NZ_CP068244.1.\n",
      "2024-02-25 13:21:04,676 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066667.1, seqID NZ_CP066667.1.\n",
      "2024-02-25 13:21:04,685 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066653.1, seqID NZ_CP066653.1.\n",
      "2024-02-25 13:21:04,694 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066659.1, seqID NZ_CP066659.1.\n",
      "2024-02-25 13:21:04,704 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066622.1, seqID NZ_CP066622.1.\n",
      "2024-02-25 13:21:04,713 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066629.1, seqID NZ_CP066629.1.\n",
      "2024-02-25 13:21:04,723 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066645.1, seqID NZ_CP066645.1.\n",
      "2024-02-25 13:21:04,732 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066598.1, seqID NZ_CP066598.1.\n",
      "2024-02-25 13:21:04,741 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066591.1, seqID NZ_CP066591.1.\n",
      "2024-02-25 13:21:04,751 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066583.1, seqID NZ_CP066583.1.\n",
      "2024-02-25 13:21:04,761 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066473.1, seqID NZ_CP066473.1.\n",
      "2024-02-25 13:21:04,770 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066480.1, seqID NZ_CP066480.1.\n",
      "2024-02-25 13:21:04,780 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066427.1, seqID NZ_CP066427.1.\n",
      "2024-02-25 13:21:04,789 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066409.1, seqID NZ_CP066409.1.\n",
      "2024-02-25 13:21:04,799 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066443.1, seqID NZ_CP066443.1.\n",
      "2024-02-25 13:21:04,809 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066457.1, seqID NZ_CP066457.1.\n",
      "2024-02-25 13:21:04,818 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066381.1, seqID NZ_CP066381.1.\n",
      "2024-02-25 13:21:04,827 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066389.1, seqID NZ_CP066389.1.\n",
      "2024-02-25 13:21:04,837 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066451.1, seqID NZ_CP066451.1.\n",
      "2024-02-25 13:21:04,846 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066421.1, seqID NZ_CP066421.1.\n",
      "2024-02-25 13:21:04,855 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066415.1, seqID NZ_CP066415.1.\n",
      "2024-02-25 13:21:04,865 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066466.1, seqID NZ_CP066466.1.\n",
      "2024-02-25 13:21:04,874 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065547.1, seqID NZ_CP065547.1.\n",
      "2024-02-25 13:21:04,884 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065523.1, seqID NZ_CP065523.1.\n",
      "2024-02-25 13:21:04,892 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065210.1, seqID NZ_CP065210.1.\n",
      "2024-02-25 13:21:04,901 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064343.1, seqID NZ_CP064343.1.\n",
      "2024-02-25 13:21:04,911 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064049.1, seqID NZ_CP064049.1.\n",
      "2024-02-25 13:21:04,920 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP062265.1, seqID NZ_CP062265.1.\n",
      "2024-02-25 13:21:04,929 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP060861.1, seqID NZ_CP060861.1.\n",
      "2024-02-25 13:21:04,938 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP053704.1, seqID NZ_CP053704.1.\n",
      "2024-02-25 13:21:04,948 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059755.1, seqID NZ_CP059755.1.\n",
      "2024-02-25 13:21:04,958 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059823.1, seqID NZ_CP059823.1.\n",
      "2024-02-25 13:21:04,967 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059764.1, seqID NZ_CP059764.1.\n",
      "2024-02-25 13:21:04,977 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059772.1, seqID NZ_CP059772.1.\n",
      "2024-02-25 13:21:04,986 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059739.1, seqID NZ_CP059739.1.\n",
      "2024-02-25 13:21:04,996 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP061817.1, seqID NZ_CP061817.1.\n",
      "2024-02-25 13:21:05,005 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059747.1, seqID NZ_CP059747.1.\n",
      "2024-02-25 13:21:05,015 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059816.1, seqID NZ_CP059816.1.\n",
      "2024-02-25 13:21:05,025 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059793.1, seqID NZ_CP059793.1.\n",
      "2024-02-25 13:21:05,034 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059806.1, seqID NZ_CP059806.1.\n",
      "2024-02-25 13:21:05,044 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP059784.1, seqID NZ_CP059784.1.\n",
      "2024-02-25 13:21:05,053 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050650.1, seqID NZ_CP050650.1.\n",
      "2024-02-25 13:21:05,063 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050648.1, seqID NZ_CP050648.1.\n",
      "2024-02-25 13:21:05,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP050255.1, seqID NZ_CP050255.1.\n",
      "2024-02-25 13:21:05,081 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP039729.1, seqID NZ_CP039729.1.\n",
      "2024-02-25 13:21:05,091 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022341.1, seqID NZ_AP022341.1.\n",
      "2024-02-25 13:21:05,100 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038995.1, seqID NZ_CP038995.1.\n",
      "2024-02-25 13:21:05,110 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP046077.1, seqID NZ_CP046077.1.\n",
      "2024-02-25 13:21:05,119 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045012.1, seqID NZ_CP045012.1.\n",
      "2024-02-25 13:21:05,129 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP044274.1, seqID NZ_CP044274.1.\n",
      "2024-02-25 13:21:05,139 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP044264.1, seqID NZ_CP044264.1.\n",
      "2024-02-25 13:21:05,148 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038174.1, seqID NZ_CP038174.1.\n",
      "2024-02-25 13:21:05,157 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038163.1, seqID NZ_CP038163.1.\n",
      "2024-02-25 13:21:05,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038164.1, seqID NZ_CP038164.1.\n",
      "2024-02-25 13:21:05,173 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038177.1, seqID NZ_CP038177.1.\n",
      "2024-02-25 13:21:05,182 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038170.1, seqID NZ_CP038170.1.\n",
      "2024-02-25 13:21:05,191 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038172.1, seqID NZ_CP038172.1.\n",
      "2024-02-25 13:21:05,200 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP041270.2, seqID NZ_CP041270.2.\n",
      "2024-02-25 13:21:05,210 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP041261.2, seqID NZ_CP041261.2.\n",
      "2024-02-25 13:21:05,219 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP019394.1, seqID NZ_AP019394.1.\n",
      "2024-02-25 13:21:05,228 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP041257.1, seqID NZ_CP041257.1.\n",
      "2024-02-25 13:21:05,237 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040904.1, seqID NZ_CP040904.1.\n",
      "2024-02-25 13:21:05,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040907.1, seqID NZ_CP040907.1.\n",
      "2024-02-25 13:21:05,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040875.1, seqID NZ_CP040875.1.\n",
      "2024-02-25 13:21:05,265 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040849.1, seqID NZ_CP040849.1.\n",
      "2024-02-25 13:21:05,274 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040740.1, seqID NZ_CP040740.1.\n",
      "2024-02-25 13:21:05,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040706.1, seqID NZ_CP040706.1.\n",
      "2024-02-25 13:21:05,293 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035666.1, seqID NZ_CP035666.1.\n",
      "2024-02-25 13:21:05,303 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035660.1, seqID NZ_CP035660.1.\n",
      "2024-02-25 13:21:05,313 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035654.1, seqID NZ_CP035654.1.\n",
      "2024-02-25 13:21:05,324 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035648.1, seqID NZ_CP035648.1.\n",
      "2024-02-25 13:21:05,334 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040368.1, seqID NZ_CP040368.1.\n",
      "2024-02-25 13:21:05,344 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP040236.1, seqID NZ_CP040236.1.\n",
      "2024-02-25 13:21:05,353 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP028727.1, seqID NZ_CP028727.1.\n",
      "2024-02-25 13:21:05,363 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP035220.1, seqID NZ_CP035220.1.\n",
      "2024-02-25 13:21:05,372 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP033206.1, seqID NZ_CP033206.1.\n",
      "2024-02-25 13:21:05,381 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP030110.1, seqID NZ_CP030110.1.\n",
      "2024-02-25 13:21:05,391 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP020488.1, seqID NZ_CP020488.1.\n",
      "2024-02-25 13:21:05,401 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP020484.1, seqID NZ_CP020484.1.\n",
      "2024-02-25 13:21:05,411 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027501.1, seqID NZ_CP027501.1.\n",
      "2024-02-25 13:21:05,420 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027517.1, seqID NZ_CP027517.1.\n",
      "2024-02-25 13:21:05,429 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027512.1, seqID NZ_CP027512.1.\n",
      "2024-02-25 13:21:05,439 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027506.1, seqID NZ_CP027506.1.\n",
      "2024-02-25 13:21:05,448 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027497.1, seqID NZ_CP027497.1.\n",
      "2024-02-25 13:21:05,456 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP027402.1, seqID NZ_CP027402.1.\n",
      "2024-02-25 13:21:05,466 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP044316.1, seqID NZ_CP044316.1.\n",
      "2024-02-25 13:21:05,476 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP025754.1, seqID NZ_CP025754.1.\n",
      "2024-02-25 13:21:05,485 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP021885.1, seqID NZ_CP021885.1.\n",
      "2024-02-25 13:21:05,495 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023808.1, seqID NZ_CP023808.1.\n",
      "2024-02-25 13:21:05,504 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023799.1, seqID NZ_CP023799.1.\n",
      "2024-02-25 13:21:05,514 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023804.1, seqID NZ_CP023804.1.\n",
      "2024-02-25 13:21:05,523 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023789.1, seqID NZ_CP023789.1.\n",
      "2024-02-25 13:21:05,533 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023794.1, seqID NZ_CP023794.1.\n",
      "2024-02-25 13:21:05,542 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023784.1, seqID NZ_CP023784.1.\n",
      "2024-02-25 13:21:05,552 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023780.1, seqID NZ_CP023780.1.\n",
      "2024-02-25 13:21:05,561 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP025425.1, seqID NZ_CP025425.1.\n",
      "2024-02-25 13:21:05,571 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP017787.1, seqID NZ_CP017787.1.\n",
      "2024-02-25 13:21:05,580 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP021849.1, seqID NZ_CP021849.1.\n",
      "2024-02-25 13:21:05,590 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP017797.1, seqID NZ_CP017797.1.\n",
      "2024-02-25 13:21:05,599 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP017792.1, seqID NZ_CP017792.1.\n",
      "2024-02-25 13:21:05,608 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018128.1, seqID NZ_CP018128.1.\n",
      "2024-02-25 13:21:05,618 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP023423.1, seqID NZ_CP023423.1.\n",
      "2024-02-25 13:21:05,628 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP019988.1, seqID NZ_CP019988.1.\n",
      "2024-02-25 13:21:05,638 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP019992.1, seqID NZ_CP019992.1.\n",
      "2024-02-25 13:21:05,647 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP019970.1, seqID NZ_CP019970.1.\n",
      "2024-02-25 13:21:05,657 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP019208.1, seqID NZ_CP019208.1.\n",
      "2024-02-25 13:21:05,667 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018830.1, seqID NZ_CP018830.1.\n",
      "2024-02-25 13:21:05,677 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018828.1, seqID NZ_CP018828.1.\n",
      "2024-02-25 13:21:05,687 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018071.1, seqID NZ_CP018071.1.\n",
      "2024-02-25 13:21:05,697 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP018065.1, seqID NZ_CP018065.1.\n",
      "2024-02-25 13:21:05,707 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014529.1, seqID NZ_CP014529.1.\n",
      "2024-02-25 13:21:05,717 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012460.1, seqID NZ_CP012460.1.\n",
      "2024-02-25 13:21:05,726 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP011281.1, seqID NZ_CP011281.1.\n",
      "2024-02-25 13:21:05,736 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP014449.1, seqID NZ_CP014449.1.\n",
      "2024-02-25 13:21:05,745 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP011828.1, seqID NZ_CP011828.1.\n",
      "2024-02-25 13:21:05,756 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP013994.1, seqID NZ_CP013994.1.\n",
      "2024-02-25 13:21:05,765 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP013009.1, seqID NZ_CP013009.1.\n",
      "2024-02-25 13:21:05,774 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012522.1, seqID NZ_CP012522.1.\n",
      "2024-02-25 13:21:05,784 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038996.1, seqID NZ_CP038996.1.\n",
      "2024-02-25 13:21:05,793 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP042289.1, seqID CP042289.1.\n",
      "2024-02-25 13:21:05,803 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP042289.1, seqID NZ_CP042289.1.\n",
      "2024-02-25 13:21:05,811 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP025473.1, seqID CP025473.1.\n",
      "2024-02-25 13:21:05,820 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP025473.1, seqID NZ_CP025473.1.\n",
      "2024-02-25 13:21:05,829 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063766.1, seqID CP063766.1.\n",
      "2024-02-25 13:21:05,838 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP063766.1, seqID NZ_CP063766.1.\n",
      "2024-02-25 13:21:05,846 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP006030.1, seqID CP006030.1.\n",
      "2024-02-25 13:21:05,854 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP006030.1, seqID NZ_CP006030.1.\n",
      "2024-02-25 13:21:05,863 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP003583.1, seqID CP003583.1.\n",
      "2024-02-25 13:21:05,872 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NC_017960.1, seqID NC_017960.1.\n",
      "2024-02-25 13:21:05,873 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135782.1.\n",
      "2024-02-25 13:21:05,873 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135782.1, seqID LR135782.1.\n",
      "2024-02-25 13:21:05,874 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135331.1.\n",
      "2024-02-25 13:21:05,874 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135331.1, seqID LR135331.1.\n",
      "2024-02-25 13:21:05,874 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135287.1.\n",
      "2024-02-25 13:21:05,874 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135287.1, seqID LR135287.1.\n",
      "2024-02-25 13:21:05,875 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135203.1.\n",
      "2024-02-25 13:21:05,875 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135203.1, seqID LR135203.1.\n",
      "2024-02-25 13:21:05,875 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135226.1.\n",
      "2024-02-25 13:21:05,875 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135226.1, seqID LR135226.1.\n",
      "2024-02-25 13:21:05,876 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135197.1.\n",
      "2024-02-25 13:21:05,876 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135197.1, seqID LR135197.1.\n",
      "2024-02-25 13:21:05,876 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135181.1.\n",
      "2024-02-25 13:21:05,876 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135181.1, seqID LR135181.1.\n",
      "2024-02-25 13:21:05,877 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for LR135170.1.\n",
      "2024-02-25 13:21:05,877 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain LR135170.1, seqID LR135170.1.\n",
      "2024-02-25 13:21:05,885 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116549.1, seqID CP116549.1.\n",
      "2024-02-25 13:21:05,893 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116546.1, seqID CP116546.1.\n",
      "2024-02-25 13:21:05,902 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116543.1, seqID CP116543.1.\n",
      "2024-02-25 13:21:05,910 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116540.1, seqID CP116540.1.\n",
      "2024-02-25 13:21:05,918 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116536.1, seqID CP116536.1.\n",
      "2024-02-25 13:21:05,927 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP026612.1, seqID AP026612.1.\n",
      "2024-02-25 13:21:05,936 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP109839.1, seqID CP109839.1.\n",
      "2024-02-25 13:21:05,944 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP088195.1, seqID CP088195.1.\n",
      "2024-02-25 13:21:05,951 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097061.1, seqID CP097061.1.\n",
      "2024-02-25 13:21:05,960 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097027.1, seqID CP097027.1.\n",
      "2024-02-25 13:21:05,969 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063639.1, seqID CP063639.1.\n",
      "2024-02-25 13:21:05,978 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063579.1, seqID CP063579.1.\n",
      "2024-02-25 13:21:05,986 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP063542.1, seqID CP063542.1.\n",
      "2024-02-25 13:21:05,994 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP084178.1, seqID CP084178.1.\n",
      "2024-02-25 13:21:06,003 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP083920.1, seqID CP083920.1.\n",
      "2024-02-25 13:21:06,011 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP083912.1, seqID CP083912.1.\n",
      "2024-02-25 13:21:06,018 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072881.1, seqID CP072881.1.\n",
      "2024-02-25 13:21:06,026 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP072878.1, seqID CP072878.1.\n",
      "2024-02-25 13:21:06,034 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066215.1, seqID CP066215.1.\n",
      "2024-02-25 13:21:06,042 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066205.1, seqID CP066205.1.\n",
      "2024-02-25 13:21:06,050 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP064424.1, seqID CP064424.1.\n",
      "2024-02-25 13:21:06,059 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066673.1, seqID CP066673.1.\n",
      "2024-02-25 13:21:06,068 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066637.1, seqID CP066637.1.\n",
      "2024-02-25 13:21:06,076 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066606.1, seqID CP066606.1.\n",
      "2024-02-25 13:21:06,084 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066614.1, seqID CP066614.1.\n",
      "2024-02-25 13:21:06,093 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066576.1, seqID CP066576.1.\n",
      "2024-02-25 13:21:06,102 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066568.1, seqID CP066568.1.\n",
      "2024-02-25 13:21:06,110 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066396.1, seqID CP066396.1.\n",
      "2024-02-25 13:21:06,119 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP066434.1, seqID CP066434.1.\n",
      "2024-02-25 13:21:06,127 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065928.1, seqID CP065928.1.\n",
      "2024-02-25 13:21:06,136 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP065527.1, seqID CP065527.1.\n",
      "2024-02-25 13:21:06,144 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP045602.1, seqID CP045602.1.\n",
      "2024-02-25 13:21:06,144 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP064279.1.\n",
      "2024-02-25 13:21:06,144 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP064279.1, seqID CP064279.1.\n",
      "2024-02-25 13:21:06,152 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP034949.1, seqID CP034949.1.\n",
      "2024-02-25 13:21:06,160 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP033041.1, seqID CP033041.1.\n",
      "2024-02-25 13:21:06,168 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP032308.1, seqID CP032308.1.\n",
      "2024-02-25 13:21:06,176 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP025077.1, seqID CP025077.1.\n",
      "2024-02-25 13:21:06,184 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP016163.1, seqID CP016163.1.\n",
      "2024-02-25 13:21:06,192 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP012430.1, seqID CP012430.1.\n",
      "2024-02-25 13:21:06,200 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135782.1, seqID NZ_LR135782.1.\n",
      "2024-02-25 13:21:06,209 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135331.1, seqID NZ_LR135331.1.\n",
      "2024-02-25 13:21:06,218 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135287.1, seqID NZ_LR135287.1.\n",
      "2024-02-25 13:21:06,227 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135203.1, seqID NZ_LR135203.1.\n",
      "2024-02-25 13:21:06,236 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135226.1, seqID NZ_LR135226.1.\n",
      "2024-02-25 13:21:06,244 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135197.1, seqID NZ_LR135197.1.\n",
      "2024-02-25 13:21:06,252 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135181.1, seqID NZ_LR135181.1.\n",
      "2024-02-25 13:21:06,260 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_LR135170.1, seqID NZ_LR135170.1.\n",
      "2024-02-25 13:21:06,269 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116549.1, seqID NZ_CP116549.1.\n",
      "2024-02-25 13:21:06,278 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116546.1, seqID NZ_CP116546.1.\n",
      "2024-02-25 13:21:06,287 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116543.1, seqID NZ_CP116543.1.\n",
      "2024-02-25 13:21:06,296 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116540.1, seqID NZ_CP116540.1.\n",
      "2024-02-25 13:21:06,304 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116536.1, seqID NZ_CP116536.1.\n",
      "2024-02-25 13:21:06,314 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP026612.1, seqID NZ_AP026612.1.\n",
      "2024-02-25 13:21:06,323 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP109839.1, seqID NZ_CP109839.1.\n",
      "2024-02-25 13:21:06,331 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP088195.1, seqID NZ_CP088195.1.\n",
      "2024-02-25 13:21:06,340 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097061.1, seqID NZ_CP097061.1.\n",
      "2024-02-25 13:21:06,349 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097027.1, seqID NZ_CP097027.1.\n",
      "2024-02-25 13:21:06,359 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP084178.1, seqID NZ_CP084178.1.\n",
      "2024-02-25 13:21:06,367 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083920.1, seqID NZ_CP083920.1.\n",
      "2024-02-25 13:21:06,376 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP083912.1, seqID NZ_CP083912.1.\n",
      "2024-02-25 13:21:06,385 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072881.1, seqID NZ_CP072881.1.\n",
      "2024-02-25 13:21:06,394 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP072878.1, seqID NZ_CP072878.1.\n",
      "2024-02-25 13:21:06,403 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066215.1, seqID NZ_CP066215.1.\n",
      "2024-02-25 13:21:06,412 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP064424.1, seqID NZ_CP064424.1.\n",
      "2024-02-25 13:21:06,421 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066673.1, seqID NZ_CP066673.1.\n",
      "2024-02-25 13:21:06,431 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066637.1, seqID NZ_CP066637.1.\n",
      "2024-02-25 13:21:06,440 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066606.1, seqID NZ_CP066606.1.\n",
      "2024-02-25 13:21:06,449 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066614.1, seqID NZ_CP066614.1.\n",
      "2024-02-25 13:21:06,459 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066576.1, seqID NZ_CP066576.1.\n",
      "2024-02-25 13:21:06,469 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066568.1, seqID NZ_CP066568.1.\n",
      "2024-02-25 13:21:06,478 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066396.1, seqID NZ_CP066396.1.\n",
      "2024-02-25 13:21:06,488 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP066434.1, seqID NZ_CP066434.1.\n",
      "2024-02-25 13:21:06,497 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065928.1, seqID NZ_CP065928.1.\n",
      "2024-02-25 13:21:06,507 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP065527.1, seqID NZ_CP065527.1.\n",
      "2024-02-25 13:21:06,515 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP045602.1, seqID NZ_CP045602.1.\n",
      "2024-02-25 13:21:06,524 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP034949.1, seqID NZ_CP034949.1.\n",
      "2024-02-25 13:21:06,533 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP033041.1, seqID NZ_CP033041.1.\n",
      "2024-02-25 13:21:06,542 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP032308.1, seqID NZ_CP032308.1.\n",
      "2024-02-25 13:21:06,550 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP025077.1, seqID NZ_CP025077.1.\n",
      "2024-02-25 13:21:06,560 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP016163.1, seqID NZ_CP016163.1.\n",
      "2024-02-25 13:21:06,570 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP012430.1, seqID NZ_CP012430.1.\n",
      "2024-02-25 13:21:06,583 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP034169.1, seqID CP034169.1.\n",
      "2024-02-25 13:21:06,594 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP024590.1, seqID CP024590.1.\n",
      "2024-02-25 13:21:06,610 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP034169.1, seqID NZ_CP034169.1.\n",
      "2024-02-25 13:21:06,622 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP024590.1, seqID NZ_CP024590.1.\n",
      "2024-02-25 13:21:06,632 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP019814.1, seqID AP019814.1.\n",
      "2024-02-25 13:21:06,644 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP019814.1, seqID NZ_AP019814.1.\n",
      "2024-02-25 13:21:06,652 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain AP022822.1, seqID AP022822.1.\n",
      "2024-02-25 13:21:06,661 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_AP022822.1, seqID NZ_AP022822.1.\n",
      "2024-02-25 13:21:06,669 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116583.1, seqID CP116583.1.\n",
      "2024-02-25 13:21:06,678 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116583.1, seqID NZ_CP116583.1.\n",
      "2024-02-25 13:21:06,686 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP060720.1, seqID CP060720.1.\n",
      "2024-02-25 13:21:06,696 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP060720.1, seqID NZ_CP060720.1.\n",
      "2024-02-25 13:21:06,697 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for CP021874.1.\n",
      "2024-02-25 13:21:06,697 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP021874.1, seqID CP021874.1.\n",
      "2024-02-25 13:21:06,708 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP021874.1, seqID NZ_CP021874.1.\n",
      "2024-02-25 13:21:06,713 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP049886.1, seqID CP049886.1.\n",
      "2024-02-25 13:21:06,718 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP049886.1, seqID NZ_CP049886.1.\n",
      "2024-02-25 13:21:06,725 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP019609.1, seqID CP019609.1.\n",
      "2024-02-25 13:21:06,732 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP019609.1, seqID NZ_CP019609.1.\n",
      "2024-02-25 13:21:06,738 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP090216.1, seqID CP090216.1.\n",
      "2024-02-25 13:21:06,745 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097045.1, seqID CP097045.1.\n",
      "2024-02-25 13:21:06,751 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097044.1, seqID CP097044.1.\n",
      "2024-02-25 13:21:06,758 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097051.1, seqID CP097051.1.\n",
      "2024-02-25 13:21:06,764 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP081833.1, seqID CP081833.1.\n",
      "2024-02-25 13:21:06,770 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP090216.1, seqID NZ_CP090216.1.\n",
      "2024-02-25 13:21:06,777 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097045.1, seqID NZ_CP097045.1.\n",
      "2024-02-25 13:21:06,785 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097044.1, seqID NZ_CP097044.1.\n",
      "2024-02-25 13:21:06,791 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097051.1, seqID NZ_CP097051.1.\n",
      "2024-02-25 13:21:06,799 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP081833.1, seqID NZ_CP081833.1.\n",
      "2024-02-25 13:21:06,805 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP110237.1, seqID CP110237.1.\n",
      "2024-02-25 13:21:06,812 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP110232.1, seqID CP110232.1.\n",
      "2024-02-25 13:21:06,820 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP110237.1, seqID NZ_CP110237.1.\n",
      "2024-02-25 13:21:06,827 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP110232.1, seqID NZ_CP110232.1.\n",
      "2024-02-25 13:21:06,834 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116507.1, seqID CP116507.1.\n",
      "2024-02-25 13:21:06,840 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097072.1, seqID CP097072.1.\n",
      "2024-02-25 13:21:06,846 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097053.1, seqID CP097053.1.\n",
      "2024-02-25 13:21:06,853 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP097017.1, seqID CP097017.1.\n",
      "2024-02-25 13:21:06,860 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116507.1, seqID NZ_CP116507.1.\n",
      "2024-02-25 13:21:06,867 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097072.1, seqID NZ_CP097072.1.\n",
      "2024-02-25 13:21:06,874 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097053.1, seqID NZ_CP097053.1.\n",
      "2024-02-25 13:21:06,881 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP097017.1, seqID NZ_CP097017.1.\n",
      "2024-02-25 13:21:06,887 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP116503.1, seqID CP116503.1.\n",
      "2024-02-25 13:21:06,894 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP116503.1, seqID NZ_CP116503.1.\n",
      "2024-02-25 13:21:06,900 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain CP038865.1, seqID CP038865.1.\n",
      "2024-02-25 13:21:06,907 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain NZ_CP038865.1, seqID NZ_CP038865.1.\n",
      "2024-02-25 13:21:06,959 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161825.\n",
      "2024-02-25 13:21:06,959 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161825, seqID GCA_902161825_CONTIG_0.\n",
      "2024-02-25 13:21:06,959 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161825.\n",
      "2024-02-25 13:21:06,959 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161825, seqID GCA_902161825_CONTIG_3.\n",
      "2024-02-25 13:21:06,960 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159855.\n",
      "2024-02-25 13:21:06,960 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159855, seqID GCA_902159855_CONTIG_0.\n",
      "2024-02-25 13:21:06,960 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159855.\n",
      "2024-02-25 13:21:06,960 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159855, seqID GCA_902159855_CONTIG_12.\n",
      "2024-02-25 13:21:06,962 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159835.\n",
      "2024-02-25 13:21:06,962 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159835, seqID GCA_902159835_CONTIG_0.\n",
      "2024-02-25 13:21:06,962 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159835.\n",
      "2024-02-25 13:21:06,962 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159835, seqID GCA_902159835_CONTIG_13.\n",
      "2024-02-25 13:21:06,963 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158985.\n",
      "2024-02-25 13:21:06,963 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158985, seqID GCA_902158985_CONTIG_0.\n",
      "2024-02-25 13:21:06,963 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158985.\n",
      "2024-02-25 13:21:06,963 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158985, seqID GCA_902158985_CONTIG_13.\n",
      "2024-02-25 13:21:06,964 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160815.\n",
      "2024-02-25 13:21:06,964 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160815, seqID GCA_902160815_CONTIG_0.\n",
      "2024-02-25 13:21:06,964 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160815.\n",
      "2024-02-25 13:21:06,964 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160815, seqID GCA_902160815_CONTIG_6.\n",
      "2024-02-25 13:21:06,965 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160805.\n",
      "2024-02-25 13:21:06,965 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160805, seqID GCA_902160805_CONTIG_0.\n",
      "2024-02-25 13:21:06,966 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160805.\n",
      "2024-02-25 13:21:06,966 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160805, seqID GCA_902160805_CONTIG_6.\n",
      "2024-02-25 13:21:06,967 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164755.\n",
      "2024-02-25 13:21:06,967 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164755, seqID GCA_902164755_CONTIG_0.\n",
      "2024-02-25 13:21:06,967 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164755.\n",
      "2024-02-25 13:21:06,967 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164755, seqID GCA_902164755_CONTIG_2.\n",
      "2024-02-25 13:21:06,969 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164735.\n",
      "2024-02-25 13:21:06,969 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164735, seqID GCA_902164735_CONTIG_0.\n",
      "2024-02-25 13:21:06,969 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164735.\n",
      "2024-02-25 13:21:06,969 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164735, seqID GCA_902164735_CONTIG_2.\n",
      "2024-02-25 13:21:06,970 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159055.\n",
      "2024-02-25 13:21:06,970 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159055, seqID GCA_902159055_CONTIG_0.\n",
      "2024-02-25 13:21:06,970 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159055.\n",
      "2024-02-25 13:21:06,970 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159055, seqID GCA_902159055_CONTIG_2.\n",
      "2024-02-25 13:21:06,971 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159465.\n",
      "2024-02-25 13:21:06,971 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159465, seqID GCA_902159465_CONTIG_2.\n",
      "2024-02-25 13:21:06,971 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159465.\n",
      "2024-02-25 13:21:06,972 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159465, seqID GCA_902159465_CONTIG_8.\n",
      "2024-02-25 13:21:06,973 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163945.\n",
      "2024-02-25 13:21:06,973 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163945, seqID GCA_902163945_CONTIG_0.\n",
      "2024-02-25 13:21:06,973 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163945.\n",
      "2024-02-25 13:21:06,973 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163945, seqID GCA_902163945_CONTIG_8.\n",
      "2024-02-25 13:21:06,974 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162075.\n",
      "2024-02-25 13:21:06,974 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162075, seqID GCA_902162075_CONTIG_0.\n",
      "2024-02-25 13:21:06,974 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162075.\n",
      "2024-02-25 13:21:06,974 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162075, seqID GCA_902162075_CONTIG_4.\n",
      "2024-02-25 13:21:06,975 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159335.\n",
      "2024-02-25 13:21:06,975 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159335, seqID GCA_902159335_CONTIG_0.\n",
      "2024-02-25 13:21:06,975 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159335.\n",
      "2024-02-25 13:21:06,975 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159335, seqID GCA_902159335_CONTIG_4.\n",
      "2024-02-25 13:21:06,976 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164295.\n",
      "2024-02-25 13:21:06,976 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164295, seqID GCA_902164295_CONTIG_11.\n",
      "2024-02-25 13:21:06,976 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164295.\n",
      "2024-02-25 13:21:06,976 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164295, seqID GCA_902164295_CONTIG_12.\n",
      "2024-02-25 13:21:06,977 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161935.\n",
      "2024-02-25 13:21:06,977 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161935, seqID GCA_902161935_CONTIG_11.\n",
      "2024-02-25 13:21:06,977 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161935.\n",
      "2024-02-25 13:21:06,977 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161935, seqID GCA_902161935_CONTIG_12.\n",
      "2024-02-25 13:21:06,979 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161645.\n",
      "2024-02-25 13:21:06,979 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161645, seqID GCA_902161645_CONTIG_2.\n",
      "2024-02-25 13:21:06,979 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161645.\n",
      "2024-02-25 13:21:06,979 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161645, seqID GCA_902161645_CONTIG_4.\n",
      "2024-02-25 13:21:06,980 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161525.\n",
      "2024-02-25 13:21:06,980 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161525, seqID GCA_902161525_CONTIG_0.\n",
      "2024-02-25 13:21:06,980 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161525.\n",
      "2024-02-25 13:21:06,980 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161525, seqID GCA_902161525_CONTIG_5.\n",
      "2024-02-25 13:21:06,981 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161515.\n",
      "2024-02-25 13:21:06,981 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161515, seqID GCA_902161515_CONTIG_0.\n",
      "2024-02-25 13:21:06,981 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161515.\n",
      "2024-02-25 13:21:06,981 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161515, seqID GCA_902161515_CONTIG_4.\n",
      "2024-02-25 13:21:06,982 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159435.\n",
      "2024-02-25 13:21:06,982 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159435, seqID GCA_902159435_CONTIG_14.\n",
      "2024-02-25 13:21:06,983 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159435.\n",
      "2024-02-25 13:21:06,983 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159435, seqID GCA_902159435_CONTIG_9.\n",
      "2024-02-25 13:21:06,984 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158815.\n",
      "2024-02-25 13:21:06,984 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158815, seqID GCA_902158815_CONTIG_14.\n",
      "2024-02-25 13:21:06,984 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158815.\n",
      "2024-02-25 13:21:06,984 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158815, seqID GCA_902158815_CONTIG_15.\n",
      "2024-02-25 13:21:06,985 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158765.\n",
      "2024-02-25 13:21:06,985 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158765, seqID GCA_902158765_CONTIG_15.\n",
      "2024-02-25 13:21:06,985 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158765.\n",
      "2024-02-25 13:21:06,985 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158765, seqID GCA_902158765_CONTIG_9.\n",
      "2024-02-25 13:21:06,986 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166645.\n",
      "2024-02-25 13:21:06,986 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166645.\n",
      "2024-02-25 13:21:06,986 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166645, seqID GCA_902166645_CONTIG_0.\n",
      "2024-02-25 13:21:06,987 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165975.\n",
      "2024-02-25 13:21:06,987 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165975, seqID GCA_902165975_CONTIG_13.\n",
      "2024-02-25 13:21:06,987 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165975.\n",
      "2024-02-25 13:21:06,987 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165975, seqID GCA_902165975_CONTIG_8.\n",
      "2024-02-25 13:21:06,988 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164015.\n",
      "2024-02-25 13:21:06,988 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164015, seqID GCA_902164015_CONTIG_1.\n",
      "2024-02-25 13:21:06,988 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164015.\n",
      "2024-02-25 13:21:06,989 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164015, seqID GCA_902164015_CONTIG_14.\n",
      "2024-02-25 13:21:06,990 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166605.\n",
      "2024-02-25 13:21:06,990 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166605.\n",
      "2024-02-25 13:21:06,990 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166605, seqID GCA_902166605_CONTIG_0.\n",
      "2024-02-25 13:21:06,991 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166575.\n",
      "2024-02-25 13:21:06,991 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166575.\n",
      "2024-02-25 13:21:06,991 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166575, seqID GCA_902166575_CONTIG_0.\n",
      "2024-02-25 13:21:06,992 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163855.\n",
      "2024-02-25 13:21:06,992 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163855, seqID GCA_902163855_CONTIG_0.\n",
      "2024-02-25 13:21:06,992 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163855.\n",
      "2024-02-25 13:21:06,992 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163855, seqID GCA_902163855_CONTIG_1.\n",
      "2024-02-25 13:21:06,993 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162785.\n",
      "2024-02-25 13:21:06,993 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162785, seqID GCA_902162785_CONTIG_0.\n",
      "2024-02-25 13:21:06,993 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162785.\n",
      "2024-02-25 13:21:06,993 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162785, seqID GCA_902162785_CONTIG_1.\n",
      "2024-02-25 13:21:06,994 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160235.\n",
      "2024-02-25 13:21:06,994 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160235, seqID GCA_902160235_CONTIG_10.\n",
      "2024-02-25 13:21:06,994 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160235.\n",
      "2024-02-25 13:21:06,994 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160235, seqID GCA_902160235_CONTIG_3.\n",
      "2024-02-25 13:21:06,995 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160135.\n",
      "2024-02-25 13:21:06,995 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160135, seqID GCA_902160135_CONTIG_10.\n",
      "2024-02-25 13:21:06,996 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160135.\n",
      "2024-02-25 13:21:06,996 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160135, seqID GCA_902160135_CONTIG_3.\n",
      "2024-02-25 13:21:06,997 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163675.\n",
      "2024-02-25 13:21:06,997 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163675, seqID GCA_902163675_CONTIG_0.\n",
      "2024-02-25 13:21:06,997 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163675.\n",
      "2024-02-25 13:21:06,997 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163675, seqID GCA_902163675_CONTIG_3.\n",
      "2024-02-25 13:21:06,998 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163635.\n",
      "2024-02-25 13:21:06,998 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163635, seqID GCA_902163635_CONTIG_0.\n",
      "2024-02-25 13:21:06,998 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163635.\n",
      "2024-02-25 13:21:06,998 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163635, seqID GCA_902163635_CONTIG_3.\n",
      "2024-02-25 13:21:06,999 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163975.\n",
      "2024-02-25 13:21:06,999 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163975, seqID GCA_902163975_CONTIG_4.\n",
      "2024-02-25 13:21:06,999 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163975.\n",
      "2024-02-25 13:21:06,999 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163975, seqID GCA_902163975_CONTIG_5.\n",
      "2024-02-25 13:21:07,001 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158855.\n",
      "2024-02-25 13:21:07,001 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158855, seqID GCA_902158855_CONTIG_4.\n",
      "2024-02-25 13:21:07,001 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158855.\n",
      "2024-02-25 13:21:07,001 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158855, seqID GCA_902158855_CONTIG_5.\n",
      "2024-02-25 13:21:07,002 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161145.\n",
      "2024-02-25 13:21:07,002 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161145, seqID GCA_902161145_CONTIG_0.\n",
      "2024-02-25 13:21:07,002 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161145.\n",
      "2024-02-25 13:21:07,002 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161145, seqID GCA_902161145_CONTIG_4.\n",
      "2024-02-25 13:21:07,003 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161115.\n",
      "2024-02-25 13:21:07,003 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161115, seqID GCA_902161115_CONTIG_0.\n",
      "2024-02-25 13:21:07,003 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161115.\n",
      "2024-02-25 13:21:07,003 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161115, seqID GCA_902161115_CONTIG_4.\n",
      "2024-02-25 13:21:07,004 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160295.\n",
      "2024-02-25 13:21:07,004 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160295, seqID GCA_902160295_CONTIG_1.\n",
      "2024-02-25 13:21:07,004 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160295.\n",
      "2024-02-25 13:21:07,004 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160295, seqID GCA_902160295_CONTIG_4.\n",
      "2024-02-25 13:21:07,005 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160285.\n",
      "2024-02-25 13:21:07,005 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160285, seqID GCA_902160285_CONTIG_1.\n",
      "2024-02-25 13:21:07,005 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160285.\n",
      "2024-02-25 13:21:07,005 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160285, seqID GCA_902160285_CONTIG_3.\n",
      "2024-02-25 13:21:07,007 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163475.\n",
      "2024-02-25 13:21:07,007 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163475, seqID GCA_902163475_CONTIG_0.\n",
      "2024-02-25 13:21:07,007 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163475.\n",
      "2024-02-25 13:21:07,007 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163475, seqID GCA_902163475_CONTIG_5.\n",
      "2024-02-25 13:21:07,008 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163415.\n",
      "2024-02-25 13:21:07,008 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163415, seqID GCA_902163415_CONTIG_0.\n",
      "2024-02-25 13:21:07,008 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163415.\n",
      "2024-02-25 13:21:07,008 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163415, seqID GCA_902163415_CONTIG_4.\n",
      "2024-02-25 13:21:07,009 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163365.\n",
      "2024-02-25 13:21:07,009 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163365, seqID GCA_902163365_CONTIG_0.\n",
      "2024-02-25 13:21:07,009 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163365.\n",
      "2024-02-25 13:21:07,009 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163365, seqID GCA_902163365_CONTIG_4.\n",
      "2024-02-25 13:21:07,010 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163355.\n",
      "2024-02-25 13:21:07,010 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163355, seqID GCA_902163355_CONTIG_0.\n",
      "2024-02-25 13:21:07,011 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163355.\n",
      "2024-02-25 13:21:07,011 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163355, seqID GCA_902163355_CONTIG_4.\n",
      "2024-02-25 13:21:07,012 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163175.\n",
      "2024-02-25 13:21:07,012 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163175, seqID GCA_902163175_CONTIG_1.\n",
      "2024-02-25 13:21:07,012 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163175.\n",
      "2024-02-25 13:21:07,012 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163175, seqID GCA_902163175_CONTIG_3.\n",
      "2024-02-25 13:21:07,013 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163785.\n",
      "2024-02-25 13:21:07,013 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163785, seqID GCA_902163785_CONTIG_11.\n",
      "2024-02-25 13:21:07,013 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163785.\n",
      "2024-02-25 13:21:07,013 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163785, seqID GCA_902163785_CONTIG_3.\n",
      "2024-02-25 13:21:07,014 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162825.\n",
      "2024-02-25 13:21:07,014 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162825, seqID GCA_902162825_CONTIG_0.\n",
      "2024-02-25 13:21:07,014 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162825.\n",
      "2024-02-25 13:21:07,014 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162825, seqID GCA_902162825_CONTIG_11.\n",
      "2024-02-25 13:21:07,016 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162335.\n",
      "2024-02-25 13:21:07,016 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162335, seqID GCA_902162335_CONTIG_0.\n",
      "2024-02-25 13:21:07,016 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162335.\n",
      "2024-02-25 13:21:07,016 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162335, seqID GCA_902162335_CONTIG_9.\n",
      "2024-02-25 13:21:07,017 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162865.\n",
      "2024-02-25 13:21:07,017 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162865, seqID GCA_902162865_CONTIG_0.\n",
      "2024-02-25 13:21:07,017 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162865.\n",
      "2024-02-25 13:21:07,017 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162865, seqID GCA_902162865_CONTIG_5.\n",
      "2024-02-25 13:21:07,018 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162815.\n",
      "2024-02-25 13:21:07,018 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162815, seqID GCA_902162815_CONTIG_0.\n",
      "2024-02-25 13:21:07,019 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162815.\n",
      "2024-02-25 13:21:07,019 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162815, seqID GCA_902162815_CONTIG_5.\n",
      "2024-02-25 13:21:07,020 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162025.\n",
      "2024-02-25 13:21:07,020 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162025, seqID GCA_902162025_CONTIG_3.\n",
      "2024-02-25 13:21:07,020 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162025.\n",
      "2024-02-25 13:21:07,020 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162025, seqID GCA_902162025_CONTIG_4.\n",
      "2024-02-25 13:21:07,021 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161975.\n",
      "2024-02-25 13:21:07,021 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161975, seqID GCA_902161975_CONTIG_4.\n",
      "2024-02-25 13:21:07,021 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161975.\n",
      "2024-02-25 13:21:07,021 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161975, seqID GCA_902161975_CONTIG_5.\n",
      "2024-02-25 13:21:07,022 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161845.\n",
      "2024-02-25 13:21:07,022 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161845, seqID GCA_902161845_CONTIG_11.\n",
      "2024-02-25 13:21:07,022 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161845.\n",
      "2024-02-25 13:21:07,022 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161845, seqID GCA_902161845_CONTIG_2.\n",
      "2024-02-25 13:21:07,023 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165945.\n",
      "2024-02-25 13:21:07,023 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165945, seqID GCA_902165945_CONTIG_0.\n",
      "2024-02-25 13:21:07,024 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165945.\n",
      "2024-02-25 13:21:07,024 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165945, seqID GCA_902165945_CONTIG_5.\n",
      "2024-02-25 13:21:07,025 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164115.\n",
      "2024-02-25 13:21:07,025 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164115, seqID GCA_902164115_CONTIG_0.\n",
      "2024-02-25 13:21:07,025 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164115.\n",
      "2024-02-25 13:21:07,025 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164115, seqID GCA_902164115_CONTIG_4.\n",
      "2024-02-25 13:21:07,026 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161475.\n",
      "2024-02-25 13:21:07,026 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161475, seqID GCA_902161475_CONTIG_1.\n",
      "2024-02-25 13:21:07,026 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161475.\n",
      "2024-02-25 13:21:07,026 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161475, seqID GCA_902161475_CONTIG_2.\n",
      "2024-02-25 13:21:07,027 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159145.\n",
      "2024-02-25 13:21:07,027 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159145, seqID GCA_902159145_CONTIG_1.\n",
      "2024-02-25 13:21:07,027 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159145.\n",
      "2024-02-25 13:21:07,027 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159145, seqID GCA_902159145_CONTIG_2.\n",
      "2024-02-25 13:21:07,028 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163125.\n",
      "2024-02-25 13:21:07,028 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163125, seqID GCA_902163125_CONTIG_10.\n",
      "2024-02-25 13:21:07,029 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163125.\n",
      "2024-02-25 13:21:07,029 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163125, seqID GCA_902163125_CONTIG_3.\n",
      "2024-02-25 13:21:07,030 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165985.\n",
      "2024-02-25 13:21:07,030 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165985, seqID GCA_902165985_CONTIG_2.\n",
      "2024-02-25 13:21:07,030 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165985.\n",
      "2024-02-25 13:21:07,030 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165985, seqID GCA_902165985_CONTIG_4.\n",
      "2024-02-25 13:21:07,031 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161335.\n",
      "2024-02-25 13:21:07,031 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161335, seqID GCA_902161335_CONTIG_1.\n",
      "2024-02-25 13:21:07,031 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161335.\n",
      "2024-02-25 13:21:07,031 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161335, seqID GCA_902161335_CONTIG_3.\n",
      "2024-02-25 13:21:07,032 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160825.\n",
      "2024-02-25 13:21:07,032 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160825, seqID GCA_902160825_CONTIG_3.\n",
      "2024-02-25 13:21:07,033 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160825.\n",
      "2024-02-25 13:21:07,033 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160825, seqID GCA_902160825_CONTIG_5.\n",
      "2024-02-25 13:21:07,034 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160795.\n",
      "2024-02-25 13:21:07,034 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160795, seqID GCA_902160795_CONTIG_2.\n",
      "2024-02-25 13:21:07,034 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160795.\n",
      "2024-02-25 13:21:07,034 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160795, seqID GCA_902160795_CONTIG_4.\n",
      "2024-02-25 13:21:07,035 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163755.\n",
      "2024-02-25 13:21:07,035 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163755, seqID GCA_902163755_CONTIG_12.\n",
      "2024-02-25 13:21:07,035 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163755.\n",
      "2024-02-25 13:21:07,035 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163755, seqID GCA_902163755_CONTIG_2.\n",
      "2024-02-25 13:21:07,036 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159175.\n",
      "2024-02-25 13:21:07,036 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159175, seqID GCA_902159175_CONTIG_12.\n",
      "2024-02-25 13:21:07,036 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159175.\n",
      "2024-02-25 13:21:07,036 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159175, seqID GCA_902159175_CONTIG_2.\n",
      "2024-02-25 13:21:07,037 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164585.\n",
      "2024-02-25 13:21:07,037 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164585, seqID GCA_902164585_CONTIG_3.\n",
      "2024-02-25 13:21:07,038 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164585.\n",
      "2024-02-25 13:21:07,038 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164585, seqID GCA_902164585_CONTIG_5.\n",
      "2024-02-25 13:21:07,039 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160105.\n",
      "2024-02-25 13:21:07,039 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160105, seqID GCA_902160105_CONTIG_4.\n",
      "2024-02-25 13:21:07,039 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160105.\n",
      "2024-02-25 13:21:07,039 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160105, seqID GCA_902160105_CONTIG_6.\n",
      "2024-02-25 13:21:07,040 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160085.\n",
      "2024-02-25 13:21:07,040 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160085, seqID GCA_902160085_CONTIG_3.\n",
      "2024-02-25 13:21:07,040 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160085.\n",
      "2024-02-25 13:21:07,040 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160085, seqID GCA_902160085_CONTIG_5.\n",
      "2024-02-25 13:21:07,041 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160075.\n",
      "2024-02-25 13:21:07,041 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160075, seqID GCA_902160075_CONTIG_3.\n",
      "2024-02-25 13:21:07,041 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160075.\n",
      "2024-02-25 13:21:07,041 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160075, seqID GCA_902160075_CONTIG_5.\n",
      "2024-02-25 13:21:07,042 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159645.\n",
      "2024-02-25 13:21:07,042 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159645, seqID GCA_902159645_CONTIG_3.\n",
      "2024-02-25 13:21:07,042 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159645.\n",
      "2024-02-25 13:21:07,042 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159645, seqID GCA_902159645_CONTIG_5.\n",
      "2024-02-25 13:21:07,044 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165485.\n",
      "2024-02-25 13:21:07,044 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165485, seqID GCA_902165485_CONTIG_1.\n",
      "2024-02-25 13:21:07,044 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165485.\n",
      "2024-02-25 13:21:07,044 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165485, seqID GCA_902165485_CONTIG_13.\n",
      "2024-02-25 13:21:07,045 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160005.\n",
      "2024-02-25 13:21:07,045 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160005, seqID GCA_902160005_CONTIG_0.\n",
      "2024-02-25 13:21:07,045 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160005.\n",
      "2024-02-25 13:21:07,045 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160005, seqID GCA_902160005_CONTIG_4.\n",
      "2024-02-25 13:21:07,046 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159135.\n",
      "2024-02-25 13:21:07,046 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159135, seqID GCA_902159135_CONTIG_1.\n",
      "2024-02-25 13:21:07,046 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159135.\n",
      "2024-02-25 13:21:07,046 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159135, seqID GCA_902159135_CONTIG_2.\n",
      "2024-02-25 13:21:07,048 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159125.\n",
      "2024-02-25 13:21:07,048 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159125, seqID GCA_902159125_CONTIG_1.\n",
      "2024-02-25 13:21:07,048 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159125.\n",
      "2024-02-25 13:21:07,048 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159125, seqID GCA_902159125_CONTIG_2.\n",
      "2024-02-25 13:21:07,049 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159115.\n",
      "2024-02-25 13:21:07,049 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159115, seqID GCA_902159115_CONTIG_1.\n",
      "2024-02-25 13:21:07,049 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159115.\n",
      "2024-02-25 13:21:07,049 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159115, seqID GCA_902159115_CONTIG_2.\n",
      "2024-02-25 13:21:07,050 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162305.\n",
      "2024-02-25 13:21:07,051 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162305, seqID GCA_902162305_CONTIG_0.\n",
      "2024-02-25 13:21:07,051 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162305.\n",
      "2024-02-25 13:21:07,051 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162305, seqID GCA_902162305_CONTIG_10.\n",
      "2024-02-25 13:21:07,052 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158825.\n",
      "2024-02-25 13:21:07,052 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158825, seqID GCA_902158825_CONTIG_2.\n",
      "2024-02-25 13:21:07,052 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158825.\n",
      "2024-02-25 13:21:07,052 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158825, seqID GCA_902158825_CONTIG_9.\n",
      "2024-02-25 13:21:07,053 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163015.\n",
      "2024-02-25 13:21:07,053 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163015, seqID GCA_902163015_CONTIG_0.\n",
      "2024-02-25 13:21:07,053 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163015.\n",
      "2024-02-25 13:21:07,053 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163015, seqID GCA_902163015_CONTIG_4.\n",
      "2024-02-25 13:21:07,054 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159885.\n",
      "2024-02-25 13:21:07,054 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159885, seqID GCA_902159885_CONTIG_0.\n",
      "2024-02-25 13:21:07,055 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159885.\n",
      "2024-02-25 13:21:07,055 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159885, seqID GCA_902159885_CONTIG_5.\n",
      "2024-02-25 13:21:07,056 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159735.\n",
      "2024-02-25 13:21:07,056 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159735, seqID GCA_902159735_CONTIG_0.\n",
      "2024-02-25 13:21:07,056 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159735.\n",
      "2024-02-25 13:21:07,056 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159735, seqID GCA_902159735_CONTIG_7.\n",
      "2024-02-25 13:21:07,057 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163505.\n",
      "2024-02-25 13:21:07,057 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163505, seqID GCA_902163505_CONTIG_0.\n",
      "2024-02-25 13:21:07,057 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163505.\n",
      "2024-02-25 13:21:07,057 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163505, seqID GCA_902163505_CONTIG_3.\n",
      "2024-02-25 13:21:07,058 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163465.\n",
      "2024-02-25 13:21:07,058 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163465, seqID GCA_902163465_CONTIG_2.\n",
      "2024-02-25 13:21:07,058 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163465.\n",
      "2024-02-25 13:21:07,059 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163465, seqID GCA_902163465_CONTIG_4.\n",
      "2024-02-25 13:21:07,060 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163825.\n",
      "2024-02-25 13:21:07,060 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163825, seqID GCA_902163825_CONTIG_0.\n",
      "2024-02-25 13:21:07,060 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163825.\n",
      "2024-02-25 13:21:07,060 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163825, seqID GCA_902163825_CONTIG_19.\n",
      "2024-02-25 13:21:07,061 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163925.\n",
      "2024-02-25 13:21:07,061 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163925, seqID GCA_902163925_CONTIG_0.\n",
      "2024-02-25 13:21:07,061 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163925.\n",
      "2024-02-25 13:21:07,061 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163925, seqID GCA_902163925_CONTIG_8.\n",
      "2024-02-25 13:21:07,062 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162345.\n",
      "2024-02-25 13:21:07,062 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162345, seqID GCA_902162345_CONTIG_0.\n",
      "2024-02-25 13:21:07,062 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162345.\n",
      "2024-02-25 13:21:07,062 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162345, seqID GCA_902162345_CONTIG_1.\n",
      "2024-02-25 13:21:07,064 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162235.\n",
      "2024-02-25 13:21:07,064 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162235, seqID GCA_902162235_CONTIG_17.\n",
      "2024-02-25 13:21:07,064 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162235.\n",
      "2024-02-25 13:21:07,064 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162235, seqID GCA_902162235_CONTIG_2.\n",
      "2024-02-25 13:21:07,065 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159495.\n",
      "2024-02-25 13:21:07,065 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159495, seqID GCA_902159495_CONTIG_0.\n",
      "2024-02-25 13:21:07,065 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159495.\n",
      "2024-02-25 13:21:07,065 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159495, seqID GCA_902159495_CONTIG_5.\n",
      "2024-02-25 13:21:07,066 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164425.\n",
      "2024-02-25 13:21:07,066 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164425, seqID GCA_902164425_CONTIG_0.\n",
      "2024-02-25 13:21:07,066 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164425.\n",
      "2024-02-25 13:21:07,066 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164425, seqID GCA_902164425_CONTIG_3.\n",
      "2024-02-25 13:21:07,067 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162275.\n",
      "2024-02-25 13:21:07,068 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162275, seqID GCA_902162275_CONTIG_0.\n",
      "2024-02-25 13:21:07,068 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162275.\n",
      "2024-02-25 13:21:07,068 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162275, seqID GCA_902162275_CONTIG_4.\n",
      "2024-02-25 13:21:07,069 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162295.\n",
      "2024-02-25 13:21:07,069 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162295, seqID GCA_902162295_CONTIG_0.\n",
      "2024-02-25 13:21:07,069 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162295.\n",
      "2024-02-25 13:21:07,069 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162295, seqID GCA_902162295_CONTIG_1.\n",
      "2024-02-25 13:21:07,070 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162885.\n",
      "2024-02-25 13:21:07,070 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162885, seqID GCA_902162885_CONTIG_0.\n",
      "2024-02-25 13:21:07,070 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162885.\n",
      "2024-02-25 13:21:07,070 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162885, seqID GCA_902162885_CONTIG_18.\n",
      "2024-02-25 13:21:07,072 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164775.\n",
      "2024-02-25 13:21:07,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164775, seqID GCA_902164775_CONTIG_0.\n",
      "2024-02-25 13:21:07,072 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164775.\n",
      "2024-02-25 13:21:07,072 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164775, seqID GCA_902164775_CONTIG_4.\n",
      "2024-02-25 13:21:07,073 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164485.\n",
      "2024-02-25 13:21:07,073 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164485, seqID GCA_902164485_CONTIG_0.\n",
      "2024-02-25 13:21:07,073 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164485.\n",
      "2024-02-25 13:21:07,073 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164485, seqID GCA_902164485_CONTIG_1.\n",
      "2024-02-25 13:21:07,074 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163805.\n",
      "2024-02-25 13:21:07,074 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163805, seqID GCA_902163805_CONTIG_0.\n",
      "2024-02-25 13:21:07,075 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163805.\n",
      "2024-02-25 13:21:07,075 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163805, seqID GCA_902163805_CONTIG_3.\n",
      "2024-02-25 13:21:07,076 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164765.\n",
      "2024-02-25 13:21:07,076 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164765, seqID GCA_902164765_CONTIG_12.\n",
      "2024-02-25 13:21:07,076 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164765.\n",
      "2024-02-25 13:21:07,076 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164765, seqID GCA_902164765_CONTIG_2.\n",
      "2024-02-25 13:21:07,077 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162145.\n",
      "2024-02-25 13:21:07,077 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162145, seqID GCA_902162145_CONTIG_12.\n",
      "2024-02-25 13:21:07,077 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162145.\n",
      "2024-02-25 13:21:07,077 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162145, seqID GCA_902162145_CONTIG_2.\n",
      "2024-02-25 13:21:07,079 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162085.\n",
      "2024-02-25 13:21:07,079 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162085, seqID GCA_902162085_CONTIG_12.\n",
      "2024-02-25 13:21:07,079 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162085.\n",
      "2024-02-25 13:21:07,079 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162085, seqID GCA_902162085_CONTIG_2.\n",
      "2024-02-25 13:21:07,080 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162095.\n",
      "2024-02-25 13:21:07,080 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162095, seqID GCA_902162095_CONTIG_14.\n",
      "2024-02-25 13:21:07,081 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162095.\n",
      "2024-02-25 13:21:07,081 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162095, seqID GCA_902162095_CONTIG_6.\n",
      "2024-02-25 13:21:07,082 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163995.\n",
      "2024-02-25 13:21:07,082 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163995, seqID GCA_902163995_CONTIG_20.\n",
      "2024-02-25 13:21:07,082 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163995.\n",
      "2024-02-25 13:21:07,082 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163995, seqID GCA_902163995_CONTIG_1.\n",
      "2024-02-25 13:21:07,083 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163935.\n",
      "2024-02-25 13:21:07,083 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163935, seqID GCA_902163935_CONTIG_20.\n",
      "2024-02-25 13:21:07,083 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163935.\n",
      "2024-02-25 13:21:07,083 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163935, seqID GCA_902163935_CONTIG_1.\n",
      "2024-02-25 13:21:07,084 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163165.\n",
      "2024-02-25 13:21:07,084 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163165, seqID GCA_902163165_CONTIG_20.\n",
      "2024-02-25 13:21:07,084 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163165.\n",
      "2024-02-25 13:21:07,084 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163165, seqID GCA_902163165_CONTIG_1.\n",
      "2024-02-25 13:21:07,086 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164315.\n",
      "2024-02-25 13:21:07,086 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164315, seqID GCA_902164315_CONTIG_0.\n",
      "2024-02-25 13:21:07,086 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164315.\n",
      "2024-02-25 13:21:07,086 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164315, seqID GCA_902164315_CONTIG_3.\n",
      "2024-02-25 13:21:07,087 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164245.\n",
      "2024-02-25 13:21:07,087 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164245, seqID GCA_902164245_CONTIG_0.\n",
      "2024-02-25 13:21:07,087 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164245.\n",
      "2024-02-25 13:21:07,087 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164245, seqID GCA_902164245_CONTIG_4.\n",
      "2024-02-25 13:21:07,088 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161705.\n",
      "2024-02-25 13:21:07,088 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161705, seqID GCA_902161705_CONTIG_1.\n",
      "2024-02-25 13:21:07,089 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161705.\n",
      "2024-02-25 13:21:07,089 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161705, seqID GCA_902161705_CONTIG_4.\n",
      "2024-02-25 13:21:07,090 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161695.\n",
      "2024-02-25 13:21:07,090 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161695, seqID GCA_902161695_CONTIG_1.\n",
      "2024-02-25 13:21:07,090 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161695.\n",
      "2024-02-25 13:21:07,090 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161695, seqID GCA_902161695_CONTIG_4.\n",
      "2024-02-25 13:21:07,091 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161565.\n",
      "2024-02-25 13:21:07,091 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161565, seqID GCA_902161565_CONTIG_0.\n",
      "2024-02-25 13:21:07,091 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161565.\n",
      "2024-02-25 13:21:07,091 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161565, seqID GCA_902161565_CONTIG_5.\n",
      "2024-02-25 13:21:07,092 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161665.\n",
      "2024-02-25 13:21:07,092 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161665, seqID GCA_902161665_CONTIG_0.\n",
      "2024-02-25 13:21:07,092 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161665.\n",
      "2024-02-25 13:21:07,092 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161665, seqID GCA_902161665_CONTIG_1.\n",
      "2024-02-25 13:21:07,094 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161575.\n",
      "2024-02-25 13:21:07,094 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161575, seqID GCA_902161575_CONTIG_0.\n",
      "2024-02-25 13:21:07,094 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161575.\n",
      "2024-02-25 13:21:07,094 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161575, seqID GCA_902161575_CONTIG_1.\n",
      "2024-02-25 13:21:07,095 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159395.\n",
      "2024-02-25 13:21:07,095 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159395, seqID GCA_902159395_CONTIG_0.\n",
      "2024-02-25 13:21:07,095 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159395.\n",
      "2024-02-25 13:21:07,095 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159395, seqID GCA_902159395_CONTIG_1.\n",
      "2024-02-25 13:21:07,096 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163875.\n",
      "2024-02-25 13:21:07,096 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163875, seqID GCA_902163875_CONTIG_17.\n",
      "2024-02-25 13:21:07,097 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163875.\n",
      "2024-02-25 13:21:07,097 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163875, seqID GCA_902163875_CONTIG_6.\n",
      "2024-02-25 13:21:07,098 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163965.\n",
      "2024-02-25 13:21:07,098 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163965, seqID GCA_902163965_CONTIG_0.\n",
      "2024-02-25 13:21:07,098 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163965.\n",
      "2024-02-25 13:21:07,098 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163965, seqID GCA_902163965_CONTIG_5.\n",
      "2024-02-25 13:21:07,099 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163815.\n",
      "2024-02-25 13:21:07,099 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163815, seqID GCA_902163815_CONTIG_10.\n",
      "2024-02-25 13:21:07,099 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163815.\n",
      "2024-02-25 13:21:07,099 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163815, seqID GCA_902163815_CONTIG_8.\n",
      "2024-02-25 13:21:07,100 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163865.\n",
      "2024-02-25 13:21:07,100 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163865, seqID GCA_902163865_CONTIG_2.\n",
      "2024-02-25 13:21:07,100 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163865.\n",
      "2024-02-25 13:21:07,100 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163865, seqID GCA_902163865_CONTIG_3.\n",
      "2024-02-25 13:21:07,102 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161385.\n",
      "2024-02-25 13:21:07,102 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161385, seqID GCA_902161385_CONTIG_25.\n",
      "2024-02-25 13:21:07,102 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161385.\n",
      "2024-02-25 13:21:07,102 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161385, seqID GCA_902161385_CONTIG_4.\n",
      "2024-02-25 13:21:07,103 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160835.\n",
      "2024-02-25 13:21:07,103 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160835, seqID GCA_902160835_CONTIG_25.\n",
      "2024-02-25 13:21:07,103 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160835.\n",
      "2024-02-25 13:21:07,103 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160835, seqID GCA_902160835_CONTIG_4.\n",
      "2024-02-25 13:21:07,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160785.\n",
      "2024-02-25 13:21:07,105 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160785, seqID GCA_902160785_CONTIG_25.\n",
      "2024-02-25 13:21:07,105 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160785.\n",
      "2024-02-25 13:21:07,105 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160785, seqID GCA_902160785_CONTIG_4.\n",
      "2024-02-25 13:21:07,106 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159275.\n",
      "2024-02-25 13:21:07,106 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159275, seqID GCA_902159275_CONTIG_25.\n",
      "2024-02-25 13:21:07,106 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159275.\n",
      "2024-02-25 13:21:07,106 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159275, seqID GCA_902159275_CONTIG_4.\n",
      "2024-02-25 13:21:07,107 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160165.\n",
      "2024-02-25 13:21:07,107 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160165, seqID GCA_902160165_CONTIG_14.\n",
      "2024-02-25 13:21:07,108 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160165.\n",
      "2024-02-25 13:21:07,108 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160165, seqID GCA_902160165_CONTIG_26.\n",
      "2024-02-25 13:21:07,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161125.\n",
      "2024-02-25 13:21:07,109 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161125, seqID GCA_902161125_CONTIG_0.\n",
      "2024-02-25 13:21:07,109 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161125.\n",
      "2024-02-25 13:21:07,109 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161125, seqID GCA_902161125_CONTIG_3.\n",
      "2024-02-25 13:21:07,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158895.\n",
      "2024-02-25 13:21:07,110 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158895, seqID GCA_902158895_CONTIG_0.\n",
      "2024-02-25 13:21:07,110 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158895.\n",
      "2024-02-25 13:21:07,110 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158895, seqID GCA_902158895_CONTIG_3.\n",
      "2024-02-25 13:21:07,111 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163885.\n",
      "2024-02-25 13:21:07,111 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163885, seqID GCA_902163885_CONTIG_0.\n",
      "2024-02-25 13:21:07,112 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163885.\n",
      "2024-02-25 13:21:07,112 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163885, seqID GCA_902163885_CONTIG_4.\n",
      "2024-02-25 13:21:07,113 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163715.\n",
      "2024-02-25 13:21:07,113 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163715, seqID GCA_902163715_CONTIG_0.\n",
      "2024-02-25 13:21:07,113 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163715.\n",
      "2024-02-25 13:21:07,113 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163715, seqID GCA_902163715_CONTIG_4.\n",
      "2024-02-25 13:21:07,114 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163705.\n",
      "2024-02-25 13:21:07,114 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163705, seqID GCA_902163705_CONTIG_0.\n",
      "2024-02-25 13:21:07,115 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163705.\n",
      "2024-02-25 13:21:07,115 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163705, seqID GCA_902163705_CONTIG_4.\n",
      "2024-02-25 13:21:07,116 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161085.\n",
      "2024-02-25 13:21:07,116 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161085, seqID GCA_902161085_CONTIG_15.\n",
      "2024-02-25 13:21:07,116 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161085.\n",
      "2024-02-25 13:21:07,116 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161085, seqID GCA_902161085_CONTIG_9.\n",
      "2024-02-25 13:21:07,117 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160185.\n",
      "2024-02-25 13:21:07,117 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160185, seqID GCA_902160185_CONTIG_11.\n",
      "2024-02-25 13:21:07,118 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160185.\n",
      "2024-02-25 13:21:07,118 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160185, seqID GCA_902160185_CONTIG_9.\n",
      "2024-02-25 13:21:07,119 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160125.\n",
      "2024-02-25 13:21:07,119 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160125, seqID GCA_902160125_CONTIG_15.\n",
      "2024-02-25 13:21:07,119 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160125.\n",
      "2024-02-25 13:21:07,119 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160125, seqID GCA_902160125_CONTIG_9.\n",
      "2024-02-25 13:21:07,120 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165815.\n",
      "2024-02-25 13:21:07,120 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165815, seqID GCA_902165815_CONTIG_0.\n",
      "2024-02-25 13:21:07,121 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165815.\n",
      "2024-02-25 13:21:07,121 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165815, seqID GCA_902165815_CONTIG_5.\n",
      "2024-02-25 13:21:07,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166775.\n",
      "2024-02-25 13:21:07,122 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166775.\n",
      "2024-02-25 13:21:07,122 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166775, seqID GCA_902166775_CONTIG_0.\n",
      "2024-02-25 13:21:07,123 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165735.\n",
      "2024-02-25 13:21:07,123 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165735, seqID GCA_902165735_CONTIG_1.\n",
      "2024-02-25 13:21:07,123 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165735.\n",
      "2024-02-25 13:21:07,123 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165735, seqID GCA_902165735_CONTIG_6.\n",
      "2024-02-25 13:21:07,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165775.\n",
      "2024-02-25 13:21:07,124 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165775, seqID GCA_902165775_CONTIG_1.\n",
      "2024-02-25 13:21:07,124 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165775.\n",
      "2024-02-25 13:21:07,124 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165775, seqID GCA_902165775_CONTIG_9.\n",
      "2024-02-25 13:21:07,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165755.\n",
      "2024-02-25 13:21:07,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165755, seqID GCA_902165755_CONTIG_2.\n",
      "2024-02-25 13:21:07,126 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165755.\n",
      "2024-02-25 13:21:07,126 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165755, seqID GCA_902165755_CONTIG_5.\n",
      "2024-02-25 13:21:07,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164855.\n",
      "2024-02-25 13:21:07,127 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164855, seqID GCA_902164855_CONTIG_0.\n",
      "2024-02-25 13:21:07,127 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164855.\n",
      "2024-02-25 13:21:07,127 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164855, seqID GCA_902164855_CONTIG_1.\n",
      "2024-02-25 13:21:07,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164845.\n",
      "2024-02-25 13:21:07,128 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164845, seqID GCA_902164845_CONTIG_0.\n",
      "2024-02-25 13:21:07,128 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164845.\n",
      "2024-02-25 13:21:07,128 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164845, seqID GCA_902164845_CONTIG_1.\n",
      "2024-02-25 13:21:07,130 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163155.\n",
      "2024-02-25 13:21:07,130 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163155, seqID GCA_902163155_CONTIG_0.\n",
      "2024-02-25 13:21:07,130 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163155.\n",
      "2024-02-25 13:21:07,130 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163155, seqID GCA_902163155_CONTIG_1.\n",
      "2024-02-25 13:21:07,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165785.\n",
      "2024-02-25 13:21:07,131 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165785, seqID GCA_902165785_CONTIG_1.\n",
      "2024-02-25 13:21:07,131 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165785.\n",
      "2024-02-25 13:21:07,131 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165785, seqID GCA_902165785_CONTIG_3.\n",
      "2024-02-25 13:21:07,132 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163735.\n",
      "2024-02-25 13:21:07,132 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163735, seqID GCA_902163735_CONTIG_1.\n",
      "2024-02-25 13:21:07,133 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163735.\n",
      "2024-02-25 13:21:07,133 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163735, seqID GCA_902163735_CONTIG_3.\n",
      "2024-02-25 13:21:07,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158935.\n",
      "2024-02-25 13:21:07,134 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158935, seqID GCA_902158935_CONTIG_2.\n",
      "2024-02-25 13:21:07,134 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158935.\n",
      "2024-02-25 13:21:07,134 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158935, seqID GCA_902158935_CONTIG_4.\n",
      "2024-02-25 13:21:07,135 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163695.\n",
      "2024-02-25 13:21:07,135 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163695, seqID GCA_902163695_CONTIG_0.\n",
      "2024-02-25 13:21:07,135 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163695.\n",
      "2024-02-25 13:21:07,135 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163695, seqID GCA_902163695_CONTIG_4.\n",
      "2024-02-25 13:21:07,136 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163655.\n",
      "2024-02-25 13:21:07,136 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163655, seqID GCA_902163655_CONTIG_0.\n",
      "2024-02-25 13:21:07,137 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163655.\n",
      "2024-02-25 13:21:07,137 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163655, seqID GCA_902163655_CONTIG_7.\n",
      "2024-02-25 13:21:07,138 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166395.\n",
      "2024-02-25 13:21:07,138 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166395.\n",
      "2024-02-25 13:21:07,138 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166395, seqID GCA_902166395_CONTIG_0.\n",
      "2024-02-25 13:21:07,139 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163915.\n",
      "2024-02-25 13:21:07,139 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163915, seqID GCA_902163915_CONTIG_1.\n",
      "2024-02-25 13:21:07,139 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163915.\n",
      "2024-02-25 13:21:07,139 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163915, seqID GCA_902163915_CONTIG_4.\n",
      "2024-02-25 13:21:07,140 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160045.\n",
      "2024-02-25 13:21:07,140 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160045, seqID GCA_902160045_CONTIG_0.\n",
      "2024-02-25 13:21:07,140 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160045.\n",
      "2024-02-25 13:21:07,141 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160045, seqID GCA_902160045_CONTIG_4.\n",
      "2024-02-25 13:21:07,142 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159955.\n",
      "2024-02-25 13:21:07,142 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159955, seqID GCA_902159955_CONTIG_0.\n",
      "2024-02-25 13:21:07,142 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159955.\n",
      "2024-02-25 13:21:07,142 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159955, seqID GCA_902159955_CONTIG_4.\n",
      "2024-02-25 13:21:07,143 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159845.\n",
      "2024-02-25 13:21:07,143 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159845, seqID GCA_902159845_CONTIG_0.\n",
      "2024-02-25 13:21:07,143 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159845.\n",
      "2024-02-25 13:21:07,143 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159845, seqID GCA_902159845_CONTIG_5.\n",
      "2024-02-25 13:21:07,144 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159825.\n",
      "2024-02-25 13:21:07,144 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159825, seqID GCA_902159825_CONTIG_0.\n",
      "2024-02-25 13:21:07,145 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159825.\n",
      "2024-02-25 13:21:07,145 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159825, seqID GCA_902159825_CONTIG_4.\n",
      "2024-02-25 13:21:07,146 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165465.\n",
      "2024-02-25 13:21:07,146 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165465, seqID GCA_902165465_CONTIG_11.\n",
      "2024-02-25 13:21:07,146 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165465.\n",
      "2024-02-25 13:21:07,146 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165465, seqID GCA_902165465_CONTIG_4.\n",
      "2024-02-25 13:21:07,147 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159295.\n",
      "2024-02-25 13:21:07,147 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159295, seqID GCA_902159295_CONTIG_1.\n",
      "2024-02-25 13:21:07,148 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159295.\n",
      "2024-02-25 13:21:07,148 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159295, seqID GCA_902159295_CONTIG_9.\n",
      "2024-02-25 13:21:07,149 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163625.\n",
      "2024-02-25 13:21:07,149 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163625, seqID GCA_902163625_CONTIG_0.\n",
      "2024-02-25 13:21:07,149 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163625.\n",
      "2024-02-25 13:21:07,149 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163625, seqID GCA_902163625_CONTIG_4.\n",
      "2024-02-25 13:21:07,150 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159315.\n",
      "2024-02-25 13:21:07,150 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159315, seqID GCA_902159315_CONTIG_4.\n",
      "2024-02-25 13:21:07,150 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159315.\n",
      "2024-02-25 13:21:07,150 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159315, seqID GCA_902159315_CONTIG_5.\n",
      "2024-02-25 13:21:07,151 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158795.\n",
      "2024-02-25 13:21:07,151 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158795, seqID GCA_902158795_CONTIG_4.\n",
      "2024-02-25 13:21:07,151 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158795.\n",
      "2024-02-25 13:21:07,151 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158795, seqID GCA_902158795_CONTIG_5.\n",
      "2024-02-25 13:21:07,152 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159155.\n",
      "2024-02-25 13:21:07,153 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159155, seqID GCA_902159155_CONTIG_0.\n",
      "2024-02-25 13:21:07,153 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159155.\n",
      "2024-02-25 13:21:07,153 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159155, seqID GCA_902159155_CONTIG_4.\n",
      "2024-02-25 13:21:07,154 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165295.\n",
      "2024-02-25 13:21:07,154 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165295, seqID GCA_902165295_CONTIG_3.\n",
      "2024-02-25 13:21:07,154 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165295.\n",
      "2024-02-25 13:21:07,154 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165295, seqID GCA_902165295_CONTIG_8.\n",
      "2024-02-25 13:21:07,155 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159755.\n",
      "2024-02-25 13:21:07,155 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159755, seqID GCA_902159755_CONTIG_6.\n",
      "2024-02-25 13:21:07,155 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159755.\n",
      "2024-02-25 13:21:07,155 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159755, seqID GCA_902159755_CONTIG_8.\n",
      "2024-02-25 13:21:07,157 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159705.\n",
      "2024-02-25 13:21:07,157 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159705, seqID GCA_902159705_CONTIG_7.\n",
      "2024-02-25 13:21:07,157 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159705.\n",
      "2024-02-25 13:21:07,157 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159705, seqID GCA_902159705_CONTIG_9.\n",
      "2024-02-25 13:21:07,158 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166505.\n",
      "2024-02-25 13:21:07,158 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166505, seqID GCA_902166505_CONTIG_0.\n",
      "2024-02-25 13:21:07,158 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166505.\n",
      "2024-02-25 13:21:07,158 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166505, seqID GCA_902166505_CONTIG_4.\n",
      "2024-02-25 13:21:07,159 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160985.\n",
      "2024-02-25 13:21:07,159 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160985, seqID GCA_902160985_CONTIG_0.\n",
      "2024-02-25 13:21:07,160 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160985.\n",
      "2024-02-25 13:21:07,160 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160985, seqID GCA_902160985_CONTIG_4.\n",
      "2024-02-25 13:21:07,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160965.\n",
      "2024-02-25 13:21:07,161 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160965, seqID GCA_902160965_CONTIG_0.\n",
      "2024-02-25 13:21:07,161 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160965.\n",
      "2024-02-25 13:21:07,161 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160965, seqID GCA_902160965_CONTIG_3.\n",
      "2024-02-25 13:21:07,162 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159935.\n",
      "2024-02-25 13:21:07,162 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159935, seqID GCA_902159935_CONTIG_0.\n",
      "2024-02-25 13:21:07,162 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159935.\n",
      "2024-02-25 13:21:07,162 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159935, seqID GCA_902159935_CONTIG_4.\n",
      "2024-02-25 13:21:07,163 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164075.\n",
      "2024-02-25 13:21:07,163 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164075, seqID GCA_902164075_CONTIG_0.\n",
      "2024-02-25 13:21:07,163 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164075.\n",
      "2024-02-25 13:21:07,163 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164075, seqID GCA_902164075_CONTIG_1.\n",
      "2024-02-25 13:21:07,165 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163585.\n",
      "2024-02-25 13:21:07,165 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163585, seqID GCA_902163585_CONTIG_0.\n",
      "2024-02-25 13:21:07,165 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163585.\n",
      "2024-02-25 13:21:07,165 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163585, seqID GCA_902163585_CONTIG_1.\n",
      "2024-02-25 13:21:07,166 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158875.\n",
      "2024-02-25 13:21:07,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158875, seqID GCA_902158875_CONTIG_0.\n",
      "2024-02-25 13:21:07,166 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158875.\n",
      "2024-02-25 13:21:07,166 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158875, seqID GCA_902158875_CONTIG_3.\n",
      "2024-02-25 13:21:07,168 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164645.\n",
      "2024-02-25 13:21:07,168 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164645.\n",
      "2024-02-25 13:21:07,168 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902164645, seqID GCA_902164645_CONTIG_0.\n",
      "2024-02-25 13:21:07,169 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164125.\n",
      "2024-02-25 13:21:07,169 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164125, seqID GCA_902164125_CONTIG_0.\n",
      "2024-02-25 13:21:07,169 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164125.\n",
      "2024-02-25 13:21:07,169 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164125, seqID GCA_902164125_CONTIG_4.\n",
      "2024-02-25 13:21:07,170 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165915.\n",
      "2024-02-25 13:21:07,170 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165915, seqID GCA_902165915_CONTIG_0.\n",
      "2024-02-25 13:21:07,170 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165915.\n",
      "2024-02-25 13:21:07,170 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165915, seqID GCA_902165915_CONTIG_4.\n",
      "2024-02-25 13:21:07,171 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159255.\n",
      "2024-02-25 13:21:07,171 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159255, seqID GCA_902159255_CONTIG_0.\n",
      "2024-02-25 13:21:07,172 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159255.\n",
      "2024-02-25 13:21:07,172 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159255, seqID GCA_902159255_CONTIG_4.\n",
      "2024-02-25 13:21:07,173 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160465.\n",
      "2024-02-25 13:21:07,173 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160465, seqID GCA_902160465_CONTIG_0.\n",
      "2024-02-25 13:21:07,173 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160465.\n",
      "2024-02-25 13:21:07,173 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160465, seqID GCA_902160465_CONTIG_1.\n",
      "2024-02-25 13:21:07,175 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159185.\n",
      "2024-02-25 13:21:07,175 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159185, seqID GCA_902159185_CONTIG_0.\n",
      "2024-02-25 13:21:07,175 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159185.\n",
      "2024-02-25 13:21:07,175 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159185, seqID GCA_902159185_CONTIG_3.\n",
      "2024-02-25 13:21:07,176 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159165.\n",
      "2024-02-25 13:21:07,176 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159165, seqID GCA_902159165_CONTIG_0.\n",
      "2024-02-25 13:21:07,176 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159165.\n",
      "2024-02-25 13:21:07,176 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159165, seqID GCA_902159165_CONTIG_3.\n",
      "2024-02-25 13:21:07,177 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163725.\n",
      "2024-02-25 13:21:07,177 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163725, seqID GCA_902163725_CONTIG_0.\n",
      "2024-02-25 13:21:07,177 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163725.\n",
      "2024-02-25 13:21:07,177 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163725, seqID GCA_902163725_CONTIG_1.\n",
      "2024-02-25 13:21:07,178 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158905.\n",
      "2024-02-25 13:21:07,178 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158905, seqID GCA_902158905_CONTIG_1.\n",
      "2024-02-25 13:21:07,179 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158905.\n",
      "2024-02-25 13:21:07,179 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158905, seqID GCA_902158905_CONTIG_2.\n",
      "2024-02-25 13:21:07,180 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158995.\n",
      "2024-02-25 13:21:07,180 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158995, seqID GCA_902158995_CONTIG_0.\n",
      "2024-02-25 13:21:07,180 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158995.\n",
      "2024-02-25 13:21:07,180 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158995, seqID GCA_902158995_CONTIG_8.\n",
      "2024-02-25 13:21:07,181 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164945.\n",
      "2024-02-25 13:21:07,181 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164945, seqID GCA_902164945_CONTIG_14.\n",
      "2024-02-25 13:21:07,181 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164945.\n",
      "2024-02-25 13:21:07,181 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164945, seqID GCA_902164945_CONTIG_9.\n",
      "2024-02-25 13:21:07,183 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164515.\n",
      "2024-02-25 13:21:07,183 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164515, seqID GCA_902164515_CONTIG_7.\n",
      "2024-02-25 13:21:07,183 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164515.\n",
      "2024-02-25 13:21:07,183 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164515, seqID GCA_902164515_CONTIG_8.\n",
      "2024-02-25 13:21:07,184 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163295.\n",
      "2024-02-25 13:21:07,184 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163295, seqID GCA_902163295_CONTIG_7.\n",
      "2024-02-25 13:21:07,184 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163295.\n",
      "2024-02-25 13:21:07,184 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163295, seqID GCA_902163295_CONTIG_8.\n",
      "2024-02-25 13:21:07,185 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163265.\n",
      "2024-02-25 13:21:07,185 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163265, seqID GCA_902163265_CONTIG_7.\n",
      "2024-02-25 13:21:07,185 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163265.\n",
      "2024-02-25 13:21:07,185 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163265, seqID GCA_902163265_CONTIG_8.\n",
      "2024-02-25 13:21:07,186 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162935.\n",
      "2024-02-25 13:21:07,186 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162935, seqID GCA_902162935_CONTIG_7.\n",
      "2024-02-25 13:21:07,186 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162935.\n",
      "2024-02-25 13:21:07,186 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162935, seqID GCA_902162935_CONTIG_8.\n",
      "2024-02-25 13:21:07,188 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166695.\n",
      "2024-02-25 13:21:07,188 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166695.\n",
      "2024-02-25 13:21:07,188 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166695, seqID GCA_902166695_CONTIG_0.\n",
      "2024-02-25 13:21:07,189 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164395.\n",
      "2024-02-25 13:21:07,189 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164395, seqID GCA_902164395_CONTIG_0.\n",
      "2024-02-25 13:21:07,189 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164395.\n",
      "2024-02-25 13:21:07,189 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164395, seqID GCA_902164395_CONTIG_1.\n",
      "2024-02-25 13:21:07,190 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164385.\n",
      "2024-02-25 13:21:07,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164385, seqID GCA_902164385_CONTIG_0.\n",
      "2024-02-25 13:21:07,190 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164385.\n",
      "2024-02-25 13:21:07,190 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164385, seqID GCA_902164385_CONTIG_4.\n",
      "2024-02-25 13:21:07,191 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161995.\n",
      "2024-02-25 13:21:07,191 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161995, seqID GCA_902161995_CONTIG_0.\n",
      "2024-02-25 13:21:07,192 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161995.\n",
      "2024-02-25 13:21:07,192 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161995, seqID GCA_902161995_CONTIG_4.\n",
      "2024-02-25 13:21:07,193 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161915.\n",
      "2024-02-25 13:21:07,193 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161915, seqID GCA_902161915_CONTIG_0.\n",
      "2024-02-25 13:21:07,193 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161915.\n",
      "2024-02-25 13:21:07,193 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161915, seqID GCA_902161915_CONTIG_3.\n",
      "2024-02-25 13:21:07,194 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161905.\n",
      "2024-02-25 13:21:07,194 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161905, seqID GCA_902161905_CONTIG_0.\n",
      "2024-02-25 13:21:07,194 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161905.\n",
      "2024-02-25 13:21:07,194 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161905, seqID GCA_902161905_CONTIG_5.\n",
      "2024-02-25 13:21:07,195 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161895.\n",
      "2024-02-25 13:21:07,195 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161895, seqID GCA_902161895_CONTIG_0.\n",
      "2024-02-25 13:21:07,196 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161895.\n",
      "2024-02-25 13:21:07,196 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161895, seqID GCA_902161895_CONTIG_5.\n",
      "2024-02-25 13:21:07,197 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161465.\n",
      "2024-02-25 13:21:07,197 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161465, seqID GCA_902161465_CONTIG_4.\n",
      "2024-02-25 13:21:07,197 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161465.\n",
      "2024-02-25 13:21:07,197 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161465, seqID GCA_902161465_CONTIG_6.\n",
      "2024-02-25 13:21:07,198 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161445.\n",
      "2024-02-25 13:21:07,198 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161445, seqID GCA_902161445_CONTIG_5.\n",
      "2024-02-25 13:21:07,198 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161445.\n",
      "2024-02-25 13:21:07,198 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161445, seqID GCA_902161445_CONTIG_8.\n",
      "2024-02-25 13:21:07,199 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159445.\n",
      "2024-02-25 13:21:07,199 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159445, seqID GCA_902159445_CONTIG_0.\n",
      "2024-02-25 13:21:07,200 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159445.\n",
      "2024-02-25 13:21:07,200 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159445, seqID GCA_902159445_CONTIG_4.\n",
      "2024-02-25 13:21:07,201 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161375.\n",
      "2024-02-25 13:21:07,201 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161375, seqID GCA_902161375_CONTIG_0.\n",
      "2024-02-25 13:21:07,201 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161375.\n",
      "2024-02-25 13:21:07,201 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161375, seqID GCA_902161375_CONTIG_6.\n",
      "2024-02-25 13:21:07,202 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161855.\n",
      "2024-02-25 13:21:07,202 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161855, seqID GCA_902161855_CONTIG_1.\n",
      "2024-02-25 13:21:07,202 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161855.\n",
      "2024-02-25 13:21:07,202 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161855, seqID GCA_902161855_CONTIG_4.\n",
      "2024-02-25 13:21:07,204 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161805.\n",
      "2024-02-25 13:21:07,204 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161805, seqID GCA_902161805_CONTIG_1.\n",
      "2024-02-25 13:21:07,204 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161805.\n",
      "2024-02-25 13:21:07,204 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161805, seqID GCA_902161805_CONTIG_4.\n",
      "2024-02-25 13:21:07,205 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165345.\n",
      "2024-02-25 13:21:07,205 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165345, seqID GCA_902165345_CONTIG_0.\n",
      "2024-02-25 13:21:07,205 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165345.\n",
      "2024-02-25 13:21:07,205 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165345, seqID GCA_902165345_CONTIG_4.\n",
      "2024-02-25 13:21:07,207 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163795.\n",
      "2024-02-25 13:21:07,207 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163795, seqID GCA_902163795_CONTIG_0.\n",
      "2024-02-25 13:21:07,207 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163795.\n",
      "2024-02-25 13:21:07,207 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163795, seqID GCA_902163795_CONTIG_1.\n",
      "2024-02-25 13:21:07,208 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163645.\n",
      "2024-02-25 13:21:07,208 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163645, seqID GCA_902163645_CONTIG_0.\n",
      "2024-02-25 13:21:07,208 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163645.\n",
      "2024-02-25 13:21:07,208 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163645, seqID GCA_902163645_CONTIG_4.\n",
      "2024-02-25 13:21:07,209 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160035.\n",
      "2024-02-25 13:21:07,209 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160035, seqID GCA_902160035_CONTIG_0.\n",
      "2024-02-25 13:21:07,209 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160035.\n",
      "2024-02-25 13:21:07,209 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160035, seqID GCA_902160035_CONTIG_4.\n",
      "2024-02-25 13:21:07,210 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159945.\n",
      "2024-02-25 13:21:07,210 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159945, seqID GCA_902159945_CONTIG_0.\n",
      "2024-02-25 13:21:07,210 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159945.\n",
      "2024-02-25 13:21:07,210 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159945, seqID GCA_902159945_CONTIG_4.\n",
      "2024-02-25 13:21:07,211 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159915.\n",
      "2024-02-25 13:21:07,212 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159915, seqID GCA_902159915_CONTIG_0.\n",
      "2024-02-25 13:21:07,212 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159915.\n",
      "2024-02-25 13:21:07,212 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159915, seqID GCA_902159915_CONTIG_4.\n",
      "2024-02-25 13:21:07,213 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159895.\n",
      "2024-02-25 13:21:07,213 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159895, seqID GCA_902159895_CONTIG_0.\n",
      "2024-02-25 13:21:07,213 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159895.\n",
      "2024-02-25 13:21:07,213 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159895, seqID GCA_902159895_CONTIG_4.\n",
      "2024-02-25 13:21:07,214 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159085.\n",
      "2024-02-25 13:21:07,214 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159085, seqID GCA_902159085_CONTIG_0.\n",
      "2024-02-25 13:21:07,214 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159085.\n",
      "2024-02-25 13:21:07,214 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159085, seqID GCA_902159085_CONTIG_4.\n",
      "2024-02-25 13:21:07,215 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160655.\n",
      "2024-02-25 13:21:07,216 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160655, seqID GCA_902160655_CONTIG_3.\n",
      "2024-02-25 13:21:07,216 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160655.\n",
      "2024-02-25 13:21:07,216 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160655, seqID GCA_902160655_CONTIG_4.\n",
      "2024-02-25 13:21:07,217 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160575.\n",
      "2024-02-25 13:21:07,217 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160575, seqID GCA_902160575_CONTIG_1.\n",
      "2024-02-25 13:21:07,217 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160575.\n",
      "2024-02-25 13:21:07,217 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160575, seqID GCA_902160575_CONTIG_7.\n",
      "2024-02-25 13:21:07,218 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160565.\n",
      "2024-02-25 13:21:07,218 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160565, seqID GCA_902160565_CONTIG_1.\n",
      "2024-02-25 13:21:07,219 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160565.\n",
      "2024-02-25 13:21:07,219 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160565, seqID GCA_902160565_CONTIG_7.\n",
      "2024-02-25 13:21:07,220 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160555.\n",
      "2024-02-25 13:21:07,220 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160555, seqID GCA_902160555_CONTIG_3.\n",
      "2024-02-25 13:21:07,220 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160555.\n",
      "2024-02-25 13:21:07,220 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160555, seqID GCA_902160555_CONTIG_4.\n",
      "2024-02-25 13:21:07,222 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166815.\n",
      "2024-02-25 13:21:07,222 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166815.\n",
      "2024-02-25 13:21:07,222 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166815, seqID GCA_902166815_CONTIG_0.\n",
      "2024-02-25 13:21:07,223 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163685.\n",
      "2024-02-25 13:21:07,223 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163685, seqID GCA_902163685_CONTIG_0.\n",
      "2024-02-25 13:21:07,223 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163685.\n",
      "2024-02-25 13:21:07,223 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163685, seqID GCA_902163685_CONTIG_4.\n",
      "2024-02-25 13:21:07,224 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160615.\n",
      "2024-02-25 13:21:07,224 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160615, seqID GCA_902160615_CONTIG_0.\n",
      "2024-02-25 13:21:07,224 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160615.\n",
      "2024-02-25 13:21:07,224 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160615, seqID GCA_902160615_CONTIG_4.\n",
      "2024-02-25 13:21:07,225 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160545.\n",
      "2024-02-25 13:21:07,225 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160545, seqID GCA_902160545_CONTIG_0.\n",
      "2024-02-25 13:21:07,225 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160545.\n",
      "2024-02-25 13:21:07,226 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160545, seqID GCA_902160545_CONTIG_4.\n",
      "2024-02-25 13:21:07,227 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160535.\n",
      "2024-02-25 13:21:07,227 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160535, seqID GCA_902160535_CONTIG_0.\n",
      "2024-02-25 13:21:07,227 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160535.\n",
      "2024-02-25 13:21:07,227 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160535, seqID GCA_902160535_CONTIG_4.\n",
      "2024-02-25 13:21:07,228 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160505.\n",
      "2024-02-25 13:21:07,228 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160505, seqID GCA_902160505_CONTIG_0.\n",
      "2024-02-25 13:21:07,228 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160505.\n",
      "2024-02-25 13:21:07,228 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160505, seqID GCA_902160505_CONTIG_4.\n",
      "2024-02-25 13:21:07,229 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160595.\n",
      "2024-02-25 13:21:07,229 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160595, seqID GCA_902160595_CONTIG_0.\n",
      "2024-02-25 13:21:07,229 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160595.\n",
      "2024-02-25 13:21:07,229 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160595, seqID GCA_902160595_CONTIG_4.\n",
      "2024-02-25 13:21:07,230 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160455.\n",
      "2024-02-25 13:21:07,230 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160455, seqID GCA_902160455_CONTIG_0.\n",
      "2024-02-25 13:21:07,231 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160455.\n",
      "2024-02-25 13:21:07,231 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160455, seqID GCA_902160455_CONTIG_4.\n",
      "2024-02-25 13:21:07,232 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159565.\n",
      "2024-02-25 13:21:07,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159565, seqID GCA_902159565_CONTIG_0.\n",
      "2024-02-25 13:21:07,232 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159565.\n",
      "2024-02-25 13:21:07,232 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159565, seqID GCA_902159565_CONTIG_4.\n",
      "2024-02-25 13:21:07,233 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162995.\n",
      "2024-02-25 13:21:07,233 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162995, seqID GCA_902162995_CONTIG_0.\n",
      "2024-02-25 13:21:07,233 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162995.\n",
      "2024-02-25 13:21:07,233 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162995, seqID GCA_902162995_CONTIG_4.\n",
      "2024-02-25 13:21:07,234 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162855.\n",
      "2024-02-25 13:21:07,234 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162855, seqID GCA_902162855_CONTIG_0.\n",
      "2024-02-25 13:21:07,235 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162855.\n",
      "2024-02-25 13:21:07,235 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162855, seqID GCA_902162855_CONTIG_3.\n",
      "2024-02-25 13:21:07,236 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158925.\n",
      "2024-02-25 13:21:07,236 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158925, seqID GCA_902158925_CONTIG_0.\n",
      "2024-02-25 13:21:07,236 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158925.\n",
      "2024-02-25 13:21:07,236 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158925, seqID GCA_902158925_CONTIG_2.\n",
      "2024-02-25 13:21:07,237 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159405.\n",
      "2024-02-25 13:21:07,237 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159405, seqID GCA_902159405_CONTIG_3.\n",
      "2024-02-25 13:21:07,238 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159405.\n",
      "2024-02-25 13:21:07,238 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159405, seqID GCA_902159405_CONTIG_8.\n",
      "2024-02-25 13:21:07,239 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163895.\n",
      "2024-02-25 13:21:07,239 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163895, seqID GCA_902163895_CONTIG_4.\n",
      "2024-02-25 13:21:07,239 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163895.\n",
      "2024-02-25 13:21:07,239 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163895, seqID GCA_902163895_CONTIG_7.\n",
      "2024-02-25 13:21:07,240 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162135.\n",
      "2024-02-25 13:21:07,240 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162135, seqID GCA_902162135_CONTIG_3.\n",
      "2024-02-25 13:21:07,240 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162135.\n",
      "2024-02-25 13:21:07,240 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162135, seqID GCA_902162135_CONTIG_7.\n",
      "2024-02-25 13:21:07,241 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161865.\n",
      "2024-02-25 13:21:07,241 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161865, seqID GCA_902161865_CONTIG_3.\n",
      "2024-02-25 13:21:07,242 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161865.\n",
      "2024-02-25 13:21:07,242 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161865, seqID GCA_902161865_CONTIG_5.\n",
      "2024-02-25 13:21:07,243 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161745.\n",
      "2024-02-25 13:21:07,243 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161745, seqID GCA_902161745_CONTIG_3.\n",
      "2024-02-25 13:21:07,243 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161745.\n",
      "2024-02-25 13:21:07,243 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161745, seqID GCA_902161745_CONTIG_4.\n",
      "2024-02-25 13:21:07,244 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162985.\n",
      "2024-02-25 13:21:07,244 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162985, seqID GCA_902162985_CONTIG_10.\n",
      "2024-02-25 13:21:07,244 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162985.\n",
      "2024-02-25 13:21:07,244 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162985, seqID GCA_902162985_CONTIG_7.\n",
      "2024-02-25 13:21:07,245 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163025.\n",
      "2024-02-25 13:21:07,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163025, seqID GCA_902163025_CONTIG_11.\n",
      "2024-02-25 13:21:07,246 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163025.\n",
      "2024-02-25 13:21:07,246 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163025, seqID GCA_902163025_CONTIG_30.\n",
      "2024-02-25 13:21:07,247 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164695.\n",
      "2024-02-25 13:21:07,247 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164695.\n",
      "2024-02-25 13:21:07,247 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902164695, seqID GCA_902164695_CONTIG_0.\n",
      "2024-02-25 13:21:07,248 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160975.\n",
      "2024-02-25 13:21:07,248 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160975, seqID GCA_902160975_CONTIG_10.\n",
      "2024-02-25 13:21:07,248 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160975.\n",
      "2024-02-25 13:21:07,248 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160975, seqID GCA_902160975_CONTIG_12.\n",
      "2024-02-25 13:21:07,250 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159005.\n",
      "2024-02-25 13:21:07,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159005, seqID GCA_902159005_CONTIG_10.\n",
      "2024-02-25 13:21:07,250 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159005.\n",
      "2024-02-25 13:21:07,250 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159005, seqID GCA_902159005_CONTIG_12.\n",
      "2024-02-25 13:21:07,251 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160015.\n",
      "2024-02-25 13:21:07,251 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160015, seqID GCA_902160015_CONTIG_0.\n",
      "2024-02-25 13:21:07,252 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160015.\n",
      "2024-02-25 13:21:07,252 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160015, seqID GCA_902160015_CONTIG_4.\n",
      "2024-02-25 13:21:07,253 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159975.\n",
      "2024-02-25 13:21:07,253 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159975, seqID GCA_902159975_CONTIG_0.\n",
      "2024-02-25 13:21:07,253 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159975.\n",
      "2024-02-25 13:21:07,253 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159975, seqID GCA_902159975_CONTIG_4.\n",
      "2024-02-25 13:21:07,254 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163845.\n",
      "2024-02-25 13:21:07,254 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163845, seqID GCA_902163845_CONTIG_13.\n",
      "2024-02-25 13:21:07,255 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163845.\n",
      "2024-02-25 13:21:07,255 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163845, seqID GCA_902163845_CONTIG_7.\n",
      "2024-02-25 13:21:07,256 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165805.\n",
      "2024-02-25 13:21:07,256 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165805, seqID GCA_902165805_CONTIG_0.\n",
      "2024-02-25 13:21:07,256 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165805.\n",
      "2024-02-25 13:21:07,256 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165805, seqID GCA_902165805_CONTIG_2.\n",
      "2024-02-25 13:21:07,257 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165575.\n",
      "2024-02-25 13:21:07,257 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165575, seqID GCA_902165575_CONTIG_4.\n",
      "2024-02-25 13:21:07,258 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165575.\n",
      "2024-02-25 13:21:07,258 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165575, seqID GCA_902165575_CONTIG_7.\n",
      "2024-02-25 13:21:07,259 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165365.\n",
      "2024-02-25 13:21:07,259 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165365, seqID GCA_902165365_CONTIG_0.\n",
      "2024-02-25 13:21:07,259 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165365.\n",
      "2024-02-25 13:21:07,259 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165365, seqID GCA_902165365_CONTIG_2.\n",
      "2024-02-25 13:21:07,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165335.\n",
      "2024-02-25 13:21:07,260 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165335, seqID GCA_902165335_CONTIG_0.\n",
      "2024-02-25 13:21:07,260 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165335.\n",
      "2024-02-25 13:21:07,260 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165335, seqID GCA_902165335_CONTIG_5.\n",
      "2024-02-25 13:21:07,261 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165185.\n",
      "2024-02-25 13:21:07,261 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165185, seqID GCA_902165185_CONTIG_0.\n",
      "2024-02-25 13:21:07,262 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165185.\n",
      "2024-02-25 13:21:07,262 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165185, seqID GCA_902165185_CONTIG_5.\n",
      "2024-02-25 13:21:07,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165105.\n",
      "2024-02-25 13:21:07,263 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165105, seqID GCA_902165105_CONTIG_0.\n",
      "2024-02-25 13:21:07,263 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165105.\n",
      "2024-02-25 13:21:07,263 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165105, seqID GCA_902165105_CONTIG_2.\n",
      "2024-02-25 13:21:07,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165055.\n",
      "2024-02-25 13:21:07,264 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165055, seqID GCA_902165055_CONTIG_0.\n",
      "2024-02-25 13:21:07,264 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165055.\n",
      "2024-02-25 13:21:07,264 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165055, seqID GCA_902165055_CONTIG_4.\n",
      "2024-02-25 13:21:07,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164955.\n",
      "2024-02-25 13:21:07,266 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164955, seqID GCA_902164955_CONTIG_0.\n",
      "2024-02-25 13:21:07,266 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164955.\n",
      "2024-02-25 13:21:07,266 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164955, seqID GCA_902164955_CONTIG_5.\n",
      "2024-02-25 13:21:07,267 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164885.\n",
      "2024-02-25 13:21:07,267 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164885, seqID GCA_902164885_CONTIG_0.\n",
      "2024-02-25 13:21:07,267 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164885.\n",
      "2024-02-25 13:21:07,267 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164885, seqID GCA_902164885_CONTIG_2.\n",
      "2024-02-25 13:21:07,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164495.\n",
      "2024-02-25 13:21:07,269 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164495, seqID GCA_902164495_CONTIG_4.\n",
      "2024-02-25 13:21:07,269 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164495.\n",
      "2024-02-25 13:21:07,269 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164495, seqID GCA_902164495_CONTIG_6.\n",
      "2024-02-25 13:21:07,270 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164435.\n",
      "2024-02-25 13:21:07,270 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164435, seqID GCA_902164435_CONTIG_4.\n",
      "2024-02-25 13:21:07,270 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164435.\n",
      "2024-02-25 13:21:07,270 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164435, seqID GCA_902164435_CONTIG_8.\n",
      "2024-02-25 13:21:07,271 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163135.\n",
      "2024-02-25 13:21:07,272 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163135, seqID GCA_902163135_CONTIG_4.\n",
      "2024-02-25 13:21:07,272 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163135.\n",
      "2024-02-25 13:21:07,272 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163135, seqID GCA_902163135_CONTIG_8.\n",
      "2024-02-25 13:21:07,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162965.\n",
      "2024-02-25 13:21:07,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162965, seqID GCA_902162965_CONTIG_4.\n",
      "2024-02-25 13:21:07,273 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162965.\n",
      "2024-02-25 13:21:07,273 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162965, seqID GCA_902162965_CONTIG_7.\n",
      "2024-02-25 13:21:07,274 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162805.\n",
      "2024-02-25 13:21:07,274 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162805, seqID GCA_902162805_CONTIG_4.\n",
      "2024-02-25 13:21:07,274 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162805.\n",
      "2024-02-25 13:21:07,274 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162805, seqID GCA_902162805_CONTIG_7.\n",
      "2024-02-25 13:21:07,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162795.\n",
      "2024-02-25 13:21:07,276 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162795, seqID GCA_902162795_CONTIG_4.\n",
      "2024-02-25 13:21:07,276 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162795.\n",
      "2024-02-25 13:21:07,276 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162795, seqID GCA_902162795_CONTIG_7.\n",
      "2024-02-25 13:21:07,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162245.\n",
      "2024-02-25 13:21:07,277 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162245, seqID GCA_902162245_CONTIG_4.\n",
      "2024-02-25 13:21:07,277 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162245.\n",
      "2024-02-25 13:21:07,277 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162245, seqID GCA_902162245_CONTIG_8.\n",
      "2024-02-25 13:21:07,278 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162265.\n",
      "2024-02-25 13:21:07,279 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162265, seqID GCA_902162265_CONTIG_0.\n",
      "2024-02-25 13:21:07,279 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162265.\n",
      "2024-02-25 13:21:07,279 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162265, seqID GCA_902162265_CONTIG_1.\n",
      "2024-02-25 13:21:07,280 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166665.\n",
      "2024-02-25 13:21:07,280 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166665.\n",
      "2024-02-25 13:21:07,280 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166665, seqID GCA_902166665_CONTIG_0.\n",
      "2024-02-25 13:21:07,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164445.\n",
      "2024-02-25 13:21:07,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164445, seqID GCA_902164445_CONTIG_4.\n",
      "2024-02-25 13:21:07,281 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164445.\n",
      "2024-02-25 13:21:07,281 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164445, seqID GCA_902164445_CONTIG_8.\n",
      "2024-02-25 13:21:07,282 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162285.\n",
      "2024-02-25 13:21:07,282 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162285, seqID GCA_902162285_CONTIG_2.\n",
      "2024-02-25 13:21:07,283 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162285.\n",
      "2024-02-25 13:21:07,283 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162285, seqID GCA_902162285_CONTIG_7.\n",
      "2024-02-25 13:21:07,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165085.\n",
      "2024-02-25 13:21:07,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165085, seqID GCA_902165085_CONTIG_0.\n",
      "2024-02-25 13:21:07,284 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165085.\n",
      "2024-02-25 13:21:07,284 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165085, seqID GCA_902165085_CONTIG_1.\n",
      "2024-02-25 13:21:07,285 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164785.\n",
      "2024-02-25 13:21:07,285 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164785, seqID GCA_902164785_CONTIG_5.\n",
      "2024-02-25 13:21:07,285 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164785.\n",
      "2024-02-25 13:21:07,285 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164785, seqID GCA_902164785_CONTIG_7.\n",
      "2024-02-25 13:21:07,287 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166685.\n",
      "2024-02-25 13:21:07,287 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166685.\n",
      "2024-02-25 13:21:07,287 [INFO - chronostrain.cli.make_db_json] - Created 2 merged markers for strain GCA_902166685, seqID GCA_902166685_CONTIG_0.\n",
      "2024-02-25 13:21:07,288 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164405.\n",
      "2024-02-25 13:21:07,288 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164405, seqID GCA_902164405_CONTIG_4.\n",
      "2024-02-25 13:21:07,288 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164405.\n",
      "2024-02-25 13:21:07,288 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164405, seqID GCA_902164405_CONTIG_8.\n",
      "2024-02-25 13:21:07,289 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164345.\n",
      "2024-02-25 13:21:07,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164345, seqID GCA_902164345_CONTIG_3.\n",
      "2024-02-25 13:21:07,290 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164345.\n",
      "2024-02-25 13:21:07,290 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164345, seqID GCA_902164345_CONTIG_6.\n",
      "2024-02-25 13:21:07,291 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164355.\n",
      "2024-02-25 13:21:07,291 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164355, seqID GCA_902164355_CONTIG_2.\n",
      "2024-02-25 13:21:07,291 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164355.\n",
      "2024-02-25 13:21:07,291 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164355, seqID GCA_902164355_CONTIG_8.\n",
      "2024-02-25 13:21:07,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164335.\n",
      "2024-02-25 13:21:07,292 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164335, seqID GCA_902164335_CONTIG_2.\n",
      "2024-02-25 13:21:07,292 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164335.\n",
      "2024-02-25 13:21:07,293 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164335, seqID GCA_902164335_CONTIG_9.\n",
      "2024-02-25 13:21:07,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162035.\n",
      "2024-02-25 13:21:07,294 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162035, seqID GCA_902162035_CONTIG_0.\n",
      "2024-02-25 13:21:07,294 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162035.\n",
      "2024-02-25 13:21:07,294 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162035, seqID GCA_902162035_CONTIG_4.\n",
      "2024-02-25 13:21:07,295 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164215.\n",
      "2024-02-25 13:21:07,295 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164215, seqID GCA_902164215_CONTIG_1.\n",
      "2024-02-25 13:21:07,295 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164215.\n",
      "2024-02-25 13:21:07,295 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164215, seqID GCA_902164215_CONTIG_6.\n",
      "2024-02-25 13:21:07,297 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161945.\n",
      "2024-02-25 13:21:07,297 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161945, seqID GCA_902161945_CONTIG_2.\n",
      "2024-02-25 13:21:07,297 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161945.\n",
      "2024-02-25 13:21:07,297 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161945, seqID GCA_902161945_CONTIG_7.\n",
      "2024-02-25 13:21:07,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161875.\n",
      "2024-02-25 13:21:07,298 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161875, seqID GCA_902161875_CONTIG_2.\n",
      "2024-02-25 13:21:07,298 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161875.\n",
      "2024-02-25 13:21:07,298 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161875, seqID GCA_902161875_CONTIG_6.\n",
      "2024-02-25 13:21:07,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161795.\n",
      "2024-02-25 13:21:07,299 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161795, seqID GCA_902161795_CONTIG_2.\n",
      "2024-02-25 13:21:07,299 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161795.\n",
      "2024-02-25 13:21:07,299 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161795, seqID GCA_902161795_CONTIG_5.\n",
      "2024-02-25 13:21:07,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161785.\n",
      "2024-02-25 13:21:07,301 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161785, seqID GCA_902161785_CONTIG_0.\n",
      "2024-02-25 13:21:07,301 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161785.\n",
      "2024-02-25 13:21:07,301 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161785, seqID GCA_902161785_CONTIG_3.\n",
      "2024-02-25 13:21:07,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164325.\n",
      "2024-02-25 13:21:07,302 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164325, seqID GCA_902164325_CONTIG_3.\n",
      "2024-02-25 13:21:07,302 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164325.\n",
      "2024-02-25 13:21:07,302 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164325, seqID GCA_902164325_CONTIG_9.\n",
      "2024-02-25 13:21:07,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164275.\n",
      "2024-02-25 13:21:07,304 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164275, seqID GCA_902164275_CONTIG_2.\n",
      "2024-02-25 13:21:07,304 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164275.\n",
      "2024-02-25 13:21:07,304 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164275, seqID GCA_902164275_CONTIG_8.\n",
      "2024-02-25 13:21:07,305 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164235.\n",
      "2024-02-25 13:21:07,305 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164235, seqID GCA_902164235_CONTIG_4.\n",
      "2024-02-25 13:21:07,305 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164235.\n",
      "2024-02-25 13:21:07,305 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164235, seqID GCA_902164235_CONTIG_7.\n",
      "2024-02-25 13:21:07,306 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159695.\n",
      "2024-02-25 13:21:07,307 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159695, seqID GCA_902159695_CONTIG_4.\n",
      "2024-02-25 13:21:07,307 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159695.\n",
      "2024-02-25 13:21:07,307 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159695, seqID GCA_902159695_CONTIG_7.\n",
      "2024-02-25 13:21:07,308 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161595.\n",
      "2024-02-25 13:21:07,308 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161595, seqID GCA_902161595_CONTIG_2.\n",
      "2024-02-25 13:21:07,308 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161595.\n",
      "2024-02-25 13:21:07,308 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161595, seqID GCA_902161595_CONTIG_8.\n",
      "2024-02-25 13:21:07,309 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159455.\n",
      "2024-02-25 13:21:07,309 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159455, seqID GCA_902159455_CONTIG_3.\n",
      "2024-02-25 13:21:07,309 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159455.\n",
      "2024-02-25 13:21:07,309 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159455, seqID GCA_902159455_CONTIG_7.\n",
      "2024-02-25 13:21:07,311 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164195.\n",
      "2024-02-25 13:21:07,311 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164195, seqID GCA_902164195_CONTIG_4.\n",
      "2024-02-25 13:21:07,311 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164195.\n",
      "2024-02-25 13:21:07,311 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164195, seqID GCA_902164195_CONTIG_7.\n",
      "2024-02-25 13:21:07,312 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164165.\n",
      "2024-02-25 13:21:07,312 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164165, seqID GCA_902164165_CONTIG_4.\n",
      "2024-02-25 13:21:07,312 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164165.\n",
      "2024-02-25 13:21:07,312 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164165, seqID GCA_902164165_CONTIG_8.\n",
      "2024-02-25 13:21:07,313 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159685.\n",
      "2024-02-25 13:21:07,314 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159685, seqID GCA_902159685_CONTIG_3.\n",
      "2024-02-25 13:21:07,314 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159685.\n",
      "2024-02-25 13:21:07,314 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159685, seqID GCA_902159685_CONTIG_6.\n",
      "2024-02-25 13:21:07,315 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164035.\n",
      "2024-02-25 13:21:07,315 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164035, seqID GCA_902164035_CONTIG_2.\n",
      "2024-02-25 13:21:07,315 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164035.\n",
      "2024-02-25 13:21:07,315 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164035, seqID GCA_902164035_CONTIG_5.\n",
      "2024-02-25 13:21:07,316 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163905.\n",
      "2024-02-25 13:21:07,316 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163905, seqID GCA_902163905_CONTIG_2.\n",
      "2024-02-25 13:21:07,317 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163905.\n",
      "2024-02-25 13:21:07,317 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163905, seqID GCA_902163905_CONTIG_8.\n",
      "2024-02-25 13:21:07,318 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163835.\n",
      "2024-02-25 13:21:07,318 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163835, seqID GCA_902163835_CONTIG_4.\n",
      "2024-02-25 13:21:07,318 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163835.\n",
      "2024-02-25 13:21:07,318 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163835, seqID GCA_902163835_CONTIG_8.\n",
      "2024-02-25 13:21:07,319 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163115.\n",
      "2024-02-25 13:21:07,319 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163115, seqID GCA_902163115_CONTIG_4.\n",
      "2024-02-25 13:21:07,319 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163115.\n",
      "2024-02-25 13:21:07,319 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163115, seqID GCA_902163115_CONTIG_9.\n",
      "2024-02-25 13:21:07,320 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161275.\n",
      "2024-02-25 13:21:07,320 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161275, seqID GCA_902161275_CONTIG_0.\n",
      "2024-02-25 13:21:07,321 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161275.\n",
      "2024-02-25 13:21:07,321 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161275, seqID GCA_902161275_CONTIG_3.\n",
      "2024-02-25 13:21:07,322 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165795.\n",
      "2024-02-25 13:21:07,322 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165795, seqID GCA_902165795_CONTIG_0.\n",
      "2024-02-25 13:21:07,322 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165795.\n",
      "2024-02-25 13:21:07,322 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165795, seqID GCA_902165795_CONTIG_5.\n",
      "2024-02-25 13:21:07,324 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164415.\n",
      "2024-02-25 13:21:07,324 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164415, seqID GCA_902164415_CONTIG_2.\n",
      "2024-02-25 13:21:07,324 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164415.\n",
      "2024-02-25 13:21:07,324 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164415, seqID GCA_902164415_CONTIG_8.\n",
      "2024-02-25 13:21:07,325 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163745.\n",
      "2024-02-25 13:21:07,325 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163745, seqID GCA_902163745_CONTIG_0.\n",
      "2024-02-25 13:21:07,325 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163745.\n",
      "2024-02-25 13:21:07,325 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163745, seqID GCA_902163745_CONTIG_4.\n",
      "2024-02-25 13:21:07,326 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163045.\n",
      "2024-02-25 13:21:07,326 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163045, seqID GCA_902163045_CONTIG_0.\n",
      "2024-02-25 13:21:07,326 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163045.\n",
      "2024-02-25 13:21:07,326 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163045, seqID GCA_902163045_CONTIG_2.\n",
      "2024-02-25 13:21:07,328 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162945.\n",
      "2024-02-25 13:21:07,328 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162945, seqID GCA_902162945_CONTIG_0.\n",
      "2024-02-25 13:21:07,328 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162945.\n",
      "2024-02-25 13:21:07,328 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162945, seqID GCA_902162945_CONTIG_4.\n",
      "2024-02-25 13:21:07,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165745.\n",
      "2024-02-25 13:21:07,329 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165745, seqID GCA_902165745_CONTIG_0.\n",
      "2024-02-25 13:21:07,329 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165745.\n",
      "2024-02-25 13:21:07,329 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165745, seqID GCA_902165745_CONTIG_2.\n",
      "2024-02-25 13:21:07,330 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165725.\n",
      "2024-02-25 13:21:07,330 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165725, seqID GCA_902165725_CONTIG_1.\n",
      "2024-02-25 13:21:07,331 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165725.\n",
      "2024-02-25 13:21:07,331 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165725, seqID GCA_902165725_CONTIG_14.\n",
      "2024-02-25 13:21:07,332 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164455.\n",
      "2024-02-25 13:21:07,332 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164455, seqID GCA_902164455_CONTIG_0.\n",
      "2024-02-25 13:21:07,332 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164455.\n",
      "2024-02-25 13:21:07,332 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164455, seqID GCA_902164455_CONTIG_2.\n",
      "2024-02-25 13:21:07,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163035.\n",
      "2024-02-25 13:21:07,333 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163035, seqID GCA_902163035_CONTIG_0.\n",
      "2024-02-25 13:21:07,333 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163035.\n",
      "2024-02-25 13:21:07,334 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163035, seqID GCA_902163035_CONTIG_1.\n",
      "2024-02-25 13:21:07,335 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163005.\n",
      "2024-02-25 13:21:07,335 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163005, seqID GCA_902163005_CONTIG_0.\n",
      "2024-02-25 13:21:07,335 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163005.\n",
      "2024-02-25 13:21:07,335 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163005, seqID GCA_902163005_CONTIG_2.\n",
      "2024-02-25 13:21:07,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162975.\n",
      "2024-02-25 13:21:07,336 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162975, seqID GCA_902162975_CONTIG_0.\n",
      "2024-02-25 13:21:07,336 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162975.\n",
      "2024-02-25 13:21:07,336 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162975, seqID GCA_902162975_CONTIG_2.\n",
      "2024-02-25 13:21:07,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158945.\n",
      "2024-02-25 13:21:07,338 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158945, seqID GCA_902158945_CONTIG_0.\n",
      "2024-02-25 13:21:07,338 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158945.\n",
      "2024-02-25 13:21:07,338 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158945, seqID GCA_902158945_CONTIG_1.\n",
      "2024-02-25 13:21:07,339 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161555.\n",
      "2024-02-25 13:21:07,339 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161555, seqID GCA_902161555_CONTIG_2.\n",
      "2024-02-25 13:21:07,339 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161555.\n",
      "2024-02-25 13:21:07,339 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161555, seqID GCA_902161555_CONTIG_6.\n",
      "2024-02-25 13:21:07,340 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161545.\n",
      "2024-02-25 13:21:07,340 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161545, seqID GCA_902161545_CONTIG_2.\n",
      "2024-02-25 13:21:07,341 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161545.\n",
      "2024-02-25 13:21:07,341 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161545, seqID GCA_902161545_CONTIG_7.\n",
      "2024-02-25 13:21:07,342 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165715.\n",
      "2024-02-25 13:21:07,342 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165715, seqID GCA_902165715_CONTIG_1.\n",
      "2024-02-25 13:21:07,342 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165715.\n",
      "2024-02-25 13:21:07,342 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165715, seqID GCA_902165715_CONTIG_14.\n",
      "2024-02-25 13:21:07,343 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165645.\n",
      "2024-02-25 13:21:07,343 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165645, seqID GCA_902165645_CONTIG_12.\n",
      "2024-02-25 13:21:07,343 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165645.\n",
      "2024-02-25 13:21:07,343 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165645, seqID GCA_902165645_CONTIG_2.\n",
      "2024-02-25 13:21:07,345 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164085.\n",
      "2024-02-25 13:21:07,345 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164085, seqID GCA_902164085_CONTIG_15.\n",
      "2024-02-25 13:21:07,345 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164085.\n",
      "2024-02-25 13:21:07,345 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164085, seqID GCA_902164085_CONTIG_3.\n",
      "2024-02-25 13:21:07,346 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163775.\n",
      "2024-02-25 13:21:07,346 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163775, seqID GCA_902163775_CONTIG_15.\n",
      "2024-02-25 13:21:07,346 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163775.\n",
      "2024-02-25 13:21:07,346 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163775, seqID GCA_902163775_CONTIG_3.\n",
      "2024-02-25 13:21:07,347 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163765.\n",
      "2024-02-25 13:21:07,347 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163765, seqID GCA_902163765_CONTIG_14.\n",
      "2024-02-25 13:21:07,348 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163765.\n",
      "2024-02-25 13:21:07,348 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163765, seqID GCA_902163765_CONTIG_3.\n",
      "2024-02-25 13:21:07,349 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162955.\n",
      "2024-02-25 13:21:07,349 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162955, seqID GCA_902162955_CONTIG_13.\n",
      "2024-02-25 13:21:07,349 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902162955.\n",
      "2024-02-25 13:21:07,349 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902162955, seqID GCA_902162955_CONTIG_2.\n",
      "2024-02-25 13:21:07,350 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159995.\n",
      "2024-02-25 13:21:07,350 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159995, seqID GCA_902159995_CONTIG_4.\n",
      "2024-02-25 13:21:07,350 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159995.\n",
      "2024-02-25 13:21:07,350 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159995, seqID GCA_902159995_CONTIG_8.\n",
      "2024-02-25 13:21:07,352 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159965.\n",
      "2024-02-25 13:21:07,352 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159965, seqID GCA_902159965_CONTIG_4.\n",
      "2024-02-25 13:21:07,352 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159965.\n",
      "2024-02-25 13:21:07,352 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159965, seqID GCA_902159965_CONTIG_7.\n",
      "2024-02-25 13:21:07,353 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164865.\n",
      "2024-02-25 13:21:07,353 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164865, seqID GCA_902164865_CONTIG_4.\n",
      "2024-02-25 13:21:07,353 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164865.\n",
      "2024-02-25 13:21:07,353 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164865, seqID GCA_902164865_CONTIG_8.\n",
      "2024-02-25 13:21:07,354 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165705.\n",
      "2024-02-25 13:21:07,355 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165705, seqID GCA_902165705_CONTIG_0.\n",
      "2024-02-25 13:21:07,355 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165705.\n",
      "2024-02-25 13:21:07,355 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165705, seqID GCA_902165705_CONTIG_1.\n",
      "2024-02-25 13:21:07,356 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165675.\n",
      "2024-02-25 13:21:07,356 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165675, seqID GCA_902165675_CONTIG_0.\n",
      "2024-02-25 13:21:07,356 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165675.\n",
      "2024-02-25 13:21:07,356 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165675, seqID GCA_902165675_CONTIG_3.\n",
      "2024-02-25 13:21:07,357 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165625.\n",
      "2024-02-25 13:21:07,357 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165625, seqID GCA_902165625_CONTIG_0.\n",
      "2024-02-25 13:21:07,358 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165625.\n",
      "2024-02-25 13:21:07,358 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165625, seqID GCA_902165625_CONTIG_4.\n",
      "2024-02-25 13:21:07,359 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165605.\n",
      "2024-02-25 13:21:07,359 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165605, seqID GCA_902165605_CONTIG_2.\n",
      "2024-02-25 13:21:07,359 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165605.\n",
      "2024-02-25 13:21:07,359 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165605, seqID GCA_902165605_CONTIG_8.\n",
      "2024-02-25 13:21:07,360 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165595.\n",
      "2024-02-25 13:21:07,360 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165595, seqID GCA_902165595_CONTIG_13.\n",
      "2024-02-25 13:21:07,360 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165595.\n",
      "2024-02-25 13:21:07,360 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165595, seqID GCA_902165595_CONTIG_3.\n",
      "2024-02-25 13:21:07,362 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165505.\n",
      "2024-02-25 13:21:07,362 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165505, seqID GCA_902165505_CONTIG_3.\n",
      "2024-02-25 13:21:07,362 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165505.\n",
      "2024-02-25 13:21:07,362 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165505, seqID GCA_902165505_CONTIG_8.\n",
      "2024-02-25 13:21:07,363 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165585.\n",
      "2024-02-25 13:21:07,363 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165585, seqID GCA_902165585_CONTIG_4.\n",
      "2024-02-25 13:21:07,363 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165585.\n",
      "2024-02-25 13:21:07,363 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165585, seqID GCA_902165585_CONTIG_9.\n",
      "2024-02-25 13:21:07,365 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165545.\n",
      "2024-02-25 13:21:07,365 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165545, seqID GCA_902165545_CONTIG_4.\n",
      "2024-02-25 13:21:07,365 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165545.\n",
      "2024-02-25 13:21:07,365 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165545, seqID GCA_902165545_CONTIG_9.\n",
      "2024-02-25 13:21:07,366 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165515.\n",
      "2024-02-25 13:21:07,366 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165515, seqID GCA_902165515_CONTIG_10.\n",
      "2024-02-25 13:21:07,366 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165515.\n",
      "2024-02-25 13:21:07,366 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165515, seqID GCA_902165515_CONTIG_2.\n",
      "2024-02-25 13:21:07,368 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165495.\n",
      "2024-02-25 13:21:07,368 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165495, seqID GCA_902165495_CONTIG_0.\n",
      "2024-02-25 13:21:07,368 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165495.\n",
      "2024-02-25 13:21:07,368 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165495, seqID GCA_902165495_CONTIG_1.\n",
      "2024-02-25 13:21:07,369 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165425.\n",
      "2024-02-25 13:21:07,369 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165425, seqID GCA_902165425_CONTIG_0.\n",
      "2024-02-25 13:21:07,369 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165425.\n",
      "2024-02-25 13:21:07,369 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165425, seqID GCA_902165425_CONTIG_1.\n",
      "2024-02-25 13:21:07,371 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159985.\n",
      "2024-02-25 13:21:07,371 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159985, seqID GCA_902159985_CONTIG_0.\n",
      "2024-02-25 13:21:07,371 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159985.\n",
      "2024-02-25 13:21:07,371 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159985, seqID GCA_902159985_CONTIG_1.\n",
      "2024-02-25 13:21:07,372 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165475.\n",
      "2024-02-25 13:21:07,372 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165475, seqID GCA_902165475_CONTIG_0.\n",
      "2024-02-25 13:21:07,373 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165475.\n",
      "2024-02-25 13:21:07,373 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165475, seqID GCA_902165475_CONTIG_1.\n",
      "2024-02-25 13:21:07,374 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165455.\n",
      "2024-02-25 13:21:07,374 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165455, seqID GCA_902165455_CONTIG_0.\n",
      "2024-02-25 13:21:07,374 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165455.\n",
      "2024-02-25 13:21:07,374 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165455, seqID GCA_902165455_CONTIG_1.\n",
      "2024-02-25 13:21:07,375 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163605.\n",
      "2024-02-25 13:21:07,375 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163605, seqID GCA_902163605_CONTIG_0.\n",
      "2024-02-25 13:21:07,375 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163605.\n",
      "2024-02-25 13:21:07,375 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163605, seqID GCA_902163605_CONTIG_4.\n",
      "2024-02-25 13:21:07,377 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163595.\n",
      "2024-02-25 13:21:07,377 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163595, seqID GCA_902163595_CONTIG_0.\n",
      "2024-02-25 13:21:07,377 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163595.\n",
      "2024-02-25 13:21:07,377 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163595, seqID GCA_902163595_CONTIG_4.\n",
      "2024-02-25 13:21:07,378 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163575.\n",
      "2024-02-25 13:21:07,378 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163575, seqID GCA_902163575_CONTIG_0.\n",
      "2024-02-25 13:21:07,378 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163575.\n",
      "2024-02-25 13:21:07,378 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163575, seqID GCA_902163575_CONTIG_3.\n",
      "2024-02-25 13:21:07,379 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165555.\n",
      "2024-02-25 13:21:07,379 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165555, seqID GCA_902165555_CONTIG_0.\n",
      "2024-02-25 13:21:07,379 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165555.\n",
      "2024-02-25 13:21:07,379 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165555, seqID GCA_902165555_CONTIG_1.\n",
      "2024-02-25 13:21:07,381 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163085.\n",
      "2024-02-25 13:21:07,381 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163085, seqID GCA_902163085_CONTIG_3.\n",
      "2024-02-25 13:21:07,381 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163085.\n",
      "2024-02-25 13:21:07,381 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163085, seqID GCA_902163085_CONTIG_7.\n",
      "2024-02-25 13:21:07,382 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164915.\n",
      "2024-02-25 13:21:07,382 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164915, seqID GCA_902164915_CONTIG_1.\n",
      "2024-02-25 13:21:07,382 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164915.\n",
      "2024-02-25 13:21:07,382 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164915, seqID GCA_902164915_CONTIG_4.\n",
      "2024-02-25 13:21:07,384 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163055.\n",
      "2024-02-25 13:21:07,384 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163055, seqID GCA_902163055_CONTIG_3.\n",
      "2024-02-25 13:21:07,384 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163055.\n",
      "2024-02-25 13:21:07,384 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163055, seqID GCA_902163055_CONTIG_7.\n",
      "2024-02-25 13:21:07,385 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158885.\n",
      "2024-02-25 13:21:07,385 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158885, seqID GCA_902158885_CONTIG_2.\n",
      "2024-02-25 13:21:07,385 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158885.\n",
      "2024-02-25 13:21:07,385 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158885, seqID GCA_902158885_CONTIG_5.\n",
      "2024-02-25 13:21:07,386 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160025.\n",
      "2024-02-25 13:21:07,386 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160025, seqID GCA_902160025_CONTIG_4.\n",
      "2024-02-25 13:21:07,386 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160025.\n",
      "2024-02-25 13:21:07,386 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160025, seqID GCA_902160025_CONTIG_7.\n",
      "2024-02-25 13:21:07,388 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165415.\n",
      "2024-02-25 13:21:07,388 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165415, seqID GCA_902165415_CONTIG_0.\n",
      "2024-02-25 13:21:07,388 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165415.\n",
      "2024-02-25 13:21:07,388 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165415, seqID GCA_902165415_CONTIG_1.\n",
      "2024-02-25 13:21:07,389 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165405.\n",
      "2024-02-25 13:21:07,389 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165405, seqID GCA_902165405_CONTIG_0.\n",
      "2024-02-25 13:21:07,389 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165405.\n",
      "2024-02-25 13:21:07,389 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165405, seqID GCA_902165405_CONTIG_1.\n",
      "2024-02-25 13:21:07,390 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165395.\n",
      "2024-02-25 13:21:07,390 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165395, seqID GCA_902165395_CONTIG_0.\n",
      "2024-02-25 13:21:07,391 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165395.\n",
      "2024-02-25 13:21:07,391 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165395, seqID GCA_902165395_CONTIG_5.\n",
      "2024-02-25 13:21:07,392 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166365.\n",
      "2024-02-25 13:21:07,392 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166365, seqID GCA_902166365_CONTIG_2.\n",
      "2024-02-25 13:21:07,392 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166365.\n",
      "2024-02-25 13:21:07,392 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166365, seqID GCA_902166365_CONTIG_7.\n",
      "2024-02-25 13:21:07,393 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165305.\n",
      "2024-02-25 13:21:07,393 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165305, seqID GCA_902165305_CONTIG_0.\n",
      "2024-02-25 13:21:07,393 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165305.\n",
      "2024-02-25 13:21:07,393 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165305, seqID GCA_902165305_CONTIG_4.\n",
      "2024-02-25 13:21:07,395 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165265.\n",
      "2024-02-25 13:21:07,395 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165265, seqID GCA_902165265_CONTIG_0.\n",
      "2024-02-25 13:21:07,395 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165265.\n",
      "2024-02-25 13:21:07,395 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165265, seqID GCA_902165265_CONTIG_4.\n",
      "2024-02-25 13:21:07,396 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165255.\n",
      "2024-02-25 13:21:07,396 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165255, seqID GCA_902165255_CONTIG_0.\n",
      "2024-02-25 13:21:07,396 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165255.\n",
      "2024-02-25 13:21:07,396 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165255, seqID GCA_902165255_CONTIG_1.\n",
      "2024-02-25 13:21:07,398 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165355.\n",
      "2024-02-25 13:21:07,398 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165355, seqID GCA_902165355_CONTIG_4.\n",
      "2024-02-25 13:21:07,398 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165355.\n",
      "2024-02-25 13:21:07,398 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165355, seqID GCA_902165355_CONTIG_8.\n",
      "2024-02-25 13:21:07,399 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165225.\n",
      "2024-02-25 13:21:07,399 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165225, seqID GCA_902165225_CONTIG_4.\n",
      "2024-02-25 13:21:07,399 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165225.\n",
      "2024-02-25 13:21:07,399 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165225, seqID GCA_902165225_CONTIG_7.\n",
      "2024-02-25 13:21:07,401 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163665.\n",
      "2024-02-25 13:21:07,401 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163665, seqID GCA_902163665_CONTIG_4.\n",
      "2024-02-25 13:21:07,401 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163665.\n",
      "2024-02-25 13:21:07,401 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163665, seqID GCA_902163665_CONTIG_9.\n",
      "2024-02-25 13:21:07,402 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158955.\n",
      "2024-02-25 13:21:07,402 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158955, seqID GCA_902158955_CONTIG_4.\n",
      "2024-02-25 13:21:07,402 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158955.\n",
      "2024-02-25 13:21:07,402 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158955, seqID GCA_902158955_CONTIG_8.\n",
      "2024-02-25 13:21:07,403 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166355.\n",
      "2024-02-25 13:21:07,403 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166355, seqID GCA_902166355_CONTIG_0.\n",
      "2024-02-25 13:21:07,403 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166355.\n",
      "2024-02-25 13:21:07,403 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166355, seqID GCA_902166355_CONTIG_1.\n",
      "2024-02-25 13:21:07,405 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165285.\n",
      "2024-02-25 13:21:07,405 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165285, seqID GCA_902165285_CONTIG_0.\n",
      "2024-02-25 13:21:07,405 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165285.\n",
      "2024-02-25 13:21:07,405 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165285, seqID GCA_902165285_CONTIG_1.\n",
      "2024-02-25 13:21:07,406 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161095.\n",
      "2024-02-25 13:21:07,406 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161095, seqID GCA_902161095_CONTIG_4.\n",
      "2024-02-25 13:21:07,407 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161095.\n",
      "2024-02-25 13:21:07,407 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161095, seqID GCA_902161095_CONTIG_9.\n",
      "2024-02-25 13:21:07,408 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161045.\n",
      "2024-02-25 13:21:07,408 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161045, seqID GCA_902161045_CONTIG_4.\n",
      "2024-02-25 13:21:07,408 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902161045.\n",
      "2024-02-25 13:21:07,408 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902161045, seqID GCA_902161045_CONTIG_9.\n",
      "2024-02-25 13:21:07,409 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166255.\n",
      "2024-02-25 13:21:07,409 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166255, seqID GCA_902166255_CONTIG_0.\n",
      "2024-02-25 13:21:07,409 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902166255.\n",
      "2024-02-25 13:21:07,409 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902166255, seqID GCA_902166255_CONTIG_9.\n",
      "2024-02-25 13:21:07,410 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165135.\n",
      "2024-02-25 13:21:07,410 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165135, seqID GCA_902165135_CONTIG_0.\n",
      "2024-02-25 13:21:07,411 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165135.\n",
      "2024-02-25 13:21:07,411 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165135, seqID GCA_902165135_CONTIG_8.\n",
      "2024-02-25 13:21:07,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163105.\n",
      "2024-02-25 13:21:07,412 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163105, seqID GCA_902163105_CONTIG_0.\n",
      "2024-02-25 13:21:07,412 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163105.\n",
      "2024-02-25 13:21:07,412 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163105, seqID GCA_902163105_CONTIG_8.\n",
      "2024-02-25 13:21:07,413 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163075.\n",
      "2024-02-25 13:21:07,413 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163075, seqID GCA_902163075_CONTIG_0.\n",
      "2024-02-25 13:21:07,414 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163075.\n",
      "2024-02-25 13:21:07,414 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163075, seqID GCA_902163075_CONTIG_8.\n",
      "2024-02-25 13:21:07,415 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163065.\n",
      "2024-02-25 13:21:07,415 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163065, seqID GCA_902163065_CONTIG_0.\n",
      "2024-02-25 13:21:07,415 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163065.\n",
      "2024-02-25 13:21:07,415 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163065, seqID GCA_902163065_CONTIG_8.\n",
      "2024-02-25 13:21:07,416 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165175.\n",
      "2024-02-25 13:21:07,416 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165175, seqID GCA_902165175_CONTIG_0.\n",
      "2024-02-25 13:21:07,416 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165175.\n",
      "2024-02-25 13:21:07,416 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165175, seqID GCA_902165175_CONTIG_2.\n",
      "2024-02-25 13:21:07,417 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165075.\n",
      "2024-02-25 13:21:07,417 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165075, seqID GCA_902165075_CONTIG_0.\n",
      "2024-02-25 13:21:07,418 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165075.\n",
      "2024-02-25 13:21:07,418 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165075, seqID GCA_902165075_CONTIG_2.\n",
      "2024-02-25 13:21:07,419 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165025.\n",
      "2024-02-25 13:21:07,419 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165025, seqID GCA_902165025_CONTIG_0.\n",
      "2024-02-25 13:21:07,419 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165025.\n",
      "2024-02-25 13:21:07,419 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165025, seqID GCA_902165025_CONTIG_5.\n",
      "2024-02-25 13:21:07,420 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158775.\n",
      "2024-02-25 13:21:07,420 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158775, seqID GCA_902158775_CONTIG_0.\n",
      "2024-02-25 13:21:07,420 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902158775.\n",
      "2024-02-25 13:21:07,421 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902158775, seqID GCA_902158775_CONTIG_4.\n",
      "2024-02-25 13:21:07,422 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160495.\n",
      "2024-02-25 13:21:07,422 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160495, seqID GCA_902160495_CONTIG_11.\n",
      "2024-02-25 13:21:07,422 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902160495.\n",
      "2024-02-25 13:21:07,422 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902160495, seqID GCA_902160495_CONTIG_14.\n",
      "2024-02-25 13:21:07,423 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159545.\n",
      "2024-02-25 13:21:07,423 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159545, seqID GCA_902159545_CONTIG_10.\n",
      "2024-02-25 13:21:07,423 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902159545.\n",
      "2024-02-25 13:21:07,423 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902159545, seqID GCA_902159545_CONTIG_3.\n",
      "2024-02-25 13:21:07,425 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164815.\n",
      "2024-02-25 13:21:07,425 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164815, seqID GCA_902164815_CONTIG_2.\n",
      "2024-02-25 13:21:07,425 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164815.\n",
      "2024-02-25 13:21:07,425 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164815, seqID GCA_902164815_CONTIG_4.\n",
      "2024-02-25 13:21:07,426 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164805.\n",
      "2024-02-25 13:21:07,427 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164805, seqID GCA_902164805_CONTIG_4.\n",
      "2024-02-25 13:21:07,427 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164805.\n",
      "2024-02-25 13:21:07,427 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164805, seqID GCA_902164805_CONTIG_8.\n",
      "2024-02-25 13:21:07,428 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165825.\n",
      "2024-02-25 13:21:07,428 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165825, seqID GCA_902165825_CONTIG_0.\n",
      "2024-02-25 13:21:07,428 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165825.\n",
      "2024-02-25 13:21:07,428 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165825, seqID GCA_902165825_CONTIG_2.\n",
      "2024-02-25 13:21:07,429 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163615.\n",
      "2024-02-25 13:21:07,429 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163615, seqID GCA_902163615_CONTIG_0.\n",
      "2024-02-25 13:21:07,429 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902163615.\n",
      "2024-02-25 13:21:07,429 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902163615, seqID GCA_902163615_CONTIG_3.\n",
      "2024-02-25 13:21:07,430 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164995.\n",
      "2024-02-25 13:21:07,431 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164995, seqID GCA_902164995_CONTIG_0.\n",
      "2024-02-25 13:21:07,431 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164995.\n",
      "2024-02-25 13:21:07,431 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164995, seqID GCA_902164995_CONTIG_3.\n",
      "2024-02-25 13:21:07,432 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165005.\n",
      "2024-02-25 13:21:07,432 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165005, seqID GCA_902165005_CONTIG_0.\n",
      "2024-02-25 13:21:07,432 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902165005.\n",
      "2024-02-25 13:21:07,432 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902165005, seqID GCA_902165005_CONTIG_5.\n",
      "2024-02-25 13:21:07,433 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164875.\n",
      "2024-02-25 13:21:07,433 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164875, seqID GCA_902164875_CONTIG_0.\n",
      "2024-02-25 13:21:07,433 [DEBUG - chronostrain.cli.make_db_json] - GFF file not found for GCA_902164875.\n",
      "2024-02-25 13:21:07,433 [INFO - chronostrain.cli.make_db_json] - Created 1 merged markers for strain GCA_902164875, seqID GCA_902164875_CONTIG_1.\n",
      "2024-02-25 13:21:08,223 [INFO - chronostrain.cli.make_db_json] - Database JSON file created: /mnt/e/infant_nt/database/chronostrain_files/efaecalis.json\n"
     ]
    }
   ],
   "source": [
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR={CHRONOSTRAIN_DB_DIR} \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      make-db \\\n",
    "      -m $MARKER_SEED_INDEX \\\n",
    "      -r $REFSEQ_INDEX \\\n",
    "      -b $BLAST_DB_NAME -bd $BLAST_DB_DIR \\\n",
    "      --min-pct-idty $MIN_PCT_IDTY \\\n",
    "      -o $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      --threads $NUM_CORES"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "fb366132-063b-4709-8e35-21ca2f0b5555",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-02-25 13:21:08,653 [DEBUG - chronostrain.logging.initialize] - Using logging configuration /home/youn/work/chronostrain/examples/database/complete_recipes/logging.ini\n",
      "2024-02-25 13:21:08,664 [INFO - chronostrain.cli.prune_json] - Pruning database via clustering\n",
      "2024-02-25 13:21:08,664 [DEBUG - chronostrain.cli.prune_json] - Src: /mnt/e/infant_nt/database/chronostrain_files/efaecalis.json, Output: /mnt/e/infant_nt/database/chronostrain_files/efaecalis.clusters.txt\n",
      "2024-02-25 13:21:08,664 [INFO - chronostrain.cli.prune_json] - Target identity threshold = 0.9999\n",
      "2024-02-25 13:21:09,311 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from chronostrain.ini.\n",
      "2024-02-25 13:21:09,400 [INFO - chronostrain.cli.prune_json] - Preprocessing step -- Loading old DB instance, using data directory: /mnt/e/infant_nt/database/chronostrain_files\n",
      "2024-02-25 13:21:09,400 [DEBUG - chronostrain.database.parser.json] - Couldn't find instance (/mnt/e/infant_nt/database/chronostrain_files/__efaecalis_/database.posix.pkl).\n",
      "2024-02-25 13:21:09,400 [INFO - chronostrain.database.parser.json] - Loading from JSON marker database file /mnt/e/infant_nt/database/chronostrain_files/efaecalis.json.\n",
      "2024-02-25 13:21:09,543 [DEBUG - chronostrain.database.parser.json] - Strain CP022488.1 (Enterococcus faecalis, ARO1/DG) loaded with 69 markers.\n",
      "2024-02-25 13:21:09,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022488.1 (Enterococcus faecalis, ARO1/DG) loaded with 69 markers.\n",
      "2024-02-25 13:21:09,568 [DEBUG - chronostrain.database.parser.json] - Strain OD940434.1 (Enterococcus faecalis, BX5936) loaded with 64 markers.\n",
      "2024-02-25 13:21:09,576 [DEBUG - chronostrain.database.parser.json] - Strain OD940440.2 (Enterococcus faecalis, WE0254) loaded with 62 markers.\n",
      "2024-02-25 13:21:09,584 [DEBUG - chronostrain.database.parser.json] - Strain OD940422.1 (Enterococcus faecalis, WE0851) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,593 [DEBUG - chronostrain.database.parser.json] - Strain LR962855.1 (Enterococcus faecalis, 27688_1#370) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,601 [DEBUG - chronostrain.database.parser.json] - Strain LR962846.1 (Enterococcus faecalis, 27688_1#63) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,609 [DEBUG - chronostrain.database.parser.json] - Strain LR962827.1 (Enterococcus faecalis, 28157_4#263) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,618 [DEBUG - chronostrain.database.parser.json] - Strain LR962806.1 (Enterococcus faecalis, 28099_2#73) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,627 [DEBUG - chronostrain.database.parser.json] - Strain LR962796.1 (Enterococcus faecalis, 28157_4#99) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,636 [DEBUG - chronostrain.database.parser.json] - Strain LR962716.1 (Enterococcus faecalis, 26975_2#34) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,644 [DEBUG - chronostrain.database.parser.json] - Strain LR962708.1 (Enterococcus faecalis, 26975_2#32) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,652 [DEBUG - chronostrain.database.parser.json] - Strain LR962700.1 (Enterococcus faecalis, 26975_2#33) loaded with 64 markers.\n",
      "2024-02-25 13:21:09,661 [DEBUG - chronostrain.database.parser.json] - Strain LR962684.1 (Enterococcus faecalis, 26975_2#27) loaded with 67 markers.\n",
      "2024-02-25 13:21:09,670 [DEBUG - chronostrain.database.parser.json] - Strain LR962642.1 (Enterococcus faecalis, 26975_1#75) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,678 [DEBUG - chronostrain.database.parser.json] - Strain LR962509.1 (Enterococcus faecalis, 28157_4#363) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,686 [DEBUG - chronostrain.database.parser.json] - Strain LR962521.1 (Enterococcus faecalis, 28099_2#292) loaded with 58 markers.\n",
      "2024-02-25 13:21:09,694 [DEBUG - chronostrain.database.parser.json] - Strain LR962474.1 (Enterococcus faecalis, 28157_4#117) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,702 [DEBUG - chronostrain.database.parser.json] - Strain LR962440.1 (Enterococcus faecalis, 28157_4#260) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,710 [DEBUG - chronostrain.database.parser.json] - Strain LR962444.1 (Enterococcus faecalis, 28157_4#281) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,717 [DEBUG - chronostrain.database.parser.json] - Strain LR962403.1 (Enterococcus faecalis, 28157_4#207) loaded with 58 markers.\n",
      "2024-02-25 13:21:09,726 [DEBUG - chronostrain.database.parser.json] - Strain LR962396.1 (Enterococcus faecalis, 28157_4#245) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,734 [DEBUG - chronostrain.database.parser.json] - Strain LR962363.1 (Enterococcus faecalis, 28157_4#7) loaded with 61 markers.\n",
      "2024-02-25 13:21:09,743 [DEBUG - chronostrain.database.parser.json] - Strain LR962361.1 (Enterococcus faecalis, 28157_4#308) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,751 [DEBUG - chronostrain.database.parser.json] - Strain LR962359.1 (Enterococcus faecalis, 28157_4#228) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,760 [DEBUG - chronostrain.database.parser.json] - Strain LR962353.1 (Enterococcus faecalis, 28157_4#11) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,768 [DEBUG - chronostrain.database.parser.json] - Strain LR962336.1 (Enterococcus faecalis, 28099_2#281) loaded with 61 markers.\n",
      "2024-02-25 13:21:09,776 [DEBUG - chronostrain.database.parser.json] - Strain LR962298.1 (Enterococcus faecalis, 28157_4#249) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,784 [DEBUG - chronostrain.database.parser.json] - Strain LR962185.1 (Enterococcus faecalis, 27688_1#269) loaded with 58 markers.\n",
      "2024-02-25 13:21:09,796 [DEBUG - chronostrain.database.parser.json] - Strain LR962115.1 (Enterococcus faecalis, 26975_1#262) loaded with 64 markers.\n",
      "2024-02-25 13:21:09,804 [DEBUG - chronostrain.database.parser.json] - Strain LR962036.1 (Enterococcus faecalis, 26975_1#182) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,813 [DEBUG - chronostrain.database.parser.json] - Strain LR962060.1 (Enterococcus faecalis, 26975_2#138) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,822 [DEBUG - chronostrain.database.parser.json] - Strain LR594051.1 (Enterococcus faecalis, NCTC8732) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,830 [DEBUG - chronostrain.database.parser.json] - Strain LR134312.1 (Enterococcus faecalis, NCTC8745) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,838 [DEBUG - chronostrain.database.parser.json] - Strain CP124918.1 (Enterococcus faecalis, EfsC9) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,845 [DEBUG - chronostrain.database.parser.json] - Strain CP124906.1 (Enterococcus faecalis, EfsC49) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,854 [DEBUG - chronostrain.database.parser.json] - Strain CP124889.1 (Enterococcus faecalis, EfsPF5) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,862 [DEBUG - chronostrain.database.parser.json] - Strain CP124886.1 (Enterococcus faecalis, EfsPF13) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,870 [DEBUG - chronostrain.database.parser.json] - Strain CP118072.1 (Enterococcus faecalis, VSI19) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,878 [DEBUG - chronostrain.database.parser.json] - Strain CP117508.1 (Enterococcus faecalis, CQJXZ21-077) loaded with 58 markers.\n",
      "2024-02-25 13:21:09,887 [DEBUG - chronostrain.database.parser.json] - Strain CP116575.1 (Enterococcus faecalis, K205-3) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,895 [DEBUG - chronostrain.database.parser.json] - Strain CP103863.1 (Enterococcus faecalis, SJ82) loaded with 59 markers.\n",
      "2024-02-25 13:21:09,904 [DEBUG - chronostrain.database.parser.json] - Strain CP091231.1 (Enterococcus faecalis, 732) loaded with 67 markers.\n",
      "2024-02-25 13:21:09,912 [DEBUG - chronostrain.database.parser.json] - Strain CP088200.1 (Enterococcus faecalis, S39-4) loaded with 66 markers.\n",
      "2024-02-25 13:21:09,920 [DEBUG - chronostrain.database.parser.json] - Strain AP025270.1 (Enterococcus faecalis, K-4) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,933 [DEBUG - chronostrain.database.parser.json] - Strain CP065784.1 (Enterococcus faecalis, 18-243) loaded with 67 markers.\n",
      "2024-02-25 13:21:09,942 [DEBUG - chronostrain.database.parser.json] - Strain CP022059.2 (Enterococcus faecalis, FDAARGOS_338) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940434.1 (Enterococcus faecalis, BX5936) loaded with 64 markers.\n",
      "2024-02-25 13:21:09,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940440.2 (Enterococcus faecalis, WE0254) loaded with 62 markers.\n",
      "2024-02-25 13:21:09,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940422.1 (Enterococcus faecalis, WE0851) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962855.1 (Enterococcus faecalis, 27688_1#370) loaded with 65 markers.\n",
      "2024-02-25 13:21:09,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962846.1 (Enterococcus faecalis, 27688_1#63) loaded with 60 markers.\n",
      "2024-02-25 13:21:09,991 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962827.1 (Enterococcus faecalis, 28157_4#263) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962806.1 (Enterococcus faecalis, 28099_2#73) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962796.1 (Enterococcus faecalis, 28157_4#99) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,017 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962716.1 (Enterococcus faecalis, 26975_2#34) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,025 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962708.1 (Enterococcus faecalis, 26975_2#32) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962700.1 (Enterococcus faecalis, 26975_2#33) loaded with 64 markers.\n",
      "2024-02-25 13:21:10,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962684.1 (Enterococcus faecalis, 26975_2#27) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962642.1 (Enterococcus faecalis, 26975_1#75) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962509.1 (Enterococcus faecalis, 28157_4#363) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,072 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962521.1 (Enterococcus faecalis, 28099_2#292) loaded with 58 markers.\n",
      "2024-02-25 13:21:10,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962474.1 (Enterococcus faecalis, 28157_4#117) loaded with 60 markers.\n",
      "2024-02-25 13:21:10,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962440.1 (Enterococcus faecalis, 28157_4#260) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962444.1 (Enterococcus faecalis, 28157_4#281) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962403.1 (Enterococcus faecalis, 28157_4#207) loaded with 58 markers.\n",
      "2024-02-25 13:21:10,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962396.1 (Enterococcus faecalis, 28157_4#245) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,121 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962363.1 (Enterococcus faecalis, 28157_4#7) loaded with 61 markers.\n",
      "2024-02-25 13:21:10,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962361.1 (Enterococcus faecalis, 28157_4#308) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962359.1 (Enterococcus faecalis, 28157_4#228) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962353.1 (Enterococcus faecalis, 28157_4#11) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,155 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962336.1 (Enterococcus faecalis, 28099_2#281) loaded with 61 markers.\n",
      "2024-02-25 13:21:10,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962298.1 (Enterococcus faecalis, 28157_4#249) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,174 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962185.1 (Enterococcus faecalis, 27688_1#269) loaded with 58 markers.\n",
      "2024-02-25 13:21:10,183 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962115.1 (Enterococcus faecalis, 26975_1#262) loaded with 64 markers.\n",
      "2024-02-25 13:21:10,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962036.1 (Enterococcus faecalis, 26975_1#182) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962060.1 (Enterococcus faecalis, 26975_2#138) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,209 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR594051.1 (Enterococcus faecalis, NCTC8732) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134312.1 (Enterococcus faecalis, NCTC8745) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124918.1 (Enterococcus faecalis, EfsC9) loaded with 60 markers.\n",
      "2024-02-25 13:21:10,233 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124906.1 (Enterococcus faecalis, EfsC49) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,241 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124889.1 (Enterococcus faecalis, EfsPF5) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124886.1 (Enterococcus faecalis, EfsPF13) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP117508.1 (Enterococcus faecalis, CQJXZ21-077) loaded with 58 markers.\n",
      "2024-02-25 13:21:10,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116575.1 (Enterococcus faecalis, K205-3) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,274 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP103863.1 (Enterococcus faecalis, SJ82) loaded with 59 markers.\n",
      "2024-02-25 13:21:10,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091231.1 (Enterococcus faecalis, 732) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088200.1 (Enterococcus faecalis, S39-4) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025270.1 (Enterococcus faecalis, K-4) loaded with 60 markers.\n",
      "2024-02-25 13:21:10,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065784.1 (Enterococcus faecalis, 18-243) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022059.1 (Enterococcus faecalis, FDAARGOS_338) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,337 [DEBUG - chronostrain.database.parser.json] - Strain AE016830.1 (Enterococcus faecalis, V583) loaded with 78 markers.\n",
      "2024-02-25 13:21:10,356 [DEBUG - chronostrain.database.parser.json] - Strain NC_004668.1 (Enterococcus faecalis, V583) loaded with 78 markers.\n",
      "2024-02-25 13:21:10,376 [DEBUG - chronostrain.database.parser.json] - Strain CP041738.1 (Enterococcus faecalis, B594) loaded with 81 markers.\n",
      "2024-02-25 13:21:10,393 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041738.1 (Enterococcus faecalis, B594) loaded with 81 markers.\n",
      "2024-02-25 13:21:10,402 [DEBUG - chronostrain.database.parser.json] - Strain CP004081.1 (Enterococcus faecalis, DENG1) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,411 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP004081.1 (Enterococcus faecalis, DENG1) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,427 [DEBUG - chronostrain.database.parser.json] - Strain LR962862.1 (Enterococcus faecalis, 28157_4#306) loaded with 72 markers.\n",
      "2024-02-25 13:21:10,443 [DEBUG - chronostrain.database.parser.json] - Strain LR962859.1 (Enterococcus faecalis, 27688_1#349) loaded with 71 markers.\n",
      "2024-02-25 13:21:10,452 [DEBUG - chronostrain.database.parser.json] - Strain LR962821.1 (Enterococcus faecalis, 28157_4#78) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,503 [DEBUG - chronostrain.database.parser.json] - Strain LR962784.1 (Enterococcus faecalis, 28157_4#120) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,512 [DEBUG - chronostrain.database.parser.json] - Strain LR962782.1 (Enterococcus faecalis, 28157_4#171) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,529 [DEBUG - chronostrain.database.parser.json] - Strain LR962731.1 (Enterococcus faecalis, 26975_2#43) loaded with 74 markers.\n",
      "2024-02-25 13:21:10,545 [DEBUG - chronostrain.database.parser.json] - Strain LR962702.1 (Enterococcus faecalis, 26975_2#40) loaded with 76 markers.\n",
      "2024-02-25 13:21:10,563 [DEBUG - chronostrain.database.parser.json] - Strain LR962639.1 (Enterococcus faecalis, 26975_2#149) loaded with 81 markers.\n",
      "2024-02-25 13:21:10,572 [DEBUG - chronostrain.database.parser.json] - Strain LR962647.1 (Enterococcus faecalis, 26975_1#29) loaded with 68 markers.\n",
      "2024-02-25 13:21:10,580 [DEBUG - chronostrain.database.parser.json] - Strain LR962636.1 (Enterococcus faecalis, 26975_1#26) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,596 [DEBUG - chronostrain.database.parser.json] - Strain LR962615.1 (Enterococcus faecalis, 27688_1#74) loaded with 70 markers.\n",
      "2024-02-25 13:21:10,613 [DEBUG - chronostrain.database.parser.json] - Strain LR962628.1 (Enterococcus faecalis, 27688_1#152) loaded with 69 markers.\n",
      "2024-02-25 13:21:10,628 [DEBUG - chronostrain.database.parser.json] - Strain LR962611.1 (Enterococcus faecalis, 27688_1#128) loaded with 69 markers.\n",
      "2024-02-25 13:21:10,648 [DEBUG - chronostrain.database.parser.json] - Strain LR962586.1 (Enterococcus faecalis, 27688_1#103) loaded with 76 markers.\n",
      "2024-02-25 13:21:10,665 [DEBUG - chronostrain.database.parser.json] - Strain LR962584.1 (Enterococcus faecalis, 27688_1#256) loaded with 78 markers.\n",
      "2024-02-25 13:21:10,683 [DEBUG - chronostrain.database.parser.json] - Strain LR962559.1 (Enterococcus faecalis, 27725_1#47) loaded with 82 markers.\n",
      "2024-02-25 13:21:10,700 [DEBUG - chronostrain.database.parser.json] - Strain LR962561.1 (Enterococcus faecalis, 27725_1#7) loaded with 70 markers.\n",
      "2024-02-25 13:21:10,708 [DEBUG - chronostrain.database.parser.json] - Strain LR962558.1 (Enterococcus faecalis, 27688_1#353) loaded with 61 markers.\n",
      "2024-02-25 13:21:10,716 [DEBUG - chronostrain.database.parser.json] - Strain LR962569.1 (Enterococcus faecalis, 27688_1#192) loaded with 64 markers.\n",
      "2024-02-25 13:21:10,725 [DEBUG - chronostrain.database.parser.json] - Strain LR962574.1 (Enterococcus faecalis, 27688_1#329) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,733 [DEBUG - chronostrain.database.parser.json] - Strain LR962565.1 (Enterococcus faecalis, 27688_1#240) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,749 [DEBUG - chronostrain.database.parser.json] - Strain LR962551.1 (Enterococcus faecalis, 27688_1#325) loaded with 70 markers.\n",
      "2024-02-25 13:21:10,765 [DEBUG - chronostrain.database.parser.json] - Strain LR962542.1 (Enterococcus faecalis, 27688_1#208) loaded with 71 markers.\n",
      "2024-02-25 13:21:10,782 [DEBUG - chronostrain.database.parser.json] - Strain LR962526.1 (Enterococcus faecalis, 28157_4#381) loaded with 78 markers.\n",
      "2024-02-25 13:21:10,799 [DEBUG - chronostrain.database.parser.json] - Strain LR962502.1 (Enterococcus faecalis, 28099_2#54) loaded with 76 markers.\n",
      "2024-02-25 13:21:10,808 [DEBUG - chronostrain.database.parser.json] - Strain LR962507.1 (Enterococcus faecalis, 28099_2#18) loaded with 69 markers.\n",
      "2024-02-25 13:21:10,824 [DEBUG - chronostrain.database.parser.json] - Strain LR962523.1 (Enterococcus faecalis, 28157_4#135) loaded with 71 markers.\n",
      "2024-02-25 13:21:10,840 [DEBUG - chronostrain.database.parser.json] - Strain LR962495.1 (Enterococcus faecalis, 28157_4#145) loaded with 73 markers.\n",
      "2024-02-25 13:21:10,858 [DEBUG - chronostrain.database.parser.json] - Strain LR962464.1 (Enterococcus faecalis, 28157_4#317) loaded with 81 markers.\n",
      "2024-02-25 13:21:10,876 [DEBUG - chronostrain.database.parser.json] - Strain LR962418.1 (Enterococcus faecalis, 28157_4#218) loaded with 78 markers.\n",
      "2024-02-25 13:21:10,885 [DEBUG - chronostrain.database.parser.json] - Strain LR962423.1 (Enterococcus faecalis, 28157_4#96) loaded with 69 markers.\n",
      "2024-02-25 13:21:10,894 [DEBUG - chronostrain.database.parser.json] - Strain LR962410.1 (Enterococcus faecalis, 28157_4#93) loaded with 63 markers.\n",
      "2024-02-25 13:21:10,906 [DEBUG - chronostrain.database.parser.json] - Strain LR962421.1 (Enterococcus faecalis, 28157_4#39) loaded with 69 markers.\n",
      "2024-02-25 13:21:10,914 [DEBUG - chronostrain.database.parser.json] - Strain LR962407.1 (Enterococcus faecalis, 28157_4#179) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,923 [DEBUG - chronostrain.database.parser.json] - Strain LR962386.1 (Enterococcus faecalis, 28157_4#65) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,941 [DEBUG - chronostrain.database.parser.json] - Strain LR962378.1 (Enterococcus faecalis, 28157_4#70) loaded with 77 markers.\n",
      "2024-02-25 13:21:10,951 [DEBUG - chronostrain.database.parser.json] - Strain LR962367.1 (Enterococcus faecalis, 28157_4#86) loaded with 66 markers.\n",
      "2024-02-25 13:21:10,959 [DEBUG - chronostrain.database.parser.json] - Strain LR962372.1 (Enterococcus faecalis, 28157_4#283) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,968 [DEBUG - chronostrain.database.parser.json] - Strain LR962356.1 (Enterococcus faecalis, 28157_4#90) loaded with 67 markers.\n",
      "2024-02-25 13:21:10,977 [DEBUG - chronostrain.database.parser.json] - Strain LR962321.1 (Enterococcus faecalis, 27688_1#215) loaded with 65 markers.\n",
      "2024-02-25 13:21:10,992 [DEBUG - chronostrain.database.parser.json] - Strain LR962331.1 (Enterococcus faecalis, 28099_2#340) loaded with 69 markers.\n",
      "2024-02-25 13:21:11,010 [DEBUG - chronostrain.database.parser.json] - Strain LR962306.1 (Enterococcus faecalis, 27725_1#8) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,030 [DEBUG - chronostrain.database.parser.json] - Strain LR962302.1 (Enterococcus faecalis, 27688_1#267) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,053 [DEBUG - chronostrain.database.parser.json] - Strain LR962267.1 (Enterococcus faecalis, 27725_1#12) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,072 [DEBUG - chronostrain.database.parser.json] - Strain LR962271.1 (Enterococcus faecalis, 27688_1#234) loaded with 74 markers.\n",
      "2024-02-25 13:21:11,089 [DEBUG - chronostrain.database.parser.json] - Strain LR962277.1 (Enterococcus faecalis, 27725_1#319) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,106 [DEBUG - chronostrain.database.parser.json] - Strain LR962248.1 (Enterococcus faecalis, 27688_1#266) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,125 [DEBUG - chronostrain.database.parser.json] - Strain LR962203.1 (Enterococcus faecalis, 27725_1#254) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,142 [DEBUG - chronostrain.database.parser.json] - Strain LR962196.1 (Enterococcus faecalis, 27688_1#88) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,159 [DEBUG - chronostrain.database.parser.json] - Strain LR962227.1 (Enterococcus faecalis, 27725_1#262) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,179 [DEBUG - chronostrain.database.parser.json] - Strain LR962174.1 (Enterococcus faecalis, 27688_1#347) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,188 [DEBUG - chronostrain.database.parser.json] - Strain LR962152.1 (Enterococcus faecalis, 26975_2#83) loaded with 69 markers.\n",
      "2024-02-25 13:21:11,196 [DEBUG - chronostrain.database.parser.json] - Strain LR962155.1 (Enterococcus faecalis, 26975_2#97) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,207 [DEBUG - chronostrain.database.parser.json] - Strain LR962169.1 (Enterococcus faecalis, 26975_2#2) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,216 [DEBUG - chronostrain.database.parser.json] - Strain LR962157.1 (Enterococcus faecalis, 26975_2#229) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,224 [DEBUG - chronostrain.database.parser.json] - Strain LR962141.1 (Enterococcus faecalis, 26975_2#69) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,241 [DEBUG - chronostrain.database.parser.json] - Strain LR962165.1 (Enterococcus faecalis, 26975_2#74) loaded with 73 markers.\n",
      "2024-02-25 13:21:11,250 [DEBUG - chronostrain.database.parser.json] - Strain LR962121.1 (Enterococcus faecalis, 26975_1#382) loaded with 64 markers.\n",
      "2024-02-25 13:21:11,267 [DEBUG - chronostrain.database.parser.json] - Strain LR962129.1 (Enterococcus faecalis, 26975_1#381) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,285 [DEBUG - chronostrain.database.parser.json] - Strain LR962119.1 (Enterococcus faecalis, 26975_1#228) loaded with 79 markers.\n",
      "2024-02-25 13:21:11,302 [DEBUG - chronostrain.database.parser.json] - Strain LR962056.1 (Enterococcus faecalis, 26975_2#200) loaded with 80 markers.\n",
      "2024-02-25 13:21:11,320 [DEBUG - chronostrain.database.parser.json] - Strain LR962025.1 (Enterococcus faecalis, 26975_1#146) loaded with 82 markers.\n",
      "2024-02-25 13:21:11,337 [DEBUG - chronostrain.database.parser.json] - Strain LR962004.1 (Enterococcus faecalis, 26975_1#89) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,354 [DEBUG - chronostrain.database.parser.json] - Strain LR962013.1 (Enterococcus faecalis, 26975_1#145) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,362 [DEBUG - chronostrain.database.parser.json] - Strain LR961991.1 (Enterococcus faecalis, 26975_1#70) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,379 [DEBUG - chronostrain.database.parser.json] - Strain LR961997.1 (Enterococcus faecalis, 26975_1#107) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,396 [DEBUG - chronostrain.database.parser.json] - Strain LR961935.1 (Enterococcus faecalis, 26975_2#145) loaded with 77 markers.\n",
      "2024-02-25 13:21:11,405 [DEBUG - chronostrain.database.parser.json] - Strain AP027297.1 (Enterococcus faecalis, L6D) loaded with 69 markers.\n",
      "2024-02-25 13:21:11,414 [DEBUG - chronostrain.database.parser.json] - Strain CP124921.1 (Enterococcus faecalis, EfsC12) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,422 [DEBUG - chronostrain.database.parser.json] - Strain CP124908.1 (Enterococcus faecalis, EfsC33) loaded with 62 markers.\n",
      "2024-02-25 13:21:11,430 [DEBUG - chronostrain.database.parser.json] - Strain CP124894.1 (Enterococcus faecalis, EfsC116) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,438 [DEBUG - chronostrain.database.parser.json] - Strain CP124884.1 (Enterococcus faecalis, EfsPF15) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,456 [DEBUG - chronostrain.database.parser.json] - Strain CP106757.1 (Enterococcus faecalis, NY13410) loaded with 80 markers.\n",
      "2024-02-25 13:21:11,475 [DEBUG - chronostrain.database.parser.json] - Strain CP119528.1 (Enterococcus faecalis, 3143) loaded with 75 markers.\n",
      "2024-02-25 13:21:11,494 [DEBUG - chronostrain.database.parser.json] - Strain CP119159.1 (Enterococcus faecalis, GENOMIC22-006) loaded with 74 markers.\n",
      "2024-02-25 13:21:11,503 [DEBUG - chronostrain.database.parser.json] - Strain CP118085.1 (Enterococcus faecalis, VSI09) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,512 [DEBUG - chronostrain.database.parser.json] - Strain CP118057.1 (Enterococcus faecalis, VSI36) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,534 [DEBUG - chronostrain.database.parser.json] - Strain CP118755.1 (Enterococcus faecalis, W5) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,544 [DEBUG - chronostrain.database.parser.json] - Strain CP113831.1 (Enterococcus faecalis, T30) loaded with 66 markers.\n",
      "2024-02-25 13:21:11,554 [DEBUG - chronostrain.database.parser.json] - Strain CP110030.1 (Enterococcus faecalis, BE65) loaded with 59 markers.\n",
      "2024-02-25 13:21:11,571 [DEBUG - chronostrain.database.parser.json] - Strain CP102070.1 (Enterococcus faecalis, JF3A-4253) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,586 [DEBUG - chronostrain.database.parser.json] - Strain CP097002.1 (Enterococcus faecalis, LS06-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:11,595 [DEBUG - chronostrain.database.parser.json] - Strain CP092576.1 (Enterococcus faecalis, VRE-WC031) loaded with 65 markers.\n",
      "2024-02-25 13:21:11,604 [DEBUG - chronostrain.database.parser.json] - Strain CP092574.1 (Enterococcus faecalis, VSE-WC032) loaded with 69 markers.\n",
      "2024-02-25 13:21:11,620 [DEBUG - chronostrain.database.parser.json] - Strain CP076498.1 (Enterococcus faecalis, UAMS_EL53) loaded with 72 markers.\n",
      "2024-02-25 13:21:11,637 [DEBUG - chronostrain.database.parser.json] - Strain CP076493.1 (Enterococcus faecalis, UAMS_EL54) loaded with 73 markers.\n",
      "2024-02-25 13:21:11,653 [DEBUG - chronostrain.database.parser.json] - Strain CP076488.1 (Enterococcus faecalis, UAMS_EL56) loaded with 73 markers.\n",
      "2024-02-25 13:21:11,662 [DEBUG - chronostrain.database.parser.json] - Strain CP091901.1 (Enterococcus faecalis, 59) loaded with 64 markers.\n",
      "2024-02-25 13:21:11,680 [DEBUG - chronostrain.database.parser.json] - Strain CP091899.1 (Enterococcus faecalis, 101-1) loaded with 74 markers.\n",
      "2024-02-25 13:21:11,688 [DEBUG - chronostrain.database.parser.json] - Strain CP091892.1 (Enterococcus faecalis, 133-1) loaded with 64 markers.\n",
      "2024-02-25 13:21:11,697 [DEBUG - chronostrain.database.parser.json] - Strain CP091889.1 (Enterococcus faecalis, 142-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:11,717 [DEBUG - chronostrain.database.parser.json] - Strain CP091198.1 (Enterococcus faecalis, UK045) loaded with 82 markers.\n",
      "2024-02-25 13:21:11,726 [DEBUG - chronostrain.database.parser.json] - Strain CP086726.1 (Enterococcus faecalis, E533) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,735 [DEBUG - chronostrain.database.parser.json] - Strain CP086592.1 (Enterococcus faecalis, Fac74) loaded with 72 markers.\n",
      "2024-02-25 13:21:11,744 [DEBUG - chronostrain.database.parser.json] - Strain CP086558.1 (Enterococcus faecalis, E512) loaded with 72 markers.\n",
      "2024-02-25 13:21:11,761 [DEBUG - chronostrain.database.parser.json] - Strain CP082231.1 (Enterococcus faecalis, E006) loaded with 72 markers.\n",
      "2024-02-25 13:21:11,779 [DEBUG - chronostrain.database.parser.json] - Strain CP075604.2 (Enterococcus faecalis, 1207/14) loaded with 80 markers.\n",
      "2024-02-25 13:21:11,794 [DEBUG - chronostrain.database.parser.json] - Strain CP046113.1 (Enterococcus faecalis, 092160007-3) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,810 [DEBUG - chronostrain.database.parser.json] - Strain CP049775.1 (Enterococcus faecalis, ES-1) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,817 [DEBUG - chronostrain.database.parser.json] - Strain CP041344.1 (Enterococcus faecalis, DM01) loaded with 59 markers.\n",
      "2024-02-25 13:21:11,834 [DEBUG - chronostrain.database.parser.json] - Strain CP039549.1 (Enterococcus faecalis, VE18395) loaded with 77 markers.\n",
      "2024-02-25 13:21:11,855 [DEBUG - chronostrain.database.parser.json] - Strain CP039548.1 (Enterococcus faecalis, VE18379) loaded with 78 markers.\n",
      "2024-02-25 13:21:11,876 [DEBUG - chronostrain.database.parser.json] - Strain CP039296.1 (Enterococcus faecalis, VE14089) loaded with 79 markers.\n",
      "2024-02-25 13:21:11,886 [DEBUG - chronostrain.database.parser.json] - Strain CP028285.1 (Enterococcus faecalis, FDAARGOS_324) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,895 [DEBUG - chronostrain.database.parser.json] - Strain AP017623.1 (Enterococcus faecalis, W11) loaded with 58 markers.\n",
      "2024-02-25 13:21:11,913 [DEBUG - chronostrain.database.parser.json] - Strain CP015883.1 (Enterococcus faecalis, sorialis) loaded with 76 markers.\n",
      "2024-02-25 13:21:11,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962862.1 (Enterococcus faecalis, 28157_4#306) loaded with 72 markers.\n",
      "2024-02-25 13:21:11,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962859.1 (Enterococcus faecalis, 27688_1#349) loaded with 71 markers.\n",
      "2024-02-25 13:21:11,958 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962821.1 (Enterococcus faecalis, 28157_4#78) loaded with 67 markers.\n",
      "2024-02-25 13:21:11,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962784.1 (Enterococcus faecalis, 28157_4#120) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962782.1 (Enterococcus faecalis, 28157_4#171) loaded with 63 markers.\n",
      "2024-02-25 13:21:11,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962731.1 (Enterococcus faecalis, 26975_2#43) loaded with 74 markers.\n",
      "2024-02-25 13:21:12,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962702.1 (Enterococcus faecalis, 26975_2#40) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962639.1 (Enterococcus faecalis, 26975_2#149) loaded with 81 markers.\n",
      "2024-02-25 13:21:12,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962647.1 (Enterococcus faecalis, 26975_1#29) loaded with 68 markers.\n",
      "2024-02-25 13:21:12,045 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962636.1 (Enterococcus faecalis, 26975_1#26) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,061 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962615.1 (Enterococcus faecalis, 27688_1#74) loaded with 70 markers.\n",
      "2024-02-25 13:21:12,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962628.1 (Enterococcus faecalis, 27688_1#152) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962611.1 (Enterococcus faecalis, 27688_1#128) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,113 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962586.1 (Enterococcus faecalis, 27688_1#103) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962584.1 (Enterococcus faecalis, 27688_1#256) loaded with 78 markers.\n",
      "2024-02-25 13:21:12,150 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962559.1 (Enterococcus faecalis, 27725_1#47) loaded with 82 markers.\n",
      "2024-02-25 13:21:12,167 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962561.1 (Enterococcus faecalis, 27725_1#7) loaded with 70 markers.\n",
      "2024-02-25 13:21:12,176 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962558.1 (Enterococcus faecalis, 27688_1#353) loaded with 61 markers.\n",
      "2024-02-25 13:21:12,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962569.1 (Enterococcus faecalis, 27688_1#192) loaded with 64 markers.\n",
      "2024-02-25 13:21:12,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962574.1 (Enterococcus faecalis, 27688_1#329) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962565.1 (Enterococcus faecalis, 27688_1#240) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962551.1 (Enterococcus faecalis, 27688_1#325) loaded with 70 markers.\n",
      "2024-02-25 13:21:12,236 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962542.1 (Enterococcus faecalis, 27688_1#208) loaded with 71 markers.\n",
      "2024-02-25 13:21:12,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962526.1 (Enterococcus faecalis, 28157_4#381) loaded with 78 markers.\n",
      "2024-02-25 13:21:12,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962502.1 (Enterococcus faecalis, 28099_2#54) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962507.1 (Enterococcus faecalis, 28099_2#18) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962523.1 (Enterococcus faecalis, 28157_4#135) loaded with 71 markers.\n",
      "2024-02-25 13:21:12,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962495.1 (Enterococcus faecalis, 28157_4#145) loaded with 73 markers.\n",
      "2024-02-25 13:21:12,334 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962464.1 (Enterococcus faecalis, 28157_4#317) loaded with 81 markers.\n",
      "2024-02-25 13:21:12,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962418.1 (Enterococcus faecalis, 28157_4#218) loaded with 78 markers.\n",
      "2024-02-25 13:21:12,361 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962423.1 (Enterococcus faecalis, 28157_4#96) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,370 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962410.1 (Enterococcus faecalis, 28157_4#93) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962421.1 (Enterococcus faecalis, 28157_4#39) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962407.1 (Enterococcus faecalis, 28157_4#179) loaded with 65 markers.\n",
      "2024-02-25 13:21:12,396 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962386.1 (Enterococcus faecalis, 28157_4#65) loaded with 66 markers.\n",
      "2024-02-25 13:21:12,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962378.1 (Enterococcus faecalis, 28157_4#70) loaded with 77 markers.\n",
      "2024-02-25 13:21:12,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962367.1 (Enterococcus faecalis, 28157_4#86) loaded with 66 markers.\n",
      "2024-02-25 13:21:12,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962372.1 (Enterococcus faecalis, 28157_4#283) loaded with 67 markers.\n",
      "2024-02-25 13:21:12,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962356.1 (Enterococcus faecalis, 28157_4#90) loaded with 67 markers.\n",
      "2024-02-25 13:21:12,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962321.1 (Enterococcus faecalis, 27688_1#215) loaded with 65 markers.\n",
      "2024-02-25 13:21:12,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962331.1 (Enterococcus faecalis, 28099_2#340) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,482 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962306.1 (Enterococcus faecalis, 27725_1#8) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,501 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962302.1 (Enterococcus faecalis, 27688_1#267) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962267.1 (Enterococcus faecalis, 27725_1#12) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962271.1 (Enterococcus faecalis, 27688_1#234) loaded with 74 markers.\n",
      "2024-02-25 13:21:12,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962277.1 (Enterococcus faecalis, 27725_1#319) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,573 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962248.1 (Enterococcus faecalis, 27688_1#266) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962203.1 (Enterococcus faecalis, 27725_1#254) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,608 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962196.1 (Enterococcus faecalis, 27688_1#88) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,626 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962227.1 (Enterococcus faecalis, 27725_1#262) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,643 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962174.1 (Enterococcus faecalis, 27688_1#347) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,652 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962152.1 (Enterococcus faecalis, 26975_2#83) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,660 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962155.1 (Enterococcus faecalis, 26975_2#97) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962169.1 (Enterococcus faecalis, 26975_2#2) loaded with 67 markers.\n",
      "2024-02-25 13:21:12,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962157.1 (Enterococcus faecalis, 26975_2#229) loaded with 67 markers.\n",
      "2024-02-25 13:21:12,686 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962141.1 (Enterococcus faecalis, 26975_2#69) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,702 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962165.1 (Enterococcus faecalis, 26975_2#74) loaded with 73 markers.\n",
      "2024-02-25 13:21:12,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962121.1 (Enterococcus faecalis, 26975_1#382) loaded with 64 markers.\n",
      "2024-02-25 13:21:12,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962129.1 (Enterococcus faecalis, 26975_1#381) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,745 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962119.1 (Enterococcus faecalis, 26975_1#228) loaded with 79 markers.\n",
      "2024-02-25 13:21:12,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962056.1 (Enterococcus faecalis, 26975_2#200) loaded with 80 markers.\n",
      "2024-02-25 13:21:12,782 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962025.1 (Enterococcus faecalis, 26975_1#146) loaded with 82 markers.\n",
      "2024-02-25 13:21:12,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962004.1 (Enterococcus faecalis, 26975_1#89) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,819 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962013.1 (Enterococcus faecalis, 26975_1#145) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961991.1 (Enterococcus faecalis, 26975_1#70) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961997.1 (Enterococcus faecalis, 26975_1#107) loaded with 76 markers.\n",
      "2024-02-25 13:21:12,863 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961935.1 (Enterococcus faecalis, 26975_2#145) loaded with 77 markers.\n",
      "2024-02-25 13:21:12,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP027297.1 (Enterococcus faecalis, L6D) loaded with 69 markers.\n",
      "2024-02-25 13:21:12,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124921.1 (Enterococcus faecalis, EfsC12) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124908.1 (Enterococcus faecalis, EfsC33) loaded with 62 markers.\n",
      "2024-02-25 13:21:12,896 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124894.1 (Enterococcus faecalis, EfsC116) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124884.1 (Enterococcus faecalis, EfsPF15) loaded with 63 markers.\n",
      "2024-02-25 13:21:12,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP106757.1 (Enterococcus faecalis, NY13410) loaded with 80 markers.\n",
      "2024-02-25 13:21:12,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP119528.1 (Enterococcus faecalis, 3143) loaded with 75 markers.\n",
      "2024-02-25 13:21:12,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP119159.1 (Enterococcus faecalis, GENOMIC22-006) loaded with 74 markers.\n",
      "2024-02-25 13:21:12,976 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118755.1 (Enterococcus faecalis, W5) loaded with 71 markers.\n",
      "2024-02-25 13:21:12,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP113831.1 (Enterococcus faecalis, T30) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,001 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102070.1 (Enterococcus faecalis, JF3A-4253) loaded with 71 markers.\n",
      "2024-02-25 13:21:13,066 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097002.1 (Enterococcus faecalis, LS06-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092576.1 (Enterococcus faecalis, VRE-WC031) loaded with 65 markers.\n",
      "2024-02-25 13:21:13,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092574.1 (Enterococcus faecalis, VSE-WC032) loaded with 69 markers.\n",
      "2024-02-25 13:21:13,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076498.1 (Enterococcus faecalis, UAMS_EL53) loaded with 72 markers.\n",
      "2024-02-25 13:21:13,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076493.1 (Enterococcus faecalis, UAMS_EL54) loaded with 73 markers.\n",
      "2024-02-25 13:21:13,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076488.1 (Enterococcus faecalis, UAMS_EL56) loaded with 73 markers.\n",
      "2024-02-25 13:21:13,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091901.1 (Enterococcus faecalis, 59) loaded with 64 markers.\n",
      "2024-02-25 13:21:13,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091899.1 (Enterococcus faecalis, 101-1) loaded with 74 markers.\n",
      "2024-02-25 13:21:13,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091892.1 (Enterococcus faecalis, 133-1) loaded with 64 markers.\n",
      "2024-02-25 13:21:13,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091889.1 (Enterococcus faecalis, 142-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091198.1 (Enterococcus faecalis, UK045) loaded with 82 markers.\n",
      "2024-02-25 13:21:13,209 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086726.1 (Enterococcus faecalis, E533) loaded with 71 markers.\n",
      "2024-02-25 13:21:13,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086592.1 (Enterococcus faecalis, Fac74) loaded with 72 markers.\n",
      "2024-02-25 13:21:13,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086558.1 (Enterococcus faecalis, E512) loaded with 72 markers.\n",
      "2024-02-25 13:21:13,246 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082231.1 (Enterococcus faecalis, E006) loaded with 72 markers.\n",
      "2024-02-25 13:21:13,264 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP075604.2 (Enterococcus faecalis, 1207/14) loaded with 80 markers.\n",
      "2024-02-25 13:21:13,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046113.1 (Enterococcus faecalis, 092160007-3) loaded with 71 markers.\n",
      "2024-02-25 13:21:13,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049775.1 (Enterococcus faecalis, ES-1) loaded with 71 markers.\n",
      "2024-02-25 13:21:13,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041344.1 (Enterococcus faecalis, DM01) loaded with 59 markers.\n",
      "2024-02-25 13:21:13,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039549.1 (Enterococcus faecalis, VE18395) loaded with 77 markers.\n",
      "2024-02-25 13:21:13,339 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039548.1 (Enterococcus faecalis, VE18379) loaded with 78 markers.\n",
      "2024-02-25 13:21:13,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039296.1 (Enterococcus faecalis, VE14089) loaded with 79 markers.\n",
      "2024-02-25 13:21:13,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028285.1 (Enterococcus faecalis, FDAARGOS_324) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,372 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP017623.1 (Enterococcus faecalis, W11) loaded with 58 markers.\n",
      "2024-02-25 13:21:13,389 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015883.1 (Enterococcus faecalis, sorialis) loaded with 76 markers.\n",
      "2024-02-25 13:21:13,397 [DEBUG - chronostrain.database.parser.json] - Strain LR962068.1 (Enterococcus faecalis, 26975_2#191) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,406 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962068.1 (Enterococcus faecalis, 26975_2#191) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,414 [DEBUG - chronostrain.database.parser.json] - Strain AP025690.1 (Enterococcus sp., PLM3) loaded with 59 markers.\n",
      "2024-02-25 13:21:13,424 [DEBUG - chronostrain.database.parser.json] - Strain HF558530.1 (Enterococcus faecalis, Symbioflor 1) loaded with 62 markers.\n",
      "2024-02-25 13:21:13,433 [DEBUG - chronostrain.database.parser.json] - Strain NC_019770.1 (Enterococcus faecalis, Symbioflor 1) loaded with 62 markers.\n",
      "2024-02-25 13:21:13,442 [DEBUG - chronostrain.database.parser.json] - Strain CP002621.1 (Enterococcus faecalis, OG1RF) loaded with 59 markers.\n",
      "2024-02-25 13:21:13,454 [DEBUG - chronostrain.database.parser.json] - Strain NC_017316.1 (Enterococcus faecalis, OG1RF) loaded with 59 markers.\n",
      "2024-02-25 13:21:13,467 [DEBUG - chronostrain.database.parser.json] - Strain CP003726.1 (Enterococcus faecalis, D32) loaded with 63 markers.\n",
      "2024-02-25 13:21:13,478 [DEBUG - chronostrain.database.parser.json] - Strain NC_018221.1 (Enterococcus faecalis, D32) loaded with 63 markers.\n",
      "2024-02-25 13:21:13,497 [DEBUG - chronostrain.database.parser.json] - Strain CP008816.1 (Enterococcus faecalis, ATCC 29212) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,514 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP008816.1 (Enterococcus faecalis, 29212) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,523 [DEBUG - chronostrain.database.parser.json] - Strain CP002491.1 (Enterococcus faecalis, 62) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,535 [DEBUG - chronostrain.database.parser.json] - Strain NC_017312.1 (Enterococcus faecalis, 62) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,545 [DEBUG - chronostrain.database.parser.json] - Strain OD940420.1 (Enterococcus faecalis, WE0438) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,561 [DEBUG - chronostrain.database.parser.json] - Strain LR962836.1 (Enterococcus faecalis, 28157_4#332) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,576 [DEBUG - chronostrain.database.parser.json] - Strain LR962830.1 (Enterococcus faecalis, 27688_1#67) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,584 [DEBUG - chronostrain.database.parser.json] - Strain LR962834.1 (Enterococcus faecalis, 28157_4#163) loaded with 60 markers.\n",
      "2024-02-25 13:21:13,593 [DEBUG - chronostrain.database.parser.json] - Strain LR962818.1 (Enterococcus faecalis, 28099_2#212) loaded with 63 markers.\n",
      "2024-02-25 13:21:13,601 [DEBUG - chronostrain.database.parser.json] - Strain LR962777.1 (Enterococcus faecalis, 28157_4#156) loaded with 63 markers.\n",
      "2024-02-25 13:21:13,609 [DEBUG - chronostrain.database.parser.json] - Strain LR962792.1 (Enterococcus faecalis, 28157_4#209) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,617 [DEBUG - chronostrain.database.parser.json] - Strain LR962811.1 (Enterococcus faecalis, 28157_4#319) loaded with 59 markers.\n",
      "2024-02-25 13:21:13,627 [DEBUG - chronostrain.database.parser.json] - Strain LR962802.1 (Enterococcus faecalis, 28099_2#176) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,637 [DEBUG - chronostrain.database.parser.json] - Strain LR962732.1 (Enterococcus faecalis, 26975_2#42) loaded with 65 markers.\n",
      "2024-02-25 13:21:13,645 [DEBUG - chronostrain.database.parser.json] - Strain LR962720.1 (Enterococcus faecalis, 26975_2#37) loaded with 61 markers.\n",
      "2024-02-25 13:21:13,653 [DEBUG - chronostrain.database.parser.json] - Strain LR962721.1 (Enterococcus faecalis, 26975_2#35) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,662 [DEBUG - chronostrain.database.parser.json] - Strain LR962717.1 (Enterococcus faecalis, 26975_2#227) loaded with 63 markers.\n",
      "2024-02-25 13:21:13,671 [DEBUG - chronostrain.database.parser.json] - Strain LR962698.1 (Enterococcus faecalis, 26975_2#31) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,681 [DEBUG - chronostrain.database.parser.json] - Strain LR962694.1 (Enterococcus faecalis, 26975_2#30) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,699 [DEBUG - chronostrain.database.parser.json] - Strain LR962695.1 (Enterococcus faecalis, 26975_2#26) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,709 [DEBUG - chronostrain.database.parser.json] - Strain LR962654.1 (Enterococcus faecalis, 26975_2#19) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,719 [DEBUG - chronostrain.database.parser.json] - Strain LR962656.1 (Enterococcus faecalis, 26975_2#20) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,736 [DEBUG - chronostrain.database.parser.json] - Strain LR962663.1 (Enterococcus faecalis, 26975_2#18) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,745 [DEBUG - chronostrain.database.parser.json] - Strain LR962670.1 (Enterococcus faecalis, 26975_2#16) loaded with 65 markers.\n",
      "2024-02-25 13:21:13,761 [DEBUG - chronostrain.database.parser.json] - Strain LR962649.1 (Enterococcus faecalis, 26975_1#39) loaded with 64 markers.\n",
      "2024-02-25 13:21:13,770 [DEBUG - chronostrain.database.parser.json] - Strain LR962620.1 (Enterococcus faecalis, 27688_1#112) loaded with 65 markers.\n",
      "2024-02-25 13:21:13,779 [DEBUG - chronostrain.database.parser.json] - Strain LR962608.1 (Enterococcus faecalis, 27688_1#285) loaded with 65 markers.\n",
      "2024-02-25 13:21:13,795 [DEBUG - chronostrain.database.parser.json] - Strain LR962589.1 (Enterococcus faecalis, 27725_1#72) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,802 [DEBUG - chronostrain.database.parser.json] - Strain LR962580.1 (Enterococcus faecalis, 27688_1#212) loaded with 56 markers.\n",
      "2024-02-25 13:21:13,818 [DEBUG - chronostrain.database.parser.json] - Strain LR962572.1 (Enterococcus faecalis, 27725_1#11) loaded with 70 markers.\n",
      "2024-02-25 13:21:13,826 [DEBUG - chronostrain.database.parser.json] - Strain LR962579.1 (Enterococcus faecalis, 27725_1#112) loaded with 62 markers.\n",
      "2024-02-25 13:21:13,834 [DEBUG - chronostrain.database.parser.json] - Strain LR962554.1 (Enterococcus faecalis, 27725_1#59) loaded with 58 markers.\n",
      "2024-02-25 13:21:13,851 [DEBUG - chronostrain.database.parser.json] - Strain LR962512.1 (Enterococcus faecalis, 28099_2#233) loaded with 72 markers.\n",
      "2024-02-25 13:21:13,860 [DEBUG - chronostrain.database.parser.json] - Strain LR962466.1 (Enterococcus faecalis, 28157_4#166) loaded with 64 markers.\n",
      "2024-02-25 13:21:13,868 [DEBUG - chronostrain.database.parser.json] - Strain LR962486.1 (Enterococcus faecalis, 28157_4#109) loaded with 64 markers.\n",
      "2024-02-25 13:21:13,878 [DEBUG - chronostrain.database.parser.json] - Strain LR962461.1 (Enterococcus faecalis, 28157_4#40) loaded with 61 markers.\n",
      "2024-02-25 13:21:13,886 [DEBUG - chronostrain.database.parser.json] - Strain LR962437.1 (Enterococcus faecalis, 28157_4#177) loaded with 62 markers.\n",
      "2024-02-25 13:21:13,903 [DEBUG - chronostrain.database.parser.json] - Strain LR962414.1 (Enterococcus faecalis, 28157_4#92) loaded with 69 markers.\n",
      "2024-02-25 13:21:13,912 [DEBUG - chronostrain.database.parser.json] - Strain LR962415.1 (Enterococcus faecalis, 28157_4#72) loaded with 67 markers.\n",
      "2024-02-25 13:21:13,921 [DEBUG - chronostrain.database.parser.json] - Strain LR962382.1 (Enterococcus faecalis, 28157_4#258) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,930 [DEBUG - chronostrain.database.parser.json] - Strain LR962400.1 (Enterococcus faecalis, 28157_4#152) loaded with 66 markers.\n",
      "2024-02-25 13:21:13,947 [DEBUG - chronostrain.database.parser.json] - Strain LR962390.1 (Enterococcus faecalis, 28157_4#191) loaded with 69 markers.\n",
      "2024-02-25 13:21:13,955 [DEBUG - chronostrain.database.parser.json] - Strain LR962370.1 (Enterococcus faecalis, 28157_4#221) loaded with 69 markers.\n",
      "2024-02-25 13:21:13,972 [DEBUG - chronostrain.database.parser.json] - Strain LR962351.1 (Enterococcus faecalis, 28157_4#38) loaded with 77 markers.\n",
      "2024-02-25 13:21:13,987 [DEBUG - chronostrain.database.parser.json] - Strain LR962319.1 (Enterococcus faecalis, 27688_1#235) loaded with 68 markers.\n",
      "2024-02-25 13:21:13,995 [DEBUG - chronostrain.database.parser.json] - Strain LR962334.1 (Enterococcus faecalis, 28099_2#67) loaded with 59 markers.\n",
      "2024-02-25 13:21:14,004 [DEBUG - chronostrain.database.parser.json] - Strain LR962340.1 (Enterococcus faecalis, 27725_1#307) loaded with 64 markers.\n",
      "2024-02-25 13:21:14,012 [DEBUG - chronostrain.database.parser.json] - Strain LR962326.1 (Enterococcus faecalis, 28157_4#208) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,023 [DEBUG - chronostrain.database.parser.json] - Strain LR962311.1 (Enterococcus faecalis, 27725_1#24) loaded with 57 markers.\n",
      "2024-02-25 13:21:14,031 [DEBUG - chronostrain.database.parser.json] - Strain LR962323.1 (Enterococcus faecalis, 27688_1#292) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,048 [DEBUG - chronostrain.database.parser.json] - Strain LR962343.1 (Enterococcus faecalis, 28157_4#168) loaded with 73 markers.\n",
      "2024-02-25 13:21:14,064 [DEBUG - chronostrain.database.parser.json] - Strain LR962293.1 (Enterococcus faecalis, 27688_1#170) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,073 [DEBUG - chronostrain.database.parser.json] - Strain LR962296.1 (Enterococcus faecalis, 27688_1#305) loaded with 65 markers.\n",
      "2024-02-25 13:21:14,085 [DEBUG - chronostrain.database.parser.json] - Strain LR962300.1 (Enterococcus faecalis, 28099_2#254) loaded with 59 markers.\n",
      "2024-02-25 13:21:14,100 [DEBUG - chronostrain.database.parser.json] - Strain LR962283.1 (Enterococcus faecalis, 27688_1#230) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,116 [DEBUG - chronostrain.database.parser.json] - Strain LR962256.1 (Enterococcus faecalis, 27688_1#202) loaded with 69 markers.\n",
      "2024-02-25 13:21:14,124 [DEBUG - chronostrain.database.parser.json] - Strain LR962242.1 (Enterococcus faecalis, 27688_1#101) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,133 [DEBUG - chronostrain.database.parser.json] - Strain LR962236.1 (Enterococcus faecalis, 27688_1#380) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,140 [DEBUG - chronostrain.database.parser.json] - Strain LR962200.1 (Enterococcus faecalis, 27688_1#265) loaded with 59 markers.\n",
      "2024-02-25 13:21:14,157 [DEBUG - chronostrain.database.parser.json] - Strain LR962190.1 (Enterococcus faecalis, 27688_1#241) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,166 [DEBUG - chronostrain.database.parser.json] - Strain LR962217.1 (Enterococcus faecalis, 27688_1#68) loaded with 69 markers.\n",
      "2024-02-25 13:21:14,181 [DEBUG - chronostrain.database.parser.json] - Strain LR962194.1 (Enterococcus faecalis, 27688_1#304) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,189 [DEBUG - chronostrain.database.parser.json] - Strain LR962180.1 (Enterococcus faecalis, 27725_1#206) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,205 [DEBUG - chronostrain.database.parser.json] - Strain LR962182.1 (Enterococcus faecalis, 27725_1#182) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,221 [DEBUG - chronostrain.database.parser.json] - Strain LR962123.1 (Enterococcus faecalis, 26975_1#372) loaded with 73 markers.\n",
      "2024-02-25 13:21:14,229 [DEBUG - chronostrain.database.parser.json] - Strain LR962097.1 (Enterococcus faecalis, 26975_2#226) loaded with 59 markers.\n",
      "2024-02-25 13:21:14,238 [DEBUG - chronostrain.database.parser.json] - Strain LR962089.1 (Enterococcus faecalis, 26975_1#241) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,254 [DEBUG - chronostrain.database.parser.json] - Strain LR962101.1 (Enterococcus faecalis, 26975_1#280) loaded with 72 markers.\n",
      "2024-02-25 13:21:14,263 [DEBUG - chronostrain.database.parser.json] - Strain LR962093.1 (Enterococcus faecalis, 26975_1#268) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,271 [DEBUG - chronostrain.database.parser.json] - Strain LR962110.1 (Enterococcus faecalis, 26975_1#236) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,279 [DEBUG - chronostrain.database.parser.json] - Strain LR962083.1 (Enterococcus faecalis, 26975_1#194) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,296 [DEBUG - chronostrain.database.parser.json] - Strain LR962043.1 (Enterococcus faecalis, 26975_2#175) loaded with 72 markers.\n",
      "2024-02-25 13:21:14,305 [DEBUG - chronostrain.database.parser.json] - Strain LR962049.1 (Enterococcus faecalis, 26975_2#201) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,321 [DEBUG - chronostrain.database.parser.json] - Strain LR962072.1 (Enterococcus faecalis, 26975_2#204) loaded with 73 markers.\n",
      "2024-02-25 13:21:14,338 [DEBUG - chronostrain.database.parser.json] - Strain LR962063.1 (Enterococcus faecalis, 26975_2#198) loaded with 65 markers.\n",
      "2024-02-25 13:21:14,352 [DEBUG - chronostrain.database.parser.json] - Strain LR962051.1 (Enterococcus faecalis, 26975_2#190) loaded with 65 markers.\n",
      "2024-02-25 13:21:14,369 [DEBUG - chronostrain.database.parser.json] - Strain LR962023.1 (Enterococcus faecalis, 26975_2#165) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,385 [DEBUG - chronostrain.database.parser.json] - Strain LR962019.1 (Enterococcus faecalis, 26975_2#162) loaded with 72 markers.\n",
      "2024-02-25 13:21:14,393 [DEBUG - chronostrain.database.parser.json] - Strain LR962021.1 (Enterococcus faecalis, 26975_1#160) loaded with 62 markers.\n",
      "2024-02-25 13:21:14,401 [DEBUG - chronostrain.database.parser.json] - Strain LR961989.1 (Enterococcus faecalis, 26975_1#111) loaded with 60 markers.\n",
      "2024-02-25 13:21:14,410 [DEBUG - chronostrain.database.parser.json] - Strain LR961999.1 (Enterococcus faecalis, 26975_1#367) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,419 [DEBUG - chronostrain.database.parser.json] - Strain LR961978.1 (Enterococcus faecalis, 26975_1#63) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,427 [DEBUG - chronostrain.database.parser.json] - Strain LR961965.1 (Enterococcus faecalis, 26975_1#31) loaded with 66 markers.\n",
      "2024-02-25 13:21:14,435 [DEBUG - chronostrain.database.parser.json] - Strain LR961945.1 (Enterococcus faecalis, 26975_1#25) loaded with 58 markers.\n",
      "2024-02-25 13:21:14,451 [DEBUG - chronostrain.database.parser.json] - Strain LR961925.1 (Enterococcus faecalis, 26975_2#147) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,467 [DEBUG - chronostrain.database.parser.json] - Strain LR961938.1 (Enterococcus faecalis, 26975_1#48) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,475 [DEBUG - chronostrain.database.parser.json] - Strain LR961975.1 (Enterococcus faecalis, 26975_1#37) loaded with 59 markers.\n",
      "2024-02-25 13:21:14,491 [DEBUG - chronostrain.database.parser.json] - Strain LR961966.1 (Enterococcus faecalis, 26975_1#33) loaded with 66 markers.\n",
      "2024-02-25 13:21:14,499 [DEBUG - chronostrain.database.parser.json] - Strain LR961933.1 (Enterococcus faecalis, 26975_1#45) loaded with 64 markers.\n",
      "2024-02-25 13:21:14,515 [DEBUG - chronostrain.database.parser.json] - Strain LR961970.1 (Enterococcus faecalis, 26975_2#148) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,523 [DEBUG - chronostrain.database.parser.json] - Strain LR961961.1 (Enterococcus faecalis, 26975_1#66) loaded with 58 markers.\n",
      "2024-02-25 13:21:14,531 [DEBUG - chronostrain.database.parser.json] - Strain LR698844.1 (Enterococcus faecalis, MGYG-HGUT-01694) loaded with 62 markers.\n",
      "2024-02-25 13:21:14,540 [DEBUG - chronostrain.database.parser.json] - Strain CP102121.1 (Enterococcus faecalis, N775_PC2) loaded with 68 markers.\n",
      "2024-02-25 13:21:14,549 [DEBUG - chronostrain.database.parser.json] - Strain CP102120.1 (Enterococcus faecalis, N775_PC7) loaded with 68 markers.\n",
      "2024-02-25 13:21:14,557 [DEBUG - chronostrain.database.parser.json] - Strain CP133450.1 (Enterococcus faecalis, JF4A2234) loaded with 62 markers.\n",
      "2024-02-25 13:21:14,565 [DEBUG - chronostrain.database.parser.json] - Strain CP124931.1 (Enterococcus faecalis, EfsPF24-2) loaded with 64 markers.\n",
      "2024-02-25 13:21:14,574 [DEBUG - chronostrain.database.parser.json] - Strain CP124953.1 (Enterococcus faecalis, EfsC23) loaded with 64 markers.\n",
      "2024-02-25 13:21:14,591 [DEBUG - chronostrain.database.parser.json] - Strain CP124947.1 (Enterococcus faecalis, EfsC43) loaded with 73 markers.\n",
      "2024-02-25 13:21:14,599 [DEBUG - chronostrain.database.parser.json] - Strain CP124928.1 (Enterococcus faecalis, EfsPF27) loaded with 66 markers.\n",
      "2024-02-25 13:21:14,608 [DEBUG - chronostrain.database.parser.json] - Strain CP128464.1 (Enterococcus faecalis, RE25) loaded with 65 markers.\n",
      "2024-02-25 13:21:14,624 [DEBUG - chronostrain.database.parser.json] - Strain AP027302.1 (Enterococcus faecalis, N4E) loaded with 77 markers.\n",
      "2024-02-25 13:21:14,640 [DEBUG - chronostrain.database.parser.json] - Strain AP027136.1 (Enterococcus faecalis, JARB-HU0796) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,656 [DEBUG - chronostrain.database.parser.json] - Strain CP124923.1 (Enterococcus faecalis, EfsC11) loaded with 72 markers.\n",
      "2024-02-25 13:21:14,665 [DEBUG - chronostrain.database.parser.json] - Strain CP124916.1 (Enterococcus faecalis, EfsC17) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,673 [DEBUG - chronostrain.database.parser.json] - Strain CP124911.1 (Enterococcus faecalis, EfsC61) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,690 [DEBUG - chronostrain.database.parser.json] - Strain CP124901.1 (Enterococcus faecalis, EfsC94) loaded with 69 markers.\n",
      "2024-02-25 13:21:14,702 [DEBUG - chronostrain.database.parser.json] - Strain CP124882.1 (Enterococcus faecalis, EfsPF20) loaded with 66 markers.\n",
      "2024-02-25 13:21:14,711 [DEBUG - chronostrain.database.parser.json] - Strain CP118962.1 (Enterococcus faecalis, DSM 20478) loaded with 68 markers.\n",
      "2024-02-25 13:21:14,718 [DEBUG - chronostrain.database.parser.json] - Strain CP117970.1 (Enterococcus faecalis, B-537) loaded with 57 markers.\n",
      "2024-02-25 13:21:14,735 [DEBUG - chronostrain.database.parser.json] - Strain CP116962.1 (Enterococcus faecalis, DM86) loaded with 78 markers.\n",
      "2024-02-25 13:21:14,745 [DEBUG - chronostrain.database.parser.json] - Strain CP116571.1 (Enterococcus faecalis, K190-1) loaded with 67 markers.\n",
      "2024-02-25 13:21:14,754 [DEBUG - chronostrain.database.parser.json] - Strain CP110074.1 (Enterococcus faecalis, BE5) loaded with 68 markers.\n",
      "2024-02-25 13:21:14,762 [DEBUG - chronostrain.database.parser.json] - Strain CP110069.1 (Enterococcus faecalis, BE11) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,770 [DEBUG - chronostrain.database.parser.json] - Strain CP110063.1 (Enterococcus faecalis, BE15) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,778 [DEBUG - chronostrain.database.parser.json] - Strain CP110050.1 (Enterococcus faecalis, BE32) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,786 [DEBUG - chronostrain.database.parser.json] - Strain CP110047.1 (Enterococcus faecalis, BE33) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,795 [DEBUG - chronostrain.database.parser.json] - Strain CP110041.1 (Enterococcus faecalis, BE43) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,804 [DEBUG - chronostrain.database.parser.json] - Strain CP110032.1 (Enterococcus faecalis, BE57) loaded with 66 markers.\n",
      "2024-02-25 13:21:14,820 [DEBUG - chronostrain.database.parser.json] - Strain CP102065.1 (Enterococcus faecalis, JF3A-223) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,836 [DEBUG - chronostrain.database.parser.json] - Strain CP100596.1 (Enterococcus faecalis, Chr-JH 2-2) loaded with 72 markers.\n",
      "2024-02-25 13:21:14,846 [DEBUG - chronostrain.database.parser.json] - Strain CP053181.1 (Enterococcus faecalis, FMAMOB6) loaded with 63 markers.\n",
      "2024-02-25 13:21:14,862 [DEBUG - chronostrain.database.parser.json] - Strain CP097048.1 (Enterococcus faecalis, AT22) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,879 [DEBUG - chronostrain.database.parser.json] - Strain CP097046.1 (Enterococcus faecalis, AT29) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,894 [DEBUG - chronostrain.database.parser.json] - Strain CP097020.1 (Enterococcus faecalis, AT46a) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,910 [DEBUG - chronostrain.database.parser.json] - Strain CP097069.1 (Enterococcus faecalis, AT40a) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,926 [DEBUG - chronostrain.database.parser.json] - Strain CP097056.1 (Enterococcus faecalis, AT09) loaded with 71 markers.\n",
      "2024-02-25 13:21:14,934 [DEBUG - chronostrain.database.parser.json] - Strain CP097010.1 (Enterococcus faecalis, AT48) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,949 [DEBUG - chronostrain.database.parser.json] - Strain CP097066.1 (Enterococcus faecalis, AT49a) loaded with 70 markers.\n",
      "2024-02-25 13:21:14,958 [DEBUG - chronostrain.database.parser.json] - Strain CP064374.1 (Enterococcus faecalis, PartL-Efaecalis-RM8376) loaded with 68 markers.\n",
      "2024-02-25 13:21:14,966 [DEBUG - chronostrain.database.parser.json] - Strain CP091906.1 (Enterococcus faecalis, 2-1) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,974 [DEBUG - chronostrain.database.parser.json] - Strain CP090558.1 (Enterococcus faecalis, PCH555) loaded with 60 markers.\n",
      "2024-02-25 13:21:14,982 [DEBUG - chronostrain.database.parser.json] - Strain CP086564.1 (Enterococcus faecalis, E512-TC1) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,990 [DEBUG - chronostrain.database.parser.json] - Strain CP086566.1 (Enterococcus faecalis, E512-TC2) loaded with 61 markers.\n",
      "2024-02-25 13:21:14,999 [DEBUG - chronostrain.database.parser.json] - Strain CP086411.1 (Enterococcus faecalis, R5) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,007 [DEBUG - chronostrain.database.parser.json] - Strain CP085294.1 (Enterococcus faecalis, EFS108) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,022 [DEBUG - chronostrain.database.parser.json] - Strain CP078162.1 (Enterococcus faecalis, NS2) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,030 [DEBUG - chronostrain.database.parser.json] - Strain CP046022.1 (Enterococcus faecalis, BFF1B1) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,047 [DEBUG - chronostrain.database.parser.json] - Strain CP046112.1 (Enterococcus faecalis, 111540027-2) loaded with 73 markers.\n",
      "2024-02-25 13:21:15,064 [DEBUG - chronostrain.database.parser.json] - Strain CP046111.1 (Enterococcus faecalis, 111540047-1) loaded with 73 markers.\n",
      "2024-02-25 13:21:15,073 [DEBUG - chronostrain.database.parser.json] - Strain CP046108.1 (Enterococcus faecalis, 133170041-3) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,081 [DEBUG - chronostrain.database.parser.json] - Strain CP041012.1 (Enterococcus faecalis, FDAARGOS_611) loaded with 57 markers.\n",
      "2024-02-25 13:21:15,090 [DEBUG - chronostrain.database.parser.json] - Strain CP030042.1 (Enterococcus faecalis, C25) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,098 [DEBUG - chronostrain.database.parser.json] - Strain CP014949.1 (Enterococcus faecalis, LD33) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,107 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940420.1 (Enterococcus faecalis, WE0438) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,123 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962836.1 (Enterococcus faecalis, 28157_4#332) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962830.1 (Enterococcus faecalis, 27688_1#67) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962834.1 (Enterococcus faecalis, 28157_4#163) loaded with 60 markers.\n",
      "2024-02-25 13:21:15,155 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962818.1 (Enterococcus faecalis, 28099_2#212) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,163 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962777.1 (Enterococcus faecalis, 28157_4#156) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962792.1 (Enterococcus faecalis, 28157_4#209) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,180 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962811.1 (Enterococcus faecalis, 28157_4#319) loaded with 59 markers.\n",
      "2024-02-25 13:21:15,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962802.1 (Enterococcus faecalis, 28099_2#176) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,197 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962732.1 (Enterococcus faecalis, 26975_2#42) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,206 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962720.1 (Enterococcus faecalis, 26975_2#37) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,214 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962721.1 (Enterococcus faecalis, 26975_2#35) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962717.1 (Enterococcus faecalis, 26975_2#227) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962698.1 (Enterococcus faecalis, 26975_2#31) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962694.1 (Enterococcus faecalis, 26975_2#30) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962695.1 (Enterococcus faecalis, 26975_2#26) loaded with 70 markers.\n",
      "2024-02-25 13:21:15,266 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962654.1 (Enterococcus faecalis, 26975_2#19) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962656.1 (Enterococcus faecalis, 26975_2#20) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962663.1 (Enterococcus faecalis, 26975_2#18) loaded with 70 markers.\n",
      "2024-02-25 13:21:15,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962670.1 (Enterococcus faecalis, 26975_2#16) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,315 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962649.1 (Enterococcus faecalis, 26975_1#39) loaded with 64 markers.\n",
      "2024-02-25 13:21:15,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962620.1 (Enterococcus faecalis, 27688_1#112) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,332 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962608.1 (Enterococcus faecalis, 27688_1#285) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,348 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962589.1 (Enterococcus faecalis, 27725_1#72) loaded with 70 markers.\n",
      "2024-02-25 13:21:15,355 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962580.1 (Enterococcus faecalis, 27688_1#212) loaded with 56 markers.\n",
      "2024-02-25 13:21:15,371 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962572.1 (Enterococcus faecalis, 27725_1#11) loaded with 70 markers.\n",
      "2024-02-25 13:21:15,379 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962579.1 (Enterococcus faecalis, 27725_1#112) loaded with 62 markers.\n",
      "2024-02-25 13:21:15,387 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962554.1 (Enterococcus faecalis, 27725_1#59) loaded with 58 markers.\n",
      "2024-02-25 13:21:15,404 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962512.1 (Enterococcus faecalis, 28099_2#233) loaded with 72 markers.\n",
      "2024-02-25 13:21:15,412 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962466.1 (Enterococcus faecalis, 28157_4#166) loaded with 64 markers.\n",
      "2024-02-25 13:21:15,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962486.1 (Enterococcus faecalis, 28157_4#109) loaded with 64 markers.\n",
      "2024-02-25 13:21:15,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962461.1 (Enterococcus faecalis, 28157_4#40) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962437.1 (Enterococcus faecalis, 28157_4#177) loaded with 62 markers.\n",
      "2024-02-25 13:21:15,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962414.1 (Enterococcus faecalis, 28157_4#92) loaded with 69 markers.\n",
      "2024-02-25 13:21:15,461 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962415.1 (Enterococcus faecalis, 28157_4#72) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,470 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962382.1 (Enterococcus faecalis, 28157_4#258) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962400.1 (Enterococcus faecalis, 28157_4#152) loaded with 66 markers.\n",
      "2024-02-25 13:21:15,494 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962390.1 (Enterococcus faecalis, 28157_4#191) loaded with 69 markers.\n",
      "2024-02-25 13:21:15,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962370.1 (Enterococcus faecalis, 28157_4#221) loaded with 69 markers.\n",
      "2024-02-25 13:21:15,519 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962351.1 (Enterococcus faecalis, 28157_4#38) loaded with 77 markers.\n",
      "2024-02-25 13:21:15,534 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962319.1 (Enterococcus faecalis, 27688_1#235) loaded with 68 markers.\n",
      "2024-02-25 13:21:15,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962334.1 (Enterococcus faecalis, 28099_2#67) loaded with 59 markers.\n",
      "2024-02-25 13:21:15,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962340.1 (Enterococcus faecalis, 27725_1#307) loaded with 64 markers.\n",
      "2024-02-25 13:21:15,559 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962326.1 (Enterococcus faecalis, 28157_4#208) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,567 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962311.1 (Enterococcus faecalis, 27725_1#24) loaded with 57 markers.\n",
      "2024-02-25 13:21:15,575 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962323.1 (Enterococcus faecalis, 27688_1#292) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962343.1 (Enterococcus faecalis, 28157_4#168) loaded with 73 markers.\n",
      "2024-02-25 13:21:15,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962293.1 (Enterococcus faecalis, 27688_1#170) loaded with 71 markers.\n",
      "2024-02-25 13:21:15,615 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962296.1 (Enterococcus faecalis, 27688_1#305) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,623 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962300.1 (Enterococcus faecalis, 28099_2#254) loaded with 59 markers.\n",
      "2024-02-25 13:21:15,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962283.1 (Enterococcus faecalis, 27688_1#230) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962256.1 (Enterococcus faecalis, 27688_1#202) loaded with 69 markers.\n",
      "2024-02-25 13:21:15,656 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962242.1 (Enterococcus faecalis, 27688_1#101) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,664 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962236.1 (Enterococcus faecalis, 27688_1#380) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,672 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962200.1 (Enterococcus faecalis, 27688_1#265) loaded with 59 markers.\n",
      "2024-02-25 13:21:15,688 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962190.1 (Enterococcus faecalis, 27688_1#241) loaded with 71 markers.\n",
      "2024-02-25 13:21:15,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962217.1 (Enterococcus faecalis, 27688_1#68) loaded with 69 markers.\n",
      "2024-02-25 13:21:15,713 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962194.1 (Enterococcus faecalis, 27688_1#304) loaded with 70 markers.\n",
      "2024-02-25 13:21:15,721 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962180.1 (Enterococcus faecalis, 27725_1#206) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,737 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962182.1 (Enterococcus faecalis, 27725_1#182) loaded with 71 markers.\n",
      "2024-02-25 13:21:15,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962123.1 (Enterococcus faecalis, 26975_1#372) loaded with 73 markers.\n",
      "2024-02-25 13:21:15,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962097.1 (Enterococcus faecalis, 26975_2#226) loaded with 59 markers.\n",
      "2024-02-25 13:21:15,771 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962089.1 (Enterococcus faecalis, 26975_1#241) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,787 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962101.1 (Enterococcus faecalis, 26975_1#280) loaded with 72 markers.\n",
      "2024-02-25 13:21:15,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962093.1 (Enterococcus faecalis, 26975_1#268) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,809 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962110.1 (Enterococcus faecalis, 26975_1#236) loaded with 63 markers.\n",
      "2024-02-25 13:21:15,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962083.1 (Enterococcus faecalis, 26975_1#194) loaded with 61 markers.\n",
      "2024-02-25 13:21:15,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962043.1 (Enterococcus faecalis, 26975_2#175) loaded with 72 markers.\n",
      "2024-02-25 13:21:15,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962049.1 (Enterococcus faecalis, 26975_2#201) loaded with 67 markers.\n",
      "2024-02-25 13:21:15,917 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962072.1 (Enterococcus faecalis, 26975_2#204) loaded with 73 markers.\n",
      "2024-02-25 13:21:15,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962063.1 (Enterococcus faecalis, 26975_2#198) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,941 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962051.1 (Enterococcus faecalis, 26975_2#190) loaded with 65 markers.\n",
      "2024-02-25 13:21:15,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962023.1 (Enterococcus faecalis, 26975_2#165) loaded with 71 markers.\n",
      "2024-02-25 13:21:15,973 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962019.1 (Enterococcus faecalis, 26975_2#162) loaded with 72 markers.\n",
      "2024-02-25 13:21:15,981 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962021.1 (Enterococcus faecalis, 26975_1#160) loaded with 62 markers.\n",
      "2024-02-25 13:21:15,989 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961989.1 (Enterococcus faecalis, 26975_1#111) loaded with 60 markers.\n",
      "2024-02-25 13:21:15,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961999.1 (Enterococcus faecalis, 26975_1#367) loaded with 71 markers.\n",
      "2024-02-25 13:21:16,007 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961978.1 (Enterococcus faecalis, 26975_1#63) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,015 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961965.1 (Enterococcus faecalis, 26975_1#31) loaded with 66 markers.\n",
      "2024-02-25 13:21:16,023 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961945.1 (Enterococcus faecalis, 26975_1#25) loaded with 58 markers.\n",
      "2024-02-25 13:21:16,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961925.1 (Enterococcus faecalis, 26975_2#147) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961938.1 (Enterococcus faecalis, 26975_1#48) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961975.1 (Enterococcus faecalis, 26975_1#37) loaded with 59 markers.\n",
      "2024-02-25 13:21:16,078 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961966.1 (Enterococcus faecalis, 26975_1#33) loaded with 66 markers.\n",
      "2024-02-25 13:21:16,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961933.1 (Enterococcus faecalis, 26975_1#45) loaded with 64 markers.\n",
      "2024-02-25 13:21:16,103 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961970.1 (Enterococcus faecalis, 26975_2#148) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,111 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961961.1 (Enterococcus faecalis, 26975_1#66) loaded with 58 markers.\n",
      "2024-02-25 13:21:16,119 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR698844.1 (Enterococcus faecalis, MGYG-HGUT-01694) loaded with 62 markers.\n",
      "2024-02-25 13:21:16,128 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102121.1 (Enterococcus faecalis, N775_PC2) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102120.1 (Enterococcus faecalis, N775_PC7) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP133450.1 (Enterococcus faecalis, JF4A2234) loaded with 62 markers.\n",
      "2024-02-25 13:21:16,153 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124931.1 (Enterococcus faecalis, EfsPF24-2) loaded with 64 markers.\n",
      "2024-02-25 13:21:16,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124953.1 (Enterococcus faecalis, EfsC23) loaded with 64 markers.\n",
      "2024-02-25 13:21:16,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124947.1 (Enterococcus faecalis, EfsC43) loaded with 73 markers.\n",
      "2024-02-25 13:21:16,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124928.1 (Enterococcus faecalis, EfsPF27) loaded with 66 markers.\n",
      "2024-02-25 13:21:16,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP128464.1 (Enterococcus faecalis, RE25) loaded with 65 markers.\n",
      "2024-02-25 13:21:16,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP027302.1 (Enterococcus faecalis, N4E) loaded with 77 markers.\n",
      "2024-02-25 13:21:16,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP027136.1 (Enterococcus faecalis, JARB-HU0796) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,248 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124923.1 (Enterococcus faecalis, EfsC11) loaded with 72 markers.\n",
      "2024-02-25 13:21:16,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124916.1 (Enterococcus faecalis, EfsC17) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124911.1 (Enterococcus faecalis, EfsC61) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,283 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124901.1 (Enterococcus faecalis, EfsC94) loaded with 69 markers.\n",
      "2024-02-25 13:21:16,291 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124882.1 (Enterococcus faecalis, EfsPF20) loaded with 66 markers.\n",
      "2024-02-25 13:21:16,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118962.1 (Enterococcus faecalis, DSM 20478) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,308 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP117970.1 (Enterococcus faecalis, B-537) loaded with 57 markers.\n",
      "2024-02-25 13:21:16,329 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116962.1 (Enterococcus faecalis, DM86) loaded with 78 markers.\n",
      "2024-02-25 13:21:16,337 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116571.1 (Enterococcus faecalis, K190-1) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,346 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110074.1 (Enterococcus faecalis, BE5) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110069.1 (Enterococcus faecalis, BE11) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,369 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110063.1 (Enterococcus faecalis, BE15) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,378 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110050.1 (Enterococcus faecalis, BE32) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,386 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110047.1 (Enterococcus faecalis, BE33) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,394 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110041.1 (Enterococcus faecalis, BE43) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,411 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102065.1 (Enterococcus faecalis, JF3A-223) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,426 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP100596.1 (Enterococcus faecalis, Chr-JH 2-2) loaded with 72 markers.\n",
      "2024-02-25 13:21:16,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053181.1 (Enterococcus faecalis, FMAMOB6) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,450 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097048.1 (Enterococcus faecalis, AT22) loaded with 71 markers.\n",
      "2024-02-25 13:21:16,470 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097046.1 (Enterococcus faecalis, AT29) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,485 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097020.1 (Enterococcus faecalis, AT46a) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,501 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097069.1 (Enterococcus faecalis, AT40a) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,517 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097056.1 (Enterococcus faecalis, AT09) loaded with 71 markers.\n",
      "2024-02-25 13:21:16,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097010.1 (Enterococcus faecalis, AT48) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,541 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097066.1 (Enterococcus faecalis, AT49a) loaded with 70 markers.\n",
      "2024-02-25 13:21:16,550 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064374.1 (Enterococcus faecalis, PartL-Efaecalis-RM8376) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,558 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091906.1 (Enterococcus faecalis, 2-1) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,566 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090558.1 (Enterococcus faecalis, PCH555) loaded with 60 markers.\n",
      "2024-02-25 13:21:16,574 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086564.1 (Enterococcus faecalis, E512-TC1) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086566.1 (Enterococcus faecalis, E512-TC2) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,591 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086411.1 (Enterococcus faecalis, R5) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,599 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085294.1 (Enterococcus faecalis, EFS108) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078162.1 (Enterococcus faecalis, NS2) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046022.1 (Enterococcus faecalis, BFF1B1) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,639 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046112.1 (Enterococcus faecalis, 111540027-2) loaded with 73 markers.\n",
      "2024-02-25 13:21:16,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046111.1 (Enterococcus faecalis, 111540047-1) loaded with 73 markers.\n",
      "2024-02-25 13:21:16,668 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046108.1 (Enterococcus faecalis, 133170041-3) loaded with 66 markers.\n",
      "2024-02-25 13:21:16,676 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041012.1 (Enterococcus faecalis, FDAARGOS_611) loaded with 57 markers.\n",
      "2024-02-25 13:21:16,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030042.1 (Enterococcus faecalis, C25) loaded with 63 markers.\n",
      "2024-02-25 13:21:16,699 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014949.1 (Enterococcus faecalis, LD33) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,711 [DEBUG - chronostrain.database.parser.json] - Strain CP092784.1 (Enterococcus sp., LX10) loaded with 59 markers.\n",
      "2024-02-25 13:21:16,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092784.1 (Enterococcus sp., LX10) loaded with 59 markers.\n",
      "2024-02-25 13:21:16,736 [DEBUG - chronostrain.database.parser.json] - Strain OD940431.1 (Enterococcus faecalis, TM6294) loaded with 76 markers.\n",
      "2024-02-25 13:21:16,752 [DEBUG - chronostrain.database.parser.json] - Strain LR962842.1 (Enterococcus faecalis, 28157_4#211) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,761 [DEBUG - chronostrain.database.parser.json] - Strain LR962843.1 (Enterococcus faecalis, 28157_4#42) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,770 [DEBUG - chronostrain.database.parser.json] - Strain LR962850.1 (Enterococcus faecalis, 27688_1#342) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,779 [DEBUG - chronostrain.database.parser.json] - Strain LR962840.1 (Enterococcus faecalis, 27725_1#165) loaded with 67 markers.\n",
      "2024-02-25 13:21:16,794 [DEBUG - chronostrain.database.parser.json] - Strain LR962857.1 (Enterococcus faecalis, 28157_4#64) loaded with 68 markers.\n",
      "2024-02-25 13:21:16,803 [DEBUG - chronostrain.database.parser.json] - Strain LR962832.1 (Enterococcus faecalis, 28157_4#150) loaded with 60 markers.\n",
      "2024-02-25 13:21:16,811 [DEBUG - chronostrain.database.parser.json] - Strain LR962814.1 (Enterococcus faecalis, 28157_4#185) loaded with 59 markers.\n",
      "2024-02-25 13:21:16,819 [DEBUG - chronostrain.database.parser.json] - Strain LR962775.1 (Enterococcus faecalis, 28099_2#75) loaded with 59 markers.\n",
      "2024-02-25 13:21:16,841 [DEBUG - chronostrain.database.parser.json] - Strain LR962788.1 (Enterococcus faecalis, 27688_1#210) loaded with 80 markers.\n",
      "2024-02-25 13:21:16,849 [DEBUG - chronostrain.database.parser.json] - Strain LR962736.1 (Enterococcus faecalis, 26975_2#41) loaded with 62 markers.\n",
      "2024-02-25 13:21:16,865 [DEBUG - chronostrain.database.parser.json] - Strain LR962729.1 (Enterococcus faecalis, 26975_2#44) loaded with 69 markers.\n",
      "2024-02-25 13:21:16,873 [DEBUG - chronostrain.database.parser.json] - Strain LR962726.1 (Enterococcus faecalis, 26975_2#242) loaded with 61 markers.\n",
      "2024-02-25 13:21:16,889 [DEBUG - chronostrain.database.parser.json] - Strain LR962711.1 (Enterococcus faecalis, 26975_2#228) loaded with 69 markers.\n",
      "2024-02-25 13:21:16,906 [DEBUG - chronostrain.database.parser.json] - Strain LR962714.1 (Enterococcus faecalis, 26975_2#38) loaded with 76 markers.\n",
      "2024-02-25 13:21:16,922 [DEBUG - chronostrain.database.parser.json] - Strain LR962658.1 (Enterococcus faecalis, 26975_1#204) loaded with 76 markers.\n",
      "2024-02-25 13:21:16,939 [DEBUG - chronostrain.database.parser.json] - Strain LR962671.1 (Enterococcus faecalis, 26975_1#150) loaded with 79 markers.\n",
      "2024-02-25 13:21:16,955 [DEBUG - chronostrain.database.parser.json] - Strain LR962632.1 (Enterococcus faecalis, 27688_1#382) loaded with 72 markers.\n",
      "2024-02-25 13:21:16,971 [DEBUG - chronostrain.database.parser.json] - Strain LR962604.1 (Enterococcus faecalis, 27688_1#124) loaded with 71 markers.\n",
      "2024-02-25 13:21:16,987 [DEBUG - chronostrain.database.parser.json] - Strain LR962606.1 (Enterococcus faecalis, 27688_1#55) loaded with 72 markers.\n",
      "2024-02-25 13:21:16,995 [DEBUG - chronostrain.database.parser.json] - Strain LR962634.1 (Enterococcus faecalis, 27688_1#136) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,003 [DEBUG - chronostrain.database.parser.json] - Strain LR962618.1 (Enterococcus faecalis, 27688_1#160) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,018 [DEBUG - chronostrain.database.parser.json] - Strain LR962602.1 (Enterococcus faecalis, 27688_1#204) loaded with 67 markers.\n",
      "2024-02-25 13:21:17,034 [DEBUG - chronostrain.database.parser.json] - Strain LR962598.1 (Enterococcus faecalis, 27688_1#341) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,043 [DEBUG - chronostrain.database.parser.json] - Strain LR962600.1 (Enterococcus faecalis, 27688_1#220) loaded with 65 markers.\n",
      "2024-02-25 13:21:17,051 [DEBUG - chronostrain.database.parser.json] - Strain LR962591.1 (Enterococcus faecalis, 27688_1#337) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,059 [DEBUG - chronostrain.database.parser.json] - Strain LR962595.1 (Enterococcus faecalis, 27688_1#260) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,077 [DEBUG - chronostrain.database.parser.json] - Strain LR962576.1 (Enterococcus faecalis, 27688_1#43) loaded with 74 markers.\n",
      "2024-02-25 13:21:17,093 [DEBUG - chronostrain.database.parser.json] - Strain LR962567.1 (Enterococcus faecalis, 27688_1#78) loaded with 72 markers.\n",
      "2024-02-25 13:21:17,108 [DEBUG - chronostrain.database.parser.json] - Strain LR962546.1 (Enterococcus faecalis, 27725_1#19) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,124 [DEBUG - chronostrain.database.parser.json] - Strain LR962545.1 (Enterococcus faecalis, 28157_4#338) loaded with 67 markers.\n",
      "2024-02-25 13:21:17,132 [DEBUG - chronostrain.database.parser.json] - Strain LR962531.1 (Enterococcus faecalis, 28099_2#221) loaded with 62 markers.\n",
      "2024-02-25 13:21:17,148 [DEBUG - chronostrain.database.parser.json] - Strain LR962539.1 (Enterococcus faecalis, 27688_1#172) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,156 [DEBUG - chronostrain.database.parser.json] - Strain LR962497.1 (Enterococcus faecalis, 28099_2#139) loaded with 64 markers.\n",
      "2024-02-25 13:21:17,173 [DEBUG - chronostrain.database.parser.json] - Strain LR962434.1 (Enterococcus faecalis, 28157_4#264) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,181 [DEBUG - chronostrain.database.parser.json] - Strain LR962454.1 (Enterococcus faecalis, 28157_4#276) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,198 [DEBUG - chronostrain.database.parser.json] - Strain LR962432.1 (Enterococcus faecalis, 28157_4#282) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,215 [DEBUG - chronostrain.database.parser.json] - Strain LR962450.1 (Enterococcus faecalis, 28099_2#314) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,230 [DEBUG - chronostrain.database.parser.json] - Strain LR962393.1 (Enterococcus faecalis, 28157_4#241) loaded with 67 markers.\n",
      "2024-02-25 13:21:17,246 [DEBUG - chronostrain.database.parser.json] - Strain LR962375.1 (Enterococcus faecalis, 28099_2#241) loaded with 75 markers.\n",
      "2024-02-25 13:21:17,258 [DEBUG - chronostrain.database.parser.json] - Strain LR962346.1 (Enterococcus faecalis, 28157_4#316) loaded with 61 markers.\n",
      "2024-02-25 13:21:17,275 [DEBUG - chronostrain.database.parser.json] - Strain LR962348.1 (Enterococcus faecalis, 28157_4#300) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,291 [DEBUG - chronostrain.database.parser.json] - Strain LR962315.1 (Enterococcus faecalis, 28157_4#164) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,299 [DEBUG - chronostrain.database.parser.json] - Strain LR962328.1 (Enterococcus faecalis, 27725_1#347) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,316 [DEBUG - chronostrain.database.parser.json] - Strain LR962239.1 (Enterococcus faecalis, 27688_1#157) loaded with 75 markers.\n",
      "2024-02-25 13:21:17,334 [DEBUG - chronostrain.database.parser.json] - Strain LR962234.1 (Enterococcus faecalis, 27688_1#21) loaded with 76 markers.\n",
      "2024-02-25 13:21:17,342 [DEBUG - chronostrain.database.parser.json] - Strain LR962225.1 (Enterococcus faecalis, 27688_1#80) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,357 [DEBUG - chronostrain.database.parser.json] - Strain LR962210.1 (Enterococcus faecalis, 27688_1#76) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,375 [DEBUG - chronostrain.database.parser.json] - Strain LR962213.1 (Enterococcus faecalis, 27688_1#45) loaded with 83 markers.\n",
      "2024-02-25 13:21:17,391 [DEBUG - chronostrain.database.parser.json] - Strain LR962189.1 (Enterococcus faecalis, 27725_1#226) loaded with 72 markers.\n",
      "2024-02-25 13:21:17,399 [DEBUG - chronostrain.database.parser.json] - Strain LR962188.1 (Enterococcus faecalis, 27688_1#303) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,408 [DEBUG - chronostrain.database.parser.json] - Strain LR962197.1 (Enterococcus faecalis, 27688_1#375) loaded with 61 markers.\n",
      "2024-02-25 13:21:17,425 [DEBUG - chronostrain.database.parser.json] - Strain LR962230.1 (Enterococcus faecalis, 27725_1#242) loaded with 72 markers.\n",
      "2024-02-25 13:21:17,433 [DEBUG - chronostrain.database.parser.json] - Strain LR962171.1 (Enterococcus faecalis, 26975_2#231) loaded with 69 markers.\n",
      "2024-02-25 13:21:17,449 [DEBUG - chronostrain.database.parser.json] - Strain LR962163.1 (Enterococcus faecalis, 26975_2#107) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,456 [DEBUG - chronostrain.database.parser.json] - Strain LR962161.1 (Enterococcus faecalis, 26975_2#91) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,473 [DEBUG - chronostrain.database.parser.json] - Strain LR962132.1 (Enterococcus faecalis, 26975_1#369) loaded with 78 markers.\n",
      "2024-02-25 13:21:17,489 [DEBUG - chronostrain.database.parser.json] - Strain LR962137.1 (Enterococcus faecalis, 26975_2#247) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,505 [DEBUG - chronostrain.database.parser.json] - Strain LR962112.1 (Enterococcus faecalis, 26975_1#283) loaded with 75 markers.\n",
      "2024-02-25 13:21:17,522 [DEBUG - chronostrain.database.parser.json] - Strain LR962103.1 (Enterococcus faecalis, 26975_1#246) loaded with 77 markers.\n",
      "2024-02-25 13:21:17,538 [DEBUG - chronostrain.database.parser.json] - Strain LR962107.1 (Enterococcus faecalis, 26975_1#214) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,554 [DEBUG - chronostrain.database.parser.json] - Strain LR962075.1 (Enterococcus faecalis, 26975_2#206) loaded with 75 markers.\n",
      "2024-02-25 13:21:17,571 [DEBUG - chronostrain.database.parser.json] - Strain LR962039.1 (Enterococcus faecalis, 26975_1#366) loaded with 78 markers.\n",
      "2024-02-25 13:21:17,579 [DEBUG - chronostrain.database.parser.json] - Strain LR962034.1 (Enterococcus faecalis, 26975_1#180) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,596 [DEBUG - chronostrain.database.parser.json] - Strain LR962030.1 (Enterococcus faecalis, 26975_2#195) loaded with 79 markers.\n",
      "2024-02-25 13:21:17,604 [DEBUG - chronostrain.database.parser.json] - Strain LR962017.1 (Enterococcus faecalis, 26975_2#160) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,612 [DEBUG - chronostrain.database.parser.json] - Strain LR962010.1 (Enterococcus faecalis, 26975_2#152) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,627 [DEBUG - chronostrain.database.parser.json] - Strain LR961996.1 (Enterococcus faecalis, 26975_1#117) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,643 [DEBUG - chronostrain.database.parser.json] - Strain LR961994.1 (Enterococcus faecalis, 26975_1#118) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,658 [DEBUG - chronostrain.database.parser.json] - Strain LR961986.1 (Enterococcus faecalis, 26975_1#129) loaded with 67 markers.\n",
      "2024-02-25 13:21:17,677 [DEBUG - chronostrain.database.parser.json] - Strain LR962008.1 (Enterococcus faecalis, 26975_1#88) loaded with 72 markers.\n",
      "2024-02-25 13:21:17,696 [DEBUG - chronostrain.database.parser.json] - Strain LR961981.1 (Enterococcus faecalis, 26975_1#105) loaded with 79 markers.\n",
      "2024-02-25 13:21:17,706 [DEBUG - chronostrain.database.parser.json] - Strain LR961957.1 (Enterococcus faecalis, 26975_1#22) loaded with 64 markers.\n",
      "2024-02-25 13:21:17,724 [DEBUG - chronostrain.database.parser.json] - Strain LR961953.1 (Enterococcus faecalis, 26975_1#18) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,734 [DEBUG - chronostrain.database.parser.json] - Strain LR961923.1 (Enterococcus faecalis, 26975_1#52) loaded with 69 markers.\n",
      "2024-02-25 13:21:17,744 [DEBUG - chronostrain.database.parser.json] - Strain LR961959.1 (Enterococcus faecalis, 26975_1#5) loaded with 62 markers.\n",
      "2024-02-25 13:21:17,759 [DEBUG - chronostrain.database.parser.json] - Strain CP124778.1 (Enterococcus faecalis, HL1) loaded with 67 markers.\n",
      "2024-02-25 13:21:17,776 [DEBUG - chronostrain.database.parser.json] - Strain CP127177.1 (Enterococcus faecalis, 90_2023) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,785 [DEBUG - chronostrain.database.parser.json] - Strain CP109611.1 (Enterococcus faecalis, AFL-109F) loaded with 60 markers.\n",
      "2024-02-25 13:21:17,793 [DEBUG - chronostrain.database.parser.json] - Strain CP124914.1 (Enterococcus faecalis, EfsC8) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,809 [DEBUG - chronostrain.database.parser.json] - Strain CP124891.1 (Enterococcus faecalis, EfsC130) loaded with 71 markers.\n",
      "2024-02-25 13:21:17,817 [DEBUG - chronostrain.database.parser.json] - Strain CP118048.1 (Enterococcus faecalis, VSI48) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,827 [DEBUG - chronostrain.database.parser.json] - Strain CP117511.1 (Enterococcus faecalis, CQNBZ21-135) loaded with 62 markers.\n",
      "2024-02-25 13:21:17,845 [DEBUG - chronostrain.database.parser.json] - Strain CP117503.1 (Enterococcus faecalis, CQJXZ21-076) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,854 [DEBUG - chronostrain.database.parser.json] - Strain CP121233.1 (Enterococcus faecalis, SVJ-EF01) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,862 [DEBUG - chronostrain.database.parser.json] - Strain CP116566.1 (Enterococcus faecalis, K191-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,870 [DEBUG - chronostrain.database.parser.json] - Strain CP116564.1 (Enterococcus faecalis, K188-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,878 [DEBUG - chronostrain.database.parser.json] - Strain CP116561.1 (Enterococcus faecalis, K72-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:17,894 [DEBUG - chronostrain.database.parser.json] - Strain CP116569.1 (Enterococcus faecalis, K198-1) loaded with 69 markers.\n",
      "2024-02-25 13:21:17,903 [DEBUG - chronostrain.database.parser.json] - Strain AP026729.1 (Enterococcus faecalis, SVR2332) loaded with 68 markers.\n",
      "2024-02-25 13:21:17,912 [DEBUG - chronostrain.database.parser.json] - Strain AP026726.1 (Enterococcus faecalis, SVR2331) loaded with 70 markers.\n",
      "2024-02-25 13:21:17,930 [DEBUG - chronostrain.database.parser.json] - Strain AP026721.1 (Enterococcus faecalis, SVR2330) loaded with 82 markers.\n",
      "2024-02-25 13:21:17,938 [DEBUG - chronostrain.database.parser.json] - Strain CP115992.1 (Enterococcus faecalis, ESC1) loaded with 61 markers.\n",
      "2024-02-25 13:21:17,947 [DEBUG - chronostrain.database.parser.json] - Strain CP110038.1 (Enterococcus faecalis, BE51) loaded with 63 markers.\n",
      "2024-02-25 13:21:17,955 [DEBUG - chronostrain.database.parser.json] - Strain CP110036.1 (Enterococcus faecalis, BE52) loaded with 61 markers.\n",
      "2024-02-25 13:21:17,970 [DEBUG - chronostrain.database.parser.json] - Strain CP110020.1 (Enterococcus faecalis, BE69) loaded with 72 markers.\n",
      "2024-02-25 13:21:17,986 [DEBUG - chronostrain.database.parser.json] - Strain CP110019.1 (Enterococcus faecalis, BE70) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,002 [DEBUG - chronostrain.database.parser.json] - Strain CP098418.1 (Enterococcus faecalis, CQ025) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,018 [DEBUG - chronostrain.database.parser.json] - Strain CP098025.1 (Enterococcus faecalis, QZ076) loaded with 73 markers.\n",
      "2024-02-25 13:21:18,033 [DEBUG - chronostrain.database.parser.json] - Strain CP091237.1 (Enterococcus faecalis, 1521) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,048 [DEBUG - chronostrain.database.parser.json] - Strain CP091240.1 (Enterococcus faecalis, 249) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,064 [DEBUG - chronostrain.database.parser.json] - Strain CP091227.1 (Enterococcus faecalis, 661) loaded with 69 markers.\n",
      "2024-02-25 13:21:18,083 [DEBUG - chronostrain.database.parser.json] - Strain CP097059.1 (Enterococcus faecalis, AT04) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,099 [DEBUG - chronostrain.database.parser.json] - Strain CP097030.1 (Enterococcus faecalis, AT43a) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,114 [DEBUG - chronostrain.database.parser.json] - Strain CP097008.1 (Enterococcus faecalis, AT50) loaded with 70 markers.\n",
      "2024-02-25 13:21:18,130 [DEBUG - chronostrain.database.parser.json] - Strain CP097038.1 (Enterococcus faecalis, AT39) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,145 [DEBUG - chronostrain.database.parser.json] - Strain CP097004.1 (Enterococcus faecalis, LS05-2) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,153 [DEBUG - chronostrain.database.parser.json] - Strain CP110289.1 (Enterococcus faecalis, AKSZ-208) loaded with 61 markers.\n",
      "2024-02-25 13:21:18,170 [DEBUG - chronostrain.database.parser.json] - Strain CP093957.1 (Enterococcus faecalis, NY13321) loaded with 74 markers.\n",
      "2024-02-25 13:21:18,185 [DEBUG - chronostrain.database.parser.json] - Strain CP092907.1 (Enterococcus faecalis, LJX 0909) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,194 [DEBUG - chronostrain.database.parser.json] - Strain CP091886.1 (Enterococcus faecalis, 143-1) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,209 [DEBUG - chronostrain.database.parser.json] - Strain CP063980.1 (Enterococcus faecalis, AR_0780) loaded with 70 markers.\n",
      "2024-02-25 13:21:18,225 [DEBUG - chronostrain.database.parser.json] - Strain CP085289.1 (Enterococcus faecalis, EFS17) loaded with 76 markers.\n",
      "2024-02-25 13:21:18,240 [DEBUG - chronostrain.database.parser.json] - Strain CP045598.1 (Enterococcus faecalis, TK-P4B) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,248 [DEBUG - chronostrain.database.parser.json] - Strain CP050491.1 (Enterococcus faecalis, EGM183) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,264 [DEBUG - chronostrain.database.parser.json] - Strain CP060804.1 (Enterococcus faecalis, SF28073) loaded with 74 markers.\n",
      "2024-02-25 13:21:18,272 [DEBUG - chronostrain.database.parser.json] - Strain CP030045.1 (Enterococcus faecalis, C54) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,280 [DEBUG - chronostrain.database.parser.json] - Strain CP039434.1 (Enterococcus faecalis, SGAir0397) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,297 [DEBUG - chronostrain.database.parser.json] - Strain CP028835.1 (Enterococcus faecalis, FC) loaded with 76 markers.\n",
      "2024-02-25 13:21:18,314 [DEBUG - chronostrain.database.parser.json] - Strain CP028720.1 (Enterococcus faecalis, CVM N48037F) loaded with 75 markers.\n",
      "2024-02-25 13:21:18,329 [DEBUG - chronostrain.database.parser.json] - Strain AP018543.1 (Enterococcus faecalis, KUB3007) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,345 [DEBUG - chronostrain.database.parser.json] - Strain AP018538.1 (Enterococcus faecalis, KUB3006) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,361 [DEBUG - chronostrain.database.parser.json] - Strain CP033787.1 (Enterococcus faecalis, FDAARGOS_528) loaded with 79 markers.\n",
      "2024-02-25 13:21:18,378 [DEBUG - chronostrain.database.parser.json] - Strain CP031027.1 (Enterococcus faecalis, TY1) loaded with 75 markers.\n",
      "2024-02-25 13:21:18,395 [DEBUG - chronostrain.database.parser.json] - Strain CP029612.1 (Enterococcus faecalis, H25) loaded with 75 markers.\n",
      "2024-02-25 13:21:18,413 [DEBUG - chronostrain.database.parser.json] - Strain CP019512.1 (Enterococcus faecalis, CLB21560) loaded with 79 markers.\n",
      "2024-02-25 13:21:18,431 [DEBUG - chronostrain.database.parser.json] - Strain CP015410.2 (Enterococcus faecalis, KB1) loaded with 75 markers.\n",
      "2024-02-25 13:21:18,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940431.1 (Enterococcus faecalis, TM6294) loaded with 76 markers.\n",
      "2024-02-25 13:21:18,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962842.1 (Enterococcus faecalis, 28157_4#211) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962843.1 (Enterococcus faecalis, 28157_4#42) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,480 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962850.1 (Enterococcus faecalis, 27688_1#342) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,488 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962840.1 (Enterococcus faecalis, 27725_1#165) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,504 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962857.1 (Enterococcus faecalis, 28157_4#64) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962832.1 (Enterococcus faecalis, 28157_4#150) loaded with 60 markers.\n",
      "2024-02-25 13:21:18,520 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962814.1 (Enterococcus faecalis, 28157_4#185) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,528 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962775.1 (Enterococcus faecalis, 28099_2#75) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,546 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962788.1 (Enterococcus faecalis, 27688_1#210) loaded with 80 markers.\n",
      "2024-02-25 13:21:18,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962736.1 (Enterococcus faecalis, 26975_2#41) loaded with 62 markers.\n",
      "2024-02-25 13:21:18,570 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962729.1 (Enterococcus faecalis, 26975_2#44) loaded with 69 markers.\n",
      "2024-02-25 13:21:18,578 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962726.1 (Enterococcus faecalis, 26975_2#242) loaded with 61 markers.\n",
      "2024-02-25 13:21:18,594 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962711.1 (Enterococcus faecalis, 26975_2#228) loaded with 69 markers.\n",
      "2024-02-25 13:21:18,611 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962714.1 (Enterococcus faecalis, 26975_2#38) loaded with 76 markers.\n",
      "2024-02-25 13:21:18,629 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962658.1 (Enterococcus faecalis, 26975_1#204) loaded with 76 markers.\n",
      "2024-02-25 13:21:18,647 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962671.1 (Enterococcus faecalis, 26975_1#150) loaded with 79 markers.\n",
      "2024-02-25 13:21:18,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962632.1 (Enterococcus faecalis, 27688_1#382) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,679 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962604.1 (Enterococcus faecalis, 27688_1#124) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,695 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962606.1 (Enterococcus faecalis, 27688_1#55) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,703 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962634.1 (Enterococcus faecalis, 27688_1#136) loaded with 60 markers.\n",
      "2024-02-25 13:21:18,711 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962618.1 (Enterococcus faecalis, 27688_1#160) loaded with 60 markers.\n",
      "2024-02-25 13:21:18,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962602.1 (Enterococcus faecalis, 27688_1#204) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962598.1 (Enterococcus faecalis, 27688_1#341) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,751 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962600.1 (Enterococcus faecalis, 27688_1#220) loaded with 65 markers.\n",
      "2024-02-25 13:21:18,758 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962591.1 (Enterococcus faecalis, 27688_1#337) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,767 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962595.1 (Enterococcus faecalis, 27688_1#260) loaded with 59 markers.\n",
      "2024-02-25 13:21:18,784 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962576.1 (Enterococcus faecalis, 27688_1#43) loaded with 74 markers.\n",
      "2024-02-25 13:21:18,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962567.1 (Enterococcus faecalis, 27688_1#78) loaded with 72 markers.\n",
      "2024-02-25 13:21:18,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962546.1 (Enterococcus faecalis, 27725_1#19) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962545.1 (Enterococcus faecalis, 28157_4#338) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962531.1 (Enterococcus faecalis, 28099_2#221) loaded with 62 markers.\n",
      "2024-02-25 13:21:18,856 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962539.1 (Enterococcus faecalis, 27688_1#172) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962497.1 (Enterococcus faecalis, 28099_2#139) loaded with 64 markers.\n",
      "2024-02-25 13:21:18,879 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962434.1 (Enterococcus faecalis, 28157_4#264) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962454.1 (Enterococcus faecalis, 28157_4#276) loaded with 60 markers.\n",
      "2024-02-25 13:21:18,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962432.1 (Enterococcus faecalis, 28157_4#282) loaded with 71 markers.\n",
      "2024-02-25 13:21:18,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962450.1 (Enterococcus faecalis, 28099_2#314) loaded with 70 markers.\n",
      "2024-02-25 13:21:18,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962393.1 (Enterococcus faecalis, 28157_4#241) loaded with 67 markers.\n",
      "2024-02-25 13:21:18,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962375.1 (Enterococcus faecalis, 28099_2#241) loaded with 75 markers.\n",
      "2024-02-25 13:21:18,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962346.1 (Enterococcus faecalis, 28157_4#316) loaded with 61 markers.\n",
      "2024-02-25 13:21:18,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962348.1 (Enterococcus faecalis, 28157_4#300) loaded with 68 markers.\n",
      "2024-02-25 13:21:18,991 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962315.1 (Enterococcus faecalis, 28157_4#164) loaded with 70 markers.\n",
      "2024-02-25 13:21:18,999 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962328.1 (Enterococcus faecalis, 27725_1#347) loaded with 60 markers.\n",
      "2024-02-25 13:21:19,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962239.1 (Enterococcus faecalis, 27688_1#157) loaded with 75 markers.\n",
      "2024-02-25 13:21:19,033 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962234.1 (Enterococcus faecalis, 27688_1#21) loaded with 76 markers.\n",
      "2024-02-25 13:21:19,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962225.1 (Enterococcus faecalis, 27688_1#80) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,058 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962210.1 (Enterococcus faecalis, 27688_1#76) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962213.1 (Enterococcus faecalis, 27688_1#45) loaded with 83 markers.\n",
      "2024-02-25 13:21:19,091 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962189.1 (Enterococcus faecalis, 27725_1#226) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,099 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962188.1 (Enterococcus faecalis, 27688_1#303) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,107 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962197.1 (Enterococcus faecalis, 27688_1#375) loaded with 61 markers.\n",
      "2024-02-25 13:21:19,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962230.1 (Enterococcus faecalis, 27725_1#242) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,131 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962171.1 (Enterococcus faecalis, 26975_2#231) loaded with 69 markers.\n",
      "2024-02-25 13:21:19,146 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962163.1 (Enterococcus faecalis, 26975_2#107) loaded with 68 markers.\n",
      "2024-02-25 13:21:19,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962161.1 (Enterococcus faecalis, 26975_2#91) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962132.1 (Enterococcus faecalis, 26975_1#369) loaded with 78 markers.\n",
      "2024-02-25 13:21:19,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962137.1 (Enterococcus faecalis, 26975_2#247) loaded with 68 markers.\n",
      "2024-02-25 13:21:19,204 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962112.1 (Enterococcus faecalis, 26975_1#283) loaded with 75 markers.\n",
      "2024-02-25 13:21:19,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962103.1 (Enterococcus faecalis, 26975_1#246) loaded with 77 markers.\n",
      "2024-02-25 13:21:19,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962107.1 (Enterococcus faecalis, 26975_1#214) loaded with 70 markers.\n",
      "2024-02-25 13:21:19,256 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962075.1 (Enterococcus faecalis, 26975_2#206) loaded with 75 markers.\n",
      "2024-02-25 13:21:19,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962039.1 (Enterococcus faecalis, 26975_1#366) loaded with 78 markers.\n",
      "2024-02-25 13:21:19,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962034.1 (Enterococcus faecalis, 26975_1#180) loaded with 60 markers.\n",
      "2024-02-25 13:21:19,302 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962030.1 (Enterococcus faecalis, 26975_2#195) loaded with 79 markers.\n",
      "2024-02-25 13:21:19,310 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962017.1 (Enterococcus faecalis, 26975_2#160) loaded with 60 markers.\n",
      "2024-02-25 13:21:19,318 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962010.1 (Enterococcus faecalis, 26975_2#152) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,335 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961996.1 (Enterococcus faecalis, 26975_1#117) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,352 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961994.1 (Enterococcus faecalis, 26975_1#118) loaded with 70 markers.\n",
      "2024-02-25 13:21:19,366 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961986.1 (Enterococcus faecalis, 26975_1#129) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,382 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962008.1 (Enterococcus faecalis, 26975_1#88) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,400 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961981.1 (Enterococcus faecalis, 26975_1#105) loaded with 79 markers.\n",
      "2024-02-25 13:21:19,408 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961957.1 (Enterococcus faecalis, 26975_1#22) loaded with 64 markers.\n",
      "2024-02-25 13:21:19,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961953.1 (Enterococcus faecalis, 26975_1#18) loaded with 68 markers.\n",
      "2024-02-25 13:21:19,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961923.1 (Enterococcus faecalis, 26975_1#52) loaded with 69 markers.\n",
      "2024-02-25 13:21:19,441 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961959.1 (Enterococcus faecalis, 26975_1#5) loaded with 62 markers.\n",
      "2024-02-25 13:21:19,456 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124778.1 (Enterococcus faecalis, HL1) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,471 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP127177.1 (Enterococcus faecalis, 90_2023) loaded with 68 markers.\n",
      "2024-02-25 13:21:19,480 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109611.1 (Enterococcus faecalis, AFL-109F) loaded with 60 markers.\n",
      "2024-02-25 13:21:19,487 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124914.1 (Enterococcus faecalis, EfsC8) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124891.1 (Enterococcus faecalis, EfsC130) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118048.1 (Enterococcus faecalis, VSI48) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP117511.1 (Enterococcus faecalis, CQNBZ21-135) loaded with 62 markers.\n",
      "2024-02-25 13:21:19,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP117503.1 (Enterococcus faecalis, CQJXZ21-076) loaded with 70 markers.\n",
      "2024-02-25 13:21:19,548 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP121233.1 (Enterococcus faecalis, SVJ-EF01) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,556 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116566.1 (Enterococcus faecalis, K191-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,564 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116564.1 (Enterococcus faecalis, K188-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,572 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116561.1 (Enterococcus faecalis, K72-1) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,587 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116569.1 (Enterococcus faecalis, K198-1) loaded with 69 markers.\n",
      "2024-02-25 13:21:19,606 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026721.1 (Enterococcus faecalis, SVR2330) loaded with 82 markers.\n",
      "2024-02-25 13:21:19,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP115992.1 (Enterococcus faecalis, ESC1) loaded with 61 markers.\n",
      "2024-02-25 13:21:19,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110036.1 (Enterococcus faecalis, BE52) loaded with 61 markers.\n",
      "2024-02-25 13:21:19,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110020.1 (Enterococcus faecalis, BE69) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,654 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110019.1 (Enterococcus faecalis, BE70) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP098418.1 (Enterococcus faecalis, CQ025) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,754 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP098025.1 (Enterococcus faecalis, QZ076) loaded with 73 markers.\n",
      "2024-02-25 13:21:19,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091237.1 (Enterococcus faecalis, 1521) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,786 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091240.1 (Enterococcus faecalis, 249) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,801 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091227.1 (Enterococcus faecalis, 661) loaded with 69 markers.\n",
      "2024-02-25 13:21:19,817 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097059.1 (Enterococcus faecalis, AT04) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097030.1 (Enterococcus faecalis, AT43a) loaded with 71 markers.\n",
      "2024-02-25 13:21:19,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097008.1 (Enterococcus faecalis, AT50) loaded with 70 markers.\n",
      "2024-02-25 13:21:19,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097038.1 (Enterococcus faecalis, AT39) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097004.1 (Enterococcus faecalis, LS05-2) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110289.1 (Enterococcus faecalis, AKSZ-208) loaded with 61 markers.\n",
      "2024-02-25 13:21:19,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093957.1 (Enterococcus faecalis, NY13321) loaded with 74 markers.\n",
      "2024-02-25 13:21:19,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092907.1 (Enterococcus faecalis, LJX 0909) loaded with 67 markers.\n",
      "2024-02-25 13:21:19,928 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091886.1 (Enterococcus faecalis, 143-1) loaded with 72 markers.\n",
      "2024-02-25 13:21:19,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063980.1 (Enterococcus faecalis, AR_0780) loaded with 70 markers.\n",
      "2024-02-25 13:21:19,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085289.1 (Enterococcus faecalis, EFS17) loaded with 76 markers.\n",
      "2024-02-25 13:21:19,974 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045598.1 (Enterococcus faecalis, TK-P4B) loaded with 68 markers.\n",
      "2024-02-25 13:21:19,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050491.1 (Enterococcus faecalis, EGM183) loaded with 59 markers.\n",
      "2024-02-25 13:21:19,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060804.1 (Enterococcus faecalis, SF28073) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,006 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030045.1 (Enterococcus faecalis, C54) loaded with 59 markers.\n",
      "2024-02-25 13:21:20,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039434.1 (Enterococcus faecalis, SGAir0397) loaded with 59 markers.\n",
      "2024-02-25 13:21:20,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028835.1 (Enterococcus faecalis, FC) loaded with 76 markers.\n",
      "2024-02-25 13:21:20,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028720.1 (Enterococcus faecalis, CVM N48037F) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,062 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018543.1 (Enterococcus faecalis, KUB3007) loaded with 68 markers.\n",
      "2024-02-25 13:21:20,077 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018538.1 (Enterococcus faecalis, KUB3006) loaded with 68 markers.\n",
      "2024-02-25 13:21:20,094 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033787.1 (Enterococcus faecalis, FDAARGOS_528) loaded with 79 markers.\n",
      "2024-02-25 13:21:20,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP031027.1 (Enterococcus faecalis, TY1) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029612.1 (Enterococcus faecalis, H25) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,144 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019512.1 (Enterococcus faecalis, CLB21560) loaded with 79 markers.\n",
      "2024-02-25 13:21:20,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015410.2 (Enterococcus faecalis, KB1) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,169 [DEBUG - chronostrain.database.parser.json] - Strain CP036249.1 (Enterococcus faecalis, R712) loaded with 67 markers.\n",
      "2024-02-25 13:21:20,178 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP036249.1 (Enterococcus faecalis, R712) loaded with 67 markers.\n",
      "2024-02-25 13:21:20,186 [DEBUG - chronostrain.database.parser.json] - Strain LR962826.1 (Enterococcus faecalis, 28157_4#56) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,194 [DEBUG - chronostrain.database.parser.json] - Strain LR962797.1 (Enterococcus faecalis, 28157_4#153) loaded with 57 markers.\n",
      "2024-02-25 13:21:20,211 [DEBUG - chronostrain.database.parser.json] - Strain LR962807.1 (Enterococcus faecalis, 28157_4#36) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,227 [DEBUG - chronostrain.database.parser.json] - Strain LR962738.1 (Enterococcus faecalis, 26975_2#45) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,243 [DEBUG - chronostrain.database.parser.json] - Strain LR962723.1 (Enterococcus faecalis, 26975_2#36) loaded with 66 markers.\n",
      "2024-02-25 13:21:20,251 [DEBUG - chronostrain.database.parser.json] - Strain LR962703.1 (Enterococcus faecalis, 26975_2#39) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,267 [DEBUG - chronostrain.database.parser.json] - Strain LR962687.1 (Enterococcus faecalis, 26975_2#25) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,276 [DEBUG - chronostrain.database.parser.json] - Strain LR962681.1 (Enterococcus faecalis, 26975_2#17) loaded with 65 markers.\n",
      "2024-02-25 13:21:20,292 [DEBUG - chronostrain.database.parser.json] - Strain LR962666.1 (Enterococcus faecalis, 26975_2#24) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,300 [DEBUG - chronostrain.database.parser.json] - Strain LR962617.1 (Enterococcus faecalis, 27688_1#59) loaded with 63 markers.\n",
      "2024-02-25 13:21:20,309 [DEBUG - chronostrain.database.parser.json] - Strain LR962550.1 (Enterococcus faecalis, 28157_4#85) loaded with 62 markers.\n",
      "2024-02-25 13:21:20,317 [DEBUG - chronostrain.database.parser.json] - Strain LR962525.1 (Enterococcus faecalis, 28099_2#118) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,324 [DEBUG - chronostrain.database.parser.json] - Strain LR962489.1 (Enterococcus faecalis, 28157_4#18) loaded with 59 markers.\n",
      "2024-02-25 13:21:20,332 [DEBUG - chronostrain.database.parser.json] - Strain LR962468.1 (Enterococcus faecalis, 28099_2#370) loaded with 60 markers.\n",
      "2024-02-25 13:21:20,340 [DEBUG - chronostrain.database.parser.json] - Strain LR962480.1 (Enterococcus faecalis, 28157_4#348) loaded with 59 markers.\n",
      "2024-02-25 13:21:20,357 [DEBUG - chronostrain.database.parser.json] - Strain LR962476.1 (Enterococcus faecalis, 28157_4#356) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,365 [DEBUG - chronostrain.database.parser.json] - Strain LR962428.1 (Enterococcus faecalis, 28157_4#173) loaded with 60 markers.\n",
      "2024-02-25 13:21:20,374 [DEBUG - chronostrain.database.parser.json] - Strain LR962447.1 (Enterococcus faecalis, 28157_4#137) loaded with 63 markers.\n",
      "2024-02-25 13:21:20,389 [DEBUG - chronostrain.database.parser.json] - Strain LR962395.1 (Enterococcus faecalis, 28157_4#315) loaded with 66 markers.\n",
      "2024-02-25 13:21:20,406 [DEBUG - chronostrain.database.parser.json] - Strain LR962284.1 (Enterococcus faecalis, 27725_1#343) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,414 [DEBUG - chronostrain.database.parser.json] - Strain LR962280.1 (Enterococcus faecalis, 27688_1#106) loaded with 64 markers.\n",
      "2024-02-25 13:21:20,422 [DEBUG - chronostrain.database.parser.json] - Strain LR962251.1 (Enterococcus faecalis, 27688_1#114) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,430 [DEBUG - chronostrain.database.parser.json] - Strain LR962254.1 (Enterococcus faecalis, 27688_1#231) loaded with 60 markers.\n",
      "2024-02-25 13:21:20,447 [DEBUG - chronostrain.database.parser.json] - Strain LR962258.1 (Enterococcus faecalis, 27688_1#194) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,463 [DEBUG - chronostrain.database.parser.json] - Strain LR962231.1 (Enterococcus faecalis, 27688_1#153) loaded with 76 markers.\n",
      "2024-02-25 13:21:20,481 [DEBUG - chronostrain.database.parser.json] - Strain LR962178.1 (Enterococcus faecalis, 27688_1#225) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,501 [DEBUG - chronostrain.database.parser.json] - Strain LR962138.1 (Enterococcus faecalis, 26975_2#114) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,509 [DEBUG - chronostrain.database.parser.json] - Strain LR962144.1 (Enterococcus faecalis, 26975_2#106) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,527 [DEBUG - chronostrain.database.parser.json] - Strain LR962158.1 (Enterococcus faecalis, 26975_2#115) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,544 [DEBUG - chronostrain.database.parser.json] - Strain LR962146.1 (Enterococcus faecalis, 26975_2#102) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,553 [DEBUG - chronostrain.database.parser.json] - Strain LR962125.1 (Enterococcus faecalis, 26975_1#368) loaded with 70 markers.\n",
      "2024-02-25 13:21:20,569 [DEBUG - chronostrain.database.parser.json] - Strain LR962091.1 (Enterococcus faecalis, 26975_1#263) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,586 [DEBUG - chronostrain.database.parser.json] - Strain LR962086.1 (Enterococcus faecalis, 26975_1#279) loaded with 79 markers.\n",
      "2024-02-25 13:21:20,599 [DEBUG - chronostrain.database.parser.json] - Strain LR962098.1 (Enterococcus faecalis, 26975_1#276) loaded with 67 markers.\n",
      "2024-02-25 13:21:20,608 [DEBUG - chronostrain.database.parser.json] - Strain LR962078.1 (Enterococcus faecalis, 26975_1#188) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,624 [DEBUG - chronostrain.database.parser.json] - Strain LR962046.1 (Enterococcus faecalis, 26975_2#180) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,633 [DEBUG - chronostrain.database.parser.json] - Strain LR962053.1 (Enterococcus faecalis, 26975_2#184) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,641 [DEBUG - chronostrain.database.parser.json] - Strain LR962029.1 (Enterococcus faecalis, 26975_1#168) loaded with 68 markers.\n",
      "2024-02-25 13:21:20,650 [DEBUG - chronostrain.database.parser.json] - Strain LR962002.1 (Enterococcus faecalis, 26975_1#132) loaded with 65 markers.\n",
      "2024-02-25 13:21:20,658 [DEBUG - chronostrain.database.parser.json] - Strain LR961951.1 (Enterococcus faecalis, 26975_1#12) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,675 [DEBUG - chronostrain.database.parser.json] - Strain LR961942.1 (Enterococcus faecalis, 26975_1#13) loaded with 70 markers.\n",
      "2024-02-25 13:21:20,684 [DEBUG - chronostrain.database.parser.json] - Strain LR961948.1 (Enterococcus faecalis, 26975_2#150) loaded with 64 markers.\n",
      "2024-02-25 13:21:20,692 [DEBUG - chronostrain.database.parser.json] - Strain CP124956.1 (Enterococcus faecalis, EfsC13) loaded with 64 markers.\n",
      "2024-02-25 13:21:20,701 [DEBUG - chronostrain.database.parser.json] - Strain AP026718.1 (Enterococcus faecalis, SVR2281) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,710 [DEBUG - chronostrain.database.parser.json] - Strain CP113832.1 (Enterococcus faecalis, M61) loaded with 63 markers.\n",
      "2024-02-25 13:21:20,726 [DEBUG - chronostrain.database.parser.json] - Strain CP085841.1 (Enterococcus faecalis, FA2-2) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,742 [DEBUG - chronostrain.database.parser.json] - Strain CP110068.1 (Enterococcus faecalis, BE13) loaded with 70 markers.\n",
      "2024-02-25 13:21:20,750 [DEBUG - chronostrain.database.parser.json] - Strain CP110059.1 (Enterococcus faecalis, BE16) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,758 [DEBUG - chronostrain.database.parser.json] - Strain CP110054.1 (Enterococcus faecalis, BE17) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,765 [DEBUG - chronostrain.database.parser.json] - Strain CP110040.1 (Enterococcus faecalis, BE45) loaded with 56 markers.\n",
      "2024-02-25 13:21:20,773 [DEBUG - chronostrain.database.parser.json] - Strain CP110039.1 (Enterococcus faecalis, BE47) loaded with 56 markers.\n",
      "2024-02-25 13:21:20,789 [DEBUG - chronostrain.database.parser.json] - Strain CP098743.1 (Enterococcus faecalis, M9) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,798 [DEBUG - chronostrain.database.parser.json] - Strain CP097031.1 (Enterococcus faecalis, AT41) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,806 [DEBUG - chronostrain.database.parser.json] - Strain CP091904.1 (Enterococcus faecalis, 43-2) loaded with 65 markers.\n",
      "2024-02-25 13:21:20,823 [DEBUG - chronostrain.database.parser.json] - Strain CP085291.1 (Enterococcus faecalis, EFS36) loaded with 78 markers.\n",
      "2024-02-25 13:21:20,839 [DEBUG - chronostrain.database.parser.json] - Strain CP081505.1 (Enterococcus faecalis, E006xJH2-2-TC1) loaded with 74 markers.\n",
      "2024-02-25 13:21:20,854 [DEBUG - chronostrain.database.parser.json] - Strain CP060796.1 (Enterococcus faecalis, EF-2001) loaded with 68 markers.\n",
      "2024-02-25 13:21:20,871 [DEBUG - chronostrain.database.parser.json] - Strain CP068249.1 (Enterococcus faecalis, T90-1) loaded with 76 markers.\n",
      "2024-02-25 13:21:20,879 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962826.1 (Enterococcus faecalis, 28157_4#56) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,887 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962797.1 (Enterococcus faecalis, 28157_4#153) loaded with 57 markers.\n",
      "2024-02-25 13:21:20,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962807.1 (Enterococcus faecalis, 28157_4#36) loaded with 75 markers.\n",
      "2024-02-25 13:21:20,922 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962738.1 (Enterococcus faecalis, 26975_2#45) loaded with 71 markers.\n",
      "2024-02-25 13:21:20,938 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962723.1 (Enterococcus faecalis, 26975_2#36) loaded with 66 markers.\n",
      "2024-02-25 13:21:20,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962703.1 (Enterococcus faecalis, 26975_2#39) loaded with 61 markers.\n",
      "2024-02-25 13:21:20,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962687.1 (Enterococcus faecalis, 26975_2#25) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962681.1 (Enterococcus faecalis, 26975_2#17) loaded with 65 markers.\n",
      "2024-02-25 13:21:20,985 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962666.1 (Enterococcus faecalis, 26975_2#24) loaded with 69 markers.\n",
      "2024-02-25 13:21:20,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962617.1 (Enterococcus faecalis, 27688_1#59) loaded with 63 markers.\n",
      "2024-02-25 13:21:21,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962550.1 (Enterococcus faecalis, 28157_4#85) loaded with 62 markers.\n",
      "2024-02-25 13:21:21,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962525.1 (Enterococcus faecalis, 28099_2#118) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962489.1 (Enterococcus faecalis, 28157_4#18) loaded with 59 markers.\n",
      "2024-02-25 13:21:21,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962468.1 (Enterococcus faecalis, 28099_2#370) loaded with 60 markers.\n",
      "2024-02-25 13:21:21,034 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962480.1 (Enterococcus faecalis, 28157_4#348) loaded with 59 markers.\n",
      "2024-02-25 13:21:21,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962476.1 (Enterococcus faecalis, 28157_4#356) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,059 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962428.1 (Enterococcus faecalis, 28157_4#173) loaded with 60 markers.\n",
      "2024-02-25 13:21:21,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962447.1 (Enterococcus faecalis, 28157_4#137) loaded with 63 markers.\n",
      "2024-02-25 13:21:21,082 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962395.1 (Enterococcus faecalis, 28157_4#315) loaded with 66 markers.\n",
      "2024-02-25 13:21:21,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962284.1 (Enterococcus faecalis, 27725_1#343) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962280.1 (Enterococcus faecalis, 27688_1#106) loaded with 64 markers.\n",
      "2024-02-25 13:21:21,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962251.1 (Enterococcus faecalis, 27688_1#114) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962254.1 (Enterococcus faecalis, 27688_1#231) loaded with 60 markers.\n",
      "2024-02-25 13:21:21,138 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962258.1 (Enterococcus faecalis, 27688_1#194) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,155 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962231.1 (Enterococcus faecalis, 27688_1#153) loaded with 76 markers.\n",
      "2024-02-25 13:21:21,172 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962178.1 (Enterococcus faecalis, 27688_1#225) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,188 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962138.1 (Enterococcus faecalis, 26975_2#114) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,196 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962144.1 (Enterococcus faecalis, 26975_2#106) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,212 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962158.1 (Enterococcus faecalis, 26975_2#115) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962146.1 (Enterococcus faecalis, 26975_2#102) loaded with 75 markers.\n",
      "2024-02-25 13:21:21,237 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962125.1 (Enterococcus faecalis, 26975_1#368) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,254 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962091.1 (Enterococcus faecalis, 26975_1#263) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,271 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962086.1 (Enterococcus faecalis, 26975_1#279) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962098.1 (Enterococcus faecalis, 26975_1#276) loaded with 67 markers.\n",
      "2024-02-25 13:21:21,290 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962078.1 (Enterococcus faecalis, 26975_1#188) loaded with 75 markers.\n",
      "2024-02-25 13:21:21,307 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962046.1 (Enterococcus faecalis, 26975_2#180) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,315 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962053.1 (Enterococcus faecalis, 26975_2#184) loaded with 69 markers.\n",
      "2024-02-25 13:21:21,324 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962029.1 (Enterococcus faecalis, 26975_1#168) loaded with 68 markers.\n",
      "2024-02-25 13:21:21,332 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962002.1 (Enterococcus faecalis, 26975_1#132) loaded with 65 markers.\n",
      "2024-02-25 13:21:21,340 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961951.1 (Enterococcus faecalis, 26975_1#12) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,356 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961942.1 (Enterococcus faecalis, 26975_1#13) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961948.1 (Enterococcus faecalis, 26975_2#150) loaded with 64 markers.\n",
      "2024-02-25 13:21:21,373 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124956.1 (Enterococcus faecalis, EfsC13) loaded with 64 markers.\n",
      "2024-02-25 13:21:21,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP113832.1 (Enterococcus faecalis, M61) loaded with 63 markers.\n",
      "2024-02-25 13:21:21,397 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085841.1 (Enterococcus faecalis, FA2-2) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,414 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110068.1 (Enterococcus faecalis, BE13) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110059.1 (Enterococcus faecalis, BE16) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110054.1 (Enterococcus faecalis, BE17) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110040.1 (Enterococcus faecalis, BE45) loaded with 56 markers.\n",
      "2024-02-25 13:21:21,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110039.1 (Enterococcus faecalis, BE47) loaded with 56 markers.\n",
      "2024-02-25 13:21:21,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP098743.1 (Enterococcus faecalis, M9) loaded with 69 markers.\n",
      "2024-02-25 13:21:21,468 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097031.1 (Enterococcus faecalis, AT41) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,476 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091904.1 (Enterococcus faecalis, 43-2) loaded with 65 markers.\n",
      "2024-02-25 13:21:21,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085291.1 (Enterococcus faecalis, EFS36) loaded with 78 markers.\n",
      "2024-02-25 13:21:21,512 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081505.1 (Enterococcus faecalis, E006xJH2-2-TC1) loaded with 74 markers.\n",
      "2024-02-25 13:21:21,527 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060796.1 (Enterococcus faecalis, EF-2001) loaded with 68 markers.\n",
      "2024-02-25 13:21:21,544 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068249.1 (Enterococcus faecalis, T90-1) loaded with 76 markers.\n",
      "2024-02-25 13:21:21,560 [DEBUG - chronostrain.database.parser.json] - Strain OD940437.1 (Enterococcus faecalis, BX8117) loaded with 80 markers.\n",
      "2024-02-25 13:21:21,576 [DEBUG - chronostrain.database.parser.json] - Strain LR962852.1 (Enterococcus faecalis, 28157_4#95) loaded with 72 markers.\n",
      "2024-02-25 13:21:21,593 [DEBUG - chronostrain.database.parser.json] - Strain LR962825.1 (Enterococcus faecalis, 28157_4#88) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,601 [DEBUG - chronostrain.database.parser.json] - Strain LR962820.1 (Enterococcus faecalis, 28157_4#299) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,618 [DEBUG - chronostrain.database.parser.json] - Strain LR962772.1 (Enterococcus faecalis, 27688_1#300) loaded with 80 markers.\n",
      "2024-02-25 13:21:21,633 [DEBUG - chronostrain.database.parser.json] - Strain LR962805.1 (Enterococcus faecalis, 28157_4#280) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,650 [DEBUG - chronostrain.database.parser.json] - Strain LR962771.1 (Enterococcus faecalis, 28157_4#255) loaded with 81 markers.\n",
      "2024-02-25 13:21:21,665 [DEBUG - chronostrain.database.parser.json] - Strain LR962691.1 (Enterococcus faecalis, 26975_2#29) loaded with 66 markers.\n",
      "2024-02-25 13:21:21,673 [DEBUG - chronostrain.database.parser.json] - Strain LR962650.1 (Enterococcus faecalis, 26975_2#21) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,689 [DEBUG - chronostrain.database.parser.json] - Strain LR962680.1 (Enterococcus faecalis, 26975_1#85) loaded with 80 markers.\n",
      "2024-02-25 13:21:21,697 [DEBUG - chronostrain.database.parser.json] - Strain LR962676.1 (Enterococcus faecalis, 26975_2#23) loaded with 61 markers.\n",
      "2024-02-25 13:21:21,712 [DEBUG - chronostrain.database.parser.json] - Strain LR962626.1 (Enterococcus faecalis, 27688_1#132) loaded with 73 markers.\n",
      "2024-02-25 13:21:21,728 [DEBUG - chronostrain.database.parser.json] - Strain LR962541.1 (Enterococcus faecalis, 28157_4#87) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,735 [DEBUG - chronostrain.database.parser.json] - Strain LR962501.1 (Enterococcus faecalis, 28099_2#361) loaded with 58 markers.\n",
      "2024-02-25 13:21:21,751 [DEBUG - chronostrain.database.parser.json] - Strain LR962519.1 (Enterococcus faecalis, 28157_4#48) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,766 [DEBUG - chronostrain.database.parser.json] - Strain LR962426.1 (Enterococcus faecalis, 28157_4#139) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,783 [DEBUG - chronostrain.database.parser.json] - Strain LR962412.1 (Enterococcus faecalis, 28157_4#140) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,798 [DEBUG - chronostrain.database.parser.json] - Strain LR962427.1 (Enterococcus faecalis, 28157_4#278) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,814 [DEBUG - chronostrain.database.parser.json] - Strain LR962288.1 (Enterococcus faecalis, 28157_4#3) loaded with 72 markers.\n",
      "2024-02-25 13:21:21,823 [DEBUG - chronostrain.database.parser.json] - Strain LR962316.1 (Enterococcus faecalis, 27688_1#93) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,839 [DEBUG - chronostrain.database.parser.json] - Strain LR962275.1 (Enterococcus faecalis, 27725_1#295) loaded with 69 markers.\n",
      "2024-02-25 13:21:21,856 [DEBUG - chronostrain.database.parser.json] - Strain LR962264.1 (Enterococcus faecalis, 27688_1#57) loaded with 82 markers.\n",
      "2024-02-25 13:21:21,864 [DEBUG - chronostrain.database.parser.json] - Strain LR962211.1 (Enterococcus faecalis, 27725_1#198) loaded with 60 markers.\n",
      "2024-02-25 13:21:21,872 [DEBUG - chronostrain.database.parser.json] - Strain LR962149.1 (Enterococcus faecalis, 26975_2#130) loaded with 59 markers.\n",
      "2024-02-25 13:21:21,888 [DEBUG - chronostrain.database.parser.json] - Strain LR962117.1 (Enterococcus faecalis, 26975_1#253) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,905 [DEBUG - chronostrain.database.parser.json] - Strain LR962085.1 (Enterococcus faecalis, 26975_2#225) loaded with 83 markers.\n",
      "2024-02-25 13:21:21,922 [DEBUG - chronostrain.database.parser.json] - Strain LR962074.1 (Enterococcus faecalis, 26975_2#221) loaded with 81 markers.\n",
      "2024-02-25 13:21:21,938 [DEBUG - chronostrain.database.parser.json] - Strain LR962016.1 (Enterococcus faecalis, 26975_1#74) loaded with 79 markers.\n",
      "2024-02-25 13:21:21,954 [DEBUG - chronostrain.database.parser.json] - Strain LR961980.1 (Enterococcus faecalis, 26975_1#32) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,969 [DEBUG - chronostrain.database.parser.json] - Strain LR961924.1 (Enterococcus faecalis, 26975_1#11) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,978 [DEBUG - chronostrain.database.parser.json] - Strain CP124960.1 (Enterococcus faecalis, EfsC1) loaded with 71 markers.\n",
      "2024-02-25 13:21:21,987 [DEBUG - chronostrain.database.parser.json] - Strain CP124950.1 (Enterococcus faecalis, EfsC29) loaded with 70 markers.\n",
      "2024-02-25 13:21:21,996 [DEBUG - chronostrain.database.parser.json] - Strain CP124939.1 (Enterococcus faecalis, EfsC129) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,005 [DEBUG - chronostrain.database.parser.json] - Strain CP124935.1 (Enterococcus faecalis, EfsPF23) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,014 [DEBUG - chronostrain.database.parser.json] - Strain CP124925.1 (Enterococcus faecalis, EfsPF36) loaded with 72 markers.\n",
      "2024-02-25 13:21:22,024 [DEBUG - chronostrain.database.parser.json] - Strain CP124913.1 (Enterococcus faecalis, EfsC20) loaded with 71 markers.\n",
      "2024-02-25 13:21:22,040 [DEBUG - chronostrain.database.parser.json] - Strain CP124905.1 (Enterococcus faecalis, EfsC85) loaded with 78 markers.\n",
      "2024-02-25 13:21:22,057 [DEBUG - chronostrain.database.parser.json] - Strain CP124898.1 (Enterococcus faecalis, EfsC109-2) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,073 [DEBUG - chronostrain.database.parser.json] - Strain CP124893.1 (Enterococcus faecalis, EfsC123) loaded with 78 markers.\n",
      "2024-02-25 13:21:22,081 [DEBUG - chronostrain.database.parser.json] - Strain CP124890.1 (Enterococcus faecalis, EfsPF2) loaded with 58 markers.\n",
      "2024-02-25 13:21:22,097 [DEBUG - chronostrain.database.parser.json] - Strain CP124887.1 (Enterococcus faecalis, EfsPNK7) loaded with 71 markers.\n",
      "2024-02-25 13:21:22,114 [DEBUG - chronostrain.database.parser.json] - Strain CP104856.1 (Enterococcus faecalis, NY13412) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,128 [DEBUG - chronostrain.database.parser.json] - Strain CP116553.1 (Enterococcus faecalis, K80-2b) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,143 [DEBUG - chronostrain.database.parser.json] - Strain CP116557.1 (Enterococcus faecalis, K194-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,158 [DEBUG - chronostrain.database.parser.json] - Strain CP116559.1 (Enterococcus faecalis, K137-1a) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,174 [DEBUG - chronostrain.database.parser.json] - Strain CP116555.1 (Enterococcus faecalis, K70-12a) loaded with 67 markers.\n",
      "2024-02-25 13:21:22,190 [DEBUG - chronostrain.database.parser.json] - Strain CP110072.1 (Enterococcus faecalis, BE7) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,206 [DEBUG - chronostrain.database.parser.json] - Strain CP110071.1 (Enterococcus faecalis, BE8) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,214 [DEBUG - chronostrain.database.parser.json] - Strain CP110053.1 (Enterococcus faecalis, BE25) loaded with 57 markers.\n",
      "2024-02-25 13:21:22,223 [DEBUG - chronostrain.database.parser.json] - Strain CP110035.1 (Enterococcus faecalis, BE54) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,232 [DEBUG - chronostrain.database.parser.json] - Strain CP110022.1 (Enterococcus faecalis, BE67) loaded with 69 markers.\n",
      "2024-02-25 13:21:22,252 [DEBUG - chronostrain.database.parser.json] - Strain CP080759.1 (Enterococcus faecalis, M817) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,268 [DEBUG - chronostrain.database.parser.json] - Strain CP091235.1 (Enterococcus faecalis, 705) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,283 [DEBUG - chronostrain.database.parser.json] - Strain CP088198.1 (Enterococcus faecalis, S11-6) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,298 [DEBUG - chronostrain.database.parser.json] - Strain CP097042.1 (Enterococcus faecalis, AT34) loaded with 67 markers.\n",
      "2024-02-25 13:21:22,314 [DEBUG - chronostrain.database.parser.json] - Strain CP097034.1 (Enterococcus faecalis, AT40b) loaded with 73 markers.\n",
      "2024-02-25 13:21:22,329 [DEBUG - chronostrain.database.parser.json] - Strain CP097006.1 (Enterococcus faecalis, LS05-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,344 [DEBUG - chronostrain.database.parser.json] - Strain CP046247.1 (Enterococcus faecalis, RF_1_1_25_1) loaded with 71 markers.\n",
      "2024-02-25 13:21:22,361 [DEBUG - chronostrain.database.parser.json] - Strain CP091897.1 (Enterococcus faecalis, 119-1) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,377 [DEBUG - chronostrain.database.parser.json] - Strain CP091895.1 (Enterococcus faecalis, 119-2) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,386 [DEBUG - chronostrain.database.parser.json] - Strain CP091884.1 (Enterococcus faecalis, 152) loaded with 72 markers.\n",
      "2024-02-25 13:21:22,403 [DEBUG - chronostrain.database.parser.json] - Strain CP086560.1 (Enterococcus faecalis, E509) loaded with 81 markers.\n",
      "2024-02-25 13:21:22,419 [DEBUG - chronostrain.database.parser.json] - Strain CP078015.1 (Enterococcus faecalis, SY-1) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,428 [DEBUG - chronostrain.database.parser.json] - Strain CP071176.1 (Enterococcus faecalis, L18) loaded with 63 markers.\n",
      "2024-02-25 13:21:22,436 [DEBUG - chronostrain.database.parser.json] - Strain CP071172.1 (Enterococcus faecalis, L13) loaded with 63 markers.\n",
      "2024-02-25 13:21:22,444 [DEBUG - chronostrain.database.parser.json] - Strain CP069185.1 (Enterococcus faecalis, L11) loaded with 63 markers.\n",
      "2024-02-25 13:21:22,461 [DEBUG - chronostrain.database.parser.json] - Strain CP051005.1 (Enterococcus faecalis, TH4125) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,479 [DEBUG - chronostrain.database.parser.json] - Strain CP045045.1 (Enterococcus faecalis, JY32) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,487 [DEBUG - chronostrain.database.parser.json] - Strain CP042216.2 (Enterococcus faecalis, L8) loaded with 63 markers.\n",
      "2024-02-25 13:21:22,503 [DEBUG - chronostrain.database.parser.json] - Strain CP040898.1 (Enterococcus faecalis, HA-1) loaded with 78 markers.\n",
      "2024-02-25 13:21:22,512 [DEBUG - chronostrain.database.parser.json] - Strain CP028724.1 (Enterococcus faecalis, CVM N60443F) loaded with 60 markers.\n",
      "2024-02-25 13:21:22,531 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OD940437.1 (Enterococcus faecalis, BX8117) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,549 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962852.1 (Enterococcus faecalis, 28157_4#95) loaded with 72 markers.\n",
      "2024-02-25 13:21:22,571 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962825.1 (Enterococcus faecalis, 28157_4#88) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962820.1 (Enterococcus faecalis, 28157_4#299) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962772.1 (Enterococcus faecalis, 27688_1#300) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,617 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962805.1 (Enterococcus faecalis, 28157_4#280) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,636 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962771.1 (Enterococcus faecalis, 28157_4#255) loaded with 81 markers.\n",
      "2024-02-25 13:21:22,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962691.1 (Enterococcus faecalis, 26975_2#29) loaded with 66 markers.\n",
      "2024-02-25 13:21:22,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962650.1 (Enterococcus faecalis, 26975_2#21) loaded with 61 markers.\n",
      "2024-02-25 13:21:22,681 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962680.1 (Enterococcus faecalis, 26975_1#85) loaded with 80 markers.\n",
      "2024-02-25 13:21:22,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962676.1 (Enterococcus faecalis, 26975_2#23) loaded with 61 markers.\n",
      "2024-02-25 13:21:22,708 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962626.1 (Enterococcus faecalis, 27688_1#132) loaded with 73 markers.\n",
      "2024-02-25 13:21:22,725 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962541.1 (Enterococcus faecalis, 28157_4#87) loaded with 71 markers.\n",
      "2024-02-25 13:21:22,734 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962501.1 (Enterococcus faecalis, 28099_2#361) loaded with 58 markers.\n",
      "2024-02-25 13:21:22,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962519.1 (Enterococcus faecalis, 28157_4#48) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962426.1 (Enterococcus faecalis, 28157_4#139) loaded with 70 markers.\n",
      "2024-02-25 13:21:22,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962412.1 (Enterococcus faecalis, 28157_4#140) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962427.1 (Enterococcus faecalis, 28157_4#278) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,825 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962288.1 (Enterococcus faecalis, 28157_4#3) loaded with 72 markers.\n",
      "2024-02-25 13:21:22,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962316.1 (Enterococcus faecalis, 27688_1#93) loaded with 71 markers.\n",
      "2024-02-25 13:21:22,853 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962275.1 (Enterococcus faecalis, 27725_1#295) loaded with 69 markers.\n",
      "2024-02-25 13:21:22,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962264.1 (Enterococcus faecalis, 27688_1#57) loaded with 82 markers.\n",
      "2024-02-25 13:21:22,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962211.1 (Enterococcus faecalis, 27725_1#198) loaded with 60 markers.\n",
      "2024-02-25 13:21:22,891 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962149.1 (Enterococcus faecalis, 26975_2#130) loaded with 59 markers.\n",
      "2024-02-25 13:21:22,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962117.1 (Enterococcus faecalis, 26975_1#253) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,929 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962085.1 (Enterococcus faecalis, 26975_2#225) loaded with 83 markers.\n",
      "2024-02-25 13:21:22,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962074.1 (Enterococcus faecalis, 26975_2#221) loaded with 81 markers.\n",
      "2024-02-25 13:21:22,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962016.1 (Enterococcus faecalis, 26975_1#74) loaded with 79 markers.\n",
      "2024-02-25 13:21:22,983 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961980.1 (Enterococcus faecalis, 26975_1#32) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR961924.1 (Enterococcus faecalis, 26975_1#11) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,011 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124960.1 (Enterococcus faecalis, EfsC1) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,021 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124950.1 (Enterococcus faecalis, EfsC29) loaded with 70 markers.\n",
      "2024-02-25 13:21:23,032 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124939.1 (Enterococcus faecalis, EfsC129) loaded with 70 markers.\n",
      "2024-02-25 13:21:23,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124935.1 (Enterococcus faecalis, EfsPF23) loaded with 70 markers.\n",
      "2024-02-25 13:21:23,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124925.1 (Enterococcus faecalis, EfsPF36) loaded with 72 markers.\n",
      "2024-02-25 13:21:23,063 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124913.1 (Enterococcus faecalis, EfsC20) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,082 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124905.1 (Enterococcus faecalis, EfsC85) loaded with 78 markers.\n",
      "2024-02-25 13:21:23,102 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124898.1 (Enterococcus faecalis, EfsC109-2) loaded with 80 markers.\n",
      "2024-02-25 13:21:23,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124893.1 (Enterococcus faecalis, EfsC123) loaded with 78 markers.\n",
      "2024-02-25 13:21:23,129 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124890.1 (Enterococcus faecalis, EfsPF2) loaded with 58 markers.\n",
      "2024-02-25 13:21:23,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124887.1 (Enterococcus faecalis, EfsPNK7) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,166 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP104856.1 (Enterococcus faecalis, NY13412) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,182 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116553.1 (Enterococcus faecalis, K80-2b) loaded with 66 markers.\n",
      "2024-02-25 13:21:23,199 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116557.1 (Enterococcus faecalis, K194-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:23,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116559.1 (Enterococcus faecalis, K137-1a) loaded with 66 markers.\n",
      "2024-02-25 13:21:23,232 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116555.1 (Enterococcus faecalis, K70-12a) loaded with 67 markers.\n",
      "2024-02-25 13:21:23,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110072.1 (Enterococcus faecalis, BE7) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110071.1 (Enterococcus faecalis, BE8) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110053.1 (Enterococcus faecalis, BE25) loaded with 57 markers.\n",
      "2024-02-25 13:21:23,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080759.1 (Enterococcus faecalis, M817) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091235.1 (Enterococcus faecalis, 705) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,331 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088198.1 (Enterococcus faecalis, S11-6) loaded with 66 markers.\n",
      "2024-02-25 13:21:23,347 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097042.1 (Enterococcus faecalis, AT34) loaded with 67 markers.\n",
      "2024-02-25 13:21:23,365 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097034.1 (Enterococcus faecalis, AT40b) loaded with 73 markers.\n",
      "2024-02-25 13:21:23,381 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097006.1 (Enterococcus faecalis, LS05-1) loaded with 66 markers.\n",
      "2024-02-25 13:21:23,398 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046247.1 (Enterococcus faecalis, RF_1_1_25_1) loaded with 71 markers.\n",
      "2024-02-25 13:21:23,417 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091897.1 (Enterococcus faecalis, 119-1) loaded with 80 markers.\n",
      "2024-02-25 13:21:23,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091895.1 (Enterococcus faecalis, 119-2) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,446 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091884.1 (Enterococcus faecalis, 152) loaded with 72 markers.\n",
      "2024-02-25 13:21:23,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086560.1 (Enterococcus faecalis, E509) loaded with 81 markers.\n",
      "2024-02-25 13:21:23,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078015.1 (Enterococcus faecalis, SY-1) loaded with 79 markers.\n",
      "2024-02-25 13:21:23,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071176.1 (Enterococcus faecalis, L18) loaded with 63 markers.\n",
      "2024-02-25 13:21:23,503 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071172.1 (Enterococcus faecalis, L13) loaded with 63 markers.\n",
      "2024-02-25 13:21:23,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP069185.1 (Enterococcus faecalis, L11) loaded with 63 markers.\n",
      "2024-02-25 13:21:23,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP051005.1 (Enterococcus faecalis, TH4125) loaded with 80 markers.\n",
      "2024-02-25 13:21:23,551 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045045.1 (Enterococcus faecalis, JY32) loaded with 80 markers.\n",
      "2024-02-25 13:21:23,560 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042216.2 (Enterococcus faecalis, L8) loaded with 63 markers.\n",
      "2024-02-25 13:21:23,579 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040898.1 (Enterococcus faecalis, HA-1) loaded with 78 markers.\n",
      "2024-02-25 13:21:23,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028724.1 (Enterococcus faecalis, CVM N60443F) loaded with 60 markers.\n",
      "2024-02-25 13:21:23,598 [DEBUG - chronostrain.database.parser.json] - Strain LR962548.1 (Enterococcus faecalis, 28157_4#330) loaded with 59 markers.\n",
      "2024-02-25 13:21:23,607 [DEBUG - chronostrain.database.parser.json] - Strain LR962517.1 (Enterococcus faecalis, 28157_4#310) loaded with 59 markers.\n",
      "2024-02-25 13:21:23,623 [DEBUG - chronostrain.database.parser.json] - Strain CP124900.1 (Enterococcus faecalis, EfsC108) loaded with 67 markers.\n",
      "2024-02-25 13:21:23,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962548.1 (Enterococcus faecalis, 28157_4#330) loaded with 59 markers.\n",
      "2024-02-25 13:21:23,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR962517.1 (Enterococcus faecalis, 28157_4#310) loaded with 59 markers.\n",
      "2024-02-25 13:21:23,653 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP124900.1 (Enterococcus faecalis, EfsC108) loaded with 67 markers.\n",
      "2024-02-25 13:21:23,670 [DEBUG - chronostrain.database.parser.json] - Strain CP068242.1 (Enterococcus faecalis, T90-2) loaded with 74 markers.\n",
      "2024-02-25 13:21:23,673 [DEBUG - chronostrain.database.parser.json] - Strain CP013614.1 (Enterococcus silesiacus, LMG 23085) loaded with 20 markers.\n",
      "2024-02-25 13:21:23,676 [DEBUG - chronostrain.database.parser.json] - Strain CP006683.1 (Melissococcus plutonius, S1) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,678 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006683.1 (Melissococcus plutonius, S1) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,680 [DEBUG - chronostrain.database.parser.json] - Strain AP021885.1 (Melissococcus plutonius, DAT1033) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,682 [DEBUG - chronostrain.database.parser.json] - Strain AP018524.1 (Melissococcus plutonius, DAT585) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,684 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP021885.1 (Melissococcus plutonius, DAT1033) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,687 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP018524.1 (Melissococcus plutonius, DAT585) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,689 [DEBUG - chronostrain.database.parser.json] - Strain CP104762.1 (Enterococcus raffinosus, ATCC 49464) loaded with 16 markers.\n",
      "2024-02-25 13:21:23,692 [DEBUG - chronostrain.database.parser.json] - Strain CP081846.1 (Enterococcus raffinosus, CX012922) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,694 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP104762.1 (Enterococcus raffinosus, ATCC 49464) loaded with 16 markers.\n",
      "2024-02-25 13:21:23,697 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081846.1 (Enterococcus raffinosus, CX012922) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,699 [DEBUG - chronostrain.database.parser.json] - Strain CP127160.1 (Enterococcus sp., FZMF) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,701 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP127160.1 (Enterococcus sp., FZMF) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,703 [DEBUG - chronostrain.database.parser.json] - Strain CP014067.2 (Enterococcus gallinarum, FDAARGOS_163) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,704 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014067.2 (Enterococcus gallinarum, FDAARGOS_163) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,706 [DEBUG - chronostrain.database.parser.json] - Strain CP116026.1 (Enterococcus casseliflavus, CQFYY22-063) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,708 [DEBUG - chronostrain.database.parser.json] - Strain CP053180.1 (Enterococcus casseliflavus, SP11) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,710 [DEBUG - chronostrain.database.parser.json] - Strain CP050490.1 (Enterococcus casseliflavus, EGM182) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,712 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116026.1 (Enterococcus casseliflavus, CQFYY22-063) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,714 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053180.1 (Enterococcus casseliflavus, SP11) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,716 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050490.1 (Enterococcus casseliflavus, EGM182) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,717 [DEBUG - chronostrain.database.parser.json] - Strain AP025635.1 (Enterococcus innesii, DB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,719 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP025635.1 (Enterococcus innesii, DB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,721 [DEBUG - chronostrain.database.parser.json] - Strain CP119393.1 (Enterococcus casseliflavus, ASE4) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,723 [DEBUG - chronostrain.database.parser.json] - Strain CP068132.1 (Enterococcus casseliflavus, FDAARGOS_1120) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,725 [DEBUG - chronostrain.database.parser.json] - Strain CP068126.1 (Enterococcus casseliflavus, FDAARGOS_1122) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,726 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP119393.1 (Enterococcus casseliflavus, ASE4) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,728 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068132.1 (Enterococcus casseliflavus, FDAARGOS_1120) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,730 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068126.1 (Enterococcus casseliflavus, FDAARGOS_1122) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,732 [DEBUG - chronostrain.database.parser.json] - Strain CP004856.1 (Enterococcus casseliflavus, EC20) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,734 [DEBUG - chronostrain.database.parser.json] - Strain NC_020995.1 (Enterococcus casseliflavus, EC20) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,735 [DEBUG - chronostrain.database.parser.json] - Strain CP066024.1 (Enterococcus casseliflavus, FDAARGOS_998) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,738 [DEBUG - chronostrain.database.parser.json] - Strain CP046123.1 (Enterococcus casseliflavus, EC291) loaded with 16 markers.\n",
      "2024-02-25 13:21:23,740 [DEBUG - chronostrain.database.parser.json] - Strain CP032739.1 (Enterococcus casseliflavus, EC-369) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,742 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066024.1 (Enterococcus casseliflavus, FDAARGOS_998) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,744 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046123.1 (Enterococcus casseliflavus, EC291) loaded with 16 markers.\n",
      "2024-02-25 13:21:23,746 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032739.1 (Enterococcus casseliflavus, EC-369) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,748 [DEBUG - chronostrain.database.parser.json] - Strain CP023515.1 (Enterococcus sp., FDAARGOS_375) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,750 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023515.1 (Enterococcus sp., FDAARGOS_375) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,752 [DEBUG - chronostrain.database.parser.json] - Strain CP025223.1 (Enterococcus sp., CR-Ec1) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,753 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025223.1 (Enterococcus sp., CR-Ec1) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,755 [DEBUG - chronostrain.database.parser.json] - Strain CP119296.1 (Enterococcus casseliflavus, ASE2) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,757 [DEBUG - chronostrain.database.parser.json] - Strain CP086066.1 (Enterococcus casseliflavus, ECB140) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,758 [DEBUG - chronostrain.database.parser.json] - Strain CP068128.1 (Enterococcus casseliflavus, FDAARGOS_1121) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,760 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP119296.1 (Enterococcus casseliflavus, ASE2) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,762 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP086066.1 (Enterococcus casseliflavus, ECB140) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,764 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068128.1 (Enterococcus casseliflavus, FDAARGOS_1121) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,766 [DEBUG - chronostrain.database.parser.json] - Strain CP088203.1 (Enterococcus durans, S8-3) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,768 [DEBUG - chronostrain.database.parser.json] - Strain CP022930.1 (Enterococcus durans, BDGP3) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,770 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088203.1 (Enterococcus durans, S8-3) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,772 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022930.1 (Enterococcus durans, BDGP3) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,774 [DEBUG - chronostrain.database.parser.json] - Strain CP116579.1 (Enterococcus durans, R109-1) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,776 [DEBUG - chronostrain.database.parser.json] - Strain CP116581.1 (Enterococcus durans, R59-1) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,778 [DEBUG - chronostrain.database.parser.json] - Strain CP065535.1 (Enterococcus durans, KCTC 13289) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,780 [DEBUG - chronostrain.database.parser.json] - Strain CP012384.1 (Enterococcus durans, KLDS 6.0930) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,782 [DEBUG - chronostrain.database.parser.json] - Strain CP012366.1 (Enterococcus durans, KLDS6.0933) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,785 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116579.1 (Enterococcus durans, R109-1) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,787 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116581.1 (Enterococcus durans, R59-1) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,789 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065535.1 (Enterococcus durans, KCTC 13289) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,791 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012384.1 (Enterococcus durans, KLDS 6.0930) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,793 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012366.1 (Enterococcus durans, KLDS 6.0933) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,794 [DEBUG - chronostrain.database.parser.json] - Strain OV996158.1 (Enterococcus mundtii, P2005) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,796 [DEBUG - chronostrain.database.parser.json] - Strain CP018061.1 (Enterococcus mundtii, DSM 4838) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,797 [DEBUG - chronostrain.database.parser.json] - Strain CP022340.1 (Enterococcus mundtii, EMB156) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,799 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OV996158.1 (Enterococcus mundtii, P2005) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022340.1 (Enterococcus mundtii, EMB156) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018061.1 (Enterococcus mundtii, DSM 4838) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,805 [DEBUG - chronostrain.database.parser.json] - Strain LR134297.1 (Enterococcus hirae, NCTC12368) loaded with 17 markers.\n",
      "2024-02-25 13:21:23,807 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134297.1 (Enterococcus hirae, NCTC12368) loaded with 17 markers.\n",
      "2024-02-25 13:21:23,809 [DEBUG - chronostrain.database.parser.json] - Strain CP040461.1 (Enterococcus sp., M190262) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040461.1 (Enterococcus sp., M190262) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,812 [DEBUG - chronostrain.database.parser.json] - Strain CP033740.1 (Enterococcus sp., FDAARGOS_553) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,813 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033740.1 (Enterococcus sp., FDAARGOS_553) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,815 [DEBUG - chronostrain.database.parser.json] - Strain CP116510.1 (Enterococcus gallinarum, K205-4a) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,816 [DEBUG - chronostrain.database.parser.json] - Strain CP078505.1 (Enterococcus gallinarum, fecal_iso1) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,818 [DEBUG - chronostrain.database.parser.json] - Strain CP091204.1 (Enterococcus gallinarum, 674) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,819 [DEBUG - chronostrain.database.parser.json] - Strain CP050816.1 (Enterococcus gallinarum, EG81) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,821 [DEBUG - chronostrain.database.parser.json] - Strain CP050485.1 (Enterococcus gallinarum, EGM181) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,823 [DEBUG - chronostrain.database.parser.json] - Strain CP046307.1 (Enterococcus gallinarum, FDAARGOS_728) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116510.1 (Enterococcus gallinarum, K205-4a) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,826 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078505.1 (Enterococcus gallinarum, fecal_iso1) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,827 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091204.1 (Enterococcus gallinarum, 674) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,829 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050816.1 (Enterococcus gallinarum, EG81) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050485.1 (Enterococcus gallinarum, EGM181) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046307.1 (Enterococcus gallinarum, FDAARGOS_728) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,834 [DEBUG - chronostrain.database.parser.json] - Strain AP013036.1 (Enterococcus mundtii, QU25) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,835 [DEBUG - chronostrain.database.parser.json] - Strain NC_022878.1 (Enterococcus mundtii, QU25) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,837 [DEBUG - chronostrain.database.parser.json] - Strain CP083695.1 (Enterococcus mundtii, P6) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,838 [DEBUG - chronostrain.database.parser.json] - Strain AP019810.1 (Enterococcus mundtii, 15-1A) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,840 [DEBUG - chronostrain.database.parser.json] - Strain CP029066.1 (Enterococcus mundtii, Pe103) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083695.1 (Enterococcus mundtii, P6) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,843 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019810.1 (Enterococcus mundtii, 15-1A) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,845 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP029066.1 (Enterococcus mundtii, Pe103) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,847 [DEBUG - chronostrain.database.parser.json] - Strain LR994474.1 (Enterococcus hirae, Enterococcus hirae canine HQ MAG) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,849 [DEBUG - chronostrain.database.parser.json] - Strain CP003504.1 (Enterococcus hirae, ATCC 9790) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,851 [DEBUG - chronostrain.database.parser.json] - Strain NC_018081.1 (Enterococcus hirae, ATCC 9790) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,853 [DEBUG - chronostrain.database.parser.json] - Strain AP027299.1 (Enterococcus hirae, L6E) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,855 [DEBUG - chronostrain.database.parser.json] - Strain CP122304.1 (Enterococcus hirae, DSM 20160) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,858 [DEBUG - chronostrain.database.parser.json] - Strain CP109808.1 (Enterococcus hirae, TF52) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,860 [DEBUG - chronostrain.database.parser.json] - Strain CP109807.1 (Enterococcus hirae, TF34) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,862 [DEBUG - chronostrain.database.parser.json] - Strain CP109803.1 (Enterococcus hirae, TF14-3) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,864 [DEBUG - chronostrain.database.parser.json] - Strain CP109802.1 (Enterococcus hirae, T16-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,866 [DEBUG - chronostrain.database.parser.json] - Strain CP091201.1 (Enterococcus hirae, 697) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,868 [DEBUG - chronostrain.database.parser.json] - Strain CP088187.1 (Enterococcus hirae, S46-3) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,870 [DEBUG - chronostrain.database.parser.json] - Strain CP088183.1 (Enterococcus hirae, S2-7) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,872 [DEBUG - chronostrain.database.parser.json] - Strain CP093926.1 (Enterococcus hirae, GDIAS-5) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,875 [DEBUG - chronostrain.database.parser.json] - Strain CP072891.1 (Enterococcus hirae, F104) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,877 [DEBUG - chronostrain.database.parser.json] - Strain CP068118.1 (Enterococcus hirae, FDAARGOS_1125) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,879 [DEBUG - chronostrain.database.parser.json] - Strain CP068122.1 (Enterococcus hirae, FDAARGOS_1124) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,881 [DEBUG - chronostrain.database.parser.json] - Strain CP065992.1 (Enterococcus hirae, FDAARGOS_1057) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,883 [DEBUG - chronostrain.database.parser.json] - Strain CP055232.1 (Enterococcus hirae, 708) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,885 [DEBUG - chronostrain.database.parser.json] - Strain CP055231.1 (Enterococcus hirae, 13144) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,887 [DEBUG - chronostrain.database.parser.json] - Strain CP055230.1 (Enterococcus hirae, 13152) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,889 [DEBUG - chronostrain.database.parser.json] - Strain CP055229.1 (Enterococcus hirae, 13344) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,891 [DEBUG - chronostrain.database.parser.json] - Strain CP037955.1 (Enterococcus hirae, CQP3-9) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,893 [DEBUG - chronostrain.database.parser.json] - Strain CP023011.2 (Enterococcus hirae, FDAARGOS_234) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,895 [DEBUG - chronostrain.database.parser.json] - Strain CP015516.1 (Enterococcus hirae, R17) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,897 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP027299.1 (Enterococcus hirae, L6E) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,899 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP122304.1 (Enterococcus hirae, DSM 20160) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109808.1 (Enterococcus hirae, TF52) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,904 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109807.1 (Enterococcus hirae, TF34) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109803.1 (Enterococcus hirae, TF14-3) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109802.1 (Enterococcus hirae, T16-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091201.1 (Enterococcus hirae, 697) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,912 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088187.1 (Enterococcus hirae, S46-3) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,914 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088183.1 (Enterococcus hirae, S2-7) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,916 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093926.1 (Enterococcus hirae, GDIAS-5) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,919 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072891.1 (Enterococcus hirae, F104) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,921 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068118.1 (Enterococcus hirae, FDAARGOS_1125) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068122.1 (Enterococcus hirae, FDAARGOS_1124) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065992.1 (Enterococcus hirae, FDAARGOS_1057) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055232.1 (Enterococcus hirae, 708) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,929 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055231.1 (Enterococcus hirae, 13144) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,931 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055230.1 (Enterococcus hirae, 13152) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,933 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055229.1 (Enterococcus hirae, 13344) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,935 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP037955.1 (Enterococcus hirae, CQP3-9) loaded with 15 markers.\n",
      "2024-02-25 13:21:23,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP015516.1 (Enterococcus hirae, R17) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023011.2 (Enterococcus hirae, FDAARGOS_234) loaded with 14 markers.\n",
      "2024-02-25 13:21:23,942 [DEBUG - chronostrain.database.parser.json] - Strain CP023074.1 (Enterococcus thailandicus, a523) loaded with 17 markers.\n",
      "2024-02-25 13:21:23,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023074.1 (Enterococcus thailandicus, a523) loaded with 17 markers.\n",
      "2024-02-25 13:21:23,949 [DEBUG - chronostrain.database.parser.json] - Strain CP042839.1 (Enterococcus sp., DA9) loaded with 7 markers.\n",
      "2024-02-25 13:21:23,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042839.1 (Enterococcus sp., DA9) loaded with 7 markers.\n",
      "2024-02-25 13:21:23,951 [DEBUG - chronostrain.database.parser.json] - Strain CP082267.1 (Enterococcus lactis, E843) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,953 [DEBUG - chronostrain.database.parser.json] - Strain CP065211.1 (Enterococcus lactis, KCTC 21015) loaded with 8 markers.\n",
      "2024-02-25 13:21:23,954 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082267.1 (Enterococcus lactis, E843) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,955 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065211.1 (Enterococcus lactis, KCTC 21015) loaded with 8 markers.\n",
      "2024-02-25 13:21:23,957 [DEBUG - chronostrain.database.parser.json] - Strain CP006620.1 (Enterococcus faecium, AUS0085) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,958 [DEBUG - chronostrain.database.parser.json] - Strain NC_021994.1 (Enterococcus faecium, AUS0085) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,960 [DEBUG - chronostrain.database.parser.json] - Strain CP003351.1 (Enterococcus faecium, Aus0004) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,962 [DEBUG - chronostrain.database.parser.json] - Strain NC_017022.1 (Enterococcus faecium, Aus0004) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,964 [DEBUG - chronostrain.database.parser.json] - Strain CP004063.1 (Enterococcus faecium, NRRL B-2354) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,966 [DEBUG - chronostrain.database.parser.json] - Strain NC_020207.1 (Enterococcus faecium, NRRL B-2354) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,968 [DEBUG - chronostrain.database.parser.json] - Strain OU016036.1 (Enterococcus faecium, USZ_VRE53_P46) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,970 [DEBUG - chronostrain.database.parser.json] - Strain OU016027.1 (Enterococcus faecium, USZ_VRE67_P60) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,971 [DEBUG - chronostrain.database.parser.json] - Strain OU015946.1 (Enterococcus faecium, USZ_VRE5_P5) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,973 [DEBUG - chronostrain.database.parser.json] - Strain OU015708.1 (Enterococcus faecium, USZ_VRE32_P32) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,975 [DEBUG - chronostrain.database.parser.json] - Strain OU015350.1 (Enterococcus faecium, AUS2002) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,976 [DEBUG - chronostrain.database.parser.json] - Strain OU015345.1 (Enterococcus faecium, AUS2001) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,978 [DEBUG - chronostrain.database.parser.json] - Strain LR215968.1 (Enterococcus faecium, 3012STDY6244127) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,980 [DEBUG - chronostrain.database.parser.json] - Strain LR135482.1 (Enterococcus faecium, E4456) loaded with 11 markers.\n",
      "2024-02-25 13:21:23,981 [DEBUG - chronostrain.database.parser.json] - Strain LR135475.1 (Enterococcus faecium, E8423) loaded with 10 markers.\n",
      "2024-02-25 13:21:23,983 [DEBUG - chronostrain.database.parser.json] - Strain LR135488.1 (Enterococcus faecium, E8414) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,985 [DEBUG - chronostrain.database.parser.json] - Strain LR135474.1 (Enterococcus faecium, E9101) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,987 [DEBUG - chronostrain.database.parser.json] - Strain LR135443.1 (Enterococcus faecium, E7114) loaded with 12 markers.\n",
      "2024-02-25 13:21:23,988 [DEBUG - chronostrain.database.parser.json] - Strain LR135435.1 (Enterococcus faecium, E8691) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,990 [DEBUG - chronostrain.database.parser.json] - Strain LR135419.1 (Enterococcus faecium, E8440) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,991 [DEBUG - chronostrain.database.parser.json] - Strain LR135428.1 (Enterococcus faecium, E8927) loaded with 9 markers.\n",
      "2024-02-25 13:21:23,993 [DEBUG - chronostrain.database.parser.json] - Strain LR135408.1 (Enterococcus faecium, E8284) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,995 [DEBUG - chronostrain.database.parser.json] - Strain LR135414.1 (Enterococcus faecium, E8328) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,997 [DEBUG - chronostrain.database.parser.json] - Strain LR135394.1 (Enterococcus faecium, E8290) loaded with 13 markers.\n",
      "2024-02-25 13:21:23,999 [DEBUG - chronostrain.database.parser.json] - Strain LR135401.1 (Enterococcus faecium, E8377) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,001 [DEBUG - chronostrain.database.parser.json] - Strain LR135357.1 (Enterococcus faecium, E7948) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,003 [DEBUG - chronostrain.database.parser.json] - Strain LR135351.1 (Enterococcus faecium, E8014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,005 [DEBUG - chronostrain.database.parser.json] - Strain LR135364.1 (Enterococcus faecium, E8195) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,006 [DEBUG - chronostrain.database.parser.json] - Strain LR135372.1 (Enterococcus faecium, E8172) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,009 [DEBUG - chronostrain.database.parser.json] - Strain LR135384.1 (Enterococcus faecium, E7933) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,011 [DEBUG - chronostrain.database.parser.json] - Strain LR135344.1 (Enterococcus faecium, E8202) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,013 [DEBUG - chronostrain.database.parser.json] - Strain LR135317.1 (Enterococcus faecium, E7663) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,015 [DEBUG - chronostrain.database.parser.json] - Strain LR135324.1 (Enterococcus faecium, E7654) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,017 [DEBUG - chronostrain.database.parser.json] - Strain LR135339.1 (Enterococcus faecium, E7356) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,019 [DEBUG - chronostrain.database.parser.json] - Strain LR135307.1 (Enterococcus faecium, E7240) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,021 [DEBUG - chronostrain.database.parser.json] - Strain LR135297.1 (Enterococcus faecium, E7429) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,023 [DEBUG - chronostrain.database.parser.json] - Strain LR135293.1 (Enterococcus faecium, E7237) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,025 [DEBUG - chronostrain.database.parser.json] - Strain LR135254.1 (Enterococcus faecium, E7098) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,026 [DEBUG - chronostrain.database.parser.json] - Strain LR135243.1 (Enterococcus faecium, E6988) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,028 [DEBUG - chronostrain.database.parser.json] - Strain LR135258.1 (Enterococcus faecium, E4457) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,030 [DEBUG - chronostrain.database.parser.json] - Strain LR135278.1 (Enterococcus faecium, E6975) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,032 [DEBUG - chronostrain.database.parser.json] - Strain LR135219.1 (Enterococcus faecium, E7040) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,033 [DEBUG - chronostrain.database.parser.json] - Strain LR135235.1 (Enterococcus faecium, E7067) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,035 [DEBUG - chronostrain.database.parser.json] - Strain LR135191.1 (Enterococcus faecium, E4438) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,037 [DEBUG - chronostrain.database.parser.json] - Strain LR135179.1 (Enterococcus faecium, E0595) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,039 [DEBUG - chronostrain.database.parser.json] - Strain LR135174.1 (Enterococcus faecium, E4402) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,041 [DEBUG - chronostrain.database.parser.json] - Strain LR135169.1 (Enterococcus faecium, E2079) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,042 [DEBUG - chronostrain.database.parser.json] - Strain LR134337.1 (Enterococcus faecium, NCTC7174) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,044 [DEBUG - chronostrain.database.parser.json] - Strain LR134105.1 (Enterococcus faecium, E6043) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,045 [DEBUG - chronostrain.database.parser.json] - Strain LR134095.1 (Enterococcus faecium, E1334) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,047 [DEBUG - chronostrain.database.parser.json] - Strain LR132067.1 (Enterococcus faecium, E0139) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,049 [DEBUG - chronostrain.database.parser.json] - Strain LT603678.1 (Enterococcus faecium, Ef_DMG1500501) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,050 [DEBUG - chronostrain.database.parser.json] - Strain LT598663.1 (Enterococcus faecium, Ef_aus00233) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,052 [DEBUG - chronostrain.database.parser.json] - Strain LN999844.1 (Enterococcus faecium, EFE10021) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,054 [DEBUG - chronostrain.database.parser.json] - Strain LN999987.1 (Enterococcus faecium, EFE11651) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,056 [DEBUG - chronostrain.database.parser.json] - Strain CP132226.1 (Enterococcus faecium, TC64_3x4995_20_TV8) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,058 [DEBUG - chronostrain.database.parser.json] - Strain CP131532.1 (Enterococcus faecium, H0019EM0023) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,060 [DEBUG - chronostrain.database.parser.json] - Strain CP131530.1 (Enterococcus faecium, H0019EM0021) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,062 [DEBUG - chronostrain.database.parser.json] - Strain CP131512.1 (Enterococcus faecium, G0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,067 [DEBUG - chronostrain.database.parser.json] - Strain CP131506.1 (Enterococcus faecium, G0019EM0015) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,069 [DEBUG - chronostrain.database.parser.json] - Strain CP131504.1 (Enterococcus faecium, G0019EM0013) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,071 [DEBUG - chronostrain.database.parser.json] - Strain CP131496.1 (Enterococcus faecium, G0019EM0006) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,073 [DEBUG - chronostrain.database.parser.json] - Strain CP131508.1 (Enterococcus faecium, G0019EM0017) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,075 [DEBUG - chronostrain.database.parser.json] - Strain CP131510.1 (Enterococcus faecium, G0019EM0019) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,077 [DEBUG - chronostrain.database.parser.json] - Strain CP131498.1 (Enterococcus faecium, G0019EM0010) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,079 [DEBUG - chronostrain.database.parser.json] - Strain CP131610.1 (Enterococcus faecium, G0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,081 [DEBUG - chronostrain.database.parser.json] - Strain CP131494.1 (Enterococcus faecium, G0019EM0002) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,083 [DEBUG - chronostrain.database.parser.json] - Strain CP131492.1 (Enterococcus faecium, G0019EM0001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,085 [DEBUG - chronostrain.database.parser.json] - Strain CP131528.1 (Enterococcus faecium, H0019EM0013) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,087 [DEBUG - chronostrain.database.parser.json] - Strain CP131522.1 (Enterococcus faecium, H0019EM0006) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,089 [DEBUG - chronostrain.database.parser.json] - Strain CP131518.1 (Enterococcus faecium, G0019EM0027) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,090 [DEBUG - chronostrain.database.parser.json] - Strain CP131520.1 (Enterococcus faecium, H0019EM0003) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,092 [DEBUG - chronostrain.database.parser.json] - Strain CP131526.1 (Enterococcus faecium, H0019EM0009) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,094 [DEBUG - chronostrain.database.parser.json] - Strain CP131524.1 (Enterococcus faecium, H0019EM0007) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,096 [DEBUG - chronostrain.database.parser.json] - Strain CP131514.1 (Enterococcus faecium, G0019EM0022) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,098 [DEBUG - chronostrain.database.parser.json] - Strain CP131516.1 (Enterococcus faecium, G0019EM0023) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,099 [DEBUG - chronostrain.database.parser.json] - Strain CP131502.1 (Enterococcus faecium, G0019EM0012) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,101 [DEBUG - chronostrain.database.parser.json] - Strain CP131500.1 (Enterococcus faecium, G0019EM0011) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,103 [DEBUG - chronostrain.database.parser.json] - Strain CP131490.1 (Enterococcus faecium, D0019EM0022) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,104 [DEBUG - chronostrain.database.parser.json] - Strain CP131489.1 (Enterococcus faecium, G0019EM0003) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,106 [DEBUG - chronostrain.database.parser.json] - Strain CP131123.1 (Enterococcus faecium, F0019EM0050) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,108 [DEBUG - chronostrain.database.parser.json] - Strain CP131125.1 (Enterococcus faecium, F0019EM0056) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,110 [DEBUG - chronostrain.database.parser.json] - Strain CP131121.1 (Enterococcus faecium, F0019EM0049) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,111 [DEBUG - chronostrain.database.parser.json] - Strain CP131119.1 (Enterococcus faecium, F0019EM0039) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,113 [DEBUG - chronostrain.database.parser.json] - Strain CP131107.1 (Enterococcus faecium, F0019EM0030) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,115 [DEBUG - chronostrain.database.parser.json] - Strain CP131117.1 (Enterococcus faecium, F0019EM0037) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,116 [DEBUG - chronostrain.database.parser.json] - Strain CP131115.1 (Enterococcus faecium, F0019EM0035) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,118 [DEBUG - chronostrain.database.parser.json] - Strain CP131113.1 (Enterococcus faecium, F0019EM0033) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,120 [DEBUG - chronostrain.database.parser.json] - Strain CP131109.1 (Enterococcus faecium, F0019EM0031) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,122 [DEBUG - chronostrain.database.parser.json] - Strain CP131111.1 (Enterococcus faecium, F0019EM0032) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,124 [DEBUG - chronostrain.database.parser.json] - Strain CP131105.1 (Enterococcus faecium, F0019EM0027) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,125 [DEBUG - chronostrain.database.parser.json] - Strain CP131103.1 (Enterococcus faecium, F0019EM0026) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,127 [DEBUG - chronostrain.database.parser.json] - Strain CP131099.1 (Enterococcus faecium, F0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,129 [DEBUG - chronostrain.database.parser.json] - Strain CP131097.1 (Enterococcus faecium, F0019EM0020) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,130 [DEBUG - chronostrain.database.parser.json] - Strain CP131101.1 (Enterococcus faecium, F0019EM0024) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,132 [DEBUG - chronostrain.database.parser.json] - Strain CP131093.1 (Enterococcus faecium, F0019EM0004) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,134 [DEBUG - chronostrain.database.parser.json] - Strain CP131095.1 (Enterococcus faecium, F0019EM0005) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,136 [DEBUG - chronostrain.database.parser.json] - Strain CP131069.1 (Enterococcus faecium, E0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,138 [DEBUG - chronostrain.database.parser.json] - Strain CP131077.1 (Enterococcus faecium, E0019EM0042) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,140 [DEBUG - chronostrain.database.parser.json] - Strain CP131075.1 (Enterococcus faecium, E0019EM0028) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,142 [DEBUG - chronostrain.database.parser.json] - Strain CP131071.1 (Enterococcus faecium, E0019EM0024) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,144 [DEBUG - chronostrain.database.parser.json] - Strain CP131087.1 (Enterococcus faecium, E0019EM0078) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,146 [DEBUG - chronostrain.database.parser.json] - Strain CP131089.1 (Enterococcus faecium, E0019EM0079) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,148 [DEBUG - chronostrain.database.parser.json] - Strain CP131079.1 (Enterococcus faecium, E0019EM0047) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,150 [DEBUG - chronostrain.database.parser.json] - Strain CP131081.1 (Enterococcus faecium, E0019EM0059) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,152 [DEBUG - chronostrain.database.parser.json] - Strain CP131091.1 (Enterococcus faecium, F0019EM0002) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,154 [DEBUG - chronostrain.database.parser.json] - Strain CP131085.1 (Enterococcus faecium, E0019EM0077) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,156 [DEBUG - chronostrain.database.parser.json] - Strain CP131083.1 (Enterococcus faecium, E0019EM0072) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,158 [DEBUG - chronostrain.database.parser.json] - Strain CP131073.1 (Enterococcus faecium, E0019EM0026) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,160 [DEBUG - chronostrain.database.parser.json] - Strain CP131067.1 (Enterococcus faecium, E0019EM0020) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,162 [DEBUG - chronostrain.database.parser.json] - Strain CP131063.1 (Enterococcus faecium, E0019EM0013) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,164 [DEBUG - chronostrain.database.parser.json] - Strain CP131065.1 (Enterococcus faecium, E0019EM0016) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,166 [DEBUG - chronostrain.database.parser.json] - Strain CP131054.1 (Enterococcus faecium, E0019EM0009) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,168 [DEBUG - chronostrain.database.parser.json] - Strain CP131052.1 (Enterococcus faecium, E0019EM0007) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,170 [DEBUG - chronostrain.database.parser.json] - Strain CP131050.1 (Enterococcus faecium, E0019EM0002) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,172 [DEBUG - chronostrain.database.parser.json] - Strain CP131048.1 (Enterococcus faecium, E0019EM0001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,174 [DEBUG - chronostrain.database.parser.json] - Strain CP131046.1 (Enterococcus faecium, D0019EM0029) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,176 [DEBUG - chronostrain.database.parser.json] - Strain CP131044.1 (Enterococcus faecium, D0019EM0026) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,178 [DEBUG - chronostrain.database.parser.json] - Strain CP131042.1 (Enterococcus faecium, D0019EM0024) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,180 [DEBUG - chronostrain.database.parser.json] - Strain CP131040.1 (Enterococcus faecium, D0019EM0023) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,182 [DEBUG - chronostrain.database.parser.json] - Strain CP131020.1 (Enterococcus faecium, F0019EM0041) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,184 [DEBUG - chronostrain.database.parser.json] - Strain CP131016.1 (Enterococcus faecium, F0019EM0011) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,186 [DEBUG - chronostrain.database.parser.json] - Strain CP131018.1 (Enterococcus faecium, F0019EM0017) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,188 [DEBUG - chronostrain.database.parser.json] - Strain CP131014.1 (Enterococcus faecium, F0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,190 [DEBUG - chronostrain.database.parser.json] - Strain CP131012.1 (Enterococcus faecium, F0019EM0006) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,192 [DEBUG - chronostrain.database.parser.json] - Strain CP131010.1 (Enterococcus faecium, F0019EM0001) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,194 [DEBUG - chronostrain.database.parser.json] - Strain CP131027.1 (Enterococcus faecium, C0019EM0035) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,196 [DEBUG - chronostrain.database.parser.json] - Strain CP131031.1 (Enterococcus faecium, D0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,198 [DEBUG - chronostrain.database.parser.json] - Strain CP131025.1 (Enterococcus faecium, C0019EM0033) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,200 [DEBUG - chronostrain.database.parser.json] - Strain CP131035.1 (Enterococcus faecium, D0019EM0012) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,202 [DEBUG - chronostrain.database.parser.json] - Strain CP131033.1 (Enterococcus faecium, D0019EM0009) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,204 [DEBUG - chronostrain.database.parser.json] - Strain CP131037.1 (Enterococcus faecium, D0019EM0013) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,206 [DEBUG - chronostrain.database.parser.json] - Strain CP131029.1 (Enterococcus faecium, D0019EM0001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,208 [DEBUG - chronostrain.database.parser.json] - Strain CP131006.1 (Enterococcus faecium, C0019EM0031) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,210 [DEBUG - chronostrain.database.parser.json] - Strain CP131000.1 (Enterococcus faecium, C0019EM0028) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,212 [DEBUG - chronostrain.database.parser.json] - Strain CP130998.1 (Enterococcus faecium, C0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,214 [DEBUG - chronostrain.database.parser.json] - Strain CP131004.1 (Enterococcus faecium, C0019EM0030) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,216 [DEBUG - chronostrain.database.parser.json] - Strain CP130996.1 (Enterococcus faecium, C0019EM0020) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,218 [DEBUG - chronostrain.database.parser.json] - Strain CP131002.1 (Enterococcus faecium, C0019EM0029) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,220 [DEBUG - chronostrain.database.parser.json] - Strain CP130992.1 (Enterococcus faecium, C0019EM0018) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,222 [DEBUG - chronostrain.database.parser.json] - Strain CP130994.1 (Enterococcus faecium, C0019EM0019) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,223 [DEBUG - chronostrain.database.parser.json] - Strain CP131024.1 (Enterococcus faecium, C0019EM0037) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,225 [DEBUG - chronostrain.database.parser.json] - Strain CP130914.1 (Enterococcus faecium, C0019EM0006) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,227 [DEBUG - chronostrain.database.parser.json] - Strain CP130912.1 (Enterococcus faecium, C0019EM0002) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,229 [DEBUG - chronostrain.database.parser.json] - Strain CP130908.1 (Enterococcus faecium, B0019EM0041) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,230 [DEBUG - chronostrain.database.parser.json] - Strain CP130910.1 (Enterococcus faecium, B0019EM0045) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,232 [DEBUG - chronostrain.database.parser.json] - Strain CP130906.1 (Enterococcus faecium, B0019EM0038) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,234 [DEBUG - chronostrain.database.parser.json] - Strain CP130883.1 (Enterococcus faecium, B0019EM0034) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,236 [DEBUG - chronostrain.database.parser.json] - Strain CP130879.1 (Enterococcus faecium, A0019EM0007) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,237 [DEBUG - chronostrain.database.parser.json] - Strain CP130875.1 (Enterococcus faecium, A0019EM0010) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,239 [DEBUG - chronostrain.database.parser.json] - Strain CP130877.1 (Enterococcus faecium, A0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,241 [DEBUG - chronostrain.database.parser.json] - Strain CP130881.1 (Enterococcus faecium, C0019EM0016) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,246 [DEBUG - chronostrain.database.parser.json] - Strain CP130882.1 (Enterococcus faecium, B0019EM0040) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,248 [DEBUG - chronostrain.database.parser.json] - Strain CP130849.1 (Enterococcus faecium, B0019EM0016) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,250 [DEBUG - chronostrain.database.parser.json] - Strain CP130833.1 (Enterococcus faecium, B0019EM0032) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,251 [DEBUG - chronostrain.database.parser.json] - Strain CP130841.1 (Enterococcus faecium, B0019EM0023) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,253 [DEBUG - chronostrain.database.parser.json] - Strain CP130837.1 (Enterococcus faecium, B0019EM0028) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,255 [DEBUG - chronostrain.database.parser.json] - Strain CP130843.1 (Enterococcus faecium, B0019EM0022) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,257 [DEBUG - chronostrain.database.parser.json] - Strain CP130835.1 (Enterococcus faecium, B0019EM0030) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,258 [DEBUG - chronostrain.database.parser.json] - Strain CP130839.1 (Enterococcus faecium, B0019EM0026) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,260 [DEBUG - chronostrain.database.parser.json] - Strain CP130845.1 (Enterococcus faecium, B0019EM0019) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,336 [DEBUG - chronostrain.database.parser.json] - Strain CP130847.1 (Enterococcus faecium, B0019EM0017) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,338 [DEBUG - chronostrain.database.parser.json] - Strain CP130851.1 (Enterococcus faecium, B0019EM0015) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,340 [DEBUG - chronostrain.database.parser.json] - Strain CP130861.1 (Enterococcus faecium, B0019EM0012) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,342 [DEBUG - chronostrain.database.parser.json] - Strain CP130859.1 (Enterococcus faecium, B0019EM0014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,344 [DEBUG - chronostrain.database.parser.json] - Strain CP130865.1 (Enterococcus faecium, B0019EM0004) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,345 [DEBUG - chronostrain.database.parser.json] - Strain CP130863.1 (Enterococcus faecium, B0019EM0006) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,347 [DEBUG - chronostrain.database.parser.json] - Strain CP130867.1 (Enterococcus faecium, A0019EM0035) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,349 [DEBUG - chronostrain.database.parser.json] - Strain CP130869.1 (Enterococcus faecium, A0019EM0022) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,351 [DEBUG - chronostrain.database.parser.json] - Strain CP130873.1 (Enterococcus faecium, A0019EM0020) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,352 [DEBUG - chronostrain.database.parser.json] - Strain CP130871.1 (Enterococcus faecium, A0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,354 [DEBUG - chronostrain.database.parser.json] - Strain CP130827.1 (Enterococcus faecium, C0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,356 [DEBUG - chronostrain.database.parser.json] - Strain CP130825.1 (Enterococcus faecium, C0019EM0014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,358 [DEBUG - chronostrain.database.parser.json] - Strain CP130829.1 (Enterococcus faecium, C0019EM0007) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,360 [DEBUG - chronostrain.database.parser.json] - Strain CP128188.1 (Enterococcus faecium, A2) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,361 [DEBUG - chronostrain.database.parser.json] - Strain AP027294.1 (Enterococcus faecium, K6D) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,363 [DEBUG - chronostrain.database.parser.json] - Strain CP109758.1 (Enterococcus faecium, XJ78NG) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,365 [DEBUG - chronostrain.database.parser.json] - Strain CP119150.1 (Enterococcus faecium, GENOMIC22-005) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,367 [DEBUG - chronostrain.database.parser.json] - Strain CP118955.1 (Enterococcus faecium, DSM 20477) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,368 [DEBUG - chronostrain.database.parser.json] - Strain CP118548.1 (Enterococcus faecium, W6) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,370 [DEBUG - chronostrain.database.parser.json] - Strain CP116528.1 (Enterococcus faecium, K188-5) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,372 [DEBUG - chronostrain.database.parser.json] - Strain CP116523.1 (Enterococcus faecium, K69-1a) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,374 [DEBUG - chronostrain.database.parser.json] - Strain CP116520.1 (Enterococcus faecium, R186-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,375 [DEBUG - chronostrain.database.parser.json] - Strain CP116517.1 (Enterococcus faecium, R39-1) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,377 [DEBUG - chronostrain.database.parser.json] - Strain CP116532.1 (Enterococcus faecium, K191-3) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,378 [DEBUG - chronostrain.database.parser.json] - Strain CP116514.1 (Enterococcus faecium, K195-6b) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,380 [DEBUG - chronostrain.database.parser.json] - Strain CP116023.1 (Enterococcus faecium, CQFYH22-006) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,381 [DEBUG - chronostrain.database.parser.json] - Strain CP115812.1 (Enterococcus faecium, DMEA09) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,383 [DEBUG - chronostrain.database.parser.json] - Strain CP114617.1 (Enterococcus faecium, EF0656) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,385 [DEBUG - chronostrain.database.parser.json] - Strain CP112862.1 (Enterococcus faecium, DVT1681) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,387 [DEBUG - chronostrain.database.parser.json] - Strain AP026666.1 (Enterococcus faecium, GK961) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,388 [DEBUG - chronostrain.database.parser.json] - Strain AP026660.1 (Enterococcus faecium, GK941) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,390 [DEBUG - chronostrain.database.parser.json] - Strain AP026655.1 (Enterococcus faecium, GK923) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,392 [DEBUG - chronostrain.database.parser.json] - Strain AP026650.1 (Enterococcus faecium, AA818) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,394 [DEBUG - chronostrain.database.parser.json] - Strain AP026641.1 (Enterococcus faecium, AA610) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,396 [DEBUG - chronostrain.database.parser.json] - Strain AP026632.1 (Enterococcus faecium, AA290) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,397 [DEBUG - chronostrain.database.parser.json] - Strain AP026628.1 (Enterococcus faecium, AA242) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,399 [DEBUG - chronostrain.database.parser.json] - Strain AP026617.1 (Enterococcus faecium, AA96) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,401 [DEBUG - chronostrain.database.parser.json] - Strain AP026604.1 (Enterococcus faecium, AA55) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,403 [DEBUG - chronostrain.database.parser.json] - Strain AP026601.1 (Enterococcus faecium, JHP80) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,404 [DEBUG - chronostrain.database.parser.json] - Strain AP026593.1 (Enterococcus faecium, JHP38) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,406 [DEBUG - chronostrain.database.parser.json] - Strain AP026586.1 (Enterococcus faecium, JHP36) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,408 [DEBUG - chronostrain.database.parser.json] - Strain AP026579.1 (Enterococcus faecium, JHP35) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,410 [DEBUG - chronostrain.database.parser.json] - Strain AP026573.1 (Enterococcus faecium, JHP10) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,411 [DEBUG - chronostrain.database.parser.json] - Strain AP026566.1 (Enterococcus faecium, JHP9) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,413 [DEBUG - chronostrain.database.parser.json] - Strain AP026772.1 (Enterococcus faecium, NUITM-VRE1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,415 [DEBUG - chronostrain.database.parser.json] - Strain CP109798.1 (Enterococcus faecium, TF51-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,417 [DEBUG - chronostrain.database.parser.json] - Strain JAOXXZ010000001.1 (Enterococcus faecium, IC25) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,419 [DEBUG - chronostrain.database.parser.json] - Strain JAOXXY010000002.1 (Enterococcus faecium, BTF71-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,420 [DEBUG - chronostrain.database.parser.json] - Strain JAOXXX010000002.1 (Enterococcus faecium, ATF64) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,421 [DEBUG - chronostrain.database.parser.json] - Strain CP102534.1 (Enterococcus faecium, ENCfa-68) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,423 [DEBUG - chronostrain.database.parser.json] - Strain CP058891.1 (Enterococcus faecium, ef332) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,425 [DEBUG - chronostrain.database.parser.json] - Strain CP091224.1 (Enterococcus faecium, 237) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,426 [DEBUG - chronostrain.database.parser.json] - Strain CP091217.1 (Enterococcus faecium, 1525) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,428 [DEBUG - chronostrain.database.parser.json] - Strain CP091213.1 (Enterococcus faecium, 264a) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,430 [DEBUG - chronostrain.database.parser.json] - Strain CP091210.1 (Enterococcus faecium, 211b) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,431 [DEBUG - chronostrain.database.parser.json] - Strain CP091206.1 (Enterococcus faecium, 1818) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,433 [DEBUG - chronostrain.database.parser.json] - Strain CP088191.1 (Enterococcus faecium, S2-6b) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,435 [DEBUG - chronostrain.database.parser.json] - Strain CP097022.1 (Enterococcus faecium, AT45b) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,437 [DEBUG - chronostrain.database.parser.json] - Strain CP097014.1 (Enterococcus faecium, AT47b) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,438 [DEBUG - chronostrain.database.parser.json] - Strain CP096219.1 (Enterococcus faecium, UCH5) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,440 [DEBUG - chronostrain.database.parser.json] - Strain CP096211.1 (Enterococcus faecium, UCH1) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,442 [DEBUG - chronostrain.database.parser.json] - Strain CP110360.1 (Enterococcus faecium, AKSZ-142) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,444 [DEBUG - chronostrain.database.parser.json] - Strain CP110294.1 (Enterococcus faecium, AKSZ-47) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,445 [DEBUG - chronostrain.database.parser.json] - Strain CP063633.1 (Enterococcus faecium, ICU-5-1) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,447 [DEBUG - chronostrain.database.parser.json] - Strain CP063626.1 (Enterococcus faecium, ICU-4-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,449 [DEBUG - chronostrain.database.parser.json] - Strain CP063618.1 (Enterococcus faecium, BMT-4-1-3) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,451 [DEBUG - chronostrain.database.parser.json] - Strain CP063611.1 (Enterococcus faecium, ICU-1-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,453 [DEBUG - chronostrain.database.parser.json] - Strain CP063647.1 (Enterococcus faecium, BMT-2-1-3) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,455 [DEBUG - chronostrain.database.parser.json] - Strain CP063604.1 (Enterococcus faecium, BMT-2-3-8) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,457 [DEBUG - chronostrain.database.parser.json] - Strain CP063598.1 (Enterococcus faecium, BMT-3-2-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,459 [DEBUG - chronostrain.database.parser.json] - Strain CP063593.1 (Enterococcus faecium, BMT-1-3-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,461 [DEBUG - chronostrain.database.parser.json] - Strain CP063586.1 (Enterococcus faecium, BMT-2-1-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,463 [DEBUG - chronostrain.database.parser.json] - Strain CP063574.1 (Enterococcus faecium, BMT-1-1-12) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,465 [DEBUG - chronostrain.database.parser.json] - Strain CP063568.1 (Enterococcus faecium, BMT-1-1-11) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,467 [DEBUG - chronostrain.database.parser.json] - Strain CP063562.1 (Enterococcus faecium, ICU-3-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,469 [DEBUG - chronostrain.database.parser.json] - Strain CP063555.1 (Enterococcus faecium, BMT-1-1-10) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,472 [DEBUG - chronostrain.database.parser.json] - Strain CP063549.1 (Enterococcus faecium, ICU-1-2) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,477 [DEBUG - chronostrain.database.parser.json] - Strain CP063537.1 (Enterococcus faecium, BMT-2-2-11) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,479 [DEBUG - chronostrain.database.parser.json] - Strain CP063283.1 (Enterococcus faecium, BMT-5-1-10) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,481 [DEBUG - chronostrain.database.parser.json] - Strain CP094230.1 (Enterococcus faecium, NY13320) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,483 [DEBUG - chronostrain.database.parser.json] - Strain CP093947.1 (Enterococcus faecium, NY11066) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,485 [DEBUG - chronostrain.database.parser.json] - Strain CP093953.1 (Enterococcus faecium, NY11067) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,487 [DEBUG - chronostrain.database.parser.json] - Strain CP093941.1 (Enterococcus faecium, NY2032) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,490 [DEBUG - chronostrain.database.parser.json] - Strain CP092569.1 (Enterococcus faecium, VRE-WC072) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,492 [DEBUG - chronostrain.database.parser.json] - Strain CP076483.1 (Enterococcus faecium, UAMS_EF55) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,494 [DEBUG - chronostrain.database.parser.json] - Strain CP076477.1 (Enterococcus faecium, UAMS_EF57) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,496 [DEBUG - chronostrain.database.parser.json] - Strain CP076471.1 (Enterococcus faecium, UAMS_EF58) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,498 [DEBUG - chronostrain.database.parser.json] - Strain CP091577.1 (Enterococcus faecium, UK040) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,500 [DEBUG - chronostrain.database.parser.json] - Strain CP091301.1 (Enterococcus faecium, UK043) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,502 [DEBUG - chronostrain.database.parser.json] - Strain CP091100.1 (Enterococcus faecium, P47) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,504 [DEBUG - chronostrain.database.parser.json] - Strain CP083179.1 (Enterococcus faecium, H7) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,506 [DEBUG - chronostrain.database.parser.json] - Strain CP089319.1 (Enterococcus faecium, NMVRE-001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,508 [DEBUG - chronostrain.database.parser.json] - Strain CP089092.2 (Enterococcus faecium, V13-21-E11-012-001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,511 [DEBUG - chronostrain.database.parser.json] - Strain CP085906.1 (Enterococcus faecium, SC1762-D) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,513 [DEBUG - chronostrain.database.parser.json] - Strain CP085894.1 (Enterococcus faecium, SC1762) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,515 [DEBUG - chronostrain.database.parser.json] - Strain CP083473.1 (Enterococcus faecium, HJP554) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,517 [DEBUG - chronostrain.database.parser.json] - Strain AP024837.1 (Enterococcus faecium, AA622) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,519 [DEBUG - chronostrain.database.parser.json] - Strain AP024831.1 (Enterococcus faecium, AA620) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,521 [DEBUG - chronostrain.database.parser.json] - Strain CP081504.1 (Enterococcus faecium, E843xGE-1-TC1) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,523 [DEBUG - chronostrain.database.parser.json] - Strain CP074109.1 (Enterococcus faecium, VBR48) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,525 [DEBUG - chronostrain.database.parser.json] - Strain CP074107.1 (Enterococcus faecium, VBO39) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,527 [DEBUG - chronostrain.database.parser.json] - Strain CP074105.1 (Enterococcus faecium, VBO96) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,528 [DEBUG - chronostrain.database.parser.json] - Strain CP079927.1 (Enterococcus faecium, 7150) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,530 [DEBUG - chronostrain.database.parser.json] - Strain CP072894.1 (Enterococcus faecium, AVS0243) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,532 [DEBUG - chronostrain.database.parser.json] - Strain CP072884.1 (Enterococcus faecium, F179) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,534 [DEBUG - chronostrain.database.parser.json] - Strain CP066212.1 (Enterococcus faecium, 18-201) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,536 [DEBUG - chronostrain.database.parser.json] - Strain CP065777.1 (Enterococcus faecium, 16-021) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,538 [DEBUG - chronostrain.database.parser.json] - Strain CP065773.1 (Enterococcus faecium, 16-164) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,540 [DEBUG - chronostrain.database.parser.json] - Strain CP065770.1 (Enterococcus faecium, 17-318) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,542 [DEBUG - chronostrain.database.parser.json] - Strain CP065767.1 (Enterococcus faecium, 17-508) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,544 [DEBUG - chronostrain.database.parser.json] - Strain CP065764.1 (Enterococcus faecium, 18-133) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,546 [DEBUG - chronostrain.database.parser.json] - Strain CP065754.1 (Enterococcus faecium, 18-276) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,548 [DEBUG - chronostrain.database.parser.json] - Strain CP065752.1 (Enterococcus faecium, 18-465) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,550 [DEBUG - chronostrain.database.parser.json] - Strain CP047326.1 (Enterococcus faecium, 116) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,551 [DEBUG - chronostrain.database.parser.json] - Strain CP064406.1 (Enterococcus faecium, PR05720-3) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,553 [DEBUG - chronostrain.database.parser.json] - Strain CP072707.1 (Enterococcus faecium, VB3378) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,556 [DEBUG - chronostrain.database.parser.json] - Strain CP072720.1 (Enterococcus faecium, VB6521) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,558 [DEBUG - chronostrain.database.parser.json] - Strain CP072731.1 (Enterococcus faecium, VB6171) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,560 [DEBUG - chronostrain.database.parser.json] - Strain CP072586.1 (Enterococcus faecium, VB976) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,562 [DEBUG - chronostrain.database.parser.json] - Strain CP072567.1 (Enterococcus faecium, VB3895) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,564 [DEBUG - chronostrain.database.parser.json] - Strain CP072579.1 (Enterococcus faecium, VB12993) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,566 [DEBUG - chronostrain.database.parser.json] - Strain CP071958.1 (Enterococcus faecium, VB13828) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,568 [DEBUG - chronostrain.database.parser.json] - Strain CP071931.1 (Enterococcus faecium, VRE3355) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,570 [DEBUG - chronostrain.database.parser.json] - Strain CP071271.1 (Enterococcus faecium, AUSMDU00011555) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,572 [DEBUG - chronostrain.database.parser.json] - Strain CP046166.1 (Enterococcus faecium, WGS1811-4-7) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,574 [DEBUG - chronostrain.database.parser.json] - Strain CP068244.1 (Enterococcus faecium, fac90) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,576 [DEBUG - chronostrain.database.parser.json] - Strain CP066667.1 (Enterococcus faecium, Dallas 158) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,577 [DEBUG - chronostrain.database.parser.json] - Strain CP066653.1 (Enterococcus faecium, Dallas 154_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,579 [DEBUG - chronostrain.database.parser.json] - Strain CP066659.1 (Enterococcus faecium, Dallas 155) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,582 [DEBUG - chronostrain.database.parser.json] - Strain CP066622.1 (Enterococcus faecium, Dallas 131_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,583 [DEBUG - chronostrain.database.parser.json] - Strain CP066629.1 (Enterococcus faecium, Dallas 137) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,585 [DEBUG - chronostrain.database.parser.json] - Strain CP066645.1 (Enterococcus faecium, Dallas 148) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,587 [DEBUG - chronostrain.database.parser.json] - Strain CP066598.1 (Enterococcus faecium, Dallas 111) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,589 [DEBUG - chronostrain.database.parser.json] - Strain CP066591.1 (Enterococcus faecium, Dallas 107_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,591 [DEBUG - chronostrain.database.parser.json] - Strain CP066583.1 (Enterococcus faecium, Dallas 103) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,594 [DEBUG - chronostrain.database.parser.json] - Strain CP066473.1 (Enterococcus faecium, Dallas 93_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,596 [DEBUG - chronostrain.database.parser.json] - Strain CP066480.1 (Enterococcus faecium, Dallas 93_3) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,598 [DEBUG - chronostrain.database.parser.json] - Strain CP066427.1 (Enterococcus faecium, Dallas 57) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,599 [DEBUG - chronostrain.database.parser.json] - Strain CP066409.1 (Enterococcus faecium, Dallas 53_2) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,601 [DEBUG - chronostrain.database.parser.json] - Strain CP066443.1 (Enterococcus faecium, Dallas 71_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,603 [DEBUG - chronostrain.database.parser.json] - Strain CP066457.1 (Enterococcus faecium, Dallas 87_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,605 [DEBUG - chronostrain.database.parser.json] - Strain CP066381.1 (Enterococcus faecium, Dallas 16_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,606 [DEBUG - chronostrain.database.parser.json] - Strain CP066389.1 (Enterococcus faecium, Dallas 17_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,608 [DEBUG - chronostrain.database.parser.json] - Strain CP066451.1 (Enterococcus faecium, Dallas 83) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,610 [DEBUG - chronostrain.database.parser.json] - Strain CP066421.1 (Enterococcus faecium, Dallas 55) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,612 [DEBUG - chronostrain.database.parser.json] - Strain CP066415.1 (Enterococcus faecium, Dallas 53_1) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,613 [DEBUG - chronostrain.database.parser.json] - Strain CP066466.1 (Enterococcus faecium, Dallas 91) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,615 [DEBUG - chronostrain.database.parser.json] - Strain CP065547.1 (Enterococcus faecium, Dallas 1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,617 [DEBUG - chronostrain.database.parser.json] - Strain CP065523.1 (Enterococcus faecium, VRE3389) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,618 [DEBUG - chronostrain.database.parser.json] - Strain CP065210.1 (Enterococcus faecium, KCCM 12118) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,620 [DEBUG - chronostrain.database.parser.json] - Strain CP064343.1 (Enterococcus faecium, VRE3363) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,622 [DEBUG - chronostrain.database.parser.json] - Strain CP064049.1 (Enterococcus faecium, VRE3370) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,623 [DEBUG - chronostrain.database.parser.json] - Strain CP062265.1 (Enterococcus faecium, VB3338) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,625 [DEBUG - chronostrain.database.parser.json] - Strain CP060861.1 (Enterococcus faecium, AML0157) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,626 [DEBUG - chronostrain.database.parser.json] - Strain CP053704.1 (Enterococcus faecium, FS 86) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,628 [DEBUG - chronostrain.database.parser.json] - Strain CP059755.1 (Enterococcus faecium, A10290) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,630 [DEBUG - chronostrain.database.parser.json] - Strain CP059823.1 (Enterococcus faecium, A15023) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,631 [DEBUG - chronostrain.database.parser.json] - Strain CP059764.1 (Enterococcus faecium, A11051) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,633 [DEBUG - chronostrain.database.parser.json] - Strain CP059772.1 (Enterococcus faecium, A3895) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,635 [DEBUG - chronostrain.database.parser.json] - Strain CP059739.1 (Enterococcus faecium, A4694) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,637 [DEBUG - chronostrain.database.parser.json] - Strain CP061817.1 (Enterococcus faecium, A6521) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,638 [DEBUG - chronostrain.database.parser.json] - Strain CP059747.1 (Enterococcus faecium, A7214) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,640 [DEBUG - chronostrain.database.parser.json] - Strain CP059816.1 (Enterococcus faecium, BP657) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,642 [DEBUG - chronostrain.database.parser.json] - Strain CP059793.1 (Enterococcus faecium, BP3378) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,644 [DEBUG - chronostrain.database.parser.json] - Strain CP059806.1 (Enterococcus faecium, BP5067) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,645 [DEBUG - chronostrain.database.parser.json] - Strain CP059784.1 (Enterococcus faecium, BA17124) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,647 [DEBUG - chronostrain.database.parser.json] - Strain CP050650.1 (Enterococcus faecium, BIOPOP-3 WT) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,648 [DEBUG - chronostrain.database.parser.json] - Strain CP050648.1 (Enterococcus faecium, BIOPOP-3 ALE) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,649 [DEBUG - chronostrain.database.parser.json] - Strain CP050255.1 (Enterococcus faecium, DT1-1) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,651 [DEBUG - chronostrain.database.parser.json] - Strain CP039729.1 (Enterococcus faecium, ZY2) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,653 [DEBUG - chronostrain.database.parser.json] - Strain AP022341.1 (Enterococcus faecium, KUHS13) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,654 [DEBUG - chronostrain.database.parser.json] - Strain CP038995.1 (Enterococcus faecium, ZY11) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,656 [DEBUG - chronostrain.database.parser.json] - Strain CP038996.1 (Enterococcus faecium, SRR24) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,658 [DEBUG - chronostrain.database.parser.json] - Strain CP046077.1 (Enterococcus faecium, VRE) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,659 [DEBUG - chronostrain.database.parser.json] - Strain CP045012.1 (Enterococcus faecium, LAC7.2) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,661 [DEBUG - chronostrain.database.parser.json] - Strain CP044274.1 (Enterococcus faecium, V2937) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,663 [DEBUG - chronostrain.database.parser.json] - Strain CP044264.1 (Enterococcus faecium, V1836) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,665 [DEBUG - chronostrain.database.parser.json] - Strain CP038174.1 (Enterococcus faecium, SDGJQ5) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,666 [DEBUG - chronostrain.database.parser.json] - Strain CP038163.1 (Enterococcus faecium, GZ8) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,668 [DEBUG - chronostrain.database.parser.json] - Strain CP038164.1 (Enterococcus faecium, HB2-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,670 [DEBUG - chronostrain.database.parser.json] - Strain CP038177.1 (Enterococcus faecium, HN11) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,671 [DEBUG - chronostrain.database.parser.json] - Strain CP038170.1 (Enterococcus faecium, SDGJP3) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,673 [DEBUG - chronostrain.database.parser.json] - Strain CP038172.1 (Enterococcus faecium, YN2-1) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,675 [DEBUG - chronostrain.database.parser.json] - Strain CP041270.2 (Enterococcus faecium, VVEswe-S) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,677 [DEBUG - chronostrain.database.parser.json] - Strain CP041261.3 (Enterococcus faecium, VVEswe-R) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,679 [DEBUG - chronostrain.database.parser.json] - Strain AP019394.1 (Enterococcus faecium, QU 50) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,680 [DEBUG - chronostrain.database.parser.json] - Strain CP041257.1 (Enterococcus faecium, 515) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,682 [DEBUG - chronostrain.database.parser.json] - Strain CP040904.1 (Enterococcus faecium, N56454) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,684 [DEBUG - chronostrain.database.parser.json] - Strain CP040907.1 (Enterococcus faecium, FB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,685 [DEBUG - chronostrain.database.parser.json] - Strain CP040875.1 (Enterococcus faecium, DB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,687 [DEBUG - chronostrain.database.parser.json] - Strain CP040849.1 (Enterococcus faecium, F17E0263) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,688 [DEBUG - chronostrain.database.parser.json] - Strain CP040740.1 (Enterococcus faecium, VRE1) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,690 [DEBUG - chronostrain.database.parser.json] - Strain CP040706.1 (Enterococcus faecium, HOU503) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,692 [DEBUG - chronostrain.database.parser.json] - Strain CP035666.1 (Enterococcus faecium, UAMSEF_20) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,694 [DEBUG - chronostrain.database.parser.json] - Strain CP035660.1 (Enterococcus faecium, UAMSEF_09) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,696 [DEBUG - chronostrain.database.parser.json] - Strain CP035654.1 (Enterococcus faecium, UAMSEF_08) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,698 [DEBUG - chronostrain.database.parser.json] - Strain CP035648.1 (Enterococcus faecium, UAMSEF_01) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,700 [DEBUG - chronostrain.database.parser.json] - Strain CP040368.1 (Enterococcus faecium, VB3240) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,702 [DEBUG - chronostrain.database.parser.json] - Strain CP040236.1 (Enterococcus faecium, VB3025) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,704 [DEBUG - chronostrain.database.parser.json] - Strain CP028727.1 (Enterococcus faecium, FSIS1608820) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,706 [DEBUG - chronostrain.database.parser.json] - Strain CP035220.1 (Enterococcus faecium, SRCM103470) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,708 [DEBUG - chronostrain.database.parser.json] - Strain CP033206.1 (Enterococcus faecium, RBWH1) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,709 [DEBUG - chronostrain.database.parser.json] - Strain CP030110.1 (Enterococcus faecium, BM4105-RF) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,711 [DEBUG - chronostrain.database.parser.json] - Strain CP020488.1 (Enterococcus faecium, CFSAN059071) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,713 [DEBUG - chronostrain.database.parser.json] - Strain CP020484.1 (Enterococcus faecium, CFSAN059070) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,715 [DEBUG - chronostrain.database.parser.json] - Strain CP027501.1 (Enterococcus faecium, AUSMDU00004142) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,717 [DEBUG - chronostrain.database.parser.json] - Strain CP027517.1 (Enterococcus faecium, AUSMDU00004024) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,719 [DEBUG - chronostrain.database.parser.json] - Strain CP027512.1 (Enterococcus faecium, AUSMDU00004028) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,721 [DEBUG - chronostrain.database.parser.json] - Strain CP027506.1 (Enterococcus faecium, AUSMDU00004055) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,723 [DEBUG - chronostrain.database.parser.json] - Strain CP027497.1 (Enterococcus faecium, AUSMDU00004167) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,725 [DEBUG - chronostrain.database.parser.json] - Strain CP027402.1 (Enterococcus faecium, FDAARGOS_323) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,727 [DEBUG - chronostrain.database.parser.json] - Strain CP044316.1 (Enterococcus faecium, 15-307-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,729 [DEBUG - chronostrain.database.parser.json] - Strain CP025754.1 (Enterococcus faecium, AALTL) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,731 [DEBUG - chronostrain.database.parser.json] - Strain CP021885.1 (Enterococcus faecium, WEFA23) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,733 [DEBUG - chronostrain.database.parser.json] - Strain CP023808.1 (Enterococcus faecium, Efaecium_ER04526.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,735 [DEBUG - chronostrain.database.parser.json] - Strain CP023799.1 (Enterococcus faecium, Efaecium_ER04526.5A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,737 [DEBUG - chronostrain.database.parser.json] - Strain CP023804.1 (Enterococcus faecium, Efaecium_ER04619.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,739 [DEBUG - chronostrain.database.parser.json] - Strain CP023789.1 (Enterococcus faecium, Efaecium_ER04462.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,741 [DEBUG - chronostrain.database.parser.json] - Strain CP023794.1 (Enterococcus faecium, Efaecium_ER04484.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,743 [DEBUG - chronostrain.database.parser.json] - Strain CP023784.1 (Enterococcus faecium, Efaecium_ER04120.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,745 [DEBUG - chronostrain.database.parser.json] - Strain CP023780.1 (Enterococcus faecium, Efaecium_ER03933.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,747 [DEBUG - chronostrain.database.parser.json] - Strain CP025425.1 (Enterococcus faecium, SC4) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,749 [DEBUG - chronostrain.database.parser.json] - Strain CP017787.1 (Enterococcus faecium, E232) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,751 [DEBUG - chronostrain.database.parser.json] - Strain CP021849.1 (Enterococcus faecium, 16-346) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,753 [DEBUG - chronostrain.database.parser.json] - Strain CP017797.1 (Enterococcus faecium, E243) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,755 [DEBUG - chronostrain.database.parser.json] - Strain CP017792.1 (Enterococcus faecium, E240) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,758 [DEBUG - chronostrain.database.parser.json] - Strain CP018128.1 (Enterococcus faecium, A_020709_82) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,760 [DEBUG - chronostrain.database.parser.json] - Strain CP023423.1 (Enterococcus faecium, K60-39) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,762 [DEBUG - chronostrain.database.parser.json] - Strain CP019988.1 (Enterococcus faecium, 2014-VREF-63) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,764 [DEBUG - chronostrain.database.parser.json] - Strain CP019992.1 (Enterococcus faecium, 2014-VREF-268) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,766 [DEBUG - chronostrain.database.parser.json] - Strain CP019970.1 (Enterococcus faecium, 2014-VREF-114) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,768 [DEBUG - chronostrain.database.parser.json] - Strain CP019208.1 (Enterococcus faecium, 2014-VREF-41) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,770 [DEBUG - chronostrain.database.parser.json] - Strain CP018830.1 (Enterococcus faecium, ISMMS_VRE_9) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,772 [DEBUG - chronostrain.database.parser.json] - Strain CP018828.1 (Enterococcus faecium, ISMMS_VRE_12) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,774 [DEBUG - chronostrain.database.parser.json] - Strain CP018071.1 (Enterococcus faecium, VRE001) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,776 [DEBUG - chronostrain.database.parser.json] - Strain CP018065.1 (Enterococcus faecium, E1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,778 [DEBUG - chronostrain.database.parser.json] - Strain CP014529.1 (Enterococcus faecium, E745) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,780 [DEBUG - chronostrain.database.parser.json] - Strain CP012460.1 (Enterococcus faecium, ISMMS_VRE_7) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,782 [DEBUG - chronostrain.database.parser.json] - Strain CP011281.1 (Enterococcus faecium, E39) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,784 [DEBUG - chronostrain.database.parser.json] - Strain CP014449.1 (Enterococcus faecium, ATCC 700221) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,786 [DEBUG - chronostrain.database.parser.json] - Strain CP011828.1 (Enterococcus faecium, UW8175) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,788 [DEBUG - chronostrain.database.parser.json] - Strain CP013994.1 (Enterococcus faecium, 6E6) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,790 [DEBUG - chronostrain.database.parser.json] - Strain CP013009.1 (Enterococcus faecium, UW7606x64/3 TC1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,792 [DEBUG - chronostrain.database.parser.json] - Strain CP012522.1 (Enterococcus faecium, 64/3) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,794 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU016036.1 (Enterococcus faecium, USZ_VRE53_P46) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,796 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU016027.1 (Enterococcus faecium, USZ_VRE67_P60) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,798 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015946.1 (Enterococcus faecium, USZ_VRE5_P5) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,800 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015708.1 (Enterococcus faecium, USZ_VRE32_P32) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,802 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015350.1 (Enterococcus faecium, AUS2002) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,804 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OU015345.1 (Enterococcus faecium, AUS2001) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,806 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR215968.1 (Enterococcus faecium, 3012STDY6244127) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,808 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135482.1 (Enterococcus faecium, E4456) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,810 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135475.1 (Enterococcus faecium, E8423) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,812 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135488.1 (Enterococcus faecium, E8414) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,814 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135474.1 (Enterococcus faecium, E9101) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,816 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135443.1 (Enterococcus faecium, E7114) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,818 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135435.1 (Enterococcus faecium, E8691) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,820 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135419.1 (Enterococcus faecium, E8440) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,822 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135428.1 (Enterococcus faecium, E8927) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,824 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135408.1 (Enterococcus faecium, E8284) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,830 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135414.1 (Enterococcus faecium, E8328) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135394.1 (Enterococcus faecium, E8290) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,834 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135401.1 (Enterococcus faecium, E8377) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,836 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135357.1 (Enterococcus faecium, E7948) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,838 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135351.1 (Enterococcus faecium, E8014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,840 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135364.1 (Enterococcus faecium, E8195) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,842 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135372.1 (Enterococcus faecium, E8172) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,844 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135384.1 (Enterococcus faecium, E7933) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,846 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135344.1 (Enterococcus faecium, E8202) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,848 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135317.1 (Enterococcus faecium, E7663) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,850 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135324.1 (Enterococcus faecium, E7654) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,852 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135339.1 (Enterococcus faecium, E7356) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,854 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135307.1 (Enterococcus faecium, E7240) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,856 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135297.1 (Enterococcus faecium, E7429) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,857 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135293.1 (Enterococcus faecium, E7237) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,859 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135254.1 (Enterococcus faecium, E7098) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,860 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135243.1 (Enterococcus faecium, E6988) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,862 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135258.1 (Enterococcus faecium, E4457) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,864 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135278.1 (Enterococcus faecium, E6975) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,865 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135219.1 (Enterococcus faecium, E7040) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,867 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135235.1 (Enterococcus faecium, E7067) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,868 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135191.1 (Enterococcus faecium, E4438) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,870 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135179.1 (Enterococcus faecium, E0595) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,872 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135174.1 (Enterococcus faecium, E4402) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,873 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135169.1 (Enterococcus faecium, E2079) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,875 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134337.1 (Enterococcus faecium, NCTC7174) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,877 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134105.1 (Enterococcus faecium, E6043) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,878 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134095.1 (Enterococcus faecium, E1334) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,880 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR132067.1 (Enterococcus faecium, E0139) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,882 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT603678.1 (Enterococcus faecium, Ef_DMG1500501) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,883 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LT598663.1 (Enterococcus faecium, Ef_aus00233) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,885 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN999844.1 (Enterococcus faecium, EFE10021) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,886 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN999987.1 (Enterococcus faecium, EFE11651) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,888 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP132226.1 (Enterococcus faecium, TC64_3x4995_20_TV8) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,890 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130914.1 (Enterococcus faecium, C0019EM0006) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,892 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130912.1 (Enterococcus faecium, C0019EM0002) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,893 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130908.1 (Enterococcus faecium, B0019EM0041) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,895 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130910.1 (Enterococcus faecium, B0019EM0045) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,897 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130906.1 (Enterococcus faecium, B0019EM0038) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,898 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130883.1 (Enterococcus faecium, B0019EM0034) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,900 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130879.1 (Enterococcus faecium, A0019EM0007) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,902 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130875.1 (Enterococcus faecium, A0019EM0010) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,903 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130877.1 (Enterococcus faecium, A0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,905 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130881.1 (Enterococcus faecium, C0019EM0016) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,906 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130882.1 (Enterococcus faecium, B0019EM0040) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,908 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130849.1 (Enterococcus faecium, B0019EM0016) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,910 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130833.1 (Enterococcus faecium, B0019EM0032) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,911 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130841.1 (Enterococcus faecium, B0019EM0023) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,913 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130837.1 (Enterococcus faecium, B0019EM0028) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,915 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130843.1 (Enterococcus faecium, B0019EM0022) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,917 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130835.1 (Enterococcus faecium, B0019EM0030) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,918 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130839.1 (Enterococcus faecium, B0019EM0026) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,920 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130845.1 (Enterococcus faecium, B0019EM0019) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,925 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130847.1 (Enterococcus faecium, B0019EM0017) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,927 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130851.1 (Enterococcus faecium, B0019EM0015) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,929 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130861.1 (Enterococcus faecium, B0019EM0012) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,930 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130859.1 (Enterococcus faecium, B0019EM0014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,932 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130865.1 (Enterococcus faecium, B0019EM0004) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,934 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130863.1 (Enterococcus faecium, B0019EM0006) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,935 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130867.1 (Enterococcus faecium, A0019EM0035) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,937 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130869.1 (Enterococcus faecium, A0019EM0022) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,939 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130873.1 (Enterococcus faecium, A0019EM0020) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,940 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130871.1 (Enterococcus faecium, A0019EM0021) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,942 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130827.1 (Enterococcus faecium, C0019EM0008) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,944 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130825.1 (Enterococcus faecium, C0019EM0014) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,946 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP130829.1 (Enterococcus faecium, C0019EM0007) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,947 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP128188.1 (Enterococcus faecium, A2) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,949 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP027294.1 (Enterococcus faecium, K6D) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,951 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109758.1 (Enterococcus faecium, XJ78NG) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,953 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP119150.1 (Enterococcus faecium, GENOMIC22-005) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,954 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118955.1 (Enterococcus faecium, DSM 20477) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,956 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118548.1 (Enterococcus faecium, W6) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,957 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116528.1 (Enterococcus faecium, K188-5) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,959 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116523.1 (Enterococcus faecium, K69-1a) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,961 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116520.1 (Enterococcus faecium, R186-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,962 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116517.1 (Enterococcus faecium, R39-1) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,964 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116532.1 (Enterococcus faecium, K191-3) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,965 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116514.1 (Enterococcus faecium, K195-6b) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,967 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116023.1 (Enterococcus faecium, CQFYH22-006) loaded with 9 markers.\n",
      "2024-02-25 13:21:24,968 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP115812.1 (Enterococcus faecium, DMEA09) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,970 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP114617.1 (Enterococcus faecium, EF0656) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,972 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP112862.1 (Enterococcus faecium, DVT1681) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,973 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026666.1 (Enterococcus faecium, GK961) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,975 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026660.1 (Enterococcus faecium, GK941) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,978 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026655.1 (Enterococcus faecium, GK923) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,980 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026604.1 (Enterococcus faecium, AA55) loaded with 12 markers.\n",
      "2024-02-25 13:21:24,982 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026601.1 (Enterococcus faecium, JHP80) loaded with 14 markers.\n",
      "2024-02-25 13:21:24,984 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026566.1 (Enterococcus faecium, JHP9) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,986 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026772.1 (Enterococcus faecium, NUITM-VRE1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,988 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109798.1 (Enterococcus faecium, TF51-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,990 [DEBUG - chronostrain.database.parser.json] - Strain NZ_JAOXXZ010000001.1 (Enterococcus faecium, IC25) loaded with 11 markers.\n",
      "2024-02-25 13:21:24,992 [DEBUG - chronostrain.database.parser.json] - Strain NZ_JAOXXY010000002.1 (Enterococcus faecium, BTF71-1) loaded with 13 markers.\n",
      "2024-02-25 13:21:24,994 [DEBUG - chronostrain.database.parser.json] - Strain NZ_JAOXXX010000002.1 (Enterococcus faecium, ATF64) loaded with 10 markers.\n",
      "2024-02-25 13:21:24,996 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102534.1 (Enterococcus faecium, ENCfa-68) loaded with 8 markers.\n",
      "2024-02-25 13:21:24,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058891.1 (Enterococcus faecium, ef332) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,000 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091224.1 (Enterococcus faecium, 237) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,002 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091217.1 (Enterococcus faecium, 1525) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,004 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091213.1 (Enterococcus faecium, 264a) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,006 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091210.1 (Enterococcus faecium, 211b) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,008 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091206.1 (Enterococcus faecium, 1818) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,010 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088191.1 (Enterococcus faecium, S2-6b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,012 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097022.1 (Enterococcus faecium, AT45b) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,014 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097014.1 (Enterococcus faecium, AT47b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,016 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP096219.1 (Enterococcus faecium, UCH5) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,018 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP096211.1 (Enterococcus faecium, UCH1) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,020 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110360.1 (Enterococcus faecium, AKSZ-142) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,022 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110294.1 (Enterococcus faecium, AKSZ-47) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,024 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP094230.1 (Enterococcus faecium, NY13320) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,026 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093947.1 (Enterococcus faecium, NY11066) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,028 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093953.1 (Enterococcus faecium, NY11067) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,030 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093941.1 (Enterococcus faecium, NY2032) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,031 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP092569.1 (Enterococcus faecium, VRE-WC072) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,033 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076483.1 (Enterococcus faecium, UAMS_EF55) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,035 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076477.1 (Enterococcus faecium, UAMS_EF57) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,037 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP076471.1 (Enterococcus faecium, UAMS_EF58) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,039 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091577.1 (Enterococcus faecium, UK040) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,041 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091301.1 (Enterococcus faecium, UK043) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,043 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091100.1 (Enterococcus faecium, P47) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,045 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083179.1 (Enterococcus faecium, H7) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,047 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089319.1 (Enterococcus faecium, NMVRE-001) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP089092.2 (Enterococcus faecium, V13-21-E11-012-001) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,051 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085906.1 (Enterococcus faecium, SC1762-D) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,053 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP085894.1 (Enterococcus faecium, SC1762) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,055 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083473.1 (Enterococcus faecium, HJP554) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,057 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024837.1 (Enterococcus faecium, AA622) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,060 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP024831.1 (Enterococcus faecium, AA620) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,062 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074109.1 (Enterococcus faecium, VBR48) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,064 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074107.1 (Enterococcus faecium, VBO39) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,066 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074105.1 (Enterococcus faecium, VBO96) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,067 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079927.1 (Enterococcus faecium, 7150) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,069 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072894.1 (Enterococcus faecium, AVS0243) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,071 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072884.1 (Enterococcus faecium, F179) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,073 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065777.1 (Enterococcus faecium, 16-021) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,075 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065773.1 (Enterococcus faecium, 16-164) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,080 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065770.1 (Enterococcus faecium, 17-318) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,083 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065767.1 (Enterococcus faecium, 17-508) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,084 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065754.1 (Enterococcus faecium, 18-276) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,086 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065752.1 (Enterococcus faecium, 18-465) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,088 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP047326.1 (Enterococcus faecium, 116) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,090 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064406.1 (Enterococcus faecium, PR05720-3) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,092 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072707.1 (Enterococcus faecium, VB3378) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,094 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072720.1 (Enterococcus faecium, VB6521) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,096 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072731.1 (Enterococcus faecium, VB6171) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,098 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072586.1 (Enterococcus faecium, VB976) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,101 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072567.1 (Enterococcus faecium, VB3895) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,103 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072579.1 (Enterococcus faecium, VB12993) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,104 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071958.1 (Enterococcus faecium, VB13828) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,106 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071931.1 (Enterococcus faecium, VRE3355) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,109 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046166.1 (Enterococcus faecium, WGS1811-4-7) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,110 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068244.1 (Enterococcus faecium, fac90) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,112 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066667.1 (Enterococcus faecium, Dallas 158) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,114 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066653.1 (Enterococcus faecium, Dallas 154_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,116 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066659.1 (Enterococcus faecium, Dallas 155) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,118 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066622.1 (Enterococcus faecium, Dallas 131_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,120 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066629.1 (Enterococcus faecium, Dallas 137) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,122 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066645.1 (Enterococcus faecium, Dallas 148) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,124 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066598.1 (Enterococcus faecium, Dallas 111) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,126 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066591.1 (Enterococcus faecium, Dallas 107_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,128 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066583.1 (Enterococcus faecium, Dallas 103) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,130 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066473.1 (Enterococcus faecium, Dallas 93_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,133 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066480.1 (Enterococcus faecium, Dallas 93_3) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,135 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066427.1 (Enterococcus faecium, Dallas 57) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,137 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066409.1 (Enterococcus faecium, Dallas 53_2) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,139 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066443.1 (Enterococcus faecium, Dallas 71_2) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,141 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066457.1 (Enterococcus faecium, Dallas 87_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,143 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066381.1 (Enterococcus faecium, Dallas 16_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,145 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066389.1 (Enterococcus faecium, Dallas 17_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,147 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066451.1 (Enterococcus faecium, Dallas 83) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,149 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066421.1 (Enterococcus faecium, Dallas 55) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,151 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066415.1 (Enterococcus faecium, Dallas 53_1) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,154 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066466.1 (Enterococcus faecium, Dallas 91) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,156 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065547.1 (Enterococcus faecium, Dallas 1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,158 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065523.1 (Enterococcus faecium, VRE3389) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,160 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065210.1 (Enterococcus faecium, KCCM 12118) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,162 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064343.1 (Enterococcus faecium, VRE3363) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,164 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064049.1 (Enterococcus faecium, VRE3370) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,166 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP062265.1 (Enterococcus faecium, VB3338) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,168 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060861.1 (Enterococcus faecium, AML0157) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,169 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053704.1 (Enterococcus faecium, FS 86) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,171 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059755.1 (Enterococcus faecium, A10290) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,173 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059823.1 (Enterococcus faecium, A15023) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,175 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059764.1 (Enterococcus faecium, A11051) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,177 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059772.1 (Enterococcus faecium, A3895) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,179 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059739.1 (Enterococcus faecium, A4694) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,181 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP061817.1 (Enterococcus faecium, A6521) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,184 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059747.1 (Enterococcus faecium, A7214) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,186 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059816.1 (Enterococcus faecium, BP657) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,187 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059793.1 (Enterococcus faecium, BP3378) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,189 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059806.1 (Enterococcus faecium, BP5067) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,191 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP059784.1 (Enterococcus faecium, BA17124) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,192 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050650.1 (Enterococcus faecium, BIOPOP-3 WT) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,193 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050648.1 (Enterococcus faecium, BIOPOP-3 ALE) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,195 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP050255.1 (Enterococcus faecium, DT1-1) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,196 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039729.1 (Enterococcus faecium, ZY2) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,198 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022341.1 (Enterococcus faecium, KUHS13) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,200 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038995.1 (Enterococcus faecium, ZY11) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,202 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046077.1 (Enterococcus faecium, VRE) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,203 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045012.1 (Enterococcus faecium, LAC7.2) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,205 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044274.1 (Enterococcus faecium, V2937) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,206 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044264.1 (Enterococcus faecium, V1836) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,208 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038174.1 (Enterococcus faecium, SDGJQ5) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,210 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038163.1 (Enterococcus faecium, GZ8) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,211 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038164.1 (Enterococcus faecium, HB2-2) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,213 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038177.1 (Enterococcus faecium, HN11) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,215 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038170.1 (Enterococcus faecium, SDGJP3) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,217 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038172.1 (Enterococcus faecium, YN2-1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,218 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041270.2 (Enterococcus faecium, VVEswe-S) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,220 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041261.2 (Enterococcus faecium, VVEswe-R) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,222 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019394.1 (Enterococcus faecium, QU 50) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041257.1 (Enterococcus faecium, 515) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,225 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040904.1 (Enterococcus faecium, N56454) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,227 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040907.1 (Enterococcus faecium, FB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,228 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040875.1 (Enterococcus faecium, DB-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,230 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040849.1 (Enterococcus faecium, F17E0263) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,231 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040740.1 (Enterococcus faecium, VRE1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,233 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040706.1 (Enterococcus faecium, HOU503) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,235 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035666.1 (Enterococcus faecium, UAMSEF_20) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,237 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035660.1 (Enterococcus faecium, UAMSEF_09) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,239 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035654.1 (Enterococcus faecium, UAMSEF_08) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,240 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035648.1 (Enterococcus faecium, UAMSEF_01) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,242 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040368.1 (Enterococcus faecium, VB3240) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,244 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040236.1 (Enterococcus faecium, VB3025) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,245 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP028727.1 (Enterococcus faecium, FSIS1608820) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035220.1 (Enterococcus faecium, SRCM103470) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,249 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033206.1 (Enterococcus faecium, RBWH1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,250 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030110.1 (Enterococcus faecium, BM4105-RF) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,252 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020488.1 (Enterococcus faecium, CFSAN059071) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,253 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020484.1 (Enterococcus faecium, CFSAN059070) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,255 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027501.1 (Enterococcus faecium, AUSMDU00004142) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,257 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027517.1 (Enterococcus faecium, AUSMDU00004024) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,258 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027512.1 (Enterococcus faecium, AUSMDU00004028) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,260 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027506.1 (Enterococcus faecium, AUSMDU00004055) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,262 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027497.1 (Enterococcus faecium, AUSMDU00004167) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,263 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027402.1 (Enterococcus faecium, FDAARGOS_323) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,265 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP044316.1 (Enterococcus faecium, 15-307-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,267 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025754.1 (Enterococcus faecium, AALTL) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,268 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021885.1 (Enterococcus faecium, WEFA23) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,270 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023808.1 (Enterococcus faecium, Efaecium_ER04526.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,272 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023799.1 (Enterococcus faecium, Efaecium_ER04526.5A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,273 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023804.1 (Enterococcus faecium, Efaecium_ER04619.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,275 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023789.1 (Enterococcus faecium, Efaecium_ER04462.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,277 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023794.1 (Enterococcus faecium, Efaecium_ER04484.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,278 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023784.1 (Enterococcus faecium, Efaecium_ER04120.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,280 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023780.1 (Enterococcus faecium, Efaecium_ER03933.3A) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,282 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025425.1 (Enterococcus faecium, SC4) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,284 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017787.1 (Enterococcus faecium, E232) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,285 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021849.1 (Enterococcus faecium, 16-346) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,287 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017797.1 (Enterococcus faecium, E243) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,289 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017792.1 (Enterococcus faecium, E240) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,290 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018128.1 (Enterococcus faecium, A_020709_82) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,292 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP023423.1 (Enterococcus faecium, K60-39) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,294 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019988.1 (Enterococcus faecium, 2014-VREF-63) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,296 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019992.1 (Enterococcus faecium, 2014-VREF-268) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,297 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019970.1 (Enterococcus faecium, 2014-VREF-114) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,299 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019208.1 (Enterococcus faecium, 2014-VREF-41) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,301 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018830.1 (Enterococcus faecium, ISMMS_VRE_9) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,303 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018828.1 (Enterococcus faecium, ISMMS_VRE_12) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,304 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018071.1 (Enterococcus faecium, VRE001) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,306 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP018065.1 (Enterococcus faecium, E1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,309 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014529.1 (Enterococcus faecium, E745) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,311 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012460.1 (Enterococcus faecium, ISMMS_VRE_7) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,312 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011281.1 (Enterococcus faecium, E39) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,314 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP014449.1 (Enterococcus faecium, ATCC 700221) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,316 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP011828.1 (Enterococcus faecium, UW8175) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,317 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013994.1 (Enterococcus faecium, 6E6) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,319 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP013009.1 (Enterococcus faecium, UW7606x64/3 TC1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,321 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012522.1 (Enterococcus faecium, 64/3) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,323 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038996.1 (Enterococcus faecium, SRR24) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,325 [DEBUG - chronostrain.database.parser.json] - Strain CP042289.1 (Enterococcus hirae, HDC14-2) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,327 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042289.1 (Enterococcus hirae, HDC14-2) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,328 [DEBUG - chronostrain.database.parser.json] - Strain CP025473.1 (Enterococcus mundtii, Pe161) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,330 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025473.1 (Enterococcus mundtii, Pe161) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,331 [DEBUG - chronostrain.database.parser.json] - Strain CP063766.1 (Enterococcus lactis, IDCC 2105) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,333 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063766.1 (Enterococcus lactis, IDCC 2105) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,335 [DEBUG - chronostrain.database.parser.json] - Strain CP006030.1 (Enterococcus faecium, T110) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,336 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP006030.1 (Enterococcus faecium, T110) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,338 [DEBUG - chronostrain.database.parser.json] - Strain CP003583.1 (Enterococcus faecium, DO) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,339 [DEBUG - chronostrain.database.parser.json] - Strain NC_017960.1 (Enterococcus faecium, DO) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,341 [DEBUG - chronostrain.database.parser.json] - Strain LR135782.1 (Enterococcus faecium, E4239) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,343 [DEBUG - chronostrain.database.parser.json] - Strain LR135331.1 (Enterococcus faecium, E7471) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,344 [DEBUG - chronostrain.database.parser.json] - Strain LR135287.1 (Enterococcus faecium, E7199) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,346 [DEBUG - chronostrain.database.parser.json] - Strain LR135203.1 (Enterococcus faecium, E7171) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,347 [DEBUG - chronostrain.database.parser.json] - Strain LR135226.1 (Enterococcus faecium, E7025) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,349 [DEBUG - chronostrain.database.parser.json] - Strain LR135197.1 (Enterococcus faecium, E6055) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,351 [DEBUG - chronostrain.database.parser.json] - Strain LR135181.1 (Enterococcus faecium, E1774) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,352 [DEBUG - chronostrain.database.parser.json] - Strain LR135170.1 (Enterococcus faecium, E4227) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,354 [DEBUG - chronostrain.database.parser.json] - Strain CP116549.1 (Enterococcus faecium, K80-15b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,355 [DEBUG - chronostrain.database.parser.json] - Strain CP116546.1 (Enterococcus faecium, K189-3) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,357 [DEBUG - chronostrain.database.parser.json] - Strain CP116543.1 (Enterococcus faecium, K70-1a) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,359 [DEBUG - chronostrain.database.parser.json] - Strain CP116540.1 (Enterococcus faecium, K75-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,360 [DEBUG - chronostrain.database.parser.json] - Strain CP116536.1 (Enterococcus faecium, K162-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,362 [DEBUG - chronostrain.database.parser.json] - Strain AP026612.1 (Enterococcus faecium, AA94) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,364 [DEBUG - chronostrain.database.parser.json] - Strain CP109839.1 (Enterococcus faecium, T17-1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,365 [DEBUG - chronostrain.database.parser.json] - Strain CP088195.1 (Enterococcus faecium, S39-7a) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,367 [DEBUG - chronostrain.database.parser.json] - Strain CP097061.1 (Enterococcus faecium, AT02) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,369 [DEBUG - chronostrain.database.parser.json] - Strain CP097027.1 (Enterococcus faecium, AT44) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,371 [DEBUG - chronostrain.database.parser.json] - Strain CP063639.1 (Enterococcus faecium, ICU-2-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,372 [DEBUG - chronostrain.database.parser.json] - Strain CP063579.1 (Enterococcus faecium, BMT-4-1-11) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,374 [DEBUG - chronostrain.database.parser.json] - Strain CP063542.1 (Enterococcus faecium, BMT-4-1-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,376 [DEBUG - chronostrain.database.parser.json] - Strain CP084178.1 (Enterococcus faecium, 4995/20) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,377 [DEBUG - chronostrain.database.parser.json] - Strain CP083920.1 (Enterococcus faecium, V1164) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,379 [DEBUG - chronostrain.database.parser.json] - Strain CP083912.1 (Enterococcus faecium, V1225) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,380 [DEBUG - chronostrain.database.parser.json] - Strain CP072881.1 (Enterococcus faecium, F39) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,382 [DEBUG - chronostrain.database.parser.json] - Strain CP072878.1 (Enterococcus faecium, F88) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,383 [DEBUG - chronostrain.database.parser.json] - Strain CP066215.1 (Enterococcus faecium, 18-042) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,385 [DEBUG - chronostrain.database.parser.json] - Strain CP066205.1 (Enterococcus faecium, 18-204) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,387 [DEBUG - chronostrain.database.parser.json] - Strain CP064424.1 (Enterococcus faecium, PR01996-12) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,388 [DEBUG - chronostrain.database.parser.json] - Strain CP066673.1 (Enterococcus faecium, Dallas 163_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,390 [DEBUG - chronostrain.database.parser.json] - Strain CP066637.1 (Enterococcus faecium, Dallas 144_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,392 [DEBUG - chronostrain.database.parser.json] - Strain CP066606.1 (Enterococcus faecium, Dallas 124_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,393 [DEBUG - chronostrain.database.parser.json] - Strain CP066614.1 (Enterococcus faecium, Dallas 124_3) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,395 [DEBUG - chronostrain.database.parser.json] - Strain CP066576.1 (Enterococcus faecium, Dallas 100_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,397 [DEBUG - chronostrain.database.parser.json] - Strain CP066568.1 (Enterococcus faecium, Dallas 97_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,399 [DEBUG - chronostrain.database.parser.json] - Strain CP066396.1 (Enterococcus faecium, Dallas 51_4) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,400 [DEBUG - chronostrain.database.parser.json] - Strain CP066434.1 (Enterococcus faecium, Dallas 66) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,402 [DEBUG - chronostrain.database.parser.json] - Strain CP065928.1 (Enterococcus faecium, Dallas 5) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,404 [DEBUG - chronostrain.database.parser.json] - Strain CP065527.1 (Enterococcus faecium, VRE3382) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,405 [DEBUG - chronostrain.database.parser.json] - Strain CP045602.1 (Enterococcus faecium, TK-P5D) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,407 [DEBUG - chronostrain.database.parser.json] - Strain CP064279.1 (Enterococcus faecium, Cairo) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,408 [DEBUG - chronostrain.database.parser.json] - Strain CP034949.1 (Enterococcus faecium, NM213) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,410 [DEBUG - chronostrain.database.parser.json] - Strain CP033041.1 (Enterococcus faecium, JE1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,411 [DEBUG - chronostrain.database.parser.json] - Strain CP032308.1 (Enterococcus faecium, HY07) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,413 [DEBUG - chronostrain.database.parser.json] - Strain CP025077.1 (Enterococcus faecium, LS170308) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,414 [DEBUG - chronostrain.database.parser.json] - Strain CP016163.1 (Enterococcus faecium, ISMMS_VRE_11) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,416 [DEBUG - chronostrain.database.parser.json] - Strain CP012430.1 (Enterococcus faecium, ISMMS_VRE_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,418 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135782.1 (Enterococcus faecium, E4239) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,419 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135331.1 (Enterococcus faecium, E7471) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,421 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135287.1 (Enterococcus faecium, E7199) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,423 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135203.1 (Enterococcus faecium, E7171) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,424 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135226.1 (Enterococcus faecium, E7025) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,426 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135197.1 (Enterococcus faecium, E6055) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,427 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135181.1 (Enterococcus faecium, E1774) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,429 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR135170.1 (Enterococcus faecium, E4227) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,430 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116549.1 (Enterococcus faecium, K80-15b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,432 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116546.1 (Enterococcus faecium, K189-3) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,434 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116543.1 (Enterococcus faecium, K70-1a) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,435 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116540.1 (Enterococcus faecium, K75-1) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,437 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116536.1 (Enterococcus faecium, K162-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,438 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP026612.1 (Enterococcus faecium, AA94) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,440 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP109839.1 (Enterococcus faecium, T17-1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,442 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP088195.1 (Enterococcus faecium, S39-7a) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,444 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097061.1 (Enterococcus faecium, AT02) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,445 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097027.1 (Enterococcus faecium, AT44) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,447 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084178.1 (Enterococcus faecium, 4995/20) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,449 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083920.1 (Enterococcus faecium, V1164) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,450 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP083912.1 (Enterococcus faecium, V1225) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,452 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072881.1 (Enterococcus faecium, F39) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,453 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072878.1 (Enterococcus faecium, F88) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,455 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066215.1 (Enterococcus faecium, 18-042) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,457 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064424.1 (Enterococcus faecium, PR01996-12) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,458 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066673.1 (Enterococcus faecium, Dallas 163_1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,460 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066637.1 (Enterococcus faecium, Dallas 144_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,462 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066606.1 (Enterococcus faecium, Dallas 124_1) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,463 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066614.1 (Enterococcus faecium, Dallas 124_3) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,465 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066576.1 (Enterococcus faecium, Dallas 100_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,467 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066568.1 (Enterococcus faecium, Dallas 97_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,469 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066396.1 (Enterococcus faecium, Dallas 51_4) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,470 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP066434.1 (Enterococcus faecium, Dallas 66) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,472 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065928.1 (Enterococcus faecium, Dallas 5) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,474 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP065527.1 (Enterococcus faecium, VRE3382) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,475 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP045602.1 (Enterococcus faecium, TK-P5D) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,477 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034949.1 (Enterococcus faecium, NM213) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,478 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033041.1 (Enterococcus faecium, Enterococcus faecium JE1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,479 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP032308.1 (Enterococcus faecium, HY07) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,481 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025077.1 (Enterococcus faecium, LS170308) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,483 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP016163.1 (Enterococcus faecium, ISMMS_VRE_11) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,484 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012430.1 (Enterococcus faecium, ISMMS_VRE_1) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,487 [DEBUG - chronostrain.database.parser.json] - Strain CP034169.1 (Enterococcus avium, 352) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,489 [DEBUG - chronostrain.database.parser.json] - Strain CP024590.1 (Enterococcus avium, FDAARGOS_184) loaded with 17 markers.\n",
      "2024-02-25 13:21:25,491 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP034169.1 (Enterococcus avium, 352) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,493 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP024590.1 (Enterococcus avium, FDAARGOS_184) loaded with 17 markers.\n",
      "2024-02-25 13:21:25,495 [DEBUG - chronostrain.database.parser.json] - Strain AP019814.1 (Enterococcus avium, G-15) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,497 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP019814.1 (Enterococcus avium, G-15) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,498 [DEBUG - chronostrain.database.parser.json] - Strain AP022822.1 (Enterococcus saigonensis, VE80) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,499 [DEBUG - chronostrain.database.parser.json] - Strain NZ_AP022822.1 (Enterococcus saigonensis, VE80) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,501 [DEBUG - chronostrain.database.parser.json] - Strain CP116583.1 (Enterococcus dispar, K205-4b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,502 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116583.1 (Enterococcus dispar, K205-4b) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,504 [DEBUG - chronostrain.database.parser.json] - Strain CP060720.1 (Vagococcus carniphilus, ATCC BAA-640) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,506 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP060720.1 (Vagococcus carniphilus, ATCC BAA-640) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,508 [DEBUG - chronostrain.database.parser.json] - Strain CP021874.1 (Enterococcus wangshanyuanii, MN05) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,510 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP021874.1 (Enterococcus wangshanyuanii, MN05) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,511 [DEBUG - chronostrain.database.parser.json] - Strain CP049886.1 (Vagococcus coleopterorum, HDW17A) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,513 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049886.1 (Vagococcus coleopterorum, HDW17A) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,514 [DEBUG - chronostrain.database.parser.json] - Strain CP019609.1 (Vagococcus penaei, CD276) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,515 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019609.1 (Vagococcus penaei, CD276T) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,516 [DEBUG - chronostrain.database.parser.json] - Strain CP090216.1 (Vagococcus lutrae, VF20) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,517 [DEBUG - chronostrain.database.parser.json] - Strain CP097045.1 (Vagococcus lutrae, AT32) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,518 [DEBUG - chronostrain.database.parser.json] - Strain CP097044.1 (Vagococcus lutrae, AT33) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,520 [DEBUG - chronostrain.database.parser.json] - Strain CP097051.1 (Vagococcus lutrae, AT19) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,521 [DEBUG - chronostrain.database.parser.json] - Strain CP081833.1 (Vagococcus lutrae, BN31) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,522 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP090216.1 (Vagococcus lutrae, VF20) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,523 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097045.1 (Vagococcus lutrae, AT32) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,524 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097044.1 (Vagococcus lutrae, AT33) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,525 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097051.1 (Vagococcus lutrae, AT19) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,526 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081833.1 (Vagococcus lutrae, BN31) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,528 [DEBUG - chronostrain.database.parser.json] - Strain CP110237.1 (Vagococcus intermedius, STAA25) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,529 [DEBUG - chronostrain.database.parser.json] - Strain CP110232.1 (Vagococcus intermedius, STAA11) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,530 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110237.1 (Vagococcus intermedius, STAA25) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,532 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP110232.1 (Vagococcus intermedius, STAA11) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,533 [DEBUG - chronostrain.database.parser.json] - Strain CP116507.1 (Vagococcus lutrae, K204-1) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,534 [DEBUG - chronostrain.database.parser.json] - Strain CP097072.1 (Vagococcus lutrae, AT03) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,535 [DEBUG - chronostrain.database.parser.json] - Strain CP097053.1 (Vagococcus lutrae, AT15) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,536 [DEBUG - chronostrain.database.parser.json] - Strain CP097017.1 (Vagococcus lutrae, AT46b) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,537 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116507.1 (Vagococcus lutrae, K204-1) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,538 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097072.1 (Vagococcus lutrae, AT03) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,539 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097053.1 (Vagococcus lutrae, AT15) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,540 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP097017.1 (Vagococcus lutrae, AT46b) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,541 [DEBUG - chronostrain.database.parser.json] - Strain CP116503.1 (Vagococcus lutrae, K136-2) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,542 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP116503.1 (Vagococcus lutrae, K136-2) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,544 [DEBUG - chronostrain.database.parser.json] - Strain CP038865.1 (Vagococcus xieshaowenii, personal::cf-49) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,545 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038865.1 (Vagococcus xieshaowenii, CF-49) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,547 [DEBUG - chronostrain.database.parser.json] - Strain CP118957.1 (Enterococcus saccharolyticus, DSM 20726) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,549 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP118957.1 (Enterococcus saccharolyticus, DSM 20726) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,551 [DEBUG - chronostrain.database.parser.json] - Strain CP131008.1 (Enterococcus faecium, C0019EM0032) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,552 [DEBUG - chronostrain.database.parser.json] - Strain CP038166.1 (Enterococcus faecium, SC3-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP038166.1 (Enterococcus faecium, SC3-1) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,555 [DEBUG - chronostrain.database.parser.json] - Strain CP027786.1 (Tetragenococcus koreensis, KCTC 3924) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,557 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027786.1 (Tetragenococcus koreensis, KCTC 3924) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,558 [DEBUG - chronostrain.database.parser.json] - Strain CP128995.1 (Enterococcus faecium, AK_C_05) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,559 [DEBUG - chronostrain.database.parser.json] - Strain CP040878.1 (Enterococcus faecium, HB-1) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,561 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP128995.1 (Enterococcus faecium, AK_C_05) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,562 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP040878.1 (Enterococcus faecium, HB-1) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,564 [DEBUG - chronostrain.database.parser.json] - Strain AP012282.1 (Melissococcus plutonius, DAT561) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,566 [DEBUG - chronostrain.database.parser.json] - Strain NC_016938.1 (Melissococcus plutonius, DAT561) loaded with 14 markers.\n",
      "2024-02-25 13:21:25,568 [DEBUG - chronostrain.database.parser.json] - Strain CP125658.1 (Enterococcus sp., ZQ21) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,569 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP125658.1 (Enterococcus sp., ZQ21) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,571 [DEBUG - chronostrain.database.parser.json] - Strain CP104764.1 (Enterococcus raffinosus, Er676) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,573 [DEBUG - chronostrain.database.parser.json] - Strain CP104392.1 (Enterococcus raffinosus, HG-5) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,576 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP104764.1 (Enterococcus raffinosus, Er676) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,582 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP104392.1 (Enterococcus raffinosus, HG-5) loaded with 15 markers.\n",
      "2024-02-25 13:21:25,583 [DEBUG - chronostrain.database.parser.json] - Strain CP129887.1 (Enterococcus lactis, SU-B46) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,584 [DEBUG - chronostrain.database.parser.json] - Strain CP084248.1 (Enterococcus lactis, DH9003) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,585 [DEBUG - chronostrain.database.parser.json] - Strain CP079880.1 (Enterococcus lactis, CX 2-6_2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,587 [DEBUG - chronostrain.database.parser.json] - Strain CP078094.1 (Enterococcus lactis, JDM1) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,588 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP129887.1 (Enterococcus lactis, SU-B46) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,589 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP084248.1 (Enterococcus lactis, DH9003) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,590 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP078094.1 (Enterococcus lactis, JDM1) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,592 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP079880.1 (Enterococcus lactis, CX 2-6_2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,593 [DEBUG - chronostrain.database.parser.json] - Strain CP041665.1 (Enterococcus faecium, CX2-6) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,594 [DEBUG - chronostrain.database.parser.json] - Strain CP099979.1 (Enterococcus faecium, WN-18) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,595 [DEBUG - chronostrain.database.parser.json] - Strain CP091221.1 (Enterococcus faecium, 642) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,597 [DEBUG - chronostrain.database.parser.json] - Strain CP035136.1 (Enterococcus faecium, SRCM103341) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,598 [DEBUG - chronostrain.database.parser.json] - Strain CP025685.1 (Enterococcus faecium, CBA7134) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,599 [DEBUG - chronostrain.database.parser.json] - Strain CP033376.1 (Enterococcus faecium, Gr17) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,600 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP041665.1 (Enterococcus faecium, CX2-6) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,602 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP099979.1 (Enterococcus faecium, WN-18) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,603 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091221.1 (Enterococcus faecium, 642) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,604 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP035136.1 (Enterococcus faecium, SRCM103341) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,605 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP025685.1 (Enterococcus faecium, CBA7134) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,607 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033376.1 (Enterococcus faecium, Gr17) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,608 [DEBUG - chronostrain.database.parser.json] - Strain CP082963.1 (Enterococcus lactis, HJS001) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,609 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP082963.1 (Enterococcus lactis, HJS001) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,610 [DEBUG - chronostrain.database.parser.json] - Strain CP043484.1 (Enterococcus faecium, DMEA02) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,611 [DEBUG - chronostrain.database.parser.json] - Strain CP042834.1 (Enterococcus faecium, FA3) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,613 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043484.1 (Enterococcus faecium, DMEA02) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,614 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042834.1 (Enterococcus faecium, FA3) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,615 [DEBUG - chronostrain.database.parser.json] - Strain CP027783.1 (Tetragenococcus osmophilus, JCM 31126) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,616 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027783.1 (Tetragenococcus osmophilus, JCM 31126) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,617 [DEBUG - chronostrain.database.parser.json] - Strain CP030932.1 (Enterococcus gilvus, CR1) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,618 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP030932.1 (Enterococcus gilvus, CR1) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,620 [DEBUG - chronostrain.database.parser.json] - Strain CP120467.1 (Enterococcus sp., CoE-012-22) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,622 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP120467.1 (Enterococcus sp., CoE-012-22) loaded with 13 markers.\n",
      "2024-02-25 13:21:25,624 [DEBUG - chronostrain.database.parser.json] - Strain OX346411.1 (Enterococcus cecorum, CIRMBP-1287) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,625 [DEBUG - chronostrain.database.parser.json] - Strain OX346408.1 (Enterococcus cecorum, CIRMBP-1302) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,627 [DEBUG - chronostrain.database.parser.json] - Strain OX346405.1 (Enterococcus cecorum, CIRMBP-1228) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,628 [DEBUG - chronostrain.database.parser.json] - Strain OX346407.1 (Enterococcus cecorum, CIRMBP-1274) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,630 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346411.1 (Enterococcus cecorum, CIRMBP-1287) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,631 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346408.1 (Enterococcus cecorum, CIRMBP-1302) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,632 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346405.1 (Enterococcus cecorum, CIRMBP-1228) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,634 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346407.1 (Enterococcus cecorum, CIRMBP-1274) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,636 [DEBUG - chronostrain.database.parser.json] - Strain OX346406.1 (Enterococcus cecorum, CIRMBP-1212) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,637 [DEBUG - chronostrain.database.parser.json] - Strain OX346401.1 (Enterococcus cecorum, CIRMBP-1246) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,638 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346406.1 (Enterococcus cecorum, CIRMBP-1212) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,640 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346401.1 (Enterococcus cecorum, CIRMBP-1246) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,641 [DEBUG - chronostrain.database.parser.json] - Strain OX346403.1 (Enterococcus cecorum, CIRMBP-1283) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,643 [DEBUG - chronostrain.database.parser.json] - Strain OX346402.1 (Enterococcus cecorum, CIRMBP-1261) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,644 [DEBUG - chronostrain.database.parser.json] - Strain OX346404.1 (Enterococcus cecorum, CIRMBP-1281) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,645 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346403.1 (Enterococcus cecorum, CIRMBP-1283) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,647 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346402.1 (Enterococcus cecorum, CIRMBP-1261) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,648 [DEBUG - chronostrain.database.parser.json] - Strain NZ_OX346404.1 (Enterococcus cecorum, CIRMBP-1281) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,650 [DEBUG - chronostrain.database.parser.json] - Strain LS483306.1 (Enterococcus cecorum, NCTC12421) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,651 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LS483306.1 (Enterococcus cecorum, NCTC12421) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,653 [DEBUG - chronostrain.database.parser.json] - Strain CP072888.1 (Enterococcus raffinosus, F162_2) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,655 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP072888.1 (Enterococcus raffinosus, F162_2) loaded with 12 markers.\n",
      "2024-02-25 13:21:25,657 [DEBUG - chronostrain.database.parser.json] - Strain CP081472.1 (Vagococcus fluvialis, 12B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,658 [DEBUG - chronostrain.database.parser.json] - Strain CP081470.1 (Vagococcus fluvialis, 25B2) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,659 [DEBUG - chronostrain.database.parser.json] - Strain CP081466.1 (Vagococcus fluvialis, 35B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,661 [DEBUG - chronostrain.database.parser.json] - Strain CP081459.1 (Vagococcus fluvialis, 110B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,662 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081472.1 (Vagococcus fluvialis, 12B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,663 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081470.1 (Vagococcus fluvialis, 25B2) loaded with 9 markers.\n",
      "2024-02-25 13:21:25,665 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081466.1 (Vagococcus fluvialis, 35B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,666 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081459.1 (Vagococcus fluvialis, 110B2) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,668 [DEBUG - chronostrain.database.parser.json] - Strain CP102451.1 (Vagococcus luciliae, G314F) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP102451.1 (Vagococcus luciliae, G314F) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,670 [DEBUG - chronostrain.database.parser.json] - Strain CP081461.1 (Vagococcus fluvialis, 36B2) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,671 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP081461.1 (Vagococcus fluvialis, 36B2) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,673 [DEBUG - chronostrain.database.parser.json] - Strain CP017267.1 (Vagococcus teuberi, DSM 21459) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,674 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017267.1 (Vagococcus teuberi, DSM 21459) loaded with 8 markers.\n",
      "2024-02-25 13:21:25,675 [DEBUG - chronostrain.database.parser.json] - Strain CP093384.1 (Vagococcus sp., CY52-2) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,677 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP093384.1 (Vagococcus sp., CY52-2) loaded with 7 markers.\n",
      "2024-02-25 13:21:25,678 [DEBUG - chronostrain.database.parser.json] - Strain CP039712.1 (Vagococcus zengguangii, MN-17) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,680 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP039712.1 (Vagococcus zengguangii, MN-17) loaded with 11 markers.\n",
      "2024-02-25 13:21:25,682 [DEBUG - chronostrain.database.parser.json] - Strain CP049887.1 (Vagococcus hydrophili, HDW17B) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,683 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP049887.1 (Vagococcus hydrophili, HDW17B) loaded with 10 markers.\n",
      "2024-02-25 13:21:25,684 [DEBUG - chronostrain.database.parser.json] - Strain CP074114.1 (Tetragenococcus halophilus, SNTH-1) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,685 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074114.1 (Tetragenococcus halophilus, SNTH-1) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,686 [DEBUG - chronostrain.database.parser.json] - Strain CP078164.1 (Tetragenococcus halophilus, CY54) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,687 [DEBUG - chronostrain.database.parser.json] - Strain AP012046.1 (Tetragenococcus halophilus, NBRC 12172) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,688 [DEBUG - chronostrain.database.parser.json] - Strain NC_016052.1 (Tetragenococcus halophilus, NBRC 12172) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,689 [DEBUG - chronostrain.database.parser.json] - Strain CP020017.1 (Tetragenococcus halophilus, KUD23) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,689 [DEBUG - chronostrain.database.parser.json] - Strain CP012047.1 (Tetragenococcus halophilus, MJ4) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP020017.1 (Tetragenococcus halophilus, KUD23) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,691 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP012047.1 (Tetragenococcus halophilus, MJ4) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,692 [DEBUG - chronostrain.database.parser.json] - Strain CP027768.1 (Tetragenococcus halophilus, LMG 26042) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,693 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027768.1 (Tetragenococcus halophilus, LMG 26042) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,694 [DEBUG - chronostrain.database.parser.json] - Strain CP046246.1 (Tetragenococcus halophilus, YJ1) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,695 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP046246.1 (Tetragenococcus halophilus, YJ1) loaded with 6 markers.\n",
      "2024-02-25 13:21:25,703 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161825 (Enterococcus faecalis, C02756_t:M_s:2) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,712 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159855 (Enterococcus faecalis, C02566_t:7_s:2) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,720 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159835 (Enterococcus faecalis, C02566_t:7_s:1) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,729 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158985 (Enterococcus faecalis, C02566_t:7_s:3) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,737 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160815 (Enterococcus faecalis, C02186_t:7_s:1) loaded with 65 markers.\n",
      "2024-02-25 13:21:25,746 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160805 (Enterococcus faecalis, C02186_t:7_s:2) loaded with 65 markers.\n",
      "2024-02-25 13:21:25,755 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164755 (Enterococcus faecalis, C01700_t:7_s:2) loaded with 62 markers.\n",
      "2024-02-25 13:21:25,763 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164735 (Enterococcus faecalis, C01700_t:4_s:1) loaded with 62 markers.\n",
      "2024-02-25 13:21:25,771 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159055 (Enterococcus faecalis, C01700_t:7_s:1) loaded with 62 markers.\n",
      "2024-02-25 13:21:25,780 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159465 (Enterococcus faecalis, C01443_t:M_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,788 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163945 (Enterococcus faecalis, C01387_t:M_s:*) loaded with 62 markers.\n",
      "2024-02-25 13:21:25,796 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162075 (Enterococcus faecalis, C01012_t:21_s:1) loaded with 58 markers.\n",
      "2024-02-25 13:21:25,804 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159335 (Enterococcus faecalis, C01012_t:21_s:2) loaded with 58 markers.\n",
      "2024-02-25 13:21:25,812 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164295 (Enterococcus faecalis, B02588_t:7_s:*) loaded with 65 markers.\n",
      "2024-02-25 13:21:25,821 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161935 (Enterococcus faecalis, B02588_t:4_s:*) loaded with 65 markers.\n",
      "2024-02-25 13:21:25,829 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161645 (Enterococcus faecalis, B01956_t:4_s:*) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,837 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161525 (Enterococcus faecalis, B01375_t:4_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,846 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161515 (Enterococcus faecalis, B01375_t:7_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,855 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159435 (Enterococcus faecalis, B01339_t:4_s:1) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,863 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158815 (Enterococcus faecalis, B01339_t:21_s:1) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,872 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158765 (Enterococcus faecalis, B01339_t:7_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,880 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166645 (Enterococcus faecalis, B01278_t:21_s:1) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,889 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165975 (Enterococcus faecalis, B01278_t:4_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,897 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164015 (Enterococcus faecalis, B01278_t:7_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:25,905 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166605 (Enterococcus faecalis, B01042_t:M_s:5) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,913 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166575 (Enterococcus faecalis, B01042_t:M_s:4) loaded with 60 markers.\n",
      "2024-02-25 13:21:25,921 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163855 (Enterococcus faecalis, B00272_t:7_s:2) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,929 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162785 (Enterococcus faecalis, B00272_t:4_s:*) loaded with 61 markers.\n",
      "2024-02-25 13:21:25,938 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160235 (Enterococcus faecalis, B00071_t:21_s:2) loaded with 71 markers.\n",
      "2024-02-25 13:21:25,947 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160135 (Enterococcus faecalis, B00071_t:7_s:2) loaded with 71 markers.\n",
      "2024-02-25 13:21:25,957 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163675 (Enterococcus faecalis, A01687_t:4_s:*) loaded with 70 markers.\n",
      "2024-02-25 13:21:25,967 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163635 (Enterococcus faecalis, A01687_t:7_s:*) loaded with 70 markers.\n",
      "2024-02-25 13:21:25,976 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163975 (Enterococcus faecalis, B00116_t:4_s:2) loaded with 67 markers.\n",
      "2024-02-25 13:21:25,989 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158855 (Enterococcus faecalis, B00116_t:7_s:3) loaded with 67 markers.\n",
      "2024-02-25 13:21:25,997 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161145 (Enterococcus faecalis, B00111_t:4_s:*) loaded with 59 markers.\n",
      "2024-02-25 13:21:26,005 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161115 (Enterococcus faecalis, B00111_t:7_s:*) loaded with 59 markers.\n",
      "2024-02-25 13:21:26,014 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160295 (Enterococcus faecalis, B00097_t:21_s:1) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,023 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160285 (Enterococcus faecalis, B00097_t:21_s:3) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,031 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163475 (Enterococcus faecalis, C02180_t:7_s:2) loaded with 65 markers.\n",
      "2024-02-25 13:21:26,040 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163415 (Enterococcus faecalis, C02180_t:7_s:1) loaded with 65 markers.\n",
      "2024-02-25 13:21:26,053 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163365 (Enterococcus faecalis, C02047_t:M_s:3) loaded with 66 markers.\n",
      "2024-02-25 13:21:26,066 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163355 (Enterococcus faecalis, C02047_t:M_s:2) loaded with 66 markers.\n",
      "2024-02-25 13:21:26,075 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163175 (Enterococcus faecalis, C01929_t:7_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,084 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163785 (Enterococcus faecalis, C01758_t:4_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,094 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162825 (Enterococcus faecalis, C01758_t:7_s:1) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,103 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162335 (Enterococcus faecalis, C01758_t:21_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,115 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162865 (Enterococcus faecalis, C01751_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,126 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162815 (Enterococcus faecalis, C01695_t:4_s:1) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,134 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162025 (Enterococcus faecalis, C01329_t:4_s:1) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,143 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161975 (Enterococcus faecalis, C01329_t:7_s:2) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,151 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161845 (Enterococcus faecalis, B02257_t:21_s:1) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,160 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165945 (Enterococcus faecalis, B01610_t:7_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,168 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164115 (Enterococcus faecalis, B01610_t:21_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,177 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161475 (Enterococcus faecalis, B01261_t:21_s:1) loaded with 66 markers.\n",
      "2024-02-25 13:21:26,186 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159145 (Enterococcus faecalis, B01261_t:7_s:1) loaded with 66 markers.\n",
      "2024-02-25 13:21:26,195 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163125 (Enterococcus faecalis, B01089_t:7_s:2) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,203 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165985 (Enterococcus faecalis, B00553_t:7_s:1) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,212 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161335 (Enterococcus faecalis, B00550_t:21_s:2) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,221 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160825 (Enterococcus faecalis, B00550_t:21_s:4) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,230 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160795 (Enterococcus faecalis, B00550_t:21_s:5) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,238 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163755 (Enterococcus faecalis, B00272_t:7_s:1) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,246 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159175 (Enterococcus faecalis, B00272_t:21_s:*) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,255 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164585 (Enterococcus faecalis, B00013_t:21_s:*) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,263 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160105 (Enterococcus faecalis, B00013_t:4_s:*) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,271 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160085 (Enterococcus faecalis, B00013_t:7_s:2) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,279 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160075 (Enterococcus faecalis, B00013_t:7_s:1) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,288 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159645 (Enterococcus faecalis, B00088_t:7_s:3) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,301 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165485 (Enterococcus faecalis, A01678_t:21_s:*) loaded with 68 markers.\n",
      "2024-02-25 13:21:26,314 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160005 (Enterococcus faecalis, A01563_t:7_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,323 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159135 (Enterococcus faecalis, A01166_t:7_s:1) loaded with 70 markers.\n",
      "2024-02-25 13:21:26,333 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159125 (Enterococcus faecalis, A01166_t:4_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:26,342 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159115 (Enterococcus faecalis, A01166_t:7_s:2) loaded with 72 markers.\n",
      "2024-02-25 13:21:26,352 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162305 (Enterococcus faecalis, A00576_t:M_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:26,360 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158825 (Enterococcus faecalis, B00157_T3_t:4_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,369 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163015 (Enterococcus faecalis, B00129_t:7_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,377 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159885 (Enterococcus faecalis, C02566_t:M_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,386 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159735 (Enterococcus faecalis, C02236_t:7_s:3) loaded with 65 markers.\n",
      "2024-02-25 13:21:26,394 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163505 (Enterococcus faecalis, C02223_T2_t:7_s:3) loaded with 59 markers.\n",
      "2024-02-25 13:21:26,407 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163465 (Enterococcus faecalis, C02062_t:21_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:26,417 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163825 (Enterococcus faecalis, C02059_t:7_s:4) loaded with 72 markers.\n",
      "2024-02-25 13:21:26,428 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163925 (Enterococcus faecalis, C01913_t:7_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,437 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162345 (Enterococcus faecalis, C01877_t:21_s:1) loaded with 68 markers.\n",
      "2024-02-25 13:21:26,445 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162235 (Enterococcus faecalis, C01871_t:M_s:*) loaded with 62 markers.\n",
      "2024-02-25 13:21:26,453 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159495 (Enterococcus faecalis, C01841_t:M_s:2) loaded with 60 markers.\n",
      "2024-02-25 13:21:26,463 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164425 (Enterococcus faecalis, C01837_t:21_s:1) loaded with 70 markers.\n",
      "2024-02-25 13:21:26,472 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162275 (Enterococcus faecalis, C01837_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,481 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162295 (Enterococcus faecalis, C01758_t:21_s:3) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,491 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162885 (Enterococcus faecalis, C01695_t:4_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,500 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164775 (Enterococcus faecalis, C01560_t:7_s:*) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,509 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164485 (Enterococcus faecalis, C01529_t:7_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,518 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163805 (Enterococcus faecalis, C01529_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,528 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164765 (Enterococcus faecalis, C01392_t:21_s:4) loaded with 76 markers.\n",
      "2024-02-25 13:21:26,538 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162145 (Enterococcus faecalis, C01392_t:21_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:26,549 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162085 (Enterococcus faecalis, C01392_t:21_s:3) loaded with 76 markers.\n",
      "2024-02-25 13:21:26,558 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162095 (Enterococcus faecalis, C01331_t:4_s:2) loaded with 72 markers.\n",
      "2024-02-25 13:21:26,567 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163995 (Enterococcus faecalis, B02215_t:21_s:2) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,577 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163935 (Enterococcus faecalis, B02215_t:21_s:3) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,586 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163165 (Enterococcus faecalis, B02215_t:21_s:1) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,597 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164315 (Enterococcus faecalis, B02211_t:7_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,608 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164245 (Enterococcus faecalis, B02211_t:21_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,617 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161705 (Enterococcus faecalis, B02156_t:21_s:2) loaded with 65 markers.\n",
      "2024-02-25 13:21:26,625 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161695 (Enterococcus faecalis, B02156_t:21_s:1) loaded with 65 markers.\n",
      "2024-02-25 13:21:26,634 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161565 (Enterococcus faecalis, B02005_t:21_s:2) loaded with 64 markers.\n",
      "2024-02-25 13:21:26,646 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161665 (Enterococcus faecalis, B01719_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,657 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161575 (Enterococcus faecalis, B01719_t:7_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,666 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159395 (Enterococcus faecalis, B01719_t:21_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,675 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163875 (Enterococcus faecalis, B01365_t:21_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,683 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163965 (Enterococcus faecalis, B01042_t:M_s:3) loaded with 62 markers.\n",
      "2024-02-25 13:21:26,693 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163815 (Enterococcus faecalis, B00922_t:7_s:2) loaded with 73 markers.\n",
      "2024-02-25 13:21:26,702 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163865 (Enterococcus faecalis, B00518_t:7_s:2) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,711 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161385 (Enterococcus faecalis, B00518_t:21_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,720 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160835 (Enterococcus faecalis, B00518_t:4_s:1) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,730 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160785 (Enterococcus faecalis, B00518_t:4_s:2) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,739 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159275 (Enterococcus faecalis, B00518_t:7_s:1) loaded with 71 markers.\n",
      "2024-02-25 13:21:26,747 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160165 (Enterococcus faecalis, B00085_t:7_s:*) loaded with 58 markers.\n",
      "2024-02-25 13:21:26,758 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161125 (Enterococcus faecalis, B00053_t:7_s:3) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,770 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158895 (Enterococcus faecalis, B00053_t:4_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,786 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163885 (Enterococcus faecalis, B00036_t:7_s:2) loaded with 78 markers.\n",
      "2024-02-25 13:21:26,799 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163715 (Enterococcus faecalis, B00036_t:21_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:26,811 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163705 (Enterococcus faecalis, B00036_t:4_s:1) loaded with 78 markers.\n",
      "2024-02-25 13:21:26,821 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161085 (Enterococcus faecalis, B00020_t:21_s:*) loaded with 78 markers.\n",
      "2024-02-25 13:21:26,831 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160185 (Enterococcus faecalis, B00020_t:7_s:*) loaded with 78 markers.\n",
      "2024-02-25 13:21:26,841 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160125 (Enterococcus faecalis, B00020_t:4_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:26,851 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165815 (Enterococcus faecalis, B00016_t:21_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,859 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166775 (Enterococcus faecalis, B00010_t:4_s:1) loaded with 60 markers.\n",
      "2024-02-25 13:21:26,869 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165735 (Enterococcus faecalis, A02819_t:7_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:26,879 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165775 (Enterococcus faecalis, A02110_t:M_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:26,889 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165755 (Enterococcus faecalis, A02110_t:M_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:26,898 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164855 (Enterococcus faecalis, A02053_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,907 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164845 (Enterococcus faecalis, A02053_t:7_s:*) loaded with 70 markers.\n",
      "2024-02-25 13:21:26,916 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163155 (Enterococcus faecalis, A02053_t:21_s:1) loaded with 69 markers.\n",
      "2024-02-25 13:21:26,925 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165785 (Enterococcus faecalis, A01966_t:7_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,933 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163735 (Enterococcus faecalis, A01966_t:21_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,942 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158935 (Enterococcus faecalis, A01966_t:4_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:26,954 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163695 (Enterococcus faecalis, A01739_t:7_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,965 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163655 (Enterococcus faecalis, A01739_t:4_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:26,973 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166395 (Enterococcus faecalis, A01676_t:4_s:3) loaded with 63 markers.\n",
      "2024-02-25 13:21:26,982 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163915 (Enterococcus faecalis, A01676_t:4_s:4) loaded with 70 markers.\n",
      "2024-02-25 13:21:26,994 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160045 (Enterococcus faecalis, A01639_t:4_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,005 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159955 (Enterococcus faecalis, A01639_t:4_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,017 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159845 (Enterococcus faecalis, A01639_t:21_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,028 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159825 (Enterococcus faecalis, A01639_t:7_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,040 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165465 (Enterococcus faecalis, A01301_t:7_s:2) loaded with 73 markers.\n",
      "2024-02-25 13:21:27,051 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159295 (Enterococcus faecalis, A01301_t:4_s:1) loaded with 73 markers.\n",
      "2024-02-25 13:21:27,061 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163625 (Enterococcus faecalis, A01176_t:4_s:2) loaded with 64 markers.\n",
      "2024-02-25 13:21:27,073 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159315 (Enterococcus faecalis, A01176_t:7_s:1) loaded with 63 markers.\n",
      "2024-02-25 13:21:27,082 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158795 (Enterococcus faecalis, A01176_t:4_s:1) loaded with 63 markers.\n",
      "2024-02-25 13:21:27,095 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159155 (Enterococcus faecalis, A01166_t:21_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,109 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165295 (Enterococcus faecalis, A01107_t:4_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:27,119 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159755 (Enterococcus faecalis, A01077_t:21_s:2) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,128 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159705 (Enterococcus faecalis, A01077_t:21_s:1) loaded with 71 markers.\n",
      "2024-02-25 13:21:27,139 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166505 (Enterococcus faecalis, A00995_t:4_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,151 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160985 (Enterococcus faecalis, A00995_t:7_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,162 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160965 (Enterococcus faecalis, A00995_t:7_s:3) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,177 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159935 (Enterococcus faecalis, A00995_t:7_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,192 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164075 (Enterococcus faecalis, A00502_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:27,203 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163585 (Enterococcus faecalis, A00502_t:7_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:27,212 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158875 (Enterococcus faecalis, A00502_t:21_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:27,231 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164645 (Enterococcus faecalis, A00190_t:21_s:1) loaded with 70 markers.\n",
      "2024-02-25 13:21:27,243 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164125 (Enterococcus faecalis, A00185_t:4_s:2) loaded with 81 markers.\n",
      "2024-02-25 13:21:27,253 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165915 (Enterococcus faecalis, B00235_t:21_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,261 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159255 (Enterococcus faecalis, B00235_t:7_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,271 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160465 (Enterococcus faecalis, B00178_t:21_s:3) loaded with 71 markers.\n",
      "2024-02-25 13:21:27,280 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159185 (Enterococcus faecalis, B00178_t:7_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:27,290 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159165 (Enterococcus faecalis, B00178_t:21_s:5) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,298 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163725 (Enterococcus faecalis, B00100_t:21_s:*) loaded with 66 markers.\n",
      "2024-02-25 13:21:27,307 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158905 (Enterococcus faecalis, B00100_t:7_s:*) loaded with 68 markers.\n",
      "2024-02-25 13:21:27,318 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158995 (Enterococcus faecalis, A00021_T2_t:7_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,329 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164945 (Enterococcus faecalis, A00021_T1_t:21_s:5) loaded with 71 markers.\n",
      "2024-02-25 13:21:27,337 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164515 (Enterococcus faecalis, C01930_t:7_s:1) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,346 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163295 (Enterococcus faecalis, C01930_t:21_s:1) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,354 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163265 (Enterococcus faecalis, C01930_t:21_s:2) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,363 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162935 (Enterococcus faecalis, C01930_t:4_s:*) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,371 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166695 (Enterococcus faecalis, C01389_t:21_s:2) loaded with 59 markers.\n",
      "2024-02-25 13:21:27,379 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164395 (Enterococcus faecalis, C01389_t:4_s:*) loaded with 60 markers.\n",
      "2024-02-25 13:21:27,387 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164385 (Enterococcus faecalis, C01389_t:7_s:*) loaded with 60 markers.\n",
      "2024-02-25 13:21:27,400 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161995 (Enterococcus faecalis, C01075_t:M_s:3) loaded with 73 markers.\n",
      "2024-02-25 13:21:27,412 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161915 (Enterococcus faecalis, B02720_t:21_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,424 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161905 (Enterococcus faecalis, B02720_t:7_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,435 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161895 (Enterococcus faecalis, B02720_t:4_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,444 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161465 (Enterococcus faecalis, B01236_t:7_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:27,452 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161445 (Enterococcus faecalis, B01236_t:21_s:*) loaded with 64 markers.\n",
      "2024-02-25 13:21:27,465 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159445 (Enterococcus faecalis, B01196_t:21_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:27,474 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161375 (Enterococcus faecalis, B01021_t:21_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,486 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161855 (Enterococcus faecalis, B02270_t:4_s:*) loaded with 84 markers.\n",
      "2024-02-25 13:21:27,497 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161805 (Enterococcus faecalis, B02270_t:7_s:*) loaded with 84 markers.\n",
      "2024-02-25 13:21:27,510 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165345 (Enterococcus faecalis, A01082_t:4_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,522 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163795 (Enterococcus faecalis, A00947_t:21_s:*) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,535 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163645 (Enterococcus faecalis, A00947_t:4_s:*) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,546 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160035 (Enterococcus faecalis, A00559_t:21_s:3) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,558 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159945 (Enterococcus faecalis, A00559_t:7_s:2) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,571 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159915 (Enterococcus faecalis, A00559_t:21_s:1) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,583 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159895 (Enterococcus faecalis, A00559_t:21_s:2) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,595 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159085 (Enterococcus faecalis, A00559_t:7_s:1) loaded with 67 markers.\n",
      "2024-02-25 13:21:27,606 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160655 (Enterococcus faecalis, B00236_t:4_s:2) loaded with 82 markers.\n",
      "2024-02-25 13:21:27,617 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160575 (Enterococcus faecalis, B00236_t:7_s:*) loaded with 84 markers.\n",
      "2024-02-25 13:21:27,628 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160565 (Enterococcus faecalis, B00236_t:4_s:1) loaded with 84 markers.\n",
      "2024-02-25 13:21:27,639 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160555 (Enterococcus faecalis, B00236_t:21_s:1) loaded with 82 markers.\n",
      "2024-02-25 13:21:27,657 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166815 (Enterococcus faecalis, B00203_T2_t:21_s:2) loaded with 71 markers.\n",
      "2024-02-25 13:21:27,669 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163685 (Enterococcus faecalis, B00203_T2_t:4_s:*) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,682 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160615 (Enterococcus faecalis, B00203_T2_t:7_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,695 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160545 (Enterococcus faecalis, B00203_T2_t:7_s:3) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,708 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160535 (Enterococcus faecalis, B00203_T2_t:7_s:2) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,721 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160505 (Enterococcus faecalis, B00203_T2_t:21_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,734 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160595 (Enterococcus faecalis, B00203_T1_t:7_s:2) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,746 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160455 (Enterococcus faecalis, B00203_T1_t:7_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,759 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159565 (Enterococcus faecalis, B00203_T1_t:21_s:5) loaded with 72 markers.\n",
      "2024-02-25 13:21:27,771 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162995 (Enterococcus faecalis, B00101_t:21_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:27,784 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162855 (Enterococcus faecalis, B00101_t:7_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,794 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158925 (Enterococcus faecalis, B00101_t:4_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,804 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159405 (Enterococcus faecalis, C01752_t:4_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,813 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163895 (Enterococcus faecalis, C01387_t:7_s:*) loaded with 68 markers.\n",
      "2024-02-25 13:21:27,822 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162135 (Enterococcus faecalis, C01387_t:4_s:*) loaded with 69 markers.\n",
      "2024-02-25 13:21:27,832 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161865 (Enterococcus faecalis, B02273_t:4_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,841 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161745 (Enterococcus faecalis, B02005_t:7_s:2) loaded with 66 markers.\n",
      "2024-02-25 13:21:27,851 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162985 (Enterococcus faecalis, A00106_T2_t:21_s:*) loaded with 81 markers.\n",
      "2024-02-25 13:21:27,861 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163025 (Enterococcus faecalis, A01862_t:4_s:4) loaded with 74 markers.\n",
      "2024-02-25 13:21:27,878 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164695 (Enterococcus faecalis, A01057_t:21_s:2) loaded with 68 markers.\n",
      "2024-02-25 13:21:27,889 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160975 (Enterococcus faecalis, A01057_t:4_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:27,899 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159005 (Enterococcus faecalis, A01057_t:21_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:27,909 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160015 (Enterococcus faecalis, A01011_t:4_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,920 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159975 (Enterococcus faecalis, A01011_t:7_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,930 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163845 (Enterococcus faecalis, A00106_T1_t:21_s:*) loaded with 80 markers.\n",
      "2024-02-25 13:21:27,942 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165805 (Enterococcus faecalis, C02892_t:7_s:3) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,952 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165575 (Enterococcus faecalis, C02756_t:7_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:27,964 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165365 (Enterococcus faecalis, C02695_t:M_s:3) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,975 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165335 (Enterococcus faecalis, C02391_t:7_s:5) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,987 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165185 (Enterococcus faecalis, C02391_t:7_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:27,998 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165105 (Enterococcus faecalis, C02391_t:7_s:3) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,011 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165055 (Enterococcus faecalis, C02391_t:7_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,023 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164955 (Enterococcus faecalis, C02295_t:M_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,035 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164885 (Enterococcus faecalis, C02295_t:M_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,044 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164495 (Enterococcus faecalis, C01930_t:7_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,054 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164435 (Enterococcus faecalis, C01930_t:7_s:3) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,064 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163135 (Enterococcus faecalis, C01919_t:21_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,074 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162965 (Enterococcus faecalis, C01919_t:7_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,084 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162805 (Enterococcus faecalis, C01919_t:4_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,094 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162795 (Enterococcus faecalis, C01919_t:7_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,103 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162245 (Enterococcus faecalis, C01877_t:21_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,116 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162265 (Enterococcus faecalis, C01839_t:7_s:*) loaded with 71 markers.\n",
      "2024-02-25 13:21:28,124 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166665 (Enterococcus faecalis, C01832_t:21_s:*) loaded with 62 markers.\n",
      "2024-02-25 13:21:28,134 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164445 (Enterococcus faecalis, C01832_t:7_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,144 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162285 (Enterococcus faecalis, C01832_t:4_s:*) loaded with 74 markers.\n",
      "2024-02-25 13:21:28,155 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165085 (Enterococcus faecalis, C01758_t:M_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,164 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164785 (Enterococcus faecalis, C01757_t:21_s:1) loaded with 70 markers.\n",
      "2024-02-25 13:21:28,173 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166685 (Enterococcus faecalis, C01700_t:4_s:2) loaded with 73 markers.\n",
      "2024-02-25 13:21:28,183 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164405 (Enterococcus faecalis, C01201_t:4_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,193 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164345 (Enterococcus faecalis, C01201_t:7_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,203 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164355 (Enterococcus faecalis, C01052_t:4_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,213 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164335 (Enterococcus faecalis, C01052_t:21_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,223 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162035 (Enterococcus faecalis, C01052_t:7_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,232 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164215 (Enterococcus faecalis, C01008_T2_t:4_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,242 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161945 (Enterococcus faecalis, B02326_t:4_s:2) loaded with 74 markers.\n",
      "2024-02-25 13:21:28,251 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161875 (Enterococcus faecalis, B02315_t:21_s:1) loaded with 74 markers.\n",
      "2024-02-25 13:21:28,261 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161795 (Enterococcus faecalis, B02315_t:21_s:2) loaded with 74 markers.\n",
      "2024-02-25 13:21:28,274 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161785 (Enterococcus faecalis, B02273_t:7_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:28,283 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164325 (Enterococcus faecalis, B02258_t:4_s:2) loaded with 73 markers.\n",
      "2024-02-25 13:21:28,292 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164275 (Enterococcus faecalis, B02258_t:4_s:1) loaded with 72 markers.\n",
      "2024-02-25 13:21:28,302 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164235 (Enterococcus faecalis, B02216_t:21_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,312 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159695 (Enterococcus faecalis, B02216_t:7_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,322 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161595 (Enterococcus faecalis, B01797_t:4_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,332 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159455 (Enterococcus faecalis, B01797_t:7_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,342 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164195 (Enterococcus faecalis, B01775_t:4_s:5) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,352 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164165 (Enterococcus faecalis, B01775_t:7_s:1) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,362 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159685 (Enterococcus faecalis, B01775_t:21_s:4) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,372 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164035 (Enterococcus faecalis, B01772_t:21_s:1) loaded with 78 markers.\n",
      "2024-02-25 13:21:28,381 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163905 (Enterococcus faecalis, B01772_t:7_s:6) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,391 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163835 (Enterococcus faecalis, B01772_t:7_s:5) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,401 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163115 (Enterococcus faecalis, B01772_t:4_s:5) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,411 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161275 (Enterococcus faecalis, B00507_t:21_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,423 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165795 (Enterococcus faecalis, B00046_t:M_s:4) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,433 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164415 (Enterococcus faecalis, B00012_t:M_s:*) loaded with 78 markers.\n",
      "2024-02-25 13:21:28,446 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163745 (Enterococcus faecalis, B00012_t:4_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,459 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163045 (Enterococcus faecalis, B00012_t:7_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,473 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162945 (Enterococcus faecalis, B00012_t:7_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,485 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165745 (Enterococcus faecalis, A02913_t:7_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,496 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165725 (Enterococcus faecalis, A02138_t:21_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,505 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164455 (Enterococcus faecalis, C01530_t:7_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,515 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163035 (Enterococcus faecalis, C01530_t:7_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,524 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163005 (Enterococcus faecalis, C01530_t:21_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,534 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162975 (Enterococcus faecalis, C01530_t:21_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,544 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158945 (Enterococcus faecalis, C01530_t:4_s:*) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,554 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161555 (Enterococcus faecalis, B01471_t:M_s:1) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,564 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161545 (Enterococcus faecalis, B01471_t:M_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,574 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165715 (Enterococcus faecalis, A02110_t:21_s:3) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,584 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165645 (Enterococcus faecalis, A02110_t:21_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,593 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164085 (Enterococcus faecalis, A02110_t:7_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,603 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163775 (Enterococcus faecalis, A02110_t:4_s:3) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,613 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163765 (Enterococcus faecalis, A02110_t:4_s:4) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,622 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902162955 (Enterococcus faecalis, A02110_t:4_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,632 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159995 (Enterococcus faecalis, A01311_t:21_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,642 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159965 (Enterococcus faecalis, A01311_t:7_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,652 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164865 (Enterococcus faecalis, A02053_t:21_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,756 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165705 (Enterococcus faecalis, A01939_t:4_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,770 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165675 (Enterococcus faecalis, A01939_t:21_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,783 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165625 (Enterococcus faecalis, A01921_t:7_s:4) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,793 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165605 (Enterococcus faecalis, A01921_t:7_s:3) loaded with 74 markers.\n",
      "2024-02-25 13:21:28,802 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165595 (Enterococcus faecalis, A01921_t:7_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,812 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165505 (Enterococcus faecalis, A01921_t:4_s:*) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,823 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165585 (Enterococcus faecalis, A01866_t:4_s:*) loaded with 81 markers.\n",
      "2024-02-25 13:21:28,834 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165545 (Enterococcus faecalis, A01866_t:7_s:3) loaded with 84 markers.\n",
      "2024-02-25 13:21:28,845 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165515 (Enterococcus faecalis, A01866_t:7_s:2) loaded with 81 markers.\n",
      "2024-02-25 13:21:28,857 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165495 (Enterococcus faecalis, A01862_t:21_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,870 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165425 (Enterococcus faecalis, A01862_t:21_s:4) loaded with 80 markers.\n",
      "2024-02-25 13:21:28,879 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159985 (Enterococcus faecalis, A01805_t:4_s:*) loaded with 73 markers.\n",
      "2024-02-25 13:21:28,891 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165475 (Enterococcus faecalis, A01676_t:7_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,904 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165455 (Enterococcus faecalis, A01676_t:7_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,914 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163605 (Enterococcus faecalis, A01671_t:21_s:3) loaded with 72 markers.\n",
      "2024-02-25 13:21:28,923 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163595 (Enterococcus faecalis, A01671_t:21_s:5) loaded with 73 markers.\n",
      "2024-02-25 13:21:28,932 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163575 (Enterococcus faecalis, A01671_t:21_s:4) loaded with 72 markers.\n",
      "2024-02-25 13:21:28,944 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165555 (Enterococcus faecalis, A01653_t:21_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:28,954 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163085 (Enterococcus faecalis, A01586_t:21_s:2) loaded with 75 markers.\n",
      "2024-02-25 13:21:28,963 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164915 (Enterococcus faecalis, A01583_t:7_s:2) loaded with 77 markers.\n",
      "2024-02-25 13:21:28,973 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163055 (Enterococcus faecalis, A01583_t:7_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,983 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158885 (Enterococcus faecalis, A01506_t:4_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:28,993 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160025 (Enterococcus faecalis, A01301_t:21_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,005 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165415 (Enterococcus faecalis, A01176_t:21_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,017 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165405 (Enterococcus faecalis, A01176_t:21_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,028 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165395 (Enterococcus faecalis, A01176_t:7_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,038 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166365 (Enterococcus faecalis, A01173_t:7_s:2) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,050 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165305 (Enterococcus faecalis, A01107_t:M_s:4) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,064 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165265 (Enterococcus faecalis, A01107_t:M_s:5) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,077 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165255 (Enterococcus faecalis, A01107_t:M_s:1) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,086 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165355 (Enterococcus faecalis, A01105_t:21_s:3) loaded with 75 markers.\n",
      "2024-02-25 13:21:29,096 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165225 (Enterococcus faecalis, A01105_t:21_s:1) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,106 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163665 (Enterococcus faecalis, A01105_t:4_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,115 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158955 (Enterococcus faecalis, A01105_t:7_s:*) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,126 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166355 (Enterococcus faecalis, A01082_t:7_s:7) loaded with 82 markers.\n",
      "2024-02-25 13:21:29,139 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165285 (Enterococcus faecalis, A01082_t:4_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,150 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161095 (Enterococcus faecalis, A01082_t:21_s:5) loaded with 82 markers.\n",
      "2024-02-25 13:21:29,161 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902161045 (Enterococcus faecalis, A01082_t:21_s:4) loaded with 82 markers.\n",
      "2024-02-25 13:21:29,172 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902166255 (Enterococcus faecalis, A00908_T1_t:4_s:9) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,183 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165135 (Enterococcus faecalis, A00908_T1_t:21_s:4) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,194 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163105 (Enterococcus faecalis, A00908_T1_t:4_s:5) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,205 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163075 (Enterococcus faecalis, A00908_T1_t:4_s:6) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,216 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163065 (Enterococcus faecalis, A00908_T1_t:4_s:7) loaded with 76 markers.\n",
      "2024-02-25 13:21:29,228 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165175 (Enterococcus faecalis, A00576_t:M_s:3) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,240 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165075 (Enterococcus faecalis, A00576_t:21_s:3) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,251 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165025 (Enterococcus faecalis, A00576_t:M_s:2) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,262 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902158775 (Enterococcus faecalis, A00219_t:21_s:1) loaded with 80 markers.\n",
      "2024-02-25 13:21:29,272 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902160495 (Enterococcus faecalis, B00119_t:7_s:3) loaded with 72 markers.\n",
      "2024-02-25 13:21:29,282 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902159545 (Enterococcus faecalis, B00116_t:7_s:4) loaded with 77 markers.\n",
      "2024-02-25 13:21:29,292 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164815 (Enterococcus faecalis, B00092_t:21_s:6) loaded with 77 markers.\n",
      "2024-02-25 13:21:29,302 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164805 (Enterococcus faecalis, B00092_t:21_s:5) loaded with 77 markers.\n",
      "2024-02-25 13:21:29,314 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165825 (Enterococcus faecalis, B00090_t:4_s:*) loaded with 80 markers.\n",
      "2024-02-25 13:21:29,323 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902163615 (Enterococcus faecalis, A00043_t:7_s:4) loaded with 73 markers.\n",
      "2024-02-25 13:21:29,332 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164995 (Enterococcus faecalis, A00031_t:21_s:1) loaded with 73 markers.\n",
      "2024-02-25 13:21:29,344 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902165005 (Enterococcus faecalis, A00021_T2_t:4_s:*) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,356 [DEBUG - chronostrain.database.parser.json] - Strain GCA_902164875 (Enterococcus faecalis, A00021_T1_t:21_s:4) loaded with 79 markers.\n",
      "2024-02-25 13:21:29,363 [INFO - chronostrain.database.database] - Instantiating database `efaecalis`.\n",
      "2024-02-25 13:21:30,190 [DEBUG - chronostrain.database.parser.json] - Using `DictionaryBackend` for database backend.\n",
      "2024-02-25 13:21:30,190 [INFO - chronostrain.cli.prune_json] - Computing all-to-all Jaccard distance calculations.\n",
      "2024-02-25 13:21:30,190 [INFO - chronostrain.cli.prune_json] - Using ProbMinHash method for sketching (sz=8192).\n",
      "2024-02-25 13:21:36,051 [WARNING - chronostrain.cli.prune_json] - Strain LR994474.1 (Enterococcus hirae, Enterococcus hirae canine HQ MAG) -- marker rpoC has an N. This strain will be excluded.\n",
      "2024-02-25 13:21:36,051 [WARNING - chronostrain.cli.prune_json] - Strain LR994474.1 (Enterococcus hirae, Enterococcus hirae canine HQ MAG) -- marker pnp has an N. This strain will be excluded.\n",
      "2024-02-25 13:21:36,052 [WARNING - chronostrain.cli.prune_json] - Strain LR994474.1 (Enterococcus hirae, Enterococcus hirae canine HQ MAG) -- marker gap-gyd has an N. This strain will be excluded.\n",
      "2024-02-25 13:22:12,783 [INFO - chronostrain.cli.prune_json] - Computing clusters.\n",
      "2024-02-25 13:22:20,756 [INFO - chronostrain.cli.prune_json] - Before clustering: 2546 genomes\n",
      "2024-02-25 13:22:20,756 [INFO - chronostrain.cli.prune_json] - After clustering: 934 genomes\n",
      "2024-02-25 13:22:20,756 [DEBUG - chronostrain.cli.prune_json] - Cleaning up. Removing tmpdir = /mnt/e/infant_nt/database/chronostrain_files/__cluster_tmp\n"
     ]
    }
   ],
   "source": [
    "# Perform clustering\n",
    "thresh = 0.9999\n",
    "!env \\\n",
    "    JAX_PLATFORM_NAME=cpu \\\n",
    "    CHRONOSTRAIN_DB_DIR=\"$CHRONOSTRAIN_DB_DIR\" \\\n",
    "    CHRONOSTRAIN_LOG_INI={_cwd}/logging.ini \\\n",
    "    chronostrain -c chronostrain.ini \\\n",
    "      cluster-db \\\n",
    "      -i $CHRONOSTRAIN_TARGET_JSON \\\n",
    "      -o $CHRONOSTRAIN_TARGET_CLUSTERS \\\n",
    "      --ident-threshold {thresh}"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ed936012-425e-4fe2-a414-6a790951dbc2",
   "metadata": {},
   "source": [
    "# OPTIONAL: Compute some statistics."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "8a6eaaa4-e4e8-4311-a22d-0e74888bb807",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2024-02-28 08:43:02,578 [INFO - chronostrain.database.database] - Instantiating database `efaecalis`.\n"
     ]
    }
   ],
   "source": [
    "from chronostrain.database import JSONParser\n",
    "src_db = JSONParser(\n",
    "    entries_file=CHRONOSTRAIN_TARGET_JSON,\n",
    "    data_dir=CHRONOSTRAIN_DB_DIR,\n",
    "    marker_max_len=50000,\n",
    "    force_refresh=False\n",
    ").parse()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "id": "c0205f9d-38ea-4bf3-8ef2-3940db05e1c8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# clusters = 934\n",
      "# efaecalis clusters = 625\n"
     ]
    }
   ],
   "source": [
    "df_entries = []\n",
    "with open(CHRONOSTRAIN_TARGET_CLUSTERS, \"rt\") as f:\n",
    "    for line in f:\n",
    "        if line.startswith(\"#\"):\n",
    "            continue\n",
    "        tokens = line.rstrip().split(\"\\t\")\n",
    "        rep = tokens[0]\n",
    "        members = tokens[1].split(\",\")\n",
    "        for member in members:\n",
    "            # if member.startswith(\"GCA\"):  # only include isolates\n",
    "            df_entries.append({'Accession': member, 'Cluster': rep})\n",
    "cluster_df = pd.DataFrame(df_entries)\n",
    "del df_entries\n",
    "print(\"# clusters = {}\".format(len(pd.unique(cluster_df['Cluster']))))\n",
    "\n",
    "\n",
    "n_efaecalis = 0\n",
    "for cluster_id in pd.unique(cluster_df['Cluster']):\n",
    "    s = src_db.get_strain(cluster_id)\n",
    "    if s.metadata.genus == 'Enterococcus' and s.metadata.species == 'faecalis':\n",
    "        n_efaecalis += 1\n",
    "print(\"# efaecalis clusters = {}\".format(n_efaecalis))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "chronostrain",
   "language": "python",
   "name": "chronostrain"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
