{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    },
    "accelerator": "GPU",
    "gpuClass": "standard"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# Install the package."
      ],
      "metadata": {
        "id": "zSobWNLTo-0m"
      }
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "-lEmmvc7blyc",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "7813a59d-6739-497f-c777-a4cb8b7341fd"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "fatal: destination path 'chronostrain' already exists and is not an empty directory.\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Processing ./chronostrain\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: torch in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.13.1+cu116)\n",
            "Collecting torch-scatter\n",
            "  Using cached torch_scatter-2.1.0.tar.gz (106 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting torch-sparse\n",
            "  Using cached torch_sparse-0.6.16.tar.gz (208 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Requirement already satisfied: pandas>=1.1.3 in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.3.5)\n",
            "Requirement already satisfied: numpy>=1.19.2 in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.21.6)\n",
            "Requirement already satisfied: numba in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (0.56.4)\n",
            "Requirement already satisfied: scipy>=1.6.1 in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.7.3)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.0.2)\n",
            "Collecting matplotlib>=3.3.2\n",
            "  Using cached matplotlib-3.6.3-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl (9.4 MB)\n",
            "Requirement already satisfied: seaborn>=0.11.0 in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (0.11.2)\n",
            "Requirement already satisfied: joblib>=1.0.1 in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (1.2.0)\n",
            "Collecting biopython>=1.78\n",
            "  Using cached biopython-1.80-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (3.1 MB)\n",
            "Requirement already satisfied: networkx in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (3.0)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.8/dist-packages (from chronostrain==0.4.0) (8.1.3)\n",
            "Collecting fonttools>=4.22.0\n",
            "  Using cached fonttools-4.38.0-py3-none-any.whl (965 kB)\n",
            "Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (23.0)\n",
            "Requirement already satisfied: python-dateutil>=2.7 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (2.8.2)\n",
            "Requirement already satisfied: cycler>=0.10 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (0.11.0)\n",
            "Requirement already satisfied: pyparsing>=2.2.1 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (3.0.9)\n",
            "Collecting contourpy>=1.0.1\n",
            "  Using cached contourpy-1.0.7-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (300 kB)\n",
            "Requirement already satisfied: pillow>=6.2.0 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (7.1.2)\n",
            "Requirement already satisfied: kiwisolver>=1.0.1 in /usr/local/lib/python3.8/dist-packages (from matplotlib>=3.3.2->chronostrain==0.4.0) (1.4.4)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.8/dist-packages (from pandas>=1.1.3->chronostrain==0.4.0) (2022.7.1)\n",
            "Requirement already satisfied: setuptools in /usr/local/lib/python3.8/dist-packages (from numba->chronostrain==0.4.0) (57.4.0)\n",
            "Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.8/dist-packages (from numba->chronostrain==0.4.0) (6.0.0)\n",
            "Requirement already satisfied: llvmlite<0.40,>=0.39.0dev0 in /usr/local/lib/python3.8/dist-packages (from numba->chronostrain==0.4.0) (0.39.1)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.8/dist-packages (from scikit-learn->chronostrain==0.4.0) (3.1.0)\n",
            "Requirement already satisfied: typing-extensions in /usr/local/lib/python3.8/dist-packages (from torch->chronostrain==0.4.0) (4.4.0)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.8/dist-packages (from python-dateutil>=2.7->matplotlib>=3.3.2->chronostrain==0.4.0) (1.15.0)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.8/dist-packages (from importlib-metadata->numba->chronostrain==0.4.0) (3.12.0)\n",
            "Building wheels for collected packages: chronostrain, torch-scatter, torch-sparse\n",
            "  Building wheel for chronostrain (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for chronostrain: filename=chronostrain-0.4.0-py3-none-any.whl size=192415 sha256=ff449040a9185b8f5788dbbc1869be12400c5fe66c73ea9b7fb0530c95aa7449\n",
            "  Stored in directory: /tmp/pip-ephem-wheel-cache-3r5cyobg/wheels/78/10/76/071ba5883eabe3611b003f4e5fa685a60311ca02af59710fc3\n"
          ]
        }
      ],
      "source": [
        "!git clone https://github.com/gibsonlab/chronostrain\n",
        "%pip install chronostrain/.\n",
        "\n",
        "# Colab-specific installation commands\n",
        "%pip install click==8.1.3 --force-reinstall\n",
        "!sudo apt -y install bwa"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "# Example usage"
      ],
      "metadata": {
        "id": "b9SUYA9OpCnL"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "!chronostrain -h"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "jDfR0UP2796v",
        "outputId": "f87ca758-9926-4faa-e26a-1ec62ad088d1"
      },
      "execution_count": 59,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Usage: chronostrain [OPTIONS] COMMAND [ARGS]...\n",
            "\n",
            "  ChronoStrain (Time-Series Metagenomic Abundance\n",
            "  Estimation) Contact: Younhun Kim\n",
            "  (younhun@mit.edu)\n",
            "\n",
            "Options:\n",
            "  -c, --config FILE  The path to a chronostrain\n",
            "                     INI configuration file.\n",
            "  -h, --help         Show this message and exit.\n",
            "\n",
            "Commands:\n",
            "  advi           Perform posterior estimation...\n",
            "  filter         Perform filtering on a...\n",
            "  filter-single  Filter a single read file.\n",
            "  make-db        Perform posterior estimation...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "Use the -c option to pass in a configuration file, along with a command."
      ],
      "metadata": {
        "id": "QrDJ8QJOpFV0"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "%env CHRONOSTRAIN_INI=test_inputs/chronostrain.ini\n",
        "%env ENTREZ_EMAIL=younhun@mit.edu\n",
        "%env CHRONOSTRAIN_LOG_INI=chronostrain/examples/example_configs/log_config.ini\n",
        "!chronostrain filter -r ./test_inputs/test_inputs.csv -o ./test_output --aligner bwa"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "NRMKiP-F8GT4",
        "outputId": "83507ed0-6ed4-42a9-9d82-e2162f04f53f"
      },
      "execution_count": 79,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "env: CHRONOSTRAIN_INI=test_inputs/chronostrain.ini\n",
            "env: ENTREZ_EMAIL=younhun@mit.edu\n",
            "env: CHRONOSTRAIN_LOG_INI=chronostrain/examples/example_configs/log_config.ini\n",
            "2023-02-02 20:18:30,160 [DEBUG - chronostrain.logging.initialize] - Using logging configuration chronostrain/examples/example_configs/log_config.ini\n",
            "2023-02-02 20:18:30,180 [INFO - chronostrain.cli] - Performing filtering to timeseries dataset `test_inputs/test_inputs.csv`.\n",
            "2023-02-02 20:18:31,924 [INFO - chronostrain.util.entrez] - Using email `younhun@mit.edu` for Entrez.\n",
            "2023-02-02 20:18:31,924 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from test_inputs/chronostrain.ini.\n",
            "2023-02-02 20:18:31,930 [INFO - chronostrain.util.alignments.pairwise.wrappers] - If invoked, bowtie2 will initialize using default setting `phred33`. (TODO: implement some flexibility here.)\n",
            "2023-02-02 20:18:31,938 [DEBUG - chronostrain.database.database] - Populating database.\n",
            "2023-02-02 20:18:31,938 [DEBUG - chronostrain.database.database] - Initializing db backend `PandasAssistedBackend`\n",
            "2023-02-02 20:18:31,938 [INFO - chronostrain.database.parser.json] - Loading from JSON marker database file test_inputs/example.json.\n",
            "2023-02-02 20:18:31,943 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP019177.1 (Salmonella enterica, ATCC_51960) loaded with 11 markers.\n",
            "2023-02-02 20:18:31,945 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP080539.1 (Citrobacter braakii, DY2019) loaded with 10 markers.\n",
            "2023-02-02 20:18:31,948 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP055315.1 (Klebsiella grimontii, RHBSTW-00555) loaded with 11 markers.\n",
            "2023-02-02 20:18:31,950 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP070534.1 (Enterobacter hormaechei, EC48293) loaded with 9 markers.\n",
            "2023-02-02 20:18:31,952 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043222.1 (Salmonella enterica, SA20114778WT) loaded with 10 markers.\n",
            "2023-02-02 20:18:31,953 [DEBUG - chronostrain.util.entrez] - [NZ_CP058207.1] Downloading entrez file (NZ_CP058207.1.fasta)...\n",
            "2023-02-02 20:18:32,613 [DEBUG - chronostrain.util.entrez] - [NZ_CP058207.1] download completed. (4.44 MB)\n",
            "2023-02-02 20:18:32,669 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP058207.1 (Escherichia marmotae, RHB42-C09) loaded with 26 markers.\n",
            "2023-02-02 20:18:32,670 [DEBUG - chronostrain.util.entrez] - [NZ_LR130543.1] Downloading entrez file (NZ_LR130543.1.fasta)...\n",
            "2023-02-02 20:18:33,287 [DEBUG - chronostrain.util.entrez] - [NZ_LR130543.1] download completed. (5.19 MB)\n",
            "2023-02-02 20:18:33,345 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR130543.1 (Klebsiella variicola, 04153260899A) loaded with 10 markers.\n",
            "2023-02-02 20:18:33,345 [DEBUG - chronostrain.util.entrez] - [NZ_CP027363.1] Downloading entrez file (NZ_CP027363.1.fasta)...\n",
            "2023-02-02 20:18:34,069 [DEBUG - chronostrain.util.entrez] - [NZ_CP027363.1] download completed. (5.03 MB)\n",
            "2023-02-02 20:18:34,127 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027363.1 (Escherichia coli, 88-3001) loaded with 39 markers.\n",
            "2023-02-02 20:18:34,127 [DEBUG - chronostrain.util.entrez] - [NZ_CP007215.2] Downloading entrez file (NZ_CP007215.2.fasta)...\n",
            "2023-02-02 20:18:34,882 [DEBUG - chronostrain.util.entrez] - [NZ_CP007215.2] download completed. (4.74 MB)\n",
            "2023-02-02 20:18:34,923 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP007215.2 (Kosakonia sacchari, SP1) loaded with 9 markers.\n",
            "2023-02-02 20:18:34,923 [DEBUG - chronostrain.util.entrez] - [NZ_CP077411.1] Downloading entrez file (NZ_CP077411.1.fasta)...\n",
            "2023-02-02 20:18:35,589 [DEBUG - chronostrain.util.entrez] - [NZ_CP077411.1] download completed. (4.55 MB)\n",
            "2023-02-02 20:18:35,619 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP077411.1 (Enterobacter asburiae, FDAARGOS_1432) loaded with 8 markers.\n",
            "2023-02-02 20:18:35,620 [DEBUG - chronostrain.util.entrez] - [NZ_CP053211.1] Downloading entrez file (NZ_CP053211.1.fasta)...\n",
            "2023-02-02 20:18:36,281 [DEBUG - chronostrain.util.entrez] - [NZ_CP053211.1] download completed. (4.43 MB)\n",
            "2023-02-02 20:18:36,360 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP053211.1 (Escherichia coli, QDFD216) loaded with 40 markers.\n",
            "2023-02-02 20:18:36,360 [DEBUG - chronostrain.util.entrez] - [NZ_LN868943.1] Downloading entrez file (NZ_LN868943.1.fasta)...\n",
            "2023-02-02 20:18:36,967 [DEBUG - chronostrain.util.entrez] - [NZ_LN868943.1] download completed. (3.62 MB)\n",
            "2023-02-02 20:18:36,998 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LN868943.1 (Salmonella enterica, NCTC10384) loaded with 10 markers.\n",
            "2023-02-02 20:18:36,998 [DEBUG - chronostrain.util.entrez] - [NZ_CP056717.1] Downloading entrez file (NZ_CP056717.1.fasta)...\n",
            "2023-02-02 20:18:37,647 [DEBUG - chronostrain.util.entrez] - [NZ_CP056717.1] download completed. (4.94 MB)\n",
            "2023-02-02 20:18:37,690 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP056717.1 (Citrobacter sp., RHBSTW-00599) loaded with 9 markers.\n",
            "2023-02-02 20:18:37,691 [DEBUG - chronostrain.util.entrez] - [NZ_CP091319.1] Downloading entrez file (NZ_CP091319.1.fasta)...\n",
            "2023-02-02 20:18:38,289 [DEBUG - chronostrain.util.entrez] - [NZ_CP091319.1] download completed. (4.56 MB)\n",
            "2023-02-02 20:18:38,325 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091319.1 (Enterobacter sp., DR25) loaded with 11 markers.\n",
            "2023-02-02 20:18:38,326 [DEBUG - chronostrain.util.entrez] - [NZ_CP033343.1] Downloading entrez file (NZ_CP033343.1.fasta)...\n",
            "2023-02-02 20:18:39,007 [DEBUG - chronostrain.util.entrez] - [NZ_CP033343.1] download completed. (4.56 MB)\n",
            "2023-02-02 20:18:39,049 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP033343.1 (Salmonella enterica, CFSAN076213) loaded with 11 markers.\n",
            "2023-02-02 20:18:39,049 [DEBUG - chronostrain.util.entrez] - [NZ_CP063369.1] Downloading entrez file (NZ_CP063369.1.fasta)...\n",
            "2023-02-02 20:18:39,782 [DEBUG - chronostrain.util.entrez] - [NZ_CP063369.1] download completed. (4.68 MB)\n",
            "2023-02-02 20:18:39,859 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063369.1 (Escherichia coli, 1409) loaded with 41 markers.\n",
            "2023-02-02 20:18:39,859 [DEBUG - chronostrain.util.entrez] - [NZ_CP027677.1] Downloading entrez file (NZ_CP027677.1.fasta)...\n",
            "2023-02-02 20:18:40,511 [DEBUG - chronostrain.util.entrez] - [NZ_CP027677.1] download completed. (4.73 MB)\n",
            "2023-02-02 20:18:40,554 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP027677.1 (Salmonella enterica, 12-01738) loaded with 11 markers.\n",
            "2023-02-02 20:18:40,554 [DEBUG - chronostrain.util.entrez] - [NZ_CP074159.1] Downloading entrez file (NZ_CP074159.1.fasta)...\n",
            "2023-02-02 20:18:41,273 [DEBUG - chronostrain.util.entrez] - [NZ_CP074159.1] download completed. (4.54 MB)\n",
            "2023-02-02 20:18:41,300 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP074159.1 (Enterobacter sp., JBIWA005) loaded with 6 markers.\n",
            "2023-02-02 20:18:41,301 [DEBUG - chronostrain.util.entrez] - [NZ_CP087122.1] Downloading entrez file (NZ_CP087122.1.fasta)...\n",
            "2023-02-02 20:18:42,234 [DEBUG - chronostrain.util.entrez] - [NZ_CP087122.1] download completed. (5.35 MB)\n",
            "2023-02-02 20:18:42,276 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP087122.1 (Klebsiella pneumoniae, TH12852) loaded with 8 markers.\n",
            "2023-02-02 20:18:42,276 [DEBUG - chronostrain.util.entrez] - [NZ_CP071393.1] Downloading entrez file (NZ_CP071393.1.fasta)...\n",
            "2023-02-02 20:18:43,212 [DEBUG - chronostrain.util.entrez] - [NZ_CP071393.1] download completed. (5.73 MB)\n",
            "2023-02-02 20:18:43,257 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP071393.1 (Klebsiella michiganensis, Kmfe267) loaded with 10 markers.\n",
            "2023-02-02 20:18:43,258 [DEBUG - chronostrain.util.entrez] - [NZ_CP042585.1] Downloading entrez file (NZ_CP042585.1.fasta)...\n",
            "2023-02-02 20:18:43,928 [DEBUG - chronostrain.util.entrez] - [NZ_CP042585.1] download completed. (4.58 MB)\n",
            "2023-02-02 20:18:44,009 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP042585.1 (Escherichia coli, LD91-1) loaded with 41 markers.\n",
            "2023-02-02 20:18:44,009 [DEBUG - chronostrain.util.entrez] - [NZ_LR134196.1] Downloading entrez file (NZ_LR134196.1.fasta)...\n",
            "2023-02-02 20:18:44,698 [DEBUG - chronostrain.util.entrez] - [NZ_LR134196.1] download completed. (4.96 MB)\n",
            "2023-02-02 20:18:44,741 [DEBUG - chronostrain.database.parser.json] - Strain NZ_LR134196.1 (Klebsiella quasipneumoniae, NCTC11357) loaded with 12 markers.\n",
            "2023-02-02 20:18:44,742 [DEBUG - chronostrain.util.entrez] - [NZ_CP064656.1] Downloading entrez file (NZ_CP064656.1.fasta)...\n",
            "2023-02-02 20:18:45,389 [DEBUG - chronostrain.util.entrez] - [NZ_CP064656.1] download completed. (4.44 MB)\n",
            "2023-02-02 20:18:45,422 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP064656.1 (Enterobacter hormaechei, 3804) loaded with 10 markers.\n",
            "2023-02-02 20:18:45,422 [DEBUG - chronostrain.util.entrez] - [NZ_CP091492.1] Downloading entrez file (NZ_CP091492.1.fasta)...\n",
            "2023-02-02 20:18:46,088 [DEBUG - chronostrain.util.entrez] - [NZ_CP091492.1] download completed. (4.74 MB)\n",
            "2023-02-02 20:18:46,119 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP091492.1 (Enterobacter cloacae, ECL405) loaded with 7 markers.\n",
            "2023-02-02 20:18:46,119 [DEBUG - chronostrain.util.entrez] - [NZ_CP022912.1] Downloading entrez file (NZ_CP022912.1.fasta)...\n",
            "2023-02-02 20:18:46,761 [DEBUG - chronostrain.util.entrez] - [NZ_CP022912.1] download completed. (4.63 MB)\n",
            "2023-02-02 20:18:46,832 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP022912.1 (Escherichia coli, 2011EL-1370-2) loaded with 39 markers.\n",
            "2023-02-02 20:18:46,832 [DEBUG - chronostrain.util.entrez] - [NZ_CP017849.1] Downloading entrez file (NZ_CP017849.1.fasta)...\n",
            "2023-02-02 20:18:48,206 [DEBUG - chronostrain.util.entrez] - [NZ_CP017849.1] download completed. (5.42 MB)\n",
            "2023-02-02 20:18:48,247 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP017849.1 (Klebsiella variicola, GJ2) loaded with 11 markers.\n",
            "2023-02-02 20:18:48,248 [DEBUG - chronostrain.util.entrez] - [NZ_CP043662.1] Downloading entrez file (NZ_CP043662.1.fasta)...\n",
            "2023-02-02 20:18:48,934 [DEBUG - chronostrain.util.entrez] - [NZ_CP043662.1] download completed. (4.74 MB)\n",
            "2023-02-02 20:18:48,971 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP043662.1 (Salmonella enterica, 159838) loaded with 10 markers.\n",
            "2023-02-02 20:18:48,972 [DEBUG - chronostrain.util.entrez] - [NZ_CP010242.1] Downloading entrez file (NZ_CP010242.1.fasta)...\n",
            "2023-02-02 20:18:49,651 [DEBUG - chronostrain.util.entrez] - [NZ_CP010242.1] download completed. (4.59 MB)\n",
            "2023-02-02 20:18:49,715 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP010242.1 (Escherichia coli, S56) loaded with 40 markers.\n",
            "2023-02-02 20:18:49,715 [DEBUG - chronostrain.util.entrez] - [NZ_CP067261.1] Downloading entrez file (NZ_CP067261.1.fasta)...\n",
            "2023-02-02 20:18:50,450 [DEBUG - chronostrain.util.entrez] - [NZ_CP067261.1] download completed. (5.3 MB)\n",
            "2023-02-02 20:18:50,495 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP067261.1 (Klebsiella variicola, E25) loaded with 11 markers.\n",
            "2023-02-02 20:18:50,495 [DEBUG - chronostrain.util.entrez] - [NZ_CP063492.1] Downloading entrez file (NZ_CP063492.1.fasta)...\n",
            "2023-02-02 20:18:51,157 [DEBUG - chronostrain.util.entrez] - [NZ_CP063492.1] download completed. (5.21 MB)\n",
            "2023-02-02 20:18:51,223 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP063492.1 (Escherichia coli, EF5-18-41) loaded with 36 markers.\n",
            "2023-02-02 20:18:51,224 [DEBUG - chronostrain.util.entrez] - [NZ_CP068155.1] Downloading entrez file (NZ_CP068155.1.fasta)...\n",
            "2023-02-02 20:18:51,897 [DEBUG - chronostrain.util.entrez] - [NZ_CP068155.1] download completed. (4.78 MB)\n",
            "2023-02-02 20:18:51,966 [DEBUG - chronostrain.database.parser.json] - Strain NZ_CP068155.1 (Escherichia coli, FDAARGOS_1077) loaded with 40 markers.\n",
            "2023-02-02 20:18:51,969 [INFO - chronostrain.database.database] - Loaded 31 strains in 0.3 minutes.\n",
            "2023-02-02 20:18:52,059 [DEBUG - chronostrain.database.database] - Multi-fasta file: chronostrain_db/__example_MARKERS/all_markers.fasta\n",
            "2023-02-02 20:18:52,065 [DEBUG - chronostrain.database.database] - Saved database to test_inputs/example.posix.pkl.\n",
            "2023-02-02 20:18:52,065 [INFO - chronostrain.cli] - Target index file: test_output/filtered_test_inputs.csv\n",
            "2023-02-02 20:18:52,065 [DEBUG - chronostrain.cli] - Parsing time-series reads from test_inputs/test_inputs.csv\n",
            "2023-02-02 20:18:52,065 [INFO - chronostrain.cli] - Applying filter to timepoint 343.0, test_inputs/t1_paired_1.fastq\n",
            "2023-02-02 20:18:52,065 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa index chronostrain_db/__example_MARKERS/all_markers.fasta\n",
            "2023-02-02 20:18:52,560 [DEBUG - chronostrain.util.external.commandline] - STDERR: [bwa_index] Pack FASTA... 0.01 sec\n",
            "[bwa_index] Construct BWT for the packed sequence...\n",
            "[bwa_index] 0.11 seconds elapse.\n",
            "[bwa_index] Update BWT... 0.00 sec\n",
            "[bwa_index] Pack forward-only FASTA... 0.00 sec\n",
            "[bwa_index] Construct SA from BWT and Occ... 0.07 sec\n",
            "[main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa index chronostrain_db/__example_MARKERS/all_markers.fasta\n",
            "[main] Real time: 0.483 sec; CPU: 0.203 sec\n",
            "2023-02-02 20:18:52,562 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t1_paired_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_paired_1.fastq\n",
            "2023-02-02 20:18:52,796 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t1_paired_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_paired_1.fastq\n",
            "[main] Real time: 0.211 sec; CPU: 0.197 sec\n",
            "2023-02-02 20:18:52,803 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t1_paired_1.sam\n",
            "2023-02-02 20:18:52,803 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 723 alignments from t1_paired_1.sam\n",
            "2023-02-02 20:18:53,346 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t1_paired_1.sam -- Total # SAM lines parsed: 723; # mapped lines: 723\n",
            "2023-02-02 20:18:53,346 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 448\n",
            "2023-02-02 20:18:53,347 [INFO - chronostrain.cli] - Applying filter to timepoint 343.0, test_inputs/t1_unmatched_1.fastq\n",
            "2023-02-02 20:18:53,347 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:53,347 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t1_unmatched_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_unmatched_1.fastq\n",
            "2023-02-02 20:18:53,377 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t1_unmatched_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_unmatched_1.fastq\n",
            "[main] Real time: 0.018 sec; CPU: 0.026 sec\n",
            "2023-02-02 20:18:53,378 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t1_unmatched_1.sam\n",
            "2023-02-02 20:18:53,379 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 96 alignments from t1_unmatched_1.sam\n",
            "2023-02-02 20:18:53,431 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t1_unmatched_1.sam -- Total # SAM lines parsed: 96; # mapped lines: 96\n",
            "2023-02-02 20:18:53,431 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 65\n",
            "2023-02-02 20:18:53,431 [INFO - chronostrain.cli] - Applying filter to timepoint 343.0, test_inputs/t1_paired_2.fastq\n",
            "2023-02-02 20:18:53,431 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:53,431 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t1_paired_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_paired_2.fastq\n",
            "2023-02-02 20:18:53,540 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t1_paired_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_paired_2.fastq\n",
            "[main] Real time: 0.097 sec; CPU: 0.182 sec\n",
            "2023-02-02 20:18:53,547 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t1_paired_2.sam\n",
            "2023-02-02 20:18:53,547 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 709 alignments from t1_paired_2.sam\n",
            "2023-02-02 20:18:53,978 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t1_paired_2.sam -- Total # SAM lines parsed: 709; # mapped lines: 709\n",
            "2023-02-02 20:18:53,978 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 453\n",
            "2023-02-02 20:18:53,978 [INFO - chronostrain.cli] - Applying filter to timepoint 343.0, test_inputs/t1_unmatched_2.fastq\n",
            "2023-02-02 20:18:53,979 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:53,979 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t1_unmatched_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_unmatched_2.fastq\n",
            "2023-02-02 20:18:53,998 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t1_unmatched_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t1_unmatched_2.fastq\n",
            "[main] Real time: 0.008 sec; CPU: 0.009 sec\n",
            "2023-02-02 20:18:53,998 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t1_unmatched_2.sam\n",
            "2023-02-02 20:18:53,999 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 0 alignments from t1_unmatched_2.sam\n",
            "2023-02-02 20:18:53,999 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t1_unmatched_2.sam -- Total # SAM lines parsed: 0; # mapped lines: 0\n",
            "2023-02-02 20:18:53,999 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 0\n",
            "2023-02-02 20:18:53,999 [INFO - chronostrain.cli] - Applying filter to timepoint 369.0, test_inputs/t2_paired_1.fastq\n",
            "2023-02-02 20:18:53,999 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:53,999 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t2_paired_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_paired_1.fastq\n",
            "2023-02-02 20:18:54,017 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t2_paired_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_paired_1.fastq\n",
            "[main] Real time: 0.007 sec; CPU: 0.009 sec\n",
            "2023-02-02 20:18:54,017 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t2_paired_1.sam\n",
            "2023-02-02 20:18:54,018 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 10 alignments from t2_paired_1.sam\n",
            "2023-02-02 20:18:54,022 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t2_paired_1.sam -- Total # SAM lines parsed: 10; # mapped lines: 10\n",
            "2023-02-02 20:18:54,022 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 4\n",
            "2023-02-02 20:18:54,023 [INFO - chronostrain.cli] - Applying filter to timepoint 369.0, test_inputs/t2_unmatched_1.fastq\n",
            "2023-02-02 20:18:54,023 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:54,023 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t2_unmatched_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_unmatched_1.fastq\n",
            "2023-02-02 20:18:54,040 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t2_unmatched_1_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_unmatched_1.fastq\n",
            "[main] Real time: 0.007 sec; CPU: 0.010 sec\n",
            "2023-02-02 20:18:54,040 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t2_unmatched_1.sam\n",
            "2023-02-02 20:18:54,040 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 13 alignments from t2_unmatched_1.sam\n",
            "2023-02-02 20:18:54,046 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t2_unmatched_1.sam -- Total # SAM lines parsed: 13; # mapped lines: 13\n",
            "2023-02-02 20:18:54,046 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 9\n",
            "2023-02-02 20:18:54,046 [INFO - chronostrain.cli] - Applying filter to timepoint 369.0, test_inputs/t2_paired_2.fastq\n",
            "2023-02-02 20:18:54,046 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:54,046 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t2_paired_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_paired_2.fastq\n",
            "2023-02-02 20:18:54,064 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t2_paired_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_paired_2.fastq\n",
            "[main] Real time: 0.007 sec; CPU: 0.010 sec\n",
            "2023-02-02 20:18:54,064 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t2_paired_2.sam\n",
            "2023-02-02 20:18:54,065 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 10 alignments from t2_paired_2.sam\n",
            "2023-02-02 20:18:54,070 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t2_paired_2.sam -- Total # SAM lines parsed: 10; # mapped lines: 10\n",
            "2023-02-02 20:18:54,070 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 5\n",
            "2023-02-02 20:18:54,070 [INFO - chronostrain.cli] - Applying filter to timepoint 369.0, test_inputs/t2_unmatched_2.fastq\n",
            "2023-02-02 20:18:54,070 [DEBUG - chronostrain.util.alignments.pairwise.wrappers] - pre-built bwa index found at chronostrain_db/__example_MARKERS/all_markers.bwa_trace\n",
            "2023-02-02 20:18:54,070 [DEBUG - chronostrain.util.external.commandline] - EXECUTE bwa mem -o test_output/tmp/t2_unmatched_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_unmatched_2.fastq\n",
            "2023-02-02 20:18:54,090 [DEBUG - chronostrain.util.external.commandline] - STDERR: [main] Version: 0.7.17-r1188\n",
            "[main] CMD: bwa mem -o test_output/tmp/t2_unmatched_2_bwa.sam -t 4 -k 15 -r 0.5 -w 10 -d 100 -A 2 -B 5 -L 0 -T 50 -U 0 -v 2 -O 0,0 -E 17,18 -Y chronostrain_db/__example_MARKERS/all_markers.fasta test_inputs/t2_unmatched_2.fastq\n",
            "[main] Real time: 0.007 sec; CPU: 0.010 sec\n",
            "2023-02-02 20:18:54,090 [DEBUG - chronostrain.cli.commands.filter.base] - Reading: t2_unmatched_2.sam\n",
            "2023-02-02 20:18:54,090 [DEBUG - chronostrain.util.alignments.pairwise.alignment] - Parsing 0 alignments from t2_unmatched_2.sam\n",
            "2023-02-02 20:18:54,091 [DEBUG - chronostrain.util.alignments.sam.sam_handler] - t2_unmatched_2.sam -- Total # SAM lines parsed: 0; # mapped lines: 0\n",
            "2023-02-02 20:18:54,091 [DEBUG - chronostrain.cli.commands.filter.base] - # passed reads: 0\n",
            "2023-02-02 20:18:54,091 [INFO - chronostrain.cli] - Finished filtering.\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%env CHRONOSTRAIN_INI=test_inputs/chronostrain.ini\n",
        "%env ENTREZ_EMAIL=younhun@mit.edu\n",
        "%env CHRONOSTRAIN_LOG_INI=chronostrain/examples/example_configs/log_config.ini\n",
        "!chronostrain advi -r test_output/filtered_test_inputs.csv -o test_output/inference"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MOhsWu7Qw0JF",
        "outputId": "894fe53d-cc04-41b5-ae07-70e70761e4da"
      },
      "execution_count": 81,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "env: CHRONOSTRAIN_INI=test_inputs/chronostrain.ini\n",
            "env: ENTREZ_EMAIL=younhun@mit.edu\n",
            "env: CHRONOSTRAIN_LOG_INI=chronostrain/examples/example_configs/log_config.ini\n",
            "2023-02-02 20:21:03,227 [DEBUG - chronostrain.logging.initialize] - Using logging configuration chronostrain/examples/example_configs/log_config.ini\n",
            "2023-02-02 20:21:03,249 [INFO - chronostrain.cli] - Pipeline for inference started.\n",
            "2023-02-02 20:21:04,678 [INFO - chronostrain.util.entrez] - Using email `younhun@mit.edu` for Entrez.\n",
            "2023-02-02 20:21:04,678 [DEBUG - chronostrain.config.initialize] - Loaded chronostrain INI from test_inputs/chronostrain.ini.\n",
            "Traceback (most recent call last):\n",
            "  File \"/usr/local/bin/chronostrain\", line 8, in <module>\n",
            "    sys.exit(main())\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/core.py\", line 1130, in __call__\n",
            "    return self.main(*args, **kwargs)\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/core.py\", line 1055, in main\n",
            "    rv = self.invoke(ctx)\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/core.py\", line 1657, in invoke\n",
            "    return _process_result(sub_ctx.command.invoke(sub_ctx))\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/core.py\", line 1404, in invoke\n",
            "    return ctx.invoke(self.callback, **ctx.params)\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/core.py\", line 760, in invoke\n",
            "    return __callback(*args, **kwargs)\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/click/decorators.py\", line 26, in new_func\n",
            "    return f(get_current_context(), *args, **kwargs)\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/cli/commands/inference/advi.py\", line 121, in main\n",
            "    import chronostrain.visualizations as viz\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/visualizations/__init__.py\", line 3, in <module>\n",
            "    from .bbvi_visualizations import *\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/visualizations/bbvi_visualizations.py\", line 9, in <module>\n",
            "    from chronostrain.algs import AbstractPosterior\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/algs/__init__.py\", line 1, in <module>\n",
            "    from .inference import AbstractModelSolver, \\\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/algs/inference/__init__.py\", line 1, in <module>\n",
            "    from .base import AbstractModelSolver\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/algs/inference/base.py\", line 11, in <module>\n",
            "    from chronostrain.model.generative import GenerativeModel\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/model/generative/__init__.py\", line 1, in <module>\n",
            "    from .generative import GenerativeModel\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/model/generative/generative.py\", line 14, in <module>\n",
            "    from chronostrain.util.math.distributions import *\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/util/math/__init__.py\", line 5, in <module>\n",
            "    from . import matrices\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/util/math/matrices/__init__.py\", line 1, in <module>\n",
            "    from .sparse_tensor import SparseMatrix\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/chronostrain/util/math/matrices/sparse_tensor.py\", line 14, in <module>\n",
            "    import torch_sparse\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/torch_sparse/__init__.py\", line 39, in <module>\n",
            "    from .storage import SparseStorage  # noqa\n",
            "  File \"/usr/local/lib/python3.8/dist-packages/torch_sparse/storage.py\", line 5, in <module>\n",
            "    from torch_scatter import segment_csr, scatter_add\n",
            "ModuleNotFoundError: No module named 'torch_scatter'\n"
          ]
        }
      ]
    }
  ]
}